{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# [Open-source software for the analysis and comprehension of high-throughput genomic data](https://www.bioconductor.org/)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "if (!requireNamespace(\"BiocManager\", quietly = TRUE))\n",
    "    install.packages(\"BiocManager\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Bioconductor version 3.7 (BiocManager 1.30.4), R 3.5.0 (2018-04-23)\n",
      "Update old packages: 'bigrquery', 'cowplot', 'DatabaseConnector', 'dbparser',\n",
      "  'deldir', 'devtools', 'digest', 'dplyr', 'DT', 'FactoMineR', 'formatR',\n",
      "  'ggplot2', 'ggstance', 'haven', 'mime', 'PerformanceAnalytics', 'pillar',\n",
      "  'plotrix', 'RcppArmadillo', 'rlang', 'skimr', 'snahelper', 'tibble', 'vctrs',\n",
      "  'zip', 'anytime', 'blob', 'blogdown', 'bold', 'bookdown', 'boot',\n",
      "  'bootstrap', 'callr', 'checkmate', 'cli', 'cluster', 'coda', 'crul',\n",
      "  'data.table', 'DatabaseConnectorJars', 'dbplyr', 'ellipsis', 'fpc', 'future',\n",
      "  'future.apply', 'geometry', 'git2r', 'hms', 'httpuv', 'later', 'markdown',\n",
      "  'mclust', 'mvtnorm', 'numDeriv', 'openssl', 'pbapply', 'pbmcapply',\n",
      "  'phangorn', 'plotly', 'processx', 'purrr', 'quantreg', 'R.utils', 'R6',\n",
      "  'recipes', 'remotes', 'reticulate', 'rmarkdown', 'rngtools', 'rotl',\n",
      "  'rsample', 'rstan', 'rstudioapi', 'segmented', 'servr', 'Seurat', 'shiny',\n",
      "  'shinyWidgets', 'SqlRender', 'StanHeaders', 'taxize', 'tinytex', 'usethis',\n",
      "  'worrms', 'xfun', 'XML'\n"
     ]
    }
   ],
   "source": [
    "BiocManager::install()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Experimental data\n",
    "\n",
    "@ https://www.bioconductor.org/packages/release/BiocViews.html#___ExperimentData\n",
    "\n",
    "For example, we'll load single cell RNA expression data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Bioconductor version 3.7 (BiocManager 1.30.4), R 3.5.0 (2018-04-23)\n",
      "Installing package(s) 'monocle'\n",
      "also installing the dependencies ‘FNN’, ‘sparsesvd’, ‘docopt’, ‘RBGL’, ‘RUnit’, ‘VGAM’, ‘DDRTree’, ‘fastICA’, ‘densityClust’, ‘qlcMatrix’, ‘biocViews’\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The downloaded binary packages are in\n",
      "\t/var/folders/j0/74wnfjb528n4qgcz2_fpzjbh0000gn/T//RtmpK3srzn/downloaded_packages\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Update old packages: 'bigrquery', 'cowplot', 'DatabaseConnector', 'dbparser',\n",
      "  'deldir', 'devtools', 'digest', 'dplyr', 'DT', 'FactoMineR', 'formatR',\n",
      "  'ggplot2', 'ggstance', 'haven', 'mime', 'PerformanceAnalytics', 'pillar',\n",
      "  'plotrix', 'RcppArmadillo', 'rlang', 'skimr', 'snahelper', 'tibble', 'vctrs',\n",
      "  'zip', 'anytime', 'blob', 'blogdown', 'bold', 'bookdown', 'boot',\n",
      "  'bootstrap', 'callr', 'checkmate', 'cli', 'cluster', 'coda', 'crul',\n",
      "  'data.table', 'DatabaseConnectorJars', 'dbplyr', 'ellipsis', 'fpc', 'future',\n",
      "  'future.apply', 'geometry', 'git2r', 'hms', 'httpuv', 'later', 'markdown',\n",
      "  'mclust', 'mvtnorm', 'numDeriv', 'openssl', 'pbapply', 'pbmcapply',\n",
      "  'phangorn', 'plotly', 'processx', 'purrr', 'quantreg', 'R.utils', 'R6',\n",
      "  'recipes', 'remotes', 'reticulate', 'rmarkdown', 'rngtools', 'rotl',\n",
      "  'rsample', 'rstan', 'rstudioapi', 'segmented', 'servr', 'Seurat', 'shiny',\n",
      "  'shinyWidgets', 'SqlRender', 'StanHeaders', 'taxize', 'tinytex', 'usethis',\n",
      "  'worrms', 'xfun', 'XML'\n"
     ]
    }
   ],
   "source": [
    "BiocManager::install(\"monocle\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading required package: Matrix\n",
      "Warning message:\n",
      "“package ‘Matrix’ was built under R version 3.5.2”Loading required package: Biobase\n",
      "Loading required package: BiocGenerics\n",
      "Loading required package: parallel\n",
      "\n",
      "Attaching package: ‘BiocGenerics’\n",
      "\n",
      "The following objects are masked from ‘package:parallel’:\n",
      "\n",
      "    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,\n",
      "    clusterExport, clusterMap, parApply, parCapply, parLapply,\n",
      "    parLapplyLB, parRapply, parSapply, parSapplyLB\n",
      "\n",
      "The following objects are masked from ‘package:Matrix’:\n",
      "\n",
      "    colMeans, colSums, rowMeans, rowSums, which\n",
      "\n",
      "The following objects are masked from ‘package:stats’:\n",
      "\n",
      "    IQR, mad, sd, var, xtabs\n",
      "\n",
      "The following objects are masked from ‘package:base’:\n",
      "\n",
      "    anyDuplicated, append, as.data.frame, basename, cbind, colMeans,\n",
      "    colnames, colSums, dirname, do.call, duplicated, eval, evalq,\n",
      "    Filter, Find, get, grep, grepl, intersect, is.unsorted, lapply,\n",
      "    lengths, Map, mapply, match, mget, order, paste, pmax, pmax.int,\n",
      "    pmin, pmin.int, Position, rank, rbind, Reduce, rowMeans, rownames,\n",
      "    rowSums, sapply, setdiff, sort, table, tapply, union, unique,\n",
      "    unsplit, which, which.max, which.min\n",
      "\n",
      "Welcome to Bioconductor\n",
      "\n",
      "    Vignettes contain introductory material; view with\n",
      "    'browseVignettes()'. To cite Bioconductor, see\n",
      "    'citation(\"Biobase\")', and for packages 'citation(\"pkgname\")'.\n",
      "\n",
      "Loading required package: ggplot2\n",
      "Warning message:\n",
      "“package ‘ggplot2’ was built under R version 3.5.2”Loading required package: VGAM\n",
      "Warning message:\n",
      "“package ‘VGAM’ was built under R version 3.5.2”Loading required package: stats4\n",
      "Loading required package: splines\n",
      "Loading required package: DDRTree\n",
      "Loading required package: irlba\n",
      "Warning message:\n",
      "“package ‘irlba’ was built under R version 3.5.2”"
     ]
    }
   ],
   "source": [
    "library(monocle)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "hsmm=monocle::load_HSMM()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 47192 × 4</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>gene_short_name</th><th scope=col>biotype</th><th scope=col>num_cells_expressed</th><th scope=col>use_for_ordering</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;fct&gt;</th><th scope=col>&lt;int&gt;</th><th scope=col>&lt;lgl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>ENSG00000000003.10</th><td>TSPAN6  </td><td>protein_coding</td><td>231</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000005.5</th><td>TNMD    </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000419.8</th><td>DPM1    </td><td>protein_coding</td><td>275</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000457.8</th><td>SCYL3   </td><td>protein_coding</td><td> 24</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000460.12</th><td>C1orf112</td><td>protein_coding</td><td> 78</td><td> TRUE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000938.8</th><td>FGR     </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000000971.11</th><td>CFH     </td><td>protein_coding</td><td>252</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001036.8</th><td>FUCA2   </td><td>protein_coding</td><td>238</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001084.6</th><td>GCLC    </td><td>protein_coding</td><td> 64</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001167.10</th><td>NFYA    </td><td>protein_coding</td><td>124</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001460.13</th><td>STPG1   </td><td>protein_coding</td><td> 32</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001461.12</th><td>NIPAL3  </td><td>protein_coding</td><td>132</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001497.12</th><td>LAS1L   </td><td>protein_coding</td><td>132</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001561.6</th><td>ENPP4   </td><td>protein_coding</td><td> 45</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001617.7</th><td>SEMA3F  </td><td>protein_coding</td><td>  4</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001626.9</th><td>CFTR    </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001629.5</th><td>ANKIB1  </td><td>protein_coding</td><td>207</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001630.11</th><td>CYP51A1 </td><td>protein_coding</td><td>331</td><td> TRUE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000001631.10</th><td>KRIT1   </td><td>protein_coding</td><td>178</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002016.11</th><td>RAD52   </td><td>protein_coding</td><td>236</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002079.8</th><td>MYH16   </td><td>pseudogene    </td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002330.9</th><td>BAD     </td><td>protein_coding</td><td>130</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002549.7</th><td>LAP3    </td><td>protein_coding</td><td>274</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002586.13</th><td>CD99    </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002587.5</th><td>HS3ST1  </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002726.15</th><td>ABP1    </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002745.8</th><td>WNT16   </td><td>protein_coding</td><td>  4</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002746.9</th><td>HECW1   </td><td>protein_coding</td><td>  0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002822.11</th><td>MAD1L1  </td><td>protein_coding</td><td> 84</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSG00000002834.13</th><td>LASP1   </td><td>protein_coding</td><td>216</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>⋮</th><td>⋮</td><td>⋮</td><td>⋮</td><td>⋮</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000182378.8</th><td>PLCXD1       </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000182484.10</th><td>WASH6P       </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000185203.7</th><td>WASIR1       </td><td>antisense           </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000185291.6</th><td>IL3RA        </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000185960.7</th><td>SHOX         </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000196433.6</th><td>ASMT         </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000197976.6</th><td>AKAP17A      </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000198223.9</th><td>CSF2RA       </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000205755.6</th><td>CRLF2        </td><td>protein_coding      </td><td>1</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000214717.4</th><td>ZBED1        </td><td>protein_coding      </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000223484.2</th><td>TRPC6P       </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000223511.1</th><td>RP13-297E16.4</td><td>lincRNA             </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000223773.2</th><td>CD99P1       </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000225661.2</th><td>RPL14P5      </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000226179.1</th><td>LINC00685    </td><td>antisense           </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000227159.3</th><td>DDX11L16     </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000228410.1</th><td>TCEB1P24     </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000228572.1</th><td>NCRNA00108   </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000229232.1</th><td>KRT18P53     </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000230542.1</th><td>LINC00102    </td><td>lincRNA             </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000234622.1</th><td>RP13-297E16.5</td><td>lincRNA             </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000234958.1</th><td>FABP5P13     </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000236017.3</th><td>ASMTL-AS1    </td><td>antisense           </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000236871.2</th><td>LINC00106    </td><td>lincRNA             </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000237040.1</th><td>DPH3P2       </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000237531.1</th><td>RP11-309M23.1</td><td>lincRNA             </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000237801.1</th><td>AMDP1        </td><td>pseudogene          </td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000248421.1</th><td>RP11-325D5.3 </td><td>processed_transcript</td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000249358.1</th><td>RP13-297E16.3</td><td>processed_transcript</td><td>0</td><td>FALSE</td></tr>\n",
       "\t<tr><th scope=row>ENSGR0000270726.1</th><td>AJ271736.10  </td><td>processed_transcript</td><td>0</td><td>FALSE</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 47192 × 4\n",
       "\\begin{tabular}{r|llll}\n",
       "  & gene\\_short\\_name & biotype & num\\_cells\\_expressed & use\\_for\\_ordering\\\\\n",
       "  & <chr> & <fct> & <int> & <lgl>\\\\\n",
       "\\hline\n",
       "\tENSG00000000003.10 & TSPAN6   & protein\\_coding & 231 & FALSE\\\\\n",
       "\tENSG00000000005.5 & TNMD     & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000000419.8 & DPM1     & protein\\_coding & 275 & FALSE\\\\\n",
       "\tENSG00000000457.8 & SCYL3    & protein\\_coding &  24 & FALSE\\\\\n",
       "\tENSG00000000460.12 & C1orf112 & protein\\_coding &  78 &  TRUE\\\\\n",
       "\tENSG00000000938.8 & FGR      & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000000971.11 & CFH      & protein\\_coding & 252 & FALSE\\\\\n",
       "\tENSG00000001036.8 & FUCA2    & protein\\_coding & 238 & FALSE\\\\\n",
       "\tENSG00000001084.6 & GCLC     & protein\\_coding &  64 & FALSE\\\\\n",
       "\tENSG00000001167.10 & NFYA     & protein\\_coding & 124 & FALSE\\\\\n",
       "\tENSG00000001460.13 & STPG1    & protein\\_coding &  32 & FALSE\\\\\n",
       "\tENSG00000001461.12 & NIPAL3   & protein\\_coding & 132 & FALSE\\\\\n",
       "\tENSG00000001497.12 & LAS1L    & protein\\_coding & 132 & FALSE\\\\\n",
       "\tENSG00000001561.6 & ENPP4    & protein\\_coding &  45 & FALSE\\\\\n",
       "\tENSG00000001617.7 & SEMA3F   & protein\\_coding &   4 & FALSE\\\\\n",
       "\tENSG00000001626.9 & CFTR     & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000001629.5 & ANKIB1   & protein\\_coding & 207 & FALSE\\\\\n",
       "\tENSG00000001630.11 & CYP51A1  & protein\\_coding & 331 &  TRUE\\\\\n",
       "\tENSG00000001631.10 & KRIT1    & protein\\_coding & 178 & FALSE\\\\\n",
       "\tENSG00000002016.11 & RAD52    & protein\\_coding & 236 & FALSE\\\\\n",
       "\tENSG00000002079.8 & MYH16    & pseudogene     &   0 & FALSE\\\\\n",
       "\tENSG00000002330.9 & BAD      & protein\\_coding & 130 & FALSE\\\\\n",
       "\tENSG00000002549.7 & LAP3     & protein\\_coding & 274 & FALSE\\\\\n",
       "\tENSG00000002586.13 & CD99     & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000002587.5 & HS3ST1   & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000002726.15 & ABP1     & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000002745.8 & WNT16    & protein\\_coding &   4 & FALSE\\\\\n",
       "\tENSG00000002746.9 & HECW1    & protein\\_coding &   0 & FALSE\\\\\n",
       "\tENSG00000002822.11 & MAD1L1   & protein\\_coding &  84 & FALSE\\\\\n",
       "\tENSG00000002834.13 & LASP1    & protein\\_coding & 216 & FALSE\\\\\n",
       "\t⋮ & ⋮ & ⋮ & ⋮ & ⋮\\\\\n",
       "\tENSGR0000182378.8 & PLCXD1        & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000182484.10 & WASH6P        & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000185203.7 & WASIR1        & antisense            & 0 & FALSE\\\\\n",
       "\tENSGR0000185291.6 & IL3RA         & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000185960.7 & SHOX          & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000196433.6 & ASMT          & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000197976.6 & AKAP17A       & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000198223.9 & CSF2RA        & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000205755.6 & CRLF2         & protein\\_coding       & 1 & FALSE\\\\\n",
       "\tENSGR0000214717.4 & ZBED1         & protein\\_coding       & 0 & FALSE\\\\\n",
       "\tENSGR0000223484.2 & TRPC6P        & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000223511.1 & RP13-297E16.4 & lincRNA              & 0 & FALSE\\\\\n",
       "\tENSGR0000223773.2 & CD99P1        & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000225661.2 & RPL14P5       & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000226179.1 & LINC00685     & antisense            & 0 & FALSE\\\\\n",
       "\tENSGR0000227159.3 & DDX11L16      & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000228410.1 & TCEB1P24      & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000228572.1 & NCRNA00108    & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000229232.1 & KRT18P53      & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000230542.1 & LINC00102     & lincRNA              & 0 & FALSE\\\\\n",
       "\tENSGR0000234622.1 & RP13-297E16.5 & lincRNA              & 0 & FALSE\\\\\n",
       "\tENSGR0000234958.1 & FABP5P13      & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000236017.3 & ASMTL-AS1     & antisense            & 0 & FALSE\\\\\n",
       "\tENSGR0000236871.2 & LINC00106     & lincRNA              & 0 & FALSE\\\\\n",
       "\tENSGR0000237040.1 & DPH3P2        & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000237531.1 & RP11-309M23.1 & lincRNA              & 0 & FALSE\\\\\n",
       "\tENSGR0000237801.1 & AMDP1         & pseudogene           & 0 & FALSE\\\\\n",
       "\tENSGR0000248421.1 & RP11-325D5.3  & processed\\_transcript & 0 & FALSE\\\\\n",
       "\tENSGR0000249358.1 & RP13-297E16.3 & processed\\_transcript & 0 & FALSE\\\\\n",
       "\tENSGR0000270726.1 & AJ271736.10   & processed\\_transcript & 0 & FALSE\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 47192 × 4\n",
       "\n",
       "| <!--/--> | gene_short_name &lt;chr&gt; | biotype &lt;fct&gt; | num_cells_expressed &lt;int&gt; | use_for_ordering &lt;lgl&gt; |\n",
       "|---|---|---|---|---|\n",
       "| ENSG00000000003.10 | TSPAN6   | protein_coding | 231 | FALSE |\n",
       "| ENSG00000000005.5 | TNMD     | protein_coding |   0 | FALSE |\n",
       "| ENSG00000000419.8 | DPM1     | protein_coding | 275 | FALSE |\n",
       "| ENSG00000000457.8 | SCYL3    | protein_coding |  24 | FALSE |\n",
       "| ENSG00000000460.12 | C1orf112 | protein_coding |  78 |  TRUE |\n",
       "| ENSG00000000938.8 | FGR      | protein_coding |   0 | FALSE |\n",
       "| ENSG00000000971.11 | CFH      | protein_coding | 252 | FALSE |\n",
       "| ENSG00000001036.8 | FUCA2    | protein_coding | 238 | FALSE |\n",
       "| ENSG00000001084.6 | GCLC     | protein_coding |  64 | FALSE |\n",
       "| ENSG00000001167.10 | NFYA     | protein_coding | 124 | FALSE |\n",
       "| ENSG00000001460.13 | STPG1    | protein_coding |  32 | FALSE |\n",
       "| ENSG00000001461.12 | NIPAL3   | protein_coding | 132 | FALSE |\n",
       "| ENSG00000001497.12 | LAS1L    | protein_coding | 132 | FALSE |\n",
       "| ENSG00000001561.6 | ENPP4    | protein_coding |  45 | FALSE |\n",
       "| ENSG00000001617.7 | SEMA3F   | protein_coding |   4 | FALSE |\n",
       "| ENSG00000001626.9 | CFTR     | protein_coding |   0 | FALSE |\n",
       "| ENSG00000001629.5 | ANKIB1   | protein_coding | 207 | FALSE |\n",
       "| ENSG00000001630.11 | CYP51A1  | protein_coding | 331 |  TRUE |\n",
       "| ENSG00000001631.10 | KRIT1    | protein_coding | 178 | FALSE |\n",
       "| ENSG00000002016.11 | RAD52    | protein_coding | 236 | FALSE |\n",
       "| ENSG00000002079.8 | MYH16    | pseudogene     |   0 | FALSE |\n",
       "| ENSG00000002330.9 | BAD      | protein_coding | 130 | FALSE |\n",
       "| ENSG00000002549.7 | LAP3     | protein_coding | 274 | FALSE |\n",
       "| ENSG00000002586.13 | CD99     | protein_coding |   0 | FALSE |\n",
       "| ENSG00000002587.5 | HS3ST1   | protein_coding |   0 | FALSE |\n",
       "| ENSG00000002726.15 | ABP1     | protein_coding |   0 | FALSE |\n",
       "| ENSG00000002745.8 | WNT16    | protein_coding |   4 | FALSE |\n",
       "| ENSG00000002746.9 | HECW1    | protein_coding |   0 | FALSE |\n",
       "| ENSG00000002822.11 | MAD1L1   | protein_coding |  84 | FALSE |\n",
       "| ENSG00000002834.13 | LASP1    | protein_coding | 216 | FALSE |\n",
       "| ⋮ | ⋮ | ⋮ | ⋮ | ⋮ |\n",
       "| ENSGR0000182378.8 | PLCXD1        | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000182484.10 | WASH6P        | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000185203.7 | WASIR1        | antisense            | 0 | FALSE |\n",
       "| ENSGR0000185291.6 | IL3RA         | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000185960.7 | SHOX          | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000196433.6 | ASMT          | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000197976.6 | AKAP17A       | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000198223.9 | CSF2RA        | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000205755.6 | CRLF2         | protein_coding       | 1 | FALSE |\n",
       "| ENSGR0000214717.4 | ZBED1         | protein_coding       | 0 | FALSE |\n",
       "| ENSGR0000223484.2 | TRPC6P        | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000223511.1 | RP13-297E16.4 | lincRNA              | 0 | FALSE |\n",
       "| ENSGR0000223773.2 | CD99P1        | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000225661.2 | RPL14P5       | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000226179.1 | LINC00685     | antisense            | 0 | FALSE |\n",
       "| ENSGR0000227159.3 | DDX11L16      | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000228410.1 | TCEB1P24      | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000228572.1 | NCRNA00108    | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000229232.1 | KRT18P53      | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000230542.1 | LINC00102     | lincRNA              | 0 | FALSE |\n",
       "| ENSGR0000234622.1 | RP13-297E16.5 | lincRNA              | 0 | FALSE |\n",
       "| ENSGR0000234958.1 | FABP5P13      | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000236017.3 | ASMTL-AS1     | antisense            | 0 | FALSE |\n",
       "| ENSGR0000236871.2 | LINC00106     | lincRNA              | 0 | FALSE |\n",
       "| ENSGR0000237040.1 | DPH3P2        | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000237531.1 | RP11-309M23.1 | lincRNA              | 0 | FALSE |\n",
       "| ENSGR0000237801.1 | AMDP1         | pseudogene           | 0 | FALSE |\n",
       "| ENSGR0000248421.1 | RP11-325D5.3  | processed_transcript | 0 | FALSE |\n",
       "| ENSGR0000249358.1 | RP13-297E16.3 | processed_transcript | 0 | FALSE |\n",
       "| ENSGR0000270726.1 | AJ271736.10   | processed_transcript | 0 | FALSE |\n",
       "\n"
      ],
      "text/plain": [
       "                   gene_short_name biotype              num_cells_expressed\n",
       "ENSG00000000003.10 TSPAN6          protein_coding       231                \n",
       "ENSG00000000005.5  TNMD            protein_coding         0                \n",
       "ENSG00000000419.8  DPM1            protein_coding       275                \n",
       "ENSG00000000457.8  SCYL3           protein_coding        24                \n",
       "ENSG00000000460.12 C1orf112        protein_coding        78                \n",
       "ENSG00000000938.8  FGR             protein_coding         0                \n",
       "ENSG00000000971.11 CFH             protein_coding       252                \n",
       "ENSG00000001036.8  FUCA2           protein_coding       238                \n",
       "ENSG00000001084.6  GCLC            protein_coding        64                \n",
       "ENSG00000001167.10 NFYA            protein_coding       124                \n",
       "ENSG00000001460.13 STPG1           protein_coding        32                \n",
       "ENSG00000001461.12 NIPAL3          protein_coding       132                \n",
       "ENSG00000001497.12 LAS1L           protein_coding       132                \n",
       "ENSG00000001561.6  ENPP4           protein_coding        45                \n",
       "ENSG00000001617.7  SEMA3F          protein_coding         4                \n",
       "ENSG00000001626.9  CFTR            protein_coding         0                \n",
       "ENSG00000001629.5  ANKIB1          protein_coding       207                \n",
       "ENSG00000001630.11 CYP51A1         protein_coding       331                \n",
       "ENSG00000001631.10 KRIT1           protein_coding       178                \n",
       "ENSG00000002016.11 RAD52           protein_coding       236                \n",
       "ENSG00000002079.8  MYH16           pseudogene             0                \n",
       "ENSG00000002330.9  BAD             protein_coding       130                \n",
       "ENSG00000002549.7  LAP3            protein_coding       274                \n",
       "ENSG00000002586.13 CD99            protein_coding         0                \n",
       "ENSG00000002587.5  HS3ST1          protein_coding         0                \n",
       "ENSG00000002726.15 ABP1            protein_coding         0                \n",
       "ENSG00000002745.8  WNT16           protein_coding         4                \n",
       "ENSG00000002746.9  HECW1           protein_coding         0                \n",
       "ENSG00000002822.11 MAD1L1          protein_coding        84                \n",
       "ENSG00000002834.13 LASP1           protein_coding       216                \n",
       "⋮                  ⋮               ⋮                    ⋮                  \n",
       "ENSGR0000182378.8  PLCXD1          protein_coding       0                  \n",
       "ENSGR0000182484.10 WASH6P          pseudogene           0                  \n",
       "ENSGR0000185203.7  WASIR1          antisense            0                  \n",
       "ENSGR0000185291.6  IL3RA           protein_coding       0                  \n",
       "ENSGR0000185960.7  SHOX            protein_coding       0                  \n",
       "ENSGR0000196433.6  ASMT            protein_coding       0                  \n",
       "ENSGR0000197976.6  AKAP17A         protein_coding       0                  \n",
       "ENSGR0000198223.9  CSF2RA          protein_coding       0                  \n",
       "ENSGR0000205755.6  CRLF2           protein_coding       1                  \n",
       "ENSGR0000214717.4  ZBED1           protein_coding       0                  \n",
       "ENSGR0000223484.2  TRPC6P          pseudogene           0                  \n",
       "ENSGR0000223511.1  RP13-297E16.4   lincRNA              0                  \n",
       "ENSGR0000223773.2  CD99P1          pseudogene           0                  \n",
       "ENSGR0000225661.2  RPL14P5         pseudogene           0                  \n",
       "ENSGR0000226179.1  LINC00685       antisense            0                  \n",
       "ENSGR0000227159.3  DDX11L16        pseudogene           0                  \n",
       "ENSGR0000228410.1  TCEB1P24        pseudogene           0                  \n",
       "ENSGR0000228572.1  NCRNA00108      pseudogene           0                  \n",
       "ENSGR0000229232.1  KRT18P53        pseudogene           0                  \n",
       "ENSGR0000230542.1  LINC00102       lincRNA              0                  \n",
       "ENSGR0000234622.1  RP13-297E16.5   lincRNA              0                  \n",
       "ENSGR0000234958.1  FABP5P13        pseudogene           0                  \n",
       "ENSGR0000236017.3  ASMTL-AS1       antisense            0                  \n",
       "ENSGR0000236871.2  LINC00106       lincRNA              0                  \n",
       "ENSGR0000237040.1  DPH3P2          pseudogene           0                  \n",
       "ENSGR0000237531.1  RP11-309M23.1   lincRNA              0                  \n",
       "ENSGR0000237801.1  AMDP1           pseudogene           0                  \n",
       "ENSGR0000248421.1  RP11-325D5.3    processed_transcript 0                  \n",
       "ENSGR0000249358.1  RP13-297E16.3   processed_transcript 0                  \n",
       "ENSGR0000270726.1  AJ271736.10     processed_transcript 0                  \n",
       "                   use_for_ordering\n",
       "ENSG00000000003.10 FALSE           \n",
       "ENSG00000000005.5  FALSE           \n",
       "ENSG00000000419.8  FALSE           \n",
       "ENSG00000000457.8  FALSE           \n",
       "ENSG00000000460.12  TRUE           \n",
       "ENSG00000000938.8  FALSE           \n",
       "ENSG00000000971.11 FALSE           \n",
       "ENSG00000001036.8  FALSE           \n",
       "ENSG00000001084.6  FALSE           \n",
       "ENSG00000001167.10 FALSE           \n",
       "ENSG00000001460.13 FALSE           \n",
       "ENSG00000001461.12 FALSE           \n",
       "ENSG00000001497.12 FALSE           \n",
       "ENSG00000001561.6  FALSE           \n",
       "ENSG00000001617.7  FALSE           \n",
       "ENSG00000001626.9  FALSE           \n",
       "ENSG00000001629.5  FALSE           \n",
       "ENSG00000001630.11  TRUE           \n",
       "ENSG00000001631.10 FALSE           \n",
       "ENSG00000002016.11 FALSE           \n",
       "ENSG00000002079.8  FALSE           \n",
       "ENSG00000002330.9  FALSE           \n",
       "ENSG00000002549.7  FALSE           \n",
       "ENSG00000002586.13 FALSE           \n",
       "ENSG00000002587.5  FALSE           \n",
       "ENSG00000002726.15 FALSE           \n",
       "ENSG00000002745.8  FALSE           \n",
       "ENSG00000002746.9  FALSE           \n",
       "ENSG00000002822.11 FALSE           \n",
       "ENSG00000002834.13 FALSE           \n",
       "⋮                  ⋮               \n",
       "ENSGR0000182378.8  FALSE           \n",
       "ENSGR0000182484.10 FALSE           \n",
       "ENSGR0000185203.7  FALSE           \n",
       "ENSGR0000185291.6  FALSE           \n",
       "ENSGR0000185960.7  FALSE           \n",
       "ENSGR0000196433.6  FALSE           \n",
       "ENSGR0000197976.6  FALSE           \n",
       "ENSGR0000198223.9  FALSE           \n",
       "ENSGR0000205755.6  FALSE           \n",
       "ENSGR0000214717.4  FALSE           \n",
       "ENSGR0000223484.2  FALSE           \n",
       "ENSGR0000223511.1  FALSE           \n",
       "ENSGR0000223773.2  FALSE           \n",
       "ENSGR0000225661.2  FALSE           \n",
       "ENSGR0000226179.1  FALSE           \n",
       "ENSGR0000227159.3  FALSE           \n",
       "ENSGR0000228410.1  FALSE           \n",
       "ENSGR0000228572.1  FALSE           \n",
       "ENSGR0000229232.1  FALSE           \n",
       "ENSGR0000230542.1  FALSE           \n",
       "ENSGR0000234622.1  FALSE           \n",
       "ENSGR0000234958.1  FALSE           \n",
       "ENSGR0000236017.3  FALSE           \n",
       "ENSGR0000236871.2  FALSE           \n",
       "ENSGR0000237040.1  FALSE           \n",
       "ENSGR0000237531.1  FALSE           \n",
       "ENSGR0000237801.1  FALSE           \n",
       "ENSGR0000248421.1  FALSE           \n",
       "ENSGR0000249358.1  FALSE           \n",
       "ENSGR0000270726.1  FALSE           "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fData(hsmm)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.5.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
