{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Exploring the TCGA liver cancer dataset for hacknights workshops/projects"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## Setup libraries"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### install Bioconductor if haven't already"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "if (!requireNamespace(\"BiocManager\", quietly = TRUE))\n",
    "    install.packages(\"BiocManager\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Bioconductor version 3.9 (BiocManager 1.30.4), R 3.6.0 (2019-04-26)\n"
     ]
    }
   ],
   "source": [
    "BiocManager::install(update=TRUE,ask=FALSE)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### install/load RTCGA libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "BiocManager::install(\"RTCGA.clinical\")\n",
    "BiocManager::install(\"RTCGA.rnaseq\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading required package: RTCGA\n",
      "Welcome to the RTCGA (version: 1.14.0).\n"
     ]
    }
   ],
   "source": [
    "library(RTCGA.clinical)\n",
    "library(RTCGA.rnaseq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### load other libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "── \u001b[1mAttaching packages\u001b[22m ─────────────────────────────────────── tidyverse 1.2.1 ──\n",
      "\u001b[32m✔\u001b[39m \u001b[34mggplot2\u001b[39m 3.2.1     \u001b[32m✔\u001b[39m \u001b[34mpurrr  \u001b[39m 0.3.2\n",
      "\u001b[32m✔\u001b[39m \u001b[34mtibble \u001b[39m 2.1.3     \u001b[32m✔\u001b[39m \u001b[34mdplyr  \u001b[39m 0.8.3\n",
      "\u001b[32m✔\u001b[39m \u001b[34mtidyr  \u001b[39m 0.8.3     \u001b[32m✔\u001b[39m \u001b[34mstringr\u001b[39m 1.4.0\n",
      "\u001b[32m✔\u001b[39m \u001b[34mreadr  \u001b[39m 1.3.1     \u001b[32m✔\u001b[39m \u001b[34mforcats\u001b[39m 0.4.0\n",
      "── \u001b[1mConflicts\u001b[22m ────────────────────────────────────────── tidyverse_conflicts() ──\n",
      "\u001b[31m✖\u001b[39m \u001b[34mdplyr\u001b[39m::\u001b[32mfilter()\u001b[39m masks \u001b[34mstats\u001b[39m::filter()\n",
      "\u001b[31m✖\u001b[39m \u001b[34mdplyr\u001b[39m::\u001b[32mlag()\u001b[39m    masks \u001b[34mstats\u001b[39m::lag()\n"
     ]
    }
   ],
   "source": [
    "library(tidyverse)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Liver clinical and rnaseq datasets\n",
    "\n",
    "We are focusing on cancers where the primary site includes the liver. \n",
    "\n",
    "http://bioconductor.org/packages/release/data/experiment/manuals/RTCGA.clinical/man/RTCGA.clinical.pdf\n",
    "\n",
    "http://bioconductor.org/packages/release/data/experiment/manuals/RTCGA.clinical/man/RTCGA.clinical.pdf"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "### Clinical"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "#### bring datasets into memory"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUAD\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LUAD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "luad_clinical <- RTCGA.clinical::LUAD.clinical"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUSC\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LUSC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "lusc_clinical <- RTCGA.clinical::LUSC.clinical"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### MESO\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-MESO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "meso_clinical <- RTCGA.clinical::MESO.clinical"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LIHC\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LIHC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "lihc_clinical <- RTCGA.clinical::LIHC.clinical"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "#####  clinical dataset list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "clinical_dataset_list <- list(\"luad\"=luad_clinical,\"lusc\"=lusc_clinical,\"meso\"=meso_clinical,\"lihc\"=lihc_clinical)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "#### View"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUAD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>522</li>\n",
       "\t<li>3046</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 522\n",
       "\\item 3046\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 522\n",
       "2. 3046\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]  522 3046"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 3046</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>admin.bcr</th><th scope=col>admin.day_of_dcc_upload</th><th scope=col>admin.disease_code</th><th scope=col>admin.file_uuid</th><th scope=col>admin.month_of_dcc_upload</th><th scope=col>admin.patient_withdrawal.withdrawn</th><th scope=col>admin.project_code</th><th scope=col>admin.year_of_dcc_upload</th><th scope=col>patient.additional_studies</th><th scope=col>patient.additional_studies.additional_study.disease_code</th><th scope=col>⋯</th><th scope=col>patient.samples.sample.preservation_method</th><th scope=col>patient.samples.sample.sample_type</th><th scope=col>patient.samples.sample.sample_type_id</th><th scope=col>patient.samples.sample.shortest_dimension</th><th scope=col>patient.samples.sample.time_between_clamping_and_freezing</th><th scope=col>patient.samples.sample.time_between_excision_and_freezing</th><th scope=col>patient.samples.sample.tissue_type</th><th scope=col>patient.samples.sample.tumor_descriptor</th><th scope=col>patient.samples.sample.tumor_pathology</th><th scope=col>patient.samples.sample.vial_number</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>⋯</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>52.70.0</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>071e3c36-bd5c-49b2-b3d3-0eb4e781810f</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>52.70.0.1</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>74423066-665a-43f2-a801-fb4f83ee15e9</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.5</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>52.70.0.2</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>b06fd27e-a052-459f-be7b-c6e336a6629d</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.4</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>52.70.0.3</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>4bdbf19f-48c2-4565-a5b6-9ada2ef93e74</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>58.73.0</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>ba34c6c3-4068-4665-8239-a5694644eab8</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>119.72.0</th><td>nationwide children's hospital</td><td>1</td><td>luad</td><td>092cfc6d-b91f-4787-8d7b-49a936d848b4</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.4</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 3046\n",
       "\\begin{tabular}{r|llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll}\n",
       "  & admin.bcr & admin.day\\_of\\_dcc\\_upload & admin.disease\\_code & admin.file\\_uuid & admin.month\\_of\\_dcc\\_upload & admin.patient\\_withdrawal.withdrawn & admin.project\\_code & admin.year\\_of\\_dcc\\_upload & patient.additional\\_studies & patient.additional\\_studies.additional\\_study.disease\\_code & patient.additional\\_studies.additional\\_study.project\\_code & patient.age\\_at\\_initial\\_pathologic\\_diagnosis & patient.anatomic\\_neoplasm\\_subdivision & patient.anatomic\\_neoplasm\\_subdivision\\_other & patient.bcr\\_patient\\_barcode & patient.bcr\\_patient\\_uuid & patient.clinical\\_cqcf.anatomic\\_neoplasm\\_subdivision & patient.clinical\\_cqcf.consent\\_or\\_death\\_status & patient.clinical\\_cqcf.country & patient.clinical\\_cqcf.days\\_to\\_consent & patient.clinical\\_cqcf.days\\_to\\_death & patient.clinical\\_cqcf.diagnosis\\_subtype & patient.clinical\\_cqcf.histological\\_type & patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.clinical\\_cqcf.history\\_of\\_prior\\_malignancy & patient.clinical\\_cqcf.normal\\_tissue\\_anatomic\\_site & patient.clinical\\_cqcf.normal\\_tissue\\_proximity & patient.clinical\\_cqcf.other\\_anatomic\\_site & patient.clinical\\_cqcf.other\\_anatomic\\_site\\_normal\\_tissue & patient.clinical\\_cqcf.prior\\_dx & patient.clinical\\_cqcf.tumor\\_type & patient.day\\_of\\_form\\_completion & patient.days\\_to\\_birth & patient.days\\_to\\_death & patient.days\\_to\\_initial\\_pathologic\\_diagnosis & patient.days\\_to\\_last\\_followup & patient.days\\_to\\_last\\_known\\_alive & patient.diagnosis & patient.dlco\\_predictive\\_percent & patient.drugs & patient.drugs.drug-10.bcr\\_drug\\_barcode & patient.drugs.drug-10.bcr\\_drug\\_uuid & patient.drugs.drug-10.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-10.day\\_of\\_form\\_completion & patient.drugs.drug-10.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-10.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-10.drug\\_name & patient.drugs.drug-10.measure\\_of\\_response & patient.drugs.drug-10.month\\_of\\_form\\_completion & patient.drugs.drug-10.number\\_cycles & patient.drugs.drug-10.prescribed\\_dose & patient.drugs.drug-10.prescribed\\_dose\\_units & patient.drugs.drug-10.regimen\\_indication & patient.drugs.drug-10.regimen\\_indication\\_notes & patient.drugs.drug-10.regimen\\_number & patient.drugs.drug-10.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-10.therapy\\_ongoing & patient.drugs.drug-10.therapy\\_types.therapy\\_type & patient.drugs.drug-10.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-10.total\\_dose & patient.drugs.drug-10.total\\_dose\\_units & patient.drugs.drug-10.tx\\_on\\_clinical\\_trial & patient.drugs.drug-10.year\\_of\\_form\\_completion & patient.drugs.drug-2.bcr\\_drug\\_barcode & patient.drugs.drug-2.bcr\\_drug\\_uuid & patient.drugs.drug-2.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-2.day\\_of\\_form\\_completion & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-2.drug\\_name & patient.drugs.drug-2.measure\\_of\\_response & patient.drugs.drug-2.month\\_of\\_form\\_completion & patient.drugs.drug-2.number\\_cycles & patient.drugs.drug-2.prescribed\\_dose & patient.drugs.drug-2.prescribed\\_dose\\_units & patient.drugs.drug-2.regimen\\_indication & patient.drugs.drug-2.regimen\\_indication\\_notes & patient.drugs.drug-2.regimen\\_number & patient.drugs.drug-2.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-2.therapy\\_ongoing & patient.drugs.drug-2.therapy\\_types.therapy\\_type & patient.drugs.drug-2.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-2.total\\_dose & patient.drugs.drug-2.total\\_dose\\_units & patient.drugs.drug-2.tx\\_on\\_clinical\\_trial & patient.drugs.drug-2.year\\_of\\_form\\_completion & patient.drugs.drug-3.bcr\\_drug\\_barcode & patient.drugs.drug-3.bcr\\_drug\\_uuid & patient.drugs.drug-3.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-3.day\\_of\\_form\\_completion & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-3.drug\\_name & patient.drugs.drug-3.measure\\_of\\_response & patient.drugs.drug-3.month\\_of\\_form\\_completion & patient.drugs.drug-3.number\\_cycles & patient.drugs.drug-3.prescribed\\_dose & patient.drugs.drug-3.prescribed\\_dose\\_units & patient.drugs.drug-3.regimen\\_indication & patient.drugs.drug-3.regimen\\_indication\\_notes & patient.drugs.drug-3.regimen\\_number & patient.drugs.drug-3.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-3.therapy\\_ongoing & patient.drugs.drug-3.therapy\\_types.therapy\\_type & patient.drugs.drug-3.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-3.total\\_dose & patient.drugs.drug-3.total\\_dose\\_units & patient.drugs.drug-3.tx\\_on\\_clinical\\_trial & patient.drugs.drug-3.year\\_of\\_form\\_completion & patient.drugs.drug-4.bcr\\_drug\\_barcode & patient.drugs.drug-4.bcr\\_drug\\_uuid & patient.drugs.drug-4.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-4.day\\_of\\_form\\_completion & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-4.drug\\_name & patient.drugs.drug-4.measure\\_of\\_response & patient.drugs.drug-4.month\\_of\\_form\\_completion & patient.drugs.drug-4.number\\_cycles & patient.drugs.drug-4.prescribed\\_dose & patient.drugs.drug-4.prescribed\\_dose\\_units & patient.drugs.drug-4.regimen\\_indication & patient.drugs.drug-4.regimen\\_indication\\_notes & patient.drugs.drug-4.regimen\\_number & patient.drugs.drug-4.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-4.therapy\\_ongoing & patient.drugs.drug-4.therapy\\_types.therapy\\_type & patient.drugs.drug-4.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-4.total\\_dose & patient.drugs.drug-4.total\\_dose\\_units & patient.drugs.drug-4.tx\\_on\\_clinical\\_trial & patient.drugs.drug-4.year\\_of\\_form\\_completion & patient.drugs.drug-5.bcr\\_drug\\_barcode & patient.drugs.drug-5.bcr\\_drug\\_uuid & patient.drugs.drug-5.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-5.day\\_of\\_form\\_completion & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-5.drug\\_name & patient.drugs.drug-5.measure\\_of\\_response & patient.drugs.drug-5.month\\_of\\_form\\_completion & patient.drugs.drug-5.number\\_cycles & patient.drugs.drug-5.prescribed\\_dose & patient.drugs.drug-5.prescribed\\_dose\\_units & patient.drugs.drug-5.regimen\\_indication & patient.drugs.drug-5.regimen\\_indication\\_notes & patient.drugs.drug-5.regimen\\_number & patient.drugs.drug-5.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-5.therapy\\_ongoing & patient.drugs.drug-5.therapy\\_types.therapy\\_type & patient.drugs.drug-5.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-5.total\\_dose & patient.drugs.drug-5.total\\_dose\\_units & patient.drugs.drug-5.tx\\_on\\_clinical\\_trial & patient.drugs.drug-5.year\\_of\\_form\\_completion & patient.drugs.drug-6.bcr\\_drug\\_barcode & patient.drugs.drug-6.bcr\\_drug\\_uuid & patient.drugs.drug-6.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-6.day\\_of\\_form\\_completion & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-6.drug\\_name & patient.drugs.drug-6.measure\\_of\\_response & patient.drugs.drug-6.month\\_of\\_form\\_completion & patient.drugs.drug-6.number\\_cycles & patient.drugs.drug-6.prescribed\\_dose & patient.drugs.drug-6.prescribed\\_dose\\_units & patient.drugs.drug-6.regimen\\_indication & patient.drugs.drug-6.regimen\\_indication\\_notes & patient.drugs.drug-6.regimen\\_number & patient.drugs.drug-6.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-6.therapy\\_ongoing & patient.drugs.drug-6.therapy\\_types.therapy\\_type & patient.drugs.drug-6.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-6.total\\_dose & patient.drugs.drug-6.total\\_dose\\_units & patient.drugs.drug-6.tx\\_on\\_clinical\\_trial & patient.drugs.drug-6.year\\_of\\_form\\_completion & patient.drugs.drug-7.bcr\\_drug\\_barcode & patient.drugs.drug-7.bcr\\_drug\\_uuid & patient.drugs.drug-7.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-7.day\\_of\\_form\\_completion & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-7.drug\\_name & patient.drugs.drug-7.measure\\_of\\_response & patient.drugs.drug-7.month\\_of\\_form\\_completion & patient.drugs.drug-7.number\\_cycles & patient.drugs.drug-7.prescribed\\_dose & patient.drugs.drug-7.prescribed\\_dose\\_units & patient.drugs.drug-7.regimen\\_indication & patient.drugs.drug-7.regimen\\_indication\\_notes & patient.drugs.drug-7.regimen\\_number & patient.drugs.drug-7.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-7.therapy\\_ongoing & patient.drugs.drug-7.therapy\\_types.therapy\\_type & patient.drugs.drug-7.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-7.total\\_dose & patient.drugs.drug-7.total\\_dose\\_units & patient.drugs.drug-7.tx\\_on\\_clinical\\_trial & patient.drugs.drug-7.year\\_of\\_form\\_completion & patient.drugs.drug-8.bcr\\_drug\\_barcode & patient.drugs.drug-8.bcr\\_drug\\_uuid & patient.drugs.drug-8.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-8.day\\_of\\_form\\_completion & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-8.drug\\_name & patient.drugs.drug-8.measure\\_of\\_response & patient.drugs.drug-8.month\\_of\\_form\\_completion & patient.drugs.drug-8.number\\_cycles & patient.drugs.drug-8.prescribed\\_dose & patient.drugs.drug-8.prescribed\\_dose\\_units & patient.drugs.drug-8.regimen\\_indication & patient.drugs.drug-8.regimen\\_indication\\_notes & patient.drugs.drug-8.regimen\\_number & patient.drugs.drug-8.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-8.therapy\\_ongoing & patient.drugs.drug-8.therapy\\_types.therapy\\_type & patient.drugs.drug-8.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-8.total\\_dose & patient.drugs.drug-8.total\\_dose\\_units & patient.drugs.drug-8.tx\\_on\\_clinical\\_trial & patient.drugs.drug-8.year\\_of\\_form\\_completion & patient.drugs.drug-9.bcr\\_drug\\_barcode & patient.drugs.drug-9.bcr\\_drug\\_uuid & patient.drugs.drug-9.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-9.day\\_of\\_form\\_completion & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-9.drug\\_name & patient.drugs.drug-9.measure\\_of\\_response & patient.drugs.drug-9.month\\_of\\_form\\_completion & patient.drugs.drug-9.number\\_cycles & patient.drugs.drug-9.prescribed\\_dose & patient.drugs.drug-9.prescribed\\_dose\\_units & patient.drugs.drug-9.regimen\\_indication & patient.drugs.drug-9.regimen\\_indication\\_notes & patient.drugs.drug-9.regimen\\_number & patient.drugs.drug-9.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-9.therapy\\_ongoing & patient.drugs.drug-9.therapy\\_types.therapy\\_type & patient.drugs.drug-9.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-9.total\\_dose & patient.drugs.drug-9.total\\_dose\\_units & patient.drugs.drug-9.tx\\_on\\_clinical\\_trial & patient.drugs.drug-9.year\\_of\\_form\\_completion & patient.drugs.drug.bcr\\_drug\\_barcode & patient.drugs.drug.bcr\\_drug\\_uuid & patient.drugs.drug.clinical\\_trail\\_drug\\_classification & patient.drugs.drug.day\\_of\\_form\\_completion & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug.drug\\_name & patient.drugs.drug.measure\\_of\\_response & patient.drugs.drug.month\\_of\\_form\\_completion & patient.drugs.drug.number\\_cycles & patient.drugs.drug.prescribed\\_dose & patient.drugs.drug.prescribed\\_dose\\_units & patient.drugs.drug.regimen\\_indication & patient.drugs.drug.regimen\\_indication\\_notes & patient.drugs.drug.regimen\\_number & patient.drugs.drug.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug.therapy\\_ongoing & patient.drugs.drug.therapy\\_types.therapy\\_type & patient.drugs.drug.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug.total\\_dose & patient.drugs.drug.total\\_dose\\_units & patient.drugs.drug.tx\\_on\\_clinical\\_trial & patient.drugs.drug.year\\_of\\_form\\_completion & patient.eastern\\_cancer\\_oncology\\_group & patient.egfr\\_mutation\\_identified & patient.egfr\\_mutation\\_performed & patient.egfr\\_mutation\\_result & patient.eml4\\_alk\\_translocation\\_identified & patient.eml4\\_alk\\_translocation\\_method & patient.eml4\\_alk\\_translocation\\_performed & patient.eml4\\_alk\\_translocation\\_result & patient.ethnicity & patient.follow\\_ups & patient.follow\\_ups.follow\\_up-2.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-2.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-2.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-2.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-2.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-2.days\\_to\\_death & patient.follow\\_ups.follow\\_up-2.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-2.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-2.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-2.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-2.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-2.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-2.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-2.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-2.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-2.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-2.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up-2.radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-2.vital\\_status & patient.follow\\_ups.follow\\_up-2.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-3.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-3.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-3.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-3.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-3.days\\_to\\_death & patient.follow\\_ups.follow\\_up-3.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-3.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-3.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-3.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-3.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-3.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-3.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-3.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-3.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-3.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up-3.progression\\_determined\\_by\\_list.progression\\_determined\\_by-3 & patient.follow\\_ups.follow\\_up-3.radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-3.vital\\_status & patient.follow\\_ups.follow\\_up-3.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-4.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-4.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-4.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-4.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-4.days\\_to\\_death & patient.follow\\_ups.follow\\_up-4.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-4.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-4.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-4.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-4.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-4.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-4.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-4.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-4.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-4.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-4.radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-4.vital\\_status & patient.follow\\_ups.follow\\_up-4.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up.days\\_to\\_death & patient.follow\\_ups.follow\\_up.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.follow\\_ups.follow\\_up.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up.radiation\\_therapy & patient.follow\\_ups.follow\\_up.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up.vital\\_status & patient.follow\\_ups.follow\\_up.year\\_of\\_form\\_completion & patient.gender & patient.histological\\_type & patient.history\\_of\\_neoadjuvant\\_treatment & patient.icd\\_10 & patient.icd\\_o\\_3\\_histology & patient.icd\\_o\\_3\\_site & patient.informed\\_consent\\_verified & patient.karnofsky\\_performance\\_score & patient.kras\\_gene\\_analysis\\_performed & patient.kras\\_mutation\\_found & patient.kras\\_mutation\\_result & patient.location\\_in\\_lung\\_parenchyma & patient.month\\_of\\_form\\_completion & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event.locoregional\\_procedure.additional\\_surgery\\_locoregional\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.locoregional\\_procedure.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.metastatic\\_procedure.additional\\_surgery\\_metastatic\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.metastatic\\_procedure.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.new\\_tumor\\_events.new\\_tumor\\_event.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.new\\_tumor\\_events.new\\_tumor\\_event.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.number\\_pack\\_years\\_smoked & patient.patient\\_id & patient.performance\\_status\\_scale\\_timing & patient.person\\_neoplasm\\_cancer\\_status & patient.post\\_bronchodilator\\_fev1\\_fvc\\_percent & patient.post\\_bronchodilator\\_fev1\\_percent & patient.pre\\_bronchodilator\\_fev1\\_fvc\\_percent & patient.pre\\_bronchodilator\\_fev1\\_percent & patient.primary\\_therapy\\_outcome\\_success & patient.prior\\_dx & patient.pulmonary\\_function\\_test\\_performed & patient.race & patient.radiation\\_therapy & patient.radiations & patient.radiations.radiation-2.anatomic\\_treatment\\_site & patient.radiations.radiation-2.bcr\\_radiation\\_barcode & patient.radiations.radiation-2.bcr\\_radiation\\_uuid & patient.radiations.radiation-2.course\\_number & patient.radiations.radiation-2.day\\_of\\_form\\_completion & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-2.measure\\_of\\_response & patient.radiations.radiation-2.month\\_of\\_form\\_completion & patient.radiations.radiation-2.numfractions & patient.radiations.radiation-2.radiation\\_dosage & patient.radiations.radiation-2.radiation\\_treatment\\_ongoing & patient.radiations.radiation-2.radiation\\_type & patient.radiations.radiation-2.radiation\\_type\\_notes & patient.radiations.radiation-2.regimen\\_indication & patient.radiations.radiation-2.regimen\\_indication\\_notes & patient.radiations.radiation-2.units & patient.radiations.radiation-2.year\\_of\\_form\\_completion & patient.radiations.radiation-3.anatomic\\_treatment\\_site & patient.radiations.radiation-3.bcr\\_radiation\\_barcode & patient.radiations.radiation-3.bcr\\_radiation\\_uuid & patient.radiations.radiation-3.course\\_number & patient.radiations.radiation-3.day\\_of\\_form\\_completion & patient.radiations.radiation-3.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-3.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-3.measure\\_of\\_response & patient.radiations.radiation-3.month\\_of\\_form\\_completion & patient.radiations.radiation-3.numfractions & patient.radiations.radiation-3.radiation\\_dosage & patient.radiations.radiation-3.radiation\\_treatment\\_ongoing & patient.radiations.radiation-3.radiation\\_type & patient.radiations.radiation-3.radiation\\_type\\_notes & patient.radiations.radiation-3.regimen\\_indication & patient.radiations.radiation-3.regimen\\_indication\\_notes & patient.radiations.radiation-3.units & patient.radiations.radiation-3.year\\_of\\_form\\_completion & patient.radiations.radiation-4.anatomic\\_treatment\\_site & patient.radiations.radiation-4.bcr\\_radiation\\_barcode & patient.radiations.radiation-4.bcr\\_radiation\\_uuid & patient.radiations.radiation-4.course\\_number & patient.radiations.radiation-4.day\\_of\\_form\\_completion & patient.radiations.radiation-4.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-4.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-4.measure\\_of\\_response & patient.radiations.radiation-4.month\\_of\\_form\\_completion & patient.radiations.radiation-4.numfractions & patient.radiations.radiation-4.radiation\\_dosage & patient.radiations.radiation-4.radiation\\_treatment\\_ongoing & patient.radiations.radiation-4.radiation\\_type & patient.radiations.radiation-4.radiation\\_type\\_notes & patient.radiations.radiation-4.regimen\\_indication & patient.radiations.radiation-4.regimen\\_indication\\_notes & patient.radiations.radiation-4.units & patient.radiations.radiation-4.year\\_of\\_form\\_completion & patient.radiations.radiation.anatomic\\_treatment\\_site & patient.radiations.radiation.bcr\\_radiation\\_barcode & patient.radiations.radiation.bcr\\_radiation\\_uuid & patient.radiations.radiation.course\\_number & patient.radiations.radiation.day\\_of\\_form\\_completion & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation.measure\\_of\\_response & patient.radiations.radiation.month\\_of\\_form\\_completion & patient.radiations.radiation.numfractions & patient.radiations.radiation.radiation\\_dosage & patient.radiations.radiation.radiation\\_treatment\\_ongoing & patient.radiations.radiation.radiation\\_type & patient.radiations.radiation.radiation\\_type\\_notes & patient.radiations.radiation.regimen\\_indication & patient.radiations.radiation.regimen\\_indication\\_notes & patient.radiations.radiation.units & patient.radiations.radiation.year\\_of\\_form\\_completion & patient.residual\\_tumor & patient.stage\\_event.ann\\_arbor.b\\_symptoms & patient.stage\\_event.ann\\_arbor.extranodal\\_involvement & patient.stage\\_event.clinical\\_stage & patient.stage\\_event.gleason\\_grading.gleason\\_score & patient.stage\\_event.gleason\\_grading.primary\\_pattern & patient.stage\\_event.gleason\\_grading.secondary\\_pattern & patient.stage\\_event.gleason\\_grading.tertiary\\_pattern & patient.stage\\_event.igcccg\\_stage & patient.stage\\_event.masaoka\\_stage & patient.stage\\_event.pathologic\\_stage & patient.stage\\_event.psa.days\\_to\\_psa & patient.stage\\_event.psa.psa\\_value & patient.stage\\_event.serum\\_markers & patient.stage\\_event.system\\_version & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_m & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_n & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_t & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t & patient.stopped\\_smoking\\_year & patient.targeted\\_molecular\\_therapy & patient.tissue\\_prospective\\_collection\\_indicator & patient.tissue\\_retrospective\\_collection\\_indicator & patient.tissue\\_source\\_site & patient.tobacco\\_smoking\\_history & patient.tumor\\_tissue\\_site & patient.vital\\_status & patient.year\\_of\\_form\\_completion & patient.year\\_of\\_initial\\_pathologic\\_diagnosis & patient.year\\_of\\_tobacco\\_smoking\\_onset & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-2 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-3 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status & patient.biospecimen\\_cqcf.b\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status & patient.biospecimen\\_cqcf.country & patient.biospecimen\\_cqcf.cytogenetic\\_report\\_submitted & patient.biospecimen\\_cqcf.days\\_to\\_pathology\\_review & patient.biospecimen\\_cqcf.days\\_to\\_sample\\_procurement & patient.biospecimen\\_cqcf.differential\\_report\\_submitted & patient.biospecimen\\_cqcf.digital\\_image\\_submitted & patient.biospecimen\\_cqcf.ethnicity & patient.biospecimen\\_cqcf.ffpe\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site & patient.biospecimen\\_cqcf.histological\\_type & patient.biospecimen\\_cqcf.histological\\_type\\_other & patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.biospecimen\\_cqcf.hiv\\_positive\\_status & patient.biospecimen\\_cqcf.laterality & patient.biospecimen\\_cqcf.maximum\\_tumor\\_dimension & patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number & patient.biospecimen\\_cqcf.other\\_method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.other\\_vessel\\_used & patient.biospecimen\\_cqcf.path\\_confirm\\_diagnosis\\_matching & patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_necrosis\\_metrics & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_nuclei\\_metrics & patient.biospecimen\\_cqcf.percent\\_myeloblasts\\_for\\_submitted\\_specimen & patient.biospecimen\\_cqcf.race & patient.biospecimen\\_cqcf.reason\\_path\\_confirm\\_diagnosis\\_not\\_matching & patient.biospecimen\\_cqcf.sample\\_prescreened & patient.biospecimen\\_cqcf.site\\_of\\_disease & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_description & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_text & patient.biospecimen\\_cqcf.submitted\\_for\\_lce & patient.biospecimen\\_cqcf.t\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.top\\_slide\\_submitted & patient.biospecimen\\_cqcf.total\\_cells\\_submitted & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.vial\\_number & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number & patient.biospecimen\\_cqcf.vessel\\_used & patient.days\\_to\\_index & patient.samples.sample-2.additional\\_studies & patient.samples.sample-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-2.bcr\\_sample\\_barcode & patient.samples.sample-2.bcr\\_sample\\_uuid & patient.samples.sample-2.biospecimen\\_sequence & patient.samples.sample-2.composition & patient.samples.sample-2.current\\_weight & patient.samples.sample-2.days\\_to\\_collection & patient.samples.sample-2.days\\_to\\_sample\\_procurement & patient.samples.sample-2.diagnostic\\_slides.additional\\_studies & patient.samples.sample-2.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample-2.freezing\\_method & patient.samples.sample-2.initial\\_weight & patient.samples.sample-2.intermediate\\_dimension & patient.samples.sample-2.is\\_ffpe & patient.samples.sample-2.longest\\_dimension & patient.samples.sample-2.method\\_of\\_sample\\_procurement & patient.samples.sample-2.ncnnct\\_othmethonsp & patient.samples.sample-2.oct\\_embedded & patient.samples.sample-2.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-2.pathology\\_report\\_file\\_name & patient.samples.sample-2.pathology\\_report\\_uuid & patient.samples.sample-2.portions.portion-2.additional\\_studies & patient.samples.sample-2.portions.portion-2.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion-2.analytes.analyte.additional\\_studies & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion-2.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion-2.analytes.analyte.amount & patient.samples.sample-2.portions.portion-2.analytes.analyte.analyte\\_type & patient.samples.sample-2.portions.portion-2.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-2.portions.portion-2.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion-2.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion-2.analytes.analyte.concentration & patient.samples.sample-2.portions.portion-2.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion-2.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion-2.analytes.analyte.gel\\_image\\_file & patient.samples.sample-2.portions.portion-2.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion-2.analytes.analyte.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion-2.analytes.analyte.rna.rinvalue & patient.samples.sample-2.portions.portion-2.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-2.portions.portion-2.analytes.analyte.subportion\\_sequence & patient.samples.sample-2.portions.portion-2.analytes.analyte.well\\_number & patient.samples.sample-2.portions.portion-2.bcr\\_portion\\_barcode & patient.samples.sample-2.portions.portion-2.bcr\\_portion\\_uuid & patient.samples.sample-2.portions.portion-2.day\\_of\\_creation & patient.samples.sample-2.portions.portion-2.is\\_ffpe & patient.samples.sample-2.portions.portion-2.lce & patient.samples.sample-2.portions.portion-2.month\\_of\\_creation & patient.samples.sample-2.portions.portion-2.portion\\_number & patient.samples.sample-2.portions.portion-2.portion\\_sequence & patient.samples.sample-2.portions.portion-2.slides.slide-2.additional\\_studies & patient.samples.sample-2.portions.portion-2.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion-2.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion-2.slides.slide-2.image\\_file\\_name & patient.samples.sample-2.portions.portion-2.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_necrosis & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion-2.slides.slide-2.section\\_location & patient.samples.sample-2.portions.portion-2.slides.slide.additional\\_studies & patient.samples.sample-2.portions.portion-2.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion-2.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion-2.slides.slide.image\\_file\\_name & patient.samples.sample-2.portions.portion-2.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion-2.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_necrosis & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion-2.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion-2.slides.slide.section\\_location & patient.samples.sample-2.portions.portion-2.weight & patient.samples.sample-2.portions.portion-2.year\\_of\\_creation & patient.samples.sample-2.portions.portion.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.amount & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-2.portions.portion.day\\_of\\_creation & patient.samples.sample-2.portions.portion.is\\_ffpe & patient.samples.sample-2.portions.portion.lce & patient.samples.sample-2.portions.portion.month\\_of\\_creation & patient.samples.sample-2.portions.portion.portion\\_number & patient.samples.sample-2.portions.portion.portion\\_sequence & patient.samples.sample-2.portions.portion.slides & patient.samples.sample-2.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide-2.section\\_location & patient.samples.sample-2.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide.section\\_location & patient.samples.sample-2.portions.portion.weight & patient.samples.sample-2.portions.portion.year\\_of\\_creation & patient.samples.sample-2.preservation\\_method & patient.samples.sample-2.sample\\_type & patient.samples.sample-2.sample\\_type\\_id & patient.samples.sample-2.shortest\\_dimension & patient.samples.sample-2.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-2.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-2.tissue\\_type & patient.samples.sample-2.tumor\\_descriptor & patient.samples.sample-2.tumor\\_pathology & patient.samples.sample-2.vial\\_number & patient.samples.sample-3.additional\\_studies & patient.samples.sample-3.bcr\\_sample\\_barcode & patient.samples.sample-3.bcr\\_sample\\_uuid & patient.samples.sample-3.biospecimen\\_sequence & patient.samples.sample-3.composition & patient.samples.sample-3.current\\_weight & patient.samples.sample-3.days\\_to\\_collection & patient.samples.sample-3.days\\_to\\_sample\\_procurement & patient.samples.sample-3.diagnostic\\_slides.additional\\_studies & patient.samples.sample-3.freezing\\_method & patient.samples.sample-3.initial\\_weight & patient.samples.sample-3.intermediate\\_dimension & patient.samples.sample-3.is\\_ffpe & patient.samples.sample-3.longest\\_dimension & patient.samples.sample-3.method\\_of\\_sample\\_procurement & patient.samples.sample-3.ncnnct\\_othmethonsp & patient.samples.sample-3.oct\\_embedded & patient.samples.sample-3.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-3.pathology\\_report\\_file\\_name & patient.samples.sample-3.pathology\\_report\\_uuid & patient.samples.sample-3.portions.portion.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-3.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample-3.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.amount & patient.samples.sample-3.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-4.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-8.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-9.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.amount & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-3.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-3.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-3.portions.portion.day\\_of\\_creation & patient.samples.sample-3.portions.portion.is\\_ffpe & patient.samples.sample-3.portions.portion.lce & patient.samples.sample-3.portions.portion.month\\_of\\_creation & patient.samples.sample-3.portions.portion.portion\\_number & patient.samples.sample-3.portions.portion.portion\\_sequence & patient.samples.sample-3.portions.portion.slides & patient.samples.sample-3.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample-3.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample-3.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion.slides.slide-2.section\\_location & patient.samples.sample-3.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-3.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-3.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion.slides.slide.section\\_location & patient.samples.sample-3.portions.portion.weight & patient.samples.sample-3.portions.portion.year\\_of\\_creation & patient.samples.sample-3.preservation\\_method & patient.samples.sample-3.sample\\_type & patient.samples.sample-3.sample\\_type\\_id & patient.samples.sample-3.shortest\\_dimension & patient.samples.sample-3.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-3.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-3.tissue\\_type & patient.samples.sample-3.tumor\\_descriptor & patient.samples.sample-3.tumor\\_pathology & patient.samples.sample-3.vial\\_number & patient.samples.sample-4.additional\\_studies & patient.samples.sample-4.bcr\\_sample\\_barcode & patient.samples.sample-4.bcr\\_sample\\_uuid & patient.samples.sample-4.biospecimen\\_sequence & patient.samples.sample-4.composition & patient.samples.sample-4.current\\_weight & patient.samples.sample-4.days\\_to\\_collection & patient.samples.sample-4.days\\_to\\_sample\\_procurement & patient.samples.sample-4.diagnostic\\_slides.additional\\_studies & patient.samples.sample-4.freezing\\_method & patient.samples.sample-4.initial\\_weight & patient.samples.sample-4.intermediate\\_dimension & patient.samples.sample-4.is\\_ffpe & patient.samples.sample-4.longest\\_dimension & patient.samples.sample-4.method\\_of\\_sample\\_procurement & patient.samples.sample-4.ncnnct\\_othmethonsp & patient.samples.sample-4.oct\\_embedded & patient.samples.sample-4.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-4.pathology\\_report\\_file\\_name & patient.samples.sample-4.pathology\\_report\\_uuid & patient.samples.sample-4.portions.portion-2.additional\\_studies & patient.samples.sample-4.portions.portion-2.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion-2.analytes.analyte.additional\\_studies & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion-2.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion-2.analytes.analyte.amount & patient.samples.sample-4.portions.portion-2.analytes.analyte.analyte\\_type & patient.samples.sample-4.portions.portion-2.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-4.portions.portion-2.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion-2.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion-2.analytes.analyte.concentration & patient.samples.sample-4.portions.portion-2.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion-2.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion-2.analytes.analyte.gel\\_image\\_file & patient.samples.sample-4.portions.portion-2.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-2.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion-2.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-4.portions.portion-2.analytes.analyte.subportion\\_sequence & patient.samples.sample-4.portions.portion-2.analytes.analyte.well\\_number & patient.samples.sample-4.portions.portion-2.bcr\\_portion\\_barcode & patient.samples.sample-4.portions.portion-2.bcr\\_portion\\_uuid & patient.samples.sample-4.portions.portion-2.day\\_of\\_creation & patient.samples.sample-4.portions.portion-2.is\\_ffpe & patient.samples.sample-4.portions.portion-2.lce & patient.samples.sample-4.portions.portion-2.month\\_of\\_creation & patient.samples.sample-4.portions.portion-2.portion\\_number & patient.samples.sample-4.portions.portion-2.portion\\_sequence & patient.samples.sample-4.portions.portion-2.slides.slide.additional\\_studies & patient.samples.sample-4.portions.portion-2.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-4.portions.portion-2.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-4.portions.portion-2.slides.slide.image\\_file\\_name & patient.samples.sample-4.portions.portion-2.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-2.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_necrosis & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_normal\\_cells & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-4.portions.portion-2.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-4.portions.portion-2.slides.slide.section\\_location & patient.samples.sample-4.portions.portion-2.weight & patient.samples.sample-4.portions.portion-2.year\\_of\\_creation & patient.samples.sample-4.portions.portion-3.additional\\_studies & patient.samples.sample-4.portions.portion-3.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion-3.analytes.analyte.additional\\_studies & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion-3.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion-3.analytes.analyte.amount & patient.samples.sample-4.portions.portion-3.analytes.analyte.analyte\\_type & patient.samples.sample-4.portions.portion-3.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-4.portions.portion-3.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion-3.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion-3.analytes.analyte.concentration & patient.samples.sample-4.portions.portion-3.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion-3.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion-3.analytes.analyte.gel\\_image\\_file & patient.samples.sample-4.portions.portion-3.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-3.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion-3.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion-3.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion-3.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-4.portions.portion-3.analytes.analyte.subportion\\_sequence & patient.samples.sample-4.portions.portion-3.analytes.analyte.well\\_number & patient.samples.sample-4.portions.portion-3.bcr\\_portion\\_barcode & patient.samples.sample-4.portions.portion-3.bcr\\_portion\\_uuid & patient.samples.sample-4.portions.portion-3.day\\_of\\_creation & patient.samples.sample-4.portions.portion-3.is\\_ffpe & patient.samples.sample-4.portions.portion-3.lce & patient.samples.sample-4.portions.portion-3.month\\_of\\_creation & patient.samples.sample-4.portions.portion-3.portion\\_number & patient.samples.sample-4.portions.portion-3.portion\\_sequence & patient.samples.sample-4.portions.portion-3.slides.slide.additional\\_studies & patient.samples.sample-4.portions.portion-3.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-4.portions.portion-3.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-4.portions.portion-3.slides.slide.image\\_file\\_name & patient.samples.sample-4.portions.portion-3.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion-3.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_necrosis & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_normal\\_cells & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-4.portions.portion-3.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-4.portions.portion-3.slides.slide.section\\_location & patient.samples.sample-4.portions.portion-3.weight & patient.samples.sample-4.portions.portion-3.year\\_of\\_creation & patient.samples.sample-4.portions.portion.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-4.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-4.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-4.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-3.amount & patient.samples.sample-4.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-4.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.amount & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-4.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-4.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-4.portions.portion.day\\_of\\_creation & patient.samples.sample-4.portions.portion.is\\_ffpe & patient.samples.sample-4.portions.portion.lce & patient.samples.sample-4.portions.portion.month\\_of\\_creation & patient.samples.sample-4.portions.portion.portion\\_number & patient.samples.sample-4.portions.portion.portion\\_sequence & patient.samples.sample-4.portions.portion.slides & patient.samples.sample-4.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample-4.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-4.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-4.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample-4.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-4.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-4.portions.portion.slides.slide-2.section\\_location & patient.samples.sample-4.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-4.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-4.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-4.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-4.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-4.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-4.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-4.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-4.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-4.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-4.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-4.portions.portion.slides.slide.section\\_location & patient.samples.sample-4.portions.portion.weight & patient.samples.sample-4.portions.portion.year\\_of\\_creation & patient.samples.sample-4.preservation\\_method & patient.samples.sample-4.sample\\_type & patient.samples.sample-4.sample\\_type\\_id & patient.samples.sample-4.shortest\\_dimension & patient.samples.sample-4.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-4.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-4.tissue\\_type & patient.samples.sample-4.tumor\\_descriptor & patient.samples.sample-4.tumor\\_pathology & patient.samples.sample-4.vial\\_number & patient.samples.sample.additional\\_studies & patient.samples.sample.bcr\\_sample\\_barcode & patient.samples.sample.bcr\\_sample\\_uuid & patient.samples.sample.biospecimen\\_sequence & patient.samples.sample.composition & patient.samples.sample.current\\_weight & patient.samples.sample.days\\_to\\_collection & patient.samples.sample.days\\_to\\_sample\\_procurement & patient.samples.sample.diagnostic\\_slides.additional\\_studies & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-10 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-2 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-3 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-4 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-5 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-6 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-7 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-8 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-9 & patient.samples.sample.freezing\\_method & patient.samples.sample.initial\\_weight & patient.samples.sample.intermediate\\_dimension & patient.samples.sample.is\\_ffpe & patient.samples.sample.longest\\_dimension & patient.samples.sample.method\\_of\\_sample\\_procurement & patient.samples.sample.ncnnct\\_othmethonsp & patient.samples.sample.oct\\_embedded & patient.samples.sample.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample.pathology\\_report\\_file\\_name & patient.samples.sample.pathology\\_report\\_uuid & patient.samples.sample.portions.portion-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion-2.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion-2.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte-3.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion-2.analytes.analyte-3.rna.rinvalue & patient.samples.sample.portions.portion-2.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-3.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-3.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-4.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion-2.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion-2.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-4.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-4.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.amount & patient.samples.sample.portions.portion-2.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion-2.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion-2.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte.well\\_number & patient.samples.sample.portions.portion-2.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion-2.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion-2.day\\_of\\_creation & patient.samples.sample.portions.portion-2.is\\_ffpe & patient.samples.sample.portions.portion-2.lce & patient.samples.sample.portions.portion-2.month\\_of\\_creation & patient.samples.sample.portions.portion-2.portion\\_number & patient.samples.sample.portions.portion-2.portion\\_sequence & patient.samples.sample.portions.portion-2.slides.slide-2.additional\\_studies & patient.samples.sample.portions.portion-2.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion-2.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion-2.slides.slide-2.image\\_file\\_name & patient.samples.sample.portions.portion-2.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_necrosis & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion-2.slides.slide-2.section\\_location & patient.samples.sample.portions.portion-2.slides.slide.additional\\_studies & patient.samples.sample.portions.portion-2.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion-2.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion-2.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion-2.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion-2.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion-2.slides.slide.section\\_location & patient.samples.sample.portions.portion-2.weight & patient.samples.sample.portions.portion-2.year\\_of\\_creation & patient.samples.sample.portions.portion-3.additional\\_studies & patient.samples.sample.portions.portion-3.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-3.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion-3.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-3.analytes.analyte.amount & patient.samples.sample.portions.portion-3.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion-3.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion-3.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-3.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-3.analytes.analyte.concentration & patient.samples.sample.portions.portion-3.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion-3.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion-3.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion-3.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-3.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-3.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-3.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-3.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion-3.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion-3.analytes.analyte.well\\_number & patient.samples.sample.portions.portion-3.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion-3.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion-3.day\\_of\\_creation & patient.samples.sample.portions.portion-3.is\\_ffpe & patient.samples.sample.portions.portion-3.lce & patient.samples.sample.portions.portion-3.month\\_of\\_creation & patient.samples.sample.portions.portion-3.portion\\_number & patient.samples.sample.portions.portion-3.portion\\_sequence & patient.samples.sample.portions.portion-3.slides.slide.additional\\_studies & patient.samples.sample.portions.portion-3.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion-3.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion-3.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion-3.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-3.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion-3.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion-3.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion-3.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion-3.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion-3.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion-3.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion-3.slides.slide.section\\_location & patient.samples.sample.portions.portion-3.weight & patient.samples.sample.portions.portion-3.year\\_of\\_creation & patient.samples.sample.portions.portion.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.amount & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.amount & patient.samples.sample.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-5.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-5.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.amount & patient.samples.sample.portions.portion.analytes.analyte-5.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-5.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-5.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-5.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-5.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-5.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-5.well\\_number & patient.samples.sample.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-14.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-15.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-16.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.disease\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies.additional\\_study.project\\_code & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.amount & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.concentration & patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte.well\\_number & patient.samples.sample.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion.day\\_of\\_creation & patient.samples.sample.portions.portion.is\\_ffpe & patient.samples.sample.portions.portion.lce & patient.samples.sample.portions.portion.month\\_of\\_creation & patient.samples.sample.portions.portion.portion\\_number & patient.samples.sample.portions.portion.portion\\_sequence & patient.samples.sample.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide-2.section\\_location & patient.samples.sample.portions.portion.slides.slide.additional\\_studies & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide.section\\_location & patient.samples.sample.portions.portion.weight & patient.samples.sample.portions.portion.year\\_of\\_creation & patient.samples.sample.portions.shipment\\_portion.additional\\_studies & patient.samples.sample.portions.shipment\\_portion.bcr\\_shipment\\_portion\\_uuid & patient.samples.sample.portions.shipment\\_portion.center\\_id & patient.samples.sample.portions.shipment\\_portion.is\\_ffpe & patient.samples.sample.portions.shipment\\_portion.plate\\_id & patient.samples.sample.portions.shipment\\_portion.portion\\_number & patient.samples.sample.portions.shipment\\_portion.portion\\_sequence & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_bcr\\_aliquot\\_barcode & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_day\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_month\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_year\\_of\\_shipment & patient.samples.sample.preservation\\_method & patient.samples.sample.sample\\_type & patient.samples.sample.sample\\_type\\_id & patient.samples.sample.shortest\\_dimension & patient.samples.sample.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample.tissue\\_type & patient.samples.sample.tumor\\_descriptor & patient.samples.sample.tumor\\_pathology & patient.samples.sample.vial\\_number\\\\\n",
       "  & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t52.70.0 & nationwide children's hospital & 1 & luad & 071e3c36-bd5c-49b2-b3d3-0eb4e781810f & 10 & false & tcga & 2015 & NA & NA & NA & 70 & r-lower & NA & tcga-05-4244 & 34040b83-7e8a-4264-a551-b16621843e28 & r-lower & consented & germany & 0 & NA & NA & lung adenocarcinoma- not otherwise specified (nos) & no & no                               & NA & NA & NA                                   & NA & NA & primary & 22 & -25752 & NA  & 0 & 0    & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA              & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA        & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA  & NA & NA                             & NA                          & NA & NA & NA & NA                 & NA & NA  & NA & NA         & NA                          & NA & NA & NA  & NA  & NA    & NA   & male   & lung adenocarcinoma- not otherwise specified (nos) & no & c34.3 & 8140/3 & c34.3 & yes & NA & NA & NA & NA & peripheral lung & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 38 & 4244 & NA & tumor free & NA & NA & NA & NA & NA & no  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                 & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & rx & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iv   & NA & NA & NA & 6th & NA & NA & NA & m1 & n2 & t2 & NA   & NA & no & yes & 05 & current reformed smoker for < or = 15 years & lung & alive & 2010 & 2009 & NA   & NA & NA & NA & canonical & NA & consented & germany & NA & 241  & 0 & NA & no & NA & NA & lung & lung adenocarcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA                                   & NA & NA & NA & NA & NA & NA         & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 4edd48ee-134e-43cd-9846-6414c4842941 & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & lobectomy       & NA & NA & yes & NA & NA & NA & NA & NA & yes & lung & lower & NA                                   & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & bac0b02d-ac3b-4784-b8bf-180eadd548a3 & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4244-10a & 4edd48ee-134e-43cd-9846-6414c4842941 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-10a-01w-1069-09 & 8d292a5c-922a-45ed-8cf8-373e1815f8b4 & 0098890740 & 09 & 0.50 & 29 & no & 06 & 3  & 1069 & e  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-10a-01w-1067-08 & 8dce8d47-4734-45ec-adc6-3120b55b4c7a & 0098888148 & 08 & 0.50 & 29 & no & 06 & 3 & 1067 & e & 40.00 & 22 & 80.00 & 2010 & 54.00  & repli-g (qiagen) dna   & w & tcga-05-4244-10a-01w & e4724a91-607e-4d2d-b6d1-0a080a1b274b & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g   & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4244-10a-01x-1070-09 & d32429a5-21ed-48e8-8d4a-d0ccf47e08e3 & 0098888340 & 09 & 0.50 & 29 & no & 06 & 3  & 1070 & e  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4244-10a-01x-1068-08 & abc25fde-3786-4fc9-b66c-a64213413cc4 & 0098891124 & 08 & 0.50 & 29 & no & 06 & 3  & 1068 & e  & 40.00 & 22 & 80.00 & 2010 & 97.00 & repli-g x (qiagen) dna & x  & tcga-05-4244-10a-01x & 504b3b86-b1ed-420f-ac3f-01657b1de872 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & tcga-05-4244-10a-01d-1105-08 & 34f50d09-4bc8-4adb-a09b-67746ecb2bdc & 0098884600 & 08 & 0.16 & 14 & no & 07 & 2 & 1105 & d & 1.60  & 22 & 10.00 & 2010 & NA & NA & NA & NA & tcga-05-4244-10a-01d-1106-09 & 719aab25-9745-4dd4-b65e-42453de0e5ce & 0098846488 & 09 & 0.16 & 14 & no & 07 & 2  & 1106 & d  & 1.07 & 22 & 6.70  & 2010 & NA & NA & tcga-05-4244-10a-01d-1877-01 & a7a90cd5-3554-4be9-9b28-8ee08f5515e3 & 1053935999 & 01 & 0.16 & 11 & no & 07 & 3  & 1877 & b  & 1.07 & 22 & 6.67 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-10a-01d-1103-02 & 2a2f93ca-a914-4309-ac80-e6b630e73d2e & 0098884976 & 02 & 0.16 & 14 & no & 07 & 2 & 1103 & f & 2.13  & 22 & 13.30 & 2010 & 49.63 & dna                  & d & tcga-05-4244-10a-01d & eeb9d239-e1a6-46e5-b02e-018b0bc911eb & 0.16 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & chemical lysis dna extraction & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4244-10a-01 & e8a10567-06ed-40fa-9bc7-bfaab40c91ac & 28 & no & NA & 05 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a & bac0b02d-ac3b-4784-b8bf-180eadd548a3 & NA & NA & NA & NA & NA & NA & d4ff32cd-38cf-40ea-8213-45c2b100ac01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.9 & no & 1.1 & NA & NA & NA & NA & tcga-05-4244.3a844132-f813-4d8e-8f7d-dbae0b23d7fd.pdf & 3a844132-f813-4d8e-8f7d-dbae0b23d7fd & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01r-1107-07 & e6a101b9-61f9-4ed1-a59f-d9db3fdb4555 & NA & 0098849126 & 07 & 0.14 & 14 & no & 07 & 1 & 1107 & a & 3.74  & 22 & 26.70 & 2010 & 140.30 & rna                    & r & tcga-05-4244-01a-01r & 7787c8a5-7d59-42e8-a509-73c8ed6ee3a2 & 0.14 & NA  & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.50 & 9.60 & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01t-1108-13 & 1575b91f-dcc8-472b-9975-781b0fa5416a & 0098846726 & 13 & 0.10 & 14 & no & 07 & 1  & 1108 & a  & 3.00 & 22 & 30.00 & 2010 & 164.00 & total rna & t  & tcga-05-4244-01a-01t & 6f3beaa6-efe6-45c0-af9e-6543b2b45e37 & 0.10 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & 1.50 & 8.50 & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01w-1069-09 & 80bca329-e1d1-4f6a-b342-f022be8e0142 & 0098890767 & 09 & 0.50 & 29 & no & 06 & 1  & 1069 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01w-1067-08 & afaab933-668c-4b76-8c1c-8c8da91e9e34 & 0098888175 & 08 & 0.50 & 29 & no & 06 & 1  & 1067 & b  & 40.00 & 22 & 80.00 & 2010 & 97.00  & repli-g (qiagen) dna & w  & tcga-05-4244-01a-01w & cdd3a7d0-1028-4509-9cc3-76852cc489df & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01x-1070-09 & e927fa15-ad47-40d6-8384-226483d18c43 & 0098888367 & 09 & 0.50 & 29 & no & 06 & 1  & 1070 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4244-01a-01x-1068-08 & b4f4f684-e105-428f-9b83-ff25370d9098 & 0098891151 & 08 & 0.50 & 29 & no & 06 & 1  & 1068 & b  & 40.00 & 22 & 80.00 & 2010 & 68.00  & repli-g x (qiagen) dna & x  & tcga-05-4244-01a-01x & 175f1c09-5343-4d6d-8bf6-397c7b80e2f2 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01d-1104-05 & 00428bbc-fe33-40c5-8c8c-dab29efef065 & NA & 0098885510 & 05 & 0.14 & 14 & no & 07 & 1 & 1104 & a & 3.74  & 22 & 26.70 & 2010 & NA & NA & NA & NA & tcga-05-4244-01a-01d-1105-08 & 738c514d-ff0f-4220-9326-236119891df5 & NA & 0098884646 & 08 & 0.14 & 14 & no & 07 & 1  & 1105 & a  & 1.40 & 22 & 10.00 & 2010 & NA & NA & tcga-05-4244-01a-01d-1106-09 & 90e94448-3ee7-4bba-8991-e24173dbaab5 & 0098846534 & 09 & 0.14 & 14 & no & 07 & 1  & 1106 & a  & 0.94 & 22 & 6.70  & 2010 & NA & NA & NA & NA & tcga-05-4244-01a-01d-1877-01 & e29ae073-4f2d-4159-b64b-8ab0400ce788 & NA & 1053935978 & 01 & 0.14 & 11 & no & 07 & 1  & 1877 & a  & 0.93 & 22 & 6.67 & 2011 & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4244-01a-01d-1103-02 & ceda4ce2-3ebf-4387-a27c-85415338d0ae & NA & 0098885026 & 02 & 0.14 & 14 & no & 07 & 1 & 1103 & a & 1.86  & 22 & 13.30 & 2010 & 64.93  & dna                  & d & tcga-05-4244-01a-01d & 42c8ab00-0c33-47a8-b93d-19c4f3734324 & 0.14 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-05-4244-01a-01 & 53eed368-b7e8-4272-8a8e-40ce5d66fff3 & 20 & no & NA & 05 & 01 & NA & NA & tcga-05-4244-01a-01-ts1 & 6c46d8c3-d7b4-4513-af31-9170244d60a7 & tcga-05-4244-01a-01-ts1.6c46d8c3-d7b4-4513-af31-9170244d60a7.svs & no & NA & NA & NA & NA & NA & NA & 0  & NA & 10 & 20 & 70 & 80 & top & NA & tcga-05-4244-01a-01-bs1 & 90f043f6-e8d5-4cfd-b4e7-84207bc499a6 & tcga-05-4244-01a-01-bs1.90f043f6-e8d5-4cfd-b4e7-84207bc499a6.svs & no & NA & NA & NA & NA & NA & NA & 5  & NA & 15 & 20 & 60 & 80 & bottom & 133.00 & 2010 & NA & 59881fe3-70f4-4fa0-9355-f50250872752 & 20 & no & 2190 & 21 & NA & tcga-05-4244-01a-21-2190-20 & 5  & 3  & 2012 & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t52.70.0.1 & nationwide children's hospital & 1 & luad & 74423066-665a-43f2-a801-fb4f83ee15e9 & 10 & false & tcga & 2015 & NA & NA & NA & 81 & l-upper & NA & tcga-05-4245 & 03d09c05-49ab-4ba6-a8d7-e7ccf71fafd2 & l-upper & consented & germany & 0 & NA & NA & lung adenocarcinoma- not otherwise specified (nos) & no & yes, history of prior malignancy & NA & NA & NA                                   & NA & NA & primary & 22 & -29647 & NA  & 0 & 395  & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA              & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA        & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & no  & NA & NA & tcga-05-4245-f36308 & 8b6363a9-c846-48d4-8051-2a0b97bad585 & 29 & NA & NA & NA & 730  & 334 & NA & scheduled follow-up submission & complete remission/response & NA & no & 10 & NA                 & NA & yes & NA & tumor free & NA                          & NA & NA & no  & no  & alive & 2012 & male   & lung adenocarcinoma- not otherwise specified (nos) & no & c34.1 & 8140/3 & c34.1 & yes & NA & NA & NA & NA & NA              & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 32 & 4245 & NA & tumor free & NA & NA & NA & NA & NA & yes & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                 & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & r2 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iiia & NA & NA & NA & 6th & NA & NA & NA & m0 & n2 & t2 & NA   & NA & no & yes & 05 & current reformed smoker for < or = 15 years & lung & alive & 2010 & 2009 & NA   & NA & NA & NA & canonical & NA & consented & germany & NA & 241  & 0 & NA & no & NA & NA & lung & lung adenocarcinoma- not otherwise specified (nos) & NA & no & NA & left  & NA & tumor resection               & NA                                   & NA & NA & NA & NA & NA & NA         & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 499df411-f9c5-45b1-96a8-4acc6d057a90 & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA              & NA & NA & no  & NA & NA & NA & NA & NA & yes & lung & upper & NA                                   & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & c9dd827b-e0ae-4807-adbf-1a0bf2aeb52f & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4245-10a & 499df411-f9c5-45b1-96a8-4acc6d057a90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-10a-01x-1070-09 & b4987fe7-db4f-46cb-af81-e2e935a2ac7b & 0098890263 & 09 & 0.50 & 29 & no & 06 & 3  & 1070 & f  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-10a-01x-1068-08 & 69b754eb-07fd-4f2c-a6e5-d576e7b407e7 & 0098891105 & 08 & 0.50 & 29 & no & 06 & 3 & 1068 & f & 40.00 & 22 & 80.00 & 2010 & 84.00  & repli-g x (qiagen) dna & x & tcga-05-4245-10a-01x & 8bd5fc26-1ba2-4b6f-a928-d39a5cbb0421 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA    & NA                     & NA & NA                   & NA                                   & NA   & NA  & NA & NA                                               & NA & NA        & NA & NA & NA & NA & NA      & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & tcga-05-4245-10a-01w-1069-09 & 5a5d9bc8-d51a-468a-a0a3-e61a4e13d3bf & 0098890721 & 09 & 0.50 & 29 & no & 06 & 3 & 1069 & f & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-10a-01w-1067-08 & c85d2f9d-798d-4801-9528-6e036cc16be5 & 0098888129 & 08 & 0.50 & 29 & no & 06 & 3 & 1067 & f & 40.00 & 22 & 80.00 & 2010 & 98.00 & repli-g (qiagen) dna & w & tcga-05-4245-10a-01w & afb7410a-2c47-4312-a721-40c343e648a2 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g                       & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4245-10a-01 & e281b433-7a5f-4d96-a01b-645f17162d0d & 28 & no & NA & 05 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-01a & c9dd827b-e0ae-4807-adbf-1a0bf2aeb52f & NA & NA & NA & NA & NA & NA & 36ff5403-d4bb-4415-b2c5-7c750d655cde & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.7 & no & 0.9 & NA & NA & NA & NA & tcga-05-4245.902fe548-5b93-49c9-81db-2af4a4a88f3c.pdf & 902fe548-5b93-49c9-81db-2af4a4a88f3c & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & tcga-05-4245-01a-01x-1070-09 & 3f94044f-d1d0-4158-85ae-65762559cfda & NA & 0098888366 & 09 & 0.50 & 29 & no & 06 & 1  & 1070 & c  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-01a-01x-1068-08 & 99eb465f-1147-4ccd-8ab2-a11304b8df6e & NA & 0098891150 & 08 & 0.50 & 29 & no & 06 & 1 & 1068 & c & 40.00 & 22 & 80.00 & 2010 & 92.00  & repli-g x (qiagen) dna & x & tcga-05-4245-01a-01x & 7d5b06ae-da9c-4e7f-8f07-f2cb96e9b3e9 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x              & NA & NA & NA   & NA   & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA     & NA        & NA & NA                   & NA                                   & NA   & NA & NA & NA                                               & NA & NA        & NA & NA & NA   & NA   & NA      & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA     & NA                   & NA & NA                   & NA                                   & NA   & NA  & NA & NA                                               & NA & NA      & NA & NA & NA      & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA     & NA                     & NA & NA                   & NA                                   & NA   & NA  & NA & NA                                               & NA & NA        & NA & NA & NA      & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-01a-01w-1069-09 & 8987f1c5-bcac-4c90-b0fb-5743d62ef804 & NA & 0098890766 & 09 & 0.50 & 29 & no & 06 & 1 & 1069 & c & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4245-01a-01w-1067-08 & 958cc18e-f4bf-4362-8887-32eb3ca0bb05 & NA & 0098889102 & 08 & 0.50 & 29 & no & 06 & 1 & 1067 & c & 40.00 & 22 & 80.00 & 2010 & 102.00 & repli-g (qiagen) dna & w & tcga-05-4245-01a-01w & 99e8ad13-a0b4-4dfa-a1fc-2bac0297c3e8 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g               & NA & NA & uv spec & NA & NA & tcga-05-4245-01a-01 & ca29fd94-0ae9-48bd-aad4-fa3e5eec37e7 & 20 & no & NA & 05 & 01 & NA & NA & tcga-05-4245-01a-01-ts1 & bf71c76b-e802-4a7a-b6c3-c5f46212fab0 & tcga-05-4245-01a-01-ts1.bf71c76b-e802-4a7a-b6c3-c5f46212fab0.svs & no & NA & NA & NA & NA & 2  & NA & 15 & NA & NA & 20 & 65 & 75 & top & NA & tcga-05-4245-01a-01-bs1 & 41d3cf23-4e36-4e42-9e08-adfea139f37e & tcga-05-4245-01a-01-bs1.41d3cf23-4e36-4e42-9e08-adfea139f37e.svs & no & NA & NA & NA & NA & 2  & NA & 15 & NA & 10 & 25 & 50 & 65 & bottom & 168.00 & 2010 & NA & NA                                   & NA & NA & NA   & NA & NA & NA                          & NA & NA & NA   & NA & primary tumor & 01 & 0.5 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t52.70.0.2 & nationwide children's hospital & 1 & luad & b06fd27e-a052-459f-be7b-c6e336a6629d & 10 & false & tcga & 2015 & NA & NA & NA & 67 & r-lower & NA & tcga-05-4249 & 4addf05f-3668-4b3f-a17f-c0227329ca52 & r-lower & consented & germany & 0 & NA & NA & lung adenocarcinoma- not otherwise specified (nos) & no & no                               & NA & NA & inferior pulmonary lobe right s viii & NA & NA & primary & 22 & -24532 & NA  & 0 & 1158 & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA              & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA        & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA & NA & tcga-05-4249-f36327 & 485963a2-753d-4e86-b3df-e3f66189abee & 29 & NA & NA & NA & 1523 & NA  & NA & scheduled follow-up submission & complete remission/response & NA & no & 10 & NA                 & NA & no  & NA & tumor free & NA                          & NA & NA & no  & no  & alive & 2012 & male   & lung adenocarcinoma- not otherwise specified (nos) & no & c34.3 & 8140/3 & c34.3 & yes & NA & NA & NA & NA & peripheral lung & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 52 & 4249 & NA & tumor free & NA & NA & NA & NA & NA & no  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                 & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ib   & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t2 & NA   & NA & no & yes & 05 & current reformed smoker for > 15 years      & lung & alive & 2010 & 2007 & NA   & NA & NA & NA & canonical & NA & consented & germany & NA & 1034 & 0 & NA & no & NA & NA & lung & lung adenocarcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA                                   & NA & NA & NA & NA & NA & NA         & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 6e8af3ce-3987-44f1-bf4c-e52f68039567 & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & lobectomy       & NA & NA & no  & NA & NA & NA & NA & NA & yes & lung & lower & inferior pulmonary lobe right s viii & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & 80f196fe-1eaf-40cb-ab58-c84795acc5c7 & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4249-10a & 6e8af3ce-3987-44f1-bf4c-e52f68039567 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-10a-01w-1069-09 & 8af9e6b9-5e30-4810-86e9-0fb9a8635c08 & 0098890716 & 09 & 0.50 & 29 & no & 06 & 3  & 1069 & g  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-10a-01w-1067-08 & cc597ad3-7bbe-4df6-adea-b1624ba2a251 & 0098888124 & 08 & 0.50 & 29 & no & 06 & 3 & 1067 & g & 40.00 & 22 & 80.00 & 2010 & 88.00  & repli-g (qiagen) dna   & w & tcga-05-4249-10a-01w & 312eeb7a-83a6-4b2a-a695-8500730bde32 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g   & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4249-10a-01x-1070-09 & 41e93e2e-19c1-419e-a810-4ce790d949ce & 0098888316 & 09 & 0.50 & 29 & no & 06 & 3  & 1070 & g  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4249-10a-01x-1068-08 & 6994f4c3-0fa5-4d7a-b72b-84a93c17b0fe & 0098891100 & 08 & 0.50 & 29 & no & 06 & 3  & 1068 & g  & 40.00 & 22 & 80.00 & 2010 & 96.00 & repli-g x (qiagen) dna & x  & tcga-05-4249-10a-01x & 165ec750-bb33-4af4-9897-e6a5a5ab2667 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & tcga-05-4249-10a-01d-1105-08 & 5d48c4d0-e1bd-4a65-9ecf-b6bad02196d7 & 0098884597 & 08 & 0.16 & 14 & no & 07 & 2 & 1105 & e & 1.60  & 22 & 10.00 & 2010 & NA & NA & NA & NA & tcga-05-4249-10a-01d-1106-09 & 0480f909-9ea5-499d-ba87-14bbefa6d68f & 0098846485 & 09 & 0.16 & 14 & no & 07 & 2  & 1106 & e  & 1.07 & 22 & 6.70  & 2010 & NA & NA & tcga-05-4249-10a-01d-1877-01 & bca406bc-71e8-4641-8ac4-cebcbd1a0c99 & 1053936004 & 01 & 0.16 & 11 & no & 07 & 3  & 1877 & c  & 1.07 & 22 & 6.67 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-10a-01d-1103-02 & e6defcc8-1e9d-46ed-b8ba-b8c62b21a5b5 & 0098886397 & 02 & 0.16 & 14 & no & 07 & 2 & 1103 & g & 2.13  & 22 & 13.30 & 2010 & 36.63 & dna                  & d & tcga-05-4249-10a-01d & fac3c29b-3693-4b38-8ecf-b6346cdb305b & 0.16 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & chemical lysis dna extraction & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4249-10a-01 & 066ac778-5418-4a4f-906a-9c5cac548a28 & 28 & no & NA & 05 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a & 80f196fe-1eaf-40cb-ab58-c84795acc5c7 & NA & NA & NA & NA & NA & NA & 9fce0297-cc19-4c04-872c-4466ee4024db & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.8 & no & 1.4 & NA & NA & NA & NA & tcga-05-4249.7e920317-d5c2-4160-9e2b-ef0101eb5a23.pdf & 7e920317-d5c2-4160-9e2b-ef0101eb5a23 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01r-1107-07 & d1a8d88d-1708-4959-9695-6f2e67853bd5 & NA & 0098849103 & 07 & 0.14 & 14 & no & 07 & 1 & 1107 & b & 3.74  & 22 & 26.70 & 2010 & 111.30 & rna                    & r & tcga-05-4249-01a-01r & 932163e9-2370-4385-bfa4-85314f40e412 & 0.14 & NA  & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.80 & 9.30 & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01t-1108-13 & 85756610-4540-4914-b228-254b2b814774 & 0098846703 & 13 & 0.10 & 14 & no & 07 & 1  & 1108 & b  & 3.00 & 22 & 30.00 & 2010 & 130.00 & total rna & t  & tcga-05-4249-01a-01t & 2546bc72-1195-45dd-b537-26ebc67a65ef & 0.10 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & 1.60 & 8.80 & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01w-1069-09 & 1e7e93e3-d75c-4bc6-b36e-377b461ddb70 & 0098890743 & 09 & 0.50 & 29 & no & 06 & 1  & 1069 & d  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01w-1067-08 & fd9e4258-0d7e-48f5-a21e-507c4709cce2 & 0098889071 & 08 & 0.50 & 29 & no & 06 & 1  & 1067 & d  & 40.00 & 22 & 80.00 & 2010 & 83.00  & repli-g (qiagen) dna & w  & tcga-05-4249-01a-01w & 20869797-8599-49e3-ae60-63865431bef3 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01x-1070-09 & 54d73fd6-569e-41d9-97c1-0e5780f1a56c & 0098888343 & 09 & 0.50 & 29 & no & 06 & 1  & 1070 & d  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4249-01a-01x-1068-08 & 8a0e4b88-cb8b-44e7-8e34-78b054f7b4d9 & 0098891127 & 08 & 0.50 & 29 & no & 06 & 1  & 1068 & d  & 40.00 & 22 & 80.00 & 2010 & 100.00 & repli-g x (qiagen) dna & x  & tcga-05-4249-01a-01x & b916d793-5816-491c-833f-533cffc62af0 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01d-1104-05 & abdf85e6-108e-4851-bcd7-ee9435d94585 & NA & 0098885487 & 05 & 0.15 & 14 & no & 07 & 1 & 1104 & b & 4.01  & 22 & 26.70 & 2010 & NA & NA & NA & NA & tcga-05-4249-01a-01d-1105-08 & 8be717b5-5b65-4631-a175-1f4c063d447e & NA & 0098884623 & 08 & 0.15 & 14 & no & 07 & 1  & 1105 & b  & 1.50 & 22 & 10.00 & 2010 & NA & NA & tcga-05-4249-01a-01d-1106-09 & b65eca35-abfb-4202-8806-c125b32f3e9a & 0098846511 & 09 & 0.15 & 14 & no & 07 & 1  & 1106 & b  & 1.01 & 22 & 6.70  & 2010 & NA & NA & NA & NA & tcga-05-4249-01a-01d-1877-01 & 65e93af7-f914-4fe6-bc55-1bd5fcfb5eb1 & NA & 1053936001 & 01 & 0.15 & 11 & no & 07 & 1  & 1877 & b  & 1.00 & 22 & 6.67 & 2011 & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4249-01a-01d-1103-02 & e3f50937-d6cf-423b-9dd3-31d715892fc7 & NA & 0098884937 & 02 & 0.15 & 14 & no & 07 & 1 & 1103 & b & 2.00  & 22 & 13.30 & 2010 & 64.93  & dna                  & d & tcga-05-4249-01a-01d & 4c30a1eb-6359-49fe-be97-49b541c34316 & 0.15 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-05-4249-01a-01 & a0db909f-b139-4065-8136-27033eedc324 & 20 & no & NA & 05 & 01 & NA & NA & tcga-05-4249-01a-01-ts1 & 912c8d26-dc9f-4bae-bfcd-559cf234c921 & tcga-05-4249-01a-01-ts1.912c8d26-dc9f-4bae-bfcd-559cf234c921.svs & no & NA & NA & NA & NA & NA & NA & 0  & NA & 0  & 20 & 80 & 80 & top & NA & tcga-05-4249-01a-01-bs1 & 38c41760-cc43-4904-9575-803d1d92b992 & tcga-05-4249-01a-01-bs1.38c41760-cc43-4904-9575-803d1d92b992.svs & no & NA & NA & NA & NA & NA & NA & 0  & NA & 0  & 20 & 80 & 80 & bottom & 111.00 & 2010 & NA & 5b46a235-49a6-43ea-91b6-82a384702477 & 20 & no & 2190 & 21 & NA & tcga-05-4249-01a-21-2190-20 & 5  & 3  & 2012 & NA & primary tumor & 01 & 0.4 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t52.70.0.3 & nationwide children's hospital & 1 & luad & 4bdbf19f-48c2-4565-a5b6-9ada2ef93e74 & 10 & false & tcga & 2015 & NA & NA & NA & 79 & r-lower & NA & tcga-05-4250 & f98ecd8a-b878-4f53-b911-20cd8e17281c & r-lower & consented & germany & 0 & NA & NA & lung adenocarcinoma- not otherwise specified (nos) & no & no                               & NA & NA & NA                                   & NA & NA & primary & 22 & -29068 & 121 & 0 & NA   & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA              & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA        & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA  & NA & NA                             & NA                          & NA & NA & NA & NA                 & NA & NA  & NA & NA         & NA                          & NA & NA & NA  & NA  & NA    & NA   & female & lung adenocarcinoma- not otherwise specified (nos) & no & c34.3 & 8140/3 & c34.3 & yes & NA & NA & NA & NA & NA              & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 47 & 4250 & NA & NA         & NA & NA & NA & NA & NA & no  & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                 & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & r2 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iiia & NA & NA & NA & 6th & NA & NA & NA & m0 & n1 & t3 & NA   & NA & no & yes & 05 & current reformed smoker for < or = 15 years & lung & dead  & 2010 & 2007 & NA   & NA & NA & NA & canonical & NA & consented & germany & NA & 911  & 0 & NA & no & NA & NA & lung & lung adenocarcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA                                   & NA & NA & NA & NA & NA & NA         & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 6ae21fea-41be-4cd2-bfcb-ec138d42cff3 & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & lobectomy       & NA & NA & no  & NA & NA & NA & NA & NA & yes & lung & lower & NA                                   & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & 8f274178-7a8e-46b6-8d2c-900338bbb946 & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4250-10a & 6ae21fea-41be-4cd2-bfcb-ec138d42cff3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-10a-01w-1069-09 & 10e5ec4c-17a7-425e-ab43-71ca9dddea5a & 0098890787 & 09 & 0.50 & 29 & no & 06 & 4  & 1069 & a  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-10a-01w-1067-08 & c2659bdd-54d8-4dd3-abbb-fb501d999809 & 0098888195 & 08 & 0.50 & 29 & no & 06 & 4 & 1067 & a & 40.00 & 22 & 80.00 & 2010 & 84.00  & repli-g (qiagen) dna   & w & tcga-05-4250-10a-01w & dc84151d-c9c2-4381-a87c-1002e9c81c7f & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g   & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4250-10a-01x-1070-09 & 1dda2ab6-dfd8-47ff-9efc-033cd1255ba2 & 0098890285 & 09 & 0.50 & 29 & no & 06 & 4  & 1070 & a  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4250-10a-01x-1068-08 & 3c02ae37-d777-4088-9892-5248502afa30 & 0098890249 & 08 & 0.50 & 29 & no & 06 & 4  & 1068 & a  & 40.00 & 22 & 80.00 & 2010 & 72.00 & repli-g x (qiagen) dna & x  & tcga-05-4250-10a-01x & 63e021bd-36bb-49e3-85f2-7bbb8a4318d6 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & tcga-05-4250-10a-01d-1105-08 & 15c63561-c425-49e3-8d90-00537cfa567f & 0098884576 & 08 & 0.16 & 14 & no & 07 & 2 & 1105 & f & 1.60  & 22 & 10.00 & 2010 & NA & NA & NA & NA & tcga-05-4250-10a-01d-1106-09 & eb06786d-8b1d-4152-bb26-66c4d340cc0b & 0098846464 & 09 & 0.16 & 14 & no & 07 & 2  & 1106 & f  & 1.07 & 22 & 6.70  & 2010 & NA & NA & tcga-05-4250-10a-01d-1877-01 & 89ee5155-db6f-486e-9452-2ce8190c91e8 & 1053936023 & 01 & 0.16 & 11 & no & 07 & 3  & 1877 & d  & 1.07 & 22 & 6.67 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-10a-01d-1103-02 & 6963dbce-5e1f-419e-8997-1178ca90d733 & 0098886376 & 02 & 0.16 & 14 & no & 07 & 2 & 1103 & h & 2.13  & 22 & 13.30 & 2010 & 14.63 & dna                  & d & tcga-05-4250-10a-01d & f1f0486c-fccb-49d4-8947-f3e9ce948bcc & 0.16 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & chemical lysis dna extraction & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4250-10a-01 & 6d0432ea-0c69-4f94-b245-aa60af946d67 & 28 & no & NA & 05 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a & 8f274178-7a8e-46b6-8d2c-900338bbb946 & NA & NA & NA & NA & NA & NA & 90f67fdf-dff9-46ca-af71-0978d7c221ba & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.7 & no & 1.2 & NA & NA & NA & NA & tcga-05-4250.5574f2f8-f247-40e6-a285-7793edcf5358.pdf & 5574f2f8-f247-40e6-a285-7793edcf5358 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01r-1107-07 & bba9333a-09f7-4585-b22e-e4ae4049f7da & NA & 0098849102 & 07 & 0.15 & 14 & no & 07 & 1 & 1107 & c & 4.01  & 22 & 26.70 & 2010 & 203.30 & rna                    & r & tcga-05-4250-01a-01r & 57d19d01-97bb-4040-af4e-795f5d600cd8 & 0.15 & NA  & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.40 & 9.20 & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01t-1108-13 & 79de28ff-c06a-4686-a82c-3a7c9b27d207 & 0098846702 & 13 & 0.10 & 14 & no & 07 & 1  & 1108 & c  & 3.00 & 22 & 30.00 & 2010 & 193.00 & total rna & t  & tcga-05-4250-01a-01t & ed208831-2283-4e55-858a-9163aa513c21 & 0.10 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & 1.30 & 8.50 & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01w-1069-09 & ef7d8062-5d9a-41d7-bc2c-f0173d5b6f7b & 0098890742 & 09 & 0.50 & 29 & no & 06 & 1  & 1069 & e  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01w-1067-08 & 00ee780b-1078-44fe-a310-96279cb9cd45 & 0098889119 & 08 & 0.50 & 29 & no & 06 & 1  & 1067 & e  & 40.00 & 22 & 80.00 & 2010 & 93.00  & repli-g (qiagen) dna & w  & tcga-05-4250-01a-01w & 1c28b799-8a6f-4726-90e9-fe44fcd9cbd9 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01x-1070-09 & 5166e96e-09d9-4b18-8f0c-37f31ad98220 & 0098888342 & 09 & 0.50 & 29 & no & 06 & 1  & 1070 & e  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4250-01a-01x-1068-08 & 116aff26-9af3-4ddd-9068-c07c52b24674 & 0098891126 & 08 & 0.50 & 29 & no & 06 & 1  & 1068 & e  & 40.00 & 22 & 80.00 & 2010 & 105.00 & repli-g x (qiagen) dna & x  & tcga-05-4250-01a-01x & 640ee12f-0490-412e-8112-1af4eab5e3ab & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01d-1104-05 & 3d6a1830-8b36-43cd-b9b6-8561e8675702 & NA & 0098885486 & 05 & 0.15 & 14 & no & 07 & 1 & 1104 & c & 4.01  & 22 & 26.70 & 2010 & NA & NA & NA & NA & tcga-05-4250-01a-01d-1105-08 & 41c4fe84-8beb-4a3a-920c-e74c7edd2182 & NA & 0098884622 & 08 & 0.15 & 14 & no & 07 & 1  & 1105 & c  & 1.50 & 22 & 10.00 & 2010 & NA & NA & tcga-05-4250-01a-01d-1106-09 & 9b0e468b-0bb7-4718-8a8c-baa9b6ef6325 & 0098846510 & 09 & 0.15 & 14 & no & 07 & 1  & 1106 & c  & 1.01 & 22 & 6.70  & 2010 & NA & NA & NA & NA & tcga-05-4250-01a-01d-1877-01 & 47ed8f8d-e76a-41e8-8bd7-580f3e0a589e & NA & 1053936002 & 01 & 0.15 & 11 & no & 07 & 1  & 1877 & c  & 1.00 & 22 & 6.67 & 2011 & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4250-01a-01d-1103-02 & a444b1e8-9fe9-492b-a361-93caab3681bc & NA & 0098886446 & 02 & 0.15 & 14 & no & 07 & 1 & 1103 & c & 2.00  & 22 & 13.30 & 2010 & 282.93 & dna                  & d & tcga-05-4250-01a-01d & 7d04fc99-4efe-407a-abf2-5211a5fb44c6 & 0.15 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-05-4250-01a-01 & 6c8028d8-1aba-4abc-b515-89cb73f54bbf & 20 & no & NA & 05 & 01 & NA & NA & tcga-05-4250-01a-01-ts1 & 5c50e0e0-1414-43f6-8fe1-13fbbcac5ec0 & tcga-05-4250-01a-01-ts1.5c50e0e0-1414-43f6-8fe1-13fbbcac5ec0.svs & no & NA & NA & NA & NA & NA & NA & 3  & NA & 10 & 5  & 82 & 85 & top & NA & tcga-05-4250-01a-01-bs1 & 7e972936-987c-4e66-b640-b889837337e9 & tcga-05-4250-01a-01-bs1.7e972936-987c-4e66-b640-b889837337e9.svs & no & NA & NA & NA & NA & NA & NA & 3  & NA & 5  & 5  & 87 & 85 & bottom & 126.00 & 2010 & NA & 3f069d62-c371-4f30-bdb4-c58093bbb1d7 & 20 & no & 2190 & 21 & NA & tcga-05-4250-01a-21-2190-20 & 5  & 3  & 2012 & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t58.73.0 & nationwide children's hospital & 1 & luad & ba34c6c3-4068-4665-8239-a5694644eab8 & 10 & false & tcga & 2015 & NA & NA & NA & 68 & l-upper & NA & tcga-05-4382 & 3434b91a-c05f-460f-a078-7b1bb6e7085d & l-upper & consented & germany & 0 & NA & NA & lung adenocarcinoma mixed subtype                  & no & yes, history of prior malignancy & NA & NA & NA                                   & NA & NA & primary & 22 & -24868 & NA  & 0 & 607  & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA              & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA        & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA & NA        & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & NA & NA & tcga-05-4382-f36329 & a8ccfebd-82aa-41e1-85e0-75e97dea2051 & 29 & NA & NA & NA & 607  & 334 & NA & scheduled follow-up submission & complete remission/response & NA & no & 10 & distant metastasis & NA & yes & NA & tumor free & complete remission/response & NA & NA & yes & no  & alive & 2012 & male   & lung adenocarcinoma mixed subtype                  & no & c34.1 & 8255/3 & c34.1 & yes & NA & NA & NA & NA & NA              & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 62 & 4382 & NA & tumor free & NA & NA & NA & NA & NA & yes & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & distant recurrence & tcga-05-4382-r36336 & 6ae3bfbc-dea0-46d9-a3b6-1ef2ce28aa24 & NA & 29 & 393 & 365 & complete response                & 10 & NA & NA & no & external & NA & NA & NA & gy & 2012 & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ib   & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t2 & NA   & NA & no & yes & 05 & current reformed smoker for < or = 15 years & lung & alive & 2010 & 2009 & NA   & NA & NA & NA & canonical & NA & consented & germany & NA & 453  & 0 & NA & no & NA & NA & lung & lung adenocarcinoma mixed subtype                  & NA & no & NA & left  & NA & other method (please specify) & ac0d08e1-31db-4faa-a5e3-116f852a63f3 & 0  & NA & NA & NA & NA & blood draw & NA & NA & NA & 2  & NA & NA & NA & NA & NA & NA & NA & adf04f24-d50f-473d-aa66-ad4778abad3b & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & lobectomy       & NA & NA & no  & NA & NA & NA & NA & NA & yes & lung & upper & NA                                   & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & cce6d71f-369e-467f-bd7e-03d20e97b7f3 & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4382-10a & adf04f24-d50f-473d-aa66-ad4778abad3b & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-10a-01w-1258-09 & ce51b743-1e22-4bc4-b8be-dc79a8f809ee & 0103961108 & 09 & 0.50 & 27 & no & 09 & 6  & 1258 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-10a-01w-1257-08 & ba2d50b4-f102-41ef-8522-36ec84542644 & 0100512884 & 08 & 0.50 & 27 & no & 09 & 6 & 1257 & b & 40.00 & 22 & 80.00 & 2010 & 94.00  & repli-g (qiagen) dna   & w & tcga-05-4382-10a-01w & 8346af3c-bedf-4eb8-b0a4-e17413df8143 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g   & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4382-10a-01x-1260-09 & c6a3435a-2d4c-466b-834d-52c6203e25fb & 103961684  & 09 & 0.50 & 27 & no & 09 & 6  & 1260 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4382-10a-01x-1259-08 & b4f4ea76-3a11-436d-a97d-d17abbc84a63 & 103959668  & 08 & 0.50 & 27 & no & 09 & 6  & 1259 & b  & 40.00 & 22 & 80.00 & 2010 & 93.00 & repli-g x (qiagen) dna & x  & tcga-05-4382-10a-01x & 30822617-2d1d-4817-97fd-9ff02ac79958 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & tcga-05-4382-10a-01d-1204-01 & ab8a5e79-daf0-49f8-a89d-2581e9ddddca & 0100501051 & 01 & 0.14 & 30 & no & 08 & 5 & 1204 & d & 0.94  & 22 & 6.70  & 2010 & NA & NA & NA & NA & tcga-05-4382-10a-01d-1265-08 & 20e316a4-d1e4-44d2-881c-527e600203f0 & 0103958516 & 08 & 0.14 & 27 & no & 09 & 6  & 1265 & b  & 1.40 & 22 & 10.00 & 2010 & NA & NA & tcga-05-4382-10a-01d-1266-09 & 468dafa9-64aa-4f9b-ad56-719d72264ec7 & 0103961972 & 09 & 0.14 & 27 & no & 09 & 6  & 1266 & b  & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-05-4382-10a-01d-1931-08 & 2eb560cb-d275-4080-b5c8-8b096112e964 & 0113627373 & 08 & 0.14 & 27 & no & 06 & 3  & 1931 & e  & 0.93 & 22 & 6.67 & 2012 & NA & NA & tcga-05-4382-10a-01d-a313-08 & e0330589-6221-4729-90e0-f472b9932452 & 0153879550 & 08 & 0.07 & 26 & no & 06 & 2  & a313 & a  & 0.94 & 23 & 13.40 & 2013 & NA & NA & tcga-05-4382-10a-01d-a46h-10 & d8e1f0e0-9915-48ef-8eb7-c30d2ead8e77 & 0167813352 & 10 & 0.07 & 22 & no & 04 & 1  & a46h & b  & 1.96 & 23 & 28.00 & 2015 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-10a-01d-1203-02 & 40acf290-13ff-44d8-8cc7-9fe9db183619 & 0100501243 & 02 & 0.14 & 30 & no & 08 & 5 & 1203 & d & 1.86  & 22 & 13.30 & 2010 & 14.23 & dna                  & d & tcga-05-4382-10a-01d & 303b777b-edb3-461d-996f-f03f4760bbf8 & 0.14 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & chemical lysis dna extraction & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4382-10a-01 & 6fa426fc-6f37-45c3-8e7a-43957a5ff4e0 & 19 & no & NA & 07 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a & cce6d71f-369e-467f-bd7e-03d20e97b7f3 & NA & NA & NA & NA & NA & NA & 76b49a4c-dbbb-48b0-b677-6d3037e5ce88 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.8 & no & 0.9 & NA & NA & NA & NA & tcga-05-4382.952c0f32-1472-49e1-8334-b0f1de4ac921.pdf & 952c0f32-1472-49e1-8334-b0f1de4ac921 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01r-1206-07 & e4177b01-6898-4bb7-b38d-0c09f85c5668 & NA & 0100500422 & 07 & 0.15 & 30 & no & 08 & 1 & 1206 & a & 4.01  & 22 & 26.70 & 2010 & 139.30 & rna                    & r & tcga-05-4382-01a-01r & 5b2440ca-c62d-4e0b-a9e9-aa233d74015e & 0.15 & NA  & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.30 & 8.40 & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01t-1207-13 & a6f483f7-324b-4588-ada2-660c08c9240f & 0100500607 & 13 & 0.10 & 30 & no & 08 & 1  & 1207 & a  & 3.00 & 22 & 30.00 & 2010 & 131.00 & total rna & t  & tcga-05-4382-01a-01t & 3d5dc3a8-47e3-4b6c-9ed0-e4ab4b6f026a & 0.10 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & 1.00 & 7.20 & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01w-1258-09 & fbf284d3-9741-4b2c-b4b4-a6e60c6d3255 & 0103961103 & 09 & 0.50 & 27 & no & 09 & 1  & 1258 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01w-1257-08 & 0e008646-6c49-47e5-a99d-4bfa7321e1df & 0100512379 & 08 & 0.50 & 27 & no & 09 & 1  & 1257 & b  & 40.00 & 22 & 80.00 & 2010 & 91.00  & repli-g (qiagen) dna & w  & tcga-05-4382-01a-01w & ac8c63d6-db2c-440e-ac16-556484765528 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01x-1260-09 & 33f9d9ac-30f4-4487-b1f6-a61418db8006 & 103961679  & 09 & 0.50 & 27 & no & 09 & 1  & 1260 & b  & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-05-4382-01a-01x-1259-08 & 549143e9-4f2f-4eed-be98-1bdbffb0b106 & 103959663  & 08 & 0.50 & 27 & no & 09 & 1  & 1259 & b  & 40.00 & 22 & 80.00 & 2010 & 72.00  & repli-g x (qiagen) dna & x  & tcga-05-4382-01a-01x & 07935f5a-4ad6-4d06-969d-5da15b054fe1 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01d-1204-01 & 5eb7654c-6209-4817-96eb-b7b0b40d6170 & NA & 0100500007 & 01 & 0.14 & 30 & no & 08 & 1 & 1204 & a & 0.94  & 22 & 6.70  & 2010 & NA & NA & NA & NA & tcga-05-4382-01a-01d-1205-05 & 4596e2e1-bc58-45be-9f07-5b365c96b593 & NA & 0100501478 & 05 & 0.14 & 30 & no & 08 & 1  & 1205 & a  & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-05-4382-01a-01d-1265-08 & 005b918d-e4a9-4971-9588-656a35c33dec & 0103958511 & 08 & 0.14 & 27 & no & 09 & 1  & 1265 & b  & 1.40 & 22 & 10.00 & 2010 & NA & NA & NA & NA & tcga-05-4382-01a-01d-1266-09 & 5881793b-f422-468e-8155-03ee9cf5772a & NA & 0103961967 & 09 & 0.14 & 27 & no & 09 & 1  & 1266 & b  & 0.94 & 22 & 6.70 & 2010 & NA & NA & NA & NA & tcga-05-4382-01a-01d-1931-08 & 43b8f335-bb32-45de-985a-85508048b237 & NA & 0113627423 & 08 & 0.14 & 27 & no & 06 & 1  & 1931 & a  & 0.93 & 22 & 6.67 & 2012 & NA & NA & NA & NA & tcga-05-4382-01a-01d-a313-08 & b3face4c-3fd1-46e2-96c4-069e5b3163fd & NA & 0153879551 & 08 & 0.07 & 26 & no & 06 & 1  & a313 & a  & 0.94 & 23 & 13.40 & 2013 & NA & NA & NA & NA & tcga-05-4382-01a-01d-a46h-10 & 58152fdd-7c80-4d93-9c79-ca078a9504fc & NA & 0167813375 & 10 & 0.07 & 22 & no & 04 & 1  & a46h & a  & 1.96 & 23 & 28.00 & 2015 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4382-01a-01d-1203-02 & ee1d241a-b371-433a-9af7-f1c9074c155f & NA & 0100501286 & 02 & 0.14 & 30 & no & 08 & 1 & 1203 & a & 1.86  & 22 & 13.30 & 2010 & 445.53 & dna                  & d & tcga-05-4382-01a-01d & c71ee997-1fe1-4fa8-a1ed-a5a50373be08 & 0.14 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-05-4382-01a-01 & d50e3b04-dd7f-4a79-b7e3-2590cef6e0ad & 02 & no & NA & 07 & 01 & NA & NA & tcga-05-4382-01a-01-ts1 & 8db4f123-fdc3-419c-a59d-fca2ca83a597 & tcga-05-4382-01a-01-ts1.8db4f123-fdc3-419c-a59d-fca2ca83a597.svs & no & NA & NA & NA & NA & NA & NA & 20 & NA & 0  & 10 & NA & 70 & top & NA & tcga-05-4382-01a-01-bs1 & ac519a18-8590-4fab-8af6-bb43fbe65310 & tcga-05-4382-01a-01-bs1.ac519a18-8590-4fab-8af6-bb43fbe65310.svs & no & NA & NA & NA & NA & NA & NA & 20 & NA & 0  & 10 & NA & 70 & bottom & 122.00 & 2010 & NA & NA                                   & NA & NA & NA   & NA & NA & NA                          & NA & NA & NA   & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t119.72.0 & nationwide children's hospital & 1 & luad & 092cfc6d-b91f-4787-8d7b-49a936d848b4 & 10 & false & tcga & 2015 & NA & NA & NA & 66 & r-lower & NA & tcga-05-4384 & 9a50e7e4-831d-489f-87d2-979e987561cc & r-lower & consented & germany & 0 & NA & NA & lung adenocarcinoma mixed subtype                  & no & yes, history of prior malignancy & NA & NA & inferior pulmonary lobe right s vi   & NA & NA & primary & 22 & -24411 & NA  & 0 & 426  & NA & lung adenocarcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-d36332 & 19860fed-a03b-407a-9081-f0cd43c3bd54 & NA & 5  & 153 & 61 & vinorelbine & clinical progressive disease & 12 & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & no & 2012 & tcga-05-4384-d36333 & 8009725b-0627-43cb-b0eb-37b12ccb3848 & NA & 5  & 426 & 395 & cisplatin & clinical progressive disease & 12 & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & no & 2012 & tcga-05-4384-d36334 & 5b01a1b0-5946-4ecc-a816-e6dc557cf563 & NA & 5  & 426 & 245 & zoledronic acid & clinical progressive disease & 12 & NA & NA & NA & NA & NA & NA & NA & no & ancillary & NA & NA & NA & no & 2012 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-d36331 & 281e8624-371d-4473-ba5e-e41e0c21d551 & NA & 5  & 153 & 61 & cisplatin & clinical progressive disease & 12 & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & no & 2012 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & NA & NA & tcga-05-4384-f36330 & f4555a19-abbe-4673-9871-083b5c01710a & 29 & NA & NA & NA & 426  & 183 & NA & scheduled follow-up submission & progressive disease         & NA & no & 10 & distant metastasis & NA & yes & NA & with tumor & progressive disease         & NA & NA & yes & yes & alive & 2012 & male   & lung adenocarcinoma mixed subtype                  & no & c34.3 & 8255/3 & c34.3 & yes & NA & NA & NA & NA & NA              & 7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 20 & 4384 & NA & NA         & NA & NA & NA & NA & NA & yes & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & distant recurrence & tcga-05-4384-r36335 & e131951d-3ade-4c06-9406-088b151b75a3 & NA & 29 & 214 & 183 & radiographic progressive disease & 10 & NA & NA & no & external & NA & NA & NA & gy & 2012 & rx & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iiia & NA & NA & NA & 6th & NA & NA & NA & m0 & n2 & t2 & 1987 & NA & no & yes & 05 & current reformed smoker for > 15 years      & lung & alive & 2010 & 2009 & 1963 & NA & NA & NA & canonical & NA & consented & germany & NA & 333  & 0 & NA & no & NA & NA & lung & lung adenocarcinoma mixed subtype                  & NA & no & NA & right & NA & other method (please specify) & NA                                   & NA & NA & NA & NA & NA & NA         & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 397704e5-f64f-4dd7-b323-b8e5b4e8b25c & 0 & NA & NA & NA & NA & blood draw & NA & NA & NA & 2 & tumor resection & NA & NA & yes & NA & NA & NA & NA & NA & yes & lung & lower & inferior pulmonary lobe right s vi   & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & e4416303-50b0-4316-bfee-030c7b29fac6 & NA & 20 & 70 & NA & 100 & 1 & cryovial & 0 & NA & NA & tcga-05-4384-10a & 397704e5-f64f-4dd7-b323-b8e5b4e8b25c & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-10a-01w-1804-08 & 628a6674-dd93-4c3a-aba8-c9d54d9589c7 & 0107042070 & 08 & 0.50 & 08 & no & 06 & 9 & 1804 & g & 40.00 & 22 & 80.00 & 2011 & 234.00 & repli-g (qiagen) dna   & w & tcga-05-4384-10a-01w & 267cb8de-836f-46c4-b45c-ded9194ca153 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g   & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA    & NA                     & NA & NA                   & NA                                   & NA   & NA  & NA & NA                                               & NA & NA        & NA & NA & NA & NA & NA      & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & tcga-05-4384-10a-01d-1752-01 & 95faa177-2bb5-43bb-b0fc-7f74fc57f7df & 1053958783 & 01 & 0.14 & 11 & no & 05 & 8 & 1752 & f & 0.93  & 22 & 6.67  & 2011 & NA & NA & NA & NA & tcga-05-4384-10a-01d-1753-08 & 2011ef98-2f99-44e0-a561-e14eb6f2d1f8 & 1053958296 & 08 & 0.05 & 11 & no & 05 & 8  & 1753 & f  & 3.00 & 22 & 60.00 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-10a-01d-1751-02 & 5e720136-5e3b-4c5e-a427-7ad349ae7c77 & 1053958104 & 02 & 0.14 & 11 & no & 05 & 8 & 1751 & f & 1.86  & 22 & 13.30 & 2011 & 74.70 & dna                  & d & tcga-05-4384-10a-01d & a3835cb4-a52c-4e3f-aff3-7237b14a01d2 & 0.14 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & chemical lysis dna extraction & NA & NA & NA & NA & uv spec & NA & NA & tcga-05-4384-10a-01 & 10421a0d-0998-4524-8ce1-2d7b18bf05fb & 04 & no & NA & 10 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2010 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a & e4416303-50b0-4316-bfee-030c7b29fac6 & NA & NA & NA & NA & NA & NA & ca68bf29-bbe3-4c8e-b48b-554431a9ee13 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.8 & no & 0.8 & NA & NA & NA & NA & tcga-05-4384.b70e95c5-2e6c-4409-8bbd-80e03e302b4c.pdf & b70e95c5-2e6c-4409-8bbd-80e03e302b4c & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a-01r-1755-07 & 7d6cf896-b04a-431a-a192-aaf540eeaf77 & NA & 1053958815 & 07 & 0.15 & 11 & no & 05 & 1 & 1755 & a & 2.00  & 22 & 13.30 & 2011 & 298.70 & rna                    & r & tcga-05-4384-01a-01r & 07fc548a-e067-45da-bce4-8bcb9d418cc4 & 0.15 & NA  & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.20 & 7.80 & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a-01t-1754-13 & 7a59e58e-0692-4d25-98a4-bf8340d59ae0 & 1053958424 & 13 & 0.10 & 11 & no & 05 & 1  & 1754 & a  & 3.00 & 22 & 30.00 & 2011 & 276.00 & total rna & t  & tcga-05-4384-01a-01t & 36831664-5b4c-40e3-b581-366c8a3d50d2 & 0.10 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & 1.40 & 7.00 & uv spec & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a-01w-1804-08 & 405b4870-515e-4431-b5df-cc2709f06e37 & 0107042127 & 08 & 0.50 & 08 & no & 06 & 1  & 1804 & b  & 40.00 & 22 & 80.00 & 2011 & 179.00 & repli-g (qiagen) dna & w  & tcga-05-4384-01a-01w & ccf0e45c-8945-4c1e-9d62-6fe037a04c28 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA    & NA & NA    & NA   & NA     & NA                     & NA & NA                   & NA                                   & NA   & NA  & NA & NA                                               & NA & NA        & NA & NA & NA      & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a-01d-1752-01 & 7d1133f5-d4d4-41e4-9903-9c42accf5889 & NA & 1053945504 & 01 & 0.15 & 11 & no & 05 & 1 & 1752 & a & 1.00  & 22 & 6.67  & 2011 & NA & NA & NA & NA & tcga-05-4384-01a-01d-1753-08 & 4c71b66b-813f-472b-b866-b34b5b9199e7 & NA & 1053958232 & 08 & 0.05 & 11 & no & 05 & 1  & 1753 & a  & 3.00 & 22 & 60.00 & 2011 & NA & NA & tcga-05-4384-01a-01d-1756-05 & 0bbdd118-120a-4820-a62e-4e11217429ee & 1053958328 & 05 & 0.15 & 11 & no & 05 & 1  & 1756 & a  & 2.00 & 22 & 13.30 & 2011 & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA   & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA                           & NA                                   & NA & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-05-4384-01a-01d-1751-02 & 5253b1a2-8c57-414a-8e19-6e8390a6c18d & NA & 1053958040 & 02 & 0.15 & 11 & no & 05 & 1 & 1751 & a & 2.00  & 22 & 13.30 & 2011 & 486.40 & dna                  & d & tcga-05-4384-01a-01d & 91a7db58-e13a-4055-8fc5-42076836c9b6 & 0.15 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-05-4384-01a-01 & 70a98141-6c1f-42d7-b2d9-48b794ee6b08 & 02 & no & NA & 07 & 01 & NA & NA & tcga-05-4384-01a-01-ts1 & e4d430ce-bc0e-4446-8a7a-bb831e1285ac & tcga-05-4384-01a-01-ts1.e4d430ce-bc0e-4446-8a7a-bb831e1285ac.svs & no & NA & NA & NA & NA & 7  & NA & 0  & NA & 15 & 10 & 75 & 70 & top & NA & tcga-05-4384-01a-01-bs1 & dae26cb5-1554-4ff8-997d-2f1c70101051 & tcga-05-4384-01a-01-bs1.dae26cb5-1554-4ff8-997d-2f1c70101051.svs & no & NA & NA & NA & NA & 5  & NA & 0  & NA & 20 & 10 & 70 & 70 & bottom & 113.00 & 2010 & NA & 5e735c28-5f2b-47ec-98fb-2f57cc8a5d6a & 20 & no & 2190 & 21 & NA & tcga-05-4384-01a-21-2190-20 & 5  & 3  & 2012 & NA & primary tumor & 01 & 0.4 & NA & NA & NA & NA & NA & a\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 3046\n",
       "\n",
       "| <!--/--> | admin.bcr &lt;chr&gt; | admin.day_of_dcc_upload &lt;chr&gt; | admin.disease_code &lt;chr&gt; | admin.file_uuid &lt;chr&gt; | admin.month_of_dcc_upload &lt;chr&gt; | admin.patient_withdrawal.withdrawn &lt;chr&gt; | admin.project_code &lt;chr&gt; | admin.year_of_dcc_upload &lt;chr&gt; | patient.additional_studies &lt;chr&gt; | patient.additional_studies.additional_study.disease_code &lt;chr&gt; | ⋯ ⋯ | patient.samples.sample.preservation_method &lt;chr&gt; | patient.samples.sample.sample_type &lt;chr&gt; | patient.samples.sample.sample_type_id &lt;chr&gt; | patient.samples.sample.shortest_dimension &lt;chr&gt; | patient.samples.sample.time_between_clamping_and_freezing &lt;chr&gt; | patient.samples.sample.time_between_excision_and_freezing &lt;chr&gt; | patient.samples.sample.tissue_type &lt;chr&gt; | patient.samples.sample.tumor_descriptor &lt;chr&gt; | patient.samples.sample.tumor_pathology &lt;chr&gt; | patient.samples.sample.vial_number &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| 52.70.0 | nationwide children's hospital | 1 | luad | 071e3c36-bd5c-49b2-b3d3-0eb4e781810f | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 52.70.0.1 | nationwide children's hospital | 1 | luad | 74423066-665a-43f2-a801-fb4f83ee15e9 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.5 | NA | NA | NA | NA | NA | a |\n",
       "| 52.70.0.2 | nationwide children's hospital | 1 | luad | b06fd27e-a052-459f-be7b-c6e336a6629d | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.4 | NA | NA | NA | NA | NA | a |\n",
       "| 52.70.0.3 | nationwide children's hospital | 1 | luad | 4bdbf19f-48c2-4565-a5b6-9ada2ef93e74 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 58.73.0 | nationwide children's hospital | 1 | luad | ba34c6c3-4068-4665-8239-a5694644eab8 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 119.72.0 | nationwide children's hospital | 1 | luad | 092cfc6d-b91f-4787-8d7b-49a936d848b4 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | 0.4 | NA | NA | NA | NA | NA | a |\n",
       "\n"
      ],
      "text/plain": [
       "          admin.bcr                      admin.day_of_dcc_upload\n",
       "52.70.0   nationwide children's hospital 1                      \n",
       "52.70.0.1 nationwide children's hospital 1                      \n",
       "52.70.0.2 nationwide children's hospital 1                      \n",
       "52.70.0.3 nationwide children's hospital 1                      \n",
       "58.73.0   nationwide children's hospital 1                      \n",
       "119.72.0  nationwide children's hospital 1                      \n",
       "          admin.disease_code admin.file_uuid                     \n",
       "52.70.0   luad               071e3c36-bd5c-49b2-b3d3-0eb4e781810f\n",
       "52.70.0.1 luad               74423066-665a-43f2-a801-fb4f83ee15e9\n",
       "52.70.0.2 luad               b06fd27e-a052-459f-be7b-c6e336a6629d\n",
       "52.70.0.3 luad               4bdbf19f-48c2-4565-a5b6-9ada2ef93e74\n",
       "58.73.0   luad               ba34c6c3-4068-4665-8239-a5694644eab8\n",
       "119.72.0  luad               092cfc6d-b91f-4787-8d7b-49a936d848b4\n",
       "          admin.month_of_dcc_upload admin.patient_withdrawal.withdrawn\n",
       "52.70.0   10                        false                             \n",
       "52.70.0.1 10                        false                             \n",
       "52.70.0.2 10                        false                             \n",
       "52.70.0.3 10                        false                             \n",
       "58.73.0   10                        false                             \n",
       "119.72.0  10                        false                             \n",
       "          admin.project_code admin.year_of_dcc_upload\n",
       "52.70.0   tcga               2015                    \n",
       "52.70.0.1 tcga               2015                    \n",
       "52.70.0.2 tcga               2015                    \n",
       "52.70.0.3 tcga               2015                    \n",
       "58.73.0   tcga               2015                    \n",
       "119.72.0  tcga               2015                    \n",
       "          patient.additional_studies\n",
       "52.70.0   NA                        \n",
       "52.70.0.1 NA                        \n",
       "52.70.0.2 NA                        \n",
       "52.70.0.3 NA                        \n",
       "58.73.0   NA                        \n",
       "119.72.0  NA                        \n",
       "          patient.additional_studies.additional_study.disease_code ⋯\n",
       "52.70.0   NA                                                       ⋯\n",
       "52.70.0.1 NA                                                       ⋯\n",
       "52.70.0.2 NA                                                       ⋯\n",
       "52.70.0.3 NA                                                       ⋯\n",
       "58.73.0   NA                                                       ⋯\n",
       "119.72.0  NA                                                       ⋯\n",
       "          patient.samples.sample.preservation_method\n",
       "52.70.0   NA                                        \n",
       "52.70.0.1 NA                                        \n",
       "52.70.0.2 NA                                        \n",
       "52.70.0.3 NA                                        \n",
       "58.73.0   NA                                        \n",
       "119.72.0  NA                                        \n",
       "          patient.samples.sample.sample_type\n",
       "52.70.0   primary tumor                     \n",
       "52.70.0.1 primary tumor                     \n",
       "52.70.0.2 primary tumor                     \n",
       "52.70.0.3 primary tumor                     \n",
       "58.73.0   primary tumor                     \n",
       "119.72.0  primary tumor                     \n",
       "          patient.samples.sample.sample_type_id\n",
       "52.70.0   01                                   \n",
       "52.70.0.1 01                                   \n",
       "52.70.0.2 01                                   \n",
       "52.70.0.3 01                                   \n",
       "58.73.0   01                                   \n",
       "119.72.0  01                                   \n",
       "          patient.samples.sample.shortest_dimension\n",
       "52.70.0   0.3                                      \n",
       "52.70.0.1 0.5                                      \n",
       "52.70.0.2 0.4                                      \n",
       "52.70.0.3 0.3                                      \n",
       "58.73.0   0.3                                      \n",
       "119.72.0  0.4                                      \n",
       "          patient.samples.sample.time_between_clamping_and_freezing\n",
       "52.70.0   NA                                                       \n",
       "52.70.0.1 NA                                                       \n",
       "52.70.0.2 NA                                                       \n",
       "52.70.0.3 NA                                                       \n",
       "58.73.0   NA                                                       \n",
       "119.72.0  NA                                                       \n",
       "          patient.samples.sample.time_between_excision_and_freezing\n",
       "52.70.0   NA                                                       \n",
       "52.70.0.1 NA                                                       \n",
       "52.70.0.2 NA                                                       \n",
       "52.70.0.3 NA                                                       \n",
       "58.73.0   NA                                                       \n",
       "119.72.0  NA                                                       \n",
       "          patient.samples.sample.tissue_type\n",
       "52.70.0   NA                                \n",
       "52.70.0.1 NA                                \n",
       "52.70.0.2 NA                                \n",
       "52.70.0.3 NA                                \n",
       "58.73.0   NA                                \n",
       "119.72.0  NA                                \n",
       "          patient.samples.sample.tumor_descriptor\n",
       "52.70.0   NA                                     \n",
       "52.70.0.1 NA                                     \n",
       "52.70.0.2 NA                                     \n",
       "52.70.0.3 NA                                     \n",
       "58.73.0   NA                                     \n",
       "119.72.0  NA                                     \n",
       "          patient.samples.sample.tumor_pathology\n",
       "52.70.0   NA                                    \n",
       "52.70.0.1 NA                                    \n",
       "52.70.0.2 NA                                    \n",
       "52.70.0.3 NA                                    \n",
       "58.73.0   NA                                    \n",
       "119.72.0  NA                                    \n",
       "          patient.samples.sample.vial_number\n",
       "52.70.0   a                                 \n",
       "52.70.0.1 a                                 \n",
       "52.70.0.2 a                                 \n",
       "52.70.0.3 a                                 \n",
       "58.73.0   a                                 \n",
       "119.72.0  a                                 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'luad'\n",
    "df = clinical_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUSC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>504</li>\n",
       "\t<li>2698</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 504\n",
       "\\item 2698\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 504\n",
       "2. 2698\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]  504 2698"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 2698</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>admin.bcr</th><th scope=col>admin.day_of_dcc_upload</th><th scope=col>admin.disease_code</th><th scope=col>admin.file_uuid</th><th scope=col>admin.month_of_dcc_upload</th><th scope=col>admin.patient_withdrawal.withdrawn</th><th scope=col>admin.project_code</th><th scope=col>admin.year_of_dcc_upload</th><th scope=col>patient.additional_studies</th><th scope=col>patient.age_at_initial_pathologic_diagnosis</th><th scope=col>⋯</th><th scope=col>patient.samples.sample.preservation_method</th><th scope=col>patient.samples.sample.sample_type</th><th scope=col>patient.samples.sample.sample_type_id</th><th scope=col>patient.samples.sample.shortest_dimension</th><th scope=col>patient.samples.sample.time_between_clamping_and_freezing</th><th scope=col>patient.samples.sample.time_between_excision_and_freezing</th><th scope=col>patient.samples.sample.tissue_type</th><th scope=col>patient.samples.sample.tumor_descriptor</th><th scope=col>patient.samples.sample.tumor_pathology</th><th scope=col>patient.samples.sample.vial_number</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>⋯</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>39.63.0</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>b34e8d44-9f2d-4ab3-a189-ecb5b77af723</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>67</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>39.63.0.1</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>2fc3e9ed-7219-4d78-8964-a5c9de65b8da</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>72</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.2</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>39.63.0.2</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>1af63f26-0d2a-463b-ba33-7b60ab118b50</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>77</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>39.63.0.3</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>730b1e78-6f9a-45a6-a246-e473ae0116e3</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>74</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.2</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>39.63.0.4</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>293cc082-7d2c-416a-be35-062bb8ee1b87</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>81</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.3</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>39.63.0.5</th><td>nationwide children's hospital</td><td>1</td><td>lusc</td><td>5eee6807-0ee7-4ad7-84a4-f36552a70695</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>63</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>0.5</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 2698\n",
       "\\begin{tabular}{r|llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll}\n",
       "  & admin.bcr & admin.day\\_of\\_dcc\\_upload & admin.disease\\_code & admin.file\\_uuid & admin.month\\_of\\_dcc\\_upload & admin.patient\\_withdrawal.withdrawn & admin.project\\_code & admin.year\\_of\\_dcc\\_upload & patient.additional\\_studies & patient.age\\_at\\_initial\\_pathologic\\_diagnosis & patient.anatomic\\_neoplasm\\_subdivision & patient.anatomic\\_neoplasm\\_subdivision\\_other & patient.bcr\\_patient\\_barcode & patient.bcr\\_patient\\_uuid & patient.clinical\\_cqcf.anatomic\\_neoplasm\\_subdivision & patient.clinical\\_cqcf.consent\\_or\\_death\\_status & patient.clinical\\_cqcf.country & patient.clinical\\_cqcf.days\\_to\\_consent & patient.clinical\\_cqcf.days\\_to\\_death & patient.clinical\\_cqcf.diagnosis\\_subtype & patient.clinical\\_cqcf.histological\\_type & patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.clinical\\_cqcf.history\\_of\\_prior\\_malignancy & patient.clinical\\_cqcf.normal\\_tissue\\_anatomic\\_site & patient.clinical\\_cqcf.normal\\_tissue\\_proximity & patient.clinical\\_cqcf.other\\_anatomic\\_site & patient.clinical\\_cqcf.other\\_anatomic\\_site\\_normal\\_tissue & patient.clinical\\_cqcf.prior\\_dx & patient.clinical\\_cqcf.tumor\\_type & patient.day\\_of\\_form\\_completion & patient.days\\_to\\_birth & patient.days\\_to\\_death & patient.days\\_to\\_initial\\_pathologic\\_diagnosis & patient.days\\_to\\_last\\_followup & patient.days\\_to\\_last\\_known\\_alive & patient.diagnosis & patient.dlco\\_predictive\\_percent & patient.drugs & patient.drugs.drug-10.bcr\\_drug\\_barcode & patient.drugs.drug-10.bcr\\_drug\\_uuid & patient.drugs.drug-10.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-10.day\\_of\\_form\\_completion & patient.drugs.drug-10.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-10.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-10.drug\\_name & patient.drugs.drug-10.measure\\_of\\_response & patient.drugs.drug-10.month\\_of\\_form\\_completion & patient.drugs.drug-10.number\\_cycles & patient.drugs.drug-10.prescribed\\_dose & patient.drugs.drug-10.prescribed\\_dose\\_units & patient.drugs.drug-10.regimen\\_indication & patient.drugs.drug-10.regimen\\_indication\\_notes & patient.drugs.drug-10.regimen\\_number & patient.drugs.drug-10.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-10.therapy\\_ongoing & patient.drugs.drug-10.therapy\\_types.therapy\\_type & patient.drugs.drug-10.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-10.total\\_dose & patient.drugs.drug-10.total\\_dose\\_units & patient.drugs.drug-10.tx\\_on\\_clinical\\_trial & patient.drugs.drug-10.year\\_of\\_form\\_completion & patient.drugs.drug-11.bcr\\_drug\\_barcode & patient.drugs.drug-11.bcr\\_drug\\_uuid & patient.drugs.drug-11.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-11.day\\_of\\_form\\_completion & patient.drugs.drug-11.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-11.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-11.drug\\_name & patient.drugs.drug-11.measure\\_of\\_response & patient.drugs.drug-11.month\\_of\\_form\\_completion & patient.drugs.drug-11.number\\_cycles & patient.drugs.drug-11.prescribed\\_dose & patient.drugs.drug-11.prescribed\\_dose\\_units & patient.drugs.drug-11.regimen\\_indication & patient.drugs.drug-11.regimen\\_indication\\_notes & patient.drugs.drug-11.regimen\\_number & patient.drugs.drug-11.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-11.therapy\\_ongoing & patient.drugs.drug-11.therapy\\_types.therapy\\_type & patient.drugs.drug-11.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-11.total\\_dose & patient.drugs.drug-11.total\\_dose\\_units & patient.drugs.drug-11.tx\\_on\\_clinical\\_trial & patient.drugs.drug-11.year\\_of\\_form\\_completion & patient.drugs.drug-12.bcr\\_drug\\_barcode & patient.drugs.drug-12.bcr\\_drug\\_uuid & patient.drugs.drug-12.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-12.day\\_of\\_form\\_completion & patient.drugs.drug-12.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-12.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-12.drug\\_name & patient.drugs.drug-12.measure\\_of\\_response & patient.drugs.drug-12.month\\_of\\_form\\_completion & patient.drugs.drug-12.number\\_cycles & patient.drugs.drug-12.prescribed\\_dose & patient.drugs.drug-12.prescribed\\_dose\\_units & patient.drugs.drug-12.regimen\\_indication & patient.drugs.drug-12.regimen\\_indication\\_notes & patient.drugs.drug-12.regimen\\_number & patient.drugs.drug-12.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-12.therapy\\_ongoing & patient.drugs.drug-12.therapy\\_types.therapy\\_type & patient.drugs.drug-12.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-12.total\\_dose & patient.drugs.drug-12.total\\_dose\\_units & patient.drugs.drug-12.tx\\_on\\_clinical\\_trial & patient.drugs.drug-12.year\\_of\\_form\\_completion & patient.drugs.drug-2.bcr\\_drug\\_barcode & patient.drugs.drug-2.bcr\\_drug\\_uuid & patient.drugs.drug-2.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-2.day\\_of\\_form\\_completion & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-2.drug\\_name & patient.drugs.drug-2.measure\\_of\\_response & patient.drugs.drug-2.month\\_of\\_form\\_completion & patient.drugs.drug-2.number\\_cycles & patient.drugs.drug-2.prescribed\\_dose & patient.drugs.drug-2.prescribed\\_dose\\_units & patient.drugs.drug-2.regimen\\_indication & patient.drugs.drug-2.regimen\\_indication\\_notes & patient.drugs.drug-2.regimen\\_number & patient.drugs.drug-2.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-2.therapy\\_ongoing & patient.drugs.drug-2.therapy\\_types.therapy\\_type & patient.drugs.drug-2.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-2.total\\_dose & patient.drugs.drug-2.total\\_dose\\_units & patient.drugs.drug-2.tx\\_on\\_clinical\\_trial & patient.drugs.drug-2.year\\_of\\_form\\_completion & patient.drugs.drug-3.bcr\\_drug\\_barcode & patient.drugs.drug-3.bcr\\_drug\\_uuid & patient.drugs.drug-3.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-3.day\\_of\\_form\\_completion & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-3.drug\\_name & patient.drugs.drug-3.measure\\_of\\_response & patient.drugs.drug-3.month\\_of\\_form\\_completion & patient.drugs.drug-3.number\\_cycles & patient.drugs.drug-3.prescribed\\_dose & patient.drugs.drug-3.prescribed\\_dose\\_units & patient.drugs.drug-3.regimen\\_indication & patient.drugs.drug-3.regimen\\_indication\\_notes & patient.drugs.drug-3.regimen\\_number & patient.drugs.drug-3.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-3.therapy\\_ongoing & patient.drugs.drug-3.therapy\\_types.therapy\\_type & patient.drugs.drug-3.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-3.total\\_dose & patient.drugs.drug-3.total\\_dose\\_units & patient.drugs.drug-3.tx\\_on\\_clinical\\_trial & patient.drugs.drug-3.year\\_of\\_form\\_completion & patient.drugs.drug-4.bcr\\_drug\\_barcode & patient.drugs.drug-4.bcr\\_drug\\_uuid & patient.drugs.drug-4.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-4.day\\_of\\_form\\_completion & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-4.drug\\_name & patient.drugs.drug-4.measure\\_of\\_response & patient.drugs.drug-4.month\\_of\\_form\\_completion & patient.drugs.drug-4.number\\_cycles & patient.drugs.drug-4.prescribed\\_dose & patient.drugs.drug-4.prescribed\\_dose\\_units & patient.drugs.drug-4.regimen\\_indication & patient.drugs.drug-4.regimen\\_indication\\_notes & patient.drugs.drug-4.regimen\\_number & patient.drugs.drug-4.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-4.therapy\\_ongoing & patient.drugs.drug-4.therapy\\_types.therapy\\_type & patient.drugs.drug-4.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-4.total\\_dose & patient.drugs.drug-4.total\\_dose\\_units & patient.drugs.drug-4.tx\\_on\\_clinical\\_trial & patient.drugs.drug-4.year\\_of\\_form\\_completion & patient.drugs.drug-5.bcr\\_drug\\_barcode & patient.drugs.drug-5.bcr\\_drug\\_uuid & patient.drugs.drug-5.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-5.day\\_of\\_form\\_completion & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-5.drug\\_name & patient.drugs.drug-5.measure\\_of\\_response & patient.drugs.drug-5.month\\_of\\_form\\_completion & patient.drugs.drug-5.number\\_cycles & patient.drugs.drug-5.prescribed\\_dose & patient.drugs.drug-5.prescribed\\_dose\\_units & patient.drugs.drug-5.regimen\\_indication & patient.drugs.drug-5.regimen\\_indication\\_notes & patient.drugs.drug-5.regimen\\_number & patient.drugs.drug-5.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-5.therapy\\_ongoing & patient.drugs.drug-5.therapy\\_types.therapy\\_type & patient.drugs.drug-5.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-5.total\\_dose & patient.drugs.drug-5.total\\_dose\\_units & patient.drugs.drug-5.tx\\_on\\_clinical\\_trial & patient.drugs.drug-5.year\\_of\\_form\\_completion & patient.drugs.drug-6.bcr\\_drug\\_barcode & patient.drugs.drug-6.bcr\\_drug\\_uuid & patient.drugs.drug-6.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-6.day\\_of\\_form\\_completion & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-6.drug\\_name & patient.drugs.drug-6.measure\\_of\\_response & patient.drugs.drug-6.month\\_of\\_form\\_completion & patient.drugs.drug-6.number\\_cycles & patient.drugs.drug-6.prescribed\\_dose & patient.drugs.drug-6.prescribed\\_dose\\_units & patient.drugs.drug-6.regimen\\_indication & patient.drugs.drug-6.regimen\\_indication\\_notes & patient.drugs.drug-6.regimen\\_number & patient.drugs.drug-6.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-6.therapy\\_ongoing & patient.drugs.drug-6.therapy\\_types.therapy\\_type & patient.drugs.drug-6.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-6.total\\_dose & patient.drugs.drug-6.total\\_dose\\_units & patient.drugs.drug-6.tx\\_on\\_clinical\\_trial & patient.drugs.drug-6.year\\_of\\_form\\_completion & patient.drugs.drug-7.bcr\\_drug\\_barcode & patient.drugs.drug-7.bcr\\_drug\\_uuid & patient.drugs.drug-7.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-7.day\\_of\\_form\\_completion & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-7.drug\\_name & patient.drugs.drug-7.measure\\_of\\_response & patient.drugs.drug-7.month\\_of\\_form\\_completion & patient.drugs.drug-7.number\\_cycles & patient.drugs.drug-7.prescribed\\_dose & patient.drugs.drug-7.prescribed\\_dose\\_units & patient.drugs.drug-7.regimen\\_indication & patient.drugs.drug-7.regimen\\_indication\\_notes & patient.drugs.drug-7.regimen\\_number & patient.drugs.drug-7.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-7.therapy\\_ongoing & patient.drugs.drug-7.therapy\\_types.therapy\\_type & patient.drugs.drug-7.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-7.total\\_dose & patient.drugs.drug-7.total\\_dose\\_units & patient.drugs.drug-7.tx\\_on\\_clinical\\_trial & patient.drugs.drug-7.year\\_of\\_form\\_completion & patient.drugs.drug-8.bcr\\_drug\\_barcode & patient.drugs.drug-8.bcr\\_drug\\_uuid & patient.drugs.drug-8.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-8.day\\_of\\_form\\_completion & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-8.drug\\_name & patient.drugs.drug-8.measure\\_of\\_response & patient.drugs.drug-8.month\\_of\\_form\\_completion & patient.drugs.drug-8.number\\_cycles & patient.drugs.drug-8.prescribed\\_dose & patient.drugs.drug-8.prescribed\\_dose\\_units & patient.drugs.drug-8.regimen\\_indication & patient.drugs.drug-8.regimen\\_indication\\_notes & patient.drugs.drug-8.regimen\\_number & patient.drugs.drug-8.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-8.therapy\\_ongoing & patient.drugs.drug-8.therapy\\_types.therapy\\_type & patient.drugs.drug-8.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-8.total\\_dose & patient.drugs.drug-8.total\\_dose\\_units & patient.drugs.drug-8.tx\\_on\\_clinical\\_trial & patient.drugs.drug-8.year\\_of\\_form\\_completion & patient.drugs.drug-9.bcr\\_drug\\_barcode & patient.drugs.drug-9.bcr\\_drug\\_uuid & patient.drugs.drug-9.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-9.day\\_of\\_form\\_completion & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-9.drug\\_name & patient.drugs.drug-9.measure\\_of\\_response & patient.drugs.drug-9.month\\_of\\_form\\_completion & patient.drugs.drug-9.number\\_cycles & patient.drugs.drug-9.prescribed\\_dose & patient.drugs.drug-9.prescribed\\_dose\\_units & patient.drugs.drug-9.regimen\\_indication & patient.drugs.drug-9.regimen\\_indication\\_notes & patient.drugs.drug-9.regimen\\_number & patient.drugs.drug-9.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-9.therapy\\_ongoing & patient.drugs.drug-9.therapy\\_types.therapy\\_type & patient.drugs.drug-9.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-9.total\\_dose & patient.drugs.drug-9.total\\_dose\\_units & patient.drugs.drug-9.tx\\_on\\_clinical\\_trial & patient.drugs.drug-9.year\\_of\\_form\\_completion & patient.drugs.drug.bcr\\_drug\\_barcode & patient.drugs.drug.bcr\\_drug\\_uuid & patient.drugs.drug.clinical\\_trail\\_drug\\_classification & patient.drugs.drug.day\\_of\\_form\\_completion & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug.drug\\_name & patient.drugs.drug.measure\\_of\\_response & patient.drugs.drug.month\\_of\\_form\\_completion & patient.drugs.drug.number\\_cycles & patient.drugs.drug.prescribed\\_dose & patient.drugs.drug.prescribed\\_dose\\_units & patient.drugs.drug.regimen\\_indication & patient.drugs.drug.regimen\\_indication\\_notes & patient.drugs.drug.regimen\\_number & patient.drugs.drug.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug.therapy\\_ongoing & patient.drugs.drug.therapy\\_types.therapy\\_type & patient.drugs.drug.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug.total\\_dose & patient.drugs.drug.total\\_dose\\_units & patient.drugs.drug.tx\\_on\\_clinical\\_trial & patient.drugs.drug.year\\_of\\_form\\_completion & patient.eastern\\_cancer\\_oncology\\_group & patient.egfr\\_mutation\\_performed & patient.egfr\\_mutation\\_result & patient.eml4\\_alk\\_translocation\\_method & patient.eml4\\_alk\\_translocation\\_performed & patient.eml4\\_alk\\_translocation\\_result & patient.ethnicity & patient.follow\\_ups & patient.follow\\_ups.follow\\_up-2.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-2.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-2.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-2.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-2.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-2.days\\_to\\_death & patient.follow\\_ups.follow\\_up-2.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-2.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-2.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-2.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-2.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-2.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-2.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-2.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-2.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-2.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-2.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up-2.radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-2.vital\\_status & patient.follow\\_ups.follow\\_up-2.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-3.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-3.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-3.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-3.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-3.days\\_to\\_death & patient.follow\\_ups.follow\\_up-3.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-3.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-3.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-3.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-3.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-3.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-3.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-3.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-3.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-3.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up-3.radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-3.vital\\_status & patient.follow\\_ups.follow\\_up-3.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-4.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-4.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-4.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up-4.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up-4.days\\_to\\_death & patient.follow\\_ups.follow\\_up-4.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-4.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-4.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-4.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up-4.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-4.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-4.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-4.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-4.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up-4.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up-4.radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up-4.vital\\_status & patient.follow\\_ups.follow\\_up-4.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up.additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.follow\\_ups.follow\\_up.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.follow\\_ups.follow\\_up.days\\_to\\_death & patient.follow\\_ups.follow\\_up.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up.followup\\_treatment\\_success & patient.follow\\_ups.follow\\_up.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.follow\\_ups.follow\\_up.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up.primary\\_therapy\\_outcome\\_success & patient.follow\\_ups.follow\\_up.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.follow\\_ups.follow\\_up.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.follow\\_ups.follow\\_up.progression\\_determined\\_by\\_list.progression\\_determined\\_by-3 & patient.follow\\_ups.follow\\_up.radiation\\_therapy & patient.follow\\_ups.follow\\_up.targeted\\_molecular\\_therapy & patient.follow\\_ups.follow\\_up.vital\\_status & patient.follow\\_ups.follow\\_up.year\\_of\\_form\\_completion & patient.gender & patient.histological\\_type & patient.history\\_of\\_neoadjuvant\\_treatment & patient.icd\\_10 & patient.icd\\_o\\_3\\_histology & patient.icd\\_o\\_3\\_site & patient.informed\\_consent\\_verified & patient.karnofsky\\_performance\\_score & patient.kras\\_gene\\_analysis\\_performed & patient.kras\\_mutation\\_found & patient.kras\\_mutation\\_result & patient.location\\_in\\_lung\\_parenchyma & patient.month\\_of\\_form\\_completion & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event.locoregional\\_procedure.additional\\_surgery\\_locoregional\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.locoregional\\_procedure.days\\_to\\_additional\\_surgery\\_locoregional\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.metastatic\\_procedure.additional\\_surgery\\_metastatic\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.metastatic\\_procedure.days\\_to\\_additional\\_surgery\\_metastatic\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_types.new\\_neoplasm\\_event\\_type-2 & patient.new\\_tumor\\_events.new\\_tumor\\_event.progression\\_determined\\_by\\_list.progression\\_determined\\_by & patient.new\\_tumor\\_events.new\\_tumor\\_event.progression\\_determined\\_by\\_list.progression\\_determined\\_by-2 & patient.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.number\\_pack\\_years\\_smoked & patient.patient\\_id & patient.performance\\_status\\_scale\\_timing & patient.person\\_neoplasm\\_cancer\\_status & patient.post\\_bronchodilator\\_fev1\\_fvc\\_percent & patient.post\\_bronchodilator\\_fev1\\_percent & patient.pre\\_bronchodilator\\_fev1\\_fvc\\_percent & patient.pre\\_bronchodilator\\_fev1\\_percent & patient.primary\\_therapy\\_outcome\\_success & patient.prior\\_dx & patient.pulmonary\\_function\\_test\\_performed & patient.race & patient.radiation\\_therapy & patient.radiations & patient.radiations.radiation-2.anatomic\\_treatment\\_site & patient.radiations.radiation-2.bcr\\_radiation\\_barcode & patient.radiations.radiation-2.bcr\\_radiation\\_uuid & patient.radiations.radiation-2.course\\_number & patient.radiations.radiation-2.day\\_of\\_form\\_completion & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-2.measure\\_of\\_response & patient.radiations.radiation-2.month\\_of\\_form\\_completion & patient.radiations.radiation-2.numfractions & patient.radiations.radiation-2.radiation\\_dosage & patient.radiations.radiation-2.radiation\\_treatment\\_ongoing & patient.radiations.radiation-2.radiation\\_type & patient.radiations.radiation-2.radiation\\_type\\_notes & patient.radiations.radiation-2.regimen\\_indication & patient.radiations.radiation-2.regimen\\_indication\\_notes & patient.radiations.radiation-2.units & patient.radiations.radiation-2.year\\_of\\_form\\_completion & patient.radiations.radiation.anatomic\\_treatment\\_site & patient.radiations.radiation.bcr\\_radiation\\_barcode & patient.radiations.radiation.bcr\\_radiation\\_uuid & patient.radiations.radiation.course\\_number & patient.radiations.radiation.day\\_of\\_form\\_completion & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation.measure\\_of\\_response & patient.radiations.radiation.month\\_of\\_form\\_completion & patient.radiations.radiation.numfractions & patient.radiations.radiation.radiation\\_dosage & patient.radiations.radiation.radiation\\_treatment\\_ongoing & patient.radiations.radiation.radiation\\_type & patient.radiations.radiation.radiation\\_type\\_notes & patient.radiations.radiation.regimen\\_indication & patient.radiations.radiation.regimen\\_indication\\_notes & patient.radiations.radiation.units & patient.radiations.radiation.year\\_of\\_form\\_completion & patient.residual\\_tumor & patient.stage\\_event.ann\\_arbor.b\\_symptoms & patient.stage\\_event.ann\\_arbor.extranodal\\_involvement & patient.stage\\_event.clinical\\_stage & patient.stage\\_event.gleason\\_grading.gleason\\_score & patient.stage\\_event.gleason\\_grading.primary\\_pattern & patient.stage\\_event.gleason\\_grading.secondary\\_pattern & patient.stage\\_event.gleason\\_grading.tertiary\\_pattern & patient.stage\\_event.igcccg\\_stage & patient.stage\\_event.masaoka\\_stage & patient.stage\\_event.pathologic\\_stage & patient.stage\\_event.psa.days\\_to\\_psa & patient.stage\\_event.psa.psa\\_value & patient.stage\\_event.serum\\_markers & patient.stage\\_event.system\\_version & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_m & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_n & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_t & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t & patient.stopped\\_smoking\\_year & patient.targeted\\_molecular\\_therapy & patient.tissue\\_prospective\\_collection\\_indicator & patient.tissue\\_retrospective\\_collection\\_indicator & patient.tissue\\_source\\_site & patient.tobacco\\_smoking\\_history & patient.tumor\\_tissue\\_site & patient.vital\\_status & patient.year\\_of\\_form\\_completion & patient.year\\_of\\_initial\\_pathologic\\_diagnosis & patient.year\\_of\\_tobacco\\_smoking\\_onset & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-2 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-3 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status & patient.biospecimen\\_cqcf.b\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status & patient.biospecimen\\_cqcf.country & patient.biospecimen\\_cqcf.cytogenetic\\_report\\_submitted & patient.biospecimen\\_cqcf.days\\_to\\_pathology\\_review & patient.biospecimen\\_cqcf.days\\_to\\_sample\\_procurement & patient.biospecimen\\_cqcf.differential\\_report\\_submitted & patient.biospecimen\\_cqcf.digital\\_image\\_submitted & patient.biospecimen\\_cqcf.ethnicity & patient.biospecimen\\_cqcf.ffpe\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site & patient.biospecimen\\_cqcf.histological\\_type & patient.biospecimen\\_cqcf.histological\\_type\\_other & patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.biospecimen\\_cqcf.hiv\\_positive\\_status & patient.biospecimen\\_cqcf.laterality & patient.biospecimen\\_cqcf.maximum\\_tumor\\_dimension & patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number & patient.biospecimen\\_cqcf.other\\_method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.other\\_vessel\\_used & patient.biospecimen\\_cqcf.path\\_confirm\\_diagnosis\\_matching & patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_necrosis\\_metrics & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_nuclei\\_metrics & patient.biospecimen\\_cqcf.percent\\_myeloblasts\\_for\\_submitted\\_specimen & patient.biospecimen\\_cqcf.race & patient.biospecimen\\_cqcf.reason\\_path\\_confirm\\_diagnosis\\_not\\_matching & patient.biospecimen\\_cqcf.sample\\_prescreened & patient.biospecimen\\_cqcf.site\\_of\\_disease & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_description & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_text & patient.biospecimen\\_cqcf.submitted\\_for\\_lce & patient.biospecimen\\_cqcf.t\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.top\\_slide\\_submitted & patient.biospecimen\\_cqcf.total\\_cells\\_submitted & patient.biospecimen\\_cqcf.tumor\\_samples & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.vial\\_number & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number & patient.biospecimen\\_cqcf.vessel\\_used & patient.days\\_to\\_index & patient.samples.sample-2.additional\\_studies & patient.samples.sample-2.bcr\\_sample\\_barcode & patient.samples.sample-2.bcr\\_sample\\_uuid & patient.samples.sample-2.biospecimen\\_sequence & patient.samples.sample-2.composition & patient.samples.sample-2.current\\_weight & patient.samples.sample-2.days\\_to\\_collection & patient.samples.sample-2.days\\_to\\_sample\\_procurement & patient.samples.sample-2.diagnostic\\_slides.additional\\_studies & patient.samples.sample-2.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample-2.freezing\\_method & patient.samples.sample-2.initial\\_weight & patient.samples.sample-2.intermediate\\_dimension & patient.samples.sample-2.is\\_ffpe & patient.samples.sample-2.longest\\_dimension & patient.samples.sample-2.method\\_of\\_sample\\_procurement & patient.samples.sample-2.ncnnct\\_othmethonsp & patient.samples.sample-2.oct\\_embedded & patient.samples.sample-2.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-2.pathology\\_report\\_file\\_name & patient.samples.sample-2.pathology\\_report\\_uuid & patient.samples.sample-2.portions.portion.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-10.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-8.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-9.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.amount & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-2.portions.portion.day\\_of\\_creation & patient.samples.sample-2.portions.portion.is\\_ffpe & patient.samples.sample-2.portions.portion.lce & patient.samples.sample-2.portions.portion.month\\_of\\_creation & patient.samples.sample-2.portions.portion.portion\\_number & patient.samples.sample-2.portions.portion.portion\\_sequence & patient.samples.sample-2.portions.portion.slides & patient.samples.sample-2.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide-2.section\\_location & patient.samples.sample-2.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide.section\\_location & patient.samples.sample-2.portions.portion.weight & patient.samples.sample-2.portions.portion.year\\_of\\_creation & patient.samples.sample-2.preservation\\_method & patient.samples.sample-2.sample\\_type & patient.samples.sample-2.sample\\_type\\_id & patient.samples.sample-2.shortest\\_dimension & patient.samples.sample-2.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-2.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-2.tissue\\_type & patient.samples.sample-2.tumor\\_descriptor & patient.samples.sample-2.tumor\\_pathology & patient.samples.sample-2.vial\\_number & patient.samples.sample-3.additional\\_studies & patient.samples.sample-3.bcr\\_sample\\_barcode & patient.samples.sample-3.bcr\\_sample\\_uuid & patient.samples.sample-3.biospecimen\\_sequence & patient.samples.sample-3.composition & patient.samples.sample-3.current\\_weight & patient.samples.sample-3.days\\_to\\_collection & patient.samples.sample-3.days\\_to\\_sample\\_procurement & patient.samples.sample-3.diagnostic\\_slides.additional\\_studies & patient.samples.sample-3.freezing\\_method & patient.samples.sample-3.initial\\_weight & patient.samples.sample-3.intermediate\\_dimension & patient.samples.sample-3.is\\_ffpe & patient.samples.sample-3.longest\\_dimension & patient.samples.sample-3.method\\_of\\_sample\\_procurement & patient.samples.sample-3.ncnnct\\_othmethonsp & patient.samples.sample-3.oct\\_embedded & patient.samples.sample-3.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-3.pathology\\_report\\_file\\_name & patient.samples.sample-3.pathology\\_report\\_uuid & patient.samples.sample-3.portions.portion.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-3.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample-3.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-4.amount & patient.samples.sample-3.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-4.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.amount & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-3.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-3.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-3.portions.portion.day\\_of\\_creation & patient.samples.sample-3.portions.portion.is\\_ffpe & patient.samples.sample-3.portions.portion.lce & patient.samples.sample-3.portions.portion.month\\_of\\_creation & patient.samples.sample-3.portions.portion.portion\\_number & patient.samples.sample-3.portions.portion.portion\\_sequence & patient.samples.sample-3.portions.portion.slides & patient.samples.sample-3.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample-3.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample-3.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion.slides.slide-2.section\\_location & patient.samples.sample-3.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-3.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-3.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion.slides.slide.section\\_location & patient.samples.sample-3.portions.portion.weight & patient.samples.sample-3.portions.portion.year\\_of\\_creation & patient.samples.sample-3.preservation\\_method & patient.samples.sample-3.sample\\_type & patient.samples.sample-3.sample\\_type\\_id & patient.samples.sample-3.shortest\\_dimension & patient.samples.sample-3.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-3.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-3.tissue\\_type & patient.samples.sample-3.tumor\\_descriptor & patient.samples.sample-3.tumor\\_pathology & patient.samples.sample-3.vial\\_number & patient.samples.sample-4.additional\\_studies & patient.samples.sample-4.bcr\\_sample\\_barcode & patient.samples.sample-4.bcr\\_sample\\_uuid & patient.samples.sample-4.biospecimen\\_sequence & patient.samples.sample-4.composition & patient.samples.sample-4.current\\_weight & patient.samples.sample-4.days\\_to\\_collection & patient.samples.sample-4.days\\_to\\_sample\\_procurement & patient.samples.sample-4.diagnostic\\_slides.additional\\_studies & patient.samples.sample-4.freezing\\_method & patient.samples.sample-4.initial\\_weight & patient.samples.sample-4.intermediate\\_dimension & patient.samples.sample-4.is\\_ffpe & patient.samples.sample-4.longest\\_dimension & patient.samples.sample-4.method\\_of\\_sample\\_procurement & patient.samples.sample-4.ncnnct\\_othmethonsp & patient.samples.sample-4.oct\\_embedded & patient.samples.sample-4.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-4.pathology\\_report\\_file\\_name & patient.samples.sample-4.pathology\\_report\\_uuid & patient.samples.sample-4.portions.portion.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-4.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.amount & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-4.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-4.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-4.portions.portion.day\\_of\\_creation & patient.samples.sample-4.portions.portion.is\\_ffpe & patient.samples.sample-4.portions.portion.lce & patient.samples.sample-4.portions.portion.month\\_of\\_creation & patient.samples.sample-4.portions.portion.portion\\_number & patient.samples.sample-4.portions.portion.portion\\_sequence & patient.samples.sample-4.portions.portion.slides & patient.samples.sample-4.portions.portion.weight & patient.samples.sample-4.portions.portion.year\\_of\\_creation & patient.samples.sample-4.preservation\\_method & patient.samples.sample-4.sample\\_type & patient.samples.sample-4.sample\\_type\\_id & patient.samples.sample-4.shortest\\_dimension & patient.samples.sample-4.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-4.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-4.tissue\\_type & patient.samples.sample-4.tumor\\_descriptor & patient.samples.sample-4.tumor\\_pathology & patient.samples.sample-4.vial\\_number & patient.samples.sample-5.additional\\_studies & patient.samples.sample-5.bcr\\_sample\\_barcode & patient.samples.sample-5.bcr\\_sample\\_uuid & patient.samples.sample-5.biospecimen\\_sequence & patient.samples.sample-5.composition & patient.samples.sample-5.current\\_weight & patient.samples.sample-5.days\\_to\\_collection & patient.samples.sample-5.days\\_to\\_sample\\_procurement & patient.samples.sample-5.diagnostic\\_slides.additional\\_studies & patient.samples.sample-5.freezing\\_method & patient.samples.sample-5.initial\\_weight & patient.samples.sample-5.intermediate\\_dimension & patient.samples.sample-5.is\\_ffpe & patient.samples.sample-5.longest\\_dimension & patient.samples.sample-5.method\\_of\\_sample\\_procurement & patient.samples.sample-5.ncnnct\\_othmethonsp & patient.samples.sample-5.oct\\_embedded & patient.samples.sample-5.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-5.pathology\\_report\\_file\\_name & patient.samples.sample-5.pathology\\_report\\_uuid & patient.samples.sample-5.portions.portion.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-5.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-5.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-2.amount & patient.samples.sample-5.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-5.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-5.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-5.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-5.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-5.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-5.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-5.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-5.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-3.amount & patient.samples.sample-5.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-5.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample-5.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample-5.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-5.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-5.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-5.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample-5.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample-5.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte-4.amount & patient.samples.sample-5.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample-5.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte-4.concentration & patient.samples.sample-5.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-5.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample-5.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample-5.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-5.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample-5.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample-5.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample-5.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample-5.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-5.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-5.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-5.portions.portion.analytes.analyte.amount & patient.samples.sample-5.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-5.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-5.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-5.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-5.portions.portion.analytes.analyte.concentration & patient.samples.sample-5.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-5.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-5.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-5.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-5.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-5.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-5.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-5.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-5.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-5.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-5.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-5.portions.portion.day\\_of\\_creation & patient.samples.sample-5.portions.portion.is\\_ffpe & patient.samples.sample-5.portions.portion.lce & patient.samples.sample-5.portions.portion.month\\_of\\_creation & patient.samples.sample-5.portions.portion.portion\\_number & patient.samples.sample-5.portions.portion.portion\\_sequence & patient.samples.sample-5.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-5.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-5.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-5.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-5.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-5.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-5.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-5.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-5.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-5.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-5.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-5.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-5.portions.portion.slides.slide.section\\_location & patient.samples.sample-5.portions.portion.weight & patient.samples.sample-5.portions.portion.year\\_of\\_creation & patient.samples.sample-5.preservation\\_method & patient.samples.sample-5.sample\\_type & patient.samples.sample-5.sample\\_type\\_id & patient.samples.sample-5.shortest\\_dimension & patient.samples.sample-5.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-5.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-5.tissue\\_type & patient.samples.sample-5.tumor\\_descriptor & patient.samples.sample-5.tumor\\_pathology & patient.samples.sample-5.vial\\_number & patient.samples.sample.additional\\_studies & patient.samples.sample.bcr\\_sample\\_barcode & patient.samples.sample.bcr\\_sample\\_uuid & patient.samples.sample.biospecimen\\_sequence & patient.samples.sample.composition & patient.samples.sample.current\\_weight & patient.samples.sample.days\\_to\\_collection & patient.samples.sample.days\\_to\\_sample\\_procurement & patient.samples.sample.diagnostic\\_slides.additional\\_studies & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-2 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-3 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-4 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-5 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-6 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-7 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-8 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-9 & patient.samples.sample.freezing\\_method & patient.samples.sample.initial\\_weight & patient.samples.sample.intermediate\\_dimension & patient.samples.sample.is\\_ffpe & patient.samples.sample.longest\\_dimension & patient.samples.sample.method\\_of\\_sample\\_procurement & patient.samples.sample.ncnnct\\_othmethonsp & patient.samples.sample.oct\\_embedded & patient.samples.sample.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample.pathology\\_report\\_file\\_name & patient.samples.sample.pathology\\_report\\_uuid & patient.samples.sample.portions.portion-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion-2.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion-2.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte-3.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion-2.analytes.analyte-3.rna.rinvalue & patient.samples.sample.portions.portion-2.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-3.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-3.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte-4.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte-4.amount & patient.samples.sample.portions.portion-2.analytes.analyte-4.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte-4.concentration & patient.samples.sample.portions.portion-2.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion-2.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion-2.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte-4.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte-4.well\\_number & patient.samples.sample.portions.portion-2.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion-2.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion-2.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion-2.analytes.analyte.amount & patient.samples.sample.portions.portion-2.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion-2.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion-2.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion-2.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion-2.analytes.analyte.concentration & patient.samples.sample.portions.portion-2.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion-2.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion-2.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion-2.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion-2.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion-2.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion-2.analytes.analyte.well\\_number & patient.samples.sample.portions.portion-2.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion-2.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion-2.day\\_of\\_creation & patient.samples.sample.portions.portion-2.is\\_ffpe & patient.samples.sample.portions.portion-2.lce & patient.samples.sample.portions.portion-2.month\\_of\\_creation & patient.samples.sample.portions.portion-2.portion\\_number & patient.samples.sample.portions.portion-2.portion\\_sequence & patient.samples.sample.portions.portion-2.slides.slide-2.additional\\_studies & patient.samples.sample.portions.portion-2.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion-2.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion-2.slides.slide-2.image\\_file\\_name & patient.samples.sample.portions.portion-2.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_necrosis & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample.portions.portion-2.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion-2.slides.slide-2.section\\_location & patient.samples.sample.portions.portion-2.slides.slide.additional\\_studies & patient.samples.sample.portions.portion-2.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion-2.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion-2.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion-2.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion-2.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion-2.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion-2.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion-2.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion-2.slides.slide.section\\_location & patient.samples.sample.portions.portion-2.weight & patient.samples.sample.portions.portion-2.year\\_of\\_creation & patient.samples.sample.portions.portion.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.amount & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-3.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-4.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-4.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-4.amount & patient.samples.sample.portions.portion.analytes.analyte-4.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-4.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-4.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-4.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-4.concentration & patient.samples.sample.portions.portion.analytes.analyte-4.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-4.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-4.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-4.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-4.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-4.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-4.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-5.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-5.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-5.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-5.amount & patient.samples.sample.portions.portion.analytes.analyte-5.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-5.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-5.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-5.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-5.concentration & patient.samples.sample.portions.portion.analytes.analyte-5.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-5.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-5.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-5.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-5.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-5.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-5.well\\_number & patient.samples.sample.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-10.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-11.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-12.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-13.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-7.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-8.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-9.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.amount & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.concentration & patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte.well\\_number & patient.samples.sample.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion.day\\_of\\_creation & patient.samples.sample.portions.portion.is\\_ffpe & patient.samples.sample.portions.portion.lce & patient.samples.sample.portions.portion.month\\_of\\_creation & patient.samples.sample.portions.portion.portion\\_number & patient.samples.sample.portions.portion.portion\\_sequence & patient.samples.sample.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide-2.section\\_location & patient.samples.sample.portions.portion.slides.slide.additional\\_studies & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide.section\\_location & patient.samples.sample.portions.portion.weight & patient.samples.sample.portions.portion.year\\_of\\_creation & patient.samples.sample.portions.shipment\\_portion.additional\\_studies & patient.samples.sample.portions.shipment\\_portion.bcr\\_shipment\\_portion\\_uuid & patient.samples.sample.portions.shipment\\_portion.center\\_id & patient.samples.sample.portions.shipment\\_portion.is\\_ffpe & patient.samples.sample.portions.shipment\\_portion.plate\\_id & patient.samples.sample.portions.shipment\\_portion.portion\\_number & patient.samples.sample.portions.shipment\\_portion.portion\\_sequence & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_bcr\\_aliquot\\_barcode & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_day\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_month\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_year\\_of\\_shipment & patient.samples.sample.preservation\\_method & patient.samples.sample.sample\\_type & patient.samples.sample.sample\\_type\\_id & patient.samples.sample.shortest\\_dimension & patient.samples.sample.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample.tissue\\_type & patient.samples.sample.tumor\\_descriptor & patient.samples.sample.tumor\\_pathology & patient.samples.sample.vial\\_number\\\\\n",
       "  & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t39.63.0 & nationwide children's hospital & 1 & lusc & b34e8d44-9f2d-4ab3-a189-ecb5b77af723 & 10 & false & tcga & 2015 & NA & 67 & l-upper & NA & tcga-18-3406 & 95b83006-02c9-4c4d-bf84-a45115f7d86d & l-upper & deceased  & canada & NA  & 371 & NA & lung squamous cell carcinoma- not otherwise specified (nos) & no & no                               & other & NA & NA & l unknown   & NA & primary & 9  & -24477 & 371 & 0 & NA   & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & not hispanic or latino & NA & NA  & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA   & NA & NA                         & NA                          & NA & NA & NA & NA                & NA & NA  & NA & NA         & NA                          & NA & NA & NA & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & no  & no & NA & tcga-18-3406-f69068 & f762cb38-45ba-4519-be01-2a508415b28d & 29 & NA & NA & 371  & NA   & 357  & NA & scheduled follow-up submission & NA                          & 0  & no & 12 & locoregional recurrence & NA & yes & NA & with tumor & progressive disease         & convincing imaging                  & NA & NA & no  & no  & dead  & 2014 & male   & lung squamous cell carcinoma- not otherwise specified (nos) & no & c34.1 & 8070/3 & c34.1 & yes & 0  & NA & NA & NA & NA              & 3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 200 & 3406 & NA & with tumor & NA & NA & NA & NA & NA & yes & NA & white & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ia   & NA & NA & NA & NA  & NA & NA & NA & m0 & n0 & t1 & NA   & NA & no & yes & 18 & current reformed smoker for < or = 15 years & lung & dead  & 2011 & 2003 & 1951 & case matched methylation control & NA & NA & canonical - plus & NA & deceased  & canada & NA & 2319 & -92 & NA & no & NA & NA & lung & lung squamous cell carcinoma- not otherwise specified (nos) & NA & no & NA & left  & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & ab8b5e65-efbe-42f1-99c3-917f1e818912 & -92 & NA & NA & NA & NA & tumor resection & NA & other & NA & 1 & completion lt. pneumonectomy & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & upper & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & 7a55a700-a706-443d-8bad-e70483c464f2 & NA & 20 & 80 & NA & 119 & 1 & cryovial & 0 & NA & tcga-18-3406-11a & ab8b5e65-efbe-42f1-99c3-917f1e818912 & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.4 & no & 0.8 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-11a-01w-0939-09 & 5a976df2-7ecd-4248-a59e-ad161a41d210 & 96492278 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & b & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-11a-01w-0938-08 & 54a0f308-5c63-48a5-851f-1b26e2a9e7e5 & 96485366 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & b & 40.00 & 22 & 80.00 & 2010 & 83.00  & repli-g (qiagen) dna & w & tcga-18-3406-11a-01w & c9c53ca0-fad0-42a2-9503-2b3aefe84feb & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3406-11a-01x-0941-09 & 3bc4a10e-c6e1-4120-b685-5fb9414580d4 & 97543318 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & b & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3406-11a-01x-0940-08 & 2203d04d-b5bf-4f37-af20-e493a89577d4 & 97544182 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & b & 40.00 & 22 & 80.00 & 2010 & 42.00  & repli-g x (qiagen) dna & x & tcga-18-3406-11a-01x & 5e692ac8-9595-4811-8525-3140ab37fd12 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-11a-01d-0979-05 & 41702f60-c945-4267-a7f0-e1ff20cdd57f & 0097557332 & 05 & 0.14 & 05 & no & 05 & 6 & 0979 & b & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3406-11a-01d-0983-08 & 61812dcb-a44c-4e00-9527-4a5146f617c1 & 0097556468 & 08 & 0.14 & 05 & no & 05 & 6 & 0983 & b & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3406-11a-01d-0984-09 & 383e4fd1-540a-400e-bdf9-43c6ae8f4d98 & 0097513364 & 09 & 0.14 & 05 & no & 05 & 6 & 0984 & b & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3406-11a-01d-1028-04 & a1243096-c1e2-4d08-a6cc-f05d2d81a67d & 0098888756 & 04 & 0.14 & 02 & no & 06 & 6 & 1028 & b & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3406-11a-01d-1029-06 & 1b781def-71dd-4572-ac61-3d83974e8a24 & 0098847572 & 06 & 0.14 & 02 & no & 06 & 6 & 1029 & b & 1.86 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3406-11a-01d-1519-02 & 8f6f7064-824a-4f0a-9450-b695ee915dc5 & 1049949210 & 02 & 0.14 & 26 & no & 01 & 5 & 1519 & b & 1.86 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-11a-01d-0978-01 & 7ac7928d-8688-4a4c-8a25-23c47ae2a06d & 0097558004 & 01 & 0.14 & 05 & no & 05 & 6 & 0978 & b & 0.94 & 22 & 6.70 & 2010 & 31.64  & dna & d & tcga-18-3406-11a-01d & 11e4b1ee-8819-4f63-ae25-df10b34abba0 & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3406-11a-01 & 2aa26aa3-186a-4a2a-aae8-78600d8c9dc0 & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3406-11a-01-ts1 & bebef82c-348c-4686-8bcb-a8117a92da8e & tcga-18-3406-11a-01-ts1.bebef82c-348c-4686-8bcb-a8117a92da8e.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3406-11a-01-bs1 & 6e0942b1-b368-4b8a-8016-cea4405d8abb & tcga-18-3406-11a-01-bs1.6e0942b1-b368-4b8a-8016-cea4405d8abb.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.2 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a & 7a55a700-a706-443d-8bad-e70483c464f2 & NA & NA & NA & NA & NA & NA & 8d07f006-425c-4724-bbb3-5ba099401234 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.8 & NA & NA & NA & NA & tcga-18-3406.a0eef850-7543-4f82-8e46-3d81e203412d.pdf & a0eef850-7543-4f82-8e46-3d81e203412d & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.70 & NA & NA & NA & tcga-18-3406-01a-01r-1030-03 & a9ffe531-c65f-414f-87af-74cb5b97c0b1 & 0098887242 & 03 & 0.15 & 02 & no & 06 & 1 & 1030 & a & 2.51 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3406-01a-01r-1031-01 & c9b700c0-9518-4edd-8db9-e75301e00c08 & 0098889350 & 01 & 0.15 & 02 & no & 06 & 1 & 1031 & a & 3.00 & 22 & 20.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01r-0980-07 & 99516b1e-9ee2-48d9-90e4-e78370eccef6 & 0097557926 & 07 & 0.15 & 05 & no & 05 & 1 & 0980 & a & 4.01 & 22 & 26.70 & 2010 & 56.60  & rna & r & tcga-18-3406-01a-01r & 39a64452-d7ba-40f6-b84d-1dbc4ef3a96e & 0.15 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 0.70 & 8.40 & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01t-1032-07 & c41c1532-2367-4774-bb8b-1014be06269c & 98848646 & 07 & 0.10 & 02 & no & 06 & 1  & 1032 & a  & 0.73 & 22 & 7.30  & 2010 & NA & NA & tcga-18-3406-01a-01t-0981-13 & 14450db2-c7aa-4d87-add6-e82988aa85d3 & 0097557446 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & a & 3.00 & 22 & 30.00 & 2010 & 0.00  & total rna & t & tcga-18-3406-01a-01t & 203521df-3899-4293-96ab-69ea56af6703 & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01w-0939-09 & b92c5e28-3d10-4037-89fa-673aa20b40a3 & 96492271 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & b & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01w-0938-08 & 2d475f0a-895a-4732-9501-befa8cc8cd5b & 96485359 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & b & 40.00 & 22 & 80.00 & 2010 & 88.00  & repli-g (qiagen) dna & w & tcga-18-3406-01a-01w & b67b8913-c639-40e3-995e-37978925a226 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01x-0941-09 & 85b61351-5fcc-4118-be3b-b045e9098a30 & 97543311 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & b & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3406-01a-01x-0940-08 & 1250686f-881a-4098-8801-dafda465d954 & 97544175 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & b & 40.00 & 22 & 80.00 & 2010 & 92.00 & repli-g x (qiagen) dna & x & tcga-18-3406-01a-01x & 9e3f225d-b920-466b-8e1e-2708bad7183c & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01d-0979-05 & 4ce86f70-ba18-490a-9b28-cbc79b8885ac & 0097557350 & 05 & 0.14 & 05 & no & 05 & 1 & 0979 & a & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3406-01a-01d-0983-08 & d3320989-71fd-425b-933e-6e8528a016ed & 0097555463 & 08 & 0.14 & 05 & no & 05 & 1 & 0983 & a & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3406-01a-01d-0984-09 & 3087303d-ca77-4629-8bee-6878c40776d6 & 0097513382 & 09 & 0.14 & 05 & no & 05 & 1 & 0984 & a & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3406-01a-01d-1028-04 & 5094fa35-5334-4930-b9f0-5e5d0ac916b0 & 0098888774 & 04 & 0.14 & 02 & no & 06 & 1 & 1028 & a & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3406-01a-01d-1029-06 & 279f967e-8dd1-46b6-b2a0-243d93295d9d & 0098847590 & 06 & 0.14 & 02 & no & 06 & 1 & 1029 & a & 1.86 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3406-01a-01d-1519-02 & 37202605-3f3a-4266-9d1f-e46aae133c15 & 1049949191 & 02 & 0.14 & 26 & no & 01 & 1 & 1519 & a & 1.86 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3406-01a-01d-0978-01 & 0f960732-d006-45f6-81d4-6ae8e659c75d & 0097558022 & 01 & 0.14 & 05 & no & 05 & 1 & 0978 & a & 0.94 & 22 & 6.70 & 2010 & 228.64 & dna & d & tcga-18-3406-01a-01d & ecc361e7-5e43-47a3-87c7-71de8d5e0b40 & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3406-01a-01 & bd616336-4b27-4dd2-a61c-288f51840cab & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3406-01a-01-ts1 & a9f60826-a270-4e3c-881f-2b8df24a5285 & tcga-18-3406-01a-01-ts1.a9f60826-a270-4e3c-881f-2b8df24a5285.svs & no & NA & NA & NA & NA & 10 & NA & 1  & NA & 0 & 19 & 80 & 80 & top & NA & tcga-18-3406-01a-01-bs1 & 64ccb223-fed0-4472-92f0-8789ac4afc59 & tcga-18-3406-01a-01-bs1.64ccb223-fed0-4472-92f0-8789ac4afc59.svs & no & NA & NA & NA & NA & 20 & NA & 1  & NA & 5  & 34 & 60 & 60 & bottom & 105.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t39.63.0.1 & nationwide children's hospital & 1 & lusc & 2fc3e9ed-7219-4d78-8964-a5c9de65b8da & 10 & false & tcga & 2015 & NA & 72 & l-upper & NA & tcga-18-3407 & 4e1ad82e-23c8-44bb-b74e-a3d0b1126b96 & l-upper & deceased  & canada & NA  & 136 & NA & lung squamous cell carcinoma- not otherwise specified (nos) & no & no                               & lung  & NA & NA & NA          & NA & primary & 9  & -26615 & 136 & 0 & NA   & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & not hispanic or latino & NA & NA  & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA   & NA & NA                         & NA                          & NA & NA & NA & NA                & NA & NA  & NA & NA         & NA                          & NA & NA & NA & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA  & NA & NA & tcga-18-3407-f69081 & 5850d534-294d-4583-ac39-37f78aa5cc5d & 29 & NA & NA & 136  & NA   & NA   & NA & scheduled follow-up submission & NA                          & 0  & no & 12 & NA                      & NA & no  & NA & tumor free & NA                          & NA                                  & NA & NA & no  & no  & dead  & 2014 & male   & lung squamous cell carcinoma- not otherwise specified (nos) & no & c34.1 & 8070/3 & c34.1 & yes & 0  & NA & NA & NA & NA              & 3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 40  & 3407 & NA & tumor free & NA & NA & NA & NA & NA & yes & NA & asian & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ib   & NA & NA & NA & NA  & NA & NA & NA & m0 & n0 & t2 & 1988 & NA & no & yes & 18 & current reformed smoker for > 15 years      & lung & dead  & 2011 & 2003 & NA   & case matched methylation control & NA & NA & canonical - plus & NA & deceased  & canada & NA & 2325 & 97  & NA & no & NA & NA & lung & lung squamous cell carcinoma- not otherwise specified (nos) & NA & no & NA & left  & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 5822fe84-c25f-4cad-8c17-4e37fc9376a3 & 97  & NA & NA & NA & NA & tumor resection & NA & lung  & NA & 1 & l upper lobectomy            & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & upper & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & 49963d3b-a50e-4086-bc94-fe2042f9fcae & NA & 15 & 70 & NA & 56  & 1 & cryovial & 0 & NA & tcga-18-3407-11a & 5822fe84-c25f-4cad-8c17-4e37fc9376a3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.7 & no & 0.9 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-11a-01w-0939-09 & 786879cb-a112-4eae-9a1b-cecba65394b6 & 96492263 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & c & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-11a-01w-0938-08 & 79be6deb-c8dc-4c99-8214-771ba6ba86b2 & 96488039 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & c & 40.00 & 22 & 80.00 & 2010 & 64.00  & repli-g (qiagen) dna & w & tcga-18-3407-11a-01w & 7c254565-f145-447f-b7da-1bf7036efb62 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3407-11a-01x-0941-09 & 6c2b64b2-f53c-4535-9490-168970181452 & 97543303 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & c & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3407-11a-01x-0940-08 & 6bf0eeea-5177-456b-a7a8-8c4cdd28fe90 & 97544167 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & c & 40.00 & 22 & 80.00 & 2010 & 84.00  & repli-g x (qiagen) dna & x & tcga-18-3407-11a-01x & 4fe1bdd2-5a50-4d79-a883-12335b641dbd & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-11a-01d-0979-05 & 31e0a946-61df-48f5-a8c3-54585e0d63ab & 0097557321 & 05 & 0.13 & 05 & no & 05 & 6 & 0979 & c & 3.47 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3407-11a-01d-0983-08 & 09e5f5f4-5e35-4b88-a266-25649c08bb77 & 0097556457 & 08 & 0.13 & 05 & no & 05 & 6 & 0983 & c & 0.87 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3407-11a-01d-0984-09 & e8cd923e-b685-48ad-9df6-455821adba60 & 0097513353 & 09 & 0.13 & 05 & no & 05 & 6 & 0984 & c & 0.87 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3407-11a-01d-1028-04 & 0dbb93a1-f926-4c91-beeb-f74c86de153f & 0098888745 & 04 & 0.13 & 02 & no & 06 & 6 & 1028 & c & 3.47 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3407-11a-01d-1029-06 & 3697599d-34bd-49b2-9a27-9bd375729e37 & 0098847561 & 06 & 0.13 & 02 & no & 06 & 6 & 1029 & c & 1.73 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3407-11a-01d-1519-02 & 3700ca30-2fef-4aab-a361-7230832d1388 & 1049949219 & 02 & 0.13 & 26 & no & 01 & 5 & 1519 & c & 1.73 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-11a-01d-0978-01 & 3e279167-a1e4-4d0c-afd4-01729c6615d7 & 0097557993 & 01 & 0.13 & 05 & no & 05 & 6 & 0978 & c & 0.87 & 22 & 6.70 & 2010 & 276.64 & dna & d & tcga-18-3407-11a-01d & 0d59982b-ed62-4a7b-b275-9c420e7c0dd2 & 0.13 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3407-11a-01 & b03373b5-218a-43cf-890f-21b19af6d038 & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3407-11a-01-ts1 & 79911757-f4d7-4e20-91f4-5779d076c666 & tcga-18-3407-11a-01-ts1.79911757-f4d7-4e20-91f4-5779d076c666.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3407-11a-01-bs1 & 6fa61035-bdb7-4cb0-becb-7d5c999ed1e3 & tcga-18-3407-11a-01-bs1.6fa61035-bdb7-4cb0-becb-7d5c999ed1e3.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.4 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a & 49963d3b-a50e-4086-bc94-fe2042f9fcae & NA & NA & NA & NA & NA & NA & e7dda7e3-45a2-4e06-a081-e6a81a672155 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.5 & NA & NA & NA & NA & tcga-18-3407.c9b161d4-e3e2-43a9-8997-195db2fcc1d7.pdf & c9b161d4-e3e2-43a9-8997-195db2fcc1d7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & tcga-18-3407-01a-01r-1030-03 & da61438c-cf66-406d-bef7-91b224752684 & 0098887407 & 03 & 0.15 & 02 & no & 06 & 1 & 1030 & b & 2.51 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3407-01a-01r-1031-01 & b90d788b-037b-4e93-88ac-bdad05e52265 & 0098889327 & 01 & 0.15 & 02 & no & 06 & 1 & 1031 & b & 3.00 & 22 & 20.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01r-0980-07 & 0aea71d4-f403-474a-91d3-f9b0929c0acf & 0097557903 & 07 & 0.15 & 05 & no & 05 & 1 & 0980 & b & 4.01 & 22 & 26.70 & 2010 & 132.60 & rna & r & tcga-18-3407-01a-01r & ddfcce1c-5b4d-4e02-aeec-44f94bb11f54 & 0.15 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.50 & 7.70 & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01t-1032-07 & ab73e14f-3fb6-4c18-a919-21d5edfbf24d & 98848623 & 07 & 0.10 & 02 & no & 06 & 1  & 1032 & b  & 0.28 & 22 & 2.80  & 2010 & NA & NA & tcga-18-3407-01a-01t-0981-13 & c23e3dc9-d382-419d-90c5-69a32c09c4fc & 0097557423 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & b & 3.00 & 22 & 30.00 & 2010 & 0.00  & total rna & t & tcga-18-3407-01a-01t & 12285ede-8f1c-4a9b-9b70-01b447f8275e & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01w-0939-09 & 665caa35-17d8-4088-b0ab-20675d207c11 & 96492270 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & c & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01w-0938-08 & 08c4a8aa-3e90-481e-96e7-e3ac2590ae5a & 96485358 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & c & 40.00 & 22 & 80.00 & 2010 & 77.00  & repli-g (qiagen) dna & w & tcga-18-3407-01a-01w & 019762d8-b8b8-4b3c-a6b2-61b72b307de8 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01x-0941-09 & 2de1712c-0530-4633-a3c0-83fdf1727721 & 97543310 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & c & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3407-01a-01x-0940-08 & 0f62ef0d-003a-4da1-a0e5-6dc48efa2b7f & 97544174 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & c & 40.00 & 22 & 80.00 & 2010 & 97.00 & repli-g x (qiagen) dna & x & tcga-18-3407-01a-01x & c7b04aac-aa8d-4db2-91b4-d1517f1f2976 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01d-0979-05 & 3cd48a12-6d39-4b43-bf94-81793f12484f & 0097557327 & 05 & 0.13 & 05 & no & 05 & 1 & 0979 & b & 3.47 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3407-01a-01d-0983-08 & c5b09119-0237-4804-a4f9-b67d676b8674 & 0097555486 & 08 & 0.13 & 05 & no & 05 & 1 & 0983 & b & 0.87 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3407-01a-01d-0984-09 & 444a6ea8-0f87-4068-9690-6fd7213dae73 & 0097513359 & 09 & 0.13 & 05 & no & 05 & 1 & 0984 & b & 0.87 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3407-01a-01d-1028-04 & 77e30af1-54fa-4d16-b6d2-c802ae54b161 & 0098888751 & 04 & 0.13 & 02 & no & 06 & 1 & 1028 & b & 3.47 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3407-01a-01d-1029-06 & babe6615-896f-40ee-8ba4-b0e8d71df57e & 0098847567 & 06 & 0.13 & 02 & no & 06 & 1 & 1029 & b & 1.73 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3407-01a-01d-1519-02 & dd570cc4-aa48-416b-b33a-d2e108066fdd & 1049949214 & 02 & 0.13 & 26 & no & 01 & 1 & 1519 & b & 1.73 & 22 & 13.30 & 2011 & NA & NA & tcga-18-3407-01a-01d-1969-01 & 85b5d841-ed1e-4a81-b157-c9ae9eb77b84 & 1053936460 & 01 & 0.13 & 22 & no & 08 & 1  & 1969 & a  & 0.87 & 22 & 6.67  & 2011 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3407-01a-01d-0978-01 & 857e37ce-e790-41be-8bd7-8b80d2763172 & 0097557999 & 01 & 0.13 & 05 & no & 05 & 1 & 0978 & b & 0.87 & 22 & 6.70 & 2010 & 265.97 & dna & d & tcga-18-3407-01a-01d & 41cb90f9-cf70-4fe3-9116-45d0ee80db03 & 0.13 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3407-01a-01 & 1d9c991b-2cb6-4e78-b787-44cd7f4607cc & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3407-01a-01-ts1 & eb6d8014-eaf9-4679-955f-8ad4c5205850 & tcga-18-3407-01a-01-ts1.eb6d8014-eaf9-4679-955f-8ad4c5205850.svs & no & NA & NA & NA & NA & 10 & NA & 10 & NA & 0 & 15 & 75 & 80 & top & NA & tcga-18-3407-01a-01-bs1 & 6115884c-8a3c-4a85-ac0f-f214fa5d4c55 & tcga-18-3407-01a-01-bs1.6115884c-8a3c-4a85-ac0f-f214fa5d4c55.svs & no & NA & NA & NA & NA & 10 & NA & 15 & NA & 0  & 15 & 70 & 75 & bottom & 82.00  & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.2 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t39.63.0.2 & nationwide children's hospital & 1 & lusc & 1af63f26-0d2a-463b-ba33-7b60ab118b50 & 10 & false & tcga & 2015 & NA & 77 & r-lower & NA & tcga-18-3408 & d4bc755a-2585-4529-ae36-7e1d88bdecfe & r-lower & consented & canada & 33  & NA  & NA & lung squamous cell carcinoma- not otherwise specified (nos) & no & no                               & lung  & NA & NA & NA          & NA & primary & 9  & -28171 & NA  & 0 & 2099 & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & not hispanic or latino & NA & NA  & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA   & NA & NA                         & NA                          & NA & NA & NA & NA                & NA & NA  & NA & NA         & NA                          & NA & NA & NA & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & yes & no & NA & tcga-18-3408-f67568 & 487467b7-493f-410c-9012-beeaa137b44f & 14 & NA & NA & 2304 & NA   & 1793 & NA & scheduled follow-up submission & progressive disease         & 0  & no & 11 & locoregional recurrence & NA & yes & NA & with tumor & NA                          & biopsy with histologic confirmation & NA & NA & no  & no  & dead  & 2014 & female & lung squamous cell carcinoma- not otherwise specified (nos) & no & c34.3 & 8070/3 & c34.3 & yes & NA & NA & NA & NA & NA              & 3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 30  & 3408 & NA & with tumor & NA & NA & NA & NA & NA & no  & NA & white & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ib   & NA & NA & NA & NA  & NA & NA & NA & m0 & n0 & t2 & 2004 & NA & no & yes & 18 & current reformed smoker for < or = 15 years & lung & alive & 2011 & 2004 & NA   & case matched methylation control & NA & NA & canonical - plus & NA & consented & canada & NA & 2107 & 53  & NA & no & NA & NA & lung & lung squamous cell carcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 786c2a76-177f-4f22-a317-20fea7e594be & 53  & NA & NA & NA & NA & tumor resection & NA & lung  & NA & 1 & r lower lobectomy            & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & lower & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & eeffadf9-2ccb-4937-ae1e-1d50aba41c4d & NA & 10 & 80 & NA & 55  & 1 & cryovial & 0 & NA & tcga-18-3408-11a & 786c2a76-177f-4f22-a317-20fea7e594be & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.5 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-11a-01w-0939-09 & b9057cbd-05af-4e6e-a4d7-79d4ad86446c & 96492254 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & d & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-11a-01w-0938-08 & 2ccc1eee-535a-47ad-b5ba-65c9c0d854cf & 96485342 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & d & 40.00 & 22 & 80.00 & 2010 & 105.00 & repli-g (qiagen) dna & w & tcga-18-3408-11a-01w & 556fbf6a-50a8-4d99-ac9a-ea1d2d860896 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3408-11a-01x-0941-09 & 1911ffa2-30cd-45a9-bc87-96b8b8f9e9a7 & 97543294 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & d & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3408-11a-01x-0940-08 & 20a2d69b-d76a-4815-a1bb-b10b13761e61 & 97544158 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & d & 40.00 & 22 & 80.00 & 2010 & 108.00 & repli-g x (qiagen) dna & x & tcga-18-3408-11a-01x & 316ca717-b1c6-4456-8077-42cbd7a28d94 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-11a-01d-0979-05 & 895fe41f-f172-4f47-b186-b2c5a3e7be53 & 0097557308 & 05 & 0.15 & 05 & no & 05 & 6 & 0979 & d & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3408-11a-01d-0983-08 & f60256d4-9359-49fc-86ff-dada8524ab7f & 0097556444 & 08 & 0.15 & 05 & no & 05 & 6 & 0983 & d & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3408-11a-01d-0984-09 & 82eb5f94-5006-467b-9f05-9264c4ede3c1 & 0097513340 & 09 & 0.15 & 05 & no & 05 & 6 & 0984 & d & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3408-11a-01d-1028-04 & d724df67-76c9-4913-af53-1930c95aa0a0 & 0098888732 & 04 & 0.15 & 02 & no & 06 & 6 & 1028 & d & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3408-11a-01d-1029-06 & a50a6155-d45d-44e0-a1ed-27379310e7c0 & 0098847548 & 06 & 0.15 & 02 & no & 06 & 6 & 1029 & d & 2.00 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3408-11a-01d-1519-02 & c40dbb91-de45-415f-853f-61d8c220ed85 & 1049949234 & 02 & 0.15 & 26 & no & 01 & 5 & 1519 & d & 2.00 & 22 & 13.30 & 2011 & NA & NA & tcga-18-3408-11a-01d-a320-08 & c1e8deaf-dda4-4d0a-9316-913a1cb736e7 & 0145635615 & 08 & 0.08 & 19 & no & 08 & 2  & a320 & c  & 1.39 & 23 & 17.40 & 2013 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-11a-01d-0978-01 & 43fac17e-a256-4ef6-a149-0b614599f3ec & 0097557980 & 01 & 0.15 & 05 & no & 05 & 6 & 0978 & d & 1.01 & 22 & 6.70 & 2010 & 149.94 & dna & d & tcga-18-3408-11a-01d & 2a6bb326-7734-4793-a2eb-3442c33639bb & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3408-11a-01 & 6adf7a63-4750-41bf-b652-ae0f1567deba & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3408-11a-01-ts1 & ee29f78a-bf26-425b-aaf0-797af774427c & tcga-18-3408-11a-01-ts1.ee29f78a-bf26-425b-aaf0-797af774427c.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3408-11a-01-bs1 & 4b8fd6fb-2f0b-4166-8554-1213e5d93758 & tcga-18-3408-11a-01-bs1.4b8fd6fb-2f0b-4166-8554-1213e5d93758.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.5 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a & eeffadf9-2ccb-4937-ae1e-1d50aba41c4d & NA & NA & NA & NA & NA & NA & b4afc08a-7460-4ee6-b033-629c6a6ca6e8 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.4 & no & 0.5 & NA & NA & NA & NA & tcga-18-3408.ef586d16-0c67-42db-9d31-86df9476d49d.pdf & ef586d16-0c67-42db-9d31-86df9476d49d & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & tcga-18-3408-01a-01r-1030-03 & 29ccf163-35c7-4418-88b9-e96e04752f28 & 0098887406 & 03 & 0.15 & 02 & no & 06 & 1 & 1030 & c & 2.51 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3408-01a-01r-1031-01 & 6c8ee55e-562d-4624-b69c-a6987301f0d0 & 0098889326 & 01 & 0.15 & 02 & no & 06 & 1 & 1031 & c & 3.00 & 22 & 20.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01r-0980-07 & 43672213-eb65-4206-bc57-56587b08259f & 0097557902 & 07 & 0.15 & 05 & no & 05 & 1 & 0980 & c & 4.01 & 22 & 26.70 & 2010 & 170.60 & rna & r & tcga-18-3408-01a-01r & 30e18002-046c-412d-9b37-3510e28fa91e & 0.15 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.40 & 8.60 & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01t-1032-07 & 2963b337-2601-4440-bcb5-b6b1cd9a808c & 98848622 & 07 & 0.10 & 02 & no & 06 & 1  & 1032 & c  & 1.00 & 22 & 10.00 & 2010 & NA & NA & tcga-18-3408-01a-01t-0981-13 & c8ac8233-424d-47c0-a27d-9fcfb6176b1b & 0097557422 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & c & 3.00 & 22 & 30.00 & 2010 & 14.00 & total rna & t & tcga-18-3408-01a-01t & 0b222e03-5bfc-4dab-8e84-43be87997b37 & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01w-0939-09 & ea7d512d-5132-450f-9bc2-7e227c85c9b2 & 96492247 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & d & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01w-0938-08 & 9e276f6c-7826-4ac9-a255-b17ac60cacd1 & 96485335 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & d & 40.00 & 22 & 80.00 & 2010 & 71.00  & repli-g (qiagen) dna & w & tcga-18-3408-01a-01w & f5957b78-8275-4ac3-a3f6-be474f1c91fb & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01x-0941-09 & 84a1aded-1f85-41e4-a4bc-7aa910191eef & 97543287 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & d & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3408-01a-01x-0940-08 & b05f71a9-b137-4560-b257-33e6f9dcf362 & 97544151 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & d & 40.00 & 22 & 80.00 & 2010 & 65.00 & repli-g x (qiagen) dna & x & tcga-18-3408-01a-01x & da4cf093-d3c8-4920-aa42-896a825c6706 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01d-0979-05 & 3ba14151-3833-4e12-aa2f-62eb2fba7ab1 & 0097557326 & 05 & 0.15 & 05 & no & 05 & 1 & 0979 & c & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3408-01a-01d-0983-08 & cab7a425-e081-4bae-b666-6cdf8ba4dd70 & 0097555520 & 08 & 0.15 & 05 & no & 05 & 1 & 0983 & c & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3408-01a-01d-0984-09 & 99ed6abf-1da0-4b69-a9f0-6ed48c7c380e & 0097513358 & 09 & 0.15 & 05 & no & 05 & 1 & 0984 & c & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3408-01a-01d-1028-04 & 25974c53-da9d-4418-8e75-201753849cc6 & 0098888750 & 04 & 0.15 & 02 & no & 06 & 1 & 1028 & c & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3408-01a-01d-1029-06 & ff0f2a92-52c1-4d08-94e6-82f7b9592594 & 0098847566 & 06 & 0.15 & 02 & no & 06 & 1 & 1029 & c & 2.00 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3408-01a-01d-1519-02 & 1349233b-17c2-474f-a6e7-c1e6e1f3de5e & 1049949215 & 02 & 0.15 & 26 & no & 01 & 1 & 1519 & c & 2.00 & 22 & 13.30 & 2011 & NA & NA & tcga-18-3408-01a-01d-a320-08 & 24badf07-8615-48a3-8d6e-45c3d783f18d & 0153878436 & 08 & 0.08 & 19 & no & 08 & 1  & a320 & c  & 1.39 & 23 & 17.40 & 2013 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3408-01a-01d-0978-01 & 93fb49c4-6433-4061-a09c-1705f23eb3c2 & 0097557998 & 01 & 0.15 & 05 & no & 05 & 1 & 0978 & c & 1.01 & 22 & 6.70 & 2010 & 385.94 & dna & d & tcga-18-3408-01a-01d & 8b42a9a2-0540-4cfc-bf4c-82e38bbb1d8d & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3408-01a-01 & ec9f9a06-02a2-4ead-a470-9b7943aa18d1 & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3408-01a-01-ts1 & 4d318c39-fdb4-4dd3-959f-e6d3f54ce0b4 & tcga-18-3408-01a-01-ts1.4d318c39-fdb4-4dd3-959f-e6d3f54ce0b4.svs & no & NA & NA & NA & NA & NA & NA & 2  & NA & 0 & 8  & 90 & 90 & top & NA & tcga-18-3408-01a-01-bs1 & 6da3d3b3-e891-4922-9135-70fc9cf8959f & tcga-18-3408-01a-01-bs1.6da3d3b3-e891-4922-9135-70fc9cf8959f.svs & no & NA & NA & NA & NA & 3  & NA & 10 & NA & 0  & 10 & 80 & 85 & bottom & 96.00  & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t39.63.0.3 & nationwide children's hospital & 1 & lusc & 730b1e78-6f9a-45a6-a246-e473ae0116e3 & 10 & false & tcga & 2015 & NA & 74 & l-upper & NA & tcga-18-3409 & b09e872a-e837-49ec-8a27-84dcdcabf347 & l-upper & consented & canada & -34 & NA  & NA & lung basaloid squamous cell carcinoma                       & no & yes, history of prior malignancy & other & NA & NA & distal lung & NA & primary & 17 & -27154 & NA  & 0 & 2417 & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & not hispanic or latino & NA & yes & yes & NA & NA & tcga-18-3409-f72297 & 80ca78f2-8900-46d4-a4e1-88a0f74ae26a & 6  & NA & NA & NA & 3747 & 2291 & NA & additional new tumor event & complete remission/response & NA & no & 5  & new primary tumor & NA & yes & NA & tumor free & complete remission/response & NA & NA & no & no & alive & 2015 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & yes & NA & NA & tcga-18-3409-f68256 & fd959514-50e3-4a23-b6ac-9d7c16b48ef7 & 14 & NA & NA & NA   & 3747 & 2291 & NA & scheduled follow-up submission & complete remission/response & NA & no & 11 & new primary tumor       & NA & yes & NA & tumor free & complete remission/response & biopsy with histologic confirmation & NA & NA & no  & no  & alive & 2014 & male   & lung basaloid squamous cell carcinoma                       & no & c34.1 & 8083/3 & c34.1 & yes & NA & NA & NA & NA & NA              & 3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 20  & 3409 & NA & tumor free & NA & NA & NA & NA & NA & yes & no & white & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ia   & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t1 & 1974 & NA & no & yes & 18 & current reformed smoker for > 15 years      & lung & alive & 2011 & 2004 & NA   & case matched methylation control & NA & NA & canonical - plus & NA & consented & canada & NA & 2031 & 0   & NA & no & NA & NA & lung & lung basaloid squamous cell carcinoma                       & NA & no & NA & left  & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 47aa271a-73c5-427f-8797-68d80de3150a & 0   & NA & NA & NA & NA & tumor resection & NA & other & NA & 1 & l upper lobectomy            & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & upper & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & f27ece99-8357-4648-9d1d-5e4127968c5f & NA & 0  & 90 & NA & 45  & 1 & cryovial & 0 & NA & tcga-18-3409-11a & 47aa271a-73c5-427f-8797-68d80de3150a & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.4 & no & 0.8 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-11a-01w-0939-09 & d3a28178-f3a2-4066-8d64-b33bb28500c9 & 96492239 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & e & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-11a-01w-0938-08 & 0b5669e9-4993-4e1c-88ef-dc99faabd563 & 96485327 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & e & 40.00 & 22 & 80.00 & 2010 & 74.00  & repli-g (qiagen) dna & w & tcga-18-3409-11a-01w & c4f7e4a5-afec-449f-b6dc-b1e26c754d69 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3409-11a-01x-0941-09 & ec700741-a842-49b6-ab26-3419b292ff95 & 97543279 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & e & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3409-11a-01x-0940-08 & 0146db62-7be8-471f-a696-cefa0c9fdb22 & 97544143 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & e & 40.00 & 22 & 80.00 & 2010 & 110.00 & repli-g x (qiagen) dna & x & tcga-18-3409-11a-01x & 2b2ff034-5063-4851-ba4b-a14c7a7491e1 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-11a-01d-0979-05 & 6110efdd-d442-4a05-a7de-b66d802234a1 & 0097557297 & 05 & 0.14 & 05 & no & 05 & 6 & 0979 & e & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3409-11a-01d-0983-08 & c56da82c-b77a-44ef-a4c0-85de66f118cc & 0097556433 & 08 & 0.14 & 05 & no & 05 & 6 & 0983 & e & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3409-11a-01d-0984-09 & 86d7b61c-8485-4df6-87f4-cd8d88d6c9ed & 0097513329 & 09 & 0.14 & 05 & no & 05 & 6 & 0984 & e & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3409-11a-01d-1028-04 & a90f489a-8705-4a8f-aba6-0e9856f57502 & 0098888721 & 04 & 0.14 & 02 & no & 06 & 6 & 1028 & e & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3409-11a-01d-1029-06 & 4fa417d8-cd24-4b17-a1d4-c8b57b6582a8 & 0098847537 & 06 & 0.14 & 02 & no & 06 & 6 & 1029 & e & 1.86 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3409-11a-01d-1519-02 & b279abc8-cb02-4bc0-a711-3f0032bb0cdd & 1049949243 & 02 & 0.14 & 26 & no & 01 & 5 & 1519 & e & 1.86 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-11a-01d-0978-01 & 6ad9661d-0493-40d3-a539-893f41c9659e & 0097557969 & 01 & 0.14 & 05 & no & 05 & 6 & 0978 & e & 0.94 & 22 & 6.70 & 2010 & 30.64  & dna & d & tcga-18-3409-11a-01d & 282bc053-8138-4250-9feb-816ed7daf97d & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3409-11a-01 & f4a34fa6-a33c-49bd-9fa9-70d38a926cbe & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3409-11a-01-ts1 & 9bd57bcc-6f43-40ff-b8f5-ab028bc17e65 & tcga-18-3409-11a-01-ts1.9bd57bcc-6f43-40ff-b8f5-ab028bc17e65.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3409-11a-01-bs1 & db463d06-2f14-4cae-86f5-480d90f5fd2c & tcga-18-3409-11a-01-bs1.db463d06-2f14-4cae-86f5-480d90f5fd2c.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.2 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a & f27ece99-8357-4648-9d1d-5e4127968c5f & NA & NA & NA & NA & NA & NA & 3e32cc78-b066-4f85-a781-edaf39de0704 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.4 & no & 0.9 & NA & NA & NA & NA & tcga-18-3409.978339c6-aea3-4ac1-87db-40388a3adf39.pdf & 978339c6-aea3-4ac1-87db-40388a3adf39 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & tcga-18-3409-01a-01r-1030-03 & 51179ffd-4551-448e-bca0-cc9e46f57a7a & 0098887383 & 03 & 0.09 & 02 & no & 06 & 1 & 1030 & d & 1.50 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3409-01a-01r-1031-01 & 8032999c-8ea9-494b-831a-30089dc606aa & 0098888657 & 01 & 0.09 & 02 & no & 06 & 1 & 1031 & d & 0.79 & 22 & 8.80  & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01r-0980-07 & c26af662-dd33-4073-bd97-1f79da7bfa43 & 0097557879 & 07 & 0.09 & 05 & no & 05 & 1 & 0980 & d & 2.40 & 22 & 26.70 & 2010 & 0.00   & rna & r & tcga-18-3409-01a-01r & abf453fb-c147-42ae-8a7e-e972e8678245 & 0.09 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.80 & 8.50 & uv spec & NA & NA & NA & NA & NA & NA & NA                           & NA                                   & NA       & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & tcga-18-3409-01a-01t-0981-13 & a6d075bc-a4c1-4aad-9236-bc1581d84d7d & 0097557399 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & d & 1.10 & 22 & 11.00 & 2010 & 0.00  & total rna & t & tcga-18-3409-01a-01t & 6a953ae7-5057-48c9-a125-663f7deec176 & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01w-0939-09 & 8b94dc52-f6a9-4c06-9aa8-cefefeea31e1 & 96492246 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & e & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01w-0938-08 & 5e9f786c-c26b-421c-81d9-5ddb3a7bb9a1 & 96485334 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & e & 40.00 & 22 & 80.00 & 2010 & 105.00 & repli-g (qiagen) dna & w & tcga-18-3409-01a-01w & 773f84c6-98cd-4d9f-8830-b9f9f7a873b0 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01x-0941-09 & c689d44e-24dc-4dd7-b13e-afc25594e166 & 97543286 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & e & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3409-01a-01x-0940-08 & 79df3f4f-46c0-47d9-ae5c-071b424ce5ca & 97544150 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & e & 40.00 & 22 & 80.00 & 2010 & 71.00 & repli-g x (qiagen) dna & x & tcga-18-3409-01a-01x & 1ad1632c-4299-4172-a296-63d48df7f625 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01d-0979-05 & a0f5f4f4-9d89-4b67-afc7-9d5bd90795da & 0097557303 & 05 & 0.14 & 05 & no & 05 & 1 & 0979 & d & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3409-01a-01d-0983-08 & aa733cb0-37a9-4fef-8d40-d57596ce9e51 & 0097555459 & 08 & 0.14 & 05 & no & 05 & 1 & 0983 & d & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3409-01a-01d-0984-09 & 0acfcd32-4bf2-4264-864e-30a268a6ac71 & 0097513335 & 09 & 0.14 & 05 & no & 05 & 1 & 0984 & d & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3409-01a-01d-1028-04 & f2c09e84-4e61-463d-a992-7b36374b3e4c & 0098888727 & 04 & 0.14 & 02 & no & 06 & 1 & 1028 & d & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3409-01a-01d-1029-06 & 46c07bef-ac8b-4e40-9c4d-b8f6d1dc5d68 & 0098847543 & 06 & 0.14 & 02 & no & 06 & 1 & 1029 & d & 1.86 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3409-01a-01d-1519-02 & dd92c79c-3087-4c19-b95a-619a62f07bfc & 1049949238 & 02 & 0.14 & 26 & no & 01 & 1 & 1519 & d & 1.86 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3409-01a-01d-0978-01 & 2bf1bed5-f248-4e23-9586-c015d4f933bb & 0097557975 & 01 & 0.14 & 05 & no & 05 & 1 & 0978 & d & 0.94 & 22 & 6.70 & 2010 & 104.64 & dna & d & tcga-18-3409-01a-01d & 4ca35e4f-9ad3-44d7-a11d-f488f0e52e58 & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3409-01a-01 & 178f51e8-ef8a-408c-af4d-757fb5eaa670 & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3409-01a-01-ts1 & a0d1ce11-2dac-48be-97bc-eb226c51190c & tcga-18-3409-01a-01-ts1.a0d1ce11-2dac-48be-97bc-eb226c51190c.svs & no & NA & NA & NA & NA & NA & NA & 5  & NA & 0 & 25 & 70 & 80 & top & NA & tcga-18-3409-01a-01-bs1 & df6b80d8-7ae7-47ae-b674-5e030cfc4999 & tcga-18-3409-01a-01-bs1.df6b80d8-7ae7-47ae-b674-5e030cfc4999.svs & no & NA & NA & NA & NA & NA & NA & 20 & NA & 0  & 20 & 60 & 70 & bottom & 69.00  & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.2 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t39.63.0.4 & nationwide children's hospital & 1 & lusc & 293cc082-7d2c-416a-be35-062bb8ee1b87 & 10 & false & tcga & 2015 & NA & 81 & r-lower & NA & tcga-18-3410 & 99599b60-4f5c-456b-8755-371b1aa7074e & r-lower & deceased  & canada & NA  & 146 & NA & lung squamous cell carcinoma- not otherwise specified (nos) & no & yes, history of prior malignancy & other & NA & NA & r unknown   & NA & primary & 4  & -29827 & 146 & 0 & NA   & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                     & NA & NA  & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA   & NA & NA                         & NA                          & NA & NA & NA & NA                & NA & NA  & NA & NA         & NA                          & NA & NA & NA & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA  & NA & NA & tcga-18-3410-f69095 & 7d67f4d1-1f56-43bf-b130-0d70280892d9 & 29 & NA & NA & 146  & NA   & NA   & NA & scheduled follow-up submission & NA                          & 0  & no & 12 & NA                      & NA & no  & NA & tumor free & NA                          & NA                                  & NA & NA & no  & no  & dead  & 2014 & male   & lung squamous cell carcinoma- not otherwise specified (nos) & no & c34.3 & 8070/3 & c34.3 & yes & 0  & NA & NA & NA & peripheral lung & 4 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & 3410 & NA & tumor free & NA & NA & NA & NA & NA & yes & NA & NA    & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iib  & NA & NA & NA & NA  & NA & NA & NA & m0 & n0 & t3 & NA   & NA & no & yes & 18 & current reformed smoker for > 15 years      & lung & dead  & 2011 & 2004 & NA   & case matched methylation control & NA & NA & canonical - plus & NA & deceased  & canada & NA & 2024 & 0   & NA & no & NA & NA & lung & lung squamous cell carcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & addd6bc0-4736-47c6-8925-abe527792391 & 0   & NA & NA & NA & NA & tumor resection & NA & other & NA & 1 & r lower lobectomy            & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & lower & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & cdffec20-96a8-4b64-9959-f701504ad186 & NA & 5  & 80 & NA & 68  & 1 & cryovial & 0 & NA & tcga-18-3410-11a & addd6bc0-4736-47c6-8925-abe527792391 & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.7 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-11a-01w-0939-09 & 6b7ebeb7-6b81-4919-ae48-c6c97a224160 & 96492230 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & f & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-11a-01w-0938-08 & 3a9797c4-9e5f-4bda-9444-59fdf5c8be5a & 96485318 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & f & 40.00 & 22 & 80.00 & 2010 & 71.00  & repli-g (qiagen) dna & w & tcga-18-3410-11a-01w & e0955263-20b1-4e39-84eb-6c7ac543fb30 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3410-11a-01x-0941-09 & 21057820-c3ae-41f8-b375-b8d225b973c0 & 97543270 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & f & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3410-11a-01x-0940-08 & 62b35495-00ad-43ea-9ccf-90723fc4deea & 97544134 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & f & 40.00 & 22 & 80.00 & 2010 & 103.00 & repli-g x (qiagen) dna & x & tcga-18-3410-11a-01x & a709a9f4-76db-4b1f-872d-b7ce1f9b9d0d & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-11a-01d-0979-05 & 0bdc33ec-6ca1-4538-8463-5317ca4ec826 & 0097557284 & 05 & 0.14 & 05 & no & 05 & 6 & 0979 & f & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3410-11a-01d-0983-08 & 8a207aee-58f5-439c-8445-f42e9f27023f & 0097556420 & 08 & 0.14 & 05 & no & 05 & 6 & 0983 & f & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3410-11a-01d-0984-09 & 96dc4705-cdfb-4a60-bca3-d757bae9e6df & 0097513316 & 09 & 0.14 & 05 & no & 05 & 6 & 0984 & f & 0.94 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3410-11a-01d-1028-04 & 8ec388d9-adf1-4588-bcdc-b5a92c7f1d89 & 0098888862 & 04 & 0.14 & 02 & no & 06 & 6 & 1028 & f & 3.74 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3410-11a-01d-1029-06 & 83d0bb71-4e12-481c-b638-5987fa4f165b & 0098847524 & 06 & 0.14 & 02 & no & 06 & 6 & 1029 & f & 1.86 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3410-11a-01d-1519-02 & e99e93c9-a59d-4a6e-963c-3d4bd5450593 & 1049949258 & 02 & 0.14 & 26 & no & 01 & 5 & 1519 & f & 1.86 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-11a-01d-0978-01 & 213a3dc3-8e9e-43dd-b7cc-f050449414b7 & 0097557956 & 01 & 0.14 & 05 & no & 05 & 6 & 0978 & f & 0.94 & 22 & 6.70 & 2010 & 63.64  & dna & d & tcga-18-3410-11a-01d & 444a4485-1e2b-4ed7-a9a0-2b483ba900da & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3410-11a-01 & 582f50ad-aa1f-416d-b936-b5cda9a83848 & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3410-11a-01-ts1 & 52656dfe-d9fe-4d19-baf9-b5d8a2c72631 & tcga-18-3410-11a-01-ts1.52656dfe-d9fe-4d19-baf9-b5d8a2c72631.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3410-11a-01-bs1 & 69eef7c1-1705-4ee6-9d87-d77b0b5062e4 & tcga-18-3410-11a-01-bs1.69eef7c1-1705-4ee6-9d87-d77b0b5062e4.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.5 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a & cdffec20-96a8-4b64-9959-f701504ad186 & NA & NA & NA & NA & NA & NA & db186d75-4aee-4e1b-83d5-5a1970f03581 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.4 & no & 0.5 & NA & NA & NA & NA & tcga-18-3410.956030d1-c74c-4bb6-93d9-04eb5c786905.pdf & 956030d1-c74c-4bb6-93d9-04eb5c786905 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & tcga-18-3410-01a-01r-1030-03 & effdf606-892e-4120-9d7f-de1ecc0a1a41 & 0098887382 & 03 & 0.15 & 02 & no & 06 & 1 & 1030 & e & 2.51 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3410-01a-01r-1031-01 & be7878de-22cb-48e2-9760-68b984833a90 & 0098889302 & 01 & 0.15 & 02 & no & 06 & 1 & 1031 & e & 3.00 & 22 & 20.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01r-0980-07 & 388eaf67-9d6d-4b9a-b404-a4ce171fbd18 & 0097557878 & 07 & 0.15 & 05 & no & 05 & 1 & 0980 & e & 4.01 & 22 & 26.70 & 2010 & 101.60 & rna & r & tcga-18-3410-01a-01r & 353d3332-0670-4546-9eec-c33c138ce3af & 0.15 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.30 & 8.60 & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01t-1032-07 & 2f372e29-eeb3-4d7f-ba17-b1c2910ae6f2 & 98848598 & 07 & 0.10 & 02 & no & 06 & 1  & 1032 & e  & 1.00 & 22 & 10.00 & 2010 & NA & NA & tcga-18-3410-01a-01t-0981-13 & 2ffd4532-d9f0-453d-937f-c3c8c47711e3 & 0097557398 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & e & 3.00 & 22 & 30.00 & 2010 & 2.00  & total rna & t & tcga-18-3410-01a-01t & 213eade9-b061-4038-8f7f-4624afc50110 & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01w-0939-09 & d99fd10e-9f45-4d88-b382-7c7e4cf8ad2b & 96492223 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & f & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01w-0938-08 & 489a9ac2-ea51-4b5a-ac01-e1750ce38c6a & 96485311 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & f & 40.00 & 22 & 80.00 & 2010 & 51.00  & repli-g (qiagen) dna & w & tcga-18-3410-01a-01w & 5c99ffcc-abe9-41a4-a41b-c700ee3fd65d & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01x-0941-09 & 679fb0a3-2dad-4121-81c0-b59307a743cc & 97543263 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & f & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3410-01a-01x-0940-08 & e2d147c5-9d8d-4225-aede-9e9c9f62b7f7 & 97544127 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & f & 40.00 & 22 & 80.00 & 2010 & 94.00 & repli-g x (qiagen) dna & x & tcga-18-3410-01a-01x & 2705691c-561c-46f5-8d58-04a90aa81f68 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01d-0979-05 & 2185e1bd-8e07-418d-8304-ab5cfe8b7185 & 0097557302 & 05 & 0.15 & 05 & no & 05 & 1 & 0979 & e & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3410-01a-01d-0983-08 & 7e6382c3-368a-43a5-9812-c58f54ceba3f & 0097555449 & 08 & 0.15 & 05 & no & 05 & 1 & 0983 & e & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3410-01a-01d-0984-09 & c458b70d-2f55-4c9b-b760-c7e3ee76e22a & 0097513334 & 09 & 0.15 & 05 & no & 05 & 1 & 0984 & e & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3410-01a-01d-1028-04 & 05071e70-f5e2-4500-a02f-6d79b586ab7b & 0098888726 & 04 & 0.15 & 02 & no & 06 & 1 & 1028 & e & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3410-01a-01d-1029-06 & dd36dda0-e4cf-4a8f-bc39-6f4ec3ecfa3b & 0098842894 & 06 & 0.15 & 02 & no & 06 & 1 & 1029 & e & 2.00 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3410-01a-01d-1519-02 & 68208823-dc93-47f9-8379-709e0bef822f & 1049949239 & 02 & 0.15 & 26 & no & 01 & 1 & 1519 & e & 2.00 & 22 & 13.30 & 2011 & NA & NA & tcga-18-3410-01a-01d-1969-01 & a658ea90-4f4c-41d5-9901-f514c1918949 & 1053936483 & 01 & 0.15 & 22 & no & 08 & 1  & 1969 & b  & 1.00 & 22 & 6.67  & 2011 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3410-01a-01d-0978-01 & 93e198f3-7d56-42c2-b66d-209c2181c8d0 & 0097557974 & 01 & 0.15 & 05 & no & 05 & 1 & 0978 & e & 1.01 & 22 & 6.70 & 2010 & 216.97 & dna & d & tcga-18-3410-01a-01d & c973496e-4ea7-4903-984d-8118675f40f8 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3410-01a-01 & c1ace207-af19-4894-a0e6-5da80fb7fb5c & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3410-01a-01-ts1 & 45efb645-1426-47fb-90ff-eebc36fb9c3c & tcga-18-3410-01a-01-ts1.45efb645-1426-47fb-90ff-eebc36fb9c3c.svs & no & NA & NA & NA & NA & 10 & NA & 5  & NA & 0 & 10 & 85 & 80 & top & NA & tcga-18-3410-01a-01-bs1 & 09c41a54-9a42-4031-ab05-6f955e109ba5 & tcga-18-3410-01a-01-bs1.09c41a54-9a42-4031-ab05-6f955e109ba5.svs & no & NA & NA & NA & NA & 5  & NA & 5  & NA & 0  & 5  & 90 & 90 & bottom & 92.00  & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.3 & NA & NA & NA & NA & NA & a\\\\\n",
       "\t39.63.0.5 & nationwide children's hospital & 1 & lusc & 5eee6807-0ee7-4ad7-84a4-f36552a70695 & 10 & false & tcga & 2015 & NA & 63 & r-upper & NA & tcga-18-3411 & f1a1bbf9-4751-4fb4-8a2b-19f8d4ba57bd & r-upper & consented & canada & 15  & NA  & NA & lung squamous cell carcinoma- not otherwise specified (nos) & no & no                               & lung  & NA & NA & NA          & NA & primary & 17 & -23370 & NA  & 0 & 1415 & NA & lung squamous cell carcinoma & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                     & NA & NA  & NA  & NA & NA & NA                  & NA                                   & NA & NA & NA & NA & NA   & NA   & NA & NA                         & NA                          & NA & NA & NA & NA                & NA & NA  & NA & NA         & NA                          & NA & NA & NA & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA  & NA & NA & tcga-18-3411-f68326 & 2d43d769-e282-4a55-9d2a-68893d9f914a & 14 & NA & NA & NA   & 3576 & NA   & NA & scheduled follow-up submission & NA                          & NA & no & 11 & NA                      & NA & NA  & NA & NA         & NA                          & NA                                  & NA & NA & yes & yes & alive & 2014 & female & lung squamous cell carcinoma- not otherwise specified (nos) & no & c34.1 & 8070/3 & c34.1 & yes & NA & NA & NA & NA & NA              & 3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 50  & 3411 & NA & tumor free & NA & NA & NA & NA & NA & no  & NA & NA    & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & r0 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iiia & NA & NA & NA & NA  & NA & NA & NA & m0 & n2 & t2 & NA   & NA & no & yes & 18 & current smoker                              & lung & alive & 2011 & 2005 & NA   & case matched methylation control & NA & NA & canonical - plus & NA & consented & canada & NA & 1835 & 36  & NA & no & NA & NA & lung & lung squamous cell carcinoma- not otherwise specified (nos) & NA & no & NA & right & NA & other method (please specify) & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & c2b83333-3dd9-4ac5-969f-4b6afc4416b3 & 36  & NA & NA & NA & NA & tumor resection & NA & lung  & NA & 1 & lobectomy                    & NA & yes & yes & yes & yes & NA & NA & NA & yes & lung & upper & NA & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & f85af662-1de5-42a8-a22d-721a9ad00ce6 & NA & 5  & 85 & NA & 89  & 1 & cryovial & 0 & NA & tcga-18-3411-11a & c2b83333-3dd9-4ac5-969f-4b6afc4416b3 & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.9 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-11a-01w-0939-09 & 8e985ba3-a0a3-44f6-b013-f2d204520f7b & 96491974 & 09 & 0.50 & 19 & no & 05 & 8 & 0939 & g & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-11a-01w-0938-08 & 8b46c8d4-b471-419c-91a5-13074f94c0b5 & 96485303 & 08 & 0.50 & 19 & no & 05 & 8 & 0938 & g & 40.00 & 22 & 80.00 & 2010 & 109.00 & repli-g (qiagen) dna & w & tcga-18-3411-11a-01w & dc19ba3c-a178-49ce-ac4f-c8682982df0b & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3411-11a-01x-0941-09 & 6cf88e29-b813-4a9a-90a6-cdaa810bab2f & 97543255 & 09 & 0.50 & 19 & no & 05 & 8 & 0941 & g & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3411-11a-01x-0940-08 & aa2a6b1f-2f3b-4d0e-8a80-309bdf81b806 & 97544120 & 08 & 0.50 & 19 & no & 05 & 8 & 0940 & g & 40.00 & 22 & 80.00 & 2010 & 107.00 & repli-g x (qiagen) dna & x & tcga-18-3411-11a-01x & 08788157-650d-4805-93e2-bc6bf4782a01 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-11a-01d-0979-05 & aa619e10-f65c-431c-a4d3-3d183976df54 & 0097557273 & 05 & 0.15 & 05 & no & 05 & 6 & 0979 & g & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3411-11a-01d-0983-08 & 5d18da80-61b7-438b-9ad1-ac2e98d0e820 & 0097556409 & 08 & 0.15 & 05 & no & 05 & 6 & 0983 & g & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3411-11a-01d-0984-09 & 8ff2ba44-e8ca-4b28-aa6a-41e7061bb464 & 0097513305 & 09 & 0.15 & 05 & no & 05 & 6 & 0984 & g & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3411-11a-01d-1028-04 & f31af08b-391c-4ba9-a854-7be99e9c0450 & 0098888697 & 04 & 0.15 & 02 & no & 06 & 6 & 1028 & g & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3411-11a-01d-1029-06 & c6b6a2a7-f772-41a3-99d6-5f6cf0a3d0a8 & 0098847513 & 06 & 0.15 & 02 & no & 06 & 6 & 1029 & g & 2.00 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3411-11a-01d-1519-02 & 89239c5c-068b-46ce-9cc2-7bed5dd1d92a & 1049949267 & 02 & 0.15 & 26 & no & 01 & 5 & 1519 & g & 2.00 & 22 & 13.30 & 2011 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-11a-01d-0978-01 & 68abdd72-6bab-4162-bf2c-62dad6721bde & 0097557945 & 01 & 0.15 & 05 & no & 05 & 6 & 0978 & g & 1.01 & 22 & 6.70 & 2010 & 121.64 & dna & d & tcga-18-3411-11a-01d & 1320f098-0f01-4c7a-ac42-93fedaa58181 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3411-11a-01 & bd7d4527-bb24-410d-bf76-01f456242e4b & 30 & no & NA & 11 & 01 & 1 & NA & NA & tcga-18-3411-11a-01-ts1 & 16765476-ae6c-4777-9d27-768bd0fd7d84 & tcga-18-3411-11a-01-ts1.16765476-ae6c-4777-9d27-768bd0fd7d84.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & top & NA & tcga-18-3411-11a-01-bs1 & b9f8d2b9-6e15-4e85-807c-3b3b12584111 & tcga-18-3411-11a-01-bs1.b9f8d2b9-6e15-4e85-807c-3b3b12584111.svs & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & bottom & 1.00 & 2012 & NA & solid tissue normal & 11 & 0.3 & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a & f85af662-1de5-42a8-a22d-721a9ad00ce6 & NA & NA & NA & NA & NA & NA & d9228674-5b13-44cb-a7ae-c64fd97a3bbb & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 0.5 & no & 0.6 & NA & NA & NA & NA & tcga-18-3411.03efa03c-f8f9-43c3-9244-265bc02cbb5c.pdf & 03efa03c-f8f9-43c3-9244-265bc02cbb5c & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 1.80 & NA & NA & NA & tcga-18-3411-01a-01r-1030-03 & 5f043518-1153-49a4-a397-ce88cf6d3fe2 & 0098887359 & 03 & 0.14 & 02 & no & 06 & 1 & 1030 & f & 2.34 & 22 & 16.70 & 2010 & NA & NA & tcga-18-3411-01a-01r-1031-01 & 56b2e492-7ba4-4c00-bbb2-bb2cdd7a2c3f & 0098889279 & 01 & 0.14 & 02 & no & 06 & 1 & 1031 & f & 2.80 & 22 & 20.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01r-0980-07 & bc493a57-6821-4607-a841-5ffc39391910 & 0097557855 & 07 & 0.14 & 05 & no & 05 & 1 & 0980 & f & 3.74 & 22 & 26.70 & 2010 & 106.60 & rna & r & tcga-18-3411-01a-01r & 9c0217af-b7d3-40fb-b2e0-b84370e65408 & 0.14 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & allprep rna extraction & NA & NA & 1.70 & 8.90 & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01t-1032-07 & 7ee9b1e7-515a-47fd-b62c-7f14c5bb0061 & 98848575 & 07 & 0.10 & 02 & no & 06 & 1  & 1032 & f  & 1.00 & 22 & 10.00 & 2010 & NA & NA & tcga-18-3411-01a-01t-0981-13 & bd823994-24e9-4ad7-9575-427e79d83d3b & 0097557375 & 13 & 0.10 & 05 & no & 05 & 1 & 0981 & f & 3.00 & 22 & 30.00 & 2010 & 4.00  & total rna & t & tcga-18-3411-01a-01t & 167e3c94-f276-4fda-aafe-948d69a1a794 & 0.10 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & total rna & NA & NA & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01w-0939-09 & 7cfefca3-0736-4221-b53c-29736a6e9690 & 96491948 & 09 & 0.50 & 19 & no & 05 & 1 & 0939 & g & 40.00 & 22 & 80.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01w-0938-08 & 2c372efb-9469-422b-a1d6-3736a699a05f & 96485310 & 08 & 0.50 & 19 & no & 05 & 1 & 0938 & g & 40.00 & 22 & 80.00 & 2010 & 83.00  & repli-g (qiagen) dna & w & tcga-18-3411-01a-01w & b97b0a2b-2dab-4d93-894a-d0f1eef27236 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & uv spec & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01x-0941-09 & bf1c38f9-86c8-49ca-b536-b3f47bd0fe17 & 97543262 & 09 & 0.50 & 19 & no & 05 & 1 & 0941 & g & 40.00 & 22 & 80.00 & 2010 & NA & NA & tcga-18-3411-01a-01x-0940-08 & c3bae3df-2a3b-4c9b-9e12-079f1606b202 & 97544126 & 08 & 0.50 & 19 & no & 05 & 1 & 0940 & g & 40.00 & 22 & 80.00 & 2010 & 44.00 & repli-g x (qiagen) dna & x & tcga-18-3411-01a-01x & d175094b-8749-4589-adf9-d4f32fb7a9c3 & 0.50 & yes & no & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g x & NA & NA & uv spec & NA & NA & 1.90 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01d-0979-05 & aa64d38d-88fd-4853-8acd-2e108a103d16 & 0097557279 & 05 & 0.15 & 05 & no & 05 & 1 & 0979 & f & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3411-01a-01d-0983-08 & 6a9cc303-c7fd-4f40-8933-1636dea99252 & 0097556415 & 08 & 0.15 & 05 & no & 05 & 1 & 0983 & f & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3411-01a-01d-0984-09 & e1a8da63-6915-40d7-bd2d-409179667e52 & 0097513311 & 09 & 0.15 & 05 & no & 05 & 1 & 0984 & f & 1.01 & 22 & 6.70 & 2010 & NA & NA & tcga-18-3411-01a-01d-1028-04 & f7759507-b07e-4d6d-933d-7761b797a608 & 0098888703 & 04 & 0.15 & 02 & no & 06 & 1 & 1028 & f & 4.01 & 22 & 26.70 & 2010 & NA & NA & tcga-18-3411-01a-01d-1029-06 & 3f100b8e-a8a8-4aa5-a948-da0f95e6a0e3 & 0098847519 & 06 & 0.15 & 02 & no & 06 & 1 & 1029 & f & 2.00 & 22 & 13.30 & 2010 & NA & NA & tcga-18-3411-01a-01d-1519-02 & 6f402d78-4826-47f1-9b0a-72b6467c44f9 & 1049949262 & 02 & 0.15 & 26 & no & 01 & 1 & 1519 & f & 2.00 & 22 & 13.30 & 2011 & NA & NA & tcga-18-3411-01a-01d-1969-01 & 8f0906f1-4501-49f2-aa8f-78d4b0893e4c & 1053936484 & 01 & 0.15 & 22 & no & 08 & 1  & 1969 & c  & 1.00 & 22 & 6.67  & 2011 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-18-3411-01a-01d-0978-01 & 34576e6b-4833-4f90-97a0-3ffb0b2d249c & 0097557951 & 01 & 0.15 & 05 & no & 05 & 1 & 0978 & f & 1.01 & 22 & 6.70 & 2010 & 157.97 & dna & d & tcga-18-3411-01a-01d & 87d5aa26-a55e-4f61-95d4-3dc5a06fe708 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & uv spec & NA & NA & tcga-18-3411-01a-01 & 1aed9853-db04-4637-b74b-ca0a73460b68 & 29 & no & NA & 01 & 01 & NA & NA & tcga-18-3411-01a-01-ts1 & c1a2993d-f8b2-4876-9af6-d481daae85de & tcga-18-3411-01a-01-ts1.c1a2993d-f8b2-4876-9af6-d481daae85de.svs & no & NA & NA & NA & NA & 10 & NA & 1  & NA & 0 & 9  & 90 & 85 & top & NA & tcga-18-3411-01a-01-bs1 & 45372e19-1f8e-4d40-8df1-76ed5e0d8bae & tcga-18-3411-01a-01-bs1.45372e19-1f8e-4d40-8df1-76ed5e0d8bae.svs & no & NA & NA & NA & NA & 15 & NA & 5  & NA & 30 & 10 & 55 & 70 & bottom & 132.00 & 2010 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & 0.5 & NA & NA & NA & NA & NA & a\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 2698\n",
       "\n",
       "| <!--/--> | admin.bcr &lt;chr&gt; | admin.day_of_dcc_upload &lt;chr&gt; | admin.disease_code &lt;chr&gt; | admin.file_uuid &lt;chr&gt; | admin.month_of_dcc_upload &lt;chr&gt; | admin.patient_withdrawal.withdrawn &lt;chr&gt; | admin.project_code &lt;chr&gt; | admin.year_of_dcc_upload &lt;chr&gt; | patient.additional_studies &lt;chr&gt; | patient.age_at_initial_pathologic_diagnosis &lt;chr&gt; | ⋯ ⋯ | patient.samples.sample.preservation_method &lt;chr&gt; | patient.samples.sample.sample_type &lt;chr&gt; | patient.samples.sample.sample_type_id &lt;chr&gt; | patient.samples.sample.shortest_dimension &lt;chr&gt; | patient.samples.sample.time_between_clamping_and_freezing &lt;chr&gt; | patient.samples.sample.time_between_excision_and_freezing &lt;chr&gt; | patient.samples.sample.tissue_type &lt;chr&gt; | patient.samples.sample.tumor_descriptor &lt;chr&gt; | patient.samples.sample.tumor_pathology &lt;chr&gt; | patient.samples.sample.vial_number &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| 39.63.0 | nationwide children's hospital | 1 | lusc | b34e8d44-9f2d-4ab3-a189-ecb5b77af723 | 10 | false | tcga | 2015 | NA | 67 | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 39.63.0.1 | nationwide children's hospital | 1 | lusc | 2fc3e9ed-7219-4d78-8964-a5c9de65b8da | 10 | false | tcga | 2015 | NA | 72 | ⋯ | NA | primary tumor | 01 | 0.2 | NA | NA | NA | NA | NA | a |\n",
       "| 39.63.0.2 | nationwide children's hospital | 1 | lusc | 1af63f26-0d2a-463b-ba33-7b60ab118b50 | 10 | false | tcga | 2015 | NA | 77 | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 39.63.0.3 | nationwide children's hospital | 1 | lusc | 730b1e78-6f9a-45a6-a246-e473ae0116e3 | 10 | false | tcga | 2015 | NA | 74 | ⋯ | NA | primary tumor | 01 | 0.2 | NA | NA | NA | NA | NA | a |\n",
       "| 39.63.0.4 | nationwide children's hospital | 1 | lusc | 293cc082-7d2c-416a-be35-062bb8ee1b87 | 10 | false | tcga | 2015 | NA | 81 | ⋯ | NA | primary tumor | 01 | 0.3 | NA | NA | NA | NA | NA | a |\n",
       "| 39.63.0.5 | nationwide children's hospital | 1 | lusc | 5eee6807-0ee7-4ad7-84a4-f36552a70695 | 10 | false | tcga | 2015 | NA | 63 | ⋯ | NA | primary tumor | 01 | 0.5 | NA | NA | NA | NA | NA | a |\n",
       "\n"
      ],
      "text/plain": [
       "          admin.bcr                      admin.day_of_dcc_upload\n",
       "39.63.0   nationwide children's hospital 1                      \n",
       "39.63.0.1 nationwide children's hospital 1                      \n",
       "39.63.0.2 nationwide children's hospital 1                      \n",
       "39.63.0.3 nationwide children's hospital 1                      \n",
       "39.63.0.4 nationwide children's hospital 1                      \n",
       "39.63.0.5 nationwide children's hospital 1                      \n",
       "          admin.disease_code admin.file_uuid                     \n",
       "39.63.0   lusc               b34e8d44-9f2d-4ab3-a189-ecb5b77af723\n",
       "39.63.0.1 lusc               2fc3e9ed-7219-4d78-8964-a5c9de65b8da\n",
       "39.63.0.2 lusc               1af63f26-0d2a-463b-ba33-7b60ab118b50\n",
       "39.63.0.3 lusc               730b1e78-6f9a-45a6-a246-e473ae0116e3\n",
       "39.63.0.4 lusc               293cc082-7d2c-416a-be35-062bb8ee1b87\n",
       "39.63.0.5 lusc               5eee6807-0ee7-4ad7-84a4-f36552a70695\n",
       "          admin.month_of_dcc_upload admin.patient_withdrawal.withdrawn\n",
       "39.63.0   10                        false                             \n",
       "39.63.0.1 10                        false                             \n",
       "39.63.0.2 10                        false                             \n",
       "39.63.0.3 10                        false                             \n",
       "39.63.0.4 10                        false                             \n",
       "39.63.0.5 10                        false                             \n",
       "          admin.project_code admin.year_of_dcc_upload\n",
       "39.63.0   tcga               2015                    \n",
       "39.63.0.1 tcga               2015                    \n",
       "39.63.0.2 tcga               2015                    \n",
       "39.63.0.3 tcga               2015                    \n",
       "39.63.0.4 tcga               2015                    \n",
       "39.63.0.5 tcga               2015                    \n",
       "          patient.additional_studies\n",
       "39.63.0   NA                        \n",
       "39.63.0.1 NA                        \n",
       "39.63.0.2 NA                        \n",
       "39.63.0.3 NA                        \n",
       "39.63.0.4 NA                        \n",
       "39.63.0.5 NA                        \n",
       "          patient.age_at_initial_pathologic_diagnosis ⋯\n",
       "39.63.0   67                                          ⋯\n",
       "39.63.0.1 72                                          ⋯\n",
       "39.63.0.2 77                                          ⋯\n",
       "39.63.0.3 74                                          ⋯\n",
       "39.63.0.4 81                                          ⋯\n",
       "39.63.0.5 63                                          ⋯\n",
       "          patient.samples.sample.preservation_method\n",
       "39.63.0   NA                                        \n",
       "39.63.0.1 NA                                        \n",
       "39.63.0.2 NA                                        \n",
       "39.63.0.3 NA                                        \n",
       "39.63.0.4 NA                                        \n",
       "39.63.0.5 NA                                        \n",
       "          patient.samples.sample.sample_type\n",
       "39.63.0   primary tumor                     \n",
       "39.63.0.1 primary tumor                     \n",
       "39.63.0.2 primary tumor                     \n",
       "39.63.0.3 primary tumor                     \n",
       "39.63.0.4 primary tumor                     \n",
       "39.63.0.5 primary tumor                     \n",
       "          patient.samples.sample.sample_type_id\n",
       "39.63.0   01                                   \n",
       "39.63.0.1 01                                   \n",
       "39.63.0.2 01                                   \n",
       "39.63.0.3 01                                   \n",
       "39.63.0.4 01                                   \n",
       "39.63.0.5 01                                   \n",
       "          patient.samples.sample.shortest_dimension\n",
       "39.63.0   0.3                                      \n",
       "39.63.0.1 0.2                                      \n",
       "39.63.0.2 0.3                                      \n",
       "39.63.0.3 0.2                                      \n",
       "39.63.0.4 0.3                                      \n",
       "39.63.0.5 0.5                                      \n",
       "          patient.samples.sample.time_between_clamping_and_freezing\n",
       "39.63.0   NA                                                       \n",
       "39.63.0.1 NA                                                       \n",
       "39.63.0.2 NA                                                       \n",
       "39.63.0.3 NA                                                       \n",
       "39.63.0.4 NA                                                       \n",
       "39.63.0.5 NA                                                       \n",
       "          patient.samples.sample.time_between_excision_and_freezing\n",
       "39.63.0   NA                                                       \n",
       "39.63.0.1 NA                                                       \n",
       "39.63.0.2 NA                                                       \n",
       "39.63.0.3 NA                                                       \n",
       "39.63.0.4 NA                                                       \n",
       "39.63.0.5 NA                                                       \n",
       "          patient.samples.sample.tissue_type\n",
       "39.63.0   NA                                \n",
       "39.63.0.1 NA                                \n",
       "39.63.0.2 NA                                \n",
       "39.63.0.3 NA                                \n",
       "39.63.0.4 NA                                \n",
       "39.63.0.5 NA                                \n",
       "          patient.samples.sample.tumor_descriptor\n",
       "39.63.0   NA                                     \n",
       "39.63.0.1 NA                                     \n",
       "39.63.0.2 NA                                     \n",
       "39.63.0.3 NA                                     \n",
       "39.63.0.4 NA                                     \n",
       "39.63.0.5 NA                                     \n",
       "          patient.samples.sample.tumor_pathology\n",
       "39.63.0   NA                                    \n",
       "39.63.0.1 NA                                    \n",
       "39.63.0.2 NA                                    \n",
       "39.63.0.3 NA                                    \n",
       "39.63.0.4 NA                                    \n",
       "39.63.0.5 NA                                    \n",
       "          patient.samples.sample.vial_number\n",
       "39.63.0   a                                 \n",
       "39.63.0.1 a                                 \n",
       "39.63.0.2 a                                 \n",
       "39.63.0.3 a                                 \n",
       "39.63.0.4 a                                 \n",
       "39.63.0.5 a                                 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'lusc'\n",
    "df = clinical_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### MESO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>87</li>\n",
       "\t<li>894</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 87\n",
       "\\item 894\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 87\n",
       "2. 894\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]  87 894"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 894</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>admin.bcr</th><th scope=col>admin.day_of_dcc_upload</th><th scope=col>admin.disease_code</th><th scope=col>admin.file_uuid</th><th scope=col>admin.month_of_dcc_upload</th><th scope=col>admin.patient_withdrawal.withdrawn</th><th scope=col>admin.project_code</th><th scope=col>admin.year_of_dcc_upload</th><th scope=col>patient.additional_studies</th><th scope=col>patient.asbestos_exposure_age</th><th scope=col>⋯</th><th scope=col>patient.samples.sample.preservation_method</th><th scope=col>patient.samples.sample.sample_type</th><th scope=col>patient.samples.sample.sample_type_id</th><th scope=col>patient.samples.sample.shortest_dimension</th><th scope=col>patient.samples.sample.time_between_clamping_and_freezing</th><th scope=col>patient.samples.sample.time_between_excision_and_freezing</th><th scope=col>patient.samples.sample.tissue_type</th><th scope=col>patient.samples.sample.tumor_descriptor</th><th scope=col>patient.samples.sample.tumor_pathology</th><th scope=col>patient.samples.sample.vial_number</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>⋯</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>418.36.0</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>be213c85-e379-42fc-918a-1fe13c4d0201</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>418.36.0.1</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>7a8ecc55-002e-4650-98f8-84806b919c17</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>418.36.0.2</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>eb988691-41bd-412e-9e6c-82bd7050dc1b</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>418.36.0.3</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>eb48bed3-00e5-45aa-8803-6ce81c7b1fff</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>25</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>418.36.0.4</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>07b76d74-59b8-4860-97b9-cef6a238d6a6</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>418.36.0.5</th><td>nationwide children's hospital</td><td>1</td><td>meso</td><td>f352e491-5cbd-4c60-8ada-0d12ea6218eb</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 894\n",
       "\\begin{tabular}{r|llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll}\n",
       "  & admin.bcr & admin.day\\_of\\_dcc\\_upload & admin.disease\\_code & admin.file\\_uuid & admin.month\\_of\\_dcc\\_upload & admin.patient\\_withdrawal.withdrawn & admin.project\\_code & admin.year\\_of\\_dcc\\_upload & patient.additional\\_studies & patient.asbestos\\_exposure\\_age & patient.asbestos\\_exposure\\_age\\_last & patient.asbestos\\_exposure\\_source & patient.asbestos\\_exposure\\_type & patient.asbestos\\_exposure\\_years & patient.bcr\\_patient\\_barcode & patient.bcr\\_patient\\_uuid & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-2.family\\_history\\_cancer\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-2.family\\_history\\_cancer\\_type\\_other & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-2.family\\_medical\\_history\\_relative\\_family\\_member\\_relationship\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-3.family\\_history\\_cancer\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-3.family\\_history\\_cancer\\_type\\_other & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-3.family\\_medical\\_history\\_relative\\_family\\_member\\_relationship\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-4.family\\_history\\_cancer\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-4.family\\_history\\_cancer\\_type\\_other & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history-4.family\\_medical\\_history\\_relative\\_family\\_member\\_relationship\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history.family\\_history\\_cancer\\_type & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history.family\\_history\\_cancer\\_type\\_other & patient.blood\\_relative\\_cancer\\_history\\_list.blood\\_relative\\_cancer\\_history.family\\_medical\\_history\\_relative\\_family\\_member\\_relationship\\_type & patient.clinical\\_cqcf.anatomic\\_neoplasm\\_subdivision & patient.clinical\\_cqcf.consent\\_or\\_death\\_status & patient.clinical\\_cqcf.country & patient.clinical\\_cqcf.days\\_to\\_consent & patient.clinical\\_cqcf.days\\_to\\_death & patient.clinical\\_cqcf.diagnosis\\_subtype & patient.clinical\\_cqcf.histological\\_type & patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.clinical\\_cqcf.normal\\_tissue\\_anatomic\\_site & patient.clinical\\_cqcf.normal\\_tissue\\_proximity & patient.clinical\\_cqcf.other\\_anatomic\\_site & patient.clinical\\_cqcf.other\\_diagnosis & patient.clinical\\_cqcf.prior\\_dx & patient.clinical\\_cqcf.tumor\\_focality & patient.clinical\\_cqcf.tumor\\_type & patient.day\\_of\\_form\\_completion & patient.days\\_to\\_birth & patient.days\\_to\\_death & patient.days\\_to\\_last\\_followup & patient.drugs & patient.drugs.drug-2.bcr\\_drug\\_barcode & patient.drugs.drug-2.bcr\\_drug\\_uuid & patient.drugs.drug-2.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-2.day\\_of\\_form\\_completion & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-2.drug\\_name & patient.drugs.drug-2.measure\\_of\\_response & patient.drugs.drug-2.month\\_of\\_form\\_completion & patient.drugs.drug-2.number\\_cycles & patient.drugs.drug-2.prescribed\\_dose & patient.drugs.drug-2.prescribed\\_dose\\_units & patient.drugs.drug-2.regimen\\_indication & patient.drugs.drug-2.regimen\\_indication\\_notes & patient.drugs.drug-2.regimen\\_number & patient.drugs.drug-2.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-2.therapy\\_ongoing & patient.drugs.drug-2.therapy\\_types.therapy\\_type & patient.drugs.drug-2.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-2.total\\_dose & patient.drugs.drug-2.total\\_dose\\_units & patient.drugs.drug-2.tx\\_on\\_clinical\\_trial & patient.drugs.drug-2.year\\_of\\_form\\_completion & patient.drugs.drug-3.bcr\\_drug\\_barcode & patient.drugs.drug-3.bcr\\_drug\\_uuid & patient.drugs.drug-3.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-3.day\\_of\\_form\\_completion & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-3.drug\\_name & patient.drugs.drug-3.measure\\_of\\_response & patient.drugs.drug-3.month\\_of\\_form\\_completion & patient.drugs.drug-3.number\\_cycles & patient.drugs.drug-3.prescribed\\_dose & patient.drugs.drug-3.prescribed\\_dose\\_units & patient.drugs.drug-3.regimen\\_indication & patient.drugs.drug-3.regimen\\_indication\\_notes & patient.drugs.drug-3.regimen\\_number & patient.drugs.drug-3.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-3.therapy\\_ongoing & patient.drugs.drug-3.therapy\\_types.therapy\\_type & patient.drugs.drug-3.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-3.total\\_dose & patient.drugs.drug-3.total\\_dose\\_units & patient.drugs.drug-3.tx\\_on\\_clinical\\_trial & patient.drugs.drug-3.year\\_of\\_form\\_completion & patient.drugs.drug-4.bcr\\_drug\\_barcode & patient.drugs.drug-4.bcr\\_drug\\_uuid & patient.drugs.drug-4.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-4.day\\_of\\_form\\_completion & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-4.drug\\_name & patient.drugs.drug-4.measure\\_of\\_response & patient.drugs.drug-4.month\\_of\\_form\\_completion & patient.drugs.drug-4.number\\_cycles & patient.drugs.drug-4.prescribed\\_dose & patient.drugs.drug-4.prescribed\\_dose\\_units & patient.drugs.drug-4.regimen\\_indication & patient.drugs.drug-4.regimen\\_indication\\_notes & patient.drugs.drug-4.regimen\\_number & patient.drugs.drug-4.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-4.therapy\\_ongoing & patient.drugs.drug-4.therapy\\_types.therapy\\_type & patient.drugs.drug-4.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-4.total\\_dose & patient.drugs.drug-4.total\\_dose\\_units & patient.drugs.drug-4.tx\\_on\\_clinical\\_trial & patient.drugs.drug-4.year\\_of\\_form\\_completion & patient.drugs.drug-5.bcr\\_drug\\_barcode & patient.drugs.drug-5.bcr\\_drug\\_uuid & patient.drugs.drug-5.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-5.day\\_of\\_form\\_completion & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-5.drug\\_name & patient.drugs.drug-5.measure\\_of\\_response & patient.drugs.drug-5.month\\_of\\_form\\_completion & patient.drugs.drug-5.number\\_cycles & patient.drugs.drug-5.prescribed\\_dose & patient.drugs.drug-5.prescribed\\_dose\\_units & patient.drugs.drug-5.regimen\\_indication & patient.drugs.drug-5.regimen\\_indication\\_notes & patient.drugs.drug-5.regimen\\_number & patient.drugs.drug-5.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-5.therapy\\_ongoing & patient.drugs.drug-5.therapy\\_types.therapy\\_type & patient.drugs.drug-5.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-5.total\\_dose & patient.drugs.drug-5.total\\_dose\\_units & patient.drugs.drug-5.tx\\_on\\_clinical\\_trial & patient.drugs.drug-5.year\\_of\\_form\\_completion & patient.drugs.drug-6.bcr\\_drug\\_barcode & patient.drugs.drug-6.bcr\\_drug\\_uuid & patient.drugs.drug-6.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-6.day\\_of\\_form\\_completion & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-6.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-6.drug\\_name & patient.drugs.drug-6.measure\\_of\\_response & patient.drugs.drug-6.month\\_of\\_form\\_completion & patient.drugs.drug-6.number\\_cycles & patient.drugs.drug-6.prescribed\\_dose & patient.drugs.drug-6.prescribed\\_dose\\_units & patient.drugs.drug-6.regimen\\_indication & patient.drugs.drug-6.regimen\\_indication\\_notes & patient.drugs.drug-6.regimen\\_number & patient.drugs.drug-6.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-6.therapy\\_ongoing & patient.drugs.drug-6.therapy\\_types.therapy\\_type & patient.drugs.drug-6.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-6.total\\_dose & patient.drugs.drug-6.total\\_dose\\_units & patient.drugs.drug-6.tx\\_on\\_clinical\\_trial & patient.drugs.drug-6.year\\_of\\_form\\_completion & patient.drugs.drug-7.bcr\\_drug\\_barcode & patient.drugs.drug-7.bcr\\_drug\\_uuid & patient.drugs.drug-7.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-7.day\\_of\\_form\\_completion & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-7.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-7.drug\\_name & patient.drugs.drug-7.measure\\_of\\_response & patient.drugs.drug-7.month\\_of\\_form\\_completion & patient.drugs.drug-7.number\\_cycles & patient.drugs.drug-7.prescribed\\_dose & patient.drugs.drug-7.prescribed\\_dose\\_units & patient.drugs.drug-7.regimen\\_indication & patient.drugs.drug-7.regimen\\_indication\\_notes & patient.drugs.drug-7.regimen\\_number & patient.drugs.drug-7.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-7.therapy\\_ongoing & patient.drugs.drug-7.therapy\\_types.therapy\\_type & patient.drugs.drug-7.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-7.total\\_dose & patient.drugs.drug-7.total\\_dose\\_units & patient.drugs.drug-7.tx\\_on\\_clinical\\_trial & patient.drugs.drug-7.year\\_of\\_form\\_completion & patient.drugs.drug-8.bcr\\_drug\\_barcode & patient.drugs.drug-8.bcr\\_drug\\_uuid & patient.drugs.drug-8.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-8.day\\_of\\_form\\_completion & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-8.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-8.drug\\_name & patient.drugs.drug-8.measure\\_of\\_response & patient.drugs.drug-8.month\\_of\\_form\\_completion & patient.drugs.drug-8.number\\_cycles & patient.drugs.drug-8.prescribed\\_dose & patient.drugs.drug-8.prescribed\\_dose\\_units & patient.drugs.drug-8.regimen\\_indication & patient.drugs.drug-8.regimen\\_indication\\_notes & patient.drugs.drug-8.regimen\\_number & patient.drugs.drug-8.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-8.therapy\\_ongoing & patient.drugs.drug-8.therapy\\_types.therapy\\_type & patient.drugs.drug-8.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-8.total\\_dose & patient.drugs.drug-8.total\\_dose\\_units & patient.drugs.drug-8.tx\\_on\\_clinical\\_trial & patient.drugs.drug-8.year\\_of\\_form\\_completion & patient.drugs.drug-9.bcr\\_drug\\_barcode & patient.drugs.drug-9.bcr\\_drug\\_uuid & patient.drugs.drug-9.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-9.day\\_of\\_form\\_completion & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-9.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-9.drug\\_name & patient.drugs.drug-9.measure\\_of\\_response & patient.drugs.drug-9.month\\_of\\_form\\_completion & patient.drugs.drug-9.number\\_cycles & patient.drugs.drug-9.prescribed\\_dose & patient.drugs.drug-9.prescribed\\_dose\\_units & patient.drugs.drug-9.regimen\\_indication & patient.drugs.drug-9.regimen\\_indication\\_notes & patient.drugs.drug-9.regimen\\_number & patient.drugs.drug-9.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-9.therapy\\_ongoing & patient.drugs.drug-9.therapy\\_types.therapy\\_type & patient.drugs.drug-9.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-9.total\\_dose & patient.drugs.drug-9.total\\_dose\\_units & patient.drugs.drug-9.tx\\_on\\_clinical\\_trial & patient.drugs.drug-9.year\\_of\\_form\\_completion & patient.drugs.drug.bcr\\_drug\\_barcode & patient.drugs.drug.bcr\\_drug\\_uuid & patient.drugs.drug.clinical\\_trail\\_drug\\_classification & patient.drugs.drug.day\\_of\\_form\\_completion & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug.drug\\_name & patient.drugs.drug.measure\\_of\\_response & patient.drugs.drug.month\\_of\\_form\\_completion & patient.drugs.drug.number\\_cycles & patient.drugs.drug.prescribed\\_dose & patient.drugs.drug.prescribed\\_dose\\_units & patient.drugs.drug.regimen\\_indication & patient.drugs.drug.regimen\\_indication\\_notes & patient.drugs.drug.regimen\\_number & patient.drugs.drug.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug.therapy\\_ongoing & patient.drugs.drug.therapy\\_types.therapy\\_type & patient.drugs.drug.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug.total\\_dose & patient.drugs.drug.total\\_dose\\_units & patient.drugs.drug.tx\\_on\\_clinical\\_trial & patient.drugs.drug.year\\_of\\_form\\_completion & patient.eastern\\_cancer\\_oncology\\_group & patient.ethnicity & patient.follow\\_ups & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-2.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.days\\_to\\_death & patient.follow\\_ups.follow\\_up-2.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-2.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up-2.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up-2.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-2.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.performance\\_status\\_assessment\\_timepoint\\_category\\_other\\_text & patient.follow\\_ups.follow\\_up-2.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up-2.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-2.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up-2.radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.vital\\_status & patient.follow\\_ups.follow\\_up-2.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.days\\_to\\_death & patient.follow\\_ups.follow\\_up.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up.eastern\\_cancer\\_oncology\\_group & patient.follow\\_ups.follow\\_up.karnofsky\\_performance\\_score & patient.follow\\_ups.follow\\_up.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.performance\\_status\\_assessment\\_timepoint\\_category\\_other\\_text & patient.follow\\_ups.follow\\_up.performance\\_status\\_scale\\_timing & patient.follow\\_ups.follow\\_up.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up.radiation\\_therapy & patient.follow\\_ups.follow\\_up.vital\\_status & patient.follow\\_ups.follow\\_up.year\\_of\\_form\\_completion & patient.gender & patient.history\\_asbestos\\_exposure & patient.history\\_of\\_neoadjuvant\\_treatment & patient.icd\\_10 & patient.icd\\_o\\_3\\_histology & patient.icd\\_o\\_3\\_site & patient.informed\\_consent\\_verified & patient.karnofsky\\_performance\\_score & patient.month\\_of\\_form\\_completion & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-5.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.patient\\_id & patient.performance\\_status\\_scale\\_timing & patient.person\\_neoplasm\\_cancer\\_status & patient.pleurodesis\\_performed\\_90\\_days & patient.pleurodesis\\_performed\\_prior & patient.postoperative\\_rx\\_tx & patient.primary\\_occupation & patient.primary\\_occupation\\_other & patient.primary\\_occupation\\_years\\_worked & patient.primary\\_pathology.age\\_at\\_initial\\_pathologic\\_diagnosis & patient.primary\\_pathology.creatinine\\_norm\\_range\\_lower & patient.primary\\_pathology.creatinine\\_norm\\_range\\_upper & patient.primary\\_pathology.creatinine\\_prior\\_tx & patient.primary\\_pathology.days\\_to\\_initial\\_pathologic\\_diagnosis & patient.primary\\_pathology.histological\\_type & patient.primary\\_pathology.laterality & patient.primary\\_pathology.mesothelioma\\_detection\\_method & patient.primary\\_pathology.residual\\_tumor & patient.primary\\_pathology.serum\\_mesothelin\\_lower\\_limit & patient.primary\\_pathology.serum\\_mesothelin\\_prior\\_tx & patient.primary\\_pathology.serum\\_mesothelin\\_upper\\_limit & patient.primary\\_pathology.suv\\_of\\_pleura\\_max & patient.primary\\_pathology.tumor\\_tissue\\_site & patient.primary\\_pathology.year\\_of\\_initial\\_pathologic\\_diagnosis & patient.prior\\_dx & patient.race & patient.radiation\\_therapy & patient.radiations & patient.radiations.radiation-2.anatomic\\_treatment\\_site & patient.radiations.radiation-2.bcr\\_radiation\\_barcode & patient.radiations.radiation-2.bcr\\_radiation\\_uuid & patient.radiations.radiation-2.course\\_number & patient.radiations.radiation-2.day\\_of\\_form\\_completion & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-2.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-2.measure\\_of\\_response & patient.radiations.radiation-2.month\\_of\\_form\\_completion & patient.radiations.radiation-2.numfractions & patient.radiations.radiation-2.radiation\\_dosage & patient.radiations.radiation-2.radiation\\_treatment\\_ongoing & patient.radiations.radiation-2.radiation\\_type & patient.radiations.radiation-2.radiation\\_type\\_notes & patient.radiations.radiation-2.regimen\\_indication & patient.radiations.radiation-2.regimen\\_indication\\_notes & patient.radiations.radiation-2.units & patient.radiations.radiation-2.year\\_of\\_form\\_completion & patient.radiations.radiation-3.anatomic\\_treatment\\_site & patient.radiations.radiation-3.bcr\\_radiation\\_barcode & patient.radiations.radiation-3.bcr\\_radiation\\_uuid & patient.radiations.radiation-3.course\\_number & patient.radiations.radiation-3.day\\_of\\_form\\_completion & patient.radiations.radiation-3.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation-3.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation-3.measure\\_of\\_response & patient.radiations.radiation-3.month\\_of\\_form\\_completion & patient.radiations.radiation-3.numfractions & patient.radiations.radiation-3.radiation\\_dosage & patient.radiations.radiation-3.radiation\\_treatment\\_ongoing & patient.radiations.radiation-3.radiation\\_type & patient.radiations.radiation-3.radiation\\_type\\_notes & patient.radiations.radiation-3.regimen\\_indication & patient.radiations.radiation-3.regimen\\_indication\\_notes & patient.radiations.radiation-3.units & patient.radiations.radiation-3.year\\_of\\_form\\_completion & patient.radiations.radiation.anatomic\\_treatment\\_site & patient.radiations.radiation.bcr\\_radiation\\_barcode & patient.radiations.radiation.bcr\\_radiation\\_uuid & patient.radiations.radiation.course\\_number & patient.radiations.radiation.day\\_of\\_form\\_completion & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation.measure\\_of\\_response & patient.radiations.radiation.month\\_of\\_form\\_completion & patient.radiations.radiation.numfractions & patient.radiations.radiation.radiation\\_dosage & patient.radiations.radiation.radiation\\_treatment\\_ongoing & patient.radiations.radiation.radiation\\_type & patient.radiations.radiation.radiation\\_type\\_notes & patient.radiations.radiation.regimen\\_indication & patient.radiations.radiation.regimen\\_indication\\_notes & patient.radiations.radiation.units & patient.radiations.radiation.year\\_of\\_form\\_completion & patient.stage\\_event.ann\\_arbor.b\\_symptoms & patient.stage\\_event.ann\\_arbor.extranodal\\_involvement & patient.stage\\_event.clinical\\_stage & patient.stage\\_event.gleason\\_grading.gleason\\_score & patient.stage\\_event.gleason\\_grading.primary\\_pattern & patient.stage\\_event.gleason\\_grading.secondary\\_pattern & patient.stage\\_event.gleason\\_grading.tertiary\\_pattern & patient.stage\\_event.igcccg\\_stage & patient.stage\\_event.masaoka\\_stage & patient.stage\\_event.pathologic\\_stage & patient.stage\\_event.psa.days\\_to\\_psa & patient.stage\\_event.psa.psa\\_value & patient.stage\\_event.serum\\_markers & patient.stage\\_event.system\\_version & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_m & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_n & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_t & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t & patient.tissue\\_prospective\\_collection\\_indicator & patient.tissue\\_retrospective\\_collection\\_indicator & patient.tissue\\_source\\_site & patient.vital\\_status & patient.year\\_of\\_form\\_completion & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status & patient.biospecimen\\_cqcf.b\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status & patient.biospecimen\\_cqcf.country & patient.biospecimen\\_cqcf.cytogenetic\\_report\\_submitted & patient.biospecimen\\_cqcf.days\\_to\\_pathology\\_review & patient.biospecimen\\_cqcf.days\\_to\\_sample\\_procurement & patient.biospecimen\\_cqcf.differential\\_report\\_submitted & patient.biospecimen\\_cqcf.digital\\_image\\_submitted & patient.biospecimen\\_cqcf.ethnicity & patient.biospecimen\\_cqcf.ffpe\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site & patient.biospecimen\\_cqcf.histological\\_type & patient.biospecimen\\_cqcf.histological\\_type\\_other & patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.biospecimen\\_cqcf.hiv\\_positive\\_status & patient.biospecimen\\_cqcf.laterality & patient.biospecimen\\_cqcf.maximum\\_tumor\\_dimension & patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number & patient.biospecimen\\_cqcf.other\\_method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.other\\_vessel\\_used & patient.biospecimen\\_cqcf.path\\_confirm\\_diagnosis\\_matching & patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_necrosis\\_metrics & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_nuclei\\_metrics & patient.biospecimen\\_cqcf.percent\\_myeloblasts\\_for\\_submitted\\_specimen & patient.biospecimen\\_cqcf.race & patient.biospecimen\\_cqcf.reason\\_path\\_confirm\\_diagnosis\\_not\\_matching & patient.biospecimen\\_cqcf.sample\\_prescreened & patient.biospecimen\\_cqcf.site\\_of\\_disease & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_description & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_text & patient.biospecimen\\_cqcf.submitted\\_for\\_lce & patient.biospecimen\\_cqcf.t\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.top\\_slide\\_submitted & patient.biospecimen\\_cqcf.total\\_cells\\_submitted & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.vial\\_number & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number & patient.biospecimen\\_cqcf.vessel\\_used & patient.days\\_to\\_index & patient.samples.sample-2.additional\\_studies & patient.samples.sample-2.bcr\\_sample\\_barcode & patient.samples.sample-2.bcr\\_sample\\_uuid & patient.samples.sample-2.biospecimen\\_sequence & patient.samples.sample-2.composition & patient.samples.sample-2.current\\_weight & patient.samples.sample-2.days\\_to\\_collection & patient.samples.sample-2.days\\_to\\_sample\\_procurement & patient.samples.sample-2.diagnostic\\_slides.additional\\_studies & patient.samples.sample-2.freezing\\_method & patient.samples.sample-2.initial\\_weight & patient.samples.sample-2.intermediate\\_dimension & patient.samples.sample-2.is\\_ffpe & patient.samples.sample-2.longest\\_dimension & patient.samples.sample-2.method\\_of\\_sample\\_procurement & patient.samples.sample-2.ncnnct\\_othmethonsp & patient.samples.sample-2.oct\\_embedded & patient.samples.sample-2.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-2.pathology\\_report\\_file\\_name & patient.samples.sample-2.pathology\\_report\\_uuid & patient.samples.sample-2.portions.portion.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.amount & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-2.portions.portion.day\\_of\\_creation & patient.samples.sample-2.portions.portion.is\\_ffpe & patient.samples.sample-2.portions.portion.lce & patient.samples.sample-2.portions.portion.month\\_of\\_creation & patient.samples.sample-2.portions.portion.portion\\_number & patient.samples.sample-2.portions.portion.portion\\_sequence & patient.samples.sample-2.portions.portion.slides & patient.samples.sample-2.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide.section\\_location & patient.samples.sample-2.portions.portion.weight & patient.samples.sample-2.portions.portion.year\\_of\\_creation & patient.samples.sample-2.preservation\\_method & patient.samples.sample-2.sample\\_type & patient.samples.sample-2.sample\\_type\\_id & patient.samples.sample-2.shortest\\_dimension & patient.samples.sample-2.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-2.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-2.tissue\\_type & patient.samples.sample-2.tumor\\_descriptor & patient.samples.sample-2.tumor\\_pathology & patient.samples.sample-2.vial\\_number & patient.samples.sample.additional\\_studies & patient.samples.sample.bcr\\_sample\\_barcode & patient.samples.sample.bcr\\_sample\\_uuid & patient.samples.sample.biospecimen\\_sequence & patient.samples.sample.composition & patient.samples.sample.current\\_weight & patient.samples.sample.days\\_to\\_collection & patient.samples.sample.days\\_to\\_sample\\_procurement & patient.samples.sample.diagnostic\\_slides.additional\\_studies & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-2 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-3 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-4 & patient.samples.sample.freezing\\_method & patient.samples.sample.initial\\_weight & patient.samples.sample.intermediate\\_dimension & patient.samples.sample.is\\_ffpe & patient.samples.sample.longest\\_dimension & patient.samples.sample.method\\_of\\_sample\\_procurement & patient.samples.sample.ncnnct\\_othmethonsp & patient.samples.sample.oct\\_embedded & patient.samples.sample.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample.pathology\\_report\\_file\\_name & patient.samples.sample.pathology\\_report\\_uuid & patient.samples.sample.portions.portion.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.amount & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.concentration & patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte.well\\_number & patient.samples.sample.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion.day\\_of\\_creation & patient.samples.sample.portions.portion.is\\_ffpe & patient.samples.sample.portions.portion.lce & patient.samples.sample.portions.portion.month\\_of\\_creation & patient.samples.sample.portions.portion.portion\\_number & patient.samples.sample.portions.portion.portion\\_sequence & patient.samples.sample.portions.portion.slides.slide.additional\\_studies & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide.section\\_location & patient.samples.sample.portions.portion.weight & patient.samples.sample.portions.portion.year\\_of\\_creation & patient.samples.sample.portions.shipment\\_portion.additional\\_studies & patient.samples.sample.portions.shipment\\_portion.bcr\\_shipment\\_portion\\_uuid & patient.samples.sample.portions.shipment\\_portion.center\\_id & patient.samples.sample.portions.shipment\\_portion.is\\_ffpe & patient.samples.sample.portions.shipment\\_portion.plate\\_id & patient.samples.sample.portions.shipment\\_portion.portion\\_number & patient.samples.sample.portions.shipment\\_portion.portion\\_sequence & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_bcr\\_aliquot\\_barcode & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_day\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_month\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_year\\_of\\_shipment & patient.samples.sample.preservation\\_method & patient.samples.sample.sample\\_type & patient.samples.sample.sample\\_type\\_id & patient.samples.sample.shortest\\_dimension & patient.samples.sample.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample.tissue\\_type & patient.samples.sample.tumor\\_descriptor & patient.samples.sample.tumor\\_pathology & patient.samples.sample.vial\\_number\\\\\n",
       "  & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t418.36.0 & nationwide children's hospital & 1 & meso & be213c85-e379-42fc-918a-1fe13c4d0201 & 10 & false & tcga & 2015 & NA & NA & NA & NA           & NA          & NA & tcga-3h-ab3k & e90bfe17-2e5b-47ed-b986-b3d4181f2bf6 & NA             & NA       & NA     & NA             & NA          & NA          & NA             & NA           & NA          & NA             & NA          & NA     & NA & consented & united states & 29 & NA & NA & diffuse malignant mesothelioma - nos         & no & NA & NA & NA & NA & NA & NA & primary & 11 & -23591 & 1715 & NA & NA & tcga-3h-ab3k-d63511 & 34c9dc36-9cd0-4339-95e6-f36ef2df34c0 & NA & 11 & 919 & 898 & cediranib  & NA                           & 8  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA               & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA           & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & tcga-3h-ab3k-d63509 & b00cc242-1ae5-4622-9840-3e1df8de9129 & NA & 11 & 71 & 52 & dasatinib & partial response             & 8 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & NA & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & male   & no  & no & c45.0 & 9050/3 & c38.4 & yes & NA & 8 & yes & no & NA & 883 & contralateral pleura & distant metastasis & NA & no & NA  & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA      & NA                 & NA & NA & no  & no  & NA & 841 & contralateral pleura & distant metastasis       & NA & no & yes & ab3k & NA                   & with tumor & no & yes & yes & other, please specify       & meat manager                                     & 30 & 64 & 0.8 & 1.5 & 1.0 & 0 & diffuse malignant mesothelioma - nos & right & biopsy & NA & NA & NA & NA & 3.7  & pleura & 2008 & no  & white & yes & NA & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA   & NA & NA       & NA & NA & NA & NA & NA   & NA            & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & regional site       & tcga-3h-ab3k-r63510 & 9d276f68-ec13-4bef-bd0c-5fae0b6384ba & NA & 11 & 190 & 155 & partial response                 & 8  & 25 & 50 & no & external & NA & NA & NA & gy & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ii  & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t2 & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 45 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - nos         & NA & no & NA & NA & NA & thoracoscopic biopsy & NA & NA & NA & NA & NA & NA & NA & NA & NA & fbb4d2cc-a002-498a-99ee-c7c6d9d92003 & 45 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & d101d5b2-4130-42cf-bbe8-ab817cc0d5cb & NA & NA & NA & NA & 108 & 1 & cryomold & 0 & NA & tcga-3h-ab3k-10a & fbb4d2cc-a002-498a-99ee-c7c6d9d92003 & NA & NA & NA & 1935 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3k-10a-01d-a39u-32 & 160fcc83-9cbf-4413-a85e-b90de9789a85 & 0167779959 & 32 & 0.16 & 14 & no & 05 & 1 & a39u & a & 3.20 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3k-10a-01d-a40g-26 & 26fa8c4f-56d7-4fff-98af-0d69b1f4cd7d & 0167823551 & 26 & 0.16 & 14 & no & 05 & 1  & a40g & a  & 1.60 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3k-10a-01d-a39t-01 & 28864803-3575-4755-86ee-37ea5c58552a & 0167789517 & 01 & 0.16 & 14 & no & 05 & 1 & a39t & a & 1.07 & 23 & 6.70 & 2014 & 55.90 & dna & d & tcga-3h-ab3k-10a-01d & 2781f2dd-29dc-4841-8a53-41b260cdad58 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3k-10a-01 & 2eb914dc-5988-4aab-b2b8-e4ab6609c097 & 20 & no & NA & 03 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3k-01a & d101d5b2-4130-42cf-bbe8-ab817cc0d5cb & NA & NA & NA & 1935 & NA & NA & NA                                   & NA                                   & NA                                   & NA & NA & 90  & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3k.2e921e98-cb9d-4fa5-b67f-a20bb5455ca2.pdf & 2e921e98-cb9d-4fa5-b67f-a20bb5455ca2 & NA & 1.86 & NA & NA & NA & tcga-3h-ab3k-01a-11r-a40a-07 & 919d8bde-c3b7-4cd9-8463-938afdb89ad7 & 0167788921 & 07 & 0.16 & 14 & no & 05 & 2 & a40a & f & 2.08 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3k-01a-11r-a404-13 & 49cab888-65e0-4582-aa5d-149f3dc652c6 & 0167811961 & 13 & 0.16 & 14 & no & 05 & 2 & a404 & f & 3.20 & 23 & 20.00 & 2014 & 29.14 & rna & r & tcga-3h-ab3k-01a-11r & 402026ba-70a4-45c8-a956-06ac7964a07d & 0.16 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.40 & 8.00 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3k-01a-11d-a39r-32 & 735c0870-dcc0-46b5-9b2f-439e9ee9791d & 0167780439 & 32 & 0.16 & 14 & no & 05 & 1 & a39r & a & 3.20 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3k-01a-11d-a39s-05 & f7b11d42-36bc-4ae7-9d9a-283b71847ea1 & 0167780447 & 05 & 0.16 & 14 & no & 05 & 1 & a39s & a & 2.08 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3k-01a-11d-a40f-26 & 5d586068-1d73-4af1-9e29-a5cb514da9f7 & 0167789078 & 26 & 0.16 & 14 & no & 05 & 1  & a40f & a  & 1.60 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3k-01a-11d-a39q-01 & 65ab6f49-d2d2-458d-82a3-3798627c0221 & 0167779967 & 01 & 0.16 & 14 & no & 05 & 1 & a39q & a & 1.07 & 23 & 6.70 & 2014 & 24.69 & dna & d & tcga-3h-ab3k-01a-11d & 4ef3f1cf-feb8-4193-9605-3fe5f8116a3e & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3k-01a-11 & 3c50eee6-63b2-43a4-804b-4c5b53919f6e & 17 & no & NA & 03 & 11 & NA & NA & tcga-3h-ab3k-01a-01-ts1 & 2c209185-171f-4bbe-b035-3f4483d58d90 & tcga-3h-ab3k-01a-01-ts1.2c209185-171f-4bbe-b035-3f4483d58d90.svs & no & NA & NA & NA & NA & 40 & 20 & 0  & 20 & 0 & 30 & 70  & 70 & top & 30.00 & 2014 & NA & fa2214bc-2f60-4767-bb8f-09679a9c0bd5 & 20 & no & a45o & 21 & NA & tcga-3h-ab3k-01a-21-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t418.36.0.1 & nationwide children's hospital & 1 & meso & 7a8ecc55-002e-4650-98f8-84806b919c17 & 10 & false & tcga & 2015 & NA & NA & NA & NA           & NA          & NA & tcga-3h-ab3l & aaace369-0a47-4176-811b-61cd42bf647f & NA             & NA       & NA     & NA             & NA          & NA          & NA             & NA           & NA          & NA             & NA          & NA     & NA & consented & united states & 43 & NA & NA & diffuse malignant mesothelioma - epithelioid & no & NA & NA & NA & NA & NA & NA & primary & 13 & -21972 & 361  & NA & NA & NA                  & NA                                   & NA & NA & NA  & NA  & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA               & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA           & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & tcga-3h-ab3l-d63585 & 3a166a84-d66d-495f-8072-540c9414542f & NA & 13 & 96 & 69 & dasatinib & clinical progressive disease & 8 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & 1  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & male   & NA  & no & c45.0 & 9052/3 & c38.4 & yes & NA & 8 & NA  & NA & NA & NA  & NA                   & NA                 & NA & NA & NA  & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA      & NA                 & NA & NA & NA  & NA  & NA & 222 & ipsilateral pleura   & intrapleural progression & NA & NA & yes & ab3l & NA                   & with tumor & no & yes & yes & construction and extraction & NA                                               & NA & 60 & 0.7 & 1.3 & 1.3 & 0 & epithelioid mesothelioma             & left  & biopsy & NA & NA & NA & NA & 16.7 & pleura & 2008 & no  & white & no  & NA & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA   & NA & NA       & NA & NA & NA & NA & NA   & NA            & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iii & NA & NA & NA & 6th & NA & NA & NA & m0 & n2 & tx & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 58 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - epithelioid & NA & no & NA & NA & NA & thoracoscopic biopsy & NA & NA & NA & NA & NA & NA & NA & NA & NA & a24af496-399d-491f-8e1e-c548997f9d21 & 58 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & ddeae462-e070-4ece-bebb-e782ba410ea8 & NA & NA & NA & NA & 113 & 1 & cryomold & 0 & NA & tcga-3h-ab3l-10a & a24af496-399d-491f-8e1e-c548997f9d21 & NA & NA & NA & 1941 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3l-10a-01d-a39u-32 & 0fe8383d-2e4b-418f-b76d-8f6962ced36c & 0167779936 & 32 & 0.15 & 14 & no & 05 & 1 & a39u & b & 3.00 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3l-10a-01d-a40g-26 & 04a81cb1-87ef-4526-9fe3-34412fccb2e2 & 0167823528 & 26 & 0.15 & 14 & no & 05 & 1  & a40g & b  & 1.50 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3l-10a-01d-a39t-01 & 880a05d8-c888-497b-933c-baabd1b7c4cc & 0167789509 & 01 & 0.15 & 14 & no & 05 & 1 & a39t & b & 1.01 & 23 & 6.70 & 2014 & 24.04 & dna & d & tcga-3h-ab3l-10a-01d & fcbaf0f7-be4c-4940-be95-6be669a93f53 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3l-10a-01 & cf871f31-36c5-4dc4-a6d0-44f86d6a230f & 20 & no & NA & 03 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3l-01a & ddeae462-e070-4ece-bebb-e782ba410ea8 & NA & NA & NA & 1941 & NA & NA & aed33a7b-eeed-468b-8892-2b7f338c5471 & NA                                   & NA                                   & NA & NA & 130 & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3l.162c8370-35c9-4b65-a9f2-6133e343390a.pdf & 162c8370-35c9-4b65-a9f2-6133e343390a & NA & 1.84 & NA & NA & NA & tcga-3h-ab3l-01a-11r-a40a-07 & 3baf5546-99d9-417b-97bb-e00dda7ebcdd & 0167788918 & 07 & 0.16 & 14 & no & 05 & 2 & a40a & g & 2.08 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3l-01a-11r-a404-13 & f0ee65bf-3d06-4559-ad29-747af0d5b244 & 0167811958 & 13 & 0.16 & 14 & no & 05 & 2 & a404 & g & 3.20 & 23 & 20.00 & 2014 & 30.43 & rna & r & tcga-3h-ab3l-01a-11r & 389ad0c7-7ef1-430d-9b20-71fc55e45fc1 & 0.16 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.66 & 8.80 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3l-01a-11d-a39r-32 & 4574d9c8-e603-46c7-b82c-09f17ce6c35e & 0167780416 & 32 & 0.15 & 14 & no & 05 & 1 & a39r & b & 3.00 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3l-01a-11d-a39s-05 & 9384a65b-2f0a-4f65-ac3f-6537aafe1ebd & 0167780424 & 05 & 0.15 & 14 & no & 05 & 1 & a39s & b & 1.95 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3l-01a-11d-a40f-26 & a01e36d2-4858-478b-9c6a-3768ed086878 & 0167789073 & 26 & 0.15 & 14 & no & 05 & 1  & a40f & b  & 1.50 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3l-01a-11d-a39q-01 & d0bd5e21-4cc2-4ac6-98c5-d03ef5cb5744 & 0167779944 & 01 & 0.15 & 14 & no & 05 & 1 & a39q & b & 1.01 & 23 & 6.70 & 2014 & 19.85 & dna & d & tcga-3h-ab3l-01a-11d & c9549f2a-07f1-484e-af5d-3c6b3289ebe4 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3l-01a-11 & e79adda9-c31e-4596-8898-1acf80a75731 & 17 & no & NA & 03 & 11 & NA & NA & tcga-3h-ab3l-01a-01-ts1 & d473695c-02b1-4663-9e73-ff25ee57796f & tcga-3h-ab3l-01a-01-ts1.d473695c-02b1-4663-9e73-ff25ee57796f.svs & no & NA & NA & NA & NA & 40 & 20 & 0  & 20 & 0 & 30 & 70  & 70 & top & 30.00 & 2014 & NA & 81f9016d-1f8d-4902-aa68-2002130cf41f & 20 & no & a45o & 21 & NA & tcga-3h-ab3l-01a-21-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t418.36.0.2 & nationwide children's hospital & 1 & meso & eb988691-41bd-412e-9e6c-82bd7050dc1b & 10 & false & tcga & 2015 & NA & NA & NA & occupational & NA          & NA & tcga-3h-ab3m & ebe927e6-0a7e-4c97-ad73-f3c302cca7bc & NA             & NA       & NA     & NA             & NA          & NA          & NA             & NA           & NA          & other, specify & ovarian     & parent & NA & consented & united states & 15 & NA & NA & diffuse malignant mesothelioma - epithelioid & no & NA & NA & NA & NA & NA & NA & primary & 11 & -19503 & 885  & NA & NA & tcga-3h-ab3m-d60588 & a7171629-f49e-44f9-8e1a-3bf45896c7c5 & NA & 11 & 386 & 120 & cisplatin  & stable disease               & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy               & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60589 & 2cf9bb9e-5a95-446a-8e83-b6436942de6a & NA & 11 & 386 & 120 & pemetrexed  & stable disease               & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60590 & 4648d25c-8e0f-4326-80d2-ff1bd51c818f & NA & 11 & 386 & 120 & imatinib   & stable disease               & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60592 & 271365e2-1da4-4b0e-8c8d-9e06ee0d87e3 & NA & 11 & 386  & 120  & everolimus & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60593 & 2031fb30-84aa-472d-ad8b-f8b00fa789c7 & NA & 11 & 800  & 736  & carboplatin & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60595 & 8017f9c5-2f43-47ae-bc78-5d2b7a4dede3 & NA & 11 & 604  & 471  & doxorubicin & stable disease   & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60596 & 3db085c4-dc87-4e33-86d3-1618dc6adf39 & NA & 11 & 604 & 471 & bevacizumab & stable disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60597 & c621d9b9-8ccc-4b79-9194-7959179b3814 & NA & 11 & 604 & 471 & temsirolimus & stable disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3m-d60586 & 429e616f-6a0c-45dc-b85a-ee65490f6d49 & NA & 11 & 66 & 38 & dasatinib & clinical progressive disease & 6 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & 1  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & female & yes & no & c45.0 & 9052/3 & c38.4 & yes & NA & 6 & yes & no & NA & 384 & lung                 & distant metastasis & NA & no & yes & no & NA & 442 & lung & intrapleural progression & NA & no & NA & NA & NA & 724 & lung & intrapleural progression & NA & no & NA & NA & NA & 822 & abdomen & distant metastasis & NA & NA & yes & no  & NA & 107 & lung                 & distant metastasis       & NA & no & yes & ab3m & preoperative         & with tumor & NA & no  & yes & other, please specify       & school district secretary                        & NA & 53 & 0.8 & 1.5 & 0.8 & 0 & epithelioid mesothelioma             & left  & biopsy & NA & NA & NA & NA & 14.2 & pleura & 2008 & no  & white & no  & NA & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA   & NA & NA       & NA & NA & NA & NA & NA   & NA            & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iii & NA & NA & NA & 6th & NA & NA & NA & m0 & n2 & t3 & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 27 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - epithelioid & NA & no & NA & NA & NA & thoracoscopic biopsy & NA & NA & NA & NA & NA & NA & NA & NA & NA & 88af3148-344e-4662-8f17-b1263fe6089f & 27 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & b71b1fb6-3c73-46c8-ab3b-bdfaa4b66b67 & NA & NA & NA & NA & 188 & 1 & cryomold & 0 & NA & tcga-3h-ab3m-10a & 88af3148-344e-4662-8f17-b1263fe6089f & NA & NA & NA & 1998 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3m-10a-01d-a39u-32 & a7a6c73a-7211-4be9-a838-1df8f7da50bc & 0167779930 & 32 & 0.14 & 14 & no & 05 & 1 & a39u & c & 2.80 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3m-10a-01d-a40g-26 & e8fb0b45-92d7-4650-9269-cb8142b400cc & 0167823527 & 26 & 0.14 & 14 & no & 05 & 1  & a40g & c  & 1.40 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3m-10a-01d-a39t-01 & 98c60894-da9b-4cb3-ba83-6a1af85deb68 & 0167789502 & 01 & 0.14 & 14 & no & 05 & 1 & a39t & c & 0.94 & 23 & 6.70 & 2014 & 20.76 & dna & d & tcga-3h-ab3m-10a-01d & cadce2a0-94bc-456c-90c4-3f2020cad5cb & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3m-10a-01 & a3e6eeaa-aaf9-4310-9ac4-ca61eee9b541 & 03 & no & NA & 04 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3m-01a & b71b1fb6-3c73-46c8-ab3b-bdfaa4b66b67 & NA & NA & NA & 1998 & NA & NA & 9cd8a3b1-0f0f-418c-b54f-cba22bb19469 & NA                                   & NA                                   & NA & NA & 160 & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3m.3d2b553b-192d-4300-af57-781143d0f10b.pdf & 3d2b553b-192d-4300-af57-781143d0f10b & NA & 1.88 & NA & NA & NA & tcga-3h-ab3m-01a-11r-a40a-07 & f0b52bcf-d7b4-4daa-82bb-e13e2d753be9 & 0167788897 & 07 & 0.17 & 14 & no & 05 & 2 & a40a & h & 2.21 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3m-01a-11r-a404-13 & 2070814e-7f62-4223-88cf-2954fac9d56c & 0167811937 & 13 & 0.17 & 14 & no & 05 & 2 & a404 & h & 3.40 & 23 & 20.00 & 2014 & 53.76 & rna & r & tcga-3h-ab3m-01a-11r & 437aeb3f-cef3-46b1-8655-a1bf8ac78c2f & 0.17 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.72 & 9.30 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3m-01a-11d-a39r-32 & 6e2fb845-7e9b-4e1e-b5ba-cc7284bfd66b & 0167780410 & 32 & 0.14 & 14 & no & 05 & 1 & a39r & c & 2.80 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3m-01a-11d-a39s-05 & 0b53dbf7-63e8-4e1d-a148-61bddb2e0698 & 0167780423 & 05 & 0.14 & 14 & no & 05 & 1 & a39s & c & 1.82 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3m-01a-11d-a40f-26 & b9c00a26-c702-48cf-9957-a2ed2624883d & 0167789054 & 26 & 0.14 & 14 & no & 05 & 1  & a40f & c  & 1.40 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3m-01a-11d-a39q-01 & be171d25-cf67-411c-9c78-7acb6942b430 & 0167779943 & 01 & 0.14 & 14 & no & 05 & 1 & a39q & c & 0.94 & 23 & 6.70 & 2014 & 40.20 & dna & d & tcga-3h-ab3m-01a-11d & 2d4b4e85-f51e-48c3-b00c-30639f4ea31c & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3m-01a-11 & 0cf46831-168a-4984-8035-50a6549cd379 & 01 & no & NA & 04 & 11 & NA & NA & tcga-3h-ab3m-01a-01-ts1 & 7992e8a0-9c2d-497b-beab-5f491e1176d2 & tcga-3h-ab3m-01a-01-ts1.7992e8a0-9c2d-497b-beab-5f491e1176d2.svs & no & NA & NA & NA & NA & 2  & 0  & 0  & 0  & 0 & 0  & 100 & 65 & top & 30.00 & 2014 & NA & f98b8a5f-23ed-477d-833e-71730a4be6b9 & 20 & no & a45o & 21 & NA & tcga-3h-ab3m-01a-21-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t418.36.0.3 & nationwide children's hospital & 1 & meso & eb48bed3-00e5-45aa-8803-6ce81c7b1fff & 10 & false & tcga & 2015 & NA & 25 & 31 & occupational & crocidolite & 6  & tcga-3h-ab3o & 2b715eb2-055e-4bc5-bc98-128e8bf6954f & NA             & NA       & NA     & NA             & NA          & NA          & NA             & NA           & NA          & NA             & NA          & NA     & NA & consented & united states & 40 & NA & NA & diffuse malignant mesothelioma - epithelioid & no & NA & NA & NA & NA & NA & NA & primary & 10 & -21423 & 1302 & NA & NA & tcga-3h-ab3o-d60482 & 622db14d-e758-4815-93ce-3ab5e9cab10a & NA & 10 & 908 & 194 & dasatinib  & partial response             & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3o-d60484 & 5d09c250-5b55-4524-8743-cd0bee1d9c9a & NA & 10 & 286 & 197 & cisplatin   & partial response             & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & no  & 2014 & tcga-3h-ab3o-d60485 & b96b4ff5-8914-4bff-9f03-f79814cba107 & NA & 10 & 286 & 197 & pemetrexed & partial response             & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy               & NA & NA & NA & no  & 2014 & tcga-3h-ab3o-d60486 & 323c388c-ac8e-4f48-bd0c-feb1b39330a1 & NA & 10 & 1265 & 1084 & cediranib  & partial response             & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3o-d60489 & 56020589-2bbe-4362-b806-2f93399c47a2 & NA & 10 & 1265 & 1084 & cisplatin   & partial response             & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3o-d60490 & d5babb86-cc75-4990-bf03-8bfd62f5ff35 & NA & 10 & 1265 & 1084 & pemetrexed  & partial response & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA           & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & tcga-3h-ab3o-d60479 & a95ea018-0f72-45ed-817f-8ce707ef7b6e & NA & 10 & 93 & 54 & dasatinib & partial response             & 6 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & 1  & NA                     & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & male   & yes & no & c45.0 & 9052/3 & c38.4 & yes & NA & 6 & NA  & NA & NA & NA  & NA                   & NA                 & NA & NA & NA  & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA      & NA                 & NA & NA & yes & yes & NA & 627 & ipsilateral pleura   & intrapleural progression & NA & NA & yes & ab3o & pre-adjuvant therapy & with tumor & NA & no  & yes & other, please specify       & engineer working on machines lined with asbestos & 6  & 58 & 0.7 & 1.3 & 0.7 & 0 & epithelioid mesothelioma             & right & biopsy & NA & NA & NA & NA & 14   & pleura & 2008 & yes & white & yes & NA & primary tumor field & tcga-3h-ab3o-r60478 & 22168352-ae35-4ec7-9ab5-ff819ac36c39 & NA & 10 & 735 & 717 & stable disease                   & 6  & 10 & 70   & no & external & NA & NA & NA & gy & 2014 & NA            & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & primary tumor field & tcga-3h-ab3o-r60476 & d05206d0-e128-469f-8356-11f9e6c84b6e & NA & 10 & 164 & 130 & partial response                 & 6  & 25 & 60 & no & external & NA & NA & NA & gy & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iii & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t3 & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 45 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - epithelioid & NA & no & NA & NA & NA & thoracoscopic biopsy & NA & NA & NA & NA & NA & NA & NA & NA & NA & da3f01e4-7c3b-4abd-b4c0-1f95e61f1df7 & 45 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 08130247-fe6d-4644-9abb-29209f441943 & NA & NA & NA & NA & 238 & 1 & cryomold & 0 & NA & tcga-3h-ab3o-10a & da3f01e4-7c3b-4abd-b4c0-1f95e61f1df7 & NA & NA & NA & 1897 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3o-10a-01d-a39u-32 & d3d27b75-5336-4c4b-874f-dcd61b80bb20 & 0167779903 & 32 & 0.16 & 14 & no & 05 & 1 & a39u & d & 3.20 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3o-10a-01d-a40g-26 & 2bec77d6-b105-44e1-85bd-9a11d45d3ba1 & 0167823504 & 26 & 0.16 & 14 & no & 05 & 1  & a40g & d  & 1.60 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3o-10a-01d-a39t-01 & f3d7566b-f665-4017-b6c0-3e8dae5c4041 & 0167789494 & 01 & 0.16 & 14 & no & 05 & 1 & a39t & d & 1.07 & 23 & 6.70 & 2014 & 35.57 & dna & d & tcga-3h-ab3o-10a-01d & fb5914c8-f036-4e53-8abb-431e8d3b571b & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3o-10a-01 & fb88aae8-6b9a-4a12-8f75-d8a9958ead7e & 03 & no & NA & 04 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3o-01a & 08130247-fe6d-4644-9abb-29209f441943 & NA & NA & NA & 1897 & NA & NA & 1458d00a-b6d7-4154-b149-9f16ae5cffc0 & NA                                   & NA                                   & NA & NA & 230 & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3o.d5828ca0-5ed9-4af6-9e3c-cbe57826e43f.pdf & d5828ca0-5ed9-4af6-9e3c-cbe57826e43f & NA & 1.89 & NA & NA & NA & tcga-3h-ab3o-01a-11r-a40a-07 & de645724-4408-4771-9814-cdeffec79758 & 0167788989 & 07 & 0.17 & 14 & no & 05 & 3 & a40a & a & 2.21 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3o-01a-11r-a404-13 & 851b4309-afb7-403c-8ad7-f0d91c942632 & 0167812029 & 13 & 0.17 & 14 & no & 05 & 3 & a404 & a & 3.40 & 23 & 20.00 & 2014 & 22.12 & rna & r & tcga-3h-ab3o-01a-11r & a98db5b6-6140-4f7a-b88a-616dc99f7c49 & 0.17 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.69 & 7.30 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3o-01a-11d-a39r-32 & 01f42235-30de-495c-8409-ab6e83afc85c & 0167780383 & 32 & 0.15 & 14 & no & 05 & 1 & a39r & d & 3.00 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3o-01a-11d-a39s-05 & dbeb5913-ab89-4266-b876-5533f5e8e775 & 0167780400 & 05 & 0.15 & 14 & no & 05 & 1 & a39s & d & 1.95 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3o-01a-11d-a40f-26 & 5e29c221-fbe9-4428-9c57-8c0b0aa28097 & 0167789049 & 26 & 0.15 & 14 & no & 05 & 1  & a40f & d  & 1.50 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3o-01a-11d-a39q-01 & 97c631a9-cd63-41b6-b023-aec0aa822d96 & 0167779920 & 01 & 0.15 & 14 & no & 05 & 1 & a39q & d & 1.01 & 23 & 6.70 & 2014 & 15.58 & dna & d & tcga-3h-ab3o-01a-11d & 0a1e3114-4bf9-4a28-921b-da6fc46f21a3 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3o-01a-11 & a7173d79-3029-4def-8a29-331b45a7870e & 01 & no & NA & 04 & 11 & NA & NA & tcga-3h-ab3o-01a-01-ts1 & 9ea61969-6557-4193-b2f9-efc8b5564976 & tcga-3h-ab3o-01a-01-ts1.9ea61969-6557-4193-b2f9-efc8b5564976.svs & no & NA & NA & NA & NA & 3  & 0  & 10 & 0  & 0 & 0  & 90  & 65 & top & 30.00 & 2014 & NA & 3b2c605a-9789-400e-b1b5-a8ceda10e484 & 20 & no & a45o & 21 & NA & tcga-3h-ab3o-01a-21-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t418.36.0.4 & nationwide children's hospital & 1 & meso & 07b76d74-59b8-4860-97b9-cef6a238d6a6 & 10 & false & tcga & 2015 & NA & NA & NA & occupational & NA          & NA & tcga-3h-ab3s & 1c848577-c3a4-4830-abfa-549992172396 & other, specify & leukemia & parent & other, specify & skin cancer & grandparent & other, specify & colon cancer & grandparent & other, specify & skin cancer & parent & NA & consented & united states & 27 & NA & NA & diffuse malignant mesothelioma - epithelioid & no & NA & NA & NA & NA & NA & NA & primary & 9  & -25471 & 385  & NA & NA & tcga-3h-ab3s-d60436 & a84d9fdf-cd5d-4db2-8b2c-16132ee4ae03 & NA & 9  & 275 & 221 & tarceva    & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3s-d60437 & d9802431-50ae-4651-b8e4-ec1fd4e7ccc9 & NA & 9  & 264 & 221 & gemcitabine & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & yes & 2014 & tcga-3h-ab3s-d60438 & 124e4ef8-8f87-4c1a-94ac-af766a3fd644 & NA & 9  & 271 & 221 & r1507      & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & NA                  & NA                                   & NA & NA & NA   & NA   & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA               & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA           & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & tcga-3h-ab3s-d60435 & b4870adf-ace0-4862-924e-213632306526 & NA & 9  & 79 & 47 & dasatinib & clinical progressive disease & 6 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & 1  & NA                     & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & male   & yes & no & c45.0 & 9052/3 & c38.4 & yes & NA & 6 & NA  & NA & NA & NA  & NA                   & NA                 & NA & NA & NA  & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA      & NA                 & NA & NA & NA  & NA  & NA & NA  & NA                   & NA                       & NA & NA & no  & ab3s & preoperative         & with tumor & NA & no  & yes & construction and extraction & NA                                               & NA & 69 & 0.7 & 1.3 & 0.8 & 0 & epithelioid mesothelioma             & right & biopsy & NA & NA & NA & NA & 9.6  & pleura & 2009 & yes & white & yes & NA & NA                  & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA   & NA & NA       & NA & NA & NA & NA & NA   & NA            & NA                  & NA                                   & NA & NA & NA  & NA  & NA                               & NA & NA & NA & NA & NA       & NA & NA & NA & NA & NA   & primary tumor field & tcga-3h-ab3s-r60439 & 523a6250-2d61-4da2-a41c-fe20dc2585b2 & NA & 9  & 152 & 132 & radiographic progressive disease & 6  & 15 & 45 & no & external & NA & NA & NA & gy & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ii  & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t2 & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 38 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - epithelioid & NA & no & NA & NA & NA & open surgery         & NA & NA & NA & NA & NA & NA & NA & NA & NA & 95089089-4fb4-45df-917a-939341f5c3ca & 38 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 00718caf-e2c8-4355-a23d-6b2def6b2278 & NA & NA & NA & NA & 189 & 1 & cryomold & 0 & NA & tcga-3h-ab3s-10a & 95089089-4fb4-45df-917a-939341f5c3ca & NA & NA & NA & 1743 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3s-10a-01d-a39u-32 & 04bce7fc-f943-4c4b-b6ff-7a597a6a2193 & 0167779886 & 32 & 0.16 & 14 & no & 05 & 1 & a39u & e & 3.20 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3s-10a-01d-a40g-26 & 4ca444f8-37bb-4b28-998d-10bbc104bcd8 & 0167823503 & 26 & 0.16 & 14 & no & 05 & 1  & a40g & e  & 1.60 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3s-10a-01d-a39t-01 & 935812e0-5a02-4e07-8188-610c05c36f07 & 0167789487 & 01 & 0.16 & 14 & no & 05 & 1 & a39t & e & 1.07 & 23 & 6.70 & 2014 & 70.99 & dna & d & tcga-3h-ab3s-10a-01d & ee3614d5-5b60-480f-bbc7-cf044b5e21d0 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3s-10a-01 & 4c64d7fe-4f60-410d-b4c7-68180cbc7227 & 03 & no & NA & 04 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3s-01a & 00718caf-e2c8-4355-a23d-6b2def6b2278 & NA & NA & NA & 1743 & NA & NA & b3743d6a-2dcc-46d5-9058-619f85b84138 & 943d90c4-82ac-4ece-982a-e4dff72b03bb & bc2dbeea-1c7c-41ad-97d4-92fbf0483200 & NA & NA & 170 & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3s.3a7d0d05-505c-4ba4-997e-e6bc9941ba72.pdf & 3a7d0d05-505c-4ba4-997e-e6bc9941ba72 & NA & 1.89 & NA & NA & NA & tcga-3h-ab3s-01a-21r-a40a-07 & 34787f03-da7e-4f3e-b33f-73b1bf51fb74 & 0167788970 & 07 & 0.17 & 14 & no & 05 & 3 & a40a & b & 2.21 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3s-01a-21r-a404-13 & c958df4e-8253-49b7-9e8e-214c9c12eb56 & 0167812010 & 13 & 0.17 & 14 & no & 05 & 3 & a404 & b & 3.40 & 23 & 20.00 & 2014 & 37.22 & rna & r & tcga-3h-ab3s-01a-21r & b0e9d19b-b731-45ab-8793-b6c6529a84af & 0.17 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.14 & 7.40 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3s-01a-21d-a39r-32 & ee0bc3e0-c05f-41c8-ba52-b159f15eaa02 & 0167780366 & 32 & 0.17 & 14 & no & 05 & 1 & a39r & e & 3.40 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3s-01a-21d-a39s-05 & 02c47e76-9119-4e9b-b634-13865e67f837 & 0167780399 & 05 & 0.17 & 14 & no & 05 & 1 & a39s & e & 2.21 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3s-01a-21d-a40f-26 & e79df96f-f79a-49f9-b0a3-e396336a980f & 0167789030 & 26 & 0.17 & 14 & no & 05 & 1  & a40f & e  & 1.70 & 23 & 10.00 & 2014 & NA & NA & tcga-3h-ab3s-01a-21d-a39q-01 & 0e45783d-8c43-4b1d-9f05-c4a439b6ea9a & 0167779919 & 01 & 0.17 & 14 & no & 05 & 1 & a39q & e & 1.14 & 23 & 6.70 & 2014 & 32.81 & dna & d & tcga-3h-ab3s-01a-21d & 768dab33-509c-4a42-b89f-4b4bf49faebb & 0.17 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3s-01a-21 & aa405c1d-1636-4eca-87ed-f09ba01df046 & 01 & no & NA & 04 & 21 & NA & NA & tcga-3h-ab3s-01a-02-ts2 & ceedf6b8-45ae-4337-9f7b-842273fa1eef & tcga-3h-ab3s-01a-02-ts2.ceedf6b8-45ae-4337-9f7b-842273fa1eef.svs & no & NA & NA & NA & NA & 3  & 0  & 0  & 0  & 0 & 0  & 100 & 65 & top & 30.00 & 2014 & NA & bf706147-4c7d-4c10-b27a-c4851b1b4dfa & 20 & no & a45o & 11 & NA & tcga-3h-ab3s-01a-11-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t418.36.0.5 & nationwide children's hospital & 1 & meso & f352e491-5cbd-4c60-8ada-0d12ea6218eb & 10 & false & tcga & 2015 & NA & NA & NA & occupational & NA          & NA & tcga-3h-ab3t & 3210027d-a9c7-4ebb-8497-e3705bed7e40 & NA             & NA       & NA     & NA             & NA          & NA          & NA             & NA           & NA          & NA             & NA          & NA     & NA & consented & united states & 21 & NA & NA & diffuse malignant mesothelioma - biphasic    & no & NA & NA & NA & NA & NA & NA & primary & 9  & -28925 & 414  & NA & NA & tcga-3h-ab3t-d60426 & 327253c9-ef77-46b9-8c37-55ec0dcc5798 & NA & 9  & 370 & 321 & vorinostat & clinical progressive disease & 6  & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA         & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA   & NA   & NA          & NA               & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA          & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & NA                  & NA                                   & NA & NA & NA  & NA  & NA           & NA             & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                         & NA & NA & NA & NA  & NA   & tcga-3h-ab3t-d60425 & d6a1ee2f-fe07-4f9f-8a8b-11e8fb7f4479 & NA & 9  & 61 & 35 & dasatinib & clinical progressive disease & 6 & NA & NA & NA & NA & NA & NA & NA & no & targeted molecular therapy & NA & NA & NA & yes & 2014 & 0  & NA                     & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & male   & yes & no & c45.0 & 9053/3 & c38.4 & yes & NA & 6 & NA  & NA & NA & NA  & NA                   & NA                 & NA & NA & NA  & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA   & NA                       & NA & NA & NA & NA & NA & NA  & NA      & NA                 & NA & NA & NA  & NA  & NA & 305 & ipsilateral pleura   & intrapleural progression & NA & no & yes & ab3t & preoperative         & with tumor & NA & no  & yes & other, please specify       & oil refinery                                     & NA & 79 & 0.7 & 1.3 & 0.9 & 0 & biphasic mesothelioma                & right & biopsy & NA & NA & NA & NA & 7.9  & pleura & 2009 & no  & white & yes & NA & regional site       & tcga-3h-ab3t-r60428 & ce82d81c-9bce-41e2-bb9f-18cfffe6f385 & NA & 9  & 152 & 117 & radiographic progressive disease & 6  & 25 & 50.4 & no & external & NA & NA & NA & gy & 2014 & regional site & tcga-3h-ab3t-r60429 & aa8b80b9-ec9f-48c9-a832-948e5b97de85 & NA & 9  & 152 & 117 & radiographic progressive disease & 6  & 25 & 45 & no & external & NA & NA & NA & gy & 2014 & primary tumor field & tcga-3h-ab3t-r60427 & ede6cfef-42e5-4872-91eb-d343e0742d98 & NA & 9  & 152 & 117 & radiographic progressive disease & 6  & 25 & 60 & no & external & NA & NA & NA & gy & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage i   & NA & NA & NA & 6th & NA & NA & NA & m0 & n0 & t1 & no & yes & 3h & dead & 2014 & canonical & NA & consented & united states & NA & NA & 25 & NA & NA & NA & NA & pleura & diffuse malignant mesothelioma - biphasic    & NA & no & NA & NA & NA & thoracoscopic biopsy & NA & NA & NA & NA & NA & NA & NA & NA & NA & 60edc839-85df-448b-93f0-5a31081f46d4 & 25 & NA & NA & NA & NA & blood draw & NA & NA & NA & 1 & NA & NA & NA & yes & NA & NA & NA & white & NA & no & pleura & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 9e43d37d-b244-4a37-9e41-cfbd27008ffe & NA & NA & NA & NA & 91  & 1 & cryomold & 0 & NA & tcga-3h-ab3t-10a & 60edc839-85df-448b-93f0-5a31081f46d4 & NA & NA & NA & 1709 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & tcga-3h-ab3t-10a-01d-a39u-32 & 45693e43-a19b-401e-906f-993fc1183055 & 0167779876 & 32 & 0.16 & 14 & no & 05 & 1 & a39u & f & 3.20 & 23 & 20.00 & 2014 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & tcga-3h-ab3t-10a-01d-a39t-01 & 6a2b7fbb-fb90-4d3e-846d-c839efe25929 & 0167789475 & 01 & 0.16 & 14 & no & 05 & 1 & a39t & f & 1.07 & 23 & 6.70 & 2014 & 46.03 & dna & d & tcga-3h-ab3t-10a-01d & 7d654d01-879c-43e8-b883-4d27f0ce5014 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & NA & NA & tcga-3h-ab3t-10a-01 & 4a80c890-71e7-48b0-9b60-d469755cefa6 & 20 & no & NA & 03 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & tcga-3h-ab3t-01a & 9e43d37d-b244-4a37-9e41-cfbd27008ffe & NA & NA & NA & 1709 & NA & NA & NA                                   & NA                                   & NA                                   & NA & NA & 80  & NA & no & NA & NA & NA & false & NA & tcga-3h-ab3t.4f0240da-bf51-4d8d-a378-1b39eefd5fea.pdf & 4f0240da-bf51-4d8d-a378-1b39eefd5fea & NA & 1.86 & NA & NA & NA & tcga-3h-ab3t-01a-11r-a40a-07 & b5bda3f8-1a43-4847-88b6-6a17ce11d29b & 0167788965 & 07 & 0.16 & 14 & no & 05 & 3 & a40a & c & 2.08 & 23 & 13.00 & 2014 & NA & NA & tcga-3h-ab3t-01a-11r-a404-13 & 0984c27a-9947-4ac5-8ec4-11eccc08674e & 0167812005 & 13 & 0.16 & 14 & no & 05 & 3 & a404 & c & 3.20 & 23 & 20.00 & 2014 & 17.84 & rna & r & tcga-3h-ab3t-01a-11r & 59dd66f3-84c6-4cb7-95cd-4108e3dd7327 & 0.16 & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.34 & 9.20 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-3h-ab3t-01a-11d-a39r-32 & 8645e7c4-b126-4f98-bc0f-44fa8787d5c8 & 0167780356 & 32 & 0.17 & 14 & no & 05 & 1 & a39r & f & 3.40 & 23 & 20.00 & 2014 & NA & NA & tcga-3h-ab3t-01a-11d-a39s-05 & 57392d32-3cf9-4f5c-926d-eacdc72aeae3 & 0167780376 & 05 & 0.17 & 14 & no & 05 & 1 & a39s & f & 2.21 & 23 & 13.00 & 2014 & NA & NA & NA                           & NA                                   & NA         & NA & NA   & NA & NA & NA & NA & NA   & NA & NA   & NA & NA    & NA   & NA & NA & tcga-3h-ab3t-01a-11d-a39q-01 & f6adb8e5-5005-42f8-92e9-3a243e992e28 & 0167779896 & 01 & 0.17 & 14 & no & 05 & 1 & a39q & f & 1.14 & 23 & 6.70 & 2014 & 9.69  & dna & d & tcga-3h-ab3t-01a-11d & 4c4c8973-9ba5-4a4f-a548-e6b4aa8a10e9 & 0.17 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-3h-ab3t-01a-11 & 6fd358e7-f06d-4b0b-b314-0b850b9473eb & 17 & no & NA & 03 & 11 & NA & NA & tcga-3h-ab3t-01a-01-ts1 & 690e2ed4-0232-4e58-b3f3-751e85261d64 & tcga-3h-ab3t-01a-01-ts1.690e2ed4-0232-4e58-b3f3-751e85261d64.svs & no & NA & NA & NA & NA & 70 & 10 & 0  & 10 & 0 & 20 & 80  & 80 & top & 30.00 & 2014 & NA & d34e3172-8faf-4be1-828d-2c158a0290ff & 20 & no & a45o & 21 & NA & tcga-3h-ab3t-01a-21-a45o-20 & 30 & 7 & 2014 & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 894\n",
       "\n",
       "| <!--/--> | admin.bcr &lt;chr&gt; | admin.day_of_dcc_upload &lt;chr&gt; | admin.disease_code &lt;chr&gt; | admin.file_uuid &lt;chr&gt; | admin.month_of_dcc_upload &lt;chr&gt; | admin.patient_withdrawal.withdrawn &lt;chr&gt; | admin.project_code &lt;chr&gt; | admin.year_of_dcc_upload &lt;chr&gt; | patient.additional_studies &lt;chr&gt; | patient.asbestos_exposure_age &lt;chr&gt; | ⋯ ⋯ | patient.samples.sample.preservation_method &lt;chr&gt; | patient.samples.sample.sample_type &lt;chr&gt; | patient.samples.sample.sample_type_id &lt;chr&gt; | patient.samples.sample.shortest_dimension &lt;chr&gt; | patient.samples.sample.time_between_clamping_and_freezing &lt;chr&gt; | patient.samples.sample.time_between_excision_and_freezing &lt;chr&gt; | patient.samples.sample.tissue_type &lt;chr&gt; | patient.samples.sample.tumor_descriptor &lt;chr&gt; | patient.samples.sample.tumor_pathology &lt;chr&gt; | patient.samples.sample.vial_number &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| 418.36.0 | nationwide children's hospital | 1 | meso | be213c85-e379-42fc-918a-1fe13c4d0201 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 418.36.0.1 | nationwide children's hospital | 1 | meso | 7a8ecc55-002e-4650-98f8-84806b919c17 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 418.36.0.2 | nationwide children's hospital | 1 | meso | eb988691-41bd-412e-9e6c-82bd7050dc1b | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 418.36.0.3 | nationwide children's hospital | 1 | meso | eb48bed3-00e5-45aa-8803-6ce81c7b1fff | 10 | false | tcga | 2015 | NA | 25 | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 418.36.0.4 | nationwide children's hospital | 1 | meso | 07b76d74-59b8-4860-97b9-cef6a238d6a6 | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 418.36.0.5 | nationwide children's hospital | 1 | meso | f352e491-5cbd-4c60-8ada-0d12ea6218eb | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "\n"
      ],
      "text/plain": [
       "           admin.bcr                      admin.day_of_dcc_upload\n",
       "418.36.0   nationwide children's hospital 1                      \n",
       "418.36.0.1 nationwide children's hospital 1                      \n",
       "418.36.0.2 nationwide children's hospital 1                      \n",
       "418.36.0.3 nationwide children's hospital 1                      \n",
       "418.36.0.4 nationwide children's hospital 1                      \n",
       "418.36.0.5 nationwide children's hospital 1                      \n",
       "           admin.disease_code admin.file_uuid                     \n",
       "418.36.0   meso               be213c85-e379-42fc-918a-1fe13c4d0201\n",
       "418.36.0.1 meso               7a8ecc55-002e-4650-98f8-84806b919c17\n",
       "418.36.0.2 meso               eb988691-41bd-412e-9e6c-82bd7050dc1b\n",
       "418.36.0.3 meso               eb48bed3-00e5-45aa-8803-6ce81c7b1fff\n",
       "418.36.0.4 meso               07b76d74-59b8-4860-97b9-cef6a238d6a6\n",
       "418.36.0.5 meso               f352e491-5cbd-4c60-8ada-0d12ea6218eb\n",
       "           admin.month_of_dcc_upload admin.patient_withdrawal.withdrawn\n",
       "418.36.0   10                        false                             \n",
       "418.36.0.1 10                        false                             \n",
       "418.36.0.2 10                        false                             \n",
       "418.36.0.3 10                        false                             \n",
       "418.36.0.4 10                        false                             \n",
       "418.36.0.5 10                        false                             \n",
       "           admin.project_code admin.year_of_dcc_upload\n",
       "418.36.0   tcga               2015                    \n",
       "418.36.0.1 tcga               2015                    \n",
       "418.36.0.2 tcga               2015                    \n",
       "418.36.0.3 tcga               2015                    \n",
       "418.36.0.4 tcga               2015                    \n",
       "418.36.0.5 tcga               2015                    \n",
       "           patient.additional_studies patient.asbestos_exposure_age ⋯\n",
       "418.36.0   NA                         NA                            ⋯\n",
       "418.36.0.1 NA                         NA                            ⋯\n",
       "418.36.0.2 NA                         NA                            ⋯\n",
       "418.36.0.3 NA                         25                            ⋯\n",
       "418.36.0.4 NA                         NA                            ⋯\n",
       "418.36.0.5 NA                         NA                            ⋯\n",
       "           patient.samples.sample.preservation_method\n",
       "418.36.0   NA                                        \n",
       "418.36.0.1 NA                                        \n",
       "418.36.0.2 NA                                        \n",
       "418.36.0.3 NA                                        \n",
       "418.36.0.4 NA                                        \n",
       "418.36.0.5 NA                                        \n",
       "           patient.samples.sample.sample_type\n",
       "418.36.0   primary tumor                     \n",
       "418.36.0.1 primary tumor                     \n",
       "418.36.0.2 primary tumor                     \n",
       "418.36.0.3 primary tumor                     \n",
       "418.36.0.4 primary tumor                     \n",
       "418.36.0.5 primary tumor                     \n",
       "           patient.samples.sample.sample_type_id\n",
       "418.36.0   01                                   \n",
       "418.36.0.1 01                                   \n",
       "418.36.0.2 01                                   \n",
       "418.36.0.3 01                                   \n",
       "418.36.0.4 01                                   \n",
       "418.36.0.5 01                                   \n",
       "           patient.samples.sample.shortest_dimension\n",
       "418.36.0   NA                                       \n",
       "418.36.0.1 NA                                       \n",
       "418.36.0.2 NA                                       \n",
       "418.36.0.3 NA                                       \n",
       "418.36.0.4 NA                                       \n",
       "418.36.0.5 NA                                       \n",
       "           patient.samples.sample.time_between_clamping_and_freezing\n",
       "418.36.0   NA                                                       \n",
       "418.36.0.1 NA                                                       \n",
       "418.36.0.2 NA                                                       \n",
       "418.36.0.3 NA                                                       \n",
       "418.36.0.4 NA                                                       \n",
       "418.36.0.5 NA                                                       \n",
       "           patient.samples.sample.time_between_excision_and_freezing\n",
       "418.36.0   NA                                                       \n",
       "418.36.0.1 NA                                                       \n",
       "418.36.0.2 NA                                                       \n",
       "418.36.0.3 NA                                                       \n",
       "418.36.0.4 NA                                                       \n",
       "418.36.0.5 NA                                                       \n",
       "           patient.samples.sample.tissue_type\n",
       "418.36.0   NA                                \n",
       "418.36.0.1 NA                                \n",
       "418.36.0.2 NA                                \n",
       "418.36.0.3 NA                                \n",
       "418.36.0.4 NA                                \n",
       "418.36.0.5 NA                                \n",
       "           patient.samples.sample.tumor_descriptor\n",
       "418.36.0   NA                                     \n",
       "418.36.0.1 NA                                     \n",
       "418.36.0.2 NA                                     \n",
       "418.36.0.3 NA                                     \n",
       "418.36.0.4 NA                                     \n",
       "418.36.0.5 NA                                     \n",
       "           patient.samples.sample.tumor_pathology\n",
       "418.36.0   NA                                    \n",
       "418.36.0.1 NA                                    \n",
       "418.36.0.2 NA                                    \n",
       "418.36.0.3 NA                                    \n",
       "418.36.0.4 NA                                    \n",
       "418.36.0.5 NA                                    \n",
       "           patient.samples.sample.vial_number\n",
       "418.36.0   a                                 \n",
       "418.36.0.1 a                                 \n",
       "418.36.0.2 a                                 \n",
       "418.36.0.3 a                                 \n",
       "418.36.0.4 a                                 \n",
       "418.36.0.5 a                                 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'meso'\n",
    "df = clinical_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LIHC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>377</li>\n",
       "\t<li>1587</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 377\n",
       "\\item 1587\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 377\n",
       "2. 1587\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]  377 1587"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 1587</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>admin.bcr</th><th scope=col>admin.day_of_dcc_upload</th><th scope=col>admin.disease_code</th><th scope=col>admin.file_uuid</th><th scope=col>admin.month_of_dcc_upload</th><th scope=col>admin.patient_withdrawal.withdrawn</th><th scope=col>admin.project_code</th><th scope=col>admin.year_of_dcc_upload</th><th scope=col>patient.ablation_embolization_tx_adjuvant</th><th scope=col>patient.additional_studies</th><th scope=col>⋯</th><th scope=col>patient.samples.sample.preservation_method</th><th scope=col>patient.samples.sample.sample_type</th><th scope=col>patient.samples.sample.sample_type_id</th><th scope=col>patient.samples.sample.shortest_dimension</th><th scope=col>patient.samples.sample.time_between_clamping_and_freezing</th><th scope=col>patient.samples.sample.time_between_excision_and_freezing</th><th scope=col>patient.samples.sample.tissue_type</th><th scope=col>patient.samples.sample.tumor_descriptor</th><th scope=col>patient.samples.sample.tumor_pathology</th><th scope=col>patient.samples.sample.vial_number</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>⋯</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>384.44.0</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>47a9da83-f87b-45a0-a501-542c5a9df212</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>no</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>399.42.0</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>ada84bc2-9724-428c-9557-7280af5f8297</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>no</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>399.42.0.1</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>b9f63d47-2293-4360-b373-5acdaa2ec7cb</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>NA</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>399.42.0.2</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>2b488ba2-f46b-4d0b-8c73-d9a761ab61b1</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>no</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>399.42.0.3</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>69fc70f4-fe6a-472a-b354-cf37515c524d</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>no</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "\t<tr><th scope=row>399.42.0.4</th><td>nationwide children's hospital</td><td>1</td><td>lihc</td><td>14e975d3-8375-462a-aa65-62b2297a360e</td><td>10</td><td>false</td><td>tcga</td><td>2015</td><td>no</td><td>NA</td><td>⋯</td><td>NA</td><td>primary tumor</td><td>01</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>NA</td><td>a</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 1587\n",
       "\\begin{tabular}{r|lllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll}\n",
       "  & admin.bcr & admin.day\\_of\\_dcc\\_upload & admin.disease\\_code & admin.file\\_uuid & admin.month\\_of\\_dcc\\_upload & admin.patient\\_withdrawal.withdrawn & admin.project\\_code & admin.year\\_of\\_dcc\\_upload & patient.ablation\\_embolization\\_tx\\_adjuvant & patient.additional\\_studies & patient.adjacent\\_hepatic\\_tissue\\_inflammation\\_extent\\_type & patient.age\\_at\\_initial\\_pathologic\\_diagnosis & patient.bcr\\_patient\\_barcode & patient.bcr\\_patient\\_uuid & patient.cancer\\_diagnosis\\_first\\_degree\\_relative\\_number & patient.child\\_pugh\\_classification\\_grade & patient.clinical\\_cqcf.anatomic\\_neoplasm\\_subdivision & patient.clinical\\_cqcf.consent\\_or\\_death\\_status & patient.clinical\\_cqcf.country & patient.clinical\\_cqcf.days\\_to\\_consent & patient.clinical\\_cqcf.days\\_to\\_death & patient.clinical\\_cqcf.diagnosis\\_subtype & patient.clinical\\_cqcf.histological\\_type & patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.clinical\\_cqcf.normal\\_tissue\\_anatomic\\_site & patient.clinical\\_cqcf.normal\\_tissue\\_proximity & patient.clinical\\_cqcf.other\\_anatomic\\_site & patient.clinical\\_cqcf.other\\_diagnosis & patient.clinical\\_cqcf.prior\\_dx & patient.clinical\\_cqcf.tumor\\_focality & patient.clinical\\_cqcf.tumor\\_type & patient.day\\_of\\_form\\_completion & patient.days\\_to\\_birth & patient.days\\_to\\_death & patient.days\\_to\\_definitive\\_surgical\\_procedure\\_performed & patient.days\\_to\\_initial\\_pathologic\\_diagnosis & patient.days\\_to\\_last\\_followup & patient.days\\_to\\_last\\_known\\_alive & patient.drugs & patient.drugs.drug-2.bcr\\_drug\\_barcode & patient.drugs.drug-2.bcr\\_drug\\_uuid & patient.drugs.drug-2.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-2.day\\_of\\_form\\_completion & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-2.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-2.drug\\_name & patient.drugs.drug-2.measure\\_of\\_response & patient.drugs.drug-2.month\\_of\\_form\\_completion & patient.drugs.drug-2.number\\_cycles & patient.drugs.drug-2.prescribed\\_dose & patient.drugs.drug-2.prescribed\\_dose\\_units & patient.drugs.drug-2.regimen\\_indication & patient.drugs.drug-2.regimen\\_indication\\_notes & patient.drugs.drug-2.regimen\\_number & patient.drugs.drug-2.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-2.therapy\\_ongoing & patient.drugs.drug-2.therapy\\_types.therapy\\_type & patient.drugs.drug-2.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-2.total\\_dose & patient.drugs.drug-2.total\\_dose\\_units & patient.drugs.drug-2.tx\\_on\\_clinical\\_trial & patient.drugs.drug-2.year\\_of\\_form\\_completion & patient.drugs.drug-3.bcr\\_drug\\_barcode & patient.drugs.drug-3.bcr\\_drug\\_uuid & patient.drugs.drug-3.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-3.day\\_of\\_form\\_completion & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-3.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-3.drug\\_name & patient.drugs.drug-3.measure\\_of\\_response & patient.drugs.drug-3.month\\_of\\_form\\_completion & patient.drugs.drug-3.number\\_cycles & patient.drugs.drug-3.prescribed\\_dose & patient.drugs.drug-3.prescribed\\_dose\\_units & patient.drugs.drug-3.regimen\\_indication & patient.drugs.drug-3.regimen\\_indication\\_notes & patient.drugs.drug-3.regimen\\_number & patient.drugs.drug-3.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-3.therapy\\_ongoing & patient.drugs.drug-3.therapy\\_types.therapy\\_type & patient.drugs.drug-3.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-3.total\\_dose & patient.drugs.drug-3.total\\_dose\\_units & patient.drugs.drug-3.tx\\_on\\_clinical\\_trial & patient.drugs.drug-3.year\\_of\\_form\\_completion & patient.drugs.drug-4.bcr\\_drug\\_barcode & patient.drugs.drug-4.bcr\\_drug\\_uuid & patient.drugs.drug-4.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-4.day\\_of\\_form\\_completion & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-4.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-4.drug\\_name & patient.drugs.drug-4.measure\\_of\\_response & patient.drugs.drug-4.month\\_of\\_form\\_completion & patient.drugs.drug-4.number\\_cycles & patient.drugs.drug-4.prescribed\\_dose & patient.drugs.drug-4.prescribed\\_dose\\_units & patient.drugs.drug-4.regimen\\_indication & patient.drugs.drug-4.regimen\\_indication\\_notes & patient.drugs.drug-4.regimen\\_number & patient.drugs.drug-4.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-4.therapy\\_ongoing & patient.drugs.drug-4.therapy\\_types.therapy\\_type & patient.drugs.drug-4.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-4.total\\_dose & patient.drugs.drug-4.total\\_dose\\_units & patient.drugs.drug-4.tx\\_on\\_clinical\\_trial & patient.drugs.drug-4.year\\_of\\_form\\_completion & patient.drugs.drug-5.bcr\\_drug\\_barcode & patient.drugs.drug-5.bcr\\_drug\\_uuid & patient.drugs.drug-5.clinical\\_trail\\_drug\\_classification & patient.drugs.drug-5.day\\_of\\_form\\_completion & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug-5.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug-5.drug\\_name & patient.drugs.drug-5.measure\\_of\\_response & patient.drugs.drug-5.month\\_of\\_form\\_completion & patient.drugs.drug-5.number\\_cycles & patient.drugs.drug-5.prescribed\\_dose & patient.drugs.drug-5.prescribed\\_dose\\_units & patient.drugs.drug-5.regimen\\_indication & patient.drugs.drug-5.regimen\\_indication\\_notes & patient.drugs.drug-5.regimen\\_number & patient.drugs.drug-5.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug-5.therapy\\_ongoing & patient.drugs.drug-5.therapy\\_types.therapy\\_type & patient.drugs.drug-5.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug-5.total\\_dose & patient.drugs.drug-5.total\\_dose\\_units & patient.drugs.drug-5.tx\\_on\\_clinical\\_trial & patient.drugs.drug-5.year\\_of\\_form\\_completion & patient.drugs.drug.bcr\\_drug\\_barcode & patient.drugs.drug.bcr\\_drug\\_uuid & patient.drugs.drug.clinical\\_trail\\_drug\\_classification & patient.drugs.drug.day\\_of\\_form\\_completion & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_end & patient.drugs.drug.days\\_to\\_drug\\_therapy\\_start & patient.drugs.drug.drug\\_name & patient.drugs.drug.measure\\_of\\_response & patient.drugs.drug.month\\_of\\_form\\_completion & patient.drugs.drug.number\\_cycles & patient.drugs.drug.prescribed\\_dose & patient.drugs.drug.prescribed\\_dose\\_units & patient.drugs.drug.regimen\\_indication & patient.drugs.drug.regimen\\_indication\\_notes & patient.drugs.drug.regimen\\_number & patient.drugs.drug.route\\_of\\_administrations.route\\_of\\_administration & patient.drugs.drug.therapy\\_ongoing & patient.drugs.drug.therapy\\_types.therapy\\_type & patient.drugs.drug.therapy\\_types.therapy\\_type\\_notes & patient.drugs.drug.total\\_dose & patient.drugs.drug.total\\_dose\\_units & patient.drugs.drug.tx\\_on\\_clinical\\_trial & patient.drugs.drug.year\\_of\\_form\\_completion & patient.eastern\\_cancer\\_oncology\\_group & patient.ethnicity & patient.follow\\_ups & patient.follow\\_ups.follow\\_up-2.ablation\\_embolization\\_tx\\_adjuvant & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-2.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-2.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.days\\_to\\_death & patient.follow\\_ups.follow\\_up-2.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-2.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-2.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-2.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event-2.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up-2.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-2.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-2.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up-2.radiation\\_therapy & patient.follow\\_ups.follow\\_up-2.vital\\_status & patient.follow\\_ups.follow\\_up-2.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.ablation\\_embolization\\_tx\\_adjuvant & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-3.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-3.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.days\\_to\\_death & patient.follow\\_ups.follow\\_up-3.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-3.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-3.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-3.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up-3.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-3.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-3.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up-3.radiation\\_therapy & patient.follow\\_ups.follow\\_up-3.vital\\_status & patient.follow\\_ups.follow\\_up-3.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.ablation\\_embolization\\_tx\\_adjuvant & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up-4.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up-4.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.days\\_to\\_death & patient.follow\\_ups.follow\\_up-4.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up-4.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up-4.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up-4.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up-4.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up-4.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up-4.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up-4.radiation\\_therapy & patient.follow\\_ups.follow\\_up-4.vital\\_status & patient.follow\\_ups.follow\\_up-4.year\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.ablation\\_embolization\\_tx\\_adjuvant & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_barcode & patient.follow\\_ups.follow\\_up.bcr\\_followup\\_uuid & patient.follow\\_ups.follow\\_up.day\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.days\\_to\\_death & patient.follow\\_ups.follow\\_up.days\\_to\\_last\\_followup & patient.follow\\_ups.follow\\_up.followup\\_case\\_report\\_form\\_submission\\_reason & patient.follow\\_ups.follow\\_up.lost\\_follow\\_up & patient.follow\\_ups.follow\\_up.month\\_of\\_form\\_completion & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-2.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-3.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event-4.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_liver\\_transplant & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.follow\\_ups.follow\\_up.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.follow\\_ups.follow\\_up.person\\_neoplasm\\_cancer\\_status & patient.follow\\_ups.follow\\_up.postoperative\\_rx\\_tx & patient.follow\\_ups.follow\\_up.radiation\\_therapy & patient.follow\\_ups.follow\\_up.vital\\_status & patient.follow\\_ups.follow\\_up.year\\_of\\_form\\_completion & patient.gender & patient.height & patient.hematology\\_serum\\_creatinine\\_laboratory\\_result\\_value\\_in\\_mg\\_dl & patient.histological\\_type & patient.history\\_hepato\\_carcinoma\\_risk\\_factors.history\\_hepato\\_carcinoma\\_risk\\_factor & patient.history\\_hepato\\_carcinoma\\_risk\\_factors.history\\_hepato\\_carcinoma\\_risk\\_factor-2 & patient.history\\_hepato\\_carcinoma\\_risk\\_factors.history\\_hepato\\_carcinoma\\_risk\\_factor-3 & patient.history\\_hepato\\_carcinoma\\_risk\\_factors.history\\_hepato\\_carcinoma\\_risk\\_factor-4 & patient.history\\_hepato\\_carcinoma\\_risk\\_factors.history\\_hepato\\_carcinoma\\_risk\\_factor\\_other & patient.history\\_of\\_neoadjuvant\\_treatment & patient.icd\\_10 & patient.icd\\_o\\_3\\_histology & patient.icd\\_o\\_3\\_site & patient.informed\\_consent\\_verified & patient.lab\\_procedure\\_platelet\\_result\\_specified\\_value & patient.laboratory\\_prcoedure\\_platelet\\_result\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_prcoedure\\_platelet\\_result\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_albumin\\_result\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_albumin\\_result\\_specified\\_value & patient.laboratory\\_procedure\\_albumin\\_result\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_alpha\\_fetoprotein\\_outcome\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_alpha\\_fetoprotein\\_outcome\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_alpha\\_fetoprotein\\_outcome\\_value & patient.laboratory\\_procedure\\_creatinine\\_result\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_creatinine\\_result\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_international\\_normalization\\_ratio\\_result\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_international\\_normalization\\_ratio\\_result\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_prothrombin\\_time\\_result\\_value & patient.laboratory\\_procedure\\_total\\_bilirubin\\_result\\_specified\\_lower\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_total\\_bilirubin\\_result\\_specified\\_upper\\_limit\\_of\\_normal\\_value & patient.laboratory\\_procedure\\_total\\_bilirubin\\_result\\_upper\\_limit\\_normal\\_value & patient.liver\\_fibrosis\\_ishak\\_score\\_category & patient.month\\_of\\_form\\_completion & patient.neoplasm\\_histologic\\_grade & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.new\\_tumor\\_event\\_liver\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-2.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.new\\_tumor\\_event\\_liver\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-3.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.new\\_tumor\\_event\\_liver\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event-4.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_pharmaceutical\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.additional\\_radiation\\_therapy & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.new\\_tumor\\_events.new\\_tumor\\_event.days\\_to\\_new\\_tumor\\_event\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_occurrence\\_anatomic\\_site & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_event\\_type & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_neoplasm\\_occurrence\\_anatomic\\_site\\_text & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_ablation\\_embo\\_tx & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_additional\\_surgery\\_procedure & patient.new\\_tumor\\_events.new\\_tumor\\_event.new\\_tumor\\_event\\_liver\\_transplant & patient.new\\_tumor\\_events.new\\_tumor\\_event.residual\\_disease\\_post\\_new\\_tumor\\_event\\_margin\\_status & patient.new\\_tumor\\_events.new\\_tumor\\_event\\_after\\_initial\\_treatment & patient.patient\\_id & patient.person\\_neoplasm\\_cancer\\_status & patient.postoperative\\_rx\\_tx & patient.prior\\_dx & patient.race & patient.radiation\\_therapy & patient.radiations & patient.radiations.radiation.anatomic\\_treatment\\_site & patient.radiations.radiation.bcr\\_radiation\\_barcode & patient.radiations.radiation.bcr\\_radiation\\_uuid & patient.radiations.radiation.course\\_number & patient.radiations.radiation.day\\_of\\_form\\_completion & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_end & patient.radiations.radiation.days\\_to\\_radiation\\_therapy\\_start & patient.radiations.radiation.measure\\_of\\_response & patient.radiations.radiation.month\\_of\\_form\\_completion & patient.radiations.radiation.numfractions & patient.radiations.radiation.radiation\\_dosage & patient.radiations.radiation.radiation\\_treatment\\_ongoing & patient.radiations.radiation.radiation\\_type & patient.radiations.radiation.radiation\\_type\\_notes & patient.radiations.radiation.regimen\\_indication & patient.radiations.radiation.regimen\\_indication\\_notes & patient.radiations.radiation.units & patient.radiations.radiation.year\\_of\\_form\\_completion & patient.relative\\_family\\_cancer\\_history\\_ind\\_3 & patient.residual\\_tumor & patient.specimen\\_collection\\_method\\_name & patient.stage\\_event.ann\\_arbor.b\\_symptoms & patient.stage\\_event.ann\\_arbor.extranodal\\_involvement & patient.stage\\_event.clinical\\_stage & patient.stage\\_event.gleason\\_grading.gleason\\_score & patient.stage\\_event.gleason\\_grading.primary\\_pattern & patient.stage\\_event.gleason\\_grading.secondary\\_pattern & patient.stage\\_event.gleason\\_grading.tertiary\\_pattern & patient.stage\\_event.igcccg\\_stage & patient.stage\\_event.masaoka\\_stage & patient.stage\\_event.pathologic\\_stage & patient.stage\\_event.psa.days\\_to\\_psa & patient.stage\\_event.psa.psa\\_value & patient.stage\\_event.serum\\_markers & patient.stage\\_event.system\\_version & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_m & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_n & patient.stage\\_event.tnm\\_categories.clinical\\_categories.clinical\\_t & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n & patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t & patient.surgical\\_procedure\\_name\\_other\\_specify\\_text & patient.tissue\\_prospective\\_collection\\_indicator & patient.tissue\\_retrospective\\_collection\\_indicator & patient.tissue\\_source\\_site & patient.tumor\\_tissue\\_site & patient.vascular\\_tumor\\_cell\\_invasion\\_type & patient.viral\\_hepatitis\\_serologies.viral\\_hepatitis\\_serology & patient.viral\\_hepatitis\\_serologies.viral\\_hepatitis\\_serology-2 & patient.viral\\_hepatitis\\_serologies.viral\\_hepatitis\\_serology-3 & patient.viral\\_hepatitis\\_serologies.viral\\_hepatitis\\_serology-4 & patient.viral\\_hepatitis\\_serologies.viral\\_hepatitis\\_serology-5 & patient.vital\\_status & patient.weight & patient.year\\_of\\_form\\_completion & patient.year\\_of\\_initial\\_pathologic\\_diagnosis & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-2 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_reasons.bcr\\_canonical\\_reason-3 & patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status & patient.biospecimen\\_cqcf.b\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status & patient.biospecimen\\_cqcf.country & patient.biospecimen\\_cqcf.cytogenetic\\_report\\_submitted & patient.biospecimen\\_cqcf.days\\_to\\_pathology\\_review & patient.biospecimen\\_cqcf.days\\_to\\_sample\\_procurement & patient.biospecimen\\_cqcf.differential\\_report\\_submitted & patient.biospecimen\\_cqcf.digital\\_image\\_submitted & patient.biospecimen\\_cqcf.ethnicity & patient.biospecimen\\_cqcf.ffpe\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site & patient.biospecimen\\_cqcf.histological\\_type & patient.biospecimen\\_cqcf.histological\\_type\\_other & patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment & patient.biospecimen\\_cqcf.hiv\\_positive\\_status & patient.biospecimen\\_cqcf.laterality & patient.biospecimen\\_cqcf.maximum\\_tumor\\_dimension & patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-2.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-3.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control-4.vial\\_number & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.days\\_to\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_conc & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qm & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_qty & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.extracted\\_dna.ncedna\\_dna\\_vol & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.method\\_of\\_normal\\_sample\\_procurement & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_control\\_type & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_anatomic\\_site & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.normal\\_tissue.normal\\_tissue\\_proximity & patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number & patient.biospecimen\\_cqcf.other\\_method\\_of\\_sample\\_procurement & patient.biospecimen\\_cqcf.other\\_vessel\\_used & patient.biospecimen\\_cqcf.path\\_confirm\\_diagnosis\\_matching & patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_necrosis\\_metrics & patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_nuclei\\_metrics & patient.biospecimen\\_cqcf.percent\\_myeloblasts\\_for\\_submitted\\_specimen & patient.biospecimen\\_cqcf.race & patient.biospecimen\\_cqcf.reason\\_path\\_confirm\\_diagnosis\\_not\\_matching & patient.biospecimen\\_cqcf.sample\\_prescreened & patient.biospecimen\\_cqcf.site\\_of\\_disease & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_description & patient.biospecimen\\_cqcf.site\\_of\\_disease\\_text & patient.biospecimen\\_cqcf.submitted\\_for\\_lce & patient.biospecimen\\_cqcf.t\\_cell\\_tumor\\_slide\\_submitted & patient.biospecimen\\_cqcf.top\\_slide\\_submitted & patient.biospecimen\\_cqcf.total\\_cells\\_submitted & patient.biospecimen\\_cqcf.tumor\\_samples & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-2.vial\\_number & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample-3.vial\\_number & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.days\\_to\\_tumor\\_sample\\_procurement & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_necrosis\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_nuclei\\_percent & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_pathology & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_weight & patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number & patient.biospecimen\\_cqcf.vessel\\_used & patient.days\\_to\\_index & patient.samples.sample-2.additional\\_studies & patient.samples.sample-2.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-2.bcr\\_sample\\_barcode & patient.samples.sample-2.bcr\\_sample\\_uuid & patient.samples.sample-2.biospecimen\\_sequence & patient.samples.sample-2.composition & patient.samples.sample-2.current\\_weight & patient.samples.sample-2.days\\_to\\_collection & patient.samples.sample-2.days\\_to\\_sample\\_procurement & patient.samples.sample-2.diagnostic\\_slides.additional\\_studies & patient.samples.sample-2.freezing\\_method & patient.samples.sample-2.initial\\_weight & patient.samples.sample-2.intermediate\\_dimension & patient.samples.sample-2.is\\_ffpe & patient.samples.sample-2.longest\\_dimension & patient.samples.sample-2.method\\_of\\_sample\\_procurement & patient.samples.sample-2.ncnnct\\_othmethonsp & patient.samples.sample-2.oct\\_embedded & patient.samples.sample-2.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-2.pathology\\_report\\_file\\_name & patient.samples.sample-2.pathology\\_report\\_uuid & patient.samples.sample-2.portions.portion.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-2.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-2.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-2.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-2.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-2.portions.portion.analytes.analyte.amount & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-2.portions.portion.analytes.analyte.concentration & patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-2.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-2.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-2.portions.portion.day\\_of\\_creation & patient.samples.sample-2.portions.portion.is\\_ffpe & patient.samples.sample-2.portions.portion.lce & patient.samples.sample-2.portions.portion.month\\_of\\_creation & patient.samples.sample-2.portions.portion.portion\\_number & patient.samples.sample-2.portions.portion.portion\\_sequence & patient.samples.sample-2.portions.portion.slides & patient.samples.sample-2.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-2.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-2.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-2.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-2.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-2.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-2.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-2.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-2.portions.portion.slides.slide.section\\_location & patient.samples.sample-2.portions.portion.weight & patient.samples.sample-2.portions.portion.year\\_of\\_creation & patient.samples.sample-2.preservation\\_method & patient.samples.sample-2.sample\\_type & patient.samples.sample-2.sample\\_type\\_id & patient.samples.sample-2.shortest\\_dimension & patient.samples.sample-2.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-2.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-2.tissue\\_type & patient.samples.sample-2.tumor\\_descriptor & patient.samples.sample-2.tumor\\_pathology & patient.samples.sample-2.vial\\_number & patient.samples.sample-3.additional\\_studies & patient.samples.sample-3.bcr\\_biospecimen\\_canonical\\_reasons.bcr\\_canonical\\_reason & patient.samples.sample-3.bcr\\_sample\\_barcode & patient.samples.sample-3.bcr\\_sample\\_uuid & patient.samples.sample-3.biospecimen\\_sequence & patient.samples.sample-3.composition & patient.samples.sample-3.current\\_weight & patient.samples.sample-3.days\\_to\\_collection & patient.samples.sample-3.days\\_to\\_sample\\_procurement & patient.samples.sample-3.diagnostic\\_slides.additional\\_studies & patient.samples.sample-3.freezing\\_method & patient.samples.sample-3.initial\\_weight & patient.samples.sample-3.intermediate\\_dimension & patient.samples.sample-3.is\\_ffpe & patient.samples.sample-3.longest\\_dimension & patient.samples.sample-3.method\\_of\\_sample\\_procurement & patient.samples.sample-3.ncnnct\\_othmethonsp & patient.samples.sample-3.oct\\_embedded & patient.samples.sample-3.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-3.pathology\\_report\\_file\\_name & patient.samples.sample-3.pathology\\_report\\_uuid & patient.samples.sample-3.portions.portion-2.additional\\_studies & patient.samples.sample-3.portions.portion-2.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion-2.analytes.analyte.additional\\_studies & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion-2.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion-2.analytes.analyte.amount & patient.samples.sample-3.portions.portion-2.analytes.analyte.analyte\\_type & patient.samples.sample-3.portions.portion-2.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-3.portions.portion-2.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion-2.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion-2.analytes.analyte.concentration & patient.samples.sample-3.portions.portion-2.analytes.analyte.gel\\_image\\_file & patient.samples.sample-3.portions.portion-2.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion-2.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion-2.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion-2.analytes.analyte.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion-2.analytes.analyte.rna.rinvalue & patient.samples.sample-3.portions.portion-2.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-3.portions.portion-2.analytes.analyte.subportion\\_sequence & patient.samples.sample-3.portions.portion-2.analytes.analyte.well\\_number & patient.samples.sample-3.portions.portion-2.bcr\\_portion\\_barcode & patient.samples.sample-3.portions.portion-2.bcr\\_portion\\_uuid & patient.samples.sample-3.portions.portion-2.day\\_of\\_creation & patient.samples.sample-3.portions.portion-2.is\\_ffpe & patient.samples.sample-3.portions.portion-2.lce & patient.samples.sample-3.portions.portion-2.month\\_of\\_creation & patient.samples.sample-3.portions.portion-2.portion\\_number & patient.samples.sample-3.portions.portion-2.portion\\_sequence & patient.samples.sample-3.portions.portion-2.slides.slide.additional\\_studies & patient.samples.sample-3.portions.portion-2.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion-2.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion-2.slides.slide.image\\_file\\_name & patient.samples.sample-3.portions.portion-2.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion-2.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_necrosis & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion-2.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion-2.slides.slide.section\\_location & patient.samples.sample-3.portions.portion-2.weight & patient.samples.sample-3.portions.portion-2.year\\_of\\_creation & patient.samples.sample-3.portions.portion.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample-3.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample-3.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample-3.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-3.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-3.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-3.portions.portion.analytes.analyte.amount & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-3.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-3.portions.portion.analytes.analyte.concentration & patient.samples.sample-3.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-3.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-3.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-3.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-3.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-3.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-3.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-3.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-3.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-3.portions.portion.day\\_of\\_creation & patient.samples.sample-3.portions.portion.is\\_ffpe & patient.samples.sample-3.portions.portion.lce & patient.samples.sample-3.portions.portion.month\\_of\\_creation & patient.samples.sample-3.portions.portion.portion\\_number & patient.samples.sample-3.portions.portion.portion\\_sequence & patient.samples.sample-3.portions.portion.slides & patient.samples.sample-3.portions.portion.slides.slide.additional\\_studies & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample-3.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample-3.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample-3.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample-3.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample-3.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample-3.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample-3.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample-3.portions.portion.slides.slide.section\\_location & patient.samples.sample-3.portions.portion.weight & patient.samples.sample-3.portions.portion.year\\_of\\_creation & patient.samples.sample-3.preservation\\_method & patient.samples.sample-3.sample\\_type & patient.samples.sample-3.sample\\_type\\_id & patient.samples.sample-3.shortest\\_dimension & patient.samples.sample-3.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-3.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-3.tissue\\_type & patient.samples.sample-3.tumor\\_descriptor & patient.samples.sample-3.tumor\\_pathology & patient.samples.sample-3.vial\\_number & patient.samples.sample-4.additional\\_studies & patient.samples.sample-4.bcr\\_sample\\_barcode & patient.samples.sample-4.bcr\\_sample\\_uuid & patient.samples.sample-4.biospecimen\\_sequence & patient.samples.sample-4.composition & patient.samples.sample-4.current\\_weight & patient.samples.sample-4.days\\_to\\_collection & patient.samples.sample-4.days\\_to\\_sample\\_procurement & patient.samples.sample-4.diagnostic\\_slides.additional\\_studies & patient.samples.sample-4.freezing\\_method & patient.samples.sample-4.initial\\_weight & patient.samples.sample-4.intermediate\\_dimension & patient.samples.sample-4.is\\_ffpe & patient.samples.sample-4.longest\\_dimension & patient.samples.sample-4.method\\_of\\_sample\\_procurement & patient.samples.sample-4.ncnnct\\_othmethonsp & patient.samples.sample-4.oct\\_embedded & patient.samples.sample-4.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample-4.pathology\\_report\\_file\\_name & patient.samples.sample-4.pathology\\_report\\_uuid & patient.samples.sample-4.portions.portion.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample-4.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample-4.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample-4.portions.portion.analytes.analyte.amount & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample-4.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample-4.portions.portion.analytes.analyte.concentration & patient.samples.sample-4.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample-4.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample-4.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample-4.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample-4.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample-4.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample-4.portions.portion.analytes.analyte.well\\_number & patient.samples.sample-4.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample-4.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample-4.portions.portion.day\\_of\\_creation & patient.samples.sample-4.portions.portion.is\\_ffpe & patient.samples.sample-4.portions.portion.lce & patient.samples.sample-4.portions.portion.month\\_of\\_creation & patient.samples.sample-4.portions.portion.portion\\_number & patient.samples.sample-4.portions.portion.portion\\_sequence & patient.samples.sample-4.portions.portion.slides & patient.samples.sample-4.portions.portion.weight & patient.samples.sample-4.portions.portion.year\\_of\\_creation & patient.samples.sample-4.preservation\\_method & patient.samples.sample-4.sample\\_type & patient.samples.sample-4.sample\\_type\\_id & patient.samples.sample-4.shortest\\_dimension & patient.samples.sample-4.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample-4.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample-4.tissue\\_type & patient.samples.sample-4.tumor\\_descriptor & patient.samples.sample-4.tumor\\_pathology & patient.samples.sample-4.vial\\_number & patient.samples.sample.additional\\_studies & patient.samples.sample.bcr\\_sample\\_barcode & patient.samples.sample.bcr\\_sample\\_uuid & patient.samples.sample.biospecimen\\_sequence & patient.samples.sample.composition & patient.samples.sample.current\\_weight & patient.samples.sample.days\\_to\\_collection & patient.samples.sample.days\\_to\\_sample\\_procurement & patient.samples.sample.diagnostic\\_slides.additional\\_studies & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-10 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-2 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-3 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-4 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-5 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-6 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-7 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-8 & patient.samples.sample.diagnostic\\_slides.ffpe\\_slide\\_uuid-9 & patient.samples.sample.freezing\\_method & patient.samples.sample.initial\\_weight & patient.samples.sample.intermediate\\_dimension & patient.samples.sample.is\\_ffpe & patient.samples.sample.longest\\_dimension & patient.samples.sample.method\\_of\\_sample\\_procurement & patient.samples.sample.ncnnct\\_othmethonsp & patient.samples.sample.oct\\_embedded & patient.samples.sample.other\\_method\\_of\\_sample\\_procurement & patient.samples.sample.pathology\\_report\\_file\\_name & patient.samples.sample.pathology\\_report\\_uuid & patient.samples.sample.portions.portion.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-2.amount & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-2.concentration & patient.samples.sample.portions.portion.analytes.analyte-2.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-2.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s & patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue & patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-2.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-2.well\\_number & patient.samples.sample.portions.portion.analytes.analyte-3.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte-3.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte-3.amount & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte-3.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte-3.concentration & patient.samples.sample.portions.portion.analytes.analyte-3.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte-3.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte-3.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte-3.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte-3.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte-3.well\\_number & patient.samples.sample.portions.portion.analytes.analyte.a260\\_a280\\_ratio & patient.samples.sample.portions.portion.analytes.analyte.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-4.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-5.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-6.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.additional\\_studies & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.amount & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume & patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment & patient.samples.sample.portions.portion.analytes.analyte.amount & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type & patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode & patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid & patient.samples.sample.portions.portion.analytes.analyte.concentration & patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match & patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful & patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file & patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_file\\_name & patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.protocol\\_name & patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method & patient.samples.sample.portions.portion.analytes.analyte.subportion\\_sequence & patient.samples.sample.portions.portion.analytes.analyte.well\\_number & patient.samples.sample.portions.portion.bcr\\_portion\\_barcode & patient.samples.sample.portions.portion.bcr\\_portion\\_uuid & patient.samples.sample.portions.portion.day\\_of\\_creation & patient.samples.sample.portions.portion.is\\_ffpe & patient.samples.sample.portions.portion.lce & patient.samples.sample.portions.portion.month\\_of\\_creation & patient.samples.sample.portions.portion.portion\\_number & patient.samples.sample.portions.portion.portion\\_sequence & patient.samples.sample.portions.portion.slides.slide-2.additional\\_studies & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide-2.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide-2.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide-2.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide-2.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide-2.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide-2.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide-2.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide-2.section\\_location & patient.samples.sample.portions.portion.slides.slide.additional\\_studies & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode & patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid & patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name & patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe & patient.samples.sample.portions.portion.slides.slide.number\\_proliferating\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_eosinophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_granulocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_inflam\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_monocyte\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis & patient.samples.sample.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration & patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells & patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei & patient.samples.sample.portions.portion.slides.slide.section\\_location & patient.samples.sample.portions.portion.weight & patient.samples.sample.portions.portion.year\\_of\\_creation & patient.samples.sample.portions.shipment\\_portion.additional\\_studies & patient.samples.sample.portions.shipment\\_portion.bcr\\_shipment\\_portion\\_uuid & patient.samples.sample.portions.shipment\\_portion.center\\_id & patient.samples.sample.portions.shipment\\_portion.is\\_ffpe & patient.samples.sample.portions.shipment\\_portion.plate\\_id & patient.samples.sample.portions.shipment\\_portion.portion\\_number & patient.samples.sample.portions.shipment\\_portion.portion\\_sequence & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_bcr\\_aliquot\\_barcode & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_day\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_month\\_of\\_shipment & patient.samples.sample.portions.shipment\\_portion.shipment\\_portion\\_year\\_of\\_shipment & patient.samples.sample.preservation\\_method & patient.samples.sample.sample\\_type & patient.samples.sample.sample\\_type\\_id & patient.samples.sample.shortest\\_dimension & patient.samples.sample.time\\_between\\_clamping\\_and\\_freezing & patient.samples.sample.time\\_between\\_excision\\_and\\_freezing & patient.samples.sample.tissue\\_type & patient.samples.sample.tumor\\_descriptor & patient.samples.sample.tumor\\_pathology & patient.samples.sample.vial\\_number\\\\\n",
       "  & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t384.44.0 & nationwide children's hospital & 1 & lihc & 47a9da83-f87b-45a0-a501-542c5a9df212 & 10 & false & tcga & 2015 & no & NA & severe & NA & tcga-2v-a95s & 2c041473-eaaa-4b7c-9b2e-b9da3543ad18 & NA & a  & NA & consented & united states & NA   & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 15 & NA     & NA   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA   & NA      & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & 4  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA         & NA & NA & NA    & NA   & male   & 173 & .6  & hepatocellular carcinoma & other               & NA          & NA          & NA & former smoker & no & c22.0 & 8170/3 & c22.0 & yes & 107 & 140 & 370 & 3.5 & 4.2 & 5.2 & 0  & 8 & 10793 & .7  & 1.2 & 2   & 3    & 2.0  & 0   & 1.5 & .2  & 6 - established cirrhosis & 6  & g3 & NA  & NA & NA & NA   & NA & NA             & NA                & NA                                & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA & NA   & NA   & NA & NA             & NA                      & NA          & NA  & NA  & NA & NA & NA  & a95s & with tumor & no  & no  & asian & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & r0 & lobectomy               & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage ii & NA & NA & NA & 7th & NA & NA & NA & mx & nx & t2 & NA & no & yes & 2v & liver & micro & NA                    & NA                          & NA                & NA      & NA & alive & 78  & 2015 & NA   & NA & NA & NA & canonical & NA & consented & united states & NA & NA  & NA & NA & NA & not hispanic or latino & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & a4fdb5c3-4314-44b5-becf-5f32758fa831 & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2  & f2ed143f-17b6-431d-88e0-5fe0bca5a59e & NA & NA & NA & NA & NA & NA & NA & NA & NA & 3  & c9c276c0-f8ae-441e-9b72-86222cb49955 & NA & NA & NA & NA & NA & NA & NA & 4  & 0ec3725f-0eb9-477a-9537-f3f763bba9c9 & NA   & NA    & NA    & NA & NA  & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & asian & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & d7873cbc-8d52-4869-b11a-4a23cb213dbd & NA & 20 & 80  & NA & 100 & 1 & cryovial                & NA & NA & NA & tcga-2v-a95s-10d & c9c276c0-f8ae-441e-9b72-86222cb49955 & NA & NA & NA & NA   & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-10d-01w-a44g-10 & 81f03dad-3c0f-40a6-9ca4-f346103e2045 & 0167788482 & 10 & 0.50 & 08 & no & 07 & 6 & a44g & c & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2v-a95s-10d-01w & c5785590-21ae-4eb1-b9dc-06536b0a60ce & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-10d-01d-a370-10 & 2143f2ba-4527-449c-975d-fbeff27126ca & 0164104044 & 10 & 0.08 & 26 & no & 02 & 6 & a370 & c & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-10d-01d-a36z-01 & db9a6405-4fc5-4922-b2f6-6f323d3a2be8 & 0164104098 & 01 & 0.16 & 26 & no & 02 & 6 & a36z & c & 1.07 & 23 & 6.70 & 2014 & 5.68  & dna & d & tcga-2v-a95s-10d-01d & 69108f0e-e4d7-479f-a71a-178361cb3aae & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type             & NA & NA & picogreen & NA & NA & tcga-2v-a95s-10d-01 & 8203f77d-e9e9-4006-8975-007149b92fbb & 22 & no & NA & 01 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 2.00 & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & d & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-01a & d7873cbc-8d52-4869-b11a-4a23cb213dbd & NA & NA & NA & NA   & NA & NA & 9dfa6fc7-87a6-4c14-9a39-e3f12b01c4d4 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 170 & NA & no & NA & NA & NA & false & NA & tcga-2v-a95s.be263dc7-2250-42e2-8ad6-d99a6df679b6.pdf & be263dc7-2250-42e2-8ad6-d99a6df679b6 & NA & 1.90 & NA & NA & NA & tcga-2v-a95s-01a-11r-a37k-07 & 53796be6-1c87-448a-9928-3c640e9d3414 & 0164104801 & 07 & 0.17 & 26 & no & 02 & 2 & a37k & h & 2.21 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-01a-11r-a37g-13 & d061cb38-713a-474e-8435-ddeed983a4d0 & 0164104897 & 13 & 0.17 & 26 & no & 02 & 2 & a37g & h & 3.40 & 23 & 20.00 & 2014 & 100.78 & rna & r & tcga-2v-a95s-01a-11r & 3dc4a74e-0e76-4c77-ab08-01c1de1dac7d & 0.17 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.33 & 7.50 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-2v-a95s-01a-11w-a44f-10 & 8a439a4e-9f0f-4d16-9762-e0e8f014a756 & 0167788674 & 10 & 0.50 & 08 & no & 07 & 6 & a44f & c & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2v-a95s-01a-11w & d6875fef-1832-4c19-89d0-3867387200cc & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 1 & NA & NA & NA & NA & NA & tcga-2v-a95s-01a-11d-a36x-10 & 163604c4-1a54-4072-9821-38e000866920 & 0164055662 & 10 & 0.07 & 26 & no & 02 & 6 & a36x & c & 2.80 & 23 & 40.00 & 2014 & NA & NA & tcga-2v-a95s-01a-11d-a36y-05 & 90af1bab-6092-469e-b5f4-8aceb11c8870 & 0164104578 & 05 & 0.14 & 26 & no & 02 & 6 & a36y & c & 1.82 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-01a-11d-a36w-01 & f25fffb2-620d-4abc-bb19-edd73be41639 & 0164055714 & 01 & 0.14 & 26 & no & 02 & 6 & a36w & c & 0.94 & 23 & 6.70 & 2014 & 28.12 & dna & d & tcga-2v-a95s-01a-11d & 0da84904-03b2-4275-806a-4fc8b0b10c2e & 0.14 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-2v-a95s-01a-11 & f95f4eab-84f9-4b99-b1d1-3eb3aa75c690 & 30 & no & NA & 12 & 11 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2v-a95s-01a-01-ts1 & 88a3bd39-b510-464c-861a-31d32ea198a8 & tcga-2v-a95s-01a-01-ts1.88a3bd39-b510-464c-861a-31d32ea198a8.svs & no & NA & NA & NA & NA & 90 & 10 & 20 & 0 & 0 & 30 & 50  & 70 & top & 30.00 & 2013 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t399.42.0 & nationwide children's hospital & 1 & lihc & ada84bc2-9724-428c-9557-7280af5f8297 & 10 & false & tcga & 2015 & no & NA & none   & 58 & tcga-2y-a9gs & 95b97198-1a1b-4f01-90f3-6c66638371d1 & NA & NA & NA & consented & united states & -5   & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 3  & -21318 & 724  & 0  & 0  & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA   & NA      & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & hispanic or latino     & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA         & NA & NA & NA    & NA   & male   & 162 & 1.6 & hepatocellular carcinoma & alcohol consumption & NA          & NA          & NA & NA            & no & c22.0 & 8170/3 & c22.0 & yes & 168 & 143 & 382 & 3.5 & 4   & 5   & 0  & 6 & 74    & 0.8 & 1.5 & 8.3 & 11.5 & 9.9  & 0.2 & 0.7 & 1.3 & NA                        & 11 & g2 & NA  & NA & NA & NA   & NA & NA             & NA                & NA                                & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & no & NA   & 102  & NA & lung           & extrahepatic recurrence & NA          & no  & no  & no & NA & yes & a9gs & with tumor & no  & no  & white & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & r0 & segmentectomy, single   & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage iv & NA & NA & NA & 6th & NA & NA & NA & mx & nx & t2 & NA & no & yes & 2y & liver & NA    & hcv genotype          & hepatitis b surface antigen & hbv core antibody & NA      & NA & dead  & 92  & 2014 & 2006 & NA & NA & NA & canonical & NA & consented & united states & NA & 179 & 0  & NA & NA & hispanic or latino     & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & c8ae9b63-ef3b-497f-aefa-d36e380b3060 & 0    & 0.07  & qubit & 20 & 285 & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & white & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 6ffc6d98-5f3c-4672-a390-98e0076d3e5a & NA & 1  & 80  & NA & 91  & 1 & biospecimen storage bag & 0  & NA & NA & tcga-2y-a9gs-10a & c8ae9b63-ef3b-497f-aefa-d36e380b3060 & NA & NA & NA & 2558 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-10a-01w-a44q-10 & 87e2f51d-a049-4618-a148-49bba494bec6 & 0167803775 & 10 & 0.50 & 08 & no & 07 & 1 & a44q & a & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gs-10a-01w & 4467681a-e92f-4239-93db-5163e64d5a58 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-10a-01d-a385-10 & 27ea2725-9c48-4965-a503-91d2302923b4 & 0164054976 & 10 & 0.08 & 26 & no & 03 & 1 & a385 & a & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-10a-01d-a384-01 & d49c2287-6dd4-485e-bf68-1a16ab5b6146 & 0164054975 & 01 & 0.15 & 26 & no & 03 & 1 & a384 & a & 1.01 & 23 & 6.70 & 2014 & 17.88 & dna & d & tcga-2y-a9gs-10a-01d & a5c27000-c79f-4b94-858d-da9c5a41e9f4 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & pre-extracted dna received by tss & NA & NA & picogreen & NA & NA & tcga-2y-a9gs-10a-01 & 4bf617e8-79b6-4131-b45f-c27e0bb444a2 & 04 & no & NA & 02 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a & 6ffc6d98-5f3c-4672-a390-98e0076d3e5a & NA & NA & NA & 2558 & NA & NA & 75c5cef1-6c8b-42ff-82f6-9a05621e9a80 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 80  & NA & no & NA & NA & NA & true  & NA & tcga-2y-a9gs.daa3d158-0488-41c9-8b3f-be6d05e036d8.pdf & daa3d158-0488-41c9-8b3f-be6d05e036d8 & NA & 1.89 & NA & NA & NA & tcga-2y-a9gs-01a-12r-a38m-13 & c1515b79-2c73-414e-9314-6ee862ab07ee & 0164104768 & 13 & 0.18 & 26 & no & 03 & 8 & a38m & c & 3.60 & 23 & 20.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a-12r-a38b-07 & 7b0d0cd1-3770-4e1c-9a64-93e41825a3b7 & 0164105824 & 07 & 0.18 & 26 & no & 03 & 8 & a38b & c & 2.34 & 23 & 13.00 & 2014 & 45.70  & rna & r & tcga-2y-a9gs-01a-12r & af96854c-218f-4d83-a73e-01ea675d75ad & 0.18 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.53 & 8.00 & uv spec & 2 & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a-12w-a44p-10 & 8380cc43-6ed7-4cd2-90a2-7cff1adeee41 & 0167788799 & 10 & 0.50 & 08 & no & 07 & 1 & a44p & a & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gs-01a-12w & 21cb95e3-d329-46c3-9916-4a82403d5f82 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 2 & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a-12d-a382-10 & 8ce44141-64e4-4d9d-9971-4dc79b316d4a & 0164054783 & 10 & 0.07 & 26 & no & 03 & 1 & a382 & a & 2.80 & 23 & 40.00 & 2014 & NA & NA & tcga-2y-a9gs-01a-12d-a383-05 & e4070aac-8ee9-4545-9436-bbfa90b3fcc6 & 0164055071 & 05 & 0.15 & 26 & no & 03 & 1 & a383 & a & 1.95 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a-12d-a381-01 & cdcb5fda-e3b4-4524-87b6-b173e2203fe3 & 0164100944 & 01 & 0.15 & 26 & no & 03 & 1 & a381 & a & 1.01 & 23 & 6.70 & 2014 & 33.29 & dna & d & tcga-2y-a9gs-01a-12d & be82b4b5-ff9c-4de5-8771-b2f2d1868ab1 & 0.15 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 2 & NA & tcga-2y-a9gs-01a-12 & 284d14c0-978c-4d42-b303-55e562ed2bf9 & 21 & no & NA & 02 & 12 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gs-01a-01-ts1 & 7ac59b6b-b3ff-4e31-b60f-a3918b3cc3f9 & tcga-2y-a9gs-01a-01-ts1.7ac59b6b-b3ff-4e31-b60f-a3918b3cc3f9.svs & no & NA & NA & NA & NA & 4  & 2  & 0  & 3 & 0 & 35 & 65  & 65 & top & 30.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t399.42.0.1 & nationwide children's hospital & 1 & lihc & b9f63d47-2293-4360-b373-5acdaa2ec7cb & 10 & false & tcga & 2015 & NA & NA & none   & 51 & tcga-2y-a9gt & 31a2ecc5-2d45-4823-8187-dc701dd412a8 & 1  & c  & NA & consented & united states & -22  & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 3  & -18768 & 1624 & 0  & 0  & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA   & NA      & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & 0  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA         & NA & NA & NA    & NA   & male   & 182 & 0.8 & hepatocellular carcinoma & alcohol consumption & hepatitis b & hepatitis c & NA & NA            & no & c22.0 & 8170/3 & c22.0 & yes & 187 & 143 & 382 & 3.5 & 3.7 & 5   & NA & 7 & 17    & 0.8 & 1.5 & 8.3 & 11.5 & 9.8  & 0.2 & 0.4 & 1.3 & NA                        & 11 & g2 & NA  & NA & NA & NA   & NA & NA             & NA                & NA                                & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & no & NA   & 1083 & NA & liver          & locoregional recurrence & NA          & yes & no  & no & NA & yes & a9gt & with tumor & no  & no  & white & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & r0 & segmentectomy, single   & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage i  & NA & NA & NA & 6th & NA & NA & NA & mx & nx & t1 & NA & no & yes & 2y & liver & none  & hepatitis c virus rna & hepatitis b surface antigen & hbv core antibody & hbv dna & NA & dead  & 122 & 2014 & 2006 & NA & NA & NA & canonical & NA & consented & united states & NA & 279 & 0  & NA & NA & not hispanic or latino & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & 1f4cc604-4b2c-459b-9466-9da83ae803d1 & 1607 & 0.217 & qubit & 20 & 92  & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & white & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & d249fffe-46a4-428c-b433-08949432ded9 & NA & 0  & 100 & NA & 83  & 1 & biospecimen storage bag & 0  & NA & NA & tcga-2y-a9gt-10a & 1f4cc604-4b2c-459b-9466-9da83ae803d1 & NA & NA & NA & 2603 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-10a-01w-a44q-10 & 10fea1b1-4d18-4f59-985c-7a7730145876 & 0167803752 & 10 & 0.50 & 08 & no & 07 & 1 & a44q & b & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gt-10a-01w & 5e803879-6d78-4cf7-a00b-12a1cabc7e7a & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-10a-01d-a385-10 & 76f29a85-4ade-4630-b69b-5a23a1c16628 & 0164055010 & 10 & 0.08 & 26 & no & 03 & 1 & a385 & b & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-10a-01d-a384-01 & c5b87805-dd81-4065-aa2d-086706850d98 & 0164054952 & 01 & 0.16 & 26 & no & 03 & 1 & a384 & b & 1.07 & 23 & 6.70 & 2014 & 18.03 & dna & d & tcga-2y-a9gt-10a-01d & 9414f586-c090-42a8-b703-ffa7a6ffd92d & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & pre-extracted dna received by tss & NA & NA & picogreen & NA & NA & tcga-2y-a9gt-10a-01 & 0b3e106f-01e7-4c41-a72c-549a58709d48 & 04 & no & NA & 02 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a & d249fffe-46a4-428c-b433-08949432ded9 & NA & NA & NA & 2603 & NA & NA & 30666775-3556-4dfe-a5ec-8ccf8eeb1803 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 60  & NA & no & NA & NA & NA & true  & NA & tcga-2y-a9gt.ac6f0803-6f8e-4708-80de-dfc708b2217d.pdf & ac6f0803-6f8e-4708-80de-dfc708b2217d & NA & 1.84 & NA & NA & NA & tcga-2y-a9gt-01a-11r-a38m-13 & c483d534-2c7c-4250-a301-cecdca9b52d4 & 0164104759 & 13 & 0.16 & 26 & no & 03 & 8 & a38m & d & 3.20 & 23 & 20.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a-11r-a38b-07 & e1ab65b5-ee17-4e86-93aa-446c8f387ef1 & 0164105815 & 07 & 0.16 & 26 & no & 03 & 8 & a38b & d & 2.08 & 23 & 13.00 & 2014 & 62.61  & rna & r & tcga-2y-a9gt-01a-11r & 1418f6eb-75ce-4698-b493-ee1b0fde7084 & 0.16 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.43 & 8.70 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a-11w-a44p-10 & bc6f6200-986e-47b9-8171-adae143e4e88 & 0167788776 & 10 & 0.50 & 08 & no & 07 & 1 & a44p & b & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gt-01a-11w & 4b5fb84f-19d6-4d34-9943-469bdc12b196 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a-11d-a382-10 & 38c5baf0-5376-42d2-b545-89f5c606b05e & 0164054760 & 10 & 0.08 & 26 & no & 03 & 1 & a382 & b & 3.20 & 23 & 40.00 & 2014 & NA & NA & tcga-2y-a9gt-01a-11d-a383-05 & 1fe4022d-53f0-412c-829f-c7c1eeab084d & 0164055048 & 05 & 0.16 & 26 & no & 03 & 1 & a383 & b & 2.08 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a-11d-a381-01 & 9725db90-fba8-4ea3-b416-405e7a9e88b6 & 0164103912 & 01 & 0.16 & 26 & no & 03 & 1 & a381 & b & 1.07 & 23 & 6.70 & 2014 & 34.27 & dna & d & tcga-2y-a9gt-01a-11d & 7ffa3ed0-0db8-46b9-8a7a-aac4db31eabe & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-2y-a9gt-01a-11 & e5a525be-3b4c-492d-a3b1-1d02707611c2 & 14 & no & NA & 02 & 11 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gt-01a-01-ts1 & bfe1248b-bb25-4828-ae6e-a4973f4a116c & tcga-2y-a9gt-01a-01-ts1.bfe1248b-bb25-4828-ae6e-a4973f4a116c.svs & no & NA & NA & NA & NA & 0  & 0  & 0  & 0 & 0 & 0  & 100 & 95 & top & 30.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t399.42.0.2 & nationwide children's hospital & 1 & lihc & 2b488ba2-f46b-4d0b-8c73-d9a761ab61b1 & 10 & false & tcga & 2015 & no & NA & none   & 55 & tcga-2y-a9gu & b27720a2-e240-413c-a0f1-0b1af3621de2 & 2  & NA & NA & consented & united states & -29  & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 3  & -20187 & NA   & 0  & 0  & 1569 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA   & NA      & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & 0  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & tcga-2y-a9gu-f69082 & fb05e6d4-a090-475a-92a3-ded79d7b91fa & 29 & NA & 1939 & NA & no & 12 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no & tumor free & no & no & alive & 2014 & female & 154 & 0.9 & hepatocellular carcinoma & none                & NA          & NA          & NA & NA            & no & c22.0 & 8170/3 & c22.0 & yes & 281 & 143 & 382 & 3.5 & 4.4 & 5   & 0  & 6 & 304   & 0.8 & 1.5 & 8.3 & 11.5 & 10.8 & 0.2 & 0.3 & 1.3 & NA                        & 11 & g2 & NA  & NA & NA & NA   & NA & NA             & NA                & NA                                & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA  & NA & NA   & NA   & NA & NA             & NA                      & NA          & NA  & NA  & NA & NA & no  & a9gu & tumor free & no  & no  & white & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & r0 & segmentectomy, single   & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage i  & NA & NA & NA & 6th & NA & NA & NA & mx & nx & t1 & NA & no & yes & 2y & liver & NA    & hepatitis c antibody  & hepatitis b surface antigen & hbv core antibody & NA      & NA & alive & 78  & 2014 & 2009 & NA & NA & NA & canonical & NA & consented & united states & NA & 33  & 0  & NA & NA & not hispanic or latino & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & 8d5e0606-2b12-4d7a-81f9-4277fb52064e & 832  & 0.248 & qubit & 20 & 80  & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & white & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 6477af21-e833-4b46-8658-5da593030fed & NA & 0  & 92  & NA & 110 & 1 & biospecimen storage bag & 0  & NA & NA & tcga-2y-a9gu-10a & 8d5e0606-2b12-4d7a-81f9-4277fb52064e & NA & NA & NA & 1752 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-10a-01w-a44q-10 & 626c6d48-57d1-45ba-8181-ac90475d4680 & 0167803751 & 10 & 0.50 & 08 & no & 07 & 1 & a44q & c & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gu-10a-01w & f4c5a6f1-9675-4f8c-ba67-c5c50a43631b & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-10a-01d-a385-10 & 745978d5-14c9-407e-9db8-1a158a071ae7 & 0164055034 & 10 & 0.08 & 26 & no & 03 & 1 & a385 & c & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-10a-01d-a384-01 & b045ce7a-0b67-4059-9221-300c395863eb & 0164054951 & 01 & 0.17 & 26 & no & 03 & 1 & a384 & c & 1.14 & 23 & 6.70 & 2014 & 22.85 & dna & d & tcga-2y-a9gu-10a-01d & 57f27a98-1593-44e1-9d3c-2bbf2a883376 & 0.17 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & pre-extracted dna received by tss & NA & NA & picogreen & NA & NA & tcga-2y-a9gu-10a-01 & f59b9aa2-7f6a-4620-bb21-731f62931ce3 & 04 & no & NA & 02 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a & 6477af21-e833-4b46-8658-5da593030fed & NA & NA & NA & 1752 & NA & NA & 700cbbd7-9f58-470d-a85b-c7bae5608018 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 120 & NA & no & NA & NA & NA & true  & NA & tcga-2y-a9gu.5585c485-b57a-4ca8-a3e3-f093a7e6ac76.pdf & 5585c485-b57a-4ca8-a3e3-f093a7e6ac76 & NA & 1.84 & NA & NA & NA & tcga-2y-a9gu-01a-11r-a38m-13 & 6b4610e6-a85c-4cf1-87f1-750b332c9bf4 & 0164104744 & 13 & 0.16 & 26 & no & 03 & 8 & a38m & e & 3.20 & 23 & 20.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a-11r-a38b-07 & 6f3f7f73-b031-424e-ba7c-263f802dd6cf & 0164105800 & 07 & 0.16 & 26 & no & 03 & 8 & a38b & e & 2.08 & 23 & 13.00 & 2014 & 63.38  & rna & r & tcga-2y-a9gu-01a-11r & 4e54e9f5-75ab-4990-a849-d784375ce2ef & 0.16 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.58 & 9.30 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a-11w-a44p-10 & 265e8083-ddbb-45ed-b702-7ca023f36c80 & 0167788775 & 10 & 0.50 & 08 & no & 07 & 1 & a44p & c & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gu-01a-11w & 0c3b5d40-65ae-403b-a2ee-1ebaa4a86368 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a-11d-a382-10 & 263e7d45-46a8-4d9d-a82a-0324d7f9a3ff & 0164054759 & 10 & 0.09 & 26 & no & 03 & 1 & a382 & c & 3.60 & 23 & 40.00 & 2014 & NA & NA & tcga-2y-a9gu-01a-11d-a383-05 & 28d0e9ef-81c3-4997-ab49-06e2514015d4 & 0164055047 & 05 & 0.17 & 26 & no & 03 & 1 & a383 & c & 2.21 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a-11d-a381-01 & afd9c5ab-fb6e-416b-988c-8e6876061b26 & 0164103911 & 01 & 0.17 & 26 & no & 03 & 1 & a381 & c & 1.14 & 23 & 6.70 & 2014 & 19.20 & dna & d & tcga-2y-a9gu-01a-11d & 1ae9208d-b015-4422-b806-9775c4c839e5 & 0.17 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-2y-a9gu-01a-11 & b00d0686-306b-4818-ada8-f5614a5519fc & 14 & no & NA & 02 & 11 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gu-01a-01-ts1 & 7018ccf5-3126-497f-81df-c5c0fdc86f05 & tcga-2y-a9gu-01a-01-ts1.7018ccf5-3126-497f-81df-c5c0fdc86f05.svs & no & NA & NA & NA & NA & 0  & 0  & 0  & 0 & 0 & 0  & 100 & 95 & top & 30.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t399.42.0.3 & nationwide children's hospital & 1 & lihc & 69fc70f4-fe6a-472a-b354-cf37515c524d & 10 & false & tcga & 2015 & no & NA & NA     & 54 & tcga-2y-a9gv & d1aa9c1a-d732-48c0-b669-8a14e0397344 & NA & NA & NA & consented & united states & -298 & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 3  & -20011 & 2532 & 0  & 0  & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-d66920 & 2ed70472-0453-4f85-9a75-c8a9a6bdd3a9 & NA & 3  & 2304 & 2233 & naxavar & clinical progressive disease & 11 & NA & NA & NA & NA & NA & NA & NA & no & chemotherapy & NA & NA & NA & no & 2014 & 2  & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA         & NA & NA & NA    & NA   & female & 167 & 0.6 & hepatocellular carcinoma & alcohol consumption & NA          & NA          & NA & NA            & no & c22.0 & 8170/3 & c22.0 & yes & 370 & 143 & 382 & 3.5 & 4.2 & 5   & NA & 7 & 6     & 0.8 & 1.5 & 8.3 & 11.5 & 9.5  & 0.2 & 0.3 & 1.3 & NA                        & 11 & g1 & yes & no & NA & 1745 & NA & other, specify & new primary tumor & renal met from prior lung primary & no & no & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & no & 2190 & 2189 & NA & other, specify & extrahepatic recurrence & soft tissue & no  & yes & no & rx & yes & a9gv & with tumor & yes & yes & white & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & r0 & segmentectomy, multiple & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage i  & NA & NA & NA & 6th & NA & NA & NA & mx & nx & t1 & NA & no & yes & 2y & liver & NA    & hepatitis c antibody  & hepatitis b surface antigen & hbv core antibody & NA      & NA & dead  & 85  & 2014 & 2007 & NA & NA & NA & canonical & NA & consented & united states & NA & 52  & 0  & NA & NA & not hispanic or latino & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & cea3a19e-6e24-4981-98f7-23a6abc4f76a & -298 & 0.507 & qubit & 20 & 39  & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & white & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & cb3afdac-4a19-4d95-bd96-42a1fa5be795 & NA & 0  & 100 & NA & 113 & 1 & biospecimen storage bag & 0  & NA & NA & tcga-2y-a9gv-10a & cea3a19e-6e24-4981-98f7-23a6abc4f76a & NA & NA & NA & 2431 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-10a-01w-a44q-10 & 35c49fe7-a709-4e6c-a034-2c1afbcbbc0a & 0167803728 & 10 & 0.50 & 08 & no & 07 & 1 & a44q & d & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gv-10a-01w & dbec8f03-fbb0-4f40-920f-91e539257282 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-10a-01d-a385-10 & c00cd9c3-fd3e-497f-94dd-30cfb18e4aa2 & 0164055013 & 10 & 0.08 & 26 & no & 03 & 1 & a385 & d & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-10a-01d-a384-01 & 644d73a4-3e81-4733-bf8d-95befb54e4e6 & 0164054928 & 01 & 0.16 & 26 & no & 03 & 1 & a384 & d & 1.07 & 23 & 6.70 & 2014 & 26.59 & dna & d & tcga-2y-a9gv-10a-01d & f56b5a1f-e042-4879-a664-57dbfd1ec489 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & pre-extracted dna received by tss & NA & NA & picogreen & NA & NA & tcga-2y-a9gv-10a-01 & 675a30db-3f73-4ab9-aa9d-03a7aeab6556 & 04 & no & NA & 02 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a & cb3afdac-4a19-4d95-bd96-42a1fa5be795 & NA & NA & NA & 2431 & NA & NA & 524fa6dd-7c84-425c-a2b5-17690df50a28 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 120 & NA & no & NA & NA & NA & true  & NA & tcga-2y-a9gv.4979af79-cf92-4d6c-87fd-158c87b5f059.pdf & 4979af79-cf92-4d6c-87fd-158c87b5f059 & NA & 1.87 & NA & NA & NA & tcga-2y-a9gv-01a-11r-a38m-13 & f6f5520d-fb6b-492b-8d0b-70027a36624b & 0164104735 & 13 & 0.17 & 26 & no & 03 & 8 & a38m & f & 3.40 & 23 & 20.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a-11r-a38b-07 & c0bcdbc7-9bed-49b7-93ca-f82ffc2929a1 & 0164105791 & 07 & 0.17 & 26 & no & 03 & 8 & a38b & f & 2.21 & 23 & 13.00 & 2014 & 42.83  & rna & r & tcga-2y-a9gv-01a-11r & 3dce6ddf-10a7-4e20-80fc-b0327eb2190d & 0.17 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.41 & 8.60 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a-11w-a44p-10 & 6cdc7d2d-e17f-4943-b62c-b1235d7ff8d2 & 0167788752 & 10 & 0.50 & 08 & no & 07 & 1 & a44p & d & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gv-01a-11w & 286b817a-76b7-412c-8a30-b2fc5f4b6c3a & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a-11d-a382-10 & bbb0ee98-a0a5-4308-96f6-1ba7f5b71fef & 0164054736 & 10 & 0.08 & 26 & no & 03 & 1 & a382 & d & 3.20 & 23 & 40.00 & 2014 & NA & NA & tcga-2y-a9gv-01a-11d-a383-05 & 93538fdf-fe84-4525-b010-0dde099b5031 & 0164055024 & 05 & 0.17 & 26 & no & 03 & 1 & a383 & d & 2.21 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a-11d-a381-01 & c1286d04-bff7-4586-ad95-e253ebc20ac7 & 0164103888 & 01 & 0.17 & 26 & no & 03 & 1 & a381 & d & 1.14 & 23 & 6.70 & 2014 & 22.06 & dna & d & tcga-2y-a9gv-01a-11d & 764647eb-d621-4868-8529-b220ed81f4bc & 0.17 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-2y-a9gv-01a-11 & 196b2998-c4c1-4e4d-973d-4b7137a1d9e7 & 14 & no & NA & 02 & 11 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gv-01a-01-ts1 & 358bacc7-2281-4653-89dc-3b1688a0492e & tcga-2y-a9gv-01a-01-ts1.358bacc7-2281-4653-89dc-3b1688a0492e.svs & no & NA & NA & NA & NA & 0  & 0  & 0  & 0 & 0 & 0  & 100 & 90 & top & 30.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\t399.42.0.4 & nationwide children's hospital & 1 & lihc & 14e975d3-8375-462a-aa65-62b2297a360e & 10 & false & tcga & 2015 & no & NA & none   & 64 & tcga-2y-a9gw & 3e1920b3-3bd4-4506-a0cf-567257154bcf & 1  & NA & NA & consented & united states & -1   & NA & NA & hepatocellular carcinoma & no & NA & NA & NA & NA & NA & NA & primary & 3  & -23587 & 1271 & 0  & 0  & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA   & NA      & NA                           & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA           & NA & NA & NA & NA & NA   & NA & not hispanic or latino & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                  & NA                                   & NA & NA & NA   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA         & NA & NA & NA    & NA   & male   & 182 & 1   & hepatocellular carcinoma & alcohol consumption & NA          & NA          & NA & NA            & no & c22.0 & 8170/3 & c22.0 & yes & 175 & 143 & 382 & 3.5 & 4.5 & 5   & 0  & 6 & 2     & 0.8 & 1.5 & 8.3 & 11.5 & 9.7  & 0.2 & 0.6 & 1.3 & NA                        & 11 & g2 & NA  & NA & NA & NA   & NA & NA             & NA                & NA                                & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & no  & no & NA   & 1229 & NA & liver          & locoregional recurrence & NA          & no  & no  & no & NA & yes & a9gw & with tumor & no  & no  & white & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & yes & r0 & segmentectomy, multiple & NA & NA & NA & NA & NA & NA & NA & NA & NA & stage i  & NA & NA & NA & 6th & NA & NA & NA & mx & n0 & t1 & NA & no & yes & 2y & liver & NA    & NA                    & NA                          & NA                & NA      & NA & dead  & 139 & 2014 & 2007 & NA & NA & NA & canonical & NA & consented & united states & NA & 65  & 0  & NA & NA & not hispanic or latino & NA & liver & hepatocellular carcinoma & NA & no & NA & NA & NA & surgical resection & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA                                   & NA & NA & NA & NA & NA & NA & NA & NA & 967a9eb1-2801-4a90-bbb3-12e1fa3a6f35 & -1   & 0.177 & qubit & 20 & 112 & blood draw & NA & NA & NA & 1 & NA & NA & yes & yes & yes & yes & NA & white & NA & yes & liver & NA & NA & no & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 5243430a-7662-4858-85ed-a0c95fd1ecdd & NA & 0  & 90  & NA & 106 & 1 & biospecimen storage bag & 0  & NA & NA & tcga-2y-a9gw-10a & 967a9eb1-2801-4a90-bbb3-12e1fa3a6f35 & NA & NA & NA & 2375 & NA & NA & NA & NA & NA & no & NA & NA & NA & false & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-10a-01w-a44q-10 & f9524b73-2abc-482d-93fe-103b67407ef2 & 0167803727 & 10 & 0.50 & 08 & no & 07 & 1 & a44q & e & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gw-10a-01w & eb6b1d0d-3865-4cfa-bf21-800d5c1dd414 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-10a-01d-a385-10 & c1a9c723-e7ce-4a26-a984-08403614980f & 0164055037 & 10 & 0.08 & 26 & no & 03 & 1 & a385 & e & 3.20 & 23 & 40.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-10a-01d-a384-01 & 272fbf05-8bd9-430f-8a01-d81d9afe79d4 & 0164054927 & 01 & 0.16 & 26 & no & 03 & 1 & a384 & e & 1.07 & 23 & 6.70 & 2014 & 23.14 & dna & d & tcga-2y-a9gw-10a-01d & c6516121-3811-4344-9ec9-f792b5d07858 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & pre-extracted dna received by tss & NA & NA & picogreen & NA & NA & tcga-2y-a9gw-10a-01 & 6aa2cc4b-5bd9-47c4-ab3e-280866b2cc28 & 04 & no & NA & 02 & 01 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA   & 2014 & NA & blood derived normal & 10 & NA & NA & NA & NA & NA & NA & a & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a & 5243430a-7662-4858-85ed-a0c95fd1ecdd & NA & NA & NA & 2375 & NA & NA & 71805205-933d-4d72-a4a2-586dc5490d76 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & 80  & NA & no & NA & NA & NA & true  & NA & tcga-2y-a9gw.5846fa3d-e3ad-4530-8a5f-5d90cd17f539.pdf & 5846fa3d-e3ad-4530-8a5f-5d90cd17f539 & NA & 1.88 & NA & NA & NA & tcga-2y-a9gw-01a-11r-a38m-13 & 764e0717-d590-4e52-96ca-96e5ba7673f7 & 0164104720 & 13 & 0.17 & 26 & no & 03 & 8 & a38m & g & 3.40 & 23 & 20.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a-11r-a38b-07 & 6d979d21-8d92-40aa-a19f-f4a6c55384cf & 0164105776 & 07 & 0.17 & 26 & no & 03 & 8 & a38b & g & 2.21 & 23 & 13.00 & 2014 & 61.92  & rna & r & tcga-2y-a9gw-01a-11r & 9fb4fb04-7e85-43f9-85c0-74484771fd0c & 0.17 & NA & NA & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & mirvana (allprep dna) rna & NA & NA & 1.28 & 8.10 & uv spec & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a-11w-a44p-10 & cf02f495-e999-42dc-a9f0-44dc7ab6a351 & 0167788751 & 10 & 0.50 & 08 & no & 07 & 1 & a44p & e & 40.00 & 23 & 80.00 & 2014 & NA & repli-g (qiagen) dna & w & tcga-2y-a9gw-01a-11w & e7cef3ee-c256-4660-a331-42e644a66d66 & NA & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & repli-g & NA & NA & NA & 1 & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a-11d-a382-10 & 8a09bb43-1c6d-4d41-901b-0dac97a29aee & 0164054735 & 10 & 0.08 & 26 & no & 03 & 1 & a382 & e & 3.20 & 23 & 40.00 & 2014 & NA & NA & tcga-2y-a9gw-01a-11d-a383-05 & 75b63df3-9d12-46f2-940f-b0bce3960b91 & 0164055023 & 05 & 0.16 & 26 & no & 03 & 1 & a383 & e & 2.08 & 23 & 13.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a-11d-a381-01 & cf78b47d-08f8-4812-80ca-66f7fc94fad2 & 0164103887 & 01 & 0.16 & 26 & no & 03 & 1 & a381 & e & 1.07 & 23 & 6.70 & 2014 & 19.22 & dna & d & tcga-2y-a9gw-01a-11d & 91e8bf59-0d0d-47ef-b8d4-37ce27024971 & 0.16 & yes & yes & https://sharedoc.nchri.org/tcga-bcr/default.aspx & no & adna preparation type & NA & NA & picogreen & 1 & NA & tcga-2y-a9gw-01a-11 & 42346ddb-34ec-449b-a88b-954127cf68b4 & 14 & no & NA & 02 & 11 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & tcga-2y-a9gw-01a-01-ts1 & 7340122f-591c-4f08-ae11-a6b34863f42f & tcga-2y-a9gw-01a-01-ts1.7340122f-591c-4f08-ae11-a6b34863f42f.svs & no & NA & NA & NA & NA & 0  & 0  & 0  & 0 & 0 & 0  & 100 & 95 & top & 30.00 & 2014 & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & NA & primary tumor & 01 & NA & NA & NA & NA & NA & NA & a\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 1587\n",
       "\n",
       "| <!--/--> | admin.bcr &lt;chr&gt; | admin.day_of_dcc_upload &lt;chr&gt; | admin.disease_code &lt;chr&gt; | admin.file_uuid &lt;chr&gt; | admin.month_of_dcc_upload &lt;chr&gt; | admin.patient_withdrawal.withdrawn &lt;chr&gt; | admin.project_code &lt;chr&gt; | admin.year_of_dcc_upload &lt;chr&gt; | patient.ablation_embolization_tx_adjuvant &lt;chr&gt; | patient.additional_studies &lt;chr&gt; | ⋯ ⋯ | patient.samples.sample.preservation_method &lt;chr&gt; | patient.samples.sample.sample_type &lt;chr&gt; | patient.samples.sample.sample_type_id &lt;chr&gt; | patient.samples.sample.shortest_dimension &lt;chr&gt; | patient.samples.sample.time_between_clamping_and_freezing &lt;chr&gt; | patient.samples.sample.time_between_excision_and_freezing &lt;chr&gt; | patient.samples.sample.tissue_type &lt;chr&gt; | patient.samples.sample.tumor_descriptor &lt;chr&gt; | patient.samples.sample.tumor_pathology &lt;chr&gt; | patient.samples.sample.vial_number &lt;chr&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| 384.44.0 | nationwide children's hospital | 1 | lihc | 47a9da83-f87b-45a0-a501-542c5a9df212 | 10 | false | tcga | 2015 | no | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 399.42.0 | nationwide children's hospital | 1 | lihc | ada84bc2-9724-428c-9557-7280af5f8297 | 10 | false | tcga | 2015 | no | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 399.42.0.1 | nationwide children's hospital | 1 | lihc | b9f63d47-2293-4360-b373-5acdaa2ec7cb | 10 | false | tcga | 2015 | NA | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 399.42.0.2 | nationwide children's hospital | 1 | lihc | 2b488ba2-f46b-4d0b-8c73-d9a761ab61b1 | 10 | false | tcga | 2015 | no | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 399.42.0.3 | nationwide children's hospital | 1 | lihc | 69fc70f4-fe6a-472a-b354-cf37515c524d | 10 | false | tcga | 2015 | no | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "| 399.42.0.4 | nationwide children's hospital | 1 | lihc | 14e975d3-8375-462a-aa65-62b2297a360e | 10 | false | tcga | 2015 | no | NA | ⋯ | NA | primary tumor | 01 | NA | NA | NA | NA | NA | NA | a |\n",
       "\n"
      ],
      "text/plain": [
       "           admin.bcr                      admin.day_of_dcc_upload\n",
       "384.44.0   nationwide children's hospital 1                      \n",
       "399.42.0   nationwide children's hospital 1                      \n",
       "399.42.0.1 nationwide children's hospital 1                      \n",
       "399.42.0.2 nationwide children's hospital 1                      \n",
       "399.42.0.3 nationwide children's hospital 1                      \n",
       "399.42.0.4 nationwide children's hospital 1                      \n",
       "           admin.disease_code admin.file_uuid                     \n",
       "384.44.0   lihc               47a9da83-f87b-45a0-a501-542c5a9df212\n",
       "399.42.0   lihc               ada84bc2-9724-428c-9557-7280af5f8297\n",
       "399.42.0.1 lihc               b9f63d47-2293-4360-b373-5acdaa2ec7cb\n",
       "399.42.0.2 lihc               2b488ba2-f46b-4d0b-8c73-d9a761ab61b1\n",
       "399.42.0.3 lihc               69fc70f4-fe6a-472a-b354-cf37515c524d\n",
       "399.42.0.4 lihc               14e975d3-8375-462a-aa65-62b2297a360e\n",
       "           admin.month_of_dcc_upload admin.patient_withdrawal.withdrawn\n",
       "384.44.0   10                        false                             \n",
       "399.42.0   10                        false                             \n",
       "399.42.0.1 10                        false                             \n",
       "399.42.0.2 10                        false                             \n",
       "399.42.0.3 10                        false                             \n",
       "399.42.0.4 10                        false                             \n",
       "           admin.project_code admin.year_of_dcc_upload\n",
       "384.44.0   tcga               2015                    \n",
       "399.42.0   tcga               2015                    \n",
       "399.42.0.1 tcga               2015                    \n",
       "399.42.0.2 tcga               2015                    \n",
       "399.42.0.3 tcga               2015                    \n",
       "399.42.0.4 tcga               2015                    \n",
       "           patient.ablation_embolization_tx_adjuvant patient.additional_studies\n",
       "384.44.0   no                                        NA                        \n",
       "399.42.0   no                                        NA                        \n",
       "399.42.0.1 NA                                        NA                        \n",
       "399.42.0.2 no                                        NA                        \n",
       "399.42.0.3 no                                        NA                        \n",
       "399.42.0.4 no                                        NA                        \n",
       "           ⋯ patient.samples.sample.preservation_method\n",
       "384.44.0   ⋯ NA                                        \n",
       "399.42.0   ⋯ NA                                        \n",
       "399.42.0.1 ⋯ NA                                        \n",
       "399.42.0.2 ⋯ NA                                        \n",
       "399.42.0.3 ⋯ NA                                        \n",
       "399.42.0.4 ⋯ NA                                        \n",
       "           patient.samples.sample.sample_type\n",
       "384.44.0   primary tumor                     \n",
       "399.42.0   primary tumor                     \n",
       "399.42.0.1 primary tumor                     \n",
       "399.42.0.2 primary tumor                     \n",
       "399.42.0.3 primary tumor                     \n",
       "399.42.0.4 primary tumor                     \n",
       "           patient.samples.sample.sample_type_id\n",
       "384.44.0   01                                   \n",
       "399.42.0   01                                   \n",
       "399.42.0.1 01                                   \n",
       "399.42.0.2 01                                   \n",
       "399.42.0.3 01                                   \n",
       "399.42.0.4 01                                   \n",
       "           patient.samples.sample.shortest_dimension\n",
       "384.44.0   NA                                       \n",
       "399.42.0   NA                                       \n",
       "399.42.0.1 NA                                       \n",
       "399.42.0.2 NA                                       \n",
       "399.42.0.3 NA                                       \n",
       "399.42.0.4 NA                                       \n",
       "           patient.samples.sample.time_between_clamping_and_freezing\n",
       "384.44.0   NA                                                       \n",
       "399.42.0   NA                                                       \n",
       "399.42.0.1 NA                                                       \n",
       "399.42.0.2 NA                                                       \n",
       "399.42.0.3 NA                                                       \n",
       "399.42.0.4 NA                                                       \n",
       "           patient.samples.sample.time_between_excision_and_freezing\n",
       "384.44.0   NA                                                       \n",
       "399.42.0   NA                                                       \n",
       "399.42.0.1 NA                                                       \n",
       "399.42.0.2 NA                                                       \n",
       "399.42.0.3 NA                                                       \n",
       "399.42.0.4 NA                                                       \n",
       "           patient.samples.sample.tissue_type\n",
       "384.44.0   NA                                \n",
       "399.42.0   NA                                \n",
       "399.42.0.1 NA                                \n",
       "399.42.0.2 NA                                \n",
       "399.42.0.3 NA                                \n",
       "399.42.0.4 NA                                \n",
       "           patient.samples.sample.tumor_descriptor\n",
       "384.44.0   NA                                     \n",
       "399.42.0   NA                                     \n",
       "399.42.0.1 NA                                     \n",
       "399.42.0.2 NA                                     \n",
       "399.42.0.3 NA                                     \n",
       "399.42.0.4 NA                                     \n",
       "           patient.samples.sample.tumor_pathology\n",
       "384.44.0   NA                                    \n",
       "399.42.0   NA                                    \n",
       "399.42.0.1 NA                                    \n",
       "399.42.0.2 NA                                    \n",
       "399.42.0.3 NA                                    \n",
       "399.42.0.4 NA                                    \n",
       "           patient.samples.sample.vial_number\n",
       "384.44.0   a                                 \n",
       "399.42.0   a                                 \n",
       "399.42.0.1 a                                 \n",
       "399.42.0.2 a                                 \n",
       "399.42.0.3 a                                 \n",
       "399.42.0.4 a                                 "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'lihc'\n",
    "df = clinical_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "### RNASeq\n",
    "\n",
    "the column names are gene names and entrez gene ids separated by a \"|\"\n",
    "\n",
    "I don't think I can remove the data load error - where the column names are too long."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "#### bring datasets into memory"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUAD\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LUAD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "luad_rnaseq <- RTCGA.rnaseq::LUAD.rnaseq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUSC\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LUSC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "lusc_rnaseq <- RTCGA.rnaseq::LUSC.rnaseq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### MESO\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-MESO\n",
    "\n",
    "No RNASeq available :/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LIHC\n",
    "\n",
    "https://portal.gdc.cancer.gov/projects/TCGA-LIHC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "lihc_rnaseq <- RTCGA.rnaseq::LIHC.rnaseq"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### rnaseq dataset list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "rnaseq_dataset_list <- list(\"luad\"=luad_rnaseq,\"lusc\"=lusc_rnaseq,\"lihc\"=lihc_rnaseq)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "#### View"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUAD"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>576</li>\n",
       "\t<li>20532</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 576\n",
       "\\item 20532\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 576\n",
       "2. 20532\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]   576 20532"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "ERROR while rich displaying an object: Error in sprintf(wrap, caption, header, body): 'fmt' length exceeds maximal format length 8192\n",
      "\n",
      "Traceback:\n",
      "1. FUN(X[[i]], ...)\n",
      "2. tryCatch(withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler), error = outer_handler)\n",
      "3. tryCatchList(expr, classes, parentenv, handlers)\n",
      "4. tryCatchOne(expr, names, parentenv, handlers[[1L]])\n",
      "5. doTryCatch(return(expr), name, parentenv, handler)\n",
      "6. withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler)\n",
      "7. repr::mime2repr[[mime]](obj)\n",
      "8. repr_latex.data.frame(obj)\n",
      "9. repr_matrix_generic(obj, sprintf(\"%%s\\n\\\\begin{tabular}{%s}\\n%%s%%s\\\\end{tabular}\\n\", \n",
      " .     cols), \"%s\\\\hline\\n\", \"%s\\\\\\\\\\n\", \"  &\", \" %s &\", \"%s\", \"\\t%s\\\\\\\\\\n\", \n",
      " .     \"%s &\", \" %s &\", escape_fun = latex_escape_vec, rows = rows, \n",
      " .     cols = cols, ...)\n",
      "10. sprintf(wrap, caption, header, body)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 20532</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>bcr_patient_barcode</th><th scope=col>?|100130426</th><th scope=col>?|100133144</th><th scope=col>?|100134869</th><th scope=col>?|10357</th><th scope=col>?|10431</th><th scope=col>?|136542</th><th scope=col>?|155060</th><th scope=col>?|26823</th><th scope=col>?|280660</th><th scope=col>⋯</th><th scope=col>ZXDA|7789</th><th scope=col>ZXDB|158586</th><th scope=col>ZXDC|79364</th><th scope=col>ZYG11A|440590</th><th scope=col>ZYG11B|79699</th><th scope=col>ZYX|7791</th><th scope=col>ZZEF1|23140</th><th scope=col>ZZZ3|26009</th><th scope=col>psiTPTE22|387590</th><th scope=col>tAKR|389932</th></tr>\n",
       "\t<tr><th scope=col>&lt;fct&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>⋯</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>TCGA-05-4244-01A-01R-1107-07</td><td>0</td><td>10.0113</td><td>11.2820</td><td>49.5994</td><td> 848.9397</td><td>0</td><td>345.2308</td><td>1.0472</td><td>0.0000</td><td>⋯</td><td>39.7940</td><td>292.5212</td><td> 959.2460</td><td> 0.6981</td><td>1088.0531</td><td>2837.944</td><td> 871.2802</td><td>575.2683</td><td>  6.6323</td><td>0.0000</td></tr>\n",
       "\t<tr><td>TCGA-05-4249-01A-01R-1107-07</td><td>0</td><td> 7.1957</td><td>12.4436</td><td>90.5117</td><td> 924.0158</td><td>0</td><td>145.2025</td><td>1.6098</td><td>0.0000</td><td>⋯</td><td>62.1376</td><td>493.5597</td><td>1179.3275</td><td>20.2833</td><td> 787.5061</td><td>2351.250</td><td>1138.1170</td><td>690.2752</td><td>179.9738</td><td>0.0000</td></tr>\n",
       "\t<tr><td>TCGA-05-4250-01A-01R-1107-07</td><td>0</td><td> 7.2453</td><td> 6.0184</td><td>49.5366</td><td>1140.6781</td><td>0</td><td> 51.7284</td><td>0.0000</td><td>0.0000</td><td>⋯</td><td>60.0182</td><td>365.4149</td><td> 843.9028</td><td>26.5274</td><td> 475.1720</td><td>5437.453</td><td>1170.5214</td><td>532.8691</td><td>  6.3003</td><td>0.0000</td></tr>\n",
       "\t<tr><td>TCGA-05-4382-01A-01R-1206-07</td><td>0</td><td>11.3311</td><td> 7.5740</td><td>82.8303</td><td> 807.1729</td><td>0</td><td>240.0221</td><td>0.4786</td><td>0.2393</td><td>⋯</td><td>40.4424</td><td>346.7517</td><td> 946.6872</td><td>64.8514</td><td> 908.1593</td><td>6770.154</td><td>1169.2401</td><td>663.8297</td><td> 35.1777</td><td>0.0000</td></tr>\n",
       "\t<tr><td>TCGA-05-4384-01A-01R-1755-07</td><td>0</td><td> 3.2254</td><td> 3.4942</td><td>72.5351</td><td> 562.0037</td><td>0</td><td>274.2822</td><td>0.6109</td><td>0.0000</td><td>⋯</td><td>80.0244</td><td>563.2254</td><td>1320.7086</td><td> 7.9414</td><td> 778.8638</td><td>3341.478</td><td>1737.3244</td><td>723.2743</td><td>378.1307</td><td>0.0000</td></tr>\n",
       "\t<tr><td>TCGA-05-4389-01A-01R-1206-07</td><td>0</td><td> 4.0000</td><td>13.7852</td><td>66.3658</td><td>1342.6174</td><td>0</td><td> 45.3020</td><td>0.3356</td><td>0.0000</td><td>⋯</td><td>79.5302</td><td>625.5034</td><td>1011.4094</td><td>71.8121</td><td> 534.8993</td><td>2565.436</td><td> 679.1946</td><td>753.0201</td><td> 11.0738</td><td>0.6711</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 20532\n",
       "\n",
       "| bcr_patient_barcode &lt;fct&gt; | ?|100130426 &lt;dbl&gt; | ?|100133144 &lt;dbl&gt; | ?|100134869 &lt;dbl&gt; | ?|10357 &lt;dbl&gt; | ?|10431 &lt;dbl&gt; | ?|136542 &lt;dbl&gt; | ?|155060 &lt;dbl&gt; | ?|26823 &lt;dbl&gt; | ?|280660 &lt;dbl&gt; | ⋯ ⋯ | ZXDA|7789 &lt;dbl&gt; | ZXDB|158586 &lt;dbl&gt; | ZXDC|79364 &lt;dbl&gt; | ZYG11A|440590 &lt;dbl&gt; | ZYG11B|79699 &lt;dbl&gt; | ZYX|7791 &lt;dbl&gt; | ZZEF1|23140 &lt;dbl&gt; | ZZZ3|26009 &lt;dbl&gt; | psiTPTE22|387590 &lt;dbl&gt; | tAKR|389932 &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| TCGA-05-4244-01A-01R-1107-07 | 0 | 10.0113 | 11.2820 | 49.5994 |  848.9397 | 0 | 345.2308 | 1.0472 | 0.0000 | ⋯ | 39.7940 | 292.5212 |  959.2460 |  0.6981 | 1088.0531 | 2837.944 |  871.2802 | 575.2683 |   6.6323 | 0.0000 |\n",
       "| TCGA-05-4249-01A-01R-1107-07 | 0 |  7.1957 | 12.4436 | 90.5117 |  924.0158 | 0 | 145.2025 | 1.6098 | 0.0000 | ⋯ | 62.1376 | 493.5597 | 1179.3275 | 20.2833 |  787.5061 | 2351.250 | 1138.1170 | 690.2752 | 179.9738 | 0.0000 |\n",
       "| TCGA-05-4250-01A-01R-1107-07 | 0 |  7.2453 |  6.0184 | 49.5366 | 1140.6781 | 0 |  51.7284 | 0.0000 | 0.0000 | ⋯ | 60.0182 | 365.4149 |  843.9028 | 26.5274 |  475.1720 | 5437.453 | 1170.5214 | 532.8691 |   6.3003 | 0.0000 |\n",
       "| TCGA-05-4382-01A-01R-1206-07 | 0 | 11.3311 |  7.5740 | 82.8303 |  807.1729 | 0 | 240.0221 | 0.4786 | 0.2393 | ⋯ | 40.4424 | 346.7517 |  946.6872 | 64.8514 |  908.1593 | 6770.154 | 1169.2401 | 663.8297 |  35.1777 | 0.0000 |\n",
       "| TCGA-05-4384-01A-01R-1755-07 | 0 |  3.2254 |  3.4942 | 72.5351 |  562.0037 | 0 | 274.2822 | 0.6109 | 0.0000 | ⋯ | 80.0244 | 563.2254 | 1320.7086 |  7.9414 |  778.8638 | 3341.478 | 1737.3244 | 723.2743 | 378.1307 | 0.0000 |\n",
       "| TCGA-05-4389-01A-01R-1206-07 | 0 |  4.0000 | 13.7852 | 66.3658 | 1342.6174 | 0 |  45.3020 | 0.3356 | 0.0000 | ⋯ | 79.5302 | 625.5034 | 1011.4094 | 71.8121 |  534.8993 | 2565.436 |  679.1946 | 753.0201 |  11.0738 | 0.6711 |\n",
       "\n"
      ],
      "text/plain": [
       "  bcr_patient_barcode          ?|100130426 ?|100133144 ?|100134869 ?|10357\n",
       "1 TCGA-05-4244-01A-01R-1107-07 0           10.0113     11.2820     49.5994\n",
       "2 TCGA-05-4249-01A-01R-1107-07 0            7.1957     12.4436     90.5117\n",
       "3 TCGA-05-4250-01A-01R-1107-07 0            7.2453      6.0184     49.5366\n",
       "4 TCGA-05-4382-01A-01R-1206-07 0           11.3311      7.5740     82.8303\n",
       "5 TCGA-05-4384-01A-01R-1755-07 0            3.2254      3.4942     72.5351\n",
       "6 TCGA-05-4389-01A-01R-1206-07 0            4.0000     13.7852     66.3658\n",
       "  ?|10431   ?|136542 ?|155060 ?|26823 ?|280660 ⋯ ZXDA|7789 ZXDB|158586\n",
       "1  848.9397 0        345.2308 1.0472  0.0000   ⋯ 39.7940   292.5212   \n",
       "2  924.0158 0        145.2025 1.6098  0.0000   ⋯ 62.1376   493.5597   \n",
       "3 1140.6781 0         51.7284 0.0000  0.0000   ⋯ 60.0182   365.4149   \n",
       "4  807.1729 0        240.0221 0.4786  0.2393   ⋯ 40.4424   346.7517   \n",
       "5  562.0037 0        274.2822 0.6109  0.0000   ⋯ 80.0244   563.2254   \n",
       "6 1342.6174 0         45.3020 0.3356  0.0000   ⋯ 79.5302   625.5034   \n",
       "  ZXDC|79364 ZYG11A|440590 ZYG11B|79699 ZYX|7791 ZZEF1|23140 ZZZ3|26009\n",
       "1  959.2460   0.6981       1088.0531    2837.944  871.2802   575.2683  \n",
       "2 1179.3275  20.2833        787.5061    2351.250 1138.1170   690.2752  \n",
       "3  843.9028  26.5274        475.1720    5437.453 1170.5214   532.8691  \n",
       "4  946.6872  64.8514        908.1593    6770.154 1169.2401   663.8297  \n",
       "5 1320.7086   7.9414        778.8638    3341.478 1737.3244   723.2743  \n",
       "6 1011.4094  71.8121        534.8993    2565.436  679.1946   753.0201  \n",
       "  psiTPTE22|387590 tAKR|389932\n",
       "1   6.6323         0.0000     \n",
       "2 179.9738         0.0000     \n",
       "3   6.3003         0.0000     \n",
       "4  35.1777         0.0000     \n",
       "5 378.1307         0.0000     \n",
       "6  11.0738         0.6711     "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'luad'\n",
    "df = rnaseq_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "col_nchars <- sapply(colnames(df),nchar)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "5"
      ],
      "text/latex": [
       "5"
      ],
      "text/markdown": [
       "5"
      ],
      "text/plain": [
       "[1] 5"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "25"
      ],
      "text/latex": [
       "25"
      ],
      "text/markdown": [
       "25"
      ],
      "text/plain": [
       "[1] 25"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "min(col_nchars);max(col_nchars)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LUSC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>552</li>\n",
       "\t<li>20532</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 552\n",
       "\\item 20532\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 552\n",
       "2. 20532\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]   552 20532"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "ERROR while rich displaying an object: Error in sprintf(wrap, caption, header, body): 'fmt' length exceeds maximal format length 8192\n",
      "\n",
      "Traceback:\n",
      "1. FUN(X[[i]], ...)\n",
      "2. tryCatch(withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler), error = outer_handler)\n",
      "3. tryCatchList(expr, classes, parentenv, handlers)\n",
      "4. tryCatchOne(expr, names, parentenv, handlers[[1L]])\n",
      "5. doTryCatch(return(expr), name, parentenv, handler)\n",
      "6. withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler)\n",
      "7. repr::mime2repr[[mime]](obj)\n",
      "8. repr_latex.data.frame(obj)\n",
      "9. repr_matrix_generic(obj, sprintf(\"%%s\\n\\\\begin{tabular}{%s}\\n%%s%%s\\\\end{tabular}\\n\", \n",
      " .     cols), \"%s\\\\hline\\n\", \"%s\\\\\\\\\\n\", \"  &\", \" %s &\", \"%s\", \"\\t%s\\\\\\\\\\n\", \n",
      " .     \"%s &\", \" %s &\", escape_fun = latex_escape_vec, rows = rows, \n",
      " .     cols = cols, ...)\n",
      "10. sprintf(wrap, caption, header, body)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 20532</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>bcr_patient_barcode</th><th scope=col>?|100130426</th><th scope=col>?|100133144</th><th scope=col>?|100134869</th><th scope=col>?|10357</th><th scope=col>?|10431</th><th scope=col>?|136542</th><th scope=col>?|155060</th><th scope=col>?|26823</th><th scope=col>?|280660</th><th scope=col>⋯</th><th scope=col>ZXDA|7789</th><th scope=col>ZXDB|158586</th><th scope=col>ZXDC|79364</th><th scope=col>ZYG11A|440590</th><th scope=col>ZYG11B|79699</th><th scope=col>ZYX|7791</th><th scope=col>ZZEF1|23140</th><th scope=col>ZZZ3|26009</th><th scope=col>psiTPTE22|387590</th><th scope=col>tAKR|389932</th></tr>\n",
       "\t<tr><th scope=col>&lt;fct&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>⋯</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>TCGA-18-3406-01A-01R-0980-07</td><td>0</td><td>75.3622</td><td>25.4252</td><td>279.9764</td><td>1510.2362</td><td>0</td><td> 23.6220</td><td>0.0000</td><td>0</td><td>⋯</td><td>75.5906</td><td>403.9370</td><td> 428.3465</td><td>  2.3622</td><td> 986.6142</td><td>1914.9606</td><td>265.3543</td><td> 738.5827</td><td> 7.0866</td><td> 0.0000</td></tr>\n",
       "\t<tr><td>TCGA-18-3407-01A-01R-0980-07</td><td>0</td><td>14.4148</td><td> 6.9933</td><td>198.2540</td><td> 968.6013</td><td>0</td><td>301.6175</td><td>0.0000</td><td>0</td><td>⋯</td><td>42.8164</td><td>303.9962</td><td>1270.2188</td><td>  5.7088</td><td> 854.4244</td><td>4221.6936</td><td>702.6641</td><td> 960.9895</td><td>11.4177</td><td> 0.4757</td></tr>\n",
       "\t<tr><td>TCGA-18-3408-01A-01R-0980-07</td><td>0</td><td> 8.4888</td><td> 3.3754</td><td>254.5929</td><td>1474.1213</td><td>0</td><td> 49.2362</td><td>1.7796</td><td>0</td><td>⋯</td><td>51.0159</td><td>377.2801</td><td> 944.9800</td><td>  8.3049</td><td> 643.0372</td><td> 526.1753</td><td>589.6485</td><td> 757.5263</td><td> 5.9321</td><td>18.3894</td></tr>\n",
       "\t<tr><td>TCGA-18-3409-01A-01R-0980-07</td><td>0</td><td> 2.5897</td><td> 7.1538</td><td>164.3744</td><td> 944.6154</td><td>0</td><td> 93.3333</td><td>1.0256</td><td>0</td><td>⋯</td><td>86.6667</td><td>616.9231</td><td>1090.2564</td><td>  1.5385</td><td>1168.2051</td><td>4388.2051</td><td>807.1795</td><td>1008.7179</td><td>48.2051</td><td> 0.0000</td></tr>\n",
       "\t<tr><td>TCGA-18-3410-01A-01R-0980-07</td><td>0</td><td>16.6653</td><td>19.5722</td><td>200.8033</td><td>1156.0568</td><td>0</td><td>120.9232</td><td>0.0000</td><td>0</td><td>⋯</td><td>40.5703</td><td>239.8769</td><td>1036.3153</td><td>161.8873</td><td> 711.7529</td><td>3029.7749</td><td>735.7800</td><td> 884.2752</td><td> 5.5144</td><td> 7.8777</td></tr>\n",
       "\t<tr><td>TCGA-18-3411-01A-01R-0980-07</td><td>0</td><td>12.9418</td><td>14.4142</td><td>277.6667</td><td>1119.5816</td><td>0</td><td> 33.3903</td><td>0.4023</td><td>0</td><td>⋯</td><td>35.8041</td><td>255.0538</td><td> 683.8982</td><td>  3.6206</td><td> 539.0727</td><td>2832.9478</td><td>321.4322</td><td> 966.7102</td><td> 9.2527</td><td> 2.0115</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 20532\n",
       "\n",
       "| bcr_patient_barcode &lt;fct&gt; | ?|100130426 &lt;dbl&gt; | ?|100133144 &lt;dbl&gt; | ?|100134869 &lt;dbl&gt; | ?|10357 &lt;dbl&gt; | ?|10431 &lt;dbl&gt; | ?|136542 &lt;dbl&gt; | ?|155060 &lt;dbl&gt; | ?|26823 &lt;dbl&gt; | ?|280660 &lt;dbl&gt; | ⋯ ⋯ | ZXDA|7789 &lt;dbl&gt; | ZXDB|158586 &lt;dbl&gt; | ZXDC|79364 &lt;dbl&gt; | ZYG11A|440590 &lt;dbl&gt; | ZYG11B|79699 &lt;dbl&gt; | ZYX|7791 &lt;dbl&gt; | ZZEF1|23140 &lt;dbl&gt; | ZZZ3|26009 &lt;dbl&gt; | psiTPTE22|387590 &lt;dbl&gt; | tAKR|389932 &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| TCGA-18-3406-01A-01R-0980-07 | 0 | 75.3622 | 25.4252 | 279.9764 | 1510.2362 | 0 |  23.6220 | 0.0000 | 0 | ⋯ | 75.5906 | 403.9370 |  428.3465 |   2.3622 |  986.6142 | 1914.9606 | 265.3543 |  738.5827 |  7.0866 |  0.0000 |\n",
       "| TCGA-18-3407-01A-01R-0980-07 | 0 | 14.4148 |  6.9933 | 198.2540 |  968.6013 | 0 | 301.6175 | 0.0000 | 0 | ⋯ | 42.8164 | 303.9962 | 1270.2188 |   5.7088 |  854.4244 | 4221.6936 | 702.6641 |  960.9895 | 11.4177 |  0.4757 |\n",
       "| TCGA-18-3408-01A-01R-0980-07 | 0 |  8.4888 |  3.3754 | 254.5929 | 1474.1213 | 0 |  49.2362 | 1.7796 | 0 | ⋯ | 51.0159 | 377.2801 |  944.9800 |   8.3049 |  643.0372 |  526.1753 | 589.6485 |  757.5263 |  5.9321 | 18.3894 |\n",
       "| TCGA-18-3409-01A-01R-0980-07 | 0 |  2.5897 |  7.1538 | 164.3744 |  944.6154 | 0 |  93.3333 | 1.0256 | 0 | ⋯ | 86.6667 | 616.9231 | 1090.2564 |   1.5385 | 1168.2051 | 4388.2051 | 807.1795 | 1008.7179 | 48.2051 |  0.0000 |\n",
       "| TCGA-18-3410-01A-01R-0980-07 | 0 | 16.6653 | 19.5722 | 200.8033 | 1156.0568 | 0 | 120.9232 | 0.0000 | 0 | ⋯ | 40.5703 | 239.8769 | 1036.3153 | 161.8873 |  711.7529 | 3029.7749 | 735.7800 |  884.2752 |  5.5144 |  7.8777 |\n",
       "| TCGA-18-3411-01A-01R-0980-07 | 0 | 12.9418 | 14.4142 | 277.6667 | 1119.5816 | 0 |  33.3903 | 0.4023 | 0 | ⋯ | 35.8041 | 255.0538 |  683.8982 |   3.6206 |  539.0727 | 2832.9478 | 321.4322 |  966.7102 |  9.2527 |  2.0115 |\n",
       "\n"
      ],
      "text/plain": [
       "  bcr_patient_barcode          ?|100130426 ?|100133144 ?|100134869 ?|10357 \n",
       "1 TCGA-18-3406-01A-01R-0980-07 0           75.3622     25.4252     279.9764\n",
       "2 TCGA-18-3407-01A-01R-0980-07 0           14.4148      6.9933     198.2540\n",
       "3 TCGA-18-3408-01A-01R-0980-07 0            8.4888      3.3754     254.5929\n",
       "4 TCGA-18-3409-01A-01R-0980-07 0            2.5897      7.1538     164.3744\n",
       "5 TCGA-18-3410-01A-01R-0980-07 0           16.6653     19.5722     200.8033\n",
       "6 TCGA-18-3411-01A-01R-0980-07 0           12.9418     14.4142     277.6667\n",
       "  ?|10431   ?|136542 ?|155060 ?|26823 ?|280660 ⋯ ZXDA|7789 ZXDB|158586\n",
       "1 1510.2362 0         23.6220 0.0000  0        ⋯ 75.5906   403.9370   \n",
       "2  968.6013 0        301.6175 0.0000  0        ⋯ 42.8164   303.9962   \n",
       "3 1474.1213 0         49.2362 1.7796  0        ⋯ 51.0159   377.2801   \n",
       "4  944.6154 0         93.3333 1.0256  0        ⋯ 86.6667   616.9231   \n",
       "5 1156.0568 0        120.9232 0.0000  0        ⋯ 40.5703   239.8769   \n",
       "6 1119.5816 0         33.3903 0.4023  0        ⋯ 35.8041   255.0538   \n",
       "  ZXDC|79364 ZYG11A|440590 ZYG11B|79699 ZYX|7791  ZZEF1|23140 ZZZ3|26009\n",
       "1  428.3465    2.3622       986.6142    1914.9606 265.3543     738.5827 \n",
       "2 1270.2188    5.7088       854.4244    4221.6936 702.6641     960.9895 \n",
       "3  944.9800    8.3049       643.0372     526.1753 589.6485     757.5263 \n",
       "4 1090.2564    1.5385      1168.2051    4388.2051 807.1795    1008.7179 \n",
       "5 1036.3153  161.8873       711.7529    3029.7749 735.7800     884.2752 \n",
       "6  683.8982    3.6206       539.0727    2832.9478 321.4322     966.7102 \n",
       "  psiTPTE22|387590 tAKR|389932\n",
       "1  7.0866           0.0000    \n",
       "2 11.4177           0.4757    \n",
       "3  5.9321          18.3894    \n",
       "4 48.2051           0.0000    \n",
       "5  5.5144           7.8777    \n",
       "6  9.2527           2.0115    "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'lusc'\n",
    "df = rnaseq_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "##### LIHC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>423</li>\n",
       "\t<li>20532</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 423\n",
       "\\item 20532\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 423\n",
       "2. 20532\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]   423 20532"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "ERROR while rich displaying an object: Error in sprintf(wrap, caption, header, body): 'fmt' length exceeds maximal format length 8192\n",
      "\n",
      "Traceback:\n",
      "1. FUN(X[[i]], ...)\n",
      "2. tryCatch(withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler), error = outer_handler)\n",
      "3. tryCatchList(expr, classes, parentenv, handlers)\n",
      "4. tryCatchOne(expr, names, parentenv, handlers[[1L]])\n",
      "5. doTryCatch(return(expr), name, parentenv, handler)\n",
      "6. withCallingHandlers({\n",
      " .     if (!mime %in% names(repr::mime2repr)) \n",
      " .         stop(\"No repr_* for mimetype \", mime, \" in repr::mime2repr\")\n",
      " .     rpr <- repr::mime2repr[[mime]](obj)\n",
      " .     if (is.null(rpr)) \n",
      " .         return(NULL)\n",
      " .     prepare_content(is.raw(rpr), rpr)\n",
      " . }, error = error_handler)\n",
      "7. repr::mime2repr[[mime]](obj)\n",
      "8. repr_latex.data.frame(obj)\n",
      "9. repr_matrix_generic(obj, sprintf(\"%%s\\n\\\\begin{tabular}{%s}\\n%%s%%s\\\\end{tabular}\\n\", \n",
      " .     cols), \"%s\\\\hline\\n\", \"%s\\\\\\\\\\n\", \"  &\", \" %s &\", \"%s\", \"\\t%s\\\\\\\\\\n\", \n",
      " .     \"%s &\", \" %s &\", escape_fun = latex_escape_vec, rows = rows, \n",
      " .     cols = cols, ...)\n",
      "10. sprintf(wrap, caption, header, body)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 20532</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>bcr_patient_barcode</th><th scope=col>?|100130426</th><th scope=col>?|100133144</th><th scope=col>?|100134869</th><th scope=col>?|10357</th><th scope=col>?|10431</th><th scope=col>?|136542</th><th scope=col>?|155060</th><th scope=col>?|26823</th><th scope=col>?|280660</th><th scope=col>⋯</th><th scope=col>ZXDA|7789</th><th scope=col>ZXDB|158586</th><th scope=col>ZXDC|79364</th><th scope=col>ZYG11A|440590</th><th scope=col>ZYG11B|79699</th><th scope=col>ZYX|7791</th><th scope=col>ZZEF1|23140</th><th scope=col>ZZZ3|26009</th><th scope=col>psiTPTE22|387590</th><th scope=col>tAKR|389932</th></tr>\n",
       "\t<tr><th scope=col>&lt;fct&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>⋯</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th><th scope=col>&lt;dbl&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>TCGA-2V-A95S-01A-11R-A37K-07</td><td>0</td><td> 1.5051</td><td>3.7074</td><td> 90.1124</td><td>1017.1038</td><td>0</td><td>141.3911</td><td>0.6516</td><td>0.0000</td><td>⋯</td><td>24.7597</td><td> 273.6602</td><td> 794.2662</td><td>18.2440</td><td> 499.1041</td><td>3172.504</td><td> 890.0472</td><td>510.1808</td><td> 3.9094</td><td>  6.5157</td></tr>\n",
       "\t<tr><td>TCGA-2Y-A9GS-01A-12R-A38B-07</td><td>0</td><td>26.4120</td><td>2.6663</td><td> 71.0054</td><td> 639.2311</td><td>0</td><td>122.7206</td><td>1.4786</td><td>0.0000</td><td>⋯</td><td>68.5067</td><td> 632.8241</td><td>1153.7703</td><td>71.4638</td><td>1000.4929</td><td>5301.134</td><td> 755.5446</td><td>860.5224</td><td> 6.4071</td><td>482.9966</td></tr>\n",
       "\t<tr><td>TCGA-2Y-A9GT-01A-11R-A38B-07</td><td>0</td><td> 0.0000</td><td>4.4833</td><td> 95.5122</td><td> 742.4344</td><td>0</td><td> 95.0460</td><td>1.7933</td><td>0.8967</td><td>⋯</td><td>46.6263</td><td>1219.4575</td><td>1133.3782</td><td>12.5532</td><td>1289.3970</td><td>3219.009</td><td> 860.7935</td><td>523.6494</td><td>14.3466</td><td> 83.3894</td></tr>\n",
       "\t<tr><td>TCGA-2Y-A9GU-01A-11R-A38B-07</td><td>0</td><td> 5.7222</td><td>5.1216</td><td> 61.6679</td><td>1186.9807</td><td>0</td><td>280.2709</td><td>0.8341</td><td>0.0000</td><td>⋯</td><td>18.3511</td><td> 285.2758</td><td>1150.2786</td><td> 9.1755</td><td> 941.7437</td><td>3092.990</td><td>1339.6283</td><td>343.6655</td><td> 2.5024</td><td>  2.5024</td></tr>\n",
       "\t<tr><td>TCGA-2Y-A9GV-01A-11R-A38B-07</td><td>0</td><td>11.4975</td><td>5.4230</td><td>104.4670</td><td> 878.1726</td><td>0</td><td>282.5719</td><td>0.0000</td><td>0.0000</td><td>⋯</td><td>41.4552</td><td> 999.1540</td><td>1631.9797</td><td> 4.2301</td><td>1380.7107</td><td>2902.707</td><td> 575.2961</td><td>665.8206</td><td> 2.5381</td><td>119.2893</td></tr>\n",
       "\t<tr><td>TCGA-2Y-A9GW-01A-11R-A38B-07</td><td>0</td><td> 3.9234</td><td>7.5709</td><td> 99.4866</td><td> 690.4215</td><td>0</td><td> 68.9655</td><td>0.0000</td><td>0.0000</td><td>⋯</td><td>19.9234</td><td>1646.7433</td><td> 706.5134</td><td>29.8851</td><td> 747.1264</td><td>8466.667</td><td> 992.3372</td><td>456.7050</td><td> 6.8966</td><td>102.6820</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 20532\n",
       "\n",
       "| bcr_patient_barcode &lt;fct&gt; | ?|100130426 &lt;dbl&gt; | ?|100133144 &lt;dbl&gt; | ?|100134869 &lt;dbl&gt; | ?|10357 &lt;dbl&gt; | ?|10431 &lt;dbl&gt; | ?|136542 &lt;dbl&gt; | ?|155060 &lt;dbl&gt; | ?|26823 &lt;dbl&gt; | ?|280660 &lt;dbl&gt; | ⋯ ⋯ | ZXDA|7789 &lt;dbl&gt; | ZXDB|158586 &lt;dbl&gt; | ZXDC|79364 &lt;dbl&gt; | ZYG11A|440590 &lt;dbl&gt; | ZYG11B|79699 &lt;dbl&gt; | ZYX|7791 &lt;dbl&gt; | ZZEF1|23140 &lt;dbl&gt; | ZZZ3|26009 &lt;dbl&gt; | psiTPTE22|387590 &lt;dbl&gt; | tAKR|389932 &lt;dbl&gt; |\n",
       "|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|\n",
       "| TCGA-2V-A95S-01A-11R-A37K-07 | 0 |  1.5051 | 3.7074 |  90.1124 | 1017.1038 | 0 | 141.3911 | 0.6516 | 0.0000 | ⋯ | 24.7597 |  273.6602 |  794.2662 | 18.2440 |  499.1041 | 3172.504 |  890.0472 | 510.1808 |  3.9094 |   6.5157 |\n",
       "| TCGA-2Y-A9GS-01A-12R-A38B-07 | 0 | 26.4120 | 2.6663 |  71.0054 |  639.2311 | 0 | 122.7206 | 1.4786 | 0.0000 | ⋯ | 68.5067 |  632.8241 | 1153.7703 | 71.4638 | 1000.4929 | 5301.134 |  755.5446 | 860.5224 |  6.4071 | 482.9966 |\n",
       "| TCGA-2Y-A9GT-01A-11R-A38B-07 | 0 |  0.0000 | 4.4833 |  95.5122 |  742.4344 | 0 |  95.0460 | 1.7933 | 0.8967 | ⋯ | 46.6263 | 1219.4575 | 1133.3782 | 12.5532 | 1289.3970 | 3219.009 |  860.7935 | 523.6494 | 14.3466 |  83.3894 |\n",
       "| TCGA-2Y-A9GU-01A-11R-A38B-07 | 0 |  5.7222 | 5.1216 |  61.6679 | 1186.9807 | 0 | 280.2709 | 0.8341 | 0.0000 | ⋯ | 18.3511 |  285.2758 | 1150.2786 |  9.1755 |  941.7437 | 3092.990 | 1339.6283 | 343.6655 |  2.5024 |   2.5024 |\n",
       "| TCGA-2Y-A9GV-01A-11R-A38B-07 | 0 | 11.4975 | 5.4230 | 104.4670 |  878.1726 | 0 | 282.5719 | 0.0000 | 0.0000 | ⋯ | 41.4552 |  999.1540 | 1631.9797 |  4.2301 | 1380.7107 | 2902.707 |  575.2961 | 665.8206 |  2.5381 | 119.2893 |\n",
       "| TCGA-2Y-A9GW-01A-11R-A38B-07 | 0 |  3.9234 | 7.5709 |  99.4866 |  690.4215 | 0 |  68.9655 | 0.0000 | 0.0000 | ⋯ | 19.9234 | 1646.7433 |  706.5134 | 29.8851 |  747.1264 | 8466.667 |  992.3372 | 456.7050 |  6.8966 | 102.6820 |\n",
       "\n"
      ],
      "text/plain": [
       "  bcr_patient_barcode          ?|100130426 ?|100133144 ?|100134869 ?|10357 \n",
       "1 TCGA-2V-A95S-01A-11R-A37K-07 0            1.5051     3.7074       90.1124\n",
       "2 TCGA-2Y-A9GS-01A-12R-A38B-07 0           26.4120     2.6663       71.0054\n",
       "3 TCGA-2Y-A9GT-01A-11R-A38B-07 0            0.0000     4.4833       95.5122\n",
       "4 TCGA-2Y-A9GU-01A-11R-A38B-07 0            5.7222     5.1216       61.6679\n",
       "5 TCGA-2Y-A9GV-01A-11R-A38B-07 0           11.4975     5.4230      104.4670\n",
       "6 TCGA-2Y-A9GW-01A-11R-A38B-07 0            3.9234     7.5709       99.4866\n",
       "  ?|10431   ?|136542 ?|155060 ?|26823 ?|280660 ⋯ ZXDA|7789 ZXDB|158586\n",
       "1 1017.1038 0        141.3911 0.6516  0.0000   ⋯ 24.7597    273.6602  \n",
       "2  639.2311 0        122.7206 1.4786  0.0000   ⋯ 68.5067    632.8241  \n",
       "3  742.4344 0         95.0460 1.7933  0.8967   ⋯ 46.6263   1219.4575  \n",
       "4 1186.9807 0        280.2709 0.8341  0.0000   ⋯ 18.3511    285.2758  \n",
       "5  878.1726 0        282.5719 0.0000  0.0000   ⋯ 41.4552    999.1540  \n",
       "6  690.4215 0         68.9655 0.0000  0.0000   ⋯ 19.9234   1646.7433  \n",
       "  ZXDC|79364 ZYG11A|440590 ZYG11B|79699 ZYX|7791 ZZEF1|23140 ZZZ3|26009\n",
       "1  794.2662  18.2440        499.1041    3172.504  890.0472   510.1808  \n",
       "2 1153.7703  71.4638       1000.4929    5301.134  755.5446   860.5224  \n",
       "3 1133.3782  12.5532       1289.3970    3219.009  860.7935   523.6494  \n",
       "4 1150.2786   9.1755        941.7437    3092.990 1339.6283   343.6655  \n",
       "5 1631.9797   4.2301       1380.7107    2902.707  575.2961   665.8206  \n",
       "6  706.5134  29.8851        747.1264    8466.667  992.3372   456.7050  \n",
       "  psiTPTE22|387590 tAKR|389932\n",
       "1  3.9094            6.5157   \n",
       "2  6.4071          482.9966   \n",
       "3 14.3466           83.3894   \n",
       "4  2.5024            2.5024   \n",
       "5  2.5381          119.2893   \n",
       "6  6.8966          102.6820   "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name = 'lihc'\n",
    "df = rnaseq_dataset_list[[name]]\n",
    "dim(df)\n",
    "head(df)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## loading biomaRt to map genes to pathways/functions\n",
    "\n",
    "https://bioconductor.org/packages/release/bioc/vignettes/biomaRt/inst/doc/biomaRt.html"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### install if haven't already"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Bioconductor version 3.9 (BiocManager 1.30.4), R 3.6.0 (2019-04-26)\n",
      "Installing package(s) 'biomaRt'\n",
      "also installing the dependencies ‘bit’, ‘bit64’, ‘blob’, ‘memoise’, ‘RSQLite’, ‘AnnotationDbi’\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The downloaded binary packages are in\n",
      "\t/var/folders/00/hrw448cj27l47l_0f2lrz1rr0000gp/T//Rtmp161D0k/downloaded_packages\n"
     ]
    }
   ],
   "source": [
    "BiocManager::install(\"biomaRt\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### load biomaRt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning message:\n",
      "“package ‘biomaRt’ was built under R version 3.6.1”"
     ]
    }
   ],
   "source": [
    "library(biomaRt)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "### get Mart and map to GO pathwys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 4 × 2</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>biomart</th><th scope=col>version</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>ENSEMBL_MART_ENSEMBL</td><td>Ensembl Genes 97     </td></tr>\n",
       "\t<tr><td>ENSEMBL_MART_MOUSE  </td><td>Mouse strains 97     </td></tr>\n",
       "\t<tr><td>ENSEMBL_MART_SNP    </td><td>Ensembl Variation 97 </td></tr>\n",
       "\t<tr><td>ENSEMBL_MART_FUNCGEN</td><td>Ensembl Regulation 97</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 4 × 2\n",
       "\\begin{tabular}{r|ll}\n",
       " biomart & version\\\\\n",
       " <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t ENSEMBL\\_MART\\_ENSEMBL & Ensembl Genes 97     \\\\\n",
       "\t ENSEMBL\\_MART\\_MOUSE   & Mouse strains 97     \\\\\n",
       "\t ENSEMBL\\_MART\\_SNP     & Ensembl Variation 97 \\\\\n",
       "\t ENSEMBL\\_MART\\_FUNCGEN & Ensembl Regulation 97\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 4 × 2\n",
       "\n",
       "| biomart &lt;chr&gt; | version &lt;chr&gt; |\n",
       "|---|---|\n",
       "| ENSEMBL_MART_ENSEMBL | Ensembl Genes 97      |\n",
       "| ENSEMBL_MART_MOUSE   | Mouse strains 97      |\n",
       "| ENSEMBL_MART_SNP     | Ensembl Variation 97  |\n",
       "| ENSEMBL_MART_FUNCGEN | Ensembl Regulation 97 |\n",
       "\n"
      ],
      "text/plain": [
       "  biomart              version              \n",
       "1 ENSEMBL_MART_ENSEMBL Ensembl Genes 97     \n",
       "2 ENSEMBL_MART_MOUSE   Mouse strains 97     \n",
       "3 ENSEMBL_MART_SNP     Ensembl Variation 97 \n",
       "4 ENSEMBL_MART_FUNCGEN Ensembl Regulation 97"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "listMarts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "ensembl = useMart(\"ensembl\",dataset=\"hsapiens_gene_ensembl\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 3</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>name</th><th scope=col>description</th><th scope=col>page</th></tr>\n",
       "\t<tr><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>ensembl_gene_id              </td><td>Gene stable ID              </td><td>feature_page</td></tr>\n",
       "\t<tr><td>ensembl_gene_id_version      </td><td>Gene stable ID version      </td><td>feature_page</td></tr>\n",
       "\t<tr><td>ensembl_transcript_id        </td><td>Transcript stable ID        </td><td>feature_page</td></tr>\n",
       "\t<tr><td>ensembl_transcript_id_version</td><td>Transcript stable ID version</td><td>feature_page</td></tr>\n",
       "\t<tr><td>ensembl_peptide_id           </td><td>Protein stable ID           </td><td>feature_page</td></tr>\n",
       "\t<tr><td>ensembl_peptide_id_version   </td><td>Protein stable ID version   </td><td>feature_page</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 3\n",
       "\\begin{tabular}{r|lll}\n",
       " name & description & page\\\\\n",
       " <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t ensembl\\_gene\\_id               & Gene stable ID               & feature\\_page\\\\\n",
       "\t ensembl\\_gene\\_id\\_version       & Gene stable ID version       & feature\\_page\\\\\n",
       "\t ensembl\\_transcript\\_id         & Transcript stable ID         & feature\\_page\\\\\n",
       "\t ensembl\\_transcript\\_id\\_version & Transcript stable ID version & feature\\_page\\\\\n",
       "\t ensembl\\_peptide\\_id            & Protein stable ID            & feature\\_page\\\\\n",
       "\t ensembl\\_peptide\\_id\\_version    & Protein stable ID version    & feature\\_page\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 3\n",
       "\n",
       "| name &lt;chr&gt; | description &lt;chr&gt; | page &lt;chr&gt; |\n",
       "|---|---|---|\n",
       "| ensembl_gene_id               | Gene stable ID               | feature_page |\n",
       "| ensembl_gene_id_version       | Gene stable ID version       | feature_page |\n",
       "| ensembl_transcript_id         | Transcript stable ID         | feature_page |\n",
       "| ensembl_transcript_id_version | Transcript stable ID version | feature_page |\n",
       "| ensembl_peptide_id            | Protein stable ID            | feature_page |\n",
       "| ensembl_peptide_id_version    | Protein stable ID version    | feature_page |\n",
       "\n"
      ],
      "text/plain": [
       "  name                          description                  page        \n",
       "1 ensembl_gene_id               Gene stable ID               feature_page\n",
       "2 ensembl_gene_id_version       Gene stable ID version       feature_page\n",
       "3 ensembl_transcript_id         Transcript stable ID         feature_page\n",
       "4 ensembl_transcript_id_version Transcript stable ID version feature_page\n",
       "5 ensembl_peptide_id            Protein stable ID            feature_page\n",
       "6 ensembl_peptide_id_version    Protein stable ID version    feature_page"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "head(listAttributes(ensembl))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "hidden": true,
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Exon stable ID'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Karyotype band'</li>\n",
       "\t<li>'Transcript start (bp)'</li>\n",
       "\t<li>'Transcript end (bp)'</li>\n",
       "\t<li>'Transcription start site (TSS)'</li>\n",
       "\t<li>'Transcript length (including UTRs and CDS)'</li>\n",
       "\t<li>'Transcript support level (TSL)'</li>\n",
       "\t<li>'GENCODE basic annotation'</li>\n",
       "\t<li>'APPRIS annotation'</li>\n",
       "\t<li>'RefSeq match transcript'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'Transcript name'</li>\n",
       "\t<li>'Source of transcript name'</li>\n",
       "\t<li>'Transcript count'</li>\n",
       "\t<li>'Gene % GC content'</li>\n",
       "\t<li>'Gene type'</li>\n",
       "\t<li>'Transcript type'</li>\n",
       "\t<li>'Source (gene)'</li>\n",
       "\t<li>'Source (transcript)'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Phenotype description'</li>\n",
       "\t<li>'Source name'</li>\n",
       "\t<li>'Study external reference'</li>\n",
       "\t<li>'Strain name'</li>\n",
       "\t<li>'Strain gender'</li>\n",
       "\t<li>'P value'</li>\n",
       "\t<li>'GO term accession'</li>\n",
       "\t<li>'GO term name'</li>\n",
       "\t<li>'GO term definition'</li>\n",
       "\t<li>'GO term evidence code'</li>\n",
       "\t<li>'GO domain'</li>\n",
       "\t<li>'GOSlim GOA Accession(s)'</li>\n",
       "\t<li>'GOSlim GOA Description'</li>\n",
       "\t<li>'CCDS ID'</li>\n",
       "\t<li>'ChEMBL ID'</li>\n",
       "\t<li>'Clone-based (Ensembl) gene ID'</li>\n",
       "\t<li>'Clone-based (Ensembl) transcript ID'</li>\n",
       "\t<li>'DataBase of Aberrant 3\\' Splice Sites name'</li>\n",
       "\t<li>'DataBase of Aberrant 3\\' Splice Sites ID'</li>\n",
       "\t<li>'DataBase of Aberrant 5\\' Splice Sites name'</li>\n",
       "\t<li>'DataBase of Aberrant 5\\' Splice Sites ID'</li>\n",
       "\t<li>'Ensembl Human Transcript ID'</li>\n",
       "\t<li>'Ensembl Human Translation ID'</li>\n",
       "\t<li>'EntrezGene transcript name ID'</li>\n",
       "\t<li>'European Nucleotide Archive ID'</li>\n",
       "\t<li>'Expression Atlas ID'</li>\n",
       "\t<li>'GeneDB ID'</li>\n",
       "\t<li>'HGNC ID'</li>\n",
       "\t<li>'HGNC symbol'</li>\n",
       "\t<li>'HGNC transcript name ID'</li>\n",
       "\t<li>'Human Protein Atlas accession'</li>\n",
       "\t<li>'Human Protein Atlas ID'</li>\n",
       "\t<li>'INSDC protein ID'</li>\n",
       "\t<li>'KEGG Pathway and Enzyme ID'</li>\n",
       "\t<li>'LRG display in Ensembl gene ID'</li>\n",
       "\t<li>'LRG display in Ensembl transcript ID'</li>\n",
       "\t<li>'MEROPS - the Peptidase Database ID'</li>\n",
       "\t<li>'MIM gene description'</li>\n",
       "\t<li>'MIM gene accession'</li>\n",
       "\t<li>'MIM morbid description'</li>\n",
       "\t<li>'MIM morbid accession'</li>\n",
       "\t<li>'miRBase accession'</li>\n",
       "\t<li>'miRBase ID'</li>\n",
       "\t<li>'miRBase transcript name ID'</li>\n",
       "\t<li>'NCBI gene description'</li>\n",
       "\t<li>'NCBI gene accession'</li>\n",
       "\t<li>'NCBI gene ID'</li>\n",
       "\t<li>'PDB ID'</li>\n",
       "\t<li>'Reactome ID'</li>\n",
       "\t<li>'Reactome gene ID'</li>\n",
       "\t<li>'Reactome transcript ID'</li>\n",
       "\t<li>'RefSeq mRNA ID'</li>\n",
       "\t<li>'RefSeq mRNA predicted ID'</li>\n",
       "\t<li>'RefSeq ncRNA ID'</li>\n",
       "\t<li>'RefSeq ncRNA predicted ID'</li>\n",
       "\t<li>'RefSeq peptide ID'</li>\n",
       "\t<li>'RefSeq peptide predicted ID'</li>\n",
       "\t<li>'RFAM ID'</li>\n",
       "\t<li>'RFAM transcript name ID'</li>\n",
       "\t<li>'RNAcentral ID'</li>\n",
       "\t<li>'UCSC Stable ID'</li>\n",
       "\t<li>'UniParc ID'</li>\n",
       "\t<li>'UniProtKB Gene Name symbol'</li>\n",
       "\t<li>'UniProtKB Gene Name ID'</li>\n",
       "\t<li>'UniProtKB/Swiss-Prot ID'</li>\n",
       "\t<li>'UniProtKB/TrEMBL ID'</li>\n",
       "\t<li>'WikiGene description'</li>\n",
       "\t<li>'WikiGene name'</li>\n",
       "\t<li>'WikiGene ID'</li>\n",
       "\t<li>'AFFY HC G110 probe'</li>\n",
       "\t<li>'AFFY HG Focus probe'</li>\n",
       "\t<li>'AFFY HG U133A probe'</li>\n",
       "\t<li>'AFFY HG U133A 2 probe'</li>\n",
       "\t<li>'AFFY HG U133B probe'</li>\n",
       "\t<li>'AFFY HG U133 Plus 2 probe'</li>\n",
       "\t<li>'AFFY HG U95A probe'</li>\n",
       "\t<li>'AFFY HG U95Av2 probe'</li>\n",
       "\t<li>'AFFY HG U95B probe'</li>\n",
       "\t<li>'AFFY HG U95C probe'</li>\n",
       "\t<li>'AFFY HG U95D probe'</li>\n",
       "\t<li>'AFFY HG U95E probe'</li>\n",
       "\t<li>'AFFY HTA 2 0 probe'</li>\n",
       "\t<li>'AFFY HuEx 1 0 st v2 probe'</li>\n",
       "\t<li>'AFFY HuGeneFL probe'</li>\n",
       "\t<li>'AFFY HuGene 1 0 st v1 probe'</li>\n",
       "\t<li>'AFFY HuGene 2 0 st v1 probe'</li>\n",
       "\t<li>'AFFY PrimeView probe'</li>\n",
       "\t<li>'AFFY U133 X3P probe'</li>\n",
       "\t<li>'AGILENT CGH 44b probe'</li>\n",
       "\t<li>'AGILENT GPL6848 probe'</li>\n",
       "\t<li>'AGILENT SurePrint G3 GE 8x60k probe'</li>\n",
       "\t<li>'AGILENT SurePrint G3 GE 8x60k v2 probe'</li>\n",
       "\t<li>'AGILENT WholeGenome probe'</li>\n",
       "\t<li>'AGILENT WholeGenome 4x44k v1 probe'</li>\n",
       "\t<li>'AGILENT WholeGenome 4x44k v2 probe'</li>\n",
       "\t<li>'CODELINK CODELINK probe'</li>\n",
       "\t<li>'ILLUMINA HumanHT 12 V3 probe'</li>\n",
       "\t<li>'ILLUMINA HumanHT 12 V4 probe'</li>\n",
       "\t<li>'ILLUMINA HumanRef 8 V3 probe'</li>\n",
       "\t<li>'ILLUMINA HumanWG 6 V1 probe'</li>\n",
       "\t<li>'ILLUMINA HumanWG 6 V2 probe'</li>\n",
       "\t<li>'ILLUMINA HumanWG 6 V3 probe'</li>\n",
       "\t<li>'PHALANX OneArray probe'</li>\n",
       "\t<li>'Ensembl Protein Family ID(s)'</li>\n",
       "\t<li>'Ensembl Family Description'</li>\n",
       "\t<li>'CDD ID'</li>\n",
       "\t<li>'CDD start'</li>\n",
       "\t<li>'CDD end'</li>\n",
       "\t<li>'Gene3D ID'</li>\n",
       "\t<li>'Gene3D start'</li>\n",
       "\t<li>'Gene3D end'</li>\n",
       "\t<li>'HAMAP ID'</li>\n",
       "\t<li>'HAMAP start'</li>\n",
       "\t<li>'HAMAP end'</li>\n",
       "\t<li>'PANTHER ID'</li>\n",
       "\t<li>'PANTHER start'</li>\n",
       "\t<li>'PANTHER end'</li>\n",
       "\t<li>'Pfam ID'</li>\n",
       "\t<li>'Pfam start'</li>\n",
       "\t<li>'Pfam end'</li>\n",
       "\t<li>'PIRSF ID'</li>\n",
       "\t<li>'PIRSF start'</li>\n",
       "\t<li>'PIRSF end'</li>\n",
       "\t<li>'Prints ID'</li>\n",
       "\t<li>'Prints start'</li>\n",
       "\t<li>'Prints end'</li>\n",
       "\t<li>'ProDom ID'</li>\n",
       "\t<li>'ProDom start'</li>\n",
       "\t<li>'ProDom end'</li>\n",
       "\t<li>'PROSITE patterns ID'</li>\n",
       "\t<li>'PROSITE patterns start'</li>\n",
       "\t<li>'PROSITE patterns end'</li>\n",
       "\t<li>'PROSITE profiles ID'</li>\n",
       "\t<li>'PROSITE profiles start'</li>\n",
       "\t<li>'PROSITE profiles end'</li>\n",
       "\t<li>'SFLD ID'</li>\n",
       "\t<li>'SFLD start'</li>\n",
       "\t<li>'SFLD end'</li>\n",
       "\t<li>'SMART ID'</li>\n",
       "\t<li>'SMART start'</li>\n",
       "\t<li>'SMART end'</li>\n",
       "\t<li>'Superfamily ID'</li>\n",
       "\t<li>'Superfamily start'</li>\n",
       "\t<li>'Superfamily end'</li>\n",
       "\t<li>'TIGRFAM ID'</li>\n",
       "\t<li>'TIGRFAM start'</li>\n",
       "\t<li>'TIGRFAM end'</li>\n",
       "\t<li>'Interpro ID'</li>\n",
       "\t<li>'Interpro Short Description'</li>\n",
       "\t<li>'Interpro Description'</li>\n",
       "\t<li>'Interpro start'</li>\n",
       "\t<li>'Interpro end'</li>\n",
       "\t<li>'MobiDB lite'</li>\n",
       "\t<li>'MobiDB lite start'</li>\n",
       "\t<li>'MobiDB lite end'</li>\n",
       "\t<li>'Coiled-coils (Ncoils)'</li>\n",
       "\t<li>'Coiled-coils (Ncoils) start'</li>\n",
       "\t<li>'Coiled-coils (Ncoils) end'</li>\n",
       "\t<li>'Low complexity (Seg)'</li>\n",
       "\t<li>'Low complexity (Seg) start'</li>\n",
       "\t<li>'Low complexity (Seg) end'</li>\n",
       "\t<li>'PDB-ENSP mappings'</li>\n",
       "\t<li>'PDB-ENSP mappings start'</li>\n",
       "\t<li>'PDB-ENSP mappings end'</li>\n",
       "\t<li>'Cleavage site (Signalp)'</li>\n",
       "\t<li>'Cleavage site (Signalp) start'</li>\n",
       "\t<li>'Cleavage site (Signalp) end'</li>\n",
       "\t<li>'Transmembrane helices'</li>\n",
       "\t<li>'Transmembrane helices start'</li>\n",
       "\t<li>'Transmembrane helices end'</li>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Transcript start (bp)'</li>\n",
       "\t<li>'Transcript end (bp)'</li>\n",
       "\t<li>'Transcription start site (TSS)'</li>\n",
       "\t<li>'Transcript length (including UTRs and CDS)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'5\\' UTR start'</li>\n",
       "\t<li>'5\\' UTR end'</li>\n",
       "\t<li>'3\\' UTR start'</li>\n",
       "\t<li>'3\\' UTR end'</li>\n",
       "\t<li>'CDS Length'</li>\n",
       "\t<li>'Transcript count'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Gene type'</li>\n",
       "\t<li>'Exon region start (bp)'</li>\n",
       "\t<li>'Exon region end (bp)'</li>\n",
       "\t<li>'Constitutive exon'</li>\n",
       "\t<li>'Exon rank in transcript'</li>\n",
       "\t<li>'Start phase'</li>\n",
       "\t<li>'End phase'</li>\n",
       "\t<li>'cDNA coding start'</li>\n",
       "\t<li>'cDNA coding end'</li>\n",
       "\t<li>'Genomic coding start'</li>\n",
       "\t<li>'Genomic coding end'</li>\n",
       "\t<li>'Exon stable ID'</li>\n",
       "\t<li>'CDS start'</li>\n",
       "\t<li>'CDS end'</li>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Karyotype band'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'Transcript count'</li>\n",
       "\t<li>'Gene % GC content'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Abingdon island giant tortoise gene stable ID'</li>\n",
       "\t<li>'Abingdon island giant tortoise gene name'</li>\n",
       "\t<li>'Abingdon island giant tortoise protein or transcript stable ID'</li>\n",
       "\t<li>'Abingdon island giant tortoise chromosome/scaffold name'</li>\n",
       "\t<li>'Abingdon island giant tortoise chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Abingdon island giant tortoise chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Abingdon island giant tortoise'</li>\n",
       "\t<li>'Abingdon island giant tortoise homology type'</li>\n",
       "\t<li>'%id. target Abingdon island giant tortoise gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Abingdon island giant tortoise gene'</li>\n",
       "\t<li>'Abingdon island giant tortoise Gene-order conservation score'</li>\n",
       "\t<li>'Abingdon island giant tortoise Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Abingdon island giant tortoise'</li>\n",
       "\t<li>'dS with Abingdon island giant tortoise'</li>\n",
       "\t<li>'Abingdon island giant tortoise orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise gene stable ID'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise gene name'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise protein or transcript stable ID'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise chromosome/scaffold name'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Agassiz\\'s desert tortoise'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise homology type'</li>\n",
       "\t<li>'%id. target Agassiz\\'s desert tortoise gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Agassiz\\'s desert tortoise gene'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise Gene-order conservation score'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Agassiz\\'s desert tortoise'</li>\n",
       "\t<li>'dS with Agassiz\\'s desert tortoise'</li>\n",
       "\t<li>'Agassiz\\'s desert tortoise orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Algerian mouse gene stable ID'</li>\n",
       "\t<li>'Algerian mouse gene name'</li>\n",
       "\t<li>'Algerian mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Algerian mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Algerian mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Algerian mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Algerian mouse'</li>\n",
       "\t<li>'Algerian mouse homology type'</li>\n",
       "\t<li>'%id. target Algerian mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Algerian mouse gene'</li>\n",
       "\t<li>'Algerian mouse Gene-order conservation score'</li>\n",
       "\t<li>'Algerian mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Algerian mouse'</li>\n",
       "\t<li>'dS with Algerian mouse'</li>\n",
       "\t<li>'Algerian mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Alpaca gene stable ID'</li>\n",
       "\t<li>'Alpaca gene name'</li>\n",
       "\t<li>'Alpaca protein or transcript stable ID'</li>\n",
       "\t<li>'Alpaca chromosome/scaffold name'</li>\n",
       "\t<li>'Alpaca chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Alpaca chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Alpaca'</li>\n",
       "\t<li>'Alpaca homology type'</li>\n",
       "\t<li>'%id. target Alpaca gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Alpaca gene'</li>\n",
       "\t<li>'Alpaca Gene-order conservation score'</li>\n",
       "\t<li>'Alpaca Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Alpaca'</li>\n",
       "\t<li>'dS with Alpaca'</li>\n",
       "\t<li>'Alpaca orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Alpine marmot gene stable ID'</li>\n",
       "\t<li>'Alpine marmot gene name'</li>\n",
       "\t<li>'Alpine marmot protein or transcript stable ID'</li>\n",
       "\t<li>'Alpine marmot chromosome/scaffold name'</li>\n",
       "\t<li>'Alpine marmot chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Alpine marmot chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Alpine marmot'</li>\n",
       "\t<li>'Alpine marmot homology type'</li>\n",
       "\t<li>'%id. target Alpine marmot gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Alpine marmot gene'</li>\n",
       "\t<li>'Alpine marmot Gene-order conservation score'</li>\n",
       "\t<li>'Alpine marmot Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Alpine marmot'</li>\n",
       "\t<li>'dS with Alpine marmot'</li>\n",
       "\t<li>'Alpine marmot orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Amazon molly gene stable ID'</li>\n",
       "\t<li>'Amazon molly gene name'</li>\n",
       "\t<li>'Amazon molly protein or transcript stable ID'</li>\n",
       "\t<li>'Amazon molly chromosome/scaffold name'</li>\n",
       "\t<li>'Amazon molly chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Amazon molly chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Amazon molly'</li>\n",
       "\t<li>'Amazon molly homology type'</li>\n",
       "\t<li>'%id. target Amazon molly gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Amazon molly gene'</li>\n",
       "\t<li>'Amazon molly Gene-order conservation score'</li>\n",
       "\t<li>'Amazon molly Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Amazon molly'</li>\n",
       "\t<li>'dS with Amazon molly'</li>\n",
       "\t<li>'Amazon molly orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'American beaver gene stable ID'</li>\n",
       "\t<li>'American beaver gene name'</li>\n",
       "\t<li>'American beaver protein or transcript stable ID'</li>\n",
       "\t<li>'American beaver chromosome/scaffold name'</li>\n",
       "\t<li>'American beaver chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'American beaver chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with American beaver'</li>\n",
       "\t<li>'American beaver homology type'</li>\n",
       "\t<li>'%id. target American beaver gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target American beaver gene'</li>\n",
       "\t<li>'American beaver Gene-order conservation score'</li>\n",
       "\t<li>'American beaver Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with American beaver'</li>\n",
       "\t<li>'dS with American beaver'</li>\n",
       "\t<li>'American beaver orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'American bison gene stable ID'</li>\n",
       "\t<li>'American bison gene name'</li>\n",
       "\t<li>'American bison protein or transcript stable ID'</li>\n",
       "\t<li>'American bison chromosome/scaffold name'</li>\n",
       "\t<li>'American bison chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'American bison chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with American bison'</li>\n",
       "\t<li>'American bison homology type'</li>\n",
       "\t<li>'%id. target American bison gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target American bison gene'</li>\n",
       "\t<li>'American bison Gene-order conservation score'</li>\n",
       "\t<li>'American bison Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with American bison'</li>\n",
       "\t<li>'dS with American bison'</li>\n",
       "\t<li>'American bison orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'American black bear gene stable ID'</li>\n",
       "\t<li>'American black bear gene name'</li>\n",
       "\t<li>'American black bear protein or transcript stable ID'</li>\n",
       "\t<li>'American black bear chromosome/scaffold name'</li>\n",
       "\t<li>'American black bear chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'American black bear chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with American black bear'</li>\n",
       "\t<li>'American black bear homology type'</li>\n",
       "\t<li>'%id. target American black bear gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target American black bear gene'</li>\n",
       "\t<li>'American black bear Gene-order conservation score'</li>\n",
       "\t<li>'American black bear Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with American black bear'</li>\n",
       "\t<li>'dS with American black bear'</li>\n",
       "\t<li>'American black bear orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'American mink gene stable ID'</li>\n",
       "\t<li>'American mink gene name'</li>\n",
       "\t<li>'American mink protein or transcript stable ID'</li>\n",
       "\t<li>'American mink chromosome/scaffold name'</li>\n",
       "\t<li>'American mink chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'American mink chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with American mink'</li>\n",
       "\t<li>'American mink homology type'</li>\n",
       "\t<li>'%id. target American mink gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target American mink gene'</li>\n",
       "\t<li>'American mink Gene-order conservation score'</li>\n",
       "\t<li>'American mink Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with American mink'</li>\n",
       "\t<li>'dS with American mink'</li>\n",
       "\t<li>'American mink orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Angola colobus gene stable ID'</li>\n",
       "\t<li>'Angola colobus gene name'</li>\n",
       "\t<li>'Angola colobus protein or transcript stable ID'</li>\n",
       "\t<li>'Angola colobus chromosome/scaffold name'</li>\n",
       "\t<li>'Angola colobus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Angola colobus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Angola colobus'</li>\n",
       "\t<li>'Angola colobus homology type'</li>\n",
       "\t<li>'%id. target Angola colobus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Angola colobus gene'</li>\n",
       "\t<li>'Angola colobus Gene-order conservation score'</li>\n",
       "\t<li>'Angola colobus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Angola colobus'</li>\n",
       "\t<li>'dS with Angola colobus'</li>\n",
       "\t<li>'Angola colobus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Anole lizard gene stable ID'</li>\n",
       "\t<li>'Anole lizard gene name'</li>\n",
       "\t<li>'Anole lizard protein or transcript stable ID'</li>\n",
       "\t<li>'Anole lizard chromosome/scaffold name'</li>\n",
       "\t<li>'Anole lizard chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Anole lizard chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Anole lizard'</li>\n",
       "\t<li>'Anole lizard homology type'</li>\n",
       "\t<li>'%id. target Anole lizard gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Anole lizard gene'</li>\n",
       "\t<li>'Anole lizard Gene-order conservation score'</li>\n",
       "\t<li>'Anole lizard Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Anole lizard'</li>\n",
       "\t<li>'dS with Anole lizard'</li>\n",
       "\t<li>'Anole lizard orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Arctic ground squirrel gene stable ID'</li>\n",
       "\t<li>'Arctic ground squirrel gene name'</li>\n",
       "\t<li>'Arctic ground squirrel protein or transcript stable ID'</li>\n",
       "\t<li>'Arctic ground squirrel chromosome/scaffold name'</li>\n",
       "\t<li>'Arctic ground squirrel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Arctic ground squirrel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Arctic ground squirrel'</li>\n",
       "\t<li>'Arctic ground squirrel homology type'</li>\n",
       "\t<li>'%id. target Arctic ground squirrel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Arctic ground squirrel gene'</li>\n",
       "\t<li>'Arctic ground squirrel Gene-order conservation score'</li>\n",
       "\t<li>'Arctic ground squirrel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Arctic ground squirrel'</li>\n",
       "\t<li>'dS with Arctic ground squirrel'</li>\n",
       "\t<li>'Arctic ground squirrel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Argentine black and white tegu gene stable ID'</li>\n",
       "\t<li>'Argentine black and white tegu gene name'</li>\n",
       "\t<li>'Argentine black and white tegu protein or transcript stable ID'</li>\n",
       "\t<li>'Argentine black and white tegu chromosome/scaffold name'</li>\n",
       "\t<li>'Argentine black and white tegu chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Argentine black and white tegu chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Argentine black and white tegu'</li>\n",
       "\t<li>'Argentine black and white tegu homology type'</li>\n",
       "\t<li>'%id. target Argentine black and white tegu gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Argentine black and white tegu gene'</li>\n",
       "\t<li>'Argentine black and white tegu Gene-order conservation score'</li>\n",
       "\t<li>'Argentine black and white tegu Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Argentine black and white tegu'</li>\n",
       "\t<li>'dS with Argentine black and white tegu'</li>\n",
       "\t<li>'Argentine black and white tegu orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Armadillo gene stable ID'</li>\n",
       "\t<li>'Armadillo gene name'</li>\n",
       "\t<li>'Armadillo protein or transcript stable ID'</li>\n",
       "\t<li>'Armadillo chromosome/scaffold name'</li>\n",
       "\t<li>'Armadillo chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Armadillo chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Armadillo'</li>\n",
       "\t<li>'Armadillo homology type'</li>\n",
       "\t<li>'%id. target Armadillo gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Armadillo gene'</li>\n",
       "\t<li>'Armadillo Gene-order conservation score'</li>\n",
       "\t<li>'Armadillo Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Armadillo'</li>\n",
       "\t<li>'dS with Armadillo'</li>\n",
       "\t<li>'Armadillo orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Asian bonytongue gene stable ID'</li>\n",
       "\t<li>'Asian bonytongue gene name'</li>\n",
       "\t<li>'Asian bonytongue protein or transcript stable ID'</li>\n",
       "\t<li>'Asian bonytongue chromosome/scaffold name'</li>\n",
       "\t<li>'Asian bonytongue chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Asian bonytongue chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Asian bonytongue'</li>\n",
       "\t<li>'Asian bonytongue homology type'</li>\n",
       "\t<li>'%id. target Asian bonytongue gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Asian bonytongue gene'</li>\n",
       "\t<li>'Asian bonytongue Gene-order conservation score'</li>\n",
       "\t<li>'Asian bonytongue Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Asian bonytongue'</li>\n",
       "\t<li>'dS with Asian bonytongue'</li>\n",
       "\t<li>'Asian bonytongue orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Australian saltwater crocodile gene stable ID'</li>\n",
       "\t<li>'Australian saltwater crocodile gene name'</li>\n",
       "\t<li>'Australian saltwater crocodile protein or transcript stable ID'</li>\n",
       "\t<li>'Australian saltwater crocodile chromosome/scaffold name'</li>\n",
       "\t<li>'Australian saltwater crocodile chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Australian saltwater crocodile chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Australian saltwater crocodile'</li>\n",
       "\t<li>'Australian saltwater crocodile homology type'</li>\n",
       "\t<li>'%id. target Australian saltwater crocodile gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Australian saltwater crocodile gene'</li>\n",
       "\t<li>'Australian saltwater crocodile Gene-order conservation score'</li>\n",
       "\t<li>'Australian saltwater crocodile Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Australian saltwater crocodile'</li>\n",
       "\t<li>'dS with Australian saltwater crocodile'</li>\n",
       "\t<li>'Australian saltwater crocodile orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ballan wrasse gene stable ID'</li>\n",
       "\t<li>'Ballan wrasse gene name'</li>\n",
       "\t<li>'Ballan wrasse protein or transcript stable ID'</li>\n",
       "\t<li>'Ballan wrasse chromosome/scaffold name'</li>\n",
       "\t<li>'Ballan wrasse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ballan wrasse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ballan wrasse'</li>\n",
       "\t<li>'Ballan wrasse homology type'</li>\n",
       "\t<li>'%id. target Ballan wrasse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ballan wrasse gene'</li>\n",
       "\t<li>'Ballan wrasse Gene-order conservation score'</li>\n",
       "\t<li>'Ballan wrasse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ballan wrasse'</li>\n",
       "\t<li>'dS with Ballan wrasse'</li>\n",
       "\t<li>'Ballan wrasse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Barramundi perch gene stable ID'</li>\n",
       "\t<li>'Barramundi perch gene name'</li>\n",
       "\t<li>'Barramundi perch protein or transcript stable ID'</li>\n",
       "\t<li>'Barramundi perch chromosome/scaffold name'</li>\n",
       "\t<li>'Barramundi perch chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Barramundi perch chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Barramundi perch'</li>\n",
       "\t<li>'Barramundi perch homology type'</li>\n",
       "\t<li>'%id. target Barramundi perch gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Barramundi perch gene'</li>\n",
       "\t<li>'Barramundi perch Gene-order conservation score'</li>\n",
       "\t<li>'Barramundi perch Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Barramundi perch'</li>\n",
       "\t<li>'dS with Barramundi perch'</li>\n",
       "\t<li>'Barramundi perch orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Bengalese finch gene stable ID'</li>\n",
       "\t<li>'Bengalese finch gene name'</li>\n",
       "\t<li>'Bengalese finch protein or transcript stable ID'</li>\n",
       "\t<li>'Bengalese finch chromosome/scaffold name'</li>\n",
       "\t<li>'Bengalese finch chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Bengalese finch chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Bengalese finch'</li>\n",
       "\t<li>'Bengalese finch homology type'</li>\n",
       "\t<li>'%id. target Bengalese finch gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Bengalese finch gene'</li>\n",
       "\t<li>'Bengalese finch Gene-order conservation score'</li>\n",
       "\t<li>'Bengalese finch Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Bengalese finch'</li>\n",
       "\t<li>'dS with Bengalese finch'</li>\n",
       "\t<li>'Bengalese finch orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Bicolor damselfish gene stable ID'</li>\n",
       "\t<li>'Bicolor damselfish gene name'</li>\n",
       "\t<li>'Bicolor damselfish protein or transcript stable ID'</li>\n",
       "\t<li>'Bicolor damselfish chromosome/scaffold name'</li>\n",
       "\t<li>'Bicolor damselfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Bicolor damselfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Bicolor damselfish'</li>\n",
       "\t<li>'Bicolor damselfish homology type'</li>\n",
       "\t<li>'%id. target Bicolor damselfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Bicolor damselfish gene'</li>\n",
       "\t<li>'Bicolor damselfish Gene-order conservation score'</li>\n",
       "\t<li>'Bicolor damselfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Bicolor damselfish'</li>\n",
       "\t<li>'dS with Bicolor damselfish'</li>\n",
       "\t<li>'Bicolor damselfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Black snub-nosed monkey gene stable ID'</li>\n",
       "\t<li>'Black snub-nosed monkey gene name'</li>\n",
       "\t<li>'Black snub-nosed monkey protein or transcript stable ID'</li>\n",
       "\t<li>'Black snub-nosed monkey chromosome/scaffold name'</li>\n",
       "\t<li>'Black snub-nosed monkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Black snub-nosed monkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Black snub-nosed monkey'</li>\n",
       "\t<li>'Black snub-nosed monkey homology type'</li>\n",
       "\t<li>'%id. target Black snub-nosed monkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Black snub-nosed monkey gene'</li>\n",
       "\t<li>'Black snub-nosed monkey Gene-order conservation score'</li>\n",
       "\t<li>'Black snub-nosed monkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Black snub-nosed monkey'</li>\n",
       "\t<li>'dS with Black snub-nosed monkey'</li>\n",
       "\t<li>'Black snub-nosed monkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Blue tit gene stable ID'</li>\n",
       "\t<li>'Blue tit gene name'</li>\n",
       "\t<li>'Blue tit protein or transcript stable ID'</li>\n",
       "\t<li>'Blue tit chromosome/scaffold name'</li>\n",
       "\t<li>'Blue tit chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Blue tit chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Blue tit'</li>\n",
       "\t<li>'Blue tit homology type'</li>\n",
       "\t<li>'%id. target Blue tit gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Blue tit gene'</li>\n",
       "\t<li>'Blue tit Gene-order conservation score'</li>\n",
       "\t<li>'Blue tit Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Blue tit'</li>\n",
       "\t<li>'dS with Blue tit'</li>\n",
       "\t<li>'Blue tit orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Blue-crowned manakin gene stable ID'</li>\n",
       "\t<li>'Blue-crowned manakin gene name'</li>\n",
       "\t<li>'Blue-crowned manakin protein or transcript stable ID'</li>\n",
       "\t<li>'Blue-crowned manakin chromosome/scaffold name'</li>\n",
       "\t<li>'Blue-crowned manakin chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Blue-crowned manakin chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Blue-crowned manakin'</li>\n",
       "\t<li>'Blue-crowned manakin homology type'</li>\n",
       "\t<li>'%id. target Blue-crowned manakin gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Blue-crowned manakin gene'</li>\n",
       "\t<li>'Blue-crowned manakin Gene-order conservation score'</li>\n",
       "\t<li>'Blue-crowned manakin Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Blue-crowned manakin'</li>\n",
       "\t<li>'dS with Blue-crowned manakin'</li>\n",
       "\t<li>'Blue-crowned manakin orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Bolivian squirrel monkey gene stable ID'</li>\n",
       "\t<li>'Bolivian squirrel monkey gene name'</li>\n",
       "\t<li>'Bolivian squirrel monkey protein or transcript stable ID'</li>\n",
       "\t<li>'Bolivian squirrel monkey chromosome/scaffold name'</li>\n",
       "\t<li>'Bolivian squirrel monkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Bolivian squirrel monkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Bolivian squirrel monkey'</li>\n",
       "\t<li>'Bolivian squirrel monkey homology type'</li>\n",
       "\t<li>'%id. target Bolivian squirrel monkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Bolivian squirrel monkey gene'</li>\n",
       "\t<li>'Bolivian squirrel monkey Gene-order conservation score'</li>\n",
       "\t<li>'Bolivian squirrel monkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Bolivian squirrel monkey'</li>\n",
       "\t<li>'dS with Bolivian squirrel monkey'</li>\n",
       "\t<li>'Bolivian squirrel monkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Bonobo gene stable ID'</li>\n",
       "\t<li>'Bonobo gene name'</li>\n",
       "\t<li>'Bonobo protein or transcript stable ID'</li>\n",
       "\t<li>'Bonobo chromosome/scaffold name'</li>\n",
       "\t<li>'Bonobo chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Bonobo chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Bonobo'</li>\n",
       "\t<li>'Bonobo homology type'</li>\n",
       "\t<li>'%id. target Bonobo gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Bonobo gene'</li>\n",
       "\t<li>'Bonobo Gene-order conservation score'</li>\n",
       "\t<li>'Bonobo Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Bonobo'</li>\n",
       "\t<li>'dS with Bonobo'</li>\n",
       "\t<li>'Bonobo orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Brazilian guinea pig gene stable ID'</li>\n",
       "\t<li>'Brazilian guinea pig gene name'</li>\n",
       "\t<li>'Brazilian guinea pig protein or transcript stable ID'</li>\n",
       "\t<li>'Brazilian guinea pig chromosome/scaffold name'</li>\n",
       "\t<li>'Brazilian guinea pig chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Brazilian guinea pig chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Brazilian guinea pig'</li>\n",
       "\t<li>'Brazilian guinea pig homology type'</li>\n",
       "\t<li>'%id. target Brazilian guinea pig gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Brazilian guinea pig gene'</li>\n",
       "\t<li>'Brazilian guinea pig Gene-order conservation score'</li>\n",
       "\t<li>'Brazilian guinea pig Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Brazilian guinea pig'</li>\n",
       "\t<li>'dS with Brazilian guinea pig'</li>\n",
       "\t<li>'Brazilian guinea pig orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Budgerigar gene stable ID'</li>\n",
       "\t<li>'Budgerigar gene name'</li>\n",
       "\t<li>'Budgerigar protein or transcript stable ID'</li>\n",
       "\t<li>'Budgerigar chromosome/scaffold name'</li>\n",
       "\t<li>'Budgerigar chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Budgerigar chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Budgerigar'</li>\n",
       "\t<li>'Budgerigar homology type'</li>\n",
       "\t<li>'%id. target Budgerigar gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Budgerigar gene'</li>\n",
       "\t<li>'Budgerigar Gene-order conservation score'</li>\n",
       "\t<li>'Budgerigar Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Budgerigar'</li>\n",
       "\t<li>'dS with Budgerigar'</li>\n",
       "\t<li>'Budgerigar orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder gene stable ID'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder gene name'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder protein or transcript stable ID'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder chromosome/scaffold name'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Burton\\'s mouthbrooder'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder homology type'</li>\n",
       "\t<li>'%id. target Burton\\'s mouthbrooder gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Burton\\'s mouthbrooder gene'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder Gene-order conservation score'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Burton\\'s mouthbrooder'</li>\n",
       "\t<li>'dS with Burton\\'s mouthbrooder'</li>\n",
       "\t<li>'Burton\\'s mouthbrooder orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Bushbaby gene stable ID'</li>\n",
       "\t<li>'Bushbaby gene name'</li>\n",
       "\t<li>'Bushbaby protein or transcript stable ID'</li>\n",
       "\t<li>'Bushbaby chromosome/scaffold name'</li>\n",
       "\t<li>'Bushbaby chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Bushbaby chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Bushbaby'</li>\n",
       "\t<li>'Bushbaby homology type'</li>\n",
       "\t<li>'%id. target Bushbaby gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Bushbaby gene'</li>\n",
       "\t<li>'Bushbaby Gene-order conservation score'</li>\n",
       "\t<li>'Bushbaby Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Bushbaby'</li>\n",
       "\t<li>'dS with Bushbaby'</li>\n",
       "\t<li>'Bushbaby orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'C.intestinalis gene stable ID'</li>\n",
       "\t<li>'C.intestinalis gene name'</li>\n",
       "\t<li>'C.intestinalis protein or transcript stable ID'</li>\n",
       "\t<li>'C.intestinalis chromosome/scaffold name'</li>\n",
       "\t<li>'C.intestinalis chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'C.intestinalis chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with C.intestinalis'</li>\n",
       "\t<li>'C.intestinalis homology type'</li>\n",
       "\t<li>'%id. target C.intestinalis gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target C.intestinalis gene'</li>\n",
       "\t<li>'C.intestinalis Gene-order conservation score'</li>\n",
       "\t<li>'C.intestinalis Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with C.intestinalis'</li>\n",
       "\t<li>'dS with C.intestinalis'</li>\n",
       "\t<li>'C.intestinalis orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'C.savignyi gene stable ID'</li>\n",
       "\t<li>'C.savignyi gene name'</li>\n",
       "\t<li>'C.savignyi protein or transcript stable ID'</li>\n",
       "\t<li>'C.savignyi chromosome/scaffold name'</li>\n",
       "\t<li>'C.savignyi chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'C.savignyi chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with C.savignyi'</li>\n",
       "\t<li>'C.savignyi homology type'</li>\n",
       "\t<li>'%id. target C.savignyi gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target C.savignyi gene'</li>\n",
       "\t<li>'C.savignyi Gene-order conservation score'</li>\n",
       "\t<li>'C.savignyi Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with C.savignyi'</li>\n",
       "\t<li>'dS with C.savignyi'</li>\n",
       "\t<li>'C.savignyi orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Caenorhabditis elegans gene stable ID'</li>\n",
       "\t<li>'Caenorhabditis elegans gene name'</li>\n",
       "\t<li>'Caenorhabditis elegans protein or transcript stable ID'</li>\n",
       "\t<li>'Caenorhabditis elegans chromosome/scaffold name'</li>\n",
       "\t<li>'Caenorhabditis elegans chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Caenorhabditis elegans chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Caenorhabditis elegans'</li>\n",
       "\t<li>'Caenorhabditis elegans homology type'</li>\n",
       "\t<li>'%id. target Caenorhabditis elegans gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Caenorhabditis elegans gene'</li>\n",
       "\t<li>'Caenorhabditis elegans Gene-order conservation score'</li>\n",
       "\t<li>'Caenorhabditis elegans Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Caenorhabditis elegans'</li>\n",
       "\t<li>'dS with Caenorhabditis elegans'</li>\n",
       "\t<li>'Caenorhabditis elegans orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Capuchin gene stable ID'</li>\n",
       "\t<li>'Capuchin gene name'</li>\n",
       "\t<li>'Capuchin protein or transcript stable ID'</li>\n",
       "\t<li>'Capuchin chromosome/scaffold name'</li>\n",
       "\t<li>'Capuchin chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Capuchin chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Capuchin'</li>\n",
       "\t<li>'Capuchin homology type'</li>\n",
       "\t<li>'%id. target Capuchin gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Capuchin gene'</li>\n",
       "\t<li>'Capuchin Gene-order conservation score'</li>\n",
       "\t<li>'Capuchin Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Capuchin'</li>\n",
       "\t<li>'dS with Capuchin'</li>\n",
       "\t<li>'Capuchin orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Cat gene stable ID'</li>\n",
       "\t<li>'Cat gene name'</li>\n",
       "\t<li>'Cat protein or transcript stable ID'</li>\n",
       "\t<li>'Cat chromosome/scaffold name'</li>\n",
       "\t<li>'Cat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Cat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Cat'</li>\n",
       "\t<li>'Cat homology type'</li>\n",
       "\t<li>'%id. target Cat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Cat gene'</li>\n",
       "\t<li>'Cat Gene-order conservation score'</li>\n",
       "\t<li>'Cat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Cat'</li>\n",
       "\t<li>'dS with Cat'</li>\n",
       "\t<li>'Cat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Central bearded dragon gene stable ID'</li>\n",
       "\t<li>'Central bearded dragon gene name'</li>\n",
       "\t<li>'Central bearded dragon protein or transcript stable ID'</li>\n",
       "\t<li>'Central bearded dragon chromosome/scaffold name'</li>\n",
       "\t<li>'Central bearded dragon chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Central bearded dragon chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Central bearded dragon'</li>\n",
       "\t<li>'Central bearded dragon homology type'</li>\n",
       "\t<li>'%id. target Central bearded dragon gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Central bearded dragon gene'</li>\n",
       "\t<li>'Central bearded dragon Gene-order conservation score'</li>\n",
       "\t<li>'Central bearded dragon Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Central bearded dragon'</li>\n",
       "\t<li>'dS with Central bearded dragon'</li>\n",
       "\t<li>'Central bearded dragon orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Channel catfish gene stable ID'</li>\n",
       "\t<li>'Channel catfish gene name'</li>\n",
       "\t<li>'Channel catfish protein or transcript stable ID'</li>\n",
       "\t<li>'Channel catfish chromosome/scaffold name'</li>\n",
       "\t<li>'Channel catfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Channel catfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Channel catfish'</li>\n",
       "\t<li>'Channel catfish homology type'</li>\n",
       "\t<li>'%id. target Channel catfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Channel catfish gene'</li>\n",
       "\t<li>'Channel catfish Gene-order conservation score'</li>\n",
       "\t<li>'Channel catfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Channel catfish'</li>\n",
       "\t<li>'dS with Channel catfish'</li>\n",
       "\t<li>'Channel catfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chicken gene stable ID'</li>\n",
       "\t<li>'Chicken gene name'</li>\n",
       "\t<li>'Chicken protein or transcript stable ID'</li>\n",
       "\t<li>'Chicken chromosome/scaffold name'</li>\n",
       "\t<li>'Chicken chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chicken chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chicken'</li>\n",
       "\t<li>'Chicken homology type'</li>\n",
       "\t<li>'%id. target Chicken gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chicken gene'</li>\n",
       "\t<li>'Chicken Gene-order conservation score'</li>\n",
       "\t<li>'Chicken Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chicken'</li>\n",
       "\t<li>'dS with Chicken'</li>\n",
       "\t<li>'Chicken orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chilean tinamou gene stable ID'</li>\n",
       "\t<li>'Chilean tinamou gene name'</li>\n",
       "\t<li>'Chilean tinamou protein or transcript stable ID'</li>\n",
       "\t<li>'Chilean tinamou chromosome/scaffold name'</li>\n",
       "\t<li>'Chilean tinamou chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chilean tinamou chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chilean tinamou'</li>\n",
       "\t<li>'Chilean tinamou homology type'</li>\n",
       "\t<li>'%id. target Chilean tinamou gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chilean tinamou gene'</li>\n",
       "\t<li>'Chilean tinamou Gene-order conservation score'</li>\n",
       "\t<li>'Chilean tinamou Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chilean tinamou'</li>\n",
       "\t<li>'dS with Chilean tinamou'</li>\n",
       "\t<li>'Chilean tinamou orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chimpanzee gene stable ID'</li>\n",
       "\t<li>'Chimpanzee gene name'</li>\n",
       "\t<li>'Chimpanzee protein or transcript stable ID'</li>\n",
       "\t<li>'Chimpanzee chromosome/scaffold name'</li>\n",
       "\t<li>'Chimpanzee chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chimpanzee chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chimpanzee'</li>\n",
       "\t<li>'Chimpanzee homology type'</li>\n",
       "\t<li>'%id. target Chimpanzee gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chimpanzee gene'</li>\n",
       "\t<li>'Chimpanzee Gene-order conservation score'</li>\n",
       "\t<li>'Chimpanzee Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chimpanzee'</li>\n",
       "\t<li>'dS with Chimpanzee'</li>\n",
       "\t<li>'Chimpanzee orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS gene stable ID'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS gene name'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS protein or transcript stable ID'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS chromosome/scaffold name'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chinese hamster CHOK1GS'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS homology type'</li>\n",
       "\t<li>'%id. target Chinese hamster CHOK1GS gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chinese hamster CHOK1GS gene'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS Gene-order conservation score'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chinese hamster CHOK1GS'</li>\n",
       "\t<li>'dS with Chinese hamster CHOK1GS'</li>\n",
       "\t<li>'Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chinese hamster CriGri gene stable ID'</li>\n",
       "\t<li>'Chinese hamster CriGri gene name'</li>\n",
       "\t<li>'Chinese hamster CriGri protein or transcript stable ID'</li>\n",
       "\t<li>'Chinese hamster CriGri chromosome/scaffold name'</li>\n",
       "\t<li>'Chinese hamster CriGri chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chinese hamster CriGri chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chinese hamster CriGri'</li>\n",
       "\t<li>'Chinese hamster CriGri homology type'</li>\n",
       "\t<li>'%id. target Chinese hamster CriGri gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chinese hamster CriGri gene'</li>\n",
       "\t<li>'Chinese hamster CriGri Gene-order conservation score'</li>\n",
       "\t<li>'Chinese hamster CriGri Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chinese hamster CriGri'</li>\n",
       "\t<li>'dS with Chinese hamster CriGri'</li>\n",
       "\t<li>'Chinese hamster CriGri orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chinese hamster PICR gene stable ID'</li>\n",
       "\t<li>'Chinese hamster PICR gene name'</li>\n",
       "\t<li>'Chinese hamster PICR protein or transcript stable ID'</li>\n",
       "\t<li>'Chinese hamster PICR chromosome/scaffold name'</li>\n",
       "\t<li>'Chinese hamster PICR chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chinese hamster PICR chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chinese hamster PICR'</li>\n",
       "\t<li>'Chinese hamster PICR homology type'</li>\n",
       "\t<li>'%id. target Chinese hamster PICR gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chinese hamster PICR gene'</li>\n",
       "\t<li>'Chinese hamster PICR Gene-order conservation score'</li>\n",
       "\t<li>'Chinese hamster PICR Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chinese hamster PICR'</li>\n",
       "\t<li>'dS with Chinese hamster PICR'</li>\n",
       "\t<li>'Chinese hamster PICR orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Chinese softshell turtle gene stable ID'</li>\n",
       "\t<li>'Chinese softshell turtle gene name'</li>\n",
       "\t<li>'Chinese softshell turtle protein or transcript stable ID'</li>\n",
       "\t<li>'Chinese softshell turtle chromosome/scaffold name'</li>\n",
       "\t<li>'Chinese softshell turtle chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Chinese softshell turtle chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Chinese softshell turtle'</li>\n",
       "\t<li>'Chinese softshell turtle homology type'</li>\n",
       "\t<li>'%id. target Chinese softshell turtle gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Chinese softshell turtle gene'</li>\n",
       "\t<li>'Chinese softshell turtle Gene-order conservation score'</li>\n",
       "\t<li>'Chinese softshell turtle Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Chinese softshell turtle'</li>\n",
       "\t<li>'dS with Chinese softshell turtle'</li>\n",
       "\t<li>'Chinese softshell turtle orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Climbing perch gene stable ID'</li>\n",
       "\t<li>'Climbing perch gene name'</li>\n",
       "\t<li>'Climbing perch protein or transcript stable ID'</li>\n",
       "\t<li>'Climbing perch chromosome/scaffold name'</li>\n",
       "\t<li>'Climbing perch chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Climbing perch chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Climbing perch'</li>\n",
       "\t<li>'Climbing perch homology type'</li>\n",
       "\t<li>'%id. target Climbing perch gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Climbing perch gene'</li>\n",
       "\t<li>'Climbing perch Gene-order conservation score'</li>\n",
       "\t<li>'Climbing perch Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Climbing perch'</li>\n",
       "\t<li>'dS with Climbing perch'</li>\n",
       "\t<li>'Climbing perch orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Clown anemonefish gene stable ID'</li>\n",
       "\t<li>'Clown anemonefish gene name'</li>\n",
       "\t<li>'Clown anemonefish protein or transcript stable ID'</li>\n",
       "\t<li>'Clown anemonefish chromosome/scaffold name'</li>\n",
       "\t<li>'Clown anemonefish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Clown anemonefish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Clown anemonefish'</li>\n",
       "\t<li>'Clown anemonefish homology type'</li>\n",
       "\t<li>'%id. target Clown anemonefish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Clown anemonefish gene'</li>\n",
       "\t<li>'Clown anemonefish Gene-order conservation score'</li>\n",
       "\t<li>'Clown anemonefish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Clown anemonefish'</li>\n",
       "\t<li>'dS with Clown anemonefish'</li>\n",
       "\t<li>'Clown anemonefish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Cod gene stable ID'</li>\n",
       "\t<li>'Cod gene name'</li>\n",
       "\t<li>'Cod protein or transcript stable ID'</li>\n",
       "\t<li>'Cod chromosome/scaffold name'</li>\n",
       "\t<li>'Cod chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Cod chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Cod'</li>\n",
       "\t<li>'Cod homology type'</li>\n",
       "\t<li>'%id. target Cod gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Cod gene'</li>\n",
       "\t<li>'Cod Gene-order conservation score'</li>\n",
       "\t<li>'Cod Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Cod'</li>\n",
       "\t<li>'dS with Cod'</li>\n",
       "\t<li>'Cod orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Coelacanth gene stable ID'</li>\n",
       "\t<li>'Coelacanth gene name'</li>\n",
       "\t<li>'Coelacanth protein or transcript stable ID'</li>\n",
       "\t<li>'Coelacanth chromosome/scaffold name'</li>\n",
       "\t<li>'Coelacanth chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Coelacanth chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Coelacanth'</li>\n",
       "\t<li>'Coelacanth homology type'</li>\n",
       "\t<li>'%id. target Coelacanth gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Coelacanth gene'</li>\n",
       "\t<li>'Coelacanth Gene-order conservation score'</li>\n",
       "\t<li>'Coelacanth Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Coelacanth'</li>\n",
       "\t<li>'dS with Coelacanth'</li>\n",
       "\t<li>'Coelacanth orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Common canary gene stable ID'</li>\n",
       "\t<li>'Common canary gene name'</li>\n",
       "\t<li>'Common canary protein or transcript stable ID'</li>\n",
       "\t<li>'Common canary chromosome/scaffold name'</li>\n",
       "\t<li>'Common canary chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Common canary chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Common canary'</li>\n",
       "\t<li>'Common canary homology type'</li>\n",
       "\t<li>'%id. target Common canary gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Common canary gene'</li>\n",
       "\t<li>'Common canary Gene-order conservation score'</li>\n",
       "\t<li>'Common canary Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Common canary'</li>\n",
       "\t<li>'dS with Common canary'</li>\n",
       "\t<li>'Common canary orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Common wombat gene stable ID'</li>\n",
       "\t<li>'Common wombat gene name'</li>\n",
       "\t<li>'Common wombat protein or transcript stable ID'</li>\n",
       "\t<li>'Common wombat chromosome/scaffold name'</li>\n",
       "\t<li>'Common wombat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Common wombat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Common wombat'</li>\n",
       "\t<li>'Common wombat homology type'</li>\n",
       "\t<li>'%id. target Common wombat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Common wombat gene'</li>\n",
       "\t<li>'Common wombat Gene-order conservation score'</li>\n",
       "\t<li>'Common wombat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Common wombat'</li>\n",
       "\t<li>'dS with Common wombat'</li>\n",
       "\t<li>'Common wombat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Coquerel\\'s sifaka gene stable ID'</li>\n",
       "\t<li>'Coquerel\\'s sifaka gene name'</li>\n",
       "\t<li>'Coquerel\\'s sifaka protein or transcript stable ID'</li>\n",
       "\t<li>'Coquerel\\'s sifaka chromosome/scaffold name'</li>\n",
       "\t<li>'Coquerel\\'s sifaka chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Coquerel\\'s sifaka chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Coquerel\\'s sifaka'</li>\n",
       "\t<li>'Coquerel\\'s sifaka homology type'</li>\n",
       "\t<li>'%id. target Coquerel\\'s sifaka gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Coquerel\\'s sifaka gene'</li>\n",
       "\t<li>'Coquerel\\'s sifaka Gene-order conservation score'</li>\n",
       "\t<li>'Coquerel\\'s sifaka Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Coquerel\\'s sifaka'</li>\n",
       "\t<li>'dS with Coquerel\\'s sifaka'</li>\n",
       "\t<li>'Coquerel\\'s sifaka orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Cow gene stable ID'</li>\n",
       "\t<li>'Cow gene name'</li>\n",
       "\t<li>'Cow protein or transcript stable ID'</li>\n",
       "\t<li>'Cow chromosome/scaffold name'</li>\n",
       "\t<li>'Cow chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Cow chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Cow'</li>\n",
       "\t<li>'Cow homology type'</li>\n",
       "\t<li>'%id. target Cow gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Cow gene'</li>\n",
       "\t<li>'Cow Gene-order conservation score'</li>\n",
       "\t<li>'Cow Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Cow'</li>\n",
       "\t<li>'dS with Cow'</li>\n",
       "\t<li>'Cow orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Crab-eating macaque gene stable ID'</li>\n",
       "\t<li>'Crab-eating macaque gene name'</li>\n",
       "\t<li>'Crab-eating macaque protein or transcript stable ID'</li>\n",
       "\t<li>'Crab-eating macaque chromosome/scaffold name'</li>\n",
       "\t<li>'Crab-eating macaque chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Crab-eating macaque chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Crab-eating macaque'</li>\n",
       "\t<li>'Crab-eating macaque homology type'</li>\n",
       "\t<li>'%id. target Crab-eating macaque gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Crab-eating macaque gene'</li>\n",
       "\t<li>'Crab-eating macaque Gene-order conservation score'</li>\n",
       "\t<li>'Crab-eating macaque Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Crab-eating macaque'</li>\n",
       "\t<li>'dS with Crab-eating macaque'</li>\n",
       "\t<li>'Crab-eating macaque orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Damara mole rat gene stable ID'</li>\n",
       "\t<li>'Damara mole rat gene name'</li>\n",
       "\t<li>'Damara mole rat protein or transcript stable ID'</li>\n",
       "\t<li>'Damara mole rat chromosome/scaffold name'</li>\n",
       "\t<li>'Damara mole rat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Damara mole rat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Damara mole rat'</li>\n",
       "\t<li>'Damara mole rat homology type'</li>\n",
       "\t<li>'%id. target Damara mole rat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Damara mole rat gene'</li>\n",
       "\t<li>'Damara mole rat Gene-order conservation score'</li>\n",
       "\t<li>'Damara mole rat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Damara mole rat'</li>\n",
       "\t<li>'dS with Damara mole rat'</li>\n",
       "\t<li>'Damara mole rat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Dark-eyed junco gene stable ID'</li>\n",
       "\t<li>'Dark-eyed junco gene name'</li>\n",
       "\t<li>'Dark-eyed junco protein or transcript stable ID'</li>\n",
       "\t<li>'Dark-eyed junco chromosome/scaffold name'</li>\n",
       "\t<li>'Dark-eyed junco chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Dark-eyed junco chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Dark-eyed junco'</li>\n",
       "\t<li>'Dark-eyed junco homology type'</li>\n",
       "\t<li>'%id. target Dark-eyed junco gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Dark-eyed junco gene'</li>\n",
       "\t<li>'Dark-eyed junco Gene-order conservation score'</li>\n",
       "\t<li>'Dark-eyed junco Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Dark-eyed junco'</li>\n",
       "\t<li>'dS with Dark-eyed junco'</li>\n",
       "\t<li>'Dark-eyed junco orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Daurian ground squirrel gene stable ID'</li>\n",
       "\t<li>'Daurian ground squirrel gene name'</li>\n",
       "\t<li>'Daurian ground squirrel protein or transcript stable ID'</li>\n",
       "\t<li>'Daurian ground squirrel chromosome/scaffold name'</li>\n",
       "\t<li>'Daurian ground squirrel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Daurian ground squirrel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Daurian ground squirrel'</li>\n",
       "\t<li>'Daurian ground squirrel homology type'</li>\n",
       "\t<li>'%id. target Daurian ground squirrel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Daurian ground squirrel gene'</li>\n",
       "\t<li>'Daurian ground squirrel Gene-order conservation score'</li>\n",
       "\t<li>'Daurian ground squirrel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Daurian ground squirrel'</li>\n",
       "\t<li>'dS with Daurian ground squirrel'</li>\n",
       "\t<li>'Daurian ground squirrel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Degu gene stable ID'</li>\n",
       "\t<li>'Degu gene name'</li>\n",
       "\t<li>'Degu protein or transcript stable ID'</li>\n",
       "\t<li>'Degu chromosome/scaffold name'</li>\n",
       "\t<li>'Degu chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Degu chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Degu'</li>\n",
       "\t<li>'Degu homology type'</li>\n",
       "\t<li>'%id. target Degu gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Degu gene'</li>\n",
       "\t<li>'Degu Gene-order conservation score'</li>\n",
       "\t<li>'Degu Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Degu'</li>\n",
       "\t<li>'dS with Degu'</li>\n",
       "\t<li>'Degu orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Dingo gene stable ID'</li>\n",
       "\t<li>'Dingo gene name'</li>\n",
       "\t<li>'Dingo protein or transcript stable ID'</li>\n",
       "\t<li>'Dingo chromosome/scaffold name'</li>\n",
       "\t<li>'Dingo chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Dingo chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Dingo'</li>\n",
       "\t<li>'Dingo homology type'</li>\n",
       "\t<li>'%id. target Dingo gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Dingo gene'</li>\n",
       "\t<li>'Dingo Gene-order conservation score'</li>\n",
       "\t<li>'Dingo Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Dingo'</li>\n",
       "\t<li>'dS with Dingo'</li>\n",
       "\t<li>'Dingo orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Dog gene stable ID'</li>\n",
       "\t<li>'Dog gene name'</li>\n",
       "\t<li>'Dog protein or transcript stable ID'</li>\n",
       "\t<li>'Dog chromosome/scaffold name'</li>\n",
       "\t<li>'Dog chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Dog chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Dog'</li>\n",
       "\t<li>'Dog homology type'</li>\n",
       "\t<li>'%id. target Dog gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Dog gene'</li>\n",
       "\t<li>'Dog Gene-order conservation score'</li>\n",
       "\t<li>'Dog Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Dog'</li>\n",
       "\t<li>'dS with Dog'</li>\n",
       "\t<li>'Dog orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Dolphin gene stable ID'</li>\n",
       "\t<li>'Dolphin gene name'</li>\n",
       "\t<li>'Dolphin protein or transcript stable ID'</li>\n",
       "\t<li>'Dolphin chromosome/scaffold name'</li>\n",
       "\t<li>'Dolphin chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Dolphin chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Dolphin'</li>\n",
       "\t<li>'Dolphin homology type'</li>\n",
       "\t<li>'%id. target Dolphin gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Dolphin gene'</li>\n",
       "\t<li>'Dolphin Gene-order conservation score'</li>\n",
       "\t<li>'Dolphin Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Dolphin'</li>\n",
       "\t<li>'dS with Dolphin'</li>\n",
       "\t<li>'Dolphin orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Donkey gene stable ID'</li>\n",
       "\t<li>'Donkey gene name'</li>\n",
       "\t<li>'Donkey protein or transcript stable ID'</li>\n",
       "\t<li>'Donkey chromosome/scaffold name'</li>\n",
       "\t<li>'Donkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Donkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Donkey'</li>\n",
       "\t<li>'Donkey homology type'</li>\n",
       "\t<li>'%id. target Donkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Donkey gene'</li>\n",
       "\t<li>'Donkey Gene-order conservation score'</li>\n",
       "\t<li>'Donkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Donkey'</li>\n",
       "\t<li>'dS with Donkey'</li>\n",
       "\t<li>'Donkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Drill gene stable ID'</li>\n",
       "\t<li>'Drill gene name'</li>\n",
       "\t<li>'Drill protein or transcript stable ID'</li>\n",
       "\t<li>'Drill chromosome/scaffold name'</li>\n",
       "\t<li>'Drill chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Drill chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Drill'</li>\n",
       "\t<li>'Drill homology type'</li>\n",
       "\t<li>'%id. target Drill gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Drill gene'</li>\n",
       "\t<li>'Drill Gene-order conservation score'</li>\n",
       "\t<li>'Drill Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Drill'</li>\n",
       "\t<li>'dS with Drill'</li>\n",
       "\t<li>'Drill orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Drosophila melanogaster gene stable ID'</li>\n",
       "\t<li>'Drosophila melanogaster gene name'</li>\n",
       "\t<li>'Drosophila melanogaster protein or transcript stable ID'</li>\n",
       "\t<li>'Drosophila melanogaster chromosome/scaffold name'</li>\n",
       "\t<li>'Drosophila melanogaster chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Drosophila melanogaster chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Drosophila melanogaster'</li>\n",
       "\t<li>'Drosophila melanogaster homology type'</li>\n",
       "\t<li>'%id. target Drosophila melanogaster gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Drosophila melanogaster gene'</li>\n",
       "\t<li>'Drosophila melanogaster Gene-order conservation score'</li>\n",
       "\t<li>'Drosophila melanogaster Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Drosophila melanogaster'</li>\n",
       "\t<li>'dS with Drosophila melanogaster'</li>\n",
       "\t<li>'Drosophila melanogaster orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Duck gene stable ID'</li>\n",
       "\t<li>'Duck gene name'</li>\n",
       "\t<li>'Duck protein or transcript stable ID'</li>\n",
       "\t<li>'Duck chromosome/scaffold name'</li>\n",
       "\t<li>'Duck chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Duck chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Duck'</li>\n",
       "\t<li>'Duck homology type'</li>\n",
       "\t<li>'%id. target Duck gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Duck gene'</li>\n",
       "\t<li>'Duck Gene-order conservation score'</li>\n",
       "\t<li>'Duck Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Duck'</li>\n",
       "\t<li>'dS with Duck'</li>\n",
       "\t<li>'Duck orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Eastern happy gene stable ID'</li>\n",
       "\t<li>'Eastern happy gene name'</li>\n",
       "\t<li>'Eastern happy protein or transcript stable ID'</li>\n",
       "\t<li>'Eastern happy chromosome/scaffold name'</li>\n",
       "\t<li>'Eastern happy chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Eastern happy chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Eastern happy'</li>\n",
       "\t<li>'Eastern happy homology type'</li>\n",
       "\t<li>'%id. target Eastern happy gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Eastern happy gene'</li>\n",
       "\t<li>'Eastern happy Gene-order conservation score'</li>\n",
       "\t<li>'Eastern happy Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Eastern happy'</li>\n",
       "\t<li>'dS with Eastern happy'</li>\n",
       "\t<li>'Eastern happy orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Electric eel gene stable ID'</li>\n",
       "\t<li>'Electric eel gene name'</li>\n",
       "\t<li>'Electric eel protein or transcript stable ID'</li>\n",
       "\t<li>'Electric eel chromosome/scaffold name'</li>\n",
       "\t<li>'Electric eel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Electric eel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Electric eel'</li>\n",
       "\t<li>'Electric eel homology type'</li>\n",
       "\t<li>'%id. target Electric eel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Electric eel gene'</li>\n",
       "\t<li>'Electric eel Gene-order conservation score'</li>\n",
       "\t<li>'Electric eel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Electric eel'</li>\n",
       "\t<li>'dS with Electric eel'</li>\n",
       "\t<li>'Electric eel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Elephant gene stable ID'</li>\n",
       "\t<li>'Elephant gene name'</li>\n",
       "\t<li>'Elephant protein or transcript stable ID'</li>\n",
       "\t<li>'Elephant chromosome/scaffold name'</li>\n",
       "\t<li>'Elephant chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Elephant chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Elephant'</li>\n",
       "\t<li>'Elephant homology type'</li>\n",
       "\t<li>'%id. target Elephant gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Elephant gene'</li>\n",
       "\t<li>'Elephant Gene-order conservation score'</li>\n",
       "\t<li>'Elephant Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Elephant'</li>\n",
       "\t<li>'dS with Elephant'</li>\n",
       "\t<li>'Elephant orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Elephant shark gene stable ID'</li>\n",
       "\t<li>'Elephant shark gene name'</li>\n",
       "\t<li>'Elephant shark protein or transcript stable ID'</li>\n",
       "\t<li>'Elephant shark chromosome/scaffold name'</li>\n",
       "\t<li>'Elephant shark chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Elephant shark chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Elephant shark'</li>\n",
       "\t<li>'Elephant shark homology type'</li>\n",
       "\t<li>'%id. target Elephant shark gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Elephant shark gene'</li>\n",
       "\t<li>'Elephant shark Gene-order conservation score'</li>\n",
       "\t<li>'Elephant shark Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Elephant shark'</li>\n",
       "\t<li>'dS with Elephant shark'</li>\n",
       "\t<li>'Elephant shark orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Emu gene stable ID'</li>\n",
       "\t<li>'Emu gene name'</li>\n",
       "\t<li>'Emu protein or transcript stable ID'</li>\n",
       "\t<li>'Emu chromosome/scaffold name'</li>\n",
       "\t<li>'Emu chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Emu chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Emu'</li>\n",
       "\t<li>'Emu homology type'</li>\n",
       "\t<li>'%id. target Emu gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Emu gene'</li>\n",
       "\t<li>'Emu Gene-order conservation score'</li>\n",
       "\t<li>'Emu Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Emu'</li>\n",
       "\t<li>'dS with Emu'</li>\n",
       "\t<li>'Emu orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ferret gene stable ID'</li>\n",
       "\t<li>'Ferret gene name'</li>\n",
       "\t<li>'Ferret protein or transcript stable ID'</li>\n",
       "\t<li>'Ferret chromosome/scaffold name'</li>\n",
       "\t<li>'Ferret chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ferret chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ferret'</li>\n",
       "\t<li>'Ferret homology type'</li>\n",
       "\t<li>'%id. target Ferret gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ferret gene'</li>\n",
       "\t<li>'Ferret Gene-order conservation score'</li>\n",
       "\t<li>'Ferret Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ferret'</li>\n",
       "\t<li>'dS with Ferret'</li>\n",
       "\t<li>'Ferret orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Flycatcher gene stable ID'</li>\n",
       "\t<li>'Flycatcher gene name'</li>\n",
       "\t<li>'Flycatcher protein or transcript stable ID'</li>\n",
       "\t<li>'Flycatcher chromosome/scaffold name'</li>\n",
       "\t<li>'Flycatcher chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Flycatcher chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Flycatcher'</li>\n",
       "\t<li>'Flycatcher homology type'</li>\n",
       "\t<li>'%id. target Flycatcher gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Flycatcher gene'</li>\n",
       "\t<li>'Flycatcher Gene-order conservation score'</li>\n",
       "\t<li>'Flycatcher Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Flycatcher'</li>\n",
       "\t<li>'dS with Flycatcher'</li>\n",
       "\t<li>'Flycatcher orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Fugu gene stable ID'</li>\n",
       "\t<li>'Fugu gene name'</li>\n",
       "\t<li>'Fugu protein or transcript stable ID'</li>\n",
       "\t<li>'Fugu chromosome/scaffold name'</li>\n",
       "\t<li>'Fugu chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Fugu chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Fugu'</li>\n",
       "\t<li>'Fugu homology type'</li>\n",
       "\t<li>'%id. target Fugu gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Fugu gene'</li>\n",
       "\t<li>'Fugu Gene-order conservation score'</li>\n",
       "\t<li>'Fugu Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Fugu'</li>\n",
       "\t<li>'dS with Fugu'</li>\n",
       "\t<li>'Fugu orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Gelada gene stable ID'</li>\n",
       "\t<li>'Gelada gene name'</li>\n",
       "\t<li>'Gelada protein or transcript stable ID'</li>\n",
       "\t<li>'Gelada chromosome/scaffold name'</li>\n",
       "\t<li>'Gelada chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Gelada chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Gelada'</li>\n",
       "\t<li>'Gelada homology type'</li>\n",
       "\t<li>'%id. target Gelada gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Gelada gene'</li>\n",
       "\t<li>'Gelada Gene-order conservation score'</li>\n",
       "\t<li>'Gelada Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Gelada'</li>\n",
       "\t<li>'dS with Gelada'</li>\n",
       "\t<li>'Gelada orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Gibbon gene stable ID'</li>\n",
       "\t<li>'Gibbon gene name'</li>\n",
       "\t<li>'Gibbon protein or transcript stable ID'</li>\n",
       "\t<li>'Gibbon chromosome/scaffold name'</li>\n",
       "\t<li>'Gibbon chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Gibbon chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Gibbon'</li>\n",
       "\t<li>'Gibbon homology type'</li>\n",
       "\t<li>'%id. target Gibbon gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Gibbon gene'</li>\n",
       "\t<li>'Gibbon Gene-order conservation score'</li>\n",
       "\t<li>'Gibbon Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Gibbon'</li>\n",
       "\t<li>'dS with Gibbon'</li>\n",
       "\t<li>'Gibbon orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Goat gene stable ID'</li>\n",
       "\t<li>'Goat gene name'</li>\n",
       "\t<li>'Goat protein or transcript stable ID'</li>\n",
       "\t<li>'Goat chromosome/scaffold name'</li>\n",
       "\t<li>'Goat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Goat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Goat'</li>\n",
       "\t<li>'Goat homology type'</li>\n",
       "\t<li>'%id. target Goat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Goat gene'</li>\n",
       "\t<li>'Goat Gene-order conservation score'</li>\n",
       "\t<li>'Goat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Goat'</li>\n",
       "\t<li>'dS with Goat'</li>\n",
       "\t<li>'Goat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Golden Hamster gene stable ID'</li>\n",
       "\t<li>'Golden Hamster gene name'</li>\n",
       "\t<li>'Golden Hamster protein or transcript stable ID'</li>\n",
       "\t<li>'Golden Hamster chromosome/scaffold name'</li>\n",
       "\t<li>'Golden Hamster chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Golden Hamster chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Golden Hamster'</li>\n",
       "\t<li>'Golden Hamster homology type'</li>\n",
       "\t<li>'%id. target Golden Hamster gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Golden Hamster gene'</li>\n",
       "\t<li>'Golden Hamster Gene-order conservation score'</li>\n",
       "\t<li>'Golden Hamster Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Golden Hamster'</li>\n",
       "\t<li>'dS with Golden Hamster'</li>\n",
       "\t<li>'Golden Hamster orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Golden snub-nosed monkey gene stable ID'</li>\n",
       "\t<li>'Golden snub-nosed monkey gene name'</li>\n",
       "\t<li>'Golden snub-nosed monkey protein or transcript stable ID'</li>\n",
       "\t<li>'Golden snub-nosed monkey chromosome/scaffold name'</li>\n",
       "\t<li>'Golden snub-nosed monkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Golden snub-nosed monkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Golden snub-nosed monkey'</li>\n",
       "\t<li>'Golden snub-nosed monkey homology type'</li>\n",
       "\t<li>'%id. target Golden snub-nosed monkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Golden snub-nosed monkey gene'</li>\n",
       "\t<li>'Golden snub-nosed monkey Gene-order conservation score'</li>\n",
       "\t<li>'Golden snub-nosed monkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Golden snub-nosed monkey'</li>\n",
       "\t<li>'dS with Golden snub-nosed monkey'</li>\n",
       "\t<li>'Golden snub-nosed monkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Golden-collared manakin gene stable ID'</li>\n",
       "\t<li>'Golden-collared manakin gene name'</li>\n",
       "\t<li>'Golden-collared manakin protein or transcript stable ID'</li>\n",
       "\t<li>'Golden-collared manakin chromosome/scaffold name'</li>\n",
       "\t<li>'Golden-collared manakin chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Golden-collared manakin chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Golden-collared manakin'</li>\n",
       "\t<li>'Golden-collared manakin homology type'</li>\n",
       "\t<li>'%id. target Golden-collared manakin gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Golden-collared manakin gene'</li>\n",
       "\t<li>'Golden-collared manakin Gene-order conservation score'</li>\n",
       "\t<li>'Golden-collared manakin Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Golden-collared manakin'</li>\n",
       "\t<li>'dS with Golden-collared manakin'</li>\n",
       "\t<li>'Golden-collared manakin orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Gorilla gene stable ID'</li>\n",
       "\t<li>'Gorilla gene name'</li>\n",
       "\t<li>'Gorilla protein or transcript stable ID'</li>\n",
       "\t<li>'Gorilla chromosome/scaffold name'</li>\n",
       "\t<li>'Gorilla chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Gorilla chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Gorilla'</li>\n",
       "\t<li>'Gorilla homology type'</li>\n",
       "\t<li>'%id. target Gorilla gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Gorilla gene'</li>\n",
       "\t<li>'Gorilla Gene-order conservation score'</li>\n",
       "\t<li>'Gorilla Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Gorilla'</li>\n",
       "\t<li>'dS with Gorilla'</li>\n",
       "\t<li>'Gorilla orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Great Tit gene stable ID'</li>\n",
       "\t<li>'Great Tit gene name'</li>\n",
       "\t<li>'Great Tit protein or transcript stable ID'</li>\n",
       "\t<li>'Great Tit chromosome/scaffold name'</li>\n",
       "\t<li>'Great Tit chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Great Tit chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Great Tit'</li>\n",
       "\t<li>'Great Tit homology type'</li>\n",
       "\t<li>'%id. target Great Tit gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Great Tit gene'</li>\n",
       "\t<li>'Great Tit Gene-order conservation score'</li>\n",
       "\t<li>'Great Tit Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Great Tit'</li>\n",
       "\t<li>'dS with Great Tit'</li>\n",
       "\t<li>'Great Tit orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Great spotted kiwi gene stable ID'</li>\n",
       "\t<li>'Great spotted kiwi gene name'</li>\n",
       "\t<li>'Great spotted kiwi protein or transcript stable ID'</li>\n",
       "\t<li>'Great spotted kiwi chromosome/scaffold name'</li>\n",
       "\t<li>'Great spotted kiwi chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Great spotted kiwi chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Great spotted kiwi'</li>\n",
       "\t<li>'Great spotted kiwi homology type'</li>\n",
       "\t<li>'%id. target Great spotted kiwi gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Great spotted kiwi gene'</li>\n",
       "\t<li>'Great spotted kiwi Gene-order conservation score'</li>\n",
       "\t<li>'Great spotted kiwi Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Great spotted kiwi'</li>\n",
       "\t<li>'dS with Great spotted kiwi'</li>\n",
       "\t<li>'Great spotted kiwi orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Greater amberjack gene stable ID'</li>\n",
       "\t<li>'Greater amberjack gene name'</li>\n",
       "\t<li>'Greater amberjack protein or transcript stable ID'</li>\n",
       "\t<li>'Greater amberjack chromosome/scaffold name'</li>\n",
       "\t<li>'Greater amberjack chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Greater amberjack chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Greater amberjack'</li>\n",
       "\t<li>'Greater amberjack homology type'</li>\n",
       "\t<li>'%id. target Greater amberjack gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Greater amberjack gene'</li>\n",
       "\t<li>'Greater amberjack Gene-order conservation score'</li>\n",
       "\t<li>'Greater amberjack Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Greater amberjack'</li>\n",
       "\t<li>'dS with Greater amberjack'</li>\n",
       "\t<li>'Greater amberjack orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Greater bamboo lemur gene stable ID'</li>\n",
       "\t<li>'Greater bamboo lemur gene name'</li>\n",
       "\t<li>'Greater bamboo lemur protein or transcript stable ID'</li>\n",
       "\t<li>'Greater bamboo lemur chromosome/scaffold name'</li>\n",
       "\t<li>'Greater bamboo lemur chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Greater bamboo lemur chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Greater bamboo lemur'</li>\n",
       "\t<li>'Greater bamboo lemur homology type'</li>\n",
       "\t<li>'%id. target Greater bamboo lemur gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Greater bamboo lemur gene'</li>\n",
       "\t<li>'Greater bamboo lemur Gene-order conservation score'</li>\n",
       "\t<li>'Greater bamboo lemur Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Greater bamboo lemur'</li>\n",
       "\t<li>'dS with Greater bamboo lemur'</li>\n",
       "\t<li>'Greater bamboo lemur orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Guinea Pig gene stable ID'</li>\n",
       "\t<li>'Guinea Pig gene name'</li>\n",
       "\t<li>'Guinea Pig protein or transcript stable ID'</li>\n",
       "\t<li>'Guinea Pig chromosome/scaffold name'</li>\n",
       "\t<li>'Guinea Pig chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Guinea Pig chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Guinea Pig'</li>\n",
       "\t<li>'Guinea Pig homology type'</li>\n",
       "\t<li>'%id. target Guinea Pig gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Guinea Pig gene'</li>\n",
       "\t<li>'Guinea Pig Gene-order conservation score'</li>\n",
       "\t<li>'Guinea Pig Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Guinea Pig'</li>\n",
       "\t<li>'dS with Guinea Pig'</li>\n",
       "\t<li>'Guinea Pig orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Guppy gene stable ID'</li>\n",
       "\t<li>'Guppy gene name'</li>\n",
       "\t<li>'Guppy protein or transcript stable ID'</li>\n",
       "\t<li>'Guppy chromosome/scaffold name'</li>\n",
       "\t<li>'Guppy chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Guppy chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Guppy'</li>\n",
       "\t<li>'Guppy homology type'</li>\n",
       "\t<li>'%id. target Guppy gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Guppy gene'</li>\n",
       "\t<li>'Guppy Gene-order conservation score'</li>\n",
       "\t<li>'Guppy Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Guppy'</li>\n",
       "\t<li>'dS with Guppy'</li>\n",
       "\t<li>'Guppy orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Hagfish gene stable ID'</li>\n",
       "\t<li>'Hagfish gene name'</li>\n",
       "\t<li>'Hagfish protein or transcript stable ID'</li>\n",
       "\t<li>'Hagfish chromosome/scaffold name'</li>\n",
       "\t<li>'Hagfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Hagfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Hagfish'</li>\n",
       "\t<li>'Hagfish homology type'</li>\n",
       "\t<li>'%id. target Hagfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Hagfish gene'</li>\n",
       "\t<li>'Hagfish Gene-order conservation score'</li>\n",
       "\t<li>'Hagfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Hagfish'</li>\n",
       "\t<li>'dS with Hagfish'</li>\n",
       "\t<li>'Hagfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Hedgehog gene stable ID'</li>\n",
       "\t<li>'Hedgehog gene name'</li>\n",
       "\t<li>'Hedgehog protein or transcript stable ID'</li>\n",
       "\t<li>'Hedgehog chromosome/scaffold name'</li>\n",
       "\t<li>'Hedgehog chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Hedgehog chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Hedgehog'</li>\n",
       "\t<li>'Hedgehog homology type'</li>\n",
       "\t<li>'%id. target Hedgehog gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Hedgehog gene'</li>\n",
       "\t<li>'Hedgehog Gene-order conservation score'</li>\n",
       "\t<li>'Hedgehog Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Hedgehog'</li>\n",
       "\t<li>'dS with Hedgehog'</li>\n",
       "\t<li>'Hedgehog orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Helmeted guineafowl gene stable ID'</li>\n",
       "\t<li>'Helmeted guineafowl gene name'</li>\n",
       "\t<li>'Helmeted guineafowl protein or transcript stable ID'</li>\n",
       "\t<li>'Helmeted guineafowl chromosome/scaffold name'</li>\n",
       "\t<li>'Helmeted guineafowl chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Helmeted guineafowl chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Helmeted guineafowl'</li>\n",
       "\t<li>'Helmeted guineafowl homology type'</li>\n",
       "\t<li>'%id. target Helmeted guineafowl gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Helmeted guineafowl gene'</li>\n",
       "\t<li>'Helmeted guineafowl Gene-order conservation score'</li>\n",
       "\t<li>'Helmeted guineafowl Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Helmeted guineafowl'</li>\n",
       "\t<li>'dS with Helmeted guineafowl'</li>\n",
       "\t<li>'Helmeted guineafowl orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Horse gene stable ID'</li>\n",
       "\t<li>'Horse gene name'</li>\n",
       "\t<li>'Horse protein or transcript stable ID'</li>\n",
       "\t<li>'Horse chromosome/scaffold name'</li>\n",
       "\t<li>'Horse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Horse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Horse'</li>\n",
       "\t<li>'Horse homology type'</li>\n",
       "\t<li>'%id. target Horse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Horse gene'</li>\n",
       "\t<li>'Horse Gene-order conservation score'</li>\n",
       "\t<li>'Horse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Horse'</li>\n",
       "\t<li>'dS with Horse'</li>\n",
       "\t<li>'Horse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Huchen gene stable ID'</li>\n",
       "\t<li>'Huchen gene name'</li>\n",
       "\t<li>'Huchen protein or transcript stable ID'</li>\n",
       "\t<li>'Huchen chromosome/scaffold name'</li>\n",
       "\t<li>'Huchen chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Huchen chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Huchen'</li>\n",
       "\t<li>'Huchen homology type'</li>\n",
       "\t<li>'%id. target Huchen gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Huchen gene'</li>\n",
       "\t<li>'Huchen Gene-order conservation score'</li>\n",
       "\t<li>'Huchen Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Huchen'</li>\n",
       "\t<li>'dS with Huchen'</li>\n",
       "\t<li>'Huchen orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Hyrax gene stable ID'</li>\n",
       "\t<li>'Hyrax gene name'</li>\n",
       "\t<li>'Hyrax protein or transcript stable ID'</li>\n",
       "\t<li>'Hyrax chromosome/scaffold name'</li>\n",
       "\t<li>'Hyrax chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Hyrax chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Hyrax'</li>\n",
       "\t<li>'Hyrax homology type'</li>\n",
       "\t<li>'%id. target Hyrax gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Hyrax gene'</li>\n",
       "\t<li>'Hyrax Gene-order conservation score'</li>\n",
       "\t<li>'Hyrax Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Hyrax'</li>\n",
       "\t<li>'dS with Hyrax'</li>\n",
       "\t<li>'Hyrax orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Indian medaka gene stable ID'</li>\n",
       "\t<li>'Indian medaka gene name'</li>\n",
       "\t<li>'Indian medaka protein or transcript stable ID'</li>\n",
       "\t<li>'Indian medaka chromosome/scaffold name'</li>\n",
       "\t<li>'Indian medaka chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Indian medaka chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Indian medaka'</li>\n",
       "\t<li>'Indian medaka homology type'</li>\n",
       "\t<li>'%id. target Indian medaka gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Indian medaka gene'</li>\n",
       "\t<li>'Indian medaka Gene-order conservation score'</li>\n",
       "\t<li>'Indian medaka Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Indian medaka'</li>\n",
       "\t<li>'dS with Indian medaka'</li>\n",
       "\t<li>'Indian medaka orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Japanese medaka HNI gene stable ID'</li>\n",
       "\t<li>'Japanese medaka HNI gene name'</li>\n",
       "\t<li>'Japanese medaka HNI protein or transcript stable ID'</li>\n",
       "\t<li>'Japanese medaka HNI chromosome/scaffold name'</li>\n",
       "\t<li>'Japanese medaka HNI chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Japanese medaka HNI chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Japanese medaka HNI'</li>\n",
       "\t<li>'Japanese medaka HNI homology type'</li>\n",
       "\t<li>'%id. target Japanese medaka HNI gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Japanese medaka HNI gene'</li>\n",
       "\t<li>'Japanese medaka HNI Gene-order conservation score'</li>\n",
       "\t<li>'Japanese medaka HNI Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Japanese medaka HNI'</li>\n",
       "\t<li>'dS with Japanese medaka HNI'</li>\n",
       "\t<li>'Japanese medaka HNI orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Japanese medaka HSOK gene stable ID'</li>\n",
       "\t<li>'Japanese medaka HSOK gene name'</li>\n",
       "\t<li>'Japanese medaka HSOK protein or transcript stable ID'</li>\n",
       "\t<li>'Japanese medaka HSOK chromosome/scaffold name'</li>\n",
       "\t<li>'Japanese medaka HSOK chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Japanese medaka HSOK chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Japanese medaka HSOK'</li>\n",
       "\t<li>'Japanese medaka HSOK homology type'</li>\n",
       "\t<li>'%id. target Japanese medaka HSOK gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Japanese medaka HSOK gene'</li>\n",
       "\t<li>'Japanese medaka HSOK Gene-order conservation score'</li>\n",
       "\t<li>'Japanese medaka HSOK Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Japanese medaka HSOK'</li>\n",
       "\t<li>'dS with Japanese medaka HSOK'</li>\n",
       "\t<li>'Japanese medaka HSOK orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Japanese medaka HdrR gene stable ID'</li>\n",
       "\t<li>'Japanese medaka HdrR gene name'</li>\n",
       "\t<li>'Japanese medaka HdrR protein or transcript stable ID'</li>\n",
       "\t<li>'Japanese medaka HdrR chromosome/scaffold name'</li>\n",
       "\t<li>'Japanese medaka HdrR chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Japanese medaka HdrR chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Japanese medaka HdrR'</li>\n",
       "\t<li>'Japanese medaka HdrR homology type'</li>\n",
       "\t<li>'%id. target Japanese medaka HdrR gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Japanese medaka HdrR gene'</li>\n",
       "\t<li>'Japanese medaka HdrR Gene-order conservation score'</li>\n",
       "\t<li>'Japanese medaka HdrR Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Japanese medaka HdrR'</li>\n",
       "\t<li>'dS with Japanese medaka HdrR'</li>\n",
       "\t<li>'Japanese medaka HdrR orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Japanese quail gene stable ID'</li>\n",
       "\t<li>'Japanese quail gene name'</li>\n",
       "\t<li>'Japanese quail protein or transcript stable ID'</li>\n",
       "\t<li>'Japanese quail chromosome/scaffold name'</li>\n",
       "\t<li>'Japanese quail chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Japanese quail chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Japanese quail'</li>\n",
       "\t<li>'Japanese quail homology type'</li>\n",
       "\t<li>'%id. target Japanese quail gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Japanese quail gene'</li>\n",
       "\t<li>'Japanese quail Gene-order conservation score'</li>\n",
       "\t<li>'Japanese quail Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Japanese quail'</li>\n",
       "\t<li>'dS with Japanese quail'</li>\n",
       "\t<li>'Japanese quail orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Kangaroo rat gene stable ID'</li>\n",
       "\t<li>'Kangaroo rat gene name'</li>\n",
       "\t<li>'Kangaroo rat protein or transcript stable ID'</li>\n",
       "\t<li>'Kangaroo rat chromosome/scaffold name'</li>\n",
       "\t<li>'Kangaroo rat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Kangaroo rat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Kangaroo rat'</li>\n",
       "\t<li>'Kangaroo rat homology type'</li>\n",
       "\t<li>'%id. target Kangaroo rat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Kangaroo rat gene'</li>\n",
       "\t<li>'Kangaroo rat Gene-order conservation score'</li>\n",
       "\t<li>'Kangaroo rat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Kangaroo rat'</li>\n",
       "\t<li>'dS with Kangaroo rat'</li>\n",
       "\t<li>'Kangaroo rat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Koala gene stable ID'</li>\n",
       "\t<li>'Koala gene name'</li>\n",
       "\t<li>'Koala protein or transcript stable ID'</li>\n",
       "\t<li>'Koala chromosome/scaffold name'</li>\n",
       "\t<li>'Koala chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Koala chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Koala'</li>\n",
       "\t<li>'Koala homology type'</li>\n",
       "\t<li>'%id. target Koala gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Koala gene'</li>\n",
       "\t<li>'Koala Gene-order conservation score'</li>\n",
       "\t<li>'Koala Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Koala'</li>\n",
       "\t<li>'dS with Koala'</li>\n",
       "\t<li>'Koala orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Lamprey gene stable ID'</li>\n",
       "\t<li>'Lamprey gene name'</li>\n",
       "\t<li>'Lamprey protein or transcript stable ID'</li>\n",
       "\t<li>'Lamprey chromosome/scaffold name'</li>\n",
       "\t<li>'Lamprey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Lamprey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Lamprey'</li>\n",
       "\t<li>'Lamprey homology type'</li>\n",
       "\t<li>'%id. target Lamprey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Lamprey gene'</li>\n",
       "\t<li>'Lamprey Gene-order conservation score'</li>\n",
       "\t<li>'Lamprey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Lamprey'</li>\n",
       "\t<li>'dS with Lamprey'</li>\n",
       "\t<li>'Lamprey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Leopard gene stable ID'</li>\n",
       "\t<li>'Leopard gene name'</li>\n",
       "\t<li>'Leopard protein or transcript stable ID'</li>\n",
       "\t<li>'Leopard chromosome/scaffold name'</li>\n",
       "\t<li>'Leopard chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Leopard chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Leopard'</li>\n",
       "\t<li>'Leopard homology type'</li>\n",
       "\t<li>'%id. target Leopard gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Leopard gene'</li>\n",
       "\t<li>'Leopard Gene-order conservation score'</li>\n",
       "\t<li>'Leopard Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Leopard'</li>\n",
       "\t<li>'dS with Leopard'</li>\n",
       "\t<li>'Leopard orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Lesser Egyptian jerboa gene stable ID'</li>\n",
       "\t<li>'Lesser Egyptian jerboa gene name'</li>\n",
       "\t<li>'Lesser Egyptian jerboa protein or transcript stable ID'</li>\n",
       "\t<li>'Lesser Egyptian jerboa chromosome/scaffold name'</li>\n",
       "\t<li>'Lesser Egyptian jerboa chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Lesser Egyptian jerboa chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Lesser Egyptian jerboa'</li>\n",
       "\t<li>'Lesser Egyptian jerboa homology type'</li>\n",
       "\t<li>'%id. target Lesser Egyptian jerboa gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Lesser Egyptian jerboa gene'</li>\n",
       "\t<li>'Lesser Egyptian jerboa Gene-order conservation score'</li>\n",
       "\t<li>'Lesser Egyptian jerboa Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Lesser Egyptian jerboa'</li>\n",
       "\t<li>'dS with Lesser Egyptian jerboa'</li>\n",
       "\t<li>'Lesser Egyptian jerboa orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Lesser hedgehog tenrec gene stable ID'</li>\n",
       "\t<li>'Lesser hedgehog tenrec gene name'</li>\n",
       "\t<li>'Lesser hedgehog tenrec protein or transcript stable ID'</li>\n",
       "\t<li>'Lesser hedgehog tenrec chromosome/scaffold name'</li>\n",
       "\t<li>'Lesser hedgehog tenrec chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Lesser hedgehog tenrec chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Lesser hedgehog tenrec'</li>\n",
       "\t<li>'Lesser hedgehog tenrec homology type'</li>\n",
       "\t<li>'%id. target Lesser hedgehog tenrec gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Lesser hedgehog tenrec gene'</li>\n",
       "\t<li>'Lesser hedgehog tenrec Gene-order conservation score'</li>\n",
       "\t<li>'Lesser hedgehog tenrec Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Lesser hedgehog tenrec'</li>\n",
       "\t<li>'dS with Lesser hedgehog tenrec'</li>\n",
       "\t<li>'Lesser hedgehog tenrec orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Little spotted kiwi gene stable ID'</li>\n",
       "\t<li>'Little spotted kiwi gene name'</li>\n",
       "\t<li>'Little spotted kiwi protein or transcript stable ID'</li>\n",
       "\t<li>'Little spotted kiwi chromosome/scaffold name'</li>\n",
       "\t<li>'Little spotted kiwi chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Little spotted kiwi chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Little spotted kiwi'</li>\n",
       "\t<li>'Little spotted kiwi homology type'</li>\n",
       "\t<li>'%id. target Little spotted kiwi gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Little spotted kiwi gene'</li>\n",
       "\t<li>'Little spotted kiwi Gene-order conservation score'</li>\n",
       "\t<li>'Little spotted kiwi Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Little spotted kiwi'</li>\n",
       "\t<li>'dS with Little spotted kiwi'</li>\n",
       "\t<li>'Little spotted kiwi orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Long-tailed chinchilla gene stable ID'</li>\n",
       "\t<li>'Long-tailed chinchilla gene name'</li>\n",
       "\t<li>'Long-tailed chinchilla protein or transcript stable ID'</li>\n",
       "\t<li>'Long-tailed chinchilla chromosome/scaffold name'</li>\n",
       "\t<li>'Long-tailed chinchilla chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Long-tailed chinchilla chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Long-tailed chinchilla'</li>\n",
       "\t<li>'Long-tailed chinchilla homology type'</li>\n",
       "\t<li>'%id. target Long-tailed chinchilla gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Long-tailed chinchilla gene'</li>\n",
       "\t<li>'Long-tailed chinchilla Gene-order conservation score'</li>\n",
       "\t<li>'Long-tailed chinchilla Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Long-tailed chinchilla'</li>\n",
       "\t<li>'dS with Long-tailed chinchilla'</li>\n",
       "\t<li>'Long-tailed chinchilla orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Lyretail cichlid gene stable ID'</li>\n",
       "\t<li>'Lyretail cichlid gene name'</li>\n",
       "\t<li>'Lyretail cichlid protein or transcript stable ID'</li>\n",
       "\t<li>'Lyretail cichlid chromosome/scaffold name'</li>\n",
       "\t<li>'Lyretail cichlid chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Lyretail cichlid chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Lyretail cichlid'</li>\n",
       "\t<li>'Lyretail cichlid homology type'</li>\n",
       "\t<li>'%id. target Lyretail cichlid gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Lyretail cichlid gene'</li>\n",
       "\t<li>'Lyretail cichlid Gene-order conservation score'</li>\n",
       "\t<li>'Lyretail cichlid Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Lyretail cichlid'</li>\n",
       "\t<li>'dS with Lyretail cichlid'</li>\n",
       "\t<li>'Lyretail cichlid orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ma\\'s night monkey gene stable ID'</li>\n",
       "\t<li>'Ma\\'s night monkey gene name'</li>\n",
       "\t<li>'Ma\\'s night monkey protein or transcript stable ID'</li>\n",
       "\t<li>'Ma\\'s night monkey chromosome/scaffold name'</li>\n",
       "\t<li>'Ma\\'s night monkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ma\\'s night monkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ma\\'s night monkey'</li>\n",
       "\t<li>'Ma\\'s night monkey homology type'</li>\n",
       "\t<li>'%id. target Ma\\'s night monkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ma\\'s night monkey gene'</li>\n",
       "\t<li>'Ma\\'s night monkey Gene-order conservation score'</li>\n",
       "\t<li>'Ma\\'s night monkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ma\\'s night monkey'</li>\n",
       "\t<li>'dS with Ma\\'s night monkey'</li>\n",
       "\t<li>'Ma\\'s night monkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Macaque gene stable ID'</li>\n",
       "\t<li>'Macaque gene name'</li>\n",
       "\t<li>'Macaque protein or transcript stable ID'</li>\n",
       "\t<li>'Macaque chromosome/scaffold name'</li>\n",
       "\t<li>'Macaque chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Macaque chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Macaque'</li>\n",
       "\t<li>'Macaque homology type'</li>\n",
       "\t<li>'%id. target Macaque gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Macaque gene'</li>\n",
       "\t<li>'Macaque Gene-order conservation score'</li>\n",
       "\t<li>'Macaque Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Macaque'</li>\n",
       "\t<li>'dS with Macaque'</li>\n",
       "\t<li>'Macaque orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mainland tiger snake gene stable ID'</li>\n",
       "\t<li>'Mainland tiger snake gene name'</li>\n",
       "\t<li>'Mainland tiger snake protein or transcript stable ID'</li>\n",
       "\t<li>'Mainland tiger snake chromosome/scaffold name'</li>\n",
       "\t<li>'Mainland tiger snake chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mainland tiger snake chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mainland tiger snake'</li>\n",
       "\t<li>'Mainland tiger snake homology type'</li>\n",
       "\t<li>'%id. target Mainland tiger snake gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mainland tiger snake gene'</li>\n",
       "\t<li>'Mainland tiger snake Gene-order conservation score'</li>\n",
       "\t<li>'Mainland tiger snake Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mainland tiger snake'</li>\n",
       "\t<li>'dS with Mainland tiger snake'</li>\n",
       "\t<li>'Mainland tiger snake orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Makobe Island cichlid gene stable ID'</li>\n",
       "\t<li>'Makobe Island cichlid gene name'</li>\n",
       "\t<li>'Makobe Island cichlid protein or transcript stable ID'</li>\n",
       "\t<li>'Makobe Island cichlid chromosome/scaffold name'</li>\n",
       "\t<li>'Makobe Island cichlid chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Makobe Island cichlid chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Makobe Island cichlid'</li>\n",
       "\t<li>'Makobe Island cichlid homology type'</li>\n",
       "\t<li>'%id. target Makobe Island cichlid gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Makobe Island cichlid gene'</li>\n",
       "\t<li>'Makobe Island cichlid Gene-order conservation score'</li>\n",
       "\t<li>'Makobe Island cichlid Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Makobe Island cichlid'</li>\n",
       "\t<li>'dS with Makobe Island cichlid'</li>\n",
       "\t<li>'Makobe Island cichlid orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mangrove rivulus gene stable ID'</li>\n",
       "\t<li>'Mangrove rivulus gene name'</li>\n",
       "\t<li>'Mangrove rivulus protein or transcript stable ID'</li>\n",
       "\t<li>'Mangrove rivulus chromosome/scaffold name'</li>\n",
       "\t<li>'Mangrove rivulus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mangrove rivulus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mangrove rivulus'</li>\n",
       "\t<li>'Mangrove rivulus homology type'</li>\n",
       "\t<li>'%id. target Mangrove rivulus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mangrove rivulus gene'</li>\n",
       "\t<li>'Mangrove rivulus Gene-order conservation score'</li>\n",
       "\t<li>'Mangrove rivulus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mangrove rivulus'</li>\n",
       "\t<li>'dS with Mangrove rivulus'</li>\n",
       "\t<li>'Mangrove rivulus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Marmoset gene stable ID'</li>\n",
       "\t<li>'Marmoset gene name'</li>\n",
       "\t<li>'Marmoset protein or transcript stable ID'</li>\n",
       "\t<li>'Marmoset chromosome/scaffold name'</li>\n",
       "\t<li>'Marmoset chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Marmoset chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Marmoset'</li>\n",
       "\t<li>'Marmoset homology type'</li>\n",
       "\t<li>'%id. target Marmoset gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Marmoset gene'</li>\n",
       "\t<li>'Marmoset Gene-order conservation score'</li>\n",
       "\t<li>'Marmoset Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Marmoset'</li>\n",
       "\t<li>'dS with Marmoset'</li>\n",
       "\t<li>'Marmoset orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Megabat gene stable ID'</li>\n",
       "\t<li>'Megabat gene name'</li>\n",
       "\t<li>'Megabat protein or transcript stable ID'</li>\n",
       "\t<li>'Megabat chromosome/scaffold name'</li>\n",
       "\t<li>'Megabat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Megabat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Megabat'</li>\n",
       "\t<li>'Megabat homology type'</li>\n",
       "\t<li>'%id. target Megabat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Megabat gene'</li>\n",
       "\t<li>'Megabat Gene-order conservation score'</li>\n",
       "\t<li>'Megabat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Megabat'</li>\n",
       "\t<li>'dS with Megabat'</li>\n",
       "\t<li>'Megabat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mexican tetra gene stable ID'</li>\n",
       "\t<li>'Mexican tetra gene name'</li>\n",
       "\t<li>'Mexican tetra protein or transcript stable ID'</li>\n",
       "\t<li>'Mexican tetra chromosome/scaffold name'</li>\n",
       "\t<li>'Mexican tetra chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mexican tetra chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mexican tetra'</li>\n",
       "\t<li>'Mexican tetra homology type'</li>\n",
       "\t<li>'%id. target Mexican tetra gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mexican tetra gene'</li>\n",
       "\t<li>'Mexican tetra Gene-order conservation score'</li>\n",
       "\t<li>'Mexican tetra Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mexican tetra'</li>\n",
       "\t<li>'dS with Mexican tetra'</li>\n",
       "\t<li>'Mexican tetra orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Microbat gene stable ID'</li>\n",
       "\t<li>'Microbat gene name'</li>\n",
       "\t<li>'Microbat protein or transcript stable ID'</li>\n",
       "\t<li>'Microbat chromosome/scaffold name'</li>\n",
       "\t<li>'Microbat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Microbat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Microbat'</li>\n",
       "\t<li>'Microbat homology type'</li>\n",
       "\t<li>'%id. target Microbat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Microbat gene'</li>\n",
       "\t<li>'Microbat Gene-order conservation score'</li>\n",
       "\t<li>'Microbat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Microbat'</li>\n",
       "\t<li>'dS with Microbat'</li>\n",
       "\t<li>'Microbat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Midas cichlid gene stable ID'</li>\n",
       "\t<li>'Midas cichlid gene name'</li>\n",
       "\t<li>'Midas cichlid protein or transcript stable ID'</li>\n",
       "\t<li>'Midas cichlid chromosome/scaffold name'</li>\n",
       "\t<li>'Midas cichlid chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Midas cichlid chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Midas cichlid'</li>\n",
       "\t<li>'Midas cichlid homology type'</li>\n",
       "\t<li>'%id. target Midas cichlid gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Midas cichlid gene'</li>\n",
       "\t<li>'Midas cichlid Gene-order conservation score'</li>\n",
       "\t<li>'Midas cichlid Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Midas cichlid'</li>\n",
       "\t<li>'dS with Midas cichlid'</li>\n",
       "\t<li>'Midas cichlid orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mongolian gerbil gene stable ID'</li>\n",
       "\t<li>'Mongolian gerbil gene name'</li>\n",
       "\t<li>'Mongolian gerbil protein or transcript stable ID'</li>\n",
       "\t<li>'Mongolian gerbil chromosome/scaffold name'</li>\n",
       "\t<li>'Mongolian gerbil chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mongolian gerbil chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mongolian gerbil'</li>\n",
       "\t<li>'Mongolian gerbil homology type'</li>\n",
       "\t<li>'%id. target Mongolian gerbil gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mongolian gerbil gene'</li>\n",
       "\t<li>'Mongolian gerbil Gene-order conservation score'</li>\n",
       "\t<li>'Mongolian gerbil Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mongolian gerbil'</li>\n",
       "\t<li>'dS with Mongolian gerbil'</li>\n",
       "\t<li>'Mongolian gerbil orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Monterrey platyfish gene stable ID'</li>\n",
       "\t<li>'Monterrey platyfish gene name'</li>\n",
       "\t<li>'Monterrey platyfish protein or transcript stable ID'</li>\n",
       "\t<li>'Monterrey platyfish chromosome/scaffold name'</li>\n",
       "\t<li>'Monterrey platyfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Monterrey platyfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Monterrey platyfish'</li>\n",
       "\t<li>'Monterrey platyfish homology type'</li>\n",
       "\t<li>'%id. target Monterrey platyfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Monterrey platyfish gene'</li>\n",
       "\t<li>'Monterrey platyfish Gene-order conservation score'</li>\n",
       "\t<li>'Monterrey platyfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Monterrey platyfish'</li>\n",
       "\t<li>'dS with Monterrey platyfish'</li>\n",
       "\t<li>'Monterrey platyfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mouse gene stable ID'</li>\n",
       "\t<li>'Mouse gene name'</li>\n",
       "\t<li>'Mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mouse'</li>\n",
       "\t<li>'Mouse homology type'</li>\n",
       "\t<li>'%id. target Mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mouse gene'</li>\n",
       "\t<li>'Mouse Gene-order conservation score'</li>\n",
       "\t<li>'Mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mouse'</li>\n",
       "\t<li>'dS with Mouse'</li>\n",
       "\t<li>'Mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mouse Lemur gene stable ID'</li>\n",
       "\t<li>'Mouse Lemur gene name'</li>\n",
       "\t<li>'Mouse Lemur protein or transcript stable ID'</li>\n",
       "\t<li>'Mouse Lemur chromosome/scaffold name'</li>\n",
       "\t<li>'Mouse Lemur chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mouse Lemur chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mouse Lemur'</li>\n",
       "\t<li>'Mouse Lemur homology type'</li>\n",
       "\t<li>'%id. target Mouse Lemur gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mouse Lemur gene'</li>\n",
       "\t<li>'Mouse Lemur Gene-order conservation score'</li>\n",
       "\t<li>'Mouse Lemur Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mouse Lemur'</li>\n",
       "\t<li>'dS with Mouse Lemur'</li>\n",
       "\t<li>'Mouse Lemur orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Mummichog gene stable ID'</li>\n",
       "\t<li>'Mummichog gene name'</li>\n",
       "\t<li>'Mummichog protein or transcript stable ID'</li>\n",
       "\t<li>'Mummichog chromosome/scaffold name'</li>\n",
       "\t<li>'Mummichog chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Mummichog chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Mummichog'</li>\n",
       "\t<li>'Mummichog homology type'</li>\n",
       "\t<li>'%id. target Mummichog gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Mummichog gene'</li>\n",
       "\t<li>'Mummichog Gene-order conservation score'</li>\n",
       "\t<li>'Mummichog Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Mummichog'</li>\n",
       "\t<li>'dS with Mummichog'</li>\n",
       "\t<li>'Mummichog orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Naked mole-rat female gene stable ID'</li>\n",
       "\t<li>'Naked mole-rat female gene name'</li>\n",
       "\t<li>'Naked mole-rat female protein or transcript stable ID'</li>\n",
       "\t<li>'Naked mole-rat female chromosome/scaffold name'</li>\n",
       "\t<li>'Naked mole-rat female chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Naked mole-rat female chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Naked mole-rat female'</li>\n",
       "\t<li>'Naked mole-rat female homology type'</li>\n",
       "\t<li>'%id. target Naked mole-rat female gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Naked mole-rat female gene'</li>\n",
       "\t<li>'Naked mole-rat female Gene-order conservation score'</li>\n",
       "\t<li>'Naked mole-rat female Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Naked mole-rat female'</li>\n",
       "\t<li>'dS with Naked mole-rat female'</li>\n",
       "\t<li>'Naked mole-rat female orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Naked mole-rat male gene stable ID'</li>\n",
       "\t<li>'Naked mole-rat male gene name'</li>\n",
       "\t<li>'Naked mole-rat male protein or transcript stable ID'</li>\n",
       "\t<li>'Naked mole-rat male chromosome/scaffold name'</li>\n",
       "\t<li>'Naked mole-rat male chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Naked mole-rat male chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Naked mole-rat male'</li>\n",
       "\t<li>'Naked mole-rat male homology type'</li>\n",
       "\t<li>'%id. target Naked mole-rat male gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Naked mole-rat male gene'</li>\n",
       "\t<li>'Naked mole-rat male Gene-order conservation score'</li>\n",
       "\t<li>'Naked mole-rat male Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Naked mole-rat male'</li>\n",
       "\t<li>'dS with Naked mole-rat male'</li>\n",
       "\t<li>'Naked mole-rat male orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Northern American deer mouse gene stable ID'</li>\n",
       "\t<li>'Northern American deer mouse gene name'</li>\n",
       "\t<li>'Northern American deer mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Northern American deer mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Northern American deer mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Northern American deer mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Northern American deer mouse'</li>\n",
       "\t<li>'Northern American deer mouse homology type'</li>\n",
       "\t<li>'%id. target Northern American deer mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Northern American deer mouse gene'</li>\n",
       "\t<li>'Northern American deer mouse Gene-order conservation score'</li>\n",
       "\t<li>'Northern American deer mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Northern American deer mouse'</li>\n",
       "\t<li>'dS with Northern American deer mouse'</li>\n",
       "\t<li>'Northern American deer mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Northern pike gene stable ID'</li>\n",
       "\t<li>'Northern pike gene name'</li>\n",
       "\t<li>'Northern pike protein or transcript stable ID'</li>\n",
       "\t<li>'Northern pike chromosome/scaffold name'</li>\n",
       "\t<li>'Northern pike chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Northern pike chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Northern pike'</li>\n",
       "\t<li>'Northern pike homology type'</li>\n",
       "\t<li>'%id. target Northern pike gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Northern pike gene'</li>\n",
       "\t<li>'Northern pike Gene-order conservation score'</li>\n",
       "\t<li>'Northern pike Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Northern pike'</li>\n",
       "\t<li>'dS with Northern pike'</li>\n",
       "\t<li>'Northern pike orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ocean sunfish gene stable ID'</li>\n",
       "\t<li>'Ocean sunfish gene name'</li>\n",
       "\t<li>'Ocean sunfish protein or transcript stable ID'</li>\n",
       "\t<li>'Ocean sunfish chromosome/scaffold name'</li>\n",
       "\t<li>'Ocean sunfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ocean sunfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ocean sunfish'</li>\n",
       "\t<li>'Ocean sunfish homology type'</li>\n",
       "\t<li>'%id. target Ocean sunfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ocean sunfish gene'</li>\n",
       "\t<li>'Ocean sunfish Gene-order conservation score'</li>\n",
       "\t<li>'Ocean sunfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ocean sunfish'</li>\n",
       "\t<li>'dS with Ocean sunfish'</li>\n",
       "\t<li>'Ocean sunfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Okarito brown kiwi gene stable ID'</li>\n",
       "\t<li>'Okarito brown kiwi gene name'</li>\n",
       "\t<li>'Okarito brown kiwi protein or transcript stable ID'</li>\n",
       "\t<li>'Okarito brown kiwi chromosome/scaffold name'</li>\n",
       "\t<li>'Okarito brown kiwi chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Okarito brown kiwi chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Okarito brown kiwi'</li>\n",
       "\t<li>'Okarito brown kiwi homology type'</li>\n",
       "\t<li>'%id. target Okarito brown kiwi gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Okarito brown kiwi gene'</li>\n",
       "\t<li>'Okarito brown kiwi Gene-order conservation score'</li>\n",
       "\t<li>'Okarito brown kiwi Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Okarito brown kiwi'</li>\n",
       "\t<li>'dS with Okarito brown kiwi'</li>\n",
       "\t<li>'Okarito brown kiwi orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Olive baboon gene stable ID'</li>\n",
       "\t<li>'Olive baboon gene name'</li>\n",
       "\t<li>'Olive baboon protein or transcript stable ID'</li>\n",
       "\t<li>'Olive baboon chromosome/scaffold name'</li>\n",
       "\t<li>'Olive baboon chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Olive baboon chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Olive baboon'</li>\n",
       "\t<li>'Olive baboon homology type'</li>\n",
       "\t<li>'%id. target Olive baboon gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Olive baboon gene'</li>\n",
       "\t<li>'Olive baboon Gene-order conservation score'</li>\n",
       "\t<li>'Olive baboon Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Olive baboon'</li>\n",
       "\t<li>'dS with Olive baboon'</li>\n",
       "\t<li>'Olive baboon orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Opossum gene stable ID'</li>\n",
       "\t<li>'Opossum gene name'</li>\n",
       "\t<li>'Opossum protein or transcript stable ID'</li>\n",
       "\t<li>'Opossum chromosome/scaffold name'</li>\n",
       "\t<li>'Opossum chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Opossum chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Opossum'</li>\n",
       "\t<li>'Opossum homology type'</li>\n",
       "\t<li>'%id. target Opossum gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Opossum gene'</li>\n",
       "\t<li>'Opossum Gene-order conservation score'</li>\n",
       "\t<li>'Opossum Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Opossum'</li>\n",
       "\t<li>'dS with Opossum'</li>\n",
       "\t<li>'Opossum orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Orange clownfish gene stable ID'</li>\n",
       "\t<li>'Orange clownfish gene name'</li>\n",
       "\t<li>'Orange clownfish protein or transcript stable ID'</li>\n",
       "\t<li>'Orange clownfish chromosome/scaffold name'</li>\n",
       "\t<li>'Orange clownfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Orange clownfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Orange clownfish'</li>\n",
       "\t<li>'Orange clownfish homology type'</li>\n",
       "\t<li>'%id. target Orange clownfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Orange clownfish gene'</li>\n",
       "\t<li>'Orange clownfish Gene-order conservation score'</li>\n",
       "\t<li>'Orange clownfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Orange clownfish'</li>\n",
       "\t<li>'dS with Orange clownfish'</li>\n",
       "\t<li>'Orange clownfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Orangutan gene stable ID'</li>\n",
       "\t<li>'Orangutan gene name'</li>\n",
       "\t<li>'Orangutan protein or transcript stable ID'</li>\n",
       "\t<li>'Orangutan chromosome/scaffold name'</li>\n",
       "\t<li>'Orangutan chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Orangutan chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Orangutan'</li>\n",
       "\t<li>'Orangutan homology type'</li>\n",
       "\t<li>'%id. target Orangutan gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Orangutan gene'</li>\n",
       "\t<li>'Orangutan Gene-order conservation score'</li>\n",
       "\t<li>'Orangutan Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Orangutan'</li>\n",
       "\t<li>'dS with Orangutan'</li>\n",
       "\t<li>'Orangutan orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Painted turtle gene stable ID'</li>\n",
       "\t<li>'Painted turtle gene name'</li>\n",
       "\t<li>'Painted turtle protein or transcript stable ID'</li>\n",
       "\t<li>'Painted turtle chromosome/scaffold name'</li>\n",
       "\t<li>'Painted turtle chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Painted turtle chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Painted turtle'</li>\n",
       "\t<li>'Painted turtle homology type'</li>\n",
       "\t<li>'%id. target Painted turtle gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Painted turtle gene'</li>\n",
       "\t<li>'Painted turtle Gene-order conservation score'</li>\n",
       "\t<li>'Painted turtle Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Painted turtle'</li>\n",
       "\t<li>'dS with Painted turtle'</li>\n",
       "\t<li>'Painted turtle orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Panda gene stable ID'</li>\n",
       "\t<li>'Panda gene name'</li>\n",
       "\t<li>'Panda protein or transcript stable ID'</li>\n",
       "\t<li>'Panda chromosome/scaffold name'</li>\n",
       "\t<li>'Panda chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Panda chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Panda'</li>\n",
       "\t<li>'Panda homology type'</li>\n",
       "\t<li>'%id. target Panda gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Panda gene'</li>\n",
       "\t<li>'Panda Gene-order conservation score'</li>\n",
       "\t<li>'Panda Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Panda'</li>\n",
       "\t<li>'dS with Panda'</li>\n",
       "\t<li>'Panda orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Paramormyrops kingsleyae gene stable ID'</li>\n",
       "\t<li>'Paramormyrops kingsleyae gene name'</li>\n",
       "\t<li>'Paramormyrops kingsleyae protein or transcript stable ID'</li>\n",
       "\t<li>'Paramormyrops kingsleyae chromosome/scaffold name'</li>\n",
       "\t<li>'Paramormyrops kingsleyae chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Paramormyrops kingsleyae chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Paramormyrops kingsleyae'</li>\n",
       "\t<li>'Paramormyrops kingsleyae homology type'</li>\n",
       "\t<li>'%id. target Paramormyrops kingsleyae gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Paramormyrops kingsleyae gene'</li>\n",
       "\t<li>'Paramormyrops kingsleyae Gene-order conservation score'</li>\n",
       "\t<li>'Paramormyrops kingsleyae Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Paramormyrops kingsleyae'</li>\n",
       "\t<li>'dS with Paramormyrops kingsleyae'</li>\n",
       "\t<li>'Paramormyrops kingsleyae orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus gene stable ID'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus gene name'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus protein or transcript stable ID'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus chromosome/scaffold name'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Periophthalmus magnuspinnatus'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus homology type'</li>\n",
       "\t<li>'%id. target Periophthalmus magnuspinnatus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Periophthalmus magnuspinnatus gene'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus Gene-order conservation score'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Periophthalmus magnuspinnatus'</li>\n",
       "\t<li>'dS with Periophthalmus magnuspinnatus'</li>\n",
       "\t<li>'Periophthalmus magnuspinnatus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Pig gene stable ID'</li>\n",
       "\t<li>'Pig gene name'</li>\n",
       "\t<li>'Pig protein or transcript stable ID'</li>\n",
       "\t<li>'Pig chromosome/scaffold name'</li>\n",
       "\t<li>'Pig chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Pig chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Pig'</li>\n",
       "\t<li>'Pig homology type'</li>\n",
       "\t<li>'%id. target Pig gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Pig gene'</li>\n",
       "\t<li>'Pig Gene-order conservation score'</li>\n",
       "\t<li>'Pig Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Pig'</li>\n",
       "\t<li>'dS with Pig'</li>\n",
       "\t<li>'Pig orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Pig USMARC gene stable ID'</li>\n",
       "\t<li>'Pig USMARC gene name'</li>\n",
       "\t<li>'Pig USMARC protein or transcript stable ID'</li>\n",
       "\t<li>'Pig USMARC chromosome/scaffold name'</li>\n",
       "\t<li>'Pig USMARC chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Pig USMARC chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Pig USMARC'</li>\n",
       "\t<li>'Pig USMARC homology type'</li>\n",
       "\t<li>'%id. target Pig USMARC gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Pig USMARC gene'</li>\n",
       "\t<li>'Pig USMARC Gene-order conservation score'</li>\n",
       "\t<li>'Pig USMARC Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Pig USMARC'</li>\n",
       "\t<li>'dS with Pig USMARC'</li>\n",
       "\t<li>'Pig USMARC orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Pig-tailed macaque gene stable ID'</li>\n",
       "\t<li>'Pig-tailed macaque gene name'</li>\n",
       "\t<li>'Pig-tailed macaque protein or transcript stable ID'</li>\n",
       "\t<li>'Pig-tailed macaque chromosome/scaffold name'</li>\n",
       "\t<li>'Pig-tailed macaque chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Pig-tailed macaque chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Pig-tailed macaque'</li>\n",
       "\t<li>'Pig-tailed macaque homology type'</li>\n",
       "\t<li>'%id. target Pig-tailed macaque gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Pig-tailed macaque gene'</li>\n",
       "\t<li>'Pig-tailed macaque Gene-order conservation score'</li>\n",
       "\t<li>'Pig-tailed macaque Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Pig-tailed macaque'</li>\n",
       "\t<li>'dS with Pig-tailed macaque'</li>\n",
       "\t<li>'Pig-tailed macaque orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Pika gene stable ID'</li>\n",
       "\t<li>'Pika gene name'</li>\n",
       "\t<li>'Pika protein or transcript stable ID'</li>\n",
       "\t<li>'Pika chromosome/scaffold name'</li>\n",
       "\t<li>'Pika chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Pika chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Pika'</li>\n",
       "\t<li>'Pika homology type'</li>\n",
       "\t<li>'%id. target Pika gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Pika gene'</li>\n",
       "\t<li>'Pika Gene-order conservation score'</li>\n",
       "\t<li>'Pika Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Pika'</li>\n",
       "\t<li>'dS with Pika'</li>\n",
       "\t<li>'Pika orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Pink-footed goose gene stable ID'</li>\n",
       "\t<li>'Pink-footed goose gene name'</li>\n",
       "\t<li>'Pink-footed goose protein or transcript stable ID'</li>\n",
       "\t<li>'Pink-footed goose chromosome/scaffold name'</li>\n",
       "\t<li>'Pink-footed goose chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Pink-footed goose chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Pink-footed goose'</li>\n",
       "\t<li>'Pink-footed goose homology type'</li>\n",
       "\t<li>'%id. target Pink-footed goose gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Pink-footed goose gene'</li>\n",
       "\t<li>'Pink-footed goose Gene-order conservation score'</li>\n",
       "\t<li>'Pink-footed goose Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Pink-footed goose'</li>\n",
       "\t<li>'dS with Pink-footed goose'</li>\n",
       "\t<li>'Pink-footed goose orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Platyfish gene stable ID'</li>\n",
       "\t<li>'Platyfish gene name'</li>\n",
       "\t<li>'Platyfish protein or transcript stable ID'</li>\n",
       "\t<li>'Platyfish chromosome/scaffold name'</li>\n",
       "\t<li>'Platyfish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Platyfish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Platyfish'</li>\n",
       "\t<li>'Platyfish homology type'</li>\n",
       "\t<li>'%id. target Platyfish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Platyfish gene'</li>\n",
       "\t<li>'Platyfish Gene-order conservation score'</li>\n",
       "\t<li>'Platyfish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Platyfish'</li>\n",
       "\t<li>'dS with Platyfish'</li>\n",
       "\t<li>'Platyfish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Platypus gene stable ID'</li>\n",
       "\t<li>'Platypus gene name'</li>\n",
       "\t<li>'Platypus protein or transcript stable ID'</li>\n",
       "\t<li>'Platypus chromosome/scaffold name'</li>\n",
       "\t<li>'Platypus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Platypus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Platypus'</li>\n",
       "\t<li>'Platypus homology type'</li>\n",
       "\t<li>'%id. target Platypus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Platypus gene'</li>\n",
       "\t<li>'Platypus Gene-order conservation score'</li>\n",
       "\t<li>'Platypus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Platypus'</li>\n",
       "\t<li>'dS with Platypus'</li>\n",
       "\t<li>'Platypus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Polar bear gene stable ID'</li>\n",
       "\t<li>'Polar bear gene name'</li>\n",
       "\t<li>'Polar bear protein or transcript stable ID'</li>\n",
       "\t<li>'Polar bear chromosome/scaffold name'</li>\n",
       "\t<li>'Polar bear chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Polar bear chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Polar bear'</li>\n",
       "\t<li>'Polar bear homology type'</li>\n",
       "\t<li>'%id. target Polar bear gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Polar bear gene'</li>\n",
       "\t<li>'Polar bear Gene-order conservation score'</li>\n",
       "\t<li>'Polar bear Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Polar bear'</li>\n",
       "\t<li>'dS with Polar bear'</li>\n",
       "\t<li>'Polar bear orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Prairie vole gene stable ID'</li>\n",
       "\t<li>'Prairie vole gene name'</li>\n",
       "\t<li>'Prairie vole protein or transcript stable ID'</li>\n",
       "\t<li>'Prairie vole chromosome/scaffold name'</li>\n",
       "\t<li>'Prairie vole chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Prairie vole chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Prairie vole'</li>\n",
       "\t<li>'Prairie vole homology type'</li>\n",
       "\t<li>'%id. target Prairie vole gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Prairie vole gene'</li>\n",
       "\t<li>'Prairie vole Gene-order conservation score'</li>\n",
       "\t<li>'Prairie vole Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Prairie vole'</li>\n",
       "\t<li>'dS with Prairie vole'</li>\n",
       "\t<li>'Prairie vole orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Rabbit gene stable ID'</li>\n",
       "\t<li>'Rabbit gene name'</li>\n",
       "\t<li>'Rabbit protein or transcript stable ID'</li>\n",
       "\t<li>'Rabbit chromosome/scaffold name'</li>\n",
       "\t<li>'Rabbit chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Rabbit chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Rabbit'</li>\n",
       "\t<li>'Rabbit homology type'</li>\n",
       "\t<li>'%id. target Rabbit gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Rabbit gene'</li>\n",
       "\t<li>'Rabbit Gene-order conservation score'</li>\n",
       "\t<li>'Rabbit Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Rabbit'</li>\n",
       "\t<li>'dS with Rabbit'</li>\n",
       "\t<li>'Rabbit orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Rat gene stable ID'</li>\n",
       "\t<li>'Rat gene name'</li>\n",
       "\t<li>'Rat protein or transcript stable ID'</li>\n",
       "\t<li>'Rat chromosome/scaffold name'</li>\n",
       "\t<li>'Rat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Rat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Rat'</li>\n",
       "\t<li>'Rat homology type'</li>\n",
       "\t<li>'%id. target Rat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Rat gene'</li>\n",
       "\t<li>'Rat Gene-order conservation score'</li>\n",
       "\t<li>'Rat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Rat'</li>\n",
       "\t<li>'dS with Rat'</li>\n",
       "\t<li>'Rat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Red fox gene stable ID'</li>\n",
       "\t<li>'Red fox gene name'</li>\n",
       "\t<li>'Red fox protein or transcript stable ID'</li>\n",
       "\t<li>'Red fox chromosome/scaffold name'</li>\n",
       "\t<li>'Red fox chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Red fox chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Red fox'</li>\n",
       "\t<li>'Red fox homology type'</li>\n",
       "\t<li>'%id. target Red fox gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Red fox gene'</li>\n",
       "\t<li>'Red fox Gene-order conservation score'</li>\n",
       "\t<li>'Red fox Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Red fox'</li>\n",
       "\t<li>'dS with Red fox'</li>\n",
       "\t<li>'Red fox orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Red-bellied piranha gene stable ID'</li>\n",
       "\t<li>'Red-bellied piranha gene name'</li>\n",
       "\t<li>'Red-bellied piranha protein or transcript stable ID'</li>\n",
       "\t<li>'Red-bellied piranha chromosome/scaffold name'</li>\n",
       "\t<li>'Red-bellied piranha chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Red-bellied piranha chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Red-bellied piranha'</li>\n",
       "\t<li>'Red-bellied piranha homology type'</li>\n",
       "\t<li>'%id. target Red-bellied piranha gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Red-bellied piranha gene'</li>\n",
       "\t<li>'Red-bellied piranha Gene-order conservation score'</li>\n",
       "\t<li>'Red-bellied piranha Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Red-bellied piranha'</li>\n",
       "\t<li>'dS with Red-bellied piranha'</li>\n",
       "\t<li>'Red-bellied piranha orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ruff gene stable ID'</li>\n",
       "\t<li>'Ruff gene name'</li>\n",
       "\t<li>'Ruff protein or transcript stable ID'</li>\n",
       "\t<li>'Ruff chromosome/scaffold name'</li>\n",
       "\t<li>'Ruff chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ruff chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ruff'</li>\n",
       "\t<li>'Ruff homology type'</li>\n",
       "\t<li>'%id. target Ruff gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ruff gene'</li>\n",
       "\t<li>'Ruff Gene-order conservation score'</li>\n",
       "\t<li>'Ruff Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ruff'</li>\n",
       "\t<li>'dS with Ruff'</li>\n",
       "\t<li>'Ruff orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ryukyu mouse gene stable ID'</li>\n",
       "\t<li>'Ryukyu mouse gene name'</li>\n",
       "\t<li>'Ryukyu mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Ryukyu mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Ryukyu mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ryukyu mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ryukyu mouse'</li>\n",
       "\t<li>'Ryukyu mouse homology type'</li>\n",
       "\t<li>'%id. target Ryukyu mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ryukyu mouse gene'</li>\n",
       "\t<li>'Ryukyu mouse Gene-order conservation score'</li>\n",
       "\t<li>'Ryukyu mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ryukyu mouse'</li>\n",
       "\t<li>'dS with Ryukyu mouse'</li>\n",
       "\t<li>'Ryukyu mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Saccharomyces cerevisiae gene stable ID'</li>\n",
       "\t<li>'Saccharomyces cerevisiae gene name'</li>\n",
       "\t<li>'Saccharomyces cerevisiae protein or transcript stable ID'</li>\n",
       "\t<li>'Saccharomyces cerevisiae chromosome/scaffold name'</li>\n",
       "\t<li>'Saccharomyces cerevisiae chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Saccharomyces cerevisiae chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Saccharomyces cerevisiae'</li>\n",
       "\t<li>'Saccharomyces cerevisiae homology type'</li>\n",
       "\t<li>'%id. target Saccharomyces cerevisiae gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Saccharomyces cerevisiae gene'</li>\n",
       "\t<li>'Saccharomyces cerevisiae Gene-order conservation score'</li>\n",
       "\t<li>'Saccharomyces cerevisiae Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Saccharomyces cerevisiae'</li>\n",
       "\t<li>'dS with Saccharomyces cerevisiae'</li>\n",
       "\t<li>'Saccharomyces cerevisiae orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Sailfin molly gene stable ID'</li>\n",
       "\t<li>'Sailfin molly gene name'</li>\n",
       "\t<li>'Sailfin molly protein or transcript stable ID'</li>\n",
       "\t<li>'Sailfin molly chromosome/scaffold name'</li>\n",
       "\t<li>'Sailfin molly chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Sailfin molly chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Sailfin molly'</li>\n",
       "\t<li>'Sailfin molly homology type'</li>\n",
       "\t<li>'%id. target Sailfin molly gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Sailfin molly gene'</li>\n",
       "\t<li>'Sailfin molly Gene-order conservation score'</li>\n",
       "\t<li>'Sailfin molly Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Sailfin molly'</li>\n",
       "\t<li>'dS with Sailfin molly'</li>\n",
       "\t<li>'Sailfin molly orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Sheep gene stable ID'</li>\n",
       "\t<li>'Sheep gene name'</li>\n",
       "\t<li>'Sheep protein or transcript stable ID'</li>\n",
       "\t<li>'Sheep chromosome/scaffold name'</li>\n",
       "\t<li>'Sheep chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Sheep chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Sheep'</li>\n",
       "\t<li>'Sheep homology type'</li>\n",
       "\t<li>'%id. target Sheep gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Sheep gene'</li>\n",
       "\t<li>'Sheep Gene-order conservation score'</li>\n",
       "\t<li>'Sheep Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Sheep'</li>\n",
       "\t<li>'dS with Sheep'</li>\n",
       "\t<li>'Sheep orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Sheepshead minnow gene stable ID'</li>\n",
       "\t<li>'Sheepshead minnow gene name'</li>\n",
       "\t<li>'Sheepshead minnow protein or transcript stable ID'</li>\n",
       "\t<li>'Sheepshead minnow chromosome/scaffold name'</li>\n",
       "\t<li>'Sheepshead minnow chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Sheepshead minnow chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Sheepshead minnow'</li>\n",
       "\t<li>'Sheepshead minnow homology type'</li>\n",
       "\t<li>'%id. target Sheepshead minnow gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Sheepshead minnow gene'</li>\n",
       "\t<li>'Sheepshead minnow Gene-order conservation score'</li>\n",
       "\t<li>'Sheepshead minnow Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Sheepshead minnow'</li>\n",
       "\t<li>'dS with Sheepshead minnow'</li>\n",
       "\t<li>'Sheepshead minnow orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Shortfin molly gene stable ID'</li>\n",
       "\t<li>'Shortfin molly gene name'</li>\n",
       "\t<li>'Shortfin molly protein or transcript stable ID'</li>\n",
       "\t<li>'Shortfin molly chromosome/scaffold name'</li>\n",
       "\t<li>'Shortfin molly chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Shortfin molly chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Shortfin molly'</li>\n",
       "\t<li>'Shortfin molly homology type'</li>\n",
       "\t<li>'%id. target Shortfin molly gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Shortfin molly gene'</li>\n",
       "\t<li>'Shortfin molly Gene-order conservation score'</li>\n",
       "\t<li>'Shortfin molly Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Shortfin molly'</li>\n",
       "\t<li>'dS with Shortfin molly'</li>\n",
       "\t<li>'Shortfin molly orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Shrew gene stable ID'</li>\n",
       "\t<li>'Shrew gene name'</li>\n",
       "\t<li>'Shrew protein or transcript stable ID'</li>\n",
       "\t<li>'Shrew chromosome/scaffold name'</li>\n",
       "\t<li>'Shrew chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Shrew chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Shrew'</li>\n",
       "\t<li>'Shrew homology type'</li>\n",
       "\t<li>'%id. target Shrew gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Shrew gene'</li>\n",
       "\t<li>'Shrew Gene-order conservation score'</li>\n",
       "\t<li>'Shrew Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Shrew'</li>\n",
       "\t<li>'dS with Shrew'</li>\n",
       "\t<li>'Shrew orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Shrew mouse gene stable ID'</li>\n",
       "\t<li>'Shrew mouse gene name'</li>\n",
       "\t<li>'Shrew mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Shrew mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Shrew mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Shrew mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Shrew mouse'</li>\n",
       "\t<li>'Shrew mouse homology type'</li>\n",
       "\t<li>'%id. target Shrew mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Shrew mouse gene'</li>\n",
       "\t<li>'Shrew mouse Gene-order conservation score'</li>\n",
       "\t<li>'Shrew mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Shrew mouse'</li>\n",
       "\t<li>'dS with Shrew mouse'</li>\n",
       "\t<li>'Shrew mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Sloth gene stable ID'</li>\n",
       "\t<li>'Sloth gene name'</li>\n",
       "\t<li>'Sloth protein or transcript stable ID'</li>\n",
       "\t<li>'Sloth chromosome/scaffold name'</li>\n",
       "\t<li>'Sloth chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Sloth chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Sloth'</li>\n",
       "\t<li>'Sloth homology type'</li>\n",
       "\t<li>'%id. target Sloth gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Sloth gene'</li>\n",
       "\t<li>'Sloth Gene-order conservation score'</li>\n",
       "\t<li>'Sloth Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Sloth'</li>\n",
       "\t<li>'dS with Sloth'</li>\n",
       "\t<li>'Sloth orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Sooty mangabey gene stable ID'</li>\n",
       "\t<li>'Sooty mangabey gene name'</li>\n",
       "\t<li>'Sooty mangabey protein or transcript stable ID'</li>\n",
       "\t<li>'Sooty mangabey chromosome/scaffold name'</li>\n",
       "\t<li>'Sooty mangabey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Sooty mangabey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Sooty mangabey'</li>\n",
       "\t<li>'Sooty mangabey homology type'</li>\n",
       "\t<li>'%id. target Sooty mangabey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Sooty mangabey gene'</li>\n",
       "\t<li>'Sooty mangabey Gene-order conservation score'</li>\n",
       "\t<li>'Sooty mangabey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Sooty mangabey'</li>\n",
       "\t<li>'dS with Sooty mangabey'</li>\n",
       "\t<li>'Sooty mangabey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Spiny chromis gene stable ID'</li>\n",
       "\t<li>'Spiny chromis gene name'</li>\n",
       "\t<li>'Spiny chromis protein or transcript stable ID'</li>\n",
       "\t<li>'Spiny chromis chromosome/scaffold name'</li>\n",
       "\t<li>'Spiny chromis chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Spiny chromis chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Spiny chromis'</li>\n",
       "\t<li>'Spiny chromis homology type'</li>\n",
       "\t<li>'%id. target Spiny chromis gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Spiny chromis gene'</li>\n",
       "\t<li>'Spiny chromis Gene-order conservation score'</li>\n",
       "\t<li>'Spiny chromis Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Spiny chromis'</li>\n",
       "\t<li>'dS with Spiny chromis'</li>\n",
       "\t<li>'Spiny chromis orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Spoon-billed sandpiper gene stable ID'</li>\n",
       "\t<li>'Spoon-billed sandpiper gene name'</li>\n",
       "\t<li>'Spoon-billed sandpiper protein or transcript stable ID'</li>\n",
       "\t<li>'Spoon-billed sandpiper chromosome/scaffold name'</li>\n",
       "\t<li>'Spoon-billed sandpiper chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Spoon-billed sandpiper chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Spoon-billed sandpiper'</li>\n",
       "\t<li>'Spoon-billed sandpiper homology type'</li>\n",
       "\t<li>'%id. target Spoon-billed sandpiper gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Spoon-billed sandpiper gene'</li>\n",
       "\t<li>'Spoon-billed sandpiper Gene-order conservation score'</li>\n",
       "\t<li>'Spoon-billed sandpiper Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Spoon-billed sandpiper'</li>\n",
       "\t<li>'dS with Spoon-billed sandpiper'</li>\n",
       "\t<li>'Spoon-billed sandpiper orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Spotted gar gene stable ID'</li>\n",
       "\t<li>'Spotted gar gene name'</li>\n",
       "\t<li>'Spotted gar protein or transcript stable ID'</li>\n",
       "\t<li>'Spotted gar chromosome/scaffold name'</li>\n",
       "\t<li>'Spotted gar chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Spotted gar chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Spotted gar'</li>\n",
       "\t<li>'Spotted gar homology type'</li>\n",
       "\t<li>'%id. target Spotted gar gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Spotted gar gene'</li>\n",
       "\t<li>'Spotted gar Gene-order conservation score'</li>\n",
       "\t<li>'Spotted gar Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Spotted gar'</li>\n",
       "\t<li>'dS with Spotted gar'</li>\n",
       "\t<li>'Spotted gar orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Squirrel gene stable ID'</li>\n",
       "\t<li>'Squirrel gene name'</li>\n",
       "\t<li>'Squirrel protein or transcript stable ID'</li>\n",
       "\t<li>'Squirrel chromosome/scaffold name'</li>\n",
       "\t<li>'Squirrel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Squirrel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Squirrel'</li>\n",
       "\t<li>'Squirrel homology type'</li>\n",
       "\t<li>'%id. target Squirrel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Squirrel gene'</li>\n",
       "\t<li>'Squirrel Gene-order conservation score'</li>\n",
       "\t<li>'Squirrel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Squirrel'</li>\n",
       "\t<li>'dS with Squirrel'</li>\n",
       "\t<li>'Squirrel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Steppe mouse gene stable ID'</li>\n",
       "\t<li>'Steppe mouse gene name'</li>\n",
       "\t<li>'Steppe mouse protein or transcript stable ID'</li>\n",
       "\t<li>'Steppe mouse chromosome/scaffold name'</li>\n",
       "\t<li>'Steppe mouse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Steppe mouse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Steppe mouse'</li>\n",
       "\t<li>'Steppe mouse homology type'</li>\n",
       "\t<li>'%id. target Steppe mouse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Steppe mouse gene'</li>\n",
       "\t<li>'Steppe mouse Gene-order conservation score'</li>\n",
       "\t<li>'Steppe mouse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Steppe mouse'</li>\n",
       "\t<li>'dS with Steppe mouse'</li>\n",
       "\t<li>'Steppe mouse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Stickleback gene stable ID'</li>\n",
       "\t<li>'Stickleback gene name'</li>\n",
       "\t<li>'Stickleback protein or transcript stable ID'</li>\n",
       "\t<li>'Stickleback chromosome/scaffold name'</li>\n",
       "\t<li>'Stickleback chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Stickleback chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Stickleback'</li>\n",
       "\t<li>'Stickleback homology type'</li>\n",
       "\t<li>'%id. target Stickleback gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Stickleback gene'</li>\n",
       "\t<li>'Stickleback Gene-order conservation score'</li>\n",
       "\t<li>'Stickleback Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Stickleback'</li>\n",
       "\t<li>'dS with Stickleback'</li>\n",
       "\t<li>'Stickleback orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Swamp eel gene stable ID'</li>\n",
       "\t<li>'Swamp eel gene name'</li>\n",
       "\t<li>'Swamp eel protein or transcript stable ID'</li>\n",
       "\t<li>'Swamp eel chromosome/scaffold name'</li>\n",
       "\t<li>'Swamp eel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Swamp eel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Swamp eel'</li>\n",
       "\t<li>'Swamp eel homology type'</li>\n",
       "\t<li>'%id. target Swamp eel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Swamp eel gene'</li>\n",
       "\t<li>'Swamp eel Gene-order conservation score'</li>\n",
       "\t<li>'Swamp eel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Swamp eel'</li>\n",
       "\t<li>'dS with Swamp eel'</li>\n",
       "\t<li>'Swamp eel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tarsier gene stable ID'</li>\n",
       "\t<li>'Tarsier gene name'</li>\n",
       "\t<li>'Tarsier protein or transcript stable ID'</li>\n",
       "\t<li>'Tarsier chromosome/scaffold name'</li>\n",
       "\t<li>'Tarsier chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tarsier chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tarsier'</li>\n",
       "\t<li>'Tarsier homology type'</li>\n",
       "\t<li>'%id. target Tarsier gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tarsier gene'</li>\n",
       "\t<li>'Tarsier Gene-order conservation score'</li>\n",
       "\t<li>'Tarsier Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tarsier'</li>\n",
       "\t<li>'dS with Tarsier'</li>\n",
       "\t<li>'Tarsier orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tasmanian devil gene stable ID'</li>\n",
       "\t<li>'Tasmanian devil gene name'</li>\n",
       "\t<li>'Tasmanian devil protein or transcript stable ID'</li>\n",
       "\t<li>'Tasmanian devil chromosome/scaffold name'</li>\n",
       "\t<li>'Tasmanian devil chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tasmanian devil chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tasmanian devil'</li>\n",
       "\t<li>'Tasmanian devil homology type'</li>\n",
       "\t<li>'%id. target Tasmanian devil gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tasmanian devil gene'</li>\n",
       "\t<li>'Tasmanian devil Gene-order conservation score'</li>\n",
       "\t<li>'Tasmanian devil Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tasmanian devil'</li>\n",
       "\t<li>'dS with Tasmanian devil'</li>\n",
       "\t<li>'Tasmanian devil orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tetraodon gene stable ID'</li>\n",
       "\t<li>'Tetraodon gene name'</li>\n",
       "\t<li>'Tetraodon protein or transcript stable ID'</li>\n",
       "\t<li>'Tetraodon chromosome/scaffold name'</li>\n",
       "\t<li>'Tetraodon chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tetraodon chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tetraodon'</li>\n",
       "\t<li>'Tetraodon homology type'</li>\n",
       "\t<li>'%id. target Tetraodon gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tetraodon gene'</li>\n",
       "\t<li>'Tetraodon Gene-order conservation score'</li>\n",
       "\t<li>'Tetraodon Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tetraodon'</li>\n",
       "\t<li>'dS with Tetraodon'</li>\n",
       "\t<li>'Tetraodon orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tiger gene stable ID'</li>\n",
       "\t<li>'Tiger gene name'</li>\n",
       "\t<li>'Tiger protein or transcript stable ID'</li>\n",
       "\t<li>'Tiger chromosome/scaffold name'</li>\n",
       "\t<li>'Tiger chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tiger chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tiger'</li>\n",
       "\t<li>'Tiger homology type'</li>\n",
       "\t<li>'%id. target Tiger gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tiger gene'</li>\n",
       "\t<li>'Tiger Gene-order conservation score'</li>\n",
       "\t<li>'Tiger Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tiger'</li>\n",
       "\t<li>'dS with Tiger'</li>\n",
       "\t<li>'Tiger orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tiger tail seahorse gene stable ID'</li>\n",
       "\t<li>'Tiger tail seahorse gene name'</li>\n",
       "\t<li>'Tiger tail seahorse protein or transcript stable ID'</li>\n",
       "\t<li>'Tiger tail seahorse chromosome/scaffold name'</li>\n",
       "\t<li>'Tiger tail seahorse chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tiger tail seahorse chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tiger tail seahorse'</li>\n",
       "\t<li>'Tiger tail seahorse homology type'</li>\n",
       "\t<li>'%id. target Tiger tail seahorse gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tiger tail seahorse gene'</li>\n",
       "\t<li>'Tiger tail seahorse Gene-order conservation score'</li>\n",
       "\t<li>'Tiger tail seahorse Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tiger tail seahorse'</li>\n",
       "\t<li>'dS with Tiger tail seahorse'</li>\n",
       "\t<li>'Tiger tail seahorse orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tilapia gene stable ID'</li>\n",
       "\t<li>'Tilapia gene name'</li>\n",
       "\t<li>'Tilapia protein or transcript stable ID'</li>\n",
       "\t<li>'Tilapia chromosome/scaffold name'</li>\n",
       "\t<li>'Tilapia chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tilapia chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tilapia'</li>\n",
       "\t<li>'Tilapia homology type'</li>\n",
       "\t<li>'%id. target Tilapia gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tilapia gene'</li>\n",
       "\t<li>'Tilapia Gene-order conservation score'</li>\n",
       "\t<li>'Tilapia Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tilapia'</li>\n",
       "\t<li>'dS with Tilapia'</li>\n",
       "\t<li>'Tilapia orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tongue sole gene stable ID'</li>\n",
       "\t<li>'Tongue sole gene name'</li>\n",
       "\t<li>'Tongue sole protein or transcript stable ID'</li>\n",
       "\t<li>'Tongue sole chromosome/scaffold name'</li>\n",
       "\t<li>'Tongue sole chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tongue sole chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tongue sole'</li>\n",
       "\t<li>'Tongue sole homology type'</li>\n",
       "\t<li>'%id. target Tongue sole gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tongue sole gene'</li>\n",
       "\t<li>'Tongue sole Gene-order conservation score'</li>\n",
       "\t<li>'Tongue sole Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tongue sole'</li>\n",
       "\t<li>'dS with Tongue sole'</li>\n",
       "\t<li>'Tongue sole orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tree Shrew gene stable ID'</li>\n",
       "\t<li>'Tree Shrew gene name'</li>\n",
       "\t<li>'Tree Shrew protein or transcript stable ID'</li>\n",
       "\t<li>'Tree Shrew chromosome/scaffold name'</li>\n",
       "\t<li>'Tree Shrew chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tree Shrew chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tree Shrew'</li>\n",
       "\t<li>'Tree Shrew homology type'</li>\n",
       "\t<li>'%id. target Tree Shrew gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tree Shrew gene'</li>\n",
       "\t<li>'Tree Shrew Gene-order conservation score'</li>\n",
       "\t<li>'Tree Shrew Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tree Shrew'</li>\n",
       "\t<li>'dS with Tree Shrew'</li>\n",
       "\t<li>'Tree Shrew orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Tuatara gene stable ID'</li>\n",
       "\t<li>'Tuatara gene name'</li>\n",
       "\t<li>'Tuatara protein or transcript stable ID'</li>\n",
       "\t<li>'Tuatara chromosome/scaffold name'</li>\n",
       "\t<li>'Tuatara chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Tuatara chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Tuatara'</li>\n",
       "\t<li>'Tuatara homology type'</li>\n",
       "\t<li>'%id. target Tuatara gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Tuatara gene'</li>\n",
       "\t<li>'Tuatara Gene-order conservation score'</li>\n",
       "\t<li>'Tuatara Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Tuatara'</li>\n",
       "\t<li>'dS with Tuatara'</li>\n",
       "\t<li>'Tuatara orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Turbot gene stable ID'</li>\n",
       "\t<li>'Turbot gene name'</li>\n",
       "\t<li>'Turbot protein or transcript stable ID'</li>\n",
       "\t<li>'Turbot chromosome/scaffold name'</li>\n",
       "\t<li>'Turbot chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Turbot chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Turbot'</li>\n",
       "\t<li>'Turbot homology type'</li>\n",
       "\t<li>'%id. target Turbot gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Turbot gene'</li>\n",
       "\t<li>'Turbot Gene-order conservation score'</li>\n",
       "\t<li>'Turbot Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Turbot'</li>\n",
       "\t<li>'dS with Turbot'</li>\n",
       "\t<li>'Turbot orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Turkey gene stable ID'</li>\n",
       "\t<li>'Turkey gene name'</li>\n",
       "\t<li>'Turkey protein or transcript stable ID'</li>\n",
       "\t<li>'Turkey chromosome/scaffold name'</li>\n",
       "\t<li>'Turkey chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Turkey chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Turkey'</li>\n",
       "\t<li>'Turkey homology type'</li>\n",
       "\t<li>'%id. target Turkey gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Turkey gene'</li>\n",
       "\t<li>'Turkey Gene-order conservation score'</li>\n",
       "\t<li>'Turkey Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Turkey'</li>\n",
       "\t<li>'dS with Turkey'</li>\n",
       "\t<li>'Turkey orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Ugandan red Colobus gene stable ID'</li>\n",
       "\t<li>'Ugandan red Colobus gene name'</li>\n",
       "\t<li>'Ugandan red Colobus protein or transcript stable ID'</li>\n",
       "\t<li>'Ugandan red Colobus chromosome/scaffold name'</li>\n",
       "\t<li>'Ugandan red Colobus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Ugandan red Colobus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Ugandan red Colobus'</li>\n",
       "\t<li>'Ugandan red Colobus homology type'</li>\n",
       "\t<li>'%id. target Ugandan red Colobus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Ugandan red Colobus gene'</li>\n",
       "\t<li>'Ugandan red Colobus Gene-order conservation score'</li>\n",
       "\t<li>'Ugandan red Colobus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Ugandan red Colobus'</li>\n",
       "\t<li>'dS with Ugandan red Colobus'</li>\n",
       "\t<li>'Ugandan red Colobus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat gene stable ID'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat gene name'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat protein or transcript stable ID'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat chromosome/scaffold name'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Upper Galilee mountains blind mole rat'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat homology type'</li>\n",
       "\t<li>'%id. target Upper Galilee mountains blind mole rat gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Upper Galilee mountains blind mole rat gene'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat Gene-order conservation score'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Upper Galilee mountains blind mole rat'</li>\n",
       "\t<li>'dS with Upper Galilee mountains blind mole rat'</li>\n",
       "\t<li>'Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Vervet-AGM gene stable ID'</li>\n",
       "\t<li>'Vervet-AGM gene name'</li>\n",
       "\t<li>'Vervet-AGM protein or transcript stable ID'</li>\n",
       "\t<li>'Vervet-AGM chromosome/scaffold name'</li>\n",
       "\t<li>'Vervet-AGM chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Vervet-AGM chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Vervet-AGM'</li>\n",
       "\t<li>'Vervet-AGM homology type'</li>\n",
       "\t<li>'%id. target Vervet-AGM gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Vervet-AGM gene'</li>\n",
       "\t<li>'Vervet-AGM Gene-order conservation score'</li>\n",
       "\t<li>'Vervet-AGM Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Vervet-AGM'</li>\n",
       "\t<li>'dS with Vervet-AGM'</li>\n",
       "\t<li>'Vervet-AGM orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Wallaby gene stable ID'</li>\n",
       "\t<li>'Wallaby gene name'</li>\n",
       "\t<li>'Wallaby protein or transcript stable ID'</li>\n",
       "\t<li>'Wallaby chromosome/scaffold name'</li>\n",
       "\t<li>'Wallaby chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Wallaby chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Wallaby'</li>\n",
       "\t<li>'Wallaby homology type'</li>\n",
       "\t<li>'%id. target Wallaby gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Wallaby gene'</li>\n",
       "\t<li>'Wallaby Gene-order conservation score'</li>\n",
       "\t<li>'Wallaby Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Wallaby'</li>\n",
       "\t<li>'dS with Wallaby'</li>\n",
       "\t<li>'Wallaby orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Western mosquitofish gene stable ID'</li>\n",
       "\t<li>'Western mosquitofish gene name'</li>\n",
       "\t<li>'Western mosquitofish protein or transcript stable ID'</li>\n",
       "\t<li>'Western mosquitofish chromosome/scaffold name'</li>\n",
       "\t<li>'Western mosquitofish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Western mosquitofish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Western mosquitofish'</li>\n",
       "\t<li>'Western mosquitofish homology type'</li>\n",
       "\t<li>'%id. target Western mosquitofish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Western mosquitofish gene'</li>\n",
       "\t<li>'Western mosquitofish Gene-order conservation score'</li>\n",
       "\t<li>'Western mosquitofish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Western mosquitofish'</li>\n",
       "\t<li>'dS with Western mosquitofish'</li>\n",
       "\t<li>'Western mosquitofish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'White-throated sparrow gene stable ID'</li>\n",
       "\t<li>'White-throated sparrow gene name'</li>\n",
       "\t<li>'White-throated sparrow protein or transcript stable ID'</li>\n",
       "\t<li>'White-throated sparrow chromosome/scaffold name'</li>\n",
       "\t<li>'White-throated sparrow chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'White-throated sparrow chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with White-throated sparrow'</li>\n",
       "\t<li>'White-throated sparrow homology type'</li>\n",
       "\t<li>'%id. target White-throated sparrow gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target White-throated sparrow gene'</li>\n",
       "\t<li>'White-throated sparrow Gene-order conservation score'</li>\n",
       "\t<li>'White-throated sparrow Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with White-throated sparrow'</li>\n",
       "\t<li>'dS with White-throated sparrow'</li>\n",
       "\t<li>'White-throated sparrow orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Wild yak gene stable ID'</li>\n",
       "\t<li>'Wild yak gene name'</li>\n",
       "\t<li>'Wild yak protein or transcript stable ID'</li>\n",
       "\t<li>'Wild yak chromosome/scaffold name'</li>\n",
       "\t<li>'Wild yak chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Wild yak chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Wild yak'</li>\n",
       "\t<li>'Wild yak homology type'</li>\n",
       "\t<li>'%id. target Wild yak gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Wild yak gene'</li>\n",
       "\t<li>'Wild yak Gene-order conservation score'</li>\n",
       "\t<li>'Wild yak Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Wild yak'</li>\n",
       "\t<li>'dS with Wild yak'</li>\n",
       "\t<li>'Wild yak orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Xenopus gene stable ID'</li>\n",
       "\t<li>'Xenopus gene name'</li>\n",
       "\t<li>'Xenopus protein or transcript stable ID'</li>\n",
       "\t<li>'Xenopus chromosome/scaffold name'</li>\n",
       "\t<li>'Xenopus chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Xenopus chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Xenopus'</li>\n",
       "\t<li>'Xenopus homology type'</li>\n",
       "\t<li>'%id. target Xenopus gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Xenopus gene'</li>\n",
       "\t<li>'Xenopus Gene-order conservation score'</li>\n",
       "\t<li>'Xenopus Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Xenopus'</li>\n",
       "\t<li>'dS with Xenopus'</li>\n",
       "\t<li>'Xenopus orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Yellowtail amberjack gene stable ID'</li>\n",
       "\t<li>'Yellowtail amberjack gene name'</li>\n",
       "\t<li>'Yellowtail amberjack protein or transcript stable ID'</li>\n",
       "\t<li>'Yellowtail amberjack chromosome/scaffold name'</li>\n",
       "\t<li>'Yellowtail amberjack chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Yellowtail amberjack chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Yellowtail amberjack'</li>\n",
       "\t<li>'Yellowtail amberjack homology type'</li>\n",
       "\t<li>'%id. target Yellowtail amberjack gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Yellowtail amberjack gene'</li>\n",
       "\t<li>'Yellowtail amberjack Gene-order conservation score'</li>\n",
       "\t<li>'Yellowtail amberjack Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Yellowtail amberjack'</li>\n",
       "\t<li>'dS with Yellowtail amberjack'</li>\n",
       "\t<li>'Yellowtail amberjack orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Zebra Finch gene stable ID'</li>\n",
       "\t<li>'Zebra Finch gene name'</li>\n",
       "\t<li>'Zebra Finch protein or transcript stable ID'</li>\n",
       "\t<li>'Zebra Finch chromosome/scaffold name'</li>\n",
       "\t<li>'Zebra Finch chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Zebra Finch chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Zebra Finch'</li>\n",
       "\t<li>'Zebra Finch homology type'</li>\n",
       "\t<li>'%id. target Zebra Finch gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Zebra Finch gene'</li>\n",
       "\t<li>'Zebra Finch Gene-order conservation score'</li>\n",
       "\t<li>'Zebra Finch Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Zebra Finch'</li>\n",
       "\t<li>'dS with Zebra Finch'</li>\n",
       "\t<li>'Zebra Finch orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Zebra mbuna gene stable ID'</li>\n",
       "\t<li>'Zebra mbuna gene name'</li>\n",
       "\t<li>'Zebra mbuna protein or transcript stable ID'</li>\n",
       "\t<li>'Zebra mbuna chromosome/scaffold name'</li>\n",
       "\t<li>'Zebra mbuna chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Zebra mbuna chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Zebra mbuna'</li>\n",
       "\t<li>'Zebra mbuna homology type'</li>\n",
       "\t<li>'%id. target Zebra mbuna gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Zebra mbuna gene'</li>\n",
       "\t<li>'Zebra mbuna Gene-order conservation score'</li>\n",
       "\t<li>'Zebra mbuna Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Zebra mbuna'</li>\n",
       "\t<li>'dS with Zebra mbuna'</li>\n",
       "\t<li>'Zebra mbuna orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Zebrafish gene stable ID'</li>\n",
       "\t<li>'Zebrafish gene name'</li>\n",
       "\t<li>'Zebrafish protein or transcript stable ID'</li>\n",
       "\t<li>'Zebrafish chromosome/scaffold name'</li>\n",
       "\t<li>'Zebrafish chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Zebrafish chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Zebrafish'</li>\n",
       "\t<li>'Zebrafish homology type'</li>\n",
       "\t<li>'%id. target Zebrafish gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Zebrafish gene'</li>\n",
       "\t<li>'Zebrafish Gene-order conservation score'</li>\n",
       "\t<li>'Zebrafish Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Zebrafish'</li>\n",
       "\t<li>'dS with Zebrafish'</li>\n",
       "\t<li>'Zebrafish orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Zig-zag eel gene stable ID'</li>\n",
       "\t<li>'Zig-zag eel gene name'</li>\n",
       "\t<li>'Zig-zag eel protein or transcript stable ID'</li>\n",
       "\t<li>'Zig-zag eel chromosome/scaffold name'</li>\n",
       "\t<li>'Zig-zag eel chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Zig-zag eel chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Query protein or transcript ID'</li>\n",
       "\t<li>'Last common ancestor with Zig-zag eel'</li>\n",
       "\t<li>'Zig-zag eel homology type'</li>\n",
       "\t<li>'%id. target Zig-zag eel gene identical to query gene'</li>\n",
       "\t<li>'%id. query gene identical to target Zig-zag eel gene'</li>\n",
       "\t<li>'Zig-zag eel Gene-order conservation score'</li>\n",
       "\t<li>'Zig-zag eel Whole-genome alignment coverage'</li>\n",
       "\t<li>'dN with Zig-zag eel'</li>\n",
       "\t<li>'dS with Zig-zag eel'</li>\n",
       "\t<li>'Zig-zag eel orthology confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Human paralogue gene stable ID'</li>\n",
       "\t<li>'Human paralogue associated gene name'</li>\n",
       "\t<li>'Human paralogue protein or transcript ID'</li>\n",
       "\t<li>'Human paralogue chromosome/scaffold name'</li>\n",
       "\t<li>'Human paralogue chromosome/scaffold start (bp)'</li>\n",
       "\t<li>'Human paralogue chromosome/scaffold end (bp)'</li>\n",
       "\t<li>'Paralogue query protein or transcript ID'</li>\n",
       "\t<li>'Paralogue last common ancestor with Human'</li>\n",
       "\t<li>'Human paralogue homology type'</li>\n",
       "\t<li>'Paralogue %id. target Human gene identical to query gene'</li>\n",
       "\t<li>'Paralogue %id. query gene identical to target Human gene'</li>\n",
       "\t<li>'Paralogue dN with Human'</li>\n",
       "\t<li>'Paralogue dS with Human'</li>\n",
       "\t<li>'Human paralogy confidence [0 low, 1 high]'</li>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Karyotype band'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'Transcript count'</li>\n",
       "\t<li>'Gene % GC content'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Variant name'</li>\n",
       "\t<li>'Variant source'</li>\n",
       "\t<li>'Variant source description'</li>\n",
       "\t<li>'Variant alleles'</li>\n",
       "\t<li>'Variant supporting evidence'</li>\n",
       "\t<li>'Mapweight'</li>\n",
       "\t<li>'Minor allele'</li>\n",
       "\t<li>'Minor allele frequency'</li>\n",
       "\t<li>'Minor allele count'</li>\n",
       "\t<li>'Clinical significance'</li>\n",
       "\t<li>'Transcript location (bp)'</li>\n",
       "\t<li>'Variant chromosome Strand'</li>\n",
       "\t<li>'Protein location (aa)'</li>\n",
       "\t<li>'chromosome/scaffold position start (bp)'</li>\n",
       "\t<li>'Chromosome/scaffold position end (bp)'</li>\n",
       "\t<li>'PolyPhen prediction'</li>\n",
       "\t<li>'PolyPhen score'</li>\n",
       "\t<li>'SIFT prediction'</li>\n",
       "\t<li>'SIFT score'</li>\n",
       "\t<li>'Distance to transcript'</li>\n",
       "\t<li>'CDS start'</li>\n",
       "\t<li>'CDS end'</li>\n",
       "\t<li>'Protein allele'</li>\n",
       "\t<li>'Variant consequence'</li>\n",
       "\t<li>'Consequence specific allele'</li>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Karyotype band'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'Transcript count'</li>\n",
       "\t<li>'Gene % GC content'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Variant name'</li>\n",
       "\t<li>'Variant source'</li>\n",
       "\t<li>'Variant source description'</li>\n",
       "\t<li>'Variant alleles'</li>\n",
       "\t<li>'Variant supporting evidence'</li>\n",
       "\t<li>'Mapweight'</li>\n",
       "\t<li>'Minor allele'</li>\n",
       "\t<li>'Minor allele frequency'</li>\n",
       "\t<li>'Minor allele count'</li>\n",
       "\t<li>'Clinical significance'</li>\n",
       "\t<li>'Transcript location (bp)'</li>\n",
       "\t<li>'Variant chromosome/scaffold strand'</li>\n",
       "\t<li>'Protein location (aa)'</li>\n",
       "\t<li>'Chromosome/scaffold position start (bp)'</li>\n",
       "\t<li>'Chromosome/scaffold position end (bp)'</li>\n",
       "\t<li>'PolyPhen prediction'</li>\n",
       "\t<li>'PolyPhen score'</li>\n",
       "\t<li>'SIFT prediction'</li>\n",
       "\t<li>'SIFT score'</li>\n",
       "\t<li>'Distance to transcript'</li>\n",
       "\t<li>'CDS start'</li>\n",
       "\t<li>'CDS end'</li>\n",
       "\t<li>'Protein allele'</li>\n",
       "\t<li>'Variant consequence'</li>\n",
       "\t<li>'Consequence specific allele'</li>\n",
       "\t<li>'Unspliced (Transcript)'</li>\n",
       "\t<li>'Unspliced (Gene)'</li>\n",
       "\t<li>'Flank (Transcript)'</li>\n",
       "\t<li>'Flank (Gene)'</li>\n",
       "\t<li>'Flank-coding region (Transcript)'</li>\n",
       "\t<li>'Flank-coding region (Gene)'</li>\n",
       "\t<li>'5\\' UTR'</li>\n",
       "\t<li>'3\\' UTR'</li>\n",
       "\t<li>'Exon sequences'</li>\n",
       "\t<li>'cDNA sequences'</li>\n",
       "\t<li>'Coding sequence'</li>\n",
       "\t<li>'Peptide'</li>\n",
       "\t<li>'upstream_flank'</li>\n",
       "\t<li>'downstream_flank'</li>\n",
       "\t<li>'Gene stable ID'</li>\n",
       "\t<li>'Gene stable ID version'</li>\n",
       "\t<li>'Gene description'</li>\n",
       "\t<li>'Gene name'</li>\n",
       "\t<li>'Source of gene name'</li>\n",
       "\t<li>'Chromosome/scaffold name'</li>\n",
       "\t<li>'Gene start (bp)'</li>\n",
       "\t<li>'Gene end (bp)'</li>\n",
       "\t<li>'Gene type'</li>\n",
       "\t<li>'Version (gene)'</li>\n",
       "\t<li>'Ensembl Protein Family ID(s)'</li>\n",
       "\t<li>'UniParc ID'</li>\n",
       "\t<li>'UniProtKB/Swiss-Prot ID'</li>\n",
       "\t<li>'UniProtKB/TrEMBL ID'</li>\n",
       "\t<li>'CDS start (within cDNA)'</li>\n",
       "\t<li>'CDS end (within cDNA)'</li>\n",
       "\t<li>'5\\' UTR start'</li>\n",
       "\t<li>'5\\' UTR end'</li>\n",
       "\t<li>'3\\' UTR start'</li>\n",
       "\t<li>'3\\' UTR end'</li>\n",
       "\t<li>'Transcript stable ID'</li>\n",
       "\t<li>'Transcript stable ID version'</li>\n",
       "\t<li>'Protein stable ID'</li>\n",
       "\t<li>'Protein stable ID version'</li>\n",
       "\t<li>'Transcript type'</li>\n",
       "\t<li>'Version (transcript)'</li>\n",
       "\t<li>'Version (protein)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Transcript start (bp)'</li>\n",
       "\t<li>'Transcript end (bp)'</li>\n",
       "\t<li>'Transcription start site (TSS)'</li>\n",
       "\t<li>'Transcript length (including UTRs and CDS)'</li>\n",
       "\t<li>'CDS Length'</li>\n",
       "\t<li>'CDS start'</li>\n",
       "\t<li>'CDS end'</li>\n",
       "\t<li>'Exon stable ID'</li>\n",
       "\t<li>'Exon region start (bp)'</li>\n",
       "\t<li>'Exon region end (bp)'</li>\n",
       "\t<li>'Strand'</li>\n",
       "\t<li>'Exon rank in transcript'</li>\n",
       "\t<li>'Start phase'</li>\n",
       "\t<li>'End phase'</li>\n",
       "\t<li>'cDNA coding start'</li>\n",
       "\t<li>'cDNA coding end'</li>\n",
       "\t<li>'Genomic coding start'</li>\n",
       "\t<li>'Genomic coding end'</li>\n",
       "\t<li>'Constitutive exon'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Exon stable ID'\n",
       "\\item 'Gene description'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Strand'\n",
       "\\item 'Karyotype band'\n",
       "\\item 'Transcript start (bp)'\n",
       "\\item 'Transcript end (bp)'\n",
       "\\item 'Transcription start site (TSS)'\n",
       "\\item 'Transcript length (including UTRs and CDS)'\n",
       "\\item 'Transcript support level (TSL)'\n",
       "\\item 'GENCODE basic annotation'\n",
       "\\item 'APPRIS annotation'\n",
       "\\item 'RefSeq match transcript'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item 'Transcript name'\n",
       "\\item 'Source of transcript name'\n",
       "\\item 'Transcript count'\n",
       "\\item 'Gene \\% GC content'\n",
       "\\item 'Gene type'\n",
       "\\item 'Transcript type'\n",
       "\\item 'Source (gene)'\n",
       "\\item 'Source (transcript)'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Phenotype description'\n",
       "\\item 'Source name'\n",
       "\\item 'Study external reference'\n",
       "\\item 'Strain name'\n",
       "\\item 'Strain gender'\n",
       "\\item 'P value'\n",
       "\\item 'GO term accession'\n",
       "\\item 'GO term name'\n",
       "\\item 'GO term definition'\n",
       "\\item 'GO term evidence code'\n",
       "\\item 'GO domain'\n",
       "\\item 'GOSlim GOA Accession(s)'\n",
       "\\item 'GOSlim GOA Description'\n",
       "\\item 'CCDS ID'\n",
       "\\item 'ChEMBL ID'\n",
       "\\item 'Clone-based (Ensembl) gene ID'\n",
       "\\item 'Clone-based (Ensembl) transcript ID'\n",
       "\\item 'DataBase of Aberrant 3\\textbackslash{}' Splice Sites name'\n",
       "\\item 'DataBase of Aberrant 3\\textbackslash{}' Splice Sites ID'\n",
       "\\item 'DataBase of Aberrant 5\\textbackslash{}' Splice Sites name'\n",
       "\\item 'DataBase of Aberrant 5\\textbackslash{}' Splice Sites ID'\n",
       "\\item 'Ensembl Human Transcript ID'\n",
       "\\item 'Ensembl Human Translation ID'\n",
       "\\item 'EntrezGene transcript name ID'\n",
       "\\item 'European Nucleotide Archive ID'\n",
       "\\item 'Expression Atlas ID'\n",
       "\\item 'GeneDB ID'\n",
       "\\item 'HGNC ID'\n",
       "\\item 'HGNC symbol'\n",
       "\\item 'HGNC transcript name ID'\n",
       "\\item 'Human Protein Atlas accession'\n",
       "\\item 'Human Protein Atlas ID'\n",
       "\\item 'INSDC protein ID'\n",
       "\\item 'KEGG Pathway and Enzyme ID'\n",
       "\\item 'LRG display in Ensembl gene ID'\n",
       "\\item 'LRG display in Ensembl transcript ID'\n",
       "\\item 'MEROPS - the Peptidase Database ID'\n",
       "\\item 'MIM gene description'\n",
       "\\item 'MIM gene accession'\n",
       "\\item 'MIM morbid description'\n",
       "\\item 'MIM morbid accession'\n",
       "\\item 'miRBase accession'\n",
       "\\item 'miRBase ID'\n",
       "\\item 'miRBase transcript name ID'\n",
       "\\item 'NCBI gene description'\n",
       "\\item 'NCBI gene accession'\n",
       "\\item 'NCBI gene ID'\n",
       "\\item 'PDB ID'\n",
       "\\item 'Reactome ID'\n",
       "\\item 'Reactome gene ID'\n",
       "\\item 'Reactome transcript ID'\n",
       "\\item 'RefSeq mRNA ID'\n",
       "\\item 'RefSeq mRNA predicted ID'\n",
       "\\item 'RefSeq ncRNA ID'\n",
       "\\item 'RefSeq ncRNA predicted ID'\n",
       "\\item 'RefSeq peptide ID'\n",
       "\\item 'RefSeq peptide predicted ID'\n",
       "\\item 'RFAM ID'\n",
       "\\item 'RFAM transcript name ID'\n",
       "\\item 'RNAcentral ID'\n",
       "\\item 'UCSC Stable ID'\n",
       "\\item 'UniParc ID'\n",
       "\\item 'UniProtKB Gene Name symbol'\n",
       "\\item 'UniProtKB Gene Name ID'\n",
       "\\item 'UniProtKB/Swiss-Prot ID'\n",
       "\\item 'UniProtKB/TrEMBL ID'\n",
       "\\item 'WikiGene description'\n",
       "\\item 'WikiGene name'\n",
       "\\item 'WikiGene ID'\n",
       "\\item 'AFFY HC G110 probe'\n",
       "\\item 'AFFY HG Focus probe'\n",
       "\\item 'AFFY HG U133A probe'\n",
       "\\item 'AFFY HG U133A 2 probe'\n",
       "\\item 'AFFY HG U133B probe'\n",
       "\\item 'AFFY HG U133 Plus 2 probe'\n",
       "\\item 'AFFY HG U95A probe'\n",
       "\\item 'AFFY HG U95Av2 probe'\n",
       "\\item 'AFFY HG U95B probe'\n",
       "\\item 'AFFY HG U95C probe'\n",
       "\\item 'AFFY HG U95D probe'\n",
       "\\item 'AFFY HG U95E probe'\n",
       "\\item 'AFFY HTA 2 0 probe'\n",
       "\\item 'AFFY HuEx 1 0 st v2 probe'\n",
       "\\item 'AFFY HuGeneFL probe'\n",
       "\\item 'AFFY HuGene 1 0 st v1 probe'\n",
       "\\item 'AFFY HuGene 2 0 st v1 probe'\n",
       "\\item 'AFFY PrimeView probe'\n",
       "\\item 'AFFY U133 X3P probe'\n",
       "\\item 'AGILENT CGH 44b probe'\n",
       "\\item 'AGILENT GPL6848 probe'\n",
       "\\item 'AGILENT SurePrint G3 GE 8x60k probe'\n",
       "\\item 'AGILENT SurePrint G3 GE 8x60k v2 probe'\n",
       "\\item 'AGILENT WholeGenome probe'\n",
       "\\item 'AGILENT WholeGenome 4x44k v1 probe'\n",
       "\\item 'AGILENT WholeGenome 4x44k v2 probe'\n",
       "\\item 'CODELINK CODELINK probe'\n",
       "\\item 'ILLUMINA HumanHT 12 V3 probe'\n",
       "\\item 'ILLUMINA HumanHT 12 V4 probe'\n",
       "\\item 'ILLUMINA HumanRef 8 V3 probe'\n",
       "\\item 'ILLUMINA HumanWG 6 V1 probe'\n",
       "\\item 'ILLUMINA HumanWG 6 V2 probe'\n",
       "\\item 'ILLUMINA HumanWG 6 V3 probe'\n",
       "\\item 'PHALANX OneArray probe'\n",
       "\\item 'Ensembl Protein Family ID(s)'\n",
       "\\item 'Ensembl Family Description'\n",
       "\\item 'CDD ID'\n",
       "\\item 'CDD start'\n",
       "\\item 'CDD end'\n",
       "\\item 'Gene3D ID'\n",
       "\\item 'Gene3D start'\n",
       "\\item 'Gene3D end'\n",
       "\\item 'HAMAP ID'\n",
       "\\item 'HAMAP start'\n",
       "\\item 'HAMAP end'\n",
       "\\item 'PANTHER ID'\n",
       "\\item 'PANTHER start'\n",
       "\\item 'PANTHER end'\n",
       "\\item 'Pfam ID'\n",
       "\\item 'Pfam start'\n",
       "\\item 'Pfam end'\n",
       "\\item 'PIRSF ID'\n",
       "\\item 'PIRSF start'\n",
       "\\item 'PIRSF end'\n",
       "\\item 'Prints ID'\n",
       "\\item 'Prints start'\n",
       "\\item 'Prints end'\n",
       "\\item 'ProDom ID'\n",
       "\\item 'ProDom start'\n",
       "\\item 'ProDom end'\n",
       "\\item 'PROSITE patterns ID'\n",
       "\\item 'PROSITE patterns start'\n",
       "\\item 'PROSITE patterns end'\n",
       "\\item 'PROSITE profiles ID'\n",
       "\\item 'PROSITE profiles start'\n",
       "\\item 'PROSITE profiles end'\n",
       "\\item 'SFLD ID'\n",
       "\\item 'SFLD start'\n",
       "\\item 'SFLD end'\n",
       "\\item 'SMART ID'\n",
       "\\item 'SMART start'\n",
       "\\item 'SMART end'\n",
       "\\item 'Superfamily ID'\n",
       "\\item 'Superfamily start'\n",
       "\\item 'Superfamily end'\n",
       "\\item 'TIGRFAM ID'\n",
       "\\item 'TIGRFAM start'\n",
       "\\item 'TIGRFAM end'\n",
       "\\item 'Interpro ID'\n",
       "\\item 'Interpro Short Description'\n",
       "\\item 'Interpro Description'\n",
       "\\item 'Interpro start'\n",
       "\\item 'Interpro end'\n",
       "\\item 'MobiDB lite'\n",
       "\\item 'MobiDB lite start'\n",
       "\\item 'MobiDB lite end'\n",
       "\\item 'Coiled-coils (Ncoils)'\n",
       "\\item 'Coiled-coils (Ncoils) start'\n",
       "\\item 'Coiled-coils (Ncoils) end'\n",
       "\\item 'Low complexity (Seg)'\n",
       "\\item 'Low complexity (Seg) start'\n",
       "\\item 'Low complexity (Seg) end'\n",
       "\\item 'PDB-ENSP mappings'\n",
       "\\item 'PDB-ENSP mappings start'\n",
       "\\item 'PDB-ENSP mappings end'\n",
       "\\item 'Cleavage site (Signalp)'\n",
       "\\item 'Cleavage site (Signalp) start'\n",
       "\\item 'Cleavage site (Signalp) end'\n",
       "\\item 'Transmembrane helices'\n",
       "\\item 'Transmembrane helices start'\n",
       "\\item 'Transmembrane helices end'\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Transcript start (bp)'\n",
       "\\item 'Transcript end (bp)'\n",
       "\\item 'Transcription start site (TSS)'\n",
       "\\item 'Transcript length (including UTRs and CDS)'\n",
       "\\item 'Strand'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item '5\\textbackslash{}' UTR start'\n",
       "\\item '5\\textbackslash{}' UTR end'\n",
       "\\item '3\\textbackslash{}' UTR start'\n",
       "\\item '3\\textbackslash{}' UTR end'\n",
       "\\item 'CDS Length'\n",
       "\\item 'Transcript count'\n",
       "\\item 'Gene description'\n",
       "\\item 'Gene type'\n",
       "\\item 'Exon region start (bp)'\n",
       "\\item 'Exon region end (bp)'\n",
       "\\item 'Constitutive exon'\n",
       "\\item 'Exon rank in transcript'\n",
       "\\item 'Start phase'\n",
       "\\item 'End phase'\n",
       "\\item 'cDNA coding start'\n",
       "\\item 'cDNA coding end'\n",
       "\\item 'Genomic coding start'\n",
       "\\item 'Genomic coding end'\n",
       "\\item 'Exon stable ID'\n",
       "\\item 'CDS start'\n",
       "\\item 'CDS end'\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Strand'\n",
       "\\item 'Karyotype band'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item 'Transcript count'\n",
       "\\item 'Gene \\% GC content'\n",
       "\\item 'Gene description'\n",
       "\\item 'Abingdon island giant tortoise gene stable ID'\n",
       "\\item 'Abingdon island giant tortoise gene name'\n",
       "\\item 'Abingdon island giant tortoise protein or transcript stable ID'\n",
       "\\item 'Abingdon island giant tortoise chromosome/scaffold name'\n",
       "\\item 'Abingdon island giant tortoise chromosome/scaffold start (bp)'\n",
       "\\item 'Abingdon island giant tortoise chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Abingdon island giant tortoise'\n",
       "\\item 'Abingdon island giant tortoise homology type'\n",
       "\\item '\\%id. target Abingdon island giant tortoise gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Abingdon island giant tortoise gene'\n",
       "\\item 'Abingdon island giant tortoise Gene-order conservation score'\n",
       "\\item 'Abingdon island giant tortoise Whole-genome alignment coverage'\n",
       "\\item 'dN with Abingdon island giant tortoise'\n",
       "\\item 'dS with Abingdon island giant tortoise'\n",
       "\\item 'Abingdon island giant tortoise orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise gene stable ID'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise gene name'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise protein or transcript stable ID'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise chromosome/scaffold name'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise chromosome/scaffold start (bp)'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Agassiz\\textbackslash{}'s desert tortoise'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise homology type'\n",
       "\\item '\\%id. target Agassiz\\textbackslash{}'s desert tortoise gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Agassiz\\textbackslash{}'s desert tortoise gene'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise Gene-order conservation score'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise Whole-genome alignment coverage'\n",
       "\\item 'dN with Agassiz\\textbackslash{}'s desert tortoise'\n",
       "\\item 'dS with Agassiz\\textbackslash{}'s desert tortoise'\n",
       "\\item 'Agassiz\\textbackslash{}'s desert tortoise orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Algerian mouse gene stable ID'\n",
       "\\item 'Algerian mouse gene name'\n",
       "\\item 'Algerian mouse protein or transcript stable ID'\n",
       "\\item 'Algerian mouse chromosome/scaffold name'\n",
       "\\item 'Algerian mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Algerian mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Algerian mouse'\n",
       "\\item 'Algerian mouse homology type'\n",
       "\\item '\\%id. target Algerian mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Algerian mouse gene'\n",
       "\\item 'Algerian mouse Gene-order conservation score'\n",
       "\\item 'Algerian mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Algerian mouse'\n",
       "\\item 'dS with Algerian mouse'\n",
       "\\item 'Algerian mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Alpaca gene stable ID'\n",
       "\\item 'Alpaca gene name'\n",
       "\\item 'Alpaca protein or transcript stable ID'\n",
       "\\item 'Alpaca chromosome/scaffold name'\n",
       "\\item 'Alpaca chromosome/scaffold start (bp)'\n",
       "\\item 'Alpaca chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Alpaca'\n",
       "\\item 'Alpaca homology type'\n",
       "\\item '\\%id. target Alpaca gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Alpaca gene'\n",
       "\\item 'Alpaca Gene-order conservation score'\n",
       "\\item 'Alpaca Whole-genome alignment coverage'\n",
       "\\item 'dN with Alpaca'\n",
       "\\item 'dS with Alpaca'\n",
       "\\item 'Alpaca orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Alpine marmot gene stable ID'\n",
       "\\item 'Alpine marmot gene name'\n",
       "\\item 'Alpine marmot protein or transcript stable ID'\n",
       "\\item 'Alpine marmot chromosome/scaffold name'\n",
       "\\item 'Alpine marmot chromosome/scaffold start (bp)'\n",
       "\\item 'Alpine marmot chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Alpine marmot'\n",
       "\\item 'Alpine marmot homology type'\n",
       "\\item '\\%id. target Alpine marmot gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Alpine marmot gene'\n",
       "\\item 'Alpine marmot Gene-order conservation score'\n",
       "\\item 'Alpine marmot Whole-genome alignment coverage'\n",
       "\\item 'dN with Alpine marmot'\n",
       "\\item 'dS with Alpine marmot'\n",
       "\\item 'Alpine marmot orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Amazon molly gene stable ID'\n",
       "\\item 'Amazon molly gene name'\n",
       "\\item 'Amazon molly protein or transcript stable ID'\n",
       "\\item 'Amazon molly chromosome/scaffold name'\n",
       "\\item 'Amazon molly chromosome/scaffold start (bp)'\n",
       "\\item 'Amazon molly chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Amazon molly'\n",
       "\\item 'Amazon molly homology type'\n",
       "\\item '\\%id. target Amazon molly gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Amazon molly gene'\n",
       "\\item 'Amazon molly Gene-order conservation score'\n",
       "\\item 'Amazon molly Whole-genome alignment coverage'\n",
       "\\item 'dN with Amazon molly'\n",
       "\\item 'dS with Amazon molly'\n",
       "\\item 'Amazon molly orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'American beaver gene stable ID'\n",
       "\\item 'American beaver gene name'\n",
       "\\item 'American beaver protein or transcript stable ID'\n",
       "\\item 'American beaver chromosome/scaffold name'\n",
       "\\item 'American beaver chromosome/scaffold start (bp)'\n",
       "\\item 'American beaver chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with American beaver'\n",
       "\\item 'American beaver homology type'\n",
       "\\item '\\%id. target American beaver gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target American beaver gene'\n",
       "\\item 'American beaver Gene-order conservation score'\n",
       "\\item 'American beaver Whole-genome alignment coverage'\n",
       "\\item 'dN with American beaver'\n",
       "\\item 'dS with American beaver'\n",
       "\\item 'American beaver orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'American bison gene stable ID'\n",
       "\\item 'American bison gene name'\n",
       "\\item 'American bison protein or transcript stable ID'\n",
       "\\item 'American bison chromosome/scaffold name'\n",
       "\\item 'American bison chromosome/scaffold start (bp)'\n",
       "\\item 'American bison chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with American bison'\n",
       "\\item 'American bison homology type'\n",
       "\\item '\\%id. target American bison gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target American bison gene'\n",
       "\\item 'American bison Gene-order conservation score'\n",
       "\\item 'American bison Whole-genome alignment coverage'\n",
       "\\item 'dN with American bison'\n",
       "\\item 'dS with American bison'\n",
       "\\item 'American bison orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'American black bear gene stable ID'\n",
       "\\item 'American black bear gene name'\n",
       "\\item 'American black bear protein or transcript stable ID'\n",
       "\\item 'American black bear chromosome/scaffold name'\n",
       "\\item 'American black bear chromosome/scaffold start (bp)'\n",
       "\\item 'American black bear chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with American black bear'\n",
       "\\item 'American black bear homology type'\n",
       "\\item '\\%id. target American black bear gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target American black bear gene'\n",
       "\\item 'American black bear Gene-order conservation score'\n",
       "\\item 'American black bear Whole-genome alignment coverage'\n",
       "\\item 'dN with American black bear'\n",
       "\\item 'dS with American black bear'\n",
       "\\item 'American black bear orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'American mink gene stable ID'\n",
       "\\item 'American mink gene name'\n",
       "\\item 'American mink protein or transcript stable ID'\n",
       "\\item 'American mink chromosome/scaffold name'\n",
       "\\item 'American mink chromosome/scaffold start (bp)'\n",
       "\\item 'American mink chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with American mink'\n",
       "\\item 'American mink homology type'\n",
       "\\item '\\%id. target American mink gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target American mink gene'\n",
       "\\item 'American mink Gene-order conservation score'\n",
       "\\item 'American mink Whole-genome alignment coverage'\n",
       "\\item 'dN with American mink'\n",
       "\\item 'dS with American mink'\n",
       "\\item 'American mink orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Angola colobus gene stable ID'\n",
       "\\item 'Angola colobus gene name'\n",
       "\\item 'Angola colobus protein or transcript stable ID'\n",
       "\\item 'Angola colobus chromosome/scaffold name'\n",
       "\\item 'Angola colobus chromosome/scaffold start (bp)'\n",
       "\\item 'Angola colobus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Angola colobus'\n",
       "\\item 'Angola colobus homology type'\n",
       "\\item '\\%id. target Angola colobus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Angola colobus gene'\n",
       "\\item 'Angola colobus Gene-order conservation score'\n",
       "\\item 'Angola colobus Whole-genome alignment coverage'\n",
       "\\item 'dN with Angola colobus'\n",
       "\\item 'dS with Angola colobus'\n",
       "\\item 'Angola colobus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Anole lizard gene stable ID'\n",
       "\\item 'Anole lizard gene name'\n",
       "\\item 'Anole lizard protein or transcript stable ID'\n",
       "\\item 'Anole lizard chromosome/scaffold name'\n",
       "\\item 'Anole lizard chromosome/scaffold start (bp)'\n",
       "\\item 'Anole lizard chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Anole lizard'\n",
       "\\item 'Anole lizard homology type'\n",
       "\\item '\\%id. target Anole lizard gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Anole lizard gene'\n",
       "\\item 'Anole lizard Gene-order conservation score'\n",
       "\\item 'Anole lizard Whole-genome alignment coverage'\n",
       "\\item 'dN with Anole lizard'\n",
       "\\item 'dS with Anole lizard'\n",
       "\\item 'Anole lizard orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Arctic ground squirrel gene stable ID'\n",
       "\\item 'Arctic ground squirrel gene name'\n",
       "\\item 'Arctic ground squirrel protein or transcript stable ID'\n",
       "\\item 'Arctic ground squirrel chromosome/scaffold name'\n",
       "\\item 'Arctic ground squirrel chromosome/scaffold start (bp)'\n",
       "\\item 'Arctic ground squirrel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Arctic ground squirrel'\n",
       "\\item 'Arctic ground squirrel homology type'\n",
       "\\item '\\%id. target Arctic ground squirrel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Arctic ground squirrel gene'\n",
       "\\item 'Arctic ground squirrel Gene-order conservation score'\n",
       "\\item 'Arctic ground squirrel Whole-genome alignment coverage'\n",
       "\\item 'dN with Arctic ground squirrel'\n",
       "\\item 'dS with Arctic ground squirrel'\n",
       "\\item 'Arctic ground squirrel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Argentine black and white tegu gene stable ID'\n",
       "\\item 'Argentine black and white tegu gene name'\n",
       "\\item 'Argentine black and white tegu protein or transcript stable ID'\n",
       "\\item 'Argentine black and white tegu chromosome/scaffold name'\n",
       "\\item 'Argentine black and white tegu chromosome/scaffold start (bp)'\n",
       "\\item 'Argentine black and white tegu chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Argentine black and white tegu'\n",
       "\\item 'Argentine black and white tegu homology type'\n",
       "\\item '\\%id. target Argentine black and white tegu gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Argentine black and white tegu gene'\n",
       "\\item 'Argentine black and white tegu Gene-order conservation score'\n",
       "\\item 'Argentine black and white tegu Whole-genome alignment coverage'\n",
       "\\item 'dN with Argentine black and white tegu'\n",
       "\\item 'dS with Argentine black and white tegu'\n",
       "\\item 'Argentine black and white tegu orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Armadillo gene stable ID'\n",
       "\\item 'Armadillo gene name'\n",
       "\\item 'Armadillo protein or transcript stable ID'\n",
       "\\item 'Armadillo chromosome/scaffold name'\n",
       "\\item 'Armadillo chromosome/scaffold start (bp)'\n",
       "\\item 'Armadillo chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Armadillo'\n",
       "\\item 'Armadillo homology type'\n",
       "\\item '\\%id. target Armadillo gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Armadillo gene'\n",
       "\\item 'Armadillo Gene-order conservation score'\n",
       "\\item 'Armadillo Whole-genome alignment coverage'\n",
       "\\item 'dN with Armadillo'\n",
       "\\item 'dS with Armadillo'\n",
       "\\item 'Armadillo orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Asian bonytongue gene stable ID'\n",
       "\\item 'Asian bonytongue gene name'\n",
       "\\item 'Asian bonytongue protein or transcript stable ID'\n",
       "\\item 'Asian bonytongue chromosome/scaffold name'\n",
       "\\item 'Asian bonytongue chromosome/scaffold start (bp)'\n",
       "\\item 'Asian bonytongue chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Asian bonytongue'\n",
       "\\item 'Asian bonytongue homology type'\n",
       "\\item '\\%id. target Asian bonytongue gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Asian bonytongue gene'\n",
       "\\item 'Asian bonytongue Gene-order conservation score'\n",
       "\\item 'Asian bonytongue Whole-genome alignment coverage'\n",
       "\\item 'dN with Asian bonytongue'\n",
       "\\item 'dS with Asian bonytongue'\n",
       "\\item 'Asian bonytongue orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Australian saltwater crocodile gene stable ID'\n",
       "\\item 'Australian saltwater crocodile gene name'\n",
       "\\item 'Australian saltwater crocodile protein or transcript stable ID'\n",
       "\\item 'Australian saltwater crocodile chromosome/scaffold name'\n",
       "\\item 'Australian saltwater crocodile chromosome/scaffold start (bp)'\n",
       "\\item 'Australian saltwater crocodile chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Australian saltwater crocodile'\n",
       "\\item 'Australian saltwater crocodile homology type'\n",
       "\\item '\\%id. target Australian saltwater crocodile gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Australian saltwater crocodile gene'\n",
       "\\item 'Australian saltwater crocodile Gene-order conservation score'\n",
       "\\item 'Australian saltwater crocodile Whole-genome alignment coverage'\n",
       "\\item 'dN with Australian saltwater crocodile'\n",
       "\\item 'dS with Australian saltwater crocodile'\n",
       "\\item 'Australian saltwater crocodile orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ballan wrasse gene stable ID'\n",
       "\\item 'Ballan wrasse gene name'\n",
       "\\item 'Ballan wrasse protein or transcript stable ID'\n",
       "\\item 'Ballan wrasse chromosome/scaffold name'\n",
       "\\item 'Ballan wrasse chromosome/scaffold start (bp)'\n",
       "\\item 'Ballan wrasse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ballan wrasse'\n",
       "\\item 'Ballan wrasse homology type'\n",
       "\\item '\\%id. target Ballan wrasse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ballan wrasse gene'\n",
       "\\item 'Ballan wrasse Gene-order conservation score'\n",
       "\\item 'Ballan wrasse Whole-genome alignment coverage'\n",
       "\\item 'dN with Ballan wrasse'\n",
       "\\item 'dS with Ballan wrasse'\n",
       "\\item 'Ballan wrasse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Barramundi perch gene stable ID'\n",
       "\\item 'Barramundi perch gene name'\n",
       "\\item 'Barramundi perch protein or transcript stable ID'\n",
       "\\item 'Barramundi perch chromosome/scaffold name'\n",
       "\\item 'Barramundi perch chromosome/scaffold start (bp)'\n",
       "\\item 'Barramundi perch chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Barramundi perch'\n",
       "\\item 'Barramundi perch homology type'\n",
       "\\item '\\%id. target Barramundi perch gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Barramundi perch gene'\n",
       "\\item 'Barramundi perch Gene-order conservation score'\n",
       "\\item 'Barramundi perch Whole-genome alignment coverage'\n",
       "\\item 'dN with Barramundi perch'\n",
       "\\item 'dS with Barramundi perch'\n",
       "\\item 'Barramundi perch orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Bengalese finch gene stable ID'\n",
       "\\item 'Bengalese finch gene name'\n",
       "\\item 'Bengalese finch protein or transcript stable ID'\n",
       "\\item 'Bengalese finch chromosome/scaffold name'\n",
       "\\item 'Bengalese finch chromosome/scaffold start (bp)'\n",
       "\\item 'Bengalese finch chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Bengalese finch'\n",
       "\\item 'Bengalese finch homology type'\n",
       "\\item '\\%id. target Bengalese finch gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Bengalese finch gene'\n",
       "\\item 'Bengalese finch Gene-order conservation score'\n",
       "\\item 'Bengalese finch Whole-genome alignment coverage'\n",
       "\\item 'dN with Bengalese finch'\n",
       "\\item 'dS with Bengalese finch'\n",
       "\\item 'Bengalese finch orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Bicolor damselfish gene stable ID'\n",
       "\\item 'Bicolor damselfish gene name'\n",
       "\\item 'Bicolor damselfish protein or transcript stable ID'\n",
       "\\item 'Bicolor damselfish chromosome/scaffold name'\n",
       "\\item 'Bicolor damselfish chromosome/scaffold start (bp)'\n",
       "\\item 'Bicolor damselfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Bicolor damselfish'\n",
       "\\item 'Bicolor damselfish homology type'\n",
       "\\item '\\%id. target Bicolor damselfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Bicolor damselfish gene'\n",
       "\\item 'Bicolor damselfish Gene-order conservation score'\n",
       "\\item 'Bicolor damselfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Bicolor damselfish'\n",
       "\\item 'dS with Bicolor damselfish'\n",
       "\\item 'Bicolor damselfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Black snub-nosed monkey gene stable ID'\n",
       "\\item 'Black snub-nosed monkey gene name'\n",
       "\\item 'Black snub-nosed monkey protein or transcript stable ID'\n",
       "\\item 'Black snub-nosed monkey chromosome/scaffold name'\n",
       "\\item 'Black snub-nosed monkey chromosome/scaffold start (bp)'\n",
       "\\item 'Black snub-nosed monkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Black snub-nosed monkey'\n",
       "\\item 'Black snub-nosed monkey homology type'\n",
       "\\item '\\%id. target Black snub-nosed monkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Black snub-nosed monkey gene'\n",
       "\\item 'Black snub-nosed monkey Gene-order conservation score'\n",
       "\\item 'Black snub-nosed monkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Black snub-nosed monkey'\n",
       "\\item 'dS with Black snub-nosed monkey'\n",
       "\\item 'Black snub-nosed monkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Blue tit gene stable ID'\n",
       "\\item 'Blue tit gene name'\n",
       "\\item 'Blue tit protein or transcript stable ID'\n",
       "\\item 'Blue tit chromosome/scaffold name'\n",
       "\\item 'Blue tit chromosome/scaffold start (bp)'\n",
       "\\item 'Blue tit chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Blue tit'\n",
       "\\item 'Blue tit homology type'\n",
       "\\item '\\%id. target Blue tit gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Blue tit gene'\n",
       "\\item 'Blue tit Gene-order conservation score'\n",
       "\\item 'Blue tit Whole-genome alignment coverage'\n",
       "\\item 'dN with Blue tit'\n",
       "\\item 'dS with Blue tit'\n",
       "\\item 'Blue tit orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Blue-crowned manakin gene stable ID'\n",
       "\\item 'Blue-crowned manakin gene name'\n",
       "\\item 'Blue-crowned manakin protein or transcript stable ID'\n",
       "\\item 'Blue-crowned manakin chromosome/scaffold name'\n",
       "\\item 'Blue-crowned manakin chromosome/scaffold start (bp)'\n",
       "\\item 'Blue-crowned manakin chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Blue-crowned manakin'\n",
       "\\item 'Blue-crowned manakin homology type'\n",
       "\\item '\\%id. target Blue-crowned manakin gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Blue-crowned manakin gene'\n",
       "\\item 'Blue-crowned manakin Gene-order conservation score'\n",
       "\\item 'Blue-crowned manakin Whole-genome alignment coverage'\n",
       "\\item 'dN with Blue-crowned manakin'\n",
       "\\item 'dS with Blue-crowned manakin'\n",
       "\\item 'Blue-crowned manakin orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Bolivian squirrel monkey gene stable ID'\n",
       "\\item 'Bolivian squirrel monkey gene name'\n",
       "\\item 'Bolivian squirrel monkey protein or transcript stable ID'\n",
       "\\item 'Bolivian squirrel monkey chromosome/scaffold name'\n",
       "\\item 'Bolivian squirrel monkey chromosome/scaffold start (bp)'\n",
       "\\item 'Bolivian squirrel monkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Bolivian squirrel monkey'\n",
       "\\item 'Bolivian squirrel monkey homology type'\n",
       "\\item '\\%id. target Bolivian squirrel monkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Bolivian squirrel monkey gene'\n",
       "\\item 'Bolivian squirrel monkey Gene-order conservation score'\n",
       "\\item 'Bolivian squirrel monkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Bolivian squirrel monkey'\n",
       "\\item 'dS with Bolivian squirrel monkey'\n",
       "\\item 'Bolivian squirrel monkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Bonobo gene stable ID'\n",
       "\\item 'Bonobo gene name'\n",
       "\\item 'Bonobo protein or transcript stable ID'\n",
       "\\item 'Bonobo chromosome/scaffold name'\n",
       "\\item 'Bonobo chromosome/scaffold start (bp)'\n",
       "\\item 'Bonobo chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Bonobo'\n",
       "\\item 'Bonobo homology type'\n",
       "\\item '\\%id. target Bonobo gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Bonobo gene'\n",
       "\\item 'Bonobo Gene-order conservation score'\n",
       "\\item 'Bonobo Whole-genome alignment coverage'\n",
       "\\item 'dN with Bonobo'\n",
       "\\item 'dS with Bonobo'\n",
       "\\item 'Bonobo orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Brazilian guinea pig gene stable ID'\n",
       "\\item 'Brazilian guinea pig gene name'\n",
       "\\item 'Brazilian guinea pig protein or transcript stable ID'\n",
       "\\item 'Brazilian guinea pig chromosome/scaffold name'\n",
       "\\item 'Brazilian guinea pig chromosome/scaffold start (bp)'\n",
       "\\item 'Brazilian guinea pig chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Brazilian guinea pig'\n",
       "\\item 'Brazilian guinea pig homology type'\n",
       "\\item '\\%id. target Brazilian guinea pig gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Brazilian guinea pig gene'\n",
       "\\item 'Brazilian guinea pig Gene-order conservation score'\n",
       "\\item 'Brazilian guinea pig Whole-genome alignment coverage'\n",
       "\\item 'dN with Brazilian guinea pig'\n",
       "\\item 'dS with Brazilian guinea pig'\n",
       "\\item 'Brazilian guinea pig orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Budgerigar gene stable ID'\n",
       "\\item 'Budgerigar gene name'\n",
       "\\item 'Budgerigar protein or transcript stable ID'\n",
       "\\item 'Budgerigar chromosome/scaffold name'\n",
       "\\item 'Budgerigar chromosome/scaffold start (bp)'\n",
       "\\item 'Budgerigar chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Budgerigar'\n",
       "\\item 'Budgerigar homology type'\n",
       "\\item '\\%id. target Budgerigar gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Budgerigar gene'\n",
       "\\item 'Budgerigar Gene-order conservation score'\n",
       "\\item 'Budgerigar Whole-genome alignment coverage'\n",
       "\\item 'dN with Budgerigar'\n",
       "\\item 'dS with Budgerigar'\n",
       "\\item 'Budgerigar orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder gene stable ID'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder gene name'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder protein or transcript stable ID'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder chromosome/scaffold name'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder chromosome/scaffold start (bp)'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Burton\\textbackslash{}'s mouthbrooder'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder homology type'\n",
       "\\item '\\%id. target Burton\\textbackslash{}'s mouthbrooder gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Burton\\textbackslash{}'s mouthbrooder gene'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder Gene-order conservation score'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder Whole-genome alignment coverage'\n",
       "\\item 'dN with Burton\\textbackslash{}'s mouthbrooder'\n",
       "\\item 'dS with Burton\\textbackslash{}'s mouthbrooder'\n",
       "\\item 'Burton\\textbackslash{}'s mouthbrooder orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Bushbaby gene stable ID'\n",
       "\\item 'Bushbaby gene name'\n",
       "\\item 'Bushbaby protein or transcript stable ID'\n",
       "\\item 'Bushbaby chromosome/scaffold name'\n",
       "\\item 'Bushbaby chromosome/scaffold start (bp)'\n",
       "\\item 'Bushbaby chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Bushbaby'\n",
       "\\item 'Bushbaby homology type'\n",
       "\\item '\\%id. target Bushbaby gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Bushbaby gene'\n",
       "\\item 'Bushbaby Gene-order conservation score'\n",
       "\\item 'Bushbaby Whole-genome alignment coverage'\n",
       "\\item 'dN with Bushbaby'\n",
       "\\item 'dS with Bushbaby'\n",
       "\\item 'Bushbaby orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'C.intestinalis gene stable ID'\n",
       "\\item 'C.intestinalis gene name'\n",
       "\\item 'C.intestinalis protein or transcript stable ID'\n",
       "\\item 'C.intestinalis chromosome/scaffold name'\n",
       "\\item 'C.intestinalis chromosome/scaffold start (bp)'\n",
       "\\item 'C.intestinalis chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with C.intestinalis'\n",
       "\\item 'C.intestinalis homology type'\n",
       "\\item '\\%id. target C.intestinalis gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target C.intestinalis gene'\n",
       "\\item 'C.intestinalis Gene-order conservation score'\n",
       "\\item 'C.intestinalis Whole-genome alignment coverage'\n",
       "\\item 'dN with C.intestinalis'\n",
       "\\item 'dS with C.intestinalis'\n",
       "\\item 'C.intestinalis orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'C.savignyi gene stable ID'\n",
       "\\item 'C.savignyi gene name'\n",
       "\\item 'C.savignyi protein or transcript stable ID'\n",
       "\\item 'C.savignyi chromosome/scaffold name'\n",
       "\\item 'C.savignyi chromosome/scaffold start (bp)'\n",
       "\\item 'C.savignyi chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with C.savignyi'\n",
       "\\item 'C.savignyi homology type'\n",
       "\\item '\\%id. target C.savignyi gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target C.savignyi gene'\n",
       "\\item 'C.savignyi Gene-order conservation score'\n",
       "\\item 'C.savignyi Whole-genome alignment coverage'\n",
       "\\item 'dN with C.savignyi'\n",
       "\\item 'dS with C.savignyi'\n",
       "\\item 'C.savignyi orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Caenorhabditis elegans gene stable ID'\n",
       "\\item 'Caenorhabditis elegans gene name'\n",
       "\\item 'Caenorhabditis elegans protein or transcript stable ID'\n",
       "\\item 'Caenorhabditis elegans chromosome/scaffold name'\n",
       "\\item 'Caenorhabditis elegans chromosome/scaffold start (bp)'\n",
       "\\item 'Caenorhabditis elegans chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Caenorhabditis elegans'\n",
       "\\item 'Caenorhabditis elegans homology type'\n",
       "\\item '\\%id. target Caenorhabditis elegans gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Caenorhabditis elegans gene'\n",
       "\\item 'Caenorhabditis elegans Gene-order conservation score'\n",
       "\\item 'Caenorhabditis elegans Whole-genome alignment coverage'\n",
       "\\item 'dN with Caenorhabditis elegans'\n",
       "\\item 'dS with Caenorhabditis elegans'\n",
       "\\item 'Caenorhabditis elegans orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Capuchin gene stable ID'\n",
       "\\item 'Capuchin gene name'\n",
       "\\item 'Capuchin protein or transcript stable ID'\n",
       "\\item 'Capuchin chromosome/scaffold name'\n",
       "\\item 'Capuchin chromosome/scaffold start (bp)'\n",
       "\\item 'Capuchin chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Capuchin'\n",
       "\\item 'Capuchin homology type'\n",
       "\\item '\\%id. target Capuchin gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Capuchin gene'\n",
       "\\item 'Capuchin Gene-order conservation score'\n",
       "\\item 'Capuchin Whole-genome alignment coverage'\n",
       "\\item 'dN with Capuchin'\n",
       "\\item 'dS with Capuchin'\n",
       "\\item 'Capuchin orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Cat gene stable ID'\n",
       "\\item 'Cat gene name'\n",
       "\\item 'Cat protein or transcript stable ID'\n",
       "\\item 'Cat chromosome/scaffold name'\n",
       "\\item 'Cat chromosome/scaffold start (bp)'\n",
       "\\item 'Cat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Cat'\n",
       "\\item 'Cat homology type'\n",
       "\\item '\\%id. target Cat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Cat gene'\n",
       "\\item 'Cat Gene-order conservation score'\n",
       "\\item 'Cat Whole-genome alignment coverage'\n",
       "\\item 'dN with Cat'\n",
       "\\item 'dS with Cat'\n",
       "\\item 'Cat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Central bearded dragon gene stable ID'\n",
       "\\item 'Central bearded dragon gene name'\n",
       "\\item 'Central bearded dragon protein or transcript stable ID'\n",
       "\\item 'Central bearded dragon chromosome/scaffold name'\n",
       "\\item 'Central bearded dragon chromosome/scaffold start (bp)'\n",
       "\\item 'Central bearded dragon chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Central bearded dragon'\n",
       "\\item 'Central bearded dragon homology type'\n",
       "\\item '\\%id. target Central bearded dragon gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Central bearded dragon gene'\n",
       "\\item 'Central bearded dragon Gene-order conservation score'\n",
       "\\item 'Central bearded dragon Whole-genome alignment coverage'\n",
       "\\item 'dN with Central bearded dragon'\n",
       "\\item 'dS with Central bearded dragon'\n",
       "\\item 'Central bearded dragon orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Channel catfish gene stable ID'\n",
       "\\item 'Channel catfish gene name'\n",
       "\\item 'Channel catfish protein or transcript stable ID'\n",
       "\\item 'Channel catfish chromosome/scaffold name'\n",
       "\\item 'Channel catfish chromosome/scaffold start (bp)'\n",
       "\\item 'Channel catfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Channel catfish'\n",
       "\\item 'Channel catfish homology type'\n",
       "\\item '\\%id. target Channel catfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Channel catfish gene'\n",
       "\\item 'Channel catfish Gene-order conservation score'\n",
       "\\item 'Channel catfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Channel catfish'\n",
       "\\item 'dS with Channel catfish'\n",
       "\\item 'Channel catfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chicken gene stable ID'\n",
       "\\item 'Chicken gene name'\n",
       "\\item 'Chicken protein or transcript stable ID'\n",
       "\\item 'Chicken chromosome/scaffold name'\n",
       "\\item 'Chicken chromosome/scaffold start (bp)'\n",
       "\\item 'Chicken chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chicken'\n",
       "\\item 'Chicken homology type'\n",
       "\\item '\\%id. target Chicken gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chicken gene'\n",
       "\\item 'Chicken Gene-order conservation score'\n",
       "\\item 'Chicken Whole-genome alignment coverage'\n",
       "\\item 'dN with Chicken'\n",
       "\\item 'dS with Chicken'\n",
       "\\item 'Chicken orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chilean tinamou gene stable ID'\n",
       "\\item 'Chilean tinamou gene name'\n",
       "\\item 'Chilean tinamou protein or transcript stable ID'\n",
       "\\item 'Chilean tinamou chromosome/scaffold name'\n",
       "\\item 'Chilean tinamou chromosome/scaffold start (bp)'\n",
       "\\item 'Chilean tinamou chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chilean tinamou'\n",
       "\\item 'Chilean tinamou homology type'\n",
       "\\item '\\%id. target Chilean tinamou gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chilean tinamou gene'\n",
       "\\item 'Chilean tinamou Gene-order conservation score'\n",
       "\\item 'Chilean tinamou Whole-genome alignment coverage'\n",
       "\\item 'dN with Chilean tinamou'\n",
       "\\item 'dS with Chilean tinamou'\n",
       "\\item 'Chilean tinamou orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chimpanzee gene stable ID'\n",
       "\\item 'Chimpanzee gene name'\n",
       "\\item 'Chimpanzee protein or transcript stable ID'\n",
       "\\item 'Chimpanzee chromosome/scaffold name'\n",
       "\\item 'Chimpanzee chromosome/scaffold start (bp)'\n",
       "\\item 'Chimpanzee chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chimpanzee'\n",
       "\\item 'Chimpanzee homology type'\n",
       "\\item '\\%id. target Chimpanzee gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chimpanzee gene'\n",
       "\\item 'Chimpanzee Gene-order conservation score'\n",
       "\\item 'Chimpanzee Whole-genome alignment coverage'\n",
       "\\item 'dN with Chimpanzee'\n",
       "\\item 'dS with Chimpanzee'\n",
       "\\item 'Chimpanzee orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chinese hamster CHOK1GS gene stable ID'\n",
       "\\item 'Chinese hamster CHOK1GS gene name'\n",
       "\\item 'Chinese hamster CHOK1GS protein or transcript stable ID'\n",
       "\\item 'Chinese hamster CHOK1GS chromosome/scaffold name'\n",
       "\\item 'Chinese hamster CHOK1GS chromosome/scaffold start (bp)'\n",
       "\\item 'Chinese hamster CHOK1GS chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chinese hamster CHOK1GS'\n",
       "\\item 'Chinese hamster CHOK1GS homology type'\n",
       "\\item '\\%id. target Chinese hamster CHOK1GS gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chinese hamster CHOK1GS gene'\n",
       "\\item 'Chinese hamster CHOK1GS Gene-order conservation score'\n",
       "\\item 'Chinese hamster CHOK1GS Whole-genome alignment coverage'\n",
       "\\item 'dN with Chinese hamster CHOK1GS'\n",
       "\\item 'dS with Chinese hamster CHOK1GS'\n",
       "\\item 'Chinese hamster CHOK1GS orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chinese hamster CriGri gene stable ID'\n",
       "\\item 'Chinese hamster CriGri gene name'\n",
       "\\item 'Chinese hamster CriGri protein or transcript stable ID'\n",
       "\\item 'Chinese hamster CriGri chromosome/scaffold name'\n",
       "\\item 'Chinese hamster CriGri chromosome/scaffold start (bp)'\n",
       "\\item 'Chinese hamster CriGri chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chinese hamster CriGri'\n",
       "\\item 'Chinese hamster CriGri homology type'\n",
       "\\item '\\%id. target Chinese hamster CriGri gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chinese hamster CriGri gene'\n",
       "\\item 'Chinese hamster CriGri Gene-order conservation score'\n",
       "\\item 'Chinese hamster CriGri Whole-genome alignment coverage'\n",
       "\\item 'dN with Chinese hamster CriGri'\n",
       "\\item 'dS with Chinese hamster CriGri'\n",
       "\\item 'Chinese hamster CriGri orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chinese hamster PICR gene stable ID'\n",
       "\\item 'Chinese hamster PICR gene name'\n",
       "\\item 'Chinese hamster PICR protein or transcript stable ID'\n",
       "\\item 'Chinese hamster PICR chromosome/scaffold name'\n",
       "\\item 'Chinese hamster PICR chromosome/scaffold start (bp)'\n",
       "\\item 'Chinese hamster PICR chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chinese hamster PICR'\n",
       "\\item 'Chinese hamster PICR homology type'\n",
       "\\item '\\%id. target Chinese hamster PICR gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chinese hamster PICR gene'\n",
       "\\item 'Chinese hamster PICR Gene-order conservation score'\n",
       "\\item 'Chinese hamster PICR Whole-genome alignment coverage'\n",
       "\\item 'dN with Chinese hamster PICR'\n",
       "\\item 'dS with Chinese hamster PICR'\n",
       "\\item 'Chinese hamster PICR orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Chinese softshell turtle gene stable ID'\n",
       "\\item 'Chinese softshell turtle gene name'\n",
       "\\item 'Chinese softshell turtle protein or transcript stable ID'\n",
       "\\item 'Chinese softshell turtle chromosome/scaffold name'\n",
       "\\item 'Chinese softshell turtle chromosome/scaffold start (bp)'\n",
       "\\item 'Chinese softshell turtle chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Chinese softshell turtle'\n",
       "\\item 'Chinese softshell turtle homology type'\n",
       "\\item '\\%id. target Chinese softshell turtle gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Chinese softshell turtle gene'\n",
       "\\item 'Chinese softshell turtle Gene-order conservation score'\n",
       "\\item 'Chinese softshell turtle Whole-genome alignment coverage'\n",
       "\\item 'dN with Chinese softshell turtle'\n",
       "\\item 'dS with Chinese softshell turtle'\n",
       "\\item 'Chinese softshell turtle orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Climbing perch gene stable ID'\n",
       "\\item 'Climbing perch gene name'\n",
       "\\item 'Climbing perch protein or transcript stable ID'\n",
       "\\item 'Climbing perch chromosome/scaffold name'\n",
       "\\item 'Climbing perch chromosome/scaffold start (bp)'\n",
       "\\item 'Climbing perch chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Climbing perch'\n",
       "\\item 'Climbing perch homology type'\n",
       "\\item '\\%id. target Climbing perch gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Climbing perch gene'\n",
       "\\item 'Climbing perch Gene-order conservation score'\n",
       "\\item 'Climbing perch Whole-genome alignment coverage'\n",
       "\\item 'dN with Climbing perch'\n",
       "\\item 'dS with Climbing perch'\n",
       "\\item 'Climbing perch orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Clown anemonefish gene stable ID'\n",
       "\\item 'Clown anemonefish gene name'\n",
       "\\item 'Clown anemonefish protein or transcript stable ID'\n",
       "\\item 'Clown anemonefish chromosome/scaffold name'\n",
       "\\item 'Clown anemonefish chromosome/scaffold start (bp)'\n",
       "\\item 'Clown anemonefish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Clown anemonefish'\n",
       "\\item 'Clown anemonefish homology type'\n",
       "\\item '\\%id. target Clown anemonefish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Clown anemonefish gene'\n",
       "\\item 'Clown anemonefish Gene-order conservation score'\n",
       "\\item 'Clown anemonefish Whole-genome alignment coverage'\n",
       "\\item 'dN with Clown anemonefish'\n",
       "\\item 'dS with Clown anemonefish'\n",
       "\\item 'Clown anemonefish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Cod gene stable ID'\n",
       "\\item 'Cod gene name'\n",
       "\\item 'Cod protein or transcript stable ID'\n",
       "\\item 'Cod chromosome/scaffold name'\n",
       "\\item 'Cod chromosome/scaffold start (bp)'\n",
       "\\item 'Cod chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Cod'\n",
       "\\item 'Cod homology type'\n",
       "\\item '\\%id. target Cod gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Cod gene'\n",
       "\\item 'Cod Gene-order conservation score'\n",
       "\\item 'Cod Whole-genome alignment coverage'\n",
       "\\item 'dN with Cod'\n",
       "\\item 'dS with Cod'\n",
       "\\item 'Cod orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Coelacanth gene stable ID'\n",
       "\\item 'Coelacanth gene name'\n",
       "\\item 'Coelacanth protein or transcript stable ID'\n",
       "\\item 'Coelacanth chromosome/scaffold name'\n",
       "\\item 'Coelacanth chromosome/scaffold start (bp)'\n",
       "\\item 'Coelacanth chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Coelacanth'\n",
       "\\item 'Coelacanth homology type'\n",
       "\\item '\\%id. target Coelacanth gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Coelacanth gene'\n",
       "\\item 'Coelacanth Gene-order conservation score'\n",
       "\\item 'Coelacanth Whole-genome alignment coverage'\n",
       "\\item 'dN with Coelacanth'\n",
       "\\item 'dS with Coelacanth'\n",
       "\\item 'Coelacanth orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Common canary gene stable ID'\n",
       "\\item 'Common canary gene name'\n",
       "\\item 'Common canary protein or transcript stable ID'\n",
       "\\item 'Common canary chromosome/scaffold name'\n",
       "\\item 'Common canary chromosome/scaffold start (bp)'\n",
       "\\item 'Common canary chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Common canary'\n",
       "\\item 'Common canary homology type'\n",
       "\\item '\\%id. target Common canary gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Common canary gene'\n",
       "\\item 'Common canary Gene-order conservation score'\n",
       "\\item 'Common canary Whole-genome alignment coverage'\n",
       "\\item 'dN with Common canary'\n",
       "\\item 'dS with Common canary'\n",
       "\\item 'Common canary orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Common wombat gene stable ID'\n",
       "\\item 'Common wombat gene name'\n",
       "\\item 'Common wombat protein or transcript stable ID'\n",
       "\\item 'Common wombat chromosome/scaffold name'\n",
       "\\item 'Common wombat chromosome/scaffold start (bp)'\n",
       "\\item 'Common wombat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Common wombat'\n",
       "\\item 'Common wombat homology type'\n",
       "\\item '\\%id. target Common wombat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Common wombat gene'\n",
       "\\item 'Common wombat Gene-order conservation score'\n",
       "\\item 'Common wombat Whole-genome alignment coverage'\n",
       "\\item 'dN with Common wombat'\n",
       "\\item 'dS with Common wombat'\n",
       "\\item 'Common wombat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka gene stable ID'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka gene name'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka protein or transcript stable ID'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka chromosome/scaffold name'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka chromosome/scaffold start (bp)'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Coquerel\\textbackslash{}'s sifaka'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka homology type'\n",
       "\\item '\\%id. target Coquerel\\textbackslash{}'s sifaka gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Coquerel\\textbackslash{}'s sifaka gene'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka Gene-order conservation score'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka Whole-genome alignment coverage'\n",
       "\\item 'dN with Coquerel\\textbackslash{}'s sifaka'\n",
       "\\item 'dS with Coquerel\\textbackslash{}'s sifaka'\n",
       "\\item 'Coquerel\\textbackslash{}'s sifaka orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Cow gene stable ID'\n",
       "\\item 'Cow gene name'\n",
       "\\item 'Cow protein or transcript stable ID'\n",
       "\\item 'Cow chromosome/scaffold name'\n",
       "\\item 'Cow chromosome/scaffold start (bp)'\n",
       "\\item 'Cow chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Cow'\n",
       "\\item 'Cow homology type'\n",
       "\\item '\\%id. target Cow gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Cow gene'\n",
       "\\item 'Cow Gene-order conservation score'\n",
       "\\item 'Cow Whole-genome alignment coverage'\n",
       "\\item 'dN with Cow'\n",
       "\\item 'dS with Cow'\n",
       "\\item 'Cow orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Crab-eating macaque gene stable ID'\n",
       "\\item 'Crab-eating macaque gene name'\n",
       "\\item 'Crab-eating macaque protein or transcript stable ID'\n",
       "\\item 'Crab-eating macaque chromosome/scaffold name'\n",
       "\\item 'Crab-eating macaque chromosome/scaffold start (bp)'\n",
       "\\item 'Crab-eating macaque chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Crab-eating macaque'\n",
       "\\item 'Crab-eating macaque homology type'\n",
       "\\item '\\%id. target Crab-eating macaque gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Crab-eating macaque gene'\n",
       "\\item 'Crab-eating macaque Gene-order conservation score'\n",
       "\\item 'Crab-eating macaque Whole-genome alignment coverage'\n",
       "\\item 'dN with Crab-eating macaque'\n",
       "\\item 'dS with Crab-eating macaque'\n",
       "\\item 'Crab-eating macaque orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Damara mole rat gene stable ID'\n",
       "\\item 'Damara mole rat gene name'\n",
       "\\item 'Damara mole rat protein or transcript stable ID'\n",
       "\\item 'Damara mole rat chromosome/scaffold name'\n",
       "\\item 'Damara mole rat chromosome/scaffold start (bp)'\n",
       "\\item 'Damara mole rat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Damara mole rat'\n",
       "\\item 'Damara mole rat homology type'\n",
       "\\item '\\%id. target Damara mole rat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Damara mole rat gene'\n",
       "\\item 'Damara mole rat Gene-order conservation score'\n",
       "\\item 'Damara mole rat Whole-genome alignment coverage'\n",
       "\\item 'dN with Damara mole rat'\n",
       "\\item 'dS with Damara mole rat'\n",
       "\\item 'Damara mole rat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Dark-eyed junco gene stable ID'\n",
       "\\item 'Dark-eyed junco gene name'\n",
       "\\item 'Dark-eyed junco protein or transcript stable ID'\n",
       "\\item 'Dark-eyed junco chromosome/scaffold name'\n",
       "\\item 'Dark-eyed junco chromosome/scaffold start (bp)'\n",
       "\\item 'Dark-eyed junco chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Dark-eyed junco'\n",
       "\\item 'Dark-eyed junco homology type'\n",
       "\\item '\\%id. target Dark-eyed junco gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Dark-eyed junco gene'\n",
       "\\item 'Dark-eyed junco Gene-order conservation score'\n",
       "\\item 'Dark-eyed junco Whole-genome alignment coverage'\n",
       "\\item 'dN with Dark-eyed junco'\n",
       "\\item 'dS with Dark-eyed junco'\n",
       "\\item 'Dark-eyed junco orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Daurian ground squirrel gene stable ID'\n",
       "\\item 'Daurian ground squirrel gene name'\n",
       "\\item 'Daurian ground squirrel protein or transcript stable ID'\n",
       "\\item 'Daurian ground squirrel chromosome/scaffold name'\n",
       "\\item 'Daurian ground squirrel chromosome/scaffold start (bp)'\n",
       "\\item 'Daurian ground squirrel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Daurian ground squirrel'\n",
       "\\item 'Daurian ground squirrel homology type'\n",
       "\\item '\\%id. target Daurian ground squirrel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Daurian ground squirrel gene'\n",
       "\\item 'Daurian ground squirrel Gene-order conservation score'\n",
       "\\item 'Daurian ground squirrel Whole-genome alignment coverage'\n",
       "\\item 'dN with Daurian ground squirrel'\n",
       "\\item 'dS with Daurian ground squirrel'\n",
       "\\item 'Daurian ground squirrel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Degu gene stable ID'\n",
       "\\item 'Degu gene name'\n",
       "\\item 'Degu protein or transcript stable ID'\n",
       "\\item 'Degu chromosome/scaffold name'\n",
       "\\item 'Degu chromosome/scaffold start (bp)'\n",
       "\\item 'Degu chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Degu'\n",
       "\\item 'Degu homology type'\n",
       "\\item '\\%id. target Degu gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Degu gene'\n",
       "\\item 'Degu Gene-order conservation score'\n",
       "\\item 'Degu Whole-genome alignment coverage'\n",
       "\\item 'dN with Degu'\n",
       "\\item 'dS with Degu'\n",
       "\\item 'Degu orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Dingo gene stable ID'\n",
       "\\item 'Dingo gene name'\n",
       "\\item 'Dingo protein or transcript stable ID'\n",
       "\\item 'Dingo chromosome/scaffold name'\n",
       "\\item 'Dingo chromosome/scaffold start (bp)'\n",
       "\\item 'Dingo chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Dingo'\n",
       "\\item 'Dingo homology type'\n",
       "\\item '\\%id. target Dingo gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Dingo gene'\n",
       "\\item 'Dingo Gene-order conservation score'\n",
       "\\item 'Dingo Whole-genome alignment coverage'\n",
       "\\item 'dN with Dingo'\n",
       "\\item 'dS with Dingo'\n",
       "\\item 'Dingo orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Dog gene stable ID'\n",
       "\\item 'Dog gene name'\n",
       "\\item 'Dog protein or transcript stable ID'\n",
       "\\item 'Dog chromosome/scaffold name'\n",
       "\\item 'Dog chromosome/scaffold start (bp)'\n",
       "\\item 'Dog chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Dog'\n",
       "\\item 'Dog homology type'\n",
       "\\item '\\%id. target Dog gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Dog gene'\n",
       "\\item 'Dog Gene-order conservation score'\n",
       "\\item 'Dog Whole-genome alignment coverage'\n",
       "\\item 'dN with Dog'\n",
       "\\item 'dS with Dog'\n",
       "\\item 'Dog orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Dolphin gene stable ID'\n",
       "\\item 'Dolphin gene name'\n",
       "\\item 'Dolphin protein or transcript stable ID'\n",
       "\\item 'Dolphin chromosome/scaffold name'\n",
       "\\item 'Dolphin chromosome/scaffold start (bp)'\n",
       "\\item 'Dolphin chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Dolphin'\n",
       "\\item 'Dolphin homology type'\n",
       "\\item '\\%id. target Dolphin gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Dolphin gene'\n",
       "\\item 'Dolphin Gene-order conservation score'\n",
       "\\item 'Dolphin Whole-genome alignment coverage'\n",
       "\\item 'dN with Dolphin'\n",
       "\\item 'dS with Dolphin'\n",
       "\\item 'Dolphin orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Donkey gene stable ID'\n",
       "\\item 'Donkey gene name'\n",
       "\\item 'Donkey protein or transcript stable ID'\n",
       "\\item 'Donkey chromosome/scaffold name'\n",
       "\\item 'Donkey chromosome/scaffold start (bp)'\n",
       "\\item 'Donkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Donkey'\n",
       "\\item 'Donkey homology type'\n",
       "\\item '\\%id. target Donkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Donkey gene'\n",
       "\\item 'Donkey Gene-order conservation score'\n",
       "\\item 'Donkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Donkey'\n",
       "\\item 'dS with Donkey'\n",
       "\\item 'Donkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Drill gene stable ID'\n",
       "\\item 'Drill gene name'\n",
       "\\item 'Drill protein or transcript stable ID'\n",
       "\\item 'Drill chromosome/scaffold name'\n",
       "\\item 'Drill chromosome/scaffold start (bp)'\n",
       "\\item 'Drill chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Drill'\n",
       "\\item 'Drill homology type'\n",
       "\\item '\\%id. target Drill gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Drill gene'\n",
       "\\item 'Drill Gene-order conservation score'\n",
       "\\item 'Drill Whole-genome alignment coverage'\n",
       "\\item 'dN with Drill'\n",
       "\\item 'dS with Drill'\n",
       "\\item 'Drill orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Drosophila melanogaster gene stable ID'\n",
       "\\item 'Drosophila melanogaster gene name'\n",
       "\\item 'Drosophila melanogaster protein or transcript stable ID'\n",
       "\\item 'Drosophila melanogaster chromosome/scaffold name'\n",
       "\\item 'Drosophila melanogaster chromosome/scaffold start (bp)'\n",
       "\\item 'Drosophila melanogaster chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Drosophila melanogaster'\n",
       "\\item 'Drosophila melanogaster homology type'\n",
       "\\item '\\%id. target Drosophila melanogaster gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Drosophila melanogaster gene'\n",
       "\\item 'Drosophila melanogaster Gene-order conservation score'\n",
       "\\item 'Drosophila melanogaster Whole-genome alignment coverage'\n",
       "\\item 'dN with Drosophila melanogaster'\n",
       "\\item 'dS with Drosophila melanogaster'\n",
       "\\item 'Drosophila melanogaster orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Duck gene stable ID'\n",
       "\\item 'Duck gene name'\n",
       "\\item 'Duck protein or transcript stable ID'\n",
       "\\item 'Duck chromosome/scaffold name'\n",
       "\\item 'Duck chromosome/scaffold start (bp)'\n",
       "\\item 'Duck chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Duck'\n",
       "\\item 'Duck homology type'\n",
       "\\item '\\%id. target Duck gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Duck gene'\n",
       "\\item 'Duck Gene-order conservation score'\n",
       "\\item 'Duck Whole-genome alignment coverage'\n",
       "\\item 'dN with Duck'\n",
       "\\item 'dS with Duck'\n",
       "\\item 'Duck orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Eastern happy gene stable ID'\n",
       "\\item 'Eastern happy gene name'\n",
       "\\item 'Eastern happy protein or transcript stable ID'\n",
       "\\item 'Eastern happy chromosome/scaffold name'\n",
       "\\item 'Eastern happy chromosome/scaffold start (bp)'\n",
       "\\item 'Eastern happy chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Eastern happy'\n",
       "\\item 'Eastern happy homology type'\n",
       "\\item '\\%id. target Eastern happy gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Eastern happy gene'\n",
       "\\item 'Eastern happy Gene-order conservation score'\n",
       "\\item 'Eastern happy Whole-genome alignment coverage'\n",
       "\\item 'dN with Eastern happy'\n",
       "\\item 'dS with Eastern happy'\n",
       "\\item 'Eastern happy orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Electric eel gene stable ID'\n",
       "\\item 'Electric eel gene name'\n",
       "\\item 'Electric eel protein or transcript stable ID'\n",
       "\\item 'Electric eel chromosome/scaffold name'\n",
       "\\item 'Electric eel chromosome/scaffold start (bp)'\n",
       "\\item 'Electric eel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Electric eel'\n",
       "\\item 'Electric eel homology type'\n",
       "\\item '\\%id. target Electric eel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Electric eel gene'\n",
       "\\item 'Electric eel Gene-order conservation score'\n",
       "\\item 'Electric eel Whole-genome alignment coverage'\n",
       "\\item 'dN with Electric eel'\n",
       "\\item 'dS with Electric eel'\n",
       "\\item 'Electric eel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Elephant gene stable ID'\n",
       "\\item 'Elephant gene name'\n",
       "\\item 'Elephant protein or transcript stable ID'\n",
       "\\item 'Elephant chromosome/scaffold name'\n",
       "\\item 'Elephant chromosome/scaffold start (bp)'\n",
       "\\item 'Elephant chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Elephant'\n",
       "\\item 'Elephant homology type'\n",
       "\\item '\\%id. target Elephant gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Elephant gene'\n",
       "\\item 'Elephant Gene-order conservation score'\n",
       "\\item 'Elephant Whole-genome alignment coverage'\n",
       "\\item 'dN with Elephant'\n",
       "\\item 'dS with Elephant'\n",
       "\\item 'Elephant orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Elephant shark gene stable ID'\n",
       "\\item 'Elephant shark gene name'\n",
       "\\item 'Elephant shark protein or transcript stable ID'\n",
       "\\item 'Elephant shark chromosome/scaffold name'\n",
       "\\item 'Elephant shark chromosome/scaffold start (bp)'\n",
       "\\item 'Elephant shark chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Elephant shark'\n",
       "\\item 'Elephant shark homology type'\n",
       "\\item '\\%id. target Elephant shark gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Elephant shark gene'\n",
       "\\item 'Elephant shark Gene-order conservation score'\n",
       "\\item 'Elephant shark Whole-genome alignment coverage'\n",
       "\\item 'dN with Elephant shark'\n",
       "\\item 'dS with Elephant shark'\n",
       "\\item 'Elephant shark orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Emu gene stable ID'\n",
       "\\item 'Emu gene name'\n",
       "\\item 'Emu protein or transcript stable ID'\n",
       "\\item 'Emu chromosome/scaffold name'\n",
       "\\item 'Emu chromosome/scaffold start (bp)'\n",
       "\\item 'Emu chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Emu'\n",
       "\\item 'Emu homology type'\n",
       "\\item '\\%id. target Emu gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Emu gene'\n",
       "\\item 'Emu Gene-order conservation score'\n",
       "\\item 'Emu Whole-genome alignment coverage'\n",
       "\\item 'dN with Emu'\n",
       "\\item 'dS with Emu'\n",
       "\\item 'Emu orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ferret gene stable ID'\n",
       "\\item 'Ferret gene name'\n",
       "\\item 'Ferret protein or transcript stable ID'\n",
       "\\item 'Ferret chromosome/scaffold name'\n",
       "\\item 'Ferret chromosome/scaffold start (bp)'\n",
       "\\item 'Ferret chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ferret'\n",
       "\\item 'Ferret homology type'\n",
       "\\item '\\%id. target Ferret gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ferret gene'\n",
       "\\item 'Ferret Gene-order conservation score'\n",
       "\\item 'Ferret Whole-genome alignment coverage'\n",
       "\\item 'dN with Ferret'\n",
       "\\item 'dS with Ferret'\n",
       "\\item 'Ferret orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Flycatcher gene stable ID'\n",
       "\\item 'Flycatcher gene name'\n",
       "\\item 'Flycatcher protein or transcript stable ID'\n",
       "\\item 'Flycatcher chromosome/scaffold name'\n",
       "\\item 'Flycatcher chromosome/scaffold start (bp)'\n",
       "\\item 'Flycatcher chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Flycatcher'\n",
       "\\item 'Flycatcher homology type'\n",
       "\\item '\\%id. target Flycatcher gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Flycatcher gene'\n",
       "\\item 'Flycatcher Gene-order conservation score'\n",
       "\\item 'Flycatcher Whole-genome alignment coverage'\n",
       "\\item 'dN with Flycatcher'\n",
       "\\item 'dS with Flycatcher'\n",
       "\\item 'Flycatcher orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Fugu gene stable ID'\n",
       "\\item 'Fugu gene name'\n",
       "\\item 'Fugu protein or transcript stable ID'\n",
       "\\item 'Fugu chromosome/scaffold name'\n",
       "\\item 'Fugu chromosome/scaffold start (bp)'\n",
       "\\item 'Fugu chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Fugu'\n",
       "\\item 'Fugu homology type'\n",
       "\\item '\\%id. target Fugu gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Fugu gene'\n",
       "\\item 'Fugu Gene-order conservation score'\n",
       "\\item 'Fugu Whole-genome alignment coverage'\n",
       "\\item 'dN with Fugu'\n",
       "\\item 'dS with Fugu'\n",
       "\\item 'Fugu orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Gelada gene stable ID'\n",
       "\\item 'Gelada gene name'\n",
       "\\item 'Gelada protein or transcript stable ID'\n",
       "\\item 'Gelada chromosome/scaffold name'\n",
       "\\item 'Gelada chromosome/scaffold start (bp)'\n",
       "\\item 'Gelada chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Gelada'\n",
       "\\item 'Gelada homology type'\n",
       "\\item '\\%id. target Gelada gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Gelada gene'\n",
       "\\item 'Gelada Gene-order conservation score'\n",
       "\\item 'Gelada Whole-genome alignment coverage'\n",
       "\\item 'dN with Gelada'\n",
       "\\item 'dS with Gelada'\n",
       "\\item 'Gelada orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Gibbon gene stable ID'\n",
       "\\item 'Gibbon gene name'\n",
       "\\item 'Gibbon protein or transcript stable ID'\n",
       "\\item 'Gibbon chromosome/scaffold name'\n",
       "\\item 'Gibbon chromosome/scaffold start (bp)'\n",
       "\\item 'Gibbon chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Gibbon'\n",
       "\\item 'Gibbon homology type'\n",
       "\\item '\\%id. target Gibbon gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Gibbon gene'\n",
       "\\item 'Gibbon Gene-order conservation score'\n",
       "\\item 'Gibbon Whole-genome alignment coverage'\n",
       "\\item 'dN with Gibbon'\n",
       "\\item 'dS with Gibbon'\n",
       "\\item 'Gibbon orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Goat gene stable ID'\n",
       "\\item 'Goat gene name'\n",
       "\\item 'Goat protein or transcript stable ID'\n",
       "\\item 'Goat chromosome/scaffold name'\n",
       "\\item 'Goat chromosome/scaffold start (bp)'\n",
       "\\item 'Goat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Goat'\n",
       "\\item 'Goat homology type'\n",
       "\\item '\\%id. target Goat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Goat gene'\n",
       "\\item 'Goat Gene-order conservation score'\n",
       "\\item 'Goat Whole-genome alignment coverage'\n",
       "\\item 'dN with Goat'\n",
       "\\item 'dS with Goat'\n",
       "\\item 'Goat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Golden Hamster gene stable ID'\n",
       "\\item 'Golden Hamster gene name'\n",
       "\\item 'Golden Hamster protein or transcript stable ID'\n",
       "\\item 'Golden Hamster chromosome/scaffold name'\n",
       "\\item 'Golden Hamster chromosome/scaffold start (bp)'\n",
       "\\item 'Golden Hamster chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Golden Hamster'\n",
       "\\item 'Golden Hamster homology type'\n",
       "\\item '\\%id. target Golden Hamster gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Golden Hamster gene'\n",
       "\\item 'Golden Hamster Gene-order conservation score'\n",
       "\\item 'Golden Hamster Whole-genome alignment coverage'\n",
       "\\item 'dN with Golden Hamster'\n",
       "\\item 'dS with Golden Hamster'\n",
       "\\item 'Golden Hamster orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Golden snub-nosed monkey gene stable ID'\n",
       "\\item 'Golden snub-nosed monkey gene name'\n",
       "\\item 'Golden snub-nosed monkey protein or transcript stable ID'\n",
       "\\item 'Golden snub-nosed monkey chromosome/scaffold name'\n",
       "\\item 'Golden snub-nosed monkey chromosome/scaffold start (bp)'\n",
       "\\item 'Golden snub-nosed monkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Golden snub-nosed monkey'\n",
       "\\item 'Golden snub-nosed monkey homology type'\n",
       "\\item '\\%id. target Golden snub-nosed monkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Golden snub-nosed monkey gene'\n",
       "\\item 'Golden snub-nosed monkey Gene-order conservation score'\n",
       "\\item 'Golden snub-nosed monkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Golden snub-nosed monkey'\n",
       "\\item 'dS with Golden snub-nosed monkey'\n",
       "\\item 'Golden snub-nosed monkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Golden-collared manakin gene stable ID'\n",
       "\\item 'Golden-collared manakin gene name'\n",
       "\\item 'Golden-collared manakin protein or transcript stable ID'\n",
       "\\item 'Golden-collared manakin chromosome/scaffold name'\n",
       "\\item 'Golden-collared manakin chromosome/scaffold start (bp)'\n",
       "\\item 'Golden-collared manakin chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Golden-collared manakin'\n",
       "\\item 'Golden-collared manakin homology type'\n",
       "\\item '\\%id. target Golden-collared manakin gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Golden-collared manakin gene'\n",
       "\\item 'Golden-collared manakin Gene-order conservation score'\n",
       "\\item 'Golden-collared manakin Whole-genome alignment coverage'\n",
       "\\item 'dN with Golden-collared manakin'\n",
       "\\item 'dS with Golden-collared manakin'\n",
       "\\item 'Golden-collared manakin orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Gorilla gene stable ID'\n",
       "\\item 'Gorilla gene name'\n",
       "\\item 'Gorilla protein or transcript stable ID'\n",
       "\\item 'Gorilla chromosome/scaffold name'\n",
       "\\item 'Gorilla chromosome/scaffold start (bp)'\n",
       "\\item 'Gorilla chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Gorilla'\n",
       "\\item 'Gorilla homology type'\n",
       "\\item '\\%id. target Gorilla gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Gorilla gene'\n",
       "\\item 'Gorilla Gene-order conservation score'\n",
       "\\item 'Gorilla Whole-genome alignment coverage'\n",
       "\\item 'dN with Gorilla'\n",
       "\\item 'dS with Gorilla'\n",
       "\\item 'Gorilla orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Great Tit gene stable ID'\n",
       "\\item 'Great Tit gene name'\n",
       "\\item 'Great Tit protein or transcript stable ID'\n",
       "\\item 'Great Tit chromosome/scaffold name'\n",
       "\\item 'Great Tit chromosome/scaffold start (bp)'\n",
       "\\item 'Great Tit chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Great Tit'\n",
       "\\item 'Great Tit homology type'\n",
       "\\item '\\%id. target Great Tit gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Great Tit gene'\n",
       "\\item 'Great Tit Gene-order conservation score'\n",
       "\\item 'Great Tit Whole-genome alignment coverage'\n",
       "\\item 'dN with Great Tit'\n",
       "\\item 'dS with Great Tit'\n",
       "\\item 'Great Tit orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Great spotted kiwi gene stable ID'\n",
       "\\item 'Great spotted kiwi gene name'\n",
       "\\item 'Great spotted kiwi protein or transcript stable ID'\n",
       "\\item 'Great spotted kiwi chromosome/scaffold name'\n",
       "\\item 'Great spotted kiwi chromosome/scaffold start (bp)'\n",
       "\\item 'Great spotted kiwi chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Great spotted kiwi'\n",
       "\\item 'Great spotted kiwi homology type'\n",
       "\\item '\\%id. target Great spotted kiwi gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Great spotted kiwi gene'\n",
       "\\item 'Great spotted kiwi Gene-order conservation score'\n",
       "\\item 'Great spotted kiwi Whole-genome alignment coverage'\n",
       "\\item 'dN with Great spotted kiwi'\n",
       "\\item 'dS with Great spotted kiwi'\n",
       "\\item 'Great spotted kiwi orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Greater amberjack gene stable ID'\n",
       "\\item 'Greater amberjack gene name'\n",
       "\\item 'Greater amberjack protein or transcript stable ID'\n",
       "\\item 'Greater amberjack chromosome/scaffold name'\n",
       "\\item 'Greater amberjack chromosome/scaffold start (bp)'\n",
       "\\item 'Greater amberjack chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Greater amberjack'\n",
       "\\item 'Greater amberjack homology type'\n",
       "\\item '\\%id. target Greater amberjack gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Greater amberjack gene'\n",
       "\\item 'Greater amberjack Gene-order conservation score'\n",
       "\\item 'Greater amberjack Whole-genome alignment coverage'\n",
       "\\item 'dN with Greater amberjack'\n",
       "\\item 'dS with Greater amberjack'\n",
       "\\item 'Greater amberjack orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Greater bamboo lemur gene stable ID'\n",
       "\\item 'Greater bamboo lemur gene name'\n",
       "\\item 'Greater bamboo lemur protein or transcript stable ID'\n",
       "\\item 'Greater bamboo lemur chromosome/scaffold name'\n",
       "\\item 'Greater bamboo lemur chromosome/scaffold start (bp)'\n",
       "\\item 'Greater bamboo lemur chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Greater bamboo lemur'\n",
       "\\item 'Greater bamboo lemur homology type'\n",
       "\\item '\\%id. target Greater bamboo lemur gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Greater bamboo lemur gene'\n",
       "\\item 'Greater bamboo lemur Gene-order conservation score'\n",
       "\\item 'Greater bamboo lemur Whole-genome alignment coverage'\n",
       "\\item 'dN with Greater bamboo lemur'\n",
       "\\item 'dS with Greater bamboo lemur'\n",
       "\\item 'Greater bamboo lemur orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Guinea Pig gene stable ID'\n",
       "\\item 'Guinea Pig gene name'\n",
       "\\item 'Guinea Pig protein or transcript stable ID'\n",
       "\\item 'Guinea Pig chromosome/scaffold name'\n",
       "\\item 'Guinea Pig chromosome/scaffold start (bp)'\n",
       "\\item 'Guinea Pig chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Guinea Pig'\n",
       "\\item 'Guinea Pig homology type'\n",
       "\\item '\\%id. target Guinea Pig gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Guinea Pig gene'\n",
       "\\item 'Guinea Pig Gene-order conservation score'\n",
       "\\item 'Guinea Pig Whole-genome alignment coverage'\n",
       "\\item 'dN with Guinea Pig'\n",
       "\\item 'dS with Guinea Pig'\n",
       "\\item 'Guinea Pig orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Guppy gene stable ID'\n",
       "\\item 'Guppy gene name'\n",
       "\\item 'Guppy protein or transcript stable ID'\n",
       "\\item 'Guppy chromosome/scaffold name'\n",
       "\\item 'Guppy chromosome/scaffold start (bp)'\n",
       "\\item 'Guppy chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Guppy'\n",
       "\\item 'Guppy homology type'\n",
       "\\item '\\%id. target Guppy gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Guppy gene'\n",
       "\\item 'Guppy Gene-order conservation score'\n",
       "\\item 'Guppy Whole-genome alignment coverage'\n",
       "\\item 'dN with Guppy'\n",
       "\\item 'dS with Guppy'\n",
       "\\item 'Guppy orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Hagfish gene stable ID'\n",
       "\\item 'Hagfish gene name'\n",
       "\\item 'Hagfish protein or transcript stable ID'\n",
       "\\item 'Hagfish chromosome/scaffold name'\n",
       "\\item 'Hagfish chromosome/scaffold start (bp)'\n",
       "\\item 'Hagfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Hagfish'\n",
       "\\item 'Hagfish homology type'\n",
       "\\item '\\%id. target Hagfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Hagfish gene'\n",
       "\\item 'Hagfish Gene-order conservation score'\n",
       "\\item 'Hagfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Hagfish'\n",
       "\\item 'dS with Hagfish'\n",
       "\\item 'Hagfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Hedgehog gene stable ID'\n",
       "\\item 'Hedgehog gene name'\n",
       "\\item 'Hedgehog protein or transcript stable ID'\n",
       "\\item 'Hedgehog chromosome/scaffold name'\n",
       "\\item 'Hedgehog chromosome/scaffold start (bp)'\n",
       "\\item 'Hedgehog chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Hedgehog'\n",
       "\\item 'Hedgehog homology type'\n",
       "\\item '\\%id. target Hedgehog gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Hedgehog gene'\n",
       "\\item 'Hedgehog Gene-order conservation score'\n",
       "\\item 'Hedgehog Whole-genome alignment coverage'\n",
       "\\item 'dN with Hedgehog'\n",
       "\\item 'dS with Hedgehog'\n",
       "\\item 'Hedgehog orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Helmeted guineafowl gene stable ID'\n",
       "\\item 'Helmeted guineafowl gene name'\n",
       "\\item 'Helmeted guineafowl protein or transcript stable ID'\n",
       "\\item 'Helmeted guineafowl chromosome/scaffold name'\n",
       "\\item 'Helmeted guineafowl chromosome/scaffold start (bp)'\n",
       "\\item 'Helmeted guineafowl chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Helmeted guineafowl'\n",
       "\\item 'Helmeted guineafowl homology type'\n",
       "\\item '\\%id. target Helmeted guineafowl gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Helmeted guineafowl gene'\n",
       "\\item 'Helmeted guineafowl Gene-order conservation score'\n",
       "\\item 'Helmeted guineafowl Whole-genome alignment coverage'\n",
       "\\item 'dN with Helmeted guineafowl'\n",
       "\\item 'dS with Helmeted guineafowl'\n",
       "\\item 'Helmeted guineafowl orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Horse gene stable ID'\n",
       "\\item 'Horse gene name'\n",
       "\\item 'Horse protein or transcript stable ID'\n",
       "\\item 'Horse chromosome/scaffold name'\n",
       "\\item 'Horse chromosome/scaffold start (bp)'\n",
       "\\item 'Horse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Horse'\n",
       "\\item 'Horse homology type'\n",
       "\\item '\\%id. target Horse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Horse gene'\n",
       "\\item 'Horse Gene-order conservation score'\n",
       "\\item 'Horse Whole-genome alignment coverage'\n",
       "\\item 'dN with Horse'\n",
       "\\item 'dS with Horse'\n",
       "\\item 'Horse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Huchen gene stable ID'\n",
       "\\item 'Huchen gene name'\n",
       "\\item 'Huchen protein or transcript stable ID'\n",
       "\\item 'Huchen chromosome/scaffold name'\n",
       "\\item 'Huchen chromosome/scaffold start (bp)'\n",
       "\\item 'Huchen chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Huchen'\n",
       "\\item 'Huchen homology type'\n",
       "\\item '\\%id. target Huchen gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Huchen gene'\n",
       "\\item 'Huchen Gene-order conservation score'\n",
       "\\item 'Huchen Whole-genome alignment coverage'\n",
       "\\item 'dN with Huchen'\n",
       "\\item 'dS with Huchen'\n",
       "\\item 'Huchen orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Hyrax gene stable ID'\n",
       "\\item 'Hyrax gene name'\n",
       "\\item 'Hyrax protein or transcript stable ID'\n",
       "\\item 'Hyrax chromosome/scaffold name'\n",
       "\\item 'Hyrax chromosome/scaffold start (bp)'\n",
       "\\item 'Hyrax chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Hyrax'\n",
       "\\item 'Hyrax homology type'\n",
       "\\item '\\%id. target Hyrax gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Hyrax gene'\n",
       "\\item 'Hyrax Gene-order conservation score'\n",
       "\\item 'Hyrax Whole-genome alignment coverage'\n",
       "\\item 'dN with Hyrax'\n",
       "\\item 'dS with Hyrax'\n",
       "\\item 'Hyrax orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Indian medaka gene stable ID'\n",
       "\\item 'Indian medaka gene name'\n",
       "\\item 'Indian medaka protein or transcript stable ID'\n",
       "\\item 'Indian medaka chromosome/scaffold name'\n",
       "\\item 'Indian medaka chromosome/scaffold start (bp)'\n",
       "\\item 'Indian medaka chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Indian medaka'\n",
       "\\item 'Indian medaka homology type'\n",
       "\\item '\\%id. target Indian medaka gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Indian medaka gene'\n",
       "\\item 'Indian medaka Gene-order conservation score'\n",
       "\\item 'Indian medaka Whole-genome alignment coverage'\n",
       "\\item 'dN with Indian medaka'\n",
       "\\item 'dS with Indian medaka'\n",
       "\\item 'Indian medaka orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Japanese medaka HNI gene stable ID'\n",
       "\\item 'Japanese medaka HNI gene name'\n",
       "\\item 'Japanese medaka HNI protein or transcript stable ID'\n",
       "\\item 'Japanese medaka HNI chromosome/scaffold name'\n",
       "\\item 'Japanese medaka HNI chromosome/scaffold start (bp)'\n",
       "\\item 'Japanese medaka HNI chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Japanese medaka HNI'\n",
       "\\item 'Japanese medaka HNI homology type'\n",
       "\\item '\\%id. target Japanese medaka HNI gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Japanese medaka HNI gene'\n",
       "\\item 'Japanese medaka HNI Gene-order conservation score'\n",
       "\\item 'Japanese medaka HNI Whole-genome alignment coverage'\n",
       "\\item 'dN with Japanese medaka HNI'\n",
       "\\item 'dS with Japanese medaka HNI'\n",
       "\\item 'Japanese medaka HNI orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Japanese medaka HSOK gene stable ID'\n",
       "\\item 'Japanese medaka HSOK gene name'\n",
       "\\item 'Japanese medaka HSOK protein or transcript stable ID'\n",
       "\\item 'Japanese medaka HSOK chromosome/scaffold name'\n",
       "\\item 'Japanese medaka HSOK chromosome/scaffold start (bp)'\n",
       "\\item 'Japanese medaka HSOK chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Japanese medaka HSOK'\n",
       "\\item 'Japanese medaka HSOK homology type'\n",
       "\\item '\\%id. target Japanese medaka HSOK gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Japanese medaka HSOK gene'\n",
       "\\item 'Japanese medaka HSOK Gene-order conservation score'\n",
       "\\item 'Japanese medaka HSOK Whole-genome alignment coverage'\n",
       "\\item 'dN with Japanese medaka HSOK'\n",
       "\\item 'dS with Japanese medaka HSOK'\n",
       "\\item 'Japanese medaka HSOK orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Japanese medaka HdrR gene stable ID'\n",
       "\\item 'Japanese medaka HdrR gene name'\n",
       "\\item 'Japanese medaka HdrR protein or transcript stable ID'\n",
       "\\item 'Japanese medaka HdrR chromosome/scaffold name'\n",
       "\\item 'Japanese medaka HdrR chromosome/scaffold start (bp)'\n",
       "\\item 'Japanese medaka HdrR chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Japanese medaka HdrR'\n",
       "\\item 'Japanese medaka HdrR homology type'\n",
       "\\item '\\%id. target Japanese medaka HdrR gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Japanese medaka HdrR gene'\n",
       "\\item 'Japanese medaka HdrR Gene-order conservation score'\n",
       "\\item 'Japanese medaka HdrR Whole-genome alignment coverage'\n",
       "\\item 'dN with Japanese medaka HdrR'\n",
       "\\item 'dS with Japanese medaka HdrR'\n",
       "\\item 'Japanese medaka HdrR orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Japanese quail gene stable ID'\n",
       "\\item 'Japanese quail gene name'\n",
       "\\item 'Japanese quail protein or transcript stable ID'\n",
       "\\item 'Japanese quail chromosome/scaffold name'\n",
       "\\item 'Japanese quail chromosome/scaffold start (bp)'\n",
       "\\item 'Japanese quail chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Japanese quail'\n",
       "\\item 'Japanese quail homology type'\n",
       "\\item '\\%id. target Japanese quail gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Japanese quail gene'\n",
       "\\item 'Japanese quail Gene-order conservation score'\n",
       "\\item 'Japanese quail Whole-genome alignment coverage'\n",
       "\\item 'dN with Japanese quail'\n",
       "\\item 'dS with Japanese quail'\n",
       "\\item 'Japanese quail orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Kangaroo rat gene stable ID'\n",
       "\\item 'Kangaroo rat gene name'\n",
       "\\item 'Kangaroo rat protein or transcript stable ID'\n",
       "\\item 'Kangaroo rat chromosome/scaffold name'\n",
       "\\item 'Kangaroo rat chromosome/scaffold start (bp)'\n",
       "\\item 'Kangaroo rat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Kangaroo rat'\n",
       "\\item 'Kangaroo rat homology type'\n",
       "\\item '\\%id. target Kangaroo rat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Kangaroo rat gene'\n",
       "\\item 'Kangaroo rat Gene-order conservation score'\n",
       "\\item 'Kangaroo rat Whole-genome alignment coverage'\n",
       "\\item 'dN with Kangaroo rat'\n",
       "\\item 'dS with Kangaroo rat'\n",
       "\\item 'Kangaroo rat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Koala gene stable ID'\n",
       "\\item 'Koala gene name'\n",
       "\\item 'Koala protein or transcript stable ID'\n",
       "\\item 'Koala chromosome/scaffold name'\n",
       "\\item 'Koala chromosome/scaffold start (bp)'\n",
       "\\item 'Koala chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Koala'\n",
       "\\item 'Koala homology type'\n",
       "\\item '\\%id. target Koala gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Koala gene'\n",
       "\\item 'Koala Gene-order conservation score'\n",
       "\\item 'Koala Whole-genome alignment coverage'\n",
       "\\item 'dN with Koala'\n",
       "\\item 'dS with Koala'\n",
       "\\item 'Koala orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Lamprey gene stable ID'\n",
       "\\item 'Lamprey gene name'\n",
       "\\item 'Lamprey protein or transcript stable ID'\n",
       "\\item 'Lamprey chromosome/scaffold name'\n",
       "\\item 'Lamprey chromosome/scaffold start (bp)'\n",
       "\\item 'Lamprey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Lamprey'\n",
       "\\item 'Lamprey homology type'\n",
       "\\item '\\%id. target Lamprey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Lamprey gene'\n",
       "\\item 'Lamprey Gene-order conservation score'\n",
       "\\item 'Lamprey Whole-genome alignment coverage'\n",
       "\\item 'dN with Lamprey'\n",
       "\\item 'dS with Lamprey'\n",
       "\\item 'Lamprey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Leopard gene stable ID'\n",
       "\\item 'Leopard gene name'\n",
       "\\item 'Leopard protein or transcript stable ID'\n",
       "\\item 'Leopard chromosome/scaffold name'\n",
       "\\item 'Leopard chromosome/scaffold start (bp)'\n",
       "\\item 'Leopard chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Leopard'\n",
       "\\item 'Leopard homology type'\n",
       "\\item '\\%id. target Leopard gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Leopard gene'\n",
       "\\item 'Leopard Gene-order conservation score'\n",
       "\\item 'Leopard Whole-genome alignment coverage'\n",
       "\\item 'dN with Leopard'\n",
       "\\item 'dS with Leopard'\n",
       "\\item 'Leopard orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Lesser Egyptian jerboa gene stable ID'\n",
       "\\item 'Lesser Egyptian jerboa gene name'\n",
       "\\item 'Lesser Egyptian jerboa protein or transcript stable ID'\n",
       "\\item 'Lesser Egyptian jerboa chromosome/scaffold name'\n",
       "\\item 'Lesser Egyptian jerboa chromosome/scaffold start (bp)'\n",
       "\\item 'Lesser Egyptian jerboa chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Lesser Egyptian jerboa'\n",
       "\\item 'Lesser Egyptian jerboa homology type'\n",
       "\\item '\\%id. target Lesser Egyptian jerboa gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Lesser Egyptian jerboa gene'\n",
       "\\item 'Lesser Egyptian jerboa Gene-order conservation score'\n",
       "\\item 'Lesser Egyptian jerboa Whole-genome alignment coverage'\n",
       "\\item 'dN with Lesser Egyptian jerboa'\n",
       "\\item 'dS with Lesser Egyptian jerboa'\n",
       "\\item 'Lesser Egyptian jerboa orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Lesser hedgehog tenrec gene stable ID'\n",
       "\\item 'Lesser hedgehog tenrec gene name'\n",
       "\\item 'Lesser hedgehog tenrec protein or transcript stable ID'\n",
       "\\item 'Lesser hedgehog tenrec chromosome/scaffold name'\n",
       "\\item 'Lesser hedgehog tenrec chromosome/scaffold start (bp)'\n",
       "\\item 'Lesser hedgehog tenrec chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Lesser hedgehog tenrec'\n",
       "\\item 'Lesser hedgehog tenrec homology type'\n",
       "\\item '\\%id. target Lesser hedgehog tenrec gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Lesser hedgehog tenrec gene'\n",
       "\\item 'Lesser hedgehog tenrec Gene-order conservation score'\n",
       "\\item 'Lesser hedgehog tenrec Whole-genome alignment coverage'\n",
       "\\item 'dN with Lesser hedgehog tenrec'\n",
       "\\item 'dS with Lesser hedgehog tenrec'\n",
       "\\item 'Lesser hedgehog tenrec orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Little spotted kiwi gene stable ID'\n",
       "\\item 'Little spotted kiwi gene name'\n",
       "\\item 'Little spotted kiwi protein or transcript stable ID'\n",
       "\\item 'Little spotted kiwi chromosome/scaffold name'\n",
       "\\item 'Little spotted kiwi chromosome/scaffold start (bp)'\n",
       "\\item 'Little spotted kiwi chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Little spotted kiwi'\n",
       "\\item 'Little spotted kiwi homology type'\n",
       "\\item '\\%id. target Little spotted kiwi gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Little spotted kiwi gene'\n",
       "\\item 'Little spotted kiwi Gene-order conservation score'\n",
       "\\item 'Little spotted kiwi Whole-genome alignment coverage'\n",
       "\\item 'dN with Little spotted kiwi'\n",
       "\\item 'dS with Little spotted kiwi'\n",
       "\\item 'Little spotted kiwi orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Long-tailed chinchilla gene stable ID'\n",
       "\\item 'Long-tailed chinchilla gene name'\n",
       "\\item 'Long-tailed chinchilla protein or transcript stable ID'\n",
       "\\item 'Long-tailed chinchilla chromosome/scaffold name'\n",
       "\\item 'Long-tailed chinchilla chromosome/scaffold start (bp)'\n",
       "\\item 'Long-tailed chinchilla chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Long-tailed chinchilla'\n",
       "\\item 'Long-tailed chinchilla homology type'\n",
       "\\item '\\%id. target Long-tailed chinchilla gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Long-tailed chinchilla gene'\n",
       "\\item 'Long-tailed chinchilla Gene-order conservation score'\n",
       "\\item 'Long-tailed chinchilla Whole-genome alignment coverage'\n",
       "\\item 'dN with Long-tailed chinchilla'\n",
       "\\item 'dS with Long-tailed chinchilla'\n",
       "\\item 'Long-tailed chinchilla orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Lyretail cichlid gene stable ID'\n",
       "\\item 'Lyretail cichlid gene name'\n",
       "\\item 'Lyretail cichlid protein or transcript stable ID'\n",
       "\\item 'Lyretail cichlid chromosome/scaffold name'\n",
       "\\item 'Lyretail cichlid chromosome/scaffold start (bp)'\n",
       "\\item 'Lyretail cichlid chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Lyretail cichlid'\n",
       "\\item 'Lyretail cichlid homology type'\n",
       "\\item '\\%id. target Lyretail cichlid gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Lyretail cichlid gene'\n",
       "\\item 'Lyretail cichlid Gene-order conservation score'\n",
       "\\item 'Lyretail cichlid Whole-genome alignment coverage'\n",
       "\\item 'dN with Lyretail cichlid'\n",
       "\\item 'dS with Lyretail cichlid'\n",
       "\\item 'Lyretail cichlid orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey gene stable ID'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey gene name'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey protein or transcript stable ID'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey chromosome/scaffold name'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey chromosome/scaffold start (bp)'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ma\\textbackslash{}'s night monkey'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey homology type'\n",
       "\\item '\\%id. target Ma\\textbackslash{}'s night monkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ma\\textbackslash{}'s night monkey gene'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey Gene-order conservation score'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Ma\\textbackslash{}'s night monkey'\n",
       "\\item 'dS with Ma\\textbackslash{}'s night monkey'\n",
       "\\item 'Ma\\textbackslash{}'s night monkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Macaque gene stable ID'\n",
       "\\item 'Macaque gene name'\n",
       "\\item 'Macaque protein or transcript stable ID'\n",
       "\\item 'Macaque chromosome/scaffold name'\n",
       "\\item 'Macaque chromosome/scaffold start (bp)'\n",
       "\\item 'Macaque chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Macaque'\n",
       "\\item 'Macaque homology type'\n",
       "\\item '\\%id. target Macaque gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Macaque gene'\n",
       "\\item 'Macaque Gene-order conservation score'\n",
       "\\item 'Macaque Whole-genome alignment coverage'\n",
       "\\item 'dN with Macaque'\n",
       "\\item 'dS with Macaque'\n",
       "\\item 'Macaque orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mainland tiger snake gene stable ID'\n",
       "\\item 'Mainland tiger snake gene name'\n",
       "\\item 'Mainland tiger snake protein or transcript stable ID'\n",
       "\\item 'Mainland tiger snake chromosome/scaffold name'\n",
       "\\item 'Mainland tiger snake chromosome/scaffold start (bp)'\n",
       "\\item 'Mainland tiger snake chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mainland tiger snake'\n",
       "\\item 'Mainland tiger snake homology type'\n",
       "\\item '\\%id. target Mainland tiger snake gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mainland tiger snake gene'\n",
       "\\item 'Mainland tiger snake Gene-order conservation score'\n",
       "\\item 'Mainland tiger snake Whole-genome alignment coverage'\n",
       "\\item 'dN with Mainland tiger snake'\n",
       "\\item 'dS with Mainland tiger snake'\n",
       "\\item 'Mainland tiger snake orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Makobe Island cichlid gene stable ID'\n",
       "\\item 'Makobe Island cichlid gene name'\n",
       "\\item 'Makobe Island cichlid protein or transcript stable ID'\n",
       "\\item 'Makobe Island cichlid chromosome/scaffold name'\n",
       "\\item 'Makobe Island cichlid chromosome/scaffold start (bp)'\n",
       "\\item 'Makobe Island cichlid chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Makobe Island cichlid'\n",
       "\\item 'Makobe Island cichlid homology type'\n",
       "\\item '\\%id. target Makobe Island cichlid gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Makobe Island cichlid gene'\n",
       "\\item 'Makobe Island cichlid Gene-order conservation score'\n",
       "\\item 'Makobe Island cichlid Whole-genome alignment coverage'\n",
       "\\item 'dN with Makobe Island cichlid'\n",
       "\\item 'dS with Makobe Island cichlid'\n",
       "\\item 'Makobe Island cichlid orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mangrove rivulus gene stable ID'\n",
       "\\item 'Mangrove rivulus gene name'\n",
       "\\item 'Mangrove rivulus protein or transcript stable ID'\n",
       "\\item 'Mangrove rivulus chromosome/scaffold name'\n",
       "\\item 'Mangrove rivulus chromosome/scaffold start (bp)'\n",
       "\\item 'Mangrove rivulus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mangrove rivulus'\n",
       "\\item 'Mangrove rivulus homology type'\n",
       "\\item '\\%id. target Mangrove rivulus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mangrove rivulus gene'\n",
       "\\item 'Mangrove rivulus Gene-order conservation score'\n",
       "\\item 'Mangrove rivulus Whole-genome alignment coverage'\n",
       "\\item 'dN with Mangrove rivulus'\n",
       "\\item 'dS with Mangrove rivulus'\n",
       "\\item 'Mangrove rivulus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Marmoset gene stable ID'\n",
       "\\item 'Marmoset gene name'\n",
       "\\item 'Marmoset protein or transcript stable ID'\n",
       "\\item 'Marmoset chromosome/scaffold name'\n",
       "\\item 'Marmoset chromosome/scaffold start (bp)'\n",
       "\\item 'Marmoset chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Marmoset'\n",
       "\\item 'Marmoset homology type'\n",
       "\\item '\\%id. target Marmoset gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Marmoset gene'\n",
       "\\item 'Marmoset Gene-order conservation score'\n",
       "\\item 'Marmoset Whole-genome alignment coverage'\n",
       "\\item 'dN with Marmoset'\n",
       "\\item 'dS with Marmoset'\n",
       "\\item 'Marmoset orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Megabat gene stable ID'\n",
       "\\item 'Megabat gene name'\n",
       "\\item 'Megabat protein or transcript stable ID'\n",
       "\\item 'Megabat chromosome/scaffold name'\n",
       "\\item 'Megabat chromosome/scaffold start (bp)'\n",
       "\\item 'Megabat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Megabat'\n",
       "\\item 'Megabat homology type'\n",
       "\\item '\\%id. target Megabat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Megabat gene'\n",
       "\\item 'Megabat Gene-order conservation score'\n",
       "\\item 'Megabat Whole-genome alignment coverage'\n",
       "\\item 'dN with Megabat'\n",
       "\\item 'dS with Megabat'\n",
       "\\item 'Megabat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mexican tetra gene stable ID'\n",
       "\\item 'Mexican tetra gene name'\n",
       "\\item 'Mexican tetra protein or transcript stable ID'\n",
       "\\item 'Mexican tetra chromosome/scaffold name'\n",
       "\\item 'Mexican tetra chromosome/scaffold start (bp)'\n",
       "\\item 'Mexican tetra chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mexican tetra'\n",
       "\\item 'Mexican tetra homology type'\n",
       "\\item '\\%id. target Mexican tetra gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mexican tetra gene'\n",
       "\\item 'Mexican tetra Gene-order conservation score'\n",
       "\\item 'Mexican tetra Whole-genome alignment coverage'\n",
       "\\item 'dN with Mexican tetra'\n",
       "\\item 'dS with Mexican tetra'\n",
       "\\item 'Mexican tetra orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Microbat gene stable ID'\n",
       "\\item 'Microbat gene name'\n",
       "\\item 'Microbat protein or transcript stable ID'\n",
       "\\item 'Microbat chromosome/scaffold name'\n",
       "\\item 'Microbat chromosome/scaffold start (bp)'\n",
       "\\item 'Microbat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Microbat'\n",
       "\\item 'Microbat homology type'\n",
       "\\item '\\%id. target Microbat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Microbat gene'\n",
       "\\item 'Microbat Gene-order conservation score'\n",
       "\\item 'Microbat Whole-genome alignment coverage'\n",
       "\\item 'dN with Microbat'\n",
       "\\item 'dS with Microbat'\n",
       "\\item 'Microbat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Midas cichlid gene stable ID'\n",
       "\\item 'Midas cichlid gene name'\n",
       "\\item 'Midas cichlid protein or transcript stable ID'\n",
       "\\item 'Midas cichlid chromosome/scaffold name'\n",
       "\\item 'Midas cichlid chromosome/scaffold start (bp)'\n",
       "\\item 'Midas cichlid chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Midas cichlid'\n",
       "\\item 'Midas cichlid homology type'\n",
       "\\item '\\%id. target Midas cichlid gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Midas cichlid gene'\n",
       "\\item 'Midas cichlid Gene-order conservation score'\n",
       "\\item 'Midas cichlid Whole-genome alignment coverage'\n",
       "\\item 'dN with Midas cichlid'\n",
       "\\item 'dS with Midas cichlid'\n",
       "\\item 'Midas cichlid orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mongolian gerbil gene stable ID'\n",
       "\\item 'Mongolian gerbil gene name'\n",
       "\\item 'Mongolian gerbil protein or transcript stable ID'\n",
       "\\item 'Mongolian gerbil chromosome/scaffold name'\n",
       "\\item 'Mongolian gerbil chromosome/scaffold start (bp)'\n",
       "\\item 'Mongolian gerbil chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mongolian gerbil'\n",
       "\\item 'Mongolian gerbil homology type'\n",
       "\\item '\\%id. target Mongolian gerbil gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mongolian gerbil gene'\n",
       "\\item 'Mongolian gerbil Gene-order conservation score'\n",
       "\\item 'Mongolian gerbil Whole-genome alignment coverage'\n",
       "\\item 'dN with Mongolian gerbil'\n",
       "\\item 'dS with Mongolian gerbil'\n",
       "\\item 'Mongolian gerbil orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Monterrey platyfish gene stable ID'\n",
       "\\item 'Monterrey platyfish gene name'\n",
       "\\item 'Monterrey platyfish protein or transcript stable ID'\n",
       "\\item 'Monterrey platyfish chromosome/scaffold name'\n",
       "\\item 'Monterrey platyfish chromosome/scaffold start (bp)'\n",
       "\\item 'Monterrey platyfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Monterrey platyfish'\n",
       "\\item 'Monterrey platyfish homology type'\n",
       "\\item '\\%id. target Monterrey platyfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Monterrey platyfish gene'\n",
       "\\item 'Monterrey platyfish Gene-order conservation score'\n",
       "\\item 'Monterrey platyfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Monterrey platyfish'\n",
       "\\item 'dS with Monterrey platyfish'\n",
       "\\item 'Monterrey platyfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mouse gene stable ID'\n",
       "\\item 'Mouse gene name'\n",
       "\\item 'Mouse protein or transcript stable ID'\n",
       "\\item 'Mouse chromosome/scaffold name'\n",
       "\\item 'Mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mouse'\n",
       "\\item 'Mouse homology type'\n",
       "\\item '\\%id. target Mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mouse gene'\n",
       "\\item 'Mouse Gene-order conservation score'\n",
       "\\item 'Mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Mouse'\n",
       "\\item 'dS with Mouse'\n",
       "\\item 'Mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mouse Lemur gene stable ID'\n",
       "\\item 'Mouse Lemur gene name'\n",
       "\\item 'Mouse Lemur protein or transcript stable ID'\n",
       "\\item 'Mouse Lemur chromosome/scaffold name'\n",
       "\\item 'Mouse Lemur chromosome/scaffold start (bp)'\n",
       "\\item 'Mouse Lemur chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mouse Lemur'\n",
       "\\item 'Mouse Lemur homology type'\n",
       "\\item '\\%id. target Mouse Lemur gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mouse Lemur gene'\n",
       "\\item 'Mouse Lemur Gene-order conservation score'\n",
       "\\item 'Mouse Lemur Whole-genome alignment coverage'\n",
       "\\item 'dN with Mouse Lemur'\n",
       "\\item 'dS with Mouse Lemur'\n",
       "\\item 'Mouse Lemur orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Mummichog gene stable ID'\n",
       "\\item 'Mummichog gene name'\n",
       "\\item 'Mummichog protein or transcript stable ID'\n",
       "\\item 'Mummichog chromosome/scaffold name'\n",
       "\\item 'Mummichog chromosome/scaffold start (bp)'\n",
       "\\item 'Mummichog chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Mummichog'\n",
       "\\item 'Mummichog homology type'\n",
       "\\item '\\%id. target Mummichog gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Mummichog gene'\n",
       "\\item 'Mummichog Gene-order conservation score'\n",
       "\\item 'Mummichog Whole-genome alignment coverage'\n",
       "\\item 'dN with Mummichog'\n",
       "\\item 'dS with Mummichog'\n",
       "\\item 'Mummichog orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Naked mole-rat female gene stable ID'\n",
       "\\item 'Naked mole-rat female gene name'\n",
       "\\item 'Naked mole-rat female protein or transcript stable ID'\n",
       "\\item 'Naked mole-rat female chromosome/scaffold name'\n",
       "\\item 'Naked mole-rat female chromosome/scaffold start (bp)'\n",
       "\\item 'Naked mole-rat female chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Naked mole-rat female'\n",
       "\\item 'Naked mole-rat female homology type'\n",
       "\\item '\\%id. target Naked mole-rat female gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Naked mole-rat female gene'\n",
       "\\item 'Naked mole-rat female Gene-order conservation score'\n",
       "\\item 'Naked mole-rat female Whole-genome alignment coverage'\n",
       "\\item 'dN with Naked mole-rat female'\n",
       "\\item 'dS with Naked mole-rat female'\n",
       "\\item 'Naked mole-rat female orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Naked mole-rat male gene stable ID'\n",
       "\\item 'Naked mole-rat male gene name'\n",
       "\\item 'Naked mole-rat male protein or transcript stable ID'\n",
       "\\item 'Naked mole-rat male chromosome/scaffold name'\n",
       "\\item 'Naked mole-rat male chromosome/scaffold start (bp)'\n",
       "\\item 'Naked mole-rat male chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Naked mole-rat male'\n",
       "\\item 'Naked mole-rat male homology type'\n",
       "\\item '\\%id. target Naked mole-rat male gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Naked mole-rat male gene'\n",
       "\\item 'Naked mole-rat male Gene-order conservation score'\n",
       "\\item 'Naked mole-rat male Whole-genome alignment coverage'\n",
       "\\item 'dN with Naked mole-rat male'\n",
       "\\item 'dS with Naked mole-rat male'\n",
       "\\item 'Naked mole-rat male orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Northern American deer mouse gene stable ID'\n",
       "\\item 'Northern American deer mouse gene name'\n",
       "\\item 'Northern American deer mouse protein or transcript stable ID'\n",
       "\\item 'Northern American deer mouse chromosome/scaffold name'\n",
       "\\item 'Northern American deer mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Northern American deer mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Northern American deer mouse'\n",
       "\\item 'Northern American deer mouse homology type'\n",
       "\\item '\\%id. target Northern American deer mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Northern American deer mouse gene'\n",
       "\\item 'Northern American deer mouse Gene-order conservation score'\n",
       "\\item 'Northern American deer mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Northern American deer mouse'\n",
       "\\item 'dS with Northern American deer mouse'\n",
       "\\item 'Northern American deer mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Northern pike gene stable ID'\n",
       "\\item 'Northern pike gene name'\n",
       "\\item 'Northern pike protein or transcript stable ID'\n",
       "\\item 'Northern pike chromosome/scaffold name'\n",
       "\\item 'Northern pike chromosome/scaffold start (bp)'\n",
       "\\item 'Northern pike chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Northern pike'\n",
       "\\item 'Northern pike homology type'\n",
       "\\item '\\%id. target Northern pike gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Northern pike gene'\n",
       "\\item 'Northern pike Gene-order conservation score'\n",
       "\\item 'Northern pike Whole-genome alignment coverage'\n",
       "\\item 'dN with Northern pike'\n",
       "\\item 'dS with Northern pike'\n",
       "\\item 'Northern pike orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ocean sunfish gene stable ID'\n",
       "\\item 'Ocean sunfish gene name'\n",
       "\\item 'Ocean sunfish protein or transcript stable ID'\n",
       "\\item 'Ocean sunfish chromosome/scaffold name'\n",
       "\\item 'Ocean sunfish chromosome/scaffold start (bp)'\n",
       "\\item 'Ocean sunfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ocean sunfish'\n",
       "\\item 'Ocean sunfish homology type'\n",
       "\\item '\\%id. target Ocean sunfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ocean sunfish gene'\n",
       "\\item 'Ocean sunfish Gene-order conservation score'\n",
       "\\item 'Ocean sunfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Ocean sunfish'\n",
       "\\item 'dS with Ocean sunfish'\n",
       "\\item 'Ocean sunfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Okarito brown kiwi gene stable ID'\n",
       "\\item 'Okarito brown kiwi gene name'\n",
       "\\item 'Okarito brown kiwi protein or transcript stable ID'\n",
       "\\item 'Okarito brown kiwi chromosome/scaffold name'\n",
       "\\item 'Okarito brown kiwi chromosome/scaffold start (bp)'\n",
       "\\item 'Okarito brown kiwi chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Okarito brown kiwi'\n",
       "\\item 'Okarito brown kiwi homology type'\n",
       "\\item '\\%id. target Okarito brown kiwi gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Okarito brown kiwi gene'\n",
       "\\item 'Okarito brown kiwi Gene-order conservation score'\n",
       "\\item 'Okarito brown kiwi Whole-genome alignment coverage'\n",
       "\\item 'dN with Okarito brown kiwi'\n",
       "\\item 'dS with Okarito brown kiwi'\n",
       "\\item 'Okarito brown kiwi orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Olive baboon gene stable ID'\n",
       "\\item 'Olive baboon gene name'\n",
       "\\item 'Olive baboon protein or transcript stable ID'\n",
       "\\item 'Olive baboon chromosome/scaffold name'\n",
       "\\item 'Olive baboon chromosome/scaffold start (bp)'\n",
       "\\item 'Olive baboon chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Olive baboon'\n",
       "\\item 'Olive baboon homology type'\n",
       "\\item '\\%id. target Olive baboon gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Olive baboon gene'\n",
       "\\item 'Olive baboon Gene-order conservation score'\n",
       "\\item 'Olive baboon Whole-genome alignment coverage'\n",
       "\\item 'dN with Olive baboon'\n",
       "\\item 'dS with Olive baboon'\n",
       "\\item 'Olive baboon orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Opossum gene stable ID'\n",
       "\\item 'Opossum gene name'\n",
       "\\item 'Opossum protein or transcript stable ID'\n",
       "\\item 'Opossum chromosome/scaffold name'\n",
       "\\item 'Opossum chromosome/scaffold start (bp)'\n",
       "\\item 'Opossum chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Opossum'\n",
       "\\item 'Opossum homology type'\n",
       "\\item '\\%id. target Opossum gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Opossum gene'\n",
       "\\item 'Opossum Gene-order conservation score'\n",
       "\\item 'Opossum Whole-genome alignment coverage'\n",
       "\\item 'dN with Opossum'\n",
       "\\item 'dS with Opossum'\n",
       "\\item 'Opossum orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Orange clownfish gene stable ID'\n",
       "\\item 'Orange clownfish gene name'\n",
       "\\item 'Orange clownfish protein or transcript stable ID'\n",
       "\\item 'Orange clownfish chromosome/scaffold name'\n",
       "\\item 'Orange clownfish chromosome/scaffold start (bp)'\n",
       "\\item 'Orange clownfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Orange clownfish'\n",
       "\\item 'Orange clownfish homology type'\n",
       "\\item '\\%id. target Orange clownfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Orange clownfish gene'\n",
       "\\item 'Orange clownfish Gene-order conservation score'\n",
       "\\item 'Orange clownfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Orange clownfish'\n",
       "\\item 'dS with Orange clownfish'\n",
       "\\item 'Orange clownfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Orangutan gene stable ID'\n",
       "\\item 'Orangutan gene name'\n",
       "\\item 'Orangutan protein or transcript stable ID'\n",
       "\\item 'Orangutan chromosome/scaffold name'\n",
       "\\item 'Orangutan chromosome/scaffold start (bp)'\n",
       "\\item 'Orangutan chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Orangutan'\n",
       "\\item 'Orangutan homology type'\n",
       "\\item '\\%id. target Orangutan gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Orangutan gene'\n",
       "\\item 'Orangutan Gene-order conservation score'\n",
       "\\item 'Orangutan Whole-genome alignment coverage'\n",
       "\\item 'dN with Orangutan'\n",
       "\\item 'dS with Orangutan'\n",
       "\\item 'Orangutan orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Painted turtle gene stable ID'\n",
       "\\item 'Painted turtle gene name'\n",
       "\\item 'Painted turtle protein or transcript stable ID'\n",
       "\\item 'Painted turtle chromosome/scaffold name'\n",
       "\\item 'Painted turtle chromosome/scaffold start (bp)'\n",
       "\\item 'Painted turtle chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Painted turtle'\n",
       "\\item 'Painted turtle homology type'\n",
       "\\item '\\%id. target Painted turtle gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Painted turtle gene'\n",
       "\\item 'Painted turtle Gene-order conservation score'\n",
       "\\item 'Painted turtle Whole-genome alignment coverage'\n",
       "\\item 'dN with Painted turtle'\n",
       "\\item 'dS with Painted turtle'\n",
       "\\item 'Painted turtle orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Panda gene stable ID'\n",
       "\\item 'Panda gene name'\n",
       "\\item 'Panda protein or transcript stable ID'\n",
       "\\item 'Panda chromosome/scaffold name'\n",
       "\\item 'Panda chromosome/scaffold start (bp)'\n",
       "\\item 'Panda chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Panda'\n",
       "\\item 'Panda homology type'\n",
       "\\item '\\%id. target Panda gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Panda gene'\n",
       "\\item 'Panda Gene-order conservation score'\n",
       "\\item 'Panda Whole-genome alignment coverage'\n",
       "\\item 'dN with Panda'\n",
       "\\item 'dS with Panda'\n",
       "\\item 'Panda orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Paramormyrops kingsleyae gene stable ID'\n",
       "\\item 'Paramormyrops kingsleyae gene name'\n",
       "\\item 'Paramormyrops kingsleyae protein or transcript stable ID'\n",
       "\\item 'Paramormyrops kingsleyae chromosome/scaffold name'\n",
       "\\item 'Paramormyrops kingsleyae chromosome/scaffold start (bp)'\n",
       "\\item 'Paramormyrops kingsleyae chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Paramormyrops kingsleyae'\n",
       "\\item 'Paramormyrops kingsleyae homology type'\n",
       "\\item '\\%id. target Paramormyrops kingsleyae gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Paramormyrops kingsleyae gene'\n",
       "\\item 'Paramormyrops kingsleyae Gene-order conservation score'\n",
       "\\item 'Paramormyrops kingsleyae Whole-genome alignment coverage'\n",
       "\\item 'dN with Paramormyrops kingsleyae'\n",
       "\\item 'dS with Paramormyrops kingsleyae'\n",
       "\\item 'Paramormyrops kingsleyae orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Periophthalmus magnuspinnatus gene stable ID'\n",
       "\\item 'Periophthalmus magnuspinnatus gene name'\n",
       "\\item 'Periophthalmus magnuspinnatus protein or transcript stable ID'\n",
       "\\item 'Periophthalmus magnuspinnatus chromosome/scaffold name'\n",
       "\\item 'Periophthalmus magnuspinnatus chromosome/scaffold start (bp)'\n",
       "\\item 'Periophthalmus magnuspinnatus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Periophthalmus magnuspinnatus'\n",
       "\\item 'Periophthalmus magnuspinnatus homology type'\n",
       "\\item '\\%id. target Periophthalmus magnuspinnatus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Periophthalmus magnuspinnatus gene'\n",
       "\\item 'Periophthalmus magnuspinnatus Gene-order conservation score'\n",
       "\\item 'Periophthalmus magnuspinnatus Whole-genome alignment coverage'\n",
       "\\item 'dN with Periophthalmus magnuspinnatus'\n",
       "\\item 'dS with Periophthalmus magnuspinnatus'\n",
       "\\item 'Periophthalmus magnuspinnatus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Pig gene stable ID'\n",
       "\\item 'Pig gene name'\n",
       "\\item 'Pig protein or transcript stable ID'\n",
       "\\item 'Pig chromosome/scaffold name'\n",
       "\\item 'Pig chromosome/scaffold start (bp)'\n",
       "\\item 'Pig chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Pig'\n",
       "\\item 'Pig homology type'\n",
       "\\item '\\%id. target Pig gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Pig gene'\n",
       "\\item 'Pig Gene-order conservation score'\n",
       "\\item 'Pig Whole-genome alignment coverage'\n",
       "\\item 'dN with Pig'\n",
       "\\item 'dS with Pig'\n",
       "\\item 'Pig orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Pig USMARC gene stable ID'\n",
       "\\item 'Pig USMARC gene name'\n",
       "\\item 'Pig USMARC protein or transcript stable ID'\n",
       "\\item 'Pig USMARC chromosome/scaffold name'\n",
       "\\item 'Pig USMARC chromosome/scaffold start (bp)'\n",
       "\\item 'Pig USMARC chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Pig USMARC'\n",
       "\\item 'Pig USMARC homology type'\n",
       "\\item '\\%id. target Pig USMARC gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Pig USMARC gene'\n",
       "\\item 'Pig USMARC Gene-order conservation score'\n",
       "\\item 'Pig USMARC Whole-genome alignment coverage'\n",
       "\\item 'dN with Pig USMARC'\n",
       "\\item 'dS with Pig USMARC'\n",
       "\\item 'Pig USMARC orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Pig-tailed macaque gene stable ID'\n",
       "\\item 'Pig-tailed macaque gene name'\n",
       "\\item 'Pig-tailed macaque protein or transcript stable ID'\n",
       "\\item 'Pig-tailed macaque chromosome/scaffold name'\n",
       "\\item 'Pig-tailed macaque chromosome/scaffold start (bp)'\n",
       "\\item 'Pig-tailed macaque chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Pig-tailed macaque'\n",
       "\\item 'Pig-tailed macaque homology type'\n",
       "\\item '\\%id. target Pig-tailed macaque gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Pig-tailed macaque gene'\n",
       "\\item 'Pig-tailed macaque Gene-order conservation score'\n",
       "\\item 'Pig-tailed macaque Whole-genome alignment coverage'\n",
       "\\item 'dN with Pig-tailed macaque'\n",
       "\\item 'dS with Pig-tailed macaque'\n",
       "\\item 'Pig-tailed macaque orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Pika gene stable ID'\n",
       "\\item 'Pika gene name'\n",
       "\\item 'Pika protein or transcript stable ID'\n",
       "\\item 'Pika chromosome/scaffold name'\n",
       "\\item 'Pika chromosome/scaffold start (bp)'\n",
       "\\item 'Pika chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Pika'\n",
       "\\item 'Pika homology type'\n",
       "\\item '\\%id. target Pika gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Pika gene'\n",
       "\\item 'Pika Gene-order conservation score'\n",
       "\\item 'Pika Whole-genome alignment coverage'\n",
       "\\item 'dN with Pika'\n",
       "\\item 'dS with Pika'\n",
       "\\item 'Pika orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Pink-footed goose gene stable ID'\n",
       "\\item 'Pink-footed goose gene name'\n",
       "\\item 'Pink-footed goose protein or transcript stable ID'\n",
       "\\item 'Pink-footed goose chromosome/scaffold name'\n",
       "\\item 'Pink-footed goose chromosome/scaffold start (bp)'\n",
       "\\item 'Pink-footed goose chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Pink-footed goose'\n",
       "\\item 'Pink-footed goose homology type'\n",
       "\\item '\\%id. target Pink-footed goose gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Pink-footed goose gene'\n",
       "\\item 'Pink-footed goose Gene-order conservation score'\n",
       "\\item 'Pink-footed goose Whole-genome alignment coverage'\n",
       "\\item 'dN with Pink-footed goose'\n",
       "\\item 'dS with Pink-footed goose'\n",
       "\\item 'Pink-footed goose orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Platyfish gene stable ID'\n",
       "\\item 'Platyfish gene name'\n",
       "\\item 'Platyfish protein or transcript stable ID'\n",
       "\\item 'Platyfish chromosome/scaffold name'\n",
       "\\item 'Platyfish chromosome/scaffold start (bp)'\n",
       "\\item 'Platyfish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Platyfish'\n",
       "\\item 'Platyfish homology type'\n",
       "\\item '\\%id. target Platyfish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Platyfish gene'\n",
       "\\item 'Platyfish Gene-order conservation score'\n",
       "\\item 'Platyfish Whole-genome alignment coverage'\n",
       "\\item 'dN with Platyfish'\n",
       "\\item 'dS with Platyfish'\n",
       "\\item 'Platyfish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Platypus gene stable ID'\n",
       "\\item 'Platypus gene name'\n",
       "\\item 'Platypus protein or transcript stable ID'\n",
       "\\item 'Platypus chromosome/scaffold name'\n",
       "\\item 'Platypus chromosome/scaffold start (bp)'\n",
       "\\item 'Platypus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Platypus'\n",
       "\\item 'Platypus homology type'\n",
       "\\item '\\%id. target Platypus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Platypus gene'\n",
       "\\item 'Platypus Gene-order conservation score'\n",
       "\\item 'Platypus Whole-genome alignment coverage'\n",
       "\\item 'dN with Platypus'\n",
       "\\item 'dS with Platypus'\n",
       "\\item 'Platypus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Polar bear gene stable ID'\n",
       "\\item 'Polar bear gene name'\n",
       "\\item 'Polar bear protein or transcript stable ID'\n",
       "\\item 'Polar bear chromosome/scaffold name'\n",
       "\\item 'Polar bear chromosome/scaffold start (bp)'\n",
       "\\item 'Polar bear chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Polar bear'\n",
       "\\item 'Polar bear homology type'\n",
       "\\item '\\%id. target Polar bear gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Polar bear gene'\n",
       "\\item 'Polar bear Gene-order conservation score'\n",
       "\\item 'Polar bear Whole-genome alignment coverage'\n",
       "\\item 'dN with Polar bear'\n",
       "\\item 'dS with Polar bear'\n",
       "\\item 'Polar bear orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Prairie vole gene stable ID'\n",
       "\\item 'Prairie vole gene name'\n",
       "\\item 'Prairie vole protein or transcript stable ID'\n",
       "\\item 'Prairie vole chromosome/scaffold name'\n",
       "\\item 'Prairie vole chromosome/scaffold start (bp)'\n",
       "\\item 'Prairie vole chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Prairie vole'\n",
       "\\item 'Prairie vole homology type'\n",
       "\\item '\\%id. target Prairie vole gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Prairie vole gene'\n",
       "\\item 'Prairie vole Gene-order conservation score'\n",
       "\\item 'Prairie vole Whole-genome alignment coverage'\n",
       "\\item 'dN with Prairie vole'\n",
       "\\item 'dS with Prairie vole'\n",
       "\\item 'Prairie vole orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Rabbit gene stable ID'\n",
       "\\item 'Rabbit gene name'\n",
       "\\item 'Rabbit protein or transcript stable ID'\n",
       "\\item 'Rabbit chromosome/scaffold name'\n",
       "\\item 'Rabbit chromosome/scaffold start (bp)'\n",
       "\\item 'Rabbit chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Rabbit'\n",
       "\\item 'Rabbit homology type'\n",
       "\\item '\\%id. target Rabbit gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Rabbit gene'\n",
       "\\item 'Rabbit Gene-order conservation score'\n",
       "\\item 'Rabbit Whole-genome alignment coverage'\n",
       "\\item 'dN with Rabbit'\n",
       "\\item 'dS with Rabbit'\n",
       "\\item 'Rabbit orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Rat gene stable ID'\n",
       "\\item 'Rat gene name'\n",
       "\\item 'Rat protein or transcript stable ID'\n",
       "\\item 'Rat chromosome/scaffold name'\n",
       "\\item 'Rat chromosome/scaffold start (bp)'\n",
       "\\item 'Rat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Rat'\n",
       "\\item 'Rat homology type'\n",
       "\\item '\\%id. target Rat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Rat gene'\n",
       "\\item 'Rat Gene-order conservation score'\n",
       "\\item 'Rat Whole-genome alignment coverage'\n",
       "\\item 'dN with Rat'\n",
       "\\item 'dS with Rat'\n",
       "\\item 'Rat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Red fox gene stable ID'\n",
       "\\item 'Red fox gene name'\n",
       "\\item 'Red fox protein or transcript stable ID'\n",
       "\\item 'Red fox chromosome/scaffold name'\n",
       "\\item 'Red fox chromosome/scaffold start (bp)'\n",
       "\\item 'Red fox chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Red fox'\n",
       "\\item 'Red fox homology type'\n",
       "\\item '\\%id. target Red fox gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Red fox gene'\n",
       "\\item 'Red fox Gene-order conservation score'\n",
       "\\item 'Red fox Whole-genome alignment coverage'\n",
       "\\item 'dN with Red fox'\n",
       "\\item 'dS with Red fox'\n",
       "\\item 'Red fox orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Red-bellied piranha gene stable ID'\n",
       "\\item 'Red-bellied piranha gene name'\n",
       "\\item 'Red-bellied piranha protein or transcript stable ID'\n",
       "\\item 'Red-bellied piranha chromosome/scaffold name'\n",
       "\\item 'Red-bellied piranha chromosome/scaffold start (bp)'\n",
       "\\item 'Red-bellied piranha chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Red-bellied piranha'\n",
       "\\item 'Red-bellied piranha homology type'\n",
       "\\item '\\%id. target Red-bellied piranha gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Red-bellied piranha gene'\n",
       "\\item 'Red-bellied piranha Gene-order conservation score'\n",
       "\\item 'Red-bellied piranha Whole-genome alignment coverage'\n",
       "\\item 'dN with Red-bellied piranha'\n",
       "\\item 'dS with Red-bellied piranha'\n",
       "\\item 'Red-bellied piranha orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ruff gene stable ID'\n",
       "\\item 'Ruff gene name'\n",
       "\\item 'Ruff protein or transcript stable ID'\n",
       "\\item 'Ruff chromosome/scaffold name'\n",
       "\\item 'Ruff chromosome/scaffold start (bp)'\n",
       "\\item 'Ruff chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ruff'\n",
       "\\item 'Ruff homology type'\n",
       "\\item '\\%id. target Ruff gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ruff gene'\n",
       "\\item 'Ruff Gene-order conservation score'\n",
       "\\item 'Ruff Whole-genome alignment coverage'\n",
       "\\item 'dN with Ruff'\n",
       "\\item 'dS with Ruff'\n",
       "\\item 'Ruff orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ryukyu mouse gene stable ID'\n",
       "\\item 'Ryukyu mouse gene name'\n",
       "\\item 'Ryukyu mouse protein or transcript stable ID'\n",
       "\\item 'Ryukyu mouse chromosome/scaffold name'\n",
       "\\item 'Ryukyu mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Ryukyu mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ryukyu mouse'\n",
       "\\item 'Ryukyu mouse homology type'\n",
       "\\item '\\%id. target Ryukyu mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ryukyu mouse gene'\n",
       "\\item 'Ryukyu mouse Gene-order conservation score'\n",
       "\\item 'Ryukyu mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Ryukyu mouse'\n",
       "\\item 'dS with Ryukyu mouse'\n",
       "\\item 'Ryukyu mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Saccharomyces cerevisiae gene stable ID'\n",
       "\\item 'Saccharomyces cerevisiae gene name'\n",
       "\\item 'Saccharomyces cerevisiae protein or transcript stable ID'\n",
       "\\item 'Saccharomyces cerevisiae chromosome/scaffold name'\n",
       "\\item 'Saccharomyces cerevisiae chromosome/scaffold start (bp)'\n",
       "\\item 'Saccharomyces cerevisiae chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Saccharomyces cerevisiae'\n",
       "\\item 'Saccharomyces cerevisiae homology type'\n",
       "\\item '\\%id. target Saccharomyces cerevisiae gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Saccharomyces cerevisiae gene'\n",
       "\\item 'Saccharomyces cerevisiae Gene-order conservation score'\n",
       "\\item 'Saccharomyces cerevisiae Whole-genome alignment coverage'\n",
       "\\item 'dN with Saccharomyces cerevisiae'\n",
       "\\item 'dS with Saccharomyces cerevisiae'\n",
       "\\item 'Saccharomyces cerevisiae orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Sailfin molly gene stable ID'\n",
       "\\item 'Sailfin molly gene name'\n",
       "\\item 'Sailfin molly protein or transcript stable ID'\n",
       "\\item 'Sailfin molly chromosome/scaffold name'\n",
       "\\item 'Sailfin molly chromosome/scaffold start (bp)'\n",
       "\\item 'Sailfin molly chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Sailfin molly'\n",
       "\\item 'Sailfin molly homology type'\n",
       "\\item '\\%id. target Sailfin molly gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Sailfin molly gene'\n",
       "\\item 'Sailfin molly Gene-order conservation score'\n",
       "\\item 'Sailfin molly Whole-genome alignment coverage'\n",
       "\\item 'dN with Sailfin molly'\n",
       "\\item 'dS with Sailfin molly'\n",
       "\\item 'Sailfin molly orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Sheep gene stable ID'\n",
       "\\item 'Sheep gene name'\n",
       "\\item 'Sheep protein or transcript stable ID'\n",
       "\\item 'Sheep chromosome/scaffold name'\n",
       "\\item 'Sheep chromosome/scaffold start (bp)'\n",
       "\\item 'Sheep chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Sheep'\n",
       "\\item 'Sheep homology type'\n",
       "\\item '\\%id. target Sheep gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Sheep gene'\n",
       "\\item 'Sheep Gene-order conservation score'\n",
       "\\item 'Sheep Whole-genome alignment coverage'\n",
       "\\item 'dN with Sheep'\n",
       "\\item 'dS with Sheep'\n",
       "\\item 'Sheep orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Sheepshead minnow gene stable ID'\n",
       "\\item 'Sheepshead minnow gene name'\n",
       "\\item 'Sheepshead minnow protein or transcript stable ID'\n",
       "\\item 'Sheepshead minnow chromosome/scaffold name'\n",
       "\\item 'Sheepshead minnow chromosome/scaffold start (bp)'\n",
       "\\item 'Sheepshead minnow chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Sheepshead minnow'\n",
       "\\item 'Sheepshead minnow homology type'\n",
       "\\item '\\%id. target Sheepshead minnow gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Sheepshead minnow gene'\n",
       "\\item 'Sheepshead minnow Gene-order conservation score'\n",
       "\\item 'Sheepshead minnow Whole-genome alignment coverage'\n",
       "\\item 'dN with Sheepshead minnow'\n",
       "\\item 'dS with Sheepshead minnow'\n",
       "\\item 'Sheepshead minnow orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Shortfin molly gene stable ID'\n",
       "\\item 'Shortfin molly gene name'\n",
       "\\item 'Shortfin molly protein or transcript stable ID'\n",
       "\\item 'Shortfin molly chromosome/scaffold name'\n",
       "\\item 'Shortfin molly chromosome/scaffold start (bp)'\n",
       "\\item 'Shortfin molly chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Shortfin molly'\n",
       "\\item 'Shortfin molly homology type'\n",
       "\\item '\\%id. target Shortfin molly gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Shortfin molly gene'\n",
       "\\item 'Shortfin molly Gene-order conservation score'\n",
       "\\item 'Shortfin molly Whole-genome alignment coverage'\n",
       "\\item 'dN with Shortfin molly'\n",
       "\\item 'dS with Shortfin molly'\n",
       "\\item 'Shortfin molly orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Shrew gene stable ID'\n",
       "\\item 'Shrew gene name'\n",
       "\\item 'Shrew protein or transcript stable ID'\n",
       "\\item 'Shrew chromosome/scaffold name'\n",
       "\\item 'Shrew chromosome/scaffold start (bp)'\n",
       "\\item 'Shrew chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Shrew'\n",
       "\\item 'Shrew homology type'\n",
       "\\item '\\%id. target Shrew gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Shrew gene'\n",
       "\\item 'Shrew Gene-order conservation score'\n",
       "\\item 'Shrew Whole-genome alignment coverage'\n",
       "\\item 'dN with Shrew'\n",
       "\\item 'dS with Shrew'\n",
       "\\item 'Shrew orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Shrew mouse gene stable ID'\n",
       "\\item 'Shrew mouse gene name'\n",
       "\\item 'Shrew mouse protein or transcript stable ID'\n",
       "\\item 'Shrew mouse chromosome/scaffold name'\n",
       "\\item 'Shrew mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Shrew mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Shrew mouse'\n",
       "\\item 'Shrew mouse homology type'\n",
       "\\item '\\%id. target Shrew mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Shrew mouse gene'\n",
       "\\item 'Shrew mouse Gene-order conservation score'\n",
       "\\item 'Shrew mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Shrew mouse'\n",
       "\\item 'dS with Shrew mouse'\n",
       "\\item 'Shrew mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Sloth gene stable ID'\n",
       "\\item 'Sloth gene name'\n",
       "\\item 'Sloth protein or transcript stable ID'\n",
       "\\item 'Sloth chromosome/scaffold name'\n",
       "\\item 'Sloth chromosome/scaffold start (bp)'\n",
       "\\item 'Sloth chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Sloth'\n",
       "\\item 'Sloth homology type'\n",
       "\\item '\\%id. target Sloth gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Sloth gene'\n",
       "\\item 'Sloth Gene-order conservation score'\n",
       "\\item 'Sloth Whole-genome alignment coverage'\n",
       "\\item 'dN with Sloth'\n",
       "\\item 'dS with Sloth'\n",
       "\\item 'Sloth orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Sooty mangabey gene stable ID'\n",
       "\\item 'Sooty mangabey gene name'\n",
       "\\item 'Sooty mangabey protein or transcript stable ID'\n",
       "\\item 'Sooty mangabey chromosome/scaffold name'\n",
       "\\item 'Sooty mangabey chromosome/scaffold start (bp)'\n",
       "\\item 'Sooty mangabey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Sooty mangabey'\n",
       "\\item 'Sooty mangabey homology type'\n",
       "\\item '\\%id. target Sooty mangabey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Sooty mangabey gene'\n",
       "\\item 'Sooty mangabey Gene-order conservation score'\n",
       "\\item 'Sooty mangabey Whole-genome alignment coverage'\n",
       "\\item 'dN with Sooty mangabey'\n",
       "\\item 'dS with Sooty mangabey'\n",
       "\\item 'Sooty mangabey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Spiny chromis gene stable ID'\n",
       "\\item 'Spiny chromis gene name'\n",
       "\\item 'Spiny chromis protein or transcript stable ID'\n",
       "\\item 'Spiny chromis chromosome/scaffold name'\n",
       "\\item 'Spiny chromis chromosome/scaffold start (bp)'\n",
       "\\item 'Spiny chromis chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Spiny chromis'\n",
       "\\item 'Spiny chromis homology type'\n",
       "\\item '\\%id. target Spiny chromis gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Spiny chromis gene'\n",
       "\\item 'Spiny chromis Gene-order conservation score'\n",
       "\\item 'Spiny chromis Whole-genome alignment coverage'\n",
       "\\item 'dN with Spiny chromis'\n",
       "\\item 'dS with Spiny chromis'\n",
       "\\item 'Spiny chromis orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Spoon-billed sandpiper gene stable ID'\n",
       "\\item 'Spoon-billed sandpiper gene name'\n",
       "\\item 'Spoon-billed sandpiper protein or transcript stable ID'\n",
       "\\item 'Spoon-billed sandpiper chromosome/scaffold name'\n",
       "\\item 'Spoon-billed sandpiper chromosome/scaffold start (bp)'\n",
       "\\item 'Spoon-billed sandpiper chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Spoon-billed sandpiper'\n",
       "\\item 'Spoon-billed sandpiper homology type'\n",
       "\\item '\\%id. target Spoon-billed sandpiper gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Spoon-billed sandpiper gene'\n",
       "\\item 'Spoon-billed sandpiper Gene-order conservation score'\n",
       "\\item 'Spoon-billed sandpiper Whole-genome alignment coverage'\n",
       "\\item 'dN with Spoon-billed sandpiper'\n",
       "\\item 'dS with Spoon-billed sandpiper'\n",
       "\\item 'Spoon-billed sandpiper orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Spotted gar gene stable ID'\n",
       "\\item 'Spotted gar gene name'\n",
       "\\item 'Spotted gar protein or transcript stable ID'\n",
       "\\item 'Spotted gar chromosome/scaffold name'\n",
       "\\item 'Spotted gar chromosome/scaffold start (bp)'\n",
       "\\item 'Spotted gar chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Spotted gar'\n",
       "\\item 'Spotted gar homology type'\n",
       "\\item '\\%id. target Spotted gar gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Spotted gar gene'\n",
       "\\item 'Spotted gar Gene-order conservation score'\n",
       "\\item 'Spotted gar Whole-genome alignment coverage'\n",
       "\\item 'dN with Spotted gar'\n",
       "\\item 'dS with Spotted gar'\n",
       "\\item 'Spotted gar orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Squirrel gene stable ID'\n",
       "\\item 'Squirrel gene name'\n",
       "\\item 'Squirrel protein or transcript stable ID'\n",
       "\\item 'Squirrel chromosome/scaffold name'\n",
       "\\item 'Squirrel chromosome/scaffold start (bp)'\n",
       "\\item 'Squirrel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Squirrel'\n",
       "\\item 'Squirrel homology type'\n",
       "\\item '\\%id. target Squirrel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Squirrel gene'\n",
       "\\item 'Squirrel Gene-order conservation score'\n",
       "\\item 'Squirrel Whole-genome alignment coverage'\n",
       "\\item 'dN with Squirrel'\n",
       "\\item 'dS with Squirrel'\n",
       "\\item 'Squirrel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Steppe mouse gene stable ID'\n",
       "\\item 'Steppe mouse gene name'\n",
       "\\item 'Steppe mouse protein or transcript stable ID'\n",
       "\\item 'Steppe mouse chromosome/scaffold name'\n",
       "\\item 'Steppe mouse chromosome/scaffold start (bp)'\n",
       "\\item 'Steppe mouse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Steppe mouse'\n",
       "\\item 'Steppe mouse homology type'\n",
       "\\item '\\%id. target Steppe mouse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Steppe mouse gene'\n",
       "\\item 'Steppe mouse Gene-order conservation score'\n",
       "\\item 'Steppe mouse Whole-genome alignment coverage'\n",
       "\\item 'dN with Steppe mouse'\n",
       "\\item 'dS with Steppe mouse'\n",
       "\\item 'Steppe mouse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Stickleback gene stable ID'\n",
       "\\item 'Stickleback gene name'\n",
       "\\item 'Stickleback protein or transcript stable ID'\n",
       "\\item 'Stickleback chromosome/scaffold name'\n",
       "\\item 'Stickleback chromosome/scaffold start (bp)'\n",
       "\\item 'Stickleback chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Stickleback'\n",
       "\\item 'Stickleback homology type'\n",
       "\\item '\\%id. target Stickleback gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Stickleback gene'\n",
       "\\item 'Stickleback Gene-order conservation score'\n",
       "\\item 'Stickleback Whole-genome alignment coverage'\n",
       "\\item 'dN with Stickleback'\n",
       "\\item 'dS with Stickleback'\n",
       "\\item 'Stickleback orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Swamp eel gene stable ID'\n",
       "\\item 'Swamp eel gene name'\n",
       "\\item 'Swamp eel protein or transcript stable ID'\n",
       "\\item 'Swamp eel chromosome/scaffold name'\n",
       "\\item 'Swamp eel chromosome/scaffold start (bp)'\n",
       "\\item 'Swamp eel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Swamp eel'\n",
       "\\item 'Swamp eel homology type'\n",
       "\\item '\\%id. target Swamp eel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Swamp eel gene'\n",
       "\\item 'Swamp eel Gene-order conservation score'\n",
       "\\item 'Swamp eel Whole-genome alignment coverage'\n",
       "\\item 'dN with Swamp eel'\n",
       "\\item 'dS with Swamp eel'\n",
       "\\item 'Swamp eel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tarsier gene stable ID'\n",
       "\\item 'Tarsier gene name'\n",
       "\\item 'Tarsier protein or transcript stable ID'\n",
       "\\item 'Tarsier chromosome/scaffold name'\n",
       "\\item 'Tarsier chromosome/scaffold start (bp)'\n",
       "\\item 'Tarsier chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tarsier'\n",
       "\\item 'Tarsier homology type'\n",
       "\\item '\\%id. target Tarsier gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tarsier gene'\n",
       "\\item 'Tarsier Gene-order conservation score'\n",
       "\\item 'Tarsier Whole-genome alignment coverage'\n",
       "\\item 'dN with Tarsier'\n",
       "\\item 'dS with Tarsier'\n",
       "\\item 'Tarsier orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tasmanian devil gene stable ID'\n",
       "\\item 'Tasmanian devil gene name'\n",
       "\\item 'Tasmanian devil protein or transcript stable ID'\n",
       "\\item 'Tasmanian devil chromosome/scaffold name'\n",
       "\\item 'Tasmanian devil chromosome/scaffold start (bp)'\n",
       "\\item 'Tasmanian devil chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tasmanian devil'\n",
       "\\item 'Tasmanian devil homology type'\n",
       "\\item '\\%id. target Tasmanian devil gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tasmanian devil gene'\n",
       "\\item 'Tasmanian devil Gene-order conservation score'\n",
       "\\item 'Tasmanian devil Whole-genome alignment coverage'\n",
       "\\item 'dN with Tasmanian devil'\n",
       "\\item 'dS with Tasmanian devil'\n",
       "\\item 'Tasmanian devil orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tetraodon gene stable ID'\n",
       "\\item 'Tetraodon gene name'\n",
       "\\item 'Tetraodon protein or transcript stable ID'\n",
       "\\item 'Tetraodon chromosome/scaffold name'\n",
       "\\item 'Tetraodon chromosome/scaffold start (bp)'\n",
       "\\item 'Tetraodon chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tetraodon'\n",
       "\\item 'Tetraodon homology type'\n",
       "\\item '\\%id. target Tetraodon gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tetraodon gene'\n",
       "\\item 'Tetraodon Gene-order conservation score'\n",
       "\\item 'Tetraodon Whole-genome alignment coverage'\n",
       "\\item 'dN with Tetraodon'\n",
       "\\item 'dS with Tetraodon'\n",
       "\\item 'Tetraodon orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tiger gene stable ID'\n",
       "\\item 'Tiger gene name'\n",
       "\\item 'Tiger protein or transcript stable ID'\n",
       "\\item 'Tiger chromosome/scaffold name'\n",
       "\\item 'Tiger chromosome/scaffold start (bp)'\n",
       "\\item 'Tiger chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tiger'\n",
       "\\item 'Tiger homology type'\n",
       "\\item '\\%id. target Tiger gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tiger gene'\n",
       "\\item 'Tiger Gene-order conservation score'\n",
       "\\item 'Tiger Whole-genome alignment coverage'\n",
       "\\item 'dN with Tiger'\n",
       "\\item 'dS with Tiger'\n",
       "\\item 'Tiger orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tiger tail seahorse gene stable ID'\n",
       "\\item 'Tiger tail seahorse gene name'\n",
       "\\item 'Tiger tail seahorse protein or transcript stable ID'\n",
       "\\item 'Tiger tail seahorse chromosome/scaffold name'\n",
       "\\item 'Tiger tail seahorse chromosome/scaffold start (bp)'\n",
       "\\item 'Tiger tail seahorse chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tiger tail seahorse'\n",
       "\\item 'Tiger tail seahorse homology type'\n",
       "\\item '\\%id. target Tiger tail seahorse gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tiger tail seahorse gene'\n",
       "\\item 'Tiger tail seahorse Gene-order conservation score'\n",
       "\\item 'Tiger tail seahorse Whole-genome alignment coverage'\n",
       "\\item 'dN with Tiger tail seahorse'\n",
       "\\item 'dS with Tiger tail seahorse'\n",
       "\\item 'Tiger tail seahorse orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tilapia gene stable ID'\n",
       "\\item 'Tilapia gene name'\n",
       "\\item 'Tilapia protein or transcript stable ID'\n",
       "\\item 'Tilapia chromosome/scaffold name'\n",
       "\\item 'Tilapia chromosome/scaffold start (bp)'\n",
       "\\item 'Tilapia chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tilapia'\n",
       "\\item 'Tilapia homology type'\n",
       "\\item '\\%id. target Tilapia gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tilapia gene'\n",
       "\\item 'Tilapia Gene-order conservation score'\n",
       "\\item 'Tilapia Whole-genome alignment coverage'\n",
       "\\item 'dN with Tilapia'\n",
       "\\item 'dS with Tilapia'\n",
       "\\item 'Tilapia orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tongue sole gene stable ID'\n",
       "\\item 'Tongue sole gene name'\n",
       "\\item 'Tongue sole protein or transcript stable ID'\n",
       "\\item 'Tongue sole chromosome/scaffold name'\n",
       "\\item 'Tongue sole chromosome/scaffold start (bp)'\n",
       "\\item 'Tongue sole chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tongue sole'\n",
       "\\item 'Tongue sole homology type'\n",
       "\\item '\\%id. target Tongue sole gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tongue sole gene'\n",
       "\\item 'Tongue sole Gene-order conservation score'\n",
       "\\item 'Tongue sole Whole-genome alignment coverage'\n",
       "\\item 'dN with Tongue sole'\n",
       "\\item 'dS with Tongue sole'\n",
       "\\item 'Tongue sole orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tree Shrew gene stable ID'\n",
       "\\item 'Tree Shrew gene name'\n",
       "\\item 'Tree Shrew protein or transcript stable ID'\n",
       "\\item 'Tree Shrew chromosome/scaffold name'\n",
       "\\item 'Tree Shrew chromosome/scaffold start (bp)'\n",
       "\\item 'Tree Shrew chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tree Shrew'\n",
       "\\item 'Tree Shrew homology type'\n",
       "\\item '\\%id. target Tree Shrew gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tree Shrew gene'\n",
       "\\item 'Tree Shrew Gene-order conservation score'\n",
       "\\item 'Tree Shrew Whole-genome alignment coverage'\n",
       "\\item 'dN with Tree Shrew'\n",
       "\\item 'dS with Tree Shrew'\n",
       "\\item 'Tree Shrew orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Tuatara gene stable ID'\n",
       "\\item 'Tuatara gene name'\n",
       "\\item 'Tuatara protein or transcript stable ID'\n",
       "\\item 'Tuatara chromosome/scaffold name'\n",
       "\\item 'Tuatara chromosome/scaffold start (bp)'\n",
       "\\item 'Tuatara chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Tuatara'\n",
       "\\item 'Tuatara homology type'\n",
       "\\item '\\%id. target Tuatara gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Tuatara gene'\n",
       "\\item 'Tuatara Gene-order conservation score'\n",
       "\\item 'Tuatara Whole-genome alignment coverage'\n",
       "\\item 'dN with Tuatara'\n",
       "\\item 'dS with Tuatara'\n",
       "\\item 'Tuatara orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Turbot gene stable ID'\n",
       "\\item 'Turbot gene name'\n",
       "\\item 'Turbot protein or transcript stable ID'\n",
       "\\item 'Turbot chromosome/scaffold name'\n",
       "\\item 'Turbot chromosome/scaffold start (bp)'\n",
       "\\item 'Turbot chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Turbot'\n",
       "\\item 'Turbot homology type'\n",
       "\\item '\\%id. target Turbot gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Turbot gene'\n",
       "\\item 'Turbot Gene-order conservation score'\n",
       "\\item 'Turbot Whole-genome alignment coverage'\n",
       "\\item 'dN with Turbot'\n",
       "\\item 'dS with Turbot'\n",
       "\\item 'Turbot orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Turkey gene stable ID'\n",
       "\\item 'Turkey gene name'\n",
       "\\item 'Turkey protein or transcript stable ID'\n",
       "\\item 'Turkey chromosome/scaffold name'\n",
       "\\item 'Turkey chromosome/scaffold start (bp)'\n",
       "\\item 'Turkey chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Turkey'\n",
       "\\item 'Turkey homology type'\n",
       "\\item '\\%id. target Turkey gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Turkey gene'\n",
       "\\item 'Turkey Gene-order conservation score'\n",
       "\\item 'Turkey Whole-genome alignment coverage'\n",
       "\\item 'dN with Turkey'\n",
       "\\item 'dS with Turkey'\n",
       "\\item 'Turkey orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Ugandan red Colobus gene stable ID'\n",
       "\\item 'Ugandan red Colobus gene name'\n",
       "\\item 'Ugandan red Colobus protein or transcript stable ID'\n",
       "\\item 'Ugandan red Colobus chromosome/scaffold name'\n",
       "\\item 'Ugandan red Colobus chromosome/scaffold start (bp)'\n",
       "\\item 'Ugandan red Colobus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Ugandan red Colobus'\n",
       "\\item 'Ugandan red Colobus homology type'\n",
       "\\item '\\%id. target Ugandan red Colobus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Ugandan red Colobus gene'\n",
       "\\item 'Ugandan red Colobus Gene-order conservation score'\n",
       "\\item 'Ugandan red Colobus Whole-genome alignment coverage'\n",
       "\\item 'dN with Ugandan red Colobus'\n",
       "\\item 'dS with Ugandan red Colobus'\n",
       "\\item 'Ugandan red Colobus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Upper Galilee mountains blind mole rat gene stable ID'\n",
       "\\item 'Upper Galilee mountains blind mole rat gene name'\n",
       "\\item 'Upper Galilee mountains blind mole rat protein or transcript stable ID'\n",
       "\\item 'Upper Galilee mountains blind mole rat chromosome/scaffold name'\n",
       "\\item 'Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)'\n",
       "\\item 'Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Upper Galilee mountains blind mole rat'\n",
       "\\item 'Upper Galilee mountains blind mole rat homology type'\n",
       "\\item '\\%id. target Upper Galilee mountains blind mole rat gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Upper Galilee mountains blind mole rat gene'\n",
       "\\item 'Upper Galilee mountains blind mole rat Gene-order conservation score'\n",
       "\\item 'Upper Galilee mountains blind mole rat Whole-genome alignment coverage'\n",
       "\\item 'dN with Upper Galilee mountains blind mole rat'\n",
       "\\item 'dS with Upper Galilee mountains blind mole rat'\n",
       "\\item 'Upper Galilee mountains blind mole rat orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Vervet-AGM gene stable ID'\n",
       "\\item 'Vervet-AGM gene name'\n",
       "\\item 'Vervet-AGM protein or transcript stable ID'\n",
       "\\item 'Vervet-AGM chromosome/scaffold name'\n",
       "\\item 'Vervet-AGM chromosome/scaffold start (bp)'\n",
       "\\item 'Vervet-AGM chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Vervet-AGM'\n",
       "\\item 'Vervet-AGM homology type'\n",
       "\\item '\\%id. target Vervet-AGM gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Vervet-AGM gene'\n",
       "\\item 'Vervet-AGM Gene-order conservation score'\n",
       "\\item 'Vervet-AGM Whole-genome alignment coverage'\n",
       "\\item 'dN with Vervet-AGM'\n",
       "\\item 'dS with Vervet-AGM'\n",
       "\\item 'Vervet-AGM orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Wallaby gene stable ID'\n",
       "\\item 'Wallaby gene name'\n",
       "\\item 'Wallaby protein or transcript stable ID'\n",
       "\\item 'Wallaby chromosome/scaffold name'\n",
       "\\item 'Wallaby chromosome/scaffold start (bp)'\n",
       "\\item 'Wallaby chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Wallaby'\n",
       "\\item 'Wallaby homology type'\n",
       "\\item '\\%id. target Wallaby gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Wallaby gene'\n",
       "\\item 'Wallaby Gene-order conservation score'\n",
       "\\item 'Wallaby Whole-genome alignment coverage'\n",
       "\\item 'dN with Wallaby'\n",
       "\\item 'dS with Wallaby'\n",
       "\\item 'Wallaby orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Western mosquitofish gene stable ID'\n",
       "\\item 'Western mosquitofish gene name'\n",
       "\\item 'Western mosquitofish protein or transcript stable ID'\n",
       "\\item 'Western mosquitofish chromosome/scaffold name'\n",
       "\\item 'Western mosquitofish chromosome/scaffold start (bp)'\n",
       "\\item 'Western mosquitofish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Western mosquitofish'\n",
       "\\item 'Western mosquitofish homology type'\n",
       "\\item '\\%id. target Western mosquitofish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Western mosquitofish gene'\n",
       "\\item 'Western mosquitofish Gene-order conservation score'\n",
       "\\item 'Western mosquitofish Whole-genome alignment coverage'\n",
       "\\item 'dN with Western mosquitofish'\n",
       "\\item 'dS with Western mosquitofish'\n",
       "\\item 'Western mosquitofish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'White-throated sparrow gene stable ID'\n",
       "\\item 'White-throated sparrow gene name'\n",
       "\\item 'White-throated sparrow protein or transcript stable ID'\n",
       "\\item 'White-throated sparrow chromosome/scaffold name'\n",
       "\\item 'White-throated sparrow chromosome/scaffold start (bp)'\n",
       "\\item 'White-throated sparrow chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with White-throated sparrow'\n",
       "\\item 'White-throated sparrow homology type'\n",
       "\\item '\\%id. target White-throated sparrow gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target White-throated sparrow gene'\n",
       "\\item 'White-throated sparrow Gene-order conservation score'\n",
       "\\item 'White-throated sparrow Whole-genome alignment coverage'\n",
       "\\item 'dN with White-throated sparrow'\n",
       "\\item 'dS with White-throated sparrow'\n",
       "\\item 'White-throated sparrow orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Wild yak gene stable ID'\n",
       "\\item 'Wild yak gene name'\n",
       "\\item 'Wild yak protein or transcript stable ID'\n",
       "\\item 'Wild yak chromosome/scaffold name'\n",
       "\\item 'Wild yak chromosome/scaffold start (bp)'\n",
       "\\item 'Wild yak chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Wild yak'\n",
       "\\item 'Wild yak homology type'\n",
       "\\item '\\%id. target Wild yak gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Wild yak gene'\n",
       "\\item 'Wild yak Gene-order conservation score'\n",
       "\\item 'Wild yak Whole-genome alignment coverage'\n",
       "\\item 'dN with Wild yak'\n",
       "\\item 'dS with Wild yak'\n",
       "\\item 'Wild yak orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Xenopus gene stable ID'\n",
       "\\item 'Xenopus gene name'\n",
       "\\item 'Xenopus protein or transcript stable ID'\n",
       "\\item 'Xenopus chromosome/scaffold name'\n",
       "\\item 'Xenopus chromosome/scaffold start (bp)'\n",
       "\\item 'Xenopus chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Xenopus'\n",
       "\\item 'Xenopus homology type'\n",
       "\\item '\\%id. target Xenopus gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Xenopus gene'\n",
       "\\item 'Xenopus Gene-order conservation score'\n",
       "\\item 'Xenopus Whole-genome alignment coverage'\n",
       "\\item 'dN with Xenopus'\n",
       "\\item 'dS with Xenopus'\n",
       "\\item 'Xenopus orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Yellowtail amberjack gene stable ID'\n",
       "\\item 'Yellowtail amberjack gene name'\n",
       "\\item 'Yellowtail amberjack protein or transcript stable ID'\n",
       "\\item 'Yellowtail amberjack chromosome/scaffold name'\n",
       "\\item 'Yellowtail amberjack chromosome/scaffold start (bp)'\n",
       "\\item 'Yellowtail amberjack chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Yellowtail amberjack'\n",
       "\\item 'Yellowtail amberjack homology type'\n",
       "\\item '\\%id. target Yellowtail amberjack gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Yellowtail amberjack gene'\n",
       "\\item 'Yellowtail amberjack Gene-order conservation score'\n",
       "\\item 'Yellowtail amberjack Whole-genome alignment coverage'\n",
       "\\item 'dN with Yellowtail amberjack'\n",
       "\\item 'dS with Yellowtail amberjack'\n",
       "\\item 'Yellowtail amberjack orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Zebra Finch gene stable ID'\n",
       "\\item 'Zebra Finch gene name'\n",
       "\\item 'Zebra Finch protein or transcript stable ID'\n",
       "\\item 'Zebra Finch chromosome/scaffold name'\n",
       "\\item 'Zebra Finch chromosome/scaffold start (bp)'\n",
       "\\item 'Zebra Finch chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Zebra Finch'\n",
       "\\item 'Zebra Finch homology type'\n",
       "\\item '\\%id. target Zebra Finch gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Zebra Finch gene'\n",
       "\\item 'Zebra Finch Gene-order conservation score'\n",
       "\\item 'Zebra Finch Whole-genome alignment coverage'\n",
       "\\item 'dN with Zebra Finch'\n",
       "\\item 'dS with Zebra Finch'\n",
       "\\item 'Zebra Finch orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Zebra mbuna gene stable ID'\n",
       "\\item 'Zebra mbuna gene name'\n",
       "\\item 'Zebra mbuna protein or transcript stable ID'\n",
       "\\item 'Zebra mbuna chromosome/scaffold name'\n",
       "\\item 'Zebra mbuna chromosome/scaffold start (bp)'\n",
       "\\item 'Zebra mbuna chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Zebra mbuna'\n",
       "\\item 'Zebra mbuna homology type'\n",
       "\\item '\\%id. target Zebra mbuna gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Zebra mbuna gene'\n",
       "\\item 'Zebra mbuna Gene-order conservation score'\n",
       "\\item 'Zebra mbuna Whole-genome alignment coverage'\n",
       "\\item 'dN with Zebra mbuna'\n",
       "\\item 'dS with Zebra mbuna'\n",
       "\\item 'Zebra mbuna orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Zebrafish gene stable ID'\n",
       "\\item 'Zebrafish gene name'\n",
       "\\item 'Zebrafish protein or transcript stable ID'\n",
       "\\item 'Zebrafish chromosome/scaffold name'\n",
       "\\item 'Zebrafish chromosome/scaffold start (bp)'\n",
       "\\item 'Zebrafish chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Zebrafish'\n",
       "\\item 'Zebrafish homology type'\n",
       "\\item '\\%id. target Zebrafish gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Zebrafish gene'\n",
       "\\item 'Zebrafish Gene-order conservation score'\n",
       "\\item 'Zebrafish Whole-genome alignment coverage'\n",
       "\\item 'dN with Zebrafish'\n",
       "\\item 'dS with Zebrafish'\n",
       "\\item 'Zebrafish orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Zig-zag eel gene stable ID'\n",
       "\\item 'Zig-zag eel gene name'\n",
       "\\item 'Zig-zag eel protein or transcript stable ID'\n",
       "\\item 'Zig-zag eel chromosome/scaffold name'\n",
       "\\item 'Zig-zag eel chromosome/scaffold start (bp)'\n",
       "\\item 'Zig-zag eel chromosome/scaffold end (bp)'\n",
       "\\item 'Query protein or transcript ID'\n",
       "\\item 'Last common ancestor with Zig-zag eel'\n",
       "\\item 'Zig-zag eel homology type'\n",
       "\\item '\\%id. target Zig-zag eel gene identical to query gene'\n",
       "\\item '\\%id. query gene identical to target Zig-zag eel gene'\n",
       "\\item 'Zig-zag eel Gene-order conservation score'\n",
       "\\item 'Zig-zag eel Whole-genome alignment coverage'\n",
       "\\item 'dN with Zig-zag eel'\n",
       "\\item 'dS with Zig-zag eel'\n",
       "\\item 'Zig-zag eel orthology confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Human paralogue gene stable ID'\n",
       "\\item 'Human paralogue associated gene name'\n",
       "\\item 'Human paralogue protein or transcript ID'\n",
       "\\item 'Human paralogue chromosome/scaffold name'\n",
       "\\item 'Human paralogue chromosome/scaffold start (bp)'\n",
       "\\item 'Human paralogue chromosome/scaffold end (bp)'\n",
       "\\item 'Paralogue query protein or transcript ID'\n",
       "\\item 'Paralogue last common ancestor with Human'\n",
       "\\item 'Human paralogue homology type'\n",
       "\\item 'Paralogue \\%id. target Human gene identical to query gene'\n",
       "\\item 'Paralogue \\%id. query gene identical to target Human gene'\n",
       "\\item 'Paralogue dN with Human'\n",
       "\\item 'Paralogue dS with Human'\n",
       "\\item 'Human paralogy confidence {[}0 low, 1 high{]}'\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Strand'\n",
       "\\item 'Karyotype band'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item 'Transcript count'\n",
       "\\item 'Gene \\% GC content'\n",
       "\\item 'Gene description'\n",
       "\\item 'Variant name'\n",
       "\\item 'Variant source'\n",
       "\\item 'Variant source description'\n",
       "\\item 'Variant alleles'\n",
       "\\item 'Variant supporting evidence'\n",
       "\\item 'Mapweight'\n",
       "\\item 'Minor allele'\n",
       "\\item 'Minor allele frequency'\n",
       "\\item 'Minor allele count'\n",
       "\\item 'Clinical significance'\n",
       "\\item 'Transcript location (bp)'\n",
       "\\item 'Variant chromosome Strand'\n",
       "\\item 'Protein location (aa)'\n",
       "\\item 'chromosome/scaffold position start (bp)'\n",
       "\\item 'Chromosome/scaffold position end (bp)'\n",
       "\\item 'PolyPhen prediction'\n",
       "\\item 'PolyPhen score'\n",
       "\\item 'SIFT prediction'\n",
       "\\item 'SIFT score'\n",
       "\\item 'Distance to transcript'\n",
       "\\item 'CDS start'\n",
       "\\item 'CDS end'\n",
       "\\item 'Protein allele'\n",
       "\\item 'Variant consequence'\n",
       "\\item 'Consequence specific allele'\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Strand'\n",
       "\\item 'Karyotype band'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item 'Transcript count'\n",
       "\\item 'Gene \\% GC content'\n",
       "\\item 'Gene description'\n",
       "\\item 'Variant name'\n",
       "\\item 'Variant source'\n",
       "\\item 'Variant source description'\n",
       "\\item 'Variant alleles'\n",
       "\\item 'Variant supporting evidence'\n",
       "\\item 'Mapweight'\n",
       "\\item 'Minor allele'\n",
       "\\item 'Minor allele frequency'\n",
       "\\item 'Minor allele count'\n",
       "\\item 'Clinical significance'\n",
       "\\item 'Transcript location (bp)'\n",
       "\\item 'Variant chromosome/scaffold strand'\n",
       "\\item 'Protein location (aa)'\n",
       "\\item 'Chromosome/scaffold position start (bp)'\n",
       "\\item 'Chromosome/scaffold position end (bp)'\n",
       "\\item 'PolyPhen prediction'\n",
       "\\item 'PolyPhen score'\n",
       "\\item 'SIFT prediction'\n",
       "\\item 'SIFT score'\n",
       "\\item 'Distance to transcript'\n",
       "\\item 'CDS start'\n",
       "\\item 'CDS end'\n",
       "\\item 'Protein allele'\n",
       "\\item 'Variant consequence'\n",
       "\\item 'Consequence specific allele'\n",
       "\\item 'Unspliced (Transcript)'\n",
       "\\item 'Unspliced (Gene)'\n",
       "\\item 'Flank (Transcript)'\n",
       "\\item 'Flank (Gene)'\n",
       "\\item 'Flank-coding region (Transcript)'\n",
       "\\item 'Flank-coding region (Gene)'\n",
       "\\item '5\\textbackslash{}' UTR'\n",
       "\\item '3\\textbackslash{}' UTR'\n",
       "\\item 'Exon sequences'\n",
       "\\item 'cDNA sequences'\n",
       "\\item 'Coding sequence'\n",
       "\\item 'Peptide'\n",
       "\\item 'upstream\\_flank'\n",
       "\\item 'downstream\\_flank'\n",
       "\\item 'Gene stable ID'\n",
       "\\item 'Gene stable ID version'\n",
       "\\item 'Gene description'\n",
       "\\item 'Gene name'\n",
       "\\item 'Source of gene name'\n",
       "\\item 'Chromosome/scaffold name'\n",
       "\\item 'Gene start (bp)'\n",
       "\\item 'Gene end (bp)'\n",
       "\\item 'Gene type'\n",
       "\\item 'Version (gene)'\n",
       "\\item 'Ensembl Protein Family ID(s)'\n",
       "\\item 'UniParc ID'\n",
       "\\item 'UniProtKB/Swiss-Prot ID'\n",
       "\\item 'UniProtKB/TrEMBL ID'\n",
       "\\item 'CDS start (within cDNA)'\n",
       "\\item 'CDS end (within cDNA)'\n",
       "\\item '5\\textbackslash{}' UTR start'\n",
       "\\item '5\\textbackslash{}' UTR end'\n",
       "\\item '3\\textbackslash{}' UTR start'\n",
       "\\item '3\\textbackslash{}' UTR end'\n",
       "\\item 'Transcript stable ID'\n",
       "\\item 'Transcript stable ID version'\n",
       "\\item 'Protein stable ID'\n",
       "\\item 'Protein stable ID version'\n",
       "\\item 'Transcript type'\n",
       "\\item 'Version (transcript)'\n",
       "\\item 'Version (protein)'\n",
       "\\item 'Strand'\n",
       "\\item 'Transcript start (bp)'\n",
       "\\item 'Transcript end (bp)'\n",
       "\\item 'Transcription start site (TSS)'\n",
       "\\item 'Transcript length (including UTRs and CDS)'\n",
       "\\item 'CDS Length'\n",
       "\\item 'CDS start'\n",
       "\\item 'CDS end'\n",
       "\\item 'Exon stable ID'\n",
       "\\item 'Exon region start (bp)'\n",
       "\\item 'Exon region end (bp)'\n",
       "\\item 'Strand'\n",
       "\\item 'Exon rank in transcript'\n",
       "\\item 'Start phase'\n",
       "\\item 'End phase'\n",
       "\\item 'cDNA coding start'\n",
       "\\item 'cDNA coding end'\n",
       "\\item 'Genomic coding start'\n",
       "\\item 'Genomic coding end'\n",
       "\\item 'Constitutive exon'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'Gene stable ID'\n",
       "2. 'Gene stable ID version'\n",
       "3. 'Transcript stable ID'\n",
       "4. 'Transcript stable ID version'\n",
       "5. 'Protein stable ID'\n",
       "6. 'Protein stable ID version'\n",
       "7. 'Exon stable ID'\n",
       "8. 'Gene description'\n",
       "9. 'Chromosome/scaffold name'\n",
       "10. 'Gene start (bp)'\n",
       "11. 'Gene end (bp)'\n",
       "12. 'Strand'\n",
       "13. 'Karyotype band'\n",
       "14. 'Transcript start (bp)'\n",
       "15. 'Transcript end (bp)'\n",
       "16. 'Transcription start site (TSS)'\n",
       "17. 'Transcript length (including UTRs and CDS)'\n",
       "18. 'Transcript support level (TSL)'\n",
       "19. 'GENCODE basic annotation'\n",
       "20. 'APPRIS annotation'\n",
       "21. 'RefSeq match transcript'\n",
       "22. 'Gene name'\n",
       "23. 'Source of gene name'\n",
       "24. 'Transcript name'\n",
       "25. 'Source of transcript name'\n",
       "26. 'Transcript count'\n",
       "27. 'Gene % GC content'\n",
       "28. 'Gene type'\n",
       "29. 'Transcript type'\n",
       "30. 'Source (gene)'\n",
       "31. 'Source (transcript)'\n",
       "32. 'Version (gene)'\n",
       "33. 'Version (transcript)'\n",
       "34. 'Version (protein)'\n",
       "35. 'Phenotype description'\n",
       "36. 'Source name'\n",
       "37. 'Study external reference'\n",
       "38. 'Strain name'\n",
       "39. 'Strain gender'\n",
       "40. 'P value'\n",
       "41. 'GO term accession'\n",
       "42. 'GO term name'\n",
       "43. 'GO term definition'\n",
       "44. 'GO term evidence code'\n",
       "45. 'GO domain'\n",
       "46. 'GOSlim GOA Accession(s)'\n",
       "47. 'GOSlim GOA Description'\n",
       "48. 'CCDS ID'\n",
       "49. 'ChEMBL ID'\n",
       "50. 'Clone-based (Ensembl) gene ID'\n",
       "51. 'Clone-based (Ensembl) transcript ID'\n",
       "52. 'DataBase of Aberrant 3\\' Splice Sites name'\n",
       "53. 'DataBase of Aberrant 3\\' Splice Sites ID'\n",
       "54. 'DataBase of Aberrant 5\\' Splice Sites name'\n",
       "55. 'DataBase of Aberrant 5\\' Splice Sites ID'\n",
       "56. 'Ensembl Human Transcript ID'\n",
       "57. 'Ensembl Human Translation ID'\n",
       "58. 'EntrezGene transcript name ID'\n",
       "59. 'European Nucleotide Archive ID'\n",
       "60. 'Expression Atlas ID'\n",
       "61. 'GeneDB ID'\n",
       "62. 'HGNC ID'\n",
       "63. 'HGNC symbol'\n",
       "64. 'HGNC transcript name ID'\n",
       "65. 'Human Protein Atlas accession'\n",
       "66. 'Human Protein Atlas ID'\n",
       "67. 'INSDC protein ID'\n",
       "68. 'KEGG Pathway and Enzyme ID'\n",
       "69. 'LRG display in Ensembl gene ID'\n",
       "70. 'LRG display in Ensembl transcript ID'\n",
       "71. 'MEROPS - the Peptidase Database ID'\n",
       "72. 'MIM gene description'\n",
       "73. 'MIM gene accession'\n",
       "74. 'MIM morbid description'\n",
       "75. 'MIM morbid accession'\n",
       "76. 'miRBase accession'\n",
       "77. 'miRBase ID'\n",
       "78. 'miRBase transcript name ID'\n",
       "79. 'NCBI gene description'\n",
       "80. 'NCBI gene accession'\n",
       "81. 'NCBI gene ID'\n",
       "82. 'PDB ID'\n",
       "83. 'Reactome ID'\n",
       "84. 'Reactome gene ID'\n",
       "85. 'Reactome transcript ID'\n",
       "86. 'RefSeq mRNA ID'\n",
       "87. 'RefSeq mRNA predicted ID'\n",
       "88. 'RefSeq ncRNA ID'\n",
       "89. 'RefSeq ncRNA predicted ID'\n",
       "90. 'RefSeq peptide ID'\n",
       "91. 'RefSeq peptide predicted ID'\n",
       "92. 'RFAM ID'\n",
       "93. 'RFAM transcript name ID'\n",
       "94. 'RNAcentral ID'\n",
       "95. 'UCSC Stable ID'\n",
       "96. 'UniParc ID'\n",
       "97. 'UniProtKB Gene Name symbol'\n",
       "98. 'UniProtKB Gene Name ID'\n",
       "99. 'UniProtKB/Swiss-Prot ID'\n",
       "100. 'UniProtKB/TrEMBL ID'\n",
       "101. 'WikiGene description'\n",
       "102. 'WikiGene name'\n",
       "103. 'WikiGene ID'\n",
       "104. 'AFFY HC G110 probe'\n",
       "105. 'AFFY HG Focus probe'\n",
       "106. 'AFFY HG U133A probe'\n",
       "107. 'AFFY HG U133A 2 probe'\n",
       "108. 'AFFY HG U133B probe'\n",
       "109. 'AFFY HG U133 Plus 2 probe'\n",
       "110. 'AFFY HG U95A probe'\n",
       "111. 'AFFY HG U95Av2 probe'\n",
       "112. 'AFFY HG U95B probe'\n",
       "113. 'AFFY HG U95C probe'\n",
       "114. 'AFFY HG U95D probe'\n",
       "115. 'AFFY HG U95E probe'\n",
       "116. 'AFFY HTA 2 0 probe'\n",
       "117. 'AFFY HuEx 1 0 st v2 probe'\n",
       "118. 'AFFY HuGeneFL probe'\n",
       "119. 'AFFY HuGene 1 0 st v1 probe'\n",
       "120. 'AFFY HuGene 2 0 st v1 probe'\n",
       "121. 'AFFY PrimeView probe'\n",
       "122. 'AFFY U133 X3P probe'\n",
       "123. 'AGILENT CGH 44b probe'\n",
       "124. 'AGILENT GPL6848 probe'\n",
       "125. 'AGILENT SurePrint G3 GE 8x60k probe'\n",
       "126. 'AGILENT SurePrint G3 GE 8x60k v2 probe'\n",
       "127. 'AGILENT WholeGenome probe'\n",
       "128. 'AGILENT WholeGenome 4x44k v1 probe'\n",
       "129. 'AGILENT WholeGenome 4x44k v2 probe'\n",
       "130. 'CODELINK CODELINK probe'\n",
       "131. 'ILLUMINA HumanHT 12 V3 probe'\n",
       "132. 'ILLUMINA HumanHT 12 V4 probe'\n",
       "133. 'ILLUMINA HumanRef 8 V3 probe'\n",
       "134. 'ILLUMINA HumanWG 6 V1 probe'\n",
       "135. 'ILLUMINA HumanWG 6 V2 probe'\n",
       "136. 'ILLUMINA HumanWG 6 V3 probe'\n",
       "137. 'PHALANX OneArray probe'\n",
       "138. 'Ensembl Protein Family ID(s)'\n",
       "139. 'Ensembl Family Description'\n",
       "140. 'CDD ID'\n",
       "141. 'CDD start'\n",
       "142. 'CDD end'\n",
       "143. 'Gene3D ID'\n",
       "144. 'Gene3D start'\n",
       "145. 'Gene3D end'\n",
       "146. 'HAMAP ID'\n",
       "147. 'HAMAP start'\n",
       "148. 'HAMAP end'\n",
       "149. 'PANTHER ID'\n",
       "150. 'PANTHER start'\n",
       "151. 'PANTHER end'\n",
       "152. 'Pfam ID'\n",
       "153. 'Pfam start'\n",
       "154. 'Pfam end'\n",
       "155. 'PIRSF ID'\n",
       "156. 'PIRSF start'\n",
       "157. 'PIRSF end'\n",
       "158. 'Prints ID'\n",
       "159. 'Prints start'\n",
       "160. 'Prints end'\n",
       "161. 'ProDom ID'\n",
       "162. 'ProDom start'\n",
       "163. 'ProDom end'\n",
       "164. 'PROSITE patterns ID'\n",
       "165. 'PROSITE patterns start'\n",
       "166. 'PROSITE patterns end'\n",
       "167. 'PROSITE profiles ID'\n",
       "168. 'PROSITE profiles start'\n",
       "169. 'PROSITE profiles end'\n",
       "170. 'SFLD ID'\n",
       "171. 'SFLD start'\n",
       "172. 'SFLD end'\n",
       "173. 'SMART ID'\n",
       "174. 'SMART start'\n",
       "175. 'SMART end'\n",
       "176. 'Superfamily ID'\n",
       "177. 'Superfamily start'\n",
       "178. 'Superfamily end'\n",
       "179. 'TIGRFAM ID'\n",
       "180. 'TIGRFAM start'\n",
       "181. 'TIGRFAM end'\n",
       "182. 'Interpro ID'\n",
       "183. 'Interpro Short Description'\n",
       "184. 'Interpro Description'\n",
       "185. 'Interpro start'\n",
       "186. 'Interpro end'\n",
       "187. 'MobiDB lite'\n",
       "188. 'MobiDB lite start'\n",
       "189. 'MobiDB lite end'\n",
       "190. 'Coiled-coils (Ncoils)'\n",
       "191. 'Coiled-coils (Ncoils) start'\n",
       "192. 'Coiled-coils (Ncoils) end'\n",
       "193. 'Low complexity (Seg)'\n",
       "194. 'Low complexity (Seg) start'\n",
       "195. 'Low complexity (Seg) end'\n",
       "196. 'PDB-ENSP mappings'\n",
       "197. 'PDB-ENSP mappings start'\n",
       "198. 'PDB-ENSP mappings end'\n",
       "199. 'Cleavage site (Signalp)'\n",
       "200. 'Cleavage site (Signalp) start'\n",
       "201. 'Cleavage site (Signalp) end'\n",
       "202. 'Transmembrane helices'\n",
       "203. 'Transmembrane helices start'\n",
       "204. 'Transmembrane helices end'\n",
       "205. 'Gene stable ID'\n",
       "206. 'Gene stable ID version'\n",
       "207. 'Version (gene)'\n",
       "208. 'Transcript stable ID'\n",
       "209. 'Transcript stable ID version'\n",
       "210. 'Version (transcript)'\n",
       "211. 'Protein stable ID'\n",
       "212. 'Protein stable ID version'\n",
       "213. 'Version (protein)'\n",
       "214. 'Chromosome/scaffold name'\n",
       "215. 'Gene start (bp)'\n",
       "216. 'Gene end (bp)'\n",
       "217. 'Transcript start (bp)'\n",
       "218. 'Transcript end (bp)'\n",
       "219. 'Transcription start site (TSS)'\n",
       "220. 'Transcript length (including UTRs and CDS)'\n",
       "221. 'Strand'\n",
       "222. 'Gene name'\n",
       "223. 'Source of gene name'\n",
       "224. '5\\' UTR start'\n",
       "225. '5\\' UTR end'\n",
       "226. '3\\' UTR start'\n",
       "227. '3\\' UTR end'\n",
       "228. 'CDS Length'\n",
       "229. 'Transcript count'\n",
       "230. 'Gene description'\n",
       "231. 'Gene type'\n",
       "232. 'Exon region start (bp)'\n",
       "233. 'Exon region end (bp)'\n",
       "234. 'Constitutive exon'\n",
       "235. 'Exon rank in transcript'\n",
       "236. 'Start phase'\n",
       "237. 'End phase'\n",
       "238. 'cDNA coding start'\n",
       "239. 'cDNA coding end'\n",
       "240. 'Genomic coding start'\n",
       "241. 'Genomic coding end'\n",
       "242. 'Exon stable ID'\n",
       "243. 'CDS start'\n",
       "244. 'CDS end'\n",
       "245. 'Gene stable ID'\n",
       "246. 'Gene stable ID version'\n",
       "247. 'Version (gene)'\n",
       "248. 'Transcript stable ID'\n",
       "249. 'Transcript stable ID version'\n",
       "250. 'Version (transcript)'\n",
       "251. 'Protein stable ID'\n",
       "252. 'Protein stable ID version'\n",
       "253. 'Version (protein)'\n",
       "254. 'Chromosome/scaffold name'\n",
       "255. 'Gene start (bp)'\n",
       "256. 'Gene end (bp)'\n",
       "257. 'Strand'\n",
       "258. 'Karyotype band'\n",
       "259. 'Gene name'\n",
       "260. 'Source of gene name'\n",
       "261. 'Transcript count'\n",
       "262. 'Gene % GC content'\n",
       "263. 'Gene description'\n",
       "264. 'Abingdon island giant tortoise gene stable ID'\n",
       "265. 'Abingdon island giant tortoise gene name'\n",
       "266. 'Abingdon island giant tortoise protein or transcript stable ID'\n",
       "267. 'Abingdon island giant tortoise chromosome/scaffold name'\n",
       "268. 'Abingdon island giant tortoise chromosome/scaffold start (bp)'\n",
       "269. 'Abingdon island giant tortoise chromosome/scaffold end (bp)'\n",
       "270. 'Query protein or transcript ID'\n",
       "271. 'Last common ancestor with Abingdon island giant tortoise'\n",
       "272. 'Abingdon island giant tortoise homology type'\n",
       "273. '%id. target Abingdon island giant tortoise gene identical to query gene'\n",
       "274. '%id. query gene identical to target Abingdon island giant tortoise gene'\n",
       "275. 'Abingdon island giant tortoise Gene-order conservation score'\n",
       "276. 'Abingdon island giant tortoise Whole-genome alignment coverage'\n",
       "277. 'dN with Abingdon island giant tortoise'\n",
       "278. 'dS with Abingdon island giant tortoise'\n",
       "279. 'Abingdon island giant tortoise orthology confidence [0 low, 1 high]'\n",
       "280. 'Agassiz\\'s desert tortoise gene stable ID'\n",
       "281. 'Agassiz\\'s desert tortoise gene name'\n",
       "282. 'Agassiz\\'s desert tortoise protein or transcript stable ID'\n",
       "283. 'Agassiz\\'s desert tortoise chromosome/scaffold name'\n",
       "284. 'Agassiz\\'s desert tortoise chromosome/scaffold start (bp)'\n",
       "285. 'Agassiz\\'s desert tortoise chromosome/scaffold end (bp)'\n",
       "286. 'Query protein or transcript ID'\n",
       "287. 'Last common ancestor with Agassiz\\'s desert tortoise'\n",
       "288. 'Agassiz\\'s desert tortoise homology type'\n",
       "289. '%id. target Agassiz\\'s desert tortoise gene identical to query gene'\n",
       "290. '%id. query gene identical to target Agassiz\\'s desert tortoise gene'\n",
       "291. 'Agassiz\\'s desert tortoise Gene-order conservation score'\n",
       "292. 'Agassiz\\'s desert tortoise Whole-genome alignment coverage'\n",
       "293. 'dN with Agassiz\\'s desert tortoise'\n",
       "294. 'dS with Agassiz\\'s desert tortoise'\n",
       "295. 'Agassiz\\'s desert tortoise orthology confidence [0 low, 1 high]'\n",
       "296. 'Algerian mouse gene stable ID'\n",
       "297. 'Algerian mouse gene name'\n",
       "298. 'Algerian mouse protein or transcript stable ID'\n",
       "299. 'Algerian mouse chromosome/scaffold name'\n",
       "300. 'Algerian mouse chromosome/scaffold start (bp)'\n",
       "301. 'Algerian mouse chromosome/scaffold end (bp)'\n",
       "302. 'Query protein or transcript ID'\n",
       "303. 'Last common ancestor with Algerian mouse'\n",
       "304. 'Algerian mouse homology type'\n",
       "305. '%id. target Algerian mouse gene identical to query gene'\n",
       "306. '%id. query gene identical to target Algerian mouse gene'\n",
       "307. 'Algerian mouse Gene-order conservation score'\n",
       "308. 'Algerian mouse Whole-genome alignment coverage'\n",
       "309. 'dN with Algerian mouse'\n",
       "310. 'dS with Algerian mouse'\n",
       "311. 'Algerian mouse orthology confidence [0 low, 1 high]'\n",
       "312. 'Alpaca gene stable ID'\n",
       "313. 'Alpaca gene name'\n",
       "314. 'Alpaca protein or transcript stable ID'\n",
       "315. 'Alpaca chromosome/scaffold name'\n",
       "316. 'Alpaca chromosome/scaffold start (bp)'\n",
       "317. 'Alpaca chromosome/scaffold end (bp)'\n",
       "318. 'Query protein or transcript ID'\n",
       "319. 'Last common ancestor with Alpaca'\n",
       "320. 'Alpaca homology type'\n",
       "321. '%id. target Alpaca gene identical to query gene'\n",
       "322. '%id. query gene identical to target Alpaca gene'\n",
       "323. 'Alpaca Gene-order conservation score'\n",
       "324. 'Alpaca Whole-genome alignment coverage'\n",
       "325. 'dN with Alpaca'\n",
       "326. 'dS with Alpaca'\n",
       "327. 'Alpaca orthology confidence [0 low, 1 high]'\n",
       "328. 'Alpine marmot gene stable ID'\n",
       "329. 'Alpine marmot gene name'\n",
       "330. 'Alpine marmot protein or transcript stable ID'\n",
       "331. 'Alpine marmot chromosome/scaffold name'\n",
       "332. 'Alpine marmot chromosome/scaffold start (bp)'\n",
       "333. 'Alpine marmot chromosome/scaffold end (bp)'\n",
       "334. 'Query protein or transcript ID'\n",
       "335. 'Last common ancestor with Alpine marmot'\n",
       "336. 'Alpine marmot homology type'\n",
       "337. '%id. target Alpine marmot gene identical to query gene'\n",
       "338. '%id. query gene identical to target Alpine marmot gene'\n",
       "339. 'Alpine marmot Gene-order conservation score'\n",
       "340. 'Alpine marmot Whole-genome alignment coverage'\n",
       "341. 'dN with Alpine marmot'\n",
       "342. 'dS with Alpine marmot'\n",
       "343. 'Alpine marmot orthology confidence [0 low, 1 high]'\n",
       "344. 'Amazon molly gene stable ID'\n",
       "345. 'Amazon molly gene name'\n",
       "346. 'Amazon molly protein or transcript stable ID'\n",
       "347. 'Amazon molly chromosome/scaffold name'\n",
       "348. 'Amazon molly chromosome/scaffold start (bp)'\n",
       "349. 'Amazon molly chromosome/scaffold end (bp)'\n",
       "350. 'Query protein or transcript ID'\n",
       "351. 'Last common ancestor with Amazon molly'\n",
       "352. 'Amazon molly homology type'\n",
       "353. '%id. target Amazon molly gene identical to query gene'\n",
       "354. '%id. query gene identical to target Amazon molly gene'\n",
       "355. 'Amazon molly Gene-order conservation score'\n",
       "356. 'Amazon molly Whole-genome alignment coverage'\n",
       "357. 'dN with Amazon molly'\n",
       "358. 'dS with Amazon molly'\n",
       "359. 'Amazon molly orthology confidence [0 low, 1 high]'\n",
       "360. 'American beaver gene stable ID'\n",
       "361. 'American beaver gene name'\n",
       "362. 'American beaver protein or transcript stable ID'\n",
       "363. 'American beaver chromosome/scaffold name'\n",
       "364. 'American beaver chromosome/scaffold start (bp)'\n",
       "365. 'American beaver chromosome/scaffold end (bp)'\n",
       "366. 'Query protein or transcript ID'\n",
       "367. 'Last common ancestor with American beaver'\n",
       "368. 'American beaver homology type'\n",
       "369. '%id. target American beaver gene identical to query gene'\n",
       "370. '%id. query gene identical to target American beaver gene'\n",
       "371. 'American beaver Gene-order conservation score'\n",
       "372. 'American beaver Whole-genome alignment coverage'\n",
       "373. 'dN with American beaver'\n",
       "374. 'dS with American beaver'\n",
       "375. 'American beaver orthology confidence [0 low, 1 high]'\n",
       "376. 'American bison gene stable ID'\n",
       "377. 'American bison gene name'\n",
       "378. 'American bison protein or transcript stable ID'\n",
       "379. 'American bison chromosome/scaffold name'\n",
       "380. 'American bison chromosome/scaffold start (bp)'\n",
       "381. 'American bison chromosome/scaffold end (bp)'\n",
       "382. 'Query protein or transcript ID'\n",
       "383. 'Last common ancestor with American bison'\n",
       "384. 'American bison homology type'\n",
       "385. '%id. target American bison gene identical to query gene'\n",
       "386. '%id. query gene identical to target American bison gene'\n",
       "387. 'American bison Gene-order conservation score'\n",
       "388. 'American bison Whole-genome alignment coverage'\n",
       "389. 'dN with American bison'\n",
       "390. 'dS with American bison'\n",
       "391. 'American bison orthology confidence [0 low, 1 high]'\n",
       "392. 'American black bear gene stable ID'\n",
       "393. 'American black bear gene name'\n",
       "394. 'American black bear protein or transcript stable ID'\n",
       "395. 'American black bear chromosome/scaffold name'\n",
       "396. 'American black bear chromosome/scaffold start (bp)'\n",
       "397. 'American black bear chromosome/scaffold end (bp)'\n",
       "398. 'Query protein or transcript ID'\n",
       "399. 'Last common ancestor with American black bear'\n",
       "400. 'American black bear homology type'\n",
       "401. '%id. target American black bear gene identical to query gene'\n",
       "402. '%id. query gene identical to target American black bear gene'\n",
       "403. 'American black bear Gene-order conservation score'\n",
       "404. 'American black bear Whole-genome alignment coverage'\n",
       "405. 'dN with American black bear'\n",
       "406. 'dS with American black bear'\n",
       "407. 'American black bear orthology confidence [0 low, 1 high]'\n",
       "408. 'American mink gene stable ID'\n",
       "409. 'American mink gene name'\n",
       "410. 'American mink protein or transcript stable ID'\n",
       "411. 'American mink chromosome/scaffold name'\n",
       "412. 'American mink chromosome/scaffold start (bp)'\n",
       "413. 'American mink chromosome/scaffold end (bp)'\n",
       "414. 'Query protein or transcript ID'\n",
       "415. 'Last common ancestor with American mink'\n",
       "416. 'American mink homology type'\n",
       "417. '%id. target American mink gene identical to query gene'\n",
       "418. '%id. query gene identical to target American mink gene'\n",
       "419. 'American mink Gene-order conservation score'\n",
       "420. 'American mink Whole-genome alignment coverage'\n",
       "421. 'dN with American mink'\n",
       "422. 'dS with American mink'\n",
       "423. 'American mink orthology confidence [0 low, 1 high]'\n",
       "424. 'Angola colobus gene stable ID'\n",
       "425. 'Angola colobus gene name'\n",
       "426. 'Angola colobus protein or transcript stable ID'\n",
       "427. 'Angola colobus chromosome/scaffold name'\n",
       "428. 'Angola colobus chromosome/scaffold start (bp)'\n",
       "429. 'Angola colobus chromosome/scaffold end (bp)'\n",
       "430. 'Query protein or transcript ID'\n",
       "431. 'Last common ancestor with Angola colobus'\n",
       "432. 'Angola colobus homology type'\n",
       "433. '%id. target Angola colobus gene identical to query gene'\n",
       "434. '%id. query gene identical to target Angola colobus gene'\n",
       "435. 'Angola colobus Gene-order conservation score'\n",
       "436. 'Angola colobus Whole-genome alignment coverage'\n",
       "437. 'dN with Angola colobus'\n",
       "438. 'dS with Angola colobus'\n",
       "439. 'Angola colobus orthology confidence [0 low, 1 high]'\n",
       "440. 'Anole lizard gene stable ID'\n",
       "441. 'Anole lizard gene name'\n",
       "442. 'Anole lizard protein or transcript stable ID'\n",
       "443. 'Anole lizard chromosome/scaffold name'\n",
       "444. 'Anole lizard chromosome/scaffold start (bp)'\n",
       "445. 'Anole lizard chromosome/scaffold end (bp)'\n",
       "446. 'Query protein or transcript ID'\n",
       "447. 'Last common ancestor with Anole lizard'\n",
       "448. 'Anole lizard homology type'\n",
       "449. '%id. target Anole lizard gene identical to query gene'\n",
       "450. '%id. query gene identical to target Anole lizard gene'\n",
       "451. 'Anole lizard Gene-order conservation score'\n",
       "452. 'Anole lizard Whole-genome alignment coverage'\n",
       "453. 'dN with Anole lizard'\n",
       "454. 'dS with Anole lizard'\n",
       "455. 'Anole lizard orthology confidence [0 low, 1 high]'\n",
       "456. 'Arctic ground squirrel gene stable ID'\n",
       "457. 'Arctic ground squirrel gene name'\n",
       "458. 'Arctic ground squirrel protein or transcript stable ID'\n",
       "459. 'Arctic ground squirrel chromosome/scaffold name'\n",
       "460. 'Arctic ground squirrel chromosome/scaffold start (bp)'\n",
       "461. 'Arctic ground squirrel chromosome/scaffold end (bp)'\n",
       "462. 'Query protein or transcript ID'\n",
       "463. 'Last common ancestor with Arctic ground squirrel'\n",
       "464. 'Arctic ground squirrel homology type'\n",
       "465. '%id. target Arctic ground squirrel gene identical to query gene'\n",
       "466. '%id. query gene identical to target Arctic ground squirrel gene'\n",
       "467. 'Arctic ground squirrel Gene-order conservation score'\n",
       "468. 'Arctic ground squirrel Whole-genome alignment coverage'\n",
       "469. 'dN with Arctic ground squirrel'\n",
       "470. 'dS with Arctic ground squirrel'\n",
       "471. 'Arctic ground squirrel orthology confidence [0 low, 1 high]'\n",
       "472. 'Argentine black and white tegu gene stable ID'\n",
       "473. 'Argentine black and white tegu gene name'\n",
       "474. 'Argentine black and white tegu protein or transcript stable ID'\n",
       "475. 'Argentine black and white tegu chromosome/scaffold name'\n",
       "476. 'Argentine black and white tegu chromosome/scaffold start (bp)'\n",
       "477. 'Argentine black and white tegu chromosome/scaffold end (bp)'\n",
       "478. 'Query protein or transcript ID'\n",
       "479. 'Last common ancestor with Argentine black and white tegu'\n",
       "480. 'Argentine black and white tegu homology type'\n",
       "481. '%id. target Argentine black and white tegu gene identical to query gene'\n",
       "482. '%id. query gene identical to target Argentine black and white tegu gene'\n",
       "483. 'Argentine black and white tegu Gene-order conservation score'\n",
       "484. 'Argentine black and white tegu Whole-genome alignment coverage'\n",
       "485. 'dN with Argentine black and white tegu'\n",
       "486. 'dS with Argentine black and white tegu'\n",
       "487. 'Argentine black and white tegu orthology confidence [0 low, 1 high]'\n",
       "488. 'Armadillo gene stable ID'\n",
       "489. 'Armadillo gene name'\n",
       "490. 'Armadillo protein or transcript stable ID'\n",
       "491. 'Armadillo chromosome/scaffold name'\n",
       "492. 'Armadillo chromosome/scaffold start (bp)'\n",
       "493. 'Armadillo chromosome/scaffold end (bp)'\n",
       "494. 'Query protein or transcript ID'\n",
       "495. 'Last common ancestor with Armadillo'\n",
       "496. 'Armadillo homology type'\n",
       "497. '%id. target Armadillo gene identical to query gene'\n",
       "498. '%id. query gene identical to target Armadillo gene'\n",
       "499. 'Armadillo Gene-order conservation score'\n",
       "500. 'Armadillo Whole-genome alignment coverage'\n",
       "501. 'dN with Armadillo'\n",
       "502. 'dS with Armadillo'\n",
       "503. 'Armadillo orthology confidence [0 low, 1 high]'\n",
       "504. 'Asian bonytongue gene stable ID'\n",
       "505. 'Asian bonytongue gene name'\n",
       "506. 'Asian bonytongue protein or transcript stable ID'\n",
       "507. 'Asian bonytongue chromosome/scaffold name'\n",
       "508. 'Asian bonytongue chromosome/scaffold start (bp)'\n",
       "509. 'Asian bonytongue chromosome/scaffold end (bp)'\n",
       "510. 'Query protein or transcript ID'\n",
       "511. 'Last common ancestor with Asian bonytongue'\n",
       "512. 'Asian bonytongue homology type'\n",
       "513. '%id. target Asian bonytongue gene identical to query gene'\n",
       "514. '%id. query gene identical to target Asian bonytongue gene'\n",
       "515. 'Asian bonytongue Gene-order conservation score'\n",
       "516. 'Asian bonytongue Whole-genome alignment coverage'\n",
       "517. 'dN with Asian bonytongue'\n",
       "518. 'dS with Asian bonytongue'\n",
       "519. 'Asian bonytongue orthology confidence [0 low, 1 high]'\n",
       "520. 'Australian saltwater crocodile gene stable ID'\n",
       "521. 'Australian saltwater crocodile gene name'\n",
       "522. 'Australian saltwater crocodile protein or transcript stable ID'\n",
       "523. 'Australian saltwater crocodile chromosome/scaffold name'\n",
       "524. 'Australian saltwater crocodile chromosome/scaffold start (bp)'\n",
       "525. 'Australian saltwater crocodile chromosome/scaffold end (bp)'\n",
       "526. 'Query protein or transcript ID'\n",
       "527. 'Last common ancestor with Australian saltwater crocodile'\n",
       "528. 'Australian saltwater crocodile homology type'\n",
       "529. '%id. target Australian saltwater crocodile gene identical to query gene'\n",
       "530. '%id. query gene identical to target Australian saltwater crocodile gene'\n",
       "531. 'Australian saltwater crocodile Gene-order conservation score'\n",
       "532. 'Australian saltwater crocodile Whole-genome alignment coverage'\n",
       "533. 'dN with Australian saltwater crocodile'\n",
       "534. 'dS with Australian saltwater crocodile'\n",
       "535. 'Australian saltwater crocodile orthology confidence [0 low, 1 high]'\n",
       "536. 'Ballan wrasse gene stable ID'\n",
       "537. 'Ballan wrasse gene name'\n",
       "538. 'Ballan wrasse protein or transcript stable ID'\n",
       "539. 'Ballan wrasse chromosome/scaffold name'\n",
       "540. 'Ballan wrasse chromosome/scaffold start (bp)'\n",
       "541. 'Ballan wrasse chromosome/scaffold end (bp)'\n",
       "542. 'Query protein or transcript ID'\n",
       "543. 'Last common ancestor with Ballan wrasse'\n",
       "544. 'Ballan wrasse homology type'\n",
       "545. '%id. target Ballan wrasse gene identical to query gene'\n",
       "546. '%id. query gene identical to target Ballan wrasse gene'\n",
       "547. 'Ballan wrasse Gene-order conservation score'\n",
       "548. 'Ballan wrasse Whole-genome alignment coverage'\n",
       "549. 'dN with Ballan wrasse'\n",
       "550. 'dS with Ballan wrasse'\n",
       "551. 'Ballan wrasse orthology confidence [0 low, 1 high]'\n",
       "552. 'Barramundi perch gene stable ID'\n",
       "553. 'Barramundi perch gene name'\n",
       "554. 'Barramundi perch protein or transcript stable ID'\n",
       "555. 'Barramundi perch chromosome/scaffold name'\n",
       "556. 'Barramundi perch chromosome/scaffold start (bp)'\n",
       "557. 'Barramundi perch chromosome/scaffold end (bp)'\n",
       "558. 'Query protein or transcript ID'\n",
       "559. 'Last common ancestor with Barramundi perch'\n",
       "560. 'Barramundi perch homology type'\n",
       "561. '%id. target Barramundi perch gene identical to query gene'\n",
       "562. '%id. query gene identical to target Barramundi perch gene'\n",
       "563. 'Barramundi perch Gene-order conservation score'\n",
       "564. 'Barramundi perch Whole-genome alignment coverage'\n",
       "565. 'dN with Barramundi perch'\n",
       "566. 'dS with Barramundi perch'\n",
       "567. 'Barramundi perch orthology confidence [0 low, 1 high]'\n",
       "568. 'Bengalese finch gene stable ID'\n",
       "569. 'Bengalese finch gene name'\n",
       "570. 'Bengalese finch protein or transcript stable ID'\n",
       "571. 'Bengalese finch chromosome/scaffold name'\n",
       "572. 'Bengalese finch chromosome/scaffold start (bp)'\n",
       "573. 'Bengalese finch chromosome/scaffold end (bp)'\n",
       "574. 'Query protein or transcript ID'\n",
       "575. 'Last common ancestor with Bengalese finch'\n",
       "576. 'Bengalese finch homology type'\n",
       "577. '%id. target Bengalese finch gene identical to query gene'\n",
       "578. '%id. query gene identical to target Bengalese finch gene'\n",
       "579. 'Bengalese finch Gene-order conservation score'\n",
       "580. 'Bengalese finch Whole-genome alignment coverage'\n",
       "581. 'dN with Bengalese finch'\n",
       "582. 'dS with Bengalese finch'\n",
       "583. 'Bengalese finch orthology confidence [0 low, 1 high]'\n",
       "584. 'Bicolor damselfish gene stable ID'\n",
       "585. 'Bicolor damselfish gene name'\n",
       "586. 'Bicolor damselfish protein or transcript stable ID'\n",
       "587. 'Bicolor damselfish chromosome/scaffold name'\n",
       "588. 'Bicolor damselfish chromosome/scaffold start (bp)'\n",
       "589. 'Bicolor damselfish chromosome/scaffold end (bp)'\n",
       "590. 'Query protein or transcript ID'\n",
       "591. 'Last common ancestor with Bicolor damselfish'\n",
       "592. 'Bicolor damselfish homology type'\n",
       "593. '%id. target Bicolor damselfish gene identical to query gene'\n",
       "594. '%id. query gene identical to target Bicolor damselfish gene'\n",
       "595. 'Bicolor damselfish Gene-order conservation score'\n",
       "596. 'Bicolor damselfish Whole-genome alignment coverage'\n",
       "597. 'dN with Bicolor damselfish'\n",
       "598. 'dS with Bicolor damselfish'\n",
       "599. 'Bicolor damselfish orthology confidence [0 low, 1 high]'\n",
       "600. 'Black snub-nosed monkey gene stable ID'\n",
       "601. 'Black snub-nosed monkey gene name'\n",
       "602. 'Black snub-nosed monkey protein or transcript stable ID'\n",
       "603. 'Black snub-nosed monkey chromosome/scaffold name'\n",
       "604. 'Black snub-nosed monkey chromosome/scaffold start (bp)'\n",
       "605. 'Black snub-nosed monkey chromosome/scaffold end (bp)'\n",
       "606. 'Query protein or transcript ID'\n",
       "607. 'Last common ancestor with Black snub-nosed monkey'\n",
       "608. 'Black snub-nosed monkey homology type'\n",
       "609. '%id. target Black snub-nosed monkey gene identical to query gene'\n",
       "610. '%id. query gene identical to target Black snub-nosed monkey gene'\n",
       "611. 'Black snub-nosed monkey Gene-order conservation score'\n",
       "612. 'Black snub-nosed monkey Whole-genome alignment coverage'\n",
       "613. 'dN with Black snub-nosed monkey'\n",
       "614. 'dS with Black snub-nosed monkey'\n",
       "615. 'Black snub-nosed monkey orthology confidence [0 low, 1 high]'\n",
       "616. 'Blue tit gene stable ID'\n",
       "617. 'Blue tit gene name'\n",
       "618. 'Blue tit protein or transcript stable ID'\n",
       "619. 'Blue tit chromosome/scaffold name'\n",
       "620. 'Blue tit chromosome/scaffold start (bp)'\n",
       "621. 'Blue tit chromosome/scaffold end (bp)'\n",
       "622. 'Query protein or transcript ID'\n",
       "623. 'Last common ancestor with Blue tit'\n",
       "624. 'Blue tit homology type'\n",
       "625. '%id. target Blue tit gene identical to query gene'\n",
       "626. '%id. query gene identical to target Blue tit gene'\n",
       "627. 'Blue tit Gene-order conservation score'\n",
       "628. 'Blue tit Whole-genome alignment coverage'\n",
       "629. 'dN with Blue tit'\n",
       "630. 'dS with Blue tit'\n",
       "631. 'Blue tit orthology confidence [0 low, 1 high]'\n",
       "632. 'Blue-crowned manakin gene stable ID'\n",
       "633. 'Blue-crowned manakin gene name'\n",
       "634. 'Blue-crowned manakin protein or transcript stable ID'\n",
       "635. 'Blue-crowned manakin chromosome/scaffold name'\n",
       "636. 'Blue-crowned manakin chromosome/scaffold start (bp)'\n",
       "637. 'Blue-crowned manakin chromosome/scaffold end (bp)'\n",
       "638. 'Query protein or transcript ID'\n",
       "639. 'Last common ancestor with Blue-crowned manakin'\n",
       "640. 'Blue-crowned manakin homology type'\n",
       "641. '%id. target Blue-crowned manakin gene identical to query gene'\n",
       "642. '%id. query gene identical to target Blue-crowned manakin gene'\n",
       "643. 'Blue-crowned manakin Gene-order conservation score'\n",
       "644. 'Blue-crowned manakin Whole-genome alignment coverage'\n",
       "645. 'dN with Blue-crowned manakin'\n",
       "646. 'dS with Blue-crowned manakin'\n",
       "647. 'Blue-crowned manakin orthology confidence [0 low, 1 high]'\n",
       "648. 'Bolivian squirrel monkey gene stable ID'\n",
       "649. 'Bolivian squirrel monkey gene name'\n",
       "650. 'Bolivian squirrel monkey protein or transcript stable ID'\n",
       "651. 'Bolivian squirrel monkey chromosome/scaffold name'\n",
       "652. 'Bolivian squirrel monkey chromosome/scaffold start (bp)'\n",
       "653. 'Bolivian squirrel monkey chromosome/scaffold end (bp)'\n",
       "654. 'Query protein or transcript ID'\n",
       "655. 'Last common ancestor with Bolivian squirrel monkey'\n",
       "656. 'Bolivian squirrel monkey homology type'\n",
       "657. '%id. target Bolivian squirrel monkey gene identical to query gene'\n",
       "658. '%id. query gene identical to target Bolivian squirrel monkey gene'\n",
       "659. 'Bolivian squirrel monkey Gene-order conservation score'\n",
       "660. 'Bolivian squirrel monkey Whole-genome alignment coverage'\n",
       "661. 'dN with Bolivian squirrel monkey'\n",
       "662. 'dS with Bolivian squirrel monkey'\n",
       "663. 'Bolivian squirrel monkey orthology confidence [0 low, 1 high]'\n",
       "664. 'Bonobo gene stable ID'\n",
       "665. 'Bonobo gene name'\n",
       "666. 'Bonobo protein or transcript stable ID'\n",
       "667. 'Bonobo chromosome/scaffold name'\n",
       "668. 'Bonobo chromosome/scaffold start (bp)'\n",
       "669. 'Bonobo chromosome/scaffold end (bp)'\n",
       "670. 'Query protein or transcript ID'\n",
       "671. 'Last common ancestor with Bonobo'\n",
       "672. 'Bonobo homology type'\n",
       "673. '%id. target Bonobo gene identical to query gene'\n",
       "674. '%id. query gene identical to target Bonobo gene'\n",
       "675. 'Bonobo Gene-order conservation score'\n",
       "676. 'Bonobo Whole-genome alignment coverage'\n",
       "677. 'dN with Bonobo'\n",
       "678. 'dS with Bonobo'\n",
       "679. 'Bonobo orthology confidence [0 low, 1 high]'\n",
       "680. 'Brazilian guinea pig gene stable ID'\n",
       "681. 'Brazilian guinea pig gene name'\n",
       "682. 'Brazilian guinea pig protein or transcript stable ID'\n",
       "683. 'Brazilian guinea pig chromosome/scaffold name'\n",
       "684. 'Brazilian guinea pig chromosome/scaffold start (bp)'\n",
       "685. 'Brazilian guinea pig chromosome/scaffold end (bp)'\n",
       "686. 'Query protein or transcript ID'\n",
       "687. 'Last common ancestor with Brazilian guinea pig'\n",
       "688. 'Brazilian guinea pig homology type'\n",
       "689. '%id. target Brazilian guinea pig gene identical to query gene'\n",
       "690. '%id. query gene identical to target Brazilian guinea pig gene'\n",
       "691. 'Brazilian guinea pig Gene-order conservation score'\n",
       "692. 'Brazilian guinea pig Whole-genome alignment coverage'\n",
       "693. 'dN with Brazilian guinea pig'\n",
       "694. 'dS with Brazilian guinea pig'\n",
       "695. 'Brazilian guinea pig orthology confidence [0 low, 1 high]'\n",
       "696. 'Budgerigar gene stable ID'\n",
       "697. 'Budgerigar gene name'\n",
       "698. 'Budgerigar protein or transcript stable ID'\n",
       "699. 'Budgerigar chromosome/scaffold name'\n",
       "700. 'Budgerigar chromosome/scaffold start (bp)'\n",
       "701. 'Budgerigar chromosome/scaffold end (bp)'\n",
       "702. 'Query protein or transcript ID'\n",
       "703. 'Last common ancestor with Budgerigar'\n",
       "704. 'Budgerigar homology type'\n",
       "705. '%id. target Budgerigar gene identical to query gene'\n",
       "706. '%id. query gene identical to target Budgerigar gene'\n",
       "707. 'Budgerigar Gene-order conservation score'\n",
       "708. 'Budgerigar Whole-genome alignment coverage'\n",
       "709. 'dN with Budgerigar'\n",
       "710. 'dS with Budgerigar'\n",
       "711. 'Budgerigar orthology confidence [0 low, 1 high]'\n",
       "712. 'Burton\\'s mouthbrooder gene stable ID'\n",
       "713. 'Burton\\'s mouthbrooder gene name'\n",
       "714. 'Burton\\'s mouthbrooder protein or transcript stable ID'\n",
       "715. 'Burton\\'s mouthbrooder chromosome/scaffold name'\n",
       "716. 'Burton\\'s mouthbrooder chromosome/scaffold start (bp)'\n",
       "717. 'Burton\\'s mouthbrooder chromosome/scaffold end (bp)'\n",
       "718. 'Query protein or transcript ID'\n",
       "719. 'Last common ancestor with Burton\\'s mouthbrooder'\n",
       "720. 'Burton\\'s mouthbrooder homology type'\n",
       "721. '%id. target Burton\\'s mouthbrooder gene identical to query gene'\n",
       "722. '%id. query gene identical to target Burton\\'s mouthbrooder gene'\n",
       "723. 'Burton\\'s mouthbrooder Gene-order conservation score'\n",
       "724. 'Burton\\'s mouthbrooder Whole-genome alignment coverage'\n",
       "725. 'dN with Burton\\'s mouthbrooder'\n",
       "726. 'dS with Burton\\'s mouthbrooder'\n",
       "727. 'Burton\\'s mouthbrooder orthology confidence [0 low, 1 high]'\n",
       "728. 'Bushbaby gene stable ID'\n",
       "729. 'Bushbaby gene name'\n",
       "730. 'Bushbaby protein or transcript stable ID'\n",
       "731. 'Bushbaby chromosome/scaffold name'\n",
       "732. 'Bushbaby chromosome/scaffold start (bp)'\n",
       "733. 'Bushbaby chromosome/scaffold end (bp)'\n",
       "734. 'Query protein or transcript ID'\n",
       "735. 'Last common ancestor with Bushbaby'\n",
       "736. 'Bushbaby homology type'\n",
       "737. '%id. target Bushbaby gene identical to query gene'\n",
       "738. '%id. query gene identical to target Bushbaby gene'\n",
       "739. 'Bushbaby Gene-order conservation score'\n",
       "740. 'Bushbaby Whole-genome alignment coverage'\n",
       "741. 'dN with Bushbaby'\n",
       "742. 'dS with Bushbaby'\n",
       "743. 'Bushbaby orthology confidence [0 low, 1 high]'\n",
       "744. 'C.intestinalis gene stable ID'\n",
       "745. 'C.intestinalis gene name'\n",
       "746. 'C.intestinalis protein or transcript stable ID'\n",
       "747. 'C.intestinalis chromosome/scaffold name'\n",
       "748. 'C.intestinalis chromosome/scaffold start (bp)'\n",
       "749. 'C.intestinalis chromosome/scaffold end (bp)'\n",
       "750. 'Query protein or transcript ID'\n",
       "751. 'Last common ancestor with C.intestinalis'\n",
       "752. 'C.intestinalis homology type'\n",
       "753. '%id. target C.intestinalis gene identical to query gene'\n",
       "754. '%id. query gene identical to target C.intestinalis gene'\n",
       "755. 'C.intestinalis Gene-order conservation score'\n",
       "756. 'C.intestinalis Whole-genome alignment coverage'\n",
       "757. 'dN with C.intestinalis'\n",
       "758. 'dS with C.intestinalis'\n",
       "759. 'C.intestinalis orthology confidence [0 low, 1 high]'\n",
       "760. 'C.savignyi gene stable ID'\n",
       "761. 'C.savignyi gene name'\n",
       "762. 'C.savignyi protein or transcript stable ID'\n",
       "763. 'C.savignyi chromosome/scaffold name'\n",
       "764. 'C.savignyi chromosome/scaffold start (bp)'\n",
       "765. 'C.savignyi chromosome/scaffold end (bp)'\n",
       "766. 'Query protein or transcript ID'\n",
       "767. 'Last common ancestor with C.savignyi'\n",
       "768. 'C.savignyi homology type'\n",
       "769. '%id. target C.savignyi gene identical to query gene'\n",
       "770. '%id. query gene identical to target C.savignyi gene'\n",
       "771. 'C.savignyi Gene-order conservation score'\n",
       "772. 'C.savignyi Whole-genome alignment coverage'\n",
       "773. 'dN with C.savignyi'\n",
       "774. 'dS with C.savignyi'\n",
       "775. 'C.savignyi orthology confidence [0 low, 1 high]'\n",
       "776. 'Caenorhabditis elegans gene stable ID'\n",
       "777. 'Caenorhabditis elegans gene name'\n",
       "778. 'Caenorhabditis elegans protein or transcript stable ID'\n",
       "779. 'Caenorhabditis elegans chromosome/scaffold name'\n",
       "780. 'Caenorhabditis elegans chromosome/scaffold start (bp)'\n",
       "781. 'Caenorhabditis elegans chromosome/scaffold end (bp)'\n",
       "782. 'Query protein or transcript ID'\n",
       "783. 'Last common ancestor with Caenorhabditis elegans'\n",
       "784. 'Caenorhabditis elegans homology type'\n",
       "785. '%id. target Caenorhabditis elegans gene identical to query gene'\n",
       "786. '%id. query gene identical to target Caenorhabditis elegans gene'\n",
       "787. 'Caenorhabditis elegans Gene-order conservation score'\n",
       "788. 'Caenorhabditis elegans Whole-genome alignment coverage'\n",
       "789. 'dN with Caenorhabditis elegans'\n",
       "790. 'dS with Caenorhabditis elegans'\n",
       "791. 'Caenorhabditis elegans orthology confidence [0 low, 1 high]'\n",
       "792. 'Capuchin gene stable ID'\n",
       "793. 'Capuchin gene name'\n",
       "794. 'Capuchin protein or transcript stable ID'\n",
       "795. 'Capuchin chromosome/scaffold name'\n",
       "796. 'Capuchin chromosome/scaffold start (bp)'\n",
       "797. 'Capuchin chromosome/scaffold end (bp)'\n",
       "798. 'Query protein or transcript ID'\n",
       "799. 'Last common ancestor with Capuchin'\n",
       "800. 'Capuchin homology type'\n",
       "801. '%id. target Capuchin gene identical to query gene'\n",
       "802. '%id. query gene identical to target Capuchin gene'\n",
       "803. 'Capuchin Gene-order conservation score'\n",
       "804. 'Capuchin Whole-genome alignment coverage'\n",
       "805. 'dN with Capuchin'\n",
       "806. 'dS with Capuchin'\n",
       "807. 'Capuchin orthology confidence [0 low, 1 high]'\n",
       "808. 'Cat gene stable ID'\n",
       "809. 'Cat gene name'\n",
       "810. 'Cat protein or transcript stable ID'\n",
       "811. 'Cat chromosome/scaffold name'\n",
       "812. 'Cat chromosome/scaffold start (bp)'\n",
       "813. 'Cat chromosome/scaffold end (bp)'\n",
       "814. 'Query protein or transcript ID'\n",
       "815. 'Last common ancestor with Cat'\n",
       "816. 'Cat homology type'\n",
       "817. '%id. target Cat gene identical to query gene'\n",
       "818. '%id. query gene identical to target Cat gene'\n",
       "819. 'Cat Gene-order conservation score'\n",
       "820. 'Cat Whole-genome alignment coverage'\n",
       "821. 'dN with Cat'\n",
       "822. 'dS with Cat'\n",
       "823. 'Cat orthology confidence [0 low, 1 high]'\n",
       "824. 'Central bearded dragon gene stable ID'\n",
       "825. 'Central bearded dragon gene name'\n",
       "826. 'Central bearded dragon protein or transcript stable ID'\n",
       "827. 'Central bearded dragon chromosome/scaffold name'\n",
       "828. 'Central bearded dragon chromosome/scaffold start (bp)'\n",
       "829. 'Central bearded dragon chromosome/scaffold end (bp)'\n",
       "830. 'Query protein or transcript ID'\n",
       "831. 'Last common ancestor with Central bearded dragon'\n",
       "832. 'Central bearded dragon homology type'\n",
       "833. '%id. target Central bearded dragon gene identical to query gene'\n",
       "834. '%id. query gene identical to target Central bearded dragon gene'\n",
       "835. 'Central bearded dragon Gene-order conservation score'\n",
       "836. 'Central bearded dragon Whole-genome alignment coverage'\n",
       "837. 'dN with Central bearded dragon'\n",
       "838. 'dS with Central bearded dragon'\n",
       "839. 'Central bearded dragon orthology confidence [0 low, 1 high]'\n",
       "840. 'Channel catfish gene stable ID'\n",
       "841. 'Channel catfish gene name'\n",
       "842. 'Channel catfish protein or transcript stable ID'\n",
       "843. 'Channel catfish chromosome/scaffold name'\n",
       "844. 'Channel catfish chromosome/scaffold start (bp)'\n",
       "845. 'Channel catfish chromosome/scaffold end (bp)'\n",
       "846. 'Query protein or transcript ID'\n",
       "847. 'Last common ancestor with Channel catfish'\n",
       "848. 'Channel catfish homology type'\n",
       "849. '%id. target Channel catfish gene identical to query gene'\n",
       "850. '%id. query gene identical to target Channel catfish gene'\n",
       "851. 'Channel catfish Gene-order conservation score'\n",
       "852. 'Channel catfish Whole-genome alignment coverage'\n",
       "853. 'dN with Channel catfish'\n",
       "854. 'dS with Channel catfish'\n",
       "855. 'Channel catfish orthology confidence [0 low, 1 high]'\n",
       "856. 'Chicken gene stable ID'\n",
       "857. 'Chicken gene name'\n",
       "858. 'Chicken protein or transcript stable ID'\n",
       "859. 'Chicken chromosome/scaffold name'\n",
       "860. 'Chicken chromosome/scaffold start (bp)'\n",
       "861. 'Chicken chromosome/scaffold end (bp)'\n",
       "862. 'Query protein or transcript ID'\n",
       "863. 'Last common ancestor with Chicken'\n",
       "864. 'Chicken homology type'\n",
       "865. '%id. target Chicken gene identical to query gene'\n",
       "866. '%id. query gene identical to target Chicken gene'\n",
       "867. 'Chicken Gene-order conservation score'\n",
       "868. 'Chicken Whole-genome alignment coverage'\n",
       "869. 'dN with Chicken'\n",
       "870. 'dS with Chicken'\n",
       "871. 'Chicken orthology confidence [0 low, 1 high]'\n",
       "872. 'Chilean tinamou gene stable ID'\n",
       "873. 'Chilean tinamou gene name'\n",
       "874. 'Chilean tinamou protein or transcript stable ID'\n",
       "875. 'Chilean tinamou chromosome/scaffold name'\n",
       "876. 'Chilean tinamou chromosome/scaffold start (bp)'\n",
       "877. 'Chilean tinamou chromosome/scaffold end (bp)'\n",
       "878. 'Query protein or transcript ID'\n",
       "879. 'Last common ancestor with Chilean tinamou'\n",
       "880. 'Chilean tinamou homology type'\n",
       "881. '%id. target Chilean tinamou gene identical to query gene'\n",
       "882. '%id. query gene identical to target Chilean tinamou gene'\n",
       "883. 'Chilean tinamou Gene-order conservation score'\n",
       "884. 'Chilean tinamou Whole-genome alignment coverage'\n",
       "885. 'dN with Chilean tinamou'\n",
       "886. 'dS with Chilean tinamou'\n",
       "887. 'Chilean tinamou orthology confidence [0 low, 1 high]'\n",
       "888. 'Chimpanzee gene stable ID'\n",
       "889. 'Chimpanzee gene name'\n",
       "890. 'Chimpanzee protein or transcript stable ID'\n",
       "891. 'Chimpanzee chromosome/scaffold name'\n",
       "892. 'Chimpanzee chromosome/scaffold start (bp)'\n",
       "893. 'Chimpanzee chromosome/scaffold end (bp)'\n",
       "894. 'Query protein or transcript ID'\n",
       "895. 'Last common ancestor with Chimpanzee'\n",
       "896. 'Chimpanzee homology type'\n",
       "897. '%id. target Chimpanzee gene identical to query gene'\n",
       "898. '%id. query gene identical to target Chimpanzee gene'\n",
       "899. 'Chimpanzee Gene-order conservation score'\n",
       "900. 'Chimpanzee Whole-genome alignment coverage'\n",
       "901. 'dN with Chimpanzee'\n",
       "902. 'dS with Chimpanzee'\n",
       "903. 'Chimpanzee orthology confidence [0 low, 1 high]'\n",
       "904. 'Chinese hamster CHOK1GS gene stable ID'\n",
       "905. 'Chinese hamster CHOK1GS gene name'\n",
       "906. 'Chinese hamster CHOK1GS protein or transcript stable ID'\n",
       "907. 'Chinese hamster CHOK1GS chromosome/scaffold name'\n",
       "908. 'Chinese hamster CHOK1GS chromosome/scaffold start (bp)'\n",
       "909. 'Chinese hamster CHOK1GS chromosome/scaffold end (bp)'\n",
       "910. 'Query protein or transcript ID'\n",
       "911. 'Last common ancestor with Chinese hamster CHOK1GS'\n",
       "912. 'Chinese hamster CHOK1GS homology type'\n",
       "913. '%id. target Chinese hamster CHOK1GS gene identical to query gene'\n",
       "914. '%id. query gene identical to target Chinese hamster CHOK1GS gene'\n",
       "915. 'Chinese hamster CHOK1GS Gene-order conservation score'\n",
       "916. 'Chinese hamster CHOK1GS Whole-genome alignment coverage'\n",
       "917. 'dN with Chinese hamster CHOK1GS'\n",
       "918. 'dS with Chinese hamster CHOK1GS'\n",
       "919. 'Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]'\n",
       "920. 'Chinese hamster CriGri gene stable ID'\n",
       "921. 'Chinese hamster CriGri gene name'\n",
       "922. 'Chinese hamster CriGri protein or transcript stable ID'\n",
       "923. 'Chinese hamster CriGri chromosome/scaffold name'\n",
       "924. 'Chinese hamster CriGri chromosome/scaffold start (bp)'\n",
       "925. 'Chinese hamster CriGri chromosome/scaffold end (bp)'\n",
       "926. 'Query protein or transcript ID'\n",
       "927. 'Last common ancestor with Chinese hamster CriGri'\n",
       "928. 'Chinese hamster CriGri homology type'\n",
       "929. '%id. target Chinese hamster CriGri gene identical to query gene'\n",
       "930. '%id. query gene identical to target Chinese hamster CriGri gene'\n",
       "931. 'Chinese hamster CriGri Gene-order conservation score'\n",
       "932. 'Chinese hamster CriGri Whole-genome alignment coverage'\n",
       "933. 'dN with Chinese hamster CriGri'\n",
       "934. 'dS with Chinese hamster CriGri'\n",
       "935. 'Chinese hamster CriGri orthology confidence [0 low, 1 high]'\n",
       "936. 'Chinese hamster PICR gene stable ID'\n",
       "937. 'Chinese hamster PICR gene name'\n",
       "938. 'Chinese hamster PICR protein or transcript stable ID'\n",
       "939. 'Chinese hamster PICR chromosome/scaffold name'\n",
       "940. 'Chinese hamster PICR chromosome/scaffold start (bp)'\n",
       "941. 'Chinese hamster PICR chromosome/scaffold end (bp)'\n",
       "942. 'Query protein or transcript ID'\n",
       "943. 'Last common ancestor with Chinese hamster PICR'\n",
       "944. 'Chinese hamster PICR homology type'\n",
       "945. '%id. target Chinese hamster PICR gene identical to query gene'\n",
       "946. '%id. query gene identical to target Chinese hamster PICR gene'\n",
       "947. 'Chinese hamster PICR Gene-order conservation score'\n",
       "948. 'Chinese hamster PICR Whole-genome alignment coverage'\n",
       "949. 'dN with Chinese hamster PICR'\n",
       "950. 'dS with Chinese hamster PICR'\n",
       "951. 'Chinese hamster PICR orthology confidence [0 low, 1 high]'\n",
       "952. 'Chinese softshell turtle gene stable ID'\n",
       "953. 'Chinese softshell turtle gene name'\n",
       "954. 'Chinese softshell turtle protein or transcript stable ID'\n",
       "955. 'Chinese softshell turtle chromosome/scaffold name'\n",
       "956. 'Chinese softshell turtle chromosome/scaffold start (bp)'\n",
       "957. 'Chinese softshell turtle chromosome/scaffold end (bp)'\n",
       "958. 'Query protein or transcript ID'\n",
       "959. 'Last common ancestor with Chinese softshell turtle'\n",
       "960. 'Chinese softshell turtle homology type'\n",
       "961. '%id. target Chinese softshell turtle gene identical to query gene'\n",
       "962. '%id. query gene identical to target Chinese softshell turtle gene'\n",
       "963. 'Chinese softshell turtle Gene-order conservation score'\n",
       "964. 'Chinese softshell turtle Whole-genome alignment coverage'\n",
       "965. 'dN with Chinese softshell turtle'\n",
       "966. 'dS with Chinese softshell turtle'\n",
       "967. 'Chinese softshell turtle orthology confidence [0 low, 1 high]'\n",
       "968. 'Climbing perch gene stable ID'\n",
       "969. 'Climbing perch gene name'\n",
       "970. 'Climbing perch protein or transcript stable ID'\n",
       "971. 'Climbing perch chromosome/scaffold name'\n",
       "972. 'Climbing perch chromosome/scaffold start (bp)'\n",
       "973. 'Climbing perch chromosome/scaffold end (bp)'\n",
       "974. 'Query protein or transcript ID'\n",
       "975. 'Last common ancestor with Climbing perch'\n",
       "976. 'Climbing perch homology type'\n",
       "977. '%id. target Climbing perch gene identical to query gene'\n",
       "978. '%id. query gene identical to target Climbing perch gene'\n",
       "979. 'Climbing perch Gene-order conservation score'\n",
       "980. 'Climbing perch Whole-genome alignment coverage'\n",
       "981. 'dN with Climbing perch'\n",
       "982. 'dS with Climbing perch'\n",
       "983. 'Climbing perch orthology confidence [0 low, 1 high]'\n",
       "984. 'Clown anemonefish gene stable ID'\n",
       "985. 'Clown anemonefish gene name'\n",
       "986. 'Clown anemonefish protein or transcript stable ID'\n",
       "987. 'Clown anemonefish chromosome/scaffold name'\n",
       "988. 'Clown anemonefish chromosome/scaffold start (bp)'\n",
       "989. 'Clown anemonefish chromosome/scaffold end (bp)'\n",
       "990. 'Query protein or transcript ID'\n",
       "991. 'Last common ancestor with Clown anemonefish'\n",
       "992. 'Clown anemonefish homology type'\n",
       "993. '%id. target Clown anemonefish gene identical to query gene'\n",
       "994. '%id. query gene identical to target Clown anemonefish gene'\n",
       "995. 'Clown anemonefish Gene-order conservation score'\n",
       "996. 'Clown anemonefish Whole-genome alignment coverage'\n",
       "997. 'dN with Clown anemonefish'\n",
       "998. 'dS with Clown anemonefish'\n",
       "999. 'Clown anemonefish orthology confidence [0 low, 1 high]'\n",
       "1000. 'Cod gene stable ID'\n",
       "1001. 'Cod gene name'\n",
       "1002. 'Cod protein or transcript stable ID'\n",
       "1003. 'Cod chromosome/scaffold name'\n",
       "1004. 'Cod chromosome/scaffold start (bp)'\n",
       "1005. 'Cod chromosome/scaffold end (bp)'\n",
       "1006. 'Query protein or transcript ID'\n",
       "1007. 'Last common ancestor with Cod'\n",
       "1008. 'Cod homology type'\n",
       "1009. '%id. target Cod gene identical to query gene'\n",
       "1010. '%id. query gene identical to target Cod gene'\n",
       "1011. 'Cod Gene-order conservation score'\n",
       "1012. 'Cod Whole-genome alignment coverage'\n",
       "1013. 'dN with Cod'\n",
       "1014. 'dS with Cod'\n",
       "1015. 'Cod orthology confidence [0 low, 1 high]'\n",
       "1016. 'Coelacanth gene stable ID'\n",
       "1017. 'Coelacanth gene name'\n",
       "1018. 'Coelacanth protein or transcript stable ID'\n",
       "1019. 'Coelacanth chromosome/scaffold name'\n",
       "1020. 'Coelacanth chromosome/scaffold start (bp)'\n",
       "1021. 'Coelacanth chromosome/scaffold end (bp)'\n",
       "1022. 'Query protein or transcript ID'\n",
       "1023. 'Last common ancestor with Coelacanth'\n",
       "1024. 'Coelacanth homology type'\n",
       "1025. '%id. target Coelacanth gene identical to query gene'\n",
       "1026. '%id. query gene identical to target Coelacanth gene'\n",
       "1027. 'Coelacanth Gene-order conservation score'\n",
       "1028. 'Coelacanth Whole-genome alignment coverage'\n",
       "1029. 'dN with Coelacanth'\n",
       "1030. 'dS with Coelacanth'\n",
       "1031. 'Coelacanth orthology confidence [0 low, 1 high]'\n",
       "1032. 'Common canary gene stable ID'\n",
       "1033. 'Common canary gene name'\n",
       "1034. 'Common canary protein or transcript stable ID'\n",
       "1035. 'Common canary chromosome/scaffold name'\n",
       "1036. 'Common canary chromosome/scaffold start (bp)'\n",
       "1037. 'Common canary chromosome/scaffold end (bp)'\n",
       "1038. 'Query protein or transcript ID'\n",
       "1039. 'Last common ancestor with Common canary'\n",
       "1040. 'Common canary homology type'\n",
       "1041. '%id. target Common canary gene identical to query gene'\n",
       "1042. '%id. query gene identical to target Common canary gene'\n",
       "1043. 'Common canary Gene-order conservation score'\n",
       "1044. 'Common canary Whole-genome alignment coverage'\n",
       "1045. 'dN with Common canary'\n",
       "1046. 'dS with Common canary'\n",
       "1047. 'Common canary orthology confidence [0 low, 1 high]'\n",
       "1048. 'Common wombat gene stable ID'\n",
       "1049. 'Common wombat gene name'\n",
       "1050. 'Common wombat protein or transcript stable ID'\n",
       "1051. 'Common wombat chromosome/scaffold name'\n",
       "1052. 'Common wombat chromosome/scaffold start (bp)'\n",
       "1053. 'Common wombat chromosome/scaffold end (bp)'\n",
       "1054. 'Query protein or transcript ID'\n",
       "1055. 'Last common ancestor with Common wombat'\n",
       "1056. 'Common wombat homology type'\n",
       "1057. '%id. target Common wombat gene identical to query gene'\n",
       "1058. '%id. query gene identical to target Common wombat gene'\n",
       "1059. 'Common wombat Gene-order conservation score'\n",
       "1060. 'Common wombat Whole-genome alignment coverage'\n",
       "1061. 'dN with Common wombat'\n",
       "1062. 'dS with Common wombat'\n",
       "1063. 'Common wombat orthology confidence [0 low, 1 high]'\n",
       "1064. 'Coquerel\\'s sifaka gene stable ID'\n",
       "1065. 'Coquerel\\'s sifaka gene name'\n",
       "1066. 'Coquerel\\'s sifaka protein or transcript stable ID'\n",
       "1067. 'Coquerel\\'s sifaka chromosome/scaffold name'\n",
       "1068. 'Coquerel\\'s sifaka chromosome/scaffold start (bp)'\n",
       "1069. 'Coquerel\\'s sifaka chromosome/scaffold end (bp)'\n",
       "1070. 'Query protein or transcript ID'\n",
       "1071. 'Last common ancestor with Coquerel\\'s sifaka'\n",
       "1072. 'Coquerel\\'s sifaka homology type'\n",
       "1073. '%id. target Coquerel\\'s sifaka gene identical to query gene'\n",
       "1074. '%id. query gene identical to target Coquerel\\'s sifaka gene'\n",
       "1075. 'Coquerel\\'s sifaka Gene-order conservation score'\n",
       "1076. 'Coquerel\\'s sifaka Whole-genome alignment coverage'\n",
       "1077. 'dN with Coquerel\\'s sifaka'\n",
       "1078. 'dS with Coquerel\\'s sifaka'\n",
       "1079. 'Coquerel\\'s sifaka orthology confidence [0 low, 1 high]'\n",
       "1080. 'Cow gene stable ID'\n",
       "1081. 'Cow gene name'\n",
       "1082. 'Cow protein or transcript stable ID'\n",
       "1083. 'Cow chromosome/scaffold name'\n",
       "1084. 'Cow chromosome/scaffold start (bp)'\n",
       "1085. 'Cow chromosome/scaffold end (bp)'\n",
       "1086. 'Query protein or transcript ID'\n",
       "1087. 'Last common ancestor with Cow'\n",
       "1088. 'Cow homology type'\n",
       "1089. '%id. target Cow gene identical to query gene'\n",
       "1090. '%id. query gene identical to target Cow gene'\n",
       "1091. 'Cow Gene-order conservation score'\n",
       "1092. 'Cow Whole-genome alignment coverage'\n",
       "1093. 'dN with Cow'\n",
       "1094. 'dS with Cow'\n",
       "1095. 'Cow orthology confidence [0 low, 1 high]'\n",
       "1096. 'Crab-eating macaque gene stable ID'\n",
       "1097. 'Crab-eating macaque gene name'\n",
       "1098. 'Crab-eating macaque protein or transcript stable ID'\n",
       "1099. 'Crab-eating macaque chromosome/scaffold name'\n",
       "1100. 'Crab-eating macaque chromosome/scaffold start (bp)'\n",
       "1101. 'Crab-eating macaque chromosome/scaffold end (bp)'\n",
       "1102. 'Query protein or transcript ID'\n",
       "1103. 'Last common ancestor with Crab-eating macaque'\n",
       "1104. 'Crab-eating macaque homology type'\n",
       "1105. '%id. target Crab-eating macaque gene identical to query gene'\n",
       "1106. '%id. query gene identical to target Crab-eating macaque gene'\n",
       "1107. 'Crab-eating macaque Gene-order conservation score'\n",
       "1108. 'Crab-eating macaque Whole-genome alignment coverage'\n",
       "1109. 'dN with Crab-eating macaque'\n",
       "1110. 'dS with Crab-eating macaque'\n",
       "1111. 'Crab-eating macaque orthology confidence [0 low, 1 high]'\n",
       "1112. 'Damara mole rat gene stable ID'\n",
       "1113. 'Damara mole rat gene name'\n",
       "1114. 'Damara mole rat protein or transcript stable ID'\n",
       "1115. 'Damara mole rat chromosome/scaffold name'\n",
       "1116. 'Damara mole rat chromosome/scaffold start (bp)'\n",
       "1117. 'Damara mole rat chromosome/scaffold end (bp)'\n",
       "1118. 'Query protein or transcript ID'\n",
       "1119. 'Last common ancestor with Damara mole rat'\n",
       "1120. 'Damara mole rat homology type'\n",
       "1121. '%id. target Damara mole rat gene identical to query gene'\n",
       "1122. '%id. query gene identical to target Damara mole rat gene'\n",
       "1123. 'Damara mole rat Gene-order conservation score'\n",
       "1124. 'Damara mole rat Whole-genome alignment coverage'\n",
       "1125. 'dN with Damara mole rat'\n",
       "1126. 'dS with Damara mole rat'\n",
       "1127. 'Damara mole rat orthology confidence [0 low, 1 high]'\n",
       "1128. 'Dark-eyed junco gene stable ID'\n",
       "1129. 'Dark-eyed junco gene name'\n",
       "1130. 'Dark-eyed junco protein or transcript stable ID'\n",
       "1131. 'Dark-eyed junco chromosome/scaffold name'\n",
       "1132. 'Dark-eyed junco chromosome/scaffold start (bp)'\n",
       "1133. 'Dark-eyed junco chromosome/scaffold end (bp)'\n",
       "1134. 'Query protein or transcript ID'\n",
       "1135. 'Last common ancestor with Dark-eyed junco'\n",
       "1136. 'Dark-eyed junco homology type'\n",
       "1137. '%id. target Dark-eyed junco gene identical to query gene'\n",
       "1138. '%id. query gene identical to target Dark-eyed junco gene'\n",
       "1139. 'Dark-eyed junco Gene-order conservation score'\n",
       "1140. 'Dark-eyed junco Whole-genome alignment coverage'\n",
       "1141. 'dN with Dark-eyed junco'\n",
       "1142. 'dS with Dark-eyed junco'\n",
       "1143. 'Dark-eyed junco orthology confidence [0 low, 1 high]'\n",
       "1144. 'Daurian ground squirrel gene stable ID'\n",
       "1145. 'Daurian ground squirrel gene name'\n",
       "1146. 'Daurian ground squirrel protein or transcript stable ID'\n",
       "1147. 'Daurian ground squirrel chromosome/scaffold name'\n",
       "1148. 'Daurian ground squirrel chromosome/scaffold start (bp)'\n",
       "1149. 'Daurian ground squirrel chromosome/scaffold end (bp)'\n",
       "1150. 'Query protein or transcript ID'\n",
       "1151. 'Last common ancestor with Daurian ground squirrel'\n",
       "1152. 'Daurian ground squirrel homology type'\n",
       "1153. '%id. target Daurian ground squirrel gene identical to query gene'\n",
       "1154. '%id. query gene identical to target Daurian ground squirrel gene'\n",
       "1155. 'Daurian ground squirrel Gene-order conservation score'\n",
       "1156. 'Daurian ground squirrel Whole-genome alignment coverage'\n",
       "1157. 'dN with Daurian ground squirrel'\n",
       "1158. 'dS with Daurian ground squirrel'\n",
       "1159. 'Daurian ground squirrel orthology confidence [0 low, 1 high]'\n",
       "1160. 'Degu gene stable ID'\n",
       "1161. 'Degu gene name'\n",
       "1162. 'Degu protein or transcript stable ID'\n",
       "1163. 'Degu chromosome/scaffold name'\n",
       "1164. 'Degu chromosome/scaffold start (bp)'\n",
       "1165. 'Degu chromosome/scaffold end (bp)'\n",
       "1166. 'Query protein or transcript ID'\n",
       "1167. 'Last common ancestor with Degu'\n",
       "1168. 'Degu homology type'\n",
       "1169. '%id. target Degu gene identical to query gene'\n",
       "1170. '%id. query gene identical to target Degu gene'\n",
       "1171. 'Degu Gene-order conservation score'\n",
       "1172. 'Degu Whole-genome alignment coverage'\n",
       "1173. 'dN with Degu'\n",
       "1174. 'dS with Degu'\n",
       "1175. 'Degu orthology confidence [0 low, 1 high]'\n",
       "1176. 'Dingo gene stable ID'\n",
       "1177. 'Dingo gene name'\n",
       "1178. 'Dingo protein or transcript stable ID'\n",
       "1179. 'Dingo chromosome/scaffold name'\n",
       "1180. 'Dingo chromosome/scaffold start (bp)'\n",
       "1181. 'Dingo chromosome/scaffold end (bp)'\n",
       "1182. 'Query protein or transcript ID'\n",
       "1183. 'Last common ancestor with Dingo'\n",
       "1184. 'Dingo homology type'\n",
       "1185. '%id. target Dingo gene identical to query gene'\n",
       "1186. '%id. query gene identical to target Dingo gene'\n",
       "1187. 'Dingo Gene-order conservation score'\n",
       "1188. 'Dingo Whole-genome alignment coverage'\n",
       "1189. 'dN with Dingo'\n",
       "1190. 'dS with Dingo'\n",
       "1191. 'Dingo orthology confidence [0 low, 1 high]'\n",
       "1192. 'Dog gene stable ID'\n",
       "1193. 'Dog gene name'\n",
       "1194. 'Dog protein or transcript stable ID'\n",
       "1195. 'Dog chromosome/scaffold name'\n",
       "1196. 'Dog chromosome/scaffold start (bp)'\n",
       "1197. 'Dog chromosome/scaffold end (bp)'\n",
       "1198. 'Query protein or transcript ID'\n",
       "1199. 'Last common ancestor with Dog'\n",
       "1200. 'Dog homology type'\n",
       "1201. '%id. target Dog gene identical to query gene'\n",
       "1202. '%id. query gene identical to target Dog gene'\n",
       "1203. 'Dog Gene-order conservation score'\n",
       "1204. 'Dog Whole-genome alignment coverage'\n",
       "1205. 'dN with Dog'\n",
       "1206. 'dS with Dog'\n",
       "1207. 'Dog orthology confidence [0 low, 1 high]'\n",
       "1208. 'Dolphin gene stable ID'\n",
       "1209. 'Dolphin gene name'\n",
       "1210. 'Dolphin protein or transcript stable ID'\n",
       "1211. 'Dolphin chromosome/scaffold name'\n",
       "1212. 'Dolphin chromosome/scaffold start (bp)'\n",
       "1213. 'Dolphin chromosome/scaffold end (bp)'\n",
       "1214. 'Query protein or transcript ID'\n",
       "1215. 'Last common ancestor with Dolphin'\n",
       "1216. 'Dolphin homology type'\n",
       "1217. '%id. target Dolphin gene identical to query gene'\n",
       "1218. '%id. query gene identical to target Dolphin gene'\n",
       "1219. 'Dolphin Gene-order conservation score'\n",
       "1220. 'Dolphin Whole-genome alignment coverage'\n",
       "1221. 'dN with Dolphin'\n",
       "1222. 'dS with Dolphin'\n",
       "1223. 'Dolphin orthology confidence [0 low, 1 high]'\n",
       "1224. 'Donkey gene stable ID'\n",
       "1225. 'Donkey gene name'\n",
       "1226. 'Donkey protein or transcript stable ID'\n",
       "1227. 'Donkey chromosome/scaffold name'\n",
       "1228. 'Donkey chromosome/scaffold start (bp)'\n",
       "1229. 'Donkey chromosome/scaffold end (bp)'\n",
       "1230. 'Query protein or transcript ID'\n",
       "1231. 'Last common ancestor with Donkey'\n",
       "1232. 'Donkey homology type'\n",
       "1233. '%id. target Donkey gene identical to query gene'\n",
       "1234. '%id. query gene identical to target Donkey gene'\n",
       "1235. 'Donkey Gene-order conservation score'\n",
       "1236. 'Donkey Whole-genome alignment coverage'\n",
       "1237. 'dN with Donkey'\n",
       "1238. 'dS with Donkey'\n",
       "1239. 'Donkey orthology confidence [0 low, 1 high]'\n",
       "1240. 'Drill gene stable ID'\n",
       "1241. 'Drill gene name'\n",
       "1242. 'Drill protein or transcript stable ID'\n",
       "1243. 'Drill chromosome/scaffold name'\n",
       "1244. 'Drill chromosome/scaffold start (bp)'\n",
       "1245. 'Drill chromosome/scaffold end (bp)'\n",
       "1246. 'Query protein or transcript ID'\n",
       "1247. 'Last common ancestor with Drill'\n",
       "1248. 'Drill homology type'\n",
       "1249. '%id. target Drill gene identical to query gene'\n",
       "1250. '%id. query gene identical to target Drill gene'\n",
       "1251. 'Drill Gene-order conservation score'\n",
       "1252. 'Drill Whole-genome alignment coverage'\n",
       "1253. 'dN with Drill'\n",
       "1254. 'dS with Drill'\n",
       "1255. 'Drill orthology confidence [0 low, 1 high]'\n",
       "1256. 'Drosophila melanogaster gene stable ID'\n",
       "1257. 'Drosophila melanogaster gene name'\n",
       "1258. 'Drosophila melanogaster protein or transcript stable ID'\n",
       "1259. 'Drosophila melanogaster chromosome/scaffold name'\n",
       "1260. 'Drosophila melanogaster chromosome/scaffold start (bp)'\n",
       "1261. 'Drosophila melanogaster chromosome/scaffold end (bp)'\n",
       "1262. 'Query protein or transcript ID'\n",
       "1263. 'Last common ancestor with Drosophila melanogaster'\n",
       "1264. 'Drosophila melanogaster homology type'\n",
       "1265. '%id. target Drosophila melanogaster gene identical to query gene'\n",
       "1266. '%id. query gene identical to target Drosophila melanogaster gene'\n",
       "1267. 'Drosophila melanogaster Gene-order conservation score'\n",
       "1268. 'Drosophila melanogaster Whole-genome alignment coverage'\n",
       "1269. 'dN with Drosophila melanogaster'\n",
       "1270. 'dS with Drosophila melanogaster'\n",
       "1271. 'Drosophila melanogaster orthology confidence [0 low, 1 high]'\n",
       "1272. 'Duck gene stable ID'\n",
       "1273. 'Duck gene name'\n",
       "1274. 'Duck protein or transcript stable ID'\n",
       "1275. 'Duck chromosome/scaffold name'\n",
       "1276. 'Duck chromosome/scaffold start (bp)'\n",
       "1277. 'Duck chromosome/scaffold end (bp)'\n",
       "1278. 'Query protein or transcript ID'\n",
       "1279. 'Last common ancestor with Duck'\n",
       "1280. 'Duck homology type'\n",
       "1281. '%id. target Duck gene identical to query gene'\n",
       "1282. '%id. query gene identical to target Duck gene'\n",
       "1283. 'Duck Gene-order conservation score'\n",
       "1284. 'Duck Whole-genome alignment coverage'\n",
       "1285. 'dN with Duck'\n",
       "1286. 'dS with Duck'\n",
       "1287. 'Duck orthology confidence [0 low, 1 high]'\n",
       "1288. 'Eastern happy gene stable ID'\n",
       "1289. 'Eastern happy gene name'\n",
       "1290. 'Eastern happy protein or transcript stable ID'\n",
       "1291. 'Eastern happy chromosome/scaffold name'\n",
       "1292. 'Eastern happy chromosome/scaffold start (bp)'\n",
       "1293. 'Eastern happy chromosome/scaffold end (bp)'\n",
       "1294. 'Query protein or transcript ID'\n",
       "1295. 'Last common ancestor with Eastern happy'\n",
       "1296. 'Eastern happy homology type'\n",
       "1297. '%id. target Eastern happy gene identical to query gene'\n",
       "1298. '%id. query gene identical to target Eastern happy gene'\n",
       "1299. 'Eastern happy Gene-order conservation score'\n",
       "1300. 'Eastern happy Whole-genome alignment coverage'\n",
       "1301. 'dN with Eastern happy'\n",
       "1302. 'dS with Eastern happy'\n",
       "1303. 'Eastern happy orthology confidence [0 low, 1 high]'\n",
       "1304. 'Electric eel gene stable ID'\n",
       "1305. 'Electric eel gene name'\n",
       "1306. 'Electric eel protein or transcript stable ID'\n",
       "1307. 'Electric eel chromosome/scaffold name'\n",
       "1308. 'Electric eel chromosome/scaffold start (bp)'\n",
       "1309. 'Electric eel chromosome/scaffold end (bp)'\n",
       "1310. 'Query protein or transcript ID'\n",
       "1311. 'Last common ancestor with Electric eel'\n",
       "1312. 'Electric eel homology type'\n",
       "1313. '%id. target Electric eel gene identical to query gene'\n",
       "1314. '%id. query gene identical to target Electric eel gene'\n",
       "1315. 'Electric eel Gene-order conservation score'\n",
       "1316. 'Electric eel Whole-genome alignment coverage'\n",
       "1317. 'dN with Electric eel'\n",
       "1318. 'dS with Electric eel'\n",
       "1319. 'Electric eel orthology confidence [0 low, 1 high]'\n",
       "1320. 'Elephant gene stable ID'\n",
       "1321. 'Elephant gene name'\n",
       "1322. 'Elephant protein or transcript stable ID'\n",
       "1323. 'Elephant chromosome/scaffold name'\n",
       "1324. 'Elephant chromosome/scaffold start (bp)'\n",
       "1325. 'Elephant chromosome/scaffold end (bp)'\n",
       "1326. 'Query protein or transcript ID'\n",
       "1327. 'Last common ancestor with Elephant'\n",
       "1328. 'Elephant homology type'\n",
       "1329. '%id. target Elephant gene identical to query gene'\n",
       "1330. '%id. query gene identical to target Elephant gene'\n",
       "1331. 'Elephant Gene-order conservation score'\n",
       "1332. 'Elephant Whole-genome alignment coverage'\n",
       "1333. 'dN with Elephant'\n",
       "1334. 'dS with Elephant'\n",
       "1335. 'Elephant orthology confidence [0 low, 1 high]'\n",
       "1336. 'Elephant shark gene stable ID'\n",
       "1337. 'Elephant shark gene name'\n",
       "1338. 'Elephant shark protein or transcript stable ID'\n",
       "1339. 'Elephant shark chromosome/scaffold name'\n",
       "1340. 'Elephant shark chromosome/scaffold start (bp)'\n",
       "1341. 'Elephant shark chromosome/scaffold end (bp)'\n",
       "1342. 'Query protein or transcript ID'\n",
       "1343. 'Last common ancestor with Elephant shark'\n",
       "1344. 'Elephant shark homology type'\n",
       "1345. '%id. target Elephant shark gene identical to query gene'\n",
       "1346. '%id. query gene identical to target Elephant shark gene'\n",
       "1347. 'Elephant shark Gene-order conservation score'\n",
       "1348. 'Elephant shark Whole-genome alignment coverage'\n",
       "1349. 'dN with Elephant shark'\n",
       "1350. 'dS with Elephant shark'\n",
       "1351. 'Elephant shark orthology confidence [0 low, 1 high]'\n",
       "1352. 'Emu gene stable ID'\n",
       "1353. 'Emu gene name'\n",
       "1354. 'Emu protein or transcript stable ID'\n",
       "1355. 'Emu chromosome/scaffold name'\n",
       "1356. 'Emu chromosome/scaffold start (bp)'\n",
       "1357. 'Emu chromosome/scaffold end (bp)'\n",
       "1358. 'Query protein or transcript ID'\n",
       "1359. 'Last common ancestor with Emu'\n",
       "1360. 'Emu homology type'\n",
       "1361. '%id. target Emu gene identical to query gene'\n",
       "1362. '%id. query gene identical to target Emu gene'\n",
       "1363. 'Emu Gene-order conservation score'\n",
       "1364. 'Emu Whole-genome alignment coverage'\n",
       "1365. 'dN with Emu'\n",
       "1366. 'dS with Emu'\n",
       "1367. 'Emu orthology confidence [0 low, 1 high]'\n",
       "1368. 'Ferret gene stable ID'\n",
       "1369. 'Ferret gene name'\n",
       "1370. 'Ferret protein or transcript stable ID'\n",
       "1371. 'Ferret chromosome/scaffold name'\n",
       "1372. 'Ferret chromosome/scaffold start (bp)'\n",
       "1373. 'Ferret chromosome/scaffold end (bp)'\n",
       "1374. 'Query protein or transcript ID'\n",
       "1375. 'Last common ancestor with Ferret'\n",
       "1376. 'Ferret homology type'\n",
       "1377. '%id. target Ferret gene identical to query gene'\n",
       "1378. '%id. query gene identical to target Ferret gene'\n",
       "1379. 'Ferret Gene-order conservation score'\n",
       "1380. 'Ferret Whole-genome alignment coverage'\n",
       "1381. 'dN with Ferret'\n",
       "1382. 'dS with Ferret'\n",
       "1383. 'Ferret orthology confidence [0 low, 1 high]'\n",
       "1384. 'Flycatcher gene stable ID'\n",
       "1385. 'Flycatcher gene name'\n",
       "1386. 'Flycatcher protein or transcript stable ID'\n",
       "1387. 'Flycatcher chromosome/scaffold name'\n",
       "1388. 'Flycatcher chromosome/scaffold start (bp)'\n",
       "1389. 'Flycatcher chromosome/scaffold end (bp)'\n",
       "1390. 'Query protein or transcript ID'\n",
       "1391. 'Last common ancestor with Flycatcher'\n",
       "1392. 'Flycatcher homology type'\n",
       "1393. '%id. target Flycatcher gene identical to query gene'\n",
       "1394. '%id. query gene identical to target Flycatcher gene'\n",
       "1395. 'Flycatcher Gene-order conservation score'\n",
       "1396. 'Flycatcher Whole-genome alignment coverage'\n",
       "1397. 'dN with Flycatcher'\n",
       "1398. 'dS with Flycatcher'\n",
       "1399. 'Flycatcher orthology confidence [0 low, 1 high]'\n",
       "1400. 'Fugu gene stable ID'\n",
       "1401. 'Fugu gene name'\n",
       "1402. 'Fugu protein or transcript stable ID'\n",
       "1403. 'Fugu chromosome/scaffold name'\n",
       "1404. 'Fugu chromosome/scaffold start (bp)'\n",
       "1405. 'Fugu chromosome/scaffold end (bp)'\n",
       "1406. 'Query protein or transcript ID'\n",
       "1407. 'Last common ancestor with Fugu'\n",
       "1408. 'Fugu homology type'\n",
       "1409. '%id. target Fugu gene identical to query gene'\n",
       "1410. '%id. query gene identical to target Fugu gene'\n",
       "1411. 'Fugu Gene-order conservation score'\n",
       "1412. 'Fugu Whole-genome alignment coverage'\n",
       "1413. 'dN with Fugu'\n",
       "1414. 'dS with Fugu'\n",
       "1415. 'Fugu orthology confidence [0 low, 1 high]'\n",
       "1416. 'Gelada gene stable ID'\n",
       "1417. 'Gelada gene name'\n",
       "1418. 'Gelada protein or transcript stable ID'\n",
       "1419. 'Gelada chromosome/scaffold name'\n",
       "1420. 'Gelada chromosome/scaffold start (bp)'\n",
       "1421. 'Gelada chromosome/scaffold end (bp)'\n",
       "1422. 'Query protein or transcript ID'\n",
       "1423. 'Last common ancestor with Gelada'\n",
       "1424. 'Gelada homology type'\n",
       "1425. '%id. target Gelada gene identical to query gene'\n",
       "1426. '%id. query gene identical to target Gelada gene'\n",
       "1427. 'Gelada Gene-order conservation score'\n",
       "1428. 'Gelada Whole-genome alignment coverage'\n",
       "1429. 'dN with Gelada'\n",
       "1430. 'dS with Gelada'\n",
       "1431. 'Gelada orthology confidence [0 low, 1 high]'\n",
       "1432. 'Gibbon gene stable ID'\n",
       "1433. 'Gibbon gene name'\n",
       "1434. 'Gibbon protein or transcript stable ID'\n",
       "1435. 'Gibbon chromosome/scaffold name'\n",
       "1436. 'Gibbon chromosome/scaffold start (bp)'\n",
       "1437. 'Gibbon chromosome/scaffold end (bp)'\n",
       "1438. 'Query protein or transcript ID'\n",
       "1439. 'Last common ancestor with Gibbon'\n",
       "1440. 'Gibbon homology type'\n",
       "1441. '%id. target Gibbon gene identical to query gene'\n",
       "1442. '%id. query gene identical to target Gibbon gene'\n",
       "1443. 'Gibbon Gene-order conservation score'\n",
       "1444. 'Gibbon Whole-genome alignment coverage'\n",
       "1445. 'dN with Gibbon'\n",
       "1446. 'dS with Gibbon'\n",
       "1447. 'Gibbon orthology confidence [0 low, 1 high]'\n",
       "1448. 'Goat gene stable ID'\n",
       "1449. 'Goat gene name'\n",
       "1450. 'Goat protein or transcript stable ID'\n",
       "1451. 'Goat chromosome/scaffold name'\n",
       "1452. 'Goat chromosome/scaffold start (bp)'\n",
       "1453. 'Goat chromosome/scaffold end (bp)'\n",
       "1454. 'Query protein or transcript ID'\n",
       "1455. 'Last common ancestor with Goat'\n",
       "1456. 'Goat homology type'\n",
       "1457. '%id. target Goat gene identical to query gene'\n",
       "1458. '%id. query gene identical to target Goat gene'\n",
       "1459. 'Goat Gene-order conservation score'\n",
       "1460. 'Goat Whole-genome alignment coverage'\n",
       "1461. 'dN with Goat'\n",
       "1462. 'dS with Goat'\n",
       "1463. 'Goat orthology confidence [0 low, 1 high]'\n",
       "1464. 'Golden Hamster gene stable ID'\n",
       "1465. 'Golden Hamster gene name'\n",
       "1466. 'Golden Hamster protein or transcript stable ID'\n",
       "1467. 'Golden Hamster chromosome/scaffold name'\n",
       "1468. 'Golden Hamster chromosome/scaffold start (bp)'\n",
       "1469. 'Golden Hamster chromosome/scaffold end (bp)'\n",
       "1470. 'Query protein or transcript ID'\n",
       "1471. 'Last common ancestor with Golden Hamster'\n",
       "1472. 'Golden Hamster homology type'\n",
       "1473. '%id. target Golden Hamster gene identical to query gene'\n",
       "1474. '%id. query gene identical to target Golden Hamster gene'\n",
       "1475. 'Golden Hamster Gene-order conservation score'\n",
       "1476. 'Golden Hamster Whole-genome alignment coverage'\n",
       "1477. 'dN with Golden Hamster'\n",
       "1478. 'dS with Golden Hamster'\n",
       "1479. 'Golden Hamster orthology confidence [0 low, 1 high]'\n",
       "1480. 'Golden snub-nosed monkey gene stable ID'\n",
       "1481. 'Golden snub-nosed monkey gene name'\n",
       "1482. 'Golden snub-nosed monkey protein or transcript stable ID'\n",
       "1483. 'Golden snub-nosed monkey chromosome/scaffold name'\n",
       "1484. 'Golden snub-nosed monkey chromosome/scaffold start (bp)'\n",
       "1485. 'Golden snub-nosed monkey chromosome/scaffold end (bp)'\n",
       "1486. 'Query protein or transcript ID'\n",
       "1487. 'Last common ancestor with Golden snub-nosed monkey'\n",
       "1488. 'Golden snub-nosed monkey homology type'\n",
       "1489. '%id. target Golden snub-nosed monkey gene identical to query gene'\n",
       "1490. '%id. query gene identical to target Golden snub-nosed monkey gene'\n",
       "1491. 'Golden snub-nosed monkey Gene-order conservation score'\n",
       "1492. 'Golden snub-nosed monkey Whole-genome alignment coverage'\n",
       "1493. 'dN with Golden snub-nosed monkey'\n",
       "1494. 'dS with Golden snub-nosed monkey'\n",
       "1495. 'Golden snub-nosed monkey orthology confidence [0 low, 1 high]'\n",
       "1496. 'Golden-collared manakin gene stable ID'\n",
       "1497. 'Golden-collared manakin gene name'\n",
       "1498. 'Golden-collared manakin protein or transcript stable ID'\n",
       "1499. 'Golden-collared manakin chromosome/scaffold name'\n",
       "1500. 'Golden-collared manakin chromosome/scaffold start (bp)'\n",
       "1501. 'Golden-collared manakin chromosome/scaffold end (bp)'\n",
       "1502. 'Query protein or transcript ID'\n",
       "1503. 'Last common ancestor with Golden-collared manakin'\n",
       "1504. 'Golden-collared manakin homology type'\n",
       "1505. '%id. target Golden-collared manakin gene identical to query gene'\n",
       "1506. '%id. query gene identical to target Golden-collared manakin gene'\n",
       "1507. 'Golden-collared manakin Gene-order conservation score'\n",
       "1508. 'Golden-collared manakin Whole-genome alignment coverage'\n",
       "1509. 'dN with Golden-collared manakin'\n",
       "1510. 'dS with Golden-collared manakin'\n",
       "1511. 'Golden-collared manakin orthology confidence [0 low, 1 high]'\n",
       "1512. 'Gorilla gene stable ID'\n",
       "1513. 'Gorilla gene name'\n",
       "1514. 'Gorilla protein or transcript stable ID'\n",
       "1515. 'Gorilla chromosome/scaffold name'\n",
       "1516. 'Gorilla chromosome/scaffold start (bp)'\n",
       "1517. 'Gorilla chromosome/scaffold end (bp)'\n",
       "1518. 'Query protein or transcript ID'\n",
       "1519. 'Last common ancestor with Gorilla'\n",
       "1520. 'Gorilla homology type'\n",
       "1521. '%id. target Gorilla gene identical to query gene'\n",
       "1522. '%id. query gene identical to target Gorilla gene'\n",
       "1523. 'Gorilla Gene-order conservation score'\n",
       "1524. 'Gorilla Whole-genome alignment coverage'\n",
       "1525. 'dN with Gorilla'\n",
       "1526. 'dS with Gorilla'\n",
       "1527. 'Gorilla orthology confidence [0 low, 1 high]'\n",
       "1528. 'Great Tit gene stable ID'\n",
       "1529. 'Great Tit gene name'\n",
       "1530. 'Great Tit protein or transcript stable ID'\n",
       "1531. 'Great Tit chromosome/scaffold name'\n",
       "1532. 'Great Tit chromosome/scaffold start (bp)'\n",
       "1533. 'Great Tit chromosome/scaffold end (bp)'\n",
       "1534. 'Query protein or transcript ID'\n",
       "1535. 'Last common ancestor with Great Tit'\n",
       "1536. 'Great Tit homology type'\n",
       "1537. '%id. target Great Tit gene identical to query gene'\n",
       "1538. '%id. query gene identical to target Great Tit gene'\n",
       "1539. 'Great Tit Gene-order conservation score'\n",
       "1540. 'Great Tit Whole-genome alignment coverage'\n",
       "1541. 'dN with Great Tit'\n",
       "1542. 'dS with Great Tit'\n",
       "1543. 'Great Tit orthology confidence [0 low, 1 high]'\n",
       "1544. 'Great spotted kiwi gene stable ID'\n",
       "1545. 'Great spotted kiwi gene name'\n",
       "1546. 'Great spotted kiwi protein or transcript stable ID'\n",
       "1547. 'Great spotted kiwi chromosome/scaffold name'\n",
       "1548. 'Great spotted kiwi chromosome/scaffold start (bp)'\n",
       "1549. 'Great spotted kiwi chromosome/scaffold end (bp)'\n",
       "1550. 'Query protein or transcript ID'\n",
       "1551. 'Last common ancestor with Great spotted kiwi'\n",
       "1552. 'Great spotted kiwi homology type'\n",
       "1553. '%id. target Great spotted kiwi gene identical to query gene'\n",
       "1554. '%id. query gene identical to target Great spotted kiwi gene'\n",
       "1555. 'Great spotted kiwi Gene-order conservation score'\n",
       "1556. 'Great spotted kiwi Whole-genome alignment coverage'\n",
       "1557. 'dN with Great spotted kiwi'\n",
       "1558. 'dS with Great spotted kiwi'\n",
       "1559. 'Great spotted kiwi orthology confidence [0 low, 1 high]'\n",
       "1560. 'Greater amberjack gene stable ID'\n",
       "1561. 'Greater amberjack gene name'\n",
       "1562. 'Greater amberjack protein or transcript stable ID'\n",
       "1563. 'Greater amberjack chromosome/scaffold name'\n",
       "1564. 'Greater amberjack chromosome/scaffold start (bp)'\n",
       "1565. 'Greater amberjack chromosome/scaffold end (bp)'\n",
       "1566. 'Query protein or transcript ID'\n",
       "1567. 'Last common ancestor with Greater amberjack'\n",
       "1568. 'Greater amberjack homology type'\n",
       "1569. '%id. target Greater amberjack gene identical to query gene'\n",
       "1570. '%id. query gene identical to target Greater amberjack gene'\n",
       "1571. 'Greater amberjack Gene-order conservation score'\n",
       "1572. 'Greater amberjack Whole-genome alignment coverage'\n",
       "1573. 'dN with Greater amberjack'\n",
       "1574. 'dS with Greater amberjack'\n",
       "1575. 'Greater amberjack orthology confidence [0 low, 1 high]'\n",
       "1576. 'Greater bamboo lemur gene stable ID'\n",
       "1577. 'Greater bamboo lemur gene name'\n",
       "1578. 'Greater bamboo lemur protein or transcript stable ID'\n",
       "1579. 'Greater bamboo lemur chromosome/scaffold name'\n",
       "1580. 'Greater bamboo lemur chromosome/scaffold start (bp)'\n",
       "1581. 'Greater bamboo lemur chromosome/scaffold end (bp)'\n",
       "1582. 'Query protein or transcript ID'\n",
       "1583. 'Last common ancestor with Greater bamboo lemur'\n",
       "1584. 'Greater bamboo lemur homology type'\n",
       "1585. '%id. target Greater bamboo lemur gene identical to query gene'\n",
       "1586. '%id. query gene identical to target Greater bamboo lemur gene'\n",
       "1587. 'Greater bamboo lemur Gene-order conservation score'\n",
       "1588. 'Greater bamboo lemur Whole-genome alignment coverage'\n",
       "1589. 'dN with Greater bamboo lemur'\n",
       "1590. 'dS with Greater bamboo lemur'\n",
       "1591. 'Greater bamboo lemur orthology confidence [0 low, 1 high]'\n",
       "1592. 'Guinea Pig gene stable ID'\n",
       "1593. 'Guinea Pig gene name'\n",
       "1594. 'Guinea Pig protein or transcript stable ID'\n",
       "1595. 'Guinea Pig chromosome/scaffold name'\n",
       "1596. 'Guinea Pig chromosome/scaffold start (bp)'\n",
       "1597. 'Guinea Pig chromosome/scaffold end (bp)'\n",
       "1598. 'Query protein or transcript ID'\n",
       "1599. 'Last common ancestor with Guinea Pig'\n",
       "1600. 'Guinea Pig homology type'\n",
       "1601. '%id. target Guinea Pig gene identical to query gene'\n",
       "1602. '%id. query gene identical to target Guinea Pig gene'\n",
       "1603. 'Guinea Pig Gene-order conservation score'\n",
       "1604. 'Guinea Pig Whole-genome alignment coverage'\n",
       "1605. 'dN with Guinea Pig'\n",
       "1606. 'dS with Guinea Pig'\n",
       "1607. 'Guinea Pig orthology confidence [0 low, 1 high]'\n",
       "1608. 'Guppy gene stable ID'\n",
       "1609. 'Guppy gene name'\n",
       "1610. 'Guppy protein or transcript stable ID'\n",
       "1611. 'Guppy chromosome/scaffold name'\n",
       "1612. 'Guppy chromosome/scaffold start (bp)'\n",
       "1613. 'Guppy chromosome/scaffold end (bp)'\n",
       "1614. 'Query protein or transcript ID'\n",
       "1615. 'Last common ancestor with Guppy'\n",
       "1616. 'Guppy homology type'\n",
       "1617. '%id. target Guppy gene identical to query gene'\n",
       "1618. '%id. query gene identical to target Guppy gene'\n",
       "1619. 'Guppy Gene-order conservation score'\n",
       "1620. 'Guppy Whole-genome alignment coverage'\n",
       "1621. 'dN with Guppy'\n",
       "1622. 'dS with Guppy'\n",
       "1623. 'Guppy orthology confidence [0 low, 1 high]'\n",
       "1624. 'Hagfish gene stable ID'\n",
       "1625. 'Hagfish gene name'\n",
       "1626. 'Hagfish protein or transcript stable ID'\n",
       "1627. 'Hagfish chromosome/scaffold name'\n",
       "1628. 'Hagfish chromosome/scaffold start (bp)'\n",
       "1629. 'Hagfish chromosome/scaffold end (bp)'\n",
       "1630. 'Query protein or transcript ID'\n",
       "1631. 'Last common ancestor with Hagfish'\n",
       "1632. 'Hagfish homology type'\n",
       "1633. '%id. target Hagfish gene identical to query gene'\n",
       "1634. '%id. query gene identical to target Hagfish gene'\n",
       "1635. 'Hagfish Gene-order conservation score'\n",
       "1636. 'Hagfish Whole-genome alignment coverage'\n",
       "1637. 'dN with Hagfish'\n",
       "1638. 'dS with Hagfish'\n",
       "1639. 'Hagfish orthology confidence [0 low, 1 high]'\n",
       "1640. 'Hedgehog gene stable ID'\n",
       "1641. 'Hedgehog gene name'\n",
       "1642. 'Hedgehog protein or transcript stable ID'\n",
       "1643. 'Hedgehog chromosome/scaffold name'\n",
       "1644. 'Hedgehog chromosome/scaffold start (bp)'\n",
       "1645. 'Hedgehog chromosome/scaffold end (bp)'\n",
       "1646. 'Query protein or transcript ID'\n",
       "1647. 'Last common ancestor with Hedgehog'\n",
       "1648. 'Hedgehog homology type'\n",
       "1649. '%id. target Hedgehog gene identical to query gene'\n",
       "1650. '%id. query gene identical to target Hedgehog gene'\n",
       "1651. 'Hedgehog Gene-order conservation score'\n",
       "1652. 'Hedgehog Whole-genome alignment coverage'\n",
       "1653. 'dN with Hedgehog'\n",
       "1654. 'dS with Hedgehog'\n",
       "1655. 'Hedgehog orthology confidence [0 low, 1 high]'\n",
       "1656. 'Helmeted guineafowl gene stable ID'\n",
       "1657. 'Helmeted guineafowl gene name'\n",
       "1658. 'Helmeted guineafowl protein or transcript stable ID'\n",
       "1659. 'Helmeted guineafowl chromosome/scaffold name'\n",
       "1660. 'Helmeted guineafowl chromosome/scaffold start (bp)'\n",
       "1661. 'Helmeted guineafowl chromosome/scaffold end (bp)'\n",
       "1662. 'Query protein or transcript ID'\n",
       "1663. 'Last common ancestor with Helmeted guineafowl'\n",
       "1664. 'Helmeted guineafowl homology type'\n",
       "1665. '%id. target Helmeted guineafowl gene identical to query gene'\n",
       "1666. '%id. query gene identical to target Helmeted guineafowl gene'\n",
       "1667. 'Helmeted guineafowl Gene-order conservation score'\n",
       "1668. 'Helmeted guineafowl Whole-genome alignment coverage'\n",
       "1669. 'dN with Helmeted guineafowl'\n",
       "1670. 'dS with Helmeted guineafowl'\n",
       "1671. 'Helmeted guineafowl orthology confidence [0 low, 1 high]'\n",
       "1672. 'Horse gene stable ID'\n",
       "1673. 'Horse gene name'\n",
       "1674. 'Horse protein or transcript stable ID'\n",
       "1675. 'Horse chromosome/scaffold name'\n",
       "1676. 'Horse chromosome/scaffold start (bp)'\n",
       "1677. 'Horse chromosome/scaffold end (bp)'\n",
       "1678. 'Query protein or transcript ID'\n",
       "1679. 'Last common ancestor with Horse'\n",
       "1680. 'Horse homology type'\n",
       "1681. '%id. target Horse gene identical to query gene'\n",
       "1682. '%id. query gene identical to target Horse gene'\n",
       "1683. 'Horse Gene-order conservation score'\n",
       "1684. 'Horse Whole-genome alignment coverage'\n",
       "1685. 'dN with Horse'\n",
       "1686. 'dS with Horse'\n",
       "1687. 'Horse orthology confidence [0 low, 1 high]'\n",
       "1688. 'Huchen gene stable ID'\n",
       "1689. 'Huchen gene name'\n",
       "1690. 'Huchen protein or transcript stable ID'\n",
       "1691. 'Huchen chromosome/scaffold name'\n",
       "1692. 'Huchen chromosome/scaffold start (bp)'\n",
       "1693. 'Huchen chromosome/scaffold end (bp)'\n",
       "1694. 'Query protein or transcript ID'\n",
       "1695. 'Last common ancestor with Huchen'\n",
       "1696. 'Huchen homology type'\n",
       "1697. '%id. target Huchen gene identical to query gene'\n",
       "1698. '%id. query gene identical to target Huchen gene'\n",
       "1699. 'Huchen Gene-order conservation score'\n",
       "1700. 'Huchen Whole-genome alignment coverage'\n",
       "1701. 'dN with Huchen'\n",
       "1702. 'dS with Huchen'\n",
       "1703. 'Huchen orthology confidence [0 low, 1 high]'\n",
       "1704. 'Hyrax gene stable ID'\n",
       "1705. 'Hyrax gene name'\n",
       "1706. 'Hyrax protein or transcript stable ID'\n",
       "1707. 'Hyrax chromosome/scaffold name'\n",
       "1708. 'Hyrax chromosome/scaffold start (bp)'\n",
       "1709. 'Hyrax chromosome/scaffold end (bp)'\n",
       "1710. 'Query protein or transcript ID'\n",
       "1711. 'Last common ancestor with Hyrax'\n",
       "1712. 'Hyrax homology type'\n",
       "1713. '%id. target Hyrax gene identical to query gene'\n",
       "1714. '%id. query gene identical to target Hyrax gene'\n",
       "1715. 'Hyrax Gene-order conservation score'\n",
       "1716. 'Hyrax Whole-genome alignment coverage'\n",
       "1717. 'dN with Hyrax'\n",
       "1718. 'dS with Hyrax'\n",
       "1719. 'Hyrax orthology confidence [0 low, 1 high]'\n",
       "1720. 'Indian medaka gene stable ID'\n",
       "1721. 'Indian medaka gene name'\n",
       "1722. 'Indian medaka protein or transcript stable ID'\n",
       "1723. 'Indian medaka chromosome/scaffold name'\n",
       "1724. 'Indian medaka chromosome/scaffold start (bp)'\n",
       "1725. 'Indian medaka chromosome/scaffold end (bp)'\n",
       "1726. 'Query protein or transcript ID'\n",
       "1727. 'Last common ancestor with Indian medaka'\n",
       "1728. 'Indian medaka homology type'\n",
       "1729. '%id. target Indian medaka gene identical to query gene'\n",
       "1730. '%id. query gene identical to target Indian medaka gene'\n",
       "1731. 'Indian medaka Gene-order conservation score'\n",
       "1732. 'Indian medaka Whole-genome alignment coverage'\n",
       "1733. 'dN with Indian medaka'\n",
       "1734. 'dS with Indian medaka'\n",
       "1735. 'Indian medaka orthology confidence [0 low, 1 high]'\n",
       "1736. 'Japanese medaka HNI gene stable ID'\n",
       "1737. 'Japanese medaka HNI gene name'\n",
       "1738. 'Japanese medaka HNI protein or transcript stable ID'\n",
       "1739. 'Japanese medaka HNI chromosome/scaffold name'\n",
       "1740. 'Japanese medaka HNI chromosome/scaffold start (bp)'\n",
       "1741. 'Japanese medaka HNI chromosome/scaffold end (bp)'\n",
       "1742. 'Query protein or transcript ID'\n",
       "1743. 'Last common ancestor with Japanese medaka HNI'\n",
       "1744. 'Japanese medaka HNI homology type'\n",
       "1745. '%id. target Japanese medaka HNI gene identical to query gene'\n",
       "1746. '%id. query gene identical to target Japanese medaka HNI gene'\n",
       "1747. 'Japanese medaka HNI Gene-order conservation score'\n",
       "1748. 'Japanese medaka HNI Whole-genome alignment coverage'\n",
       "1749. 'dN with Japanese medaka HNI'\n",
       "1750. 'dS with Japanese medaka HNI'\n",
       "1751. 'Japanese medaka HNI orthology confidence [0 low, 1 high]'\n",
       "1752. 'Japanese medaka HSOK gene stable ID'\n",
       "1753. 'Japanese medaka HSOK gene name'\n",
       "1754. 'Japanese medaka HSOK protein or transcript stable ID'\n",
       "1755. 'Japanese medaka HSOK chromosome/scaffold name'\n",
       "1756. 'Japanese medaka HSOK chromosome/scaffold start (bp)'\n",
       "1757. 'Japanese medaka HSOK chromosome/scaffold end (bp)'\n",
       "1758. 'Query protein or transcript ID'\n",
       "1759. 'Last common ancestor with Japanese medaka HSOK'\n",
       "1760. 'Japanese medaka HSOK homology type'\n",
       "1761. '%id. target Japanese medaka HSOK gene identical to query gene'\n",
       "1762. '%id. query gene identical to target Japanese medaka HSOK gene'\n",
       "1763. 'Japanese medaka HSOK Gene-order conservation score'\n",
       "1764. 'Japanese medaka HSOK Whole-genome alignment coverage'\n",
       "1765. 'dN with Japanese medaka HSOK'\n",
       "1766. 'dS with Japanese medaka HSOK'\n",
       "1767. 'Japanese medaka HSOK orthology confidence [0 low, 1 high]'\n",
       "1768. 'Japanese medaka HdrR gene stable ID'\n",
       "1769. 'Japanese medaka HdrR gene name'\n",
       "1770. 'Japanese medaka HdrR protein or transcript stable ID'\n",
       "1771. 'Japanese medaka HdrR chromosome/scaffold name'\n",
       "1772. 'Japanese medaka HdrR chromosome/scaffold start (bp)'\n",
       "1773. 'Japanese medaka HdrR chromosome/scaffold end (bp)'\n",
       "1774. 'Query protein or transcript ID'\n",
       "1775. 'Last common ancestor with Japanese medaka HdrR'\n",
       "1776. 'Japanese medaka HdrR homology type'\n",
       "1777. '%id. target Japanese medaka HdrR gene identical to query gene'\n",
       "1778. '%id. query gene identical to target Japanese medaka HdrR gene'\n",
       "1779. 'Japanese medaka HdrR Gene-order conservation score'\n",
       "1780. 'Japanese medaka HdrR Whole-genome alignment coverage'\n",
       "1781. 'dN with Japanese medaka HdrR'\n",
       "1782. 'dS with Japanese medaka HdrR'\n",
       "1783. 'Japanese medaka HdrR orthology confidence [0 low, 1 high]'\n",
       "1784. 'Japanese quail gene stable ID'\n",
       "1785. 'Japanese quail gene name'\n",
       "1786. 'Japanese quail protein or transcript stable ID'\n",
       "1787. 'Japanese quail chromosome/scaffold name'\n",
       "1788. 'Japanese quail chromosome/scaffold start (bp)'\n",
       "1789. 'Japanese quail chromosome/scaffold end (bp)'\n",
       "1790. 'Query protein or transcript ID'\n",
       "1791. 'Last common ancestor with Japanese quail'\n",
       "1792. 'Japanese quail homology type'\n",
       "1793. '%id. target Japanese quail gene identical to query gene'\n",
       "1794. '%id. query gene identical to target Japanese quail gene'\n",
       "1795. 'Japanese quail Gene-order conservation score'\n",
       "1796. 'Japanese quail Whole-genome alignment coverage'\n",
       "1797. 'dN with Japanese quail'\n",
       "1798. 'dS with Japanese quail'\n",
       "1799. 'Japanese quail orthology confidence [0 low, 1 high]'\n",
       "1800. 'Kangaroo rat gene stable ID'\n",
       "1801. 'Kangaroo rat gene name'\n",
       "1802. 'Kangaroo rat protein or transcript stable ID'\n",
       "1803. 'Kangaroo rat chromosome/scaffold name'\n",
       "1804. 'Kangaroo rat chromosome/scaffold start (bp)'\n",
       "1805. 'Kangaroo rat chromosome/scaffold end (bp)'\n",
       "1806. 'Query protein or transcript ID'\n",
       "1807. 'Last common ancestor with Kangaroo rat'\n",
       "1808. 'Kangaroo rat homology type'\n",
       "1809. '%id. target Kangaroo rat gene identical to query gene'\n",
       "1810. '%id. query gene identical to target Kangaroo rat gene'\n",
       "1811. 'Kangaroo rat Gene-order conservation score'\n",
       "1812. 'Kangaroo rat Whole-genome alignment coverage'\n",
       "1813. 'dN with Kangaroo rat'\n",
       "1814. 'dS with Kangaroo rat'\n",
       "1815. 'Kangaroo rat orthology confidence [0 low, 1 high]'\n",
       "1816. 'Koala gene stable ID'\n",
       "1817. 'Koala gene name'\n",
       "1818. 'Koala protein or transcript stable ID'\n",
       "1819. 'Koala chromosome/scaffold name'\n",
       "1820. 'Koala chromosome/scaffold start (bp)'\n",
       "1821. 'Koala chromosome/scaffold end (bp)'\n",
       "1822. 'Query protein or transcript ID'\n",
       "1823. 'Last common ancestor with Koala'\n",
       "1824. 'Koala homology type'\n",
       "1825. '%id. target Koala gene identical to query gene'\n",
       "1826. '%id. query gene identical to target Koala gene'\n",
       "1827. 'Koala Gene-order conservation score'\n",
       "1828. 'Koala Whole-genome alignment coverage'\n",
       "1829. 'dN with Koala'\n",
       "1830. 'dS with Koala'\n",
       "1831. 'Koala orthology confidence [0 low, 1 high]'\n",
       "1832. 'Lamprey gene stable ID'\n",
       "1833. 'Lamprey gene name'\n",
       "1834. 'Lamprey protein or transcript stable ID'\n",
       "1835. 'Lamprey chromosome/scaffold name'\n",
       "1836. 'Lamprey chromosome/scaffold start (bp)'\n",
       "1837. 'Lamprey chromosome/scaffold end (bp)'\n",
       "1838. 'Query protein or transcript ID'\n",
       "1839. 'Last common ancestor with Lamprey'\n",
       "1840. 'Lamprey homology type'\n",
       "1841. '%id. target Lamprey gene identical to query gene'\n",
       "1842. '%id. query gene identical to target Lamprey gene'\n",
       "1843. 'Lamprey Gene-order conservation score'\n",
       "1844. 'Lamprey Whole-genome alignment coverage'\n",
       "1845. 'dN with Lamprey'\n",
       "1846. 'dS with Lamprey'\n",
       "1847. 'Lamprey orthology confidence [0 low, 1 high]'\n",
       "1848. 'Leopard gene stable ID'\n",
       "1849. 'Leopard gene name'\n",
       "1850. 'Leopard protein or transcript stable ID'\n",
       "1851. 'Leopard chromosome/scaffold name'\n",
       "1852. 'Leopard chromosome/scaffold start (bp)'\n",
       "1853. 'Leopard chromosome/scaffold end (bp)'\n",
       "1854. 'Query protein or transcript ID'\n",
       "1855. 'Last common ancestor with Leopard'\n",
       "1856. 'Leopard homology type'\n",
       "1857. '%id. target Leopard gene identical to query gene'\n",
       "1858. '%id. query gene identical to target Leopard gene'\n",
       "1859. 'Leopard Gene-order conservation score'\n",
       "1860. 'Leopard Whole-genome alignment coverage'\n",
       "1861. 'dN with Leopard'\n",
       "1862. 'dS with Leopard'\n",
       "1863. 'Leopard orthology confidence [0 low, 1 high]'\n",
       "1864. 'Lesser Egyptian jerboa gene stable ID'\n",
       "1865. 'Lesser Egyptian jerboa gene name'\n",
       "1866. 'Lesser Egyptian jerboa protein or transcript stable ID'\n",
       "1867. 'Lesser Egyptian jerboa chromosome/scaffold name'\n",
       "1868. 'Lesser Egyptian jerboa chromosome/scaffold start (bp)'\n",
       "1869. 'Lesser Egyptian jerboa chromosome/scaffold end (bp)'\n",
       "1870. 'Query protein or transcript ID'\n",
       "1871. 'Last common ancestor with Lesser Egyptian jerboa'\n",
       "1872. 'Lesser Egyptian jerboa homology type'\n",
       "1873. '%id. target Lesser Egyptian jerboa gene identical to query gene'\n",
       "1874. '%id. query gene identical to target Lesser Egyptian jerboa gene'\n",
       "1875. 'Lesser Egyptian jerboa Gene-order conservation score'\n",
       "1876. 'Lesser Egyptian jerboa Whole-genome alignment coverage'\n",
       "1877. 'dN with Lesser Egyptian jerboa'\n",
       "1878. 'dS with Lesser Egyptian jerboa'\n",
       "1879. 'Lesser Egyptian jerboa orthology confidence [0 low, 1 high]'\n",
       "1880. 'Lesser hedgehog tenrec gene stable ID'\n",
       "1881. 'Lesser hedgehog tenrec gene name'\n",
       "1882. 'Lesser hedgehog tenrec protein or transcript stable ID'\n",
       "1883. 'Lesser hedgehog tenrec chromosome/scaffold name'\n",
       "1884. 'Lesser hedgehog tenrec chromosome/scaffold start (bp)'\n",
       "1885. 'Lesser hedgehog tenrec chromosome/scaffold end (bp)'\n",
       "1886. 'Query protein or transcript ID'\n",
       "1887. 'Last common ancestor with Lesser hedgehog tenrec'\n",
       "1888. 'Lesser hedgehog tenrec homology type'\n",
       "1889. '%id. target Lesser hedgehog tenrec gene identical to query gene'\n",
       "1890. '%id. query gene identical to target Lesser hedgehog tenrec gene'\n",
       "1891. 'Lesser hedgehog tenrec Gene-order conservation score'\n",
       "1892. 'Lesser hedgehog tenrec Whole-genome alignment coverage'\n",
       "1893. 'dN with Lesser hedgehog tenrec'\n",
       "1894. 'dS with Lesser hedgehog tenrec'\n",
       "1895. 'Lesser hedgehog tenrec orthology confidence [0 low, 1 high]'\n",
       "1896. 'Little spotted kiwi gene stable ID'\n",
       "1897. 'Little spotted kiwi gene name'\n",
       "1898. 'Little spotted kiwi protein or transcript stable ID'\n",
       "1899. 'Little spotted kiwi chromosome/scaffold name'\n",
       "1900. 'Little spotted kiwi chromosome/scaffold start (bp)'\n",
       "1901. 'Little spotted kiwi chromosome/scaffold end (bp)'\n",
       "1902. 'Query protein or transcript ID'\n",
       "1903. 'Last common ancestor with Little spotted kiwi'\n",
       "1904. 'Little spotted kiwi homology type'\n",
       "1905. '%id. target Little spotted kiwi gene identical to query gene'\n",
       "1906. '%id. query gene identical to target Little spotted kiwi gene'\n",
       "1907. 'Little spotted kiwi Gene-order conservation score'\n",
       "1908. 'Little spotted kiwi Whole-genome alignment coverage'\n",
       "1909. 'dN with Little spotted kiwi'\n",
       "1910. 'dS with Little spotted kiwi'\n",
       "1911. 'Little spotted kiwi orthology confidence [0 low, 1 high]'\n",
       "1912. 'Long-tailed chinchilla gene stable ID'\n",
       "1913. 'Long-tailed chinchilla gene name'\n",
       "1914. 'Long-tailed chinchilla protein or transcript stable ID'\n",
       "1915. 'Long-tailed chinchilla chromosome/scaffold name'\n",
       "1916. 'Long-tailed chinchilla chromosome/scaffold start (bp)'\n",
       "1917. 'Long-tailed chinchilla chromosome/scaffold end (bp)'\n",
       "1918. 'Query protein or transcript ID'\n",
       "1919. 'Last common ancestor with Long-tailed chinchilla'\n",
       "1920. 'Long-tailed chinchilla homology type'\n",
       "1921. '%id. target Long-tailed chinchilla gene identical to query gene'\n",
       "1922. '%id. query gene identical to target Long-tailed chinchilla gene'\n",
       "1923. 'Long-tailed chinchilla Gene-order conservation score'\n",
       "1924. 'Long-tailed chinchilla Whole-genome alignment coverage'\n",
       "1925. 'dN with Long-tailed chinchilla'\n",
       "1926. 'dS with Long-tailed chinchilla'\n",
       "1927. 'Long-tailed chinchilla orthology confidence [0 low, 1 high]'\n",
       "1928. 'Lyretail cichlid gene stable ID'\n",
       "1929. 'Lyretail cichlid gene name'\n",
       "1930. 'Lyretail cichlid protein or transcript stable ID'\n",
       "1931. 'Lyretail cichlid chromosome/scaffold name'\n",
       "1932. 'Lyretail cichlid chromosome/scaffold start (bp)'\n",
       "1933. 'Lyretail cichlid chromosome/scaffold end (bp)'\n",
       "1934. 'Query protein or transcript ID'\n",
       "1935. 'Last common ancestor with Lyretail cichlid'\n",
       "1936. 'Lyretail cichlid homology type'\n",
       "1937. '%id. target Lyretail cichlid gene identical to query gene'\n",
       "1938. '%id. query gene identical to target Lyretail cichlid gene'\n",
       "1939. 'Lyretail cichlid Gene-order conservation score'\n",
       "1940. 'Lyretail cichlid Whole-genome alignment coverage'\n",
       "1941. 'dN with Lyretail cichlid'\n",
       "1942. 'dS with Lyretail cichlid'\n",
       "1943. 'Lyretail cichlid orthology confidence [0 low, 1 high]'\n",
       "1944. 'Ma\\'s night monkey gene stable ID'\n",
       "1945. 'Ma\\'s night monkey gene name'\n",
       "1946. 'Ma\\'s night monkey protein or transcript stable ID'\n",
       "1947. 'Ma\\'s night monkey chromosome/scaffold name'\n",
       "1948. 'Ma\\'s night monkey chromosome/scaffold start (bp)'\n",
       "1949. 'Ma\\'s night monkey chromosome/scaffold end (bp)'\n",
       "1950. 'Query protein or transcript ID'\n",
       "1951. 'Last common ancestor with Ma\\'s night monkey'\n",
       "1952. 'Ma\\'s night monkey homology type'\n",
       "1953. '%id. target Ma\\'s night monkey gene identical to query gene'\n",
       "1954. '%id. query gene identical to target Ma\\'s night monkey gene'\n",
       "1955. 'Ma\\'s night monkey Gene-order conservation score'\n",
       "1956. 'Ma\\'s night monkey Whole-genome alignment coverage'\n",
       "1957. 'dN with Ma\\'s night monkey'\n",
       "1958. 'dS with Ma\\'s night monkey'\n",
       "1959. 'Ma\\'s night monkey orthology confidence [0 low, 1 high]'\n",
       "1960. 'Macaque gene stable ID'\n",
       "1961. 'Macaque gene name'\n",
       "1962. 'Macaque protein or transcript stable ID'\n",
       "1963. 'Macaque chromosome/scaffold name'\n",
       "1964. 'Macaque chromosome/scaffold start (bp)'\n",
       "1965. 'Macaque chromosome/scaffold end (bp)'\n",
       "1966. 'Query protein or transcript ID'\n",
       "1967. 'Last common ancestor with Macaque'\n",
       "1968. 'Macaque homology type'\n",
       "1969. '%id. target Macaque gene identical to query gene'\n",
       "1970. '%id. query gene identical to target Macaque gene'\n",
       "1971. 'Macaque Gene-order conservation score'\n",
       "1972. 'Macaque Whole-genome alignment coverage'\n",
       "1973. 'dN with Macaque'\n",
       "1974. 'dS with Macaque'\n",
       "1975. 'Macaque orthology confidence [0 low, 1 high]'\n",
       "1976. 'Mainland tiger snake gene stable ID'\n",
       "1977. 'Mainland tiger snake gene name'\n",
       "1978. 'Mainland tiger snake protein or transcript stable ID'\n",
       "1979. 'Mainland tiger snake chromosome/scaffold name'\n",
       "1980. 'Mainland tiger snake chromosome/scaffold start (bp)'\n",
       "1981. 'Mainland tiger snake chromosome/scaffold end (bp)'\n",
       "1982. 'Query protein or transcript ID'\n",
       "1983. 'Last common ancestor with Mainland tiger snake'\n",
       "1984. 'Mainland tiger snake homology type'\n",
       "1985. '%id. target Mainland tiger snake gene identical to query gene'\n",
       "1986. '%id. query gene identical to target Mainland tiger snake gene'\n",
       "1987. 'Mainland tiger snake Gene-order conservation score'\n",
       "1988. 'Mainland tiger snake Whole-genome alignment coverage'\n",
       "1989. 'dN with Mainland tiger snake'\n",
       "1990. 'dS with Mainland tiger snake'\n",
       "1991. 'Mainland tiger snake orthology confidence [0 low, 1 high]'\n",
       "1992. 'Makobe Island cichlid gene stable ID'\n",
       "1993. 'Makobe Island cichlid gene name'\n",
       "1994. 'Makobe Island cichlid protein or transcript stable ID'\n",
       "1995. 'Makobe Island cichlid chromosome/scaffold name'\n",
       "1996. 'Makobe Island cichlid chromosome/scaffold start (bp)'\n",
       "1997. 'Makobe Island cichlid chromosome/scaffold end (bp)'\n",
       "1998. 'Query protein or transcript ID'\n",
       "1999. 'Last common ancestor with Makobe Island cichlid'\n",
       "2000. 'Makobe Island cichlid homology type'\n",
       "2001. '%id. target Makobe Island cichlid gene identical to query gene'\n",
       "2002. '%id. query gene identical to target Makobe Island cichlid gene'\n",
       "2003. 'Makobe Island cichlid Gene-order conservation score'\n",
       "2004. 'Makobe Island cichlid Whole-genome alignment coverage'\n",
       "2005. 'dN with Makobe Island cichlid'\n",
       "2006. 'dS with Makobe Island cichlid'\n",
       "2007. 'Makobe Island cichlid orthology confidence [0 low, 1 high]'\n",
       "2008. 'Mangrove rivulus gene stable ID'\n",
       "2009. 'Mangrove rivulus gene name'\n",
       "2010. 'Mangrove rivulus protein or transcript stable ID'\n",
       "2011. 'Mangrove rivulus chromosome/scaffold name'\n",
       "2012. 'Mangrove rivulus chromosome/scaffold start (bp)'\n",
       "2013. 'Mangrove rivulus chromosome/scaffold end (bp)'\n",
       "2014. 'Query protein or transcript ID'\n",
       "2015. 'Last common ancestor with Mangrove rivulus'\n",
       "2016. 'Mangrove rivulus homology type'\n",
       "2017. '%id. target Mangrove rivulus gene identical to query gene'\n",
       "2018. '%id. query gene identical to target Mangrove rivulus gene'\n",
       "2019. 'Mangrove rivulus Gene-order conservation score'\n",
       "2020. 'Mangrove rivulus Whole-genome alignment coverage'\n",
       "2021. 'dN with Mangrove rivulus'\n",
       "2022. 'dS with Mangrove rivulus'\n",
       "2023. 'Mangrove rivulus orthology confidence [0 low, 1 high]'\n",
       "2024. 'Marmoset gene stable ID'\n",
       "2025. 'Marmoset gene name'\n",
       "2026. 'Marmoset protein or transcript stable ID'\n",
       "2027. 'Marmoset chromosome/scaffold name'\n",
       "2028. 'Marmoset chromosome/scaffold start (bp)'\n",
       "2029. 'Marmoset chromosome/scaffold end (bp)'\n",
       "2030. 'Query protein or transcript ID'\n",
       "2031. 'Last common ancestor with Marmoset'\n",
       "2032. 'Marmoset homology type'\n",
       "2033. '%id. target Marmoset gene identical to query gene'\n",
       "2034. '%id. query gene identical to target Marmoset gene'\n",
       "2035. 'Marmoset Gene-order conservation score'\n",
       "2036. 'Marmoset Whole-genome alignment coverage'\n",
       "2037. 'dN with Marmoset'\n",
       "2038. 'dS with Marmoset'\n",
       "2039. 'Marmoset orthology confidence [0 low, 1 high]'\n",
       "2040. 'Megabat gene stable ID'\n",
       "2041. 'Megabat gene name'\n",
       "2042. 'Megabat protein or transcript stable ID'\n",
       "2043. 'Megabat chromosome/scaffold name'\n",
       "2044. 'Megabat chromosome/scaffold start (bp)'\n",
       "2045. 'Megabat chromosome/scaffold end (bp)'\n",
       "2046. 'Query protein or transcript ID'\n",
       "2047. 'Last common ancestor with Megabat'\n",
       "2048. 'Megabat homology type'\n",
       "2049. '%id. target Megabat gene identical to query gene'\n",
       "2050. '%id. query gene identical to target Megabat gene'\n",
       "2051. 'Megabat Gene-order conservation score'\n",
       "2052. 'Megabat Whole-genome alignment coverage'\n",
       "2053. 'dN with Megabat'\n",
       "2054. 'dS with Megabat'\n",
       "2055. 'Megabat orthology confidence [0 low, 1 high]'\n",
       "2056. 'Mexican tetra gene stable ID'\n",
       "2057. 'Mexican tetra gene name'\n",
       "2058. 'Mexican tetra protein or transcript stable ID'\n",
       "2059. 'Mexican tetra chromosome/scaffold name'\n",
       "2060. 'Mexican tetra chromosome/scaffold start (bp)'\n",
       "2061. 'Mexican tetra chromosome/scaffold end (bp)'\n",
       "2062. 'Query protein or transcript ID'\n",
       "2063. 'Last common ancestor with Mexican tetra'\n",
       "2064. 'Mexican tetra homology type'\n",
       "2065. '%id. target Mexican tetra gene identical to query gene'\n",
       "2066. '%id. query gene identical to target Mexican tetra gene'\n",
       "2067. 'Mexican tetra Gene-order conservation score'\n",
       "2068. 'Mexican tetra Whole-genome alignment coverage'\n",
       "2069. 'dN with Mexican tetra'\n",
       "2070. 'dS with Mexican tetra'\n",
       "2071. 'Mexican tetra orthology confidence [0 low, 1 high]'\n",
       "2072. 'Microbat gene stable ID'\n",
       "2073. 'Microbat gene name'\n",
       "2074. 'Microbat protein or transcript stable ID'\n",
       "2075. 'Microbat chromosome/scaffold name'\n",
       "2076. 'Microbat chromosome/scaffold start (bp)'\n",
       "2077. 'Microbat chromosome/scaffold end (bp)'\n",
       "2078. 'Query protein or transcript ID'\n",
       "2079. 'Last common ancestor with Microbat'\n",
       "2080. 'Microbat homology type'\n",
       "2081. '%id. target Microbat gene identical to query gene'\n",
       "2082. '%id. query gene identical to target Microbat gene'\n",
       "2083. 'Microbat Gene-order conservation score'\n",
       "2084. 'Microbat Whole-genome alignment coverage'\n",
       "2085. 'dN with Microbat'\n",
       "2086. 'dS with Microbat'\n",
       "2087. 'Microbat orthology confidence [0 low, 1 high]'\n",
       "2088. 'Midas cichlid gene stable ID'\n",
       "2089. 'Midas cichlid gene name'\n",
       "2090. 'Midas cichlid protein or transcript stable ID'\n",
       "2091. 'Midas cichlid chromosome/scaffold name'\n",
       "2092. 'Midas cichlid chromosome/scaffold start (bp)'\n",
       "2093. 'Midas cichlid chromosome/scaffold end (bp)'\n",
       "2094. 'Query protein or transcript ID'\n",
       "2095. 'Last common ancestor with Midas cichlid'\n",
       "2096. 'Midas cichlid homology type'\n",
       "2097. '%id. target Midas cichlid gene identical to query gene'\n",
       "2098. '%id. query gene identical to target Midas cichlid gene'\n",
       "2099. 'Midas cichlid Gene-order conservation score'\n",
       "2100. 'Midas cichlid Whole-genome alignment coverage'\n",
       "2101. 'dN with Midas cichlid'\n",
       "2102. 'dS with Midas cichlid'\n",
       "2103. 'Midas cichlid orthology confidence [0 low, 1 high]'\n",
       "2104. 'Mongolian gerbil gene stable ID'\n",
       "2105. 'Mongolian gerbil gene name'\n",
       "2106. 'Mongolian gerbil protein or transcript stable ID'\n",
       "2107. 'Mongolian gerbil chromosome/scaffold name'\n",
       "2108. 'Mongolian gerbil chromosome/scaffold start (bp)'\n",
       "2109. 'Mongolian gerbil chromosome/scaffold end (bp)'\n",
       "2110. 'Query protein or transcript ID'\n",
       "2111. 'Last common ancestor with Mongolian gerbil'\n",
       "2112. 'Mongolian gerbil homology type'\n",
       "2113. '%id. target Mongolian gerbil gene identical to query gene'\n",
       "2114. '%id. query gene identical to target Mongolian gerbil gene'\n",
       "2115. 'Mongolian gerbil Gene-order conservation score'\n",
       "2116. 'Mongolian gerbil Whole-genome alignment coverage'\n",
       "2117. 'dN with Mongolian gerbil'\n",
       "2118. 'dS with Mongolian gerbil'\n",
       "2119. 'Mongolian gerbil orthology confidence [0 low, 1 high]'\n",
       "2120. 'Monterrey platyfish gene stable ID'\n",
       "2121. 'Monterrey platyfish gene name'\n",
       "2122. 'Monterrey platyfish protein or transcript stable ID'\n",
       "2123. 'Monterrey platyfish chromosome/scaffold name'\n",
       "2124. 'Monterrey platyfish chromosome/scaffold start (bp)'\n",
       "2125. 'Monterrey platyfish chromosome/scaffold end (bp)'\n",
       "2126. 'Query protein or transcript ID'\n",
       "2127. 'Last common ancestor with Monterrey platyfish'\n",
       "2128. 'Monterrey platyfish homology type'\n",
       "2129. '%id. target Monterrey platyfish gene identical to query gene'\n",
       "2130. '%id. query gene identical to target Monterrey platyfish gene'\n",
       "2131. 'Monterrey platyfish Gene-order conservation score'\n",
       "2132. 'Monterrey platyfish Whole-genome alignment coverage'\n",
       "2133. 'dN with Monterrey platyfish'\n",
       "2134. 'dS with Monterrey platyfish'\n",
       "2135. 'Monterrey platyfish orthology confidence [0 low, 1 high]'\n",
       "2136. 'Mouse gene stable ID'\n",
       "2137. 'Mouse gene name'\n",
       "2138. 'Mouse protein or transcript stable ID'\n",
       "2139. 'Mouse chromosome/scaffold name'\n",
       "2140. 'Mouse chromosome/scaffold start (bp)'\n",
       "2141. 'Mouse chromosome/scaffold end (bp)'\n",
       "2142. 'Query protein or transcript ID'\n",
       "2143. 'Last common ancestor with Mouse'\n",
       "2144. 'Mouse homology type'\n",
       "2145. '%id. target Mouse gene identical to query gene'\n",
       "2146. '%id. query gene identical to target Mouse gene'\n",
       "2147. 'Mouse Gene-order conservation score'\n",
       "2148. 'Mouse Whole-genome alignment coverage'\n",
       "2149. 'dN with Mouse'\n",
       "2150. 'dS with Mouse'\n",
       "2151. 'Mouse orthology confidence [0 low, 1 high]'\n",
       "2152. 'Mouse Lemur gene stable ID'\n",
       "2153. 'Mouse Lemur gene name'\n",
       "2154. 'Mouse Lemur protein or transcript stable ID'\n",
       "2155. 'Mouse Lemur chromosome/scaffold name'\n",
       "2156. 'Mouse Lemur chromosome/scaffold start (bp)'\n",
       "2157. 'Mouse Lemur chromosome/scaffold end (bp)'\n",
       "2158. 'Query protein or transcript ID'\n",
       "2159. 'Last common ancestor with Mouse Lemur'\n",
       "2160. 'Mouse Lemur homology type'\n",
       "2161. '%id. target Mouse Lemur gene identical to query gene'\n",
       "2162. '%id. query gene identical to target Mouse Lemur gene'\n",
       "2163. 'Mouse Lemur Gene-order conservation score'\n",
       "2164. 'Mouse Lemur Whole-genome alignment coverage'\n",
       "2165. 'dN with Mouse Lemur'\n",
       "2166. 'dS with Mouse Lemur'\n",
       "2167. 'Mouse Lemur orthology confidence [0 low, 1 high]'\n",
       "2168. 'Mummichog gene stable ID'\n",
       "2169. 'Mummichog gene name'\n",
       "2170. 'Mummichog protein or transcript stable ID'\n",
       "2171. 'Mummichog chromosome/scaffold name'\n",
       "2172. 'Mummichog chromosome/scaffold start (bp)'\n",
       "2173. 'Mummichog chromosome/scaffold end (bp)'\n",
       "2174. 'Query protein or transcript ID'\n",
       "2175. 'Last common ancestor with Mummichog'\n",
       "2176. 'Mummichog homology type'\n",
       "2177. '%id. target Mummichog gene identical to query gene'\n",
       "2178. '%id. query gene identical to target Mummichog gene'\n",
       "2179. 'Mummichog Gene-order conservation score'\n",
       "2180. 'Mummichog Whole-genome alignment coverage'\n",
       "2181. 'dN with Mummichog'\n",
       "2182. 'dS with Mummichog'\n",
       "2183. 'Mummichog orthology confidence [0 low, 1 high]'\n",
       "2184. 'Naked mole-rat female gene stable ID'\n",
       "2185. 'Naked mole-rat female gene name'\n",
       "2186. 'Naked mole-rat female protein or transcript stable ID'\n",
       "2187. 'Naked mole-rat female chromosome/scaffold name'\n",
       "2188. 'Naked mole-rat female chromosome/scaffold start (bp)'\n",
       "2189. 'Naked mole-rat female chromosome/scaffold end (bp)'\n",
       "2190. 'Query protein or transcript ID'\n",
       "2191. 'Last common ancestor with Naked mole-rat female'\n",
       "2192. 'Naked mole-rat female homology type'\n",
       "2193. '%id. target Naked mole-rat female gene identical to query gene'\n",
       "2194. '%id. query gene identical to target Naked mole-rat female gene'\n",
       "2195. 'Naked mole-rat female Gene-order conservation score'\n",
       "2196. 'Naked mole-rat female Whole-genome alignment coverage'\n",
       "2197. 'dN with Naked mole-rat female'\n",
       "2198. 'dS with Naked mole-rat female'\n",
       "2199. 'Naked mole-rat female orthology confidence [0 low, 1 high]'\n",
       "2200. 'Naked mole-rat male gene stable ID'\n",
       "2201. 'Naked mole-rat male gene name'\n",
       "2202. 'Naked mole-rat male protein or transcript stable ID'\n",
       "2203. 'Naked mole-rat male chromosome/scaffold name'\n",
       "2204. 'Naked mole-rat male chromosome/scaffold start (bp)'\n",
       "2205. 'Naked mole-rat male chromosome/scaffold end (bp)'\n",
       "2206. 'Query protein or transcript ID'\n",
       "2207. 'Last common ancestor with Naked mole-rat male'\n",
       "2208. 'Naked mole-rat male homology type'\n",
       "2209. '%id. target Naked mole-rat male gene identical to query gene'\n",
       "2210. '%id. query gene identical to target Naked mole-rat male gene'\n",
       "2211. 'Naked mole-rat male Gene-order conservation score'\n",
       "2212. 'Naked mole-rat male Whole-genome alignment coverage'\n",
       "2213. 'dN with Naked mole-rat male'\n",
       "2214. 'dS with Naked mole-rat male'\n",
       "2215. 'Naked mole-rat male orthology confidence [0 low, 1 high]'\n",
       "2216. 'Northern American deer mouse gene stable ID'\n",
       "2217. 'Northern American deer mouse gene name'\n",
       "2218. 'Northern American deer mouse protein or transcript stable ID'\n",
       "2219. 'Northern American deer mouse chromosome/scaffold name'\n",
       "2220. 'Northern American deer mouse chromosome/scaffold start (bp)'\n",
       "2221. 'Northern American deer mouse chromosome/scaffold end (bp)'\n",
       "2222. 'Query protein or transcript ID'\n",
       "2223. 'Last common ancestor with Northern American deer mouse'\n",
       "2224. 'Northern American deer mouse homology type'\n",
       "2225. '%id. target Northern American deer mouse gene identical to query gene'\n",
       "2226. '%id. query gene identical to target Northern American deer mouse gene'\n",
       "2227. 'Northern American deer mouse Gene-order conservation score'\n",
       "2228. 'Northern American deer mouse Whole-genome alignment coverage'\n",
       "2229. 'dN with Northern American deer mouse'\n",
       "2230. 'dS with Northern American deer mouse'\n",
       "2231. 'Northern American deer mouse orthology confidence [0 low, 1 high]'\n",
       "2232. 'Northern pike gene stable ID'\n",
       "2233. 'Northern pike gene name'\n",
       "2234. 'Northern pike protein or transcript stable ID'\n",
       "2235. 'Northern pike chromosome/scaffold name'\n",
       "2236. 'Northern pike chromosome/scaffold start (bp)'\n",
       "2237. 'Northern pike chromosome/scaffold end (bp)'\n",
       "2238. 'Query protein or transcript ID'\n",
       "2239. 'Last common ancestor with Northern pike'\n",
       "2240. 'Northern pike homology type'\n",
       "2241. '%id. target Northern pike gene identical to query gene'\n",
       "2242. '%id. query gene identical to target Northern pike gene'\n",
       "2243. 'Northern pike Gene-order conservation score'\n",
       "2244. 'Northern pike Whole-genome alignment coverage'\n",
       "2245. 'dN with Northern pike'\n",
       "2246. 'dS with Northern pike'\n",
       "2247. 'Northern pike orthology confidence [0 low, 1 high]'\n",
       "2248. 'Ocean sunfish gene stable ID'\n",
       "2249. 'Ocean sunfish gene name'\n",
       "2250. 'Ocean sunfish protein or transcript stable ID'\n",
       "2251. 'Ocean sunfish chromosome/scaffold name'\n",
       "2252. 'Ocean sunfish chromosome/scaffold start (bp)'\n",
       "2253. 'Ocean sunfish chromosome/scaffold end (bp)'\n",
       "2254. 'Query protein or transcript ID'\n",
       "2255. 'Last common ancestor with Ocean sunfish'\n",
       "2256. 'Ocean sunfish homology type'\n",
       "2257. '%id. target Ocean sunfish gene identical to query gene'\n",
       "2258. '%id. query gene identical to target Ocean sunfish gene'\n",
       "2259. 'Ocean sunfish Gene-order conservation score'\n",
       "2260. 'Ocean sunfish Whole-genome alignment coverage'\n",
       "2261. 'dN with Ocean sunfish'\n",
       "2262. 'dS with Ocean sunfish'\n",
       "2263. 'Ocean sunfish orthology confidence [0 low, 1 high]'\n",
       "2264. 'Okarito brown kiwi gene stable ID'\n",
       "2265. 'Okarito brown kiwi gene name'\n",
       "2266. 'Okarito brown kiwi protein or transcript stable ID'\n",
       "2267. 'Okarito brown kiwi chromosome/scaffold name'\n",
       "2268. 'Okarito brown kiwi chromosome/scaffold start (bp)'\n",
       "2269. 'Okarito brown kiwi chromosome/scaffold end (bp)'\n",
       "2270. 'Query protein or transcript ID'\n",
       "2271. 'Last common ancestor with Okarito brown kiwi'\n",
       "2272. 'Okarito brown kiwi homology type'\n",
       "2273. '%id. target Okarito brown kiwi gene identical to query gene'\n",
       "2274. '%id. query gene identical to target Okarito brown kiwi gene'\n",
       "2275. 'Okarito brown kiwi Gene-order conservation score'\n",
       "2276. 'Okarito brown kiwi Whole-genome alignment coverage'\n",
       "2277. 'dN with Okarito brown kiwi'\n",
       "2278. 'dS with Okarito brown kiwi'\n",
       "2279. 'Okarito brown kiwi orthology confidence [0 low, 1 high]'\n",
       "2280. 'Olive baboon gene stable ID'\n",
       "2281. 'Olive baboon gene name'\n",
       "2282. 'Olive baboon protein or transcript stable ID'\n",
       "2283. 'Olive baboon chromosome/scaffold name'\n",
       "2284. 'Olive baboon chromosome/scaffold start (bp)'\n",
       "2285. 'Olive baboon chromosome/scaffold end (bp)'\n",
       "2286. 'Query protein or transcript ID'\n",
       "2287. 'Last common ancestor with Olive baboon'\n",
       "2288. 'Olive baboon homology type'\n",
       "2289. '%id. target Olive baboon gene identical to query gene'\n",
       "2290. '%id. query gene identical to target Olive baboon gene'\n",
       "2291. 'Olive baboon Gene-order conservation score'\n",
       "2292. 'Olive baboon Whole-genome alignment coverage'\n",
       "2293. 'dN with Olive baboon'\n",
       "2294. 'dS with Olive baboon'\n",
       "2295. 'Olive baboon orthology confidence [0 low, 1 high]'\n",
       "2296. 'Opossum gene stable ID'\n",
       "2297. 'Opossum gene name'\n",
       "2298. 'Opossum protein or transcript stable ID'\n",
       "2299. 'Opossum chromosome/scaffold name'\n",
       "2300. 'Opossum chromosome/scaffold start (bp)'\n",
       "2301. 'Opossum chromosome/scaffold end (bp)'\n",
       "2302. 'Query protein or transcript ID'\n",
       "2303. 'Last common ancestor with Opossum'\n",
       "2304. 'Opossum homology type'\n",
       "2305. '%id. target Opossum gene identical to query gene'\n",
       "2306. '%id. query gene identical to target Opossum gene'\n",
       "2307. 'Opossum Gene-order conservation score'\n",
       "2308. 'Opossum Whole-genome alignment coverage'\n",
       "2309. 'dN with Opossum'\n",
       "2310. 'dS with Opossum'\n",
       "2311. 'Opossum orthology confidence [0 low, 1 high]'\n",
       "2312. 'Orange clownfish gene stable ID'\n",
       "2313. 'Orange clownfish gene name'\n",
       "2314. 'Orange clownfish protein or transcript stable ID'\n",
       "2315. 'Orange clownfish chromosome/scaffold name'\n",
       "2316. 'Orange clownfish chromosome/scaffold start (bp)'\n",
       "2317. 'Orange clownfish chromosome/scaffold end (bp)'\n",
       "2318. 'Query protein or transcript ID'\n",
       "2319. 'Last common ancestor with Orange clownfish'\n",
       "2320. 'Orange clownfish homology type'\n",
       "2321. '%id. target Orange clownfish gene identical to query gene'\n",
       "2322. '%id. query gene identical to target Orange clownfish gene'\n",
       "2323. 'Orange clownfish Gene-order conservation score'\n",
       "2324. 'Orange clownfish Whole-genome alignment coverage'\n",
       "2325. 'dN with Orange clownfish'\n",
       "2326. 'dS with Orange clownfish'\n",
       "2327. 'Orange clownfish orthology confidence [0 low, 1 high]'\n",
       "2328. 'Orangutan gene stable ID'\n",
       "2329. 'Orangutan gene name'\n",
       "2330. 'Orangutan protein or transcript stable ID'\n",
       "2331. 'Orangutan chromosome/scaffold name'\n",
       "2332. 'Orangutan chromosome/scaffold start (bp)'\n",
       "2333. 'Orangutan chromosome/scaffold end (bp)'\n",
       "2334. 'Query protein or transcript ID'\n",
       "2335. 'Last common ancestor with Orangutan'\n",
       "2336. 'Orangutan homology type'\n",
       "2337. '%id. target Orangutan gene identical to query gene'\n",
       "2338. '%id. query gene identical to target Orangutan gene'\n",
       "2339. 'Orangutan Gene-order conservation score'\n",
       "2340. 'Orangutan Whole-genome alignment coverage'\n",
       "2341. 'dN with Orangutan'\n",
       "2342. 'dS with Orangutan'\n",
       "2343. 'Orangutan orthology confidence [0 low, 1 high]'\n",
       "2344. 'Painted turtle gene stable ID'\n",
       "2345. 'Painted turtle gene name'\n",
       "2346. 'Painted turtle protein or transcript stable ID'\n",
       "2347. 'Painted turtle chromosome/scaffold name'\n",
       "2348. 'Painted turtle chromosome/scaffold start (bp)'\n",
       "2349. 'Painted turtle chromosome/scaffold end (bp)'\n",
       "2350. 'Query protein or transcript ID'\n",
       "2351. 'Last common ancestor with Painted turtle'\n",
       "2352. 'Painted turtle homology type'\n",
       "2353. '%id. target Painted turtle gene identical to query gene'\n",
       "2354. '%id. query gene identical to target Painted turtle gene'\n",
       "2355. 'Painted turtle Gene-order conservation score'\n",
       "2356. 'Painted turtle Whole-genome alignment coverage'\n",
       "2357. 'dN with Painted turtle'\n",
       "2358. 'dS with Painted turtle'\n",
       "2359. 'Painted turtle orthology confidence [0 low, 1 high]'\n",
       "2360. 'Panda gene stable ID'\n",
       "2361. 'Panda gene name'\n",
       "2362. 'Panda protein or transcript stable ID'\n",
       "2363. 'Panda chromosome/scaffold name'\n",
       "2364. 'Panda chromosome/scaffold start (bp)'\n",
       "2365. 'Panda chromosome/scaffold end (bp)'\n",
       "2366. 'Query protein or transcript ID'\n",
       "2367. 'Last common ancestor with Panda'\n",
       "2368. 'Panda homology type'\n",
       "2369. '%id. target Panda gene identical to query gene'\n",
       "2370. '%id. query gene identical to target Panda gene'\n",
       "2371. 'Panda Gene-order conservation score'\n",
       "2372. 'Panda Whole-genome alignment coverage'\n",
       "2373. 'dN with Panda'\n",
       "2374. 'dS with Panda'\n",
       "2375. 'Panda orthology confidence [0 low, 1 high]'\n",
       "2376. 'Paramormyrops kingsleyae gene stable ID'\n",
       "2377. 'Paramormyrops kingsleyae gene name'\n",
       "2378. 'Paramormyrops kingsleyae protein or transcript stable ID'\n",
       "2379. 'Paramormyrops kingsleyae chromosome/scaffold name'\n",
       "2380. 'Paramormyrops kingsleyae chromosome/scaffold start (bp)'\n",
       "2381. 'Paramormyrops kingsleyae chromosome/scaffold end (bp)'\n",
       "2382. 'Query protein or transcript ID'\n",
       "2383. 'Last common ancestor with Paramormyrops kingsleyae'\n",
       "2384. 'Paramormyrops kingsleyae homology type'\n",
       "2385. '%id. target Paramormyrops kingsleyae gene identical to query gene'\n",
       "2386. '%id. query gene identical to target Paramormyrops kingsleyae gene'\n",
       "2387. 'Paramormyrops kingsleyae Gene-order conservation score'\n",
       "2388. 'Paramormyrops kingsleyae Whole-genome alignment coverage'\n",
       "2389. 'dN with Paramormyrops kingsleyae'\n",
       "2390. 'dS with Paramormyrops kingsleyae'\n",
       "2391. 'Paramormyrops kingsleyae orthology confidence [0 low, 1 high]'\n",
       "2392. 'Periophthalmus magnuspinnatus gene stable ID'\n",
       "2393. 'Periophthalmus magnuspinnatus gene name'\n",
       "2394. 'Periophthalmus magnuspinnatus protein or transcript stable ID'\n",
       "2395. 'Periophthalmus magnuspinnatus chromosome/scaffold name'\n",
       "2396. 'Periophthalmus magnuspinnatus chromosome/scaffold start (bp)'\n",
       "2397. 'Periophthalmus magnuspinnatus chromosome/scaffold end (bp)'\n",
       "2398. 'Query protein or transcript ID'\n",
       "2399. 'Last common ancestor with Periophthalmus magnuspinnatus'\n",
       "2400. 'Periophthalmus magnuspinnatus homology type'\n",
       "2401. '%id. target Periophthalmus magnuspinnatus gene identical to query gene'\n",
       "2402. '%id. query gene identical to target Periophthalmus magnuspinnatus gene'\n",
       "2403. 'Periophthalmus magnuspinnatus Gene-order conservation score'\n",
       "2404. 'Periophthalmus magnuspinnatus Whole-genome alignment coverage'\n",
       "2405. 'dN with Periophthalmus magnuspinnatus'\n",
       "2406. 'dS with Periophthalmus magnuspinnatus'\n",
       "2407. 'Periophthalmus magnuspinnatus orthology confidence [0 low, 1 high]'\n",
       "2408. 'Pig gene stable ID'\n",
       "2409. 'Pig gene name'\n",
       "2410. 'Pig protein or transcript stable ID'\n",
       "2411. 'Pig chromosome/scaffold name'\n",
       "2412. 'Pig chromosome/scaffold start (bp)'\n",
       "2413. 'Pig chromosome/scaffold end (bp)'\n",
       "2414. 'Query protein or transcript ID'\n",
       "2415. 'Last common ancestor with Pig'\n",
       "2416. 'Pig homology type'\n",
       "2417. '%id. target Pig gene identical to query gene'\n",
       "2418. '%id. query gene identical to target Pig gene'\n",
       "2419. 'Pig Gene-order conservation score'\n",
       "2420. 'Pig Whole-genome alignment coverage'\n",
       "2421. 'dN with Pig'\n",
       "2422. 'dS with Pig'\n",
       "2423. 'Pig orthology confidence [0 low, 1 high]'\n",
       "2424. 'Pig USMARC gene stable ID'\n",
       "2425. 'Pig USMARC gene name'\n",
       "2426. 'Pig USMARC protein or transcript stable ID'\n",
       "2427. 'Pig USMARC chromosome/scaffold name'\n",
       "2428. 'Pig USMARC chromosome/scaffold start (bp)'\n",
       "2429. 'Pig USMARC chromosome/scaffold end (bp)'\n",
       "2430. 'Query protein or transcript ID'\n",
       "2431. 'Last common ancestor with Pig USMARC'\n",
       "2432. 'Pig USMARC homology type'\n",
       "2433. '%id. target Pig USMARC gene identical to query gene'\n",
       "2434. '%id. query gene identical to target Pig USMARC gene'\n",
       "2435. 'Pig USMARC Gene-order conservation score'\n",
       "2436. 'Pig USMARC Whole-genome alignment coverage'\n",
       "2437. 'dN with Pig USMARC'\n",
       "2438. 'dS with Pig USMARC'\n",
       "2439. 'Pig USMARC orthology confidence [0 low, 1 high]'\n",
       "2440. 'Pig-tailed macaque gene stable ID'\n",
       "2441. 'Pig-tailed macaque gene name'\n",
       "2442. 'Pig-tailed macaque protein or transcript stable ID'\n",
       "2443. 'Pig-tailed macaque chromosome/scaffold name'\n",
       "2444. 'Pig-tailed macaque chromosome/scaffold start (bp)'\n",
       "2445. 'Pig-tailed macaque chromosome/scaffold end (bp)'\n",
       "2446. 'Query protein or transcript ID'\n",
       "2447. 'Last common ancestor with Pig-tailed macaque'\n",
       "2448. 'Pig-tailed macaque homology type'\n",
       "2449. '%id. target Pig-tailed macaque gene identical to query gene'\n",
       "2450. '%id. query gene identical to target Pig-tailed macaque gene'\n",
       "2451. 'Pig-tailed macaque Gene-order conservation score'\n",
       "2452. 'Pig-tailed macaque Whole-genome alignment coverage'\n",
       "2453. 'dN with Pig-tailed macaque'\n",
       "2454. 'dS with Pig-tailed macaque'\n",
       "2455. 'Pig-tailed macaque orthology confidence [0 low, 1 high]'\n",
       "2456. 'Pika gene stable ID'\n",
       "2457. 'Pika gene name'\n",
       "2458. 'Pika protein or transcript stable ID'\n",
       "2459. 'Pika chromosome/scaffold name'\n",
       "2460. 'Pika chromosome/scaffold start (bp)'\n",
       "2461. 'Pika chromosome/scaffold end (bp)'\n",
       "2462. 'Query protein or transcript ID'\n",
       "2463. 'Last common ancestor with Pika'\n",
       "2464. 'Pika homology type'\n",
       "2465. '%id. target Pika gene identical to query gene'\n",
       "2466. '%id. query gene identical to target Pika gene'\n",
       "2467. 'Pika Gene-order conservation score'\n",
       "2468. 'Pika Whole-genome alignment coverage'\n",
       "2469. 'dN with Pika'\n",
       "2470. 'dS with Pika'\n",
       "2471. 'Pika orthology confidence [0 low, 1 high]'\n",
       "2472. 'Pink-footed goose gene stable ID'\n",
       "2473. 'Pink-footed goose gene name'\n",
       "2474. 'Pink-footed goose protein or transcript stable ID'\n",
       "2475. 'Pink-footed goose chromosome/scaffold name'\n",
       "2476. 'Pink-footed goose chromosome/scaffold start (bp)'\n",
       "2477. 'Pink-footed goose chromosome/scaffold end (bp)'\n",
       "2478. 'Query protein or transcript ID'\n",
       "2479. 'Last common ancestor with Pink-footed goose'\n",
       "2480. 'Pink-footed goose homology type'\n",
       "2481. '%id. target Pink-footed goose gene identical to query gene'\n",
       "2482. '%id. query gene identical to target Pink-footed goose gene'\n",
       "2483. 'Pink-footed goose Gene-order conservation score'\n",
       "2484. 'Pink-footed goose Whole-genome alignment coverage'\n",
       "2485. 'dN with Pink-footed goose'\n",
       "2486. 'dS with Pink-footed goose'\n",
       "2487. 'Pink-footed goose orthology confidence [0 low, 1 high]'\n",
       "2488. 'Platyfish gene stable ID'\n",
       "2489. 'Platyfish gene name'\n",
       "2490. 'Platyfish protein or transcript stable ID'\n",
       "2491. 'Platyfish chromosome/scaffold name'\n",
       "2492. 'Platyfish chromosome/scaffold start (bp)'\n",
       "2493. 'Platyfish chromosome/scaffold end (bp)'\n",
       "2494. 'Query protein or transcript ID'\n",
       "2495. 'Last common ancestor with Platyfish'\n",
       "2496. 'Platyfish homology type'\n",
       "2497. '%id. target Platyfish gene identical to query gene'\n",
       "2498. '%id. query gene identical to target Platyfish gene'\n",
       "2499. 'Platyfish Gene-order conservation score'\n",
       "2500. 'Platyfish Whole-genome alignment coverage'\n",
       "2501. 'dN with Platyfish'\n",
       "2502. 'dS with Platyfish'\n",
       "2503. 'Platyfish orthology confidence [0 low, 1 high]'\n",
       "2504. 'Platypus gene stable ID'\n",
       "2505. 'Platypus gene name'\n",
       "2506. 'Platypus protein or transcript stable ID'\n",
       "2507. 'Platypus chromosome/scaffold name'\n",
       "2508. 'Platypus chromosome/scaffold start (bp)'\n",
       "2509. 'Platypus chromosome/scaffold end (bp)'\n",
       "2510. 'Query protein or transcript ID'\n",
       "2511. 'Last common ancestor with Platypus'\n",
       "2512. 'Platypus homology type'\n",
       "2513. '%id. target Platypus gene identical to query gene'\n",
       "2514. '%id. query gene identical to target Platypus gene'\n",
       "2515. 'Platypus Gene-order conservation score'\n",
       "2516. 'Platypus Whole-genome alignment coverage'\n",
       "2517. 'dN with Platypus'\n",
       "2518. 'dS with Platypus'\n",
       "2519. 'Platypus orthology confidence [0 low, 1 high]'\n",
       "2520. 'Polar bear gene stable ID'\n",
       "2521. 'Polar bear gene name'\n",
       "2522. 'Polar bear protein or transcript stable ID'\n",
       "2523. 'Polar bear chromosome/scaffold name'\n",
       "2524. 'Polar bear chromosome/scaffold start (bp)'\n",
       "2525. 'Polar bear chromosome/scaffold end (bp)'\n",
       "2526. 'Query protein or transcript ID'\n",
       "2527. 'Last common ancestor with Polar bear'\n",
       "2528. 'Polar bear homology type'\n",
       "2529. '%id. target Polar bear gene identical to query gene'\n",
       "2530. '%id. query gene identical to target Polar bear gene'\n",
       "2531. 'Polar bear Gene-order conservation score'\n",
       "2532. 'Polar bear Whole-genome alignment coverage'\n",
       "2533. 'dN with Polar bear'\n",
       "2534. 'dS with Polar bear'\n",
       "2535. 'Polar bear orthology confidence [0 low, 1 high]'\n",
       "2536. 'Prairie vole gene stable ID'\n",
       "2537. 'Prairie vole gene name'\n",
       "2538. 'Prairie vole protein or transcript stable ID'\n",
       "2539. 'Prairie vole chromosome/scaffold name'\n",
       "2540. 'Prairie vole chromosome/scaffold start (bp)'\n",
       "2541. 'Prairie vole chromosome/scaffold end (bp)'\n",
       "2542. 'Query protein or transcript ID'\n",
       "2543. 'Last common ancestor with Prairie vole'\n",
       "2544. 'Prairie vole homology type'\n",
       "2545. '%id. target Prairie vole gene identical to query gene'\n",
       "2546. '%id. query gene identical to target Prairie vole gene'\n",
       "2547. 'Prairie vole Gene-order conservation score'\n",
       "2548. 'Prairie vole Whole-genome alignment coverage'\n",
       "2549. 'dN with Prairie vole'\n",
       "2550. 'dS with Prairie vole'\n",
       "2551. 'Prairie vole orthology confidence [0 low, 1 high]'\n",
       "2552. 'Rabbit gene stable ID'\n",
       "2553. 'Rabbit gene name'\n",
       "2554. 'Rabbit protein or transcript stable ID'\n",
       "2555. 'Rabbit chromosome/scaffold name'\n",
       "2556. 'Rabbit chromosome/scaffold start (bp)'\n",
       "2557. 'Rabbit chromosome/scaffold end (bp)'\n",
       "2558. 'Query protein or transcript ID'\n",
       "2559. 'Last common ancestor with Rabbit'\n",
       "2560. 'Rabbit homology type'\n",
       "2561. '%id. target Rabbit gene identical to query gene'\n",
       "2562. '%id. query gene identical to target Rabbit gene'\n",
       "2563. 'Rabbit Gene-order conservation score'\n",
       "2564. 'Rabbit Whole-genome alignment coverage'\n",
       "2565. 'dN with Rabbit'\n",
       "2566. 'dS with Rabbit'\n",
       "2567. 'Rabbit orthology confidence [0 low, 1 high]'\n",
       "2568. 'Rat gene stable ID'\n",
       "2569. 'Rat gene name'\n",
       "2570. 'Rat protein or transcript stable ID'\n",
       "2571. 'Rat chromosome/scaffold name'\n",
       "2572. 'Rat chromosome/scaffold start (bp)'\n",
       "2573. 'Rat chromosome/scaffold end (bp)'\n",
       "2574. 'Query protein or transcript ID'\n",
       "2575. 'Last common ancestor with Rat'\n",
       "2576. 'Rat homology type'\n",
       "2577. '%id. target Rat gene identical to query gene'\n",
       "2578. '%id. query gene identical to target Rat gene'\n",
       "2579. 'Rat Gene-order conservation score'\n",
       "2580. 'Rat Whole-genome alignment coverage'\n",
       "2581. 'dN with Rat'\n",
       "2582. 'dS with Rat'\n",
       "2583. 'Rat orthology confidence [0 low, 1 high]'\n",
       "2584. 'Red fox gene stable ID'\n",
       "2585. 'Red fox gene name'\n",
       "2586. 'Red fox protein or transcript stable ID'\n",
       "2587. 'Red fox chromosome/scaffold name'\n",
       "2588. 'Red fox chromosome/scaffold start (bp)'\n",
       "2589. 'Red fox chromosome/scaffold end (bp)'\n",
       "2590. 'Query protein or transcript ID'\n",
       "2591. 'Last common ancestor with Red fox'\n",
       "2592. 'Red fox homology type'\n",
       "2593. '%id. target Red fox gene identical to query gene'\n",
       "2594. '%id. query gene identical to target Red fox gene'\n",
       "2595. 'Red fox Gene-order conservation score'\n",
       "2596. 'Red fox Whole-genome alignment coverage'\n",
       "2597. 'dN with Red fox'\n",
       "2598. 'dS with Red fox'\n",
       "2599. 'Red fox orthology confidence [0 low, 1 high]'\n",
       "2600. 'Red-bellied piranha gene stable ID'\n",
       "2601. 'Red-bellied piranha gene name'\n",
       "2602. 'Red-bellied piranha protein or transcript stable ID'\n",
       "2603. 'Red-bellied piranha chromosome/scaffold name'\n",
       "2604. 'Red-bellied piranha chromosome/scaffold start (bp)'\n",
       "2605. 'Red-bellied piranha chromosome/scaffold end (bp)'\n",
       "2606. 'Query protein or transcript ID'\n",
       "2607. 'Last common ancestor with Red-bellied piranha'\n",
       "2608. 'Red-bellied piranha homology type'\n",
       "2609. '%id. target Red-bellied piranha gene identical to query gene'\n",
       "2610. '%id. query gene identical to target Red-bellied piranha gene'\n",
       "2611. 'Red-bellied piranha Gene-order conservation score'\n",
       "2612. 'Red-bellied piranha Whole-genome alignment coverage'\n",
       "2613. 'dN with Red-bellied piranha'\n",
       "2614. 'dS with Red-bellied piranha'\n",
       "2615. 'Red-bellied piranha orthology confidence [0 low, 1 high]'\n",
       "2616. 'Ruff gene stable ID'\n",
       "2617. 'Ruff gene name'\n",
       "2618. 'Ruff protein or transcript stable ID'\n",
       "2619. 'Ruff chromosome/scaffold name'\n",
       "2620. 'Ruff chromosome/scaffold start (bp)'\n",
       "2621. 'Ruff chromosome/scaffold end (bp)'\n",
       "2622. 'Query protein or transcript ID'\n",
       "2623. 'Last common ancestor with Ruff'\n",
       "2624. 'Ruff homology type'\n",
       "2625. '%id. target Ruff gene identical to query gene'\n",
       "2626. '%id. query gene identical to target Ruff gene'\n",
       "2627. 'Ruff Gene-order conservation score'\n",
       "2628. 'Ruff Whole-genome alignment coverage'\n",
       "2629. 'dN with Ruff'\n",
       "2630. 'dS with Ruff'\n",
       "2631. 'Ruff orthology confidence [0 low, 1 high]'\n",
       "2632. 'Ryukyu mouse gene stable ID'\n",
       "2633. 'Ryukyu mouse gene name'\n",
       "2634. 'Ryukyu mouse protein or transcript stable ID'\n",
       "2635. 'Ryukyu mouse chromosome/scaffold name'\n",
       "2636. 'Ryukyu mouse chromosome/scaffold start (bp)'\n",
       "2637. 'Ryukyu mouse chromosome/scaffold end (bp)'\n",
       "2638. 'Query protein or transcript ID'\n",
       "2639. 'Last common ancestor with Ryukyu mouse'\n",
       "2640. 'Ryukyu mouse homology type'\n",
       "2641. '%id. target Ryukyu mouse gene identical to query gene'\n",
       "2642. '%id. query gene identical to target Ryukyu mouse gene'\n",
       "2643. 'Ryukyu mouse Gene-order conservation score'\n",
       "2644. 'Ryukyu mouse Whole-genome alignment coverage'\n",
       "2645. 'dN with Ryukyu mouse'\n",
       "2646. 'dS with Ryukyu mouse'\n",
       "2647. 'Ryukyu mouse orthology confidence [0 low, 1 high]'\n",
       "2648. 'Saccharomyces cerevisiae gene stable ID'\n",
       "2649. 'Saccharomyces cerevisiae gene name'\n",
       "2650. 'Saccharomyces cerevisiae protein or transcript stable ID'\n",
       "2651. 'Saccharomyces cerevisiae chromosome/scaffold name'\n",
       "2652. 'Saccharomyces cerevisiae chromosome/scaffold start (bp)'\n",
       "2653. 'Saccharomyces cerevisiae chromosome/scaffold end (bp)'\n",
       "2654. 'Query protein or transcript ID'\n",
       "2655. 'Last common ancestor with Saccharomyces cerevisiae'\n",
       "2656. 'Saccharomyces cerevisiae homology type'\n",
       "2657. '%id. target Saccharomyces cerevisiae gene identical to query gene'\n",
       "2658. '%id. query gene identical to target Saccharomyces cerevisiae gene'\n",
       "2659. 'Saccharomyces cerevisiae Gene-order conservation score'\n",
       "2660. 'Saccharomyces cerevisiae Whole-genome alignment coverage'\n",
       "2661. 'dN with Saccharomyces cerevisiae'\n",
       "2662. 'dS with Saccharomyces cerevisiae'\n",
       "2663. 'Saccharomyces cerevisiae orthology confidence [0 low, 1 high]'\n",
       "2664. 'Sailfin molly gene stable ID'\n",
       "2665. 'Sailfin molly gene name'\n",
       "2666. 'Sailfin molly protein or transcript stable ID'\n",
       "2667. 'Sailfin molly chromosome/scaffold name'\n",
       "2668. 'Sailfin molly chromosome/scaffold start (bp)'\n",
       "2669. 'Sailfin molly chromosome/scaffold end (bp)'\n",
       "2670. 'Query protein or transcript ID'\n",
       "2671. 'Last common ancestor with Sailfin molly'\n",
       "2672. 'Sailfin molly homology type'\n",
       "2673. '%id. target Sailfin molly gene identical to query gene'\n",
       "2674. '%id. query gene identical to target Sailfin molly gene'\n",
       "2675. 'Sailfin molly Gene-order conservation score'\n",
       "2676. 'Sailfin molly Whole-genome alignment coverage'\n",
       "2677. 'dN with Sailfin molly'\n",
       "2678. 'dS with Sailfin molly'\n",
       "2679. 'Sailfin molly orthology confidence [0 low, 1 high]'\n",
       "2680. 'Sheep gene stable ID'\n",
       "2681. 'Sheep gene name'\n",
       "2682. 'Sheep protein or transcript stable ID'\n",
       "2683. 'Sheep chromosome/scaffold name'\n",
       "2684. 'Sheep chromosome/scaffold start (bp)'\n",
       "2685. 'Sheep chromosome/scaffold end (bp)'\n",
       "2686. 'Query protein or transcript ID'\n",
       "2687. 'Last common ancestor with Sheep'\n",
       "2688. 'Sheep homology type'\n",
       "2689. '%id. target Sheep gene identical to query gene'\n",
       "2690. '%id. query gene identical to target Sheep gene'\n",
       "2691. 'Sheep Gene-order conservation score'\n",
       "2692. 'Sheep Whole-genome alignment coverage'\n",
       "2693. 'dN with Sheep'\n",
       "2694. 'dS with Sheep'\n",
       "2695. 'Sheep orthology confidence [0 low, 1 high]'\n",
       "2696. 'Sheepshead minnow gene stable ID'\n",
       "2697. 'Sheepshead minnow gene name'\n",
       "2698. 'Sheepshead minnow protein or transcript stable ID'\n",
       "2699. 'Sheepshead minnow chromosome/scaffold name'\n",
       "2700. 'Sheepshead minnow chromosome/scaffold start (bp)'\n",
       "2701. 'Sheepshead minnow chromosome/scaffold end (bp)'\n",
       "2702. 'Query protein or transcript ID'\n",
       "2703. 'Last common ancestor with Sheepshead minnow'\n",
       "2704. 'Sheepshead minnow homology type'\n",
       "2705. '%id. target Sheepshead minnow gene identical to query gene'\n",
       "2706. '%id. query gene identical to target Sheepshead minnow gene'\n",
       "2707. 'Sheepshead minnow Gene-order conservation score'\n",
       "2708. 'Sheepshead minnow Whole-genome alignment coverage'\n",
       "2709. 'dN with Sheepshead minnow'\n",
       "2710. 'dS with Sheepshead minnow'\n",
       "2711. 'Sheepshead minnow orthology confidence [0 low, 1 high]'\n",
       "2712. 'Shortfin molly gene stable ID'\n",
       "2713. 'Shortfin molly gene name'\n",
       "2714. 'Shortfin molly protein or transcript stable ID'\n",
       "2715. 'Shortfin molly chromosome/scaffold name'\n",
       "2716. 'Shortfin molly chromosome/scaffold start (bp)'\n",
       "2717. 'Shortfin molly chromosome/scaffold end (bp)'\n",
       "2718. 'Query protein or transcript ID'\n",
       "2719. 'Last common ancestor with Shortfin molly'\n",
       "2720. 'Shortfin molly homology type'\n",
       "2721. '%id. target Shortfin molly gene identical to query gene'\n",
       "2722. '%id. query gene identical to target Shortfin molly gene'\n",
       "2723. 'Shortfin molly Gene-order conservation score'\n",
       "2724. 'Shortfin molly Whole-genome alignment coverage'\n",
       "2725. 'dN with Shortfin molly'\n",
       "2726. 'dS with Shortfin molly'\n",
       "2727. 'Shortfin molly orthology confidence [0 low, 1 high]'\n",
       "2728. 'Shrew gene stable ID'\n",
       "2729. 'Shrew gene name'\n",
       "2730. 'Shrew protein or transcript stable ID'\n",
       "2731. 'Shrew chromosome/scaffold name'\n",
       "2732. 'Shrew chromosome/scaffold start (bp)'\n",
       "2733. 'Shrew chromosome/scaffold end (bp)'\n",
       "2734. 'Query protein or transcript ID'\n",
       "2735. 'Last common ancestor with Shrew'\n",
       "2736. 'Shrew homology type'\n",
       "2737. '%id. target Shrew gene identical to query gene'\n",
       "2738. '%id. query gene identical to target Shrew gene'\n",
       "2739. 'Shrew Gene-order conservation score'\n",
       "2740. 'Shrew Whole-genome alignment coverage'\n",
       "2741. 'dN with Shrew'\n",
       "2742. 'dS with Shrew'\n",
       "2743. 'Shrew orthology confidence [0 low, 1 high]'\n",
       "2744. 'Shrew mouse gene stable ID'\n",
       "2745. 'Shrew mouse gene name'\n",
       "2746. 'Shrew mouse protein or transcript stable ID'\n",
       "2747. 'Shrew mouse chromosome/scaffold name'\n",
       "2748. 'Shrew mouse chromosome/scaffold start (bp)'\n",
       "2749. 'Shrew mouse chromosome/scaffold end (bp)'\n",
       "2750. 'Query protein or transcript ID'\n",
       "2751. 'Last common ancestor with Shrew mouse'\n",
       "2752. 'Shrew mouse homology type'\n",
       "2753. '%id. target Shrew mouse gene identical to query gene'\n",
       "2754. '%id. query gene identical to target Shrew mouse gene'\n",
       "2755. 'Shrew mouse Gene-order conservation score'\n",
       "2756. 'Shrew mouse Whole-genome alignment coverage'\n",
       "2757. 'dN with Shrew mouse'\n",
       "2758. 'dS with Shrew mouse'\n",
       "2759. 'Shrew mouse orthology confidence [0 low, 1 high]'\n",
       "2760. 'Sloth gene stable ID'\n",
       "2761. 'Sloth gene name'\n",
       "2762. 'Sloth protein or transcript stable ID'\n",
       "2763. 'Sloth chromosome/scaffold name'\n",
       "2764. 'Sloth chromosome/scaffold start (bp)'\n",
       "2765. 'Sloth chromosome/scaffold end (bp)'\n",
       "2766. 'Query protein or transcript ID'\n",
       "2767. 'Last common ancestor with Sloth'\n",
       "2768. 'Sloth homology type'\n",
       "2769. '%id. target Sloth gene identical to query gene'\n",
       "2770. '%id. query gene identical to target Sloth gene'\n",
       "2771. 'Sloth Gene-order conservation score'\n",
       "2772. 'Sloth Whole-genome alignment coverage'\n",
       "2773. 'dN with Sloth'\n",
       "2774. 'dS with Sloth'\n",
       "2775. 'Sloth orthology confidence [0 low, 1 high]'\n",
       "2776. 'Sooty mangabey gene stable ID'\n",
       "2777. 'Sooty mangabey gene name'\n",
       "2778. 'Sooty mangabey protein or transcript stable ID'\n",
       "2779. 'Sooty mangabey chromosome/scaffold name'\n",
       "2780. 'Sooty mangabey chromosome/scaffold start (bp)'\n",
       "2781. 'Sooty mangabey chromosome/scaffold end (bp)'\n",
       "2782. 'Query protein or transcript ID'\n",
       "2783. 'Last common ancestor with Sooty mangabey'\n",
       "2784. 'Sooty mangabey homology type'\n",
       "2785. '%id. target Sooty mangabey gene identical to query gene'\n",
       "2786. '%id. query gene identical to target Sooty mangabey gene'\n",
       "2787. 'Sooty mangabey Gene-order conservation score'\n",
       "2788. 'Sooty mangabey Whole-genome alignment coverage'\n",
       "2789. 'dN with Sooty mangabey'\n",
       "2790. 'dS with Sooty mangabey'\n",
       "2791. 'Sooty mangabey orthology confidence [0 low, 1 high]'\n",
       "2792. 'Spiny chromis gene stable ID'\n",
       "2793. 'Spiny chromis gene name'\n",
       "2794. 'Spiny chromis protein or transcript stable ID'\n",
       "2795. 'Spiny chromis chromosome/scaffold name'\n",
       "2796. 'Spiny chromis chromosome/scaffold start (bp)'\n",
       "2797. 'Spiny chromis chromosome/scaffold end (bp)'\n",
       "2798. 'Query protein or transcript ID'\n",
       "2799. 'Last common ancestor with Spiny chromis'\n",
       "2800. 'Spiny chromis homology type'\n",
       "2801. '%id. target Spiny chromis gene identical to query gene'\n",
       "2802. '%id. query gene identical to target Spiny chromis gene'\n",
       "2803. 'Spiny chromis Gene-order conservation score'\n",
       "2804. 'Spiny chromis Whole-genome alignment coverage'\n",
       "2805. 'dN with Spiny chromis'\n",
       "2806. 'dS with Spiny chromis'\n",
       "2807. 'Spiny chromis orthology confidence [0 low, 1 high]'\n",
       "2808. 'Spoon-billed sandpiper gene stable ID'\n",
       "2809. 'Spoon-billed sandpiper gene name'\n",
       "2810. 'Spoon-billed sandpiper protein or transcript stable ID'\n",
       "2811. 'Spoon-billed sandpiper chromosome/scaffold name'\n",
       "2812. 'Spoon-billed sandpiper chromosome/scaffold start (bp)'\n",
       "2813. 'Spoon-billed sandpiper chromosome/scaffold end (bp)'\n",
       "2814. 'Query protein or transcript ID'\n",
       "2815. 'Last common ancestor with Spoon-billed sandpiper'\n",
       "2816. 'Spoon-billed sandpiper homology type'\n",
       "2817. '%id. target Spoon-billed sandpiper gene identical to query gene'\n",
       "2818. '%id. query gene identical to target Spoon-billed sandpiper gene'\n",
       "2819. 'Spoon-billed sandpiper Gene-order conservation score'\n",
       "2820. 'Spoon-billed sandpiper Whole-genome alignment coverage'\n",
       "2821. 'dN with Spoon-billed sandpiper'\n",
       "2822. 'dS with Spoon-billed sandpiper'\n",
       "2823. 'Spoon-billed sandpiper orthology confidence [0 low, 1 high]'\n",
       "2824. 'Spotted gar gene stable ID'\n",
       "2825. 'Spotted gar gene name'\n",
       "2826. 'Spotted gar protein or transcript stable ID'\n",
       "2827. 'Spotted gar chromosome/scaffold name'\n",
       "2828. 'Spotted gar chromosome/scaffold start (bp)'\n",
       "2829. 'Spotted gar chromosome/scaffold end (bp)'\n",
       "2830. 'Query protein or transcript ID'\n",
       "2831. 'Last common ancestor with Spotted gar'\n",
       "2832. 'Spotted gar homology type'\n",
       "2833. '%id. target Spotted gar gene identical to query gene'\n",
       "2834. '%id. query gene identical to target Spotted gar gene'\n",
       "2835. 'Spotted gar Gene-order conservation score'\n",
       "2836. 'Spotted gar Whole-genome alignment coverage'\n",
       "2837. 'dN with Spotted gar'\n",
       "2838. 'dS with Spotted gar'\n",
       "2839. 'Spotted gar orthology confidence [0 low, 1 high]'\n",
       "2840. 'Squirrel gene stable ID'\n",
       "2841. 'Squirrel gene name'\n",
       "2842. 'Squirrel protein or transcript stable ID'\n",
       "2843. 'Squirrel chromosome/scaffold name'\n",
       "2844. 'Squirrel chromosome/scaffold start (bp)'\n",
       "2845. 'Squirrel chromosome/scaffold end (bp)'\n",
       "2846. 'Query protein or transcript ID'\n",
       "2847. 'Last common ancestor with Squirrel'\n",
       "2848. 'Squirrel homology type'\n",
       "2849. '%id. target Squirrel gene identical to query gene'\n",
       "2850. '%id. query gene identical to target Squirrel gene'\n",
       "2851. 'Squirrel Gene-order conservation score'\n",
       "2852. 'Squirrel Whole-genome alignment coverage'\n",
       "2853. 'dN with Squirrel'\n",
       "2854. 'dS with Squirrel'\n",
       "2855. 'Squirrel orthology confidence [0 low, 1 high]'\n",
       "2856. 'Steppe mouse gene stable ID'\n",
       "2857. 'Steppe mouse gene name'\n",
       "2858. 'Steppe mouse protein or transcript stable ID'\n",
       "2859. 'Steppe mouse chromosome/scaffold name'\n",
       "2860. 'Steppe mouse chromosome/scaffold start (bp)'\n",
       "2861. 'Steppe mouse chromosome/scaffold end (bp)'\n",
       "2862. 'Query protein or transcript ID'\n",
       "2863. 'Last common ancestor with Steppe mouse'\n",
       "2864. 'Steppe mouse homology type'\n",
       "2865. '%id. target Steppe mouse gene identical to query gene'\n",
       "2866. '%id. query gene identical to target Steppe mouse gene'\n",
       "2867. 'Steppe mouse Gene-order conservation score'\n",
       "2868. 'Steppe mouse Whole-genome alignment coverage'\n",
       "2869. 'dN with Steppe mouse'\n",
       "2870. 'dS with Steppe mouse'\n",
       "2871. 'Steppe mouse orthology confidence [0 low, 1 high]'\n",
       "2872. 'Stickleback gene stable ID'\n",
       "2873. 'Stickleback gene name'\n",
       "2874. 'Stickleback protein or transcript stable ID'\n",
       "2875. 'Stickleback chromosome/scaffold name'\n",
       "2876. 'Stickleback chromosome/scaffold start (bp)'\n",
       "2877. 'Stickleback chromosome/scaffold end (bp)'\n",
       "2878. 'Query protein or transcript ID'\n",
       "2879. 'Last common ancestor with Stickleback'\n",
       "2880. 'Stickleback homology type'\n",
       "2881. '%id. target Stickleback gene identical to query gene'\n",
       "2882. '%id. query gene identical to target Stickleback gene'\n",
       "2883. 'Stickleback Gene-order conservation score'\n",
       "2884. 'Stickleback Whole-genome alignment coverage'\n",
       "2885. 'dN with Stickleback'\n",
       "2886. 'dS with Stickleback'\n",
       "2887. 'Stickleback orthology confidence [0 low, 1 high]'\n",
       "2888. 'Swamp eel gene stable ID'\n",
       "2889. 'Swamp eel gene name'\n",
       "2890. 'Swamp eel protein or transcript stable ID'\n",
       "2891. 'Swamp eel chromosome/scaffold name'\n",
       "2892. 'Swamp eel chromosome/scaffold start (bp)'\n",
       "2893. 'Swamp eel chromosome/scaffold end (bp)'\n",
       "2894. 'Query protein or transcript ID'\n",
       "2895. 'Last common ancestor with Swamp eel'\n",
       "2896. 'Swamp eel homology type'\n",
       "2897. '%id. target Swamp eel gene identical to query gene'\n",
       "2898. '%id. query gene identical to target Swamp eel gene'\n",
       "2899. 'Swamp eel Gene-order conservation score'\n",
       "2900. 'Swamp eel Whole-genome alignment coverage'\n",
       "2901. 'dN with Swamp eel'\n",
       "2902. 'dS with Swamp eel'\n",
       "2903. 'Swamp eel orthology confidence [0 low, 1 high]'\n",
       "2904. 'Tarsier gene stable ID'\n",
       "2905. 'Tarsier gene name'\n",
       "2906. 'Tarsier protein or transcript stable ID'\n",
       "2907. 'Tarsier chromosome/scaffold name'\n",
       "2908. 'Tarsier chromosome/scaffold start (bp)'\n",
       "2909. 'Tarsier chromosome/scaffold end (bp)'\n",
       "2910. 'Query protein or transcript ID'\n",
       "2911. 'Last common ancestor with Tarsier'\n",
       "2912. 'Tarsier homology type'\n",
       "2913. '%id. target Tarsier gene identical to query gene'\n",
       "2914. '%id. query gene identical to target Tarsier gene'\n",
       "2915. 'Tarsier Gene-order conservation score'\n",
       "2916. 'Tarsier Whole-genome alignment coverage'\n",
       "2917. 'dN with Tarsier'\n",
       "2918. 'dS with Tarsier'\n",
       "2919. 'Tarsier orthology confidence [0 low, 1 high]'\n",
       "2920. 'Tasmanian devil gene stable ID'\n",
       "2921. 'Tasmanian devil gene name'\n",
       "2922. 'Tasmanian devil protein or transcript stable ID'\n",
       "2923. 'Tasmanian devil chromosome/scaffold name'\n",
       "2924. 'Tasmanian devil chromosome/scaffold start (bp)'\n",
       "2925. 'Tasmanian devil chromosome/scaffold end (bp)'\n",
       "2926. 'Query protein or transcript ID'\n",
       "2927. 'Last common ancestor with Tasmanian devil'\n",
       "2928. 'Tasmanian devil homology type'\n",
       "2929. '%id. target Tasmanian devil gene identical to query gene'\n",
       "2930. '%id. query gene identical to target Tasmanian devil gene'\n",
       "2931. 'Tasmanian devil Gene-order conservation score'\n",
       "2932. 'Tasmanian devil Whole-genome alignment coverage'\n",
       "2933. 'dN with Tasmanian devil'\n",
       "2934. 'dS with Tasmanian devil'\n",
       "2935. 'Tasmanian devil orthology confidence [0 low, 1 high]'\n",
       "2936. 'Tetraodon gene stable ID'\n",
       "2937. 'Tetraodon gene name'\n",
       "2938. 'Tetraodon protein or transcript stable ID'\n",
       "2939. 'Tetraodon chromosome/scaffold name'\n",
       "2940. 'Tetraodon chromosome/scaffold start (bp)'\n",
       "2941. 'Tetraodon chromosome/scaffold end (bp)'\n",
       "2942. 'Query protein or transcript ID'\n",
       "2943. 'Last common ancestor with Tetraodon'\n",
       "2944. 'Tetraodon homology type'\n",
       "2945. '%id. target Tetraodon gene identical to query gene'\n",
       "2946. '%id. query gene identical to target Tetraodon gene'\n",
       "2947. 'Tetraodon Gene-order conservation score'\n",
       "2948. 'Tetraodon Whole-genome alignment coverage'\n",
       "2949. 'dN with Tetraodon'\n",
       "2950. 'dS with Tetraodon'\n",
       "2951. 'Tetraodon orthology confidence [0 low, 1 high]'\n",
       "2952. 'Tiger gene stable ID'\n",
       "2953. 'Tiger gene name'\n",
       "2954. 'Tiger protein or transcript stable ID'\n",
       "2955. 'Tiger chromosome/scaffold name'\n",
       "2956. 'Tiger chromosome/scaffold start (bp)'\n",
       "2957. 'Tiger chromosome/scaffold end (bp)'\n",
       "2958. 'Query protein or transcript ID'\n",
       "2959. 'Last common ancestor with Tiger'\n",
       "2960. 'Tiger homology type'\n",
       "2961. '%id. target Tiger gene identical to query gene'\n",
       "2962. '%id. query gene identical to target Tiger gene'\n",
       "2963. 'Tiger Gene-order conservation score'\n",
       "2964. 'Tiger Whole-genome alignment coverage'\n",
       "2965. 'dN with Tiger'\n",
       "2966. 'dS with Tiger'\n",
       "2967. 'Tiger orthology confidence [0 low, 1 high]'\n",
       "2968. 'Tiger tail seahorse gene stable ID'\n",
       "2969. 'Tiger tail seahorse gene name'\n",
       "2970. 'Tiger tail seahorse protein or transcript stable ID'\n",
       "2971. 'Tiger tail seahorse chromosome/scaffold name'\n",
       "2972. 'Tiger tail seahorse chromosome/scaffold start (bp)'\n",
       "2973. 'Tiger tail seahorse chromosome/scaffold end (bp)'\n",
       "2974. 'Query protein or transcript ID'\n",
       "2975. 'Last common ancestor with Tiger tail seahorse'\n",
       "2976. 'Tiger tail seahorse homology type'\n",
       "2977. '%id. target Tiger tail seahorse gene identical to query gene'\n",
       "2978. '%id. query gene identical to target Tiger tail seahorse gene'\n",
       "2979. 'Tiger tail seahorse Gene-order conservation score'\n",
       "2980. 'Tiger tail seahorse Whole-genome alignment coverage'\n",
       "2981. 'dN with Tiger tail seahorse'\n",
       "2982. 'dS with Tiger tail seahorse'\n",
       "2983. 'Tiger tail seahorse orthology confidence [0 low, 1 high]'\n",
       "2984. 'Tilapia gene stable ID'\n",
       "2985. 'Tilapia gene name'\n",
       "2986. 'Tilapia protein or transcript stable ID'\n",
       "2987. 'Tilapia chromosome/scaffold name'\n",
       "2988. 'Tilapia chromosome/scaffold start (bp)'\n",
       "2989. 'Tilapia chromosome/scaffold end (bp)'\n",
       "2990. 'Query protein or transcript ID'\n",
       "2991. 'Last common ancestor with Tilapia'\n",
       "2992. 'Tilapia homology type'\n",
       "2993. '%id. target Tilapia gene identical to query gene'\n",
       "2994. '%id. query gene identical to target Tilapia gene'\n",
       "2995. 'Tilapia Gene-order conservation score'\n",
       "2996. 'Tilapia Whole-genome alignment coverage'\n",
       "2997. 'dN with Tilapia'\n",
       "2998. 'dS with Tilapia'\n",
       "2999. 'Tilapia orthology confidence [0 low, 1 high]'\n",
       "3000. 'Tongue sole gene stable ID'\n",
       "3001. 'Tongue sole gene name'\n",
       "3002. 'Tongue sole protein or transcript stable ID'\n",
       "3003. 'Tongue sole chromosome/scaffold name'\n",
       "3004. 'Tongue sole chromosome/scaffold start (bp)'\n",
       "3005. 'Tongue sole chromosome/scaffold end (bp)'\n",
       "3006. 'Query protein or transcript ID'\n",
       "3007. 'Last common ancestor with Tongue sole'\n",
       "3008. 'Tongue sole homology type'\n",
       "3009. '%id. target Tongue sole gene identical to query gene'\n",
       "3010. '%id. query gene identical to target Tongue sole gene'\n",
       "3011. 'Tongue sole Gene-order conservation score'\n",
       "3012. 'Tongue sole Whole-genome alignment coverage'\n",
       "3013. 'dN with Tongue sole'\n",
       "3014. 'dS with Tongue sole'\n",
       "3015. 'Tongue sole orthology confidence [0 low, 1 high]'\n",
       "3016. 'Tree Shrew gene stable ID'\n",
       "3017. 'Tree Shrew gene name'\n",
       "3018. 'Tree Shrew protein or transcript stable ID'\n",
       "3019. 'Tree Shrew chromosome/scaffold name'\n",
       "3020. 'Tree Shrew chromosome/scaffold start (bp)'\n",
       "3021. 'Tree Shrew chromosome/scaffold end (bp)'\n",
       "3022. 'Query protein or transcript ID'\n",
       "3023. 'Last common ancestor with Tree Shrew'\n",
       "3024. 'Tree Shrew homology type'\n",
       "3025. '%id. target Tree Shrew gene identical to query gene'\n",
       "3026. '%id. query gene identical to target Tree Shrew gene'\n",
       "3027. 'Tree Shrew Gene-order conservation score'\n",
       "3028. 'Tree Shrew Whole-genome alignment coverage'\n",
       "3029. 'dN with Tree Shrew'\n",
       "3030. 'dS with Tree Shrew'\n",
       "3031. 'Tree Shrew orthology confidence [0 low, 1 high]'\n",
       "3032. 'Tuatara gene stable ID'\n",
       "3033. 'Tuatara gene name'\n",
       "3034. 'Tuatara protein or transcript stable ID'\n",
       "3035. 'Tuatara chromosome/scaffold name'\n",
       "3036. 'Tuatara chromosome/scaffold start (bp)'\n",
       "3037. 'Tuatara chromosome/scaffold end (bp)'\n",
       "3038. 'Query protein or transcript ID'\n",
       "3039. 'Last common ancestor with Tuatara'\n",
       "3040. 'Tuatara homology type'\n",
       "3041. '%id. target Tuatara gene identical to query gene'\n",
       "3042. '%id. query gene identical to target Tuatara gene'\n",
       "3043. 'Tuatara Gene-order conservation score'\n",
       "3044. 'Tuatara Whole-genome alignment coverage'\n",
       "3045. 'dN with Tuatara'\n",
       "3046. 'dS with Tuatara'\n",
       "3047. 'Tuatara orthology confidence [0 low, 1 high]'\n",
       "3048. 'Turbot gene stable ID'\n",
       "3049. 'Turbot gene name'\n",
       "3050. 'Turbot protein or transcript stable ID'\n",
       "3051. 'Turbot chromosome/scaffold name'\n",
       "3052. 'Turbot chromosome/scaffold start (bp)'\n",
       "3053. 'Turbot chromosome/scaffold end (bp)'\n",
       "3054. 'Query protein or transcript ID'\n",
       "3055. 'Last common ancestor with Turbot'\n",
       "3056. 'Turbot homology type'\n",
       "3057. '%id. target Turbot gene identical to query gene'\n",
       "3058. '%id. query gene identical to target Turbot gene'\n",
       "3059. 'Turbot Gene-order conservation score'\n",
       "3060. 'Turbot Whole-genome alignment coverage'\n",
       "3061. 'dN with Turbot'\n",
       "3062. 'dS with Turbot'\n",
       "3063. 'Turbot orthology confidence [0 low, 1 high]'\n",
       "3064. 'Turkey gene stable ID'\n",
       "3065. 'Turkey gene name'\n",
       "3066. 'Turkey protein or transcript stable ID'\n",
       "3067. 'Turkey chromosome/scaffold name'\n",
       "3068. 'Turkey chromosome/scaffold start (bp)'\n",
       "3069. 'Turkey chromosome/scaffold end (bp)'\n",
       "3070. 'Query protein or transcript ID'\n",
       "3071. 'Last common ancestor with Turkey'\n",
       "3072. 'Turkey homology type'\n",
       "3073. '%id. target Turkey gene identical to query gene'\n",
       "3074. '%id. query gene identical to target Turkey gene'\n",
       "3075. 'Turkey Gene-order conservation score'\n",
       "3076. 'Turkey Whole-genome alignment coverage'\n",
       "3077. 'dN with Turkey'\n",
       "3078. 'dS with Turkey'\n",
       "3079. 'Turkey orthology confidence [0 low, 1 high]'\n",
       "3080. 'Ugandan red Colobus gene stable ID'\n",
       "3081. 'Ugandan red Colobus gene name'\n",
       "3082. 'Ugandan red Colobus protein or transcript stable ID'\n",
       "3083. 'Ugandan red Colobus chromosome/scaffold name'\n",
       "3084. 'Ugandan red Colobus chromosome/scaffold start (bp)'\n",
       "3085. 'Ugandan red Colobus chromosome/scaffold end (bp)'\n",
       "3086. 'Query protein or transcript ID'\n",
       "3087. 'Last common ancestor with Ugandan red Colobus'\n",
       "3088. 'Ugandan red Colobus homology type'\n",
       "3089. '%id. target Ugandan red Colobus gene identical to query gene'\n",
       "3090. '%id. query gene identical to target Ugandan red Colobus gene'\n",
       "3091. 'Ugandan red Colobus Gene-order conservation score'\n",
       "3092. 'Ugandan red Colobus Whole-genome alignment coverage'\n",
       "3093. 'dN with Ugandan red Colobus'\n",
       "3094. 'dS with Ugandan red Colobus'\n",
       "3095. 'Ugandan red Colobus orthology confidence [0 low, 1 high]'\n",
       "3096. 'Upper Galilee mountains blind mole rat gene stable ID'\n",
       "3097. 'Upper Galilee mountains blind mole rat gene name'\n",
       "3098. 'Upper Galilee mountains blind mole rat protein or transcript stable ID'\n",
       "3099. 'Upper Galilee mountains blind mole rat chromosome/scaffold name'\n",
       "3100. 'Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)'\n",
       "3101. 'Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)'\n",
       "3102. 'Query protein or transcript ID'\n",
       "3103. 'Last common ancestor with Upper Galilee mountains blind mole rat'\n",
       "3104. 'Upper Galilee mountains blind mole rat homology type'\n",
       "3105. '%id. target Upper Galilee mountains blind mole rat gene identical to query gene'\n",
       "3106. '%id. query gene identical to target Upper Galilee mountains blind mole rat gene'\n",
       "3107. 'Upper Galilee mountains blind mole rat Gene-order conservation score'\n",
       "3108. 'Upper Galilee mountains blind mole rat Whole-genome alignment coverage'\n",
       "3109. 'dN with Upper Galilee mountains blind mole rat'\n",
       "3110. 'dS with Upper Galilee mountains blind mole rat'\n",
       "3111. 'Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]'\n",
       "3112. 'Vervet-AGM gene stable ID'\n",
       "3113. 'Vervet-AGM gene name'\n",
       "3114. 'Vervet-AGM protein or transcript stable ID'\n",
       "3115. 'Vervet-AGM chromosome/scaffold name'\n",
       "3116. 'Vervet-AGM chromosome/scaffold start (bp)'\n",
       "3117. 'Vervet-AGM chromosome/scaffold end (bp)'\n",
       "3118. 'Query protein or transcript ID'\n",
       "3119. 'Last common ancestor with Vervet-AGM'\n",
       "3120. 'Vervet-AGM homology type'\n",
       "3121. '%id. target Vervet-AGM gene identical to query gene'\n",
       "3122. '%id. query gene identical to target Vervet-AGM gene'\n",
       "3123. 'Vervet-AGM Gene-order conservation score'\n",
       "3124. 'Vervet-AGM Whole-genome alignment coverage'\n",
       "3125. 'dN with Vervet-AGM'\n",
       "3126. 'dS with Vervet-AGM'\n",
       "3127. 'Vervet-AGM orthology confidence [0 low, 1 high]'\n",
       "3128. 'Wallaby gene stable ID'\n",
       "3129. 'Wallaby gene name'\n",
       "3130. 'Wallaby protein or transcript stable ID'\n",
       "3131. 'Wallaby chromosome/scaffold name'\n",
       "3132. 'Wallaby chromosome/scaffold start (bp)'\n",
       "3133. 'Wallaby chromosome/scaffold end (bp)'\n",
       "3134. 'Query protein or transcript ID'\n",
       "3135. 'Last common ancestor with Wallaby'\n",
       "3136. 'Wallaby homology type'\n",
       "3137. '%id. target Wallaby gene identical to query gene'\n",
       "3138. '%id. query gene identical to target Wallaby gene'\n",
       "3139. 'Wallaby Gene-order conservation score'\n",
       "3140. 'Wallaby Whole-genome alignment coverage'\n",
       "3141. 'dN with Wallaby'\n",
       "3142. 'dS with Wallaby'\n",
       "3143. 'Wallaby orthology confidence [0 low, 1 high]'\n",
       "3144. 'Western mosquitofish gene stable ID'\n",
       "3145. 'Western mosquitofish gene name'\n",
       "3146. 'Western mosquitofish protein or transcript stable ID'\n",
       "3147. 'Western mosquitofish chromosome/scaffold name'\n",
       "3148. 'Western mosquitofish chromosome/scaffold start (bp)'\n",
       "3149. 'Western mosquitofish chromosome/scaffold end (bp)'\n",
       "3150. 'Query protein or transcript ID'\n",
       "3151. 'Last common ancestor with Western mosquitofish'\n",
       "3152. 'Western mosquitofish homology type'\n",
       "3153. '%id. target Western mosquitofish gene identical to query gene'\n",
       "3154. '%id. query gene identical to target Western mosquitofish gene'\n",
       "3155. 'Western mosquitofish Gene-order conservation score'\n",
       "3156. 'Western mosquitofish Whole-genome alignment coverage'\n",
       "3157. 'dN with Western mosquitofish'\n",
       "3158. 'dS with Western mosquitofish'\n",
       "3159. 'Western mosquitofish orthology confidence [0 low, 1 high]'\n",
       "3160. 'White-throated sparrow gene stable ID'\n",
       "3161. 'White-throated sparrow gene name'\n",
       "3162. 'White-throated sparrow protein or transcript stable ID'\n",
       "3163. 'White-throated sparrow chromosome/scaffold name'\n",
       "3164. 'White-throated sparrow chromosome/scaffold start (bp)'\n",
       "3165. 'White-throated sparrow chromosome/scaffold end (bp)'\n",
       "3166. 'Query protein or transcript ID'\n",
       "3167. 'Last common ancestor with White-throated sparrow'\n",
       "3168. 'White-throated sparrow homology type'\n",
       "3169. '%id. target White-throated sparrow gene identical to query gene'\n",
       "3170. '%id. query gene identical to target White-throated sparrow gene'\n",
       "3171. 'White-throated sparrow Gene-order conservation score'\n",
       "3172. 'White-throated sparrow Whole-genome alignment coverage'\n",
       "3173. 'dN with White-throated sparrow'\n",
       "3174. 'dS with White-throated sparrow'\n",
       "3175. 'White-throated sparrow orthology confidence [0 low, 1 high]'\n",
       "3176. 'Wild yak gene stable ID'\n",
       "3177. 'Wild yak gene name'\n",
       "3178. 'Wild yak protein or transcript stable ID'\n",
       "3179. 'Wild yak chromosome/scaffold name'\n",
       "3180. 'Wild yak chromosome/scaffold start (bp)'\n",
       "3181. 'Wild yak chromosome/scaffold end (bp)'\n",
       "3182. 'Query protein or transcript ID'\n",
       "3183. 'Last common ancestor with Wild yak'\n",
       "3184. 'Wild yak homology type'\n",
       "3185. '%id. target Wild yak gene identical to query gene'\n",
       "3186. '%id. query gene identical to target Wild yak gene'\n",
       "3187. 'Wild yak Gene-order conservation score'\n",
       "3188. 'Wild yak Whole-genome alignment coverage'\n",
       "3189. 'dN with Wild yak'\n",
       "3190. 'dS with Wild yak'\n",
       "3191. 'Wild yak orthology confidence [0 low, 1 high]'\n",
       "3192. 'Xenopus gene stable ID'\n",
       "3193. 'Xenopus gene name'\n",
       "3194. 'Xenopus protein or transcript stable ID'\n",
       "3195. 'Xenopus chromosome/scaffold name'\n",
       "3196. 'Xenopus chromosome/scaffold start (bp)'\n",
       "3197. 'Xenopus chromosome/scaffold end (bp)'\n",
       "3198. 'Query protein or transcript ID'\n",
       "3199. 'Last common ancestor with Xenopus'\n",
       "3200. 'Xenopus homology type'\n",
       "3201. '%id. target Xenopus gene identical to query gene'\n",
       "3202. '%id. query gene identical to target Xenopus gene'\n",
       "3203. 'Xenopus Gene-order conservation score'\n",
       "3204. 'Xenopus Whole-genome alignment coverage'\n",
       "3205. 'dN with Xenopus'\n",
       "3206. 'dS with Xenopus'\n",
       "3207. 'Xenopus orthology confidence [0 low, 1 high]'\n",
       "3208. 'Yellowtail amberjack gene stable ID'\n",
       "3209. 'Yellowtail amberjack gene name'\n",
       "3210. 'Yellowtail amberjack protein or transcript stable ID'\n",
       "3211. 'Yellowtail amberjack chromosome/scaffold name'\n",
       "3212. 'Yellowtail amberjack chromosome/scaffold start (bp)'\n",
       "3213. 'Yellowtail amberjack chromosome/scaffold end (bp)'\n",
       "3214. 'Query protein or transcript ID'\n",
       "3215. 'Last common ancestor with Yellowtail amberjack'\n",
       "3216. 'Yellowtail amberjack homology type'\n",
       "3217. '%id. target Yellowtail amberjack gene identical to query gene'\n",
       "3218. '%id. query gene identical to target Yellowtail amberjack gene'\n",
       "3219. 'Yellowtail amberjack Gene-order conservation score'\n",
       "3220. 'Yellowtail amberjack Whole-genome alignment coverage'\n",
       "3221. 'dN with Yellowtail amberjack'\n",
       "3222. 'dS with Yellowtail amberjack'\n",
       "3223. 'Yellowtail amberjack orthology confidence [0 low, 1 high]'\n",
       "3224. 'Zebra Finch gene stable ID'\n",
       "3225. 'Zebra Finch gene name'\n",
       "3226. 'Zebra Finch protein or transcript stable ID'\n",
       "3227. 'Zebra Finch chromosome/scaffold name'\n",
       "3228. 'Zebra Finch chromosome/scaffold start (bp)'\n",
       "3229. 'Zebra Finch chromosome/scaffold end (bp)'\n",
       "3230. 'Query protein or transcript ID'\n",
       "3231. 'Last common ancestor with Zebra Finch'\n",
       "3232. 'Zebra Finch homology type'\n",
       "3233. '%id. target Zebra Finch gene identical to query gene'\n",
       "3234. '%id. query gene identical to target Zebra Finch gene'\n",
       "3235. 'Zebra Finch Gene-order conservation score'\n",
       "3236. 'Zebra Finch Whole-genome alignment coverage'\n",
       "3237. 'dN with Zebra Finch'\n",
       "3238. 'dS with Zebra Finch'\n",
       "3239. 'Zebra Finch orthology confidence [0 low, 1 high]'\n",
       "3240. 'Zebra mbuna gene stable ID'\n",
       "3241. 'Zebra mbuna gene name'\n",
       "3242. 'Zebra mbuna protein or transcript stable ID'\n",
       "3243. 'Zebra mbuna chromosome/scaffold name'\n",
       "3244. 'Zebra mbuna chromosome/scaffold start (bp)'\n",
       "3245. 'Zebra mbuna chromosome/scaffold end (bp)'\n",
       "3246. 'Query protein or transcript ID'\n",
       "3247. 'Last common ancestor with Zebra mbuna'\n",
       "3248. 'Zebra mbuna homology type'\n",
       "3249. '%id. target Zebra mbuna gene identical to query gene'\n",
       "3250. '%id. query gene identical to target Zebra mbuna gene'\n",
       "3251. 'Zebra mbuna Gene-order conservation score'\n",
       "3252. 'Zebra mbuna Whole-genome alignment coverage'\n",
       "3253. 'dN with Zebra mbuna'\n",
       "3254. 'dS with Zebra mbuna'\n",
       "3255. 'Zebra mbuna orthology confidence [0 low, 1 high]'\n",
       "3256. 'Zebrafish gene stable ID'\n",
       "3257. 'Zebrafish gene name'\n",
       "3258. 'Zebrafish protein or transcript stable ID'\n",
       "3259. 'Zebrafish chromosome/scaffold name'\n",
       "3260. 'Zebrafish chromosome/scaffold start (bp)'\n",
       "3261. 'Zebrafish chromosome/scaffold end (bp)'\n",
       "3262. 'Query protein or transcript ID'\n",
       "3263. 'Last common ancestor with Zebrafish'\n",
       "3264. 'Zebrafish homology type'\n",
       "3265. '%id. target Zebrafish gene identical to query gene'\n",
       "3266. '%id. query gene identical to target Zebrafish gene'\n",
       "3267. 'Zebrafish Gene-order conservation score'\n",
       "3268. 'Zebrafish Whole-genome alignment coverage'\n",
       "3269. 'dN with Zebrafish'\n",
       "3270. 'dS with Zebrafish'\n",
       "3271. 'Zebrafish orthology confidence [0 low, 1 high]'\n",
       "3272. 'Zig-zag eel gene stable ID'\n",
       "3273. 'Zig-zag eel gene name'\n",
       "3274. 'Zig-zag eel protein or transcript stable ID'\n",
       "3275. 'Zig-zag eel chromosome/scaffold name'\n",
       "3276. 'Zig-zag eel chromosome/scaffold start (bp)'\n",
       "3277. 'Zig-zag eel chromosome/scaffold end (bp)'\n",
       "3278. 'Query protein or transcript ID'\n",
       "3279. 'Last common ancestor with Zig-zag eel'\n",
       "3280. 'Zig-zag eel homology type'\n",
       "3281. '%id. target Zig-zag eel gene identical to query gene'\n",
       "3282. '%id. query gene identical to target Zig-zag eel gene'\n",
       "3283. 'Zig-zag eel Gene-order conservation score'\n",
       "3284. 'Zig-zag eel Whole-genome alignment coverage'\n",
       "3285. 'dN with Zig-zag eel'\n",
       "3286. 'dS with Zig-zag eel'\n",
       "3287. 'Zig-zag eel orthology confidence [0 low, 1 high]'\n",
       "3288. 'Human paralogue gene stable ID'\n",
       "3289. 'Human paralogue associated gene name'\n",
       "3290. 'Human paralogue protein or transcript ID'\n",
       "3291. 'Human paralogue chromosome/scaffold name'\n",
       "3292. 'Human paralogue chromosome/scaffold start (bp)'\n",
       "3293. 'Human paralogue chromosome/scaffold end (bp)'\n",
       "3294. 'Paralogue query protein or transcript ID'\n",
       "3295. 'Paralogue last common ancestor with Human'\n",
       "3296. 'Human paralogue homology type'\n",
       "3297. 'Paralogue %id. target Human gene identical to query gene'\n",
       "3298. 'Paralogue %id. query gene identical to target Human gene'\n",
       "3299. 'Paralogue dN with Human'\n",
       "3300. 'Paralogue dS with Human'\n",
       "3301. 'Human paralogy confidence [0 low, 1 high]'\n",
       "3302. 'Gene stable ID'\n",
       "3303. 'Gene stable ID version'\n",
       "3304. 'Version (gene)'\n",
       "3305. 'Transcript stable ID'\n",
       "3306. 'Transcript stable ID version'\n",
       "3307. 'Version (transcript)'\n",
       "3308. 'Protein stable ID'\n",
       "3309. 'Protein stable ID version'\n",
       "3310. 'Version (protein)'\n",
       "3311. 'Chromosome/scaffold name'\n",
       "3312. 'Gene start (bp)'\n",
       "3313. 'Gene end (bp)'\n",
       "3314. 'Strand'\n",
       "3315. 'Karyotype band'\n",
       "3316. 'Gene name'\n",
       "3317. 'Source of gene name'\n",
       "3318. 'Transcript count'\n",
       "3319. 'Gene % GC content'\n",
       "3320. 'Gene description'\n",
       "3321. 'Variant name'\n",
       "3322. 'Variant source'\n",
       "3323. 'Variant source description'\n",
       "3324. 'Variant alleles'\n",
       "3325. 'Variant supporting evidence'\n",
       "3326. 'Mapweight'\n",
       "3327. 'Minor allele'\n",
       "3328. 'Minor allele frequency'\n",
       "3329. 'Minor allele count'\n",
       "3330. 'Clinical significance'\n",
       "3331. 'Transcript location (bp)'\n",
       "3332. 'Variant chromosome Strand'\n",
       "3333. 'Protein location (aa)'\n",
       "3334. 'chromosome/scaffold position start (bp)'\n",
       "3335. 'Chromosome/scaffold position end (bp)'\n",
       "3336. 'PolyPhen prediction'\n",
       "3337. 'PolyPhen score'\n",
       "3338. 'SIFT prediction'\n",
       "3339. 'SIFT score'\n",
       "3340. 'Distance to transcript'\n",
       "3341. 'CDS start'\n",
       "3342. 'CDS end'\n",
       "3343. 'Protein allele'\n",
       "3344. 'Variant consequence'\n",
       "3345. 'Consequence specific allele'\n",
       "3346. 'Gene stable ID'\n",
       "3347. 'Gene stable ID version'\n",
       "3348. 'Version (gene)'\n",
       "3349. 'Transcript stable ID'\n",
       "3350. 'Transcript stable ID version'\n",
       "3351. 'Version (transcript)'\n",
       "3352. 'Protein stable ID'\n",
       "3353. 'Protein stable ID version'\n",
       "3354. 'Version (protein)'\n",
       "3355. 'Chromosome/scaffold name'\n",
       "3356. 'Gene start (bp)'\n",
       "3357. 'Gene end (bp)'\n",
       "3358. 'Strand'\n",
       "3359. 'Karyotype band'\n",
       "3360. 'Gene name'\n",
       "3361. 'Source of gene name'\n",
       "3362. 'Transcript count'\n",
       "3363. 'Gene % GC content'\n",
       "3364. 'Gene description'\n",
       "3365. 'Variant name'\n",
       "3366. 'Variant source'\n",
       "3367. 'Variant source description'\n",
       "3368. 'Variant alleles'\n",
       "3369. 'Variant supporting evidence'\n",
       "3370. 'Mapweight'\n",
       "3371. 'Minor allele'\n",
       "3372. 'Minor allele frequency'\n",
       "3373. 'Minor allele count'\n",
       "3374. 'Clinical significance'\n",
       "3375. 'Transcript location (bp)'\n",
       "3376. 'Variant chromosome/scaffold strand'\n",
       "3377. 'Protein location (aa)'\n",
       "3378. 'Chromosome/scaffold position start (bp)'\n",
       "3379. 'Chromosome/scaffold position end (bp)'\n",
       "3380. 'PolyPhen prediction'\n",
       "3381. 'PolyPhen score'\n",
       "3382. 'SIFT prediction'\n",
       "3383. 'SIFT score'\n",
       "3384. 'Distance to transcript'\n",
       "3385. 'CDS start'\n",
       "3386. 'CDS end'\n",
       "3387. 'Protein allele'\n",
       "3388. 'Variant consequence'\n",
       "3389. 'Consequence specific allele'\n",
       "3390. 'Unspliced (Transcript)'\n",
       "3391. 'Unspliced (Gene)'\n",
       "3392. 'Flank (Transcript)'\n",
       "3393. 'Flank (Gene)'\n",
       "3394. 'Flank-coding region (Transcript)'\n",
       "3395. 'Flank-coding region (Gene)'\n",
       "3396. '5\\' UTR'\n",
       "3397. '3\\' UTR'\n",
       "3398. 'Exon sequences'\n",
       "3399. 'cDNA sequences'\n",
       "3400. 'Coding sequence'\n",
       "3401. 'Peptide'\n",
       "3402. 'upstream_flank'\n",
       "3403. 'downstream_flank'\n",
       "3404. 'Gene stable ID'\n",
       "3405. 'Gene stable ID version'\n",
       "3406. 'Gene description'\n",
       "3407. 'Gene name'\n",
       "3408. 'Source of gene name'\n",
       "3409. 'Chromosome/scaffold name'\n",
       "3410. 'Gene start (bp)'\n",
       "3411. 'Gene end (bp)'\n",
       "3412. 'Gene type'\n",
       "3413. 'Version (gene)'\n",
       "3414. 'Ensembl Protein Family ID(s)'\n",
       "3415. 'UniParc ID'\n",
       "3416. 'UniProtKB/Swiss-Prot ID'\n",
       "3417. 'UniProtKB/TrEMBL ID'\n",
       "3418. 'CDS start (within cDNA)'\n",
       "3419. 'CDS end (within cDNA)'\n",
       "3420. '5\\' UTR start'\n",
       "3421. '5\\' UTR end'\n",
       "3422. '3\\' UTR start'\n",
       "3423. '3\\' UTR end'\n",
       "3424. 'Transcript stable ID'\n",
       "3425. 'Transcript stable ID version'\n",
       "3426. 'Protein stable ID'\n",
       "3427. 'Protein stable ID version'\n",
       "3428. 'Transcript type'\n",
       "3429. 'Version (transcript)'\n",
       "3430. 'Version (protein)'\n",
       "3431. 'Strand'\n",
       "3432. 'Transcript start (bp)'\n",
       "3433. 'Transcript end (bp)'\n",
       "3434. 'Transcription start site (TSS)'\n",
       "3435. 'Transcript length (including UTRs and CDS)'\n",
       "3436. 'CDS Length'\n",
       "3437. 'CDS start'\n",
       "3438. 'CDS end'\n",
       "3439. 'Exon stable ID'\n",
       "3440. 'Exon region start (bp)'\n",
       "3441. 'Exon region end (bp)'\n",
       "3442. 'Strand'\n",
       "3443. 'Exon rank in transcript'\n",
       "3444. 'Start phase'\n",
       "3445. 'End phase'\n",
       "3446. 'cDNA coding start'\n",
       "3447. 'cDNA coding end'\n",
       "3448. 'Genomic coding start'\n",
       "3449. 'Genomic coding end'\n",
       "3450. 'Constitutive exon'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "   [1] \"Gene stable ID\"                                                                 \n",
       "   [2] \"Gene stable ID version\"                                                         \n",
       "   [3] \"Transcript stable ID\"                                                           \n",
       "   [4] \"Transcript stable ID version\"                                                   \n",
       "   [5] \"Protein stable ID\"                                                              \n",
       "   [6] \"Protein stable ID version\"                                                      \n",
       "   [7] \"Exon stable ID\"                                                                 \n",
       "   [8] \"Gene description\"                                                               \n",
       "   [9] \"Chromosome/scaffold name\"                                                       \n",
       "  [10] \"Gene start (bp)\"                                                                \n",
       "  [11] \"Gene end (bp)\"                                                                  \n",
       "  [12] \"Strand\"                                                                         \n",
       "  [13] \"Karyotype band\"                                                                 \n",
       "  [14] \"Transcript start (bp)\"                                                          \n",
       "  [15] \"Transcript end (bp)\"                                                            \n",
       "  [16] \"Transcription start site (TSS)\"                                                 \n",
       "  [17] \"Transcript length (including UTRs and CDS)\"                                     \n",
       "  [18] \"Transcript support level (TSL)\"                                                 \n",
       "  [19] \"GENCODE basic annotation\"                                                       \n",
       "  [20] \"APPRIS annotation\"                                                              \n",
       "  [21] \"RefSeq match transcript\"                                                        \n",
       "  [22] \"Gene name\"                                                                      \n",
       "  [23] \"Source of gene name\"                                                            \n",
       "  [24] \"Transcript name\"                                                                \n",
       "  [25] \"Source of transcript name\"                                                      \n",
       "  [26] \"Transcript count\"                                                               \n",
       "  [27] \"Gene % GC content\"                                                              \n",
       "  [28] \"Gene type\"                                                                      \n",
       "  [29] \"Transcript type\"                                                                \n",
       "  [30] \"Source (gene)\"                                                                  \n",
       "  [31] \"Source (transcript)\"                                                            \n",
       "  [32] \"Version (gene)\"                                                                 \n",
       "  [33] \"Version (transcript)\"                                                           \n",
       "  [34] \"Version (protein)\"                                                              \n",
       "  [35] \"Phenotype description\"                                                          \n",
       "  [36] \"Source name\"                                                                    \n",
       "  [37] \"Study external reference\"                                                       \n",
       "  [38] \"Strain name\"                                                                    \n",
       "  [39] \"Strain gender\"                                                                  \n",
       "  [40] \"P value\"                                                                        \n",
       "  [41] \"GO term accession\"                                                              \n",
       "  [42] \"GO term name\"                                                                   \n",
       "  [43] \"GO term definition\"                                                             \n",
       "  [44] \"GO term evidence code\"                                                          \n",
       "  [45] \"GO domain\"                                                                      \n",
       "  [46] \"GOSlim GOA Accession(s)\"                                                        \n",
       "  [47] \"GOSlim GOA Description\"                                                         \n",
       "  [48] \"CCDS ID\"                                                                        \n",
       "  [49] \"ChEMBL ID\"                                                                      \n",
       "  [50] \"Clone-based (Ensembl) gene ID\"                                                  \n",
       "  [51] \"Clone-based (Ensembl) transcript ID\"                                            \n",
       "  [52] \"DataBase of Aberrant 3' Splice Sites name\"                                      \n",
       "  [53] \"DataBase of Aberrant 3' Splice Sites ID\"                                        \n",
       "  [54] \"DataBase of Aberrant 5' Splice Sites name\"                                      \n",
       "  [55] \"DataBase of Aberrant 5' Splice Sites ID\"                                        \n",
       "  [56] \"Ensembl Human Transcript ID\"                                                    \n",
       "  [57] \"Ensembl Human Translation ID\"                                                   \n",
       "  [58] \"EntrezGene transcript name ID\"                                                  \n",
       "  [59] \"European Nucleotide Archive ID\"                                                 \n",
       "  [60] \"Expression Atlas ID\"                                                            \n",
       "  [61] \"GeneDB ID\"                                                                      \n",
       "  [62] \"HGNC ID\"                                                                        \n",
       "  [63] \"HGNC symbol\"                                                                    \n",
       "  [64] \"HGNC transcript name ID\"                                                        \n",
       "  [65] \"Human Protein Atlas accession\"                                                  \n",
       "  [66] \"Human Protein Atlas ID\"                                                         \n",
       "  [67] \"INSDC protein ID\"                                                               \n",
       "  [68] \"KEGG Pathway and Enzyme ID\"                                                     \n",
       "  [69] \"LRG display in Ensembl gene ID\"                                                 \n",
       "  [70] \"LRG display in Ensembl transcript ID\"                                           \n",
       "  [71] \"MEROPS - the Peptidase Database ID\"                                             \n",
       "  [72] \"MIM gene description\"                                                           \n",
       "  [73] \"MIM gene accession\"                                                             \n",
       "  [74] \"MIM morbid description\"                                                         \n",
       "  [75] \"MIM morbid accession\"                                                           \n",
       "  [76] \"miRBase accession\"                                                              \n",
       "  [77] \"miRBase ID\"                                                                     \n",
       "  [78] \"miRBase transcript name ID\"                                                     \n",
       "  [79] \"NCBI gene description\"                                                          \n",
       "  [80] \"NCBI gene accession\"                                                            \n",
       "  [81] \"NCBI gene ID\"                                                                   \n",
       "  [82] \"PDB ID\"                                                                         \n",
       "  [83] \"Reactome ID\"                                                                    \n",
       "  [84] \"Reactome gene ID\"                                                               \n",
       "  [85] \"Reactome transcript ID\"                                                         \n",
       "  [86] \"RefSeq mRNA ID\"                                                                 \n",
       "  [87] \"RefSeq mRNA predicted ID\"                                                       \n",
       "  [88] \"RefSeq ncRNA ID\"                                                                \n",
       "  [89] \"RefSeq ncRNA predicted ID\"                                                      \n",
       "  [90] \"RefSeq peptide ID\"                                                              \n",
       "  [91] \"RefSeq peptide predicted ID\"                                                    \n",
       "  [92] \"RFAM ID\"                                                                        \n",
       "  [93] \"RFAM transcript name ID\"                                                        \n",
       "  [94] \"RNAcentral ID\"                                                                  \n",
       "  [95] \"UCSC Stable ID\"                                                                 \n",
       "  [96] \"UniParc ID\"                                                                     \n",
       "  [97] \"UniProtKB Gene Name symbol\"                                                     \n",
       "  [98] \"UniProtKB Gene Name ID\"                                                         \n",
       "  [99] \"UniProtKB/Swiss-Prot ID\"                                                        \n",
       " [100] \"UniProtKB/TrEMBL ID\"                                                            \n",
       " [101] \"WikiGene description\"                                                           \n",
       " [102] \"WikiGene name\"                                                                  \n",
       " [103] \"WikiGene ID\"                                                                    \n",
       " [104] \"AFFY HC G110 probe\"                                                             \n",
       " [105] \"AFFY HG Focus probe\"                                                            \n",
       " [106] \"AFFY HG U133A probe\"                                                            \n",
       " [107] \"AFFY HG U133A 2 probe\"                                                          \n",
       " [108] \"AFFY HG U133B probe\"                                                            \n",
       " [109] \"AFFY HG U133 Plus 2 probe\"                                                      \n",
       " [110] \"AFFY HG U95A probe\"                                                             \n",
       " [111] \"AFFY HG U95Av2 probe\"                                                           \n",
       " [112] \"AFFY HG U95B probe\"                                                             \n",
       " [113] \"AFFY HG U95C probe\"                                                             \n",
       " [114] \"AFFY HG U95D probe\"                                                             \n",
       " [115] \"AFFY HG U95E probe\"                                                             \n",
       " [116] \"AFFY HTA 2 0 probe\"                                                             \n",
       " [117] \"AFFY HuEx 1 0 st v2 probe\"                                                      \n",
       " [118] \"AFFY HuGeneFL probe\"                                                            \n",
       " [119] \"AFFY HuGene 1 0 st v1 probe\"                                                    \n",
       " [120] \"AFFY HuGene 2 0 st v1 probe\"                                                    \n",
       " [121] \"AFFY PrimeView probe\"                                                           \n",
       " [122] \"AFFY U133 X3P probe\"                                                            \n",
       " [123] \"AGILENT CGH 44b probe\"                                                          \n",
       " [124] \"AGILENT GPL6848 probe\"                                                          \n",
       " [125] \"AGILENT SurePrint G3 GE 8x60k probe\"                                            \n",
       " [126] \"AGILENT SurePrint G3 GE 8x60k v2 probe\"                                         \n",
       " [127] \"AGILENT WholeGenome probe\"                                                      \n",
       " [128] \"AGILENT WholeGenome 4x44k v1 probe\"                                             \n",
       " [129] \"AGILENT WholeGenome 4x44k v2 probe\"                                             \n",
       " [130] \"CODELINK CODELINK probe\"                                                        \n",
       " [131] \"ILLUMINA HumanHT 12 V3 probe\"                                                   \n",
       " [132] \"ILLUMINA HumanHT 12 V4 probe\"                                                   \n",
       " [133] \"ILLUMINA HumanRef 8 V3 probe\"                                                   \n",
       " [134] \"ILLUMINA HumanWG 6 V1 probe\"                                                    \n",
       " [135] \"ILLUMINA HumanWG 6 V2 probe\"                                                    \n",
       " [136] \"ILLUMINA HumanWG 6 V3 probe\"                                                    \n",
       " [137] \"PHALANX OneArray probe\"                                                         \n",
       " [138] \"Ensembl Protein Family ID(s)\"                                                   \n",
       " [139] \"Ensembl Family Description\"                                                     \n",
       " [140] \"CDD ID\"                                                                         \n",
       " [141] \"CDD start\"                                                                      \n",
       " [142] \"CDD end\"                                                                        \n",
       " [143] \"Gene3D ID\"                                                                      \n",
       " [144] \"Gene3D start\"                                                                   \n",
       " [145] \"Gene3D end\"                                                                     \n",
       " [146] \"HAMAP ID\"                                                                       \n",
       " [147] \"HAMAP start\"                                                                    \n",
       " [148] \"HAMAP end\"                                                                      \n",
       " [149] \"PANTHER ID\"                                                                     \n",
       " [150] \"PANTHER start\"                                                                  \n",
       " [151] \"PANTHER end\"                                                                    \n",
       " [152] \"Pfam ID\"                                                                        \n",
       " [153] \"Pfam start\"                                                                     \n",
       " [154] \"Pfam end\"                                                                       \n",
       " [155] \"PIRSF ID\"                                                                       \n",
       " [156] \"PIRSF start\"                                                                    \n",
       " [157] \"PIRSF end\"                                                                      \n",
       " [158] \"Prints ID\"                                                                      \n",
       " [159] \"Prints start\"                                                                   \n",
       " [160] \"Prints end\"                                                                     \n",
       " [161] \"ProDom ID\"                                                                      \n",
       " [162] \"ProDom start\"                                                                   \n",
       " [163] \"ProDom end\"                                                                     \n",
       " [164] \"PROSITE patterns ID\"                                                            \n",
       " [165] \"PROSITE patterns start\"                                                         \n",
       " [166] \"PROSITE patterns end\"                                                           \n",
       " [167] \"PROSITE profiles ID\"                                                            \n",
       " [168] \"PROSITE profiles start\"                                                         \n",
       " [169] \"PROSITE profiles end\"                                                           \n",
       " [170] \"SFLD ID\"                                                                        \n",
       " [171] \"SFLD start\"                                                                     \n",
       " [172] \"SFLD end\"                                                                       \n",
       " [173] \"SMART ID\"                                                                       \n",
       " [174] \"SMART start\"                                                                    \n",
       " [175] \"SMART end\"                                                                      \n",
       " [176] \"Superfamily ID\"                                                                 \n",
       " [177] \"Superfamily start\"                                                              \n",
       " [178] \"Superfamily end\"                                                                \n",
       " [179] \"TIGRFAM ID\"                                                                     \n",
       " [180] \"TIGRFAM start\"                                                                  \n",
       " [181] \"TIGRFAM end\"                                                                    \n",
       " [182] \"Interpro ID\"                                                                    \n",
       " [183] \"Interpro Short Description\"                                                     \n",
       " [184] \"Interpro Description\"                                                           \n",
       " [185] \"Interpro start\"                                                                 \n",
       " [186] \"Interpro end\"                                                                   \n",
       " [187] \"MobiDB lite\"                                                                    \n",
       " [188] \"MobiDB lite start\"                                                              \n",
       " [189] \"MobiDB lite end\"                                                                \n",
       " [190] \"Coiled-coils (Ncoils)\"                                                          \n",
       " [191] \"Coiled-coils (Ncoils) start\"                                                    \n",
       " [192] \"Coiled-coils (Ncoils) end\"                                                      \n",
       " [193] \"Low complexity (Seg)\"                                                           \n",
       " [194] \"Low complexity (Seg) start\"                                                     \n",
       " [195] \"Low complexity (Seg) end\"                                                       \n",
       " [196] \"PDB-ENSP mappings\"                                                              \n",
       " [197] \"PDB-ENSP mappings start\"                                                        \n",
       " [198] \"PDB-ENSP mappings end\"                                                          \n",
       " [199] \"Cleavage site (Signalp)\"                                                        \n",
       " [200] \"Cleavage site (Signalp) start\"                                                  \n",
       " [201] \"Cleavage site (Signalp) end\"                                                    \n",
       " [202] \"Transmembrane helices\"                                                          \n",
       " [203] \"Transmembrane helices start\"                                                    \n",
       " [204] \"Transmembrane helices end\"                                                      \n",
       " [205] \"Gene stable ID\"                                                                 \n",
       " [206] \"Gene stable ID version\"                                                         \n",
       " [207] \"Version (gene)\"                                                                 \n",
       " [208] \"Transcript stable ID\"                                                           \n",
       " [209] \"Transcript stable ID version\"                                                   \n",
       " [210] \"Version (transcript)\"                                                           \n",
       " [211] \"Protein stable ID\"                                                              \n",
       " [212] \"Protein stable ID version\"                                                      \n",
       " [213] \"Version (protein)\"                                                              \n",
       " [214] \"Chromosome/scaffold name\"                                                       \n",
       " [215] \"Gene start (bp)\"                                                                \n",
       " [216] \"Gene end (bp)\"                                                                  \n",
       " [217] \"Transcript start (bp)\"                                                          \n",
       " [218] \"Transcript end (bp)\"                                                            \n",
       " [219] \"Transcription start site (TSS)\"                                                 \n",
       " [220] \"Transcript length (including UTRs and CDS)\"                                     \n",
       " [221] \"Strand\"                                                                         \n",
       " [222] \"Gene name\"                                                                      \n",
       " [223] \"Source of gene name\"                                                            \n",
       " [224] \"5' UTR start\"                                                                   \n",
       " [225] \"5' UTR end\"                                                                     \n",
       " [226] \"3' UTR start\"                                                                   \n",
       " [227] \"3' UTR end\"                                                                     \n",
       " [228] \"CDS Length\"                                                                     \n",
       " [229] \"Transcript count\"                                                               \n",
       " [230] \"Gene description\"                                                               \n",
       " [231] \"Gene type\"                                                                      \n",
       " [232] \"Exon region start (bp)\"                                                         \n",
       " [233] \"Exon region end (bp)\"                                                           \n",
       " [234] \"Constitutive exon\"                                                              \n",
       " [235] \"Exon rank in transcript\"                                                        \n",
       " [236] \"Start phase\"                                                                    \n",
       " [237] \"End phase\"                                                                      \n",
       " [238] \"cDNA coding start\"                                                              \n",
       " [239] \"cDNA coding end\"                                                                \n",
       " [240] \"Genomic coding start\"                                                           \n",
       " [241] \"Genomic coding end\"                                                             \n",
       " [242] \"Exon stable ID\"                                                                 \n",
       " [243] \"CDS start\"                                                                      \n",
       " [244] \"CDS end\"                                                                        \n",
       " [245] \"Gene stable ID\"                                                                 \n",
       " [246] \"Gene stable ID version\"                                                         \n",
       " [247] \"Version (gene)\"                                                                 \n",
       " [248] \"Transcript stable ID\"                                                           \n",
       " [249] \"Transcript stable ID version\"                                                   \n",
       " [250] \"Version (transcript)\"                                                           \n",
       " [251] \"Protein stable ID\"                                                              \n",
       " [252] \"Protein stable ID version\"                                                      \n",
       " [253] \"Version (protein)\"                                                              \n",
       " [254] \"Chromosome/scaffold name\"                                                       \n",
       " [255] \"Gene start (bp)\"                                                                \n",
       " [256] \"Gene end (bp)\"                                                                  \n",
       " [257] \"Strand\"                                                                         \n",
       " [258] \"Karyotype band\"                                                                 \n",
       " [259] \"Gene name\"                                                                      \n",
       " [260] \"Source of gene name\"                                                            \n",
       " [261] \"Transcript count\"                                                               \n",
       " [262] \"Gene % GC content\"                                                              \n",
       " [263] \"Gene description\"                                                               \n",
       " [264] \"Abingdon island giant tortoise gene stable ID\"                                  \n",
       " [265] \"Abingdon island giant tortoise gene name\"                                       \n",
       " [266] \"Abingdon island giant tortoise protein or transcript stable ID\"                 \n",
       " [267] \"Abingdon island giant tortoise chromosome/scaffold name\"                        \n",
       " [268] \"Abingdon island giant tortoise chromosome/scaffold start (bp)\"                  \n",
       " [269] \"Abingdon island giant tortoise chromosome/scaffold end (bp)\"                    \n",
       " [270] \"Query protein or transcript ID\"                                                 \n",
       " [271] \"Last common ancestor with Abingdon island giant tortoise\"                       \n",
       " [272] \"Abingdon island giant tortoise homology type\"                                   \n",
       " [273] \"%id. target Abingdon island giant tortoise gene identical to query gene\"        \n",
       " [274] \"%id. query gene identical to target Abingdon island giant tortoise gene\"        \n",
       " [275] \"Abingdon island giant tortoise Gene-order conservation score\"                   \n",
       " [276] \"Abingdon island giant tortoise Whole-genome alignment coverage\"                 \n",
       " [277] \"dN with Abingdon island giant tortoise\"                                         \n",
       " [278] \"dS with Abingdon island giant tortoise\"                                         \n",
       " [279] \"Abingdon island giant tortoise orthology confidence [0 low, 1 high]\"            \n",
       " [280] \"Agassiz's desert tortoise gene stable ID\"                                       \n",
       " [281] \"Agassiz's desert tortoise gene name\"                                            \n",
       " [282] \"Agassiz's desert tortoise protein or transcript stable ID\"                      \n",
       " [283] \"Agassiz's desert tortoise chromosome/scaffold name\"                             \n",
       " [284] \"Agassiz's desert tortoise chromosome/scaffold start (bp)\"                       \n",
       " [285] \"Agassiz's desert tortoise chromosome/scaffold end (bp)\"                         \n",
       " [286] \"Query protein or transcript ID\"                                                 \n",
       " [287] \"Last common ancestor with Agassiz's desert tortoise\"                            \n",
       " [288] \"Agassiz's desert tortoise homology type\"                                        \n",
       " [289] \"%id. target Agassiz's desert tortoise gene identical to query gene\"             \n",
       " [290] \"%id. query gene identical to target Agassiz's desert tortoise gene\"             \n",
       " [291] \"Agassiz's desert tortoise Gene-order conservation score\"                        \n",
       " [292] \"Agassiz's desert tortoise Whole-genome alignment coverage\"                      \n",
       " [293] \"dN with Agassiz's desert tortoise\"                                              \n",
       " [294] \"dS with Agassiz's desert tortoise\"                                              \n",
       " [295] \"Agassiz's desert tortoise orthology confidence [0 low, 1 high]\"                 \n",
       " [296] \"Algerian mouse gene stable ID\"                                                  \n",
       " [297] \"Algerian mouse gene name\"                                                       \n",
       " [298] \"Algerian mouse protein or transcript stable ID\"                                 \n",
       " [299] \"Algerian mouse chromosome/scaffold name\"                                        \n",
       " [300] \"Algerian mouse chromosome/scaffold start (bp)\"                                  \n",
       " [301] \"Algerian mouse chromosome/scaffold end (bp)\"                                    \n",
       " [302] \"Query protein or transcript ID\"                                                 \n",
       " [303] \"Last common ancestor with Algerian mouse\"                                       \n",
       " [304] \"Algerian mouse homology type\"                                                   \n",
       " [305] \"%id. target Algerian mouse gene identical to query gene\"                        \n",
       " [306] \"%id. query gene identical to target Algerian mouse gene\"                        \n",
       " [307] \"Algerian mouse Gene-order conservation score\"                                   \n",
       " [308] \"Algerian mouse Whole-genome alignment coverage\"                                 \n",
       " [309] \"dN with Algerian mouse\"                                                         \n",
       " [310] \"dS with Algerian mouse\"                                                         \n",
       " [311] \"Algerian mouse orthology confidence [0 low, 1 high]\"                            \n",
       " [312] \"Alpaca gene stable ID\"                                                          \n",
       " [313] \"Alpaca gene name\"                                                               \n",
       " [314] \"Alpaca protein or transcript stable ID\"                                         \n",
       " [315] \"Alpaca chromosome/scaffold name\"                                                \n",
       " [316] \"Alpaca chromosome/scaffold start (bp)\"                                          \n",
       " [317] \"Alpaca chromosome/scaffold end (bp)\"                                            \n",
       " [318] \"Query protein or transcript ID\"                                                 \n",
       " [319] \"Last common ancestor with Alpaca\"                                               \n",
       " [320] \"Alpaca homology type\"                                                           \n",
       " [321] \"%id. target Alpaca gene identical to query gene\"                                \n",
       " [322] \"%id. query gene identical to target Alpaca gene\"                                \n",
       " [323] \"Alpaca Gene-order conservation score\"                                           \n",
       " [324] \"Alpaca Whole-genome alignment coverage\"                                         \n",
       " [325] \"dN with Alpaca\"                                                                 \n",
       " [326] \"dS with Alpaca\"                                                                 \n",
       " [327] \"Alpaca orthology confidence [0 low, 1 high]\"                                    \n",
       " [328] \"Alpine marmot gene stable ID\"                                                   \n",
       " [329] \"Alpine marmot gene name\"                                                        \n",
       " [330] \"Alpine marmot protein or transcript stable ID\"                                  \n",
       " [331] \"Alpine marmot chromosome/scaffold name\"                                         \n",
       " [332] \"Alpine marmot chromosome/scaffold start (bp)\"                                   \n",
       " [333] \"Alpine marmot chromosome/scaffold end (bp)\"                                     \n",
       " [334] \"Query protein or transcript ID\"                                                 \n",
       " [335] \"Last common ancestor with Alpine marmot\"                                        \n",
       " [336] \"Alpine marmot homology type\"                                                    \n",
       " [337] \"%id. target Alpine marmot gene identical to query gene\"                         \n",
       " [338] \"%id. query gene identical to target Alpine marmot gene\"                         \n",
       " [339] \"Alpine marmot Gene-order conservation score\"                                    \n",
       " [340] \"Alpine marmot Whole-genome alignment coverage\"                                  \n",
       " [341] \"dN with Alpine marmot\"                                                          \n",
       " [342] \"dS with Alpine marmot\"                                                          \n",
       " [343] \"Alpine marmot orthology confidence [0 low, 1 high]\"                             \n",
       " [344] \"Amazon molly gene stable ID\"                                                    \n",
       " [345] \"Amazon molly gene name\"                                                         \n",
       " [346] \"Amazon molly protein or transcript stable ID\"                                   \n",
       " [347] \"Amazon molly chromosome/scaffold name\"                                          \n",
       " [348] \"Amazon molly chromosome/scaffold start (bp)\"                                    \n",
       " [349] \"Amazon molly chromosome/scaffold end (bp)\"                                      \n",
       " [350] \"Query protein or transcript ID\"                                                 \n",
       " [351] \"Last common ancestor with Amazon molly\"                                         \n",
       " [352] \"Amazon molly homology type\"                                                     \n",
       " [353] \"%id. target Amazon molly gene identical to query gene\"                          \n",
       " [354] \"%id. query gene identical to target Amazon molly gene\"                          \n",
       " [355] \"Amazon molly Gene-order conservation score\"                                     \n",
       " [356] \"Amazon molly Whole-genome alignment coverage\"                                   \n",
       " [357] \"dN with Amazon molly\"                                                           \n",
       " [358] \"dS with Amazon molly\"                                                           \n",
       " [359] \"Amazon molly orthology confidence [0 low, 1 high]\"                              \n",
       " [360] \"American beaver gene stable ID\"                                                 \n",
       " [361] \"American beaver gene name\"                                                      \n",
       " [362] \"American beaver protein or transcript stable ID\"                                \n",
       " [363] \"American beaver chromosome/scaffold name\"                                       \n",
       " [364] \"American beaver chromosome/scaffold start (bp)\"                                 \n",
       " [365] \"American beaver chromosome/scaffold end (bp)\"                                   \n",
       " [366] \"Query protein or transcript ID\"                                                 \n",
       " [367] \"Last common ancestor with American beaver\"                                      \n",
       " [368] \"American beaver homology type\"                                                  \n",
       " [369] \"%id. target American beaver gene identical to query gene\"                       \n",
       " [370] \"%id. query gene identical to target American beaver gene\"                       \n",
       " [371] \"American beaver Gene-order conservation score\"                                  \n",
       " [372] \"American beaver Whole-genome alignment coverage\"                                \n",
       " [373] \"dN with American beaver\"                                                        \n",
       " [374] \"dS with American beaver\"                                                        \n",
       " [375] \"American beaver orthology confidence [0 low, 1 high]\"                           \n",
       " [376] \"American bison gene stable ID\"                                                  \n",
       " [377] \"American bison gene name\"                                                       \n",
       " [378] \"American bison protein or transcript stable ID\"                                 \n",
       " [379] \"American bison chromosome/scaffold name\"                                        \n",
       " [380] \"American bison chromosome/scaffold start (bp)\"                                  \n",
       " [381] \"American bison chromosome/scaffold end (bp)\"                                    \n",
       " [382] \"Query protein or transcript ID\"                                                 \n",
       " [383] \"Last common ancestor with American bison\"                                       \n",
       " [384] \"American bison homology type\"                                                   \n",
       " [385] \"%id. target American bison gene identical to query gene\"                        \n",
       " [386] \"%id. query gene identical to target American bison gene\"                        \n",
       " [387] \"American bison Gene-order conservation score\"                                   \n",
       " [388] \"American bison Whole-genome alignment coverage\"                                 \n",
       " [389] \"dN with American bison\"                                                         \n",
       " [390] \"dS with American bison\"                                                         \n",
       " [391] \"American bison orthology confidence [0 low, 1 high]\"                            \n",
       " [392] \"American black bear gene stable ID\"                                             \n",
       " [393] \"American black bear gene name\"                                                  \n",
       " [394] \"American black bear protein or transcript stable ID\"                            \n",
       " [395] \"American black bear chromosome/scaffold name\"                                   \n",
       " [396] \"American black bear chromosome/scaffold start (bp)\"                             \n",
       " [397] \"American black bear chromosome/scaffold end (bp)\"                               \n",
       " [398] \"Query protein or transcript ID\"                                                 \n",
       " [399] \"Last common ancestor with American black bear\"                                  \n",
       " [400] \"American black bear homology type\"                                              \n",
       " [401] \"%id. target American black bear gene identical to query gene\"                   \n",
       " [402] \"%id. query gene identical to target American black bear gene\"                   \n",
       " [403] \"American black bear Gene-order conservation score\"                              \n",
       " [404] \"American black bear Whole-genome alignment coverage\"                            \n",
       " [405] \"dN with American black bear\"                                                    \n",
       " [406] \"dS with American black bear\"                                                    \n",
       " [407] \"American black bear orthology confidence [0 low, 1 high]\"                       \n",
       " [408] \"American mink gene stable ID\"                                                   \n",
       " [409] \"American mink gene name\"                                                        \n",
       " [410] \"American mink protein or transcript stable ID\"                                  \n",
       " [411] \"American mink chromosome/scaffold name\"                                         \n",
       " [412] \"American mink chromosome/scaffold start (bp)\"                                   \n",
       " [413] \"American mink chromosome/scaffold end (bp)\"                                     \n",
       " [414] \"Query protein or transcript ID\"                                                 \n",
       " [415] \"Last common ancestor with American mink\"                                        \n",
       " [416] \"American mink homology type\"                                                    \n",
       " [417] \"%id. target American mink gene identical to query gene\"                         \n",
       " [418] \"%id. query gene identical to target American mink gene\"                         \n",
       " [419] \"American mink Gene-order conservation score\"                                    \n",
       " [420] \"American mink Whole-genome alignment coverage\"                                  \n",
       " [421] \"dN with American mink\"                                                          \n",
       " [422] \"dS with American mink\"                                                          \n",
       " [423] \"American mink orthology confidence [0 low, 1 high]\"                             \n",
       " [424] \"Angola colobus gene stable ID\"                                                  \n",
       " [425] \"Angola colobus gene name\"                                                       \n",
       " [426] \"Angola colobus protein or transcript stable ID\"                                 \n",
       " [427] \"Angola colobus chromosome/scaffold name\"                                        \n",
       " [428] \"Angola colobus chromosome/scaffold start (bp)\"                                  \n",
       " [429] \"Angola colobus chromosome/scaffold end (bp)\"                                    \n",
       " [430] \"Query protein or transcript ID\"                                                 \n",
       " [431] \"Last common ancestor with Angola colobus\"                                       \n",
       " [432] \"Angola colobus homology type\"                                                   \n",
       " [433] \"%id. target Angola colobus gene identical to query gene\"                        \n",
       " [434] \"%id. query gene identical to target Angola colobus gene\"                        \n",
       " [435] \"Angola colobus Gene-order conservation score\"                                   \n",
       " [436] \"Angola colobus Whole-genome alignment coverage\"                                 \n",
       " [437] \"dN with Angola colobus\"                                                         \n",
       " [438] \"dS with Angola colobus\"                                                         \n",
       " [439] \"Angola colobus orthology confidence [0 low, 1 high]\"                            \n",
       " [440] \"Anole lizard gene stable ID\"                                                    \n",
       " [441] \"Anole lizard gene name\"                                                         \n",
       " [442] \"Anole lizard protein or transcript stable ID\"                                   \n",
       " [443] \"Anole lizard chromosome/scaffold name\"                                          \n",
       " [444] \"Anole lizard chromosome/scaffold start (bp)\"                                    \n",
       " [445] \"Anole lizard chromosome/scaffold end (bp)\"                                      \n",
       " [446] \"Query protein or transcript ID\"                                                 \n",
       " [447] \"Last common ancestor with Anole lizard\"                                         \n",
       " [448] \"Anole lizard homology type\"                                                     \n",
       " [449] \"%id. target Anole lizard gene identical to query gene\"                          \n",
       " [450] \"%id. query gene identical to target Anole lizard gene\"                          \n",
       " [451] \"Anole lizard Gene-order conservation score\"                                     \n",
       " [452] \"Anole lizard Whole-genome alignment coverage\"                                   \n",
       " [453] \"dN with Anole lizard\"                                                           \n",
       " [454] \"dS with Anole lizard\"                                                           \n",
       " [455] \"Anole lizard orthology confidence [0 low, 1 high]\"                              \n",
       " [456] \"Arctic ground squirrel gene stable ID\"                                          \n",
       " [457] \"Arctic ground squirrel gene name\"                                               \n",
       " [458] \"Arctic ground squirrel protein or transcript stable ID\"                         \n",
       " [459] \"Arctic ground squirrel chromosome/scaffold name\"                                \n",
       " [460] \"Arctic ground squirrel chromosome/scaffold start (bp)\"                          \n",
       " [461] \"Arctic ground squirrel chromosome/scaffold end (bp)\"                            \n",
       " [462] \"Query protein or transcript ID\"                                                 \n",
       " [463] \"Last common ancestor with Arctic ground squirrel\"                               \n",
       " [464] \"Arctic ground squirrel homology type\"                                           \n",
       " [465] \"%id. target Arctic ground squirrel gene identical to query gene\"                \n",
       " [466] \"%id. query gene identical to target Arctic ground squirrel gene\"                \n",
       " [467] \"Arctic ground squirrel Gene-order conservation score\"                           \n",
       " [468] \"Arctic ground squirrel Whole-genome alignment coverage\"                         \n",
       " [469] \"dN with Arctic ground squirrel\"                                                 \n",
       " [470] \"dS with Arctic ground squirrel\"                                                 \n",
       " [471] \"Arctic ground squirrel orthology confidence [0 low, 1 high]\"                    \n",
       " [472] \"Argentine black and white tegu gene stable ID\"                                  \n",
       " [473] \"Argentine black and white tegu gene name\"                                       \n",
       " [474] \"Argentine black and white tegu protein or transcript stable ID\"                 \n",
       " [475] \"Argentine black and white tegu chromosome/scaffold name\"                        \n",
       " [476] \"Argentine black and white tegu chromosome/scaffold start (bp)\"                  \n",
       " [477] \"Argentine black and white tegu chromosome/scaffold end (bp)\"                    \n",
       " [478] \"Query protein or transcript ID\"                                                 \n",
       " [479] \"Last common ancestor with Argentine black and white tegu\"                       \n",
       " [480] \"Argentine black and white tegu homology type\"                                   \n",
       " [481] \"%id. target Argentine black and white tegu gene identical to query gene\"        \n",
       " [482] \"%id. query gene identical to target Argentine black and white tegu gene\"        \n",
       " [483] \"Argentine black and white tegu Gene-order conservation score\"                   \n",
       " [484] \"Argentine black and white tegu Whole-genome alignment coverage\"                 \n",
       " [485] \"dN with Argentine black and white tegu\"                                         \n",
       " [486] \"dS with Argentine black and white tegu\"                                         \n",
       " [487] \"Argentine black and white tegu orthology confidence [0 low, 1 high]\"            \n",
       " [488] \"Armadillo gene stable ID\"                                                       \n",
       " [489] \"Armadillo gene name\"                                                            \n",
       " [490] \"Armadillo protein or transcript stable ID\"                                      \n",
       " [491] \"Armadillo chromosome/scaffold name\"                                             \n",
       " [492] \"Armadillo chromosome/scaffold start (bp)\"                                       \n",
       " [493] \"Armadillo chromosome/scaffold end (bp)\"                                         \n",
       " [494] \"Query protein or transcript ID\"                                                 \n",
       " [495] \"Last common ancestor with Armadillo\"                                            \n",
       " [496] \"Armadillo homology type\"                                                        \n",
       " [497] \"%id. target Armadillo gene identical to query gene\"                             \n",
       " [498] \"%id. query gene identical to target Armadillo gene\"                             \n",
       " [499] \"Armadillo Gene-order conservation score\"                                        \n",
       " [500] \"Armadillo Whole-genome alignment coverage\"                                      \n",
       " [501] \"dN with Armadillo\"                                                              \n",
       " [502] \"dS with Armadillo\"                                                              \n",
       " [503] \"Armadillo orthology confidence [0 low, 1 high]\"                                 \n",
       " [504] \"Asian bonytongue gene stable ID\"                                                \n",
       " [505] \"Asian bonytongue gene name\"                                                     \n",
       " [506] \"Asian bonytongue protein or transcript stable ID\"                               \n",
       " [507] \"Asian bonytongue chromosome/scaffold name\"                                      \n",
       " [508] \"Asian bonytongue chromosome/scaffold start (bp)\"                                \n",
       " [509] \"Asian bonytongue chromosome/scaffold end (bp)\"                                  \n",
       " [510] \"Query protein or transcript ID\"                                                 \n",
       " [511] \"Last common ancestor with Asian bonytongue\"                                     \n",
       " [512] \"Asian bonytongue homology type\"                                                 \n",
       " [513] \"%id. target Asian bonytongue gene identical to query gene\"                      \n",
       " [514] \"%id. query gene identical to target Asian bonytongue gene\"                      \n",
       " [515] \"Asian bonytongue Gene-order conservation score\"                                 \n",
       " [516] \"Asian bonytongue Whole-genome alignment coverage\"                               \n",
       " [517] \"dN with Asian bonytongue\"                                                       \n",
       " [518] \"dS with Asian bonytongue\"                                                       \n",
       " [519] \"Asian bonytongue orthology confidence [0 low, 1 high]\"                          \n",
       " [520] \"Australian saltwater crocodile gene stable ID\"                                  \n",
       " [521] \"Australian saltwater crocodile gene name\"                                       \n",
       " [522] \"Australian saltwater crocodile protein or transcript stable ID\"                 \n",
       " [523] \"Australian saltwater crocodile chromosome/scaffold name\"                        \n",
       " [524] \"Australian saltwater crocodile chromosome/scaffold start (bp)\"                  \n",
       " [525] \"Australian saltwater crocodile chromosome/scaffold end (bp)\"                    \n",
       " [526] \"Query protein or transcript ID\"                                                 \n",
       " [527] \"Last common ancestor with Australian saltwater crocodile\"                       \n",
       " [528] \"Australian saltwater crocodile homology type\"                                   \n",
       " [529] \"%id. target Australian saltwater crocodile gene identical to query gene\"        \n",
       " [530] \"%id. query gene identical to target Australian saltwater crocodile gene\"        \n",
       " [531] \"Australian saltwater crocodile Gene-order conservation score\"                   \n",
       " [532] \"Australian saltwater crocodile Whole-genome alignment coverage\"                 \n",
       " [533] \"dN with Australian saltwater crocodile\"                                         \n",
       " [534] \"dS with Australian saltwater crocodile\"                                         \n",
       " [535] \"Australian saltwater crocodile orthology confidence [0 low, 1 high]\"            \n",
       " [536] \"Ballan wrasse gene stable ID\"                                                   \n",
       " [537] \"Ballan wrasse gene name\"                                                        \n",
       " [538] \"Ballan wrasse protein or transcript stable ID\"                                  \n",
       " [539] \"Ballan wrasse chromosome/scaffold name\"                                         \n",
       " [540] \"Ballan wrasse chromosome/scaffold start (bp)\"                                   \n",
       " [541] \"Ballan wrasse chromosome/scaffold end (bp)\"                                     \n",
       " [542] \"Query protein or transcript ID\"                                                 \n",
       " [543] \"Last common ancestor with Ballan wrasse\"                                        \n",
       " [544] \"Ballan wrasse homology type\"                                                    \n",
       " [545] \"%id. target Ballan wrasse gene identical to query gene\"                         \n",
       " [546] \"%id. query gene identical to target Ballan wrasse gene\"                         \n",
       " [547] \"Ballan wrasse Gene-order conservation score\"                                    \n",
       " [548] \"Ballan wrasse Whole-genome alignment coverage\"                                  \n",
       " [549] \"dN with Ballan wrasse\"                                                          \n",
       " [550] \"dS with Ballan wrasse\"                                                          \n",
       " [551] \"Ballan wrasse orthology confidence [0 low, 1 high]\"                             \n",
       " [552] \"Barramundi perch gene stable ID\"                                                \n",
       " [553] \"Barramundi perch gene name\"                                                     \n",
       " [554] \"Barramundi perch protein or transcript stable ID\"                               \n",
       " [555] \"Barramundi perch chromosome/scaffold name\"                                      \n",
       " [556] \"Barramundi perch chromosome/scaffold start (bp)\"                                \n",
       " [557] \"Barramundi perch chromosome/scaffold end (bp)\"                                  \n",
       " [558] \"Query protein or transcript ID\"                                                 \n",
       " [559] \"Last common ancestor with Barramundi perch\"                                     \n",
       " [560] \"Barramundi perch homology type\"                                                 \n",
       " [561] \"%id. target Barramundi perch gene identical to query gene\"                      \n",
       " [562] \"%id. query gene identical to target Barramundi perch gene\"                      \n",
       " [563] \"Barramundi perch Gene-order conservation score\"                                 \n",
       " [564] \"Barramundi perch Whole-genome alignment coverage\"                               \n",
       " [565] \"dN with Barramundi perch\"                                                       \n",
       " [566] \"dS with Barramundi perch\"                                                       \n",
       " [567] \"Barramundi perch orthology confidence [0 low, 1 high]\"                          \n",
       " [568] \"Bengalese finch gene stable ID\"                                                 \n",
       " [569] \"Bengalese finch gene name\"                                                      \n",
       " [570] \"Bengalese finch protein or transcript stable ID\"                                \n",
       " [571] \"Bengalese finch chromosome/scaffold name\"                                       \n",
       " [572] \"Bengalese finch chromosome/scaffold start (bp)\"                                 \n",
       " [573] \"Bengalese finch chromosome/scaffold end (bp)\"                                   \n",
       " [574] \"Query protein or transcript ID\"                                                 \n",
       " [575] \"Last common ancestor with Bengalese finch\"                                      \n",
       " [576] \"Bengalese finch homology type\"                                                  \n",
       " [577] \"%id. target Bengalese finch gene identical to query gene\"                       \n",
       " [578] \"%id. query gene identical to target Bengalese finch gene\"                       \n",
       " [579] \"Bengalese finch Gene-order conservation score\"                                  \n",
       " [580] \"Bengalese finch Whole-genome alignment coverage\"                                \n",
       " [581] \"dN with Bengalese finch\"                                                        \n",
       " [582] \"dS with Bengalese finch\"                                                        \n",
       " [583] \"Bengalese finch orthology confidence [0 low, 1 high]\"                           \n",
       " [584] \"Bicolor damselfish gene stable ID\"                                              \n",
       " [585] \"Bicolor damselfish gene name\"                                                   \n",
       " [586] \"Bicolor damselfish protein or transcript stable ID\"                             \n",
       " [587] \"Bicolor damselfish chromosome/scaffold name\"                                    \n",
       " [588] \"Bicolor damselfish chromosome/scaffold start (bp)\"                              \n",
       " [589] \"Bicolor damselfish chromosome/scaffold end (bp)\"                                \n",
       " [590] \"Query protein or transcript ID\"                                                 \n",
       " [591] \"Last common ancestor with Bicolor damselfish\"                                   \n",
       " [592] \"Bicolor damselfish homology type\"                                               \n",
       " [593] \"%id. target Bicolor damselfish gene identical to query gene\"                    \n",
       " [594] \"%id. query gene identical to target Bicolor damselfish gene\"                    \n",
       " [595] \"Bicolor damselfish Gene-order conservation score\"                               \n",
       " [596] \"Bicolor damselfish Whole-genome alignment coverage\"                             \n",
       " [597] \"dN with Bicolor damselfish\"                                                     \n",
       " [598] \"dS with Bicolor damselfish\"                                                     \n",
       " [599] \"Bicolor damselfish orthology confidence [0 low, 1 high]\"                        \n",
       " [600] \"Black snub-nosed monkey gene stable ID\"                                         \n",
       " [601] \"Black snub-nosed monkey gene name\"                                              \n",
       " [602] \"Black snub-nosed monkey protein or transcript stable ID\"                        \n",
       " [603] \"Black snub-nosed monkey chromosome/scaffold name\"                               \n",
       " [604] \"Black snub-nosed monkey chromosome/scaffold start (bp)\"                         \n",
       " [605] \"Black snub-nosed monkey chromosome/scaffold end (bp)\"                           \n",
       " [606] \"Query protein or transcript ID\"                                                 \n",
       " [607] \"Last common ancestor with Black snub-nosed monkey\"                              \n",
       " [608] \"Black snub-nosed monkey homology type\"                                          \n",
       " [609] \"%id. target Black snub-nosed monkey gene identical to query gene\"               \n",
       " [610] \"%id. query gene identical to target Black snub-nosed monkey gene\"               \n",
       " [611] \"Black snub-nosed monkey Gene-order conservation score\"                          \n",
       " [612] \"Black snub-nosed monkey Whole-genome alignment coverage\"                        \n",
       " [613] \"dN with Black snub-nosed monkey\"                                                \n",
       " [614] \"dS with Black snub-nosed monkey\"                                                \n",
       " [615] \"Black snub-nosed monkey orthology confidence [0 low, 1 high]\"                   \n",
       " [616] \"Blue tit gene stable ID\"                                                        \n",
       " [617] \"Blue tit gene name\"                                                             \n",
       " [618] \"Blue tit protein or transcript stable ID\"                                       \n",
       " [619] \"Blue tit chromosome/scaffold name\"                                              \n",
       " [620] \"Blue tit chromosome/scaffold start (bp)\"                                        \n",
       " [621] \"Blue tit chromosome/scaffold end (bp)\"                                          \n",
       " [622] \"Query protein or transcript ID\"                                                 \n",
       " [623] \"Last common ancestor with Blue tit\"                                             \n",
       " [624] \"Blue tit homology type\"                                                         \n",
       " [625] \"%id. target Blue tit gene identical to query gene\"                              \n",
       " [626] \"%id. query gene identical to target Blue tit gene\"                              \n",
       " [627] \"Blue tit Gene-order conservation score\"                                         \n",
       " [628] \"Blue tit Whole-genome alignment coverage\"                                       \n",
       " [629] \"dN with Blue tit\"                                                               \n",
       " [630] \"dS with Blue tit\"                                                               \n",
       " [631] \"Blue tit orthology confidence [0 low, 1 high]\"                                  \n",
       " [632] \"Blue-crowned manakin gene stable ID\"                                            \n",
       " [633] \"Blue-crowned manakin gene name\"                                                 \n",
       " [634] \"Blue-crowned manakin protein or transcript stable ID\"                           \n",
       " [635] \"Blue-crowned manakin chromosome/scaffold name\"                                  \n",
       " [636] \"Blue-crowned manakin chromosome/scaffold start (bp)\"                            \n",
       " [637] \"Blue-crowned manakin chromosome/scaffold end (bp)\"                              \n",
       " [638] \"Query protein or transcript ID\"                                                 \n",
       " [639] \"Last common ancestor with Blue-crowned manakin\"                                 \n",
       " [640] \"Blue-crowned manakin homology type\"                                             \n",
       " [641] \"%id. target Blue-crowned manakin gene identical to query gene\"                  \n",
       " [642] \"%id. query gene identical to target Blue-crowned manakin gene\"                  \n",
       " [643] \"Blue-crowned manakin Gene-order conservation score\"                             \n",
       " [644] \"Blue-crowned manakin Whole-genome alignment coverage\"                           \n",
       " [645] \"dN with Blue-crowned manakin\"                                                   \n",
       " [646] \"dS with Blue-crowned manakin\"                                                   \n",
       " [647] \"Blue-crowned manakin orthology confidence [0 low, 1 high]\"                      \n",
       " [648] \"Bolivian squirrel monkey gene stable ID\"                                        \n",
       " [649] \"Bolivian squirrel monkey gene name\"                                             \n",
       " [650] \"Bolivian squirrel monkey protein or transcript stable ID\"                       \n",
       " [651] \"Bolivian squirrel monkey chromosome/scaffold name\"                              \n",
       " [652] \"Bolivian squirrel monkey chromosome/scaffold start (bp)\"                        \n",
       " [653] \"Bolivian squirrel monkey chromosome/scaffold end (bp)\"                          \n",
       " [654] \"Query protein or transcript ID\"                                                 \n",
       " [655] \"Last common ancestor with Bolivian squirrel monkey\"                             \n",
       " [656] \"Bolivian squirrel monkey homology type\"                                         \n",
       " [657] \"%id. target Bolivian squirrel monkey gene identical to query gene\"              \n",
       " [658] \"%id. query gene identical to target Bolivian squirrel monkey gene\"              \n",
       " [659] \"Bolivian squirrel monkey Gene-order conservation score\"                         \n",
       " [660] \"Bolivian squirrel monkey Whole-genome alignment coverage\"                       \n",
       " [661] \"dN with Bolivian squirrel monkey\"                                               \n",
       " [662] \"dS with Bolivian squirrel monkey\"                                               \n",
       " [663] \"Bolivian squirrel monkey orthology confidence [0 low, 1 high]\"                  \n",
       " [664] \"Bonobo gene stable ID\"                                                          \n",
       " [665] \"Bonobo gene name\"                                                               \n",
       " [666] \"Bonobo protein or transcript stable ID\"                                         \n",
       " [667] \"Bonobo chromosome/scaffold name\"                                                \n",
       " [668] \"Bonobo chromosome/scaffold start (bp)\"                                          \n",
       " [669] \"Bonobo chromosome/scaffold end (bp)\"                                            \n",
       " [670] \"Query protein or transcript ID\"                                                 \n",
       " [671] \"Last common ancestor with Bonobo\"                                               \n",
       " [672] \"Bonobo homology type\"                                                           \n",
       " [673] \"%id. target Bonobo gene identical to query gene\"                                \n",
       " [674] \"%id. query gene identical to target Bonobo gene\"                                \n",
       " [675] \"Bonobo Gene-order conservation score\"                                           \n",
       " [676] \"Bonobo Whole-genome alignment coverage\"                                         \n",
       " [677] \"dN with Bonobo\"                                                                 \n",
       " [678] \"dS with Bonobo\"                                                                 \n",
       " [679] \"Bonobo orthology confidence [0 low, 1 high]\"                                    \n",
       " [680] \"Brazilian guinea pig gene stable ID\"                                            \n",
       " [681] \"Brazilian guinea pig gene name\"                                                 \n",
       " [682] \"Brazilian guinea pig protein or transcript stable ID\"                           \n",
       " [683] \"Brazilian guinea pig chromosome/scaffold name\"                                  \n",
       " [684] \"Brazilian guinea pig chromosome/scaffold start (bp)\"                            \n",
       " [685] \"Brazilian guinea pig chromosome/scaffold end (bp)\"                              \n",
       " [686] \"Query protein or transcript ID\"                                                 \n",
       " [687] \"Last common ancestor with Brazilian guinea pig\"                                 \n",
       " [688] \"Brazilian guinea pig homology type\"                                             \n",
       " [689] \"%id. target Brazilian guinea pig gene identical to query gene\"                  \n",
       " [690] \"%id. query gene identical to target Brazilian guinea pig gene\"                  \n",
       " [691] \"Brazilian guinea pig Gene-order conservation score\"                             \n",
       " [692] \"Brazilian guinea pig Whole-genome alignment coverage\"                           \n",
       " [693] \"dN with Brazilian guinea pig\"                                                   \n",
       " [694] \"dS with Brazilian guinea pig\"                                                   \n",
       " [695] \"Brazilian guinea pig orthology confidence [0 low, 1 high]\"                      \n",
       " [696] \"Budgerigar gene stable ID\"                                                      \n",
       " [697] \"Budgerigar gene name\"                                                           \n",
       " [698] \"Budgerigar protein or transcript stable ID\"                                     \n",
       " [699] \"Budgerigar chromosome/scaffold name\"                                            \n",
       " [700] \"Budgerigar chromosome/scaffold start (bp)\"                                      \n",
       " [701] \"Budgerigar chromosome/scaffold end (bp)\"                                        \n",
       " [702] \"Query protein or transcript ID\"                                                 \n",
       " [703] \"Last common ancestor with Budgerigar\"                                           \n",
       " [704] \"Budgerigar homology type\"                                                       \n",
       " [705] \"%id. target Budgerigar gene identical to query gene\"                            \n",
       " [706] \"%id. query gene identical to target Budgerigar gene\"                            \n",
       " [707] \"Budgerigar Gene-order conservation score\"                                       \n",
       " [708] \"Budgerigar Whole-genome alignment coverage\"                                     \n",
       " [709] \"dN with Budgerigar\"                                                             \n",
       " [710] \"dS with Budgerigar\"                                                             \n",
       " [711] \"Budgerigar orthology confidence [0 low, 1 high]\"                                \n",
       " [712] \"Burton's mouthbrooder gene stable ID\"                                           \n",
       " [713] \"Burton's mouthbrooder gene name\"                                                \n",
       " [714] \"Burton's mouthbrooder protein or transcript stable ID\"                          \n",
       " [715] \"Burton's mouthbrooder chromosome/scaffold name\"                                 \n",
       " [716] \"Burton's mouthbrooder chromosome/scaffold start (bp)\"                           \n",
       " [717] \"Burton's mouthbrooder chromosome/scaffold end (bp)\"                             \n",
       " [718] \"Query protein or transcript ID\"                                                 \n",
       " [719] \"Last common ancestor with Burton's mouthbrooder\"                                \n",
       " [720] \"Burton's mouthbrooder homology type\"                                            \n",
       " [721] \"%id. target Burton's mouthbrooder gene identical to query gene\"                 \n",
       " [722] \"%id. query gene identical to target Burton's mouthbrooder gene\"                 \n",
       " [723] \"Burton's mouthbrooder Gene-order conservation score\"                            \n",
       " [724] \"Burton's mouthbrooder Whole-genome alignment coverage\"                          \n",
       " [725] \"dN with Burton's mouthbrooder\"                                                  \n",
       " [726] \"dS with Burton's mouthbrooder\"                                                  \n",
       " [727] \"Burton's mouthbrooder orthology confidence [0 low, 1 high]\"                     \n",
       " [728] \"Bushbaby gene stable ID\"                                                        \n",
       " [729] \"Bushbaby gene name\"                                                             \n",
       " [730] \"Bushbaby protein or transcript stable ID\"                                       \n",
       " [731] \"Bushbaby chromosome/scaffold name\"                                              \n",
       " [732] \"Bushbaby chromosome/scaffold start (bp)\"                                        \n",
       " [733] \"Bushbaby chromosome/scaffold end (bp)\"                                          \n",
       " [734] \"Query protein or transcript ID\"                                                 \n",
       " [735] \"Last common ancestor with Bushbaby\"                                             \n",
       " [736] \"Bushbaby homology type\"                                                         \n",
       " [737] \"%id. target Bushbaby gene identical to query gene\"                              \n",
       " [738] \"%id. query gene identical to target Bushbaby gene\"                              \n",
       " [739] \"Bushbaby Gene-order conservation score\"                                         \n",
       " [740] \"Bushbaby Whole-genome alignment coverage\"                                       \n",
       " [741] \"dN with Bushbaby\"                                                               \n",
       " [742] \"dS with Bushbaby\"                                                               \n",
       " [743] \"Bushbaby orthology confidence [0 low, 1 high]\"                                  \n",
       " [744] \"C.intestinalis gene stable ID\"                                                  \n",
       " [745] \"C.intestinalis gene name\"                                                       \n",
       " [746] \"C.intestinalis protein or transcript stable ID\"                                 \n",
       " [747] \"C.intestinalis chromosome/scaffold name\"                                        \n",
       " [748] \"C.intestinalis chromosome/scaffold start (bp)\"                                  \n",
       " [749] \"C.intestinalis chromosome/scaffold end (bp)\"                                    \n",
       " [750] \"Query protein or transcript ID\"                                                 \n",
       " [751] \"Last common ancestor with C.intestinalis\"                                       \n",
       " [752] \"C.intestinalis homology type\"                                                   \n",
       " [753] \"%id. target C.intestinalis gene identical to query gene\"                        \n",
       " [754] \"%id. query gene identical to target C.intestinalis gene\"                        \n",
       " [755] \"C.intestinalis Gene-order conservation score\"                                   \n",
       " [756] \"C.intestinalis Whole-genome alignment coverage\"                                 \n",
       " [757] \"dN with C.intestinalis\"                                                         \n",
       " [758] \"dS with C.intestinalis\"                                                         \n",
       " [759] \"C.intestinalis orthology confidence [0 low, 1 high]\"                            \n",
       " [760] \"C.savignyi gene stable ID\"                                                      \n",
       " [761] \"C.savignyi gene name\"                                                           \n",
       " [762] \"C.savignyi protein or transcript stable ID\"                                     \n",
       " [763] \"C.savignyi chromosome/scaffold name\"                                            \n",
       " [764] \"C.savignyi chromosome/scaffold start (bp)\"                                      \n",
       " [765] \"C.savignyi chromosome/scaffold end (bp)\"                                        \n",
       " [766] \"Query protein or transcript ID\"                                                 \n",
       " [767] \"Last common ancestor with C.savignyi\"                                           \n",
       " [768] \"C.savignyi homology type\"                                                       \n",
       " [769] \"%id. target C.savignyi gene identical to query gene\"                            \n",
       " [770] \"%id. query gene identical to target C.savignyi gene\"                            \n",
       " [771] \"C.savignyi Gene-order conservation score\"                                       \n",
       " [772] \"C.savignyi Whole-genome alignment coverage\"                                     \n",
       " [773] \"dN with C.savignyi\"                                                             \n",
       " [774] \"dS with C.savignyi\"                                                             \n",
       " [775] \"C.savignyi orthology confidence [0 low, 1 high]\"                                \n",
       " [776] \"Caenorhabditis elegans gene stable ID\"                                          \n",
       " [777] \"Caenorhabditis elegans gene name\"                                               \n",
       " [778] \"Caenorhabditis elegans protein or transcript stable ID\"                         \n",
       " [779] \"Caenorhabditis elegans chromosome/scaffold name\"                                \n",
       " [780] \"Caenorhabditis elegans chromosome/scaffold start (bp)\"                          \n",
       " [781] \"Caenorhabditis elegans chromosome/scaffold end (bp)\"                            \n",
       " [782] \"Query protein or transcript ID\"                                                 \n",
       " [783] \"Last common ancestor with Caenorhabditis elegans\"                               \n",
       " [784] \"Caenorhabditis elegans homology type\"                                           \n",
       " [785] \"%id. target Caenorhabditis elegans gene identical to query gene\"                \n",
       " [786] \"%id. query gene identical to target Caenorhabditis elegans gene\"                \n",
       " [787] \"Caenorhabditis elegans Gene-order conservation score\"                           \n",
       " [788] \"Caenorhabditis elegans Whole-genome alignment coverage\"                         \n",
       " [789] \"dN with Caenorhabditis elegans\"                                                 \n",
       " [790] \"dS with Caenorhabditis elegans\"                                                 \n",
       " [791] \"Caenorhabditis elegans orthology confidence [0 low, 1 high]\"                    \n",
       " [792] \"Capuchin gene stable ID\"                                                        \n",
       " [793] \"Capuchin gene name\"                                                             \n",
       " [794] \"Capuchin protein or transcript stable ID\"                                       \n",
       " [795] \"Capuchin chromosome/scaffold name\"                                              \n",
       " [796] \"Capuchin chromosome/scaffold start (bp)\"                                        \n",
       " [797] \"Capuchin chromosome/scaffold end (bp)\"                                          \n",
       " [798] \"Query protein or transcript ID\"                                                 \n",
       " [799] \"Last common ancestor with Capuchin\"                                             \n",
       " [800] \"Capuchin homology type\"                                                         \n",
       " [801] \"%id. target Capuchin gene identical to query gene\"                              \n",
       " [802] \"%id. query gene identical to target Capuchin gene\"                              \n",
       " [803] \"Capuchin Gene-order conservation score\"                                         \n",
       " [804] \"Capuchin Whole-genome alignment coverage\"                                       \n",
       " [805] \"dN with Capuchin\"                                                               \n",
       " [806] \"dS with Capuchin\"                                                               \n",
       " [807] \"Capuchin orthology confidence [0 low, 1 high]\"                                  \n",
       " [808] \"Cat gene stable ID\"                                                             \n",
       " [809] \"Cat gene name\"                                                                  \n",
       " [810] \"Cat protein or transcript stable ID\"                                            \n",
       " [811] \"Cat chromosome/scaffold name\"                                                   \n",
       " [812] \"Cat chromosome/scaffold start (bp)\"                                             \n",
       " [813] \"Cat chromosome/scaffold end (bp)\"                                               \n",
       " [814] \"Query protein or transcript ID\"                                                 \n",
       " [815] \"Last common ancestor with Cat\"                                                  \n",
       " [816] \"Cat homology type\"                                                              \n",
       " [817] \"%id. target Cat gene identical to query gene\"                                   \n",
       " [818] \"%id. query gene identical to target Cat gene\"                                   \n",
       " [819] \"Cat Gene-order conservation score\"                                              \n",
       " [820] \"Cat Whole-genome alignment coverage\"                                            \n",
       " [821] \"dN with Cat\"                                                                    \n",
       " [822] \"dS with Cat\"                                                                    \n",
       " [823] \"Cat orthology confidence [0 low, 1 high]\"                                       \n",
       " [824] \"Central bearded dragon gene stable ID\"                                          \n",
       " [825] \"Central bearded dragon gene name\"                                               \n",
       " [826] \"Central bearded dragon protein or transcript stable ID\"                         \n",
       " [827] \"Central bearded dragon chromosome/scaffold name\"                                \n",
       " [828] \"Central bearded dragon chromosome/scaffold start (bp)\"                          \n",
       " [829] \"Central bearded dragon chromosome/scaffold end (bp)\"                            \n",
       " [830] \"Query protein or transcript ID\"                                                 \n",
       " [831] \"Last common ancestor with Central bearded dragon\"                               \n",
       " [832] \"Central bearded dragon homology type\"                                           \n",
       " [833] \"%id. target Central bearded dragon gene identical to query gene\"                \n",
       " [834] \"%id. query gene identical to target Central bearded dragon gene\"                \n",
       " [835] \"Central bearded dragon Gene-order conservation score\"                           \n",
       " [836] \"Central bearded dragon Whole-genome alignment coverage\"                         \n",
       " [837] \"dN with Central bearded dragon\"                                                 \n",
       " [838] \"dS with Central bearded dragon\"                                                 \n",
       " [839] \"Central bearded dragon orthology confidence [0 low, 1 high]\"                    \n",
       " [840] \"Channel catfish gene stable ID\"                                                 \n",
       " [841] \"Channel catfish gene name\"                                                      \n",
       " [842] \"Channel catfish protein or transcript stable ID\"                                \n",
       " [843] \"Channel catfish chromosome/scaffold name\"                                       \n",
       " [844] \"Channel catfish chromosome/scaffold start (bp)\"                                 \n",
       " [845] \"Channel catfish chromosome/scaffold end (bp)\"                                   \n",
       " [846] \"Query protein or transcript ID\"                                                 \n",
       " [847] \"Last common ancestor with Channel catfish\"                                      \n",
       " [848] \"Channel catfish homology type\"                                                  \n",
       " [849] \"%id. target Channel catfish gene identical to query gene\"                       \n",
       " [850] \"%id. query gene identical to target Channel catfish gene\"                       \n",
       " [851] \"Channel catfish Gene-order conservation score\"                                  \n",
       " [852] \"Channel catfish Whole-genome alignment coverage\"                                \n",
       " [853] \"dN with Channel catfish\"                                                        \n",
       " [854] \"dS with Channel catfish\"                                                        \n",
       " [855] \"Channel catfish orthology confidence [0 low, 1 high]\"                           \n",
       " [856] \"Chicken gene stable ID\"                                                         \n",
       " [857] \"Chicken gene name\"                                                              \n",
       " [858] \"Chicken protein or transcript stable ID\"                                        \n",
       " [859] \"Chicken chromosome/scaffold name\"                                               \n",
       " [860] \"Chicken chromosome/scaffold start (bp)\"                                         \n",
       " [861] \"Chicken chromosome/scaffold end (bp)\"                                           \n",
       " [862] \"Query protein or transcript ID\"                                                 \n",
       " [863] \"Last common ancestor with Chicken\"                                              \n",
       " [864] \"Chicken homology type\"                                                          \n",
       " [865] \"%id. target Chicken gene identical to query gene\"                               \n",
       " [866] \"%id. query gene identical to target Chicken gene\"                               \n",
       " [867] \"Chicken Gene-order conservation score\"                                          \n",
       " [868] \"Chicken Whole-genome alignment coverage\"                                        \n",
       " [869] \"dN with Chicken\"                                                                \n",
       " [870] \"dS with Chicken\"                                                                \n",
       " [871] \"Chicken orthology confidence [0 low, 1 high]\"                                   \n",
       " [872] \"Chilean tinamou gene stable ID\"                                                 \n",
       " [873] \"Chilean tinamou gene name\"                                                      \n",
       " [874] \"Chilean tinamou protein or transcript stable ID\"                                \n",
       " [875] \"Chilean tinamou chromosome/scaffold name\"                                       \n",
       " [876] \"Chilean tinamou chromosome/scaffold start (bp)\"                                 \n",
       " [877] \"Chilean tinamou chromosome/scaffold end (bp)\"                                   \n",
       " [878] \"Query protein or transcript ID\"                                                 \n",
       " [879] \"Last common ancestor with Chilean tinamou\"                                      \n",
       " [880] \"Chilean tinamou homology type\"                                                  \n",
       " [881] \"%id. target Chilean tinamou gene identical to query gene\"                       \n",
       " [882] \"%id. query gene identical to target Chilean tinamou gene\"                       \n",
       " [883] \"Chilean tinamou Gene-order conservation score\"                                  \n",
       " [884] \"Chilean tinamou Whole-genome alignment coverage\"                                \n",
       " [885] \"dN with Chilean tinamou\"                                                        \n",
       " [886] \"dS with Chilean tinamou\"                                                        \n",
       " [887] \"Chilean tinamou orthology confidence [0 low, 1 high]\"                           \n",
       " [888] \"Chimpanzee gene stable ID\"                                                      \n",
       " [889] \"Chimpanzee gene name\"                                                           \n",
       " [890] \"Chimpanzee protein or transcript stable ID\"                                     \n",
       " [891] \"Chimpanzee chromosome/scaffold name\"                                            \n",
       " [892] \"Chimpanzee chromosome/scaffold start (bp)\"                                      \n",
       " [893] \"Chimpanzee chromosome/scaffold end (bp)\"                                        \n",
       " [894] \"Query protein or transcript ID\"                                                 \n",
       " [895] \"Last common ancestor with Chimpanzee\"                                           \n",
       " [896] \"Chimpanzee homology type\"                                                       \n",
       " [897] \"%id. target Chimpanzee gene identical to query gene\"                            \n",
       " [898] \"%id. query gene identical to target Chimpanzee gene\"                            \n",
       " [899] \"Chimpanzee Gene-order conservation score\"                                       \n",
       " [900] \"Chimpanzee Whole-genome alignment coverage\"                                     \n",
       " [901] \"dN with Chimpanzee\"                                                             \n",
       " [902] \"dS with Chimpanzee\"                                                             \n",
       " [903] \"Chimpanzee orthology confidence [0 low, 1 high]\"                                \n",
       " [904] \"Chinese hamster CHOK1GS gene stable ID\"                                         \n",
       " [905] \"Chinese hamster CHOK1GS gene name\"                                              \n",
       " [906] \"Chinese hamster CHOK1GS protein or transcript stable ID\"                        \n",
       " [907] \"Chinese hamster CHOK1GS chromosome/scaffold name\"                               \n",
       " [908] \"Chinese hamster CHOK1GS chromosome/scaffold start (bp)\"                         \n",
       " [909] \"Chinese hamster CHOK1GS chromosome/scaffold end (bp)\"                           \n",
       " [910] \"Query protein or transcript ID\"                                                 \n",
       " [911] \"Last common ancestor with Chinese hamster CHOK1GS\"                              \n",
       " [912] \"Chinese hamster CHOK1GS homology type\"                                          \n",
       " [913] \"%id. target Chinese hamster CHOK1GS gene identical to query gene\"               \n",
       " [914] \"%id. query gene identical to target Chinese hamster CHOK1GS gene\"               \n",
       " [915] \"Chinese hamster CHOK1GS Gene-order conservation score\"                          \n",
       " [916] \"Chinese hamster CHOK1GS Whole-genome alignment coverage\"                        \n",
       " [917] \"dN with Chinese hamster CHOK1GS\"                                                \n",
       " [918] \"dS with Chinese hamster CHOK1GS\"                                                \n",
       " [919] \"Chinese hamster CHOK1GS orthology confidence [0 low, 1 high]\"                   \n",
       " [920] \"Chinese hamster CriGri gene stable ID\"                                          \n",
       " [921] \"Chinese hamster CriGri gene name\"                                               \n",
       " [922] \"Chinese hamster CriGri protein or transcript stable ID\"                         \n",
       " [923] \"Chinese hamster CriGri chromosome/scaffold name\"                                \n",
       " [924] \"Chinese hamster CriGri chromosome/scaffold start (bp)\"                          \n",
       " [925] \"Chinese hamster CriGri chromosome/scaffold end (bp)\"                            \n",
       " [926] \"Query protein or transcript ID\"                                                 \n",
       " [927] \"Last common ancestor with Chinese hamster CriGri\"                               \n",
       " [928] \"Chinese hamster CriGri homology type\"                                           \n",
       " [929] \"%id. target Chinese hamster CriGri gene identical to query gene\"                \n",
       " [930] \"%id. query gene identical to target Chinese hamster CriGri gene\"                \n",
       " [931] \"Chinese hamster CriGri Gene-order conservation score\"                           \n",
       " [932] \"Chinese hamster CriGri Whole-genome alignment coverage\"                         \n",
       " [933] \"dN with Chinese hamster CriGri\"                                                 \n",
       " [934] \"dS with Chinese hamster CriGri\"                                                 \n",
       " [935] \"Chinese hamster CriGri orthology confidence [0 low, 1 high]\"                    \n",
       " [936] \"Chinese hamster PICR gene stable ID\"                                            \n",
       " [937] \"Chinese hamster PICR gene name\"                                                 \n",
       " [938] \"Chinese hamster PICR protein or transcript stable ID\"                           \n",
       " [939] \"Chinese hamster PICR chromosome/scaffold name\"                                  \n",
       " [940] \"Chinese hamster PICR chromosome/scaffold start (bp)\"                            \n",
       " [941] \"Chinese hamster PICR chromosome/scaffold end (bp)\"                              \n",
       " [942] \"Query protein or transcript ID\"                                                 \n",
       " [943] \"Last common ancestor with Chinese hamster PICR\"                                 \n",
       " [944] \"Chinese hamster PICR homology type\"                                             \n",
       " [945] \"%id. target Chinese hamster PICR gene identical to query gene\"                  \n",
       " [946] \"%id. query gene identical to target Chinese hamster PICR gene\"                  \n",
       " [947] \"Chinese hamster PICR Gene-order conservation score\"                             \n",
       " [948] \"Chinese hamster PICR Whole-genome alignment coverage\"                           \n",
       " [949] \"dN with Chinese hamster PICR\"                                                   \n",
       " [950] \"dS with Chinese hamster PICR\"                                                   \n",
       " [951] \"Chinese hamster PICR orthology confidence [0 low, 1 high]\"                      \n",
       " [952] \"Chinese softshell turtle gene stable ID\"                                        \n",
       " [953] \"Chinese softshell turtle gene name\"                                             \n",
       " [954] \"Chinese softshell turtle protein or transcript stable ID\"                       \n",
       " [955] \"Chinese softshell turtle chromosome/scaffold name\"                              \n",
       " [956] \"Chinese softshell turtle chromosome/scaffold start (bp)\"                        \n",
       " [957] \"Chinese softshell turtle chromosome/scaffold end (bp)\"                          \n",
       " [958] \"Query protein or transcript ID\"                                                 \n",
       " [959] \"Last common ancestor with Chinese softshell turtle\"                             \n",
       " [960] \"Chinese softshell turtle homology type\"                                         \n",
       " [961] \"%id. target Chinese softshell turtle gene identical to query gene\"              \n",
       " [962] \"%id. query gene identical to target Chinese softshell turtle gene\"              \n",
       " [963] \"Chinese softshell turtle Gene-order conservation score\"                         \n",
       " [964] \"Chinese softshell turtle Whole-genome alignment coverage\"                       \n",
       " [965] \"dN with Chinese softshell turtle\"                                               \n",
       " [966] \"dS with Chinese softshell turtle\"                                               \n",
       " [967] \"Chinese softshell turtle orthology confidence [0 low, 1 high]\"                  \n",
       " [968] \"Climbing perch gene stable ID\"                                                  \n",
       " [969] \"Climbing perch gene name\"                                                       \n",
       " [970] \"Climbing perch protein or transcript stable ID\"                                 \n",
       " [971] \"Climbing perch chromosome/scaffold name\"                                        \n",
       " [972] \"Climbing perch chromosome/scaffold start (bp)\"                                  \n",
       " [973] \"Climbing perch chromosome/scaffold end (bp)\"                                    \n",
       " [974] \"Query protein or transcript ID\"                                                 \n",
       " [975] \"Last common ancestor with Climbing perch\"                                       \n",
       " [976] \"Climbing perch homology type\"                                                   \n",
       " [977] \"%id. target Climbing perch gene identical to query gene\"                        \n",
       " [978] \"%id. query gene identical to target Climbing perch gene\"                        \n",
       " [979] \"Climbing perch Gene-order conservation score\"                                   \n",
       " [980] \"Climbing perch Whole-genome alignment coverage\"                                 \n",
       " [981] \"dN with Climbing perch\"                                                         \n",
       " [982] \"dS with Climbing perch\"                                                         \n",
       " [983] \"Climbing perch orthology confidence [0 low, 1 high]\"                            \n",
       " [984] \"Clown anemonefish gene stable ID\"                                               \n",
       " [985] \"Clown anemonefish gene name\"                                                    \n",
       " [986] \"Clown anemonefish protein or transcript stable ID\"                              \n",
       " [987] \"Clown anemonefish chromosome/scaffold name\"                                     \n",
       " [988] \"Clown anemonefish chromosome/scaffold start (bp)\"                               \n",
       " [989] \"Clown anemonefish chromosome/scaffold end (bp)\"                                 \n",
       " [990] \"Query protein or transcript ID\"                                                 \n",
       " [991] \"Last common ancestor with Clown anemonefish\"                                    \n",
       " [992] \"Clown anemonefish homology type\"                                                \n",
       " [993] \"%id. target Clown anemonefish gene identical to query gene\"                     \n",
       " [994] \"%id. query gene identical to target Clown anemonefish gene\"                     \n",
       " [995] \"Clown anemonefish Gene-order conservation score\"                                \n",
       " [996] \"Clown anemonefish Whole-genome alignment coverage\"                              \n",
       " [997] \"dN with Clown anemonefish\"                                                      \n",
       " [998] \"dS with Clown anemonefish\"                                                      \n",
       " [999] \"Clown anemonefish orthology confidence [0 low, 1 high]\"                         \n",
       "[1000] \"Cod gene stable ID\"                                                             \n",
       "[1001] \"Cod gene name\"                                                                  \n",
       "[1002] \"Cod protein or transcript stable ID\"                                            \n",
       "[1003] \"Cod chromosome/scaffold name\"                                                   \n",
       "[1004] \"Cod chromosome/scaffold start (bp)\"                                             \n",
       "[1005] \"Cod chromosome/scaffold end (bp)\"                                               \n",
       "[1006] \"Query protein or transcript ID\"                                                 \n",
       "[1007] \"Last common ancestor with Cod\"                                                  \n",
       "[1008] \"Cod homology type\"                                                              \n",
       "[1009] \"%id. target Cod gene identical to query gene\"                                   \n",
       "[1010] \"%id. query gene identical to target Cod gene\"                                   \n",
       "[1011] \"Cod Gene-order conservation score\"                                              \n",
       "[1012] \"Cod Whole-genome alignment coverage\"                                            \n",
       "[1013] \"dN with Cod\"                                                                    \n",
       "[1014] \"dS with Cod\"                                                                    \n",
       "[1015] \"Cod orthology confidence [0 low, 1 high]\"                                       \n",
       "[1016] \"Coelacanth gene stable ID\"                                                      \n",
       "[1017] \"Coelacanth gene name\"                                                           \n",
       "[1018] \"Coelacanth protein or transcript stable ID\"                                     \n",
       "[1019] \"Coelacanth chromosome/scaffold name\"                                            \n",
       "[1020] \"Coelacanth chromosome/scaffold start (bp)\"                                      \n",
       "[1021] \"Coelacanth chromosome/scaffold end (bp)\"                                        \n",
       "[1022] \"Query protein or transcript ID\"                                                 \n",
       "[1023] \"Last common ancestor with Coelacanth\"                                           \n",
       "[1024] \"Coelacanth homology type\"                                                       \n",
       "[1025] \"%id. target Coelacanth gene identical to query gene\"                            \n",
       "[1026] \"%id. query gene identical to target Coelacanth gene\"                            \n",
       "[1027] \"Coelacanth Gene-order conservation score\"                                       \n",
       "[1028] \"Coelacanth Whole-genome alignment coverage\"                                     \n",
       "[1029] \"dN with Coelacanth\"                                                             \n",
       "[1030] \"dS with Coelacanth\"                                                             \n",
       "[1031] \"Coelacanth orthology confidence [0 low, 1 high]\"                                \n",
       "[1032] \"Common canary gene stable ID\"                                                   \n",
       "[1033] \"Common canary gene name\"                                                        \n",
       "[1034] \"Common canary protein or transcript stable ID\"                                  \n",
       "[1035] \"Common canary chromosome/scaffold name\"                                         \n",
       "[1036] \"Common canary chromosome/scaffold start (bp)\"                                   \n",
       "[1037] \"Common canary chromosome/scaffold end (bp)\"                                     \n",
       "[1038] \"Query protein or transcript ID\"                                                 \n",
       "[1039] \"Last common ancestor with Common canary\"                                        \n",
       "[1040] \"Common canary homology type\"                                                    \n",
       "[1041] \"%id. target Common canary gene identical to query gene\"                         \n",
       "[1042] \"%id. query gene identical to target Common canary gene\"                         \n",
       "[1043] \"Common canary Gene-order conservation score\"                                    \n",
       "[1044] \"Common canary Whole-genome alignment coverage\"                                  \n",
       "[1045] \"dN with Common canary\"                                                          \n",
       "[1046] \"dS with Common canary\"                                                          \n",
       "[1047] \"Common canary orthology confidence [0 low, 1 high]\"                             \n",
       "[1048] \"Common wombat gene stable ID\"                                                   \n",
       "[1049] \"Common wombat gene name\"                                                        \n",
       "[1050] \"Common wombat protein or transcript stable ID\"                                  \n",
       "[1051] \"Common wombat chromosome/scaffold name\"                                         \n",
       "[1052] \"Common wombat chromosome/scaffold start (bp)\"                                   \n",
       "[1053] \"Common wombat chromosome/scaffold end (bp)\"                                     \n",
       "[1054] \"Query protein or transcript ID\"                                                 \n",
       "[1055] \"Last common ancestor with Common wombat\"                                        \n",
       "[1056] \"Common wombat homology type\"                                                    \n",
       "[1057] \"%id. target Common wombat gene identical to query gene\"                         \n",
       "[1058] \"%id. query gene identical to target Common wombat gene\"                         \n",
       "[1059] \"Common wombat Gene-order conservation score\"                                    \n",
       "[1060] \"Common wombat Whole-genome alignment coverage\"                                  \n",
       "[1061] \"dN with Common wombat\"                                                          \n",
       "[1062] \"dS with Common wombat\"                                                          \n",
       "[1063] \"Common wombat orthology confidence [0 low, 1 high]\"                             \n",
       "[1064] \"Coquerel's sifaka gene stable ID\"                                               \n",
       "[1065] \"Coquerel's sifaka gene name\"                                                    \n",
       "[1066] \"Coquerel's sifaka protein or transcript stable ID\"                              \n",
       "[1067] \"Coquerel's sifaka chromosome/scaffold name\"                                     \n",
       "[1068] \"Coquerel's sifaka chromosome/scaffold start (bp)\"                               \n",
       "[1069] \"Coquerel's sifaka chromosome/scaffold end (bp)\"                                 \n",
       "[1070] \"Query protein or transcript ID\"                                                 \n",
       "[1071] \"Last common ancestor with Coquerel's sifaka\"                                    \n",
       "[1072] \"Coquerel's sifaka homology type\"                                                \n",
       "[1073] \"%id. target Coquerel's sifaka gene identical to query gene\"                     \n",
       "[1074] \"%id. query gene identical to target Coquerel's sifaka gene\"                     \n",
       "[1075] \"Coquerel's sifaka Gene-order conservation score\"                                \n",
       "[1076] \"Coquerel's sifaka Whole-genome alignment coverage\"                              \n",
       "[1077] \"dN with Coquerel's sifaka\"                                                      \n",
       "[1078] \"dS with Coquerel's sifaka\"                                                      \n",
       "[1079] \"Coquerel's sifaka orthology confidence [0 low, 1 high]\"                         \n",
       "[1080] \"Cow gene stable ID\"                                                             \n",
       "[1081] \"Cow gene name\"                                                                  \n",
       "[1082] \"Cow protein or transcript stable ID\"                                            \n",
       "[1083] \"Cow chromosome/scaffold name\"                                                   \n",
       "[1084] \"Cow chromosome/scaffold start (bp)\"                                             \n",
       "[1085] \"Cow chromosome/scaffold end (bp)\"                                               \n",
       "[1086] \"Query protein or transcript ID\"                                                 \n",
       "[1087] \"Last common ancestor with Cow\"                                                  \n",
       "[1088] \"Cow homology type\"                                                              \n",
       "[1089] \"%id. target Cow gene identical to query gene\"                                   \n",
       "[1090] \"%id. query gene identical to target Cow gene\"                                   \n",
       "[1091] \"Cow Gene-order conservation score\"                                              \n",
       "[1092] \"Cow Whole-genome alignment coverage\"                                            \n",
       "[1093] \"dN with Cow\"                                                                    \n",
       "[1094] \"dS with Cow\"                                                                    \n",
       "[1095] \"Cow orthology confidence [0 low, 1 high]\"                                       \n",
       "[1096] \"Crab-eating macaque gene stable ID\"                                             \n",
       "[1097] \"Crab-eating macaque gene name\"                                                  \n",
       "[1098] \"Crab-eating macaque protein or transcript stable ID\"                            \n",
       "[1099] \"Crab-eating macaque chromosome/scaffold name\"                                   \n",
       "[1100] \"Crab-eating macaque chromosome/scaffold start (bp)\"                             \n",
       "[1101] \"Crab-eating macaque chromosome/scaffold end (bp)\"                               \n",
       "[1102] \"Query protein or transcript ID\"                                                 \n",
       "[1103] \"Last common ancestor with Crab-eating macaque\"                                  \n",
       "[1104] \"Crab-eating macaque homology type\"                                              \n",
       "[1105] \"%id. target Crab-eating macaque gene identical to query gene\"                   \n",
       "[1106] \"%id. query gene identical to target Crab-eating macaque gene\"                   \n",
       "[1107] \"Crab-eating macaque Gene-order conservation score\"                              \n",
       "[1108] \"Crab-eating macaque Whole-genome alignment coverage\"                            \n",
       "[1109] \"dN with Crab-eating macaque\"                                                    \n",
       "[1110] \"dS with Crab-eating macaque\"                                                    \n",
       "[1111] \"Crab-eating macaque orthology confidence [0 low, 1 high]\"                       \n",
       "[1112] \"Damara mole rat gene stable ID\"                                                 \n",
       "[1113] \"Damara mole rat gene name\"                                                      \n",
       "[1114] \"Damara mole rat protein or transcript stable ID\"                                \n",
       "[1115] \"Damara mole rat chromosome/scaffold name\"                                       \n",
       "[1116] \"Damara mole rat chromosome/scaffold start (bp)\"                                 \n",
       "[1117] \"Damara mole rat chromosome/scaffold end (bp)\"                                   \n",
       "[1118] \"Query protein or transcript ID\"                                                 \n",
       "[1119] \"Last common ancestor with Damara mole rat\"                                      \n",
       "[1120] \"Damara mole rat homology type\"                                                  \n",
       "[1121] \"%id. target Damara mole rat gene identical to query gene\"                       \n",
       "[1122] \"%id. query gene identical to target Damara mole rat gene\"                       \n",
       "[1123] \"Damara mole rat Gene-order conservation score\"                                  \n",
       "[1124] \"Damara mole rat Whole-genome alignment coverage\"                                \n",
       "[1125] \"dN with Damara mole rat\"                                                        \n",
       "[1126] \"dS with Damara mole rat\"                                                        \n",
       "[1127] \"Damara mole rat orthology confidence [0 low, 1 high]\"                           \n",
       "[1128] \"Dark-eyed junco gene stable ID\"                                                 \n",
       "[1129] \"Dark-eyed junco gene name\"                                                      \n",
       "[1130] \"Dark-eyed junco protein or transcript stable ID\"                                \n",
       "[1131] \"Dark-eyed junco chromosome/scaffold name\"                                       \n",
       "[1132] \"Dark-eyed junco chromosome/scaffold start (bp)\"                                 \n",
       "[1133] \"Dark-eyed junco chromosome/scaffold end (bp)\"                                   \n",
       "[1134] \"Query protein or transcript ID\"                                                 \n",
       "[1135] \"Last common ancestor with Dark-eyed junco\"                                      \n",
       "[1136] \"Dark-eyed junco homology type\"                                                  \n",
       "[1137] \"%id. target Dark-eyed junco gene identical to query gene\"                       \n",
       "[1138] \"%id. query gene identical to target Dark-eyed junco gene\"                       \n",
       "[1139] \"Dark-eyed junco Gene-order conservation score\"                                  \n",
       "[1140] \"Dark-eyed junco Whole-genome alignment coverage\"                                \n",
       "[1141] \"dN with Dark-eyed junco\"                                                        \n",
       "[1142] \"dS with Dark-eyed junco\"                                                        \n",
       "[1143] \"Dark-eyed junco orthology confidence [0 low, 1 high]\"                           \n",
       "[1144] \"Daurian ground squirrel gene stable ID\"                                         \n",
       "[1145] \"Daurian ground squirrel gene name\"                                              \n",
       "[1146] \"Daurian ground squirrel protein or transcript stable ID\"                        \n",
       "[1147] \"Daurian ground squirrel chromosome/scaffold name\"                               \n",
       "[1148] \"Daurian ground squirrel chromosome/scaffold start (bp)\"                         \n",
       "[1149] \"Daurian ground squirrel chromosome/scaffold end (bp)\"                           \n",
       "[1150] \"Query protein or transcript ID\"                                                 \n",
       "[1151] \"Last common ancestor with Daurian ground squirrel\"                              \n",
       "[1152] \"Daurian ground squirrel homology type\"                                          \n",
       "[1153] \"%id. target Daurian ground squirrel gene identical to query gene\"               \n",
       "[1154] \"%id. query gene identical to target Daurian ground squirrel gene\"               \n",
       "[1155] \"Daurian ground squirrel Gene-order conservation score\"                          \n",
       "[1156] \"Daurian ground squirrel Whole-genome alignment coverage\"                        \n",
       "[1157] \"dN with Daurian ground squirrel\"                                                \n",
       "[1158] \"dS with Daurian ground squirrel\"                                                \n",
       "[1159] \"Daurian ground squirrel orthology confidence [0 low, 1 high]\"                   \n",
       "[1160] \"Degu gene stable ID\"                                                            \n",
       "[1161] \"Degu gene name\"                                                                 \n",
       "[1162] \"Degu protein or transcript stable ID\"                                           \n",
       "[1163] \"Degu chromosome/scaffold name\"                                                  \n",
       "[1164] \"Degu chromosome/scaffold start (bp)\"                                            \n",
       "[1165] \"Degu chromosome/scaffold end (bp)\"                                              \n",
       "[1166] \"Query protein or transcript ID\"                                                 \n",
       "[1167] \"Last common ancestor with Degu\"                                                 \n",
       "[1168] \"Degu homology type\"                                                             \n",
       "[1169] \"%id. target Degu gene identical to query gene\"                                  \n",
       "[1170] \"%id. query gene identical to target Degu gene\"                                  \n",
       "[1171] \"Degu Gene-order conservation score\"                                             \n",
       "[1172] \"Degu Whole-genome alignment coverage\"                                           \n",
       "[1173] \"dN with Degu\"                                                                   \n",
       "[1174] \"dS with Degu\"                                                                   \n",
       "[1175] \"Degu orthology confidence [0 low, 1 high]\"                                      \n",
       "[1176] \"Dingo gene stable ID\"                                                           \n",
       "[1177] \"Dingo gene name\"                                                                \n",
       "[1178] \"Dingo protein or transcript stable ID\"                                          \n",
       "[1179] \"Dingo chromosome/scaffold name\"                                                 \n",
       "[1180] \"Dingo chromosome/scaffold start (bp)\"                                           \n",
       "[1181] \"Dingo chromosome/scaffold end (bp)\"                                             \n",
       "[1182] \"Query protein or transcript ID\"                                                 \n",
       "[1183] \"Last common ancestor with Dingo\"                                                \n",
       "[1184] \"Dingo homology type\"                                                            \n",
       "[1185] \"%id. target Dingo gene identical to query gene\"                                 \n",
       "[1186] \"%id. query gene identical to target Dingo gene\"                                 \n",
       "[1187] \"Dingo Gene-order conservation score\"                                            \n",
       "[1188] \"Dingo Whole-genome alignment coverage\"                                          \n",
       "[1189] \"dN with Dingo\"                                                                  \n",
       "[1190] \"dS with Dingo\"                                                                  \n",
       "[1191] \"Dingo orthology confidence [0 low, 1 high]\"                                     \n",
       "[1192] \"Dog gene stable ID\"                                                             \n",
       "[1193] \"Dog gene name\"                                                                  \n",
       "[1194] \"Dog protein or transcript stable ID\"                                            \n",
       "[1195] \"Dog chromosome/scaffold name\"                                                   \n",
       "[1196] \"Dog chromosome/scaffold start (bp)\"                                             \n",
       "[1197] \"Dog chromosome/scaffold end (bp)\"                                               \n",
       "[1198] \"Query protein or transcript ID\"                                                 \n",
       "[1199] \"Last common ancestor with Dog\"                                                  \n",
       "[1200] \"Dog homology type\"                                                              \n",
       "[1201] \"%id. target Dog gene identical to query gene\"                                   \n",
       "[1202] \"%id. query gene identical to target Dog gene\"                                   \n",
       "[1203] \"Dog Gene-order conservation score\"                                              \n",
       "[1204] \"Dog Whole-genome alignment coverage\"                                            \n",
       "[1205] \"dN with Dog\"                                                                    \n",
       "[1206] \"dS with Dog\"                                                                    \n",
       "[1207] \"Dog orthology confidence [0 low, 1 high]\"                                       \n",
       "[1208] \"Dolphin gene stable ID\"                                                         \n",
       "[1209] \"Dolphin gene name\"                                                              \n",
       "[1210] \"Dolphin protein or transcript stable ID\"                                        \n",
       "[1211] \"Dolphin chromosome/scaffold name\"                                               \n",
       "[1212] \"Dolphin chromosome/scaffold start (bp)\"                                         \n",
       "[1213] \"Dolphin chromosome/scaffold end (bp)\"                                           \n",
       "[1214] \"Query protein or transcript ID\"                                                 \n",
       "[1215] \"Last common ancestor with Dolphin\"                                              \n",
       "[1216] \"Dolphin homology type\"                                                          \n",
       "[1217] \"%id. target Dolphin gene identical to query gene\"                               \n",
       "[1218] \"%id. query gene identical to target Dolphin gene\"                               \n",
       "[1219] \"Dolphin Gene-order conservation score\"                                          \n",
       "[1220] \"Dolphin Whole-genome alignment coverage\"                                        \n",
       "[1221] \"dN with Dolphin\"                                                                \n",
       "[1222] \"dS with Dolphin\"                                                                \n",
       "[1223] \"Dolphin orthology confidence [0 low, 1 high]\"                                   \n",
       "[1224] \"Donkey gene stable ID\"                                                          \n",
       "[1225] \"Donkey gene name\"                                                               \n",
       "[1226] \"Donkey protein or transcript stable ID\"                                         \n",
       "[1227] \"Donkey chromosome/scaffold name\"                                                \n",
       "[1228] \"Donkey chromosome/scaffold start (bp)\"                                          \n",
       "[1229] \"Donkey chromosome/scaffold end (bp)\"                                            \n",
       "[1230] \"Query protein or transcript ID\"                                                 \n",
       "[1231] \"Last common ancestor with Donkey\"                                               \n",
       "[1232] \"Donkey homology type\"                                                           \n",
       "[1233] \"%id. target Donkey gene identical to query gene\"                                \n",
       "[1234] \"%id. query gene identical to target Donkey gene\"                                \n",
       "[1235] \"Donkey Gene-order conservation score\"                                           \n",
       "[1236] \"Donkey Whole-genome alignment coverage\"                                         \n",
       "[1237] \"dN with Donkey\"                                                                 \n",
       "[1238] \"dS with Donkey\"                                                                 \n",
       "[1239] \"Donkey orthology confidence [0 low, 1 high]\"                                    \n",
       "[1240] \"Drill gene stable ID\"                                                           \n",
       "[1241] \"Drill gene name\"                                                                \n",
       "[1242] \"Drill protein or transcript stable ID\"                                          \n",
       "[1243] \"Drill chromosome/scaffold name\"                                                 \n",
       "[1244] \"Drill chromosome/scaffold start (bp)\"                                           \n",
       "[1245] \"Drill chromosome/scaffold end (bp)\"                                             \n",
       "[1246] \"Query protein or transcript ID\"                                                 \n",
       "[1247] \"Last common ancestor with Drill\"                                                \n",
       "[1248] \"Drill homology type\"                                                            \n",
       "[1249] \"%id. target Drill gene identical to query gene\"                                 \n",
       "[1250] \"%id. query gene identical to target Drill gene\"                                 \n",
       "[1251] \"Drill Gene-order conservation score\"                                            \n",
       "[1252] \"Drill Whole-genome alignment coverage\"                                          \n",
       "[1253] \"dN with Drill\"                                                                  \n",
       "[1254] \"dS with Drill\"                                                                  \n",
       "[1255] \"Drill orthology confidence [0 low, 1 high]\"                                     \n",
       "[1256] \"Drosophila melanogaster gene stable ID\"                                         \n",
       "[1257] \"Drosophila melanogaster gene name\"                                              \n",
       "[1258] \"Drosophila melanogaster protein or transcript stable ID\"                        \n",
       "[1259] \"Drosophila melanogaster chromosome/scaffold name\"                               \n",
       "[1260] \"Drosophila melanogaster chromosome/scaffold start (bp)\"                         \n",
       "[1261] \"Drosophila melanogaster chromosome/scaffold end (bp)\"                           \n",
       "[1262] \"Query protein or transcript ID\"                                                 \n",
       "[1263] \"Last common ancestor with Drosophila melanogaster\"                              \n",
       "[1264] \"Drosophila melanogaster homology type\"                                          \n",
       "[1265] \"%id. target Drosophila melanogaster gene identical to query gene\"               \n",
       "[1266] \"%id. query gene identical to target Drosophila melanogaster gene\"               \n",
       "[1267] \"Drosophila melanogaster Gene-order conservation score\"                          \n",
       "[1268] \"Drosophila melanogaster Whole-genome alignment coverage\"                        \n",
       "[1269] \"dN with Drosophila melanogaster\"                                                \n",
       "[1270] \"dS with Drosophila melanogaster\"                                                \n",
       "[1271] \"Drosophila melanogaster orthology confidence [0 low, 1 high]\"                   \n",
       "[1272] \"Duck gene stable ID\"                                                            \n",
       "[1273] \"Duck gene name\"                                                                 \n",
       "[1274] \"Duck protein or transcript stable ID\"                                           \n",
       "[1275] \"Duck chromosome/scaffold name\"                                                  \n",
       "[1276] \"Duck chromosome/scaffold start (bp)\"                                            \n",
       "[1277] \"Duck chromosome/scaffold end (bp)\"                                              \n",
       "[1278] \"Query protein or transcript ID\"                                                 \n",
       "[1279] \"Last common ancestor with Duck\"                                                 \n",
       "[1280] \"Duck homology type\"                                                             \n",
       "[1281] \"%id. target Duck gene identical to query gene\"                                  \n",
       "[1282] \"%id. query gene identical to target Duck gene\"                                  \n",
       "[1283] \"Duck Gene-order conservation score\"                                             \n",
       "[1284] \"Duck Whole-genome alignment coverage\"                                           \n",
       "[1285] \"dN with Duck\"                                                                   \n",
       "[1286] \"dS with Duck\"                                                                   \n",
       "[1287] \"Duck orthology confidence [0 low, 1 high]\"                                      \n",
       "[1288] \"Eastern happy gene stable ID\"                                                   \n",
       "[1289] \"Eastern happy gene name\"                                                        \n",
       "[1290] \"Eastern happy protein or transcript stable ID\"                                  \n",
       "[1291] \"Eastern happy chromosome/scaffold name\"                                         \n",
       "[1292] \"Eastern happy chromosome/scaffold start (bp)\"                                   \n",
       "[1293] \"Eastern happy chromosome/scaffold end (bp)\"                                     \n",
       "[1294] \"Query protein or transcript ID\"                                                 \n",
       "[1295] \"Last common ancestor with Eastern happy\"                                        \n",
       "[1296] \"Eastern happy homology type\"                                                    \n",
       "[1297] \"%id. target Eastern happy gene identical to query gene\"                         \n",
       "[1298] \"%id. query gene identical to target Eastern happy gene\"                         \n",
       "[1299] \"Eastern happy Gene-order conservation score\"                                    \n",
       "[1300] \"Eastern happy Whole-genome alignment coverage\"                                  \n",
       "[1301] \"dN with Eastern happy\"                                                          \n",
       "[1302] \"dS with Eastern happy\"                                                          \n",
       "[1303] \"Eastern happy orthology confidence [0 low, 1 high]\"                             \n",
       "[1304] \"Electric eel gene stable ID\"                                                    \n",
       "[1305] \"Electric eel gene name\"                                                         \n",
       "[1306] \"Electric eel protein or transcript stable ID\"                                   \n",
       "[1307] \"Electric eel chromosome/scaffold name\"                                          \n",
       "[1308] \"Electric eel chromosome/scaffold start (bp)\"                                    \n",
       "[1309] \"Electric eel chromosome/scaffold end (bp)\"                                      \n",
       "[1310] \"Query protein or transcript ID\"                                                 \n",
       "[1311] \"Last common ancestor with Electric eel\"                                         \n",
       "[1312] \"Electric eel homology type\"                                                     \n",
       "[1313] \"%id. target Electric eel gene identical to query gene\"                          \n",
       "[1314] \"%id. query gene identical to target Electric eel gene\"                          \n",
       "[1315] \"Electric eel Gene-order conservation score\"                                     \n",
       "[1316] \"Electric eel Whole-genome alignment coverage\"                                   \n",
       "[1317] \"dN with Electric eel\"                                                           \n",
       "[1318] \"dS with Electric eel\"                                                           \n",
       "[1319] \"Electric eel orthology confidence [0 low, 1 high]\"                              \n",
       "[1320] \"Elephant gene stable ID\"                                                        \n",
       "[1321] \"Elephant gene name\"                                                             \n",
       "[1322] \"Elephant protein or transcript stable ID\"                                       \n",
       "[1323] \"Elephant chromosome/scaffold name\"                                              \n",
       "[1324] \"Elephant chromosome/scaffold start (bp)\"                                        \n",
       "[1325] \"Elephant chromosome/scaffold end (bp)\"                                          \n",
       "[1326] \"Query protein or transcript ID\"                                                 \n",
       "[1327] \"Last common ancestor with Elephant\"                                             \n",
       "[1328] \"Elephant homology type\"                                                         \n",
       "[1329] \"%id. target Elephant gene identical to query gene\"                              \n",
       "[1330] \"%id. query gene identical to target Elephant gene\"                              \n",
       "[1331] \"Elephant Gene-order conservation score\"                                         \n",
       "[1332] \"Elephant Whole-genome alignment coverage\"                                       \n",
       "[1333] \"dN with Elephant\"                                                               \n",
       "[1334] \"dS with Elephant\"                                                               \n",
       "[1335] \"Elephant orthology confidence [0 low, 1 high]\"                                  \n",
       "[1336] \"Elephant shark gene stable ID\"                                                  \n",
       "[1337] \"Elephant shark gene name\"                                                       \n",
       "[1338] \"Elephant shark protein or transcript stable ID\"                                 \n",
       "[1339] \"Elephant shark chromosome/scaffold name\"                                        \n",
       "[1340] \"Elephant shark chromosome/scaffold start (bp)\"                                  \n",
       "[1341] \"Elephant shark chromosome/scaffold end (bp)\"                                    \n",
       "[1342] \"Query protein or transcript ID\"                                                 \n",
       "[1343] \"Last common ancestor with Elephant shark\"                                       \n",
       "[1344] \"Elephant shark homology type\"                                                   \n",
       "[1345] \"%id. target Elephant shark gene identical to query gene\"                        \n",
       "[1346] \"%id. query gene identical to target Elephant shark gene\"                        \n",
       "[1347] \"Elephant shark Gene-order conservation score\"                                   \n",
       "[1348] \"Elephant shark Whole-genome alignment coverage\"                                 \n",
       "[1349] \"dN with Elephant shark\"                                                         \n",
       "[1350] \"dS with Elephant shark\"                                                         \n",
       "[1351] \"Elephant shark orthology confidence [0 low, 1 high]\"                            \n",
       "[1352] \"Emu gene stable ID\"                                                             \n",
       "[1353] \"Emu gene name\"                                                                  \n",
       "[1354] \"Emu protein or transcript stable ID\"                                            \n",
       "[1355] \"Emu chromosome/scaffold name\"                                                   \n",
       "[1356] \"Emu chromosome/scaffold start (bp)\"                                             \n",
       "[1357] \"Emu chromosome/scaffold end (bp)\"                                               \n",
       "[1358] \"Query protein or transcript ID\"                                                 \n",
       "[1359] \"Last common ancestor with Emu\"                                                  \n",
       "[1360] \"Emu homology type\"                                                              \n",
       "[1361] \"%id. target Emu gene identical to query gene\"                                   \n",
       "[1362] \"%id. query gene identical to target Emu gene\"                                   \n",
       "[1363] \"Emu Gene-order conservation score\"                                              \n",
       "[1364] \"Emu Whole-genome alignment coverage\"                                            \n",
       "[1365] \"dN with Emu\"                                                                    \n",
       "[1366] \"dS with Emu\"                                                                    \n",
       "[1367] \"Emu orthology confidence [0 low, 1 high]\"                                       \n",
       "[1368] \"Ferret gene stable ID\"                                                          \n",
       "[1369] \"Ferret gene name\"                                                               \n",
       "[1370] \"Ferret protein or transcript stable ID\"                                         \n",
       "[1371] \"Ferret chromosome/scaffold name\"                                                \n",
       "[1372] \"Ferret chromosome/scaffold start (bp)\"                                          \n",
       "[1373] \"Ferret chromosome/scaffold end (bp)\"                                            \n",
       "[1374] \"Query protein or transcript ID\"                                                 \n",
       "[1375] \"Last common ancestor with Ferret\"                                               \n",
       "[1376] \"Ferret homology type\"                                                           \n",
       "[1377] \"%id. target Ferret gene identical to query gene\"                                \n",
       "[1378] \"%id. query gene identical to target Ferret gene\"                                \n",
       "[1379] \"Ferret Gene-order conservation score\"                                           \n",
       "[1380] \"Ferret Whole-genome alignment coverage\"                                         \n",
       "[1381] \"dN with Ferret\"                                                                 \n",
       "[1382] \"dS with Ferret\"                                                                 \n",
       "[1383] \"Ferret orthology confidence [0 low, 1 high]\"                                    \n",
       "[1384] \"Flycatcher gene stable ID\"                                                      \n",
       "[1385] \"Flycatcher gene name\"                                                           \n",
       "[1386] \"Flycatcher protein or transcript stable ID\"                                     \n",
       "[1387] \"Flycatcher chromosome/scaffold name\"                                            \n",
       "[1388] \"Flycatcher chromosome/scaffold start (bp)\"                                      \n",
       "[1389] \"Flycatcher chromosome/scaffold end (bp)\"                                        \n",
       "[1390] \"Query protein or transcript ID\"                                                 \n",
       "[1391] \"Last common ancestor with Flycatcher\"                                           \n",
       "[1392] \"Flycatcher homology type\"                                                       \n",
       "[1393] \"%id. target Flycatcher gene identical to query gene\"                            \n",
       "[1394] \"%id. query gene identical to target Flycatcher gene\"                            \n",
       "[1395] \"Flycatcher Gene-order conservation score\"                                       \n",
       "[1396] \"Flycatcher Whole-genome alignment coverage\"                                     \n",
       "[1397] \"dN with Flycatcher\"                                                             \n",
       "[1398] \"dS with Flycatcher\"                                                             \n",
       "[1399] \"Flycatcher orthology confidence [0 low, 1 high]\"                                \n",
       "[1400] \"Fugu gene stable ID\"                                                            \n",
       "[1401] \"Fugu gene name\"                                                                 \n",
       "[1402] \"Fugu protein or transcript stable ID\"                                           \n",
       "[1403] \"Fugu chromosome/scaffold name\"                                                  \n",
       "[1404] \"Fugu chromosome/scaffold start (bp)\"                                            \n",
       "[1405] \"Fugu chromosome/scaffold end (bp)\"                                              \n",
       "[1406] \"Query protein or transcript ID\"                                                 \n",
       "[1407] \"Last common ancestor with Fugu\"                                                 \n",
       "[1408] \"Fugu homology type\"                                                             \n",
       "[1409] \"%id. target Fugu gene identical to query gene\"                                  \n",
       "[1410] \"%id. query gene identical to target Fugu gene\"                                  \n",
       "[1411] \"Fugu Gene-order conservation score\"                                             \n",
       "[1412] \"Fugu Whole-genome alignment coverage\"                                           \n",
       "[1413] \"dN with Fugu\"                                                                   \n",
       "[1414] \"dS with Fugu\"                                                                   \n",
       "[1415] \"Fugu orthology confidence [0 low, 1 high]\"                                      \n",
       "[1416] \"Gelada gene stable ID\"                                                          \n",
       "[1417] \"Gelada gene name\"                                                               \n",
       "[1418] \"Gelada protein or transcript stable ID\"                                         \n",
       "[1419] \"Gelada chromosome/scaffold name\"                                                \n",
       "[1420] \"Gelada chromosome/scaffold start (bp)\"                                          \n",
       "[1421] \"Gelada chromosome/scaffold end (bp)\"                                            \n",
       "[1422] \"Query protein or transcript ID\"                                                 \n",
       "[1423] \"Last common ancestor with Gelada\"                                               \n",
       "[1424] \"Gelada homology type\"                                                           \n",
       "[1425] \"%id. target Gelada gene identical to query gene\"                                \n",
       "[1426] \"%id. query gene identical to target Gelada gene\"                                \n",
       "[1427] \"Gelada Gene-order conservation score\"                                           \n",
       "[1428] \"Gelada Whole-genome alignment coverage\"                                         \n",
       "[1429] \"dN with Gelada\"                                                                 \n",
       "[1430] \"dS with Gelada\"                                                                 \n",
       "[1431] \"Gelada orthology confidence [0 low, 1 high]\"                                    \n",
       "[1432] \"Gibbon gene stable ID\"                                                          \n",
       "[1433] \"Gibbon gene name\"                                                               \n",
       "[1434] \"Gibbon protein or transcript stable ID\"                                         \n",
       "[1435] \"Gibbon chromosome/scaffold name\"                                                \n",
       "[1436] \"Gibbon chromosome/scaffold start (bp)\"                                          \n",
       "[1437] \"Gibbon chromosome/scaffold end (bp)\"                                            \n",
       "[1438] \"Query protein or transcript ID\"                                                 \n",
       "[1439] \"Last common ancestor with Gibbon\"                                               \n",
       "[1440] \"Gibbon homology type\"                                                           \n",
       "[1441] \"%id. target Gibbon gene identical to query gene\"                                \n",
       "[1442] \"%id. query gene identical to target Gibbon gene\"                                \n",
       "[1443] \"Gibbon Gene-order conservation score\"                                           \n",
       "[1444] \"Gibbon Whole-genome alignment coverage\"                                         \n",
       "[1445] \"dN with Gibbon\"                                                                 \n",
       "[1446] \"dS with Gibbon\"                                                                 \n",
       "[1447] \"Gibbon orthology confidence [0 low, 1 high]\"                                    \n",
       "[1448] \"Goat gene stable ID\"                                                            \n",
       "[1449] \"Goat gene name\"                                                                 \n",
       "[1450] \"Goat protein or transcript stable ID\"                                           \n",
       "[1451] \"Goat chromosome/scaffold name\"                                                  \n",
       "[1452] \"Goat chromosome/scaffold start (bp)\"                                            \n",
       "[1453] \"Goat chromosome/scaffold end (bp)\"                                              \n",
       "[1454] \"Query protein or transcript ID\"                                                 \n",
       "[1455] \"Last common ancestor with Goat\"                                                 \n",
       "[1456] \"Goat homology type\"                                                             \n",
       "[1457] \"%id. target Goat gene identical to query gene\"                                  \n",
       "[1458] \"%id. query gene identical to target Goat gene\"                                  \n",
       "[1459] \"Goat Gene-order conservation score\"                                             \n",
       "[1460] \"Goat Whole-genome alignment coverage\"                                           \n",
       "[1461] \"dN with Goat\"                                                                   \n",
       "[1462] \"dS with Goat\"                                                                   \n",
       "[1463] \"Goat orthology confidence [0 low, 1 high]\"                                      \n",
       "[1464] \"Golden Hamster gene stable ID\"                                                  \n",
       "[1465] \"Golden Hamster gene name\"                                                       \n",
       "[1466] \"Golden Hamster protein or transcript stable ID\"                                 \n",
       "[1467] \"Golden Hamster chromosome/scaffold name\"                                        \n",
       "[1468] \"Golden Hamster chromosome/scaffold start (bp)\"                                  \n",
       "[1469] \"Golden Hamster chromosome/scaffold end (bp)\"                                    \n",
       "[1470] \"Query protein or transcript ID\"                                                 \n",
       "[1471] \"Last common ancestor with Golden Hamster\"                                       \n",
       "[1472] \"Golden Hamster homology type\"                                                   \n",
       "[1473] \"%id. target Golden Hamster gene identical to query gene\"                        \n",
       "[1474] \"%id. query gene identical to target Golden Hamster gene\"                        \n",
       "[1475] \"Golden Hamster Gene-order conservation score\"                                   \n",
       "[1476] \"Golden Hamster Whole-genome alignment coverage\"                                 \n",
       "[1477] \"dN with Golden Hamster\"                                                         \n",
       "[1478] \"dS with Golden Hamster\"                                                         \n",
       "[1479] \"Golden Hamster orthology confidence [0 low, 1 high]\"                            \n",
       "[1480] \"Golden snub-nosed monkey gene stable ID\"                                        \n",
       "[1481] \"Golden snub-nosed monkey gene name\"                                             \n",
       "[1482] \"Golden snub-nosed monkey protein or transcript stable ID\"                       \n",
       "[1483] \"Golden snub-nosed monkey chromosome/scaffold name\"                              \n",
       "[1484] \"Golden snub-nosed monkey chromosome/scaffold start (bp)\"                        \n",
       "[1485] \"Golden snub-nosed monkey chromosome/scaffold end (bp)\"                          \n",
       "[1486] \"Query protein or transcript ID\"                                                 \n",
       "[1487] \"Last common ancestor with Golden snub-nosed monkey\"                             \n",
       "[1488] \"Golden snub-nosed monkey homology type\"                                         \n",
       "[1489] \"%id. target Golden snub-nosed monkey gene identical to query gene\"              \n",
       "[1490] \"%id. query gene identical to target Golden snub-nosed monkey gene\"              \n",
       "[1491] \"Golden snub-nosed monkey Gene-order conservation score\"                         \n",
       "[1492] \"Golden snub-nosed monkey Whole-genome alignment coverage\"                       \n",
       "[1493] \"dN with Golden snub-nosed monkey\"                                               \n",
       "[1494] \"dS with Golden snub-nosed monkey\"                                               \n",
       "[1495] \"Golden snub-nosed monkey orthology confidence [0 low, 1 high]\"                  \n",
       "[1496] \"Golden-collared manakin gene stable ID\"                                         \n",
       "[1497] \"Golden-collared manakin gene name\"                                              \n",
       "[1498] \"Golden-collared manakin protein or transcript stable ID\"                        \n",
       "[1499] \"Golden-collared manakin chromosome/scaffold name\"                               \n",
       "[1500] \"Golden-collared manakin chromosome/scaffold start (bp)\"                         \n",
       "[1501] \"Golden-collared manakin chromosome/scaffold end (bp)\"                           \n",
       "[1502] \"Query protein or transcript ID\"                                                 \n",
       "[1503] \"Last common ancestor with Golden-collared manakin\"                              \n",
       "[1504] \"Golden-collared manakin homology type\"                                          \n",
       "[1505] \"%id. target Golden-collared manakin gene identical to query gene\"               \n",
       "[1506] \"%id. query gene identical to target Golden-collared manakin gene\"               \n",
       "[1507] \"Golden-collared manakin Gene-order conservation score\"                          \n",
       "[1508] \"Golden-collared manakin Whole-genome alignment coverage\"                        \n",
       "[1509] \"dN with Golden-collared manakin\"                                                \n",
       "[1510] \"dS with Golden-collared manakin\"                                                \n",
       "[1511] \"Golden-collared manakin orthology confidence [0 low, 1 high]\"                   \n",
       "[1512] \"Gorilla gene stable ID\"                                                         \n",
       "[1513] \"Gorilla gene name\"                                                              \n",
       "[1514] \"Gorilla protein or transcript stable ID\"                                        \n",
       "[1515] \"Gorilla chromosome/scaffold name\"                                               \n",
       "[1516] \"Gorilla chromosome/scaffold start (bp)\"                                         \n",
       "[1517] \"Gorilla chromosome/scaffold end (bp)\"                                           \n",
       "[1518] \"Query protein or transcript ID\"                                                 \n",
       "[1519] \"Last common ancestor with Gorilla\"                                              \n",
       "[1520] \"Gorilla homology type\"                                                          \n",
       "[1521] \"%id. target Gorilla gene identical to query gene\"                               \n",
       "[1522] \"%id. query gene identical to target Gorilla gene\"                               \n",
       "[1523] \"Gorilla Gene-order conservation score\"                                          \n",
       "[1524] \"Gorilla Whole-genome alignment coverage\"                                        \n",
       "[1525] \"dN with Gorilla\"                                                                \n",
       "[1526] \"dS with Gorilla\"                                                                \n",
       "[1527] \"Gorilla orthology confidence [0 low, 1 high]\"                                   \n",
       "[1528] \"Great Tit gene stable ID\"                                                       \n",
       "[1529] \"Great Tit gene name\"                                                            \n",
       "[1530] \"Great Tit protein or transcript stable ID\"                                      \n",
       "[1531] \"Great Tit chromosome/scaffold name\"                                             \n",
       "[1532] \"Great Tit chromosome/scaffold start (bp)\"                                       \n",
       "[1533] \"Great Tit chromosome/scaffold end (bp)\"                                         \n",
       "[1534] \"Query protein or transcript ID\"                                                 \n",
       "[1535] \"Last common ancestor with Great Tit\"                                            \n",
       "[1536] \"Great Tit homology type\"                                                        \n",
       "[1537] \"%id. target Great Tit gene identical to query gene\"                             \n",
       "[1538] \"%id. query gene identical to target Great Tit gene\"                             \n",
       "[1539] \"Great Tit Gene-order conservation score\"                                        \n",
       "[1540] \"Great Tit Whole-genome alignment coverage\"                                      \n",
       "[1541] \"dN with Great Tit\"                                                              \n",
       "[1542] \"dS with Great Tit\"                                                              \n",
       "[1543] \"Great Tit orthology confidence [0 low, 1 high]\"                                 \n",
       "[1544] \"Great spotted kiwi gene stable ID\"                                              \n",
       "[1545] \"Great spotted kiwi gene name\"                                                   \n",
       "[1546] \"Great spotted kiwi protein or transcript stable ID\"                             \n",
       "[1547] \"Great spotted kiwi chromosome/scaffold name\"                                    \n",
       "[1548] \"Great spotted kiwi chromosome/scaffold start (bp)\"                              \n",
       "[1549] \"Great spotted kiwi chromosome/scaffold end (bp)\"                                \n",
       "[1550] \"Query protein or transcript ID\"                                                 \n",
       "[1551] \"Last common ancestor with Great spotted kiwi\"                                   \n",
       "[1552] \"Great spotted kiwi homology type\"                                               \n",
       "[1553] \"%id. target Great spotted kiwi gene identical to query gene\"                    \n",
       "[1554] \"%id. query gene identical to target Great spotted kiwi gene\"                    \n",
       "[1555] \"Great spotted kiwi Gene-order conservation score\"                               \n",
       "[1556] \"Great spotted kiwi Whole-genome alignment coverage\"                             \n",
       "[1557] \"dN with Great spotted kiwi\"                                                     \n",
       "[1558] \"dS with Great spotted kiwi\"                                                     \n",
       "[1559] \"Great spotted kiwi orthology confidence [0 low, 1 high]\"                        \n",
       "[1560] \"Greater amberjack gene stable ID\"                                               \n",
       "[1561] \"Greater amberjack gene name\"                                                    \n",
       "[1562] \"Greater amberjack protein or transcript stable ID\"                              \n",
       "[1563] \"Greater amberjack chromosome/scaffold name\"                                     \n",
       "[1564] \"Greater amberjack chromosome/scaffold start (bp)\"                               \n",
       "[1565] \"Greater amberjack chromosome/scaffold end (bp)\"                                 \n",
       "[1566] \"Query protein or transcript ID\"                                                 \n",
       "[1567] \"Last common ancestor with Greater amberjack\"                                    \n",
       "[1568] \"Greater amberjack homology type\"                                                \n",
       "[1569] \"%id. target Greater amberjack gene identical to query gene\"                     \n",
       "[1570] \"%id. query gene identical to target Greater amberjack gene\"                     \n",
       "[1571] \"Greater amberjack Gene-order conservation score\"                                \n",
       "[1572] \"Greater amberjack Whole-genome alignment coverage\"                              \n",
       "[1573] \"dN with Greater amberjack\"                                                      \n",
       "[1574] \"dS with Greater amberjack\"                                                      \n",
       "[1575] \"Greater amberjack orthology confidence [0 low, 1 high]\"                         \n",
       "[1576] \"Greater bamboo lemur gene stable ID\"                                            \n",
       "[1577] \"Greater bamboo lemur gene name\"                                                 \n",
       "[1578] \"Greater bamboo lemur protein or transcript stable ID\"                           \n",
       "[1579] \"Greater bamboo lemur chromosome/scaffold name\"                                  \n",
       "[1580] \"Greater bamboo lemur chromosome/scaffold start (bp)\"                            \n",
       "[1581] \"Greater bamboo lemur chromosome/scaffold end (bp)\"                              \n",
       "[1582] \"Query protein or transcript ID\"                                                 \n",
       "[1583] \"Last common ancestor with Greater bamboo lemur\"                                 \n",
       "[1584] \"Greater bamboo lemur homology type\"                                             \n",
       "[1585] \"%id. target Greater bamboo lemur gene identical to query gene\"                  \n",
       "[1586] \"%id. query gene identical to target Greater bamboo lemur gene\"                  \n",
       "[1587] \"Greater bamboo lemur Gene-order conservation score\"                             \n",
       "[1588] \"Greater bamboo lemur Whole-genome alignment coverage\"                           \n",
       "[1589] \"dN with Greater bamboo lemur\"                                                   \n",
       "[1590] \"dS with Greater bamboo lemur\"                                                   \n",
       "[1591] \"Greater bamboo lemur orthology confidence [0 low, 1 high]\"                      \n",
       "[1592] \"Guinea Pig gene stable ID\"                                                      \n",
       "[1593] \"Guinea Pig gene name\"                                                           \n",
       "[1594] \"Guinea Pig protein or transcript stable ID\"                                     \n",
       "[1595] \"Guinea Pig chromosome/scaffold name\"                                            \n",
       "[1596] \"Guinea Pig chromosome/scaffold start (bp)\"                                      \n",
       "[1597] \"Guinea Pig chromosome/scaffold end (bp)\"                                        \n",
       "[1598] \"Query protein or transcript ID\"                                                 \n",
       "[1599] \"Last common ancestor with Guinea Pig\"                                           \n",
       "[1600] \"Guinea Pig homology type\"                                                       \n",
       "[1601] \"%id. target Guinea Pig gene identical to query gene\"                            \n",
       "[1602] \"%id. query gene identical to target Guinea Pig gene\"                            \n",
       "[1603] \"Guinea Pig Gene-order conservation score\"                                       \n",
       "[1604] \"Guinea Pig Whole-genome alignment coverage\"                                     \n",
       "[1605] \"dN with Guinea Pig\"                                                             \n",
       "[1606] \"dS with Guinea Pig\"                                                             \n",
       "[1607] \"Guinea Pig orthology confidence [0 low, 1 high]\"                                \n",
       "[1608] \"Guppy gene stable ID\"                                                           \n",
       "[1609] \"Guppy gene name\"                                                                \n",
       "[1610] \"Guppy protein or transcript stable ID\"                                          \n",
       "[1611] \"Guppy chromosome/scaffold name\"                                                 \n",
       "[1612] \"Guppy chromosome/scaffold start (bp)\"                                           \n",
       "[1613] \"Guppy chromosome/scaffold end (bp)\"                                             \n",
       "[1614] \"Query protein or transcript ID\"                                                 \n",
       "[1615] \"Last common ancestor with Guppy\"                                                \n",
       "[1616] \"Guppy homology type\"                                                            \n",
       "[1617] \"%id. target Guppy gene identical to query gene\"                                 \n",
       "[1618] \"%id. query gene identical to target Guppy gene\"                                 \n",
       "[1619] \"Guppy Gene-order conservation score\"                                            \n",
       "[1620] \"Guppy Whole-genome alignment coverage\"                                          \n",
       "[1621] \"dN with Guppy\"                                                                  \n",
       "[1622] \"dS with Guppy\"                                                                  \n",
       "[1623] \"Guppy orthology confidence [0 low, 1 high]\"                                     \n",
       "[1624] \"Hagfish gene stable ID\"                                                         \n",
       "[1625] \"Hagfish gene name\"                                                              \n",
       "[1626] \"Hagfish protein or transcript stable ID\"                                        \n",
       "[1627] \"Hagfish chromosome/scaffold name\"                                               \n",
       "[1628] \"Hagfish chromosome/scaffold start (bp)\"                                         \n",
       "[1629] \"Hagfish chromosome/scaffold end (bp)\"                                           \n",
       "[1630] \"Query protein or transcript ID\"                                                 \n",
       "[1631] \"Last common ancestor with Hagfish\"                                              \n",
       "[1632] \"Hagfish homology type\"                                                          \n",
       "[1633] \"%id. target Hagfish gene identical to query gene\"                               \n",
       "[1634] \"%id. query gene identical to target Hagfish gene\"                               \n",
       "[1635] \"Hagfish Gene-order conservation score\"                                          \n",
       "[1636] \"Hagfish Whole-genome alignment coverage\"                                        \n",
       "[1637] \"dN with Hagfish\"                                                                \n",
       "[1638] \"dS with Hagfish\"                                                                \n",
       "[1639] \"Hagfish orthology confidence [0 low, 1 high]\"                                   \n",
       "[1640] \"Hedgehog gene stable ID\"                                                        \n",
       "[1641] \"Hedgehog gene name\"                                                             \n",
       "[1642] \"Hedgehog protein or transcript stable ID\"                                       \n",
       "[1643] \"Hedgehog chromosome/scaffold name\"                                              \n",
       "[1644] \"Hedgehog chromosome/scaffold start (bp)\"                                        \n",
       "[1645] \"Hedgehog chromosome/scaffold end (bp)\"                                          \n",
       "[1646] \"Query protein or transcript ID\"                                                 \n",
       "[1647] \"Last common ancestor with Hedgehog\"                                             \n",
       "[1648] \"Hedgehog homology type\"                                                         \n",
       "[1649] \"%id. target Hedgehog gene identical to query gene\"                              \n",
       "[1650] \"%id. query gene identical to target Hedgehog gene\"                              \n",
       "[1651] \"Hedgehog Gene-order conservation score\"                                         \n",
       "[1652] \"Hedgehog Whole-genome alignment coverage\"                                       \n",
       "[1653] \"dN with Hedgehog\"                                                               \n",
       "[1654] \"dS with Hedgehog\"                                                               \n",
       "[1655] \"Hedgehog orthology confidence [0 low, 1 high]\"                                  \n",
       "[1656] \"Helmeted guineafowl gene stable ID\"                                             \n",
       "[1657] \"Helmeted guineafowl gene name\"                                                  \n",
       "[1658] \"Helmeted guineafowl protein or transcript stable ID\"                            \n",
       "[1659] \"Helmeted guineafowl chromosome/scaffold name\"                                   \n",
       "[1660] \"Helmeted guineafowl chromosome/scaffold start (bp)\"                             \n",
       "[1661] \"Helmeted guineafowl chromosome/scaffold end (bp)\"                               \n",
       "[1662] \"Query protein or transcript ID\"                                                 \n",
       "[1663] \"Last common ancestor with Helmeted guineafowl\"                                  \n",
       "[1664] \"Helmeted guineafowl homology type\"                                              \n",
       "[1665] \"%id. target Helmeted guineafowl gene identical to query gene\"                   \n",
       "[1666] \"%id. query gene identical to target Helmeted guineafowl gene\"                   \n",
       "[1667] \"Helmeted guineafowl Gene-order conservation score\"                              \n",
       "[1668] \"Helmeted guineafowl Whole-genome alignment coverage\"                            \n",
       "[1669] \"dN with Helmeted guineafowl\"                                                    \n",
       "[1670] \"dS with Helmeted guineafowl\"                                                    \n",
       "[1671] \"Helmeted guineafowl orthology confidence [0 low, 1 high]\"                       \n",
       "[1672] \"Horse gene stable ID\"                                                           \n",
       "[1673] \"Horse gene name\"                                                                \n",
       "[1674] \"Horse protein or transcript stable ID\"                                          \n",
       "[1675] \"Horse chromosome/scaffold name\"                                                 \n",
       "[1676] \"Horse chromosome/scaffold start (bp)\"                                           \n",
       "[1677] \"Horse chromosome/scaffold end (bp)\"                                             \n",
       "[1678] \"Query protein or transcript ID\"                                                 \n",
       "[1679] \"Last common ancestor with Horse\"                                                \n",
       "[1680] \"Horse homology type\"                                                            \n",
       "[1681] \"%id. target Horse gene identical to query gene\"                                 \n",
       "[1682] \"%id. query gene identical to target Horse gene\"                                 \n",
       "[1683] \"Horse Gene-order conservation score\"                                            \n",
       "[1684] \"Horse Whole-genome alignment coverage\"                                          \n",
       "[1685] \"dN with Horse\"                                                                  \n",
       "[1686] \"dS with Horse\"                                                                  \n",
       "[1687] \"Horse orthology confidence [0 low, 1 high]\"                                     \n",
       "[1688] \"Huchen gene stable ID\"                                                          \n",
       "[1689] \"Huchen gene name\"                                                               \n",
       "[1690] \"Huchen protein or transcript stable ID\"                                         \n",
       "[1691] \"Huchen chromosome/scaffold name\"                                                \n",
       "[1692] \"Huchen chromosome/scaffold start (bp)\"                                          \n",
       "[1693] \"Huchen chromosome/scaffold end (bp)\"                                            \n",
       "[1694] \"Query protein or transcript ID\"                                                 \n",
       "[1695] \"Last common ancestor with Huchen\"                                               \n",
       "[1696] \"Huchen homology type\"                                                           \n",
       "[1697] \"%id. target Huchen gene identical to query gene\"                                \n",
       "[1698] \"%id. query gene identical to target Huchen gene\"                                \n",
       "[1699] \"Huchen Gene-order conservation score\"                                           \n",
       "[1700] \"Huchen Whole-genome alignment coverage\"                                         \n",
       "[1701] \"dN with Huchen\"                                                                 \n",
       "[1702] \"dS with Huchen\"                                                                 \n",
       "[1703] \"Huchen orthology confidence [0 low, 1 high]\"                                    \n",
       "[1704] \"Hyrax gene stable ID\"                                                           \n",
       "[1705] \"Hyrax gene name\"                                                                \n",
       "[1706] \"Hyrax protein or transcript stable ID\"                                          \n",
       "[1707] \"Hyrax chromosome/scaffold name\"                                                 \n",
       "[1708] \"Hyrax chromosome/scaffold start (bp)\"                                           \n",
       "[1709] \"Hyrax chromosome/scaffold end (bp)\"                                             \n",
       "[1710] \"Query protein or transcript ID\"                                                 \n",
       "[1711] \"Last common ancestor with Hyrax\"                                                \n",
       "[1712] \"Hyrax homology type\"                                                            \n",
       "[1713] \"%id. target Hyrax gene identical to query gene\"                                 \n",
       "[1714] \"%id. query gene identical to target Hyrax gene\"                                 \n",
       "[1715] \"Hyrax Gene-order conservation score\"                                            \n",
       "[1716] \"Hyrax Whole-genome alignment coverage\"                                          \n",
       "[1717] \"dN with Hyrax\"                                                                  \n",
       "[1718] \"dS with Hyrax\"                                                                  \n",
       "[1719] \"Hyrax orthology confidence [0 low, 1 high]\"                                     \n",
       "[1720] \"Indian medaka gene stable ID\"                                                   \n",
       "[1721] \"Indian medaka gene name\"                                                        \n",
       "[1722] \"Indian medaka protein or transcript stable ID\"                                  \n",
       "[1723] \"Indian medaka chromosome/scaffold name\"                                         \n",
       "[1724] \"Indian medaka chromosome/scaffold start (bp)\"                                   \n",
       "[1725] \"Indian medaka chromosome/scaffold end (bp)\"                                     \n",
       "[1726] \"Query protein or transcript ID\"                                                 \n",
       "[1727] \"Last common ancestor with Indian medaka\"                                        \n",
       "[1728] \"Indian medaka homology type\"                                                    \n",
       "[1729] \"%id. target Indian medaka gene identical to query gene\"                         \n",
       "[1730] \"%id. query gene identical to target Indian medaka gene\"                         \n",
       "[1731] \"Indian medaka Gene-order conservation score\"                                    \n",
       "[1732] \"Indian medaka Whole-genome alignment coverage\"                                  \n",
       "[1733] \"dN with Indian medaka\"                                                          \n",
       "[1734] \"dS with Indian medaka\"                                                          \n",
       "[1735] \"Indian medaka orthology confidence [0 low, 1 high]\"                             \n",
       "[1736] \"Japanese medaka HNI gene stable ID\"                                             \n",
       "[1737] \"Japanese medaka HNI gene name\"                                                  \n",
       "[1738] \"Japanese medaka HNI protein or transcript stable ID\"                            \n",
       "[1739] \"Japanese medaka HNI chromosome/scaffold name\"                                   \n",
       "[1740] \"Japanese medaka HNI chromosome/scaffold start (bp)\"                             \n",
       "[1741] \"Japanese medaka HNI chromosome/scaffold end (bp)\"                               \n",
       "[1742] \"Query protein or transcript ID\"                                                 \n",
       "[1743] \"Last common ancestor with Japanese medaka HNI\"                                  \n",
       "[1744] \"Japanese medaka HNI homology type\"                                              \n",
       "[1745] \"%id. target Japanese medaka HNI gene identical to query gene\"                   \n",
       "[1746] \"%id. query gene identical to target Japanese medaka HNI gene\"                   \n",
       "[1747] \"Japanese medaka HNI Gene-order conservation score\"                              \n",
       "[1748] \"Japanese medaka HNI Whole-genome alignment coverage\"                            \n",
       "[1749] \"dN with Japanese medaka HNI\"                                                    \n",
       "[1750] \"dS with Japanese medaka HNI\"                                                    \n",
       "[1751] \"Japanese medaka HNI orthology confidence [0 low, 1 high]\"                       \n",
       "[1752] \"Japanese medaka HSOK gene stable ID\"                                            \n",
       "[1753] \"Japanese medaka HSOK gene name\"                                                 \n",
       "[1754] \"Japanese medaka HSOK protein or transcript stable ID\"                           \n",
       "[1755] \"Japanese medaka HSOK chromosome/scaffold name\"                                  \n",
       "[1756] \"Japanese medaka HSOK chromosome/scaffold start (bp)\"                            \n",
       "[1757] \"Japanese medaka HSOK chromosome/scaffold end (bp)\"                              \n",
       "[1758] \"Query protein or transcript ID\"                                                 \n",
       "[1759] \"Last common ancestor with Japanese medaka HSOK\"                                 \n",
       "[1760] \"Japanese medaka HSOK homology type\"                                             \n",
       "[1761] \"%id. target Japanese medaka HSOK gene identical to query gene\"                  \n",
       "[1762] \"%id. query gene identical to target Japanese medaka HSOK gene\"                  \n",
       "[1763] \"Japanese medaka HSOK Gene-order conservation score\"                             \n",
       "[1764] \"Japanese medaka HSOK Whole-genome alignment coverage\"                           \n",
       "[1765] \"dN with Japanese medaka HSOK\"                                                   \n",
       "[1766] \"dS with Japanese medaka HSOK\"                                                   \n",
       "[1767] \"Japanese medaka HSOK orthology confidence [0 low, 1 high]\"                      \n",
       "[1768] \"Japanese medaka HdrR gene stable ID\"                                            \n",
       "[1769] \"Japanese medaka HdrR gene name\"                                                 \n",
       "[1770] \"Japanese medaka HdrR protein or transcript stable ID\"                           \n",
       "[1771] \"Japanese medaka HdrR chromosome/scaffold name\"                                  \n",
       "[1772] \"Japanese medaka HdrR chromosome/scaffold start (bp)\"                            \n",
       "[1773] \"Japanese medaka HdrR chromosome/scaffold end (bp)\"                              \n",
       "[1774] \"Query protein or transcript ID\"                                                 \n",
       "[1775] \"Last common ancestor with Japanese medaka HdrR\"                                 \n",
       "[1776] \"Japanese medaka HdrR homology type\"                                             \n",
       "[1777] \"%id. target Japanese medaka HdrR gene identical to query gene\"                  \n",
       "[1778] \"%id. query gene identical to target Japanese medaka HdrR gene\"                  \n",
       "[1779] \"Japanese medaka HdrR Gene-order conservation score\"                             \n",
       "[1780] \"Japanese medaka HdrR Whole-genome alignment coverage\"                           \n",
       "[1781] \"dN with Japanese medaka HdrR\"                                                   \n",
       "[1782] \"dS with Japanese medaka HdrR\"                                                   \n",
       "[1783] \"Japanese medaka HdrR orthology confidence [0 low, 1 high]\"                      \n",
       "[1784] \"Japanese quail gene stable ID\"                                                  \n",
       "[1785] \"Japanese quail gene name\"                                                       \n",
       "[1786] \"Japanese quail protein or transcript stable ID\"                                 \n",
       "[1787] \"Japanese quail chromosome/scaffold name\"                                        \n",
       "[1788] \"Japanese quail chromosome/scaffold start (bp)\"                                  \n",
       "[1789] \"Japanese quail chromosome/scaffold end (bp)\"                                    \n",
       "[1790] \"Query protein or transcript ID\"                                                 \n",
       "[1791] \"Last common ancestor with Japanese quail\"                                       \n",
       "[1792] \"Japanese quail homology type\"                                                   \n",
       "[1793] \"%id. target Japanese quail gene identical to query gene\"                        \n",
       "[1794] \"%id. query gene identical to target Japanese quail gene\"                        \n",
       "[1795] \"Japanese quail Gene-order conservation score\"                                   \n",
       "[1796] \"Japanese quail Whole-genome alignment coverage\"                                 \n",
       "[1797] \"dN with Japanese quail\"                                                         \n",
       "[1798] \"dS with Japanese quail\"                                                         \n",
       "[1799] \"Japanese quail orthology confidence [0 low, 1 high]\"                            \n",
       "[1800] \"Kangaroo rat gene stable ID\"                                                    \n",
       "[1801] \"Kangaroo rat gene name\"                                                         \n",
       "[1802] \"Kangaroo rat protein or transcript stable ID\"                                   \n",
       "[1803] \"Kangaroo rat chromosome/scaffold name\"                                          \n",
       "[1804] \"Kangaroo rat chromosome/scaffold start (bp)\"                                    \n",
       "[1805] \"Kangaroo rat chromosome/scaffold end (bp)\"                                      \n",
       "[1806] \"Query protein or transcript ID\"                                                 \n",
       "[1807] \"Last common ancestor with Kangaroo rat\"                                         \n",
       "[1808] \"Kangaroo rat homology type\"                                                     \n",
       "[1809] \"%id. target Kangaroo rat gene identical to query gene\"                          \n",
       "[1810] \"%id. query gene identical to target Kangaroo rat gene\"                          \n",
       "[1811] \"Kangaroo rat Gene-order conservation score\"                                     \n",
       "[1812] \"Kangaroo rat Whole-genome alignment coverage\"                                   \n",
       "[1813] \"dN with Kangaroo rat\"                                                           \n",
       "[1814] \"dS with Kangaroo rat\"                                                           \n",
       "[1815] \"Kangaroo rat orthology confidence [0 low, 1 high]\"                              \n",
       "[1816] \"Koala gene stable ID\"                                                           \n",
       "[1817] \"Koala gene name\"                                                                \n",
       "[1818] \"Koala protein or transcript stable ID\"                                          \n",
       "[1819] \"Koala chromosome/scaffold name\"                                                 \n",
       "[1820] \"Koala chromosome/scaffold start (bp)\"                                           \n",
       "[1821] \"Koala chromosome/scaffold end (bp)\"                                             \n",
       "[1822] \"Query protein or transcript ID\"                                                 \n",
       "[1823] \"Last common ancestor with Koala\"                                                \n",
       "[1824] \"Koala homology type\"                                                            \n",
       "[1825] \"%id. target Koala gene identical to query gene\"                                 \n",
       "[1826] \"%id. query gene identical to target Koala gene\"                                 \n",
       "[1827] \"Koala Gene-order conservation score\"                                            \n",
       "[1828] \"Koala Whole-genome alignment coverage\"                                          \n",
       "[1829] \"dN with Koala\"                                                                  \n",
       "[1830] \"dS with Koala\"                                                                  \n",
       "[1831] \"Koala orthology confidence [0 low, 1 high]\"                                     \n",
       "[1832] \"Lamprey gene stable ID\"                                                         \n",
       "[1833] \"Lamprey gene name\"                                                              \n",
       "[1834] \"Lamprey protein or transcript stable ID\"                                        \n",
       "[1835] \"Lamprey chromosome/scaffold name\"                                               \n",
       "[1836] \"Lamprey chromosome/scaffold start (bp)\"                                         \n",
       "[1837] \"Lamprey chromosome/scaffold end (bp)\"                                           \n",
       "[1838] \"Query protein or transcript ID\"                                                 \n",
       "[1839] \"Last common ancestor with Lamprey\"                                              \n",
       "[1840] \"Lamprey homology type\"                                                          \n",
       "[1841] \"%id. target Lamprey gene identical to query gene\"                               \n",
       "[1842] \"%id. query gene identical to target Lamprey gene\"                               \n",
       "[1843] \"Lamprey Gene-order conservation score\"                                          \n",
       "[1844] \"Lamprey Whole-genome alignment coverage\"                                        \n",
       "[1845] \"dN with Lamprey\"                                                                \n",
       "[1846] \"dS with Lamprey\"                                                                \n",
       "[1847] \"Lamprey orthology confidence [0 low, 1 high]\"                                   \n",
       "[1848] \"Leopard gene stable ID\"                                                         \n",
       "[1849] \"Leopard gene name\"                                                              \n",
       "[1850] \"Leopard protein or transcript stable ID\"                                        \n",
       "[1851] \"Leopard chromosome/scaffold name\"                                               \n",
       "[1852] \"Leopard chromosome/scaffold start (bp)\"                                         \n",
       "[1853] \"Leopard chromosome/scaffold end (bp)\"                                           \n",
       "[1854] \"Query protein or transcript ID\"                                                 \n",
       "[1855] \"Last common ancestor with Leopard\"                                              \n",
       "[1856] \"Leopard homology type\"                                                          \n",
       "[1857] \"%id. target Leopard gene identical to query gene\"                               \n",
       "[1858] \"%id. query gene identical to target Leopard gene\"                               \n",
       "[1859] \"Leopard Gene-order conservation score\"                                          \n",
       "[1860] \"Leopard Whole-genome alignment coverage\"                                        \n",
       "[1861] \"dN with Leopard\"                                                                \n",
       "[1862] \"dS with Leopard\"                                                                \n",
       "[1863] \"Leopard orthology confidence [0 low, 1 high]\"                                   \n",
       "[1864] \"Lesser Egyptian jerboa gene stable ID\"                                          \n",
       "[1865] \"Lesser Egyptian jerboa gene name\"                                               \n",
       "[1866] \"Lesser Egyptian jerboa protein or transcript stable ID\"                         \n",
       "[1867] \"Lesser Egyptian jerboa chromosome/scaffold name\"                                \n",
       "[1868] \"Lesser Egyptian jerboa chromosome/scaffold start (bp)\"                          \n",
       "[1869] \"Lesser Egyptian jerboa chromosome/scaffold end (bp)\"                            \n",
       "[1870] \"Query protein or transcript ID\"                                                 \n",
       "[1871] \"Last common ancestor with Lesser Egyptian jerboa\"                               \n",
       "[1872] \"Lesser Egyptian jerboa homology type\"                                           \n",
       "[1873] \"%id. target Lesser Egyptian jerboa gene identical to query gene\"                \n",
       "[1874] \"%id. query gene identical to target Lesser Egyptian jerboa gene\"                \n",
       "[1875] \"Lesser Egyptian jerboa Gene-order conservation score\"                           \n",
       "[1876] \"Lesser Egyptian jerboa Whole-genome alignment coverage\"                         \n",
       "[1877] \"dN with Lesser Egyptian jerboa\"                                                 \n",
       "[1878] \"dS with Lesser Egyptian jerboa\"                                                 \n",
       "[1879] \"Lesser Egyptian jerboa orthology confidence [0 low, 1 high]\"                    \n",
       "[1880] \"Lesser hedgehog tenrec gene stable ID\"                                          \n",
       "[1881] \"Lesser hedgehog tenrec gene name\"                                               \n",
       "[1882] \"Lesser hedgehog tenrec protein or transcript stable ID\"                         \n",
       "[1883] \"Lesser hedgehog tenrec chromosome/scaffold name\"                                \n",
       "[1884] \"Lesser hedgehog tenrec chromosome/scaffold start (bp)\"                          \n",
       "[1885] \"Lesser hedgehog tenrec chromosome/scaffold end (bp)\"                            \n",
       "[1886] \"Query protein or transcript ID\"                                                 \n",
       "[1887] \"Last common ancestor with Lesser hedgehog tenrec\"                               \n",
       "[1888] \"Lesser hedgehog tenrec homology type\"                                           \n",
       "[1889] \"%id. target Lesser hedgehog tenrec gene identical to query gene\"                \n",
       "[1890] \"%id. query gene identical to target Lesser hedgehog tenrec gene\"                \n",
       "[1891] \"Lesser hedgehog tenrec Gene-order conservation score\"                           \n",
       "[1892] \"Lesser hedgehog tenrec Whole-genome alignment coverage\"                         \n",
       "[1893] \"dN with Lesser hedgehog tenrec\"                                                 \n",
       "[1894] \"dS with Lesser hedgehog tenrec\"                                                 \n",
       "[1895] \"Lesser hedgehog tenrec orthology confidence [0 low, 1 high]\"                    \n",
       "[1896] \"Little spotted kiwi gene stable ID\"                                             \n",
       "[1897] \"Little spotted kiwi gene name\"                                                  \n",
       "[1898] \"Little spotted kiwi protein or transcript stable ID\"                            \n",
       "[1899] \"Little spotted kiwi chromosome/scaffold name\"                                   \n",
       "[1900] \"Little spotted kiwi chromosome/scaffold start (bp)\"                             \n",
       "[1901] \"Little spotted kiwi chromosome/scaffold end (bp)\"                               \n",
       "[1902] \"Query protein or transcript ID\"                                                 \n",
       "[1903] \"Last common ancestor with Little spotted kiwi\"                                  \n",
       "[1904] \"Little spotted kiwi homology type\"                                              \n",
       "[1905] \"%id. target Little spotted kiwi gene identical to query gene\"                   \n",
       "[1906] \"%id. query gene identical to target Little spotted kiwi gene\"                   \n",
       "[1907] \"Little spotted kiwi Gene-order conservation score\"                              \n",
       "[1908] \"Little spotted kiwi Whole-genome alignment coverage\"                            \n",
       "[1909] \"dN with Little spotted kiwi\"                                                    \n",
       "[1910] \"dS with Little spotted kiwi\"                                                    \n",
       "[1911] \"Little spotted kiwi orthology confidence [0 low, 1 high]\"                       \n",
       "[1912] \"Long-tailed chinchilla gene stable ID\"                                          \n",
       "[1913] \"Long-tailed chinchilla gene name\"                                               \n",
       "[1914] \"Long-tailed chinchilla protein or transcript stable ID\"                         \n",
       "[1915] \"Long-tailed chinchilla chromosome/scaffold name\"                                \n",
       "[1916] \"Long-tailed chinchilla chromosome/scaffold start (bp)\"                          \n",
       "[1917] \"Long-tailed chinchilla chromosome/scaffold end (bp)\"                            \n",
       "[1918] \"Query protein or transcript ID\"                                                 \n",
       "[1919] \"Last common ancestor with Long-tailed chinchilla\"                               \n",
       "[1920] \"Long-tailed chinchilla homology type\"                                           \n",
       "[1921] \"%id. target Long-tailed chinchilla gene identical to query gene\"                \n",
       "[1922] \"%id. query gene identical to target Long-tailed chinchilla gene\"                \n",
       "[1923] \"Long-tailed chinchilla Gene-order conservation score\"                           \n",
       "[1924] \"Long-tailed chinchilla Whole-genome alignment coverage\"                         \n",
       "[1925] \"dN with Long-tailed chinchilla\"                                                 \n",
       "[1926] \"dS with Long-tailed chinchilla\"                                                 \n",
       "[1927] \"Long-tailed chinchilla orthology confidence [0 low, 1 high]\"                    \n",
       "[1928] \"Lyretail cichlid gene stable ID\"                                                \n",
       "[1929] \"Lyretail cichlid gene name\"                                                     \n",
       "[1930] \"Lyretail cichlid protein or transcript stable ID\"                               \n",
       "[1931] \"Lyretail cichlid chromosome/scaffold name\"                                      \n",
       "[1932] \"Lyretail cichlid chromosome/scaffold start (bp)\"                                \n",
       "[1933] \"Lyretail cichlid chromosome/scaffold end (bp)\"                                  \n",
       "[1934] \"Query protein or transcript ID\"                                                 \n",
       "[1935] \"Last common ancestor with Lyretail cichlid\"                                     \n",
       "[1936] \"Lyretail cichlid homology type\"                                                 \n",
       "[1937] \"%id. target Lyretail cichlid gene identical to query gene\"                      \n",
       "[1938] \"%id. query gene identical to target Lyretail cichlid gene\"                      \n",
       "[1939] \"Lyretail cichlid Gene-order conservation score\"                                 \n",
       "[1940] \"Lyretail cichlid Whole-genome alignment coverage\"                               \n",
       "[1941] \"dN with Lyretail cichlid\"                                                       \n",
       "[1942] \"dS with Lyretail cichlid\"                                                       \n",
       "[1943] \"Lyretail cichlid orthology confidence [0 low, 1 high]\"                          \n",
       "[1944] \"Ma's night monkey gene stable ID\"                                               \n",
       "[1945] \"Ma's night monkey gene name\"                                                    \n",
       "[1946] \"Ma's night monkey protein or transcript stable ID\"                              \n",
       "[1947] \"Ma's night monkey chromosome/scaffold name\"                                     \n",
       "[1948] \"Ma's night monkey chromosome/scaffold start (bp)\"                               \n",
       "[1949] \"Ma's night monkey chromosome/scaffold end (bp)\"                                 \n",
       "[1950] \"Query protein or transcript ID\"                                                 \n",
       "[1951] \"Last common ancestor with Ma's night monkey\"                                    \n",
       "[1952] \"Ma's night monkey homology type\"                                                \n",
       "[1953] \"%id. target Ma's night monkey gene identical to query gene\"                     \n",
       "[1954] \"%id. query gene identical to target Ma's night monkey gene\"                     \n",
       "[1955] \"Ma's night monkey Gene-order conservation score\"                                \n",
       "[1956] \"Ma's night monkey Whole-genome alignment coverage\"                              \n",
       "[1957] \"dN with Ma's night monkey\"                                                      \n",
       "[1958] \"dS with Ma's night monkey\"                                                      \n",
       "[1959] \"Ma's night monkey orthology confidence [0 low, 1 high]\"                         \n",
       "[1960] \"Macaque gene stable ID\"                                                         \n",
       "[1961] \"Macaque gene name\"                                                              \n",
       "[1962] \"Macaque protein or transcript stable ID\"                                        \n",
       "[1963] \"Macaque chromosome/scaffold name\"                                               \n",
       "[1964] \"Macaque chromosome/scaffold start (bp)\"                                         \n",
       "[1965] \"Macaque chromosome/scaffold end (bp)\"                                           \n",
       "[1966] \"Query protein or transcript ID\"                                                 \n",
       "[1967] \"Last common ancestor with Macaque\"                                              \n",
       "[1968] \"Macaque homology type\"                                                          \n",
       "[1969] \"%id. target Macaque gene identical to query gene\"                               \n",
       "[1970] \"%id. query gene identical to target Macaque gene\"                               \n",
       "[1971] \"Macaque Gene-order conservation score\"                                          \n",
       "[1972] \"Macaque Whole-genome alignment coverage\"                                        \n",
       "[1973] \"dN with Macaque\"                                                                \n",
       "[1974] \"dS with Macaque\"                                                                \n",
       "[1975] \"Macaque orthology confidence [0 low, 1 high]\"                                   \n",
       "[1976] \"Mainland tiger snake gene stable ID\"                                            \n",
       "[1977] \"Mainland tiger snake gene name\"                                                 \n",
       "[1978] \"Mainland tiger snake protein or transcript stable ID\"                           \n",
       "[1979] \"Mainland tiger snake chromosome/scaffold name\"                                  \n",
       "[1980] \"Mainland tiger snake chromosome/scaffold start (bp)\"                            \n",
       "[1981] \"Mainland tiger snake chromosome/scaffold end (bp)\"                              \n",
       "[1982] \"Query protein or transcript ID\"                                                 \n",
       "[1983] \"Last common ancestor with Mainland tiger snake\"                                 \n",
       "[1984] \"Mainland tiger snake homology type\"                                             \n",
       "[1985] \"%id. target Mainland tiger snake gene identical to query gene\"                  \n",
       "[1986] \"%id. query gene identical to target Mainland tiger snake gene\"                  \n",
       "[1987] \"Mainland tiger snake Gene-order conservation score\"                             \n",
       "[1988] \"Mainland tiger snake Whole-genome alignment coverage\"                           \n",
       "[1989] \"dN with Mainland tiger snake\"                                                   \n",
       "[1990] \"dS with Mainland tiger snake\"                                                   \n",
       "[1991] \"Mainland tiger snake orthology confidence [0 low, 1 high]\"                      \n",
       "[1992] \"Makobe Island cichlid gene stable ID\"                                           \n",
       "[1993] \"Makobe Island cichlid gene name\"                                                \n",
       "[1994] \"Makobe Island cichlid protein or transcript stable ID\"                          \n",
       "[1995] \"Makobe Island cichlid chromosome/scaffold name\"                                 \n",
       "[1996] \"Makobe Island cichlid chromosome/scaffold start (bp)\"                           \n",
       "[1997] \"Makobe Island cichlid chromosome/scaffold end (bp)\"                             \n",
       "[1998] \"Query protein or transcript ID\"                                                 \n",
       "[1999] \"Last common ancestor with Makobe Island cichlid\"                                \n",
       "[2000] \"Makobe Island cichlid homology type\"                                            \n",
       "[2001] \"%id. target Makobe Island cichlid gene identical to query gene\"                 \n",
       "[2002] \"%id. query gene identical to target Makobe Island cichlid gene\"                 \n",
       "[2003] \"Makobe Island cichlid Gene-order conservation score\"                            \n",
       "[2004] \"Makobe Island cichlid Whole-genome alignment coverage\"                          \n",
       "[2005] \"dN with Makobe Island cichlid\"                                                  \n",
       "[2006] \"dS with Makobe Island cichlid\"                                                  \n",
       "[2007] \"Makobe Island cichlid orthology confidence [0 low, 1 high]\"                     \n",
       "[2008] \"Mangrove rivulus gene stable ID\"                                                \n",
       "[2009] \"Mangrove rivulus gene name\"                                                     \n",
       "[2010] \"Mangrove rivulus protein or transcript stable ID\"                               \n",
       "[2011] \"Mangrove rivulus chromosome/scaffold name\"                                      \n",
       "[2012] \"Mangrove rivulus chromosome/scaffold start (bp)\"                                \n",
       "[2013] \"Mangrove rivulus chromosome/scaffold end (bp)\"                                  \n",
       "[2014] \"Query protein or transcript ID\"                                                 \n",
       "[2015] \"Last common ancestor with Mangrove rivulus\"                                     \n",
       "[2016] \"Mangrove rivulus homology type\"                                                 \n",
       "[2017] \"%id. target Mangrove rivulus gene identical to query gene\"                      \n",
       "[2018] \"%id. query gene identical to target Mangrove rivulus gene\"                      \n",
       "[2019] \"Mangrove rivulus Gene-order conservation score\"                                 \n",
       "[2020] \"Mangrove rivulus Whole-genome alignment coverage\"                               \n",
       "[2021] \"dN with Mangrove rivulus\"                                                       \n",
       "[2022] \"dS with Mangrove rivulus\"                                                       \n",
       "[2023] \"Mangrove rivulus orthology confidence [0 low, 1 high]\"                          \n",
       "[2024] \"Marmoset gene stable ID\"                                                        \n",
       "[2025] \"Marmoset gene name\"                                                             \n",
       "[2026] \"Marmoset protein or transcript stable ID\"                                       \n",
       "[2027] \"Marmoset chromosome/scaffold name\"                                              \n",
       "[2028] \"Marmoset chromosome/scaffold start (bp)\"                                        \n",
       "[2029] \"Marmoset chromosome/scaffold end (bp)\"                                          \n",
       "[2030] \"Query protein or transcript ID\"                                                 \n",
       "[2031] \"Last common ancestor with Marmoset\"                                             \n",
       "[2032] \"Marmoset homology type\"                                                         \n",
       "[2033] \"%id. target Marmoset gene identical to query gene\"                              \n",
       "[2034] \"%id. query gene identical to target Marmoset gene\"                              \n",
       "[2035] \"Marmoset Gene-order conservation score\"                                         \n",
       "[2036] \"Marmoset Whole-genome alignment coverage\"                                       \n",
       "[2037] \"dN with Marmoset\"                                                               \n",
       "[2038] \"dS with Marmoset\"                                                               \n",
       "[2039] \"Marmoset orthology confidence [0 low, 1 high]\"                                  \n",
       "[2040] \"Megabat gene stable ID\"                                                         \n",
       "[2041] \"Megabat gene name\"                                                              \n",
       "[2042] \"Megabat protein or transcript stable ID\"                                        \n",
       "[2043] \"Megabat chromosome/scaffold name\"                                               \n",
       "[2044] \"Megabat chromosome/scaffold start (bp)\"                                         \n",
       "[2045] \"Megabat chromosome/scaffold end (bp)\"                                           \n",
       "[2046] \"Query protein or transcript ID\"                                                 \n",
       "[2047] \"Last common ancestor with Megabat\"                                              \n",
       "[2048] \"Megabat homology type\"                                                          \n",
       "[2049] \"%id. target Megabat gene identical to query gene\"                               \n",
       "[2050] \"%id. query gene identical to target Megabat gene\"                               \n",
       "[2051] \"Megabat Gene-order conservation score\"                                          \n",
       "[2052] \"Megabat Whole-genome alignment coverage\"                                        \n",
       "[2053] \"dN with Megabat\"                                                                \n",
       "[2054] \"dS with Megabat\"                                                                \n",
       "[2055] \"Megabat orthology confidence [0 low, 1 high]\"                                   \n",
       "[2056] \"Mexican tetra gene stable ID\"                                                   \n",
       "[2057] \"Mexican tetra gene name\"                                                        \n",
       "[2058] \"Mexican tetra protein or transcript stable ID\"                                  \n",
       "[2059] \"Mexican tetra chromosome/scaffold name\"                                         \n",
       "[2060] \"Mexican tetra chromosome/scaffold start (bp)\"                                   \n",
       "[2061] \"Mexican tetra chromosome/scaffold end (bp)\"                                     \n",
       "[2062] \"Query protein or transcript ID\"                                                 \n",
       "[2063] \"Last common ancestor with Mexican tetra\"                                        \n",
       "[2064] \"Mexican tetra homology type\"                                                    \n",
       "[2065] \"%id. target Mexican tetra gene identical to query gene\"                         \n",
       "[2066] \"%id. query gene identical to target Mexican tetra gene\"                         \n",
       "[2067] \"Mexican tetra Gene-order conservation score\"                                    \n",
       "[2068] \"Mexican tetra Whole-genome alignment coverage\"                                  \n",
       "[2069] \"dN with Mexican tetra\"                                                          \n",
       "[2070] \"dS with Mexican tetra\"                                                          \n",
       "[2071] \"Mexican tetra orthology confidence [0 low, 1 high]\"                             \n",
       "[2072] \"Microbat gene stable ID\"                                                        \n",
       "[2073] \"Microbat gene name\"                                                             \n",
       "[2074] \"Microbat protein or transcript stable ID\"                                       \n",
       "[2075] \"Microbat chromosome/scaffold name\"                                              \n",
       "[2076] \"Microbat chromosome/scaffold start (bp)\"                                        \n",
       "[2077] \"Microbat chromosome/scaffold end (bp)\"                                          \n",
       "[2078] \"Query protein or transcript ID\"                                                 \n",
       "[2079] \"Last common ancestor with Microbat\"                                             \n",
       "[2080] \"Microbat homology type\"                                                         \n",
       "[2081] \"%id. target Microbat gene identical to query gene\"                              \n",
       "[2082] \"%id. query gene identical to target Microbat gene\"                              \n",
       "[2083] \"Microbat Gene-order conservation score\"                                         \n",
       "[2084] \"Microbat Whole-genome alignment coverage\"                                       \n",
       "[2085] \"dN with Microbat\"                                                               \n",
       "[2086] \"dS with Microbat\"                                                               \n",
       "[2087] \"Microbat orthology confidence [0 low, 1 high]\"                                  \n",
       "[2088] \"Midas cichlid gene stable ID\"                                                   \n",
       "[2089] \"Midas cichlid gene name\"                                                        \n",
       "[2090] \"Midas cichlid protein or transcript stable ID\"                                  \n",
       "[2091] \"Midas cichlid chromosome/scaffold name\"                                         \n",
       "[2092] \"Midas cichlid chromosome/scaffold start (bp)\"                                   \n",
       "[2093] \"Midas cichlid chromosome/scaffold end (bp)\"                                     \n",
       "[2094] \"Query protein or transcript ID\"                                                 \n",
       "[2095] \"Last common ancestor with Midas cichlid\"                                        \n",
       "[2096] \"Midas cichlid homology type\"                                                    \n",
       "[2097] \"%id. target Midas cichlid gene identical to query gene\"                         \n",
       "[2098] \"%id. query gene identical to target Midas cichlid gene\"                         \n",
       "[2099] \"Midas cichlid Gene-order conservation score\"                                    \n",
       "[2100] \"Midas cichlid Whole-genome alignment coverage\"                                  \n",
       "[2101] \"dN with Midas cichlid\"                                                          \n",
       "[2102] \"dS with Midas cichlid\"                                                          \n",
       "[2103] \"Midas cichlid orthology confidence [0 low, 1 high]\"                             \n",
       "[2104] \"Mongolian gerbil gene stable ID\"                                                \n",
       "[2105] \"Mongolian gerbil gene name\"                                                     \n",
       "[2106] \"Mongolian gerbil protein or transcript stable ID\"                               \n",
       "[2107] \"Mongolian gerbil chromosome/scaffold name\"                                      \n",
       "[2108] \"Mongolian gerbil chromosome/scaffold start (bp)\"                                \n",
       "[2109] \"Mongolian gerbil chromosome/scaffold end (bp)\"                                  \n",
       "[2110] \"Query protein or transcript ID\"                                                 \n",
       "[2111] \"Last common ancestor with Mongolian gerbil\"                                     \n",
       "[2112] \"Mongolian gerbil homology type\"                                                 \n",
       "[2113] \"%id. target Mongolian gerbil gene identical to query gene\"                      \n",
       "[2114] \"%id. query gene identical to target Mongolian gerbil gene\"                      \n",
       "[2115] \"Mongolian gerbil Gene-order conservation score\"                                 \n",
       "[2116] \"Mongolian gerbil Whole-genome alignment coverage\"                               \n",
       "[2117] \"dN with Mongolian gerbil\"                                                       \n",
       "[2118] \"dS with Mongolian gerbil\"                                                       \n",
       "[2119] \"Mongolian gerbil orthology confidence [0 low, 1 high]\"                          \n",
       "[2120] \"Monterrey platyfish gene stable ID\"                                             \n",
       "[2121] \"Monterrey platyfish gene name\"                                                  \n",
       "[2122] \"Monterrey platyfish protein or transcript stable ID\"                            \n",
       "[2123] \"Monterrey platyfish chromosome/scaffold name\"                                   \n",
       "[2124] \"Monterrey platyfish chromosome/scaffold start (bp)\"                             \n",
       "[2125] \"Monterrey platyfish chromosome/scaffold end (bp)\"                               \n",
       "[2126] \"Query protein or transcript ID\"                                                 \n",
       "[2127] \"Last common ancestor with Monterrey platyfish\"                                  \n",
       "[2128] \"Monterrey platyfish homology type\"                                              \n",
       "[2129] \"%id. target Monterrey platyfish gene identical to query gene\"                   \n",
       "[2130] \"%id. query gene identical to target Monterrey platyfish gene\"                   \n",
       "[2131] \"Monterrey platyfish Gene-order conservation score\"                              \n",
       "[2132] \"Monterrey platyfish Whole-genome alignment coverage\"                            \n",
       "[2133] \"dN with Monterrey platyfish\"                                                    \n",
       "[2134] \"dS with Monterrey platyfish\"                                                    \n",
       "[2135] \"Monterrey platyfish orthology confidence [0 low, 1 high]\"                       \n",
       "[2136] \"Mouse gene stable ID\"                                                           \n",
       "[2137] \"Mouse gene name\"                                                                \n",
       "[2138] \"Mouse protein or transcript stable ID\"                                          \n",
       "[2139] \"Mouse chromosome/scaffold name\"                                                 \n",
       "[2140] \"Mouse chromosome/scaffold start (bp)\"                                           \n",
       "[2141] \"Mouse chromosome/scaffold end (bp)\"                                             \n",
       "[2142] \"Query protein or transcript ID\"                                                 \n",
       "[2143] \"Last common ancestor with Mouse\"                                                \n",
       "[2144] \"Mouse homology type\"                                                            \n",
       "[2145] \"%id. target Mouse gene identical to query gene\"                                 \n",
       "[2146] \"%id. query gene identical to target Mouse gene\"                                 \n",
       "[2147] \"Mouse Gene-order conservation score\"                                            \n",
       "[2148] \"Mouse Whole-genome alignment coverage\"                                          \n",
       "[2149] \"dN with Mouse\"                                                                  \n",
       "[2150] \"dS with Mouse\"                                                                  \n",
       "[2151] \"Mouse orthology confidence [0 low, 1 high]\"                                     \n",
       "[2152] \"Mouse Lemur gene stable ID\"                                                     \n",
       "[2153] \"Mouse Lemur gene name\"                                                          \n",
       "[2154] \"Mouse Lemur protein or transcript stable ID\"                                    \n",
       "[2155] \"Mouse Lemur chromosome/scaffold name\"                                           \n",
       "[2156] \"Mouse Lemur chromosome/scaffold start (bp)\"                                     \n",
       "[2157] \"Mouse Lemur chromosome/scaffold end (bp)\"                                       \n",
       "[2158] \"Query protein or transcript ID\"                                                 \n",
       "[2159] \"Last common ancestor with Mouse Lemur\"                                          \n",
       "[2160] \"Mouse Lemur homology type\"                                                      \n",
       "[2161] \"%id. target Mouse Lemur gene identical to query gene\"                           \n",
       "[2162] \"%id. query gene identical to target Mouse Lemur gene\"                           \n",
       "[2163] \"Mouse Lemur Gene-order conservation score\"                                      \n",
       "[2164] \"Mouse Lemur Whole-genome alignment coverage\"                                    \n",
       "[2165] \"dN with Mouse Lemur\"                                                            \n",
       "[2166] \"dS with Mouse Lemur\"                                                            \n",
       "[2167] \"Mouse Lemur orthology confidence [0 low, 1 high]\"                               \n",
       "[2168] \"Mummichog gene stable ID\"                                                       \n",
       "[2169] \"Mummichog gene name\"                                                            \n",
       "[2170] \"Mummichog protein or transcript stable ID\"                                      \n",
       "[2171] \"Mummichog chromosome/scaffold name\"                                             \n",
       "[2172] \"Mummichog chromosome/scaffold start (bp)\"                                       \n",
       "[2173] \"Mummichog chromosome/scaffold end (bp)\"                                         \n",
       "[2174] \"Query protein or transcript ID\"                                                 \n",
       "[2175] \"Last common ancestor with Mummichog\"                                            \n",
       "[2176] \"Mummichog homology type\"                                                        \n",
       "[2177] \"%id. target Mummichog gene identical to query gene\"                             \n",
       "[2178] \"%id. query gene identical to target Mummichog gene\"                             \n",
       "[2179] \"Mummichog Gene-order conservation score\"                                        \n",
       "[2180] \"Mummichog Whole-genome alignment coverage\"                                      \n",
       "[2181] \"dN with Mummichog\"                                                              \n",
       "[2182] \"dS with Mummichog\"                                                              \n",
       "[2183] \"Mummichog orthology confidence [0 low, 1 high]\"                                 \n",
       "[2184] \"Naked mole-rat female gene stable ID\"                                           \n",
       "[2185] \"Naked mole-rat female gene name\"                                                \n",
       "[2186] \"Naked mole-rat female protein or transcript stable ID\"                          \n",
       "[2187] \"Naked mole-rat female chromosome/scaffold name\"                                 \n",
       "[2188] \"Naked mole-rat female chromosome/scaffold start (bp)\"                           \n",
       "[2189] \"Naked mole-rat female chromosome/scaffold end (bp)\"                             \n",
       "[2190] \"Query protein or transcript ID\"                                                 \n",
       "[2191] \"Last common ancestor with Naked mole-rat female\"                                \n",
       "[2192] \"Naked mole-rat female homology type\"                                            \n",
       "[2193] \"%id. target Naked mole-rat female gene identical to query gene\"                 \n",
       "[2194] \"%id. query gene identical to target Naked mole-rat female gene\"                 \n",
       "[2195] \"Naked mole-rat female Gene-order conservation score\"                            \n",
       "[2196] \"Naked mole-rat female Whole-genome alignment coverage\"                          \n",
       "[2197] \"dN with Naked mole-rat female\"                                                  \n",
       "[2198] \"dS with Naked mole-rat female\"                                                  \n",
       "[2199] \"Naked mole-rat female orthology confidence [0 low, 1 high]\"                     \n",
       "[2200] \"Naked mole-rat male gene stable ID\"                                             \n",
       "[2201] \"Naked mole-rat male gene name\"                                                  \n",
       "[2202] \"Naked mole-rat male protein or transcript stable ID\"                            \n",
       "[2203] \"Naked mole-rat male chromosome/scaffold name\"                                   \n",
       "[2204] \"Naked mole-rat male chromosome/scaffold start (bp)\"                             \n",
       "[2205] \"Naked mole-rat male chromosome/scaffold end (bp)\"                               \n",
       "[2206] \"Query protein or transcript ID\"                                                 \n",
       "[2207] \"Last common ancestor with Naked mole-rat male\"                                  \n",
       "[2208] \"Naked mole-rat male homology type\"                                              \n",
       "[2209] \"%id. target Naked mole-rat male gene identical to query gene\"                   \n",
       "[2210] \"%id. query gene identical to target Naked mole-rat male gene\"                   \n",
       "[2211] \"Naked mole-rat male Gene-order conservation score\"                              \n",
       "[2212] \"Naked mole-rat male Whole-genome alignment coverage\"                            \n",
       "[2213] \"dN with Naked mole-rat male\"                                                    \n",
       "[2214] \"dS with Naked mole-rat male\"                                                    \n",
       "[2215] \"Naked mole-rat male orthology confidence [0 low, 1 high]\"                       \n",
       "[2216] \"Northern American deer mouse gene stable ID\"                                    \n",
       "[2217] \"Northern American deer mouse gene name\"                                         \n",
       "[2218] \"Northern American deer mouse protein or transcript stable ID\"                   \n",
       "[2219] \"Northern American deer mouse chromosome/scaffold name\"                          \n",
       "[2220] \"Northern American deer mouse chromosome/scaffold start (bp)\"                    \n",
       "[2221] \"Northern American deer mouse chromosome/scaffold end (bp)\"                      \n",
       "[2222] \"Query protein or transcript ID\"                                                 \n",
       "[2223] \"Last common ancestor with Northern American deer mouse\"                         \n",
       "[2224] \"Northern American deer mouse homology type\"                                     \n",
       "[2225] \"%id. target Northern American deer mouse gene identical to query gene\"          \n",
       "[2226] \"%id. query gene identical to target Northern American deer mouse gene\"          \n",
       "[2227] \"Northern American deer mouse Gene-order conservation score\"                     \n",
       "[2228] \"Northern American deer mouse Whole-genome alignment coverage\"                   \n",
       "[2229] \"dN with Northern American deer mouse\"                                           \n",
       "[2230] \"dS with Northern American deer mouse\"                                           \n",
       "[2231] \"Northern American deer mouse orthology confidence [0 low, 1 high]\"              \n",
       "[2232] \"Northern pike gene stable ID\"                                                   \n",
       "[2233] \"Northern pike gene name\"                                                        \n",
       "[2234] \"Northern pike protein or transcript stable ID\"                                  \n",
       "[2235] \"Northern pike chromosome/scaffold name\"                                         \n",
       "[2236] \"Northern pike chromosome/scaffold start (bp)\"                                   \n",
       "[2237] \"Northern pike chromosome/scaffold end (bp)\"                                     \n",
       "[2238] \"Query protein or transcript ID\"                                                 \n",
       "[2239] \"Last common ancestor with Northern pike\"                                        \n",
       "[2240] \"Northern pike homology type\"                                                    \n",
       "[2241] \"%id. target Northern pike gene identical to query gene\"                         \n",
       "[2242] \"%id. query gene identical to target Northern pike gene\"                         \n",
       "[2243] \"Northern pike Gene-order conservation score\"                                    \n",
       "[2244] \"Northern pike Whole-genome alignment coverage\"                                  \n",
       "[2245] \"dN with Northern pike\"                                                          \n",
       "[2246] \"dS with Northern pike\"                                                          \n",
       "[2247] \"Northern pike orthology confidence [0 low, 1 high]\"                             \n",
       "[2248] \"Ocean sunfish gene stable ID\"                                                   \n",
       "[2249] \"Ocean sunfish gene name\"                                                        \n",
       "[2250] \"Ocean sunfish protein or transcript stable ID\"                                  \n",
       "[2251] \"Ocean sunfish chromosome/scaffold name\"                                         \n",
       "[2252] \"Ocean sunfish chromosome/scaffold start (bp)\"                                   \n",
       "[2253] \"Ocean sunfish chromosome/scaffold end (bp)\"                                     \n",
       "[2254] \"Query protein or transcript ID\"                                                 \n",
       "[2255] \"Last common ancestor with Ocean sunfish\"                                        \n",
       "[2256] \"Ocean sunfish homology type\"                                                    \n",
       "[2257] \"%id. target Ocean sunfish gene identical to query gene\"                         \n",
       "[2258] \"%id. query gene identical to target Ocean sunfish gene\"                         \n",
       "[2259] \"Ocean sunfish Gene-order conservation score\"                                    \n",
       "[2260] \"Ocean sunfish Whole-genome alignment coverage\"                                  \n",
       "[2261] \"dN with Ocean sunfish\"                                                          \n",
       "[2262] \"dS with Ocean sunfish\"                                                          \n",
       "[2263] \"Ocean sunfish orthology confidence [0 low, 1 high]\"                             \n",
       "[2264] \"Okarito brown kiwi gene stable ID\"                                              \n",
       "[2265] \"Okarito brown kiwi gene name\"                                                   \n",
       "[2266] \"Okarito brown kiwi protein or transcript stable ID\"                             \n",
       "[2267] \"Okarito brown kiwi chromosome/scaffold name\"                                    \n",
       "[2268] \"Okarito brown kiwi chromosome/scaffold start (bp)\"                              \n",
       "[2269] \"Okarito brown kiwi chromosome/scaffold end (bp)\"                                \n",
       "[2270] \"Query protein or transcript ID\"                                                 \n",
       "[2271] \"Last common ancestor with Okarito brown kiwi\"                                   \n",
       "[2272] \"Okarito brown kiwi homology type\"                                               \n",
       "[2273] \"%id. target Okarito brown kiwi gene identical to query gene\"                    \n",
       "[2274] \"%id. query gene identical to target Okarito brown kiwi gene\"                    \n",
       "[2275] \"Okarito brown kiwi Gene-order conservation score\"                               \n",
       "[2276] \"Okarito brown kiwi Whole-genome alignment coverage\"                             \n",
       "[2277] \"dN with Okarito brown kiwi\"                                                     \n",
       "[2278] \"dS with Okarito brown kiwi\"                                                     \n",
       "[2279] \"Okarito brown kiwi orthology confidence [0 low, 1 high]\"                        \n",
       "[2280] \"Olive baboon gene stable ID\"                                                    \n",
       "[2281] \"Olive baboon gene name\"                                                         \n",
       "[2282] \"Olive baboon protein or transcript stable ID\"                                   \n",
       "[2283] \"Olive baboon chromosome/scaffold name\"                                          \n",
       "[2284] \"Olive baboon chromosome/scaffold start (bp)\"                                    \n",
       "[2285] \"Olive baboon chromosome/scaffold end (bp)\"                                      \n",
       "[2286] \"Query protein or transcript ID\"                                                 \n",
       "[2287] \"Last common ancestor with Olive baboon\"                                         \n",
       "[2288] \"Olive baboon homology type\"                                                     \n",
       "[2289] \"%id. target Olive baboon gene identical to query gene\"                          \n",
       "[2290] \"%id. query gene identical to target Olive baboon gene\"                          \n",
       "[2291] \"Olive baboon Gene-order conservation score\"                                     \n",
       "[2292] \"Olive baboon Whole-genome alignment coverage\"                                   \n",
       "[2293] \"dN with Olive baboon\"                                                           \n",
       "[2294] \"dS with Olive baboon\"                                                           \n",
       "[2295] \"Olive baboon orthology confidence [0 low, 1 high]\"                              \n",
       "[2296] \"Opossum gene stable ID\"                                                         \n",
       "[2297] \"Opossum gene name\"                                                              \n",
       "[2298] \"Opossum protein or transcript stable ID\"                                        \n",
       "[2299] \"Opossum chromosome/scaffold name\"                                               \n",
       "[2300] \"Opossum chromosome/scaffold start (bp)\"                                         \n",
       "[2301] \"Opossum chromosome/scaffold end (bp)\"                                           \n",
       "[2302] \"Query protein or transcript ID\"                                                 \n",
       "[2303] \"Last common ancestor with Opossum\"                                              \n",
       "[2304] \"Opossum homology type\"                                                          \n",
       "[2305] \"%id. target Opossum gene identical to query gene\"                               \n",
       "[2306] \"%id. query gene identical to target Opossum gene\"                               \n",
       "[2307] \"Opossum Gene-order conservation score\"                                          \n",
       "[2308] \"Opossum Whole-genome alignment coverage\"                                        \n",
       "[2309] \"dN with Opossum\"                                                                \n",
       "[2310] \"dS with Opossum\"                                                                \n",
       "[2311] \"Opossum orthology confidence [0 low, 1 high]\"                                   \n",
       "[2312] \"Orange clownfish gene stable ID\"                                                \n",
       "[2313] \"Orange clownfish gene name\"                                                     \n",
       "[2314] \"Orange clownfish protein or transcript stable ID\"                               \n",
       "[2315] \"Orange clownfish chromosome/scaffold name\"                                      \n",
       "[2316] \"Orange clownfish chromosome/scaffold start (bp)\"                                \n",
       "[2317] \"Orange clownfish chromosome/scaffold end (bp)\"                                  \n",
       "[2318] \"Query protein or transcript ID\"                                                 \n",
       "[2319] \"Last common ancestor with Orange clownfish\"                                     \n",
       "[2320] \"Orange clownfish homology type\"                                                 \n",
       "[2321] \"%id. target Orange clownfish gene identical to query gene\"                      \n",
       "[2322] \"%id. query gene identical to target Orange clownfish gene\"                      \n",
       "[2323] \"Orange clownfish Gene-order conservation score\"                                 \n",
       "[2324] \"Orange clownfish Whole-genome alignment coverage\"                               \n",
       "[2325] \"dN with Orange clownfish\"                                                       \n",
       "[2326] \"dS with Orange clownfish\"                                                       \n",
       "[2327] \"Orange clownfish orthology confidence [0 low, 1 high]\"                          \n",
       "[2328] \"Orangutan gene stable ID\"                                                       \n",
       "[2329] \"Orangutan gene name\"                                                            \n",
       "[2330] \"Orangutan protein or transcript stable ID\"                                      \n",
       "[2331] \"Orangutan chromosome/scaffold name\"                                             \n",
       "[2332] \"Orangutan chromosome/scaffold start (bp)\"                                       \n",
       "[2333] \"Orangutan chromosome/scaffold end (bp)\"                                         \n",
       "[2334] \"Query protein or transcript ID\"                                                 \n",
       "[2335] \"Last common ancestor with Orangutan\"                                            \n",
       "[2336] \"Orangutan homology type\"                                                        \n",
       "[2337] \"%id. target Orangutan gene identical to query gene\"                             \n",
       "[2338] \"%id. query gene identical to target Orangutan gene\"                             \n",
       "[2339] \"Orangutan Gene-order conservation score\"                                        \n",
       "[2340] \"Orangutan Whole-genome alignment coverage\"                                      \n",
       "[2341] \"dN with Orangutan\"                                                              \n",
       "[2342] \"dS with Orangutan\"                                                              \n",
       "[2343] \"Orangutan orthology confidence [0 low, 1 high]\"                                 \n",
       "[2344] \"Painted turtle gene stable ID\"                                                  \n",
       "[2345] \"Painted turtle gene name\"                                                       \n",
       "[2346] \"Painted turtle protein or transcript stable ID\"                                 \n",
       "[2347] \"Painted turtle chromosome/scaffold name\"                                        \n",
       "[2348] \"Painted turtle chromosome/scaffold start (bp)\"                                  \n",
       "[2349] \"Painted turtle chromosome/scaffold end (bp)\"                                    \n",
       "[2350] \"Query protein or transcript ID\"                                                 \n",
       "[2351] \"Last common ancestor with Painted turtle\"                                       \n",
       "[2352] \"Painted turtle homology type\"                                                   \n",
       "[2353] \"%id. target Painted turtle gene identical to query gene\"                        \n",
       "[2354] \"%id. query gene identical to target Painted turtle gene\"                        \n",
       "[2355] \"Painted turtle Gene-order conservation score\"                                   \n",
       "[2356] \"Painted turtle Whole-genome alignment coverage\"                                 \n",
       "[2357] \"dN with Painted turtle\"                                                         \n",
       "[2358] \"dS with Painted turtle\"                                                         \n",
       "[2359] \"Painted turtle orthology confidence [0 low, 1 high]\"                            \n",
       "[2360] \"Panda gene stable ID\"                                                           \n",
       "[2361] \"Panda gene name\"                                                                \n",
       "[2362] \"Panda protein or transcript stable ID\"                                          \n",
       "[2363] \"Panda chromosome/scaffold name\"                                                 \n",
       "[2364] \"Panda chromosome/scaffold start (bp)\"                                           \n",
       "[2365] \"Panda chromosome/scaffold end (bp)\"                                             \n",
       "[2366] \"Query protein or transcript ID\"                                                 \n",
       "[2367] \"Last common ancestor with Panda\"                                                \n",
       "[2368] \"Panda homology type\"                                                            \n",
       "[2369] \"%id. target Panda gene identical to query gene\"                                 \n",
       "[2370] \"%id. query gene identical to target Panda gene\"                                 \n",
       "[2371] \"Panda Gene-order conservation score\"                                            \n",
       "[2372] \"Panda Whole-genome alignment coverage\"                                          \n",
       "[2373] \"dN with Panda\"                                                                  \n",
       "[2374] \"dS with Panda\"                                                                  \n",
       "[2375] \"Panda orthology confidence [0 low, 1 high]\"                                     \n",
       "[2376] \"Paramormyrops kingsleyae gene stable ID\"                                        \n",
       "[2377] \"Paramormyrops kingsleyae gene name\"                                             \n",
       "[2378] \"Paramormyrops kingsleyae protein or transcript stable ID\"                       \n",
       "[2379] \"Paramormyrops kingsleyae chromosome/scaffold name\"                              \n",
       "[2380] \"Paramormyrops kingsleyae chromosome/scaffold start (bp)\"                        \n",
       "[2381] \"Paramormyrops kingsleyae chromosome/scaffold end (bp)\"                          \n",
       "[2382] \"Query protein or transcript ID\"                                                 \n",
       "[2383] \"Last common ancestor with Paramormyrops kingsleyae\"                             \n",
       "[2384] \"Paramormyrops kingsleyae homology type\"                                         \n",
       "[2385] \"%id. target Paramormyrops kingsleyae gene identical to query gene\"              \n",
       "[2386] \"%id. query gene identical to target Paramormyrops kingsleyae gene\"              \n",
       "[2387] \"Paramormyrops kingsleyae Gene-order conservation score\"                         \n",
       "[2388] \"Paramormyrops kingsleyae Whole-genome alignment coverage\"                       \n",
       "[2389] \"dN with Paramormyrops kingsleyae\"                                               \n",
       "[2390] \"dS with Paramormyrops kingsleyae\"                                               \n",
       "[2391] \"Paramormyrops kingsleyae orthology confidence [0 low, 1 high]\"                  \n",
       "[2392] \"Periophthalmus magnuspinnatus gene stable ID\"                                   \n",
       "[2393] \"Periophthalmus magnuspinnatus gene name\"                                        \n",
       "[2394] \"Periophthalmus magnuspinnatus protein or transcript stable ID\"                  \n",
       "[2395] \"Periophthalmus magnuspinnatus chromosome/scaffold name\"                         \n",
       "[2396] \"Periophthalmus magnuspinnatus chromosome/scaffold start (bp)\"                   \n",
       "[2397] \"Periophthalmus magnuspinnatus chromosome/scaffold end (bp)\"                     \n",
       "[2398] \"Query protein or transcript ID\"                                                 \n",
       "[2399] \"Last common ancestor with Periophthalmus magnuspinnatus\"                        \n",
       "[2400] \"Periophthalmus magnuspinnatus homology type\"                                    \n",
       "[2401] \"%id. target Periophthalmus magnuspinnatus gene identical to query gene\"         \n",
       "[2402] \"%id. query gene identical to target Periophthalmus magnuspinnatus gene\"         \n",
       "[2403] \"Periophthalmus magnuspinnatus Gene-order conservation score\"                    \n",
       "[2404] \"Periophthalmus magnuspinnatus Whole-genome alignment coverage\"                  \n",
       "[2405] \"dN with Periophthalmus magnuspinnatus\"                                          \n",
       "[2406] \"dS with Periophthalmus magnuspinnatus\"                                          \n",
       "[2407] \"Periophthalmus magnuspinnatus orthology confidence [0 low, 1 high]\"             \n",
       "[2408] \"Pig gene stable ID\"                                                             \n",
       "[2409] \"Pig gene name\"                                                                  \n",
       "[2410] \"Pig protein or transcript stable ID\"                                            \n",
       "[2411] \"Pig chromosome/scaffold name\"                                                   \n",
       "[2412] \"Pig chromosome/scaffold start (bp)\"                                             \n",
       "[2413] \"Pig chromosome/scaffold end (bp)\"                                               \n",
       "[2414] \"Query protein or transcript ID\"                                                 \n",
       "[2415] \"Last common ancestor with Pig\"                                                  \n",
       "[2416] \"Pig homology type\"                                                              \n",
       "[2417] \"%id. target Pig gene identical to query gene\"                                   \n",
       "[2418] \"%id. query gene identical to target Pig gene\"                                   \n",
       "[2419] \"Pig Gene-order conservation score\"                                              \n",
       "[2420] \"Pig Whole-genome alignment coverage\"                                            \n",
       "[2421] \"dN with Pig\"                                                                    \n",
       "[2422] \"dS with Pig\"                                                                    \n",
       "[2423] \"Pig orthology confidence [0 low, 1 high]\"                                       \n",
       "[2424] \"Pig USMARC gene stable ID\"                                                      \n",
       "[2425] \"Pig USMARC gene name\"                                                           \n",
       "[2426] \"Pig USMARC protein or transcript stable ID\"                                     \n",
       "[2427] \"Pig USMARC chromosome/scaffold name\"                                            \n",
       "[2428] \"Pig USMARC chromosome/scaffold start (bp)\"                                      \n",
       "[2429] \"Pig USMARC chromosome/scaffold end (bp)\"                                        \n",
       "[2430] \"Query protein or transcript ID\"                                                 \n",
       "[2431] \"Last common ancestor with Pig USMARC\"                                           \n",
       "[2432] \"Pig USMARC homology type\"                                                       \n",
       "[2433] \"%id. target Pig USMARC gene identical to query gene\"                            \n",
       "[2434] \"%id. query gene identical to target Pig USMARC gene\"                            \n",
       "[2435] \"Pig USMARC Gene-order conservation score\"                                       \n",
       "[2436] \"Pig USMARC Whole-genome alignment coverage\"                                     \n",
       "[2437] \"dN with Pig USMARC\"                                                             \n",
       "[2438] \"dS with Pig USMARC\"                                                             \n",
       "[2439] \"Pig USMARC orthology confidence [0 low, 1 high]\"                                \n",
       "[2440] \"Pig-tailed macaque gene stable ID\"                                              \n",
       "[2441] \"Pig-tailed macaque gene name\"                                                   \n",
       "[2442] \"Pig-tailed macaque protein or transcript stable ID\"                             \n",
       "[2443] \"Pig-tailed macaque chromosome/scaffold name\"                                    \n",
       "[2444] \"Pig-tailed macaque chromosome/scaffold start (bp)\"                              \n",
       "[2445] \"Pig-tailed macaque chromosome/scaffold end (bp)\"                                \n",
       "[2446] \"Query protein or transcript ID\"                                                 \n",
       "[2447] \"Last common ancestor with Pig-tailed macaque\"                                   \n",
       "[2448] \"Pig-tailed macaque homology type\"                                               \n",
       "[2449] \"%id. target Pig-tailed macaque gene identical to query gene\"                    \n",
       "[2450] \"%id. query gene identical to target Pig-tailed macaque gene\"                    \n",
       "[2451] \"Pig-tailed macaque Gene-order conservation score\"                               \n",
       "[2452] \"Pig-tailed macaque Whole-genome alignment coverage\"                             \n",
       "[2453] \"dN with Pig-tailed macaque\"                                                     \n",
       "[2454] \"dS with Pig-tailed macaque\"                                                     \n",
       "[2455] \"Pig-tailed macaque orthology confidence [0 low, 1 high]\"                        \n",
       "[2456] \"Pika gene stable ID\"                                                            \n",
       "[2457] \"Pika gene name\"                                                                 \n",
       "[2458] \"Pika protein or transcript stable ID\"                                           \n",
       "[2459] \"Pika chromosome/scaffold name\"                                                  \n",
       "[2460] \"Pika chromosome/scaffold start (bp)\"                                            \n",
       "[2461] \"Pika chromosome/scaffold end (bp)\"                                              \n",
       "[2462] \"Query protein or transcript ID\"                                                 \n",
       "[2463] \"Last common ancestor with Pika\"                                                 \n",
       "[2464] \"Pika homology type\"                                                             \n",
       "[2465] \"%id. target Pika gene identical to query gene\"                                  \n",
       "[2466] \"%id. query gene identical to target Pika gene\"                                  \n",
       "[2467] \"Pika Gene-order conservation score\"                                             \n",
       "[2468] \"Pika Whole-genome alignment coverage\"                                           \n",
       "[2469] \"dN with Pika\"                                                                   \n",
       "[2470] \"dS with Pika\"                                                                   \n",
       "[2471] \"Pika orthology confidence [0 low, 1 high]\"                                      \n",
       "[2472] \"Pink-footed goose gene stable ID\"                                               \n",
       "[2473] \"Pink-footed goose gene name\"                                                    \n",
       "[2474] \"Pink-footed goose protein or transcript stable ID\"                              \n",
       "[2475] \"Pink-footed goose chromosome/scaffold name\"                                     \n",
       "[2476] \"Pink-footed goose chromosome/scaffold start (bp)\"                               \n",
       "[2477] \"Pink-footed goose chromosome/scaffold end (bp)\"                                 \n",
       "[2478] \"Query protein or transcript ID\"                                                 \n",
       "[2479] \"Last common ancestor with Pink-footed goose\"                                    \n",
       "[2480] \"Pink-footed goose homology type\"                                                \n",
       "[2481] \"%id. target Pink-footed goose gene identical to query gene\"                     \n",
       "[2482] \"%id. query gene identical to target Pink-footed goose gene\"                     \n",
       "[2483] \"Pink-footed goose Gene-order conservation score\"                                \n",
       "[2484] \"Pink-footed goose Whole-genome alignment coverage\"                              \n",
       "[2485] \"dN with Pink-footed goose\"                                                      \n",
       "[2486] \"dS with Pink-footed goose\"                                                      \n",
       "[2487] \"Pink-footed goose orthology confidence [0 low, 1 high]\"                         \n",
       "[2488] \"Platyfish gene stable ID\"                                                       \n",
       "[2489] \"Platyfish gene name\"                                                            \n",
       "[2490] \"Platyfish protein or transcript stable ID\"                                      \n",
       "[2491] \"Platyfish chromosome/scaffold name\"                                             \n",
       "[2492] \"Platyfish chromosome/scaffold start (bp)\"                                       \n",
       "[2493] \"Platyfish chromosome/scaffold end (bp)\"                                         \n",
       "[2494] \"Query protein or transcript ID\"                                                 \n",
       "[2495] \"Last common ancestor with Platyfish\"                                            \n",
       "[2496] \"Platyfish homology type\"                                                        \n",
       "[2497] \"%id. target Platyfish gene identical to query gene\"                             \n",
       "[2498] \"%id. query gene identical to target Platyfish gene\"                             \n",
       "[2499] \"Platyfish Gene-order conservation score\"                                        \n",
       "[2500] \"Platyfish Whole-genome alignment coverage\"                                      \n",
       "[2501] \"dN with Platyfish\"                                                              \n",
       "[2502] \"dS with Platyfish\"                                                              \n",
       "[2503] \"Platyfish orthology confidence [0 low, 1 high]\"                                 \n",
       "[2504] \"Platypus gene stable ID\"                                                        \n",
       "[2505] \"Platypus gene name\"                                                             \n",
       "[2506] \"Platypus protein or transcript stable ID\"                                       \n",
       "[2507] \"Platypus chromosome/scaffold name\"                                              \n",
       "[2508] \"Platypus chromosome/scaffold start (bp)\"                                        \n",
       "[2509] \"Platypus chromosome/scaffold end (bp)\"                                          \n",
       "[2510] \"Query protein or transcript ID\"                                                 \n",
       "[2511] \"Last common ancestor with Platypus\"                                             \n",
       "[2512] \"Platypus homology type\"                                                         \n",
       "[2513] \"%id. target Platypus gene identical to query gene\"                              \n",
       "[2514] \"%id. query gene identical to target Platypus gene\"                              \n",
       "[2515] \"Platypus Gene-order conservation score\"                                         \n",
       "[2516] \"Platypus Whole-genome alignment coverage\"                                       \n",
       "[2517] \"dN with Platypus\"                                                               \n",
       "[2518] \"dS with Platypus\"                                                               \n",
       "[2519] \"Platypus orthology confidence [0 low, 1 high]\"                                  \n",
       "[2520] \"Polar bear gene stable ID\"                                                      \n",
       "[2521] \"Polar bear gene name\"                                                           \n",
       "[2522] \"Polar bear protein or transcript stable ID\"                                     \n",
       "[2523] \"Polar bear chromosome/scaffold name\"                                            \n",
       "[2524] \"Polar bear chromosome/scaffold start (bp)\"                                      \n",
       "[2525] \"Polar bear chromosome/scaffold end (bp)\"                                        \n",
       "[2526] \"Query protein or transcript ID\"                                                 \n",
       "[2527] \"Last common ancestor with Polar bear\"                                           \n",
       "[2528] \"Polar bear homology type\"                                                       \n",
       "[2529] \"%id. target Polar bear gene identical to query gene\"                            \n",
       "[2530] \"%id. query gene identical to target Polar bear gene\"                            \n",
       "[2531] \"Polar bear Gene-order conservation score\"                                       \n",
       "[2532] \"Polar bear Whole-genome alignment coverage\"                                     \n",
       "[2533] \"dN with Polar bear\"                                                             \n",
       "[2534] \"dS with Polar bear\"                                                             \n",
       "[2535] \"Polar bear orthology confidence [0 low, 1 high]\"                                \n",
       "[2536] \"Prairie vole gene stable ID\"                                                    \n",
       "[2537] \"Prairie vole gene name\"                                                         \n",
       "[2538] \"Prairie vole protein or transcript stable ID\"                                   \n",
       "[2539] \"Prairie vole chromosome/scaffold name\"                                          \n",
       "[2540] \"Prairie vole chromosome/scaffold start (bp)\"                                    \n",
       "[2541] \"Prairie vole chromosome/scaffold end (bp)\"                                      \n",
       "[2542] \"Query protein or transcript ID\"                                                 \n",
       "[2543] \"Last common ancestor with Prairie vole\"                                         \n",
       "[2544] \"Prairie vole homology type\"                                                     \n",
       "[2545] \"%id. target Prairie vole gene identical to query gene\"                          \n",
       "[2546] \"%id. query gene identical to target Prairie vole gene\"                          \n",
       "[2547] \"Prairie vole Gene-order conservation score\"                                     \n",
       "[2548] \"Prairie vole Whole-genome alignment coverage\"                                   \n",
       "[2549] \"dN with Prairie vole\"                                                           \n",
       "[2550] \"dS with Prairie vole\"                                                           \n",
       "[2551] \"Prairie vole orthology confidence [0 low, 1 high]\"                              \n",
       "[2552] \"Rabbit gene stable ID\"                                                          \n",
       "[2553] \"Rabbit gene name\"                                                               \n",
       "[2554] \"Rabbit protein or transcript stable ID\"                                         \n",
       "[2555] \"Rabbit chromosome/scaffold name\"                                                \n",
       "[2556] \"Rabbit chromosome/scaffold start (bp)\"                                          \n",
       "[2557] \"Rabbit chromosome/scaffold end (bp)\"                                            \n",
       "[2558] \"Query protein or transcript ID\"                                                 \n",
       "[2559] \"Last common ancestor with Rabbit\"                                               \n",
       "[2560] \"Rabbit homology type\"                                                           \n",
       "[2561] \"%id. target Rabbit gene identical to query gene\"                                \n",
       "[2562] \"%id. query gene identical to target Rabbit gene\"                                \n",
       "[2563] \"Rabbit Gene-order conservation score\"                                           \n",
       "[2564] \"Rabbit Whole-genome alignment coverage\"                                         \n",
       "[2565] \"dN with Rabbit\"                                                                 \n",
       "[2566] \"dS with Rabbit\"                                                                 \n",
       "[2567] \"Rabbit orthology confidence [0 low, 1 high]\"                                    \n",
       "[2568] \"Rat gene stable ID\"                                                             \n",
       "[2569] \"Rat gene name\"                                                                  \n",
       "[2570] \"Rat protein or transcript stable ID\"                                            \n",
       "[2571] \"Rat chromosome/scaffold name\"                                                   \n",
       "[2572] \"Rat chromosome/scaffold start (bp)\"                                             \n",
       "[2573] \"Rat chromosome/scaffold end (bp)\"                                               \n",
       "[2574] \"Query protein or transcript ID\"                                                 \n",
       "[2575] \"Last common ancestor with Rat\"                                                  \n",
       "[2576] \"Rat homology type\"                                                              \n",
       "[2577] \"%id. target Rat gene identical to query gene\"                                   \n",
       "[2578] \"%id. query gene identical to target Rat gene\"                                   \n",
       "[2579] \"Rat Gene-order conservation score\"                                              \n",
       "[2580] \"Rat Whole-genome alignment coverage\"                                            \n",
       "[2581] \"dN with Rat\"                                                                    \n",
       "[2582] \"dS with Rat\"                                                                    \n",
       "[2583] \"Rat orthology confidence [0 low, 1 high]\"                                       \n",
       "[2584] \"Red fox gene stable ID\"                                                         \n",
       "[2585] \"Red fox gene name\"                                                              \n",
       "[2586] \"Red fox protein or transcript stable ID\"                                        \n",
       "[2587] \"Red fox chromosome/scaffold name\"                                               \n",
       "[2588] \"Red fox chromosome/scaffold start (bp)\"                                         \n",
       "[2589] \"Red fox chromosome/scaffold end (bp)\"                                           \n",
       "[2590] \"Query protein or transcript ID\"                                                 \n",
       "[2591] \"Last common ancestor with Red fox\"                                              \n",
       "[2592] \"Red fox homology type\"                                                          \n",
       "[2593] \"%id. target Red fox gene identical to query gene\"                               \n",
       "[2594] \"%id. query gene identical to target Red fox gene\"                               \n",
       "[2595] \"Red fox Gene-order conservation score\"                                          \n",
       "[2596] \"Red fox Whole-genome alignment coverage\"                                        \n",
       "[2597] \"dN with Red fox\"                                                                \n",
       "[2598] \"dS with Red fox\"                                                                \n",
       "[2599] \"Red fox orthology confidence [0 low, 1 high]\"                                   \n",
       "[2600] \"Red-bellied piranha gene stable ID\"                                             \n",
       "[2601] \"Red-bellied piranha gene name\"                                                  \n",
       "[2602] \"Red-bellied piranha protein or transcript stable ID\"                            \n",
       "[2603] \"Red-bellied piranha chromosome/scaffold name\"                                   \n",
       "[2604] \"Red-bellied piranha chromosome/scaffold start (bp)\"                             \n",
       "[2605] \"Red-bellied piranha chromosome/scaffold end (bp)\"                               \n",
       "[2606] \"Query protein or transcript ID\"                                                 \n",
       "[2607] \"Last common ancestor with Red-bellied piranha\"                                  \n",
       "[2608] \"Red-bellied piranha homology type\"                                              \n",
       "[2609] \"%id. target Red-bellied piranha gene identical to query gene\"                   \n",
       "[2610] \"%id. query gene identical to target Red-bellied piranha gene\"                   \n",
       "[2611] \"Red-bellied piranha Gene-order conservation score\"                              \n",
       "[2612] \"Red-bellied piranha Whole-genome alignment coverage\"                            \n",
       "[2613] \"dN with Red-bellied piranha\"                                                    \n",
       "[2614] \"dS with Red-bellied piranha\"                                                    \n",
       "[2615] \"Red-bellied piranha orthology confidence [0 low, 1 high]\"                       \n",
       "[2616] \"Ruff gene stable ID\"                                                            \n",
       "[2617] \"Ruff gene name\"                                                                 \n",
       "[2618] \"Ruff protein or transcript stable ID\"                                           \n",
       "[2619] \"Ruff chromosome/scaffold name\"                                                  \n",
       "[2620] \"Ruff chromosome/scaffold start (bp)\"                                            \n",
       "[2621] \"Ruff chromosome/scaffold end (bp)\"                                              \n",
       "[2622] \"Query protein or transcript ID\"                                                 \n",
       "[2623] \"Last common ancestor with Ruff\"                                                 \n",
       "[2624] \"Ruff homology type\"                                                             \n",
       "[2625] \"%id. target Ruff gene identical to query gene\"                                  \n",
       "[2626] \"%id. query gene identical to target Ruff gene\"                                  \n",
       "[2627] \"Ruff Gene-order conservation score\"                                             \n",
       "[2628] \"Ruff Whole-genome alignment coverage\"                                           \n",
       "[2629] \"dN with Ruff\"                                                                   \n",
       "[2630] \"dS with Ruff\"                                                                   \n",
       "[2631] \"Ruff orthology confidence [0 low, 1 high]\"                                      \n",
       "[2632] \"Ryukyu mouse gene stable ID\"                                                    \n",
       "[2633] \"Ryukyu mouse gene name\"                                                         \n",
       "[2634] \"Ryukyu mouse protein or transcript stable ID\"                                   \n",
       "[2635] \"Ryukyu mouse chromosome/scaffold name\"                                          \n",
       "[2636] \"Ryukyu mouse chromosome/scaffold start (bp)\"                                    \n",
       "[2637] \"Ryukyu mouse chromosome/scaffold end (bp)\"                                      \n",
       "[2638] \"Query protein or transcript ID\"                                                 \n",
       "[2639] \"Last common ancestor with Ryukyu mouse\"                                         \n",
       "[2640] \"Ryukyu mouse homology type\"                                                     \n",
       "[2641] \"%id. target Ryukyu mouse gene identical to query gene\"                          \n",
       "[2642] \"%id. query gene identical to target Ryukyu mouse gene\"                          \n",
       "[2643] \"Ryukyu mouse Gene-order conservation score\"                                     \n",
       "[2644] \"Ryukyu mouse Whole-genome alignment coverage\"                                   \n",
       "[2645] \"dN with Ryukyu mouse\"                                                           \n",
       "[2646] \"dS with Ryukyu mouse\"                                                           \n",
       "[2647] \"Ryukyu mouse orthology confidence [0 low, 1 high]\"                              \n",
       "[2648] \"Saccharomyces cerevisiae gene stable ID\"                                        \n",
       "[2649] \"Saccharomyces cerevisiae gene name\"                                             \n",
       "[2650] \"Saccharomyces cerevisiae protein or transcript stable ID\"                       \n",
       "[2651] \"Saccharomyces cerevisiae chromosome/scaffold name\"                              \n",
       "[2652] \"Saccharomyces cerevisiae chromosome/scaffold start (bp)\"                        \n",
       "[2653] \"Saccharomyces cerevisiae chromosome/scaffold end (bp)\"                          \n",
       "[2654] \"Query protein or transcript ID\"                                                 \n",
       "[2655] \"Last common ancestor with Saccharomyces cerevisiae\"                             \n",
       "[2656] \"Saccharomyces cerevisiae homology type\"                                         \n",
       "[2657] \"%id. target Saccharomyces cerevisiae gene identical to query gene\"              \n",
       "[2658] \"%id. query gene identical to target Saccharomyces cerevisiae gene\"              \n",
       "[2659] \"Saccharomyces cerevisiae Gene-order conservation score\"                         \n",
       "[2660] \"Saccharomyces cerevisiae Whole-genome alignment coverage\"                       \n",
       "[2661] \"dN with Saccharomyces cerevisiae\"                                               \n",
       "[2662] \"dS with Saccharomyces cerevisiae\"                                               \n",
       "[2663] \"Saccharomyces cerevisiae orthology confidence [0 low, 1 high]\"                  \n",
       "[2664] \"Sailfin molly gene stable ID\"                                                   \n",
       "[2665] \"Sailfin molly gene name\"                                                        \n",
       "[2666] \"Sailfin molly protein or transcript stable ID\"                                  \n",
       "[2667] \"Sailfin molly chromosome/scaffold name\"                                         \n",
       "[2668] \"Sailfin molly chromosome/scaffold start (bp)\"                                   \n",
       "[2669] \"Sailfin molly chromosome/scaffold end (bp)\"                                     \n",
       "[2670] \"Query protein or transcript ID\"                                                 \n",
       "[2671] \"Last common ancestor with Sailfin molly\"                                        \n",
       "[2672] \"Sailfin molly homology type\"                                                    \n",
       "[2673] \"%id. target Sailfin molly gene identical to query gene\"                         \n",
       "[2674] \"%id. query gene identical to target Sailfin molly gene\"                         \n",
       "[2675] \"Sailfin molly Gene-order conservation score\"                                    \n",
       "[2676] \"Sailfin molly Whole-genome alignment coverage\"                                  \n",
       "[2677] \"dN with Sailfin molly\"                                                          \n",
       "[2678] \"dS with Sailfin molly\"                                                          \n",
       "[2679] \"Sailfin molly orthology confidence [0 low, 1 high]\"                             \n",
       "[2680] \"Sheep gene stable ID\"                                                           \n",
       "[2681] \"Sheep gene name\"                                                                \n",
       "[2682] \"Sheep protein or transcript stable ID\"                                          \n",
       "[2683] \"Sheep chromosome/scaffold name\"                                                 \n",
       "[2684] \"Sheep chromosome/scaffold start (bp)\"                                           \n",
       "[2685] \"Sheep chromosome/scaffold end (bp)\"                                             \n",
       "[2686] \"Query protein or transcript ID\"                                                 \n",
       "[2687] \"Last common ancestor with Sheep\"                                                \n",
       "[2688] \"Sheep homology type\"                                                            \n",
       "[2689] \"%id. target Sheep gene identical to query gene\"                                 \n",
       "[2690] \"%id. query gene identical to target Sheep gene\"                                 \n",
       "[2691] \"Sheep Gene-order conservation score\"                                            \n",
       "[2692] \"Sheep Whole-genome alignment coverage\"                                          \n",
       "[2693] \"dN with Sheep\"                                                                  \n",
       "[2694] \"dS with Sheep\"                                                                  \n",
       "[2695] \"Sheep orthology confidence [0 low, 1 high]\"                                     \n",
       "[2696] \"Sheepshead minnow gene stable ID\"                                               \n",
       "[2697] \"Sheepshead minnow gene name\"                                                    \n",
       "[2698] \"Sheepshead minnow protein or transcript stable ID\"                              \n",
       "[2699] \"Sheepshead minnow chromosome/scaffold name\"                                     \n",
       "[2700] \"Sheepshead minnow chromosome/scaffold start (bp)\"                               \n",
       "[2701] \"Sheepshead minnow chromosome/scaffold end (bp)\"                                 \n",
       "[2702] \"Query protein or transcript ID\"                                                 \n",
       "[2703] \"Last common ancestor with Sheepshead minnow\"                                    \n",
       "[2704] \"Sheepshead minnow homology type\"                                                \n",
       "[2705] \"%id. target Sheepshead minnow gene identical to query gene\"                     \n",
       "[2706] \"%id. query gene identical to target Sheepshead minnow gene\"                     \n",
       "[2707] \"Sheepshead minnow Gene-order conservation score\"                                \n",
       "[2708] \"Sheepshead minnow Whole-genome alignment coverage\"                              \n",
       "[2709] \"dN with Sheepshead minnow\"                                                      \n",
       "[2710] \"dS with Sheepshead minnow\"                                                      \n",
       "[2711] \"Sheepshead minnow orthology confidence [0 low, 1 high]\"                         \n",
       "[2712] \"Shortfin molly gene stable ID\"                                                  \n",
       "[2713] \"Shortfin molly gene name\"                                                       \n",
       "[2714] \"Shortfin molly protein or transcript stable ID\"                                 \n",
       "[2715] \"Shortfin molly chromosome/scaffold name\"                                        \n",
       "[2716] \"Shortfin molly chromosome/scaffold start (bp)\"                                  \n",
       "[2717] \"Shortfin molly chromosome/scaffold end (bp)\"                                    \n",
       "[2718] \"Query protein or transcript ID\"                                                 \n",
       "[2719] \"Last common ancestor with Shortfin molly\"                                       \n",
       "[2720] \"Shortfin molly homology type\"                                                   \n",
       "[2721] \"%id. target Shortfin molly gene identical to query gene\"                        \n",
       "[2722] \"%id. query gene identical to target Shortfin molly gene\"                        \n",
       "[2723] \"Shortfin molly Gene-order conservation score\"                                   \n",
       "[2724] \"Shortfin molly Whole-genome alignment coverage\"                                 \n",
       "[2725] \"dN with Shortfin molly\"                                                         \n",
       "[2726] \"dS with Shortfin molly\"                                                         \n",
       "[2727] \"Shortfin molly orthology confidence [0 low, 1 high]\"                            \n",
       "[2728] \"Shrew gene stable ID\"                                                           \n",
       "[2729] \"Shrew gene name\"                                                                \n",
       "[2730] \"Shrew protein or transcript stable ID\"                                          \n",
       "[2731] \"Shrew chromosome/scaffold name\"                                                 \n",
       "[2732] \"Shrew chromosome/scaffold start (bp)\"                                           \n",
       "[2733] \"Shrew chromosome/scaffold end (bp)\"                                             \n",
       "[2734] \"Query protein or transcript ID\"                                                 \n",
       "[2735] \"Last common ancestor with Shrew\"                                                \n",
       "[2736] \"Shrew homology type\"                                                            \n",
       "[2737] \"%id. target Shrew gene identical to query gene\"                                 \n",
       "[2738] \"%id. query gene identical to target Shrew gene\"                                 \n",
       "[2739] \"Shrew Gene-order conservation score\"                                            \n",
       "[2740] \"Shrew Whole-genome alignment coverage\"                                          \n",
       "[2741] \"dN with Shrew\"                                                                  \n",
       "[2742] \"dS with Shrew\"                                                                  \n",
       "[2743] \"Shrew orthology confidence [0 low, 1 high]\"                                     \n",
       "[2744] \"Shrew mouse gene stable ID\"                                                     \n",
       "[2745] \"Shrew mouse gene name\"                                                          \n",
       "[2746] \"Shrew mouse protein or transcript stable ID\"                                    \n",
       "[2747] \"Shrew mouse chromosome/scaffold name\"                                           \n",
       "[2748] \"Shrew mouse chromosome/scaffold start (bp)\"                                     \n",
       "[2749] \"Shrew mouse chromosome/scaffold end (bp)\"                                       \n",
       "[2750] \"Query protein or transcript ID\"                                                 \n",
       "[2751] \"Last common ancestor with Shrew mouse\"                                          \n",
       "[2752] \"Shrew mouse homology type\"                                                      \n",
       "[2753] \"%id. target Shrew mouse gene identical to query gene\"                           \n",
       "[2754] \"%id. query gene identical to target Shrew mouse gene\"                           \n",
       "[2755] \"Shrew mouse Gene-order conservation score\"                                      \n",
       "[2756] \"Shrew mouse Whole-genome alignment coverage\"                                    \n",
       "[2757] \"dN with Shrew mouse\"                                                            \n",
       "[2758] \"dS with Shrew mouse\"                                                            \n",
       "[2759] \"Shrew mouse orthology confidence [0 low, 1 high]\"                               \n",
       "[2760] \"Sloth gene stable ID\"                                                           \n",
       "[2761] \"Sloth gene name\"                                                                \n",
       "[2762] \"Sloth protein or transcript stable ID\"                                          \n",
       "[2763] \"Sloth chromosome/scaffold name\"                                                 \n",
       "[2764] \"Sloth chromosome/scaffold start (bp)\"                                           \n",
       "[2765] \"Sloth chromosome/scaffold end (bp)\"                                             \n",
       "[2766] \"Query protein or transcript ID\"                                                 \n",
       "[2767] \"Last common ancestor with Sloth\"                                                \n",
       "[2768] \"Sloth homology type\"                                                            \n",
       "[2769] \"%id. target Sloth gene identical to query gene\"                                 \n",
       "[2770] \"%id. query gene identical to target Sloth gene\"                                 \n",
       "[2771] \"Sloth Gene-order conservation score\"                                            \n",
       "[2772] \"Sloth Whole-genome alignment coverage\"                                          \n",
       "[2773] \"dN with Sloth\"                                                                  \n",
       "[2774] \"dS with Sloth\"                                                                  \n",
       "[2775] \"Sloth orthology confidence [0 low, 1 high]\"                                     \n",
       "[2776] \"Sooty mangabey gene stable ID\"                                                  \n",
       "[2777] \"Sooty mangabey gene name\"                                                       \n",
       "[2778] \"Sooty mangabey protein or transcript stable ID\"                                 \n",
       "[2779] \"Sooty mangabey chromosome/scaffold name\"                                        \n",
       "[2780] \"Sooty mangabey chromosome/scaffold start (bp)\"                                  \n",
       "[2781] \"Sooty mangabey chromosome/scaffold end (bp)\"                                    \n",
       "[2782] \"Query protein or transcript ID\"                                                 \n",
       "[2783] \"Last common ancestor with Sooty mangabey\"                                       \n",
       "[2784] \"Sooty mangabey homology type\"                                                   \n",
       "[2785] \"%id. target Sooty mangabey gene identical to query gene\"                        \n",
       "[2786] \"%id. query gene identical to target Sooty mangabey gene\"                        \n",
       "[2787] \"Sooty mangabey Gene-order conservation score\"                                   \n",
       "[2788] \"Sooty mangabey Whole-genome alignment coverage\"                                 \n",
       "[2789] \"dN with Sooty mangabey\"                                                         \n",
       "[2790] \"dS with Sooty mangabey\"                                                         \n",
       "[2791] \"Sooty mangabey orthology confidence [0 low, 1 high]\"                            \n",
       "[2792] \"Spiny chromis gene stable ID\"                                                   \n",
       "[2793] \"Spiny chromis gene name\"                                                        \n",
       "[2794] \"Spiny chromis protein or transcript stable ID\"                                  \n",
       "[2795] \"Spiny chromis chromosome/scaffold name\"                                         \n",
       "[2796] \"Spiny chromis chromosome/scaffold start (bp)\"                                   \n",
       "[2797] \"Spiny chromis chromosome/scaffold end (bp)\"                                     \n",
       "[2798] \"Query protein or transcript ID\"                                                 \n",
       "[2799] \"Last common ancestor with Spiny chromis\"                                        \n",
       "[2800] \"Spiny chromis homology type\"                                                    \n",
       "[2801] \"%id. target Spiny chromis gene identical to query gene\"                         \n",
       "[2802] \"%id. query gene identical to target Spiny chromis gene\"                         \n",
       "[2803] \"Spiny chromis Gene-order conservation score\"                                    \n",
       "[2804] \"Spiny chromis Whole-genome alignment coverage\"                                  \n",
       "[2805] \"dN with Spiny chromis\"                                                          \n",
       "[2806] \"dS with Spiny chromis\"                                                          \n",
       "[2807] \"Spiny chromis orthology confidence [0 low, 1 high]\"                             \n",
       "[2808] \"Spoon-billed sandpiper gene stable ID\"                                          \n",
       "[2809] \"Spoon-billed sandpiper gene name\"                                               \n",
       "[2810] \"Spoon-billed sandpiper protein or transcript stable ID\"                         \n",
       "[2811] \"Spoon-billed sandpiper chromosome/scaffold name\"                                \n",
       "[2812] \"Spoon-billed sandpiper chromosome/scaffold start (bp)\"                          \n",
       "[2813] \"Spoon-billed sandpiper chromosome/scaffold end (bp)\"                            \n",
       "[2814] \"Query protein or transcript ID\"                                                 \n",
       "[2815] \"Last common ancestor with Spoon-billed sandpiper\"                               \n",
       "[2816] \"Spoon-billed sandpiper homology type\"                                           \n",
       "[2817] \"%id. target Spoon-billed sandpiper gene identical to query gene\"                \n",
       "[2818] \"%id. query gene identical to target Spoon-billed sandpiper gene\"                \n",
       "[2819] \"Spoon-billed sandpiper Gene-order conservation score\"                           \n",
       "[2820] \"Spoon-billed sandpiper Whole-genome alignment coverage\"                         \n",
       "[2821] \"dN with Spoon-billed sandpiper\"                                                 \n",
       "[2822] \"dS with Spoon-billed sandpiper\"                                                 \n",
       "[2823] \"Spoon-billed sandpiper orthology confidence [0 low, 1 high]\"                    \n",
       "[2824] \"Spotted gar gene stable ID\"                                                     \n",
       "[2825] \"Spotted gar gene name\"                                                          \n",
       "[2826] \"Spotted gar protein or transcript stable ID\"                                    \n",
       "[2827] \"Spotted gar chromosome/scaffold name\"                                           \n",
       "[2828] \"Spotted gar chromosome/scaffold start (bp)\"                                     \n",
       "[2829] \"Spotted gar chromosome/scaffold end (bp)\"                                       \n",
       "[2830] \"Query protein or transcript ID\"                                                 \n",
       "[2831] \"Last common ancestor with Spotted gar\"                                          \n",
       "[2832] \"Spotted gar homology type\"                                                      \n",
       "[2833] \"%id. target Spotted gar gene identical to query gene\"                           \n",
       "[2834] \"%id. query gene identical to target Spotted gar gene\"                           \n",
       "[2835] \"Spotted gar Gene-order conservation score\"                                      \n",
       "[2836] \"Spotted gar Whole-genome alignment coverage\"                                    \n",
       "[2837] \"dN with Spotted gar\"                                                            \n",
       "[2838] \"dS with Spotted gar\"                                                            \n",
       "[2839] \"Spotted gar orthology confidence [0 low, 1 high]\"                               \n",
       "[2840] \"Squirrel gene stable ID\"                                                        \n",
       "[2841] \"Squirrel gene name\"                                                             \n",
       "[2842] \"Squirrel protein or transcript stable ID\"                                       \n",
       "[2843] \"Squirrel chromosome/scaffold name\"                                              \n",
       "[2844] \"Squirrel chromosome/scaffold start (bp)\"                                        \n",
       "[2845] \"Squirrel chromosome/scaffold end (bp)\"                                          \n",
       "[2846] \"Query protein or transcript ID\"                                                 \n",
       "[2847] \"Last common ancestor with Squirrel\"                                             \n",
       "[2848] \"Squirrel homology type\"                                                         \n",
       "[2849] \"%id. target Squirrel gene identical to query gene\"                              \n",
       "[2850] \"%id. query gene identical to target Squirrel gene\"                              \n",
       "[2851] \"Squirrel Gene-order conservation score\"                                         \n",
       "[2852] \"Squirrel Whole-genome alignment coverage\"                                       \n",
       "[2853] \"dN with Squirrel\"                                                               \n",
       "[2854] \"dS with Squirrel\"                                                               \n",
       "[2855] \"Squirrel orthology confidence [0 low, 1 high]\"                                  \n",
       "[2856] \"Steppe mouse gene stable ID\"                                                    \n",
       "[2857] \"Steppe mouse gene name\"                                                         \n",
       "[2858] \"Steppe mouse protein or transcript stable ID\"                                   \n",
       "[2859] \"Steppe mouse chromosome/scaffold name\"                                          \n",
       "[2860] \"Steppe mouse chromosome/scaffold start (bp)\"                                    \n",
       "[2861] \"Steppe mouse chromosome/scaffold end (bp)\"                                      \n",
       "[2862] \"Query protein or transcript ID\"                                                 \n",
       "[2863] \"Last common ancestor with Steppe mouse\"                                         \n",
       "[2864] \"Steppe mouse homology type\"                                                     \n",
       "[2865] \"%id. target Steppe mouse gene identical to query gene\"                          \n",
       "[2866] \"%id. query gene identical to target Steppe mouse gene\"                          \n",
       "[2867] \"Steppe mouse Gene-order conservation score\"                                     \n",
       "[2868] \"Steppe mouse Whole-genome alignment coverage\"                                   \n",
       "[2869] \"dN with Steppe mouse\"                                                           \n",
       "[2870] \"dS with Steppe mouse\"                                                           \n",
       "[2871] \"Steppe mouse orthology confidence [0 low, 1 high]\"                              \n",
       "[2872] \"Stickleback gene stable ID\"                                                     \n",
       "[2873] \"Stickleback gene name\"                                                          \n",
       "[2874] \"Stickleback protein or transcript stable ID\"                                    \n",
       "[2875] \"Stickleback chromosome/scaffold name\"                                           \n",
       "[2876] \"Stickleback chromosome/scaffold start (bp)\"                                     \n",
       "[2877] \"Stickleback chromosome/scaffold end (bp)\"                                       \n",
       "[2878] \"Query protein or transcript ID\"                                                 \n",
       "[2879] \"Last common ancestor with Stickleback\"                                          \n",
       "[2880] \"Stickleback homology type\"                                                      \n",
       "[2881] \"%id. target Stickleback gene identical to query gene\"                           \n",
       "[2882] \"%id. query gene identical to target Stickleback gene\"                           \n",
       "[2883] \"Stickleback Gene-order conservation score\"                                      \n",
       "[2884] \"Stickleback Whole-genome alignment coverage\"                                    \n",
       "[2885] \"dN with Stickleback\"                                                            \n",
       "[2886] \"dS with Stickleback\"                                                            \n",
       "[2887] \"Stickleback orthology confidence [0 low, 1 high]\"                               \n",
       "[2888] \"Swamp eel gene stable ID\"                                                       \n",
       "[2889] \"Swamp eel gene name\"                                                            \n",
       "[2890] \"Swamp eel protein or transcript stable ID\"                                      \n",
       "[2891] \"Swamp eel chromosome/scaffold name\"                                             \n",
       "[2892] \"Swamp eel chromosome/scaffold start (bp)\"                                       \n",
       "[2893] \"Swamp eel chromosome/scaffold end (bp)\"                                         \n",
       "[2894] \"Query protein or transcript ID\"                                                 \n",
       "[2895] \"Last common ancestor with Swamp eel\"                                            \n",
       "[2896] \"Swamp eel homology type\"                                                        \n",
       "[2897] \"%id. target Swamp eel gene identical to query gene\"                             \n",
       "[2898] \"%id. query gene identical to target Swamp eel gene\"                             \n",
       "[2899] \"Swamp eel Gene-order conservation score\"                                        \n",
       "[2900] \"Swamp eel Whole-genome alignment coverage\"                                      \n",
       "[2901] \"dN with Swamp eel\"                                                              \n",
       "[2902] \"dS with Swamp eel\"                                                              \n",
       "[2903] \"Swamp eel orthology confidence [0 low, 1 high]\"                                 \n",
       "[2904] \"Tarsier gene stable ID\"                                                         \n",
       "[2905] \"Tarsier gene name\"                                                              \n",
       "[2906] \"Tarsier protein or transcript stable ID\"                                        \n",
       "[2907] \"Tarsier chromosome/scaffold name\"                                               \n",
       "[2908] \"Tarsier chromosome/scaffold start (bp)\"                                         \n",
       "[2909] \"Tarsier chromosome/scaffold end (bp)\"                                           \n",
       "[2910] \"Query protein or transcript ID\"                                                 \n",
       "[2911] \"Last common ancestor with Tarsier\"                                              \n",
       "[2912] \"Tarsier homology type\"                                                          \n",
       "[2913] \"%id. target Tarsier gene identical to query gene\"                               \n",
       "[2914] \"%id. query gene identical to target Tarsier gene\"                               \n",
       "[2915] \"Tarsier Gene-order conservation score\"                                          \n",
       "[2916] \"Tarsier Whole-genome alignment coverage\"                                        \n",
       "[2917] \"dN with Tarsier\"                                                                \n",
       "[2918] \"dS with Tarsier\"                                                                \n",
       "[2919] \"Tarsier orthology confidence [0 low, 1 high]\"                                   \n",
       "[2920] \"Tasmanian devil gene stable ID\"                                                 \n",
       "[2921] \"Tasmanian devil gene name\"                                                      \n",
       "[2922] \"Tasmanian devil protein or transcript stable ID\"                                \n",
       "[2923] \"Tasmanian devil chromosome/scaffold name\"                                       \n",
       "[2924] \"Tasmanian devil chromosome/scaffold start (bp)\"                                 \n",
       "[2925] \"Tasmanian devil chromosome/scaffold end (bp)\"                                   \n",
       "[2926] \"Query protein or transcript ID\"                                                 \n",
       "[2927] \"Last common ancestor with Tasmanian devil\"                                      \n",
       "[2928] \"Tasmanian devil homology type\"                                                  \n",
       "[2929] \"%id. target Tasmanian devil gene identical to query gene\"                       \n",
       "[2930] \"%id. query gene identical to target Tasmanian devil gene\"                       \n",
       "[2931] \"Tasmanian devil Gene-order conservation score\"                                  \n",
       "[2932] \"Tasmanian devil Whole-genome alignment coverage\"                                \n",
       "[2933] \"dN with Tasmanian devil\"                                                        \n",
       "[2934] \"dS with Tasmanian devil\"                                                        \n",
       "[2935] \"Tasmanian devil orthology confidence [0 low, 1 high]\"                           \n",
       "[2936] \"Tetraodon gene stable ID\"                                                       \n",
       "[2937] \"Tetraodon gene name\"                                                            \n",
       "[2938] \"Tetraodon protein or transcript stable ID\"                                      \n",
       "[2939] \"Tetraodon chromosome/scaffold name\"                                             \n",
       "[2940] \"Tetraodon chromosome/scaffold start (bp)\"                                       \n",
       "[2941] \"Tetraodon chromosome/scaffold end (bp)\"                                         \n",
       "[2942] \"Query protein or transcript ID\"                                                 \n",
       "[2943] \"Last common ancestor with Tetraodon\"                                            \n",
       "[2944] \"Tetraodon homology type\"                                                        \n",
       "[2945] \"%id. target Tetraodon gene identical to query gene\"                             \n",
       "[2946] \"%id. query gene identical to target Tetraodon gene\"                             \n",
       "[2947] \"Tetraodon Gene-order conservation score\"                                        \n",
       "[2948] \"Tetraodon Whole-genome alignment coverage\"                                      \n",
       "[2949] \"dN with Tetraodon\"                                                              \n",
       "[2950] \"dS with Tetraodon\"                                                              \n",
       "[2951] \"Tetraodon orthology confidence [0 low, 1 high]\"                                 \n",
       "[2952] \"Tiger gene stable ID\"                                                           \n",
       "[2953] \"Tiger gene name\"                                                                \n",
       "[2954] \"Tiger protein or transcript stable ID\"                                          \n",
       "[2955] \"Tiger chromosome/scaffold name\"                                                 \n",
       "[2956] \"Tiger chromosome/scaffold start (bp)\"                                           \n",
       "[2957] \"Tiger chromosome/scaffold end (bp)\"                                             \n",
       "[2958] \"Query protein or transcript ID\"                                                 \n",
       "[2959] \"Last common ancestor with Tiger\"                                                \n",
       "[2960] \"Tiger homology type\"                                                            \n",
       "[2961] \"%id. target Tiger gene identical to query gene\"                                 \n",
       "[2962] \"%id. query gene identical to target Tiger gene\"                                 \n",
       "[2963] \"Tiger Gene-order conservation score\"                                            \n",
       "[2964] \"Tiger Whole-genome alignment coverage\"                                          \n",
       "[2965] \"dN with Tiger\"                                                                  \n",
       "[2966] \"dS with Tiger\"                                                                  \n",
       "[2967] \"Tiger orthology confidence [0 low, 1 high]\"                                     \n",
       "[2968] \"Tiger tail seahorse gene stable ID\"                                             \n",
       "[2969] \"Tiger tail seahorse gene name\"                                                  \n",
       "[2970] \"Tiger tail seahorse protein or transcript stable ID\"                            \n",
       "[2971] \"Tiger tail seahorse chromosome/scaffold name\"                                   \n",
       "[2972] \"Tiger tail seahorse chromosome/scaffold start (bp)\"                             \n",
       "[2973] \"Tiger tail seahorse chromosome/scaffold end (bp)\"                               \n",
       "[2974] \"Query protein or transcript ID\"                                                 \n",
       "[2975] \"Last common ancestor with Tiger tail seahorse\"                                  \n",
       "[2976] \"Tiger tail seahorse homology type\"                                              \n",
       "[2977] \"%id. target Tiger tail seahorse gene identical to query gene\"                   \n",
       "[2978] \"%id. query gene identical to target Tiger tail seahorse gene\"                   \n",
       "[2979] \"Tiger tail seahorse Gene-order conservation score\"                              \n",
       "[2980] \"Tiger tail seahorse Whole-genome alignment coverage\"                            \n",
       "[2981] \"dN with Tiger tail seahorse\"                                                    \n",
       "[2982] \"dS with Tiger tail seahorse\"                                                    \n",
       "[2983] \"Tiger tail seahorse orthology confidence [0 low, 1 high]\"                       \n",
       "[2984] \"Tilapia gene stable ID\"                                                         \n",
       "[2985] \"Tilapia gene name\"                                                              \n",
       "[2986] \"Tilapia protein or transcript stable ID\"                                        \n",
       "[2987] \"Tilapia chromosome/scaffold name\"                                               \n",
       "[2988] \"Tilapia chromosome/scaffold start (bp)\"                                         \n",
       "[2989] \"Tilapia chromosome/scaffold end (bp)\"                                           \n",
       "[2990] \"Query protein or transcript ID\"                                                 \n",
       "[2991] \"Last common ancestor with Tilapia\"                                              \n",
       "[2992] \"Tilapia homology type\"                                                          \n",
       "[2993] \"%id. target Tilapia gene identical to query gene\"                               \n",
       "[2994] \"%id. query gene identical to target Tilapia gene\"                               \n",
       "[2995] \"Tilapia Gene-order conservation score\"                                          \n",
       "[2996] \"Tilapia Whole-genome alignment coverage\"                                        \n",
       "[2997] \"dN with Tilapia\"                                                                \n",
       "[2998] \"dS with Tilapia\"                                                                \n",
       "[2999] \"Tilapia orthology confidence [0 low, 1 high]\"                                   \n",
       "[3000] \"Tongue sole gene stable ID\"                                                     \n",
       "[3001] \"Tongue sole gene name\"                                                          \n",
       "[3002] \"Tongue sole protein or transcript stable ID\"                                    \n",
       "[3003] \"Tongue sole chromosome/scaffold name\"                                           \n",
       "[3004] \"Tongue sole chromosome/scaffold start (bp)\"                                     \n",
       "[3005] \"Tongue sole chromosome/scaffold end (bp)\"                                       \n",
       "[3006] \"Query protein or transcript ID\"                                                 \n",
       "[3007] \"Last common ancestor with Tongue sole\"                                          \n",
       "[3008] \"Tongue sole homology type\"                                                      \n",
       "[3009] \"%id. target Tongue sole gene identical to query gene\"                           \n",
       "[3010] \"%id. query gene identical to target Tongue sole gene\"                           \n",
       "[3011] \"Tongue sole Gene-order conservation score\"                                      \n",
       "[3012] \"Tongue sole Whole-genome alignment coverage\"                                    \n",
       "[3013] \"dN with Tongue sole\"                                                            \n",
       "[3014] \"dS with Tongue sole\"                                                            \n",
       "[3015] \"Tongue sole orthology confidence [0 low, 1 high]\"                               \n",
       "[3016] \"Tree Shrew gene stable ID\"                                                      \n",
       "[3017] \"Tree Shrew gene name\"                                                           \n",
       "[3018] \"Tree Shrew protein or transcript stable ID\"                                     \n",
       "[3019] \"Tree Shrew chromosome/scaffold name\"                                            \n",
       "[3020] \"Tree Shrew chromosome/scaffold start (bp)\"                                      \n",
       "[3021] \"Tree Shrew chromosome/scaffold end (bp)\"                                        \n",
       "[3022] \"Query protein or transcript ID\"                                                 \n",
       "[3023] \"Last common ancestor with Tree Shrew\"                                           \n",
       "[3024] \"Tree Shrew homology type\"                                                       \n",
       "[3025] \"%id. target Tree Shrew gene identical to query gene\"                            \n",
       "[3026] \"%id. query gene identical to target Tree Shrew gene\"                            \n",
       "[3027] \"Tree Shrew Gene-order conservation score\"                                       \n",
       "[3028] \"Tree Shrew Whole-genome alignment coverage\"                                     \n",
       "[3029] \"dN with Tree Shrew\"                                                             \n",
       "[3030] \"dS with Tree Shrew\"                                                             \n",
       "[3031] \"Tree Shrew orthology confidence [0 low, 1 high]\"                                \n",
       "[3032] \"Tuatara gene stable ID\"                                                         \n",
       "[3033] \"Tuatara gene name\"                                                              \n",
       "[3034] \"Tuatara protein or transcript stable ID\"                                        \n",
       "[3035] \"Tuatara chromosome/scaffold name\"                                               \n",
       "[3036] \"Tuatara chromosome/scaffold start (bp)\"                                         \n",
       "[3037] \"Tuatara chromosome/scaffold end (bp)\"                                           \n",
       "[3038] \"Query protein or transcript ID\"                                                 \n",
       "[3039] \"Last common ancestor with Tuatara\"                                              \n",
       "[3040] \"Tuatara homology type\"                                                          \n",
       "[3041] \"%id. target Tuatara gene identical to query gene\"                               \n",
       "[3042] \"%id. query gene identical to target Tuatara gene\"                               \n",
       "[3043] \"Tuatara Gene-order conservation score\"                                          \n",
       "[3044] \"Tuatara Whole-genome alignment coverage\"                                        \n",
       "[3045] \"dN with Tuatara\"                                                                \n",
       "[3046] \"dS with Tuatara\"                                                                \n",
       "[3047] \"Tuatara orthology confidence [0 low, 1 high]\"                                   \n",
       "[3048] \"Turbot gene stable ID\"                                                          \n",
       "[3049] \"Turbot gene name\"                                                               \n",
       "[3050] \"Turbot protein or transcript stable ID\"                                         \n",
       "[3051] \"Turbot chromosome/scaffold name\"                                                \n",
       "[3052] \"Turbot chromosome/scaffold start (bp)\"                                          \n",
       "[3053] \"Turbot chromosome/scaffold end (bp)\"                                            \n",
       "[3054] \"Query protein or transcript ID\"                                                 \n",
       "[3055] \"Last common ancestor with Turbot\"                                               \n",
       "[3056] \"Turbot homology type\"                                                           \n",
       "[3057] \"%id. target Turbot gene identical to query gene\"                                \n",
       "[3058] \"%id. query gene identical to target Turbot gene\"                                \n",
       "[3059] \"Turbot Gene-order conservation score\"                                           \n",
       "[3060] \"Turbot Whole-genome alignment coverage\"                                         \n",
       "[3061] \"dN with Turbot\"                                                                 \n",
       "[3062] \"dS with Turbot\"                                                                 \n",
       "[3063] \"Turbot orthology confidence [0 low, 1 high]\"                                    \n",
       "[3064] \"Turkey gene stable ID\"                                                          \n",
       "[3065] \"Turkey gene name\"                                                               \n",
       "[3066] \"Turkey protein or transcript stable ID\"                                         \n",
       "[3067] \"Turkey chromosome/scaffold name\"                                                \n",
       "[3068] \"Turkey chromosome/scaffold start (bp)\"                                          \n",
       "[3069] \"Turkey chromosome/scaffold end (bp)\"                                            \n",
       "[3070] \"Query protein or transcript ID\"                                                 \n",
       "[3071] \"Last common ancestor with Turkey\"                                               \n",
       "[3072] \"Turkey homology type\"                                                           \n",
       "[3073] \"%id. target Turkey gene identical to query gene\"                                \n",
       "[3074] \"%id. query gene identical to target Turkey gene\"                                \n",
       "[3075] \"Turkey Gene-order conservation score\"                                           \n",
       "[3076] \"Turkey Whole-genome alignment coverage\"                                         \n",
       "[3077] \"dN with Turkey\"                                                                 \n",
       "[3078] \"dS with Turkey\"                                                                 \n",
       "[3079] \"Turkey orthology confidence [0 low, 1 high]\"                                    \n",
       "[3080] \"Ugandan red Colobus gene stable ID\"                                             \n",
       "[3081] \"Ugandan red Colobus gene name\"                                                  \n",
       "[3082] \"Ugandan red Colobus protein or transcript stable ID\"                            \n",
       "[3083] \"Ugandan red Colobus chromosome/scaffold name\"                                   \n",
       "[3084] \"Ugandan red Colobus chromosome/scaffold start (bp)\"                             \n",
       "[3085] \"Ugandan red Colobus chromosome/scaffold end (bp)\"                               \n",
       "[3086] \"Query protein or transcript ID\"                                                 \n",
       "[3087] \"Last common ancestor with Ugandan red Colobus\"                                  \n",
       "[3088] \"Ugandan red Colobus homology type\"                                              \n",
       "[3089] \"%id. target Ugandan red Colobus gene identical to query gene\"                   \n",
       "[3090] \"%id. query gene identical to target Ugandan red Colobus gene\"                   \n",
       "[3091] \"Ugandan red Colobus Gene-order conservation score\"                              \n",
       "[3092] \"Ugandan red Colobus Whole-genome alignment coverage\"                            \n",
       "[3093] \"dN with Ugandan red Colobus\"                                                    \n",
       "[3094] \"dS with Ugandan red Colobus\"                                                    \n",
       "[3095] \"Ugandan red Colobus orthology confidence [0 low, 1 high]\"                       \n",
       "[3096] \"Upper Galilee mountains blind mole rat gene stable ID\"                          \n",
       "[3097] \"Upper Galilee mountains blind mole rat gene name\"                               \n",
       "[3098] \"Upper Galilee mountains blind mole rat protein or transcript stable ID\"         \n",
       "[3099] \"Upper Galilee mountains blind mole rat chromosome/scaffold name\"                \n",
       "[3100] \"Upper Galilee mountains blind mole rat chromosome/scaffold start (bp)\"          \n",
       "[3101] \"Upper Galilee mountains blind mole rat chromosome/scaffold end (bp)\"            \n",
       "[3102] \"Query protein or transcript ID\"                                                 \n",
       "[3103] \"Last common ancestor with Upper Galilee mountains blind mole rat\"               \n",
       "[3104] \"Upper Galilee mountains blind mole rat homology type\"                           \n",
       "[3105] \"%id. target Upper Galilee mountains blind mole rat gene identical to query gene\"\n",
       "[3106] \"%id. query gene identical to target Upper Galilee mountains blind mole rat gene\"\n",
       "[3107] \"Upper Galilee mountains blind mole rat Gene-order conservation score\"           \n",
       "[3108] \"Upper Galilee mountains blind mole rat Whole-genome alignment coverage\"         \n",
       "[3109] \"dN with Upper Galilee mountains blind mole rat\"                                 \n",
       "[3110] \"dS with Upper Galilee mountains blind mole rat\"                                 \n",
       "[3111] \"Upper Galilee mountains blind mole rat orthology confidence [0 low, 1 high]\"    \n",
       "[3112] \"Vervet-AGM gene stable ID\"                                                      \n",
       "[3113] \"Vervet-AGM gene name\"                                                           \n",
       "[3114] \"Vervet-AGM protein or transcript stable ID\"                                     \n",
       "[3115] \"Vervet-AGM chromosome/scaffold name\"                                            \n",
       "[3116] \"Vervet-AGM chromosome/scaffold start (bp)\"                                      \n",
       "[3117] \"Vervet-AGM chromosome/scaffold end (bp)\"                                        \n",
       "[3118] \"Query protein or transcript ID\"                                                 \n",
       "[3119] \"Last common ancestor with Vervet-AGM\"                                           \n",
       "[3120] \"Vervet-AGM homology type\"                                                       \n",
       "[3121] \"%id. target Vervet-AGM gene identical to query gene\"                            \n",
       "[3122] \"%id. query gene identical to target Vervet-AGM gene\"                            \n",
       "[3123] \"Vervet-AGM Gene-order conservation score\"                                       \n",
       "[3124] \"Vervet-AGM Whole-genome alignment coverage\"                                     \n",
       "[3125] \"dN with Vervet-AGM\"                                                             \n",
       "[3126] \"dS with Vervet-AGM\"                                                             \n",
       "[3127] \"Vervet-AGM orthology confidence [0 low, 1 high]\"                                \n",
       "[3128] \"Wallaby gene stable ID\"                                                         \n",
       "[3129] \"Wallaby gene name\"                                                              \n",
       "[3130] \"Wallaby protein or transcript stable ID\"                                        \n",
       "[3131] \"Wallaby chromosome/scaffold name\"                                               \n",
       "[3132] \"Wallaby chromosome/scaffold start (bp)\"                                         \n",
       "[3133] \"Wallaby chromosome/scaffold end (bp)\"                                           \n",
       "[3134] \"Query protein or transcript ID\"                                                 \n",
       "[3135] \"Last common ancestor with Wallaby\"                                              \n",
       "[3136] \"Wallaby homology type\"                                                          \n",
       "[3137] \"%id. target Wallaby gene identical to query gene\"                               \n",
       "[3138] \"%id. query gene identical to target Wallaby gene\"                               \n",
       "[3139] \"Wallaby Gene-order conservation score\"                                          \n",
       "[3140] \"Wallaby Whole-genome alignment coverage\"                                        \n",
       "[3141] \"dN with Wallaby\"                                                                \n",
       "[3142] \"dS with Wallaby\"                                                                \n",
       "[3143] \"Wallaby orthology confidence [0 low, 1 high]\"                                   \n",
       "[3144] \"Western mosquitofish gene stable ID\"                                            \n",
       "[3145] \"Western mosquitofish gene name\"                                                 \n",
       "[3146] \"Western mosquitofish protein or transcript stable ID\"                           \n",
       "[3147] \"Western mosquitofish chromosome/scaffold name\"                                  \n",
       "[3148] \"Western mosquitofish chromosome/scaffold start (bp)\"                            \n",
       "[3149] \"Western mosquitofish chromosome/scaffold end (bp)\"                              \n",
       "[3150] \"Query protein or transcript ID\"                                                 \n",
       "[3151] \"Last common ancestor with Western mosquitofish\"                                 \n",
       "[3152] \"Western mosquitofish homology type\"                                             \n",
       "[3153] \"%id. target Western mosquitofish gene identical to query gene\"                  \n",
       "[3154] \"%id. query gene identical to target Western mosquitofish gene\"                  \n",
       "[3155] \"Western mosquitofish Gene-order conservation score\"                             \n",
       "[3156] \"Western mosquitofish Whole-genome alignment coverage\"                           \n",
       "[3157] \"dN with Western mosquitofish\"                                                   \n",
       "[3158] \"dS with Western mosquitofish\"                                                   \n",
       "[3159] \"Western mosquitofish orthology confidence [0 low, 1 high]\"                      \n",
       "[3160] \"White-throated sparrow gene stable ID\"                                          \n",
       "[3161] \"White-throated sparrow gene name\"                                               \n",
       "[3162] \"White-throated sparrow protein or transcript stable ID\"                         \n",
       "[3163] \"White-throated sparrow chromosome/scaffold name\"                                \n",
       "[3164] \"White-throated sparrow chromosome/scaffold start (bp)\"                          \n",
       "[3165] \"White-throated sparrow chromosome/scaffold end (bp)\"                            \n",
       "[3166] \"Query protein or transcript ID\"                                                 \n",
       "[3167] \"Last common ancestor with White-throated sparrow\"                               \n",
       "[3168] \"White-throated sparrow homology type\"                                           \n",
       "[3169] \"%id. target White-throated sparrow gene identical to query gene\"                \n",
       "[3170] \"%id. query gene identical to target White-throated sparrow gene\"                \n",
       "[3171] \"White-throated sparrow Gene-order conservation score\"                           \n",
       "[3172] \"White-throated sparrow Whole-genome alignment coverage\"                         \n",
       "[3173] \"dN with White-throated sparrow\"                                                 \n",
       "[3174] \"dS with White-throated sparrow\"                                                 \n",
       "[3175] \"White-throated sparrow orthology confidence [0 low, 1 high]\"                    \n",
       "[3176] \"Wild yak gene stable ID\"                                                        \n",
       "[3177] \"Wild yak gene name\"                                                             \n",
       "[3178] \"Wild yak protein or transcript stable ID\"                                       \n",
       "[3179] \"Wild yak chromosome/scaffold name\"                                              \n",
       "[3180] \"Wild yak chromosome/scaffold start (bp)\"                                        \n",
       "[3181] \"Wild yak chromosome/scaffold end (bp)\"                                          \n",
       "[3182] \"Query protein or transcript ID\"                                                 \n",
       "[3183] \"Last common ancestor with Wild yak\"                                             \n",
       "[3184] \"Wild yak homology type\"                                                         \n",
       "[3185] \"%id. target Wild yak gene identical to query gene\"                              \n",
       "[3186] \"%id. query gene identical to target Wild yak gene\"                              \n",
       "[3187] \"Wild yak Gene-order conservation score\"                                         \n",
       "[3188] \"Wild yak Whole-genome alignment coverage\"                                       \n",
       "[3189] \"dN with Wild yak\"                                                               \n",
       "[3190] \"dS with Wild yak\"                                                               \n",
       "[3191] \"Wild yak orthology confidence [0 low, 1 high]\"                                  \n",
       "[3192] \"Xenopus gene stable ID\"                                                         \n",
       "[3193] \"Xenopus gene name\"                                                              \n",
       "[3194] \"Xenopus protein or transcript stable ID\"                                        \n",
       "[3195] \"Xenopus chromosome/scaffold name\"                                               \n",
       "[3196] \"Xenopus chromosome/scaffold start (bp)\"                                         \n",
       "[3197] \"Xenopus chromosome/scaffold end (bp)\"                                           \n",
       "[3198] \"Query protein or transcript ID\"                                                 \n",
       "[3199] \"Last common ancestor with Xenopus\"                                              \n",
       "[3200] \"Xenopus homology type\"                                                          \n",
       "[3201] \"%id. target Xenopus gene identical to query gene\"                               \n",
       "[3202] \"%id. query gene identical to target Xenopus gene\"                               \n",
       "[3203] \"Xenopus Gene-order conservation score\"                                          \n",
       "[3204] \"Xenopus Whole-genome alignment coverage\"                                        \n",
       "[3205] \"dN with Xenopus\"                                                                \n",
       "[3206] \"dS with Xenopus\"                                                                \n",
       "[3207] \"Xenopus orthology confidence [0 low, 1 high]\"                                   \n",
       "[3208] \"Yellowtail amberjack gene stable ID\"                                            \n",
       "[3209] \"Yellowtail amberjack gene name\"                                                 \n",
       "[3210] \"Yellowtail amberjack protein or transcript stable ID\"                           \n",
       "[3211] \"Yellowtail amberjack chromosome/scaffold name\"                                  \n",
       "[3212] \"Yellowtail amberjack chromosome/scaffold start (bp)\"                            \n",
       "[3213] \"Yellowtail amberjack chromosome/scaffold end (bp)\"                              \n",
       "[3214] \"Query protein or transcript ID\"                                                 \n",
       "[3215] \"Last common ancestor with Yellowtail amberjack\"                                 \n",
       "[3216] \"Yellowtail amberjack homology type\"                                             \n",
       "[3217] \"%id. target Yellowtail amberjack gene identical to query gene\"                  \n",
       "[3218] \"%id. query gene identical to target Yellowtail amberjack gene\"                  \n",
       "[3219] \"Yellowtail amberjack Gene-order conservation score\"                             \n",
       "[3220] \"Yellowtail amberjack Whole-genome alignment coverage\"                           \n",
       "[3221] \"dN with Yellowtail amberjack\"                                                   \n",
       "[3222] \"dS with Yellowtail amberjack\"                                                   \n",
       "[3223] \"Yellowtail amberjack orthology confidence [0 low, 1 high]\"                      \n",
       "[3224] \"Zebra Finch gene stable ID\"                                                     \n",
       "[3225] \"Zebra Finch gene name\"                                                          \n",
       "[3226] \"Zebra Finch protein or transcript stable ID\"                                    \n",
       "[3227] \"Zebra Finch chromosome/scaffold name\"                                           \n",
       "[3228] \"Zebra Finch chromosome/scaffold start (bp)\"                                     \n",
       "[3229] \"Zebra Finch chromosome/scaffold end (bp)\"                                       \n",
       "[3230] \"Query protein or transcript ID\"                                                 \n",
       "[3231] \"Last common ancestor with Zebra Finch\"                                          \n",
       "[3232] \"Zebra Finch homology type\"                                                      \n",
       "[3233] \"%id. target Zebra Finch gene identical to query gene\"                           \n",
       "[3234] \"%id. query gene identical to target Zebra Finch gene\"                           \n",
       "[3235] \"Zebra Finch Gene-order conservation score\"                                      \n",
       "[3236] \"Zebra Finch Whole-genome alignment coverage\"                                    \n",
       "[3237] \"dN with Zebra Finch\"                                                            \n",
       "[3238] \"dS with Zebra Finch\"                                                            \n",
       "[3239] \"Zebra Finch orthology confidence [0 low, 1 high]\"                               \n",
       "[3240] \"Zebra mbuna gene stable ID\"                                                     \n",
       "[3241] \"Zebra mbuna gene name\"                                                          \n",
       "[3242] \"Zebra mbuna protein or transcript stable ID\"                                    \n",
       "[3243] \"Zebra mbuna chromosome/scaffold name\"                                           \n",
       "[3244] \"Zebra mbuna chromosome/scaffold start (bp)\"                                     \n",
       "[3245] \"Zebra mbuna chromosome/scaffold end (bp)\"                                       \n",
       "[3246] \"Query protein or transcript ID\"                                                 \n",
       "[3247] \"Last common ancestor with Zebra mbuna\"                                          \n",
       "[3248] \"Zebra mbuna homology type\"                                                      \n",
       "[3249] \"%id. target Zebra mbuna gene identical to query gene\"                           \n",
       "[3250] \"%id. query gene identical to target Zebra mbuna gene\"                           \n",
       "[3251] \"Zebra mbuna Gene-order conservation score\"                                      \n",
       "[3252] \"Zebra mbuna Whole-genome alignment coverage\"                                    \n",
       "[3253] \"dN with Zebra mbuna\"                                                            \n",
       "[3254] \"dS with Zebra mbuna\"                                                            \n",
       "[3255] \"Zebra mbuna orthology confidence [0 low, 1 high]\"                               \n",
       "[3256] \"Zebrafish gene stable ID\"                                                       \n",
       "[3257] \"Zebrafish gene name\"                                                            \n",
       "[3258] \"Zebrafish protein or transcript stable ID\"                                      \n",
       "[3259] \"Zebrafish chromosome/scaffold name\"                                             \n",
       "[3260] \"Zebrafish chromosome/scaffold start (bp)\"                                       \n",
       "[3261] \"Zebrafish chromosome/scaffold end (bp)\"                                         \n",
       "[3262] \"Query protein or transcript ID\"                                                 \n",
       "[3263] \"Last common ancestor with Zebrafish\"                                            \n",
       "[3264] \"Zebrafish homology type\"                                                        \n",
       "[3265] \"%id. target Zebrafish gene identical to query gene\"                             \n",
       "[3266] \"%id. query gene identical to target Zebrafish gene\"                             \n",
       "[3267] \"Zebrafish Gene-order conservation score\"                                        \n",
       "[3268] \"Zebrafish Whole-genome alignment coverage\"                                      \n",
       "[3269] \"dN with Zebrafish\"                                                              \n",
       "[3270] \"dS with Zebrafish\"                                                              \n",
       "[3271] \"Zebrafish orthology confidence [0 low, 1 high]\"                                 \n",
       "[3272] \"Zig-zag eel gene stable ID\"                                                     \n",
       "[3273] \"Zig-zag eel gene name\"                                                          \n",
       "[3274] \"Zig-zag eel protein or transcript stable ID\"                                    \n",
       "[3275] \"Zig-zag eel chromosome/scaffold name\"                                           \n",
       "[3276] \"Zig-zag eel chromosome/scaffold start (bp)\"                                     \n",
       "[3277] \"Zig-zag eel chromosome/scaffold end (bp)\"                                       \n",
       "[3278] \"Query protein or transcript ID\"                                                 \n",
       "[3279] \"Last common ancestor with Zig-zag eel\"                                          \n",
       "[3280] \"Zig-zag eel homology type\"                                                      \n",
       "[3281] \"%id. target Zig-zag eel gene identical to query gene\"                           \n",
       "[3282] \"%id. query gene identical to target Zig-zag eel gene\"                           \n",
       "[3283] \"Zig-zag eel Gene-order conservation score\"                                      \n",
       "[3284] \"Zig-zag eel Whole-genome alignment coverage\"                                    \n",
       "[3285] \"dN with Zig-zag eel\"                                                            \n",
       "[3286] \"dS with Zig-zag eel\"                                                            \n",
       "[3287] \"Zig-zag eel orthology confidence [0 low, 1 high]\"                               \n",
       "[3288] \"Human paralogue gene stable ID\"                                                 \n",
       "[3289] \"Human paralogue associated gene name\"                                           \n",
       "[3290] \"Human paralogue protein or transcript ID\"                                       \n",
       "[3291] \"Human paralogue chromosome/scaffold name\"                                       \n",
       "[3292] \"Human paralogue chromosome/scaffold start (bp)\"                                 \n",
       "[3293] \"Human paralogue chromosome/scaffold end (bp)\"                                   \n",
       "[3294] \"Paralogue query protein or transcript ID\"                                       \n",
       "[3295] \"Paralogue last common ancestor with Human\"                                      \n",
       "[3296] \"Human paralogue homology type\"                                                  \n",
       "[3297] \"Paralogue %id. target Human gene identical to query gene\"                       \n",
       "[3298] \"Paralogue %id. query gene identical to target Human gene\"                       \n",
       "[3299] \"Paralogue dN with Human\"                                                        \n",
       "[3300] \"Paralogue dS with Human\"                                                        \n",
       "[3301] \"Human paralogy confidence [0 low, 1 high]\"                                      \n",
       "[3302] \"Gene stable ID\"                                                                 \n",
       "[3303] \"Gene stable ID version\"                                                         \n",
       "[3304] \"Version (gene)\"                                                                 \n",
       "[3305] \"Transcript stable ID\"                                                           \n",
       "[3306] \"Transcript stable ID version\"                                                   \n",
       "[3307] \"Version (transcript)\"                                                           \n",
       "[3308] \"Protein stable ID\"                                                              \n",
       "[3309] \"Protein stable ID version\"                                                      \n",
       "[3310] \"Version (protein)\"                                                              \n",
       "[3311] \"Chromosome/scaffold name\"                                                       \n",
       "[3312] \"Gene start (bp)\"                                                                \n",
       "[3313] \"Gene end (bp)\"                                                                  \n",
       "[3314] \"Strand\"                                                                         \n",
       "[3315] \"Karyotype band\"                                                                 \n",
       "[3316] \"Gene name\"                                                                      \n",
       "[3317] \"Source of gene name\"                                                            \n",
       "[3318] \"Transcript count\"                                                               \n",
       "[3319] \"Gene % GC content\"                                                              \n",
       "[3320] \"Gene description\"                                                               \n",
       "[3321] \"Variant name\"                                                                   \n",
       "[3322] \"Variant source\"                                                                 \n",
       "[3323] \"Variant source description\"                                                     \n",
       "[3324] \"Variant alleles\"                                                                \n",
       "[3325] \"Variant supporting evidence\"                                                    \n",
       "[3326] \"Mapweight\"                                                                      \n",
       "[3327] \"Minor allele\"                                                                   \n",
       "[3328] \"Minor allele frequency\"                                                         \n",
       "[3329] \"Minor allele count\"                                                             \n",
       "[3330] \"Clinical significance\"                                                          \n",
       "[3331] \"Transcript location (bp)\"                                                       \n",
       "[3332] \"Variant chromosome Strand\"                                                      \n",
       "[3333] \"Protein location (aa)\"                                                          \n",
       "[3334] \"chromosome/scaffold position start (bp)\"                                        \n",
       "[3335] \"Chromosome/scaffold position end (bp)\"                                          \n",
       "[3336] \"PolyPhen prediction\"                                                            \n",
       "[3337] \"PolyPhen score\"                                                                 \n",
       "[3338] \"SIFT prediction\"                                                                \n",
       "[3339] \"SIFT score\"                                                                     \n",
       "[3340] \"Distance to transcript\"                                                         \n",
       "[3341] \"CDS start\"                                                                      \n",
       "[3342] \"CDS end\"                                                                        \n",
       "[3343] \"Protein allele\"                                                                 \n",
       "[3344] \"Variant consequence\"                                                            \n",
       "[3345] \"Consequence specific allele\"                                                    \n",
       "[3346] \"Gene stable ID\"                                                                 \n",
       "[3347] \"Gene stable ID version\"                                                         \n",
       "[3348] \"Version (gene)\"                                                                 \n",
       "[3349] \"Transcript stable ID\"                                                           \n",
       "[3350] \"Transcript stable ID version\"                                                   \n",
       "[3351] \"Version (transcript)\"                                                           \n",
       "[3352] \"Protein stable ID\"                                                              \n",
       "[3353] \"Protein stable ID version\"                                                      \n",
       "[3354] \"Version (protein)\"                                                              \n",
       "[3355] \"Chromosome/scaffold name\"                                                       \n",
       "[3356] \"Gene start (bp)\"                                                                \n",
       "[3357] \"Gene end (bp)\"                                                                  \n",
       "[3358] \"Strand\"                                                                         \n",
       "[3359] \"Karyotype band\"                                                                 \n",
       "[3360] \"Gene name\"                                                                      \n",
       "[3361] \"Source of gene name\"                                                            \n",
       "[3362] \"Transcript count\"                                                               \n",
       "[3363] \"Gene % GC content\"                                                              \n",
       "[3364] \"Gene description\"                                                               \n",
       "[3365] \"Variant name\"                                                                   \n",
       "[3366] \"Variant source\"                                                                 \n",
       "[3367] \"Variant source description\"                                                     \n",
       "[3368] \"Variant alleles\"                                                                \n",
       "[3369] \"Variant supporting evidence\"                                                    \n",
       "[3370] \"Mapweight\"                                                                      \n",
       "[3371] \"Minor allele\"                                                                   \n",
       "[3372] \"Minor allele frequency\"                                                         \n",
       "[3373] \"Minor allele count\"                                                             \n",
       "[3374] \"Clinical significance\"                                                          \n",
       "[3375] \"Transcript location (bp)\"                                                       \n",
       "[3376] \"Variant chromosome/scaffold strand\"                                             \n",
       "[3377] \"Protein location (aa)\"                                                          \n",
       "[3378] \"Chromosome/scaffold position start (bp)\"                                        \n",
       "[3379] \"Chromosome/scaffold position end (bp)\"                                          \n",
       "[3380] \"PolyPhen prediction\"                                                            \n",
       "[3381] \"PolyPhen score\"                                                                 \n",
       "[3382] \"SIFT prediction\"                                                                \n",
       "[3383] \"SIFT score\"                                                                     \n",
       "[3384] \"Distance to transcript\"                                                         \n",
       "[3385] \"CDS start\"                                                                      \n",
       "[3386] \"CDS end\"                                                                        \n",
       "[3387] \"Protein allele\"                                                                 \n",
       "[3388] \"Variant consequence\"                                                            \n",
       "[3389] \"Consequence specific allele\"                                                    \n",
       "[3390] \"Unspliced (Transcript)\"                                                         \n",
       "[3391] \"Unspliced (Gene)\"                                                               \n",
       "[3392] \"Flank (Transcript)\"                                                             \n",
       "[3393] \"Flank (Gene)\"                                                                   \n",
       "[3394] \"Flank-coding region (Transcript)\"                                               \n",
       "[3395] \"Flank-coding region (Gene)\"                                                     \n",
       "[3396] \"5' UTR\"                                                                         \n",
       "[3397] \"3' UTR\"                                                                         \n",
       "[3398] \"Exon sequences\"                                                                 \n",
       "[3399] \"cDNA sequences\"                                                                 \n",
       "[3400] \"Coding sequence\"                                                                \n",
       "[3401] \"Peptide\"                                                                        \n",
       "[3402] \"upstream_flank\"                                                                 \n",
       "[3403] \"downstream_flank\"                                                               \n",
       "[3404] \"Gene stable ID\"                                                                 \n",
       "[3405] \"Gene stable ID version\"                                                         \n",
       "[3406] \"Gene description\"                                                               \n",
       "[3407] \"Gene name\"                                                                      \n",
       "[3408] \"Source of gene name\"                                                            \n",
       "[3409] \"Chromosome/scaffold name\"                                                       \n",
       "[3410] \"Gene start (bp)\"                                                                \n",
       "[3411] \"Gene end (bp)\"                                                                  \n",
       "[3412] \"Gene type\"                                                                      \n",
       "[3413] \"Version (gene)\"                                                                 \n",
       "[3414] \"Ensembl Protein Family ID(s)\"                                                   \n",
       "[3415] \"UniParc ID\"                                                                     \n",
       "[3416] \"UniProtKB/Swiss-Prot ID\"                                                        \n",
       "[3417] \"UniProtKB/TrEMBL ID\"                                                            \n",
       "[3418] \"CDS start (within cDNA)\"                                                        \n",
       "[3419] \"CDS end (within cDNA)\"                                                          \n",
       "[3420] \"5' UTR start\"                                                                   \n",
       "[3421] \"5' UTR end\"                                                                     \n",
       "[3422] \"3' UTR start\"                                                                   \n",
       "[3423] \"3' UTR end\"                                                                     \n",
       "[3424] \"Transcript stable ID\"                                                           \n",
       "[3425] \"Transcript stable ID version\"                                                   \n",
       "[3426] \"Protein stable ID\"                                                              \n",
       "[3427] \"Protein stable ID version\"                                                      \n",
       "[3428] \"Transcript type\"                                                                \n",
       "[3429] \"Version (transcript)\"                                                           \n",
       "[3430] \"Version (protein)\"                                                              \n",
       "[3431] \"Strand\"                                                                         \n",
       "[3432] \"Transcript start (bp)\"                                                          \n",
       "[3433] \"Transcript end (bp)\"                                                            \n",
       "[3434] \"Transcription start site (TSS)\"                                                 \n",
       "[3435] \"Transcript length (including UTRs and CDS)\"                                     \n",
       "[3436] \"CDS Length\"                                                                     \n",
       "[3437] \"CDS start\"                                                                      \n",
       "[3438] \"CDS end\"                                                                        \n",
       "[3439] \"Exon stable ID\"                                                                 \n",
       "[3440] \"Exon region start (bp)\"                                                         \n",
       "[3441] \"Exon region end (bp)\"                                                           \n",
       "[3442] \"Strand\"                                                                         \n",
       "[3443] \"Exon rank in transcript\"                                                        \n",
       "[3444] \"Start phase\"                                                                    \n",
       "[3445] \"End phase\"                                                                      \n",
       "[3446] \"cDNA coding start\"                                                              \n",
       "[3447] \"cDNA coding end\"                                                                \n",
       "[3448] \"Genomic coding start\"                                                           \n",
       "[3449] \"Genomic coding end\"                                                             \n",
       "[3450] \"Constitutive exon\"                                                              "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "listAttributes(ensembl)$description"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "hidden": true,
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 3 × 3</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>name</th><th scope=col>description</th><th scope=col>page</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>83</th><td>reactome           </td><td>Reactome ID           </td><td>feature_page</td></tr>\n",
       "\t<tr><th scope=row>84</th><td>reactome_gene      </td><td>Reactome gene ID      </td><td>feature_page</td></tr>\n",
       "\t<tr><th scope=row>85</th><td>reactome_transcript</td><td>Reactome transcript ID</td><td>feature_page</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 3 × 3\n",
       "\\begin{tabular}{r|lll}\n",
       "  & name & description & page\\\\\n",
       "  & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t83 & reactome            & Reactome ID            & feature\\_page\\\\\n",
       "\t84 & reactome\\_gene       & Reactome gene ID       & feature\\_page\\\\\n",
       "\t85 & reactome\\_transcript & Reactome transcript ID & feature\\_page\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 3 × 3\n",
       "\n",
       "| <!--/--> | name &lt;chr&gt; | description &lt;chr&gt; | page &lt;chr&gt; |\n",
       "|---|---|---|---|\n",
       "| 83 | reactome            | Reactome ID            | feature_page |\n",
       "| 84 | reactome_gene       | Reactome gene ID       | feature_page |\n",
       "| 85 | reactome_transcript | Reactome transcript ID | feature_page |\n",
       "\n"
      ],
      "text/plain": [
       "   name                description            page        \n",
       "83 reactome            Reactome ID            feature_page\n",
       "84 reactome_gene       Reactome gene ID       feature_page\n",
       "85 reactome_transcript Reactome transcript ID feature_page"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "listAttributes(ensembl)[grepl('Reactome',listAttributes(ensembl)$description),]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "hidden": true,
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 4 × 3</caption>\n",
       "<thead>\n",
       "\t<tr><th></th><th scope=col>name</th><th scope=col>description</th><th scope=col>page</th></tr>\n",
       "\t<tr><th></th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><th scope=row>41</th><td>go_id          </td><td>GO term accession    </td><td>feature_page</td></tr>\n",
       "\t<tr><th scope=row>42</th><td>name_1006      </td><td>GO term name         </td><td>feature_page</td></tr>\n",
       "\t<tr><th scope=row>43</th><td>definition_1006</td><td>GO term definition   </td><td>feature_page</td></tr>\n",
       "\t<tr><th scope=row>44</th><td>go_linkage_type</td><td>GO term evidence code</td><td>feature_page</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 4 × 3\n",
       "\\begin{tabular}{r|lll}\n",
       "  & name & description & page\\\\\n",
       "  & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t41 & go\\_id           & GO term accession     & feature\\_page\\\\\n",
       "\t42 & name\\_1006       & GO term name          & feature\\_page\\\\\n",
       "\t43 & definition\\_1006 & GO term definition    & feature\\_page\\\\\n",
       "\t44 & go\\_linkage\\_type & GO term evidence code & feature\\_page\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 4 × 3\n",
       "\n",
       "| <!--/--> | name &lt;chr&gt; | description &lt;chr&gt; | page &lt;chr&gt; |\n",
       "|---|---|---|---|\n",
       "| 41 | go_id           | GO term accession     | feature_page |\n",
       "| 42 | name_1006       | GO term name          | feature_page |\n",
       "| 43 | definition_1006 | GO term definition    | feature_page |\n",
       "| 44 | go_linkage_type | GO term evidence code | feature_page |\n",
       "\n"
      ],
      "text/plain": [
       "   name            description           page        \n",
       "41 go_id           GO term accession     feature_page\n",
       "42 name_1006       GO term name          feature_page\n",
       "43 definition_1006 GO term definition    feature_page\n",
       "44 go_linkage_type GO term evidence code feature_page"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "listAttributes(ensembl)[grepl('GO term',listAttributes(ensembl)$description),]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'100130426'</li>\n",
       "\t<li>'100133144'</li>\n",
       "\t<li>'100134869'</li>\n",
       "\t<li>'10357'</li>\n",
       "\t<li>'10431'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item '100130426'\n",
       "\\item '100133144'\n",
       "\\item '100134869'\n",
       "\\item '10357'\n",
       "\\item '10431'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. '100130426'\n",
       "2. '100133144'\n",
       "3. '100134869'\n",
       "4. '10357'\n",
       "5. '10431'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] \"100130426\" \"100133144\" \"100134869\" \"10357\"     \"10431\"    "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "name='lihc'\n",
    "ids=sapply(strsplit(colnames(rnaseq_dataset_list[[name]])[-c(1)],\"[|]\"),function(x){x[2]})\n",
    "ids[1:5]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "hidden": true
   },
   "source": [
    "http://geneontology.org/docs/guide-go-evidence-codes/\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "                                                                      \r"
     ]
    },
    {
     "data": {
      "text/html": [
       "<table>\n",
       "<caption>A data.frame: 6 × 5</caption>\n",
       "<thead>\n",
       "\t<tr><th scope=col>entrezgene_id</th><th scope=col>hgnc_symbol</th><th scope=col>go_id</th><th scope=col>name_1006</th><th scope=col>go_linkage_type</th></tr>\n",
       "\t<tr><th scope=col>&lt;int&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th><th scope=col>&lt;chr&gt;</th></tr>\n",
       "</thead>\n",
       "<tbody>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>          </td><td>                     </td><td>   </td></tr>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>GO:0005576</td><td>extracellular region </td><td>IDA</td></tr>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>GO:0005576</td><td>extracellular region </td><td>IEA</td></tr>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>GO:0005615</td><td>extracellular space  </td><td>HDA</td></tr>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>GO:0005576</td><td>extracellular region </td><td>TAS</td></tr>\n",
       "\t<tr><td>1</td><td>A1BG</td><td>GO:0070062</td><td>extracellular exosome</td><td>HDA</td></tr>\n",
       "</tbody>\n",
       "</table>\n"
      ],
      "text/latex": [
       "A data.frame: 6 × 5\n",
       "\\begin{tabular}{r|lllll}\n",
       " entrezgene\\_id & hgnc\\_symbol & go\\_id & name\\_1006 & go\\_linkage\\_type\\\\\n",
       " <int> & <chr> & <chr> & <chr> & <chr>\\\\\n",
       "\\hline\n",
       "\t 1 & A1BG &            &                       &    \\\\\n",
       "\t 1 & A1BG & GO:0005576 & extracellular region  & IDA\\\\\n",
       "\t 1 & A1BG & GO:0005576 & extracellular region  & IEA\\\\\n",
       "\t 1 & A1BG & GO:0005615 & extracellular space   & HDA\\\\\n",
       "\t 1 & A1BG & GO:0005576 & extracellular region  & TAS\\\\\n",
       "\t 1 & A1BG & GO:0070062 & extracellular exosome & HDA\\\\\n",
       "\\end{tabular}\n"
      ],
      "text/markdown": [
       "\n",
       "A data.frame: 6 × 5\n",
       "\n",
       "| entrezgene_id &lt;int&gt; | hgnc_symbol &lt;chr&gt; | go_id &lt;chr&gt; | name_1006 &lt;chr&gt; | go_linkage_type &lt;chr&gt; |\n",
       "|---|---|---|---|---|\n",
       "| 1 | A1BG | <!----> | <!----> | <!----> |\n",
       "| 1 | A1BG | GO:0005576 | extracellular region  | IDA |\n",
       "| 1 | A1BG | GO:0005576 | extracellular region  | IEA |\n",
       "| 1 | A1BG | GO:0005615 | extracellular space   | HDA |\n",
       "| 1 | A1BG | GO:0005576 | extracellular region  | TAS |\n",
       "| 1 | A1BG | GO:0070062 | extracellular exosome | HDA |\n",
       "\n"
      ],
      "text/plain": [
       "  entrezgene_id hgnc_symbol go_id      name_1006             go_linkage_type\n",
       "1 1             A1BG                                                        \n",
       "2 1             A1BG        GO:0005576 extracellular region  IDA            \n",
       "3 1             A1BG        GO:0005576 extracellular region  IEA            \n",
       "4 1             A1BG        GO:0005615 extracellular space   HDA            \n",
       "5 1             A1BG        GO:0005576 extracellular region  TAS            \n",
       "6 1             A1BG        GO:0070062 extracellular exosome HDA            "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "go_map <- getBM(attributes = c('entrezgene_id', 'hgnc_symbol', \n",
    "                     'go_id', 'name_1006','go_linkage_type'),\n",
    "      filters = 'entrezgene_id', \n",
    "      values = ids, \n",
    "      mart = ensembl)\n",
    "head(go_map)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "The downloaded binary packages are in\n",
      "\t/var/folders/00/hrw448cj27l47l_0f2lrz1rr0000gp/T//RtmpHgXX6D/downloaded_packages\n"
     ]
    }
   ],
   "source": [
    "install.packages('jjb')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "library(jjb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "mkdir('data/')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "go_map %>% write_csv('data/gene_ontology_mappings.csv')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "## RNA level distributions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "name = 'lihc'\n",
    "df <- rnaseq_dataset_list[[name]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>423</li>\n",
       "\t<li>20532</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 423\n",
       "\\item 20532\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 423\n",
       "2. 20532\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1]   423 20532"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "dim(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df_rownames <- df[,1]\n",
    "df <- df %>% select(-bcr_patient_barcode)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'A'</li>\n",
       "\t<li>'A'</li>\n",
       "\t<li>'N'</li>\n",
       "\t<li>'A'</li>\n",
       "\t<li>'T'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'A'\n",
       "\\item 'A'\n",
       "\\item 'N'\n",
       "\\item 'A'\n",
       "\\item 'T'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'A'\n",
       "2. 'A'\n",
       "3. 'N'\n",
       "4. 'A'\n",
       "5. 'T'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "[1] \"A\" \"A\" \"N\" \"A\" \"T\""
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "x <- strsplit(colnames(df),\"|\")[[50]]\n",
    "paste(x[1:(which(x %in% \"|\")-1)],collapse='')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {
    "hidden": true,
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "df_colnames_entrez <- sapply(strsplit(colnames(df),\"|\"),\n",
    "                      function(x){\n",
    "                          as.integer(paste(x[(which(x %in% \"|\")+1):length(x)],collapse=''))\n",
    "                      })\n",
    "df_colnames_genes <- sapply(strsplit(colnames(df),\"|\"),\n",
    "                      function(x){\n",
    "                          paste(x[1:(which(x %in% \"|\")-1)],collapse='')\n",
    "                      })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Parsed with column specification:\n",
      "cols(\n",
      "  entrezgene_id = \u001b[32mcol_double()\u001b[39m,\n",
      "  hgnc_symbol = \u001b[31mcol_character()\u001b[39m,\n",
      "  go_id = \u001b[31mcol_character()\u001b[39m,\n",
      "  name_1006 = \u001b[31mcol_character()\u001b[39m,\n",
      "  go_linkage_type = \u001b[31mcol_character()\u001b[39m\n",
      ")\n"
     ]
    }
   ],
   "source": [
    "go_map <- read_csv('data/gene_ontology_mappings.csv') %>% select(entrezgene_id,hgnc_symbol) %>% distinct()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df_genes <- df[,df_colnames_genes %in% go_map$hgnc_symbol]\n",
    "colnames(df_genes) <- df_colnames_genes[which(df_colnames_genes %in% go_map$hgnc_symbol)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA0gAAANICAYAAAD958/bAAAEGWlDQ1BrQ0dDb2xvclNwYWNl\nR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi\n6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lp\nurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZP\nC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q4\n4WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23B\naIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys\n2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y\n5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrl\nSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98\nhTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7C\nlP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmK\nPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZf\nsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJ\nxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19\nzn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNC\nUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU\n97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KT\nYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyA\ngccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/\nqwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7N0LvG1lXS/8TXITFQQCQxS8ZPl6IREUwQCjpMxU\nfEmTOl7KS3UyPerpeLJEKbB8U3m99b4UR6EUs5N5SYUS84ACXtEwDVKQO4bcBRQBOb8/rOkZ\nYznX2nOuvebaa47n+3w+P+Z8xhhzzPF8n70267/HmGNu2KARIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ\nIECAAAECBAgQIECAAAECBAgQIECAAAECBAiME9hi3ELLCBAgQGBdCtwjR/XgzpF9Pc9v7vS7\nTx+Rzo8sLPhmHq/srKx91L6q/cdC7uz4zyAF6v/1D032S+6WnJv8W3JNohEgQIAAAQIECBCY\nW4EDc+R3dLL/MiO5pbPdqxdtd2Zn3R8vWrcp3YfnxS/dlB147aoLVCF8etL9c1PP/3zV38kO\nCRAgMBCB0b8uDmQ4hkGAAAECm0Fgx7znW5IvJQdvhvf3lksLVAFchfXi9sXFC/QJECBA4C6B\nLUEQIECAQHMCl2fEFyyM+tpVGP0bs49fX4X92MXqCzyys8sz8vw3krrkri6t1AgQIEBgjIAC\naQyKRQQIEBi4wC+v8vi6n2ety7e09SPwoM6hfDDP/73T95QAAQIExggokMagWESAAIGBCzwr\n47vvwhg/ncf6TFK3/XQ6v5Dskdw7qRs81BmndycXJaO2Q548P6nPHo3aQ/Lk5cl3kv9vtHDh\nsS7rfmLypKT2XWevzklOS+ryvKXaVllxRPLYZKekjrd+2b8k+d2k1lc7PrnhzmcbNvxWHrdb\neP6OPNYx/qfktuTk5J8Wnudhw/2TZycPTnZN6ti/mZya/EPSLfpqDP8lGbVj8+RHk8OSn06+\nnXwq+buk3quKx19Kfia5T3JGckpSntO2afyenp0/MNm58yblV3NzXVImGgECBAgQIECAAIG5\nFqjPktQv66Os9k0aqtCowmO0/8WPt2fd4cmo1dmJxduM+lePNlp43D6PVYyN1ncfa79VaNw9\nWdzqPT6bdLev51clNf4bO+v2zPNRq6Ju9JrfzvNbO/26vGz0D4S1rgqZ0baLH9+ddVsno1ZG\n3W32Tb8Kte6yev7e5J4Lj4vXVWH4sGSaNq1fFYGL33fU//o0b2xbAgQIECBAgAABAutVYHGB\n9IYc6O8ske4v/a9eNKA6AzP6ZfmPO+te31led8H7fPKRpIqR0fb1fLek2qQF0j2zbfc9a19V\nFI32OXr85yyrMy6jVmdM/iUZra/HKnwuXlh2cx6749wz/VHrFki1XXcfb13YqAqs73fWfTfP\nv5zU7a+723f9FhdIo31XkdQt1ur1o/2UZfd4at03khrfJG0lfgqkSWRtQ4AAAQIECBAgMNcC\niwuk7i/xyz3v/oJfAN1ipVsg1edTRvt5fEeqfpH/ZFIFxOeT5yXV6sxK3QSge9bp1IVl3TMk\nH86y0X6/nefPS+qX/l2SP0m6RUpdGjdqz8+T0evqsS6nG7Wfz5PaV3f9A0Yr87i4IPnbLHta\n8obk0Um1Nyej9/54nt+9FqZVEfSZZLTvuiRu1BYXSN/Lip9dWHmvPF6QjF5Xjx9Ldki2SWqs\n3XXdSxOzasm2Er+6hPLHkyuS0XvWn4NatmeiESBAgAABAgQIEJh7gVkXSKMzM/UL9SeS/zu5\n94LaPfI4uixtYdEPHt6ZZ/Wayvt+sPSuJz+xsHy0vlsAjTY9qbPN5aOFeeyeBakCZnE7JgtG\n+63HB3Q26BZIF2X5tp113adV1ByQVEHRbX+Yzmjfn+2sWFwgvbGzrp6+LRm9roqv+9XChVbv\nMVpXj09YWL7cw6b41X4vSUbv+aLl3sg6AgQIELhLYKn/2fEhQIAAgfUv8Jc5xCpqxrXXZuHd\nxq1YZlkVIc9bWP+EPFbql/yzk7rUrs4UfTGZpj2hs/Htef6OTn/0tMZxxEKnLt+rM0vfSh60\nsKwexhVIdWbnVZ1tlnr6D1lRZ7/GtToLdWZSxcszkv2SA5PHJKNWZ3+Wap9btOL6Tr/m5tJO\nvy6567bl9jva7gmjJ3mc1q/zUk8JECBAYFIBBdKkUrYjQIDA+hN4Zw7prCUO69VZPm2B9NK8\nZq9kdAla7fpHkn0X8po8VqFUZyIuTyZp3TMol+UF3xvzorosrdsels5pyQM6C/+j83z0tC4f\nm6TVGaSl2i9lxdHJTy21QZZXkbhU6xZEtc2tnQ3rbnHdNm7s3fXjnm+K37j9WUaAAAECGxGo\n//FpBAgQIECgBG5I6lbQhydVCN2YLG5PzoKTFi9cpt8tEupytnGtPqPTbVctdLrFxy7dDRae\nj1s2ZrMNdSOFce25WfiBZFQcfTnP65K5pyS/n4zacgVStyCq7etytlFbSUE0eu3ocVP8Rvvw\nSIAAAQJTCCiQpsCyKQECBBoQqMu4qmioMys7Jo9PjkouTEbt4DzZedRZ9Lj4/ysXdtbX/h7c\n6Y+e1hmqUaui4t8XOheOFubxcZ3no6dPGD3ZyGPdRW5ce1UWjs6yvT3P6+zZf00+nGyRjFqZ\nbK52YeeNp/XrvNRTAgQIEJhUYPH/yCZ9ne0IECBAYFgCdRblg8m5SZ1x2Se5LanP57w2+dWk\n27budGq7Uesur2X/mNw0WpnHOkNTNzoYtToL9IejTh5PTUZnZT7SWf7UPK+ibdSqqPpvo85G\nHrvHN9r0R/PkJ0adPFZR1G0HdTqb83L0TfHrDGHqp+VTZwtH6RaMU+/MCwgQIDBPApvzL/15\ncnKsBAgQGLrA1zLAOls0OjN0Yp5XMVNf7vqI5EXJqNUZnu7nf749WpHHJyR1d7n7J89Jqjiq\nz0O9Kan2tKT2+f7kHsmzk92TarXt79z57K7/vD0Pv5fUdvULehVwdfvtOqNTN1PoFlrpLtm6\nl72NNro2T76TjG7t/Zo8H13O9sI8/4Vk1O45erIZHjfFb1MOtz6H1i0a6/eFctcIECBAgAAB\nAgQIrBuBA3Mk9cv+KPsvc2R1Wdlou1cv2u7Mzro/7qyroqDu9jZ63bjH2u/iy90OW+I122f5\nqL04T+qzPOP2WctuTH4lWdwOzoLrk8Wvq7Ncr1y0fM/0R617m+/njhYueqyibfF+R/2LO+vK\nZFQkVVE22qYefy7pttekM1rfvT14bVNXbYzW1ePP18IJ20r9Lsn+R+/ZLXI39raHdl5Xrx9d\nirix11lPgACBuRdwid3cT6EBECBAYNUETsmeqgj7UDLuxgQfyfI6c1NngLqtlr+zuyDPr0p2\n6ix7W54/Kzk5qYJn1Krg+sdkr+S9o4Wdx9Py/OCk9v/1pM5c1XaHJN0zHOneeUaoHidtv58N\nj026l+B9J/3/N/nJ5KKk2jZJFYGbs63Ub3Mes/cmQIAAAQIECBAgMBiBOmNSt9t+QvLQZHQp\nWp4u2eryvLpM735LbnHXivrHuZ9KHp4sd6n3Llk/OnOTpz/UFp9R2/aHtphsQZ3p2jd5ZLLc\n8Uy2t9lvNanf7I/EOxAgQIAAAQIECBAgsGYCr8s71eVdVydnJz+TdNtr06n1lfMTjQABAgQI\nbLLAPPxL2SYP0g4IECBAYC4Fzlg46rpUr3JSUpfV3ZDUXfZ+Ohm140dPPBIgQIAAAQIECBAg\nQGCoAu/KwEZniZZ6rLNLWw8VwLgIECBAgAABAgQIECAwEqjP2zw9+VhyYVI3dbg9qbuznZ78\n58TVEEHQCBAgQIAAAQIECBBoT6C+D0lB1N68GzEBAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBA\ngAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgMA6FthiHR/brA5tx+x4h2Sb5MbkuuSmRCNA\ngAABAgQIECBAgEATAntnlMcnVyZ3jMn5WXZcskuiESBAgAABAgQIECBAYLACR2Zko6Loojw/\nM/lw8jfJyclnkiuS2uaq5FcTjQABAgQIECBAgAABAoMTeEZGVIVPFUKPXmZ0danhQcnnktr+\ngEQjQIAAAQIECBAgQIDAoATendHU5XP1eaNJWn0+6Ybk/59kY9sQIECAAAECBAgQIDAsgR8Z\n1nB+aDR7ZclZyS0/tGb8gmuz+Jxk9/GrLSVAgAABAgQIECBAYMgCQy+Q6rNF+yRbTTiJdQap\niqpzJ9zeZgQIECBAgAABAgQIEJgbgV/LkdZnij6U7LfMUddnkA5M6oYNtyWPTzQCBAgQIECA\nAAECBAgMSqAKn5cl9T1HVShdmnw6+UjynoXHugTv8qTW35q8NNEIECBAgAABAgQIECAwWIEH\nZWRVEF2WVCHUTRVPX0vekNw/0QgQIECAAAECBAgQaFSgzrC01rbPgHdItk3qi2OvTzQCBAgQ\nIECAAAECBAhsaLFA6k77nun8ZFKF0nnJdxKNAAECBAgQIECAAIFGBYZeIP1m5vXg5PlJt/h5\nZPrvSPZNRq3OJP1JUpfa3T5aOLDH+2Y8jxrYmJYazgVZ4W6ES+lYToAAAQIECBAgMFZg6AVS\nFUG/ntw7GV1KV58z+nJSl9l9PvlCUpfdHZTsnhybvDwZYjs+g3pu8r0hDq4zprqt+78kj+ks\n85QAAQIECBAgQIBA8wJVINUNGaoYGrV350kte/FowcLjdnkcrfu5RetW0q3ic5qs5D2mfc0J\neUFl6O1lGeDZQx+k8REgQIAAAQIECKy+wNC/KHac2AFZ+NnkbYtW3pz+C5Krk0MWrZu2++C8\noM7SfH+K1GV9Lc5Hhq0RIECAAAECBAgQWB8CW66Pw1jTo6jL6T6+xDvW55TqcyuPWGL9pIsv\nyIYHJltP+IKHZ7s/T2o+hn7524QkNiNAgAABAgQIECCw9gItFkj1maO6ScO4tnMW1udWThi3\ncopldQlffSHtpK2+oFYjQIAAAQIECBAgQGAzC7RySVddUlefL6qbL5yZ1N3rnpp02x7p1GV3\nddbntO4KzwkQIECAAAECBAgQIDAEgV/OIP4+qUve6qxONxenP2pPzpM6i1Prz0jq5gpr2fbP\nm9V7T3pJ3kqP7YS8sDL05iYNQ59h4yNAgAABAgQIzEhg6JfY/V3cKtXqTnb1HUCjdIugu2V5\nff7oPUn9cl3FikaAAAECBAgQIECAQGMCQy+QutN5fTp16dy4y+c+luX1+SOfBQqCRoAAAQIE\nCBAgQKBVgVY+gzSa36XGW2ePqjiqM0k7JtsmGgECBAgQIECAAAECjQksVTAMieE+Gcx7k2uS\nG5JPJI9PxrW6u11t98pxKy0jQIAAAQIECBAgQGDYAkMvkO6Z6ftc8sykzg5dmhycnJ4ck2gE\nCBAgQIAAAQIECBD4gcDQC6Tfy0jvnxyV3C95aFLfc/SvyauSNyUaAQIECBAgQIAAAQIE7hQY\neoF0QEZ5ZXJ08u07R7xhQ31R7EHJJ5O6Y10VURoBAgQIECBAgAABAgQ2DL1A2j1zXIXQbYvm\nuu5o90vJOcnrk7oETyNAgAABAgQIECBAoHGBoRdIF2V+fy4Zd1e6umHDLyb1uaQTk6Vu3JBV\nGgECBAgQIECAAAECLQgMvUD6eCaxviD2dcl9x0zoZVn2xKQuv/to8uREI0CAAAECBAgQIECg\nUYGhF0hvy7x+NanPGl2SPCtZ3M7LgkOT7yf1WaVqW9z14L8ECBAgQIAAAQIECLQkMPQC6buZ\nzP2StyQXJ99LxrUvZeG+ySnjVlpGgAABAgQIECBAgEAbAls2MMwbM8aXLmS5gvD8bPOkpG4D\nXoWVRoAAAQIECBAgQIBAYwItFEjdKa3L6DbW6otlNQIECBAgQIAAAQIEGhRY7oxKgxyGTIAA\nAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaN\nnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6\nHDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIK\npJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACB\nnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIE\nCLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAAB\nAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIE\nCBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQAB\nAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoE\nCBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn\n39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoAC\nqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQs\noEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQI\nEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAg\nQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQI\nECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAg\nQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJ\nECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqceh\nQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBq\nefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJ\nKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBA\nywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBA\ngACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAA\nAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBA\ngAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnoACqcehQ4AA\nAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6HDoECBAgQIAAAQIECLQsoEBqefaN\nnQABAgQIECBAgACBnoACqcehQ4AAAQIECBAgQIBAywIKpJZn39gJECBAgAABAgQIEOgJKJB6\nHDoECBAgQIAAAQIECLQsoEBqefaNnQABAgQIECBAgACBnsCWvV4bnR0zzB2SbZIbk+uSmxKN\nAAECBAgQIECAAIHGBVo5g7R35vn45MrkmuQbybnJpUkVSecnxyW7JBoBAgQIECBAgAABAo0K\ntHAG6cjM7VEL83txHs9KqkiqwqjOJO2U7JG8KDk8eUlyUqIRIECAAAECBAgQIEBgUALPyGju\nSE5OHr3MyLbIuoOSzyW1/QHJWrb982b1vlvP+E1PyP4rQ28vywDPHvogjY8AAQIECBAgQGD1\nBYZ+id1hIbsgqcflfmGu4uT05NDk28lzEo0AAQIECBAgQIAAgcYEhl4g7ZX5rEvqbplwXq/N\nducku0+4vc0IECBAgAABAgQIEBiQwNALpCsyV/skW004Z3WHuyqq6gYOGgECBAgQIECAAAEC\njQkMvUA6MfP50OR9yX7LzG19BunA5JRku+QDiUaAAAECBAgQIECAQGMCQ7+LXd2Nbtfk6OQp\nyWVJ3dr76uSGZPuk7mK3Z7JbclvyiuSMRCNAgAABAgQIECBAoDGBoRdIdfOFY5MPJsckdae6\nxWeSbs6yy5M3Jm9OLkk0AgQIECBAgAABAgQaFBh6gTSa0rqT3RELnTprVN9/tG1SXxx7faIR\nIECAAAECBAgQIEBgw9A/gzRuiu+WhZUa+z2TeyQaAQIECBAgQIAAAQIEmimQ9s5cH5/UGaNr\nkm8kdae6+jzSjcn5yXHJLolGgAABAgQIECBAgECjAi1cYndk5vaohfm9OI/1vUhVJFVhVJfa\n1U0a9khelByevCSpmztoBAgQIECAAAECBAgQGJTAMzKaulHDycmjlxlZ3ea7buDwuaS2PyBZ\ny7Z/3qzed+sZv+kJ2X9l6O1lGeDZQx+k8REgQIAAAQIECKy+wNDPIB0WsrpBQz3esgxfFSen\nJ4cmFyXPSc5MVtp2zwv/ZzJpweNzUCuV9joCBAgQIECAAAECqygw9AJpr1jVJXXLFUddzmvT\nOSepAmdTWl3C9zfJVhPuZM9sV19oqxEgQIAAAQIECBAgsBkFhl4gXRHbfZIqVG6dwHnHbFNF\nVd2wYVPad/Lit0yxg7rE7nen2N6mBAgQIECAAAECBAjMQGDot/k+MWZ1ZuZ9yeIviO1y1meQ\nDkxOSbZLPpBoBAgQIECAAAECBAg0JjD0M0gnZT53TY5OnpJcllyaXJ3ckGyf7JTUJW67Jbcl\nr0jOSDQCBAgQIECAAAECBBoTGHqBVDdfODb5YHJMUneqW3wm6eYsuzx5Y/Lm5JJEI0CAAAEC\nBAgQIECgQYGhF0ijKa072R2x0KmzRvX9R9sm9cWx1ycaAQIECBAgQIAAAQIENrRSII2muj5z\nVZfWVca1u2VhFVB1k4XvjtvAMgIECBAgQIAAAQIEhisw9Js01MzdJ3lvck1ShdEnkscn49oj\ns7C2e+W4lZYRIECAAAECBAgQIDBsgaEXSPfM9H0ueWZSZ4fqBg0HJ6cn9ZkkjQABAgQIECBA\ngAABAj8QGHqB9HsZ6f2To5L7JXXL78ck/5q8KnlTohEgQIAAAQIECBAgQOBOgaEXSAdklHUj\nhqOTb9854g0bvpDHupvdJ5OXJVVEaQQIECBAgAABAgQIENgw9AJp98xxFUL1/UbdVneu+6Xk\nnOT1SV2CpxEgQIAAAQIECBAg0LjA0AukizK/P5fULb0Xt7phwy8m9bmkE5OlbtyQVRoBAgQI\nECBAgAABAi0IDL1A+ngmsb7z6HXJfcdM6GVZ9sSkLr/7aPLkRCNAgAABAgQIECBAoFGBoRdI\nb8u8fjWpzxpdkjwrWdzOy4JDk+8n9Vmlalvc9eC/BAgQIECAAAECBAi0JDD0Aqm+7HW/5C3J\nxcn3knHtS1m4b3LKuJWWESBAgAABAgQIECDQhsCWDQzzxozxpQtZriA8P9s8KanbgFdhpREg\nQIAAAQIECBAg0JhACwVSd0rrMrqNtfpiWY0AAQIECBAgQIAAgQYFljuj0iCHIRMgQIAAAQIE\nCBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAAB\nAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIE\nCBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2\nAgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx\n6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQ\nWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6\nAgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg\n0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQI\nECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAg\nQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQI\nECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAg\nQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59\nYydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6Agqk\nHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKA\nAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBA\noCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAA\nAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBA\ngAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAA\nAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydA\ngAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocO\nAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl\n2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCeg\nQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQIt\nCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAAB\nAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCegQOpx6BAgQIAAAQIE\nCBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIECBAgQIAAAQItCyiQWp59YydAgAAB\nAgQIECBAoCegQOpx6BAgQIAAAQIECBAg0LKAAqnl2Td2AgQIECBAgAABAgR6AgqkHocOAQIE\nCBAgQIAAAQItCyiQWp59YydAgAABAgQIECBAoCewZa/XRmfHDHOHZJvkxuS65KZEI0CAAAEC\nBAgQIECgcYFWziDtnXk+PrkyuSb5RnJucmlSRdL5yXHJLolGgAABAgQIECBAgECjAi2cQToy\nc3vUwvxenMezkiqSqjCqM0k7JXskL0oOT16SnJRoBAgQIECAAAECBAgQGJTAMzKaO5KTk0cv\nM7Itsu6g5HNJbX9AspZt/7xZve/WM37TE7L/ytDbyzLAs4c+SOMjQIAAAQIECBBYfYGhX2J3\nWMguSOpxuV+Yqzg5PTk0+XbynEQjQIAAAQIECBAgQKAxgaEXSHtlPuuSulsmnNdrs905ye4T\nbm8zAgQIECBAgAABAgQGJDD0AumKzNU+yVYTzlnd4a6KqrqBg0aAAAECBAgQIECAQGMCQy+Q\nTsx8PjR5X7LfMnNbn0E6MDkl2S75QKIRIECAAAECBAgQINCYwNDvYld3o9s1OTp5SnJZUrf2\nvjq5Idk+qbvY7ZnsltyWvCI5I9EIECBAgAABAgQIEGhMYOgFUt184djkg8kxSd2pbvGZpJuz\n7PLkjcmbk0sSjQABAgQIECBAgACBBgWGXiCNprTuZHfEQqfOGtX3H22b1BfHXp9oBAgQIECA\nAAECBAgQ2NBKgdSd6rq0rlKtLq2rM0pVKJ2XfCfRCBAgQIAAAQIECBBoVGDoN2n4zcxrfQ7p\n7ovm95Hp15fCXpj8Y/LFpO5498rkbolGgAABAgQIECBAgECDAkMvkOrsUF1at3Vnbu+f559M\n9k0+nxyXvCe5MfnT5M8SjQABAgQIECBAgACBBgVavMSuiqD6DNLvJm/rzHnd3vsvk5clH01O\nTTQCBAgQIECAAAECBBoSaLFAOiDz+9mkWxzVlNfd7F6Q/HxySLIpBdK98/rXJt0zV+ku2e6z\n5BorCBAgQIAAAQIECBBYM4EWC6S6i93HlxCumzScmzxiifWTLi7XnZNJC6Q6o6URIECAAAEC\nBAgQILCZBVoskL4Q87pJw7hWRc1jkhPGrZxi2VXZ9tlTbL9/tv3ZKba3KQECBAgQIECAAAEC\nMxAY+k0aRmR1Sd27k5cnZyZ1g4anJt22Rzp12V2d9Tmtu8JzAgQIECBAgAABAgQIDEHglzOI\nv0/qi2LvWJSL0x+1J+fJrUltc0ayRbKWrc4g1XtPekneSo/thLywMvRWN9o4e+iDND4CBAgQ\nIECAAIHVFxj6JXZ/F7JKtfqcz6M66RZB9d1H9fmjut13/XJdxYpGgAABAgQIECBAgEBjAkMv\nkLrTeX06dencuMvnPpbl9fmjOoukESBAgAABAgQIECDQqEBLBdJyU1xnjzQCBAgQIECAAAEC\nBBoXaOUmDY1Ps+ETIECAAAECBAgQIDCJgAJpEiXbECBAgAABAgQIECDQhMC0l9i9NSqnJh9N\n5uHzOi/McdYXw07b6lbgZ037ItsTIECAAAECBAgQIDDfAtMWSL+Y4b44+VZyUnJC8qVkvbb/\nnAOrO9dN216bFyiQplWzPQECBAgQIECAAIE5F5i2QKrv6zkieU7y0oX8Sx5PSKpgujJZT+1J\nOZj6HqQ67g8m70gmaedNspFtCBAgQIAAAQIECBAgMBJ4eJ68Prk0qe8Nqkvuqgh5erJVsl7a\nNjmQTye3JHuvl4NadBxVwPmi2EUom9B9WV7ri2I3AdBLCRAgQIAAAQKtCmzKTRq+ErRXJnsk\nT0jq80mPS+qMzeXJm5KHJJu7VWH0/IWDqGPUCBAgQIAAAQIECBAgMFZgUwqk0Q4fnCcHJQcn\nuyZ1JqQutfsvybnJkcnmblXMvSrZPnnk5j4Y70+AAAECBAgQIECAwLAEdslwfjf5TFIFUeXC\n5KjkQUm1ByYfSGrd8xJtaQGX2C1ts5I1LrFbiZrXECBAgAABAgQITC1weF7xkaQ+b1SFz83J\nu5KfTbZIFrf7ZEFt99eLV+j3BBRIPY5N7iiQNpnQDggQIECAAAECbQpMexe7PwtTnRmqmx68\nM3lvcn2yVLstKy5KfGB+KSHLCRAgQIAAAQIECBBYNwLTFkhvy5HXl8TWZ4smaVdnowdMsqFt\nCBAgQIAAAQIECBAgsLkFpr1JQ92ZroqjuqTuwZ2Dv2+e/9XC8s5iTwkQIECAAAECBAgQIDA/\nAtMWSFUIfSg5NdmvM8y6McOzF5b/UWe5pwQIECBAgAABAgQIEJgbgWkLpDdmZE9K6lK7f+qM\n8lN5fmjyyeTVyQGJRoAAAQIECBAgQIAAgbkSmKZAqrvUPS15f1K3+L4q6baPpfMrye3Js7or\nPCdAgAABAgQIECBAgMA8CExTIN0rA7p78vFlBnZF1n0+2WOZbawiQIAAAQIECBAgQIDAuhSY\npkC6ISP49+RRy4xkq6x7UHL+MttYRYAAAQIECBAgQIAAgXUpME2BVAP4RPLC5IjqLGr3TP+4\nZJekbuKgESBAgAABAgQIECBAYK4Epv0epCMzun2Sk5LXJF9Nrkvq7naPTXZM/jo5OdEIECBA\ngAABAgQIECAwVwLTFkhXZnQ/k7wleUJyWFI3b6h2afKq5C+roxEgQIAAAQIECBAgQGDeBKYt\nkGp8Nya/sTDQHfJYN2S4KKnPKGkECBAgQIAAAQIECBCYW4GVFEjdwV6fzpe7CzwnQIAAAQIE\nCBAgQIDAvAqspEA6JIN9drJrUrf9Hl1il6c/aCfk2Yk/6HlCgAABAgQIECBAgACBORCYtkB6\nZsb03gnGddoE29iEAAECBAgQIECAAAEC60pg2gLpj3P0NyUvSuqW33XThnHt++MWWkaAAAEC\nBAgQIECAAIH1LDBNgXSPDOQhSX3XUd3mWyNAgAABAgQIECBAgMCgBKb5otjvZOR1p7o6g6QR\nIECAAAECBAgQIEBgcALTFEh12Vx9tuiIZJrXDQ7NgAgQIECAAAECBAgQGKbAtIXOC8Nwc/J3\nyUFJfQfSzmNSd7fTCBAgQIAAAQIECBAgMFcC0xZIH8ro6vbeT0/qbNJFyVVj8sos0wgQIECA\nAAECBAgQIDBXAtPcpKEG9sXk8glG+G8TbGMTAgQIECBAgAABAgQIrCuBaQuk315XR+9gCBAg\nQIAAAQIECBAgsIoC015i133r+pzRI5P9FhbWbcA1AgQIECBAgAABAgQIzK3ASgqkujHD3yZ1\nu+9zkjck1d6VHJ1sUx2NAAECBAgQIECAAAEC8yYw7SV2u2WAZyd157r6nNF2yahtkSd/kByW\n7Jt8N9EIECBAgAABAgQIECAwNwLTnkF6S0ZWl9YdmDwsqWJp1A7Pk2OShyfPHS30SIAAAQIE\nCBAgQIAAgXkRmLZA+tkM7O3Jp8YM8PYsOyq5PnncmPUWESBAgAABAgQIECBAYF0LTFMgbZ+R\n7Jict8yIbs26ryxst8xmVhEgQIAAAQIECBAgQGD9CUxTIN2Qw/9m8phlhlFFVF1id+4y21hF\ngAABAgQIECBAgACBdSkwTYFUAzg5eUHy4uSeSbfdO52/SnZIPtZd4TkBAgQIECBAgAABAgTm\nQWDaAunlGdTlyVuTy5IDkgclH0jOT56WnJB8PNEIECBAgAABAgQIECAwVwLTFkjXZXSPTo5L\ntk3uk9w3qcKo2kuSOsOkESBAgAABAgQIECBAYO4Epv0epBrgVclvJb+T7Jn8WHJhUmeWNAIE\nCBAgQIAAAQIECMytwEoKpNFg67beFyxktMwjAQIECBAgQIAAAQIE5lZg2kvs5nagDpwAAQIE\nCBAgQIAAAQIbE5j2DFJ99qg+d7Sx9jfZoKIRIECAAAECBAgQIEBgbgSmLZCemJE9cCOjuzTr\nT9vINlYTIECAAAECBAgQIEBg3QlMWyDtnREsviyv+vdLHpEcm9SZo3rUCBAgQIAAAQIECBAg\nMFcC0xZI1y8xuquz/F+SryRfTD6ZfCjRCBAgQIAAAQIECBAgMDcCi88GbeqBfyk7uCipS/E0\nAgQIECBAgAABAgQIzJXAahdI22T0Oye7zpWCgyVAgAABAgQIECBAgEAEpr3Ebtu8ZosxcrWf\nXZKjk3smn080AgQIECBAgAABAgQIzJXAtAXSVzO6jd3Frr489i/mSsHBEiBAgAABAgQIECBA\nIALTFkin5zX/Pkbu+1l2Q3JOcnyy1M0cskojQIAAAQIECBAgQIDA+hSYtkB63vochqMiQIAA\nAQIECBAgQIDApgus9k0aNv2I7IEAAQIECBAgQIAAAQKbSWDaM0jH5Tjvs4Jj/eu85n0reJ2X\nECBAgAABAgQIECBAYM0Epi2Q9sqRPSKpO9VVuz25LtkpGXd3uyy+s31m9MQjgTUQ2C7vce/k\nWWvwXpv7La7MAfzz5j4I70+AAAECBAgQGIrAtAXSszPwTyX1C9lRSd2U4bZk6+SQ5E1J3azh\nqUktH7WbR088ElgDgX3yHg9Ijl2D99qcb1E/d/dK6lEjQIAAAQIECBDYDAKn5T1PTpb67NL9\nsu6m5LcSbXKB/bPpHcmsf9E9Ie9RGXp7fwZYfw6H3uofJeosrkaAAAECBAgQILBKAksVOuN2\nv00WPi55V1K39R7XLs3CLyUHj1tpGQECBAgQIECAAAECBNazwDQFUl0yd2Oy+zID2irrHpxc\nvcw2VhEgQIAAAQIECBAgQGBdCkxTINWlPB9LXpU8dsxo6oPxo7vcnTJmvUUECBAgQIAAAQIE\nCBBY1wLT3qThdRnNTyd1V7rTk39Lvp3UZ48OSXZN/kfy4UQjQIAAAQIECBAgQIDAXAlMWyDV\nXesek7wjOTA5KBm1/8iT30yOHy3wSIAAAQIECBAgQIAAgXkSmLZAqrFdkTwpqcvzfiKpL449\nP7ksqTuxaQQIECBAgAABAgQIEJhLgWk+g7R4gHVXu7opw3eTuntdfQZJI0CAAAECBAgQIECA\nwNwKrKRA2iOj/dukvmemLrl7Q1LtXcnRSRVOGgECBAgQIECAAAECBOZOYNpL7HbLCM9Odk7q\nBg3ds0ZbpP8HyWHJvkmdWdIIECBAgAABAgQIECAwNwLTnkF6S0Z296Ru0PCwpIqlUTs8T45J\nHp48d7TQIwECBAgQIECAAAECBOZFYNoC6WczsLcnnxozwNuz7Kjk+uRxY9ZbRIAAAQIECBAg\nQIAAgXUtME2BtH1GsmNy3jIjujXrvrKw3TKbWUWAAAECBAgQIECAAIH1JzBNgXRDDv+bSX0P\n0lKtiqi6xO7cpTawnAABAgQIECBAgAABAutVYJoCqcZwcvKC5MXJPZNuu3c6f5XskHysu8Jz\nAgQIECBAgAABAgQIzIPAtAXSyzOoy5O3JvXFsAckD0o+kNSXxT4tOSH5eKIRIECAAAECBAgQ\nIEBgrgSmLZCuy+genRyXbJvcJ7lvUoVRtZckdYZJI0CAAAECBAgQIECAwNwJTPs9SDXAq5Lf\nSn4n2TP5seTCpM4saQQIECBAgAABAgQIEJhbgWkLpLrF93eS/57cllywkDxoBAgQIECAAAEC\nBAgQmG+BaS6x2yZDrS+A/aWkiiONAAECBAgQIECAAAECgxKYpkD6Xkb+7WS7ZItBKRgMAQIE\nCBAgQIAAAQIEIjBNgXRHtn/6gtqH8vjzyYOT+u6jxamzTRoBAgQIECBAgAABAgTmSmCaAqkG\n9oakziDVZXanJF9Prh+T388yjQABAgQIECBAgAABAnMlMO1NGs7N6K6dYITnTbCNTQgQIECA\nAAECBAgQILCuBKYtkF6wro7ewRAgQIAAAQIECBAgQGAVBTZ2id1Bea9DVvH97IoAAQIECBAg\nQIAAAQLrVmBjZ5DekiPfIXngohE8Mv2dk/+1aLkuAQIECBAgQIAAAQIE5lZgY2eQlhrY0Vnx\niaVWWk6AAAECBAgQIECAAIF5FFhpgTSPY3XMBAgQIECAAAECBAgQWFZAgbQsj5UECBAgQIAA\nAQIECLQkoEBqabaNlQABAgQIECBAgACBZQU2dpOGZV88pyt3zHHXjSe2SW5MrktuSjQCBAgQ\nIECAAAECBBoXaOUM0t6Z5+OTK5Nrkm8k9aW3lyZVJJ2fHJfskmgECBAgQIAAAQIECDQqMMkZ\npDrj8vpFPg9b6C9ePtrsY3ly6qizmR+PzPsftXAMF+fxrKSKpCqM6kzSTskeyYuSw5OXJCcl\nGgECBAgQIECAAAECBHoCX0rvjhXkNb29bL7OMxaO/eQ8PnqZw9gi6+pLcT+X1HgPSNay7Z83\nq/fdesZvekL2Xxl6e38G2MJlk4dknLcPfTKNjwABAgQIECCwlgIbO4P0BzmYe6/ggM5ZwWtm\n8ZLDstMLknq8ZZk3qOLk9OTQ5KLkOcmZiUaAAAECBAgQIECAQEMCGyuQPjLnFnvl+OuSuuWK\no+4Qr02nirvduws9J0CAAAECBAgQIECgDYGh36ThikzjPslWE05nfd6qiqq6gYNGgAABAgQI\nECBAgEBjAkMvkE7MfD40eV+y3zJzW59BOjA5Jdku+UCiESBAgAABAgQIECDQmMDGLrGbd466\nG92uydHJU5LLkkuTq5Mbku2TnZI9k92S25JXJGckGgECBAgQIECAAAECjQkMvUCqmy8cm3ww\nOSapO9UtPpN0c5ZdnrwxeXNySaIRIECAAAECBAgQINCgwNALpNGU1p3sjljo1Fmj+v6jbZP6\n4tjrE40AAQIECBAgQIAAAQIbhv4ZpHFTfLcsrNTY75ncI9EIECBAgAABAgQIECDQTIG0d+b6\n+KTOGF2TfCOpO9XV55FuTM5Pjkt2STQCBAgQIECAAAECBBoVaOESuyMzt0ctzO/FeazvRaoi\nqQqjutSubtKwR/Ki5PDkJUnd3EEjQIAAAQIECBAgQIDAoASekdHUjRpOTh69zMjqNt91A4fP\nJbX9Aclatv3zZvW+W8/4TU/I/itDb+/PAG8a+iAzvkOS2xsYpyESIECAAAECBNZMYOhnkA6L\nZN2goR5vWUa1ipPTk0OTi5LnJGcmK21b5YXPTOpxkvbgSTayDQECBAgQIECAAAECsxUYeoG0\nV/jqkrrliqOu8LXpnJPs3l24guf3zWuOTCY9I7TNCt7DSwgQIECAAAECBAgQWGWBoRdIV8Rr\nn6TO5Nw6gd2O2aaKqrphw6a0Ogv1k1PsoC6x25QzVlO8lU0JECBAgAABAgQIEFhKoG51PeR2\nYgb30OR9yeIviO2Ouz6DdGBySrJd8oFEI0CAAAECBAgQIECgMYGhn0E6KfO5a3J08pTksuTS\n5OrkhmT7pO5it2eyW3Jb8orkjEQjQIAAAQIECBAgQKAxgaEXSHXzhWOTDybHJHWnusVnkm7O\nssuTNyZvTi5JNAIECBAgQIAAAQIEGhQYeoE0mtK6k90RC506a1Tff7RtUl8ce32iESBAgAAB\nAgQIECBAYEMrBVJ3quvSuopGgAABAgQIECBAgACBnsDQb9LQG6wOAQIECBAgQIAAAQIElhNQ\nIC2nYx0BAgQIECBAgAABAk0JDP0SuxdmNuszR9O2+k6i+oJZjQABAgQIECBAgACBhgSGXiD9\n58zlo1Ywn6/NaxRIK4DzEgIECBAgQIAAAQLzLDD0AulJmZy/T/ZP6lbf70gmaedNspFtCBAg\nQIAAAQIECBAYlsDQC6RvZrp+JjktqWLpqOSLiUaAAAECBAgQIECAAIEfEmjhJg23ZNTPXxj5\nW39IwAICBAgQIECAAAECBAgsCLRQINVQv5K8KqkbNjwy0QgQIECAAAECBAgQIPBDAq0USDXw\nNyZ7JV+ujkaAAAECBAgQIECAAIHFAi0VSIvHrk+AAAECBAgQIECAAIGegAKpx6FDgAABAgQI\nECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAEC\nBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQI\nECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff\n2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKp\nx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCyg\nQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ\n6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBA\ngEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQ\nIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBA\ngAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQ\nIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FD\ngAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp5\n9o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIEBc1cK+AAAoWUlE\nQVSAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FD\ngAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp5\n9o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6Ako\nkHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDL\nAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECA\nAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAAB\nAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECA\nAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAAB\nAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2d\nAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHoc\nOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqk\nlmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGe\ngAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQI\ntCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAEC\nBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAECBAgQIECAAIGegAKpx6FDgAABAgQI\nECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQIECBAgAABAgQItCygQGp59o2dAAEC\nBAgQIECAAIGegAKpx6FDgAABAgQIECBAgEDLAgqklmff2AkQIECAAAECBAgQ6AkokHocOgQI\nECBAgAABAgQItCywZYOD3zFj3iHZJrkxuS65KdEIECBAgAABAgQIEGhcoJUzSHtnno9Prkyu\nSb6RnJtcmlSRdH5yXLJLohEgQIAAAQIECBAg0KhAC2eQjszcHrUwvxfn8aykiqQqjOpM0k7J\nHsmLksOTlyQnJRoBAgQIECBAgAABAgQGJfCMjOaO5OTk0cuMbIusOyj5XFLbH5CsZds/b1bv\nu/WM3/SE7L8y9Pb+DLCFyyYPyThvH/pkGh8BAgQIECBAYC0Fhn6J3WHBvCCpx7OXga3i5PTk\n0OTbyXMSjQABAgQIECBAgACBxgSGXiDtlfmsS+pumXBer8125yS7T7i9zQgQIECAAAECBAgQ\nGJDA0AukKzJX+yRbTThndYe7KqrqBg4aAQIECBAgQIAAAQKNCQy9QDox8/nQ5H3JfsvMbX0G\n6cDklGS75AOJRoAAAQIECBAgQIBAYwJDv4td3Y1u1+To5CnJZUnd2vvq5IZk+6TuYrdnslty\nW/KK5IxEI0CAAAECBAgQIECgMYGhF0h184Vjkw8mxyR1p7rFZ5JuzrLLkzcmb04uSVajPSQ7\nmfSudA9cjTe0DwIECBAgQIAAAQIENk1g6AXSSKfuZHfEQqfOGtX3H22b1BfHXp+sdvvx7PC8\npC7d0wgQIECAAAECBAgQmBOBVgqk7nTUpXWVanVpXZ1RqkKpCprvJKvRvp6d1KV7k/rum21P\nXo03tg8CBAgQIECAAAECBFYuMOkv8Ct/h837yt/M2x+cPD/pFj+PTP8dSRUmo1Znkv4keUOy\nGl++ed1oxxM8zuIs1gRvaxMCBAgQIECAAAECBLoCQ7+LXZ0dqkvrup8Fun/6n0yqOPp8clzy\nnuTG5E+TP0s0AgQIECBAgAABAgQaFBj6GaRxU1pFUH0G6XeTt3U2qNt7/2XysuSjyamJRoAA\nAQIECBAgQIBAQwJDP4M0bioPyMLPJt3iqLaru9m9IKlbgB+SaAQIECBAgAABAgQINCbQYoFU\nd7H78hLzXJ9TOjd5xBLrLSZAgAABAgQIECBAYMACLRZIX8h81k0axrWds/AxyRXjVlpGgAAB\nAgQIECBAgMCwBVopkOqSuncnL0/OTOoGDU9Num2PdOqyu7qhw2ndFZ4TIECAAAECBAgQIEBg\nCAK/nEH8fVJfFHvHolyc/qg9OU9uTWqbM5K1/oLX/Rfeu3u3vSxa9XZC9lgZent/BnjT0AeZ\n8R2SrMYt6RugMkQCBAgQIECAwGQCQ7+L3d+FoVKt7lz3qE66RdDdsrw+f1S3+6672FWhpBEg\nQIAAAQIECBAg0JjA0Auk7nTWl7HWpXPjLp/7WJbX54/qLJJGgAABAgQIECBAgECjAi0VSMtN\ncZ090ggQIECAAAECBAgQaFyglZs0ND7Nhk+AAAECBAgQIECAwCQCCqRJlGxDgAABAgQIECBA\ngEATAgqkJqbZIAkQIECAAAECBAgQmERAgTSJkm0IECBAgAABAgQIEGhCQIHUxDQbJAECBAgQ\nIECAAAECkwgokCZRsg0BAgQIECBAgAABAk0IKJCamGaDJECAAAECBAgQIEBgEgEF0iRKtiFA\ngAABAgQIECBAoAkBBVIT02yQBAgQIECAAAECBAhMIqBAmkTJNgQIECBAgAABAgQINCGgQGpi\nmg2SAAECBAgQIECAAIFJBBRIkyjZhgABAgQIECBAgACBJgQUSE1Ms0ESIECAAAECBAgQIDCJ\ngAJpEiXbECBAgAABAgQIECDQhIACqYlpNkgCBAgQIECAAAECBCYRUCBNomQbAgQIECBAgAAB\nAgSaENiyiVEaJIFhCtx9YVi/P8zh9Ub13fTeltzaW6pDgAABAgQIEFhlAQXSKoPaHYE1FPjJ\nvFedBX7KGr7n5nirrfKm+yb/lHxlcxyA9yRAgAABAgTaEVAgtTPXRjpMgTsyrAOGObQfjOpH\n8+xbP+h5QoAAAQIECBCYoYDPIM0Q164JECBAgAABAgQIEJgvAQXSfM2XoyVAgAABAgQIECBA\nYIYCCqQZ4to1AQIECBAgQIAAAQLzJaBAmq/5crQECBAgQIAAAQIECMxQQIE0Q1y7JkCAAAEC\nBAgQIEBgvgQUSPM1X46WAAECBAgQIECAAIEZCiiQZohr1wQIECBAgAABAgQIzJeAAmm+5svR\nEiBAgAABAgQIECAwQwEF0gxx7ZoAAQIECBAgQIAAgfkSUCDN13w5WgIECBAgQIAAAQIEZiig\nQJohrl0TIECAAAECBAgQIDBfAgqk+ZovR0uAAAECBAgQIECAwAwFFEgzxLVrAgQIECBAgAAB\nAgTmS0CBNF/z5WgJECBAgAABAgQIEJihgAJphrh2TYAAAQIECBAgQIDAfAkokOZrvhwtAQIE\nCBAgQIAAAQIzFFAgzRDXrgkQIECAAAECBAgQmC8BBdJ8zZejJUCAAAECBAgQIEBghgIKpBni\n2jUBAgQIECBAgAABAvMloECar/lytAQIECBAgAABAgQIzFBAgTRDXLsmQIAAAQIECBAgQGC+\nBBRI8zVfjpYAAQIECBAgQIAAgRkKKJBmiGvXBAgQIECAAAECBAjMl4ACab7my9ESIECAAAEC\nBAgQIDBDAQXSDHHtmgABAgQIECBAgACB+RJQIM3XfDlaAgQIECBAgAABAgRmKKBAmiGuXRMg\nQIAAAQIECBAgMF8CCqT5mi9HS4AAAQIECBAgQIDADAUUSDPEtWsCBAgQIECAAAECBOZLQIE0\nX/PlaAkQIECAAAECBAgQmKGAAmmGuHZNgAABAgQIECBAgMB8CSiQ5mu+HC0BAgQIECBAgAAB\nAjMUUCDNENeuCRAgQIAAAQIECBCYLwEF0nzNl6MlQIAAAQIECBAgQGCGAgqkGeLaNQECBAgQ\nIECAAAEC8yWgQJqv+XK0BAgQIECAAAECBAjMUECBNENcuyZAgAABAgQIECBAYL4EFEjzNV+O\nlgABAgQIECBAgACBGQookGaIa9cECBAgQIAAAQIECMyXgAJpvubL0RIgQIAAAQIECBAgMEMB\nBdIMce2aAAECBAgQIECAAIH5ElAgzdd8OVoCBAgQIECAAAECBGYooECaIa5dEyBAgAABAgQI\nECAwXwIKpPmaL0dLgAABAgQIECBAgMAMBRRIM8S1awIECBAgQIAAAQIE5ktAgTRf8+VoCRAg\nQIAAAQIECBCYoYACaYa4dk2AAAECBAgQIECAwHwJKJDma74cLQECBAgQIECAAAECMxRQIM0Q\n164JECBAgAABAgQIEJgvAQXSfM2XoyVAgAABAgQIECBAYIYCCqQZ4to1AQIECBAgQIAAAQLz\nJaBAmq/5crQECBAgQIAAAQIECMxQQIE0Q1y7JkCAAAECBAgQIEBgvgQUSPM1X46WAAECBAgQ\nIECAAIEZCiiQZohr1wQIECBAgAABAgQIzJeAAmm+5svREiBAgAABAgQIECAwQ4EtZ7hvuyZA\ngMBqCuyYne2ymjtch/v6fo7p6nV4XA6JAAECBAg0I6BAamaqDZTA3ArcY+HIPzm3I5juwJ+S\nzT883UtsTYAAAQIECKyWgAJptSTthwCBWQlss7DjF+fxE7N6k3Wy3/+V49h+nRyLwyBAgAAB\nAk0KKJCanHaDJjCXApfkqL86l0c++UHfOvmmtiRAgAABAgRmIeAmDbNQtU8CBAgQIECAAAEC\nBOZSQIE0l9PmoAkQIECAAAECBAgQmIWAAmkWqvZJgAABAgQIECBAgMBcCiiQ5nLaHDQBAgQI\nECBAgAABArMQUCDNQtU+CRAgQIAAAQIECBCYSwEF0lxOm4MmQIAAAQIECBAgQGAWAgqkWaja\nJwECBAgQIECAAAECcymgQJrLaXPQBAgQIECAAAECBAjMQkCBNAtV+yRAgAABAgQIECBAYC4F\nFEhzOW0OmgABAgQIECBAgACBWQgokGahap8ECBAgQIAAAQIECMylgAJpLqfNQRMgQIAAAQIE\nCBAgMAsBBdIsVO2TAAECBAgQIECAAIG5FFAgzeW0OWgCBAgQIECAAAECBGYhoECahap9EiBA\ngAABAgQIECAwlwIKpLmcNgdNgAABAgQIECBAgMAsBBRIs1C1TwIECBAgQIAAAQIE5lJgy7k8\nagdNgAABAgTWv8DeOcRnrP/DXJUj/Fr28s5V2ZOdECBAYDMLtFgg7RjzHZJtkhuT65KbEo0A\nAQIECKymwK9kZ89Pzl7Nna7Dfe2WY9o5USCtw8lxSAQITC/QSoFU/4r3O8lTk13GMF2QZacm\nf5h8a8x6iwgQIECAwEoEPp8XPWklL5yj1/xqjvXP5uh4HSqBkcCeeVJpoX01g7yqhYGuxhhb\nKJCODNRRC1gX5/Gs5Jqkzh7VmaSdkj2SFyWHJy9JTko0AgQIECBAgACB4Qr8bYb22OEOrzey\n49N7YW+JzpICQy+Q6trvKo5OSf4gWeoyhy2y7sDkjcm7kwuTMxONAAECBAgQIEBgmAJbZViv\nSI4d5vB+MKq6/LXGqk0oUIXBkFsVO49LHpbcMsFA6/NJFyV1Bum3Jth+qU0elBVfSLZeaoNF\ny+tugvWZqNr+tkXrVrNb/3rw3OR7q7nTdbivsizT76zDY1vNQ6p/4Kg/Mzev5k7X4b7q76m7\nJ/UzfPs6PL7VPKQaZ/18Dn2c9T/q7zcyzrtlnN9NhtxqPlsYZ83htsnQ57PGWf8frb+L7qjO\ngFuNs8bYwu9FJ2ScL0i0CQSGfgZprxiclUxSHBXXtck5ye7V2YR2YV77zGTSar1+AfzRZJbF\nUXa/oS43/Pt6MvBWv2TWpZOXDXycVQQ+ILlg4OOs4f148vUGxrlnxnh5cuvAx/pjGV8V9jcM\nfJzbZ3zbJd8c+Djr/3X3TeofGIfeHpIB1h37ht4enAF+I6l/yBhyq9/3rkmG/g+qNYdfqv9o\nBErgn5J/SyYtVOoMUv0P24dNg6ARIECAAAECBAgQIDAsgV/LcOrU6YeS/ZYZWp3Bqc8gfSap\nsziPTzQCBAgQIECAAAECBAgMSqAKn5cl9T1HVShdmnw6+UjynoXHugSvLmmp9bcmL000AgQI\nECBAgAABAgQIDFagbppQBVF9JqUKoW6qePpa8obk/olGgAABAgQIECBAgECjAnWGpbVWH5qt\n7z+qO9FcmVyfaAQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgMDaCWyxdm/lndaJwNbr5DgcBgECBAgQIEBgcwrc\nkTe/dXMegPdenwIKpPU5L7M6qk9nx/vNauf2S4AAAQIECBCYM4G353hfPGfH7HBnLLDljPdv\n9+tL4IIcztXJ0evrsBzNCgX2yevenPz0Cl/vZetP4IM5pLclH1t/h+aIViDw23nNQ5KXr+C1\nXrL+BB6YQ3p38qTk+vV3eI5oBQLvzGsuWsHrvGTgAgqkgU/wouF9L/1vJWctWq47nwJ3Xzhs\n8zmf8zfuqOtSj68n5nSczvwte1oOeRfzOX8Tt8QR37Cw/PN5vGqJbSyeL4Gbc7i3zdchO9q1\nEPiRtXgT70GAAAECBAgQIECAAIF5EFAgzcMsOUYCBAgQIECAAAECBNZEQIG0JszehAABAgQI\nECBAgACBeRBQIM3DLDlGAgQIECBAgAABAgTWRECBtCbM3oQAAQIECBAgQIAAgXkQUCDNwyw5\nRgIECBAgQIAAAQIE1kRAgbQmzN6EAAECBAgQIECAAIF5EFAgzcMsOUYCBAgQIECAAAECBNZE\nQIG0JszehAABAgQIECBAgACBeRDYch4O0jGumsD3Vm1PdrQeBG7NQZjT9TATq3cM5nT1LNfD\nnsznepiF1TuGms87knrUhiHgZ3QY82gUBDZJYKe8uqINQ2CLDONBwxiKUSwI7JlH/3A1nD8O\n98pQ7jOc4RhJBH6cwqAEds9oth3UiAyGAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC8yWwxXwd7uCP9n4Z4d7JTclnFh7zsOZtmuOY\nZts1H8g6esPDcixfS76yGY7pbnnP/ZLdknOSOo5xbfss3HHciiyrP5NXLbGupcUPymAfmmyV\nnJucl2yONunP3R45uKX+nr8s627bHAe/jt7zXjmWRyf1577+zr0i2Rxt0vn0Mzr57Nw3m9bf\ne6cl10z+slXZ0t+5q8K4YT39/TXpz+h6OubVmQV7IbCZBY7K+9+a3LGQ+sXlvyVr3aY5jmm2\nXetxrKf3e2EOpub1FZvhoB6S9/y3hfcf/dmqIu3+Y47lzxdtN9q+Hk8as31Li34sg/1A0jWp\n5/+cVNG0lm3Sn7tdc1CLj7fb/4m1POh1+F5H5Ji+tcjozPTLbS3bpPNZx+RndLKZqQKl5rL+\nvO8/2UtWbSt/564O5Xr6+2vSn9H1dMyrMwsN72XLhse+nob+xBzMkcn7kz9Otkr+KHl98p3k\nrclatGmOY5pt1+LY1+t7PC0H9vbNdHBb5H3/R7J78uzk08nPJG9OPpU8LKkzQ6P2qDy5MTl+\ntKDz+IXO89ae/kgG/DfJwcnfJickNyfPSX49+VCyb/LdZNZtmp+7ms9qpyb/euez/n+u7Xeb\n6h2U0f518o3kVclZyeHJf0/OSB6R3JLMuk0zn3UsfkYnm5E/yGZrXRjVkfk7d7L5mWSr9fL3\n1zQ/o+vlmCfxtQ2BdS+wXY6w/id9aVL/6jVqW+dJLb8k6S4frV/JY/2P/5glXjjNcUyz7RJv\nN/jFO2eE70ruSOoX53pc7TNI9ct5/WL3wGRc++0srPf9zUUrXzhmeRUBVRx9YtG2uncVRuVY\n/yK9uH0kC2rdMxavWEF/Y/M57c/dKxeO7eAVHMvQX/LhBZsnLxroOxeW1y9Fq9FW6+/cOhY/\no5PNyGOzWV2NcWVSP5urWSht7GfU37kBX6W2Fn9/bWw+/Z27SpM5j7upv3C1zStQv7w8IKlf\npm9PRu17eXJSUte9/sJoYefxIXlev/i+KXlJsleysfa4bPATS2w0zXFMs+0Sbzf4xR/NCH8t\n+Z/JiyYY7Urms/5s1JzWX+Lj2vOysP4V/L2LVla/irYXdJbX+98j+Xxnmad3CTwgDxcm70gW\ntzoLUe1hdz384L9b5lmdPXxN8rrkmcndk+XaxuZz2p+7+tfMO5Kzl3vTRtd9MOP+f5L6Oe22\nf17o/F/dhQvPV/Izulp/59Yh+BkdMymLFtXfYe9O6mz5iQvr6mdgXFvJfG7sZ/R5eSN/547T\nnn7ZtH9/+Tt3emOvWEag/kBpm1eg/rWr2mfveuj9d7SsLt+pf6ketToTUb90bZVcluyWbJH8\nafKHyVL/Q8iqJds0xzHNtku+4cBX1CVpdZnHqclTNzLWWcxn/dmo/8Gcl1y36P1vSP/c5KeS\n2q7+tbW2rVbHfUCyT1Lb1S8atY+WW/2iNfpla7HDgxYWnN9ZUcvek9TPSRmW785JfRbsWck5\nyUratD93Naf/nmydHJH8WPLV5PSkLt1tuf3lmMHX36FPX1j+8UXrZ/EzupL5rMPyM7pocjrd\nY/P8PsmhyeIz553N7jybv9r/D/V3bld4059P8/eXv3M33dseCKw7gT/PEVVBc/CYIztwYd1f\ndNY9ZWHZaXm878Lye+XxpIXlz11YNu6hzlDVGY1xbZrjmGbbce/V2rIqkGqO65esxW1T5vPV\n2Vnt9+GLd5r+rgvrPjFmXS2qXwDrtaM/Q3+y0K9fqGv5KPVn5k2Jf0wJwqL2o+l/K7k+qeKj\nWv2S/bmk3P7TQj8PG56YXJ1UsVkFy7i23HzW9tP83G2X7esYvplUkTaaz3qsOX5sot0l8LA8\n/FFSZ9rK7L8m3bYpP6O1v9X4O7eOx89od1Z++Hmdsa0/37+xsKr+wbD6j1vojx42ZT6X+xn1\nd+5IeNMfp/n7y9+5m+5tDwTWpcC7clT1l/i4X3JrWa17dzJq5+ZJLat/4e+2e6Rzc3J5Un9h\nVDsruaiTel19KL+77DXpV5vmOKbZ9q69t/3f5QqkaebzV8LYnbs6M1RzWnPeXb59+j++sG6p\nX85qeb32IUm1U5Lq19mFJyV1KUk9/mtSy+vMpPZ/BOrnrc6ulc3z/8/iO88Q1bJ/6CwbPa1f\nwmvd6F+2p5nP2sc0P3f1S2G91y3J7yV1yVgVAq9Lbkv+I9kp0TZs+IsglFXla8kjk26b5md0\nVn/n/u/27j3UlqqOA3hdih7kIzIwoltRkCDd0q50o0jp8U+B0buQ6I9SAumPkiL6pyCDIiSq\nPyIkIYiK7GFBlEFeKTUq0izzEVFQRpGVlUJJof2+3lkwZ5/Zj7l7747nzmfB787MmjWz1/6s\nWevM65yb+uij/VbZOX96Ld5VcVUve94F0pj2HNNHjbk9/DVnj9T2q45fb+rKGnPXRLf5ToFH\n7Fy0tAcC/+4+88DAZ7c/zpC7kEmnVjyrIj/E89rOoYp++kkt5K8z5anAH7pIuZYO1kwuonIy\n3dLd3cyYeowp2z7HdLfA2Pa8t3bRb7u05ykVuUBKu7Z0f80saqOUmz22Plx5X674QkXb9s6a\nv6ni9opcIH28IhfYU095cvTNiudXfLLisxUtHelmrqnpbP+8tVt3uKafqRjTntm0tcsqY8Vv\nqnxeq/t9xfUVLb2/ZtL27614d4UL32NPj/Jk4FUV76y4seLiilw4je2jGXe3MebWbh/8Azv6\naCR2pysqK+PehbtX7cgZ255j+uii/plKGHN3NMXChTHjlzF3IaWVBPavQLurfO7AVziv8nIX\n5VPdunO65eQtiqF9ZRe50LoyMwNpTD3GlB34qMllnV/fOO11ycw3X7c9c1KX/Z45s98s5uZH\nThiOZmEgXVt52fYJA+tms3LMpGzqO/X0jALIDYp4XDqA8a1uXdbPi6MD2yVrUXtm/ab6XY6X\n1C11lXYKNJtfdNnr9tFNjbk7a7l7acp99OLiyPGcpz15NavFZV3+eV3ew2u6bnsu6qPG3AL+\nP6TWR9v4lem8sbblH51Tr0XtmU2MuXPgppCdDi3trUDu/icNve7S8nJXMqndobq65j/2YM7w\nP7cMZy/MHVOPMWUXfujEV26zPfMa1Z8r2jE0S538PHX6++yKgeW7uryTB9ZNKSv/N853K55Y\ncVHF5RWzqbXpBbUir7ENpX8OZa6Qt6l+pz3nY/+yVv2oIk8HD1a09tzrMbeqsjBNuU1f28l8\naY5QOzk+o9Zvsz2NuXMaYMPZs8d6a1Nj7oahp747F0h7fwTc1lXh3Jp+faY6yUv68bHJw35d\n0wcqTqv4XpfXn+SHeu5Y3tPPXHF+TD3GlF3x4ydZbJvtGdC004sqcrz8paKlnODnd1Ly+xI5\nXk6quLbivoqUz5OnfsqJRdIdxyaT/PdwfeucJD+y4pUVuVAaSr/qMnMRNNtH83pP9vOnrszY\nyZh+967a+TsqPljxxYp+mnp7Pq4wflbxu4qX9GG6+Xb831vLORl7KIy5+uhAQ3VZ+bl5y8Dq\nF1be2RVXVqTP3V2Rn43bas/atTE3CBtIY8YvY+4GwO2CwENV4OdVsT9W9O/Qn1LLGdRvquhf\nyH6nljPAv6Kin86shZzg3lyRk7ih9NXKfM/Qii5vTD3GlF3wkZNYNe8Vu3z5ddrzDbV9Tg6e\nkh0NpNdUXo6V/L5JP72vFpL/ul5mXilKXvbZTznJyAnj7Ml+v8yJPv+Y+oK/rcidyhcs+bJZ\nH68bKtrvHLRNPl8zQ8Zt/bL2TLlV+13uquezcuKYV4taynw75l7cMic4/Wl959wcOGvmu6f9\nkp9xt6Xmtddjrj7aWmS16UeqWPrAkZni67Tnsj5qzJ3BPs7FMeOXMfc4kW1GYD8IvLkqmYE8\nP7Rz0vr6ihsr/ltxdkU/5Y80/KuLD9T05RU5Ac7TiJQ/XHG8aUw9xpQ93vqcKNstukDaZnse\nKMBbK3LC96GKl1Vc2i1/rab99NJaSLk8abqsImVzXN1T8deKQxVTTe099LzqetWceHsP54qa\nT3++ruKNFa+u+FxF8r5RsU5atd/l4uyainzm0Yq3VKQeefKVvMsrppzypPQ/FXkN9aMVOf5z\n8+gfFfdVnFXR0jb76Krtmbroo61FVpvOu0DaZnsac1drm2Wlxo5fxtxlotYT2McCF1Td/1aR\nk5dE5t9WMZTOqMzvV+SEtpW/s+bfWrFuGlOPMWXXrdd+3v78qnza6ZI5X2Kb7Xlafea3K+6v\naMfK1TV/esVsyh3yvK7QyuWC+wcVT6+YcsrThGYyb/qJHlBOknKynd/vauXj/5WKIffKHpVW\n7XePr71+uiLt2OqRC+BFT5Fr9WRSbgLcXtFsMv1hxXMqZtM2++iq7Zk66aOzLTN/ed4FUrbY\nZnsac+e3yZg1Y8YvY+4YWWUJ7EOBvP7yzIq8LveoFer/2Crz3IqnVuSOy6bSmHqMKbup+p2o\n+9lWe8brpIrnVaxygv6kKpcnl6mPtJ7Awdo8J9wnr7ebXVuP6XePrq2fXfG0XXuREYEnV5xT\ncWoWlqRt9dEx7Zkq6qNLGmrF1dtqz3y8MXfFRlhSbOz4ZcxdAmo1AQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAg\nQIAAgbUEDtXWD1TcvNZebEyAAAECkxA4MIlv6UsSIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAEC\nBAgQIECAAAECUxb4H2RU+HrjhJ+lAAAAAElFTkSuQmCC",
      "text/plain": [
       "Plot with title “Histogram of .”"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "df_genes %>% select(A1BG) %>% unlist  %>% unname %>% hist()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Choosing liver cancer clinical outcomes/characteristics for supervised learning"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 178,
   "metadata": {},
   "outputs": [],
   "source": [
    "characterized = list()\n",
    "for(name in names(clinical_dataset_list)){\n",
    "    characterized[[name]] = na.omit(colnames(clinical_dataset_list[[name]])[sapply(clinical_dataset_list[[name]],function(x){as.integer(table(!is.na(x))['TRUE'])==nrow(clinical_dataset_list[[name]])})])\n",
    "    }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<dl>\n",
       "\t<dt>$luad</dt>\n",
       "\t\t<dd><ol class=list-inline>\n",
       "\t<li>'admin.bcr'</li>\n",
       "\t<li>'admin.day_of_dcc_upload'</li>\n",
       "\t<li>'admin.disease_code'</li>\n",
       "\t<li>'admin.file_uuid'</li>\n",
       "\t<li>'admin.month_of_dcc_upload'</li>\n",
       "\t<li>'admin.patient_withdrawal.withdrawn'</li>\n",
       "\t<li>'admin.project_code'</li>\n",
       "\t<li>'admin.year_of_dcc_upload'</li>\n",
       "\t<li>'patient.bcr_patient_barcode'</li>\n",
       "\t<li>'patient.bcr_patient_uuid'</li>\n",
       "\t<li>'patient.clinical_cqcf.anatomic_neoplasm_subdivision'</li>\n",
       "\t<li>'patient.clinical_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.clinical_cqcf.histological_type'</li>\n",
       "\t<li>'patient.clinical_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.clinical_cqcf.tumor_type'</li>\n",
       "\t<li>'patient.diagnosis'</li>\n",
       "\t<li>'patient.gender'</li>\n",
       "\t<li>'patient.histological_type'</li>\n",
       "\t<li>'patient.icd_10'</li>\n",
       "\t<li>'patient.icd_o_3_histology'</li>\n",
       "\t<li>'patient.icd_o_3_site'</li>\n",
       "\t<li>'patient.informed_consent_verified'</li>\n",
       "\t<li>'patient.month_of_form_completion'</li>\n",
       "\t<li>'patient.patient_id'</li>\n",
       "\t<li>'patient.prior_dx'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'</li>\n",
       "\t<li>'patient.tissue_source_site'</li>\n",
       "\t<li>'patient.tumor_tissue_site'</li>\n",
       "\t<li>'patient.vital_status'</li>\n",
       "\t<li>'patient.year_of_form_completion'</li>\n",
       "\t<li>'patient.bcr_canonical_check.bcr_patient_canonical_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.digital_image_submitted'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.histological_type'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.method_of_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.vial_number'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_report_attached'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.sample_prescreened'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.site_of_disease'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_nuclei_percent'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.vessel_used'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.vial_number'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_file_name'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.section_location'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.weight'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.sample_type'</li>\n",
       "\t<li>'patient.samples.sample.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample.vial_number'</li>\n",
       "</ol>\n",
       "</dd>\n",
       "\t<dt>$lusc</dt>\n",
       "\t\t<dd><ol class=list-inline>\n",
       "\t<li>'admin.bcr'</li>\n",
       "\t<li>'admin.day_of_dcc_upload'</li>\n",
       "\t<li>'admin.disease_code'</li>\n",
       "\t<li>'admin.month_of_dcc_upload'</li>\n",
       "\t<li>'admin.patient_withdrawal.withdrawn'</li>\n",
       "\t<li>'admin.year_of_dcc_upload'</li>\n",
       "\t<li>'patient.bcr_patient_barcode'</li>\n",
       "\t<li>'patient.bcr_patient_uuid'</li>\n",
       "\t<li>'patient.clinical_cqcf.country'</li>\n",
       "\t<li>'patient.clinical_cqcf.tumor_type'</li>\n",
       "\t<li>'patient.diagnosis'</li>\n",
       "\t<li>'patient.gender'</li>\n",
       "\t<li>'patient.histological_type'</li>\n",
       "\t<li>'patient.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.icd_10'</li>\n",
       "\t<li>'patient.icd_o_3_histology'</li>\n",
       "\t<li>'patient.icd_o_3_site'</li>\n",
       "\t<li>'patient.informed_consent_verified'</li>\n",
       "\t<li>'patient.month_of_form_completion'</li>\n",
       "\t<li>'patient.patient_id'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_n'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'</li>\n",
       "\t<li>'patient.tissue_source_site'</li>\n",
       "\t<li>'patient.tumor_tissue_site'</li>\n",
       "\t<li>'patient.vital_status'</li>\n",
       "\t<li>'patient.year_of_form_completion'</li>\n",
       "\t<li>'patient.bcr_canonical_check.bcr_patient_canonical_status'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.vial_number'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_file_name'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_uuid'</li>\n",
       "\t<li>'patient.samples.sample.sample_type'</li>\n",
       "\t<li>'patient.samples.sample.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample.vial_number'</li>\n",
       "</ol>\n",
       "</dd>\n",
       "\t<dt>$meso</dt>\n",
       "\t\t<dd><ol class=list-inline>\n",
       "\t<li>'admin.bcr'</li>\n",
       "\t<li>'admin.day_of_dcc_upload'</li>\n",
       "\t<li>'admin.disease_code'</li>\n",
       "\t<li>'admin.file_uuid'</li>\n",
       "\t<li>'admin.month_of_dcc_upload'</li>\n",
       "\t<li>'admin.patient_withdrawal.withdrawn'</li>\n",
       "\t<li>'admin.project_code'</li>\n",
       "\t<li>'admin.year_of_dcc_upload'</li>\n",
       "\t<li>'patient.bcr_patient_barcode'</li>\n",
       "\t<li>'patient.bcr_patient_uuid'</li>\n",
       "\t<li>'patient.clinical_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.clinical_cqcf.country'</li>\n",
       "\t<li>'patient.clinical_cqcf.histological_type'</li>\n",
       "\t<li>'patient.clinical_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.clinical_cqcf.tumor_type'</li>\n",
       "\t<li>'patient.day_of_form_completion'</li>\n",
       "\t<li>'patient.days_to_birth'</li>\n",
       "\t<li>'patient.gender'</li>\n",
       "\t<li>'patient.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.icd_10'</li>\n",
       "\t<li>'patient.icd_o_3_histology'</li>\n",
       "\t<li>'patient.icd_o_3_site'</li>\n",
       "\t<li>'patient.informed_consent_verified'</li>\n",
       "\t<li>'patient.month_of_form_completion'</li>\n",
       "\t<li>'patient.patient_id'</li>\n",
       "\t<li>'patient.primary_pathology.age_at_initial_pathologic_diagnosis'</li>\n",
       "\t<li>'patient.primary_pathology.days_to_initial_pathologic_diagnosis'</li>\n",
       "\t<li>'patient.primary_pathology.histological_type'</li>\n",
       "\t<li>'patient.primary_pathology.laterality'</li>\n",
       "\t<li>'patient.primary_pathology.tumor_tissue_site'</li>\n",
       "\t<li>'patient.primary_pathology.year_of_initial_pathologic_diagnosis'</li>\n",
       "\t<li>'patient.prior_dx'</li>\n",
       "\t<li>'patient.race'</li>\n",
       "\t<li>'patient.stage_event.pathologic_stage'</li>\n",
       "\t<li>'patient.stage_event.system_version'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_m'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_n'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'</li>\n",
       "\t<li>'patient.tissue_prospective_collection_indicator'</li>\n",
       "\t<li>'patient.tissue_retrospective_collection_indicator'</li>\n",
       "\t<li>'patient.tissue_source_site'</li>\n",
       "\t<li>'patient.vital_status'</li>\n",
       "\t<li>'patient.year_of_form_completion'</li>\n",
       "\t<li>'patient.bcr_canonical_check.bcr_patient_canonical_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.country'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.days_to_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.histological_type'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.method_of_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.days_to_normal_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.method_of_normal_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.normal_controls.normal_control.vial_number'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_report_attached'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.sample_prescreened'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.site_of_disease'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_weight'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.vessel_used'</li>\n",
       "\t<li>'patient.days_to_index'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.days_to_collection'</li>\n",
       "\t<li>'patient.samples.sample-2.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.oct_embedded'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.vial_number'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample.days_to_collection'</li>\n",
       "\t<li>'patient.samples.sample.initial_weight'</li>\n",
       "\t<li>'patient.samples.sample.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.oct_embedded'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_file_name'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.a260_a280_ratio'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.amount'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio_28s_18s'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.image_file_name'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_lymphocyte_infiltration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_monocyte_infiltration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_necrosis'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_neutrophil_infiltration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_normal_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.section_location'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.weight'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.sample_type'</li>\n",
       "\t<li>'patient.samples.sample.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample.vial_number'</li>\n",
       "</ol>\n",
       "</dd>\n",
       "\t<dt>$lihc</dt>\n",
       "\t\t<dd><ol class=list-inline>\n",
       "\t<li>'admin.bcr'</li>\n",
       "\t<li>'admin.day_of_dcc_upload'</li>\n",
       "\t<li>'admin.disease_code'</li>\n",
       "\t<li>'admin.file_uuid'</li>\n",
       "\t<li>'admin.month_of_dcc_upload'</li>\n",
       "\t<li>'admin.patient_withdrawal.withdrawn'</li>\n",
       "\t<li>'admin.project_code'</li>\n",
       "\t<li>'admin.year_of_dcc_upload'</li>\n",
       "\t<li>'patient.bcr_patient_barcode'</li>\n",
       "\t<li>'patient.bcr_patient_uuid'</li>\n",
       "\t<li>'patient.clinical_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.clinical_cqcf.histological_type'</li>\n",
       "\t<li>'patient.clinical_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.clinical_cqcf.tumor_type'</li>\n",
       "\t<li>'patient.day_of_form_completion'</li>\n",
       "\t<li>'patient.gender'</li>\n",
       "\t<li>'patient.histological_type'</li>\n",
       "\t<li>'patient.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.icd_10'</li>\n",
       "\t<li>'patient.icd_o_3_histology'</li>\n",
       "\t<li>'patient.icd_o_3_site'</li>\n",
       "\t<li>'patient.informed_consent_verified'</li>\n",
       "\t<li>'patient.month_of_form_completion'</li>\n",
       "\t<li>'patient.patient_id'</li>\n",
       "\t<li>'patient.prior_dx'</li>\n",
       "\t<li>'patient.stage_event.system_version'</li>\n",
       "\t<li>'patient.stage_event.tnm_categories.pathologic_categories.pathologic_m'</li>\n",
       "\t<li>'patient.tissue_prospective_collection_indicator'</li>\n",
       "\t<li>'patient.tissue_retrospective_collection_indicator'</li>\n",
       "\t<li>'patient.tissue_source_site'</li>\n",
       "\t<li>'patient.tumor_tissue_site'</li>\n",
       "\t<li>'patient.vital_status'</li>\n",
       "\t<li>'patient.year_of_form_completion'</li>\n",
       "\t<li>'patient.bcr_canonical_check.bcr_patient_canonical_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.consent_or_death_status'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.histological_type'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.method_of_sample_procurement'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_diagnosis_matching'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_report_attached'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_tumor_necrosis_metrics'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.path_confirm_tumor_nuclei_metrics'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.sample_prescreened'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.site_of_disease'</li>\n",
       "\t<li>'patient.biospecimen_cqcf.vessel_used'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.oct_embedded'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.vial_number'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample.initial_weight'</li>\n",
       "\t<li>'patient.samples.sample.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.oct_embedded'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_file_name'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.image_file_name'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_necrosis'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_normal_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.slides.slide.section_location'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.weight'</li>\n",
       "\t<li>'patient.samples.sample.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample.sample_type'</li>\n",
       "\t<li>'patient.samples.sample.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample.vial_number'</li>\n",
       "</ol>\n",
       "</dd>\n",
       "</dl>\n"
      ],
      "text/latex": [
       "\\begin{description}\n",
       "\\item[\\$luad] \\begin{enumerate*}\n",
       "\\item 'admin.bcr'\n",
       "\\item 'admin.day\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.disease\\_code'\n",
       "\\item 'admin.file\\_uuid'\n",
       "\\item 'admin.month\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.patient\\_withdrawal.withdrawn'\n",
       "\\item 'admin.project\\_code'\n",
       "\\item 'admin.year\\_of\\_dcc\\_upload'\n",
       "\\item 'patient.bcr\\_patient\\_barcode'\n",
       "\\item 'patient.bcr\\_patient\\_uuid'\n",
       "\\item 'patient.clinical\\_cqcf.anatomic\\_neoplasm\\_subdivision'\n",
       "\\item 'patient.clinical\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.clinical\\_cqcf.histological\\_type'\n",
       "\\item 'patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.clinical\\_cqcf.tumor\\_type'\n",
       "\\item 'patient.diagnosis'\n",
       "\\item 'patient.gender'\n",
       "\\item 'patient.histological\\_type'\n",
       "\\item 'patient.icd\\_10'\n",
       "\\item 'patient.icd\\_o\\_3\\_histology'\n",
       "\\item 'patient.icd\\_o\\_3\\_site'\n",
       "\\item 'patient.informed\\_consent\\_verified'\n",
       "\\item 'patient.month\\_of\\_form\\_completion'\n",
       "\\item 'patient.patient\\_id'\n",
       "\\item 'patient.prior\\_dx'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t'\n",
       "\\item 'patient.tissue\\_source\\_site'\n",
       "\\item 'patient.tumor\\_tissue\\_site'\n",
       "\\item 'patient.vital\\_status'\n",
       "\\item 'patient.year\\_of\\_form\\_completion'\n",
       "\\item 'patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.digital\\_image\\_submitted'\n",
       "\\item 'patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site'\n",
       "\\item 'patient.biospecimen\\_cqcf.histological\\_type'\n",
       "\\item 'patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached'\n",
       "\\item 'patient.biospecimen\\_cqcf.sample\\_prescreened'\n",
       "\\item 'patient.biospecimen\\_cqcf.site\\_of\\_disease'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_nuclei\\_percent'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number'\n",
       "\\item 'patient.biospecimen\\_cqcf.vessel\\_used'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample-2.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample-2.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.sample\\_type'\n",
       "\\item 'patient.samples.sample-2.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.vial\\_number'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_file\\_name'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.section\\_location'\n",
       "\\item 'patient.samples.sample.portions.portion.weight'\n",
       "\\item 'patient.samples.sample.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.sample\\_type'\n",
       "\\item 'patient.samples.sample.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample.vial\\_number'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item[\\$lusc] \\begin{enumerate*}\n",
       "\\item 'admin.bcr'\n",
       "\\item 'admin.day\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.disease\\_code'\n",
       "\\item 'admin.month\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.patient\\_withdrawal.withdrawn'\n",
       "\\item 'admin.year\\_of\\_dcc\\_upload'\n",
       "\\item 'patient.bcr\\_patient\\_barcode'\n",
       "\\item 'patient.bcr\\_patient\\_uuid'\n",
       "\\item 'patient.clinical\\_cqcf.country'\n",
       "\\item 'patient.clinical\\_cqcf.tumor\\_type'\n",
       "\\item 'patient.diagnosis'\n",
       "\\item 'patient.gender'\n",
       "\\item 'patient.histological\\_type'\n",
       "\\item 'patient.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.icd\\_10'\n",
       "\\item 'patient.icd\\_o\\_3\\_histology'\n",
       "\\item 'patient.icd\\_o\\_3\\_site'\n",
       "\\item 'patient.informed\\_consent\\_verified'\n",
       "\\item 'patient.month\\_of\\_form\\_completion'\n",
       "\\item 'patient.patient\\_id'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t'\n",
       "\\item 'patient.tissue\\_source\\_site'\n",
       "\\item 'patient.tumor\\_tissue\\_site'\n",
       "\\item 'patient.vital\\_status'\n",
       "\\item 'patient.year\\_of\\_form\\_completion'\n",
       "\\item 'patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample-2.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample-2.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.sample\\_type'\n",
       "\\item 'patient.samples.sample-2.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.vial\\_number'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_file\\_name'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_uuid'\n",
       "\\item 'patient.samples.sample.sample\\_type'\n",
       "\\item 'patient.samples.sample.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample.vial\\_number'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item[\\$meso] \\begin{enumerate*}\n",
       "\\item 'admin.bcr'\n",
       "\\item 'admin.day\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.disease\\_code'\n",
       "\\item 'admin.file\\_uuid'\n",
       "\\item 'admin.month\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.patient\\_withdrawal.withdrawn'\n",
       "\\item 'admin.project\\_code'\n",
       "\\item 'admin.year\\_of\\_dcc\\_upload'\n",
       "\\item 'patient.bcr\\_patient\\_barcode'\n",
       "\\item 'patient.bcr\\_patient\\_uuid'\n",
       "\\item 'patient.clinical\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.clinical\\_cqcf.country'\n",
       "\\item 'patient.clinical\\_cqcf.histological\\_type'\n",
       "\\item 'patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.clinical\\_cqcf.tumor\\_type'\n",
       "\\item 'patient.day\\_of\\_form\\_completion'\n",
       "\\item 'patient.days\\_to\\_birth'\n",
       "\\item 'patient.gender'\n",
       "\\item 'patient.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.icd\\_10'\n",
       "\\item 'patient.icd\\_o\\_3\\_histology'\n",
       "\\item 'patient.icd\\_o\\_3\\_site'\n",
       "\\item 'patient.informed\\_consent\\_verified'\n",
       "\\item 'patient.month\\_of\\_form\\_completion'\n",
       "\\item 'patient.patient\\_id'\n",
       "\\item 'patient.primary\\_pathology.age\\_at\\_initial\\_pathologic\\_diagnosis'\n",
       "\\item 'patient.primary\\_pathology.days\\_to\\_initial\\_pathologic\\_diagnosis'\n",
       "\\item 'patient.primary\\_pathology.histological\\_type'\n",
       "\\item 'patient.primary\\_pathology.laterality'\n",
       "\\item 'patient.primary\\_pathology.tumor\\_tissue\\_site'\n",
       "\\item 'patient.primary\\_pathology.year\\_of\\_initial\\_pathologic\\_diagnosis'\n",
       "\\item 'patient.prior\\_dx'\n",
       "\\item 'patient.race'\n",
       "\\item 'patient.stage\\_event.pathologic\\_stage'\n",
       "\\item 'patient.stage\\_event.system\\_version'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_n'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_t'\n",
       "\\item 'patient.tissue\\_prospective\\_collection\\_indicator'\n",
       "\\item 'patient.tissue\\_retrospective\\_collection\\_indicator'\n",
       "\\item 'patient.tissue\\_source\\_site'\n",
       "\\item 'patient.vital\\_status'\n",
       "\\item 'patient.year\\_of\\_form\\_completion'\n",
       "\\item 'patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.country'\n",
       "\\item 'patient.biospecimen\\_cqcf.days\\_to\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site'\n",
       "\\item 'patient.biospecimen\\_cqcf.histological\\_type'\n",
       "\\item 'patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.days\\_to\\_normal\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.method\\_of\\_normal\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.normal\\_controls.normal\\_control.vial\\_number'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached'\n",
       "\\item 'patient.biospecimen\\_cqcf.sample\\_prescreened'\n",
       "\\item 'patient.biospecimen\\_cqcf.site\\_of\\_disease'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.tumor\\_weight'\n",
       "\\item 'patient.biospecimen\\_cqcf.tumor\\_samples.tumor\\_sample.vial\\_number'\n",
       "\\item 'patient.biospecimen\\_cqcf.vessel\\_used'\n",
       "\\item 'patient.days\\_to\\_index'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample-2.days\\_to\\_collection'\n",
       "\\item 'patient.samples.sample-2.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.oct\\_embedded'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample-2.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.sample\\_type'\n",
       "\\item 'patient.samples.sample-2.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.vial\\_number'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample.days\\_to\\_collection'\n",
       "\\item 'patient.samples.sample.initial\\_weight'\n",
       "\\item 'patient.samples.sample.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.oct\\_embedded'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_file\\_name'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.a260\\_a280\\_ratio'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.amount'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio\\_28s\\_18s'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_lymphocyte\\_infiltration'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_monocyte\\_infiltration'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_neutrophil\\_infiltration'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.section\\_location'\n",
       "\\item 'patient.samples.sample.portions.portion.weight'\n",
       "\\item 'patient.samples.sample.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.sample\\_type'\n",
       "\\item 'patient.samples.sample.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample.vial\\_number'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\item[\\$lihc] \\begin{enumerate*}\n",
       "\\item 'admin.bcr'\n",
       "\\item 'admin.day\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.disease\\_code'\n",
       "\\item 'admin.file\\_uuid'\n",
       "\\item 'admin.month\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.patient\\_withdrawal.withdrawn'\n",
       "\\item 'admin.project\\_code'\n",
       "\\item 'admin.year\\_of\\_dcc\\_upload'\n",
       "\\item 'patient.bcr\\_patient\\_barcode'\n",
       "\\item 'patient.bcr\\_patient\\_uuid'\n",
       "\\item 'patient.clinical\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.clinical\\_cqcf.histological\\_type'\n",
       "\\item 'patient.clinical\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.clinical\\_cqcf.tumor\\_type'\n",
       "\\item 'patient.day\\_of\\_form\\_completion'\n",
       "\\item 'patient.gender'\n",
       "\\item 'patient.histological\\_type'\n",
       "\\item 'patient.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.icd\\_10'\n",
       "\\item 'patient.icd\\_o\\_3\\_histology'\n",
       "\\item 'patient.icd\\_o\\_3\\_site'\n",
       "\\item 'patient.informed\\_consent\\_verified'\n",
       "\\item 'patient.month\\_of\\_form\\_completion'\n",
       "\\item 'patient.patient\\_id'\n",
       "\\item 'patient.prior\\_dx'\n",
       "\\item 'patient.stage\\_event.system\\_version'\n",
       "\\item 'patient.stage\\_event.tnm\\_categories.pathologic\\_categories.pathologic\\_m'\n",
       "\\item 'patient.tissue\\_prospective\\_collection\\_indicator'\n",
       "\\item 'patient.tissue\\_retrospective\\_collection\\_indicator'\n",
       "\\item 'patient.tissue\\_source\\_site'\n",
       "\\item 'patient.tumor\\_tissue\\_site'\n",
       "\\item 'patient.vital\\_status'\n",
       "\\item 'patient.year\\_of\\_form\\_completion'\n",
       "\\item 'patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.consent\\_or\\_death\\_status'\n",
       "\\item 'patient.biospecimen\\_cqcf.frozen\\_specimen\\_anatomic\\_site'\n",
       "\\item 'patient.biospecimen\\_cqcf.histological\\_type'\n",
       "\\item 'patient.biospecimen\\_cqcf.history\\_of\\_neoadjuvant\\_treatment'\n",
       "\\item 'patient.biospecimen\\_cqcf.method\\_of\\_sample\\_procurement'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_diagnosis\\_matching'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_report\\_attached'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_necrosis\\_metrics'\n",
       "\\item 'patient.biospecimen\\_cqcf.path\\_confirm\\_tumor\\_nuclei\\_metrics'\n",
       "\\item 'patient.biospecimen\\_cqcf.sample\\_prescreened'\n",
       "\\item 'patient.biospecimen\\_cqcf.site\\_of\\_disease'\n",
       "\\item 'patient.biospecimen\\_cqcf.vessel\\_used'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample-2.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.oct\\_embedded'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample-2.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.sample\\_type'\n",
       "\\item 'patient.samples.sample-2.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.vial\\_number'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample.initial\\_weight'\n",
       "\\item 'patient.samples.sample.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.oct\\_embedded'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_file\\_name'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen\\_barcode\\_bottom'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal\\_tumor\\_genotype\\_match'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr\\_amplification\\_successful'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_barcode'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.bcr\\_slide\\_uuid'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.image\\_file\\_name'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_necrosis'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_normal\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_stromal\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_cells'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.percent\\_tumor\\_nuclei'\n",
       "\\item 'patient.samples.sample.portions.portion.slides.slide.section\\_location'\n",
       "\\item 'patient.samples.sample.portions.portion.weight'\n",
       "\\item 'patient.samples.sample.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample.sample\\_type'\n",
       "\\item 'patient.samples.sample.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample.vial\\_number'\n",
       "\\end{enumerate*}\n",
       "\n",
       "\\end{description}\n"
      ],
      "text/markdown": [
       "$luad\n",
       ":   1. 'admin.bcr'\n",
       "2. 'admin.day_of_dcc_upload'\n",
       "3. 'admin.disease_code'\n",
       "4. 'admin.file_uuid'\n",
       "5. 'admin.month_of_dcc_upload'\n",
       "6. 'admin.patient_withdrawal.withdrawn'\n",
       "7. 'admin.project_code'\n",
       "8. 'admin.year_of_dcc_upload'\n",
       "9. 'patient.bcr_patient_barcode'\n",
       "10. 'patient.bcr_patient_uuid'\n",
       "11. 'patient.clinical_cqcf.anatomic_neoplasm_subdivision'\n",
       "12. 'patient.clinical_cqcf.consent_or_death_status'\n",
       "13. 'patient.clinical_cqcf.histological_type'\n",
       "14. 'patient.clinical_cqcf.history_of_neoadjuvant_treatment'\n",
       "15. 'patient.clinical_cqcf.tumor_type'\n",
       "16. 'patient.diagnosis'\n",
       "17. 'patient.gender'\n",
       "18. 'patient.histological_type'\n",
       "19. 'patient.icd_10'\n",
       "20. 'patient.icd_o_3_histology'\n",
       "21. 'patient.icd_o_3_site'\n",
       "22. 'patient.informed_consent_verified'\n",
       "23. 'patient.month_of_form_completion'\n",
       "24. 'patient.patient_id'\n",
       "25. 'patient.prior_dx'\n",
       "26. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'\n",
       "27. 'patient.tissue_source_site'\n",
       "28. 'patient.tumor_tissue_site'\n",
       "29. 'patient.vital_status'\n",
       "30. 'patient.year_of_form_completion'\n",
       "31. 'patient.bcr_canonical_check.bcr_patient_canonical_status'\n",
       "32. 'patient.biospecimen_cqcf.consent_or_death_status'\n",
       "33. 'patient.biospecimen_cqcf.digital_image_submitted'\n",
       "34. 'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'\n",
       "35. 'patient.biospecimen_cqcf.histological_type'\n",
       "36. 'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'\n",
       "37. 'patient.biospecimen_cqcf.method_of_sample_procurement'\n",
       "38. 'patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid'\n",
       "39. 'patient.biospecimen_cqcf.normal_controls.normal_control.vial_number'\n",
       "40. 'patient.biospecimen_cqcf.path_confirm_report_attached'\n",
       "41. 'patient.biospecimen_cqcf.sample_prescreened'\n",
       "42. 'patient.biospecimen_cqcf.site_of_disease'\n",
       "43. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid'\n",
       "44. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_nuclei_percent'\n",
       "45. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number'\n",
       "46. 'patient.biospecimen_cqcf.vessel_used'\n",
       "47. 'patient.samples.sample-2.bcr_sample_barcode'\n",
       "48. 'patient.samples.sample-2.bcr_sample_uuid'\n",
       "49. 'patient.samples.sample-2.is_ffpe'\n",
       "50. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "51. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "52. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "53. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "54. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "55. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "56. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "57. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "58. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "59. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "60. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "61. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "62. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "63. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "64. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "65. 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "66. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'\n",
       "67. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'\n",
       "68. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "69. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "70. 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "71. 'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'\n",
       "72. 'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "73. 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "74. 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "75. 'patient.samples.sample-2.portions.portion.bcr_portion_barcode'\n",
       "76. 'patient.samples.sample-2.portions.portion.bcr_portion_uuid'\n",
       "77. 'patient.samples.sample-2.portions.portion.day_of_creation'\n",
       "78. 'patient.samples.sample-2.portions.portion.is_ffpe'\n",
       "79. 'patient.samples.sample-2.portions.portion.month_of_creation'\n",
       "80. 'patient.samples.sample-2.portions.portion.portion_number'\n",
       "81. 'patient.samples.sample-2.portions.portion.year_of_creation'\n",
       "82. 'patient.samples.sample-2.sample_type'\n",
       "83. 'patient.samples.sample-2.sample_type_id'\n",
       "84. 'patient.samples.sample-2.vial_number'\n",
       "85. 'patient.samples.sample.bcr_sample_barcode'\n",
       "86. 'patient.samples.sample.bcr_sample_uuid'\n",
       "87. 'patient.samples.sample.is_ffpe'\n",
       "88. 'patient.samples.sample.pathology_report_file_name'\n",
       "89. 'patient.samples.sample.pathology_report_uuid'\n",
       "90. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "91. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "92. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "93. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "94. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "95. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "96. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "97. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "98. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "99. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "100. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "101. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "102. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "103. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "104. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "105. 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "106. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'\n",
       "107. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'\n",
       "108. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "109. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "110. 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "111. 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "112. 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "113. 'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'\n",
       "114. 'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "115. 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "116. 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "117. 'patient.samples.sample.portions.portion.bcr_portion_barcode'\n",
       "118. 'patient.samples.sample.portions.portion.bcr_portion_uuid'\n",
       "119. 'patient.samples.sample.portions.portion.day_of_creation'\n",
       "120. 'patient.samples.sample.portions.portion.is_ffpe'\n",
       "121. 'patient.samples.sample.portions.portion.month_of_creation'\n",
       "122. 'patient.samples.sample.portions.portion.portion_number'\n",
       "123. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'\n",
       "124. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'\n",
       "125. 'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'\n",
       "126. 'patient.samples.sample.portions.portion.slides.slide.section_location'\n",
       "127. 'patient.samples.sample.portions.portion.weight'\n",
       "128. 'patient.samples.sample.portions.portion.year_of_creation'\n",
       "129. 'patient.samples.sample.sample_type'\n",
       "130. 'patient.samples.sample.sample_type_id'\n",
       "131. 'patient.samples.sample.vial_number'\n",
       "\n",
       "\n",
       "\n",
       "$lusc\n",
       ":   1. 'admin.bcr'\n",
       "2. 'admin.day_of_dcc_upload'\n",
       "3. 'admin.disease_code'\n",
       "4. 'admin.month_of_dcc_upload'\n",
       "5. 'admin.patient_withdrawal.withdrawn'\n",
       "6. 'admin.year_of_dcc_upload'\n",
       "7. 'patient.bcr_patient_barcode'\n",
       "8. 'patient.bcr_patient_uuid'\n",
       "9. 'patient.clinical_cqcf.country'\n",
       "10. 'patient.clinical_cqcf.tumor_type'\n",
       "11. 'patient.diagnosis'\n",
       "12. 'patient.gender'\n",
       "13. 'patient.histological_type'\n",
       "14. 'patient.history_of_neoadjuvant_treatment'\n",
       "15. 'patient.icd_10'\n",
       "16. 'patient.icd_o_3_histology'\n",
       "17. 'patient.icd_o_3_site'\n",
       "18. 'patient.informed_consent_verified'\n",
       "19. 'patient.month_of_form_completion'\n",
       "20. 'patient.patient_id'\n",
       "21. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_n'\n",
       "22. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'\n",
       "23. 'patient.tissue_source_site'\n",
       "24. 'patient.tumor_tissue_site'\n",
       "25. 'patient.vital_status'\n",
       "26. 'patient.year_of_form_completion'\n",
       "27. 'patient.bcr_canonical_check.bcr_patient_canonical_status'\n",
       "28. 'patient.samples.sample-2.bcr_sample_barcode'\n",
       "29. 'patient.samples.sample-2.bcr_sample_uuid'\n",
       "30. 'patient.samples.sample-2.is_ffpe'\n",
       "31. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "32. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "33. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "34. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "35. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "36. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "37. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "38. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "39. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "40. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "41. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "42. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "43. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "44. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "45. 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "46. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'\n",
       "47. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'\n",
       "48. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "49. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "50. 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "51. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "52. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "53. 'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'\n",
       "54. 'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "55. 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "56. 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "57. 'patient.samples.sample-2.portions.portion.bcr_portion_barcode'\n",
       "58. 'patient.samples.sample-2.portions.portion.bcr_portion_uuid'\n",
       "59. 'patient.samples.sample-2.portions.portion.day_of_creation'\n",
       "60. 'patient.samples.sample-2.portions.portion.is_ffpe'\n",
       "61. 'patient.samples.sample-2.portions.portion.month_of_creation'\n",
       "62. 'patient.samples.sample-2.portions.portion.portion_number'\n",
       "63. 'patient.samples.sample-2.portions.portion.year_of_creation'\n",
       "64. 'patient.samples.sample-2.sample_type'\n",
       "65. 'patient.samples.sample-2.sample_type_id'\n",
       "66. 'patient.samples.sample-2.vial_number'\n",
       "67. 'patient.samples.sample.bcr_sample_barcode'\n",
       "68. 'patient.samples.sample.bcr_sample_uuid'\n",
       "69. 'patient.samples.sample.is_ffpe'\n",
       "70. 'patient.samples.sample.pathology_report_file_name'\n",
       "71. 'patient.samples.sample.pathology_report_uuid'\n",
       "72. 'patient.samples.sample.sample_type'\n",
       "73. 'patient.samples.sample.sample_type_id'\n",
       "74. 'patient.samples.sample.vial_number'\n",
       "\n",
       "\n",
       "\n",
       "$meso\n",
       ":   1. 'admin.bcr'\n",
       "2. 'admin.day_of_dcc_upload'\n",
       "3. 'admin.disease_code'\n",
       "4. 'admin.file_uuid'\n",
       "5. 'admin.month_of_dcc_upload'\n",
       "6. 'admin.patient_withdrawal.withdrawn'\n",
       "7. 'admin.project_code'\n",
       "8. 'admin.year_of_dcc_upload'\n",
       "9. 'patient.bcr_patient_barcode'\n",
       "10. 'patient.bcr_patient_uuid'\n",
       "11. 'patient.clinical_cqcf.consent_or_death_status'\n",
       "12. 'patient.clinical_cqcf.country'\n",
       "13. 'patient.clinical_cqcf.histological_type'\n",
       "14. 'patient.clinical_cqcf.history_of_neoadjuvant_treatment'\n",
       "15. 'patient.clinical_cqcf.tumor_type'\n",
       "16. 'patient.day_of_form_completion'\n",
       "17. 'patient.days_to_birth'\n",
       "18. 'patient.gender'\n",
       "19. 'patient.history_of_neoadjuvant_treatment'\n",
       "20. 'patient.icd_10'\n",
       "21. 'patient.icd_o_3_histology'\n",
       "22. 'patient.icd_o_3_site'\n",
       "23. 'patient.informed_consent_verified'\n",
       "24. 'patient.month_of_form_completion'\n",
       "25. 'patient.patient_id'\n",
       "26. 'patient.primary_pathology.age_at_initial_pathologic_diagnosis'\n",
       "27. 'patient.primary_pathology.days_to_initial_pathologic_diagnosis'\n",
       "28. 'patient.primary_pathology.histological_type'\n",
       "29. 'patient.primary_pathology.laterality'\n",
       "30. 'patient.primary_pathology.tumor_tissue_site'\n",
       "31. 'patient.primary_pathology.year_of_initial_pathologic_diagnosis'\n",
       "32. 'patient.prior_dx'\n",
       "33. 'patient.race'\n",
       "34. 'patient.stage_event.pathologic_stage'\n",
       "35. 'patient.stage_event.system_version'\n",
       "36. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_m'\n",
       "37. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_n'\n",
       "38. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_t'\n",
       "39. 'patient.tissue_prospective_collection_indicator'\n",
       "40. 'patient.tissue_retrospective_collection_indicator'\n",
       "41. 'patient.tissue_source_site'\n",
       "42. 'patient.vital_status'\n",
       "43. 'patient.year_of_form_completion'\n",
       "44. 'patient.bcr_canonical_check.bcr_patient_canonical_status'\n",
       "45. 'patient.biospecimen_cqcf.consent_or_death_status'\n",
       "46. 'patient.biospecimen_cqcf.country'\n",
       "47. 'patient.biospecimen_cqcf.days_to_sample_procurement'\n",
       "48. 'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'\n",
       "49. 'patient.biospecimen_cqcf.histological_type'\n",
       "50. 'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'\n",
       "51. 'patient.biospecimen_cqcf.method_of_sample_procurement'\n",
       "52. 'patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid'\n",
       "53. 'patient.biospecimen_cqcf.normal_controls.normal_control.days_to_normal_sample_procurement'\n",
       "54. 'patient.biospecimen_cqcf.normal_controls.normal_control.method_of_normal_sample_procurement'\n",
       "55. 'patient.biospecimen_cqcf.normal_controls.normal_control.vial_number'\n",
       "56. 'patient.biospecimen_cqcf.path_confirm_report_attached'\n",
       "57. 'patient.biospecimen_cqcf.sample_prescreened'\n",
       "58. 'patient.biospecimen_cqcf.site_of_disease'\n",
       "59. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid'\n",
       "60. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_weight'\n",
       "61. 'patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number'\n",
       "62. 'patient.biospecimen_cqcf.vessel_used'\n",
       "63. 'patient.days_to_index'\n",
       "64. 'patient.samples.sample-2.bcr_sample_barcode'\n",
       "65. 'patient.samples.sample-2.bcr_sample_uuid'\n",
       "66. 'patient.samples.sample-2.days_to_collection'\n",
       "67. 'patient.samples.sample-2.is_ffpe'\n",
       "68. 'patient.samples.sample-2.oct_embedded'\n",
       "69. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'\n",
       "70. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'\n",
       "71. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'\n",
       "72. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'\n",
       "73. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "74. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'\n",
       "75. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'\n",
       "76. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'\n",
       "77. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'\n",
       "78. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'\n",
       "79. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'\n",
       "80. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "81. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'\n",
       "82. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "83. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'\n",
       "84. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "85. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "86. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "87. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "88. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "89. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "90. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "91. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "92. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "93. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "94. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "95. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "96. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "97. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "98. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "99. 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "100. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'\n",
       "101. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'\n",
       "102. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "103. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "104. 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "105. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "106. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "107. 'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'\n",
       "108. 'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "109. 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "110. 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "111. 'patient.samples.sample-2.portions.portion.bcr_portion_barcode'\n",
       "112. 'patient.samples.sample-2.portions.portion.bcr_portion_uuid'\n",
       "113. 'patient.samples.sample-2.portions.portion.day_of_creation'\n",
       "114. 'patient.samples.sample-2.portions.portion.is_ffpe'\n",
       "115. 'patient.samples.sample-2.portions.portion.month_of_creation'\n",
       "116. 'patient.samples.sample-2.portions.portion.portion_number'\n",
       "117. 'patient.samples.sample-2.portions.portion.year_of_creation'\n",
       "118. 'patient.samples.sample-2.sample_type'\n",
       "119. 'patient.samples.sample-2.sample_type_id'\n",
       "120. 'patient.samples.sample-2.vial_number'\n",
       "121. 'patient.samples.sample.bcr_sample_barcode'\n",
       "122. 'patient.samples.sample.bcr_sample_uuid'\n",
       "123. 'patient.samples.sample.days_to_collection'\n",
       "124. 'patient.samples.sample.initial_weight'\n",
       "125. 'patient.samples.sample.is_ffpe'\n",
       "126. 'patient.samples.sample.oct_embedded'\n",
       "127. 'patient.samples.sample.pathology_report_file_name'\n",
       "128. 'patient.samples.sample.pathology_report_uuid'\n",
       "129. 'patient.samples.sample.portions.portion.analytes.analyte-2.a260_a280_ratio'\n",
       "130. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_barcode'\n",
       "131. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_uuid'\n",
       "132. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen_barcode_bottom'\n",
       "133. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center_id'\n",
       "134. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration'\n",
       "135. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day_of_shipment'\n",
       "136. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is_derived_from_ffpe'\n",
       "137. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month_of_shipment'\n",
       "138. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_column'\n",
       "139. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_id'\n",
       "140. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_row'\n",
       "141. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity'\n",
       "142. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source_center'\n",
       "143. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume'\n",
       "144. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year_of_shipment'\n",
       "145. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "146. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "147. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "148. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id'\n",
       "149. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'\n",
       "150. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment'\n",
       "151. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe'\n",
       "152. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment'\n",
       "153. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column'\n",
       "154. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id'\n",
       "155. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row'\n",
       "156. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'\n",
       "157. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center'\n",
       "158. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'\n",
       "159. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment'\n",
       "160. 'patient.samples.sample.portions.portion.analytes.analyte-2.amount'\n",
       "161. 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type'\n",
       "162. 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id'\n",
       "163. 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode'\n",
       "164. 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid'\n",
       "165. 'patient.samples.sample.portions.portion.analytes.analyte-2.concentration'\n",
       "166. 'patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file'\n",
       "167. 'patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe'\n",
       "168. 'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type'\n",
       "169. 'patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio_28s_18s'\n",
       "170. 'patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue'\n",
       "171. 'patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer_method'\n",
       "172. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'\n",
       "173. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'\n",
       "174. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'\n",
       "175. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'\n",
       "176. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "177. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'\n",
       "178. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'\n",
       "179. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'\n",
       "180. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'\n",
       "181. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'\n",
       "182. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'\n",
       "183. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "184. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'\n",
       "185. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "186. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'\n",
       "187. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode'\n",
       "188. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid'\n",
       "189. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom'\n",
       "190. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id'\n",
       "191. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'\n",
       "192. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment'\n",
       "193. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe'\n",
       "194. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment'\n",
       "195. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column'\n",
       "196. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id'\n",
       "197. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row'\n",
       "198. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'\n",
       "199. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center'\n",
       "200. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'\n",
       "201. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment'\n",
       "202. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "203. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "204. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "205. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "206. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "207. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "208. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "209. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "210. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "211. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "212. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "213. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "214. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "215. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "216. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "217. 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "218. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'\n",
       "219. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'\n",
       "220. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "221. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "222. 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "223. 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "224. 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "225. 'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'\n",
       "226. 'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "227. 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "228. 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "229. 'patient.samples.sample.portions.portion.bcr_portion_barcode'\n",
       "230. 'patient.samples.sample.portions.portion.bcr_portion_uuid'\n",
       "231. 'patient.samples.sample.portions.portion.day_of_creation'\n",
       "232. 'patient.samples.sample.portions.portion.is_ffpe'\n",
       "233. 'patient.samples.sample.portions.portion.month_of_creation'\n",
       "234. 'patient.samples.sample.portions.portion.portion_number'\n",
       "235. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'\n",
       "236. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'\n",
       "237. 'patient.samples.sample.portions.portion.slides.slide.image_file_name'\n",
       "238. 'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'\n",
       "239. 'patient.samples.sample.portions.portion.slides.slide.percent_lymphocyte_infiltration'\n",
       "240. 'patient.samples.sample.portions.portion.slides.slide.percent_monocyte_infiltration'\n",
       "241. 'patient.samples.sample.portions.portion.slides.slide.percent_necrosis'\n",
       "242. 'patient.samples.sample.portions.portion.slides.slide.percent_neutrophil_infiltration'\n",
       "243. 'patient.samples.sample.portions.portion.slides.slide.percent_normal_cells'\n",
       "244. 'patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells'\n",
       "245. 'patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells'\n",
       "246. 'patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei'\n",
       "247. 'patient.samples.sample.portions.portion.slides.slide.section_location'\n",
       "248. 'patient.samples.sample.portions.portion.weight'\n",
       "249. 'patient.samples.sample.portions.portion.year_of_creation'\n",
       "250. 'patient.samples.sample.sample_type'\n",
       "251. 'patient.samples.sample.sample_type_id'\n",
       "252. 'patient.samples.sample.vial_number'\n",
       "\n",
       "\n",
       "\n",
       "$lihc\n",
       ":   1. 'admin.bcr'\n",
       "2. 'admin.day_of_dcc_upload'\n",
       "3. 'admin.disease_code'\n",
       "4. 'admin.file_uuid'\n",
       "5. 'admin.month_of_dcc_upload'\n",
       "6. 'admin.patient_withdrawal.withdrawn'\n",
       "7. 'admin.project_code'\n",
       "8. 'admin.year_of_dcc_upload'\n",
       "9. 'patient.bcr_patient_barcode'\n",
       "10. 'patient.bcr_patient_uuid'\n",
       "11. 'patient.clinical_cqcf.consent_or_death_status'\n",
       "12. 'patient.clinical_cqcf.histological_type'\n",
       "13. 'patient.clinical_cqcf.history_of_neoadjuvant_treatment'\n",
       "14. 'patient.clinical_cqcf.tumor_type'\n",
       "15. 'patient.day_of_form_completion'\n",
       "16. 'patient.gender'\n",
       "17. 'patient.histological_type'\n",
       "18. 'patient.history_of_neoadjuvant_treatment'\n",
       "19. 'patient.icd_10'\n",
       "20. 'patient.icd_o_3_histology'\n",
       "21. 'patient.icd_o_3_site'\n",
       "22. 'patient.informed_consent_verified'\n",
       "23. 'patient.month_of_form_completion'\n",
       "24. 'patient.patient_id'\n",
       "25. 'patient.prior_dx'\n",
       "26. 'patient.stage_event.system_version'\n",
       "27. 'patient.stage_event.tnm_categories.pathologic_categories.pathologic_m'\n",
       "28. 'patient.tissue_prospective_collection_indicator'\n",
       "29. 'patient.tissue_retrospective_collection_indicator'\n",
       "30. 'patient.tissue_source_site'\n",
       "31. 'patient.tumor_tissue_site'\n",
       "32. 'patient.vital_status'\n",
       "33. 'patient.year_of_form_completion'\n",
       "34. 'patient.bcr_canonical_check.bcr_patient_canonical_status'\n",
       "35. 'patient.biospecimen_cqcf.consent_or_death_status'\n",
       "36. 'patient.biospecimen_cqcf.frozen_specimen_anatomic_site'\n",
       "37. 'patient.biospecimen_cqcf.histological_type'\n",
       "38. 'patient.biospecimen_cqcf.history_of_neoadjuvant_treatment'\n",
       "39. 'patient.biospecimen_cqcf.method_of_sample_procurement'\n",
       "40. 'patient.biospecimen_cqcf.path_confirm_diagnosis_matching'\n",
       "41. 'patient.biospecimen_cqcf.path_confirm_report_attached'\n",
       "42. 'patient.biospecimen_cqcf.path_confirm_tumor_necrosis_metrics'\n",
       "43. 'patient.biospecimen_cqcf.path_confirm_tumor_nuclei_metrics'\n",
       "44. 'patient.biospecimen_cqcf.sample_prescreened'\n",
       "45. 'patient.biospecimen_cqcf.site_of_disease'\n",
       "46. 'patient.biospecimen_cqcf.vessel_used'\n",
       "47. 'patient.samples.sample-2.bcr_sample_barcode'\n",
       "48. 'patient.samples.sample-2.bcr_sample_uuid'\n",
       "49. 'patient.samples.sample-2.is_ffpe'\n",
       "50. 'patient.samples.sample-2.oct_embedded'\n",
       "51. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'\n",
       "52. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'\n",
       "53. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'\n",
       "54. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'\n",
       "55. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "56. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'\n",
       "57. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'\n",
       "58. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'\n",
       "59. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'\n",
       "60. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'\n",
       "61. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'\n",
       "62. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "63. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'\n",
       "64. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "65. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'\n",
       "66. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "67. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "68. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "69. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "70. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "71. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "72. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "73. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "74. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "75. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "76. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "77. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "78. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "79. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "80. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "81. 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "82. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'\n",
       "83. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'\n",
       "84. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "85. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "86. 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "87. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "88. 'patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "89. 'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'\n",
       "90. 'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "91. 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "92. 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "93. 'patient.samples.sample-2.portions.portion.bcr_portion_barcode'\n",
       "94. 'patient.samples.sample-2.portions.portion.bcr_portion_uuid'\n",
       "95. 'patient.samples.sample-2.portions.portion.day_of_creation'\n",
       "96. 'patient.samples.sample-2.portions.portion.is_ffpe'\n",
       "97. 'patient.samples.sample-2.portions.portion.month_of_creation'\n",
       "98. 'patient.samples.sample-2.portions.portion.portion_number'\n",
       "99. 'patient.samples.sample-2.portions.portion.year_of_creation'\n",
       "100. 'patient.samples.sample-2.sample_type'\n",
       "101. 'patient.samples.sample-2.sample_type_id'\n",
       "102. 'patient.samples.sample-2.vial_number'\n",
       "103. 'patient.samples.sample.bcr_sample_barcode'\n",
       "104. 'patient.samples.sample.bcr_sample_uuid'\n",
       "105. 'patient.samples.sample.initial_weight'\n",
       "106. 'patient.samples.sample.is_ffpe'\n",
       "107. 'patient.samples.sample.oct_embedded'\n",
       "108. 'patient.samples.sample.pathology_report_file_name'\n",
       "109. 'patient.samples.sample.pathology_report_uuid'\n",
       "110. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "111. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "112. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "113. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id'\n",
       "114. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration'\n",
       "115. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment'\n",
       "116. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe'\n",
       "117. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment'\n",
       "118. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column'\n",
       "119. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id'\n",
       "120. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row'\n",
       "121. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity'\n",
       "122. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center'\n",
       "123. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume'\n",
       "124. 'patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment'\n",
       "125. 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type'\n",
       "126. 'patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id'\n",
       "127. 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode'\n",
       "128. 'patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid'\n",
       "129. 'patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file'\n",
       "130. 'patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe'\n",
       "131. 'patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type'\n",
       "132. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode'\n",
       "133. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid'\n",
       "134. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom'\n",
       "135. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id'\n",
       "136. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration'\n",
       "137. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment'\n",
       "138. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe'\n",
       "139. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment'\n",
       "140. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column'\n",
       "141. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id'\n",
       "142. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row'\n",
       "143. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity'\n",
       "144. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center'\n",
       "145. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume'\n",
       "146. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment'\n",
       "147. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode'\n",
       "148. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid'\n",
       "149. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom'\n",
       "150. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id'\n",
       "151. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration'\n",
       "152. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment'\n",
       "153. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe'\n",
       "154. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment'\n",
       "155. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column'\n",
       "156. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id'\n",
       "157. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row'\n",
       "158. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity'\n",
       "159. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center'\n",
       "160. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume'\n",
       "161. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment'\n",
       "162. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "163. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "164. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom'\n",
       "165. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "166. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "167. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "168. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "169. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "170. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "171. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "172. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "173. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "174. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "175. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "176. 'patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "177. 'patient.samples.sample.portions.portion.analytes.analyte.amount'\n",
       "178. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type'\n",
       "179. 'patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id'\n",
       "180. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "181. 'patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "182. 'patient.samples.sample.portions.portion.analytes.analyte.concentration'\n",
       "183. 'patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match'\n",
       "184. 'patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful'\n",
       "185. 'patient.samples.sample.portions.portion.analytes.analyte.gel_image_file'\n",
       "186. 'patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "187. 'patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "188. 'patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "189. 'patient.samples.sample.portions.portion.bcr_portion_barcode'\n",
       "190. 'patient.samples.sample.portions.portion.bcr_portion_uuid'\n",
       "191. 'patient.samples.sample.portions.portion.day_of_creation'\n",
       "192. 'patient.samples.sample.portions.portion.is_ffpe'\n",
       "193. 'patient.samples.sample.portions.portion.month_of_creation'\n",
       "194. 'patient.samples.sample.portions.portion.portion_number'\n",
       "195. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode'\n",
       "196. 'patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid'\n",
       "197. 'patient.samples.sample.portions.portion.slides.slide.image_file_name'\n",
       "198. 'patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe'\n",
       "199. 'patient.samples.sample.portions.portion.slides.slide.percent_necrosis'\n",
       "200. 'patient.samples.sample.portions.portion.slides.slide.percent_normal_cells'\n",
       "201. 'patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells'\n",
       "202. 'patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells'\n",
       "203. 'patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei'\n",
       "204. 'patient.samples.sample.portions.portion.slides.slide.section_location'\n",
       "205. 'patient.samples.sample.portions.portion.weight'\n",
       "206. 'patient.samples.sample.portions.portion.year_of_creation'\n",
       "207. 'patient.samples.sample.sample_type'\n",
       "208. 'patient.samples.sample.sample_type_id'\n",
       "209. 'patient.samples.sample.vial_number'\n",
       "\n",
       "\n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "$luad\n",
       "  [1] \"admin.bcr\"                                                                                               \n",
       "  [2] \"admin.day_of_dcc_upload\"                                                                                 \n",
       "  [3] \"admin.disease_code\"                                                                                      \n",
       "  [4] \"admin.file_uuid\"                                                                                         \n",
       "  [5] \"admin.month_of_dcc_upload\"                                                                               \n",
       "  [6] \"admin.patient_withdrawal.withdrawn\"                                                                      \n",
       "  [7] \"admin.project_code\"                                                                                      \n",
       "  [8] \"admin.year_of_dcc_upload\"                                                                                \n",
       "  [9] \"patient.bcr_patient_barcode\"                                                                             \n",
       " [10] \"patient.bcr_patient_uuid\"                                                                                \n",
       " [11] \"patient.clinical_cqcf.anatomic_neoplasm_subdivision\"                                                     \n",
       " [12] \"patient.clinical_cqcf.consent_or_death_status\"                                                           \n",
       " [13] \"patient.clinical_cqcf.histological_type\"                                                                 \n",
       " [14] \"patient.clinical_cqcf.history_of_neoadjuvant_treatment\"                                                  \n",
       " [15] \"patient.clinical_cqcf.tumor_type\"                                                                        \n",
       " [16] \"patient.diagnosis\"                                                                                       \n",
       " [17] \"patient.gender\"                                                                                          \n",
       " [18] \"patient.histological_type\"                                                                               \n",
       " [19] \"patient.icd_10\"                                                                                          \n",
       " [20] \"patient.icd_o_3_histology\"                                                                               \n",
       " [21] \"patient.icd_o_3_site\"                                                                                    \n",
       " [22] \"patient.informed_consent_verified\"                                                                       \n",
       " [23] \"patient.month_of_form_completion\"                                                                        \n",
       " [24] \"patient.patient_id\"                                                                                      \n",
       " [25] \"patient.prior_dx\"                                                                                        \n",
       " [26] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_t\"                                   \n",
       " [27] \"patient.tissue_source_site\"                                                                              \n",
       " [28] \"patient.tumor_tissue_site\"                                                                               \n",
       " [29] \"patient.vital_status\"                                                                                    \n",
       " [30] \"patient.year_of_form_completion\"                                                                         \n",
       " [31] \"patient.bcr_canonical_check.bcr_patient_canonical_status\"                                                \n",
       " [32] \"patient.biospecimen_cqcf.consent_or_death_status\"                                                        \n",
       " [33] \"patient.biospecimen_cqcf.digital_image_submitted\"                                                        \n",
       " [34] \"patient.biospecimen_cqcf.frozen_specimen_anatomic_site\"                                                  \n",
       " [35] \"patient.biospecimen_cqcf.histological_type\"                                                              \n",
       " [36] \"patient.biospecimen_cqcf.history_of_neoadjuvant_treatment\"                                               \n",
       " [37] \"patient.biospecimen_cqcf.method_of_sample_procurement\"                                                   \n",
       " [38] \"patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid\"                                 \n",
       " [39] \"patient.biospecimen_cqcf.normal_controls.normal_control.vial_number\"                                     \n",
       " [40] \"patient.biospecimen_cqcf.path_confirm_report_attached\"                                                   \n",
       " [41] \"patient.biospecimen_cqcf.sample_prescreened\"                                                             \n",
       " [42] \"patient.biospecimen_cqcf.site_of_disease\"                                                                \n",
       " [43] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid\"                                     \n",
       " [44] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_nuclei_percent\"                                \n",
       " [45] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number\"                                         \n",
       " [46] \"patient.biospecimen_cqcf.vessel_used\"                                                                    \n",
       " [47] \"patient.samples.sample-2.bcr_sample_barcode\"                                                             \n",
       " [48] \"patient.samples.sample-2.bcr_sample_uuid\"                                                                \n",
       " [49] \"patient.samples.sample-2.is_ffpe\"                                                                        \n",
       " [50] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       " [51] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       " [52] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"  \n",
       " [53] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                   \n",
       " [54] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"               \n",
       " [55] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"             \n",
       " [56] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       " [57] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"           \n",
       " [58] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                \n",
       " [59] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                    \n",
       " [60] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                   \n",
       " [61] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                    \n",
       " [62] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"               \n",
       " [63] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                      \n",
       " [64] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"            \n",
       " [65] \"patient.samples.sample-2.portions.portion.analytes.analyte.amount\"                                       \n",
       " [66] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type\"                                 \n",
       " [67] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id\"                              \n",
       " [68] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode\"                          \n",
       " [69] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid\"                             \n",
       " [70] \"patient.samples.sample-2.portions.portion.analytes.analyte.concentration\"                                \n",
       " [71] \"patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file\"                               \n",
       " [72] \"patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe\"                         \n",
       " [73] \"patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"\n",
       " [74] \"patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method\"                     \n",
       " [75] \"patient.samples.sample-2.portions.portion.bcr_portion_barcode\"                                           \n",
       " [76] \"patient.samples.sample-2.portions.portion.bcr_portion_uuid\"                                              \n",
       " [77] \"patient.samples.sample-2.portions.portion.day_of_creation\"                                               \n",
       " [78] \"patient.samples.sample-2.portions.portion.is_ffpe\"                                                       \n",
       " [79] \"patient.samples.sample-2.portions.portion.month_of_creation\"                                             \n",
       " [80] \"patient.samples.sample-2.portions.portion.portion_number\"                                                \n",
       " [81] \"patient.samples.sample-2.portions.portion.year_of_creation\"                                              \n",
       " [82] \"patient.samples.sample-2.sample_type\"                                                                    \n",
       " [83] \"patient.samples.sample-2.sample_type_id\"                                                                 \n",
       " [84] \"patient.samples.sample-2.vial_number\"                                                                    \n",
       " [85] \"patient.samples.sample.bcr_sample_barcode\"                                                               \n",
       " [86] \"patient.samples.sample.bcr_sample_uuid\"                                                                  \n",
       " [87] \"patient.samples.sample.is_ffpe\"                                                                          \n",
       " [88] \"patient.samples.sample.pathology_report_file_name\"                                                       \n",
       " [89] \"patient.samples.sample.pathology_report_uuid\"                                                            \n",
       " [90] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"           \n",
       " [91] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"              \n",
       " [92] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"    \n",
       " [93] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                     \n",
       " [94] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"                 \n",
       " [95] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"               \n",
       " [96] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"          \n",
       " [97] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"             \n",
       " [98] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                  \n",
       " [99] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                      \n",
       "[100] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                     \n",
       "[101] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                      \n",
       "[102] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"                 \n",
       "[103] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                        \n",
       "[104] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"              \n",
       "[105] \"patient.samples.sample.portions.portion.analytes.analyte.amount\"                                         \n",
       "[106] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type\"                                   \n",
       "[107] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id\"                                \n",
       "[108] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode\"                            \n",
       "[109] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid\"                               \n",
       "[110] \"patient.samples.sample.portions.portion.analytes.analyte.concentration\"                                  \n",
       "[111] \"patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"                \n",
       "[112] \"patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"               \n",
       "[113] \"patient.samples.sample.portions.portion.analytes.analyte.gel_image_file\"                                 \n",
       "[114] \"patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe\"                           \n",
       "[115] \"patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"  \n",
       "[116] \"patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method\"                       \n",
       "[117] \"patient.samples.sample.portions.portion.bcr_portion_barcode\"                                             \n",
       "[118] \"patient.samples.sample.portions.portion.bcr_portion_uuid\"                                                \n",
       "[119] \"patient.samples.sample.portions.portion.day_of_creation\"                                                 \n",
       "[120] \"patient.samples.sample.portions.portion.is_ffpe\"                                                         \n",
       "[121] \"patient.samples.sample.portions.portion.month_of_creation\"                                               \n",
       "[122] \"patient.samples.sample.portions.portion.portion_number\"                                                  \n",
       "[123] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode\"                                  \n",
       "[124] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid\"                                     \n",
       "[125] \"patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe\"                               \n",
       "[126] \"patient.samples.sample.portions.portion.slides.slide.section_location\"                                   \n",
       "[127] \"patient.samples.sample.portions.portion.weight\"                                                          \n",
       "[128] \"patient.samples.sample.portions.portion.year_of_creation\"                                                \n",
       "[129] \"patient.samples.sample.sample_type\"                                                                      \n",
       "[130] \"patient.samples.sample.sample_type_id\"                                                                   \n",
       "[131] \"patient.samples.sample.vial_number\"                                                                      \n",
       "attr(,\"na.action\")\n",
       "  [1]   9  14  17  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34\n",
       " [19]  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52\n",
       " [37]  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70\n",
       " [55]  80  81  82  83  84  85  86  87  88  89  90  91  92  93  94  95  96  97\n",
       " [73]  98  99 106 108 109 111 114 116 117 119 120 122 125 128 129 130 131 133\n",
       " [91] 135 138 141 142 143 144 145 146 148 149 150 151 152 153 154 155 156 157\n",
       "[109] 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175\n",
       "[127] 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192 193\n",
       "[145] 194 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211\n",
       "[163] 212 213 214 215 216 217 218 243 244 246 251 254 255 256 257 258 259 260\n",
       "[181] 261 262 263 264 266 269 270 271 272 273 275 276 277 278 279 280 281 282\n",
       "[199] 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300\n",
       "[217] 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318\n",
       "[235] 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336\n",
       "[253] 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354\n",
       "[271] 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372\n",
       "[289] 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390\n",
       "[307] 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408\n",
       "[325] 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426\n",
       "[343] 427 428 429 430 431 432 433 434 435 436 437 438 439 440 441 442 443 444\n",
       "[361] 445 446 447 448 449 450 451 452 453 454 455 456 457 458 459 460 461 464\n",
       "[379] 465 466 467 468 469 471 472 473 476 477 478 479 480 481 482 483 484 485\n",
       "[397] 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503\n",
       "[415] 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521\n",
       "[433] 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539\n",
       "[451] 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557\n",
       "[469] 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575\n",
       "[487] 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593\n",
       "[505] 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 609 610 637\n",
       "[523] 638 640 647 648 649 650 651 652 656 657 658 659 663 664 667 668 669 670\n",
       "[541] 671\n",
       "attr(,\"class\")\n",
       "[1] \"omit\"\n",
       "\n",
       "$lusc\n",
       " [1] \"admin.bcr\"                                                                                               \n",
       " [2] \"admin.day_of_dcc_upload\"                                                                                 \n",
       " [3] \"admin.disease_code\"                                                                                      \n",
       " [4] \"admin.month_of_dcc_upload\"                                                                               \n",
       " [5] \"admin.patient_withdrawal.withdrawn\"                                                                      \n",
       " [6] \"admin.year_of_dcc_upload\"                                                                                \n",
       " [7] \"patient.bcr_patient_barcode\"                                                                             \n",
       " [8] \"patient.bcr_patient_uuid\"                                                                                \n",
       " [9] \"patient.clinical_cqcf.country\"                                                                           \n",
       "[10] \"patient.clinical_cqcf.tumor_type\"                                                                        \n",
       "[11] \"patient.diagnosis\"                                                                                       \n",
       "[12] \"patient.gender\"                                                                                          \n",
       "[13] \"patient.histological_type\"                                                                               \n",
       "[14] \"patient.history_of_neoadjuvant_treatment\"                                                                \n",
       "[15] \"patient.icd_10\"                                                                                          \n",
       "[16] \"patient.icd_o_3_histology\"                                                                               \n",
       "[17] \"patient.icd_o_3_site\"                                                                                    \n",
       "[18] \"patient.informed_consent_verified\"                                                                       \n",
       "[19] \"patient.month_of_form_completion\"                                                                        \n",
       "[20] \"patient.patient_id\"                                                                                      \n",
       "[21] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_n\"                                   \n",
       "[22] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_t\"                                   \n",
       "[23] \"patient.tissue_source_site\"                                                                              \n",
       "[24] \"patient.tumor_tissue_site\"                                                                               \n",
       "[25] \"patient.vital_status\"                                                                                    \n",
       "[26] \"patient.year_of_form_completion\"                                                                         \n",
       "[27] \"patient.bcr_canonical_check.bcr_patient_canonical_status\"                                                \n",
       "[28] \"patient.samples.sample-2.bcr_sample_barcode\"                                                             \n",
       "[29] \"patient.samples.sample-2.bcr_sample_uuid\"                                                                \n",
       "[30] \"patient.samples.sample-2.is_ffpe\"                                                                        \n",
       "[31] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       "[32] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       "[33] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                   \n",
       "[34] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"               \n",
       "[35] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"             \n",
       "[36] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       "[37] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"           \n",
       "[38] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                \n",
       "[39] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                    \n",
       "[40] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                   \n",
       "[41] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                    \n",
       "[42] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"               \n",
       "[43] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                      \n",
       "[44] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"            \n",
       "[45] \"patient.samples.sample-2.portions.portion.analytes.analyte.amount\"                                       \n",
       "[46] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type\"                                 \n",
       "[47] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id\"                              \n",
       "[48] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode\"                          \n",
       "[49] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid\"                             \n",
       "[50] \"patient.samples.sample-2.portions.portion.analytes.analyte.concentration\"                                \n",
       "[51] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"              \n",
       "[52] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"             \n",
       "[53] \"patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file\"                               \n",
       "[54] \"patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe\"                         \n",
       "[55] \"patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"\n",
       "[56] \"patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method\"                     \n",
       "[57] \"patient.samples.sample-2.portions.portion.bcr_portion_barcode\"                                           \n",
       "[58] \"patient.samples.sample-2.portions.portion.bcr_portion_uuid\"                                              \n",
       "[59] \"patient.samples.sample-2.portions.portion.day_of_creation\"                                               \n",
       "[60] \"patient.samples.sample-2.portions.portion.is_ffpe\"                                                       \n",
       "[61] \"patient.samples.sample-2.portions.portion.month_of_creation\"                                             \n",
       "[62] \"patient.samples.sample-2.portions.portion.portion_number\"                                                \n",
       "[63] \"patient.samples.sample-2.portions.portion.year_of_creation\"                                              \n",
       "[64] \"patient.samples.sample-2.sample_type\"                                                                    \n",
       "[65] \"patient.samples.sample-2.sample_type_id\"                                                                 \n",
       "[66] \"patient.samples.sample-2.vial_number\"                                                                    \n",
       "[67] \"patient.samples.sample.bcr_sample_barcode\"                                                               \n",
       "[68] \"patient.samples.sample.bcr_sample_uuid\"                                                                  \n",
       "[69] \"patient.samples.sample.is_ffpe\"                                                                          \n",
       "[70] \"patient.samples.sample.pathology_report_file_name\"                                                       \n",
       "[71] \"patient.samples.sample.pathology_report_uuid\"                                                            \n",
       "[72] \"patient.samples.sample.sample_type\"                                                                      \n",
       "[73] \"patient.samples.sample.sample_type_id\"                                                                   \n",
       "[74] \"patient.samples.sample.vial_number\"                                                                      \n",
       "attr(,\"na.action\")\n",
       "  [1]   7  11  12  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29\n",
       " [19]  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47\n",
       " [37]  48  49  50  51  52  53  54  55  56  57  58  59  60  61  62  72  73  74\n",
       " [55]  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89  97  98  99\n",
       " [73] 100 101 102 103 104 105 106 107 108 109 110 111 112 113 114 115 116 119\n",
       " [91] 120 121 122 123 124 126 127 128 129 130 131 132 133 134 135 136 137 138\n",
       "[109] 139 140 141 142 143 144 145 146 147 148 149 150 151 152 153 154 155 156\n",
       "[127] 157 158 159 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174\n",
       "[145] 175 176 177 203 204 206 211 214 215 216 217 218 219 220 221 222 223 224\n",
       "[163] 225 226 227 228 229 230 231 232 234 237 238 239 240 241 243 244 245 246\n",
       "[181] 247 248 249 250 251 252 253 254 255 256 257 258 259 260 261 262 263 264\n",
       "[199] 265 266 267 268 269 270 271 272 273 274 275 276 277 278 279 280 281 282\n",
       "[217] 283 284 285 286 287 288 289 290 291 292 293 294 295 296 297 298 299 300\n",
       "[235] 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318\n",
       "[253] 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336\n",
       "[271] 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354\n",
       "[289] 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372\n",
       "[307] 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390\n",
       "[325] 391 392 393 394 395 396 397 398 399 400 401 402 403 404 405 406 407 408\n",
       "[343] 409 410 411 412 413 414 415 416 417 418 419 420 421 422 423 424 425 426\n",
       "[361] 427 428 429 430 431 432 433 434 435 436 437 438 439 442 443 444 445 446\n",
       "[379] 447 449 450 451 454 455 456 457 458 459 460 461 462 463 464 465 466 467\n",
       "[397] 468 469 470 471 472 473 474 475 476 477 478 479 480 481 482 483 484 485\n",
       "[415] 486 487 488 489 490 491 492 493 494 495 496 497 498 499 500 501 502 503\n",
       "[433] 504 505 506 507 508 509 510 511 512 513 514 515 516 517 518 519 520 521\n",
       "[451] 522 523 524 525 526 527 528 529 530 531 532 533 534 535 536 537 538 539\n",
       "[469] 540 541 542 543 544 545 546 547 548 549 550 551 552 553 554 555 556 557\n",
       "[487] 558 559 560 561 562 563 564 565 566 567 568 569 570 571 572 573 574 575\n",
       "[505] 576 577 578 579 580 581 582 583 584 585 586 587 588 589 590 591 592 593\n",
       "[523] 594 595 596 597 598 599 600 601 602 603 604 605 606 607 608 611 612 613\n",
       "[541] 614 615\n",
       "attr(,\"class\")\n",
       "[1] \"omit\"\n",
       "\n",
       "$meso\n",
       "  [1] \"admin.bcr\"                                                                                               \n",
       "  [2] \"admin.day_of_dcc_upload\"                                                                                 \n",
       "  [3] \"admin.disease_code\"                                                                                      \n",
       "  [4] \"admin.file_uuid\"                                                                                         \n",
       "  [5] \"admin.month_of_dcc_upload\"                                                                               \n",
       "  [6] \"admin.patient_withdrawal.withdrawn\"                                                                      \n",
       "  [7] \"admin.project_code\"                                                                                      \n",
       "  [8] \"admin.year_of_dcc_upload\"                                                                                \n",
       "  [9] \"patient.bcr_patient_barcode\"                                                                             \n",
       " [10] \"patient.bcr_patient_uuid\"                                                                                \n",
       " [11] \"patient.clinical_cqcf.consent_or_death_status\"                                                           \n",
       " [12] \"patient.clinical_cqcf.country\"                                                                           \n",
       " [13] \"patient.clinical_cqcf.histological_type\"                                                                 \n",
       " [14] \"patient.clinical_cqcf.history_of_neoadjuvant_treatment\"                                                  \n",
       " [15] \"patient.clinical_cqcf.tumor_type\"                                                                        \n",
       " [16] \"patient.day_of_form_completion\"                                                                          \n",
       " [17] \"patient.days_to_birth\"                                                                                   \n",
       " [18] \"patient.gender\"                                                                                          \n",
       " [19] \"patient.history_of_neoadjuvant_treatment\"                                                                \n",
       " [20] \"patient.icd_10\"                                                                                          \n",
       " [21] \"patient.icd_o_3_histology\"                                                                               \n",
       " [22] \"patient.icd_o_3_site\"                                                                                    \n",
       " [23] \"patient.informed_consent_verified\"                                                                       \n",
       " [24] \"patient.month_of_form_completion\"                                                                        \n",
       " [25] \"patient.patient_id\"                                                                                      \n",
       " [26] \"patient.primary_pathology.age_at_initial_pathologic_diagnosis\"                                           \n",
       " [27] \"patient.primary_pathology.days_to_initial_pathologic_diagnosis\"                                          \n",
       " [28] \"patient.primary_pathology.histological_type\"                                                             \n",
       " [29] \"patient.primary_pathology.laterality\"                                                                    \n",
       " [30] \"patient.primary_pathology.tumor_tissue_site\"                                                             \n",
       " [31] \"patient.primary_pathology.year_of_initial_pathologic_diagnosis\"                                          \n",
       " [32] \"patient.prior_dx\"                                                                                        \n",
       " [33] \"patient.race\"                                                                                            \n",
       " [34] \"patient.stage_event.pathologic_stage\"                                                                    \n",
       " [35] \"patient.stage_event.system_version\"                                                                      \n",
       " [36] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_m\"                                   \n",
       " [37] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_n\"                                   \n",
       " [38] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_t\"                                   \n",
       " [39] \"patient.tissue_prospective_collection_indicator\"                                                         \n",
       " [40] \"patient.tissue_retrospective_collection_indicator\"                                                       \n",
       " [41] \"patient.tissue_source_site\"                                                                              \n",
       " [42] \"patient.vital_status\"                                                                                    \n",
       " [43] \"patient.year_of_form_completion\"                                                                         \n",
       " [44] \"patient.bcr_canonical_check.bcr_patient_canonical_status\"                                                \n",
       " [45] \"patient.biospecimen_cqcf.consent_or_death_status\"                                                        \n",
       " [46] \"patient.biospecimen_cqcf.country\"                                                                        \n",
       " [47] \"patient.biospecimen_cqcf.days_to_sample_procurement\"                                                     \n",
       " [48] \"patient.biospecimen_cqcf.frozen_specimen_anatomic_site\"                                                  \n",
       " [49] \"patient.biospecimen_cqcf.histological_type\"                                                              \n",
       " [50] \"patient.biospecimen_cqcf.history_of_neoadjuvant_treatment\"                                               \n",
       " [51] \"patient.biospecimen_cqcf.method_of_sample_procurement\"                                                   \n",
       " [52] \"patient.biospecimen_cqcf.normal_controls.normal_control.bcr_sample_uuid\"                                 \n",
       " [53] \"patient.biospecimen_cqcf.normal_controls.normal_control.days_to_normal_sample_procurement\"               \n",
       " [54] \"patient.biospecimen_cqcf.normal_controls.normal_control.method_of_normal_sample_procurement\"             \n",
       " [55] \"patient.biospecimen_cqcf.normal_controls.normal_control.vial_number\"                                     \n",
       " [56] \"patient.biospecimen_cqcf.path_confirm_report_attached\"                                                   \n",
       " [57] \"patient.biospecimen_cqcf.sample_prescreened\"                                                             \n",
       " [58] \"patient.biospecimen_cqcf.site_of_disease\"                                                                \n",
       " [59] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.bcr_sample_uuid\"                                     \n",
       " [60] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.tumor_weight\"                                        \n",
       " [61] \"patient.biospecimen_cqcf.tumor_samples.tumor_sample.vial_number\"                                         \n",
       " [62] \"patient.biospecimen_cqcf.vessel_used\"                                                                    \n",
       " [63] \"patient.days_to_index\"                                                                                   \n",
       " [64] \"patient.samples.sample-2.bcr_sample_barcode\"                                                             \n",
       " [65] \"patient.samples.sample-2.bcr_sample_uuid\"                                                                \n",
       " [66] \"patient.samples.sample-2.days_to_collection\"                                                             \n",
       " [67] \"patient.samples.sample-2.is_ffpe\"                                                                        \n",
       " [68] \"patient.samples.sample-2.oct_embedded\"                                                                   \n",
       " [69] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode\"       \n",
       " [70] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid\"          \n",
       " [71] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom\"\n",
       " [72] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id\"                 \n",
       " [73] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration\"             \n",
       " [74] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment\"           \n",
       " [75] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe\"      \n",
       " [76] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment\"         \n",
       " [77] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column\"              \n",
       " [78] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id\"                  \n",
       " [79] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row\"                 \n",
       " [80] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity\"                  \n",
       " [81] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center\"             \n",
       " [82] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume\"                    \n",
       " [83] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment\"          \n",
       " [84] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       " [85] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       " [86] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"  \n",
       " [87] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                   \n",
       " [88] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"               \n",
       " [89] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"             \n",
       " [90] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       " [91] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"           \n",
       " [92] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                \n",
       " [93] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                    \n",
       " [94] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                   \n",
       " [95] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                    \n",
       " [96] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"               \n",
       " [97] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                      \n",
       " [98] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"            \n",
       " [99] \"patient.samples.sample-2.portions.portion.analytes.analyte.amount\"                                       \n",
       "[100] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type\"                                 \n",
       "[101] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id\"                              \n",
       "[102] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode\"                          \n",
       "[103] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid\"                             \n",
       "[104] \"patient.samples.sample-2.portions.portion.analytes.analyte.concentration\"                                \n",
       "[105] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"              \n",
       "[106] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"             \n",
       "[107] \"patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file\"                               \n",
       "[108] \"patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe\"                         \n",
       "[109] \"patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"\n",
       "[110] \"patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method\"                     \n",
       "[111] \"patient.samples.sample-2.portions.portion.bcr_portion_barcode\"                                           \n",
       "[112] \"patient.samples.sample-2.portions.portion.bcr_portion_uuid\"                                              \n",
       "[113] \"patient.samples.sample-2.portions.portion.day_of_creation\"                                               \n",
       "[114] \"patient.samples.sample-2.portions.portion.is_ffpe\"                                                       \n",
       "[115] \"patient.samples.sample-2.portions.portion.month_of_creation\"                                             \n",
       "[116] \"patient.samples.sample-2.portions.portion.portion_number\"                                                \n",
       "[117] \"patient.samples.sample-2.portions.portion.year_of_creation\"                                              \n",
       "[118] \"patient.samples.sample-2.sample_type\"                                                                    \n",
       "[119] \"patient.samples.sample-2.sample_type_id\"                                                                 \n",
       "[120] \"patient.samples.sample-2.vial_number\"                                                                    \n",
       "[121] \"patient.samples.sample.bcr_sample_barcode\"                                                               \n",
       "[122] \"patient.samples.sample.bcr_sample_uuid\"                                                                  \n",
       "[123] \"patient.samples.sample.days_to_collection\"                                                               \n",
       "[124] \"patient.samples.sample.initial_weight\"                                                                   \n",
       "[125] \"patient.samples.sample.is_ffpe\"                                                                          \n",
       "[126] \"patient.samples.sample.oct_embedded\"                                                                     \n",
       "[127] \"patient.samples.sample.pathology_report_file_name\"                                                       \n",
       "[128] \"patient.samples.sample.pathology_report_uuid\"                                                            \n",
       "[129] \"patient.samples.sample.portions.portion.analytes.analyte-2.a260_a280_ratio\"                              \n",
       "[130] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_barcode\"       \n",
       "[131] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.bcr_aliquot_uuid\"          \n",
       "[132] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.biospecimen_barcode_bottom\"\n",
       "[133] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.center_id\"                 \n",
       "[134] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.concentration\"             \n",
       "[135] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.day_of_shipment\"           \n",
       "[136] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.is_derived_from_ffpe\"      \n",
       "[137] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.month_of_shipment\"         \n",
       "[138] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_column\"              \n",
       "[139] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_id\"                  \n",
       "[140] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.plate_row\"                 \n",
       "[141] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.quantity\"                  \n",
       "[142] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.source_center\"             \n",
       "[143] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.volume\"                    \n",
       "[144] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot-2.year_of_shipment\"          \n",
       "[145] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       "[146] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       "[147] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom\"  \n",
       "[148] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id\"                   \n",
       "[149] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration\"               \n",
       "[150] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment\"             \n",
       "[151] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       "[152] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment\"           \n",
       "[153] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column\"                \n",
       "[154] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id\"                    \n",
       "[155] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row\"                   \n",
       "[156] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity\"                    \n",
       "[157] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center\"               \n",
       "[158] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume\"                      \n",
       "[159] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment\"            \n",
       "[160] \"patient.samples.sample.portions.portion.analytes.analyte-2.amount\"                                       \n",
       "[161] \"patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type\"                                 \n",
       "[162] \"patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id\"                              \n",
       "[163] \"patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode\"                          \n",
       "[164] \"patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid\"                             \n",
       "[165] \"patient.samples.sample.portions.portion.analytes.analyte-2.concentration\"                                \n",
       "[166] \"patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file\"                               \n",
       "[167] \"patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe\"                         \n",
       "[168] \"patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type\"\n",
       "[169] \"patient.samples.sample.portions.portion.analytes.analyte-2.rna.ratio_28s_18s\"                            \n",
       "[170] \"patient.samples.sample.portions.portion.analytes.analyte-2.rna.rinvalue\"                                 \n",
       "[171] \"patient.samples.sample.portions.portion.analytes.analyte-2.spectrophotometer_method\"                     \n",
       "[172] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode\"         \n",
       "[173] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid\"            \n",
       "[174] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom\"  \n",
       "[175] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id\"                   \n",
       "[176] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration\"               \n",
       "[177] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment\"             \n",
       "[178] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe\"        \n",
       "[179] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment\"           \n",
       "[180] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column\"                \n",
       "[181] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id\"                    \n",
       "[182] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row\"                   \n",
       "[183] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity\"                    \n",
       "[184] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center\"               \n",
       "[185] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume\"                      \n",
       "[186] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment\"            \n",
       "[187] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode\"         \n",
       "[188] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid\"            \n",
       "[189] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom\"  \n",
       "[190] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id\"                   \n",
       "[191] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration\"               \n",
       "[192] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment\"             \n",
       "[193] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe\"        \n",
       "[194] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment\"           \n",
       "[195] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column\"                \n",
       "[196] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id\"                    \n",
       "[197] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row\"                   \n",
       "[198] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity\"                    \n",
       "[199] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center\"               \n",
       "[200] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume\"                      \n",
       "[201] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment\"            \n",
       "[202] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"           \n",
       "[203] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"              \n",
       "[204] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"    \n",
       "[205] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                     \n",
       "[206] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"                 \n",
       "[207] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"               \n",
       "[208] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"          \n",
       "[209] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"             \n",
       "[210] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                  \n",
       "[211] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                      \n",
       "[212] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                     \n",
       "[213] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                      \n",
       "[214] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"                 \n",
       "[215] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                        \n",
       "[216] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"              \n",
       "[217] \"patient.samples.sample.portions.portion.analytes.analyte.amount\"                                         \n",
       "[218] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type\"                                   \n",
       "[219] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id\"                                \n",
       "[220] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode\"                            \n",
       "[221] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid\"                               \n",
       "[222] \"patient.samples.sample.portions.portion.analytes.analyte.concentration\"                                  \n",
       "[223] \"patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"                \n",
       "[224] \"patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"               \n",
       "[225] \"patient.samples.sample.portions.portion.analytes.analyte.gel_image_file\"                                 \n",
       "[226] \"patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe\"                           \n",
       "[227] \"patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"  \n",
       "[228] \"patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method\"                       \n",
       "[229] \"patient.samples.sample.portions.portion.bcr_portion_barcode\"                                             \n",
       "[230] \"patient.samples.sample.portions.portion.bcr_portion_uuid\"                                                \n",
       "[231] \"patient.samples.sample.portions.portion.day_of_creation\"                                                 \n",
       "[232] \"patient.samples.sample.portions.portion.is_ffpe\"                                                         \n",
       "[233] \"patient.samples.sample.portions.portion.month_of_creation\"                                               \n",
       "[234] \"patient.samples.sample.portions.portion.portion_number\"                                                  \n",
       "[235] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode\"                                  \n",
       "[236] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid\"                                     \n",
       "[237] \"patient.samples.sample.portions.portion.slides.slide.image_file_name\"                                    \n",
       "[238] \"patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe\"                               \n",
       "[239] \"patient.samples.sample.portions.portion.slides.slide.percent_lymphocyte_infiltration\"                    \n",
       "[240] \"patient.samples.sample.portions.portion.slides.slide.percent_monocyte_infiltration\"                      \n",
       "[241] \"patient.samples.sample.portions.portion.slides.slide.percent_necrosis\"                                   \n",
       "[242] \"patient.samples.sample.portions.portion.slides.slide.percent_neutrophil_infiltration\"                    \n",
       "[243] \"patient.samples.sample.portions.portion.slides.slide.percent_normal_cells\"                               \n",
       "[244] \"patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells\"                              \n",
       "[245] \"patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells\"                                \n",
       "[246] \"patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei\"                               \n",
       "[247] \"patient.samples.sample.portions.portion.slides.slide.section_location\"                                   \n",
       "[248] \"patient.samples.sample.portions.portion.weight\"                                                          \n",
       "[249] \"patient.samples.sample.portions.portion.year_of_creation\"                                                \n",
       "[250] \"patient.samples.sample.sample_type\"                                                                      \n",
       "[251] \"patient.samples.sample.sample_type_id\"                                                                   \n",
       "[252] \"patient.samples.sample.vial_number\"                                                                      \n",
       "attr(,\"na.action\")\n",
       "  [1]   9  12  15  18  19  20  21  25  26  27  28  29  30  31  32  33  34  35\n",
       " [19]  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53\n",
       " [37]  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69  70  71\n",
       " [55]  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87  88  89\n",
       " [73]  90  91  92  93  94  95  96  97  98  99 100 101 102 103 104 105 106 107\n",
       " [91] 108 109 110 111 112 113 114 115 116 117 118 119 120 121 122 123 124 125\n",
       "[109] 126 127 135 136 137 138 139 140 141 142 143 144 154 155 156 157 158 159\n",
       "[127] 160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175 177 178\n",
       "[145] 179 181 182 183 193 196 198 199 200 203 205 206 207 209 213 216 217 220\n",
       "[163] 221 222 223 224 225 226 228 229 234 237 238 239 241 242 243 244 246 247\n",
       "[181] 248 250 251 252 253 254 255 256 257 273 274 275 276 303 304 306 311 314\n",
       "[199] 315 316 317 318 319 321 324 325 326 327 328 329 331 334 335 336 338 339\n",
       "[217] 340 342 344 345 346 348 351 353 354 355 371 372 397 398 402 403 404 405\n",
       "[235] 406 422 423 439 440 441 442 469 470 472 477 480 485 486 487 488 500 501\n",
       "[253] 504 505 506 507 508 509\n",
       "attr(,\"class\")\n",
       "[1] \"omit\"\n",
       "\n",
       "$lihc\n",
       "  [1] \"admin.bcr\"                                                                                               \n",
       "  [2] \"admin.day_of_dcc_upload\"                                                                                 \n",
       "  [3] \"admin.disease_code\"                                                                                      \n",
       "  [4] \"admin.file_uuid\"                                                                                         \n",
       "  [5] \"admin.month_of_dcc_upload\"                                                                               \n",
       "  [6] \"admin.patient_withdrawal.withdrawn\"                                                                      \n",
       "  [7] \"admin.project_code\"                                                                                      \n",
       "  [8] \"admin.year_of_dcc_upload\"                                                                                \n",
       "  [9] \"patient.bcr_patient_barcode\"                                                                             \n",
       " [10] \"patient.bcr_patient_uuid\"                                                                                \n",
       " [11] \"patient.clinical_cqcf.consent_or_death_status\"                                                           \n",
       " [12] \"patient.clinical_cqcf.histological_type\"                                                                 \n",
       " [13] \"patient.clinical_cqcf.history_of_neoadjuvant_treatment\"                                                  \n",
       " [14] \"patient.clinical_cqcf.tumor_type\"                                                                        \n",
       " [15] \"patient.day_of_form_completion\"                                                                          \n",
       " [16] \"patient.gender\"                                                                                          \n",
       " [17] \"patient.histological_type\"                                                                               \n",
       " [18] \"patient.history_of_neoadjuvant_treatment\"                                                                \n",
       " [19] \"patient.icd_10\"                                                                                          \n",
       " [20] \"patient.icd_o_3_histology\"                                                                               \n",
       " [21] \"patient.icd_o_3_site\"                                                                                    \n",
       " [22] \"patient.informed_consent_verified\"                                                                       \n",
       " [23] \"patient.month_of_form_completion\"                                                                        \n",
       " [24] \"patient.patient_id\"                                                                                      \n",
       " [25] \"patient.prior_dx\"                                                                                        \n",
       " [26] \"patient.stage_event.system_version\"                                                                      \n",
       " [27] \"patient.stage_event.tnm_categories.pathologic_categories.pathologic_m\"                                   \n",
       " [28] \"patient.tissue_prospective_collection_indicator\"                                                         \n",
       " [29] \"patient.tissue_retrospective_collection_indicator\"                                                       \n",
       " [30] \"patient.tissue_source_site\"                                                                              \n",
       " [31] \"patient.tumor_tissue_site\"                                                                               \n",
       " [32] \"patient.vital_status\"                                                                                    \n",
       " [33] \"patient.year_of_form_completion\"                                                                         \n",
       " [34] \"patient.bcr_canonical_check.bcr_patient_canonical_status\"                                                \n",
       " [35] \"patient.biospecimen_cqcf.consent_or_death_status\"                                                        \n",
       " [36] \"patient.biospecimen_cqcf.frozen_specimen_anatomic_site\"                                                  \n",
       " [37] \"patient.biospecimen_cqcf.histological_type\"                                                              \n",
       " [38] \"patient.biospecimen_cqcf.history_of_neoadjuvant_treatment\"                                               \n",
       " [39] \"patient.biospecimen_cqcf.method_of_sample_procurement\"                                                   \n",
       " [40] \"patient.biospecimen_cqcf.path_confirm_diagnosis_matching\"                                                \n",
       " [41] \"patient.biospecimen_cqcf.path_confirm_report_attached\"                                                   \n",
       " [42] \"patient.biospecimen_cqcf.path_confirm_tumor_necrosis_metrics\"                                            \n",
       " [43] \"patient.biospecimen_cqcf.path_confirm_tumor_nuclei_metrics\"                                              \n",
       " [44] \"patient.biospecimen_cqcf.sample_prescreened\"                                                             \n",
       " [45] \"patient.biospecimen_cqcf.site_of_disease\"                                                                \n",
       " [46] \"patient.biospecimen_cqcf.vessel_used\"                                                                    \n",
       " [47] \"patient.samples.sample-2.bcr_sample_barcode\"                                                             \n",
       " [48] \"patient.samples.sample-2.bcr_sample_uuid\"                                                                \n",
       " [49] \"patient.samples.sample-2.is_ffpe\"                                                                        \n",
       " [50] \"patient.samples.sample-2.oct_embedded\"                                                                   \n",
       " [51] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode\"       \n",
       " [52] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid\"          \n",
       " [53] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom\"\n",
       " [54] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id\"                 \n",
       " [55] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration\"             \n",
       " [56] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment\"           \n",
       " [57] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe\"      \n",
       " [58] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment\"         \n",
       " [59] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column\"              \n",
       " [60] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id\"                  \n",
       " [61] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row\"                 \n",
       " [62] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity\"                  \n",
       " [63] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center\"             \n",
       " [64] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.volume\"                    \n",
       " [65] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment\"          \n",
       " [66] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       " [67] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       " [68] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"  \n",
       " [69] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                   \n",
       " [70] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"               \n",
       " [71] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"             \n",
       " [72] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       " [73] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"           \n",
       " [74] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                \n",
       " [75] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                    \n",
       " [76] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                   \n",
       " [77] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                    \n",
       " [78] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"               \n",
       " [79] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                      \n",
       " [80] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"            \n",
       " [81] \"patient.samples.sample-2.portions.portion.analytes.analyte.amount\"                                       \n",
       " [82] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type\"                                 \n",
       " [83] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id\"                              \n",
       " [84] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode\"                          \n",
       " [85] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid\"                             \n",
       " [86] \"patient.samples.sample-2.portions.portion.analytes.analyte.concentration\"                                \n",
       " [87] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"              \n",
       " [88] \"patient.samples.sample-2.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"             \n",
       " [89] \"patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file\"                               \n",
       " [90] \"patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe\"                         \n",
       " [91] \"patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"\n",
       " [92] \"patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method\"                     \n",
       " [93] \"patient.samples.sample-2.portions.portion.bcr_portion_barcode\"                                           \n",
       " [94] \"patient.samples.sample-2.portions.portion.bcr_portion_uuid\"                                              \n",
       " [95] \"patient.samples.sample-2.portions.portion.day_of_creation\"                                               \n",
       " [96] \"patient.samples.sample-2.portions.portion.is_ffpe\"                                                       \n",
       " [97] \"patient.samples.sample-2.portions.portion.month_of_creation\"                                             \n",
       " [98] \"patient.samples.sample-2.portions.portion.portion_number\"                                                \n",
       " [99] \"patient.samples.sample-2.portions.portion.year_of_creation\"                                              \n",
       "[100] \"patient.samples.sample-2.sample_type\"                                                                    \n",
       "[101] \"patient.samples.sample-2.sample_type_id\"                                                                 \n",
       "[102] \"patient.samples.sample-2.vial_number\"                                                                    \n",
       "[103] \"patient.samples.sample.bcr_sample_barcode\"                                                               \n",
       "[104] \"patient.samples.sample.bcr_sample_uuid\"                                                                  \n",
       "[105] \"patient.samples.sample.initial_weight\"                                                                   \n",
       "[106] \"patient.samples.sample.is_ffpe\"                                                                          \n",
       "[107] \"patient.samples.sample.oct_embedded\"                                                                     \n",
       "[108] \"patient.samples.sample.pathology_report_file_name\"                                                       \n",
       "[109] \"patient.samples.sample.pathology_report_uuid\"                                                            \n",
       "[110] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       "[111] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       "[112] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.biospecimen_barcode_bottom\"  \n",
       "[113] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.center_id\"                   \n",
       "[114] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.concentration\"               \n",
       "[115] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.day_of_shipment\"             \n",
       "[116] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       "[117] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.month_of_shipment\"           \n",
       "[118] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_column\"                \n",
       "[119] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_id\"                    \n",
       "[120] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.plate_row\"                   \n",
       "[121] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.quantity\"                    \n",
       "[122] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.source_center\"               \n",
       "[123] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.volume\"                      \n",
       "[124] \"patient.samples.sample.portions.portion.analytes.analyte-2.aliquots.aliquot.year_of_shipment\"            \n",
       "[125] \"patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type\"                                 \n",
       "[126] \"patient.samples.sample.portions.portion.analytes.analyte-2.analyte_type_id\"                              \n",
       "[127] \"patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_barcode\"                          \n",
       "[128] \"patient.samples.sample.portions.portion.analytes.analyte-2.bcr_analyte_uuid\"                             \n",
       "[129] \"patient.samples.sample.portions.portion.analytes.analyte-2.gel_image_file\"                               \n",
       "[130] \"patient.samples.sample.portions.portion.analytes.analyte-2.is_derived_from_ffpe\"                         \n",
       "[131] \"patient.samples.sample.portions.portion.analytes.analyte-2.protocols.protocol.experimental_protocol_type\"\n",
       "[132] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_barcode\"         \n",
       "[133] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.bcr_aliquot_uuid\"            \n",
       "[134] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.biospecimen_barcode_bottom\"  \n",
       "[135] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.center_id\"                   \n",
       "[136] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.concentration\"               \n",
       "[137] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.day_of_shipment\"             \n",
       "[138] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.is_derived_from_ffpe\"        \n",
       "[139] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.month_of_shipment\"           \n",
       "[140] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_column\"                \n",
       "[141] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_id\"                    \n",
       "[142] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.plate_row\"                   \n",
       "[143] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.quantity\"                    \n",
       "[144] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.source_center\"               \n",
       "[145] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.volume\"                      \n",
       "[146] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-2.year_of_shipment\"            \n",
       "[147] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_barcode\"         \n",
       "[148] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.bcr_aliquot_uuid\"            \n",
       "[149] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.biospecimen_barcode_bottom\"  \n",
       "[150] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.center_id\"                   \n",
       "[151] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.concentration\"               \n",
       "[152] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.day_of_shipment\"             \n",
       "[153] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.is_derived_from_ffpe\"        \n",
       "[154] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.month_of_shipment\"           \n",
       "[155] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_column\"                \n",
       "[156] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_id\"                    \n",
       "[157] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.plate_row\"                   \n",
       "[158] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.quantity\"                    \n",
       "[159] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.source_center\"               \n",
       "[160] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.volume\"                      \n",
       "[161] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot-3.year_of_shipment\"            \n",
       "[162] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"           \n",
       "[163] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"              \n",
       "[164] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.biospecimen_barcode_bottom\"    \n",
       "[165] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                     \n",
       "[166] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"                 \n",
       "[167] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"               \n",
       "[168] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"          \n",
       "[169] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"             \n",
       "[170] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                  \n",
       "[171] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                      \n",
       "[172] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                     \n",
       "[173] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                      \n",
       "[174] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"                 \n",
       "[175] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                        \n",
       "[176] \"patient.samples.sample.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"              \n",
       "[177] \"patient.samples.sample.portions.portion.analytes.analyte.amount\"                                         \n",
       "[178] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type\"                                   \n",
       "[179] \"patient.samples.sample.portions.portion.analytes.analyte.analyte_type_id\"                                \n",
       "[180] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_barcode\"                            \n",
       "[181] \"patient.samples.sample.portions.portion.analytes.analyte.bcr_analyte_uuid\"                               \n",
       "[182] \"patient.samples.sample.portions.portion.analytes.analyte.concentration\"                                  \n",
       "[183] \"patient.samples.sample.portions.portion.analytes.analyte.dna.normal_tumor_genotype_match\"                \n",
       "[184] \"patient.samples.sample.portions.portion.analytes.analyte.dna.pcr_amplification_successful\"               \n",
       "[185] \"patient.samples.sample.portions.portion.analytes.analyte.gel_image_file\"                                 \n",
       "[186] \"patient.samples.sample.portions.portion.analytes.analyte.is_derived_from_ffpe\"                           \n",
       "[187] \"patient.samples.sample.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"  \n",
       "[188] \"patient.samples.sample.portions.portion.analytes.analyte.spectrophotometer_method\"                       \n",
       "[189] \"patient.samples.sample.portions.portion.bcr_portion_barcode\"                                             \n",
       "[190] \"patient.samples.sample.portions.portion.bcr_portion_uuid\"                                                \n",
       "[191] \"patient.samples.sample.portions.portion.day_of_creation\"                                                 \n",
       "[192] \"patient.samples.sample.portions.portion.is_ffpe\"                                                         \n",
       "[193] \"patient.samples.sample.portions.portion.month_of_creation\"                                               \n",
       "[194] \"patient.samples.sample.portions.portion.portion_number\"                                                  \n",
       "[195] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_barcode\"                                  \n",
       "[196] \"patient.samples.sample.portions.portion.slides.slide.bcr_slide_uuid\"                                     \n",
       "[197] \"patient.samples.sample.portions.portion.slides.slide.image_file_name\"                                    \n",
       "[198] \"patient.samples.sample.portions.portion.slides.slide.is_derived_from_ffpe\"                               \n",
       "[199] \"patient.samples.sample.portions.portion.slides.slide.percent_necrosis\"                                   \n",
       "[200] \"patient.samples.sample.portions.portion.slides.slide.percent_normal_cells\"                               \n",
       "[201] \"patient.samples.sample.portions.portion.slides.slide.percent_stromal_cells\"                              \n",
       "[202] \"patient.samples.sample.portions.portion.slides.slide.percent_tumor_cells\"                                \n",
       "[203] \"patient.samples.sample.portions.portion.slides.slide.percent_tumor_nuclei\"                               \n",
       "[204] \"patient.samples.sample.portions.portion.slides.slide.section_location\"                                   \n",
       "[205] \"patient.samples.sample.portions.portion.weight\"                                                          \n",
       "[206] \"patient.samples.sample.portions.portion.year_of_creation\"                                                \n",
       "[207] \"patient.samples.sample.sample_type\"                                                                      \n",
       "[208] \"patient.samples.sample.sample_type_id\"                                                                   \n",
       "[209] \"patient.samples.sample.vial_number\"                                                                      \n",
       "attr(,\"na.action\")\n",
       "  [1]   9  12  14  17  18  19  20  23  24  25  26  27  28  29  30  31  32  33\n",
       " [19]  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51\n",
       " [37]  52  53  54  55  56  57  58  59  60  61  62  63  64  65  66  67  68  69\n",
       " [55]  70  71  72  73  74  75  76  77  78  79  80  81  82  83  84  85  86  87\n",
       " [73]  88  89  90  99 100 101 102 103 104 105 108 109 110 111 112 113 114 115\n",
       " [91] 116 117 118 119 120 121 122 123 124 126 127 128 137 139 140 141 144 146\n",
       "[109] 147 148 150 151 152 153 154 155 156 161 164 165 166 167 168 169 170 171\n",
       "[127] 172 173 174 176 179 180 181 182 183 184 185 187 188 189 191 192 193 194\n",
       "[145] 195 196 197 198 199 200 201 202 203 204 205 206 207 208 209 210 211 212\n",
       "[163] 213 229 230 231 232 233 234 235 236 237 238 265 266 268 273 276 277 278\n",
       "[181] 279 280 281 283 286 287 288 289 290 291 293 294 295 296 297 298 299 300\n",
       "[199] 301 302 303 304 305 306 307 308 309 310 311 312 313 314 315 316 317 318\n",
       "[217] 319 320 321 322 323 324 325 326 327 328 329 330 331 332 333 334 335 336\n",
       "[235] 337 338 339 340 341 342 343 344 345 346 347 348 349 350 351 352 353 354\n",
       "[253] 355 356 357 358 359 360 361 362 363 364 365 366 367 368 369 370 371 372\n",
       "[271] 373 374 375 376 377 378 379 380 381 382 383 384 385 386 387 388 389 390\n",
       "[289] 391 392 393 394 395 396 397 398 399 400 401 402 403 404 407 408 409 410\n",
       "[307] 411 412 414 416 417 418 420 423 424 425 426 427 428 429 430 453 454 455\n",
       "[325] 456 457 458 459 460 461 462 463 464 465 466 467 468 484 485 501 502 503\n",
       "[343] 504 505 506 507 508 535 536 538 543 546 547 548 549 550 551 556 557 558\n",
       "[361] 559 568 569 572 573 574 575 576 577\n",
       "attr(,\"class\")\n",
       "[1] \"omit\"\n"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "characterized"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "metadata": {},
   "outputs": [],
   "source": [
    "fully_characterized_clinical_characteristics <- Reduce(intersect,characterized)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<ol class=list-inline>\n",
       "\t<li>'admin.bcr'</li>\n",
       "\t<li>'admin.day_of_dcc_upload'</li>\n",
       "\t<li>'admin.disease_code'</li>\n",
       "\t<li>'admin.month_of_dcc_upload'</li>\n",
       "\t<li>'admin.patient_withdrawal.withdrawn'</li>\n",
       "\t<li>'admin.year_of_dcc_upload'</li>\n",
       "\t<li>'patient.bcr_patient_barcode'</li>\n",
       "\t<li>'patient.bcr_patient_uuid'</li>\n",
       "\t<li>'patient.clinical_cqcf.tumor_type'</li>\n",
       "\t<li>'patient.gender'</li>\n",
       "\t<li>'patient.icd_10'</li>\n",
       "\t<li>'patient.icd_o_3_histology'</li>\n",
       "\t<li>'patient.icd_o_3_site'</li>\n",
       "\t<li>'patient.informed_consent_verified'</li>\n",
       "\t<li>'patient.month_of_form_completion'</li>\n",
       "\t<li>'patient.patient_id'</li>\n",
       "\t<li>'patient.tissue_source_site'</li>\n",
       "\t<li>'patient.vital_status'</li>\n",
       "\t<li>'patient.year_of_form_completion'</li>\n",
       "\t<li>'patient.bcr_canonical_check.bcr_patient_canonical_status'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.amount'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_barcode'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.bcr_portion_uuid'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.day_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.month_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.portion_number'</li>\n",
       "\t<li>'patient.samples.sample-2.portions.portion.year_of_creation'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type'</li>\n",
       "\t<li>'patient.samples.sample-2.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample-2.vial_number'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_barcode'</li>\n",
       "\t<li>'patient.samples.sample.bcr_sample_uuid'</li>\n",
       "\t<li>'patient.samples.sample.is_ffpe'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_file_name'</li>\n",
       "\t<li>'patient.samples.sample.pathology_report_uuid'</li>\n",
       "\t<li>'patient.samples.sample.sample_type'</li>\n",
       "\t<li>'patient.samples.sample.sample_type_id'</li>\n",
       "\t<li>'patient.samples.sample.vial_number'</li>\n",
       "</ol>\n"
      ],
      "text/latex": [
       "\\begin{enumerate*}\n",
       "\\item 'admin.bcr'\n",
       "\\item 'admin.day\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.disease\\_code'\n",
       "\\item 'admin.month\\_of\\_dcc\\_upload'\n",
       "\\item 'admin.patient\\_withdrawal.withdrawn'\n",
       "\\item 'admin.year\\_of\\_dcc\\_upload'\n",
       "\\item 'patient.bcr\\_patient\\_barcode'\n",
       "\\item 'patient.bcr\\_patient\\_uuid'\n",
       "\\item 'patient.clinical\\_cqcf.tumor\\_type'\n",
       "\\item 'patient.gender'\n",
       "\\item 'patient.icd\\_10'\n",
       "\\item 'patient.icd\\_o\\_3\\_histology'\n",
       "\\item 'patient.icd\\_o\\_3\\_site'\n",
       "\\item 'patient.informed\\_consent\\_verified'\n",
       "\\item 'patient.month\\_of\\_form\\_completion'\n",
       "\\item 'patient.patient\\_id'\n",
       "\\item 'patient.tissue\\_source\\_site'\n",
       "\\item 'patient.vital\\_status'\n",
       "\\item 'patient.year\\_of\\_form\\_completion'\n",
       "\\item 'patient.bcr\\_canonical\\_check.bcr\\_patient\\_canonical\\_status'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample-2.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample-2.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr\\_aliquot\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_column'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate\\_row'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source\\_center'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year\\_of\\_shipment'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr\\_analyte\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.gel\\_image\\_file'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.is\\_derived\\_from\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental\\_protocol\\_type'\n",
       "\\item 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer\\_method'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_barcode'\n",
       "\\item 'patient.samples.sample-2.portions.portion.bcr\\_portion\\_uuid'\n",
       "\\item 'patient.samples.sample-2.portions.portion.day\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.is\\_ffpe'\n",
       "\\item 'patient.samples.sample-2.portions.portion.month\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.portions.portion.portion\\_number'\n",
       "\\item 'patient.samples.sample-2.portions.portion.year\\_of\\_creation'\n",
       "\\item 'patient.samples.sample-2.sample\\_type'\n",
       "\\item 'patient.samples.sample-2.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample-2.vial\\_number'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_barcode'\n",
       "\\item 'patient.samples.sample.bcr\\_sample\\_uuid'\n",
       "\\item 'patient.samples.sample.is\\_ffpe'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_file\\_name'\n",
       "\\item 'patient.samples.sample.pathology\\_report\\_uuid'\n",
       "\\item 'patient.samples.sample.sample\\_type'\n",
       "\\item 'patient.samples.sample.sample\\_type\\_id'\n",
       "\\item 'patient.samples.sample.vial\\_number'\n",
       "\\end{enumerate*}\n"
      ],
      "text/markdown": [
       "1. 'admin.bcr'\n",
       "2. 'admin.day_of_dcc_upload'\n",
       "3. 'admin.disease_code'\n",
       "4. 'admin.month_of_dcc_upload'\n",
       "5. 'admin.patient_withdrawal.withdrawn'\n",
       "6. 'admin.year_of_dcc_upload'\n",
       "7. 'patient.bcr_patient_barcode'\n",
       "8. 'patient.bcr_patient_uuid'\n",
       "9. 'patient.clinical_cqcf.tumor_type'\n",
       "10. 'patient.gender'\n",
       "11. 'patient.icd_10'\n",
       "12. 'patient.icd_o_3_histology'\n",
       "13. 'patient.icd_o_3_site'\n",
       "14. 'patient.informed_consent_verified'\n",
       "15. 'patient.month_of_form_completion'\n",
       "16. 'patient.patient_id'\n",
       "17. 'patient.tissue_source_site'\n",
       "18. 'patient.vital_status'\n",
       "19. 'patient.year_of_form_completion'\n",
       "20. 'patient.bcr_canonical_check.bcr_patient_canonical_status'\n",
       "21. 'patient.samples.sample-2.bcr_sample_barcode'\n",
       "22. 'patient.samples.sample-2.bcr_sample_uuid'\n",
       "23. 'patient.samples.sample-2.is_ffpe'\n",
       "24. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode'\n",
       "25. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid'\n",
       "26. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id'\n",
       "27. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration'\n",
       "28. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment'\n",
       "29. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe'\n",
       "30. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment'\n",
       "31. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column'\n",
       "32. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id'\n",
       "33. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row'\n",
       "34. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity'\n",
       "35. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center'\n",
       "36. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume'\n",
       "37. 'patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment'\n",
       "38. 'patient.samples.sample-2.portions.portion.analytes.analyte.amount'\n",
       "39. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type'\n",
       "40. 'patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id'\n",
       "41. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode'\n",
       "42. 'patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid'\n",
       "43. 'patient.samples.sample-2.portions.portion.analytes.analyte.concentration'\n",
       "44. 'patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file'\n",
       "45. 'patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe'\n",
       "46. 'patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type'\n",
       "47. 'patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method'\n",
       "48. 'patient.samples.sample-2.portions.portion.bcr_portion_barcode'\n",
       "49. 'patient.samples.sample-2.portions.portion.bcr_portion_uuid'\n",
       "50. 'patient.samples.sample-2.portions.portion.day_of_creation'\n",
       "51. 'patient.samples.sample-2.portions.portion.is_ffpe'\n",
       "52. 'patient.samples.sample-2.portions.portion.month_of_creation'\n",
       "53. 'patient.samples.sample-2.portions.portion.portion_number'\n",
       "54. 'patient.samples.sample-2.portions.portion.year_of_creation'\n",
       "55. 'patient.samples.sample-2.sample_type'\n",
       "56. 'patient.samples.sample-2.sample_type_id'\n",
       "57. 'patient.samples.sample-2.vial_number'\n",
       "58. 'patient.samples.sample.bcr_sample_barcode'\n",
       "59. 'patient.samples.sample.bcr_sample_uuid'\n",
       "60. 'patient.samples.sample.is_ffpe'\n",
       "61. 'patient.samples.sample.pathology_report_file_name'\n",
       "62. 'patient.samples.sample.pathology_report_uuid'\n",
       "63. 'patient.samples.sample.sample_type'\n",
       "64. 'patient.samples.sample.sample_type_id'\n",
       "65. 'patient.samples.sample.vial_number'\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       " [1] \"admin.bcr\"                                                                                               \n",
       " [2] \"admin.day_of_dcc_upload\"                                                                                 \n",
       " [3] \"admin.disease_code\"                                                                                      \n",
       " [4] \"admin.month_of_dcc_upload\"                                                                               \n",
       " [5] \"admin.patient_withdrawal.withdrawn\"                                                                      \n",
       " [6] \"admin.year_of_dcc_upload\"                                                                                \n",
       " [7] \"patient.bcr_patient_barcode\"                                                                             \n",
       " [8] \"patient.bcr_patient_uuid\"                                                                                \n",
       " [9] \"patient.clinical_cqcf.tumor_type\"                                                                        \n",
       "[10] \"patient.gender\"                                                                                          \n",
       "[11] \"patient.icd_10\"                                                                                          \n",
       "[12] \"patient.icd_o_3_histology\"                                                                               \n",
       "[13] \"patient.icd_o_3_site\"                                                                                    \n",
       "[14] \"patient.informed_consent_verified\"                                                                       \n",
       "[15] \"patient.month_of_form_completion\"                                                                        \n",
       "[16] \"patient.patient_id\"                                                                                      \n",
       "[17] \"patient.tissue_source_site\"                                                                              \n",
       "[18] \"patient.vital_status\"                                                                                    \n",
       "[19] \"patient.year_of_form_completion\"                                                                         \n",
       "[20] \"patient.bcr_canonical_check.bcr_patient_canonical_status\"                                                \n",
       "[21] \"patient.samples.sample-2.bcr_sample_barcode\"                                                             \n",
       "[22] \"patient.samples.sample-2.bcr_sample_uuid\"                                                                \n",
       "[23] \"patient.samples.sample-2.is_ffpe\"                                                                        \n",
       "[24] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_barcode\"         \n",
       "[25] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.bcr_aliquot_uuid\"            \n",
       "[26] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.center_id\"                   \n",
       "[27] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.concentration\"               \n",
       "[28] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.day_of_shipment\"             \n",
       "[29] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.is_derived_from_ffpe\"        \n",
       "[30] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.month_of_shipment\"           \n",
       "[31] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_column\"                \n",
       "[32] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_id\"                    \n",
       "[33] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.plate_row\"                   \n",
       "[34] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.quantity\"                    \n",
       "[35] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.source_center\"               \n",
       "[36] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.volume\"                      \n",
       "[37] \"patient.samples.sample-2.portions.portion.analytes.analyte.aliquots.aliquot.year_of_shipment\"            \n",
       "[38] \"patient.samples.sample-2.portions.portion.analytes.analyte.amount\"                                       \n",
       "[39] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type\"                                 \n",
       "[40] \"patient.samples.sample-2.portions.portion.analytes.analyte.analyte_type_id\"                              \n",
       "[41] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_barcode\"                          \n",
       "[42] \"patient.samples.sample-2.portions.portion.analytes.analyte.bcr_analyte_uuid\"                             \n",
       "[43] \"patient.samples.sample-2.portions.portion.analytes.analyte.concentration\"                                \n",
       "[44] \"patient.samples.sample-2.portions.portion.analytes.analyte.gel_image_file\"                               \n",
       "[45] \"patient.samples.sample-2.portions.portion.analytes.analyte.is_derived_from_ffpe\"                         \n",
       "[46] \"patient.samples.sample-2.portions.portion.analytes.analyte.protocols.protocol.experimental_protocol_type\"\n",
       "[47] \"patient.samples.sample-2.portions.portion.analytes.analyte.spectrophotometer_method\"                     \n",
       "[48] \"patient.samples.sample-2.portions.portion.bcr_portion_barcode\"                                           \n",
       "[49] \"patient.samples.sample-2.portions.portion.bcr_portion_uuid\"                                              \n",
       "[50] \"patient.samples.sample-2.portions.portion.day_of_creation\"                                               \n",
       "[51] \"patient.samples.sample-2.portions.portion.is_ffpe\"                                                       \n",
       "[52] \"patient.samples.sample-2.portions.portion.month_of_creation\"                                             \n",
       "[53] \"patient.samples.sample-2.portions.portion.portion_number\"                                                \n",
       "[54] \"patient.samples.sample-2.portions.portion.year_of_creation\"                                              \n",
       "[55] \"patient.samples.sample-2.sample_type\"                                                                    \n",
       "[56] \"patient.samples.sample-2.sample_type_id\"                                                                 \n",
       "[57] \"patient.samples.sample-2.vial_number\"                                                                    \n",
       "[58] \"patient.samples.sample.bcr_sample_barcode\"                                                               \n",
       "[59] \"patient.samples.sample.bcr_sample_uuid\"                                                                  \n",
       "[60] \"patient.samples.sample.is_ffpe\"                                                                          \n",
       "[61] \"patient.samples.sample.pathology_report_file_name\"                                                       \n",
       "[62] \"patient.samples.sample.pathology_report_uuid\"                                                            \n",
       "[63] \"patient.samples.sample.sample_type\"                                                                      \n",
       "[64] \"patient.samples.sample.sample_type_id\"                                                                   \n",
       "[65] \"patient.samples.sample.vial_number\"                                                                      "
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fully_characterized_clinical_characteristics"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "could use icd10 (same for all luad, lihc, etc. samples), gender, tissue source site, vital status (dead/alive), icd o 3 site/histology, ..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA0gAAANICAYAAAD958/bAAAEGWlDQ1BrQ0dDb2xvclNwYWNl\nR2VuZXJpY1JHQgAAOI2NVV1oHFUUPrtzZyMkzlNsNIV0qD8NJQ2TVjShtLp/3d02bpZJNtoi\n6GT27s6Yyc44M7v9oU9FUHwx6psUxL+3gCAo9Q/bPrQvlQol2tQgKD60+INQ6Ium65k7M5lp\nurHeZe58853vnnvuuWfvBei5qliWkRQBFpquLRcy4nOHj4g9K5CEh6AXBqFXUR0rXalMAjZP\nC3e1W99Dwntf2dXd/p+tt0YdFSBxH2Kz5qgLiI8B8KdVy3YBevqRHz/qWh72Yui3MUDEL3q4\n4WPXw3M+fo1pZuQs4tOIBVVTaoiXEI/MxfhGDPsxsNZfoE1q66ro5aJim3XdoLFw72H+n23B\naIXzbcOnz5mfPoTvYVz7KzUl5+FRxEuqkp9G/Ajia219thzg25abkRE/BpDc3pqvphHvRFys\n2weqvp+krbWKIX7nhDbzLOItiM8358pTwdirqpPFnMF2xLc1WvLyOwTAibpbmvHHcvttU57y\n5+XqNZrLe3lE/Pq8eUj2fXKfOe3pfOjzhJYtB/yll5SDFcSDiH+hRkH25+L+sdxKEAMZahrl\nSX8ukqMOWy/jXW2m6M9LDBc31B9LFuv6gVKg/0Szi3KAr1kGq1GMjU/aLbnq6/lRxc4XfJ98\nhTargX++DbMJBSiYMIe9Ck1YAxFkKEAG3xbYaKmDDgYyFK0UGYpfoWYXG+fAPPI6tJnNwb7C\nlP7IyF+D+bjOtCpkhz6CFrIa/I6sFtNl8auFXGMTP34sNwI/JhkgEtmDz14ySfaRcTIBInmK\nPE32kxyyE2Tv+thKbEVePDfW/byMM1Kmm0XdObS7oGD/MypMXFPXrCwOtoYjyyn7BV29/MZf\nsVzpLDdRtuIZnbpXzvlf+ev8MvYr/Gqk4H/kV/G3csdazLuyTMPsbFhzd1UabQbjFvDRmcWJ\nxR3zcfHkVw9GfpbJmeev9F08WW8uDkaslwX6avlWGU6NRKz0g/SHtCy9J30o/ca9zX3Kfc19\nzn3BXQKRO8ud477hLnAfc1/G9mrzGlrfexZ5GLdn6ZZrrEohI2wVHhZywjbhUWEy8icMCGNC\nUdiBlq3r+xafL549HQ5jH+an+1y+LlYBifuxAvRN/lVVVOlwlCkdVm9NOL5BE4wkQ2SMlDZU\n97hX86EilU/lUmkQUztTE6mx1EEPh7OmdqBtAvv8HdWpbrJS6tJj3n0CWdM6busNzRV3S9KT\nYhqvNiqWmuroiKgYhshMjmhTh9ptWhsF7970j/SbMrsPE1suR5z7DMC+P/Hs+y7ijrQAlhyA\ngccjbhjPygfeBTjzhNqy28EdkUh8C+DU9+z2v/oyeH791OncxHOs5y2AtTc7nb/f73TWPkD/\nqwBnjX8BoJ98VQNcC+8AAEAASURBVHgB7d0PsO51XSfwewUE0UAwKCBht2xjN2RlQUlNKkta\nI1wahzbMoWYsnK0RxthyRzaMXWhyXGJImo2NbWJ3wbURB9xyMXVVDChRNCyDiv8XaC1A/uoV\nVvb9wfM4z/d47vF37wW5/D6v78yb53yf5/ece76v7/Fc3/f3PL+zYYNBgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAAB\nAgQIECBAgAABAgQIENiRBDbuSF/MN+lr2St/zp7JrsmDyReShxKDAAECBAgQIECAAAECLQQO\nyyovSD6fPLZGbsx95yf7JAYBAgQIECBAgAABAgRmK3B6VrYoRbfm46uSP0z+Z/K/kz9L7krq\nmH9IXpcYBAgQIECAAAECBAg0FJj7S+yOz57+QXJ5clpybbLWKIdXJGcnRyQvT6pIGQQIECBA\nYFsFviVPPGRbn+x5BAgQeAIFbsjnuucJ/Hw+1dNY4KJ87fXyuXq/0ZRR70+6P/mdKQc7hgAB\nAgQIrCPwH/PY4hUMbln4HvA98FR+D/yXdX5WeWiVwM6r5nObHpoFXZ1snriwe3PcdckBE493\nGAECBAgQ2JLALocccsjmt73tbVP/kW5Ln8f9BAgQ2GaB3/7t397w0Y9+9Jnb/AkaPnHuBane\nW3R4skvyyIT9rTNIVarqgg0GAQIECBDYLoFnPOMZG3bZpf4KMggQIPDUCGzcOPd31Dzxrs94\n4j/lDvUZL8xXc3BySXLkOl/Z4j1I9V6l3ZNL1znWQwQIECBAgAABAgQIzFRg7meQLs6+7Zuc\nmRyb3JFsSu5O6r1GeyR7Jwcl+yWPJqcmVyYGAQIECBAgQIAAAQLNBOZekOrNcOcklyVnJUcl\nq88kPZz77kzOTs5Nbk8MAgQIECBAgAABAgQaCsy9IC229KZ8cMLKpM4a7ZnsltQvjr0vMQgQ\nIECAAAECBAgQILChS0Fa3up6aV2lRr20rs4oVVG6IfliYhAgQIAAAQIECBAg0FRg7hdpeGP2\ntd6H9KxV+/vCzK9Jbkk+kHw6qSvevSXZKTEIECBAgAABAgQIEGgoMPeCVGeH6qV1y9d+f37m\nH0+OSD6ZnJ+8K3kw+Y3kHYlBgAABAgQIECBAgEBDgY4vsasSVO9BelNy3tKe1+W9fzd5c/L+\n5EOJQYAAAQIECBAgQIBAI4GOBell2d9PJMvlqLa8rmb3c8mPJq9Mtrcg1S+cXT5zlem6o14G\nWGe2DAIECBAgQIAAAQIEniKBjgWprmL34S1410Uark8O2cLjU+9+QQ6s9zVtzUsY65LkVajq\ndzEZBAgQIECAAAECBAg8BQJb83/gn4Iv70n5Iz+Vz1oXaVhrPC93vjipCzZsz/jbPPnZyXMm\nps5YbUw67keWbRAgQIAAAQIECBDYMQS6nEGql9TVBRmqHF2V/GrymuR9yWIcmA/entRZnI8t\n7tyO2y9txXO35tit+LQOJUCAAAECBAgQIEBgawTmXpDqYgvPTV6UvG4luXl81HuQFgXpmHx8\naVIeVaDqqnYGAQIECBAgQIAAAQLNBOZekN6T/azUqCvXVVFapF7Sthj1u4/q/UdVjOoqdvV+\nIIMAAQIECBAgQIAAgWYCcy9Iy9t5Xyb10rm1Xj73wdxf7z96JDEIECBAgAABAgQIEGgq0Kkg\nrbfFdfbIIECAAAECBAgQIECguUC3q6Z9o/XWS+32SnZr/n1h+QQIECBAgAABAgRaCnyjwjAH\nlG/LIt6d3JPcn3wkeXmy1qjLf9dxb1nrQfcRIECAAAECBAgQIDBvgbkXpPo9RNckP5nU2aFN\nyQ8kVyRnJQYBAgQIECBAgAABAgS+JjD3gvTLWenzkzOS70gOTuoXwf5F8tbkNxODAAECBAgQ\nIECAAAECjwvMvSC9LKv8fHJm8sDjK/7qL4s9Kh9/PKlLeleJMggQIECAAAECBAgQILBh7gXp\ngOxxFaFHV+11XfL7x5Prkrcn9RI8gwABAgQIECBAgACB5gJzL0i3Zn9/JFnrqnR1wYYfS+p9\nSRcmW7pwQx4yCBAgQIAAAQIECBDoIDD3gvThbOKeya8n+6+xoXfkvlcl9fK79yfHJAYBAgQI\nECBAgAABAk0F5l6Qzsu+fi6p9xrdnvxUsnrckDuOTr6S1HuVamz86o3/EiBAgAABAgQIECDQ\nSWDuBelL2cwjk99Kbku+nKw1PpM7j0guX+tB9xEgQIAAAQIECBAg0ENg5wbLfDBrPGUl6xXC\nG3PMq5O6DHgVK4MAAQIECBAgQIAAgWYCHQrS8pbWy+i+0ahfLGsQIECAAAECBAgQINBQYL0z\nKg05LJkAAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCz\ngILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBA\nYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAA\ngc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQIECBA\ngACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAgQIAA\nAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQIECBA\ngAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAA\nAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJECBA\ngAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAA\nAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydA\ngAABAgQIECBAYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QA\nAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefet\nnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYO\nEwIECBAgQIAAAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn\n3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBB\nGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAgQIAAAQIEOgso\nSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQG\nAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDo\nLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJECBAgAABAgQI\nEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAg\nQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQI\nECBAYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAg\nQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQI\nECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAg\nQIAAAQIEOgsoSJ1339oJECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQI\nECBAgAABAgQGAQVp4DAhQIAAAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAg\nQIAAAQIECBDoLKAgdd59aydAgAABAgQIECBAYBBQkAYOEwIECBAgQIAAAQIEOgsoSJ1339oJ\nECBAgAABAgQIEBgEFKSBw4QAAQIECBAgQIAAgc4CClLn3bd2AgQIECBAgAABAgQGAQVp4DAh\nQIAAAQIECBAgQKCzgILUefetnQABAgQIECBAgACBQUBBGjhMCBAgQIAAAQIECBDoLKAgdd59\naydAgAABAgQIECBAYBDYeZj1mOyVZe6Z7Jo8mHwheSgxCBAgQIAAAQIECBBoLtDlDNJh2ecL\nks8n9yQ3J9cnm5IqSTcm5yf7JAYBAgQIECBAgAABAk0FOpxBOj17e8bK/t6W26uTKklVjOpM\n0t7JgclJyWuTk5OLE4MAAQIECBAgQIAAgWYCcy9Ix2c/qxxdnpyWXJusNTbmzlckZycXJbck\nVyUGAQIECBAgQIAAAQKNBOb+Ervjspc3JXW7pXJU2/1YckVydPJAcmJiECBAgAABAgQIECDQ\nTGDuBenQ7Ge9pG7zxH29N8ddlxww8XiHESBAgAABAgQIECAwI4G5F6S7sleHJ7tM3LO6wl2V\nqrqAg0GAAAECBAgQIECAQDOBuRekC7OfByeXJEeus7eL9yDVe5V2Ty5d51gPESBAgAABAgQI\nECAwU4G5X6Shrka3b3JmcmxyR7IpuTu5P9kjqavYHZTslzyanJpcmRgECBAgQIAAAQIECDQT\nmHtBqosvnJNclpyVHJWsPpP0cO67M6kr2J2b3J4YBAgQIECAAAECBAg0FJh7QVpsaV3J7oSV\nSZ01qt9/tFtSvzj2vsQgQIAAAQIECBAgQIDAhrm/B2mtLd4pd1Zq7c9Jnp0YBAgQIECAAAEC\nBAgQaFOQDsteX5DUGaN7kpuTulJdvR/pweTG5Pxkn8QgQIAAAQIECBAgQKCpQIeX2J2evT1j\nZX9vy239XqQqSVWM6qV2dZGGA5OTktcmJyd1cQeDAAECBAgQIECAAIFmAnMvSMdnP6sc1eW7\nT0uuTdYai8t814UaLkpuSa5KDAIECBAgQIAAAQIEGgnMvSAdl72sCzTU7eZ19rWudndFcnRy\na3Jisj0F6bvy/M8luyQGAQIECBAgQIAAAQJPE4G5F6RDsw/1krr1ytHyVt2byXXJAct3bsPH\nVcpemTxz4nP/WY47b+KxDiNAgAABAgQIECBA4EkSmHtBuituhyd1JueRCYZ75ZgqVXXBhu0Z\ndUbqyq34BF/aimMdSoAAAQIECBAgQIDAkyRQl7qe87gwizs4uSRZ/Qtil9e9eA9SvVdp9+TS\n5Qd9TIAAAQIECBAgQIBAD4G5n0G6ONu4b3JmcmxyR7IpuTu5P9kjqavYHZTslzyanJpszdmf\nHG4QIECAAAECBAgQIDAHgbkXpHqp2znJZclZyVHJ6jNJD+e+O5O6gt25ye2JQYAAAQIECBAg\nQIBAQ4G5F6TFltZFE05YmdRZo/r9R7sl9Ytj70sMAgQIECBAgAABAgQIbOhSkBZbXe+5qpfW\nVdYaO+XOKlBfTFw4YS0h9xEgQIAAAQIECBCYscDcL9JQW/dtybuTe5IqRh9JXp6sNV6YO+u4\nt6z1oPsIECBAgAABAgQIEJi3wNwL0nOyfdckP5nU2aG6QMMPJFck9Z4kgwABAgQIECBAgAAB\nAl8TmHtB+uWs9PnJGcl3JHXJ7xcnf5G8NfnNxCBAgAABAgQIECBAgMDjAnMvSC/LKutCDGcm\nDzy+4g0bPpXbuprdx5M3J1WiDAIECBAgQIAAAQIECGyYe0E6IHtcRah+v9HyqCvX/XhyXfL2\npF6CZxAgQIAAAQIECBAg0Fxg7gXp1uzvjyR1Se/Voy7Y8GNJvS/pwmRLF27IQwYBAgQIECBA\ngAABAh0E5l6QPpxNrN959OvJ/mts6B2571VJvfzu/ckxiUGAAAECBAgQIECAQFOBuRek87Kv\nn0vqvUa3Jz+VrB435I6jk68k9V6lGhu/euO/BAgQIECAAAECBAh0Eph7Qapf9npk8lvJbcmX\nk7XGZ3LnEcnlaz3oPgIECBAgQIAAAQIEegjs3GCZD2aNp6xkvUJ4Y455dVKXAa9iZRAgQIAA\nAQIECBAg0EygQ0Fa3tJ6Gd03GvWLZQ0CBAgQIECAAAECBBoKrHdGpSGHJRMgQIAAAQIECBAg\n0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQI\nECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAg\nQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQI\nECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAg\nQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQI\nECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAg\nQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQI\nECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAg\nQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761\nEyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBh\nQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8\n+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChI\nA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEF\nqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAg\noCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKd\nBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAAB\nAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIE\nCHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAAB\nAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIE\nCBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAAB\nAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIE\nCBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAAB\nAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIE\nCBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsB\nAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYE\nCBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv\n7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRw\nmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA6\n7761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAI7\nD7Mek72yzD2TXZMHky8kDyUGAQIECBAgQIAAAQLNBbqcQTos+3xB8vnknuTm5PpkU1Il6cbk\n/GSfxCBAgAABAgQIECBAoKlAhzNIp2dvz1jZ39tye3VSJamKUZ1J2js5MDkpeW1ycnJxYhAg\nQIAAAQIECBAg0Exg7gXp+OxnlaPLk9OSa5O1xsbc+Yrk7OSi5JbkqsQgQIAAAQIECBAgQKCR\nwNxfYndc9vKmpG63VI5qux9LrkiOTh5ITkwMAgQIECBAgAABAgSaCcy9IB2a/ayX1G2euK/3\n5rjrkgMmHu8wAgQIECBAgAABAgRmJDD3gnRX9urwZJeJe1ZXuKtSVRdwMAgQIECAAAECBAgQ\naCYw94J0Yfbz4OSS5Mh19nbxHqR6r9LuyaXrHOshAgQIECBAgAABAgRmKjD3izTU1ej2Tc5M\njk3uSDYldyf3J3skdRW7g5L9kkeTU5MrE4MAAQIECBAgQIAAgWYCcy9IdfGFc5LLkrOSo5LV\nZ5Iezn13JnUFu3OT2xODAAECBAgQIECAAIGGAnMvSIstrSvZnbAyqbNG9fuPdkvqF8felxgE\nCBAgQIAAAQIECBDY0KUgLW91vbSuUqNeWldnlKoo3ZB8MTEIECBAgAABAgQIEGgqMPeLNLwx\n+1rvQ3rWqv19YebXJLckH0g+ndQV796S7JQYBAgQIECAAAECBAg0FJh7QaqzQ/XSumcu7e3z\n8/HHkyOSTybnJ+9KHkx+I3lHYhAgQIAAAQIECBAg0FCg40vsqgTVe5DelJy3tOd1ee/fTd6c\nvD/5UGIQIECAAAECBAgQINBIoGNBeln29xPJcjmqLa+r2f1c8qPJK5PtKUj75/nvTqb+gtrn\n5FiDAAECBAgQIECAAIGnWKBjQaqr2H14C+51kYbrk0O28PjUu+/Nge9Nll/at95zD8yD37ve\nAR4jQIAAAQIECBAgQODJF+hYkD4V1rpIw1rjebnzxcnvr/XgVtxXReucrTj+pTn2F7bieIcS\nIECAAAECBAgQIPAkCMz9Ig0LsnpJ3UXJLyVXJXWBhtcky6PO4tTL7uqsz8eWH/AxAQIECBAg\nQIAAAQI9BOZ+BqkutvDc5EXJ61aSm8dHlaH3rXx8TG4vTcqjClRd1c4gQIAAAQIECBAgQKCZ\nwNwL0nuyn5UadeW6KkqLbKw7V0b97qN6WVwVo7qK3WOJQYAAAQIECBAgQIBAM4G5F6Tl7bwv\nk3rp3Fovn/tg7q/3Hz2SGAQIECBAgAABAgQINBXoVJDW2+I6e2QQIECAAAECBAgQINBcoMtF\nGppvs+UTIECAAAECBAgQIDBFQEGaouQYAgQIECBAgAABAgRaCMz9JXY/n12sXwy7taOuZHf1\n1j7J8QQIECBAgAABAgQIPL0F5l6Q6pev1lXrtnb8Wp6gIG2tmuMJECBAgAABAgQIPM0F5l6Q\nXp39eW/y0uSy5PeSKeOGKQc5hgABAgQIECBAgACBeQnMvSD9Xbbrh5K6tHeVpTOSTycGAQIE\nCBAgQIAAAQIEvk6gw0UaNmfVb1hZ+Tu/TsAdBAgQIECAAAECBAgQWBHoUJBqqX+ZvDWpCza8\nMDEIECBAgAABAgQIECDwdQJdClIt/Ozk0OSzNTEIECBAgAABAgQIECCwWqBTQVq9dnMCBAgQ\nIECAAAECBAgMAgrSwGFCgAABAgQIECBAgEBnAQWp8+5bOwECBAgQIECAAAECg4CCNHCYECBA\ngAABAgQIECDQWUBB6rz71k6AAAECBAgQIECAwCCgIA0cJgQIECBAgAABAgQIdBZQkDrvvrUT\nIECAAAECBAgQIDAIKEgDhwkBAgQIECBAgAABAp0FFKTOu2/tBAgQIECAAAECBAgMAgrSwGFC\ngAABAgQIECBAgEBnAQWp8+5bOwECBAgQIECAAAECg4CCNHCYECBAgAABAgQIECDQWUBB6rz7\n1k6AAAECBAgQIECAwCCgIA0cJgQIECBAgAABAgQIdBZQkDrvvrUTIECAAAECBAgQIDAIKEgD\nhwkBAgQIECBAgAABAp0FFKTOu2/tBAgQIECAAAECBAgMAgrSwGFCgAABAgQIECBAgEBnAQWp\n8+5bOwECBAgQIECAAAECg4CCNHCYECBAgAABAgQIECDQWUBB6rz71k6AAAECBAgQIECAwCCg\nIA0cJgQIECBAgAABAgQIdBZQkDrvvrUTIECAAAECBAgQIDAIKEgDhwkBAgQIECBAgAABAp0F\nFKTOu2/tBAgQIECAAAECBAgMAgrSwGFCgAABAgQIECBAgEBnAQWp8+5bOwECBAgQIECAAAEC\ng4CCNHCYECBAgAABAgQIECDQWUBB6rz71k6AAAECBAgQIECAwCCgIA0cJgQIECBAgAABAgQI\ndBZQkDrvvrUTIECAAAECBAgQIDAIKEgDhwkBAgQIECBAgAABAp0FFKTOu2/tBAgQIECAAAEC\nBAgMAgrSwGFCgAABAgQIECBAgEBnAQWp8+5bOwECBAgQIECAAAECg4CCNHCYECBAgAABAgQI\nECDQWUBB6rz71k6AAAECBAgQIECAwCCgIA0cJgQIECBAgAABAgQIdBZQkDrvvrUTIECAAAEC\nBAgQIDAIKEgDhwkBAgQIECBAgAABAp0FFKTOu2/tBAgQIECAAAECBAgMAgrSwGFCgAABAgQI\nECBAgEBnAQWp8+5bOwECBAgQIECAAAECg4CCNHCYECBAgAABAgQIECDQWUBB6rz71k6AAAEC\nBAgQIECAwCCgIA0cJgQIECBAgAABAgQIdBZQkDrvvrUTIECAAAECBAgQIDAIKEgDhwkBAgQI\nECBAgAABAp0FFKTOu2/tBAgQIECAAAECBAgMAgrSwGFCgAABAgQIECBAgEBngZ07L77h2o/I\nmn+44botmQCBHU/gffmS/mrH+7J8RQQIECDQXUBB6vUd8IvPfvazX7/PPvs80mvZVkuAwI4k\ncNddd+2yefPmvfI1/bsd6evytRAgQIAAgRJQkHp9H2x8yUtesvMpp5xi33vtu9US2KEETj/9\n9M3XXXfdDvU1+WIIECBAgMBCwHuQFhJuCRAgQIAAAQIECBBoL6Agtf8WAECAAAECBAgQIECA\nwEJAQVpIuCVAgAABAgQIECBAoL2AgtT+WwAAAQIECBAgQIAAAQILAQVpIeGWAAECBAgQIECA\nAIH2AgpS+28BAAQIECBAgAABAgQILAQUpIWEWwIECBAgQIAAAQIE2gsoSO2/BQAQIECAAAEC\nBAgQILAQUJAWEm4JECBAgAABAgQIEGgvoCC1/xYAQIAAAQIECBAgQIDAQkBBWki4JUCAAAEC\nBAgQIECgvYCC1P5bAAABAgQIECBAgAABAgsBBWkh4ZYAAQIECBAgQIAAgfYCClL7bwEABAgQ\nIECAAAECBAgsBBSkhYRbAgQIECBAgAABAgTaCyhI7b8FABAgQIAAAQIECBAgsBBQkBYSbgkQ\nIECAAAECBAgQaC+gILX/FgBAgAABAgQIECBAgMBCQEFaSLglQIAAAQIECBAgQKC9gILU/lsA\nAAECBAgQIECAAAECCwEFaSHhlgABAgQIECBAgACB9gIKUvtvAQAECBAgQIAAAQIECCwEFKSF\nhFsCBAgQIECAAAECBNoLKEjtvwUAECBAgAABAgQIECCwEFCQFhJuCRAgQIAAAQIECBBoL6Ag\ntf8WAECAAAECBAgQIECAwEJAQVpIuCVAgAABAgQIECBAoL2AgtT+WwAAAQIECBAgQIAAAQIL\nAQVpIeGWAAECBAgQIECAAIH2AgpS+28BAAQIECBAgAABAgQILAQUpIWEWwIECBAgQIAAAQIE\n2gsoSO2/BQAQIECAAAECBAgQILAQUJAWEm4JECBAgAABAgQIEGgvoCC1/xYAQIAAAQIECBAg\nQIDAQkBBWki4JUCAAAECBAgQIECgvcDODQX2ypr3THZNHky+kDyUGAQIECBAgAABAgQINBfo\ncgbpsOzzBcnnk3uSm5Prk01JlaQbk/OTfRKDAAECBAgQIECAAIGmAh3OIJ2evT1jZX9vy+3V\nSZWkKkZ1Jmnv5MDkpOS1ycnJxYlBgAABAgQIECBAgEAzgbkXpOOzn1WOLk9OS65N1hobc+cr\nkrOTi5JbkqsSgwABAgQIECBAgACBRgJzf4ndcdnLm5K63VI5qu1+LLkiOTp5IDkxMQgQIECA\nAAECBAgQaCYw94J0aPazXlK3eeK+3pvjrksOmHi8wwgQIECAAAECBAgQmJHA3AvSXdmrw5Nd\nJu5ZXeGuSlVdwMEgQIAAAQIECBAgQKCZwNwL0oXZz4OTS5Ij19nbxXuQ6r1KuyeXrnOshwgQ\nIECAAAECBAgQmKnA3C/SUFej2zc5Mzk2uSPZlNyd3J/skdRV7A5K9kseTU5NrkwMAgQIECBA\ngAABAgSaCcy9INXFF85JLkvOSo5KVp9Jejj33ZnUFezOTW5PDAIECBAgQIAAAQIEGgrMvSAt\ntrSuZHfCyqTOGtXvP9otqV8ce19iECBAgAABAgQIECBAYMPc34O01hbvlDsrtfbnJM9ODAIE\nCBAgQIAAAQIECLQpSIdlry9I6ozRPcnNSV2prt6P9GByY3J+sk9iECBAgAABAgQIECDQVKDD\nS+xOz96esbK/t+W2fi9SlaQqRvVSu7pIw4HJSclrk5OTuriDQYAAAQIECBAgQIBAM4G5F6Tj\ns59Vjury3acl1yZrjcVlvutCDRcltyRXJQYBAgQIECBAgAABAo0E5l6Qjste1gUa6nbzOvta\nV7u7Ijk6uTU5MdmeglRnpurM1TOTKePbpxzkGAIECBAgQIAAAQIEnlyBuRekQ8NXL6lbrxwt\nC9+byXXJAct3bsPHVYz2T3aZ+Nx6mZ9BgAABAgQIECBAgMBTLDD3gnRXfA9Pqqg8MsF6rxxT\npaou2LA94+/z5BO24hO8NMduzxmrrfijHEqAAAECBAgQIECAwJYE5n6Z7wuz8IOTS5LVvyB2\n2WTxHqR6r9LuyaXLD/qYAAECBAgQIECAAIEeAnM/g3RxtnHf5Mzk2OSOZFNyd3J/skdSL287\nKNkveTQ5NbkyMQgQIECAAAECBAgQaCYw94JUF184J7ksOSs5Kll9Junh3HdnUlewOze5PTEI\nECBAgAABAgQIEGgoMPeCtNjSupLd4j1BddaorjK3W1K/OPa+xCBAgAABAgQIECBAgMCGLgVp\neavrpXUVgwABAgQIECBAgAABAoPA3C/SMCzWhAABAgQIECBAgAABAusJKEjr6XiMAAECBAgQ\nIECAAIFWAnN/id3PZzfrPUdbO+p3EtUvmDUIECBAgAABAgQIEGgkMPeC9AvZyxdtw37+Wp6j\nIG0DnKcQIECAAAECBAgQeDoLzL0gvTqb897kpUld6vv3kinjhikHOYYAAQIECBAgQIAAgXkJ\nzL0g/V2264eSjyVVls5IPp0YBAgQIECAAAECBAgQ+DqBDhdp2JxVv2Fl5e/8OgF3ECBAgAAB\nAgQIECBAYEWgQ0Gqpf5l8takLtjwwsQgQIAAAQIECBAgQIDA1wl0KUi18LOTQ5PP1sQgQIAA\nAQIECBAgQIDAaoFOBWn12s0JECBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv\n7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRw\nmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA6\n7761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK\n0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlA\nQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAw\nCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBA\nZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBA\ngMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAA\nAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBA\ngAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAA\nAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBA\ngAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAA\nAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBA\ngAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAA\nAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZO\ngAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJ\nAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPu\nWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCAN\nHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSk\nzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOA\ngjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQW\nUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQI\nDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg\n0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQI\nECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAg\nQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQI\nECBAgMAgoCANHCYECBAgQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAg\nQIAAAQKdBRSkzrtv7QQIECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQI\nECBAgAABAoOAgjRwmBAgQIAAAQIECBAg0FlAQeq8+9ZOgAABAgQIECBAgMAgoCANHCYECBAg\nQIAAAQIECHQWUJA67761EyBAgAABAgQIECAwCChIA4cJAQIECBAgQIAAAQKdBRSkzrtv7QQI\nECBAgAABAgQIDAIK0sBhQoAAAQIECBAgQIBAZwEFqfPuWzsBAgQIECBAgAABAoOAgjRwmBAg\nQIAAAQIECBAg0Flg54aL3ytr3jPZNXkw+ULyUGIQIECAAAECBAgQINBcoMsZpMOyzxckn0/u\nSW5Ork82JVWSbkzOT/ZJDAIECBAgQIAAAQIEmgp0OIN0evb2jJX9vS23VydVkqoY1ZmkvZMD\nk5OS1yYnJxcnBgECBAgQIECAAAECzQTmXpCOz35WObo8OS25NllrbMydr0jOTi5KbkmuSgwC\nBAgQIECAAAECBBoJzP0ldsdlL29K6nZL5ai2+7HkiuTo5IHkxMQgQIAAAQIECBAgQKCZwNwL\n0qHZz3pJ3eaJ+3pvjrsuOWDi8Q4jQIAAAQIECBAgQGBGAnMvSHdlrw5Pdpm4Z3WFuypVdQEH\ngwABAgQIECBAgACBZgJzL0gXZj8PTi5JjlxnbxfvQar3Ku2eXLrOsR4iQIAAAQIECBAgQGCm\nAnO/SENdjW7f5Mzk2OSOZFNyd3J/skdSV7E7KNkveTQ5NbkyMQgQIECAAAECBAgQaCYw94JU\nF184J7ksOSs5Kll9Junh3HdnUlewOze5PdneUS/pq0uG1+2U8YIpBzmGAAECBAgQIECAAIEn\nV2DuBWmhV1eyO2FlUmeN6vcf7ZbUL469L3mix/75hGcmz5z4iaceN/HTOYwAAQIECBAgQIAA\ngW0R6FKQlm3qpXWVGvXSujqjVEXphuSLyRMxbs0n2ZqzQi/N8Vc9EX+wz0GAAAECBAgQIECA\nwLYLzP0iDW8MzcXJs1YRvTDza5Jbkg8kn07qindvSXZKDAIECBAgQIAAAQIEGgrMvSDV2aF6\nad3yS9ien/nHkyOSTybnJ+9KHkx+I3lHYhAgQIAAAQIECBAg0FCg40vsqgTVe5DelJy3tOd1\nee/fTd6cvD/5UGIQIECAAAECBAgQINBIYO5nkNbaypflzk8ky+Wojqur2f1cUpcAf2ViECBA\ngAABAgQIECDQTKBjQaqr2H12C/tcF2m4PjlkC4+7mwABAgQIECBAgACBGQt0LEifyn7WRRrW\nGs/LnS9O6oINBgECBAgQIECAAAECzQS6FKR6Sd1FyS8ldTntukDDa5LlcWAm9bK7uqDDx5Yf\n8DEBAgQIECBAgAABAj0E5n6RhrrYwnOTFyWvW0luHh9Vht638vExub00KY8qUHVVO4MAAQIE\nCBAgQIAAgWYCcy9I78l+VmrUleuqKC2yse5cGfW7j+r9R1WM6ip2jyUGAQIECBAgQIAAAQLN\nBOZekJa3875M6qVza7187oO5v95/9EhiECBAgAABAgQIECDQVKBTQVpvi+vskUGAAAECBAgQ\nIECAQHOBLhdpaL7Nlk+AAAECBAgQIECAwBQBBWmKkmMIECBAgAABAgQIEGghoCC12GaLJECA\nAAECBAgQIEBgioCCNEXJMQQIECBAgAABAgQItBBQkFpss0USIECAAAECBAgQIDBFQEGaouQY\nAgQIECBAgAABAgRaCChILbbZIgkQIECAAAECBAgQmCKgIE1RcgwBAgQIECBAgAABAi0EFKQW\n22yRBAgQIECAAAECBAhMEVCQpig5hgABAgQIECBAgACBFgIKUotttkgCBAgQIECAAAECBKYI\nKEhTlBxDgAABAgQIECBAgEALAQWpxTZbJAECBAgQIECAAAECUwQUpClKjiFAgAABAgQIECBA\noIWAgtRimy2SAAECBAgQIECAAIEpAgrSFCXHECBAgAABAgQIECDQQkBBarHNFkmAAAECBAgQ\nIECAwBQBBWmKkmMIECBAgAABAgQIEGghoCC12GaLJECAAAECBAgQIEBgioCCNEXJMQQIECBA\ngAABAgQItBBQkFpss0USIECAAAECBAgQIDBFQEGaouQYAgQIECBAgAABAgRaCChILbbZIgkQ\nIECAAAECBAgQmCKgIE1RcgwBAgQIECBAgAABAi0EFKQW22yRBAgQIECAAAECBAhMEVCQpig5\nhgABAgQIECBAgACBFgIKUotttkgCBAgQIECAAAECBKYIKEhTlBxDgAABAgQIECBAgEALAQWp\nxTZbJAECBAgQIECAAAECUwQUpClKjiFAgAABAgQIECBAoIWAgtRimy2SAAECBAgQIECAAIEp\nAgrSFCXHECBAgAABAgQIECDQQkBBarHNFkmAAAECBAgQIECAwBQBBWmKkmMIECBAgAABAgQI\nEGghoCC12GaLJECAAAECBAgQIEBgioCCNEXJMQQIECBAgAABAgQItBBQkFpss0USIECAAAEC\nBAgQIDBFQEGaouQYAgQIECBAgAABAgRaCChILbbZIgkQIECAAAECBAgQmCKgIE1RcgwBAgQI\nECBAgAABAi0EFKQW22yRBAgQIECAAAECBAhMEVCQpig5hgABAgQIECBAgACBFgIKUotttkgC\nBAgQIECAAAECBKYIKEhTlBxDgAABAgQIECBAgEALAQWpxTZbJAECBAgQIECAAAECUwQUpClK\njiFAgAABAgQIECBAoIWAgtRimy2SAAECBAgQIECAAIEpAgrSFCXHECBAgAABAgQIECDQQkBB\narHNFkmAAAECBAgQIECAwBQBBWmKkmMIECBAgAABAgQIEGghoCC12GaLJECAAAECBAgQIEBg\nioCCNEXJMQQIECBAgAABAgQItBBQkFpss0USIECAAAECBAgQIDBFQEGaouQYAgQIECBAgAAB\nAgRaCChILbbZIgkQIECAAAECBAgQmCKgIE1RcgwBAgQIECBAgAABAi0EFKQW22yRBAgQIECA\nAAECBAhMEVCQpig5hgABAgQIECBAgACBFgIKUotttkgCBAgQIECAAAECBKYIKEhTlBxDgAAB\nAgQIECBAgEALAQWpxTZbJAECBAgQIECAAAECUwQUpClKjiFAgAABAgQIECBAoIWAgtRimy2S\nAAECBAgQIECAAIEpAgrSFCXHECBAgAABAgQIECDQQkBBarHNFkmAAAECBAgQIECAwBQBBWmK\nkmMIECBAgAABAgQIEGghoCC12GaLJECAAAECBAgQIEBgioCCNEXJMQQIECBAgAABAgQItBBQ\nkFpss0USIECAAAECBAgQIDBFQEGaouQYAgQIECBAgAABAgRaCChILbbZIgkQIECAAAECBAgQ\nmCKgIE1RcgwBAgQIECBAgAABAi0EFKQW22yRBAgQIECAAAECBAhMEVCQpig5hgABAgQIECBA\ngACBFgIKUotttkgCBAgQIECAAAECBKYIKEhTlBxDgAABAgQIECBAgEALAQWpxTZbJAECBAgQ\nIECAAAECUwQUpClKjiFAgAABAgQIECBAoIWAgtRimy2SAAECBAgQIECAAIEpAgrSFCXHECBA\ngAABAgQIECDQQkBBarHNFkmAAAECBAgQIECAwBQBBWmKkmMIECBAgAABAgQIEGghoCC12GaL\nJECAAAECBAgQIEBgioCCNEXJMQQIECBAgAABAgQItBBQkFpss0USIECAAAECBAgQIDBFQEGa\nouQYAgQIECBAgAABAgRaCChILbbZIgkQIECAAAECBAgQmCKgIE1RcgwBAgQIECBAgAABAi0E\nFKQW22yRBAgQIECAAAECBAhMEVCQpig5hgABAgQIECBAgACBFgIKUotttkgCBAgQIECAAAEC\nBKYIKEhTlBxDgAABAgQIECBAgEALAQWpxTZbJAECBAgQIECAAAECUwQUpClKjiFAgAABAgQI\nECBAoIWAgtRimy2SAAECBAgQIECAAIEpAgrSFCXHECBAgAABAgQIECDQQkBBarHNFkmAAAEC\nBAgQIECAwBQBBWmKkmMIECBAgAABAgQIEGghoCC12GaLJECAAAECBAgQIECWR6HOAAAO30lE\nQVRgioCCNEXJMQQIECBAgAABAgQItBBQkFpss0USIECAAAECBAgQIDBFQEGaouQYAgQIECBA\ngAABAgRaCChILbbZIgkQIECAAAECBAgQmCKw85SDZnbMXlnPnsmuyYPJF5KHEoMAAQIECBAg\nQIAAgeYCXc4gHZZ9viD5fHJPcnNyfbIpqZJ0Y3J+sk9iECBAgAABAgQIECDQVKDDGaTTs7dn\nrOzvbbm9OqmSVMWoziTtnRyYnJS8Njk5uTgxCBAgQIAAAQIECBBoJjD3gnR89rPK0eXJacm1\nyVpjY+58RXJ2clFyS3JVsj3jO/PkXSZ+gipo35Tx8MMPb7jjjju+KX+WP4QAAQJrCWzevLl+\n5rYYtVY/c1tstUUS2GEF6v/7GQSWBars1Mvn6v1GU0a9P+n+5HemHLzOMS/IY49tZb6S43da\n53M+EQ+9cyu/pq1dg+O3ft+ZMev6PVD/aDX3UWvsur/Wbe99D+xY3wO/NfcfuE/k+ub+r3if\nDdafJ6/fCrQ/ybH3JsduxXPWOvR5uXPqGaR6fhWkeo/UkznqjOG3Ppl/gM9NgACBCQL1f5z+\nPqmfe3Me9T7fem/r3P+unfMeWhuBuQj8Qxby6FwWYx3bJ/DHefpfJVOLyuIM0ju274/1bAIE\nCBAgQIAAAQIECOx4Aj+dL6n+pfJ9yZHrfHmL9yD9WY6pdv3ydY71EAECBAgQIECAAAECBJ6W\nAlV83pzU7zmqorQp+dPkj5J3rdzWVe3uTOrxR5JTEoMAAQIECBAgQIAAAQKzFagrylUhqsu3\nVRFaTpWnv0n+U/L8xCBAgAABAgQIECBAoKlAxzeO7pG9rt9/tFtSF0W4LzEIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI\nECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABArMQ2GkWq7AIAr0FnpXl/8vk9clL\nkr9JHkp2xLFXvqhXJ/Wz5+93xC/Q10SAAIGnkYCfqU+jzfKlEiBAgMA3R2Bj/phLk8eWst83\n54/epj/l+1a+zrdv07M9iQABAgSWBfxMXdbwMYEnSOAZT9Dn8WkIEHhqBL4zf+y/Sv4qeVFy\nQHJXYhAgQIAAAQIECGyDwM7b8BxPIUBgxxGoQlTjj5I/f/wj/yFAgAABAgQIENhmAQVpm+k8\nkcBTLnBsvoLvX/kq6kzSzyRVkj6zcl/97/uYpM4s7ZrU/f8r+WKyPH4wk29N3pO8OPnhpJ77\nkeTKpMb3JPU+p+cnn0zendTL+pZHfY56bh373ORvkz9JrkumjO/OQa9M6vm3JB9Npj43hxoE\nCBB4ygXqPUGvSa5K/iH50eRfJPWz7ANJvfdyt+TopH5+35vUz9ObktXDz9TVIuYECBAgQOAb\nCHw6jy+/9+grmZ+x8pwqTH+28vh9ua2/qOvYzyWHJsvjvZncnvxqUsd8eeW2Pv43yU8km5Pl\n+9+V+fKol/nVX/z1nPrzvrTy8f/L7a8ki7Gl18ufmgPqz6g11NfyaFLPPSup91kZBAgQeDoI\n1M/X+jl4ZlL/SFQfP7Jy+9e5rbP+H1mZL36m1kV1DkuWh5+pyxo+JkCAAAECWyFQZ1zqL+B/\nv/ScKhTXJFUwXp8sCsar8vHdyQ3JM5PFqIJUxeSBpP61c5ek/nWzik6VlnuSNyV1VugfJfWX\nfP2Z/zSpsUdyf1Kf+4hkp+TZyXFJPf/hZM+kxloFqc6E1ef7WLJ/UuNbkouTuv9nEoMAAQJP\nB4FFQaqfqf8jqZ9p35b8t6R+nlUZen/ysqR+Dv/bpO6vs0iL4WfqQsItAQIECBDYBoG1CtJP\n5fPUX7j1crrV4z/kjnrsjUsPvHflvl9cuq8+/MOV++s5y6PKWH2OOrNUo0rP5ckbarJq1NdQ\nxx6ycv9aBen6lWMOXzlmcVMlq8rVncmi5C0ec0uAAIEdUWBRkOpM+K5LX2D941H9LKx/iKoC\ntBj1s+3LyZ8v7sitn6lLGD4k8FQI1PsMDAIE5iVQf7nW+D9J/WW9PD63Mqm/rM9ffiAff2rV\n/LOZH5N8YtX9d63M6yxPjT9N6v1Ji1FnoL4reVFS/3JaY/ev3nzdf+us1Pckf5PUy1BWf73X\n5L6jkvpX2DsSgwABAk8Hgb/MF1ln0Bejfp7WuCGpM+6LUaXp/yaLn6d1v5+ppWAQeAoFFKSn\nEN8fTeBJEvjulc/7m+t8/hes8ditq+6rl+jV+MJXb77238X9X7sjH9TnOzX5wZWP62fL4mV7\n+XCLZ4AWX2vdLv8Laj1nedTnV5CWRXxMgMCOLDD152mtwc/UHXknfW0tBer/xBgECMxL4Esr\ny/np3Na/TK41lv8Fc/F4vcxjW8bBedJVSf0L6AeS/558JqmzP7+W/EKypbH4Wut579jSQbn/\nL9Z5zEMECBDY0QS29edprcPP1B1tN3097QQUpHZbbsENBP56ZY1Vgj68ar31krZ6ed3frbp/\ne6Yn58l7JT+bXJgsj3+yMtlp+c6ljxdXefrW3Lf6a63DjkzqX1frdfsGAQIEOgj4mdphl61x\nhxZ4xg791fniCBDYFoH35Un1uva3JquLyXm574PJS5MnavzjlU9086pPeFjmiz9nl1WPLab1\nO5n+ODk8+bHFnSu335vbK5L/mtR6DAIECHQQ8DO1wy5b4w4toCDt0NvjiyOwTQJX51m/n1Q5\n+Vjyr5OfSOrsTr3srgrUHyRP1KiLQdQ4JzkheVnylqSKz+akxt5fvVnzv6fk3nqp3SXJ25JX\nJb+SXJZUwXtDUhdwMAgQINBBwM/UDrtsjQQIECDwpAm8Mp+5zq4s/x6k+sPqHz9+OakLLNTj\nlbpownuSb0+Wx3szqceft3xnPj5z5f7vX3X/z67cf+LK/VVi/nPy6Mr99bnqJXw/n9TL+Wr+\nO0mN70tq/vaaLI16zX2dLaqX09XjlU3JzyQGAQIEni4Ch+YLrZ9f71z1BddbGur+D626v6a3\nJDfVByvDz9SFhFsCBAgQIPAkCRyYz/vPkz2epM+/+LT1+V+U7L+4Yxtu63Lg9TkOSur/JBgE\nCBDoKuBnatedt24CBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECA\nAAECBAgQIECAAAECBAgQIECAAIEdXOD/AwcbDIDmqz1WAAAAAElFTkSuQmCC",
      "text/plain": [
       "plot without title"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "barplot(table(clinical_dataset_list[['lihc']][,'patient.gender']))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "R",
   "language": "R",
   "name": "ir"
  },
  "language_info": {
   "codemirror_mode": "r",
   "file_extension": ".r",
   "mimetype": "text/x-r-source",
   "name": "R",
   "pygments_lexer": "r",
   "version": "3.6.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
