{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Healthcare Hack Nights: Part I"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## RNA-Seq\n",
    "Questions that can be answered by RNA-seq:\n",
    "    - What genes are differentially expressed between group samples?\n",
    "    - How does gene expression change across time or conditions? (eg, in benign vs malignant tumors)\n",
    "    - What pathways or processes are enriched under a condition?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(423, 20532)"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lihc = pd.read_csv('../lihc_rnaseq.csv')\n",
    "lihc.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>bcr_patient_barcode</th>\n",
       "      <th>?|100130426</th>\n",
       "      <th>?|100133144</th>\n",
       "      <th>?|100134869</th>\n",
       "      <th>?|10357</th>\n",
       "      <th>?|10431</th>\n",
       "      <th>?|136542</th>\n",
       "      <th>?|155060</th>\n",
       "      <th>?|26823</th>\n",
       "      <th>?|280660</th>\n",
       "      <th>...</th>\n",
       "      <th>ZXDA|7789</th>\n",
       "      <th>ZXDB|158586</th>\n",
       "      <th>ZXDC|79364</th>\n",
       "      <th>ZYG11A|440590</th>\n",
       "      <th>ZYG11B|79699</th>\n",
       "      <th>ZYX|7791</th>\n",
       "      <th>ZZEF1|23140</th>\n",
       "      <th>ZZZ3|26009</th>\n",
       "      <th>psiTPTE22|387590</th>\n",
       "      <th>tAKR|389932</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>TCGA-2V-A95S-01A-11R-A37K-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>1.5051</td>\n",
       "      <td>3.7074</td>\n",
       "      <td>90.1124</td>\n",
       "      <td>1017.1038</td>\n",
       "      <td>0.0</td>\n",
       "      <td>141.3911</td>\n",
       "      <td>0.6516</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>...</td>\n",
       "      <td>24.7597</td>\n",
       "      <td>273.6602</td>\n",
       "      <td>794.2662</td>\n",
       "      <td>18.2440</td>\n",
       "      <td>499.1041</td>\n",
       "      <td>3172.5037</td>\n",
       "      <td>890.0472</td>\n",
       "      <td>510.1808</td>\n",
       "      <td>3.9094</td>\n",
       "      <td>6.5157</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>TCGA-2Y-A9GS-01A-12R-A38B-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>26.4120</td>\n",
       "      <td>2.6663</td>\n",
       "      <td>71.0054</td>\n",
       "      <td>639.2311</td>\n",
       "      <td>0.0</td>\n",
       "      <td>122.7206</td>\n",
       "      <td>1.4786</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>...</td>\n",
       "      <td>68.5067</td>\n",
       "      <td>632.8241</td>\n",
       "      <td>1153.7703</td>\n",
       "      <td>71.4638</td>\n",
       "      <td>1000.4929</td>\n",
       "      <td>5301.1336</td>\n",
       "      <td>755.5446</td>\n",
       "      <td>860.5224</td>\n",
       "      <td>6.4071</td>\n",
       "      <td>482.9966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>TCGA-2Y-A9GT-01A-11R-A38B-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>4.4833</td>\n",
       "      <td>95.5122</td>\n",
       "      <td>742.4344</td>\n",
       "      <td>0.0</td>\n",
       "      <td>95.0460</td>\n",
       "      <td>1.7933</td>\n",
       "      <td>0.8967</td>\n",
       "      <td>...</td>\n",
       "      <td>46.6263</td>\n",
       "      <td>1219.4575</td>\n",
       "      <td>1133.3782</td>\n",
       "      <td>12.5532</td>\n",
       "      <td>1289.3970</td>\n",
       "      <td>3219.0092</td>\n",
       "      <td>860.7935</td>\n",
       "      <td>523.6494</td>\n",
       "      <td>14.3466</td>\n",
       "      <td>83.3894</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>TCGA-2Y-A9GU-01A-11R-A38B-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>5.7222</td>\n",
       "      <td>5.1216</td>\n",
       "      <td>61.6679</td>\n",
       "      <td>1186.9807</td>\n",
       "      <td>0.0</td>\n",
       "      <td>280.2709</td>\n",
       "      <td>0.8341</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>...</td>\n",
       "      <td>18.3511</td>\n",
       "      <td>285.2758</td>\n",
       "      <td>1150.2786</td>\n",
       "      <td>9.1755</td>\n",
       "      <td>941.7437</td>\n",
       "      <td>3092.9899</td>\n",
       "      <td>1339.6283</td>\n",
       "      <td>343.6655</td>\n",
       "      <td>2.5024</td>\n",
       "      <td>2.5024</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>TCGA-2Y-A9GV-01A-11R-A38B-07</td>\n",
       "      <td>0.0</td>\n",
       "      <td>11.4975</td>\n",
       "      <td>5.4230</td>\n",
       "      <td>104.4670</td>\n",
       "      <td>878.1726</td>\n",
       "      <td>0.0</td>\n",
       "      <td>282.5719</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>0.0000</td>\n",
       "      <td>...</td>\n",
       "      <td>41.4552</td>\n",
       "      <td>999.1540</td>\n",
       "      <td>1631.9797</td>\n",
       "      <td>4.2301</td>\n",
       "      <td>1380.7107</td>\n",
       "      <td>2902.7073</td>\n",
       "      <td>575.2961</td>\n",
       "      <td>665.8206</td>\n",
       "      <td>2.5381</td>\n",
       "      <td>119.2893</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 20532 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            bcr_patient_barcode  ?|100130426  ?|100133144  ?|100134869  \\\n",
       "0  TCGA-2V-A95S-01A-11R-A37K-07          0.0       1.5051       3.7074   \n",
       "1  TCGA-2Y-A9GS-01A-12R-A38B-07          0.0      26.4120       2.6663   \n",
       "2  TCGA-2Y-A9GT-01A-11R-A38B-07          0.0       0.0000       4.4833   \n",
       "3  TCGA-2Y-A9GU-01A-11R-A38B-07          0.0       5.7222       5.1216   \n",
       "4  TCGA-2Y-A9GV-01A-11R-A38B-07          0.0      11.4975       5.4230   \n",
       "\n",
       "    ?|10357    ?|10431  ?|136542  ?|155060  ?|26823  ?|280660  ...  ZXDA|7789  \\\n",
       "0   90.1124  1017.1038       0.0  141.3911   0.6516    0.0000  ...    24.7597   \n",
       "1   71.0054   639.2311       0.0  122.7206   1.4786    0.0000  ...    68.5067   \n",
       "2   95.5122   742.4344       0.0   95.0460   1.7933    0.8967  ...    46.6263   \n",
       "3   61.6679  1186.9807       0.0  280.2709   0.8341    0.0000  ...    18.3511   \n",
       "4  104.4670   878.1726       0.0  282.5719   0.0000    0.0000  ...    41.4552   \n",
       "\n",
       "   ZXDB|158586  ZXDC|79364  ZYG11A|440590  ZYG11B|79699   ZYX|7791  \\\n",
       "0     273.6602    794.2662        18.2440      499.1041  3172.5037   \n",
       "1     632.8241   1153.7703        71.4638     1000.4929  5301.1336   \n",
       "2    1219.4575   1133.3782        12.5532     1289.3970  3219.0092   \n",
       "3     285.2758   1150.2786         9.1755      941.7437  3092.9899   \n",
       "4     999.1540   1631.9797         4.2301     1380.7107  2902.7073   \n",
       "\n",
       "   ZZEF1|23140  ZZZ3|26009  psiTPTE22|387590  tAKR|389932  \n",
       "0     890.0472    510.1808            3.9094       6.5157  \n",
       "1     755.5446    860.5224            6.4071     482.9966  \n",
       "2     860.7935    523.6494           14.3466      83.3894  \n",
       "3    1339.6283    343.6655            2.5024       2.5024  \n",
       "4     575.2961    665.8206            2.5381     119.2893  \n",
       "\n",
       "[5 rows x 20532 columns]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lihc.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The dataset contains the number of counts for ~20k genes defined by their Entrez transcript ID x 423 deidentified patients. \n",
    "\n",
    "Entrez (https://www.ncbi.nlm.nih.gov/Web/Search/entrezfs.html) is a data retrieval system that provides users access to NCBI’s databases such as PubMed, GenBank, GEO, and many others. You can access Entrez from a web browser to manually enter queries, or you can use Biopython’s Bio.Entrez module for programmatic access to Entrez. \n",
    "\n",
    "Entrez gene IDs are unique gene identifiers that can be used to trace a particular gene or transcript to the genome."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['100130426', '100133144', '100134869', '10357', '10431'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Get Entrez transcript IDs\n",
    "\n",
    "ids = pd.Series(lihc.columns.values[1:]).apply(lambda x: x.split('|')[1]).values\n",
    "ids[:5]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# BioMart\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from pybiomart import Server, Dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene stable ID</th>\n",
       "      <th>Gene stable ID version</th>\n",
       "      <th>Transcript stable ID</th>\n",
       "      <th>Transcript stable ID version</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>ENSG00000200036</td>\n",
       "      <td>ENSG00000200036.1</td>\n",
       "      <td>ENST00000363166</td>\n",
       "      <td>ENST00000363166.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>ENSG00000252396</td>\n",
       "      <td>ENSG00000252396.1</td>\n",
       "      <td>ENST00000516587</td>\n",
       "      <td>ENST00000516587.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>ENSG00000252429</td>\n",
       "      <td>ENSG00000252429.2</td>\n",
       "      <td>ENST00000516620</td>\n",
       "      <td>ENST00000516620.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>ENSG00000221643</td>\n",
       "      <td>ENSG00000221643.1</td>\n",
       "      <td>ENST00000408716</td>\n",
       "      <td>ENST00000408716.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>ENSG00000264371</td>\n",
       "      <td>ENSG00000264371.1</td>\n",
       "      <td>ENST00000580572</td>\n",
       "      <td>ENST00000580572.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36496</td>\n",
       "      <td>ENSG00000196290</td>\n",
       "      <td>ENSG00000196290.15</td>\n",
       "      <td>ENST00000409357</td>\n",
       "      <td>ENST00000409357.5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36497</td>\n",
       "      <td>ENSG00000196290</td>\n",
       "      <td>ENSG00000196290.15</td>\n",
       "      <td>ENST00000409129</td>\n",
       "      <td>ENST00000409129.2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36498</td>\n",
       "      <td>ENSG00000196290</td>\n",
       "      <td>ENSG00000196290.15</td>\n",
       "      <td>ENST00000409588</td>\n",
       "      <td>ENST00000409588.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36499</td>\n",
       "      <td>ENSG00000196290</td>\n",
       "      <td>ENSG00000196290.15</td>\n",
       "      <td>ENST00000436412</td>\n",
       "      <td>ENST00000436412.1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36500</td>\n",
       "      <td>ENSG00000196290</td>\n",
       "      <td>ENSG00000196290.15</td>\n",
       "      <td>ENST00000416651</td>\n",
       "      <td>ENST00000416651.5</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>36501 rows × 4 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        Gene stable ID Gene stable ID version Transcript stable ID  \\\n",
       "0      ENSG00000200036      ENSG00000200036.1      ENST00000363166   \n",
       "1      ENSG00000252396      ENSG00000252396.1      ENST00000516587   \n",
       "2      ENSG00000252429      ENSG00000252429.2      ENST00000516620   \n",
       "3      ENSG00000221643      ENSG00000221643.1      ENST00000408716   \n",
       "4      ENSG00000264371      ENSG00000264371.1      ENST00000580572   \n",
       "...                ...                    ...                  ...   \n",
       "36496  ENSG00000196290     ENSG00000196290.15      ENST00000409357   \n",
       "36497  ENSG00000196290     ENSG00000196290.15      ENST00000409129   \n",
       "36498  ENSG00000196290     ENSG00000196290.15      ENST00000409588   \n",
       "36499  ENSG00000196290     ENSG00000196290.15      ENST00000436412   \n",
       "36500  ENSG00000196290     ENSG00000196290.15      ENST00000416651   \n",
       "\n",
       "      Transcript stable ID version  \n",
       "0                ENST00000363166.1  \n",
       "1                ENST00000516587.1  \n",
       "2                ENST00000516620.2  \n",
       "3                ENST00000408716.1  \n",
       "4                ENST00000580572.1  \n",
       "...                            ...  \n",
       "36496            ENST00000409357.5  \n",
       "36497            ENST00000409129.2  \n",
       "36498            ENST00000409588.1  \n",
       "36499            ENST00000436412.1  \n",
       "36500            ENST00000416651.5  \n",
       "\n",
       "[36501 rows x 4 columns]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Retrieving a dataset directly with known dataset name\n",
    "\n",
    "dataset = Dataset(name='hsapiens_gene_ensembl',\n",
    "                  host='http://www.ensembl.org')\n",
    "\n",
    "dataset.query(\n",
    "              filters={'chromosome_name': ['1','2']})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The `attributes` attribute can be used to pull up a list of additional fields available from the dataset"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['ensembl_gene_id',\n",
       " 'ensembl_gene_id_version',\n",
       " 'ensembl_transcript_id',\n",
       " 'ensembl_transcript_id_version',\n",
       " 'ensembl_peptide_id',\n",
       " 'ensembl_peptide_id_version',\n",
       " 'ensembl_exon_id',\n",
       " 'description',\n",
       " 'chromosome_name',\n",
       " 'start_position',\n",
       " 'end_position',\n",
       " 'strand',\n",
       " 'band',\n",
       " 'transcript_start',\n",
       " 'transcript_end',\n",
       " 'transcription_start_site',\n",
       " 'transcript_length',\n",
       " 'transcript_tsl',\n",
       " 'transcript_gencode_basic',\n",
       " 'transcript_appris',\n",
       " 'transcript_mane_select',\n",
       " 'external_gene_name',\n",
       " 'external_gene_source',\n",
       " 'external_transcript_name',\n",
       " 'external_transcript_source_name',\n",
       " 'transcript_count',\n",
       " 'percentage_gene_gc_content',\n",
       " 'gene_biotype',\n",
       " 'transcript_biotype',\n",
       " 'source',\n",
       " 'transcript_source',\n",
       " 'version',\n",
       " 'transcript_version',\n",
       " 'peptide_version',\n",
       " 'phenotype_description',\n",
       " 'Source_name',\n",
       " 'study_external_id',\n",
       " 'strain_name',\n",
       " 'strain_gender',\n",
       " 'p_value',\n",
       " 'go_id',\n",
       " 'name_1006',\n",
       " 'definition_1006',\n",
       " 'go_linkage_type',\n",
       " 'namespace_1003',\n",
       " 'goslim_goa_accession',\n",
       " 'goslim_goa_description',\n",
       " 'ccds',\n",
       " 'chembl',\n",
       " 'clone_based_ensembl_gene',\n",
       " 'clone_based_ensembl_transcript',\n",
       " 'dbass3_name',\n",
       " 'dbass3_id',\n",
       " 'dbass5_name',\n",
       " 'dbass5_id',\n",
       " 'ens_hs_transcript',\n",
       " 'ens_hs_translation',\n",
       " 'entrezgene_trans_name',\n",
       " 'embl',\n",
       " 'arrayexpress',\n",
       " 'genedb',\n",
       " 'hgnc_id',\n",
       " 'hgnc_symbol',\n",
       " 'hgnc_trans_name',\n",
       " 'hpa_accession',\n",
       " 'hpa_id',\n",
       " 'protein_id',\n",
       " 'kegg_enzyme',\n",
       " 'ens_lrg_gene',\n",
       " 'ens_lrg_transcript',\n",
       " 'merops',\n",
       " 'mim_gene_description',\n",
       " 'mim_gene_accession',\n",
       " 'mim_morbid_description',\n",
       " 'mim_morbid_accession',\n",
       " 'mirbase_accession',\n",
       " 'mirbase_id',\n",
       " 'mirbase_trans_name',\n",
       " 'entrezgene_description',\n",
       " 'entrezgene_accession',\n",
       " 'entrezgene_id',\n",
       " 'pdb',\n",
       " 'reactome',\n",
       " 'reactome_gene',\n",
       " 'reactome_transcript',\n",
       " 'refseq_mrna',\n",
       " 'refseq_mrna_predicted',\n",
       " 'refseq_ncrna',\n",
       " 'refseq_ncrna_predicted',\n",
       " 'refseq_peptide',\n",
       " 'refseq_peptide_predicted',\n",
       " 'rfam',\n",
       " 'rfam_trans_name',\n",
       " 'rnacentral',\n",
       " 'ucsc',\n",
       " 'uniparc',\n",
       " 'uniprot_gn_symbol',\n",
       " 'uniprot_gn_id',\n",
       " 'uniprotswissprot',\n",
       " 'uniprotsptrembl',\n",
       " 'wikigene_description',\n",
       " 'wikigene_name',\n",
       " 'wikigene_id',\n",
       " 'affy_hc_g110',\n",
       " 'affy_hg_focus',\n",
       " 'affy_hg_u133a',\n",
       " 'affy_hg_u133a_2',\n",
       " 'affy_hg_u133b',\n",
       " 'affy_hg_u133_plus_2',\n",
       " 'affy_hg_u95a',\n",
       " 'affy_hg_u95av2',\n",
       " 'affy_hg_u95b',\n",
       " 'affy_hg_u95c',\n",
       " 'affy_hg_u95d',\n",
       " 'affy_hg_u95e',\n",
       " 'affy_hta_2_0',\n",
       " 'affy_huex_1_0_st_v2',\n",
       " 'affy_hugenefl',\n",
       " 'affy_hugene_1_0_st_v1',\n",
       " 'affy_hugene_2_0_st_v1',\n",
       " 'affy_primeview',\n",
       " 'affy_u133_x3p',\n",
       " 'agilent_cgh_44b',\n",
       " 'agilent_gpl6848',\n",
       " 'agilent_sureprint_g3_ge_8x60k',\n",
       " 'agilent_sureprint_g3_ge_8x60k_v2',\n",
       " 'agilent_wholegenome',\n",
       " 'agilent_wholegenome_4x44k_v1',\n",
       " 'agilent_wholegenome_4x44k_v2',\n",
       " 'codelink_codelink',\n",
       " 'illumina_humanht_12_v3',\n",
       " 'illumina_humanht_12_v4',\n",
       " 'illumina_humanref_8_v3',\n",
       " 'illumina_humanwg_6_v1',\n",
       " 'illumina_humanwg_6_v2',\n",
       " 'illumina_humanwg_6_v3',\n",
       " 'phalanx_onearray',\n",
       " 'family',\n",
       " 'family_description',\n",
       " 'cdd',\n",
       " 'cdd_start',\n",
       " 'cdd_end',\n",
       " 'gene3d',\n",
       " 'gene3d_start',\n",
       " 'gene3d_end',\n",
       " 'hamap',\n",
       " 'hamap_start',\n",
       " 'hamap_end',\n",
       " 'hmmpanther',\n",
       " 'hmmpanther_start',\n",
       " 'hmmpanther_end',\n",
       " 'pfam',\n",
       " 'pfam_start',\n",
       " 'pfam_end',\n",
       " 'pirsf',\n",
       " 'pirsf_start',\n",
       " 'pirsf_end',\n",
       " 'prints',\n",
       " 'prints_start',\n",
       " 'prints_end',\n",
       " 'blastprodom',\n",
       " 'blastprodom_start',\n",
       " 'blastprodom_end',\n",
       " 'scanprosite',\n",
       " 'scanprosite_start',\n",
       " 'scanprosite_end',\n",
       " 'pfscan',\n",
       " 'pfscan_start',\n",
       " 'pfscan_end',\n",
       " 'sfld',\n",
       " 'sfld_start',\n",
       " 'sfld_end',\n",
       " 'smart',\n",
       " 'smart_start',\n",
       " 'smart_end',\n",
       " 'superfamily',\n",
       " 'superfamily_start',\n",
       " 'superfamily_end',\n",
       " 'tigrfam',\n",
       " 'tigrfam_start',\n",
       " 'tigrfam_end',\n",
       " 'interpro',\n",
       " 'interpro_short_description',\n",
       " 'interpro_description',\n",
       " 'interpro_start',\n",
       " 'interpro_end',\n",
       " 'mobidblite',\n",
       " 'mobidblite_start',\n",
       " 'mobidblite_end',\n",
       " 'ncoils',\n",
       " 'ncoils_start',\n",
       " 'ncoils_end',\n",
       " 'seg',\n",
       " 'seg_start',\n",
       " 'seg_end',\n",
       " 'sifts_import',\n",
       " 'sifts_import_start',\n",
       " 'sifts_import_end',\n",
       " 'signalp',\n",
       " 'signalp_start',\n",
       " 'signalp_end',\n",
       " 'tmhmm',\n",
       " 'tmhmm_start',\n",
       " 'tmhmm_end',\n",
       " 'structure_gene_stable_id',\n",
       " 'structure_gene_stable_id_version',\n",
       " 'structure_gene_version',\n",
       " 'structure_transcript_stable_id',\n",
       " 'structure_transcript_stable_id_version',\n",
       " 'structure_transcript_version',\n",
       " 'structure_translation_stable_id',\n",
       " 'structure_translation_stable_id_version',\n",
       " 'structure_peptide_version',\n",
       " 'structure_canonical_transcript_id',\n",
       " 'structure_chrom_name',\n",
       " 'structure_gene_chrom_start',\n",
       " 'structure_gene_chrom_end',\n",
       " 'structure_transcript_chrom_start',\n",
       " 'structure_transcript_chrom_end',\n",
       " 'structure_transcription_start_site',\n",
       " 'structure_transcript_length',\n",
       " 'structure_transcript_chrom_strand',\n",
       " 'structure_external_gene_name',\n",
       " 'structure_external_source_name',\n",
       " 'structure_5_utr_start',\n",
       " 'structure_5_utr_end',\n",
       " 'structure_3_utr_start',\n",
       " 'structure_3_utr_end',\n",
       " 'structure_cds_length',\n",
       " 'structure_cdna_length',\n",
       " 'structure_peptide_length',\n",
       " 'struct_transcript_count',\n",
       " 'structure_translation_count',\n",
       " 'structure_description',\n",
       " 'structure_biotype',\n",
       " 'structure_ensembl_exon_id',\n",
       " 'exon_chrom_start',\n",
       " 'exon_chrom_end',\n",
       " 'is_constitutive',\n",
       " 'rank',\n",
       " 'phase',\n",
       " 'end_phase',\n",
       " 'cdna_coding_start',\n",
       " 'cdna_coding_end',\n",
       " 'genomic_coding_start',\n",
       " 'genomic_coding_end',\n",
       " 'structure_cds_start',\n",
       " 'structure_cds_end',\n",
       " 'homologs_ensembl_gene_id',\n",
       " 'homologs_gene_stable_id_version',\n",
       " 'homologs_gene_version',\n",
       " 'homologs_ensembl_transcript_id',\n",
       " 'homologs_transcript_stable_id_version',\n",
       " 'homologs_transcript_version',\n",
       " 'homologs_ensembl_peptide_id',\n",
       " 'homologs_translation_stable_id_version',\n",
       " 'homologs_peptide_version',\n",
       " 'homologs_canonical_transcript_id',\n",
       " 'homologs_chromosome_name',\n",
       " 'homologs_start_position',\n",
       " 'homologs_end_position',\n",
       " 'homologs_strand',\n",
       " 'homologs_band',\n",
       " 'homologs_external_gene_name',\n",
       " 'homologs_external_gene_source',\n",
       " 'homologs_ensembl_CDS_length',\n",
       " 'homologs_ensembl_cDNA_length',\n",
       " 'homologs_ensembl_peptide_length',\n",
       " 'homologs_transcript_count',\n",
       " 'homologs_percentage_gc_content',\n",
       " 'homologs_description',\n",
       " 'cabingdonii_homolog_ensembl_gene',\n",
       " 'cabingdonii_homolog_associated_gene_name',\n",
       " 'cabingdonii_homolog_ensembl_peptide',\n",
       " 'cabingdonii_homolog_chromosome',\n",
       " 'cabingdonii_homolog_chrom_start',\n",
       " 'cabingdonii_homolog_chrom_end',\n",
       " 'cabingdonii_homolog_canonical_transcript_protein',\n",
       " 'cabingdonii_homolog_subtype',\n",
       " 'cabingdonii_homolog_orthology_type',\n",
       " 'cabingdonii_homolog_perc_id',\n",
       " 'cabingdonii_homolog_perc_id_r1',\n",
       " 'cabingdonii_homolog_goc_score',\n",
       " 'cabingdonii_homolog_wga_coverage',\n",
       " 'cabingdonii_homolog_dn',\n",
       " 'cabingdonii_homolog_ds',\n",
       " 'cabingdonii_homolog_orthology_confidence',\n",
       " 'gagassizii_homolog_ensembl_gene',\n",
       " 'gagassizii_homolog_associated_gene_name',\n",
       " 'gagassizii_homolog_ensembl_peptide',\n",
       " 'gagassizii_homolog_chromosome',\n",
       " 'gagassizii_homolog_chrom_start',\n",
       " 'gagassizii_homolog_chrom_end',\n",
       " 'gagassizii_homolog_canonical_transcript_protein',\n",
       " 'gagassizii_homolog_subtype',\n",
       " 'gagassizii_homolog_orthology_type',\n",
       " 'gagassizii_homolog_perc_id',\n",
       " 'gagassizii_homolog_perc_id_r1',\n",
       " 'gagassizii_homolog_goc_score',\n",
       " 'gagassizii_homolog_wga_coverage',\n",
       " 'gagassizii_homolog_dn',\n",
       " 'gagassizii_homolog_ds',\n",
       " 'gagassizii_homolog_orthology_confidence',\n",
       " 'mspretus_homolog_ensembl_gene',\n",
       " 'mspretus_homolog_associated_gene_name',\n",
       " 'mspretus_homolog_ensembl_peptide',\n",
       " 'mspretus_homolog_chromosome',\n",
       " 'mspretus_homolog_chrom_start',\n",
       " 'mspretus_homolog_chrom_end',\n",
       " 'mspretus_homolog_canonical_transcript_protein',\n",
       " 'mspretus_homolog_subtype',\n",
       " 'mspretus_homolog_orthology_type',\n",
       " 'mspretus_homolog_perc_id',\n",
       " 'mspretus_homolog_perc_id_r1',\n",
       " 'mspretus_homolog_goc_score',\n",
       " 'mspretus_homolog_wga_coverage',\n",
       " 'mspretus_homolog_dn',\n",
       " 'mspretus_homolog_ds',\n",
       " 'mspretus_homolog_orthology_confidence',\n",
       " 'vpacos_homolog_ensembl_gene',\n",
       " 'vpacos_homolog_associated_gene_name',\n",
       " 'vpacos_homolog_ensembl_peptide',\n",
       " 'vpacos_homolog_chromosome',\n",
       " 'vpacos_homolog_chrom_start',\n",
       " 'vpacos_homolog_chrom_end',\n",
       " 'vpacos_homolog_canonical_transcript_protein',\n",
       " 'vpacos_homolog_subtype',\n",
       " 'vpacos_homolog_orthology_type',\n",
       " 'vpacos_homolog_perc_id',\n",
       " 'vpacos_homolog_perc_id_r1',\n",
       " 'vpacos_homolog_goc_score',\n",
       " 'vpacos_homolog_wga_coverage',\n",
       " 'vpacos_homolog_dn',\n",
       " 'vpacos_homolog_ds',\n",
       " 'vpacos_homolog_orthology_confidence',\n",
       " 'mmarmota_homolog_ensembl_gene',\n",
       " 'mmarmota_homolog_associated_gene_name',\n",
       " 'mmarmota_homolog_ensembl_peptide',\n",
       " 'mmarmota_homolog_chromosome',\n",
       " 'mmarmota_homolog_chrom_start',\n",
       " 'mmarmota_homolog_chrom_end',\n",
       " 'mmarmota_homolog_canonical_transcript_protein',\n",
       " 'mmarmota_homolog_subtype',\n",
       " 'mmarmota_homolog_orthology_type',\n",
       " 'mmarmota_homolog_perc_id',\n",
       " 'mmarmota_homolog_perc_id_r1',\n",
       " 'mmarmota_homolog_goc_score',\n",
       " 'mmarmota_homolog_wga_coverage',\n",
       " 'mmarmota_homolog_dn',\n",
       " 'mmarmota_homolog_ds',\n",
       " 'mmarmota_homolog_orthology_confidence',\n",
       " 'pformosa_homolog_ensembl_gene',\n",
       " 'pformosa_homolog_associated_gene_name',\n",
       " 'pformosa_homolog_ensembl_peptide',\n",
       " 'pformosa_homolog_chromosome',\n",
       " 'pformosa_homolog_chrom_start',\n",
       " 'pformosa_homolog_chrom_end',\n",
       " 'pformosa_homolog_canonical_transcript_protein',\n",
       " 'pformosa_homolog_subtype',\n",
       " 'pformosa_homolog_orthology_type',\n",
       " 'pformosa_homolog_perc_id',\n",
       " 'pformosa_homolog_perc_id_r1',\n",
       " 'pformosa_homolog_goc_score',\n",
       " 'pformosa_homolog_wga_coverage',\n",
       " 'pformosa_homolog_dn',\n",
       " 'pformosa_homolog_ds',\n",
       " 'pformosa_homolog_orthology_confidence',\n",
       " 'ccanadensis_homolog_ensembl_gene',\n",
       " 'ccanadensis_homolog_associated_gene_name',\n",
       " 'ccanadensis_homolog_ensembl_peptide',\n",
       " 'ccanadensis_homolog_chromosome',\n",
       " 'ccanadensis_homolog_chrom_start',\n",
       " 'ccanadensis_homolog_chrom_end',\n",
       " 'ccanadensis_homolog_canonical_transcript_protein',\n",
       " 'ccanadensis_homolog_subtype',\n",
       " 'ccanadensis_homolog_orthology_type',\n",
       " 'ccanadensis_homolog_perc_id',\n",
       " 'ccanadensis_homolog_perc_id_r1',\n",
       " 'ccanadensis_homolog_goc_score',\n",
       " 'ccanadensis_homolog_wga_coverage',\n",
       " 'ccanadensis_homolog_dn',\n",
       " 'ccanadensis_homolog_ds',\n",
       " 'ccanadensis_homolog_orthology_confidence',\n",
       " 'bbison_homolog_ensembl_gene',\n",
       " 'bbison_homolog_associated_gene_name',\n",
       " 'bbison_homolog_ensembl_peptide',\n",
       " 'bbison_homolog_chromosome',\n",
       " 'bbison_homolog_chrom_start',\n",
       " 'bbison_homolog_chrom_end',\n",
       " 'bbison_homolog_canonical_transcript_protein',\n",
       " 'bbison_homolog_subtype',\n",
       " 'bbison_homolog_orthology_type',\n",
       " 'bbison_homolog_perc_id',\n",
       " 'bbison_homolog_perc_id_r1',\n",
       " 'bbison_homolog_goc_score',\n",
       " 'bbison_homolog_wga_coverage',\n",
       " 'bbison_homolog_dn',\n",
       " 'bbison_homolog_ds',\n",
       " 'bbison_homolog_orthology_confidence',\n",
       " 'uamericanus_homolog_ensembl_gene',\n",
       " 'uamericanus_homolog_associated_gene_name',\n",
       " 'uamericanus_homolog_ensembl_peptide',\n",
       " 'uamericanus_homolog_chromosome',\n",
       " 'uamericanus_homolog_chrom_start',\n",
       " 'uamericanus_homolog_chrom_end',\n",
       " 'uamericanus_homolog_canonical_transcript_protein',\n",
       " 'uamericanus_homolog_subtype',\n",
       " 'uamericanus_homolog_orthology_type',\n",
       " 'uamericanus_homolog_perc_id',\n",
       " 'uamericanus_homolog_perc_id_r1',\n",
       " 'uamericanus_homolog_goc_score',\n",
       " 'uamericanus_homolog_wga_coverage',\n",
       " 'uamericanus_homolog_dn',\n",
       " 'uamericanus_homolog_ds',\n",
       " 'uamericanus_homolog_orthology_confidence',\n",
       " 'nvison_homolog_ensembl_gene',\n",
       " 'nvison_homolog_associated_gene_name',\n",
       " 'nvison_homolog_ensembl_peptide',\n",
       " 'nvison_homolog_chromosome',\n",
       " 'nvison_homolog_chrom_start',\n",
       " 'nvison_homolog_chrom_end',\n",
       " 'nvison_homolog_canonical_transcript_protein',\n",
       " 'nvison_homolog_subtype',\n",
       " 'nvison_homolog_orthology_type',\n",
       " 'nvison_homolog_perc_id',\n",
       " 'nvison_homolog_perc_id_r1',\n",
       " 'nvison_homolog_goc_score',\n",
       " 'nvison_homolog_wga_coverage',\n",
       " 'nvison_homolog_dn',\n",
       " 'nvison_homolog_ds',\n",
       " 'nvison_homolog_orthology_confidence',\n",
       " 'cpalliatus_homolog_ensembl_gene',\n",
       " 'cpalliatus_homolog_associated_gene_name',\n",
       " 'cpalliatus_homolog_ensembl_peptide',\n",
       " 'cpalliatus_homolog_chromosome',\n",
       " 'cpalliatus_homolog_chrom_start',\n",
       " 'cpalliatus_homolog_chrom_end',\n",
       " 'cpalliatus_homolog_canonical_transcript_protein',\n",
       " 'cpalliatus_homolog_subtype',\n",
       " 'cpalliatus_homolog_orthology_type',\n",
       " 'cpalliatus_homolog_perc_id',\n",
       " 'cpalliatus_homolog_perc_id_r1',\n",
       " 'cpalliatus_homolog_goc_score',\n",
       " 'cpalliatus_homolog_wga_coverage',\n",
       " 'cpalliatus_homolog_dn',\n",
       " 'cpalliatus_homolog_ds',\n",
       " 'cpalliatus_homolog_orthology_confidence',\n",
       " 'acarolinensis_homolog_ensembl_gene',\n",
       " 'acarolinensis_homolog_associated_gene_name',\n",
       " 'acarolinensis_homolog_ensembl_peptide',\n",
       " 'acarolinensis_homolog_chromosome',\n",
       " 'acarolinensis_homolog_chrom_start',\n",
       " 'acarolinensis_homolog_chrom_end',\n",
       " 'acarolinensis_homolog_canonical_transcript_protein',\n",
       " 'acarolinensis_homolog_subtype',\n",
       " 'acarolinensis_homolog_orthology_type',\n",
       " 'acarolinensis_homolog_perc_id',\n",
       " 'acarolinensis_homolog_perc_id_r1',\n",
       " 'acarolinensis_homolog_goc_score',\n",
       " 'acarolinensis_homolog_wga_coverage',\n",
       " 'acarolinensis_homolog_dn',\n",
       " 'acarolinensis_homolog_ds',\n",
       " 'acarolinensis_homolog_orthology_confidence',\n",
       " 'uparryii_homolog_ensembl_gene',\n",
       " 'uparryii_homolog_associated_gene_name',\n",
       " 'uparryii_homolog_ensembl_peptide',\n",
       " 'uparryii_homolog_chromosome',\n",
       " 'uparryii_homolog_chrom_start',\n",
       " 'uparryii_homolog_chrom_end',\n",
       " 'uparryii_homolog_canonical_transcript_protein',\n",
       " 'uparryii_homolog_subtype',\n",
       " 'uparryii_homolog_orthology_type',\n",
       " 'uparryii_homolog_perc_id',\n",
       " 'uparryii_homolog_perc_id_r1',\n",
       " 'uparryii_homolog_goc_score',\n",
       " 'uparryii_homolog_wga_coverage',\n",
       " 'uparryii_homolog_dn',\n",
       " 'uparryii_homolog_ds',\n",
       " 'uparryii_homolog_orthology_confidence',\n",
       " 'smerianae_homolog_ensembl_gene',\n",
       " 'smerianae_homolog_associated_gene_name',\n",
       " 'smerianae_homolog_ensembl_peptide',\n",
       " 'smerianae_homolog_chromosome',\n",
       " 'smerianae_homolog_chrom_start',\n",
       " 'smerianae_homolog_chrom_end',\n",
       " 'smerianae_homolog_canonical_transcript_protein',\n",
       " 'smerianae_homolog_subtype',\n",
       " 'smerianae_homolog_orthology_type',\n",
       " 'smerianae_homolog_perc_id',\n",
       " 'smerianae_homolog_perc_id_r1',\n",
       " 'smerianae_homolog_goc_score',\n",
       " 'smerianae_homolog_wga_coverage',\n",
       " 'smerianae_homolog_dn',\n",
       " 'smerianae_homolog_ds',\n",
       " 'smerianae_homolog_orthology_confidence',\n",
       " 'dnovemcinctus_homolog_ensembl_gene',\n",
       " 'dnovemcinctus_homolog_associated_gene_name',\n",
       " 'dnovemcinctus_homolog_ensembl_peptide',\n",
       " 'dnovemcinctus_homolog_chromosome',\n",
       " 'dnovemcinctus_homolog_chrom_start',\n",
       " 'dnovemcinctus_homolog_chrom_end',\n",
       " 'dnovemcinctus_homolog_canonical_transcript_protein',\n",
       " 'dnovemcinctus_homolog_subtype',\n",
       " 'dnovemcinctus_homolog_orthology_type',\n",
       " 'dnovemcinctus_homolog_perc_id',\n",
       " 'dnovemcinctus_homolog_perc_id_r1',\n",
       " 'dnovemcinctus_homolog_goc_score',\n",
       " 'dnovemcinctus_homolog_wga_coverage',\n",
       " 'dnovemcinctus_homolog_dn',\n",
       " 'dnovemcinctus_homolog_ds',\n",
       " 'dnovemcinctus_homolog_orthology_confidence',\n",
       " 'sformosus_homolog_ensembl_gene',\n",
       " 'sformosus_homolog_associated_gene_name',\n",
       " 'sformosus_homolog_ensembl_peptide',\n",
       " 'sformosus_homolog_chromosome',\n",
       " 'sformosus_homolog_chrom_start',\n",
       " 'sformosus_homolog_chrom_end',\n",
       " 'sformosus_homolog_canonical_transcript_protein',\n",
       " 'sformosus_homolog_subtype',\n",
       " 'sformosus_homolog_orthology_type',\n",
       " 'sformosus_homolog_perc_id',\n",
       " 'sformosus_homolog_perc_id_r1',\n",
       " 'sformosus_homolog_goc_score',\n",
       " 'sformosus_homolog_wga_coverage',\n",
       " 'sformosus_homolog_dn',\n",
       " 'sformosus_homolog_ds',\n",
       " 'sformosus_homolog_orthology_confidence',\n",
       " 'cporosus_homolog_ensembl_gene',\n",
       " 'cporosus_homolog_associated_gene_name',\n",
       " 'cporosus_homolog_ensembl_peptide',\n",
       " 'cporosus_homolog_chromosome',\n",
       " 'cporosus_homolog_chrom_start',\n",
       " 'cporosus_homolog_chrom_end',\n",
       " 'cporosus_homolog_canonical_transcript_protein',\n",
       " 'cporosus_homolog_subtype',\n",
       " 'cporosus_homolog_orthology_type',\n",
       " 'cporosus_homolog_perc_id',\n",
       " 'cporosus_homolog_perc_id_r1',\n",
       " 'cporosus_homolog_goc_score',\n",
       " 'cporosus_homolog_wga_coverage',\n",
       " 'cporosus_homolog_dn',\n",
       " 'cporosus_homolog_ds',\n",
       " 'cporosus_homolog_orthology_confidence',\n",
       " 'lbergylta_homolog_ensembl_gene',\n",
       " 'lbergylta_homolog_associated_gene_name',\n",
       " 'lbergylta_homolog_ensembl_peptide',\n",
       " 'lbergylta_homolog_chromosome',\n",
       " 'lbergylta_homolog_chrom_start',\n",
       " 'lbergylta_homolog_chrom_end',\n",
       " 'lbergylta_homolog_canonical_transcript_protein',\n",
       " 'lbergylta_homolog_subtype',\n",
       " 'lbergylta_homolog_orthology_type',\n",
       " 'lbergylta_homolog_perc_id',\n",
       " 'lbergylta_homolog_perc_id_r1',\n",
       " 'lbergylta_homolog_goc_score',\n",
       " 'lbergylta_homolog_wga_coverage',\n",
       " 'lbergylta_homolog_dn',\n",
       " 'lbergylta_homolog_ds',\n",
       " 'lbergylta_homolog_orthology_confidence',\n",
       " 'lcalcarifer_homolog_ensembl_gene',\n",
       " 'lcalcarifer_homolog_associated_gene_name',\n",
       " 'lcalcarifer_homolog_ensembl_peptide',\n",
       " 'lcalcarifer_homolog_chromosome',\n",
       " 'lcalcarifer_homolog_chrom_start',\n",
       " 'lcalcarifer_homolog_chrom_end',\n",
       " 'lcalcarifer_homolog_canonical_transcript_protein',\n",
       " 'lcalcarifer_homolog_subtype',\n",
       " 'lcalcarifer_homolog_orthology_type',\n",
       " 'lcalcarifer_homolog_perc_id',\n",
       " 'lcalcarifer_homolog_perc_id_r1',\n",
       " 'lcalcarifer_homolog_goc_score',\n",
       " 'lcalcarifer_homolog_wga_coverage',\n",
       " 'lcalcarifer_homolog_dn',\n",
       " 'lcalcarifer_homolog_ds',\n",
       " 'lcalcarifer_homolog_orthology_confidence',\n",
       " 'ldomestica_homolog_ensembl_gene',\n",
       " 'ldomestica_homolog_associated_gene_name',\n",
       " 'ldomestica_homolog_ensembl_peptide',\n",
       " 'ldomestica_homolog_chromosome',\n",
       " 'ldomestica_homolog_chrom_start',\n",
       " 'ldomestica_homolog_chrom_end',\n",
       " 'ldomestica_homolog_canonical_transcript_protein',\n",
       " 'ldomestica_homolog_subtype',\n",
       " 'ldomestica_homolog_orthology_type',\n",
       " 'ldomestica_homolog_perc_id',\n",
       " 'ldomestica_homolog_perc_id_r1',\n",
       " 'ldomestica_homolog_goc_score',\n",
       " 'ldomestica_homolog_wga_coverage',\n",
       " 'ldomestica_homolog_dn',\n",
       " 'ldomestica_homolog_ds',\n",
       " 'ldomestica_homolog_orthology_confidence',\n",
       " 'spartitus_homolog_ensembl_gene',\n",
       " 'spartitus_homolog_associated_gene_name',\n",
       " 'spartitus_homolog_ensembl_peptide',\n",
       " 'spartitus_homolog_chromosome',\n",
       " 'spartitus_homolog_chrom_start',\n",
       " 'spartitus_homolog_chrom_end',\n",
       " 'spartitus_homolog_canonical_transcript_protein',\n",
       " 'spartitus_homolog_subtype',\n",
       " 'spartitus_homolog_orthology_type',\n",
       " 'spartitus_homolog_perc_id',\n",
       " 'spartitus_homolog_perc_id_r1',\n",
       " 'spartitus_homolog_goc_score',\n",
       " 'spartitus_homolog_wga_coverage',\n",
       " 'spartitus_homolog_dn',\n",
       " 'spartitus_homolog_ds',\n",
       " 'spartitus_homolog_orthology_confidence',\n",
       " 'rbieti_homolog_ensembl_gene',\n",
       " 'rbieti_homolog_associated_gene_name',\n",
       " 'rbieti_homolog_ensembl_peptide',\n",
       " 'rbieti_homolog_chromosome',\n",
       " 'rbieti_homolog_chrom_start',\n",
       " 'rbieti_homolog_chrom_end',\n",
       " 'rbieti_homolog_canonical_transcript_protein',\n",
       " 'rbieti_homolog_subtype',\n",
       " 'rbieti_homolog_orthology_type',\n",
       " 'rbieti_homolog_perc_id',\n",
       " 'rbieti_homolog_perc_id_r1',\n",
       " 'rbieti_homolog_goc_score',\n",
       " 'rbieti_homolog_wga_coverage',\n",
       " 'rbieti_homolog_dn',\n",
       " 'rbieti_homolog_ds',\n",
       " 'rbieti_homolog_orthology_confidence',\n",
       " 'ccaeruleus_homolog_ensembl_gene',\n",
       " 'ccaeruleus_homolog_associated_gene_name',\n",
       " 'ccaeruleus_homolog_ensembl_peptide',\n",
       " 'ccaeruleus_homolog_chromosome',\n",
       " 'ccaeruleus_homolog_chrom_start',\n",
       " 'ccaeruleus_homolog_chrom_end',\n",
       " 'ccaeruleus_homolog_canonical_transcript_protein',\n",
       " 'ccaeruleus_homolog_subtype',\n",
       " 'ccaeruleus_homolog_orthology_type',\n",
       " 'ccaeruleus_homolog_perc_id',\n",
       " 'ccaeruleus_homolog_perc_id_r1',\n",
       " 'ccaeruleus_homolog_goc_score',\n",
       " 'ccaeruleus_homolog_wga_coverage',\n",
       " 'ccaeruleus_homolog_dn',\n",
       " 'ccaeruleus_homolog_ds',\n",
       " 'ccaeruleus_homolog_orthology_confidence',\n",
       " 'lcoronata_homolog_ensembl_gene',\n",
       " 'lcoronata_homolog_associated_gene_name',\n",
       " 'lcoronata_homolog_ensembl_peptide',\n",
       " 'lcoronata_homolog_chromosome',\n",
       " 'lcoronata_homolog_chrom_start',\n",
       " 'lcoronata_homolog_chrom_end',\n",
       " 'lcoronata_homolog_canonical_transcript_protein',\n",
       " 'lcoronata_homolog_subtype',\n",
       " 'lcoronata_homolog_orthology_type',\n",
       " 'lcoronata_homolog_perc_id',\n",
       " 'lcoronata_homolog_perc_id_r1',\n",
       " 'lcoronata_homolog_goc_score',\n",
       " 'lcoronata_homolog_wga_coverage',\n",
       " 'lcoronata_homolog_dn',\n",
       " 'lcoronata_homolog_ds',\n",
       " 'lcoronata_homolog_orthology_confidence',\n",
       " 'sboliviensis_homolog_ensembl_gene',\n",
       " 'sboliviensis_homolog_associated_gene_name',\n",
       " 'sboliviensis_homolog_ensembl_peptide',\n",
       " 'sboliviensis_homolog_chromosome',\n",
       " 'sboliviensis_homolog_chrom_start',\n",
       " 'sboliviensis_homolog_chrom_end',\n",
       " 'sboliviensis_homolog_canonical_transcript_protein',\n",
       " 'sboliviensis_homolog_subtype',\n",
       " 'sboliviensis_homolog_orthology_type',\n",
       " 'sboliviensis_homolog_perc_id',\n",
       " 'sboliviensis_homolog_perc_id_r1',\n",
       " 'sboliviensis_homolog_goc_score',\n",
       " 'sboliviensis_homolog_wga_coverage',\n",
       " 'sboliviensis_homolog_dn',\n",
       " 'sboliviensis_homolog_ds',\n",
       " 'sboliviensis_homolog_orthology_confidence',\n",
       " 'ppaniscus_homolog_ensembl_gene',\n",
       " 'ppaniscus_homolog_associated_gene_name',\n",
       " 'ppaniscus_homolog_ensembl_peptide',\n",
       " 'ppaniscus_homolog_chromosome',\n",
       " 'ppaniscus_homolog_chrom_start',\n",
       " 'ppaniscus_homolog_chrom_end',\n",
       " 'ppaniscus_homolog_canonical_transcript_protein',\n",
       " 'ppaniscus_homolog_subtype',\n",
       " 'ppaniscus_homolog_orthology_type',\n",
       " 'ppaniscus_homolog_perc_id',\n",
       " 'ppaniscus_homolog_perc_id_r1',\n",
       " 'ppaniscus_homolog_goc_score',\n",
       " 'ppaniscus_homolog_wga_coverage',\n",
       " 'ppaniscus_homolog_dn',\n",
       " 'ppaniscus_homolog_ds',\n",
       " 'ppaniscus_homolog_orthology_confidence',\n",
       " 'caperea_homolog_ensembl_gene',\n",
       " 'caperea_homolog_associated_gene_name',\n",
       " 'caperea_homolog_ensembl_peptide',\n",
       " 'caperea_homolog_chromosome',\n",
       " 'caperea_homolog_chrom_start',\n",
       " 'caperea_homolog_chrom_end',\n",
       " 'caperea_homolog_canonical_transcript_protein',\n",
       " 'caperea_homolog_subtype',\n",
       " 'caperea_homolog_orthology_type',\n",
       " 'caperea_homolog_perc_id',\n",
       " 'caperea_homolog_perc_id_r1',\n",
       " 'caperea_homolog_goc_score',\n",
       " 'caperea_homolog_wga_coverage',\n",
       " 'caperea_homolog_dn',\n",
       " 'caperea_homolog_ds',\n",
       " 'caperea_homolog_orthology_confidence',\n",
       " 'mundulatus_homolog_ensembl_gene',\n",
       " 'mundulatus_homolog_associated_gene_name',\n",
       " 'mundulatus_homolog_ensembl_peptide',\n",
       " 'mundulatus_homolog_chromosome',\n",
       " 'mundulatus_homolog_chrom_start',\n",
       " 'mundulatus_homolog_chrom_end',\n",
       " 'mundulatus_homolog_canonical_transcript_protein',\n",
       " 'mundulatus_homolog_subtype',\n",
       " 'mundulatus_homolog_orthology_type',\n",
       " 'mundulatus_homolog_perc_id',\n",
       " 'mundulatus_homolog_perc_id_r1',\n",
       " 'mundulatus_homolog_goc_score',\n",
       " 'mundulatus_homolog_wga_coverage',\n",
       " 'mundulatus_homolog_dn',\n",
       " 'mundulatus_homolog_ds',\n",
       " 'mundulatus_homolog_orthology_confidence',\n",
       " 'hburtoni_homolog_ensembl_gene',\n",
       " 'hburtoni_homolog_associated_gene_name',\n",
       " 'hburtoni_homolog_ensembl_peptide',\n",
       " 'hburtoni_homolog_chromosome',\n",
       " 'hburtoni_homolog_chrom_start',\n",
       " 'hburtoni_homolog_chrom_end',\n",
       " 'hburtoni_homolog_canonical_transcript_protein',\n",
       " 'hburtoni_homolog_subtype',\n",
       " 'hburtoni_homolog_orthology_type',\n",
       " 'hburtoni_homolog_perc_id',\n",
       " 'hburtoni_homolog_perc_id_r1',\n",
       " 'hburtoni_homolog_goc_score',\n",
       " 'hburtoni_homolog_wga_coverage',\n",
       " 'hburtoni_homolog_dn',\n",
       " 'hburtoni_homolog_ds',\n",
       " 'hburtoni_homolog_orthology_confidence',\n",
       " 'ogarnettii_homolog_ensembl_gene',\n",
       " 'ogarnettii_homolog_associated_gene_name',\n",
       " 'ogarnettii_homolog_ensembl_peptide',\n",
       " 'ogarnettii_homolog_chromosome',\n",
       " 'ogarnettii_homolog_chrom_start',\n",
       " 'ogarnettii_homolog_chrom_end',\n",
       " 'ogarnettii_homolog_canonical_transcript_protein',\n",
       " 'ogarnettii_homolog_subtype',\n",
       " 'ogarnettii_homolog_orthology_type',\n",
       " 'ogarnettii_homolog_perc_id',\n",
       " 'ogarnettii_homolog_perc_id_r1',\n",
       " 'ogarnettii_homolog_goc_score',\n",
       " 'ogarnettii_homolog_wga_coverage',\n",
       " 'ogarnettii_homolog_dn',\n",
       " 'ogarnettii_homolog_ds',\n",
       " 'ogarnettii_homolog_orthology_confidence',\n",
       " 'cintestinalis_homolog_ensembl_gene',\n",
       " 'cintestinalis_homolog_associated_gene_name',\n",
       " 'cintestinalis_homolog_ensembl_peptide',\n",
       " 'cintestinalis_homolog_chromosome',\n",
       " 'cintestinalis_homolog_chrom_start',\n",
       " 'cintestinalis_homolog_chrom_end',\n",
       " 'cintestinalis_homolog_canonical_transcript_protein',\n",
       " 'cintestinalis_homolog_subtype',\n",
       " 'cintestinalis_homolog_orthology_type',\n",
       " 'cintestinalis_homolog_perc_id',\n",
       " 'cintestinalis_homolog_perc_id_r1',\n",
       " 'cintestinalis_homolog_goc_score',\n",
       " 'cintestinalis_homolog_wga_coverage',\n",
       " 'cintestinalis_homolog_dn',\n",
       " 'cintestinalis_homolog_ds',\n",
       " 'cintestinalis_homolog_orthology_confidence',\n",
       " 'csavignyi_homolog_ensembl_gene',\n",
       " 'csavignyi_homolog_associated_gene_name',\n",
       " 'csavignyi_homolog_ensembl_peptide',\n",
       " 'csavignyi_homolog_chromosome',\n",
       " 'csavignyi_homolog_chrom_start',\n",
       " 'csavignyi_homolog_chrom_end',\n",
       " 'csavignyi_homolog_canonical_transcript_protein',\n",
       " 'csavignyi_homolog_subtype',\n",
       " 'csavignyi_homolog_orthology_type',\n",
       " 'csavignyi_homolog_perc_id',\n",
       " 'csavignyi_homolog_perc_id_r1',\n",
       " 'csavignyi_homolog_goc_score',\n",
       " 'csavignyi_homolog_wga_coverage',\n",
       " 'csavignyi_homolog_dn',\n",
       " 'csavignyi_homolog_ds',\n",
       " 'csavignyi_homolog_orthology_confidence',\n",
       " 'celegans_homolog_ensembl_gene',\n",
       " 'celegans_homolog_associated_gene_name',\n",
       " 'celegans_homolog_ensembl_peptide',\n",
       " 'celegans_homolog_chromosome',\n",
       " 'celegans_homolog_chrom_start',\n",
       " 'celegans_homolog_chrom_end',\n",
       " 'celegans_homolog_canonical_transcript_protein',\n",
       " 'celegans_homolog_subtype',\n",
       " 'celegans_homolog_orthology_type',\n",
       " 'celegans_homolog_perc_id',\n",
       " 'celegans_homolog_perc_id_r1',\n",
       " 'celegans_homolog_goc_score',\n",
       " 'celegans_homolog_wga_coverage',\n",
       " 'celegans_homolog_dn',\n",
       " 'celegans_homolog_ds',\n",
       " 'celegans_homolog_orthology_confidence',\n",
       " 'ccapucinus_homolog_ensembl_gene',\n",
       " 'ccapucinus_homolog_associated_gene_name',\n",
       " 'ccapucinus_homolog_ensembl_peptide',\n",
       " 'ccapucinus_homolog_chromosome',\n",
       " 'ccapucinus_homolog_chrom_start',\n",
       " 'ccapucinus_homolog_chrom_end',\n",
       " 'ccapucinus_homolog_canonical_transcript_protein',\n",
       " 'ccapucinus_homolog_subtype',\n",
       " 'ccapucinus_homolog_orthology_type',\n",
       " 'ccapucinus_homolog_perc_id',\n",
       " 'ccapucinus_homolog_perc_id_r1',\n",
       " 'ccapucinus_homolog_goc_score',\n",
       " 'ccapucinus_homolog_wga_coverage',\n",
       " 'ccapucinus_homolog_dn',\n",
       " 'ccapucinus_homolog_ds',\n",
       " 'ccapucinus_homolog_orthology_confidence',\n",
       " 'fcatus_homolog_ensembl_gene',\n",
       " 'fcatus_homolog_associated_gene_name',\n",
       " 'fcatus_homolog_ensembl_peptide',\n",
       " 'fcatus_homolog_chromosome',\n",
       " 'fcatus_homolog_chrom_start',\n",
       " 'fcatus_homolog_chrom_end',\n",
       " 'fcatus_homolog_canonical_transcript_protein',\n",
       " 'fcatus_homolog_subtype',\n",
       " 'fcatus_homolog_orthology_type',\n",
       " 'fcatus_homolog_perc_id',\n",
       " 'fcatus_homolog_perc_id_r1',\n",
       " 'fcatus_homolog_goc_score',\n",
       " 'fcatus_homolog_wga_coverage',\n",
       " 'fcatus_homolog_dn',\n",
       " 'fcatus_homolog_ds',\n",
       " 'fcatus_homolog_orthology_confidence',\n",
       " 'pvitticeps_homolog_ensembl_gene',\n",
       " 'pvitticeps_homolog_associated_gene_name',\n",
       " 'pvitticeps_homolog_ensembl_peptide',\n",
       " 'pvitticeps_homolog_chromosome',\n",
       " 'pvitticeps_homolog_chrom_start',\n",
       " 'pvitticeps_homolog_chrom_end',\n",
       " 'pvitticeps_homolog_canonical_transcript_protein',\n",
       " 'pvitticeps_homolog_subtype',\n",
       " 'pvitticeps_homolog_orthology_type',\n",
       " 'pvitticeps_homolog_perc_id',\n",
       " 'pvitticeps_homolog_perc_id_r1',\n",
       " 'pvitticeps_homolog_goc_score',\n",
       " 'pvitticeps_homolog_wga_coverage',\n",
       " 'pvitticeps_homolog_dn',\n",
       " 'pvitticeps_homolog_ds',\n",
       " 'pvitticeps_homolog_orthology_confidence',\n",
       " 'ipunctatus_homolog_ensembl_gene',\n",
       " 'ipunctatus_homolog_associated_gene_name',\n",
       " 'ipunctatus_homolog_ensembl_peptide',\n",
       " 'ipunctatus_homolog_chromosome',\n",
       " 'ipunctatus_homolog_chrom_start',\n",
       " 'ipunctatus_homolog_chrom_end',\n",
       " 'ipunctatus_homolog_canonical_transcript_protein',\n",
       " 'ipunctatus_homolog_subtype',\n",
       " 'ipunctatus_homolog_orthology_type',\n",
       " 'ipunctatus_homolog_perc_id',\n",
       " 'ipunctatus_homolog_perc_id_r1',\n",
       " 'ipunctatus_homolog_goc_score',\n",
       " 'ipunctatus_homolog_wga_coverage',\n",
       " 'ipunctatus_homolog_dn',\n",
       " 'ipunctatus_homolog_ds',\n",
       " 'ipunctatus_homolog_orthology_confidence',\n",
       " 'ggallus_homolog_ensembl_gene',\n",
       " 'ggallus_homolog_associated_gene_name',\n",
       " 'ggallus_homolog_ensembl_peptide',\n",
       " 'ggallus_homolog_chromosome',\n",
       " 'ggallus_homolog_chrom_start',\n",
       " 'ggallus_homolog_chrom_end',\n",
       " 'ggallus_homolog_canonical_transcript_protein',\n",
       " 'ggallus_homolog_subtype',\n",
       " 'ggallus_homolog_orthology_type',\n",
       " 'ggallus_homolog_perc_id',\n",
       " 'ggallus_homolog_perc_id_r1',\n",
       " 'ggallus_homolog_goc_score',\n",
       " 'ggallus_homolog_wga_coverage',\n",
       " 'ggallus_homolog_dn',\n",
       " 'ggallus_homolog_ds',\n",
       " 'ggallus_homolog_orthology_confidence',\n",
       " 'nperdicaria_homolog_ensembl_gene',\n",
       " 'nperdicaria_homolog_associated_gene_name',\n",
       " 'nperdicaria_homolog_ensembl_peptide',\n",
       " 'nperdicaria_homolog_chromosome',\n",
       " 'nperdicaria_homolog_chrom_start',\n",
       " 'nperdicaria_homolog_chrom_end',\n",
       " 'nperdicaria_homolog_canonical_transcript_protein',\n",
       " 'nperdicaria_homolog_subtype',\n",
       " 'nperdicaria_homolog_orthology_type',\n",
       " 'nperdicaria_homolog_perc_id',\n",
       " 'nperdicaria_homolog_perc_id_r1',\n",
       " 'nperdicaria_homolog_goc_score',\n",
       " 'nperdicaria_homolog_wga_coverage',\n",
       " 'nperdicaria_homolog_dn',\n",
       " 'nperdicaria_homolog_ds',\n",
       " 'nperdicaria_homolog_orthology_confidence',\n",
       " 'ptroglodytes_homolog_ensembl_gene',\n",
       " 'ptroglodytes_homolog_associated_gene_name',\n",
       " 'ptroglodytes_homolog_ensembl_peptide',\n",
       " 'ptroglodytes_homolog_chromosome',\n",
       " 'ptroglodytes_homolog_chrom_start',\n",
       " 'ptroglodytes_homolog_chrom_end',\n",
       " 'ptroglodytes_homolog_canonical_transcript_protein',\n",
       " 'ptroglodytes_homolog_subtype',\n",
       " 'ptroglodytes_homolog_orthology_type',\n",
       " 'ptroglodytes_homolog_perc_id',\n",
       " 'ptroglodytes_homolog_perc_id_r1',\n",
       " 'ptroglodytes_homolog_goc_score',\n",
       " 'ptroglodytes_homolog_wga_coverage',\n",
       " 'ptroglodytes_homolog_dn',\n",
       " 'ptroglodytes_homolog_ds',\n",
       " 'ptroglodytes_homolog_orthology_confidence',\n",
       " 'cchok1gshd_homolog_ensembl_gene',\n",
       " 'cchok1gshd_homolog_associated_gene_name',\n",
       " 'cchok1gshd_homolog_ensembl_peptide',\n",
       " 'cchok1gshd_homolog_chromosome',\n",
       " 'cchok1gshd_homolog_chrom_start',\n",
       " 'cchok1gshd_homolog_chrom_end',\n",
       " 'cchok1gshd_homolog_canonical_transcript_protein',\n",
       " 'cchok1gshd_homolog_subtype',\n",
       " 'cchok1gshd_homolog_orthology_type',\n",
       " 'cchok1gshd_homolog_perc_id',\n",
       " 'cchok1gshd_homolog_perc_id_r1',\n",
       " 'cchok1gshd_homolog_goc_score',\n",
       " 'cchok1gshd_homolog_wga_coverage',\n",
       " 'cchok1gshd_homolog_dn',\n",
       " 'cchok1gshd_homolog_ds',\n",
       " 'cchok1gshd_homolog_orthology_confidence',\n",
       " 'ccrigri_homolog_ensembl_gene',\n",
       " 'ccrigri_homolog_associated_gene_name',\n",
       " 'ccrigri_homolog_ensembl_peptide',\n",
       " 'ccrigri_homolog_chromosome',\n",
       " 'ccrigri_homolog_chrom_start',\n",
       " 'ccrigri_homolog_chrom_end',\n",
       " 'ccrigri_homolog_canonical_transcript_protein',\n",
       " 'ccrigri_homolog_subtype',\n",
       " 'ccrigri_homolog_orthology_type',\n",
       " 'ccrigri_homolog_perc_id',\n",
       " 'ccrigri_homolog_perc_id_r1',\n",
       " 'ccrigri_homolog_goc_score',\n",
       " 'ccrigri_homolog_wga_coverage',\n",
       " 'ccrigri_homolog_dn',\n",
       " 'ccrigri_homolog_ds',\n",
       " 'ccrigri_homolog_orthology_confidence',\n",
       " 'cpicr_homolog_ensembl_gene',\n",
       " 'cpicr_homolog_associated_gene_name',\n",
       " 'cpicr_homolog_ensembl_peptide',\n",
       " 'cpicr_homolog_chromosome',\n",
       " 'cpicr_homolog_chrom_start',\n",
       " 'cpicr_homolog_chrom_end',\n",
       " 'cpicr_homolog_canonical_transcript_protein',\n",
       " 'cpicr_homolog_subtype',\n",
       " 'cpicr_homolog_orthology_type',\n",
       " 'cpicr_homolog_perc_id',\n",
       " 'cpicr_homolog_perc_id_r1',\n",
       " 'cpicr_homolog_goc_score',\n",
       " 'cpicr_homolog_wga_coverage',\n",
       " 'cpicr_homolog_dn',\n",
       " 'cpicr_homolog_ds',\n",
       " 'cpicr_homolog_orthology_confidence',\n",
       " 'psinensis_homolog_ensembl_gene',\n",
       " 'psinensis_homolog_associated_gene_name',\n",
       " 'psinensis_homolog_ensembl_peptide',\n",
       " 'psinensis_homolog_chromosome',\n",
       " 'psinensis_homolog_chrom_start',\n",
       " 'psinensis_homolog_chrom_end',\n",
       " 'psinensis_homolog_canonical_transcript_protein',\n",
       " 'psinensis_homolog_subtype',\n",
       " 'psinensis_homolog_orthology_type',\n",
       " 'psinensis_homolog_perc_id',\n",
       " 'psinensis_homolog_perc_id_r1',\n",
       " 'psinensis_homolog_goc_score',\n",
       " 'psinensis_homolog_wga_coverage',\n",
       " 'psinensis_homolog_dn',\n",
       " 'psinensis_homolog_ds',\n",
       " 'psinensis_homolog_orthology_confidence',\n",
       " 'atestudineus_homolog_ensembl_gene',\n",
       " 'atestudineus_homolog_associated_gene_name',\n",
       " 'atestudineus_homolog_ensembl_peptide',\n",
       " 'atestudineus_homolog_chromosome',\n",
       " 'atestudineus_homolog_chrom_start',\n",
       " 'atestudineus_homolog_chrom_end',\n",
       " 'atestudineus_homolog_canonical_transcript_protein',\n",
       " 'atestudineus_homolog_subtype',\n",
       " 'atestudineus_homolog_orthology_type',\n",
       " 'atestudineus_homolog_perc_id',\n",
       " 'atestudineus_homolog_perc_id_r1',\n",
       " 'atestudineus_homolog_goc_score',\n",
       " 'atestudineus_homolog_wga_coverage',\n",
       " 'atestudineus_homolog_dn',\n",
       " 'atestudineus_homolog_ds',\n",
       " 'atestudineus_homolog_orthology_confidence',\n",
       " 'aocellaris_homolog_ensembl_gene',\n",
       " 'aocellaris_homolog_associated_gene_name',\n",
       " 'aocellaris_homolog_ensembl_peptide',\n",
       " 'aocellaris_homolog_chromosome',\n",
       " 'aocellaris_homolog_chrom_start',\n",
       " 'aocellaris_homolog_chrom_end',\n",
       " 'aocellaris_homolog_canonical_transcript_protein',\n",
       " 'aocellaris_homolog_subtype',\n",
       " 'aocellaris_homolog_orthology_type',\n",
       " ...]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(dataset.attributes)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can map the gene stable ID with the mappings from the to get a sense of which pathways are linked to a particular gene:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'link_so_mini_closure': <biomart.Filter name='link_so_mini_closure', type='list'>,\n",
       " 'link_go_closure': <biomart.Filter name='link_go_closure', type='text'>,\n",
       " 'link_ensembl_transcript_stable_id': <biomart.Filter name='link_ensembl_transcript_stable_id', type='text'>,\n",
       " 'gene_id': <biomart.Filter name='gene_id', type='text'>,\n",
       " 'transcript_id': <biomart.Filter name='transcript_id', type='text'>,\n",
       " 'link_ensembl_gene_id': <biomart.Filter name='link_ensembl_gene_id', type='text'>,\n",
       " 'chromosome_name': <biomart.Filter name='chromosome_name', type='text'>,\n",
       " 'start': <biomart.Filter name='start', type='text'>,\n",
       " 'end': <biomart.Filter name='end', type='text'>,\n",
       " 'band_start': <biomart.Filter name='band_start', type='drop_down_basic_filter'>,\n",
       " 'band_end': <biomart.Filter name='band_end', type='drop_down_basic_filter'>,\n",
       " 'marker_start': <biomart.Filter name='marker_start', type='drop_down_basic_filter'>,\n",
       " 'marker_end': <biomart.Filter name='marker_end', type=''>,\n",
       " 'hsapiens_encode.type': <biomart.Filter name='hsapiens_encode.type', type='text'>,\n",
       " 'hsapiens_encode.encode_region': <biomart.Filter name='hsapiens_encode.encode_region', type=''>,\n",
       " 'strand': <biomart.Filter name='strand', type='text'>,\n",
       " 'chromosomal_region': <biomart.Filter name='chromosomal_region', type='text'>,\n",
       " 'id_list_xrefs_filters': <biomart.Filter name='id_list_xrefs_filters', type='boolean_list'>,\n",
       " 'id_list_limit_xrefs_filters': <biomart.Filter name='id_list_limit_xrefs_filters', type='id_list'>,\n",
       " 'id_list_microarray_filters': <biomart.Filter name='id_list_microarray_filters', type='boolean_list'>,\n",
       " 'id_list_limit_microarray_filters': <biomart.Filter name='id_list_limit_microarray_filters', type='id_list'>,\n",
       " 'transcript_count_greater_than': <biomart.Filter name='transcript_count_greater_than', type='text'>,\n",
       " 'transcript_count_less_than': <biomart.Filter name='transcript_count_less_than', type='text'>,\n",
       " 'biotype': <biomart.Filter name='biotype', type='list'>,\n",
       " 'transcript_biotype': <biomart.Filter name='transcript_biotype', type='list'>,\n",
       " 'source': <biomart.Filter name='source', type='list'>,\n",
       " 'transcript_source': <biomart.Filter name='transcript_source', type='list'>,\n",
       " 'transcript_tsl': <biomart.Filter name='transcript_tsl', type='boolean'>,\n",
       " 'transcript_gencode_basic': <biomart.Filter name='transcript_gencode_basic', type='boolean'>,\n",
       " 'transcript_appris': <biomart.Filter name='transcript_appris', type='boolean'>,\n",
       " 'mane_select': <biomart.Filter name='mane_select', type='boolean'>,\n",
       " 'phenotype_description': <biomart.Filter name='phenotype_description', type='list'>,\n",
       " 'phenotype_source': <biomart.Filter name='phenotype_source', type='list'>,\n",
       " 'go_clos': <biomart.Filter name='go_clos', type=''>,\n",
       " 'go_name': <biomart.Filter name='go_name', type=''>,\n",
       " 'go_evidence_code': <biomart.Filter name='go_evidence_code', type='list'>,\n",
       " 'homolog_filters': <biomart.Filter name='homolog_filters', type='boolean_list'>,\n",
       " 'id_list_protein_domain_and_feature_filters': <biomart.Filter name='id_list_protein_domain_and_feature_filters', type='boolean_list'>,\n",
       " 'id_list_limit_protein_domain_filters': <biomart.Filter name='id_list_limit_protein_domain_filters', type='id_list'>,\n",
       " 'germ_line_variation_source': <biomart.Filter name='germ_line_variation_source', type='list'>,\n",
       " 'somatic_variation_source': <biomart.Filter name='somatic_variation_source', type='list'>,\n",
       " 'with_validated_snp': <biomart.Filter name='with_validated_snp', type='boolean'>,\n",
       " 'so_consequence_name': <biomart.Filter name='so_consequence_name', type='list'>}"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.filters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Transcript stable ID</th>\n",
       "      <th>NCBI gene ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>251176</td>\n",
       "      <td>ENST00000644207</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251177</td>\n",
       "      <td>ENST00000647544</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251178</td>\n",
       "      <td>ENST00000642596</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251179</td>\n",
       "      <td>ENST00000643537</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251180</td>\n",
       "      <td>ENST00000644633</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251181</td>\n",
       "      <td>ENST00000642800</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251182</td>\n",
       "      <td>ENST00000645112</td>\n",
       "      <td>56169.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251183</td>\n",
       "      <td>ENST00000642712</td>\n",
       "      <td>56169.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251184</td>\n",
       "      <td>ENST00000646090</td>\n",
       "      <td>56169.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251185</td>\n",
       "      <td>ENST00000643960</td>\n",
       "      <td>56169.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "       Transcript stable ID  NCBI gene ID\n",
       "251176      ENST00000644207           NaN\n",
       "251177      ENST00000647544           NaN\n",
       "251178      ENST00000642596           NaN\n",
       "251179      ENST00000643537           NaN\n",
       "251180      ENST00000644633           NaN\n",
       "251181      ENST00000642800           NaN\n",
       "251182      ENST00000645112       56169.0\n",
       "251183      ENST00000642712       56169.0\n",
       "251184      ENST00000646090       56169.0\n",
       "251185      ENST00000643960       56169.0"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ensmbl_entrez_gene_ids = dataset.query(attributes=['ensembl_transcript_id', 'entrezgene_id'])\n",
    "ensmbl_entrez_gene_ids.tail(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 88,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{4535: 'ENST00000361390',\n",
       " 4536: 'ENST00000361453',\n",
       " 4512: 'ENST00000361624',\n",
       " 113219467: 'ENST00000387416',\n",
       " 4513: 'ENST00000361739',\n",
       " 4509: 'ENST00000361851',\n",
       " 4508: 'ENST00000361899',\n",
       " 4514: 'ENST00000362079',\n",
       " 4537: 'ENST00000361227',\n",
       " 4539: 'ENST00000361335',\n",
       " 4538: 'ENST00000361381',\n",
       " 4540: 'ENST00000361567',\n",
       " 4541: 'ENST00000361681',\n",
       " 4519: 'ENST00000361789',\n",
       " 1028: 'ENST00000471157',\n",
       " 51621: 'ENST00000616962',\n",
       " 255027: 'ENST00000567442',\n",
       " 9665: 'ENST00000549219',\n",
       " 4849: 'ENST00000611667',\n",
       " 102723475: 'ENST00000622690',\n",
       " 9093: 'ENST00000431375',\n",
       " 79165: 'ENST00000619669',\n",
       " 57348: 'ENST00000423529',\n",
       " 1113: 'ENST00000556876',\n",
       " 5165: 'ENST00000493226',\n",
       " 89927: 'ENST00000565913',\n",
       " 94104: 'ENST00000464256',\n",
       " 1007: 'ENST00000511822',\n",
       " 147798: 'ENST00000620520',\n",
       " 23532: 'ENST00000398741',\n",
       " 338755: 'ENST00000338569',\n",
       " 144125: 'ENST00000307401',\n",
       " 54664: 'ENST00000462754',\n",
       " 51277: 'ENST00000534855',\n",
       " 440243: 'ENST00000612056',\n",
       " 161725: 'ENST00000560598',\n",
       " 2558: 'ENST00000400081',\n",
       " 285148: 'ENST00000490621',\n",
       " 83787: 'ENST00000425331',\n",
       " 643401: 'ENST00000671061',\n",
       " 7726: 'ENST00000477569',\n",
       " 2533: 'ENST00000503065',\n",
       " 56245: 'ENST00000479548',\n",
       " 26531: 'ENST00000546665',\n",
       " 442194: 'ENST00000550017',\n",
       " 222236: 'ENST00000425379',\n",
       " 9962: 'ENST00000496331',\n",
       " 102723996: 'ENST00000620481',\n",
       " 81696: 'ENST00000230223',\n",
       " 26530: 'ENST00000509836',\n",
       " 81797: 'ENST00000550681',\n",
       " 2562: 'ENST00000622697',\n",
       " 26529: 'ENST00000377140',\n",
       " 8590: 'ENST00000332601',\n",
       " 6868: 'ENST00000648002',\n",
       " 51314: 'ENST00000476435',\n",
       " 7761: 'ENST00000536068',\n",
       " 102724023: 'ENST00000642815',\n",
       " 120775: 'ENST00000317834',\n",
       " 5002: 'ENST00000495518',\n",
       " 7762: 'ENST00000610573',\n",
       " 90527: 'ENST00000613425',\n",
       " 6424: 'ENST00000447200',\n",
       " 144124: 'ENST00000299454',\n",
       " 120776: 'ENST00000299459',\n",
       " 283297: 'ENST00000379829',\n",
       " 341276: 'ENST00000307322',\n",
       " 54749: 'ENST00000425345',\n",
       " 102724159: 'ENST00000634021',\n",
       " 5003: 'ENST00000455942',\n",
       " 8340: 'ENST00000377401',\n",
       " 51439: 'ENST00000259963',\n",
       " 8329: 'ENST00000358739',\n",
       " 8357: 'ENST00000369163',\n",
       " 353345: 'ENST00000334425',\n",
       " 8227: 'ENST00000381261',\n",
       " 3009: 'ENST00000331442',\n",
       " 55975: 'ENST00000469845',\n",
       " 8332: 'ENST00000613174',\n",
       " 221806: 'ENST00000614403',\n",
       " 9104: 'ENST00000457380',\n",
       " 8362: 'ENST00000611927',\n",
       " 8354: 'ENST00000616365',\n",
       " 3705: 'ENST00000553655',\n",
       " 8330: 'ENST00000618958',\n",
       " 8342: 'ENST00000621112',\n",
       " 56265: 'ENST00000380605',\n",
       " 57718: 'ENST00000556470',\n",
       " 8336: 'ENST00000359611',\n",
       " 8363: 'ENST00000355057',\n",
       " 8356: 'ENST00000479986',\n",
       " 8368: 'ENST00000618305',\n",
       " 8331: 'ENST00000333151',\n",
       " 8341: 'ENST00000396980',\n",
       " 90525: 'ENST00000561278',\n",
       " 223075: 'ENST00000451887',\n",
       " 105375283: 'ENST00000653190',\n",
       " 1022: 'ENST00000629350',\n",
       " 10669: 'ENST00000260595',\n",
       " 405753: 'ENST00000491993',\n",
       " 57407: 'ENST00000572559',\n",
       " 26716: 'ENST00000400689',\n",
       " 10398: 'ENST00000346786',\n",
       " 3795: 'ENST00000464371',\n",
       " 50506: 'ENST00000603300',\n",
       " 26015: 'ENST00000569829',\n",
       " 22839: 'ENST00000339266',\n",
       " 51156: 'ENST00000393096',\n",
       " 79143: 'ENST00000620311',\n",
       " 8100: 'ENST00000470695',\n",
       " 59307: 'ENST00000525070',\n",
       " 53905: 'ENST00000558744',\n",
       " 3163: 'ENST00000613539',\n",
       " 116511: 'ENST00000383549',\n",
       " 8468: 'ENST00000252037',\n",
       " 105372749: 'ENST00000662613',\n",
       " 105375009: 'ENST00000416126',\n",
       " 8348: 'ENST00000616182',\n",
       " 81697: 'ENST00000303324',\n",
       " 10842: 'ENST00000498609',\n",
       " 11215: 'ENST00000025301',\n",
       " 651746: 'ENST00000296657',\n",
       " 64112: 'ENST00000556883',\n",
       " 219771: 'ENST00000339497',\n",
       " 54765: 'ENST00000532066',\n",
       " 80831: 'ENST00000249044',\n",
       " 26212: 'ENST00000244623',\n",
       " 55644: 'ENST00000556252',\n",
       " 202658: 'ENST00000552136',\n",
       " 57477: 'ENST00000376020',\n",
       " 101930100: 'ENST00000624813',\n",
       " 56658: 'ENST00000552337',\n",
       " 438: 'ENST00000432523',\n",
       " 3460: 'ENST00000421802',\n",
       " 160287: 'ENST00000615355',\n",
       " 283080: 'ENST00000614533',\n",
       " 56243: 'ENST00000430453',\n",
       " 7932: 'ENST00000453513',\n",
       " 10537: 'ENST00000414436',\n",
       " 9246: 'ENST00000526659',\n",
       " 100288797: 'ENST00000380585',\n",
       " 390598: 'ENST00000341418',\n",
       " 128653: 'ENST00000380589',\n",
       " 79944: 'ENST00000556393',\n",
       " 2550: 'ENST00000546913',\n",
       " 3092: 'ENST00000616821',\n",
       " 7251: 'ENST00000535077',\n",
       " 100131244: 'ENST00000434396',\n",
       " 5330: 'ENST00000559634',\n",
       " 866: 'ENST00000557225',\n",
       " 757: 'ENST00000459909',\n",
       " 219770: 'ENST00000321660',\n",
       " 79897: 'ENST00000552136',\n",
       " 8325: 'ENST00000374694',\n",
       " 390502: 'ENST00000553483',\n",
       " 8326: 'ENST00000344575',\n",
       " 3812: 'ENST00000639252',\n",
       " 170954: 'ENST00000443517',\n",
       " 107987457: 'ENST00000443517',\n",
       " 7140: 'ENST00000673505',\n",
       " 167153: 'ENST00000453514',\n",
       " 11033: 'ENST00000617043',\n",
       " 200958: 'ENST00000625621',\n",
       " 80741: 'ENST00000494996',\n",
       " 55148: 'ENST00000555329',\n",
       " 3106: 'ENST00000640475',\n",
       " 3120: 'ENST00000417780',\n",
       " 51604: 'ENST00000639058',\n",
       " 29947: 'ENST00000431166',\n",
       " 5269: 'ENST00000644875',\n",
       " 140831: 'ENST00000255152',\n",
       " 84520: 'ENST00000556566',\n",
       " 8600: 'ENST00000239849',\n",
       " 26175: 'ENST00000415050',\n",
       " 9210: 'ENST00000252677',\n",
       " 140873: 'ENST00000444723',\n",
       " 387357: 'ENST00000368248',\n",
       " 2492: 'ENST00000419927',\n",
       " 140890: 'ENST00000612404',\n",
       " 56975: 'ENST00000673583',\n",
       " 90204: 'ENST00000372520',\n",
       " 23564: 'ENST00000483514',\n",
       " 79042: 'ENST00000616209',\n",
       " 255967: 'ENST00000483842',\n",
       " 7407: 'ENST00000455956',\n",
       " 100507170: 'ENST00000419982',\n",
       " 6227: 'ENST00000450116',\n",
       " 6725: 'ENST00000217188',\n",
       " 55388: 'ENST00000485659',\n",
       " 6048: 'ENST00000488204',\n",
       " 54962: 'ENST00000561773',\n",
       " 64795: 'ENST00000471113',\n",
       " 245938: 'ENST00000382410',\n",
       " 55611: 'ENST00000422031',\n",
       " 30818: 'ENST00000360990',\n",
       " 4585: 'ENST00000478685',\n",
       " 255313: 'ENST00000457020',\n",
       " 105375118: 'ENST00000670013',\n",
       " 84569: 'ENST00000375500',\n",
       " 2000: 'ENST00000615377',\n",
       " 7844: 'ENST00000472680',\n",
       " 81623: 'ENST00000542572',\n",
       " 728036: 'ENST00000430448',\n",
       " 6535: 'ENST00000457723',\n",
       " 153478: 'ENST00000283426',\n",
       " 3196: 'ENST00000621092',\n",
       " 2019: 'ENST00000546667',\n",
       " 90639: 'ENST00000466853',\n",
       " 140688: 'ENST00000616976',\n",
       " 9031: 'ENST00000466844',\n",
       " 896: 'ENST00000616010',\n",
       " 81850: 'ENST00000344363',\n",
       " 338323: 'ENST00000299481',\n",
       " 284428: 'ENST00000329753',\n",
       " 79644: 'ENST00000514398',\n",
       " 81851: 'ENST00000306271',\n",
       " 10382: 'ENST00000598635',\n",
       " 3133: 'ENST00000462399',\n",
       " 81872: 'ENST00000542137',\n",
       " 132001: 'ENST00000630288',\n",
       " 728042: 'ENST00000417256',\n",
       " 6538: 'ENST00000464828',\n",
       " 79591: 'ENST00000311122',\n",
       " 1237: 'ENST00000414803',\n",
       " 653656: 'ENST00000381394',\n",
       " 728279: 'ENST00000542910',\n",
       " 100526767: 'ENST00000440757',\n",
       " 143458: 'ENST00000529759',\n",
       " 114823: 'ENST00000616932',\n",
       " 730755: 'ENST00000391418',\n",
       " 125997: 'ENST00000381393',\n",
       " 51652: 'ENST00000485465',\n",
       " 401138: 'ENST00000504451',\n",
       " 122258: 'ENST00000631817',\n",
       " 140850: 'ENST00000382388',\n",
       " 54977: 'ENST00000642442',\n",
       " 442179: 'ENST00000402365',\n",
       " 102724889: 'ENST00000605945',\n",
       " 245939: 'ENST00000334391',\n",
       " 51617: 'ENST00000521146',\n",
       " 85461: 'ENST00000496406',\n",
       " 84993: 'ENST00000673596',\n",
       " 128497: 'ENST00000372519',\n",
       " 80345: 'ENST00000340487',\n",
       " 401251: 'ENST00000489292',\n",
       " 4987: 'ENST00000672146',\n",
       " 23769: 'ENST00000246841',\n",
       " 7718: 'ENST00000377325',\n",
       " 53335: 'ENST00000647038',\n",
       " 29928: 'ENST00000617832',\n",
       " 28992: 'ENST00000538595',\n",
       " 6203: 'ENST00000625297',\n",
       " 23423: 'ENST00000650754',\n",
       " 29: 'ENST00000628631',\n",
       " 54939: 'ENST00000567935',\n",
       " 222: 'ENST00000349015',\n",
       " 85294: 'ENST00000394015',\n",
       " 94059: 'ENST00000611161',\n",
       " 2683: 'ENST00000535206',\n",
       " 55667: 'ENST00000380424',\n",
       " 811: 'ENST00000586967',\n",
       " 10771: 'ENST00000381604',\n",
       " 139728: 'ENST00000484705',\n",
       " 23251: 'ENST00000559272',\n",
       " 10949: 'ENST00000314940',\n",
       " 8685: 'ENST00000494979',\n",
       " 729020: 'ENST00000441178',\n",
       " 57189: 'ENST00000482493',\n",
       " 8814: 'ENST00000534566',\n",
       " 3601: 'ENST00000397251',\n",
       " 5306: 'ENST00000539476',\n",
       " 1351: 'ENST00000314133',\n",
       " 9159: 'ENST00000540028',\n",
       " 9118: 'ENST00000369849',\n",
       " 5407: 'ENST00000534513',\n",
       " 55750: 'ENST00000629555',\n",
       " 10071: 'ENST00000536621',\n",
       " 5624: 'ENST00000402125',\n",
       " 55553: 'ENST00000396356',\n",
       " 2832: 'ENST00000369768',\n",
       " 10371: 'ENST00000490883',\n",
       " 11025: 'ENST00000445347',\n",
       " 102725035: 'ENST00000445347',\n",
       " 107987425: 'ENST00000445347',\n",
       " 107987462: 'ENST00000445347',\n",
       " 92521: 'ENST00000492188',\n",
       " 9499: 'ENST00000508938',\n",
       " 79803: 'ENST00000299238',\n",
       " 168090: 'ENST00000491176',\n",
       " 9542: 'ENST00000541337',\n",
       " 3107: 'ENST00000618255',\n",
       " 100233156: 'ENST00000559466',\n",
       " 64210: 'ENST00000355839',\n",
       " 27109: 'ENST00000672612',\n",
       " 100996750: 'ENST00000571574',\n",
       " 129881: 'ENST00000419478',\n",
       " 10752: 'ENST00000620033',\n",
       " 114799: 'ENST00000622333',\n",
       " 728224: 'ENST00000318329',\n",
       " 123606: 'ENST00000557930',\n",
       " 100132386: 'ENST00000617453',\n",
       " 653240: 'ENST00000391413',\n",
       " 83755: 'ENST00000394014',\n",
       " 3908: 'ENST00000617695',\n",
       " 112577516: 'ENST00000652090',\n",
       " 100533177: 'ENST00000391353',\n",
       " 2334: 'ENST00000286437',\n",
       " 399814: 'ENST00000329446',\n",
       " 1192: 'ENST00000614982',\n",
       " 157506: 'ENST00000521013',\n",
       " 51306: 'ENST00000510804',\n",
       " 27290: 'ENST00000379721',\n",
       " 202333: 'ENST00000505466',\n",
       " 116985: 'ENST00000455638',\n",
       " 105375501: 'ENST00000660838',\n",
       " 6129: 'ENST00000431653',\n",
       " 7402: 'ENST00000460618',\n",
       " 3111: 'ENST00000492557',\n",
       " 573: 'ENST00000673069',\n",
       " 11270: 'ENST00000610775',\n",
       " 79616: 'ENST00000541762',\n",
       " 83732: 'ENST00000493691',\n",
       " 51092: 'ENST00000431081',\n",
       " 51274: 'ENST00000482150',\n",
       " 5265: 'ENST00000355814',\n",
       " 10247: 'ENST00000519155',\n",
       " 8722: 'ENST00000526010',\n",
       " 126917: 'ENST00000416166',\n",
       " 55900: 'ENST00000627982',\n",
       " 57695: 'ENST00000338465',\n",
       " 5476: 'ENST00000606000',\n",
       " 112936: 'ENST00000531741',\n",
       " 80036: 'ENST00000360823',\n",
       " 9991: 'ENST00000334318',\n",
       " 91801: 'ENST00000530933',\n",
       " 26285: 'ENST00000286808',\n",
       " 6347: 'ENST00000582017',\n",
       " 23352: 'ENST00000497018',\n",
       " 7328: 'ENST00000490974',\n",
       " 9073: 'ENST00000399899',\n",
       " 10944: 'ENST00000524508',\n",
       " 23310: 'ENST00000532445',\n",
       " 339665: 'ENST00000451479',\n",
       " 23095: 'ENST00000622724',\n",
       " 643803: 'ENST00000340345',\n",
       " 150290: 'ENST00000442752',\n",
       " 100131902: 'ENST00000416044',\n",
       " 340252: 'ENST00000473601',\n",
       " 6840: 'ENST00000375398',\n",
       " 85477: 'ENST00000476649',\n",
       " 3113: 'ENST00000448154',\n",
       " 6354: 'ENST00000394630',\n",
       " 326: 'ENST00000337909',\n",
       " 25987: 'ENST00000612930',\n",
       " 110599588: 'ENST00000607690',\n",
       " 4439: 'ENST00000548003',\n",
       " 10940: 'ENST00000517435',\n",
       " 54529: 'ENST00000420250',\n",
       " 3115: 'ENST00000488076',\n",
       " 148170: 'ENST00000472703',\n",
       " 151525: 'ENST00000409124',\n",
       " 57583: 'ENST00000367090',\n",
       " 10075: 'ENST00000262854',\n",
       " 207063: 'ENST00000430536',\n",
       " 55735: 'ENST00000465911',\n",
       " 79168: 'ENST00000270464',\n",
       " 1776: 'ENST00000463694',\n",
       " 9437: 'ENST00000638974',\n",
       " 84893: 'ENST00000397269',\n",
       " 6632: 'ENST00000577906',\n",
       " 1643: 'ENST00000614884',\n",
       " 27039: 'ENST00000511176',\n",
       " 9189: 'ENST00000381223',\n",
       " 388818: 'ENST00000360542',\n",
       " 100528021: 'ENST00000494999',\n",
       " 10588: 'ENST00000560919',\n",
       " 2563: 'ENST00000640688',\n",
       " 29058: 'ENST00000379276',\n",
       " 221044: 'ENST00000463405',\n",
       " 1734: 'ENST00000422005',\n",
       " 338440: 'ENST00000525857',\n",
       " 8861: 'ENST00000425280',\n",
       " 64318: 'ENST00000461562',\n",
       " 79025: 'ENST00000370097',\n",
       " 25837: 'ENST00000566724',\n",
       " 64419: 'ENST00000606184',\n",
       " 2204: 'ENST00000640830',\n",
       " 6356: 'ENST00000305869',\n",
       " 30012: 'ENST00000296921',\n",
       " 5368: 'ENST00000522209',\n",
       " 57176: 'ENST00000550773',\n",
       " 79603: 'ENST00000599275',\n",
       " 728049: 'ENST00000457977',\n",
       " 6355: 'ENST00000394620',\n",
       " 100996455: 'ENST00000316124',\n",
       " 81793: 'ENST00000622002',\n",
       " 653282: 'ENST00000434883',\n",
       " 155051: 'ENST00000478106',\n",
       " 3904: 'ENST00000351841',\n",
       " 4798: 'ENST00000531318',\n",
       " 100507608: 'ENST00000391355',\n",
       " 85280: 'ENST00000334109',\n",
       " 81870: 'ENST00000394008',\n",
       " 728062: 'ENST00000673456',\n",
       " 83700: 'ENST00000533711',\n",
       " 151230: 'ENST00000448589',\n",
       " 100526832: 'ENST00000448589',\n",
       " 9081: 'ENST00000472391',\n",
       " 5228: 'ENST00000555234',\n",
       " 6876: 'ENST00000529622',\n",
       " 221527: 'ENST00000375527',\n",
       " 222950: 'ENST00000489641',\n",
       " 90326: 'ENST00000480647',\n",
       " 629: 'ENST00000486534',\n",
       " 255426: 'ENST00000615330',\n",
       " 4650: 'ENST00000397274',\n",
       " 389257: 'ENST00000328278',\n",
       " 7096: 'ENST00000308979',\n",
       " 493911: 'ENST00000617738',\n",
       " 140881: 'ENST00000246105',\n",
       " 7262: 'ENST00000314222',\n",
       " 1021: 'ENST00000491250',\n",
       " 4123: 'ENST00000618257',\n",
       " 123036: 'ENST00000555302',\n",
       " 9782: 'ENST00000502394',\n",
       " 9061: 'ENST00000504987',\n",
       " 219464: 'ENST00000313264',\n",
       " 390154: 'ENST00000303059',\n",
       " 400830: 'ENST00000382376',\n",
       " 83900: 'ENST00000411528',\n",
       " 83899: 'ENST00000377721',\n",
       " 7429: 'ENST00000419986',\n",
       " 10333: 'ENST00000610323',\n",
       " 653657: 'ENST00000333843',\n",
       " 55727: 'ENST00000554968',\n",
       " 5886: 'ENST00000591467',\n",
       " 92092: 'ENST00000275766',\n",
       " 196996: 'ENST00000569450',\n",
       " 6993: 'ENST00000367085',\n",
       " 55296: 'ENST00000513596',\n",
       " 255725: 'ENST00000530810',\n",
       " 4773: 'ENST00000496054',\n",
       " 81847: 'ENST00000616343',\n",
       " 196335: 'ENST00000316529',\n",
       " 4999: 'ENST00000457595',\n",
       " 120787: 'ENST00000311352',\n",
       " 5360: 'ENST00000420868',\n",
       " 81928: 'ENST00000453274',\n",
       " 94120: 'ENST00000611299',\n",
       " 100507632: 'ENST00000653444',\n",
       " 643812: 'ENST00000382835',\n",
       " 165257: 'ENST00000272520',\n",
       " 337963: 'ENST00000334160',\n",
       " 729597: 'ENST00000563237',\n",
       " 6742: 'ENST00000612337',\n",
       " 337959: 'ENST00000399889',\n",
       " 5179: 'ENST00000521153',\n",
       " 140258: 'ENST00000355459',\n",
       " 55312: 'ENST00000479197',\n",
       " 9575: 'ENST00000513033',\n",
       " 337960: 'ENST00000390690',\n",
       " 728072: 'ENST00000439466',\n",
       " 133957: 'ENST00000441693',\n",
       " 284827: 'ENST00000334068',\n",
       " 254950: 'ENST00000334067',\n",
       " 3206: 'ENST00000613671',\n",
       " 165545: 'ENST00000498552',\n",
       " 5154: 'ENST00000426681',\n",
       " 337882: 'ENST00000390689',\n",
       " 79820: 'ENST00000553676',\n",
       " 2556: 'ENST00000417858',\n",
       " 90480: 'ENST00000316939',\n",
       " 337969: 'ENST00000334055',\n",
       " 337970: 'ENST00000334063',\n",
       " 100505724: 'ENST00000391354',\n",
       " 10869: 'ENST00000465902',\n",
       " 54809: 'ENST00000620985',\n",
       " 337971: 'ENST00000334058',\n",
       " 258: 'ENST00000613447',\n",
       " 23244: 'ENST00000506262',\n",
       " 337972: 'ENST00000334151',\n",
       " 23016: 'ENST00000459856',\n",
       " 710: 'ENST00000619430',\n",
       " 1004: 'ENST00000504835',\n",
       " 4709: 'ENST00000454214',\n",
       " 717: 'ENST00000549972',\n",
       " 100289561: 'ENST00000492837',\n",
       " 7145: 'ENST00000615025',\n",
       " 390155: 'ENST00000641368',\n",
       " 65125: 'ENST00000530271',\n",
       " 56729: 'ENST00000629642',\n",
       " 728075: 'ENST00000672380',\n",
       " 219469: 'ENST00000610894',\n",
       " 79706: 'ENST00000468316',\n",
       " 105379487: 'ENST00000623227',\n",
       " 84071: 'ENST00000368972',\n",
       " 10942: 'ENST00000635038',\n",
       " 6357: 'ENST00000577681',\n",
       " 285172: 'ENST00000494881',\n",
       " 7174: 'ENST00000651823',\n",
       " 3205: 'ENST00000489695',\n",
       " 92689: 'ENST00000513966',\n",
       " 337973: 'ENST00000334046',\n",
       " 337974: 'ENST00000334849',\n",
       " 100288287: 'ENST00000382830',\n",
       " 50809: 'ENST00000443615',\n",
       " 3921: 'ENST00000495394',\n",
       " 337968: 'ENST00000391624',\n",
       " 337967: 'ENST00000334897',\n",
       " 337979: 'ENST00000334680',\n",
       " 494551: 'ENST00000493845',\n",
       " 337966: 'ENST00000329122',\n",
       " 337975: 'ENST00000334664',\n",
       " 5111: 'ENST00000379160',\n",
       " 51304: 'ENST00000478669',\n",
       " 5813: 'ENST00000331327',\n",
       " 345757: 'ENST00000505792',\n",
       " 55251: 'ENST00000299468',\n",
       " 56829: 'ENST00000471652',\n",
       " 54539: 'ENST00000276062',\n",
       " 57491: 'ENST00000512529',\n",
       " 80017: 'ENST00000298858',\n",
       " 140825: 'ENST00000372518',\n",
       " 219285: 'ENST00000610760',\n",
       " 177: 'ENST00000551827',\n",
       " 23287: 'ENST00000337006',\n",
       " 144809: 'ENST00000537894',\n",
       " 160298: 'ENST00000316375',\n",
       " 8241: 'ENST00000329236',\n",
       " 5077: 'ENST00000258387',\n",
       " 55531: 'ENST00000534236',\n",
       " 9319: 'ENST00000508430',\n",
       " 101927378: 'ENST00000446000',\n",
       " 9083: 'ENST00000618574',\n",
       " 442867: 'ENST00000618574',\n",
       " 442868: 'ENST00000618574',\n",
       " 10124: 'ENST00000396663',\n",
       " 27288: 'ENST00000306904',\n",
       " 22907: 'ENST00000619982',\n",
       " 10344: 'ENST00000394905',\n",
       " 4661: 'ENST00000360149',\n",
       " 151835: 'ENST00000613455',\n",
       " 55858: 'ENST00000515591',\n",
       " 100151643: 'ENST00000382828',\n",
       " 22827: 'ENST00000533362',\n",
       " 7301: 'ENST00000568490',\n",
       " 337976: 'ENST00000330798',\n",
       " 79631: 'ENST00000558974',\n",
       " 337985: 'ENST00000382826',\n",
       " 100288323: 'ENST00000444335',\n",
       " 337977: 'ENST00000335093',\n",
       " 337978: 'ENST00000333892',\n",
       " 10016: 'ENST00000628729',\n",
       " 116535: 'ENST00000441623',\n",
       " 7123: 'ENST00000428034',\n",
       " 5634: 'ENST00000461630',\n",
       " 6194: 'ENST00000380381',\n",
       " 1139: 'ENST00000645153',\n",
       " 102724631: 'ENST00000612601',\n",
       " 79815: 'ENST00000519324',\n",
       " 199675: 'ENST00000598851',\n",
       " 5926: 'ENST00000348476',\n",
       " 79230: 'ENST00000414706',\n",
       " 619279: 'ENST00000519936',\n",
       " 389610: 'ENST00000647027',\n",
       " 6892: 'ENST00000456592',\n",
       " 401: 'ENST00000544057',\n",
       " 6138: 'ENST00000611050',\n",
       " 348654: 'ENST00000381254',\n",
       " 126003: 'ENST00000595985',\n",
       " 1672: 'ENST00000642865',\n",
       " 3636: 'ENST00000541756',\n",
       " 1823: 'ENST00000257198',\n",
       " 2968: 'ENST00000400450',\n",
       " 55203: 'ENST00000512108',\n",
       " 55824: 'ENST00000519326',\n",
       " 124935: 'ENST00000613070',\n",
       " 4122: 'ENST00000360468',\n",
       " 2173: 'ENST00000356535',\n",
       " 6588: 'ENST00000305991',\n",
       " 22906: 'ENST00000613405',\n",
       " 115701: 'ENST00000590642',\n",
       " 54790: 'ENST00000540549',\n",
       " 54476: 'ENST00000476345',\n",
       " 513: 'ENST00000614837',\n",
       " 149775: 'ENST00000443966',\n",
       " 151278: 'ENST00000295226',\n",
       " 344752: 'ENST00000356517',\n",
       " 54784: 'ENST00000498283',\n",
       " 81490: 'ENST00000632264',\n",
       " 2517: 'ENST00000374479',\n",
       " 6346: 'ENST00000225842',\n",
       " 6389: 'ENST00000617470',\n",
       " 1671: 'ENST00000646187',\n",
       " 55620: 'ENST00000601482',\n",
       " 29062: 'ENST00000485942',\n",
       " 54510: 'ENST00000617302',\n",
       " 1824: 'ENST00000648081',\n",
       " 80228: 'ENST00000356387',\n",
       " 55655: 'ENST00000616903',\n",
       " 2778: 'ENST00000306090',\n",
       " 5834: 'ENST00000471359',\n",
       " 79839: 'ENST00000581103',\n",
       " 6850: 'ENST00000375746',\n",
       " 129049: 'ENST00000610372',\n",
       " 114898: 'ENST00000393975',\n",
       " 728082: 'ENST00000441330',\n",
       " 63920: 'ENST00000523213',\n",
       " 728090: 'ENST00000458218',\n",
       " 285761: 'ENST00000487076',\n",
       " 728096: 'ENST00000672229',\n",
       " 11027: 'ENST00000616629',\n",
       " 6598: 'ENST00000626269',\n",
       " 79677: 'ENST00000489535',\n",
       " 51530: 'ENST00000311873',\n",
       " 140893: 'ENST00000252998',\n",
       " 55879: 'ENST00000598523',\n",
       " 157777: 'ENST00000415737',\n",
       " 6654: 'ENST00000402219',\n",
       " 9975: 'ENST00000472780',\n",
       " 57406: 'ENST00000480457',\n",
       " 1611: 'ENST00000508646',\n",
       " 80008: 'ENST00000495029',\n",
       " 2747: 'ENST00000673493',\n",
       " 11046: 'ENST00000643789',\n",
       " 8892: 'ENST00000556668',\n",
       " 26090: 'ENST00000461204',\n",
       " 4488: 'ENST00000507785',\n",
       " 26154: 'ENST00000412081',\n",
       " 57120: 'ENST00000535237',\n",
       " 7920: 'ENST00000548592',\n",
       " 2208: 'ENST00000360067',\n",
       " 54872: 'ENST00000505800',\n",
       " 23082: 'ENST00000413464',\n",
       " 79290: 'ENST00000612185',\n",
       " 202243: 'ENST00000396499',\n",
       " 7033: 'ENST00000291525',\n",
       " 29099: 'ENST00000526789',\n",
       " 25942: 'ENST00000565264',\n",
       " 115273: 'ENST00000465518',\n",
       " 10924: 'ENST00000539041',\n",
       " 6883: 'ENST00000495422',\n",
       " 3605: 'ENST00000648244',\n",
       " 28512: 'ENST00000614374',\n",
       " 219473: 'ENST00000623845',\n",
       " 8449: 'ENST00000446074',\n",
       " 122553: 'ENST00000556765',\n",
       " 51015: 'ENST00000173527',\n",
       " 3293: 'ENST00000650386',\n",
       " 105375851: 'ENST00000661064',\n",
       " 6369: 'ENST00000416943',\n",
       " 90342: 'ENST00000641731',\n",
       " 3005: 'ENST00000340857',\n",
       " 6098: 'ENST00000403284',\n",
       " 64374: 'ENST00000508744',\n",
       " 1244: 'ENST00000648523',\n",
       " 8225: 'ENST00000326153',\n",
       " 219: 'ENST00000635162',\n",
       " 55607: 'ENST00000456331',\n",
       " 729288: 'ENST00000668878',\n",
       " 51175: 'ENST00000603722',\n",
       " 51284: 'ENST00000484204',\n",
       " 157574: 'ENST00000519471',\n",
       " 254272: 'ENST00000405044',\n",
       " 347252: 'ENST00000377694',\n",
       " 100507303: 'ENST00000563192',\n",
       " 6854: 'ENST00000425297',\n",
       " 240: 'ENST00000622238',\n",
       " 51311: 'ENST00000311912',\n",
       " 3112: 'ENST00000413252',\n",
       " 55344: 'ENST00000381663',\n",
       " 221143: 'ENST00000460374',\n",
       " 158405: 'ENST00000536272',\n",
       " 4869: 'ENST00000524204',\n",
       " 10809: 'ENST00000540587',\n",
       " 81614: 'ENST00000398014',\n",
       " 91319: 'ENST00000626238',\n",
       " 158471: 'ENST00000223609',\n",
       " 2310: 'ENST00000454745',\n",
       " 53342: 'ENST00000468605',\n",
       " 84263: 'ENST00000480881',\n",
       " 101926971: 'ENST00000665808',\n",
       " 54928: 'ENST00000517461',\n",
       " 8938: 'ENST00000628027',\n",
       " 1269: 'ENST00000374472',\n",
       " 199713: 'ENST00000615699',\n",
       " 124944: 'ENST00000547709',\n",
       " 7697: 'ENST00000359735',\n",
       " 55629: 'ENST00000579103',\n",
       " 340602: 'ENST00000342995',\n",
       " 150160: 'ENST00000359963',\n",
       " 170685: 'ENST00000356450',\n",
       " 55893: 'ENST00000519730',\n",
       " 8838: 'ENST00000409166',\n",
       " 79623: 'ENST00000485468',\n",
       " 9938: 'ENST00000479844',\n",
       " 221545: 'ENST00000468808',\n",
       " 84902: 'ENST00000592401',\n",
       " 203611: 'ENST00000544303',\n",
       " 51011: 'ENST00000470100',\n",
       " 23191: 'ENST00000627816',\n",
       " 107983955: 'ENST00000620688',\n",
       " 134829: 'ENST00000368438',\n",
       " 9063: 'ENST00000590944',\n",
       " 51427: 'ENST00000423627',\n",
       " 23657: 'ENST00000509248',\n",
       " 203074: 'ENST00000646980',\n",
       " 7079: 'ENST00000287814',\n",
       " 79661: 'ENST00000569758',\n",
       " 199699: 'ENST00000317060',\n",
       " 390157: 'ENST00000279783',\n",
       " 8744: 'ENST00000245817',\n",
       " 80179: 'ENST00000634006',\n",
       " 55190: 'ENST00000375992',\n",
       " 140680: 'ENST00000400269',\n",
       " 7032: 'ENST00000475297',\n",
       " 100507436: 'ENST00000546529',\n",
       " 1669: 'ENST00000642522',\n",
       " 440400: 'ENST00000552321',\n",
       " 2650: 'ENST00000442371',\n",
       " 55683: 'ENST00000478492',\n",
       " 10117: 'ENST00000472597',\n",
       " 3643: 'ENST00000600492',\n",
       " 94240: 'ENST00000536042',\n",
       " 5411: 'ENST00000557680',\n",
       " 4771: 'ENST00000413209',\n",
       " 11177: 'ENST00000360310',\n",
       " 149951: 'ENST00000645058',\n",
       " 27430: 'ENST00000521838',\n",
       " 140453: 'ENST00000497859',\n",
       " 339318: 'ENST00000492450',\n",
       " 4340: 'ENST00000551993',\n",
       " 7127: 'ENST00000333007',\n",
       " 3512: 'ENST00000543780',\n",
       " 2171: 'ENST00000481695',\n",
       " 8495: 'ENST00000534552',\n",
       " 102724219: 'ENST00000611026',\n",
       " 619208: 'ENST00000604268',\n",
       " 162699: 'ENST00000620881',\n",
       " 51224: 'ENST00000620522',\n",
       " 1828: 'ENST00000462981',\n",
       " 284805: 'ENST00000360785',\n",
       " 91404: 'ENST00000452991',\n",
       " 728929: 'ENST00000611323',\n",
       " 7031: 'ENST00000291527',\n",
       " 100506888: 'ENST00000330682',\n",
       " 10569: 'ENST00000520664',\n",
       " 83473: 'ENST00000591522',\n",
       " 9618: 'ENST00000618771',\n",
       " 54943: 'ENST00000617313',\n",
       " 105375622: 'ENST00000650562',\n",
       " 159963: 'ENST00000528822',\n",
       " 10630: 'ENST00000617617',\n",
       " 81844: 'ENST00000487252',\n",
       " 79148: 'ENST00000612672',\n",
       " 219477: 'ENST00000303039',\n",
       " 345778: 'ENST00000617335',\n",
       " 147409: 'ENST00000359747',\n",
       " 504189: 'ENST00000627176',\n",
       " 85364: 'ENST00000500893',\n",
       " 504190: 'ENST00000626074',\n",
       " 1667: 'ENST00000644528',\n",
       " 440934: 'ENST00000439237',\n",
       " 80023: 'ENST00000621012',\n",
       " 5725: 'ENST00000627714',\n",
       " 970: 'ENST00000597430',\n",
       " 57819: 'ENST00000478692',\n",
       " 23066: 'ENST00000626378',\n",
       " 79722: 'ENST00000519114',\n",
       " 7430: 'ENST00000392177',\n",
       " 387758: 'ENST00000318627',\n",
       " 23514: 'ENST00000541342',\n",
       " 51091: 'ENST00000513285',\n",
       " 92799: 'ENST00000602239',\n",
       " 441308: 'ENST00000320901',\n",
       " 7253: 'ENST00000298171',\n",
       " 55240: 'ENST00000393107',\n",
       " 54905: 'ENST00000415893',\n",
       " 2321: 'ENST00000543394',\n",
       " 8424: 'ENST00000263182',\n",
       " 9468: 'ENST00000496020',\n",
       " 57414: 'ENST00000318622',\n",
       " 239: 'ENST00000406228',\n",
       " 169270: 'ENST00000320552',\n",
       " 3572: 'ENST00000396816',\n",
       " 346673: 'ENST00000275764',\n",
       " 401027: 'ENST00000673386',\n",
       " 27346: 'ENST00000336687',\n",
       " 1832: 'ENST00000506617',\n",
       " 5375: 'ENST00000519260',\n",
       " 196792: 'ENST00000489000',\n",
       " 55328: 'ENST00000437752',\n",
       " 130074: 'ENST00000389915',\n",
       " 101928359: 'ENST00000665977',\n",
       " 6666: 'ENST00000342665',\n",
       " 170949: 'ENST00000453425',\n",
       " 100287083: 'ENST00000453425',\n",
       " 90410: 'ENST00000588477',\n",
       " 54733: 'ENST00000524991',\n",
       " 728358: 'ENST00000644617',\n",
       " 646486: 'ENST00000360464',\n",
       " 5408: 'ENST00000591655',\n",
       " 102724334: 'ENST00000464664',\n",
       " 50624: 'ENST00000338948',\n",
       " 646480: 'ENST00000379071',\n",
       " 64328: 'ENST00000400602',\n",
       " 91584: 'ENST00000321063',\n",
       " 7114: 'ENST00000380633',\n",
       " 818: 'ENST00000423381',\n",
       " 6161: 'ENST00000396957',\n",
       " 123720: 'ENST00000560964',\n",
       " 2167: 'ENST00000521734',\n",
       " 107986954: 'ENST00000656790',\n",
       " 51272: 'ENST00000479463',\n",
       " 9252: 'ENST00000556594',\n",
       " 4967: 'ENST00000631326',\n",
       " 255349: 'ENST00000423535',\n",
       " 64866: 'ENST00000490471',\n",
       " 203076: 'ENST00000643037',\n",
       " 142910: 'ENST00000531458',\n",
       " 6891: 'ENST00000468636',\n",
       " 85481: 'ENST00000523010',\n",
       " 9972: 'ENST00000537253',\n",
       " 55231: 'ENST00000333861',\n",
       " 4336: 'ENST00000479860',\n",
       " 94137: 'ENST00000329335',\n",
       " 245972: 'ENST00000285393',\n",
       " 10892: 'ENST00000587438',\n",
       " 117583: 'ENST00000622699',\n",
       " 1668: 'ENST00000642311',\n",
       " 147166: 'ENST00000395671',\n",
       " 100463482: 'ENST00000633925',\n",
       " 6741: 'ENST00000494051',\n",
       " 113746: 'ENST00000342593',\n",
       " 166929: 'ENST00000504754',\n",
       " 8740: 'ENST00000599359',\n",
       " 441239: 'ENST00000340779',\n",
       " 145508: 'ENST00000281129',\n",
       " 10793: 'ENST00000395375',\n",
       " 1794: 'ENST00000540750',\n",
       " 80742: 'ENST00000475643',\n",
       " 541565: 'ENST00000615223',\n",
       " 64711: 'ENST00000293937',\n",
       " 337879: 'ENST00000329621',\n",
       " 1670: 'ENST00000647159',\n",
       " 337878: 'ENST00000621162',\n",
       " 64116: 'ENST00000502903',\n",
       " 147199: 'ENST00000342878',\n",
       " 337880: 'ENST00000332378',\n",
       " 91057: 'ENST00000317945',\n",
       " 29081: 'ENST00000471560',\n",
       " 728299: 'ENST00000382822',\n",
       " 113612: 'ENST00000508453',\n",
       " 102157402: 'ENST00000618980',\n",
       " 6150: 'ENST00000673399',\n",
       " 107987373: 'ENST00000673399',\n",
       " 9786: 'ENST00000619722',\n",
       " 5159: 'ENST00000523456',\n",
       " 107984345: 'ENST00000641280',\n",
       " 2794: 'ENST00000478986',\n",
       " 390174: 'ENST00000312153',\n",
       " 107986298: 'ENST00000668384',\n",
       " 219931: 'ENST00000442692',\n",
       " 504191: 'ENST00000571130',\n",
       " 3268: 'ENST00000429987',\n",
       " 51351: 'ENST00000610793',\n",
       " 109504726: 'ENST00000610793',\n",
       " 9326: 'ENST00000634200',\n",
       " 148103: 'ENST00000588788',\n",
       " 116449: 'ENST00000442825',\n",
       " 283189: 'ENST00000302957',\n",
       " 2086: 'ENST00000535538',\n",
       " 6880: 'ENST00000328663',\n",
       " 2618: 'ENST00000441403',\n",
       " 64236: 'ENST00000622702',\n",
       " 150165: 'ENST00000331428',\n",
       " 55677: 'ENST00000436740',\n",
       " 101927322: 'ENST00000588144',\n",
       " 85379: 'ENST00000358431',\n",
       " 3207: 'ENST00000006015',\n",
       " 440270: 'ENST00000438958',\n",
       " 124842: 'ENST00000577271',\n",
       " 7384: 'ENST00000493806',\n",
       " 353149: 'ENST00000612583',\n",
       " 25907: 'ENST00000503771',\n",
       " 5884: 'ENST00000616683',\n",
       " 57584: 'ENST00000320481',\n",
       " 100505573: 'ENST00000560415',\n",
       " 107986888: 'ENST00000653093',\n",
       " 9131: 'ENST00000535724',\n",
       " 102724398: 'ENST00000624872',\n",
       " 154007: 'ENST00000496946',\n",
       " 84067: 'ENST00000532797',\n",
       " 115653: 'ENST00000638267',\n",
       " 344387: 'ENST00000419111',\n",
       " 4102: 'ENST00000598245',\n",
       " 256364: 'ENST00000419857',\n",
       " 284098: 'ENST00000616581',\n",
       " 79977: 'ENST00000474338',\n",
       " 492311: 'ENST00000333305',\n",
       " 128954: 'ENST00000523144',\n",
       " 8760: 'ENST00000379070',\n",
       " 3788: 'ENST00000521839',\n",
       " 57050: 'ENST00000254803',\n",
       " 102723547: 'ENST00000535353',\n",
       " 9439: 'ENST00000539158',\n",
       " 3804: 'ENST00000640389',\n",
       " 10666: 'ENST00000580335',\n",
       " 143425: 'ENST00000532592',\n",
       " 57524: 'ENST00000562055',\n",
       " 6788: 'ENST00000617590',\n",
       " 11102: 'ENST00000462046',\n",
       " 90362: 'ENST00000521413',\n",
       " 197131: 'ENST00000627960',\n",
       " 283298: 'ENST00000528758',\n",
       " 81622: 'ENST00000531152',\n",
       " 1617: 'ENST00000426000',\n",
       " 84174: 'ENST00000360672',\n",
       " 22902: 'ENST00000513593',\n",
       " 91442: 'ENST00000591791',\n",
       " 9275: 'ENST00000481667',\n",
       " 55300: 'ENST00000264864',\n",
       " 109703458: 'ENST00000476007',\n",
       " 84231: 'ENST00000570169',\n",
       " 10331: 'ENST00000595387',\n",
       " 60436: 'ENST00000611732',\n",
       " 220164: 'ENST00000577609',\n",
       " 3150: 'ENST00000463631',\n",
       " 84196: 'ENST00000400301',\n",
       " 150051: 'ENST00000433071',\n",
       " 79133: 'ENST00000486772',\n",
       " 7074: 'ENST00000541036',\n",
       " 57335: 'ENST00000593105',\n",
       " 23365: 'ENST00000528681',\n",
       " 27000: 'ENST00000412522',\n",
       " 7179: 'ENST00000612746',\n",
       " 1830: 'ENST00000257189',\n",
       " 257062: 'ENST00000448307',\n",
       " 101927120: 'ENST00000526436',\n",
       " 1946: 'ENST00000611503',\n",
       " 102724428: 'ENST00000624077',\n",
       " 171586: 'ENST00000579982',\n",
       " 1387: 'ENST00000636895',\n",
       " 100131897: 'ENST00000523970',\n",
       " 3811: 'ENST00000640111',\n",
       " 85460: 'ENST00000503068',\n",
       " 653486: 'ENST00000613221',\n",
       " 55969: 'ENST00000639924',\n",
       " 5089: 'ENST00000466640',\n",
       " 54984: 'ENST00000647329',\n",
       " 51614: 'ENST00000640748',\n",
       " 23008: 'ENST00000495724',\n",
       " 8447: 'ENST00000633635',\n",
       " 55284: 'ENST00000650817',\n",
       " 54919: 'ENST00000461576',\n",
       " 157724: 'ENST00000520624',\n",
       " 10910: 'ENST00000609175',\n",
       " 54927: 'ENST00000457942',\n",
       " 163154: 'ENST00000419421',\n",
       " 9656: 'ENST00000550500',\n",
       " 64172: 'ENST00000521630',\n",
       " 23762: 'ENST00000403222',\n",
       " 1302: 'ENST00000486982',\n",
       " 83595: 'ENST00000646538',\n",
       " 10497: 'ENST00000617908',\n",
       " 283726: 'ENST00000565006',\n",
       " 56931: 'ENST00000589854',\n",
       " 23258: 'ENST00000530780',\n",
       " 100527943: 'ENST00000558530',\n",
       " 132884: 'ENST00000509670',\n",
       " 5575: 'ENST00000544935',\n",
       " 347549: 'ENST00000417339',\n",
       " 29063: 'ENST00000612982',\n",
       " 150709: 'ENST00000313581',\n",
       " 115548: 'ENST00000508431',\n",
       " 6094: 'ENST00000525947',\n",
       " 375611: 'ENST00000354356',\n",
       " 115939: 'ENST00000566296',\n",
       " 654: 'ENST00000283147',\n",
       " 105375624: 'ENST00000668794',\n",
       " 22941: 'ENST00000449833',\n",
       " 51115: 'ENST00000518772',\n",
       " 3802: 'ENST00000639974',\n",
       " 100129654: 'ENST00000566129',\n",
       " 63915: 'ENST00000543936',\n",
       " 7060: 'ENST00000511888',\n",
       " 253650: 'ENST00000475234',\n",
       " 116969: 'ENST00000425767',\n",
       " 81567: 'ENST00000469459',\n",
       " 3809: 'ENST00000640257',\n",
       " 118670: 'ENST00000368894',\n",
       " 4034: 'ENST00000619071',\n",
       " 5201: 'ENST00000507185',\n",
       " 3805: 'ENST00000638428',\n",
       " 112268354: 'ENST00000638428',\n",
       " 3613: 'ENST00000625802',\n",
       " 4784: 'ENST00000358552',\n",
       " 259217: 'ENST00000434227',\n",
       " 79664: 'ENST00000560520',\n",
       " 27067: 'ENST00000523160',\n",
       " ...}"
      ]
     },
     "execution_count": 88,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ensmbl_entrez_gene_ids.dropna(inplace=True)\n",
    "ensmbl_entrez_gene_ids['NCBI gene ID'] = ensmbl_entrez_gene_ids['NCBI gene ID'].astype(int)\n",
    "ensmbl_entrez_gene_ids = ensmbl_entrez_gene_ids.set_index('NCBI gene ID').to_dict()['Transcript stable ID']\n",
    "ensmbl_entrez_gene_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {},
   "outputs": [],
   "source": [
    "ensmbl_ids = pd.Series(lihc.columns.values[1:]).apply(lambda x: x.split('|')[1]).astype(int).map(ensmbl_entrez_gene_ids).dropna()\n",
    "\n",
    "# Had to drop ~2k that didn't align, is there a better way?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'link_so_mini_closure': <biomart.Filter name='link_so_mini_closure', type='list'>,\n",
       " 'link_go_closure': <biomart.Filter name='link_go_closure', type='text'>,\n",
       " 'link_ensembl_transcript_stable_id': <biomart.Filter name='link_ensembl_transcript_stable_id', type='text'>,\n",
       " 'gene_id': <biomart.Filter name='gene_id', type='text'>,\n",
       " 'transcript_id': <biomart.Filter name='transcript_id', type='text'>,\n",
       " 'link_ensembl_gene_id': <biomart.Filter name='link_ensembl_gene_id', type='text'>,\n",
       " 'chromosome_name': <biomart.Filter name='chromosome_name', type='text'>,\n",
       " 'start': <biomart.Filter name='start', type='text'>,\n",
       " 'end': <biomart.Filter name='end', type='text'>,\n",
       " 'band_start': <biomart.Filter name='band_start', type='drop_down_basic_filter'>,\n",
       " 'band_end': <biomart.Filter name='band_end', type='drop_down_basic_filter'>,\n",
       " 'marker_start': <biomart.Filter name='marker_start', type='drop_down_basic_filter'>,\n",
       " 'marker_end': <biomart.Filter name='marker_end', type=''>,\n",
       " 'hsapiens_encode.type': <biomart.Filter name='hsapiens_encode.type', type='text'>,\n",
       " 'hsapiens_encode.encode_region': <biomart.Filter name='hsapiens_encode.encode_region', type=''>,\n",
       " 'strand': <biomart.Filter name='strand', type='text'>,\n",
       " 'chromosomal_region': <biomart.Filter name='chromosomal_region', type='text'>,\n",
       " 'id_list_xrefs_filters': <biomart.Filter name='id_list_xrefs_filters', type='boolean_list'>,\n",
       " 'id_list_limit_xrefs_filters': <biomart.Filter name='id_list_limit_xrefs_filters', type='id_list'>,\n",
       " 'id_list_microarray_filters': <biomart.Filter name='id_list_microarray_filters', type='boolean_list'>,\n",
       " 'id_list_limit_microarray_filters': <biomart.Filter name='id_list_limit_microarray_filters', type='id_list'>,\n",
       " 'transcript_count_greater_than': <biomart.Filter name='transcript_count_greater_than', type='text'>,\n",
       " 'transcript_count_less_than': <biomart.Filter name='transcript_count_less_than', type='text'>,\n",
       " 'biotype': <biomart.Filter name='biotype', type='list'>,\n",
       " 'transcript_biotype': <biomart.Filter name='transcript_biotype', type='list'>,\n",
       " 'source': <biomart.Filter name='source', type='list'>,\n",
       " 'transcript_source': <biomart.Filter name='transcript_source', type='list'>,\n",
       " 'transcript_tsl': <biomart.Filter name='transcript_tsl', type='boolean'>,\n",
       " 'transcript_gencode_basic': <biomart.Filter name='transcript_gencode_basic', type='boolean'>,\n",
       " 'transcript_appris': <biomart.Filter name='transcript_appris', type='boolean'>,\n",
       " 'mane_select': <biomart.Filter name='mane_select', type='boolean'>,\n",
       " 'phenotype_description': <biomart.Filter name='phenotype_description', type='list'>,\n",
       " 'phenotype_source': <biomart.Filter name='phenotype_source', type='list'>,\n",
       " 'go_clos': <biomart.Filter name='go_clos', type=''>,\n",
       " 'go_name': <biomart.Filter name='go_name', type=''>,\n",
       " 'go_evidence_code': <biomart.Filter name='go_evidence_code', type='list'>,\n",
       " 'homolog_filters': <biomart.Filter name='homolog_filters', type='boolean_list'>,\n",
       " 'id_list_protein_domain_and_feature_filters': <biomart.Filter name='id_list_protein_domain_and_feature_filters', type='boolean_list'>,\n",
       " 'id_list_limit_protein_domain_filters': <biomart.Filter name='id_list_limit_protein_domain_filters', type='id_list'>,\n",
       " 'germ_line_variation_source': <biomart.Filter name='germ_line_variation_source', type='list'>,\n",
       " 'somatic_variation_source': <biomart.Filter name='somatic_variation_source', type='list'>,\n",
       " 'with_validated_snp': <biomart.Filter name='with_validated_snp', type='boolean'>,\n",
       " 'so_consequence_name': <biomart.Filter name='so_consequence_name', type='list'>}"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dataset.filters"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Find a faster way to do this\n",
    "\n",
    "attributes = [\n",
    "#     'gene_id',\n",
    "    'entrezgene_id',\n",
    "    'ensembl_gene_id',\n",
    "    'ensembl_transcript_id',\n",
    "    'go_id',\n",
    "    'name_1006',\n",
    "    'definition_1006',\n",
    "    'go_linkage_type',\n",
    "    'hgnc_id',\n",
    "    'hgnc_symbol',\n",
    "#     'hgnc_trans_name',\n",
    "]\n",
    "go_mappings = dataset.query(attributes=attributes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "go_mappings.to_csv('data/go_mappings.csv', index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>NCBI gene ID</th>\n",
       "      <th>Gene stable ID</th>\n",
       "      <th>Transcript stable ID</th>\n",
       "      <th>GO term accession</th>\n",
       "      <th>GO term name</th>\n",
       "      <th>GO term definition</th>\n",
       "      <th>GO term evidence code</th>\n",
       "      <th>HGNC ID</th>\n",
       "      <th>HGNC symbol</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [NCBI gene ID, Gene stable ID, Transcript stable ID, GO term accession, GO term name, GO term definition, GO term evidence code, HGNC ID, HGNC symbol]\n",
       "Index: []"
      ]
     },
     "execution_count": 92,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Figure out better way to do this\n",
    "attributes = [\n",
    "#     'gene_id',\n",
    "    'entrezgene_id',\n",
    "    'ensembl_gene_id',\n",
    "    'ensembl_transcript_id',\n",
    "    'go_id',\n",
    "    'name_1006',\n",
    "    'definition_1006',\n",
    "    'go_linkage_type',\n",
    "    'hgnc_id',\n",
    "    'hgnc_symbol',\n",
    "#     'hgnc_trans_name',\n",
    "]\n",
    "dataset.query(attributes=attributes,\n",
    "              filters={'transcript_id': ensmbl_ids.values}\n",
    "             )"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Examine Gene Counts\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/mattheweng/Desktop/Galvanize/Projects/hacknights/env-tcga/lib/python3.6/site-packages/IPython/core/interactiveshell.py:3058: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  interactivity=interactivity, compiler=compiler, result=result)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(1392800, 9)"
      ]
     },
     "execution_count": 98,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "go_mappings = pd.read_csv('data/go_mappings.csv')\n",
    "\n",
    "go_mappings.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>NCBI gene ID</th>\n",
       "      <th>Gene stable ID</th>\n",
       "      <th>Transcript stable ID</th>\n",
       "      <th>GO term accession</th>\n",
       "      <th>GO term name</th>\n",
       "      <th>GO term definition</th>\n",
       "      <th>GO term evidence code</th>\n",
       "      <th>HGNC ID</th>\n",
       "      <th>HGNC symbol</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENSG00000210049</td>\n",
       "      <td>ENST00000387314</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7481</td>\n",
       "      <td>MT-TF</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENSG00000211459</td>\n",
       "      <td>ENST00000389680</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7470</td>\n",
       "      <td>MT-RNR1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENSG00000210077</td>\n",
       "      <td>ENST00000387342</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7500</td>\n",
       "      <td>MT-TV</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENSG00000210082</td>\n",
       "      <td>ENST00000387347</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7471</td>\n",
       "      <td>MT-RNR2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ENSG00000209082</td>\n",
       "      <td>ENST00000386347</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7490</td>\n",
       "      <td>MT-TL1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  NCBI gene ID   Gene stable ID Transcript stable ID GO term accession  \\\n",
       "0          NaN  ENSG00000210049      ENST00000387314               NaN   \n",
       "1          NaN  ENSG00000211459      ENST00000389680               NaN   \n",
       "2          NaN  ENSG00000210077      ENST00000387342               NaN   \n",
       "3          NaN  ENSG00000210082      ENST00000387347               NaN   \n",
       "4          NaN  ENSG00000209082      ENST00000386347               NaN   \n",
       "\n",
       "  GO term name GO term definition GO term evidence code    HGNC ID HGNC symbol  \n",
       "0          NaN                NaN                   NaN  HGNC:7481       MT-TF  \n",
       "1          NaN                NaN                   NaN  HGNC:7470     MT-RNR1  \n",
       "2          NaN                NaN                   NaN  HGNC:7500       MT-TV  \n",
       "3          NaN                NaN                   NaN  HGNC:7471     MT-RNR2  \n",
       "4          NaN                NaN                   NaN  HGNC:7490      MT-TL1  "
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "go_mappings.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>NCBI gene ID</th>\n",
       "      <th>Gene stable ID</th>\n",
       "      <th>Transcript stable ID</th>\n",
       "      <th>GO term accession</th>\n",
       "      <th>GO term name</th>\n",
       "      <th>GO term definition</th>\n",
       "      <th>GO term evidence code</th>\n",
       "      <th>HGNC ID</th>\n",
       "      <th>HGNC symbol</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>4535</td>\n",
       "      <td>ENSG00000198888</td>\n",
       "      <td>ENST00000361390</td>\n",
       "      <td>GO:0016020</td>\n",
       "      <td>membrane</td>\n",
       "      <td>A lipid bilayer along with all the proteins an...</td>\n",
       "      <td>IEA</td>\n",
       "      <td>HGNC:7455</td>\n",
       "      <td>MT-ND1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      <td>4535</td>\n",
       "      <td>ENSG00000198888</td>\n",
       "      <td>ENST00000361390</td>\n",
       "      <td>GO:0016021</td>\n",
       "      <td>integral component of membrane</td>\n",
       "      <td>The component of a membrane consisting of the ...</td>\n",
       "      <td>IEA</td>\n",
       "      <td>HGNC:7455</td>\n",
       "      <td>MT-ND1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>4535</td>\n",
       "      <td>ENSG00000198888</td>\n",
       "      <td>ENST00000361390</td>\n",
       "      <td>GO:0055114</td>\n",
       "      <td>oxidation-reduction process</td>\n",
       "      <td>A metabolic process that results in the remova...</td>\n",
       "      <td>IEA</td>\n",
       "      <td>HGNC:7455</td>\n",
       "      <td>MT-ND1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      <td>4535</td>\n",
       "      <td>ENSG00000198888</td>\n",
       "      <td>ENST00000361390</td>\n",
       "      <td>GO:0005743</td>\n",
       "      <td>mitochondrial inner membrane</td>\n",
       "      <td>The inner, i.e. lumen-facing, lipid bilayer of...</td>\n",
       "      <td>IEA</td>\n",
       "      <td>HGNC:7455</td>\n",
       "      <td>MT-ND1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>4535</td>\n",
       "      <td>ENSG00000198888</td>\n",
       "      <td>ENST00000361390</td>\n",
       "      <td>GO:0005739</td>\n",
       "      <td>mitochondrion</td>\n",
       "      <td>A semiautonomous, self replicating organelle t...</td>\n",
       "      <td>IEA</td>\n",
       "      <td>HGNC:7455</td>\n",
       "      <td>MT-ND1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1392787</td>\n",
       "      <td>60678</td>\n",
       "      <td>ENSG00000284869</td>\n",
       "      <td>ENST00000646013</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:24614</td>\n",
       "      <td>EEFSEC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1392796</td>\n",
       "      <td>56169</td>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>ENST00000645112</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7151</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1392797</td>\n",
       "      <td>56169</td>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>ENST00000642712</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7151</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1392798</td>\n",
       "      <td>56169</td>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>ENST00000646090</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7151</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1392799</td>\n",
       "      <td>56169</td>\n",
       "      <td>ENSG00000285114</td>\n",
       "      <td>ENST00000643960</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HGNC:7151</td>\n",
       "      <td>GSDMC</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1306714 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        NCBI gene ID   Gene stable ID Transcript stable ID GO term accession  \\\n",
       "5               4535  ENSG00000198888      ENST00000361390        GO:0016020   \n",
       "6               4535  ENSG00000198888      ENST00000361390        GO:0016021   \n",
       "7               4535  ENSG00000198888      ENST00000361390        GO:0055114   \n",
       "8               4535  ENSG00000198888      ENST00000361390        GO:0005743   \n",
       "9               4535  ENSG00000198888      ENST00000361390        GO:0005739   \n",
       "...              ...              ...                  ...               ...   \n",
       "1392787        60678  ENSG00000284869      ENST00000646013               NaN   \n",
       "1392796        56169  ENSG00000285114      ENST00000645112               NaN   \n",
       "1392797        56169  ENSG00000285114      ENST00000642712               NaN   \n",
       "1392798        56169  ENSG00000285114      ENST00000646090               NaN   \n",
       "1392799        56169  ENSG00000285114      ENST00000643960               NaN   \n",
       "\n",
       "                           GO term name  \\\n",
       "5                              membrane   \n",
       "6        integral component of membrane   \n",
       "7           oxidation-reduction process   \n",
       "8          mitochondrial inner membrane   \n",
       "9                         mitochondrion   \n",
       "...                                 ...   \n",
       "1392787                             NaN   \n",
       "1392796                             NaN   \n",
       "1392797                             NaN   \n",
       "1392798                             NaN   \n",
       "1392799                             NaN   \n",
       "\n",
       "                                        GO term definition  \\\n",
       "5        A lipid bilayer along with all the proteins an...   \n",
       "6        The component of a membrane consisting of the ...   \n",
       "7        A metabolic process that results in the remova...   \n",
       "8        The inner, i.e. lumen-facing, lipid bilayer of...   \n",
       "9        A semiautonomous, self replicating organelle t...   \n",
       "...                                                    ...   \n",
       "1392787                                                NaN   \n",
       "1392796                                                NaN   \n",
       "1392797                                                NaN   \n",
       "1392798                                                NaN   \n",
       "1392799                                                NaN   \n",
       "\n",
       "        GO term evidence code     HGNC ID HGNC symbol  \n",
       "5                         IEA   HGNC:7455      MT-ND1  \n",
       "6                         IEA   HGNC:7455      MT-ND1  \n",
       "7                         IEA   HGNC:7455      MT-ND1  \n",
       "8                         IEA   HGNC:7455      MT-ND1  \n",
       "9                         IEA   HGNC:7455      MT-ND1  \n",
       "...                       ...         ...         ...  \n",
       "1392787                   NaN  HGNC:24614      EEFSEC  \n",
       "1392796                   NaN   HGNC:7151       GSDMC  \n",
       "1392797                   NaN   HGNC:7151       GSDMC  \n",
       "1392798                   NaN   HGNC:7151       GSDMC  \n",
       "1392799                   NaN   HGNC:7151       GSDMC  \n",
       "\n",
       "[1306714 rows x 9 columns]"
      ]
     },
     "execution_count": 104,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "go_mappings.loc[~go_mappings['NCBI gene ID'].isnull()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[1,\n",
       " 2,\n",
       " 131076,\n",
       " 9,\n",
       " 10,\n",
       " 12,\n",
       " 13,\n",
       " 14,\n",
       " 15,\n",
       " 16,\n",
       " 26054,\n",
       " 18,\n",
       " 19,\n",
       " 20,\n",
       " 21,\n",
       " 22,\n",
       " 23,\n",
       " 24,\n",
       " 25,\n",
       " 26,\n",
       " 27,\n",
       " 28,\n",
       " 29,\n",
       " 30,\n",
       " 31,\n",
       " 32,\n",
       " 33,\n",
       " 34,\n",
       " 35,\n",
       " 36,\n",
       " 37,\n",
       " 38,\n",
       " 39,\n",
       " 40,\n",
       " 41,\n",
       " 163882,\n",
       " 43,\n",
       " 319139,\n",
       " 131118,\n",
       " 47,\n",
       " 48,\n",
       " 49,\n",
       " 50,\n",
       " 51,\n",
       " 52,\n",
       " 53,\n",
       " 54,\n",
       " 55,\n",
       " 56,\n",
       " 58,\n",
       " 59,\n",
       " 60,\n",
       " 6525,\n",
       " 83935,\n",
       " 70,\n",
       " 71,\n",
       " 72,\n",
       " 131149,\n",
       " 81,\n",
       " 86,\n",
       " 87,\n",
       " 88,\n",
       " 89,\n",
       " 90,\n",
       " 91,\n",
       " 92,\n",
       " 93,\n",
       " 94,\n",
       " 95,\n",
       " 163933,\n",
       " 97,\n",
       " 98,\n",
       " 100,\n",
       " 101,\n",
       " 102,\n",
       " 103,\n",
       " 104,\n",
       " 105,\n",
       " 131177,\n",
       " 107,\n",
       " 108,\n",
       " 109,\n",
       " 111,\n",
       " 112,\n",
       " 113,\n",
       " 114,\n",
       " 115,\n",
       " 116,\n",
       " 117,\n",
       " 118,\n",
       " 119,\n",
       " 120,\n",
       " 6536,\n",
       " 123,\n",
       " 124,\n",
       " 125,\n",
       " 126,\n",
       " 127,\n",
       " 128,\n",
       " 130,\n",
       " 131,\n",
       " 132,\n",
       " 133,\n",
       " 134,\n",
       " 135,\n",
       " 136,\n",
       " 196740,\n",
       " 196743,\n",
       " 140,\n",
       " 141,\n",
       " 142,\n",
       " 143,\n",
       " 146,\n",
       " 147,\n",
       " 148,\n",
       " 150,\n",
       " 151,\n",
       " 152,\n",
       " 153,\n",
       " 154,\n",
       " 155,\n",
       " 156,\n",
       " 157,\n",
       " 158,\n",
       " 159,\n",
       " 160,\n",
       " 161,\n",
       " 162,\n",
       " 163,\n",
       " 164,\n",
       " 165,\n",
       " 166,\n",
       " 167,\n",
       " 84671,\n",
       " 149830,\n",
       " 172,\n",
       " 173,\n",
       " 174,\n",
       " 175,\n",
       " 176,\n",
       " 177,\n",
       " 178,\n",
       " 6548,\n",
       " 181,\n",
       " 182,\n",
       " 183,\n",
       " 196792,\n",
       " 185,\n",
       " 186,\n",
       " 187,\n",
       " 6549,\n",
       " 189,\n",
       " 190,\n",
       " 191,\n",
       " 6550,\n",
       " 89882,\n",
       " 196,\n",
       " 197,\n",
       " 199,\n",
       " 202,\n",
       " 203,\n",
       " 204,\n",
       " 205,\n",
       " 164045,\n",
       " 207,\n",
       " 208,\n",
       " 6553,\n",
       " 210,\n",
       " 211,\n",
       " 212,\n",
       " 213,\n",
       " 214,\n",
       " 215,\n",
       " 216,\n",
       " 217,\n",
       " 218,\n",
       " 219,\n",
       " 220,\n",
       " 221,\n",
       " 222,\n",
       " 223,\n",
       " 224,\n",
       " 225,\n",
       " 226,\n",
       " 229,\n",
       " 230,\n",
       " 231,\n",
       " 238,\n",
       " 239,\n",
       " 240,\n",
       " 241,\n",
       " 242,\n",
       " 244,\n",
       " 245,\n",
       " 246,\n",
       " 247,\n",
       " 248,\n",
       " 249,\n",
       " 250,\n",
       " 251,\n",
       " 164091,\n",
       " 257,\n",
       " 258,\n",
       " 259,\n",
       " 262,\n",
       " 196872,\n",
       " 265,\n",
       " 266,\n",
       " 267,\n",
       " 268,\n",
       " 269,\n",
       " 270,\n",
       " 271,\n",
       " 272,\n",
       " 273,\n",
       " 274,\n",
       " 196883,\n",
       " 275,\n",
       " 276,\n",
       " 164118,\n",
       " 279,\n",
       " 280,\n",
       " 283,\n",
       " 284,\n",
       " 285,\n",
       " 286,\n",
       " 287,\n",
       " 288,\n",
       " 164127,\n",
       " 290,\n",
       " 291,\n",
       " 292,\n",
       " 293,\n",
       " 117289,\n",
       " 131368,\n",
       " 301,\n",
       " 302,\n",
       " 303,\n",
       " 304,\n",
       " 305,\n",
       " 306,\n",
       " 307,\n",
       " 308,\n",
       " 309,\n",
       " 310,\n",
       " 311,\n",
       " 312,\n",
       " 313,\n",
       " 314,\n",
       " 164153,\n",
       " 316,\n",
       " 317,\n",
       " 318,\n",
       " 319,\n",
       " 320,\n",
       " 321,\n",
       " 322,\n",
       " 323,\n",
       " 324,\n",
       " 325,\n",
       " 326,\n",
       " 327,\n",
       " 328,\n",
       " 329,\n",
       " 330,\n",
       " 331,\n",
       " 332,\n",
       " 333,\n",
       " 334,\n",
       " 335,\n",
       " 336,\n",
       " 337,\n",
       " 338,\n",
       " 339,\n",
       " 131408,\n",
       " 341,\n",
       " 342,\n",
       " 343,\n",
       " 344,\n",
       " 345,\n",
       " 346,\n",
       " 347,\n",
       " 348,\n",
       " 196951,\n",
       " 350,\n",
       " 351,\n",
       " 353,\n",
       " 354,\n",
       " 355,\n",
       " 356,\n",
       " 357,\n",
       " 358,\n",
       " 359,\n",
       " 360,\n",
       " 361,\n",
       " 362,\n",
       " 363,\n",
       " 364,\n",
       " 196968,\n",
       " 366,\n",
       " 367,\n",
       " 368,\n",
       " 369,\n",
       " 285268,\n",
       " 372,\n",
       " 373,\n",
       " 374,\n",
       " 375,\n",
       " 55751,\n",
       " 377,\n",
       " 378,\n",
       " 379,\n",
       " 131450,\n",
       " 381,\n",
       " 382,\n",
       " 383,\n",
       " 384,\n",
       " 196993,\n",
       " 387,\n",
       " 196996,\n",
       " 388,\n",
       " 389,\n",
       " 391,\n",
       " 392,\n",
       " 393,\n",
       " 394,\n",
       " 395,\n",
       " 396,\n",
       " 397,\n",
       " 398,\n",
       " 197003,\n",
       " 400,\n",
       " 401,\n",
       " 402,\n",
       " 403,\n",
       " 131474,\n",
       " 405,\n",
       " 406,\n",
       " 407,\n",
       " 408,\n",
       " 409,\n",
       " 410,\n",
       " 411,\n",
       " 412,\n",
       " 197021,\n",
       " 414,\n",
       " 415,\n",
       " 416,\n",
       " 417,\n",
       " 419,\n",
       " 420,\n",
       " 421,\n",
       " 89891,\n",
       " 54970,\n",
       " 427,\n",
       " 429,\n",
       " 430,\n",
       " 432,\n",
       " 433,\n",
       " 434,\n",
       " 435,\n",
       " 438,\n",
       " 439,\n",
       " 440,\n",
       " 65977,\n",
       " 65975,\n",
       " 443,\n",
       " 164284,\n",
       " 444,\n",
       " 445,\n",
       " 65980,\n",
       " 65981,\n",
       " 65985,\n",
       " 65983,\n",
       " 65987,\n",
       " 65979,\n",
       " 65989,\n",
       " 65990,\n",
       " 65991,\n",
       " 65992,\n",
       " 65993,\n",
       " 65986,\n",
       " 65988,\n",
       " 460,\n",
       " 65996,\n",
       " 65998,\n",
       " 65999,\n",
       " 65997,\n",
       " 462,\n",
       " 466,\n",
       " 467,\n",
       " 468,\n",
       " 66005,\n",
       " 66002,\n",
       " 471,\n",
       " 472,\n",
       " 131544,\n",
       " 474,\n",
       " 475,\n",
       " 476,\n",
       " 477,\n",
       " 478,\n",
       " 479,\n",
       " 480,\n",
       " 481,\n",
       " 482,\n",
       " 483,\n",
       " 26147,\n",
       " 729540,\n",
       " 486,\n",
       " 487,\n",
       " 488,\n",
       " 489,\n",
       " 490,\n",
       " 491,\n",
       " 492,\n",
       " 493,\n",
       " 131566,\n",
       " 495,\n",
       " 496,\n",
       " 26150,\n",
       " 498,\n",
       " 66035,\n",
       " 66036,\n",
       " 501,\n",
       " 66037,\n",
       " 6612,\n",
       " 6613,\n",
       " 506,\n",
       " 131578,\n",
       " 26152,\n",
       " 509,\n",
       " 131583,\n",
       " 513,\n",
       " 514,\n",
       " 515,\n",
       " 516,\n",
       " 517,\n",
       " 518,\n",
       " 338773,\n",
       " 521,\n",
       " 522,\n",
       " 523,\n",
       " 197131,\n",
       " 525,\n",
       " 526,\n",
       " 527,\n",
       " 528,\n",
       " 529,\n",
       " 197135,\n",
       " 131601,\n",
       " 533,\n",
       " 534,\n",
       " 535,\n",
       " 537,\n",
       " 538,\n",
       " 539,\n",
       " 540,\n",
       " 164380,\n",
       " 131616,\n",
       " 545,\n",
       " 546,\n",
       " 547,\n",
       " 549,\n",
       " 550,\n",
       " 551,\n",
       " 552,\n",
       " 553,\n",
       " 554,\n",
       " 164395,\n",
       " 558,\n",
       " 563,\n",
       " 566,\n",
       " 567,\n",
       " 570,\n",
       " 571,\n",
       " 572,\n",
       " 573,\n",
       " 574,\n",
       " 575,\n",
       " 576,\n",
       " 577,\n",
       " 578,\n",
       " 197187,\n",
       " 580,\n",
       " 581,\n",
       " 582,\n",
       " 583,\n",
       " 579,\n",
       " 585,\n",
       " 586,\n",
       " 587,\n",
       " 163081,\n",
       " 590,\n",
       " 593,\n",
       " 594,\n",
       " 595,\n",
       " 596,\n",
       " 597,\n",
       " 598,\n",
       " 599,\n",
       " 254122,\n",
       " 131669,\n",
       " 602,\n",
       " 604,\n",
       " 605,\n",
       " 606,\n",
       " 607,\n",
       " 608,\n",
       " 100303453,\n",
       " 610,\n",
       " 611,\n",
       " 613,\n",
       " 617,\n",
       " 618,\n",
       " 622,\n",
       " 623,\n",
       " 624,\n",
       " 627,\n",
       " 629,\n",
       " 100270710,\n",
       " 631,\n",
       " 632,\n",
       " 633,\n",
       " 634,\n",
       " 635,\n",
       " 636,\n",
       " 637,\n",
       " 638,\n",
       " 639,\n",
       " 640,\n",
       " 641,\n",
       " 642,\n",
       " 643,\n",
       " 644,\n",
       " 645,\n",
       " 646,\n",
       " 648,\n",
       " 649,\n",
       " 650,\n",
       " 651,\n",
       " 652,\n",
       " 653,\n",
       " 654,\n",
       " 655,\n",
       " 656,\n",
       " 657,\n",
       " 658,\n",
       " 659,\n",
       " 660,\n",
       " 661,\n",
       " 662,\n",
       " 663,\n",
       " 664,\n",
       " 665,\n",
       " 666,\n",
       " 667,\n",
       " 668,\n",
       " 669,\n",
       " 670,\n",
       " 671,\n",
       " 672,\n",
       " 673,\n",
       " 100270746,\n",
       " 675,\n",
       " 676,\n",
       " 677,\n",
       " 678,\n",
       " 680,\n",
       " 682,\n",
       " 683,\n",
       " 684,\n",
       " 685,\n",
       " 686,\n",
       " 687,\n",
       " 688,\n",
       " 689,\n",
       " 690,\n",
       " 694,\n",
       " 695,\n",
       " 696,\n",
       " 699,\n",
       " 701,\n",
       " 705,\n",
       " 706,\n",
       " 708,\n",
       " 710,\n",
       " 711,\n",
       " 712,\n",
       " 713,\n",
       " 197322,\n",
       " 714,\n",
       " 715,\n",
       " 716,\n",
       " 717,\n",
       " 719,\n",
       " 718,\n",
       " 720,\n",
       " 722,\n",
       " 6656,\n",
       " 100270804,\n",
       " 725,\n",
       " 726,\n",
       " 727,\n",
       " 728,\n",
       " 729,\n",
       " 730,\n",
       " 731,\n",
       " 732,\n",
       " 733,\n",
       " 197342,\n",
       " 735,\n",
       " 734,\n",
       " 6658,\n",
       " 738,\n",
       " 6659,\n",
       " 740,\n",
       " 741,\n",
       " 744,\n",
       " 745,\n",
       " 746,\n",
       " 747,\n",
       " 750,\n",
       " 197358,\n",
       " 164592,\n",
       " 753,\n",
       " 752,\n",
       " 755,\n",
       " 754,\n",
       " 757,\n",
       " 758,\n",
       " 759,\n",
       " 760,\n",
       " 761,\n",
       " 762,\n",
       " 763,\n",
       " 131831,\n",
       " 765,\n",
       " 766,\n",
       " 767,\n",
       " 768,\n",
       " 197370,\n",
       " 770,\n",
       " 771,\n",
       " 729597,\n",
       " 773,\n",
       " 774,\n",
       " 775,\n",
       " 776,\n",
       " 777,\n",
       " 778,\n",
       " 779,\n",
       " 780,\n",
       " 781,\n",
       " 782,\n",
       " 783,\n",
       " 784,\n",
       " 785,\n",
       " 786,\n",
       " 492307,\n",
       " 492303,\n",
       " 788,\n",
       " 790,\n",
       " 492311,\n",
       " 164633,\n",
       " 793,\n",
       " 794,\n",
       " 796,\n",
       " 797,\n",
       " 131870,\n",
       " 799,\n",
       " 800,\n",
       " 801,\n",
       " 131873,\n",
       " 795,\n",
       " 197407,\n",
       " 805,\n",
       " 808,\n",
       " 810,\n",
       " 811,\n",
       " 813,\n",
       " 814,\n",
       " 815,\n",
       " 816,\n",
       " 817,\n",
       " 818,\n",
       " 819,\n",
       " 820,\n",
       " 821,\n",
       " 822,\n",
       " 823,\n",
       " 824,\n",
       " 825,\n",
       " 826,\n",
       " 827,\n",
       " 164668,\n",
       " 828,\n",
       " 829,\n",
       " 830,\n",
       " 832,\n",
       " 833,\n",
       " 834,\n",
       " 835,\n",
       " 836,\n",
       " 837,\n",
       " 838,\n",
       " 839,\n",
       " 840,\n",
       " 841,\n",
       " 842,\n",
       " 843,\n",
       " 844,\n",
       " 845,\n",
       " 846,\n",
       " 847,\n",
       " 131920,\n",
       " 164684,\n",
       " 254173,\n",
       " 857,\n",
       " 858,\n",
       " 859,\n",
       " 860,\n",
       " 861,\n",
       " 862,\n",
       " 863,\n",
       " 864,\n",
       " 865,\n",
       " 866,\n",
       " 867,\n",
       " 868,\n",
       " 869,\n",
       " 871,\n",
       " 873,\n",
       " 874,\n",
       " 875,\n",
       " 164714,\n",
       " 100303728,\n",
       " 881,\n",
       " 883,\n",
       " 26227,\n",
       " 885,\n",
       " 886,\n",
       " 887,\n",
       " 26228,\n",
       " 889,\n",
       " 890,\n",
       " 891,\n",
       " 892,\n",
       " 131965,\n",
       " 894,\n",
       " 896,\n",
       " 898,\n",
       " 899,\n",
       " 900,\n",
       " 901,\n",
       " 902,\n",
       " 26231,\n",
       " 904,\n",
       " 905,\n",
       " 26230,\n",
       " 908,\n",
       " 909,\n",
       " 910,\n",
       " 911,\n",
       " 912,\n",
       " 913,\n",
       " 914,\n",
       " 915,\n",
       " 916,\n",
       " 917,\n",
       " 919,\n",
       " 920,\n",
       " 921,\n",
       " 922,\n",
       " 923,\n",
       " 924,\n",
       " 925,\n",
       " 926,\n",
       " 6697,\n",
       " 928,\n",
       " 132001,\n",
       " 929,\n",
       " 930,\n",
       " 931,\n",
       " 933,\n",
       " 932,\n",
       " 100129424,\n",
       " 254187,\n",
       " 939,\n",
       " 940,\n",
       " 941,\n",
       " 942,\n",
       " 132014,\n",
       " 943,\n",
       " 945,\n",
       " 946,\n",
       " 947,\n",
       " 948,\n",
       " 949,\n",
       " 950,\n",
       " 951,\n",
       " 952,\n",
       " 953,\n",
       " 954,\n",
       " 955,\n",
       " 956,\n",
       " 957,\n",
       " 958,\n",
       " 959,\n",
       " 960,\n",
       " 961,\n",
       " 962,\n",
       " 963,\n",
       " 965,\n",
       " 966,\n",
       " 967,\n",
       " 968,\n",
       " 969,\n",
       " 970,\n",
       " 971,\n",
       " 972,\n",
       " 973,\n",
       " 974,\n",
       " 975,\n",
       " 976,\n",
       " 977,\n",
       " 978,\n",
       " 26246,\n",
       " 26245,\n",
       " 983,\n",
       " 984,\n",
       " 987,\n",
       " 988,\n",
       " 989,\n",
       " 990,\n",
       " 991,\n",
       " 164832,\n",
       " 993,\n",
       " 994,\n",
       " 995,\n",
       " 996,\n",
       " 997,\n",
       " 998,\n",
       " 999,\n",
       " 1000,\n",
       " 1001,\n",
       " 1002,\n",
       " 1003,\n",
       " 1004,\n",
       " 1005,\n",
       " 1006,\n",
       " 1007,\n",
       " 1008,\n",
       " 1009,\n",
       " 1010,\n",
       " 1012,\n",
       " 1013,\n",
       " 1014,\n",
       " 1015,\n",
       " 1016,\n",
       " 1017,\n",
       " 1018,\n",
       " 1019,\n",
       " 1020,\n",
       " 1021,\n",
       " 1022,\n",
       " 1024,\n",
       " 1025,\n",
       " 1026,\n",
       " 1027,\n",
       " 1028,\n",
       " 1029,\n",
       " 1030,\n",
       " 1031,\n",
       " 1032,\n",
       " 1033,\n",
       " 1036,\n",
       " 1038,\n",
       " 1039,\n",
       " 1040,\n",
       " 1041,\n",
       " 132112,\n",
       " 1043,\n",
       " 1044,\n",
       " 1045,\n",
       " 1046,\n",
       " 1047,\n",
       " 1048,\n",
       " 394263,\n",
       " 1050,\n",
       " 1051,\n",
       " 1052,\n",
       " 1053,\n",
       " 1054,\n",
       " 26092,\n",
       " 1056,\n",
       " 1057,\n",
       " 1058,\n",
       " 1059,\n",
       " 1060,\n",
       " 26262,\n",
       " 1062,\n",
       " 1063,\n",
       " 1066,\n",
       " 1068,\n",
       " 1069,\n",
       " 1070,\n",
       " 1071,\n",
       " 1072,\n",
       " 1073,\n",
       " 132141,\n",
       " 1075,\n",
       " 6726,\n",
       " 1080,\n",
       " 1081,\n",
       " 1082,\n",
       " 1084,\n",
       " 132158,\n",
       " 1087,\n",
       " 1088,\n",
       " 1089,\n",
       " 132160,\n",
       " 1101,\n",
       " 1102,\n",
       " 1103,\n",
       " 1104,\n",
       " 1105,\n",
       " 1106,\n",
       " 1107,\n",
       " 1108,\n",
       " 1109,\n",
       " 1111,\n",
       " 1112,\n",
       " 1113,\n",
       " 1114,\n",
       " 1116,\n",
       " 1117,\n",
       " 1118,\n",
       " 1119,\n",
       " 1120,\n",
       " 1121,\n",
       " 1122,\n",
       " 1123,\n",
       " 1124,\n",
       " 221662,\n",
       " 132200,\n",
       " 1128,\n",
       " 1129,\n",
       " 1131,\n",
       " 1132,\n",
       " 1133,\n",
       " 1134,\n",
       " 1135,\n",
       " 1136,\n",
       " 1137,\n",
       " 1138,\n",
       " 1139,\n",
       " 1140,\n",
       " 1141,\n",
       " 1142,\n",
       " 1143,\n",
       " 1144,\n",
       " 1145,\n",
       " 1146,\n",
       " 1147,\n",
       " 1149,\n",
       " 1152,\n",
       " 1153,\n",
       " 1154,\n",
       " 1155,\n",
       " 132228,\n",
       " 1158,\n",
       " 1159,\n",
       " 1160,\n",
       " 1161,\n",
       " 1163,\n",
       " 1164,\n",
       " 132241,\n",
       " 132243,\n",
       " 1173,\n",
       " 1174,\n",
       " 1175,\n",
       " 1176,\n",
       " 1178,\n",
       " 1179,\n",
       " 1180,\n",
       " 1181,\n",
       " 1182,\n",
       " 1183,\n",
       " 1184,\n",
       " 1185,\n",
       " 1186,\n",
       " 1187,\n",
       " 1188,\n",
       " 1191,\n",
       " 1192,\n",
       " 1193,\n",
       " 6750,\n",
       " 1195,\n",
       " 1196,\n",
       " ...]"
      ]
     },
     "execution_count": 142,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "entrez_ids = list(pd.Series(lihc.columns.values[1:]).apply(lambda x: x.split('|')[1]).values)\n",
    "entrez_ids = list(set(pd.Series(entrez_ids).astype(int).values))\n",
    "entrez_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 136,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get list of transcript ids from GO mapping\n",
    "transcript_ids_mapping = go_mappings['NCBI gene ID'].dropna().unique()\n",
    "def check_type(x):\n",
    "    try:\n",
    "        \n",
    "        return int(x)\n",
    "    except:\n",
    "        return -1\n",
    "\n",
    "transcript_ids_mapping = list(map(lambda x: check_type(x), transcript_ids_mapping))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "18742"
      ]
     },
     "execution_count": 148,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(set(transcript_ids_mapping).intersection(set(entrez_ids)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "bcr_patient_barcode    TCGA-2Y-A9GS-01A-12R-A38B-07TCGA-2Y-A9GT-01A-1...\n",
       "?|100130426                                                      21.0025\n",
       "?|100133144                                                      1870.33\n",
       "?|100134869                                                      1948.12\n",
       "?|10357                                                          40000.7\n",
       "                                             ...                        \n",
       "ZYX|7791                                                     1.59719e+06\n",
       "ZZEF1|23140                                                       294653\n",
       "ZZZ3|26009                                                        242955\n",
       "psiTPTE22|387590                                                 3631.78\n",
       "tAKR|389932                                                      39635.7\n",
       "Length: 20532, dtype: object"
      ]
     },
     "execution_count": 150,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "lihc[1:].sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df_genes <- df[,df_colnames_genes %in% go_map$hgnc_symbol]\n",
    "colnames(df_genes) <- df_colnames_genes[which(df_colnames_genes %in% go_map$hgnc_symbol)]\n",
    "df_genes %>% dim()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## TCGA\n",
    "For the RNA-Seq analysis workflow, we obtain the raw FASTQ sequencing files from the sequencing facility. We assess the quality of our sequence reads for each sample, then determine from where on the genome the reads originated by performing alignment. We will use the information about where the reads align to generate the count matrix, which we will be using to start the differential expression analysis.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "Differential gene expression analysis\n",
    "- https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8\n",
    "- http://bioconductor.org/packages/release/bioc/html/DESeq.htmlb\n",
    "- https://nbviewer.jupyter.org/github/maayanlab/Zika-RNAseq-Pipeline/blob/master/Zika.ipynbb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## Load expression matrix\n",
    "expr_df = pd.read_csv(os.path.join(os.environ['WORKDIR'], 'repCpmMatrix_featureCounts.csv'))\n",
    "expr_df = expr_df.set_index(expr_df.columns[0])\n",
    "expr_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## Filter out non-expressed genes\n",
    "expr_df = expr_df.loc[expr_df.sum(axis=1) > 0, :]\n",
    "print expr_df.shape\n",
    "\n",
    "## Filter out lowly expressed genes\n",
    "mask_low_vals = (expr_df > 0.3).sum(axis=1) > 2\n",
    "expr_df = expr_df.loc[mask_low_vals, :]\n",
    "\n",
    "print expr_df.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "meta_df = pd.read_csv(os.path.join(os.environ['WORKDIR'], 'SraRunTable.txt'), sep='\\t').set_index('Run_s')\n",
    "print meta_df.shape\n",
    "# re-order the index to make it the same with expr_df\n",
    "meta_df = meta_df.ix[expr_df.columns]\n",
    "meta_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## PCA Plots\n",
    "- Generate PCA plots to observe whether samples cluster as expected (controls with controls and treatments with treatments)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import RNAseq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# plot PCA\n",
    "%matplotlib inline\n",
    "RNAseq.PCA_plot(expr_df.values, meta_df['infection_status_s'], \n",
    "         standardize=2, log=True, \n",
    "         show_text=False, sep=' ', legend_loc='upper right')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Differential Gene Expression\n",
    "Now we are ready to identify the differentially expressed genes between the two sets of samples: control vs. treatment. We will achieve this using the Characteristic Direction method[6](#ref6) that we developed and published in BMC Bioinformatics in 2014.\n",
    "\n",
    "An implementation in Python of the Characteristic Direction method can be downloaded and installed from here: https://github.com/wangz10/geode.\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import geode\n",
    "d_platform_cd = {} # to top up/down genes\n",
    "cd_results = pd.DataFrame(index=expr_df.index)\n",
    "\n",
    "sample_classes = {}\n",
    "for layout in meta_df['LibraryLayout_s'].unique():\n",
    "    ## make sample_class \n",
    "    sample_class = np.zeros(expr_df.shape[1], dtype=np.int32)\n",
    "    sample_class[meta_df['LibraryLayout_s'].values == layout] = 1\n",
    "    sample_class[(meta_df['LibraryLayout_s'].values == layout) & \n",
    "                 (meta_df['infection_status_s'].values == 'Zika infected')] = 2\n",
    "    platform = d_layout_platform[layout]\n",
    "    sample_classes[platform] = sample_class\n",
    "\n",
    "sample_classes['combined'] = sample_classes['MiSeq'] + sample_classes['NextSeq 500']\n",
    "print sample_classes\n",
    "\n",
    "for platform, sample_class in sample_classes.items():\n",
    "    cd_res = geode.chdir(expr_df.values, sample_class, expr_df.index, \n",
    "                      gamma=.5, sort=False, calculate_sig=False)\n",
    "    cd_coefs = np.array(map(lambda x: x[0], cd_res))\n",
    "    cd_results[platform] = cd_coefs\n",
    "    \n",
    "    # sort CD in by absolute values in descending order\n",
    "    srt_idx = np.abs(cd_coefs).argsort()[::-1]\n",
    "    cd_coefs = cd_coefs[srt_idx][:600]\n",
    "    sorted_DEGs = expr_df.index[srt_idx][:600]\n",
    "    # split up and down\n",
    "    up_genes = dict(zip(sorted_DEGs[cd_coefs > 0], cd_coefs[cd_coefs > 0]))\n",
    "    dn_genes = dict(zip(sorted_DEGs[cd_coefs < 0], cd_coefs[cd_coefs < 0]))\n",
    "    d_platform_cd[platform+'-up'] = up_genes\n",
    "    d_platform_cd[platform+'-dn'] = dn_genes\n",
    "\n",
    "print cd_results.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## Check the cosine distance between the two signatures\n",
    "from scipy.spatial.distance import cosine\n",
    "from itertools import combinations\n",
    "for col1, col2 in combinations(cd_results.columns, 2):\n",
    "    print col1, col2, cosine(cd_results[col1], cd_results[col2])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## Prepare count matrices\n",
    "expect input data as obtained, e.g., from RNA-seq or another high-throughput sequencing experiment, in the form of a matrix of integer values. The value in the i-th row and the j-th column of the matrix tells how many reads (or fragments, for paired-end RNA-seq) have been assigned to gene i in sample j. Analogously, for other types of assays, the rows of the matrix might correspond e.g., to binding regions (with ChIP-Seq), species of bacteria (with metagenomic datasets), or peptide sequences (with quantitative mass spectrometry).\n",
    "\n",
    "The values in the matrix should be counts of sequencing reads/fragments. This is important for DESeq2’s statistical model to hold, as only counts allow assessing the measurement precision correctly. It is important to never provide counts that were pre-normalized for sequencing depth/library size, as the statistical model is most powerful when applied to un-normalized counts, and is designed to account for library size differences internally.\n",
    "\n",
    "## Align Reads to reference genome\n",
    "The computational analysis of an RNA-seq experiment begins from the FASTQ files that contain the nucleotide sequence of each read and a quality score at each position. These reads must first be aligned to a reference genome or transcriptome, or the abundances and estimated counts per transcript can be estimated without alignment, as described above. In either case, it is important to know if the sequencing experiment was single-end or paired-end, as the alignment software will require the user to specify both FASTQ files for a paired-end experiment. The output of this alignment step is commonly stored in a file format called SAM/BAM.\n",
    "\n",
    "\n",
    "## Define gene models\n",
    "\n",
    "## Plot counts\n",
    "\n",
    "## PCA Plot \n",
    "\n",
    "## Differential Expression Analysis\n",
    "## Gene Clustering\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Load expression matrix\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from diffexp.py_deseq import py_DESeq2\n",
    "\n",
    "dds = py_DESeq2(count_matrix = df,\n",
    "               design_matrix = sample_df,\n",
    "               design_formula = '~ sample',\n",
    "               gene_column = 'id') # <- telling DESeq2 this should be the gene ID column\n",
    "    \n",
    "dds.run_deseq() \n",
    "dds.get_deseq_result()\n",
    "res = dds.deseq_result \n",
    "res.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "plt.scatter(res.log2FoldChange, -np.log2(res.padj))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "ename": "ModuleNotFoundError",
     "evalue": "No module named 'dgeclust'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-2-8d6855043f8a>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfrom\u001b[0m \u001b[0mdgeclust\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mCountData\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mSimulationManager\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'dgeclust'"
     ]
    }
   ],
   "source": [
    "from dgeclust import CountData, SimulationManager"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "env-tcga",
   "language": "python",
   "name": "env-tcga"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
